{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "IndabaChallenge.ipynb",
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3.7.6 64-bit ('rosetta': conda)",
      "language": "python",
      "name": "python37664bitrosettacondab668476472174f758405ff25d9da075f"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.7.6"
    },
    "toc": {
      "base_numbering": 1,
      "nav_menu": {},
      "number_sections": true,
      "sideBar": true,
      "skip_h1_title": false,
      "title_cell": "Table of Contents",
      "title_sidebar": "Contents",
      "toc_cell": false,
      "toc_position": {},
      "toc_section_display": true,
      "toc_window_display": false
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "3nxPHFOsov4x",
        "colab_type": "text"
      },
      "source": [
        "<div align=\"center\">\n",
        " <img alt=\"Deep Learning Indaba\" src=\"https://gitlab.com/instadeep-public/grandchallenge/-/raw/master/dlindaba.jpg\" />\n",
        "    <h1>Grand Challenge</h1>\n",
        "        <h2>Curing Leishmaniasis</h2>\n",
        "    <br />\n",
        "</div>\n",
        "\n",
        "<h1> </h1>\n",
        "\n",
        "\n",
        "This notebbok focuses on the evaluation of an arbitrary structure formed by a Leishmania protein (either from a defined species, or a protein present in the proteome of one or more of the Leishmania species) and a small molecule (ligand) which comes from a set of approved drugs.\n",
        "\n",
        "\n",
        "Authors : Dr Marcin Skwark (m.skwark@instadeep.com) and Dr Nicolas Lopez Carranza (n.lopezcarranza@instadeep.com)\n",
        "\n",
        "\n",
        "*This notebook is based on [PyRosetta](https://RosettaCommons.github.io/PyRosetta.notebooks) notebooks;\n",
        "content is available [on Github](https://github.com/RosettaCommons/PyRosetta.notebooks.git).*\n",
        "In order to run it, you need to install  [PyRosetta](http://www.pyrosetta.org/dow);"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "tAkgHWj8pFxy",
        "colab_type": "text"
      },
      "source": [
        "**Setup guide**\n",
        "\n",
        "\n",
        "1.   Get [PyRosetta license](https://els2.comotion.uw.edu/product/pyrosetta) (for free).\n",
        "2.   Download the Linux [PyRosetta package](http://www.pyrosetta.org/dow) that is called “Python-3.6.MinSizeRel” (~1.3GB) and upload it to your Google drive.\n",
        "3.   Make a new folder called “Indaba_grand_challenge” in your top level directory on Google Drive. Inside this folder make a folder called “PyRosetta”.\n",
        "4.  Upload the \"PyRosetta4.MinSizeRel.python36.linux.release-259.tar.bz2\" tar file you downloaded in step 2 to the PyRosetta folder. \n",
        "5.   Setup PyRosetta on your Google drive by following the steps below.\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "3yKjnrUJrLpG",
        "colab_type": "text"
      },
      "source": [
        "## PyRosetta Google Drive Setup\n",
        "\n",
        "Run this notebook to install PyRosetta in Colab. You will only need to do this once. Afterwards, you will be able to initialize and use PyRosetta in any notebook in Colab. \n",
        "\n",
        "**NOTE**: running the code below will prompt you to visit a specific URL and enter an authorization code. This is safe, simply click on the provided link, copy the code and paste it in the box below."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab_type": "code",
        "id": "7avg_MNH3myr",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "2403ed4d-e257-4cf4-9249-a2895f3ce06e"
      },
      "source": [
        "# Mounting Google Drive and add it to Python sys path  \n",
        "\n",
        "google_drive_mount_point = '/content/google_drive'\n",
        "\n",
        "import os, sys, time\n",
        "\n",
        "if 'google.colab' in sys.modules:\n",
        "    from google.colab import drive\n",
        "    drive.mount(google_drive_mount_point)\n",
        "\n",
        "google_drive = google_drive_mount_point + '/My Drive'\n",
        "google_drive_prefix = google_drive + '/prefix'\n",
        "\n",
        "if not os.path.isdir(google_drive_prefix):\n",
        "    os.mkdir(google_drive_prefix)\n",
        "\n",
        "pyrosetta_install_prefix_path = '/content/prefix'\n",
        "if os.path.islink(pyrosetta_install_prefix_path):\n",
        "    os.unlink(pyrosetta_install_prefix_path)\n",
        "os.symlink(google_drive_prefix, pyrosetta_install_prefix_path)\n",
        "\n",
        "sys.path.insert(0, pyrosetta_install_prefix_path)"
      ],
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Drive already mounted at /content/google_drive; to attempt to forcibly remount, call drive.mount(\"/content/google_drive\", force_remount=True).\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "OW8mmsuEsMaG",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "os.environ[\"PYTHONPATH\"] = os.environ[\"PYTHONPATH\"] + ':' + pyrosetta_install_prefix_path"
      ],
      "execution_count": 2,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "e9DadOsQsNKn",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "fde76e20-814e-472e-a3b3-1fae16d9bf89"
      },
      "source": [
        "# installing PyRosetta\n",
        "if sys.version_info.major != 3 or sys.version_info.minor != 6:\n",
        "    print('Need Python-3.6 to run!')\n",
        "    sys.exit(1)\n",
        "\n",
        "import_start_time = time.time()\n",
        "\n",
        "# upload PyRosetta Linux package into your google drive and put it path here\n",
        "pyrosetta_distr_path = google_drive.replace(' ', '\\\\ ') + '/Indaba_grand_challenge/PyRosetta/*'\n",
        "print(pyrosetta_distr_path)\n",
        "\n",
        "!mkdir _PyRosetta_\n",
        "!tar xjf $pyrosetta_distr_path -C _PyRosetta_ --strip-components=1\n",
        "!cd _PyRosetta_/setup && python setup.py install --install-lib=$pyrosetta_install_prefix_path\n",
        "!rm -rf _PyRosetta_\n",
        "\n",
        "import_end_time = time.time()\n",
        "\n",
        "print(f'PyRosetta setup took: {import_end_time - import_start_time:.1f}s...')"
      ],
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna/dna_cutpoint_upper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna/5prime_thioethanolamine_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna/5prime_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna/3prime_deoxy.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna/dna_cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna/5prime_methylphosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna/three_prime_azide.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna/LowerDNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/dna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/3prime5prime_methyl_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/rna_cutpoint_upper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Protonated_N3_Cytosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/3prime_packable_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Virtual_RNA_Residue_Exclude_Phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Protonated_N3_Adenosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/UpperConnectionRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Virtual_Phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/BlockStackAbove.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Virtual_Ribose.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/LowerRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/LowerConnectRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/BulgeRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/5prime_packable_triphosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/5prime_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Protonated_N1_Adenosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/3prime_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/5prime_cap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/VirtualBaseHeavyAtom.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/3prime_end_OH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Deoxy_O2prime.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/5prime_end_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/2prime_connection.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/L_RNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/rna_cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Virtual_Backbone_Except_C1prime.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Virtual_O2prime_hydrogen.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/VirtualBase.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/UpperRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/7deaza.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/3prime2prime_cyclic_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/BlockStackBelow.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Repulsive_Phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/5prime_end_OH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/5prime_packable_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna/Virtual_RNA_Residue.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/a3b_hbs_post.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/triazolamerC.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/tyr_sulfated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/RPEPTOID.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/AcetylatedProteinNterm.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/NtermPeptoidFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/N_acetylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/lys_monomethylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/NtermProteinMethylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/N_methylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/spinlabel_mg2_coordination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/MethylatedProteinCterm.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/lys_acetylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/spinlabel_fe2_coordination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/spinlabel_co2_coordination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/CtermTruncation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/C-linked_glycosylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/O1-lactylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/2-reduction.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/6-branch.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/N2-acetylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/lower_terminus.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/4-branch.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/4-reduction.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/6-sulfation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/N-R-3'-hydroxybutyryl-3-amination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/omega-oxidation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/2-branch.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/2-sulfation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/O2-lactylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/N-linked_glycosylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/3-methylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/N4-acetylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/O6-acetylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/Me_glycoside.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/3-amination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/cutpoint_upper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/3-branch.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/O3-lactylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/upper_terminus.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/O6-lactylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/N5-acetylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/O4-lactylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/N-acetyl-2-amination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/2-sulfamidation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/2-amination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates/cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/SpecialRotamer.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/coenzymes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/coenzymes/non-conjugated_nucleotide_diphosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/coenzymes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/VirtualProteinSideChain.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/thr_phosphorylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/VirtualBB.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/RepulsiveOnly_fa.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/tyr_phosphorylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/a3b_hbs_pre.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/lys_trimethylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/triazolamerN.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/VirtualProteinResidue.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/protein_cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/ShoveBB.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/oop_pre.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/AcetylatedPeptoidNterm.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/terpenes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/terpenes/terpene_lower_term.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/terpenes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/terpenes/terpene_upper_term.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/terpenes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/CtermPeptoidFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/spinlabel_cu2_coordination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/pro_hydroxylated_case2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/CtermProteinFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/hbs_pre.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches/VirtualNterm.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/aromatic_trifluoromethylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/connect.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/aromatic_iodination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/aromatic_ethylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/aromatic_chlorination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/pruneH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/aromatic_bromination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/aromatic_methoxylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/aromatic_hydroxylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/aromatic_methylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/metal_connect.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches/aromatic_fluorination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/how_to_add_new_residue_types.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/ASN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/THR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/CYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/HIS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/CYT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/THY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/ADE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/ASP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/LEU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/PRO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/ILE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/GLN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/VAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/PHE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/TYR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/HIS_D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/ARG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/MET.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/SER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/VRT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/LYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/TRP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/GLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/GLY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types/GUA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/residue_types.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/metapatches.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/patches.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/atom_properties_coarse.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/coarse_rsd_list.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/patches/NtermProteinFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/patches/CtermProteinFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_two_bead/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types/RGU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types/URA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types/RAD.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types/RX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types/VRT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types/RCY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/residue_types.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/metapatches.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches/Virtual_Phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches/coarse_rna_cutpoint_upper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches/LowerRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches/Virtual_Base.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches/UpperRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches/coarse_rna_cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/coarse_rna/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches.txt.slim -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/water\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/water/TP3.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/water\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/water/HOH.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/water\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/water/TP5.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/water\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-amino-3-methylamino-propanoic_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4.5-dihydroxy-isoleucine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2.4-dimethyl-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/MAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/beta-(1-naphthyl)-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/beta-iodo-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha-methyl-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-hydroxy-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha-methyl-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha.gama-diaminobutyric_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/tert-butyl-cysteine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-aminomethyl-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-tert-butyl-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/ornithine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/5-methyl-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/trifluoro-leucine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/S56.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/V04.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/9-anthryl-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/beta-cyclopentyl-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-fluoro-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/BCS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/6-chloro-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/V01.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-amino-tetrahydropyran-4-carboxylic_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/ABA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-amino-tetrahydrothiopyran-4-carboxylic_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/0TD.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/3-methyl-histidine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/fluoro-leucine_ent1.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/homocysteine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-fluoro-proline.params_rot -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/3-amino-tyrosine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/HPR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha-amino-glycine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/homophenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4.5-dehydro-leucine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/cyclohexyl-glycine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/5-chloro-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/V02.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-methyl-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/phenylglycine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-amino-4-bromo-4-pentenoic_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-indanyl-glycine_puck2.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/dipropyl-glycine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/7-bromo-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-indanyl-glycine_puck1.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/diphenylglycine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha-methyl-tyrosine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/BZP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-fluoro-proline_puck.params_rot -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4.5-dehydro-lysine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/1-methyl-histidine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-allyl-glycine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/beta-fluoro-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha.beta-diaminopropionic_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/3-methyl-histidine_prot.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/hexafluoro-leucine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/HIP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-amino-piperidine-4-carboxylic-acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/HTY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/beta-beta-dicyclohexyl-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha-methyl-proline.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/5-fluoro-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha-methyl-leucine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/beta-cyclohexyl-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-carboxy-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/n-in-methyl-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/HP2.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/6-bromo-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha-methyl-valine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/YPN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/5-hydroxy-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/beta-chloro-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-phenyl-phenylalanine_tyr_rot.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/tert-butyl-glycine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/TES.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-amino-heptanoic_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/3-hydroxy-tyrosine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/amino-ethyl-cysteine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-methyl-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/V03.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/beta.beta-diphenyl-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/APA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-methyl-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/allo-threonine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/beta-(2-naphthyl)-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/trifluoro-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/phenyl-serine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-phenyl-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/MTP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/6-methyl-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-amino-octanoic_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/7-azatryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/6-fluoro-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha-methyl-histidine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/ethionine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-amino-5-phenyl-pentanoic_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/2-amino-2-phenylbutyric_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-hydroxy-phenylglycine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/trifluoro-leucine_ent2.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/3-hydroxy-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-fluoro-proline_puck.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/3-aminomethyl-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/allo-isoleucine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-aminomethyl-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/homoserine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/fluoro-leucine_ent2.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/tert-butyl-alanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/NLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/5-bromo-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/1-amino-cyclopentane-carboxylic_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/SAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/penicillamine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/NVL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/3-chloro-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/1-methyl-histidine_prot.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/4-fluoro-proline.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/3-methyl-phenylalanine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha-aminoadipic_acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/HLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/beta-hydroxy-norvaline.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/alpha-methyl-3-hydroxy-tyrosine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/MPA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa/7-methyl-tryptophan.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-ncaa\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terminal\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terminal/BZA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terminal\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terminal/NME.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terminal\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terminal/BZO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terminal\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terminal/ACE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terminal\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/BoronateLDopa.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/1.3.5_trisbromomethylbenzene.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/1.8_bisbromomethylnaphthalene.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/1.2_bisbromomethylbenzene.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/2.6_bisbromomethylpyridine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/1.4_bisbromomethylbenzene.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/1.3.5_trisbromomethylbenzene_symm.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/1.3_bisbromomethylbenzene.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/2.7_bisbromomethylnaphthalene.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/2.6_bisbromomethylnaphthalene.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker/triazolemer_linker.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/crosslinker\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/cyclosporin\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/cyclosporin/LMA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/cyclosporin\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/cyclosporin/BMT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/cyclosporin\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/lipids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/lipids/ONonOMe.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/lipids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/lipids/sphingosine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/lipids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/lipids/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/lipids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/lipids/linoleate.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/lipids\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/spin_labels\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/spin_labels/R1A_rotamers_nonclashing_moe_saved.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/spin_labels\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/spin_labels/R1A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/spin_labels\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/G4TYR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/4D3D2L_GALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/GABA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/4L3L_GALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/G4GLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/4L3L2L_GALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/4L3D2L_GALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/G4LEU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/G4PHE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/G4ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/G4LYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/G4VAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/4D3L_GALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/4L2L_GALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/3L2L_GALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/G3ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/4D2L_GALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/3D2L_GALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/4D3L2L_GALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/G4ASP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide/G2ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/gamma-peptide\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/OHH.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/OC3.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/HHA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/CO3.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/HYD.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/Hice.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/Oice.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/PHO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/CAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/OPH.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/OSi.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/Si.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface/PHA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/mineral_surface\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/SSC.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/2SU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/3MC.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/7MG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/I6A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/MA6.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/52C.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/5MU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/IGU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/OMU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/CNM.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/SRU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/1MG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/S4A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/K2C.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/31U.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/S4C.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/2MU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/SSA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/APU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/6MZ.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/3TD.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/7DA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/M5U.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/MPU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/RIA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/OMC.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/MOT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/2ST.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/MIA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/S2C.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/PSU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/4OC.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/Q.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/M26.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/1MA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/BRU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/O2W.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/S4G.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/5MC.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/S4U.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/FA7.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/NST.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/FMU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/M1I.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/M2G.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/F5C.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/MST.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/M3G.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/CNS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/H2U.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/NMT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/1PU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/CNT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/FMA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/INO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/MOU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/CMT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/FMG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/2MA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/ICY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/OMG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/A4C.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/C5T.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/4SU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/T6A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/2MG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/SSG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/MTA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/OAU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/SSU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/SRG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/yW.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/IHA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/FMC.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/52U.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/OMA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/1AP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/MHU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/UR3.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/SRC.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural/SRA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_nonnatural\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna/GDP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna/GTP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna/CYT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna/THY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna/ADE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna/GNP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna/GUA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/dna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/URA_s.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/URA_n.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/RAD_n.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry/U_s_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry/U_n_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry/C_s_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry/G_n_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry/A_n_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry/G_s_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry/C_n_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry/A_s_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/ideal_geometry\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/RGU_n.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/RAD_s.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/RCY_n.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/RGU_s.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix/RCY_s.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_phenix\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_legacy\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_legacy/RGU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_legacy\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_legacy/URA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_legacy\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_legacy/RAD.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_legacy\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_legacy/RCY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/nucleic/rna_legacy\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/membrane\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/membrane/MEM.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/membrane\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/membrane/EMB.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/membrane\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/CU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/MG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/ZN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/NA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/MN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/LN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/CO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/FE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/CA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/K.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions/FE2.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_ions\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states/LYS_D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states/GLU_P1.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states/HIS_P.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states/ASP_P1.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states/ASP_P2.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states/GLU_P2.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states/TYR_D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/protonation_states\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/ptm-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/ptm-caa/PTR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/ptm-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/ptm-caa/SEP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/ptm-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/ptm-caa/TPO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/ptm-caa\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-alpha-L-Rhap.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-beta-Neup.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-beta-D-Manp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-beta-Bacp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-alpha-D-Xylp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-alpha-L-Rhap.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to6-alpha-D-Galp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to6-alpha-D-Glcp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-alpha-Neup.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-beta-D-Xylp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-alpha-D-Galp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-alpha-L-Fucp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-beta-L-Gulp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-alpha-D-Abep.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-alpha-D-Galp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-alpha-L-Fucp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to6-beta-D-Galp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to2-alpha-D-Manp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-beta-D-Galp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-alpha-D-Manp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to6-beta-D-Manp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-beta-D-Fruf.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-beta-L-Manp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to8-alpha-Neup.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-beta-D-Glcp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-alpha-Daup.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-alpha-D-Manp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-alpha-D-Glcp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to6-alpha-D-Manp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-beta-L-Fucp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-alpha-D-Glcp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to5-alpha-L-Araf.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-beta-D-Galp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-beta-D-Glcp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-alpha-L-Idop.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-beta-L-Altp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to2-beta-D-Glcp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to4-beta-D-Quip.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to2-beta-D-Galp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to3-beta-D-Manp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to6-beta-D-Glcp.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates/to5-beta-D-Psip.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/carbohydrates\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/coenzymes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/coenzymes/flavin_reduced.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/coenzymes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/coenzymes/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/coenzymes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/coenzymes/flavin_oxidized.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/coenzymes\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_binding_ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_binding_ncaa/2S-2-amino-7-sulfanylheptanoic-acid.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_binding_ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_binding_ncaa/BPY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_binding_ncaa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_binding_ncaa/6-mercapto-norleucine.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/metal_binding_ncaa\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3I.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/cisACHC.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3C.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3Q.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3N.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3K.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3T.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3F.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3G.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3R.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3E.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3H.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3M.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/cisACPC.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3L.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3V.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3S.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3O.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3P.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/transACPC.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3X.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3W.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide/B3Y.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/beta-peptide\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna/TPN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna/GPN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna/CPN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna/APN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna/UPN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/pna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/staple\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/staple/08B.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/staple\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/staple/08A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/staple\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/ASN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/THR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/CYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/CYV.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/HIS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/ASP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/LEU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/PRO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/ILE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/GLN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/VAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/PHE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/TYR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/HIS_D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/ARG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/MET.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/SER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/LYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/TRP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/CYZ.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/GLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa/GLY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/l-caa\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/extra\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/extra/CRO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/extra\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/extra/C.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/extra\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific/INV_VRT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific/VRT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific/GB_AA_PLACEHOLDER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific/SUCK.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific/VRTBB.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific/VRT1.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/rosetta_specific\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terpenes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terpenes/isoprene.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/terpenes\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/204.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/125.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/413.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/402.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/126.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/412.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/132.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/506.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/104.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/014.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/016.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/509.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/403.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/702.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/006.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/110.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/211.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/010.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/205.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/118.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/003.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/112.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/013.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/317.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/206.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/sar.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/320.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/504.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/105.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/130.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/007.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/623.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/129.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/704.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/122.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/002.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/131.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/309.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/116.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/106.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/407.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/111.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/119.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/121.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/107.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/314.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/316.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/017.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/404.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/303.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/123.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/203.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/208.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/117.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/703.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/302.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/109.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/005.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/127.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/621.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/008.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/103.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/406.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/410.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/304.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/313.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/503.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/505.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/209.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/124.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/411.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/001.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/315.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/401.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/101.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/102.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/601.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/113.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/507.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/308.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/305.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/307.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/201.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/015.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/114.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/701.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/210.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/012.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/501.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/502.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/318.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/108.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/405.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/011.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/409.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/207.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/021.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/115.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/020.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/408.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/202.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/128.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/019.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/009.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/120.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/508.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/306.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/004.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids/018.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/peptoids\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/VOR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/PET.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/VHA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/DLYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/DASX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/ALT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/ASX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/GLX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/DGLX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/VLS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/VDP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/CYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/LYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation/VSR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types/sidechain_conjugation\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/exclude_pdb_component_list.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/merge_residue_behaviors.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/residue_types.txt.slim -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/cys_acetylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/water\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/water/VirtualWater.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/water\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/SC_Fragment.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/N_formylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/disulfide.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/NtermProteinFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/tyr_diiodinated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/NtermCtermDipeptoid.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/D_AA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/lys_dimethylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/protein_cutpoint_upper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/PtermProteinFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/hbs_post.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/NtermTruncation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/glu_carboxylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/Cterm_amidation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/pro_hydroxylated_case1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/his_methylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/ReplsBB.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching/aryl-C-conjugated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching/S-linked_conjugation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching/phg_cd1_conjugation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching/O-linked_conjugation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching/sidechain_electrophile_conjugation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching/N-linked_conjugation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching/C-terminal_conjugation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching/phg_cd2_conjugation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching/aryl-O-conjugated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/branching\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/ser_phosphorylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/C_methylamidated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/oop_post.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna/UpperDNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna/VirtualDNAPhosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna/dna_cutpoint_upper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna/dna_cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna/LowerDNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/dna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/3prime5prime_methyl_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/rna_cutpoint_upper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/3prime_packable_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/Virtual_RNA_Residue_Exclude_Phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/Protonated_N3_Adenosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/UpperConnectionRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/Virtual_Phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/BlockStackAbove.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/Virtual_Ribose.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/LowerRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/LowerConnectRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/BulgeRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/5prime_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/Protonated_N1_Adenosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/3prime_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/VirtualBaseHeavyAtom.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/3prime_end_OH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/Deoxy_O2prime.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/5prime_end_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/2prime_connection.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/L_RNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/rna_cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/Virtual_Backbone_Except_C1prime.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/Virtual_O2prime_hydrogen.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/VirtualBase.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/UpperRNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/7deaza.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/BlockStackBelow.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/Repulsive_Phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/5prime_end_OH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/5prime_packable_phosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna/Virtual_RNA_Residue.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/nucleic/rna\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/a3b_hbs_post.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/triazolamerC.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/tyr_sulfated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/AcetylatedProteinNterm.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/NtermPeptoidFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/N_acetylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/lys_monomethylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/NtermProteinMethylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/N_methylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/MethylatedProteinCterm.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/lys_acetylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/CtermTruncation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/C-linked_glycosylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/2-reduction.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/6-branch.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/N2-acetylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/lower_terminus.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/4-branch.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/4-reduction.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/6-sulfation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/N-R-3'-hydroxybutyryl-3-amination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/omega-oxidation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/2-branch.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/2-sulfation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/N-linked_glycosylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/3-methylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/N4-acetylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/Me_glycoside.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/3-amination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/cutpoint_upper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/3-branch.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/upper_terminus.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/N5-acetylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/N-acetyl-2-amination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/2-sulfamidation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/2-amination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates/cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/carbohydrates\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/SpecialRotamer.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/VirtualProteinSideChain.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/thr_phosphorylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/VirtualBB.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/RepulsiveOnly_fa.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/tyr_phosphorylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/a3b_hbs_pre.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/lys_trimethylated.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/triazolamerN.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/VirtualProteinResidue.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/protein_cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/ShoveBB.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/oop_pre.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/AcetylatedPeptoidNterm.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/terpenes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/terpenes/terpene_lower_term.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/terpenes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/terpenes/terpene_upper_term.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/terpenes\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/CtermPeptoidFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/pro_hydroxylated_case2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/CtermProteinFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/hbs_pre.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches/VirtualNterm.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/aromatic_trifluoromethylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/connect.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/aromatic_iodination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/aromatic_ethylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/aromatic_chlorination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/pruneH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/aromatic_bromination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/aromatic_methoxylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/aromatic_hydroxylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/aromatic_methylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/metal_connect.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches/aromatic_fluorination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_talaris/metapatches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/ASN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/THR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/CYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/HIS_DE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/HIS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/ASP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/LEU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/PRO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/ILE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/GLN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/VAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/PHE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/TYR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/HIS_D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/ARG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/MET.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/SER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/LYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/TRP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/GLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS/GLY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/homogres.sh -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/homogres_revert.sh -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/ASN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/TP3.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/GDP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/THR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/CYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/GTP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/HIS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/TP5.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/CYT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/THY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/ADE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/ASP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/08B.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/LEU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/PRO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions/MG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions/ZN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions/NA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions/MN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions/FE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions/CA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions/K.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions/FE2.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/metal_ions\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/ILE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/GLN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/VAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/PHE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/TYR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/HIS_D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/CYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/ARG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/MET.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/SER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/VRT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/LYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/LYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/TRP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/CYZ.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/GB_AA_PLACEHOLDER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/GNP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/SUCK.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/GLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/GLY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/08A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/GUA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types/VRT1.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/exclude_pdb_component_list.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/merge_residue_behaviors.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/residue_types_HOMOGENEOUS.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/UpperDNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/VirtualDNAPhosphate.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/SC_Fragment.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/disulfide.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/NtermProteinFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/D_AA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/protein_cutpoint_upper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/NtermTruncation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/CtermTruncation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/VirtualBB.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/RepulsiveOnly_fa.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/protein_cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/ShoveBB.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/LowerDNA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches/CtermProteinFull.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches/connect.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches/aromatic_iodination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches/aromatic_ethylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches/aromatic_chlorination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches/aromatic_bromination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches/aromatic_methoxylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches/aromatic_hydroxylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches/aromatic_methylation.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches/metal_connect.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches/aromatic_fluorination.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/fa_standard_05.2009_icoor/metapatches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/ASN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/THR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/CYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/HIS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/ASP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/LEU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/PRO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/ILE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/GLN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/VAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/PHE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/TYR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/ARG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/MET.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/SER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/VRT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/LYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/TRP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/CYZ.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/GLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types/GLY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/residue_types.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/merge_residue_behaviors.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/metapatches.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches/disulfide.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches/D_AA.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches/protein_cutpoint_upper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches/NtermProtein.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches/protein_cutpoint_lower.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches\n",
            "copying build/lib/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches/CtermProtein.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/residue_type_sets/centroid_rot/patches\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/element_sets\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/element_sets/default\n",
            "copying build/lib/pyrosetta/database/chemical/element_sets/default/element_properties.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/element_sets/default\n",
            "copying build/lib/pyrosetta/database/chemical/element_sets/default/para_ion_properties.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/element_sets/default\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/gpu\n",
            "copying build/lib/pyrosetta/database/gpu/DARC_VERSION2.cl -> build/bdist.linux-x86_64/egg/pyrosetta/database/gpu\n",
            "copying build/lib/pyrosetta/database/gpu/DARC_PSO_PHIPSI.cl -> build/bdist.linux-x86_64/egg/pyrosetta/database/gpu\n",
            "copying build/lib/pyrosetta/database/gpu/DARC_PSO.cl -> build/bdist.linux-x86_64/egg/pyrosetta/database/gpu\n",
            "copying build/lib/pyrosetta/database/gpu/sc.cl -> build/bdist.linux-x86_64/egg/pyrosetta/database/gpu\n",
            "copying build/lib/pyrosetta/database/gpu/darc.cl -> build/bdist.linux-x86_64/egg/pyrosetta/database/gpu\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/citations\n",
            "copying build/lib/pyrosetta/database/citations/rosetta_citations.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/citations\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/AtomVDW\n",
            "copying build/lib/pyrosetta/database/scoring/AtomVDW/atom_vdw.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/AtomVDW\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/dfire\n",
            "copying build/lib/pyrosetta/database/scoring/dfire/dfire_pair.lib.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/dfire\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_02\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_02/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_02\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_02/potential.bin.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_02\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_02/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_02\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_03\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_03/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_03\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_03/potential.bin.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_03\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_03/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_03\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_00\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_00/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_00\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_00/potential.bin.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_00\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_00/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_00\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_01\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_01/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_01\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_01/potential.bin.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_01\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_01/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_01\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_09\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_09/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_09\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_09/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_09\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_06\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_06/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_06\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_06/potential.bin.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_06\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_06/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_06\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/.gitignore -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_08\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_08/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_08\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_08/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_08\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_05\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_05/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_05\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_05/potential.bin.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_05\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_05/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_05\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_07\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_07/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_07\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_07/potential.bin.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_07\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_07/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_07\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_04\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_04/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_04\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_04/potential.bin.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_04\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_04/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_04\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_10\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_10/potential.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_10\n",
            "copying build/lib/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_10/potential.txt.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/loop_close/6D_potentials/rna/loop_10\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/constraints\n",
            "copying build/lib/pyrosetta/database/scoring/constraints/epr_distance_potential.histogram -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/constraints\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/qsar\n",
            "copying build/lib/pyrosetta/database/scoring/qsar/lj_table.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/qsar\n",
            "copying build/lib/pyrosetta/database/scoring/qsar/shape_histogram_data.js -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/qsar\n",
            "copying build/lib/pyrosetta/database/scoring/qsar/hb_table.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/qsar\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/qsar/weights\n",
            "copying build/lib/pyrosetta/database/scoring/qsar/weights/standard.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/qsar/weights\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/sucker\n",
            "copying build/lib/pyrosetta/database/scoring/sucker/pusher.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/sucker\n",
            "copying build/lib/pyrosetta/database/scoring/sucker/sucker.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/sucker\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/sasa_scores\n",
            "copying build/lib/pyrosetta/database/scoring/sasa_scores/sasapack_datafile_v1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/sasa_scores\n",
            "copying build/lib/pyrosetta/database/scoring/sasa_scores/avge_datafile_score12prime_v1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/sasa_scores\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQM_model.4.linear -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQ2_model.1.linear -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQ2_model.3.linear_retrained_for_rosetta -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQ2_model.5.linear -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQ2_model.1.linear_retrained_for_rosetta -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQM_model.2.linear -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQM_model.2.linear_retrained_for_rosetta -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQM_model.3.linear -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQ2_model.4.linear_retrained_for_rosetta -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQ2_model.2.linear_retrained_for_rosetta -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQM_model.1.linear_retrained_for_rosetta -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQ2_model.2.linear -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQ2_model.5.linear_retrained_for_rosetta -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQ2_model.4.linear -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQ2_model.3.linear -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQM_model.1.linear -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQM_model.3.linear_retrained_for_rosetta -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQM_model.5.linear_retrained_for_rosetta -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQM_model.4.linear_retrained_for_rosetta -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "copying build/lib/pyrosetta/database/scoring/ProQ/ProQM_model.5.linear -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/ProQ\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rosettaholes\n",
            "copying build/lib/pyrosetta/database/scoring/rosettaholes/decoy15.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rosettaholes\n",
            "copying build/lib/pyrosetta/database/scoring/rosettaholes/resl.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rosettaholes\n",
            "copying build/lib/pyrosetta/database/scoring/rosettaholes/decoy25.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rosettaholes\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/dna\n",
            "copying build/lib/pyrosetta/database/scoring/dna/bound_dna_dihedrals.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/dna\n",
            "copying build/lib/pyrosetta/database/scoring/dna/dna_bs_bp.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/dna\n",
            "copying build/lib/pyrosetta/database/scoring/dna/dna_env_pair.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/dna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/nmr\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/nmr/spinlabel\n",
            "copying build/lib/pyrosetta/database/scoring/nmr/spinlabel/r1a_pre_hn_distance_potential.histogram -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/nmr/spinlabel\n",
            "copying build/lib/pyrosetta/database/scoring/nmr/spinlabel/spinlabel_properties.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/nmr/spinlabel\n",
            "copying build/lib/pyrosetta/database/scoring/nmr/spinlabel/r1a_pre_n_distance_potential.histogram -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/nmr/spinlabel\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/sc\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/sc/sc_radii.lib -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/sc\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/p_aa_ss.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/etable\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/etable/resolve_etable.twobead.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/etable\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/etable/etable.twobead.dlj.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/etable\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/etable/etable.twobead.lj.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/etable\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/disulfides\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/disulfides/fa_CbSSCb_dihedral_score -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/disulfides\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/disulfides/centroid_distance_score -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/disulfides\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/disulfides/centroid_CaCbCbCa_dihedral_score -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/disulfides\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/disulfides/fa_SS_distance_score -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/disulfides\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/disulfides/centroid_backbone_dihedral_score -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/disulfides\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/disulfides/fa_CaCbS_angle_score -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/disulfides\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/disulfides/centroid_CaCbCb_angle_score -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/disulfides\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/unique_database_mm_std_mean -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/rosetta_database_mm_std_mean -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/pdb_database_talaris2013_median -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/ele_database_mm_std_boltz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/pdb_database_talaris2013_mean -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/ele_database_talaris2013_boltz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/rosetta_database_mm_std_boltz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/ele_database_mm_std_mean -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/rosetta_database_talaris2013_mode -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/pdb_database_talaris2013_boltz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/pdb_database_talaris2013_mode -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/unique_database_talaris2013_boltz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/mm_database_mm_std_mean -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/mm_database_mm_std_mode -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/mm_database_mm_std_boltz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/rosetta_database_mm_std_mode -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/unique_database_mm_std_median -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/rosetta_database_mm_std_median -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/split_unfolded_two_body_default -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/ele_database_mm_std_median -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/ele_database_talaris2013_mean -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/rosetta_database_talaris2013_boltz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/mm_database_talaris2013_boltz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/unique_database_mm_std_boltz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/rosetta_database_talaris2013_median -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/unique_database_talaris2013_mode -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/mm_database_talaris2013_median -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/split_unfolded_one_body_talaris2013 -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/mm_database_talaris2013_mean -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/ele_database_talaris2013_median -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/split_unfolded_one_body_mm_std -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/unique_database_talaris2013_mean -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/mm_database_mm_std_median -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/mm_database_talaris2013_mode -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/rosetta_database_talaris2013_mean -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/ele_database_mm_std_mode -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/unique_database_mm_std_mode -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/unique_database_talaris2013_median -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/split_unfolded/ele_database_talaris2013_mode -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/split_unfolded\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_SS_cen6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_SS_cen6/P_AA_SS.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_SS_cen6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_SS_cen6/P_AA_SS_cen6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_SS_cen6\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/EnvPairPotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/EnvPairPotential/pair_log.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/EnvPairPotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/EnvPairPotential/env_log.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/EnvPairPotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/EnvPairPotential/cenpack_log.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/EnvPairPotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/EnvPairPotential/cbeta_den.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/EnvPairPotential\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/unfolded/unfolded_state_residue_energies_rna -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/unfolded/unfolded_state_residue_energies_mm_std -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/unfolded\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/unfolded/unfolded_state_residue_energies_score12 -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/unfolded\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle_talaris\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle_talaris/default-improper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle_talaris\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle_talaris/default-torsions.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle_talaris\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle_talaris/default-lengths.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle_talaris\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle_talaris/default-angles.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle_talaris\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/SecondaryStructurePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/SecondaryStructurePotential/rsigma_dot.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/SecondaryStructurePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/SecondaryStructurePotential/phi.theta.36.SS.resmooth -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/SecondaryStructurePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/SecondaryStructurePotential/phi.theta.36.HS.resmooth -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/SecondaryStructurePotential\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/goap\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/goap/angle_definitions.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/goap\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/goap/dist_table.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/goap\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/goap/angle_table.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/goap\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/GLN__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/ASN__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/LYS__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/PRO__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/ASP__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/SER__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/ff-rosetta.cfg -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/GLU__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/MET__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/THR__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/ff-rosetta-fa.cfg -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/LEU__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/ILE__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/CYS__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/N.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/TYR__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/always_one.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/TRP__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/ARG__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/O.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/C.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/H.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/HIS__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/VAL__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/ff-rosetta-cen.cfg -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/saxs/PHE__CEN.ff -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/saxs\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/ELazaridis_polynom_table.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/CEN10_mem_env_log.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/CEN6_mem_env_log.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/CEN10_Menv_smooth_log.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/spline_test_cen.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/spline_test_fa.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/elazar_spline_mp_span_ins_fa.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/ELazaridis_cen_polynom_table.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/mem_env_log.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/memcbeta_den.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/MembranePotential/mem_pair_log.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/MembranePotential\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DHO_Hpol_bbOrbH.large_bins -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DHO_Haro_scOrbH.large_bins -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_AOH_Haro_scOrbH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DHO_Haro_scOrbH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DHO_Hpol_scOrbH.large_bins -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DHO_Haro_scOrbH.small_angle_bins -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DHO_Hpol_scOrbH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_AOO_OrbOrb.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_ACO.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DHO_Hpol_bbOrbH.small_angle_bins -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_AOH_Hpol_scOrbH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_AOD_OrbOrb.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DOA_OrbOrb.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_AOH_Hpol_bbOrbH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DHO_Hpol_bbOrbH.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DOO_OrbOrb.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/orbitals/BiCubic_DHO_Hpol_scOrbH.small_angle_bins -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/orbitals\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle_beta\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle_beta/default-improper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle_beta\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/InterchainPotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/InterchainPotential/interchain_env_log.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/InterchainPotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/InterchainPotential/interchain_pair_log.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/InterchainPotential\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/NV\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/NV/neighbor_vector_score.histogram -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/NV\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/elec_cp_reps.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle/bbdep-graphdata-pro.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle/hydroxyl_torsion_preference.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle/bbdep-graphdata-gly.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle/default-improper.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle/bbdep-graphdata-xpro.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle/default-torsions.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle/bbdep-all-but-gly-pro-xpro-ile-val.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle/default-lengths.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle/bbdep-graphdata-ile-val.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/bondlength_bondangle/default-angles.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/bondlength_bondangle\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/netcharge\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/netcharge/net_zero.charge -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/netcharge\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/netcharge/negative_one.charge -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/netcharge\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/Paa -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/Paa_n -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/P_AA -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/P_AA_n -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/P_AA_pp -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/P_AA_pp_08.2009 -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/argumentsExample.shapovalov.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/kappa50\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/kappa50/a20.prop -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/kappa50\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/kappa50/AcademicLicensePropensityMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/kappa50\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/AcademicLicensePropensityMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/kappa131\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/kappa131/a20.prop -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/kappa131\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/kappa131/AcademicLicensePropensityMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/kappa131\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg/Taken.From.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/2.5deg\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/AcademicLicensePropensityMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa50\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa50/a20.prop -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa50\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa50/AcademicLicensePropensityMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa50\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/AcademicLicensePropensityMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa131\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa131/a20.prop -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa131\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa131/AcademicLicensePropensityMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa131\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg/Taken.From.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/P_AA_pp/shapovalov/10deg\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/custom_pair_distance\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/custom_pair_distance/pos_charge_repulsion -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/custom_pair_distance\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/custom_pair_distance/neg_charge_repulsion -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/custom_pair_distance\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/custom_pair_distance/fa_custom_pair_distance -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/custom_pair_distance\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/His_Phil_fix\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/BBasSC_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/helix_hb_06_2009\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/AccStrength.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params/DonStrength.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/ref2015_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/.gitignore -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/score12_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/score12_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_elec_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_no_lr_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/sp2_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/sp2_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/params.db3 -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/OLF_params_4\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/extended_BAH_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/experimental_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/experimental_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/correct_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/correct_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newOH_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newOH_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/HBAccChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/HBondWeightType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/HBAccHybridization.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/HBDonChemType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/HBFadeIntervals.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/schema.sql -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/HBSeqSep.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/HybridizationType.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/HBPoly1D.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params/HBEval.csv -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/hbonds/newCHI_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/envsmooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/envsmooth/notes_on_envdata.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/envsmooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/envsmooth/envdata.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/envsmooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/envsmooth/envdata_zero_negatives.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/envsmooth\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/alpha_ncaa\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/alpha_ncaa/AIB_prepro.old.rama -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/alpha_ncaa\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/alpha_ncaa/AIB_general.rama -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/alpha_ncaa\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/alpha_ncaa/AIB_general.old.rama -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/alpha_ncaa\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/alpha_ncaa/AIB_prepro.rama -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/alpha_ncaa\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd_beta_nov2016\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd_beta_nov2016/prepro.ramaProb -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd_beta_nov2016\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd_beta_nov2016/all.ramaProb -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd_beta_nov2016\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/aramid/OAA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/aramid/UAA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/aramid/PAA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/aramid/QAA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/aramid/RAA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/aramid/KAA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/aramid/TAA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/aramid/SAA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/aramid/MAA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/aramid/LAA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/aramid\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/Rama09_noEH_kernel25_it08.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd/prepro.ramaProb -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd/prepro.ramaProb.loose -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd/all.ramaProb.rb.100_2_0_20 -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd/all.ramaProb.shapovalov.kappa25 -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd/all.ramaProb.loose -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd/prepro.ramaProb.rb -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd/all.ramaProb.rb -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd/prepro.ramaProb.rb.100_2_0_20 -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/fd/all.ramaProb -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/fd\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/oligourea\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/oligourea/OU3_PRO_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/oligourea\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/oligourea/OU3_PRO_rama_preproline.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/oligourea\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/oligourea/OU3_ALA_rama.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/oligourea\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/oligourea/OU3_ALA_rama_preproline.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/oligourea\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/n_methyl\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/n_methyl/sarcosine_ramatable_MM.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/n_methyl\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/n_methyl/sarcosine_ramatable_prepro_MM.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/n_methyl\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/n_methyl/Generic_L_Alpha_Nmethyl_AA_prepro_ramatable_QM.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/n_methyl\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/n_methyl/Generic_L_Alpha_Nmethyl_AA_ramatable_QM.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/n_methyl\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/flat\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/flat/sym_all_rama_str.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/flat\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/flat/sym_all_rama.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/flat\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/flat/sym_P_rama_str.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/flat\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/flat/avg_L_rama_str.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/flat\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/flat/sym_G_rama.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/flat\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/flat/avg_L_rama.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/flat\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/flat/sym_G_rama_str.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/flat\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/flat/sym_P_rama.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/flat\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/Rama08.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/AcademicLicenseRamaMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa25\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa25/AcademicLicenseRamaMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa25\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa25/Suggested.Default.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa25\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa25/all.ramaProb -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa25\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/.gitignore -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/argumentsExample.shapovalov.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa50\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa50/AcademicLicenseRamaMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa50\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa50/all.ramaProb -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa50\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa75\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa75/AcademicLicenseRamaMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa75\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa75/all.ramaProb -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa75\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/Suggested.Default.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa37.5\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa37.5/AcademicLicenseRamaMap.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa37.5\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa37.5/all.ramaProb -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov/kappa37.5\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/shapovalov/Taken.From.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama/shapovalov\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/Rama.10.2009.yfsong.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/Rama_XPG_3level.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/rama/Rama_smooth_dyn.dat_ss_6.4 -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/rama\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbohydrates\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/carbohydrates/CHI_energy_function_for_beta6_linkages.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbohydrates\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/carbohydrates/CHI_energy_function_for_2ax_3eq_4ax_linkages.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbohydrates\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/carbohydrates/CHI_energy_function_for_alpha_linkages.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbohydrates\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/carbohydrates/CHI_energy_function_for_2eq_3ax_4eq_linkages.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbohydrates\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/carbohydrates/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbohydrates\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/carbohydrates/CHI_energy_function_for_alpha6_linkages.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbohydrates\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/carbohydrates/CHI_energy_function_for_beta_linkages.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbohydrates\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/aa_repeat_energy\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/aa_repeat_energy/default_repeat_penalty_table.rpt_pen -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/aa_repeat_energy\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/generic_potential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/generic_potential/generic_bonded.aamerge.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/generic_potential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/generic_potential/generic_bonded.round6p.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/generic_potential\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/centroid_smooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/centroid_smooth/cen_rot_pair_dih_params.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/centroid_smooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/centroid_smooth/cen_rot_cbeta_params.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/centroid_smooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/centroid_smooth/cen_smooth_params.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/centroid_smooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/centroid_smooth/cen_rot_dfire_params.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/centroid_smooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/centroid_smooth/cen_rot_dpair_params.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/centroid_smooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/centroid_smooth/cen_hb_params.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/centroid_smooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/centroid_smooth/cen_rot_pair_params.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/centroid_smooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/centroid_smooth/cen_rot_env_params.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/centroid_smooth\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/centroid_smooth/cen_rot_pair_ang_params.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/centroid_smooth\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/ASN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/TP3.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/GDP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/THR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/GLY_org.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/CYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/GTP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/ASN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/TP3.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/GDP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/THR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/CYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/GTP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/HIS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/TP5.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/CYT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/CYD.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/THY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/ADE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/ASP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/08B.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/LEU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/PRO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/GLU_mod.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/ILE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/GLN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/VAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/PHE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/TYR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/HIS_D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/CYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/ARG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/MET.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/SER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/VRT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/LYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/LYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/TRP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/CYZ.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/GB_AA_PLACEHOLDER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/GNP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/SUCK.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/GLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/GLY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/08A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/GUA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/modified/VRT1.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/modified\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/HIS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/ASN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/TP3.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/FACTS_AA_PLACEHOLDER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/GDP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/THR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/CYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/GTP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/HIS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/TP5.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/CYT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/CYD.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/THY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/ADE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/ASP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/08B.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/LEU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/PRO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/ILE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/GLN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/VAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/PHE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/TYR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/HIS_D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/CYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/ARG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/MET.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/SER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/VRT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/LYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/LYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/TRP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/CYZ.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/GB_AA_PLACEHOLDER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/GNP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/SUCK.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/GLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/GLY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/08A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/GUA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/eff/VRT1.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/eff\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/TP5.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/CYT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/CYD.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/THY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/ADE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/ASP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/08B.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/LEU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/PRO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/GLU_mod.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/ILE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/GLN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/VAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/PHE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/TYR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/HIS_D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/CYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/ARG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/MET.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/SER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/VRT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/LYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/LYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/TRP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/CYZ.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/GB_AA_PLACEHOLDER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/GNP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/ASN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/TP3.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/GDP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/THR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/ALA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/GLY_org.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/CYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/GTP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/HIS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/TP5.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/CYT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/CYD.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/THY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/ADE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/ASP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/08B.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/LEU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/PRO.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/GLU_mod.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/ILE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/GLN.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/VAL.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/PHE.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/TYR.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/HIS_D.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/CYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/ARG.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/MET.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/SER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/VRT.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/LYS.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/LYX.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/TRP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/CYZ.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/GB_AA_PLACEHOLDER.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/GNP.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/SUCK.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/GLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/GLY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/08A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/GUA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/test/VRT1.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts/test\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/SUCK.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/GLU.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/GLY.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/08A.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/GUA.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/facts/VRT1.params -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/facts\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/aa_composition\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/aa_composition/sample_composition.comp -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/aa_composition\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/aa_composition/ten_percent_aromatic.comp -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/aa_composition\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/aa_composition/two_aromatics.comp -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/aa_composition\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/aa_composition/exactly_one_ala.comp -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/aa_composition\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/aa_composition/exactly_one_trp.comp -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/aa_composition\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/aa_composition/twenty_percent_pro.comp -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/aa_composition\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/PairEPotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/PairEPotential/pdb_pair_stats_fine -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/PairEPotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/jump_templates_SSpairs_v2.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/DDPscore\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/DDPscore/interface_ddp_score.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/DDPscore\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/p_aa_abego3.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/omega\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/omega/omega_ppdep.pro.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/omega\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/omega/omega_ppdep.gly.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/omega\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/omega/notes_on_data.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/omega\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/omega/omega_ppdep.prepro.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/omega\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/omega/omega_ppdep.all.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/omega\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/omega/omega_ppdep.valile.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/omega\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbon_hbond\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/carbon_hbond/ch_o_bond_potential_near_min_yf.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbon_hbond\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/carbon_hbond/ch_o_bond_potential_2.5Acutoff_smooth.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbon_hbond\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/carbon_hbond/ch_o_bond_potential.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/carbon_hbond\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ARG_1HH2_NH2.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ASP_OD2_CG.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_LYS_1HZ_NZ.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ARG_1HH1_NH1.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_GLU_OE2_CD.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ALL_O_C.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ARG_2HH1_NH1.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ASN_1HD2_ND2.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_GLN_OE1_CG.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ARG_2HH2_NH2.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_THR_OG1_CB.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_LIG_ACC_SP3.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_LIG_DON.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_GLU_OE1_CD.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_HIS_HD1_ND1.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_SER_OG_CB.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_TRP_HE1_NE1.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ASN_OD1_CG.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ARG_HE_NE.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ALL_H_N.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_HIS_ND1_XXX.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_LYS_3HZ_NZ.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_LIG_ACC_SP2.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ASP_OD1_CG.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_HIS_HE2_NE2.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_TYR_OH_CZ.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_GLN_1HE2_NE2.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_GLN_2HE2_NE2.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_ASN_2HD2_ND2.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6/E_LYS_2HZ_NZ.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/pointwater/norm_trunc6\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/mhc_epitope\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/mhc_epitope/iedb_data.db -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/mhc_epitope\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/mhc_epitope/propred8.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/mhc_epitope\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/mhc_epitope/iedb_data.mhc -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/mhc_epitope\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/mhc_epitope/iedb_data.info -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/mhc_epitope\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/mhc_epitope/refresh_iedb_database.sh -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/mhc_epitope\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/mhc_epitope/Distribution of Propred matrices with Rosetta.pdf -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/mhc_epitope\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/mhc_epitope/propred8_5.mhc -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/mhc_epitope\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/mhc_epitope/Distribution of IEDB-derived data with Rosetta.pdf -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/mhc_epitope\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/elec_group_def.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/SurfacePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/SurfacePotential/average_hASA_by_res_and_neighbor.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/SurfacePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/SurfacePotential/surface_score.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/SurfacePotential\n",
            "copying build/lib/pyrosetta/database/scoring/score_functions/SurfacePotential/hpatch_score.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/score_functions/SurfacePotential\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mm_std_no_hbond.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/scorefacts_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mm_std_fa_elec_dslf_fa13.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/corrections_conway2016.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_nov16_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score2_covalentlabeling.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/membrane_highres.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score4_cenrot_relax.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score4_smooth_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mm_std_fa_elec_dslf_fa13_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpdocking_fa_14-7-23_no-penalties.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score_membrane_center_normal.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/nv_env_design.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/scorefacts.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2013_calibrated.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_genpot.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score5_smooth.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2013_dun02.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score0_memb.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/genpot_full.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/small_radii.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2014.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/membrane_highres_Menv_smooth.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score4_smooth_cen_relax.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ddg_monomer.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpframework_smooth_fa_2012.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score4L.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2013_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_nov16.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score4_smooth.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_peptide_soft_rep_mm.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_july15_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/sp2_correction.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/interchain_cen.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ligand_soft_rep.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/enzdes_polyA_min.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_soft_rep.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ref2015_cart_cst.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score2.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/orbitals.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/occ_Hbond_sol_exact_talaris2014.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/antibody_design.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/motif_dock_score.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12_justdisulfides.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/occ_Hbond_sol_exact_talaris2013.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ligand_gen_born.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ref2015_cst.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ref2015_soft.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/genpot_full_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna_res_level_energy4.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score1.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score4_cenrot_cartmin.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/smooth_etable.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/membrane_parameters.patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12_w_corrections.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/cen_std_smooth.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/pb_elec.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/hpol_wcpd_relaxed_set_4.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2014_cst.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score4_cenrot_desgin.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/remodel_cen.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpframework_symdock_fa_2015.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/allfold_soft_rep.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_july15.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_nov16_soft.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score_membrane_env.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/gen_born.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/sp2_correction_dun02.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/burial.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/fldsgn_cen.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/interface.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/protein\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/protein/pack_no_hb_env_dep.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/protein\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/protein/protein_res_level_energy.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/protein\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/stepwise_res_level_energy.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_lores_for_stepwise.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_loop_hires_04092010.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/LOG.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_res_level_energy4.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_res_level_energy7beta.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_res_level_energy_with_intra2.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_res_level_energy7alpha.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_hires_07232011_with_intra_base_phosphate.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/CD_geom_sol_rna_loop_hires_04092010.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_hires_elec_dens.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_res_level_energy_with_intra.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/turner.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_hires_fang_with_unfolded.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_res_level_energy.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna/deprecated\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/deprecated/rna_res_level_energy_AFKmod.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna/deprecated\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/deprecated/rna_res_level_energy6.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna/deprecated\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/deprecated/rna_res_level_energy5.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna/deprecated\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/turner_no_zeros.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_hires_fang.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_helix.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_hires_elec_dens_FCC2012.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/stepwise/rna/rna_hires_elec_dens_CG2016.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/stepwise/rna\n",
            "copying build/lib/pyrosetta/database/scoring/weights/dna_gb.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2014_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/hpol_only_relaxed_set_5.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score0_covalentlabeling.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/make_rot_lib_orig.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mainchain_potential_generation.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score1_smooth.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/docking.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/make_rot_lib.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_cst.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score3_covalentlabeling.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rnp_lores.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_nov15_soft.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score13.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12_unfolded.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/sp2_correction_dun02_no_bicubic_spline.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/pre_talaris_2013_standard.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2013_occ_sol.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ProQ2.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2013_cst.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/loop_fragsample.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpframework_smooth_fa_2012_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/make_rot_lib_orig_bl_ba.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_peptide.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score1_smooth.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_peptide_mm.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12_no_hb_env_dep.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_nov15_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_nov15.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/none.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score2_memb.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/gauss.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_nov15_cst.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpframework_docking_fa_2015.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/covalent_labeling_fa.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2013.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/orbitals_talaris2013_softrep.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/HBNet.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2013_split_unfolded.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpframework_docking_cen_2015.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/occ_Hbond_sol_exact__deprecated.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ProQM.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score1_covalentlabeling.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/hrf_ms_labeling.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/occ_Hbond_sol_fitted_tal.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score5_smooth.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mm_std_fa_elec_dslf_fa13_split_unfolded_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/talaris2013_downAla.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mm_std_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12_full.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score3_smooth.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpframework_cen_2006.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ref2015.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/docking_cen.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score3_smooth.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/latest_greatest.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score3_cenrot.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/cen_std.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/occ_Hbond_sol_fitted_talaris2014.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rnp_ddg_stepwise.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/soft_rep.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/docking_occ_exact.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/pb_elec.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpdocking_cen_14-7-25_low-penalties.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/abinitio_rob.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ligand_soft_rep_gen_born.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_genpot_soft.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score0.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/enzdes.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpframework_fa_2007.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/occ_Hbond_sol_fitted_fast.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score5_covalentlabeling.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score3_rob.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rnp_ddg.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/paa_abego3.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/docking_min.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_genpot_cst.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_july15_cst.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/soft_rep_gen_born.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/occ_Hbond_sol_fitted_cst.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/empty.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/sasa_only.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/franklin2019.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12_w_corrections.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/soft_rep_design.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_peptide_soft_rep_design.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mm_std_cart_no_hbond.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpdocking_fa_14-7-25_low-penalties.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score0_membrane.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/dna_no_gb.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/abinitio_remodel_cen.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mm_std_fa_elec_dslf_fa13_split_unfolded.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_nov16_cst.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/orbitals_talaris2013.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score4_cenrot_repack.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpframework_pHmode_fa_2015.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/antibody_design_talaris2013.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score3_memb.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score3.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12minpack.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12prime.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/orbitals_dun10.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/design_hpatch.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ligandprime.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpdocking_cen_14-7-23_no-penalties.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna_minimize.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ref2015_memb.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_soft.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score1_memb.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ref2015_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score12_w_bicubic_interpolation_corrections.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score5.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score2_smooth.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/beta_genpot_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna/denovo/rna_lores.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna/denovo/rna_lores_plus_hires.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna/denovo/rna_hires_with_protein.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna/denovo/rna_and_rnp_vdw.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna/denovo/rna_lores_with_rnp_aug.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna/denovo/rna_lores_with_rnp_june05.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna/denovo/rna_lores_with_rnp.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna/denovo/rna_hires_nextgen.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna/denovo/coarse_rna.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/rna/denovo/rna_hires.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights/rna/denovo\n",
            "copying build/lib/pyrosetta/database/scoring/weights/franklin2018.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score4_cenrot_relax_cart.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/docking.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/dna.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score13_env_hb.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ligand.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/opte.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score_membrane.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score_membrane_cen.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/occ_Hbond_sol_fitted_talaris2013.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/membrane_penalties.patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/gauss1.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mm_std.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/mpframework_symdock_cen_2015.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score5_memb.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/RS_centroid.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/PPI_discrimination.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score2_smooth.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/score0.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/hydrate_score12.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/ddg.wts -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "copying build/lib/pyrosetta/database/scoring/weights/occ_Hbond_sol_fitted.wts_patch -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/weights\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_pair_dist_potential_renorm.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rna_backbone_backbone.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rna_atom_vdw.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/beta_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/nu2_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/o2h_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/zeta_alpha_ap_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/chi_purine_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/epsilon_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/zeta_alpha_sc_minus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/alpha_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/nu2_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/chi_pyrimidine_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/nu1_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/zeta_alpha_sc_plus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/chi_purine_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/nu1_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/gamma_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/o2h_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/delta_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/epsilon_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/delta_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new/chi_pyrimidine_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rotamer_based_2012_new\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/beta_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/nu2_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/zeta_alpha_ap_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/epsilon_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/zeta_alpha_sc_minus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/alpha_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/chi_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/nu2_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/nu1_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/chi_north_potential_guanosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/zeta_alpha_sc_plus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/chi_north_potential_others.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/nu1_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/gamma_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/chi_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/delta_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/chi_south_potential_others.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/chi_south_potential_guanosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/epsilon_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi/delta_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FINAL_Mar_24_2010_new_delta_zeta_chi\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/beta_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/nu2_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/o2h_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/zeta_alpha_ap_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/chi_purine_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/epsilon_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/zeta_alpha_sc_minus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/alpha_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/nu2_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/chi_pyrimidine_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/nu1_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/zeta_alpha_sc_plus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/chi_purine_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/nu1_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/gamma_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/o2h_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/delta_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/epsilon_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/delta_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new/chi_pyrimidine_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA11_based_new\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/beta_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/nu2_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/zeta_alpha_ap_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/epsilon_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/zeta_alpha_sc_minus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/alpha_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/chi_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/nu2_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/nu1_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/zeta_alpha_sc_plus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/nu1_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/gamma_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/chi_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/delta_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/epsilon_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd2008/delta_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd2008\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/beta_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/nu2_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/zeta_alpha_ap_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/epsilon_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/zeta_alpha_sc_minus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/alpha_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/chi_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/nu2_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/nu1_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/zeta_alpha_sc_plus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/nu1_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/gamma_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/chi_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/delta_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/epsilon_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps2009/delta_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps2009\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/beta_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/nu2_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/zeta_alpha_ap_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/epsilon_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/zeta_alpha_sc_minus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/alpha_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/chi_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/nu2_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/nu1_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/zeta_alpha_sc_plus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/nu1_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/gamma_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/chi_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/delta_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/epsilon_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010/delta_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/rd_ps_2010\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/beta_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/nu2_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/o2h_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/zeta_alpha_ap_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/chi_purine_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/epsilon_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/zeta_alpha_sc_minus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/alpha_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/nu2_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/chi_pyrimidine_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/nu1_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/zeta_alpha_sc_plus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/chi_purine_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/nu1_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/gamma_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/o2h_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/delta_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/epsilon_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/delta_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new/chi_pyrimidine_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/FCC2012_RNA11_based_new\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/beta_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/nu2_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/o2h_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/zeta_alpha_ap_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/chi_purine_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/epsilon_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/zeta_alpha_sc_minus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/alpha_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/nu2_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/chi_pyrimidine_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/nu1_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/zeta_alpha_sc_plus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/chi_purine_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/nu1_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/gamma_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/o2h_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/delta_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/epsilon_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/delta_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new/chi_pyrimidine_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/RNA09_based_2012_new\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/beta_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/nu2_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/zeta_alpha_ap_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/epsilon_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/zeta_alpha_sc_minus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/alpha_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/nu2_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/nu1_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/chi_north_potential_guanosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/zeta_alpha_sc_plus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/chi_north_potential_others.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/nu1_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/gamma_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/delta_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/chi_south_potential_others.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/chi_south_potential_guanosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/epsilon_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010/delta_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_03242010\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/beta_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/nu2_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/zeta_alpha_ap_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/epsilon_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/zeta_alpha_sc_minus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/alpha_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/nu2_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/nu1_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/chi_north_potential_guanosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/zeta_alpha_sc_plus_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/chi_north_potential_others.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/nu1_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/gamma_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/delta_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/chi_south_potential_others.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/chi_south_potential_guanosine.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/epsilon_north_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011/delta_south_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/torsion_potentials/ps_04282011\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_backbone_potential.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_map_lores\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_map_lores/dms\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_map_lores/dms/ade_lores_WC_protected_logstats.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_map_lores/dms\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_map_lores/dms/ade_lores_not_WC_protected_logstats.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_map_lores/dms\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_atom_vdw_min_distances_reformat_MIN_actual_centroid.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/larmord_2.0_10.0_parameters.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/larmord_1.0_nocut_accuracy.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/larmord_2.0_10.0_accuracy.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/larmord_noweight.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/larmord_1.0_neighbor_atoms.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/larmord_noweight.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/larmord_nuchemics_1.0_nocut_accuracy.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/larmord_noweights.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/noweight.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/larmord_1.0_reference_shifts.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_shift_larmor_d/larmord_1.0_nocut_parameters.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_shift_larmor_d\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_pair_dist_potential_renorm_actual_centroid.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_base_pair.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/README -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/Richardson_refit\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/Richardson_refit/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/Richardson_refit\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/Richardson\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/Richardson/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/Richardson\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/Denovo_60\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/Denovo_60/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/Denovo_60\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/1b_4b_0i_7r_6p_2[_0a_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/1b_4b_0i_7r_6p_2[_0a_bonus/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/1b_4b_0i_7r_6p_2[_0a_bonus\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_2.4x_1z_6n_2[\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_2.4x_1z_6n_2[/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_2.4x_1z_6n_2[\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/1z_6n_2[_refit_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/1z_6n_2[_refit_bonus/inv_cov.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/1z_6n_2[_refit_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/1z_6n_2[_refit_bonus/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/1z_6n_2[_refit_bonus\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/1e_5z_4s_#a_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/1e_5z_4s_#a_bonus/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/1e_5z_4s_#a_bonus\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/1z_6n_2[_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/1z_6n_2[_bonus/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/1z_6n_2[_bonus\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_2.7x_1z_6n_2[\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_2.7x_1z_6n_2[/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_2.7x_1z_6n_2[\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_double_1z_6n_2[\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_double_1z_6n_2[/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_double_1z_6n_2[\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/5z_4s_#a_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/5z_4s_#a_bonus/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/5z_4s_#a_bonus\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_triple_1z_6n_2[\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_triple_1z_6n_2[/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suiteness_bonus/test/all_suites_triple_1z_6n_2[\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_stack.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_atom_vdw_min_distances.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_atom_vdw_min_distances_reformat_MIN.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rna_vdw_parameter.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_pair_protein_notburied_actual_centroid.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_map\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_map/dms\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_map/dms/ade_N1_bonded_logstats.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_map/dms\n",
            "copying build/lib/pyrosetta/database/scoring/rna/chem_map/dms/ade_N1_not_bonded_logstats.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/chem_map/dms\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_backbone_potential_actual_centroid.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_pair_protein_buried.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_pair_protein_notburied.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/README -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/tna_suite_potential\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/tna_suite_potential/inv_cov.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/tna_suite_potential\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/tna_suite_potential/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/tna_suite_potential\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/Richardson_refit\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/Richardson_refit/inv_cov.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/Richardson_refit\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/Richardson_refit/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/Richardson_refit\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/Richardson\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/Richardson/inv_cov.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/Richardson\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/Richardson/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/Richardson\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/Richardson_suites.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/Denovo_60\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/Denovo_60/inv_cov.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/Denovo_60\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/Denovo_60/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/Denovo_60\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/test\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/test/1z_6n_2[_refit_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/test/1z_6n_2[_refit_bonus/inv_cov.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/test/1z_6n_2[_refit_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/test/1z_6n_2[_refit_bonus/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/test/1z_6n_2[_refit_bonus\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/test/1z_6n_2[_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/test/1z_6n_2[_bonus/inv_cov.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/test/1z_6n_2[_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/suite_potentials/test/1z_6n_2[_bonus/centers.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna/suite_potentials/test/1z_6n_2[_bonus\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rna_base_pair_xy.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_pair_protein_buried_actual_centroid.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rnp_base_pair_actual_centroid.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "copying build/lib/pyrosetta/database/scoring/rna/rna_base_backbone_xy.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/scoring/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer\n",
            "copying build/lib/pyrosetta/database/rotamer/.gitignore -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer\n",
            "copying build/lib/pyrosetta/database/rotamer/cenrot_dunbrack.lib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer\n",
            "copying build/lib/pyrosetta/database/rotamer/bbdep02.May.sortlib.Dunbrack02.lib.bin -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/met.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/trp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/gln.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/asn.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/gln.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/AcademicLicenseBBDep2010.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/pro.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/asp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/leu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/his.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/his.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/phe.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/glu.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/lys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/his.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/val.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/asn.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/tyr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/phe.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/phe.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/Description.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/tyr.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/ile.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/glu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/Default-5StepDown.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/trp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/ser.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/asp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/glu.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/asn.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/gln.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/tyr.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/arg.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/cys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/thr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/asp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/corrections_conway2016/trp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/corrections_conway2016\n",
            "copying build/lib/pyrosetta/database/rotamer/bbdep02.May.sortlib-correct.12.2010.Dunbrack02.lib.bin -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/met.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/trp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/gln.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/asn.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/gln.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/pro.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/asp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/leu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/his.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/his.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/phe.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/glu.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/lys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/his.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/val.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/asn.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/tyr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/phe.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/phe.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/tyr.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/ile.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/glu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/trp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/ser.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/asp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/glu.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/asn.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/gln.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/Dunbrack10.lib.bin -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/tyr.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/arg.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/cys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/thr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/asp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "copying build/lib/pyrosetta/database/rotamer/beta_nov2016/trp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/beta_nov2016\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/met.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/trp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/gln.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/asn.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/gln.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/AcademicLicenseBBDep2010.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/pro.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/asp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/leu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/his.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/his.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/phe.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/glu.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/.gitignore -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/lys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/his.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/val.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/asn.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/tyr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/phe.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/phe.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/Description.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/tyr.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/ile.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/glu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/Default-5StepDown.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/trp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/ser.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/asp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/glu.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/asn.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/gln.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/Dunbrack10.lib.bin -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/tyr.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/arg.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/cys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/thr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/asp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "copying build/lib/pyrosetta/database/rotamer/ExtendedOpt1-5/trp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ExtendedOpt1-5\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/nvl.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/D_aminoacid_README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/Readme -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/NAP.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/hlu.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/oligourea\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/oligourea/OU3_PRO.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/oligourea\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/oligourea/OU3_VAL.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/oligourea\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/beta_amino_acid\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/beta_amino_acid/B3C_30.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/beta_amino_acid\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/beta_amino_acid/B3N_30.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/beta_amino_acid\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/n_methyl_amino_acid\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/n_methyl_amino_acid/TRP_Nmethylated.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/n_methyl_amino_acid\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/alpha_amino_acid\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/alpha_amino_acid/B51.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/alpha_amino_acid\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/alpha_amino_acid/C40.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/alpha_amino_acid\n",
            "copying build/lib/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/alpha_amino_acid/B53.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/ncaa_rotlibs/ncaa_rotamer_libraries/alpha_amino_acid\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/dna\n",
            "copying build/lib/pyrosetta/database/rotamer/dna/VQ-DNA-256.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/dna\n",
            "copying build/lib/pyrosetta/database/rotamer/dna/VQ-DNA-512.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/dna\n",
            "copying build/lib/pyrosetta/database/rotamer/dna/VQ-DNA-128.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/dna\n",
            "copying build/lib/pyrosetta/database/rotamer/dna/VQ-DNA-64.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/dna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/peptoid_rotlibs\n",
            "copying build/lib/pyrosetta/database/rotamer/peptoid_rotlibs/001.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/peptoid_rotlibs\n",
            "copying build/lib/pyrosetta/database/rotamer/peptoid_rotlibs/601.rotlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/peptoid_rotlibs\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/AcademicLicenseBBDep2010.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/.gitignore -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/argumentsExample.shapovalov.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/Description.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/met.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/trp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/gln.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/asn.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/gln.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/AcademicLicenseBBDep2010.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/pro.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/asp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/leu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/his.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/his.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/phe.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/glu.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/lys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/his.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/val.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/asn.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/tyr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/phe.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/phe.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/Description.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/tyr.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/ile.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/glu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/trp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/ser.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/asp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/glu.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/asn.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/gln.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/Taken.From.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/tyr.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/arg.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/cys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/thr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/asp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25/trp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_25-25-25\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/met.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/trp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/gln.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/asn.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/gln.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/AcademicLicenseBBDep2010.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/pro.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/asp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/leu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/his.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/his.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/phe.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/glu.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/lys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/his.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/val.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/asn.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/tyr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/phe.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/phe.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/Description.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/tyr.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/ile.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/glu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/trp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/ser.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/asp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/glu.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/asn.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/gln.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/Taken.From.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/tyr.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/arg.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/cys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/thr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/asp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10/trp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_10-10-10\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/Suggested.Default.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/met.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/trp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/gln.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/asn.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/gln.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/AcademicLicenseBBDep2010.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/pro.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/asp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/leu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/his.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/his.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/phe.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/glu.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/lys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/his.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/val.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/asn.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/tyr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/phe.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/phe.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/Description.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/tyr.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/ile.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/glu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/trp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/ser.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/asp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/glu.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/asn.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/gln.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/Taken.From.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/tyr.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/arg.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/cys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/thr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/asp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2/trp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_2-2-2\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/met.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/trp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/gln.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/asn.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/gln.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/AcademicLicenseBBDep2010.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/pro.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/asp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/leu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/his.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/his.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/phe.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/glu.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/lys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/his.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/val.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/asn.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/tyr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/phe.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/phe.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/Description.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/tyr.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/ile.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/glu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/trp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/ser.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/asp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/glu.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/asn.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/gln.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/Taken.From.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/tyr.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/arg.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/cys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/thr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/asp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20/trp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_20-20-20\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/Taken.From.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/met.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/trp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/gln.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/asn.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/gln.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/AcademicLicenseBBDep2010.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/pro.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/asp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/leu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/his.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/his.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/phe.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/glu.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/lys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/his.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/val.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/asn.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/tyr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/phe.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/phe.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/Description.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/tyr.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/ile.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/glu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/trp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/ser.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/asp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/glu.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/asn.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/gln.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/Dunbrack10.lib.bin -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/Taken.From.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/tyr.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/arg.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/cys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/thr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/asp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/trp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/met.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/trp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/gln.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/asn.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/gln.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/AcademicLicenseBBDep2010.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/pro.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/asp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/leu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/his.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/his.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/phe.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/glu.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/lys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/his.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/val.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/asn.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/tyr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/phe.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/phe.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/Description.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/tyr.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/ile.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/glu.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/Suggested.Default.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/trp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/ser.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/asp.bbdep.densities.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/glu.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/asn.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/gln.bbind.chi3.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/Taken.From.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/tyr.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/arg.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/cys.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/thr.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/asp.bbdep.rotamers.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5/trp.bbind.chi2.Definitions.lib.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer/shapovalov/StpDwn_5-5-5\n",
            "copying build/lib/pyrosetta/database/rotamer/bbdep02.May.sortlib-correct.12.2010 -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer\n",
            "copying build/lib/pyrosetta/database/rotamer/bbdep02.May.sortlib -> build/bdist.linux-x86_64/egg/pyrosetta/database/rotamer\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/glycosyltransferases\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/glycosyltransferases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/glycosyltransferases/h_sapiens/OGT -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/glycosyltransferases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/glycosyltransferases/h_sapiens/generic -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/glycosyltransferases/h_sapiens\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/glycosyltransferases/c_jejuni\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/glycosyltransferases/c_jejuni/generic -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/glycosyltransferases/c_jejuni\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/glycosyltransferases/c_jejuni/PglB -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/glycosyltransferases/c_jejuni\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PhK -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PKCeta -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/AMPK -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PKCepsilon -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CDK2 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PKCzeta -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/DMPK-E -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/DCK1-b2 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/RSK1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CK2 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PKCalpha -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/SGK1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PKCdelta -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CK1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CK1-beta-catenin -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PKCmu -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/NIMA -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PKA -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PKB -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/SLK1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CaMK4 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PKCgamma -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CHK1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/MAPKAPK2 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PAK -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/ZIPK -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/generic -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/MSK2 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/MSK1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CDK1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/PKCbeta -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/MAPK -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CaMK1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CDK5 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CDK4 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CLK1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/GSK3B -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/CaMK2 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/DYRK1A -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/Pim1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/kinases/h_sapiens/Akt -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/kinases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/README -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/DNA_methyltransferases\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/DNA_methyltransferases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/DNA_methyltransferases/h_sapiens/generic -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/DNA_methyltransferases/h_sapiens\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/N-terminal_acetyltransferases\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/N-terminal_acetyltransferases/s_solfataricus\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/N-terminal_acetyltransferases/s_solfataricus/Ard1 -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/N-terminal_acetyltransferases/s_solfataricus\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/N-terminal_acetyltransferases/h_sapiens\n",
            "copying build/lib/pyrosetta/database/virtual_enzymes/N-terminal_acetyltransferases/h_sapiens/generic -> build/bdist.linux-x86_64/egg/pyrosetta/database/virtual_enzymes/N-terminal_acetyltransferases/h_sapiens\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/membrane\n",
            "copying build/lib/pyrosetta/database/membrane/implicit_lipid_parameters.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/membrane\n",
            "copying build/lib/pyrosetta/database/membrane/memb_fa_params_2019.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/membrane\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/splice\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/splice/antibodies\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/splice/antibodies/db\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/splice/antibodies/db/2BRR\n",
            "copying build/lib/pyrosetta/database/protocol_data/splice/antibodies/db/2BRR/H3.db -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/splice/antibodies/db/2BRR\n",
            "copying build/lib/pyrosetta/database/protocol_data/splice/antibodies/db/2BRR/L3.db -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/splice/antibodies/db/2BRR\n",
            "copying build/lib/pyrosetta/database/protocol_data/splice/antibodies/db/2BRR/H1_H2.db -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/splice/antibodies/db/2BRR\n",
            "copying build/lib/pyrosetta/database/protocol_data/splice/antibodies/db/2BRR/L1_L2.db -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/splice/antibodies/db/2BRR\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/denovo_design\n",
            "copying build/lib/pyrosetta/database/protocol_data/denovo_design/aa_abego_frequencies.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/denovo_design\n",
            "copying build/lib/pyrosetta/database/protocol_data/denovo_design/preproline_normalized.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/denovo_design\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/SDE\n",
            "copying build/lib/pyrosetta/database/protocol_data/SDE/GUIP.matrix -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/SDE\n",
            "copying build/lib/pyrosetta/database/protocol_data/SDE/spc216.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/SDE\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/Z6P_2_zigzag.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/a3b.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/A_form_RNA_helix.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/Z6P_zigzag.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/neutral_beta_strand.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/R_PPII.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/L_alpha_helix.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/alpha_helix.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/L_PPI.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/PPII.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/README.TXT -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/alpha_helix_100.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/14_helix.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/PPI.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/three_10.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/collagen.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/beta_strand.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "copying build/lib/pyrosetta/database/protocol_data/crick_parameters/daa_beta_strand.crick_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/crick_parameters\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/generalizedKIC\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/generalizedKIC/bin_params\n",
            "copying build/lib/pyrosetta/database/protocol_data/generalizedKIC/bin_params/ABEGO.bin_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/generalizedKIC/bin_params\n",
            "copying build/lib/pyrosetta/database/protocol_data/generalizedKIC/bin_params/ABBA.bin_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/generalizedKIC/bin_params\n",
            "copying build/lib/pyrosetta/database/protocol_data/generalizedKIC/bin_params/PRO_DPRO.bin_params -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/generalizedKIC/bin_params\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/saved_model.pb -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/variables\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/variables/variables.data-00000-of-00001 -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/variables\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/variables/checkpoint -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/variables\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/variables/variables.index -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/variables\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/assets\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/assets/saved_model.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/debug/assets\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/saved_model.pb -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/variables\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/variables/variables.data-00000-of-00001 -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/variables\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/variables/checkpoint -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/variables\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/variables/variables.index -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/variables\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/assets\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/assets/saved_model.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation1/assets\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/saved_model.pb -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/variables\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/variables/variables.data-00000-of-00001 -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/variables\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/variables/checkpoint -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/variables\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/variables/variables.index -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/variables\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/assets\n",
            "copying build/lib/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/assets/saved_model.json -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/tensorflow_graphs/smart_annealer/generation2/assets\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/antibody\n",
            "copying build/lib/pyrosetta/database/protocol_data/antibody/comparisons.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/antibody\n",
            "copying build/lib/pyrosetta/database/protocol_data/antibody/cdr_regex.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/antibody\n",
            "copying build/lib/pyrosetta/database/protocol_data/antibody/outlier_list -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/antibody\n",
            "copying build/lib/pyrosetta/database/protocol_data/antibody/list_bfactor50 -> build/bdist.linux-x86_64/egg/pyrosetta/database/protocol_data/antibody\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C26_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C94_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C57_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C25_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C82_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C71_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C56_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C70_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C92_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C7_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C22_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C31_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C18_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C21_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C95_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C86_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C23_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C36_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C19_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C72_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C78_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C6_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C30_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C59_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C69_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C81_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C3_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C29_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C52_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C5_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C27_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C87_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C61_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C28_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C15_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C88_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C10_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C47_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C48_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C79_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C37_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C8_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C68_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C49_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C20_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C16_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C93_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C50_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C63_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C24_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C35_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C53_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C34_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C46_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C80_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C75_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C76_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C99_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C13_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C32_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C84_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C96_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C54_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C98_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C65_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C91_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C12_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C41_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C77_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C4_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C11_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C83_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C43_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C38_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C73_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C2_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C89_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C39_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C67_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C51_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C58_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C14_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C66_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C33_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C97_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C62_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C60_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C9_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C44_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C64_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C74_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C90_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C42_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C85_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C17_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C45_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C40_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "copying build/lib/pyrosetta/database/symmetry/cyclic/C55_Z.sym -> build/bdist.linux-x86_64/egg/pyrosetta/database/symmetry/cyclic\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling\n",
            "copying build/lib/pyrosetta/database/sampling/disulfide_jump_database_wip.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/branch_angle\n",
            "copying build/lib/pyrosetta/database/sampling/branch_angle/branch_angle_1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/branch_angle\n",
            "copying build/lib/pyrosetta/database/sampling/branch_angle/branch_angle_2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/branch_angle\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/ELE.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/HEH.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/LLL.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/EEH.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/LLE.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/HLL.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/HHL.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/HLE.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/LEH.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/ELH.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/LEE.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/ELL.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/HEL.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/LHH.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/EHH.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/HHH.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/HEE.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/EEL.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/HHE.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/EEE.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/LLH.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/LEL.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/ss_fragfiles/HLH.fragfile -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/ss_fragfiles\n",
            "copying build/lib/pyrosetta/database/sampling/vall.dat.2001-02-02.torsions.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling\n",
            "copying build/lib/pyrosetta/database/sampling/SASA-masks.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling\n",
            "copying build/lib/pyrosetta/database/sampling/small.vall.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/cen_relax\n",
            "copying build/lib/pyrosetta/database/sampling/cen_relax/default_relax_parameters.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/cen_relax\n",
            "copying build/lib/pyrosetta/database/sampling/cen_relax/format_notes.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/cen_relax\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/notes.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/ExpandSet.cpp -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u12083.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u158763.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u932999.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u7111.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u427333.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u5879.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u815.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u2947.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u40003.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u898517.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u367113.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u4701.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u309.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u493799.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u28329.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u735777.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u110143.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u25039.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u676103.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u118647.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u330023.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u248571.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u49397.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u648549.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u798587.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u32081.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u233569.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u31793.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u568499.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u148395.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u518377.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u44709.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u448437.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u1045817.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u830491.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u21863.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u566819.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u12251.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u10305.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c600v.quat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u312831.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u101477.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u128249.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u527.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u347617.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u157.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c600vec.quat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u8649.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u3733.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u2867.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u519.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u1153.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u295333.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u93701.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u83.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u765729.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u65985.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u169757.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u1.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u137809.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u706351.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c600vc.quat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u970447.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48n309.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u1006449.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u386211.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u72521.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u1201.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u279565.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u27.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u619981.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u1083955.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u181.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u9.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u263339.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u54799.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u35851.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u181909.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u193767.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u542361.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u79099.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48n9.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u2219.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u86451.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48n527.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u14251.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u220121.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u4749.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u207023.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u16533.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u407099.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u60279.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u101917.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u865149.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u19181.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u1641.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u593755.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "copying build/lib/pyrosetta/database/sampling/orientations/orientgridall/data/c48u471503.grid.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/orientations/orientgridall/data\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/waterrots\n",
            "copying build/lib/pyrosetta/database/sampling/waterrots/water.rots -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/waterrots\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/rosettacon2018.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/MonomerRelax2019.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/no_cst_ramping.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/rosettacon2018.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.beta_nov16.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.talaris2014.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/legacy.beta_nov16.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.ref2015.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/no_cst_ramping.beta_nov16.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.ref2015.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.score12.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.talaris2014.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/index.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/MonomerRelax2019.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/InterfaceDesign2019.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/MonomerDesign2019.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.talaris2013.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/legacy.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/README.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/InterfaceDesign2019.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/MonomerDesign2019.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.score12.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/legacy.beta_nov16.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/legacy.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.beta_nov16.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/PolarDesign2019.talaris2013.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/InterfaceRelax2019.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/no_cst_ramping.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/no_cst_ramping.beta_nov16.dualspace.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/relax_scripts/InterfaceRelax2019.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/relax_scripts\n",
            "copying build/lib/pyrosetta/database/sampling/filtered.vall.dat.2006-05-05.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/db_structure_loading.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/ab_db_for_tests.db -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_center_dihedrals.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/design\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/design/cluster_framework_mutations.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/design\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/design/default_instructions.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/design\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/numbering_schemes\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/numbering_schemes/tcr_cdr_definitions.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/numbering_schemes\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/numbering_schemes/numbering_scheme_definitions_original.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/numbering_schemes\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/numbering_schemes/numbering_scheme_definitions.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/numbering_schemes\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/numbering_schemes/adding_schemes_and_definitions.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/numbering_schemes\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/numbering_schemes/cdr_definitions.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/numbering_schemes\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/Fragments\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/Fragments/H3_CTERM -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/Fragments\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-5-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-11-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-12-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-14-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-18-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-8.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-10-cis8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-7-cis4-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-5-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-11.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-10-7.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-13-cis8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-17-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-9-cis7-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-9-cis7-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-19-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-7-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-16-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L2-8-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L2-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-10-8.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-cis9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-9-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-10-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/MEAN_SD.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-9-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L2-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-14-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-26-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-17-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-14-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-10-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-10-cis7,8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-11-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-10-6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-10-9.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L2-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-9.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L2-8-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-18-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-11-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-12-cis9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-15-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-8-cis6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-7-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L3-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L2-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-22-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-10-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-11-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-13-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-14-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-10-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-20-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-7.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H2-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L1-15-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H3-13-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/L2-8-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal/H1-13-10.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L3-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-11-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H2-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L3-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-14-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-13-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L3-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-13-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-21-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-18-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-13-8.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-5-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-17-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L3-9-cis7-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-19-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-7-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-16-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L2-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-13-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H2-10-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L3-9-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L2-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-13-6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-24-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-14-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-17-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-14-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H2-10-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H2-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H2-10-6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-13-9.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L3-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-18-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-11-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-15-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L3-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H2-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L3-8-cis6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L3-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-7-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L3-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H2-10-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-11-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-13-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L1-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-20-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H1-13-7.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H2-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/H3-13-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original/L2-8-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/original\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-5-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-11-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-12-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-14-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-21-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-18-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-8.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-10-cis8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-7-cis4-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-5-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-11.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-10-7.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-13-cis8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-17-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-9-cis7-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-9-cis7-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-19-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-7-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-16-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L2-8-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L2-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-10-8.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-9-cis6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-cis9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-9-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-10-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/MEAN_SD.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-9-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L2-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-24-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-14-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-26-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-17-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-14-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-10-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-10-cis7,8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-11-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-10-6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-10-9.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L2-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-9.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L2-8-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-18-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-11-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-12-cis9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-15-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-8-cis6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-7-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L3-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L2-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-22-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-10-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-11-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-13-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-14-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-10-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-20-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-7.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H2-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L1-15-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H3-13-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/L2-8-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means/H1-13-10.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true_use_means\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-5-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-11-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-12-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-14-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-18-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-8.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-10-cis8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-7-cis4-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-5-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-11.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-10-7.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-13-cis8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-17-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-9-cis7-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-9-cis7-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-19-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-7-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-16-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L2-8-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L2-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-10-8.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-cis9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-9-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-10-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/MEAN_SD.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-9-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L2-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-14-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-26-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-17-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-14-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-10-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-10-cis7,8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-11-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-10-6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-10-9.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L2-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-9.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L2-8-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-18-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-11-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-12-cis9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-15-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-8-cis6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-7-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L3-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L2-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-22-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-10-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-11-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-13-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-14-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-10-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-20-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-7.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H2-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L1-15-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H3-13-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/L2-8-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means/H1-13-10.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_false_liberal_use_means\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/notes_on_data.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-5-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-11-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-12-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-14-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-21-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-18-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-8.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-10-cis8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-7-cis4-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-5-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-11.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-10-7.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-13-cis8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-17-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-9-cis7-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-9-cis7-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-19-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-7-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-16-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L2-8-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L2-8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-10-8.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-9-cis6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-cis9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-9-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-10-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/MEAN_SD.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-16-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-9-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L2-8-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-24-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-14-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-10-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-26-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-14-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-17-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-14-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-10-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-13-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-10-cis7,8-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-11-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-10-6.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-10-9.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L2-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-9.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L2-8-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-18-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-13-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-11-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-12-cis9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-15-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-11-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-8-cis6-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-12-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-9-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-7-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L3-9-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L2-12-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-22-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-10-5.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-11-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-13-cis7-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-14-4.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-10-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-20-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-7.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H2-10-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L1-15-2.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-15-1.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H3-13-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/L2-8-3.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true/H1-13-10.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies/cluster_based_constraints/CircularHarmonic/outliers_true\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/antibody_database_rosetta_design_north_paper.db -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/antibody_database_notes.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies\n",
            "copying build/lib/pyrosetta/database/sampling/antibodies/cluster_center_dihedrals_info.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/antibodies\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_132.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_672.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_282.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_4092_neighbors.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_252.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_632.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_612.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_1032.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_492.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_92.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/README -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_122.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_672_neighbors.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_12.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_hierarchy.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_8192.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_522.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_362.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_162.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_72.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_192.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_17282.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_642.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_272.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_572.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_1812_neighbors.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_32.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_372.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_4092.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_432.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_392.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_482.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_1032_neighbors.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_8192_neighbors.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_212.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_1812.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_17282_neighbors.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/spheres/sphere_312.dat.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/spheres\n",
            "copying build/lib/pyrosetta/database/sampling/vall.jul19.2011.torsions.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_PP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_W.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_V.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_T.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_Y.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_dcP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_P.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_F.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_N.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_YP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_QP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_M.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_K.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_HP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_R.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_NP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_KP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_LP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_H.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_N.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_EP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_V.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_R.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_VP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_R.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_A.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_IP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_dc.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_TP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_L.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_C.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_FP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_CP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_E.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_dcP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_NP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_K.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_dc.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_H.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_LP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_P.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_L.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_EP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_C.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_W.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_DP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_RP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_M.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_H.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_P.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_GP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_PP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_SP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_I.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_E.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_HP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_G.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_DP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_MP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_PP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_Q.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_CP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_QP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_NP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_TP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_IP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_Y.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_A.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_S.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_WP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_L.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_dc.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_T.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_GP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_CP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_HP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_SP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_VP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_MP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_D.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_KP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_VP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_IP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_Q.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_K.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_AP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_D.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_YP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_LP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_F.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_V.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_N.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_FP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_KP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_T.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_Q.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_E.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_FP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_G.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_W.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_TP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_I.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_RP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_M.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_SP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_A.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_AP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_DP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_F.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_C.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_WP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_WP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_MP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_I.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_EP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_G.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_AP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_YP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_S.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/E_D.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_GP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_QP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_S.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_Y.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/H_RP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "copying build/lib/pyrosetta/database/sampling/fragpicker_rama_tables/L_dcP.counts.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/fragpicker_rama_tables\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/RNA11_full.torsions -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_ga.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_au.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_gg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ca_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_cu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_ua.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ga_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_ac.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gc_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uc_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/au_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/uu_ag.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cg_cg.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cu_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_uc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ug_ug.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ag_gu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ac_ca.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gu_aa.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/aa_uu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/gg_gc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/ua_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/standard/cc_cc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/standard\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_anu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ug_gna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gc_cnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uu_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ac_gnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ua_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ac_cng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gu_ang.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_gnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uu_anu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cu_cng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cc_cna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gu_anu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ac_gna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cu_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_cnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ca_ang.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cc_gnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ca_unc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_ung.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cc_gna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_ung.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_cna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uc_gnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ac_cnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gg_gnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cu_una.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ug_cna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uu_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_gna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gc_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_unu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ug_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_cnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_gna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_gnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_ang.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_anc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ua_gna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ug_una.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cg_cng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_una.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gu_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/au_ung.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cg_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ca_cng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cc_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uc_cng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ac_gnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ac_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cc_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ua_una.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_cna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cu_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_unc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uc_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gg_gna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gg_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_unu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uu_ang.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cc_una.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_ung.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ug_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ua_unc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ug_unu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_unc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/au_anu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uc_anc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gg_cna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ca_una.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gc_anc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gg_unc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/au_ang.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/au_cna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ug_ung.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_unu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cu_unu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cg_cna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ua_ang.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ca_ung.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uc_unc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gg_anc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gu_unc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uc_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ac_anu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_gnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ca_gnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ua_cnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_gna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cg_ung.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gg_cnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ua_unu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cc_ang.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_anc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gu_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ca_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_una.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gc_cna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cu_ang.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gg_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uc_cna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_gng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_anu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/au_anc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gu_gnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cu_anc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cg_una.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_anu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_una.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cc_cng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ug_ang.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_unc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gu_anc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cg_ang.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_cna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ag_cng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cg_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/ga_cnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/cc_gnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/uc_gna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gc_cng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gc_gnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/aa_ana.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt/gc_gnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_1nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/README -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cc_gnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/au_annnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/aa_unnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ca_annng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/aa_unnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gc_unnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gg_unnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ua_annna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gu_unnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cg_unnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ca_gnnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ac_gnnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ga_unnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/aa_cnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cc_annng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ag_cnnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gg_cnnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gg_annna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ac_gnnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ac_gnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ag_cnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cg_gnnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/aa_cnnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ua_unnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gu_annnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ac_gnnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gc_unnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/uu_unnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ga_cnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ua_annng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ca_unnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cc_gnnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gg_annnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/au_cnnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gu_unnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gu_gnnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/uu_annna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ua_gnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ug_cnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cu_annnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ag_gnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cg_cnnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cc_gnnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cg_cnnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ca_gnnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ga_unnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ca_cnnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/uu_annng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cg_cnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ca_cnnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gg_gnnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/uu_cnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gc_gnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/uu_unnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/uc_gnnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ga_annna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cc_annna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ga_annnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ca_unnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ca_unnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cu_annnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ug_gnnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gg_cnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ug_annna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gc_gnnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/au_unnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gu_gnnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/aa_gnnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/uc_gnnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gc_gnnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ga_gnnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/gg_gnnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ga_gnnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/au_gnnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ug_unnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cu_unnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/ag_cnnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt/cu_annng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_3nt\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/uu_gnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/au_annu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/uc_unng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ac_unng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ac_gnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ca_gnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ca_gnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/uc_unna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ga_gnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cg_anng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/uc_gnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ac_gnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ag_unnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ug_cnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cc_gnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gc_gnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/uu_gnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/uu_unng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cg_unng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cu_annc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gu_unna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gc_gnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/uc_anna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ug_cnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ug_cnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ga_unna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ac_gnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/aa_unnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ca_cnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gg_cnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gg_annc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ag_unna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ac_annu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ga_anna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ua_anna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ac_anng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gg_cnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cu_gnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/uc_anng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cu_unng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cc_gnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ug_gnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ga_cnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cc_anna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gc_unnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cc_gnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ua_gnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/uc_gnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ac_gnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gg_cnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ag_cnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gc_annc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gg_annu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gu_unng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ca_unng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cg_cnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gg_unnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ca_anng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ug_gnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gc_anng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gu_anng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cu_anng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cc_annc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ua_cnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ua_unng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ag_anna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/au_anna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cg_cnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gc_gnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gc_unnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cg_cnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gu_annc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ga_cnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ga_unnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ag_cnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ag_cnnc.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gg_cnnu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ug_unna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gu_gnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ag_cnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/cu_cnng.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/uu_anna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/aa_cnna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/ca_unna.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt/gu_annu.out.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/general/bulge_2nt\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ug_ua.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/uu_ag.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/uu_aa.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/gu_ac.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ua_ug.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/gu_gc.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/au_gu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/gu_gu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ga_uu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/gc_gc.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ag_uu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/uu_ga.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/aa_uu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/gc_gu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ug_ca.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/gg_uu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/uc_gg.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/cg_ug.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/cg_cg.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ug_cg.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ga_uc.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ag_cu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/uu_gg.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/uc_ga.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ac_gu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/gg_cu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/gg_cc.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ca_ug.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/gu_au.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/cu_gg.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/cc_gg.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ug_ug.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/cu_ag.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/au_au.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/ua_ua.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard/gg_uc.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/base_pair_steps/canonical/standard\n",
            "copying build/lib/pyrosetta/database/sampling/rna/cg_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/RNA11_filter_all.jump -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/ug.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/cu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/ac.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/au.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/gu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/gc.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/gg.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/uu.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/uc.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/ga.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/ag.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/ua.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/ca.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/aa.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/cg.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/precomputed/cc.out -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/precomputed\n",
            "copying build/lib/pyrosetta/database/sampling/rna/RICHARDSON_RNA09.torsions -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/RNA11_full.jump -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/ua_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/RNA11_filter_pucker.jump -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/au_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/RNA18_HUB_2.154_2.5.torsions -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/RNA18_HUB_2.154_2.5.jump -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/1jj2_coarse_jumps.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/srl\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/srl/GUA_GA_430d.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/srl\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/dinucleotides/cc.out.3.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/dinucleotides/uu.out.5.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/dinucleotides/uu.out.2.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/dinucleotides/uu.out.3.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/dinucleotides/cc.out.4.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/dinucleotides/uu.out.1.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/dinucleotides/cc.out.1.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/dinucleotides/uu.out.4.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/dinucleotides/cc.out.2.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/dinucleotides/cc.out.5.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/dinucleotides\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/submotifs.txt -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/uturns\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/uturns/uturn_t_loop_3l0u_RNA.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/uturns\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/uturns/uturn_t_loop_3l0u_RNA_thread_to_uaa.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/uturns\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/base_pairs\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/base_pairs/ua_handle_1q9a.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/base_pairs\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/base_pairs/cg_standard.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/base_pairs\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/base_pairs/gg_1raw_RNA.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/base_pairs\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/base_pairs/gg_2grb_RNA.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/base_pairs\n",
            "copying build/lib/pyrosetta/database/sampling/rna/submotif/base_pairs/G_GAI.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna/submotif/base_pairs\n",
            "copying build/lib/pyrosetta/database/sampling/rna/RNA11_filter_all.torsions -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/RNA11_filter_pucker.torsions -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/1jj2.torsions -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/1jj2_RNA_jump_library.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/rna/gc_std.pdb -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling/rna\n",
            "copying build/lib/pyrosetta/database/sampling/SASA-angles.dat -> build/bdist.linux-x86_64/egg/pyrosetta/database/sampling\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/README -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/homology.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/N.level1.WL1.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/fitting.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/HA.level1.WL1.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/weight.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/CB.level1.WL1.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/C.level1.WL2.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/HA.level1.WI.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/CA.level1.WL1.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/rcadj.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/CA.level1.WL2.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/C.level1.WI.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/C.level1.WL1.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/N.level1.WI.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/BLOSUM62.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/HA.level1.WL2.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/CA.level1.WI.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/CB.level1.WL2.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/HN.level1.WL1.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/HN.level1.WI.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/CB.level1.WI.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/randcoil.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/rcnext.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/Q..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/M..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/I..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/T..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/H..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/E..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/K..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/A..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/S..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/Y..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/C..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/R..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/P..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/D..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/W..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/V..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/L..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/N..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB/F..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CB\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/Q..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/M..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/I..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/T..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/G..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/H..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/E..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/K..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/A..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/S..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/Y..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/C..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/R..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/P..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/D..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/W..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/V..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/L..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/N..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA/F..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HA\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/Q..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/M..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/I..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/T..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/G..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/H..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/E..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/K..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/A..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/S..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/Y..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/C..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/R..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/D..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/W..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/V..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/L..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/N..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN/F..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/HN\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/Q..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/M..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/I..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/T..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/G..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/H..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/E..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/K..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/A..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/S..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/Y..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/C..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/R..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/P..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/D..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/W..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/V..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/L..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/N..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/C/F..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/C\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/Q..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/M..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/I..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/T..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/G..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/H..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/E..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/K..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/A..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/S..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/Y..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/C..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/R..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/P..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/D..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/W..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/V..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/L..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/N..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA/F..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/CA\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/Q..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/M..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/I..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/T..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/G..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/H..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/E..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/K..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/A..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/S..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/Y..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/C..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/R..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/P..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/D..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/W..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/V..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/L..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/N..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/errorSurface/N/F..A450.S5.RMS.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab/errorSurface/N\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/HN.level1.WL2.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/rcprev.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/N.level1.WL2.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "copying build/lib/pyrosetta/database/external/SPARTA+/tab/sslimit.tab -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/SPARTA+/tab\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/svm_models\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/svm_models/ss_prediction\n",
            "copying build/lib/pyrosetta/database/external/svm_models/ss_prediction/ABEGO_rd1_model -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/svm_models/ss_prediction\n",
            "copying build/lib/pyrosetta/database/external/svm_models/ss_prediction/ABEGO_rd2_model -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/svm_models/ss_prediction\n",
            "copying build/lib/pyrosetta/database/external/svm_models/ss_prediction/README -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/svm_models/ss_prediction\n",
            "copying build/lib/pyrosetta/database/external/svm_models/ss_prediction/HLE_rd2_model -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/svm_models/ss_prediction\n",
            "copying build/lib/pyrosetta/database/external/svm_models/ss_prediction/HLE_rd1_model -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/svm_models/ss_prediction\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/external/svm_models/helix_cap\n",
            "copying build/lib/pyrosetta/database/external/svm_models/helix_cap/Ncap_svm_model -> build/bdist.linux-x86_64/egg/pyrosetta/database/external/svm_models/helix_cap\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sequence\n",
            "copying build/lib/pyrosetta/database/sequence/resinfo2.db -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB50101.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB11501.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB10802.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/H-2-IAd.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB30101.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB40101.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DPA10201-DPB10101.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DQA10301-DQB10302.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DPA10201-DPB10501.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB10101.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DQA10102-DQB10602.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB10405.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DQA10401-DQB10402.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB10401.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB10301.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB10701.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB11101.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB10901.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DPA10301-DPB10402.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DPA10103-DPB10201.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DPA101-DPB10401.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB11302.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DRB10404.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DPB10301-DPB10401.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DQA10101-DQB10501.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/H-2-IAb.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DQA10501-DQB10201.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_rank_svm_scores/HLA-DQA10501-DQB10301.libsvm.test.out.sort.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_rank_svm_scores\n",
            "copying build/lib/pyrosetta/database/sequence/resinfo.db -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DPA10103-DPB10201.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DQA10101-DQB10501.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DQA10501-DQB10201.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/README -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB10901.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB10401.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB50101.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/H-2-IAd.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/H-2-IAb.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB10701.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB40101.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DQA10102-DQB10602.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB11302.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB11101.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB10101.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DQA10301-DQB10302.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB30101.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB10404.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB10301.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB10405.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB11501.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/tcell_ep_9mers/HLA-DRB10802.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/tcell_ep_9mers\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB10802_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB50101_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB10401_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DQA10401-DQB10402_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB30101_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DPB10301-DPB10401_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DPA10201-DPB10501_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/H-2-IAd_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB40101_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DQA10101-DQB10501_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB10301_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB11302_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DPA101-DPB10401_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DPA10103-DPB10201_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB10404_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DQA10501-DQB10301_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DQA10501-DQB10201_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DPA10201-DPB10101_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB11101_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DQA10301-DQB10302_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB10405_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/H-2-IAb_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB10101_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB10701_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DPA10301-DPB10402_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB11501_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DRB10901_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_svms/HLA-DQA10102-DQB10602_nooverlap.libsvm.dat.noscale.nu0.5.min_mse.model -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_svms\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DPA101-DPB10401_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB10701_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB50101_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB11101_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB10901_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB10301_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/H-2-IAb_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DQA10401-DQB10402_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DQA10501-DQB10301_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DQA10301-DQB10302_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB10101_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB10404_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB11302_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DQA10101-DQB10501_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB40101_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB10401_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB10405_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DPA10201-DPB10501_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB30101_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DPB10301-DPB10401_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/H-2-IAd_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DPA10301-DPB10402_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB11501_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DPA10201-DPB10101_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DPA10103-DPB10201_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DRB10802_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DQA10102-DQB10602_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "copying build/lib/pyrosetta/database/sequence/mhc_pssms/HLA-DQA10501-DQB10201_nooverlap.9mer.norm.pssm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/mhc_pssms\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/substitution_matrix\n",
            "copying build/lib/pyrosetta/database/sequence/substitution_matrix/PAM30 -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/substitution_matrix\n",
            "copying build/lib/pyrosetta/database/sequence/substitution_matrix/IDENTITY -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/substitution_matrix\n",
            "copying build/lib/pyrosetta/database/sequence/substitution_matrix/BLOSUM62 -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/substitution_matrix\n",
            "copying build/lib/pyrosetta/database/sequence/substitution_matrix/BLOSUM62_aa -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/substitution_matrix\n",
            "copying build/lib/pyrosetta/database/sequence/substitution_matrix/pepspec_background.binding-prot-0.1.pwm -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/substitution_matrix\n",
            "copying build/lib/pyrosetta/database/sequence/substitution_matrix/MATCH -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/substitution_matrix\n",
            "copying build/lib/pyrosetta/database/sequence/substitution_matrix/BLOSUM62.prob.rescale -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/substitution_matrix\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/genome_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/genome_9mers/Homo_sapiens.GRCh37.66.pep.all.9mers.sort.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/genome_9mers\n",
            "copying build/lib/pyrosetta/database/sequence/genome_9mers/Mus_musculus.NCBIM37.66.pep.all.9mers.sort.refscore.gz -> build/bdist.linux-x86_64/egg/pyrosetta/database/sequence/genome_9mers\n",
            "copying build/lib/pyrosetta/rosetta.so -> build/bdist.linux-x86_64/egg/pyrosetta\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "copying build/lib/pyrosetta/toolbox/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "copying build/lib/pyrosetta/toolbox/cleaning.py -> build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "copying build/lib/pyrosetta/toolbox/load_ligand.py -> build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "copying build/lib/pyrosetta/toolbox/py_jobdistributor.py -> build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "copying build/lib/pyrosetta/toolbox/atom_pair_energy.py -> build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "copying build/lib/pyrosetta/toolbox/numpy_utils.py -> build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "copying build/lib/pyrosetta/toolbox/extract_coords_pose.py -> build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "copying build/lib/pyrosetta/toolbox/mutants.py -> build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "copying build/lib/pyrosetta/toolbox/rcsb.py -> build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "copying build/lib/pyrosetta/toolbox/generate_resfile.py -> build/bdist.linux-x86_64/egg/pyrosetta/toolbox\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/utility\n",
            "copying build/lib/pyrosetta/utility/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/utility\n",
            "copying build/lib/pyrosetta/utility/array.py -> build/bdist.linux-x86_64/egg/pyrosetta/utility\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/io\n",
            "copying build/lib/pyrosetta/io/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/io\n",
            "copying build/lib/pyrosetta/io/silent_file_map.py -> build/bdist.linux-x86_64/egg/pyrosetta/io\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/tests\n",
            "copying build/lib/pyrosetta/tests/__main__.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests\n",
            "copying build/lib/pyrosetta/tests/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/tests/numeric\n",
            "copying build/lib/pyrosetta/tests/numeric/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/numeric\n",
            "copying build/lib/pyrosetta/tests/numeric/test_alignment.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/numeric\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed\n",
            "copying build/lib/pyrosetta/tests/distributed/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed\n",
            "copying build/lib/pyrosetta/tests/distributed/test_smoke.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed\n",
            "copying build/lib/pyrosetta/tests/distributed/test_gil.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed\n",
            "copying build/lib/pyrosetta/tests/distributed/test_dask_worker.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed\n",
            "copying build/lib/pyrosetta/tests/distributed/test_dask.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed\n",
            "copying build/lib/pyrosetta/tests/distributed/test_concurrency.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed\n",
            "copying build/lib/pyrosetta/tests/distributed/test_viewer.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols\n",
            "copying build/lib/pyrosetta/tests/protocols/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols/indexed_structure_store\n",
            "copying build/lib/pyrosetta/tests/protocols/indexed_structure_store/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols/indexed_structure_store\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols/indexed_structure_store/data\n",
            "copying build/lib/pyrosetta/tests/protocols/indexed_structure_store/data/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols/indexed_structure_store/data\n",
            "copying build/lib/pyrosetta/tests/protocols/indexed_structure_store/test_search.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols/indexed_structure_store\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/tests/bindings\n",
            "copying build/lib/pyrosetta/tests/bindings/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/bindings\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/tests/bindings/core\n",
            "copying build/lib/pyrosetta/tests/bindings/core/test_pose.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/bindings/core\n",
            "copying build/lib/pyrosetta/tests/bindings/core/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/tests/bindings/core\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/protocols\n",
            "copying build/lib/pyrosetta/protocols/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/protocols\n",
            "copying build/lib/pyrosetta/protocols/h5_fragment_store_provider.py -> build/bdist.linux-x86_64/egg/pyrosetta/protocols\n",
            "copying build/lib/pyrosetta/protocols/h5_structure_store_provider.py -> build/bdist.linux-x86_64/egg/pyrosetta/protocols\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/protocols/structure_search\n",
            "copying build/lib/pyrosetta/protocols/structure_search/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/protocols/structure_search\n",
            "copying build/lib/pyrosetta/protocols/structure_search/alignment.py -> build/bdist.linux-x86_64/egg/pyrosetta/protocols/structure_search\n",
            "copying build/lib/pyrosetta/teaching.py -> build/bdist.linux-x86_64/egg/pyrosetta\n",
            "creating build/bdist.linux-x86_64/egg/pyrosetta/bindings\n",
            "copying build/lib/pyrosetta/bindings/pose.py -> build/bdist.linux-x86_64/egg/pyrosetta/bindings\n",
            "copying build/lib/pyrosetta/bindings/__init__.py -> build/bdist.linux-x86_64/egg/pyrosetta/bindings\n",
            "copying build/lib/pyrosetta/bindings/rt.py -> build/bdist.linux-x86_64/egg/pyrosetta/bindings\n",
            "copying build/lib/pyrosetta/bindings/energies.py -> build/bdist.linux-x86_64/egg/pyrosetta/bindings\n",
            "copying build/lib/pyrosetta/bindings/homogeneous_transform.py -> build/bdist.linux-x86_64/egg/pyrosetta/bindings\n",
            "copying build/lib/pyrosetta/bindings/utility.py -> build/bdist.linux-x86_64/egg/pyrosetta/bindings\n",
            "byte-compiling build/bdist.linux-x86_64/egg/rosetta/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/logging_support.py to logging_support.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/PyMOLRosettaServer.py to PyMOLRosettaServer.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/numeric/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/numeric/alignment/rmsd_calc.py to rmsd_calc.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/numeric/alignment/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/mpi.py to mpi.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/packed_pose/pandas.py to pandas.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/packed_pose/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/packed_pose/core.py to core.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/dask/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/dask/worker.py to worker.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/io.py to io.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/viewer/presets/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/viewer/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/viewer/core.py to core.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/viewer/modules.py to modules.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/utility/log.py to log.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/utility/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/utility/pickle.py to pickle.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/tasks/score.py to score.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/tasks/rosetta_scripts.py to rosetta_scripts.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/tasks/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/tasks/taskbase.py to taskbase.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/distributed/docs/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/carbohydrates/reformat_original_gly_relax_file.py to reformat_original_gly_relax_file.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/database/chemical/pdb_components/split_components.py to split_components.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/toolbox/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/toolbox/cleaning.py to cleaning.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/toolbox/load_ligand.py to load_ligand.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/toolbox/py_jobdistributor.py to py_jobdistributor.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/toolbox/atom_pair_energy.py to atom_pair_energy.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/toolbox/numpy_utils.py to numpy_utils.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/toolbox/extract_coords_pose.py to extract_coords_pose.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/toolbox/mutants.py to mutants.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/toolbox/rcsb.py to rcsb.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/toolbox/generate_resfile.py to generate_resfile.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/utility/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/utility/array.py to array.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/io/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/io/silent_file_map.py to silent_file_map.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/__main__.py to __main__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/numeric/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/numeric/test_alignment.py to test_alignment.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed/test_smoke.py to test_smoke.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed/test_gil.py to test_gil.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed/test_dask_worker.py to test_dask_worker.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed/test_dask.py to test_dask.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed/test_concurrency.py to test_concurrency.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/distributed/test_viewer.py to test_viewer.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols/indexed_structure_store/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols/indexed_structure_store/data/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/protocols/indexed_structure_store/test_search.py to test_search.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/bindings/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/bindings/core/test_pose.py to test_pose.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/tests/bindings/core/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/protocols/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/protocols/h5_fragment_store_provider.py to h5_fragment_store_provider.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/protocols/h5_structure_store_provider.py to h5_structure_store_provider.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/protocols/structure_search/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/protocols/structure_search/alignment.py to alignment.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/teaching.py to teaching.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/bindings/pose.py to pose.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/bindings/__init__.py to __init__.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/bindings/rt.py to rt.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/bindings/energies.py to energies.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/bindings/homogeneous_transform.py to homogeneous_transform.cpython-36.pyc\n",
            "byte-compiling build/bdist.linux-x86_64/egg/pyrosetta/bindings/utility.py to utility.cpython-36.pyc\n",
            "creating build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pyrosetta.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pyrosetta.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pyrosetta.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pyrosetta.egg-info/not-zip-safe -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pyrosetta.egg-info/requires.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "copying pyrosetta.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO\n",
            "writing build/bdist.linux-x86_64/egg/EGG-INFO/native_libs.txt\n",
            "creating dist\n",
            "creating 'dist/pyrosetta-2020.27+release.cd4c8de-py3.6-linux-x86_64.egg' and adding 'build/bdist.linux-x86_64/egg' to it\n",
            "removing 'build/bdist.linux-x86_64/egg' (and everything under it)\n",
            "Processing pyrosetta-2020.27+release.cd4c8de-py3.6-linux-x86_64.egg\n",
            "removing '/content/google_drive/My Drive/prefix/pyrosetta-2020.27+release.cd4c8de-py3.6-linux-x86_64.egg' (and everything under it)\n",
            "creating /content/google_drive/My Drive/prefix/pyrosetta-2020.27+release.cd4c8de-py3.6-linux-x86_64.egg\n",
            "Extracting pyrosetta-2020.27+release.cd4c8de-py3.6-linux-x86_64.egg to /content/google_drive/My Drive/prefix\n",
            "pyrosetta 2020.27+release.cd4c8de is already the active version in easy-install.pth\n",
            "\n",
            "Installed /content/google_drive/My Drive/prefix/pyrosetta-2020.27+release.cd4c8de-py3.6-linux-x86_64.egg\n",
            "Processing dependencies for pyrosetta==2020.27+release.cd4c8de\n",
            "Finished processing dependencies for pyrosetta==2020.27+release.cd4c8de\n",
            "PyRosetta setup took: 605.4s...\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "imtzpWkKtyhi",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "for e in os.listdir(pyrosetta_install_prefix_path):\n",
        "    sys.path.append(pyrosetta_install_prefix_path + '/' + e)"
      ],
      "execution_count": 4,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "a6GAfULyy6iA",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 136
        },
        "outputId": "a85ef5c6-2544-497d-a12f-8eb1f695f6fe"
      },
      "source": [
        "from pyrosetta import *\n",
        "pyrosetta.init()"
      ],
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "PyRosetta-4 2020 [Rosetta PyRosetta4.MinSizeRel.python36.linux 2020.27+release.cd4c8de0d49b3a523091f756785370c2adf91ff4 2020-06-29T13:41:46] retrieved from: http://www.pyrosetta.org\n",
            "(C) Copyright Rosetta Commons Member Institutions. Created in JHU by Sergey Lyskov and PyRosetta Team.\n",
            "core.init: Checking for fconfig files in pwd and ./rosetta/flags\n",
            "core.init: Rosetta version: PyRosetta4.MinSizeRel.python36.linux r259 2020.27+release.cd4c8de cd4c8de0d49b3a523091f756785370c2adf91ff4 http://www.pyrosetta.org 2020-06-29T13:41:46\n",
            "core.init: command: PyRosetta -ex1 -ex2aro -database /content/prefix/pyrosetta-2020.27+release.cd4c8de-py3.6-linux-x86_64.egg/pyrosetta/database\n",
            "basic.random.init_random_generator: 'RNG device' seed mode, using '/dev/urandom', seed=-1339639536 seed_offset=0 real_seed=-1339639536\n",
            "basic.random.init_random_generator: RandomGenerator:init: Normal mode, seed=-1339639536 RG_type=mt19937\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "cue2RPZFtzd-",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "f12b771f-93d9-49e2-81bd-a2c3e4a4071c"
      },
      "source": [
        "!pip install biopython blosc dask dask-jobqueue distributed graphviz jupyter matplotlib numpy pandas scipy seaborn traitlets py3Dmol"
      ],
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Requirement already satisfied: biopython in /root/.local/lib/python3.6/site-packages (1.77)\n",
            "Requirement already satisfied: blosc in /root/.local/lib/python3.6/site-packages (1.9.1)\n",
            "Requirement already satisfied: dask in /usr/local/lib/python3.6/dist-packages (2.12.0)\n",
            "Requirement already satisfied: dask-jobqueue in /root/.local/lib/python3.6/site-packages (0.7.1)\n",
            "Requirement already satisfied: distributed in /usr/local/lib/python3.6/dist-packages (1.25.3)\n",
            "Requirement already satisfied: graphviz in /usr/local/lib/python3.6/dist-packages (0.10.1)\n",
            "Requirement already satisfied: jupyter in /usr/local/lib/python3.6/dist-packages (1.0.0)\n",
            "Requirement already satisfied: matplotlib in /usr/local/lib/python3.6/dist-packages (3.2.2)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (1.18.5)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.6/dist-packages (1.0.5)\n",
            "Requirement already satisfied: scipy in /usr/local/lib/python3.6/dist-packages (1.4.1)\n",
            "Requirement already satisfied: seaborn in /usr/local/lib/python3.6/dist-packages (0.10.1)\n",
            "Requirement already satisfied: traitlets in /usr/local/lib/python3.6/dist-packages (4.3.3)\n",
            "Requirement already satisfied: py3Dmol in /root/.local/lib/python3.6/site-packages (0.8.0)\n",
            "Requirement already satisfied: psutil>=5.0 in /usr/local/lib/python3.6/dist-packages (from distributed) (5.4.8)\n",
            "Requirement already satisfied: toolz>=0.7.4 in /usr/local/lib/python3.6/dist-packages (from distributed) (0.10.0)\n",
            "Requirement already satisfied: msgpack in /usr/local/lib/python3.6/dist-packages (from distributed) (1.0.0)\n",
            "Requirement already satisfied: pyyaml in /usr/local/lib/python3.6/dist-packages (from distributed) (3.13)\n",
            "Requirement already satisfied: click>=6.6 in /usr/local/lib/python3.6/dist-packages (from distributed) (7.1.2)\n",
            "Requirement already satisfied: sortedcontainers!=2.0.0,!=2.0.1 in /usr/local/lib/python3.6/dist-packages (from distributed) (2.2.2)\n",
            "Requirement already satisfied: zict>=0.1.3 in /usr/local/lib/python3.6/dist-packages (from distributed) (2.0.0)\n",
            "Requirement already satisfied: tornado>=4.5.1 in /usr/local/lib/python3.6/dist-packages (from distributed) (4.5.3)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.6/dist-packages (from distributed) (1.12.0)\n",
            "Requirement already satisfied: cloudpickle>=0.2.2 in /usr/local/lib/python3.6/dist-packages (from distributed) (1.3.0)\n",
            "Requirement already satisfied: tblib in /usr/local/lib/python3.6/dist-packages (from distributed) (1.6.0)\n",
            "Requirement already satisfied: notebook in /usr/local/lib/python3.6/dist-packages (from jupyter) (5.2.2)\n",
            "Requirement already satisfied: jupyter-console in /usr/local/lib/python3.6/dist-packages (from jupyter) (5.2.0)\n",
            "Requirement already satisfied: qtconsole in /usr/local/lib/python3.6/dist-packages (from jupyter) (4.7.5)\n",
            "Requirement already satisfied: ipykernel in /usr/local/lib/python3.6/dist-packages (from jupyter) (4.10.1)\n",
            "Requirement already satisfied: ipywidgets in /usr/local/lib/python3.6/dist-packages (from jupyter) (7.5.1)\n",
            "Requirement already satisfied: nbconvert in /usr/local/lib/python3.6/dist-packages (from jupyter) (5.6.1)\n",
            "Requirement already satisfied: python-dateutil>=2.1 in /usr/local/lib/python3.6/dist-packages (from matplotlib) (2.8.1)\n",
            "Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.6/dist-packages (from matplotlib) (1.2.0)\n",
            "Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.6/dist-packages (from matplotlib) (0.10.0)\n",
            "Requirement already satisfied: pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1 in /usr/local/lib/python3.6/dist-packages (from matplotlib) (2.4.7)\n",
            "Requirement already satisfied: pytz>=2017.2 in /usr/local/lib/python3.6/dist-packages (from pandas) (2018.9)\n",
            "Requirement already satisfied: ipython-genutils in /usr/local/lib/python3.6/dist-packages (from traitlets) (0.2.0)\n",
            "Requirement already satisfied: decorator in /usr/local/lib/python3.6/dist-packages (from traitlets) (4.4.2)\n",
            "Requirement already satisfied: idisplay in /root/.local/lib/python3.6/site-packages (from py3Dmol) (0.1.2)\n",
            "Requirement already satisfied: heapdict in /usr/local/lib/python3.6/dist-packages (from zict>=0.1.3->distributed) (1.0.1)\n",
            "Requirement already satisfied: nbformat in /usr/local/lib/python3.6/dist-packages (from notebook->jupyter) (5.0.7)\n",
            "Requirement already satisfied: terminado>=0.3.3; sys_platform != \"win32\" in /usr/local/lib/python3.6/dist-packages (from notebook->jupyter) (0.8.3)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.6/dist-packages (from notebook->jupyter) (2.11.2)\n",
            "Requirement already satisfied: jupyter-client in /usr/local/lib/python3.6/dist-packages (from notebook->jupyter) (5.3.4)\n",
            "Requirement already satisfied: jupyter-core in /usr/local/lib/python3.6/dist-packages (from notebook->jupyter) (4.6.3)\n",
            "Requirement already satisfied: pygments in /usr/local/lib/python3.6/dist-packages (from jupyter-console->jupyter) (2.1.3)\n",
            "Requirement already satisfied: prompt-toolkit<2.0.0,>=1.0.0 in /usr/local/lib/python3.6/dist-packages (from jupyter-console->jupyter) (1.0.18)\n",
            "Requirement already satisfied: ipython in /usr/local/lib/python3.6/dist-packages (from jupyter-console->jupyter) (5.5.0)\n",
            "Requirement already satisfied: qtpy in /usr/local/lib/python3.6/dist-packages (from qtconsole->jupyter) (1.9.0)\n",
            "Requirement already satisfied: pyzmq>=17.1 in /usr/local/lib/python3.6/dist-packages (from qtconsole->jupyter) (19.0.1)\n",
            "Requirement already satisfied: widgetsnbextension~=3.5.0 in /usr/local/lib/python3.6/dist-packages (from ipywidgets->jupyter) (3.5.1)\n",
            "Requirement already satisfied: testpath in /usr/local/lib/python3.6/dist-packages (from nbconvert->jupyter) (0.4.4)\n",
            "Requirement already satisfied: entrypoints>=0.2.2 in /usr/local/lib/python3.6/dist-packages (from nbconvert->jupyter) (0.3)\n",
            "Requirement already satisfied: pandocfilters>=1.4.1 in /usr/local/lib/python3.6/dist-packages (from nbconvert->jupyter) (1.4.2)\n",
            "Requirement already satisfied: mistune<2,>=0.8.1 in /usr/local/lib/python3.6/dist-packages (from nbconvert->jupyter) (0.8.4)\n",
            "Requirement already satisfied: bleach in /usr/local/lib/python3.6/dist-packages (from nbconvert->jupyter) (3.1.5)\n",
            "Requirement already satisfied: defusedxml in /usr/local/lib/python3.6/dist-packages (from nbconvert->jupyter) (0.6.0)\n",
            "Requirement already satisfied: jsonschema!=2.5.0,>=2.4 in /usr/local/lib/python3.6/dist-packages (from nbformat->notebook->jupyter) (2.6.0)\n",
            "Requirement already satisfied: ptyprocess; os_name != \"nt\" in /usr/local/lib/python3.6/dist-packages (from terminado>=0.3.3; sys_platform != \"win32\"->notebook->jupyter) (0.6.0)\n",
            "Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.6/dist-packages (from jinja2->notebook->jupyter) (1.1.1)\n",
            "Requirement already satisfied: wcwidth in /usr/local/lib/python3.6/dist-packages (from prompt-toolkit<2.0.0,>=1.0.0->jupyter-console->jupyter) (0.2.5)\n",
            "Requirement already satisfied: simplegeneric>0.8 in /usr/local/lib/python3.6/dist-packages (from ipython->jupyter-console->jupyter) (0.8.1)\n",
            "Requirement already satisfied: pickleshare in /usr/local/lib/python3.6/dist-packages (from ipython->jupyter-console->jupyter) (0.7.5)\n",
            "Requirement already satisfied: pexpect; sys_platform != \"win32\" in /usr/local/lib/python3.6/dist-packages (from ipython->jupyter-console->jupyter) (4.8.0)\n",
            "Requirement already satisfied: setuptools>=18.5 in /usr/local/lib/python3.6/dist-packages (from ipython->jupyter-console->jupyter) (47.3.1)\n",
            "Requirement already satisfied: webencodings in /usr/local/lib/python3.6/dist-packages (from bleach->nbconvert->jupyter) (0.5.1)\n",
            "Requirement already satisfied: packaging in /usr/local/lib/python3.6/dist-packages (from bleach->nbconvert->jupyter) (20.4)\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "AgoY0R0ouHLw",
        "colab_type": "text"
      },
      "source": [
        "Now you should be ready to use the notebook with Google Colab! :)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "colab_type": "text",
        "id": "QV9XsAnJ5gCO"
      },
      "source": [
        "*Warning*: This notebook uses `pyrosetta.distributed.viewer` code, which runs in `jupyter notebook` and might not run if you're using `jupyterlab`. Consider installing `! jupyter labextension install jupyterlab_3dmol`"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab_type": "code",
        "id": "49vXPJbg5gCP",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 71
        },
        "outputId": "2a6049de-36fe-451d-881c-ea6e48841b27"
      },
      "source": [
        "import logging\n",
        "logging.basicConfig(level=logging.INFO)\n",
        "import matplotlib\n",
        "%matplotlib inline\n",
        "import zipfile\n",
        "import os\n",
        "import pandas as pd\n",
        "import pyrosetta\n",
        "import pyrosetta.distributed.viewer as viewer\n",
        "import seaborn\n",
        "seaborn.set()\n",
        "import sys\n",
        "import requests\n",
        "working_dir = os.getcwd()\n",
        "DATA_FOLDER = './data'"
      ],
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/html": [
              "<style>.container { width:100% !important; }</style>"
            ],
            "text/plain": [
              "<IPython.core.display.HTML object>"
            ]
          },
          "metadata": {
            "tags": []
          }
        },
        {
          "output_type": "stream",
          "text": [
            "/usr/local/lib/python3.6/dist-packages/statsmodels/tools/_testing.py:19: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.\n",
            "  import pandas.util.testing as tm\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "T3NwjMr1ov43",
        "colab_type": "text"
      },
      "source": [
        "The pdb files describing the possible targets for the Leishmania protein are stored in a (google bucket)[https://storage.googleapis.com/indaba-challenge/]. A useful (but not mandatory) tool to explore the bucket is (gsutil)[https://cloud.google.com/storage/docs/gsutil_install#mac]. For example we can list the contents of the bucket:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "gB8KM0Pzov44",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "# Example of gsutil\n",
        "# List all files\n",
        "# !gsutil ls -r gs://indaba-challenge/*\n",
        "# Copy a file locally\n",
        "# !gsutil cp gs://indaba-challenge/A0A088RHB8.0.apo.pdb ."
      ],
      "execution_count": 8,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "nxIToXUjov46",
        "colab_type": "text"
      },
      "source": [
        "In the bucket there is a list of approved drugs. Lets download the zip folder containing the data of all the approved drugs and extracti it locally."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "_UO3JI2lov46",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "url = 'https://storage.googleapis.com/indaba-challenge/molecules.zip'\n",
        "r = requests.get(url, allow_redirects=True)\n",
        "open('molecules.zip', 'wb').write(r.content)\n",
        "\n",
        "# # Extract all the files with \n",
        "with zipfile.ZipFile('molecules.zip', 'r') as zip_ref:\n",
        "    zip_ref.extractall(DATA_FOLDER)"
      ],
      "execution_count": 9,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "17kx9CDNov48",
        "colab_type": "text"
      },
      "source": [
        "From the bucket, lets pick as target protein `A0A088RHB8.0.apo` (88RHB8 Leishmania panamensis) and a particular drug we want to test (a.k.a ligand, compound). Lets pick [Erlotinib](https://pubchem.ncbi.nlm.nih.gov/compound/176870) `AAKJLRGGTJKAMG-UHFFFAOYSA-N`. You can search for compounds [here](https://www.ncbi.nlm.nih.gov/pccompound/?term=AAKJLRGGTJKAMG-UHFFF) and check if they are in the zip file."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "0IOO1TrPov49",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "5921b454-f7f0-4816-9710-48bdec64739e"
      },
      "source": [
        "target_protein = 'A0A088RHB8.0.apo'\n",
        "target_protein_file = '{}/{}.pdb'.format(DATA_FOLDER, target_protein)\n",
        "\n",
        "url = 'https://storage.googleapis.com/indaba-challenge/{}.pdb'.format(target_protein)\n",
        "r = requests.get(url, allow_redirects=True)\n",
        "open(target_protein_file, 'wb').write(r.content)\n",
        "\n"
      ],
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "333562"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 10
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "pByp1OZpov5A",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "compound = 'AAKJLRGGTJKAMG-UHFFFAOYSA-N'\n",
        "ligand_params = \"{}/{}.params\".format(DATA_FOLDER,compound)\n",
        "ligand_file = \"{}/{}.pdb\".format(DATA_FOLDER,compound)"
      ],
      "execution_count": 11,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "FkPZHfMlov5C",
        "colab_type": "text"
      },
      "source": [
        "Now that we have our ligand and target structure we need to create poses with rosetta and create a target-ligand pose by appending the receptor protein and the ligand (drug). The [Pose](https://www.rosettacommons.org/manuals/latest/core+protocols/d3/d3c/classcore_1_1pose_1_1_pose.html#details) class in Rosetta represents a molecular system (protein-ligand) as a container of Residues. \n",
        "\n",
        "We also need the [params](https://www.rosettacommons.org/docs/latest/rosetta_basics/file_types/Residue-Params-file) file for the ligand. Params files store a variety of chemical and geometric information used to define the shape and chemical connectivity of an amino acid building block or other small molecule. This will be useful to create our scoring function."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab_type": "code",
        "id": "KxQ5aZnn5gCU",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "a887f59e-31f6-41d2-cea7-02b5b154b167"
      },
      "source": [
        "flags = f\"\"\"\n",
        "-ignore_unrecognized_res 1\n",
        "-extra_res_fa {ligand_params}\n",
        "\"\"\"\n",
        "pyrosetta.distributed.init(flags)\n",
        "pose_ligand = pyrosetta.io.pose_from_file(filename=ligand_file)\n",
        "pose = pyrosetta.io.pose_from_file(filename=target_protein_file)\n",
        "pose.append_pose_by_jump(pose_ligand,pose.total_residue())"
      ],
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "core.chemical.GlobalResidueTypeSet: Finished initializing fa_standard residue type set.  Created 981 residue types\n",
            "core.chemical.GlobalResidueTypeSet: Total time to initialize 1.66982 seconds.\n",
            "core.import_pose.import_pose: File './data/AAKJLRGGTJKAMG-UHFFFAOYSA-N.pdb' automatically determined to be of type PDB\n",
            "core.chemical.GlobalResidueTypeSet: Loading (but possibly not actually using) 'LG1' from the PDB components dictionary for residue type 'pdb_LG1'\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] discarding 40 atoms at position 1 in file ./data/AAKJLRGGTJKAMG-UHFFFAOYSA-N.pdb. Best match rsd_type:  pdb_LG1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C05 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C06 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C01 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: N14 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: F30 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C23 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C24 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: NP5 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C02 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C03 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C04 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: F08 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C34 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C27 on residue pdb_LG1 1\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom: C33 on residue pdb_LG1 1\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C1  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C10 (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C14 (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C2  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C3  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C4  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C5  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C6  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C7  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C8  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  C9  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  N1  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  N2  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  N3  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  O1  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  O2  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  O3  (trying to store temperature in PDBInfo)\n",
            "core.io.pose_from_sfr.PoseFromSFRBuilder: [ WARNING ] can't find pose atom for file-residue 1 atom  O4  (trying to store temperature in PDBInfo)\n",
            "core.pack.pack_missing_sidechains: packing residue number 1 because of missing atom number 1 atom name C05\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "core.scoring.etable: Starting energy table calculation\n",
            "core.scoring.etable: smooth_etable: changing atr/rep split to bottom of energy well\n",
            "core.scoring.etable: smooth_etable: spline smoothing lj etables (maxdis = 6)\n",
            "core.scoring.etable: smooth_etable: spline smoothing solvation etables (max_dis = 6)\n",
            "core.scoring.etable: Finished calculating energy tables.\n",
            "basic.io.database: Database file opened: scoring/score_functions/hbonds/ref2015_params/HBPoly1D.csv\n",
            "basic.io.database: Database file opened: scoring/score_functions/hbonds/ref2015_params/HBFadeIntervals.csv\n",
            "basic.io.database: Database file opened: scoring/score_functions/hbonds/ref2015_params/HBEval.csv\n",
            "basic.io.database: Database file opened: scoring/score_functions/hbonds/ref2015_params/DonStrength.csv\n",
            "basic.io.database: Database file opened: scoring/score_functions/hbonds/ref2015_params/AccStrength.csv\n",
            "basic.io.database: Database file opened: scoring/score_functions/rama/fd/all.ramaProb\n",
            "basic.io.database: Database file opened: scoring/score_functions/rama/fd/prepro.ramaProb\n",
            "basic.io.database: Database file opened: scoring/score_functions/omega/omega_ppdep.all.txt\n",
            "basic.io.database: Database file opened: scoring/score_functions/omega/omega_ppdep.gly.txt\n",
            "basic.io.database: Database file opened: scoring/score_functions/omega/omega_ppdep.pro.txt\n",
            "basic.io.database: Database file opened: scoring/score_functions/omega/omega_ppdep.valile.txt\n",
            "basic.io.database: Database file opened: scoring/score_functions/P_AA_pp/P_AA\n",
            "basic.io.database: Database file opened: scoring/score_functions/P_AA_pp/P_AA_n\n",
            "core.scoring.P_AA: shapovalov_lib::shap_p_aa_pp_smooth_level of 1( aka low_smooth ) got activated.\n",
            "basic.io.database: Database file opened: scoring/score_functions/P_AA_pp/shapovalov/10deg/kappa131/a20.prop\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 1 rotamers at 1 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.import_pose.import_pose: File './data/A0A088RHB8.0.apo.pdb' automatically determined to be of type PDB\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ALA:CtermProteinFull 255\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  CG  on residue GLN 348\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue GLN 348\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  OE1 on residue GLN 348\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  NE2 on residue GLN 348\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  CG  on residue GLU 385\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  CD  on residue GLU 385\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  OE1 on residue GLU 385\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  OE2 on residue GLU 385\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  CE  on residue LYS 489\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  NZ  on residue LYS 489\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ALA:CtermProteinFull 510\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ASP:CtermProteinFull 516\n",
            "core.conformation.Conformation: [ WARNING ] missing heavyatom:  OXT on residue ASP:CtermProteinFull 522\n",
            "core.pack.pack_missing_sidechains: packing residue number 348 because of missing atom number 6 atom name  CG\n",
            "core.pack.pack_missing_sidechains: packing residue number 385 because of missing atom number 6 atom name  CG\n",
            "core.pack.pack_missing_sidechains: packing residue number 489 because of missing atom number 8 atom name  CE\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "basic.io.database: Database file opened: scoring/score_functions/elec_cp_reps.dat\n",
            "core.scoring.elec.util: Read 40 countpair representative atoms\n",
            "core.pack.dunbrack.RotamerLibrary: shapovalov_lib_fixes_enable option is true.\n",
            "core.pack.dunbrack.RotamerLibrary: shapovalov_lib::shap_dun10_smooth_level of 1( aka lowest_smooth ) got activated.\n",
            "core.pack.dunbrack.RotamerLibrary: Binary rotamer library selected: /content/prefix/pyrosetta-2020.27+release.cd4c8de-py3.6-linux-x86_64.egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/Dunbrack10.lib.bin\n",
            "core.pack.dunbrack.RotamerLibrary: Using Dunbrack library binary file '/content/prefix/pyrosetta-2020.27+release.cd4c8de-py3.6-linux-x86_64.egg/pyrosetta/database/rotamer/shapovalov/StpDwn_0-0-0/Dunbrack10.lib.bin'.\n",
            "core.pack.dunbrack.RotamerLibrary: Dunbrack 2010 library took 0.26957 seconds to load from binary\n",
            "core.pack.pack_rotamers: built 55 rotamers at 3 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "colab_type": "text",
        "id": "PkisG-6I5gCY"
      },
      "source": [
        "Rosetta can be used to predict the bound structure of a ligand and a target protein starting from unbound structures. This is what we are going to do. To achieve that, we need to define a [Rosetta Ligand docking protocol](https://www.rosettacommons.org/demos/latest/tutorials/Protein-Protein-Docking/Protein-Protein-Docking). This is a script that describes the docking and scoring."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab_type": "code",
        "id": "RW1uus_15gCY",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "988d5331-1200-4873-e9dc-7905cc1fe92a"
      },
      "source": [
        "xml = pyrosetta.rosetta.protocols.rosetta_scripts.XmlObjects.create_from_string(\"\"\"\n",
        "<ROSETTASCRIPTS>\n",
        "  <SCOREFXNS>\n",
        "    <ScoreFunction name=\"fa_standard\" weights=\"ref2015.wts\"/>\n",
        "  </SCOREFXNS>\n",
        "  <RESIDUE_SELECTORS>\n",
        "    <Chain name=\"chX\" chains=\"X\"/>\n",
        "  </RESIDUE_SELECTORS>\n",
        "  <SIMPLE_METRICS>\n",
        "    <RMSDMetric name=\"rmsd_chX\" residue_selector=\"chX\" reference_name=\"store_native\" residue_selector_ref=\"chX\" robust=\"true\" rmsd_type=\"rmsd_all\" />\n",
        "  </SIMPLE_METRICS>\n",
        "  <SCORINGGRIDS ligand_chain=\"X\" width=\"25\">\n",
        "    <ClassicGrid grid_name=\"vdw\" weight=\"1.0\"/>\n",
        "  </SCORINGGRIDS>\n",
        "  <LIGAND_AREAS>\n",
        "    <LigandArea name=\"docking_sidechain_X\" chain=\"X\" cutoff=\"6.0\" add_nbr_radius=\"true\" all_atom_mode=\"true\" minimize_ligand=\"10\"/>\n",
        "    <LigandArea name=\"final_sidechain_X\" chain=\"X\" cutoff=\"6.0\" add_nbr_radius=\"true\" all_atom_mode=\"true\"/>\n",
        "    <LigandArea name=\"final_backbone_X\" chain=\"X\" cutoff=\"7.0\" add_nbr_radius=\"false\" all_atom_mode=\"true\" Calpha_restraints=\"0.3\"/>\n",
        "  </LIGAND_AREAS>\n",
        "  <INTERFACE_BUILDERS>\n",
        "    <InterfaceBuilder name=\"side_chain_for_docking\" ligand_areas=\"docking_sidechain_X\"/>\n",
        "    <InterfaceBuilder name=\"side_chain_for_final\" ligand_areas=\"final_sidechain_X\"/>\n",
        "    <InterfaceBuilder name=\"backbone\" ligand_areas=\"final_backbone_X\" extension_window=\"3\"/>\n",
        "  </INTERFACE_BUILDERS>\n",
        "  <MOVEMAP_BUILDERS>\n",
        "    <MoveMapBuilder name=\"docking\" sc_interface=\"side_chain_for_docking\" minimize_water=\"true\"/>\n",
        "    <MoveMapBuilder name=\"final\" sc_interface=\"side_chain_for_final\" bb_interface=\"backbone\" minimize_water=\"true\"/>\n",
        "  </MOVEMAP_BUILDERS>\n",
        "  <MOVERS>\n",
        "    <SavePoseMover name=\"spm\" restore_pose=\"0\" reference_name=\"store_native\"/>\n",
        "    <Transform name=\"transform\" chain=\"X\" box_size=\"20.0\" move_distance=\"10\" angle=\"360\" initial_perturb=\"2\" cycles=\"500\" repeats=\"5\" temperature=\"1000\"/>\n",
        "    <HighResDocker name=\"high_res_docker\" cycles=\"9\" repack_every_Nth=\"3\" scorefxn=\"fa_standard\" movemap_builder=\"docking\"/>\n",
        "    <FinalMinimizer name=\"final\" scorefxn=\"fa_standard\" movemap_builder=\"final\"/>\n",
        "  </MOVERS>\n",
        "  <FILTERS>\n",
        "      <LigInterfaceEnergy name=\"interfE\" scorefxn=\"fa_standard\" energy_cutoff=\"0.0\" confidence=\"0\"/>\n",
        "      <SimpleMetricFilter name=\"rmsd_chX\" metric=\"rmsd_chX\" cutoff=\"999999.\" comparison_type=\"lt\" confidence=\"0\"/>\n",
        "  </FILTERS>\n",
        "  <PROTOCOLS>\n",
        "    <Add mover=\"spm\"/>\n",
        "    <Add mover=\"transform\"/>\n",
        "    <Add mover=\"high_res_docker\"/>\n",
        "    <Add mover=\"final\"/>\n",
        "    <Add filter=\"interfE\"/>\n",
        "    <Add filter=\"rmsd_chX\"/>\n",
        "  </PROTOCOLS>\n",
        "</ROSETTASCRIPTS>\n",
        "\"\"\").get_mover(\"ParsedProtocol\")"
      ],
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "protocols.rosetta_scripts.RosettaScriptsParser: Generating XML Schema for rosetta_scripts...\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Initializing schema validator...\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Validating input script...\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: ...done\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Parsed script:\n",
            "<ROSETTASCRIPTS>\n",
            "\t<SCOREFXNS>\n",
            "\t\t<ScoreFunction name=\"fa_standard\" weights=\"ref2015.wts\"/>\n",
            "\t</SCOREFXNS>\n",
            "\t<RESIDUE_SELECTORS>\n",
            "\t\t<Chain chains=\"X\" name=\"chX\"/>\n",
            "\t</RESIDUE_SELECTORS>\n",
            "\t<SIMPLE_METRICS>\n",
            "\t\t<RMSDMetric name=\"rmsd_chX\" reference_name=\"store_native\" residue_selector=\"chX\" residue_selector_ref=\"chX\" rmsd_type=\"rmsd_all\" robust=\"true\"/>\n",
            "\t</SIMPLE_METRICS>\n",
            "\t<SCORINGGRIDS ligand_chain=\"X\" width=\"25\">\n",
            "\t\t<ClassicGrid grid_name=\"vdw\" weight=\"1.0\"/>\n",
            "\t</SCORINGGRIDS>\n",
            "\t<LIGAND_AREAS>\n",
            "\t\t<LigandArea add_nbr_radius=\"true\" all_atom_mode=\"true\" chain=\"X\" cutoff=\"6.0\" minimize_ligand=\"10\" name=\"docking_sidechain_X\"/>\n",
            "\t\t<LigandArea add_nbr_radius=\"true\" all_atom_mode=\"true\" chain=\"X\" cutoff=\"6.0\" name=\"final_sidechain_X\"/>\n",
            "\t\t<LigandArea Calpha_restraints=\"0.3\" add_nbr_radius=\"false\" all_atom_mode=\"true\" chain=\"X\" cutoff=\"7.0\" name=\"final_backbone_X\"/>\n",
            "\t</LIGAND_AREAS>\n",
            "\t<INTERFACE_BUILDERS>\n",
            "\t\t<InterfaceBuilder ligand_areas=\"docking_sidechain_X\" name=\"side_chain_for_docking\"/>\n",
            "\t\t<InterfaceBuilder ligand_areas=\"final_sidechain_X\" name=\"side_chain_for_final\"/>\n",
            "\t\t<InterfaceBuilder extension_window=\"3\" ligand_areas=\"final_backbone_X\" name=\"backbone\"/>\n",
            "\t</INTERFACE_BUILDERS>\n",
            "\t<MOVEMAP_BUILDERS>\n",
            "\t\t<MoveMapBuilder minimize_water=\"true\" name=\"docking\" sc_interface=\"side_chain_for_docking\"/>\n",
            "\t\t<MoveMapBuilder bb_interface=\"backbone\" minimize_water=\"true\" name=\"final\" sc_interface=\"side_chain_for_final\"/>\n",
            "\t</MOVEMAP_BUILDERS>\n",
            "\t<MOVERS>\n",
            "\t\t<SavePoseMover name=\"spm\" reference_name=\"store_native\" restore_pose=\"0\"/>\n",
            "\t\t<Transform angle=\"360\" box_size=\"20.0\" chain=\"X\" cycles=\"500\" initial_perturb=\"2\" move_distance=\"10\" name=\"transform\" repeats=\"5\" temperature=\"1000\"/>\n",
            "\t\t<HighResDocker cycles=\"9\" movemap_builder=\"docking\" name=\"high_res_docker\" repack_every_Nth=\"3\" scorefxn=\"fa_standard\"/>\n",
            "\t\t<FinalMinimizer movemap_builder=\"final\" name=\"final\" scorefxn=\"fa_standard\"/>\n",
            "\t</MOVERS>\n",
            "\t<FILTERS>\n",
            "\t\t<LigInterfaceEnergy confidence=\"0\" energy_cutoff=\"0.0\" name=\"interfE\" scorefxn=\"fa_standard\"/>\n",
            "\t\t<SimpleMetricFilter comparison_type=\"lt\" confidence=\"0\" cutoff=\"999999.\" metric=\"rmsd_chX\" name=\"rmsd_chX\"/>\n",
            "\t</FILTERS>\n",
            "\t<PROTOCOLS>\n",
            "\t\t<Add mover=\"spm\"/>\n",
            "\t\t<Add mover=\"transform\"/>\n",
            "\t\t<Add mover=\"high_res_docker\"/>\n",
            "\t\t<Add mover=\"final\"/>\n",
            "\t\t<Add filter=\"interfE\"/>\n",
            "\t\t<Add filter=\"rmsd_chX\"/>\n",
            "\t</PROTOCOLS>\n",
            "</ROSETTASCRIPTS>\n",
            "core.scoring.ScoreFunctionFactory: SCOREFUNCTION: ref2015\n",
            "core.scoring.etable: Starting energy table calculation\n",
            "core.scoring.etable: smooth_etable: changing atr/rep split to bottom of energy well\n",
            "core.scoring.etable: smooth_etable: spline smoothing lj etables (maxdis = 6)\n",
            "core.scoring.etable: smooth_etable: spline smoothing solvation etables (max_dis = 6)\n",
            "core.scoring.etable: Finished calculating energy tables.\n",
            "basic.io.database: Database file opened: scoring/score_functions/PairEPotential/pdb_pair_stats_fine\n",
            "basic.io.database: Database file opened: scoring/score_functions/InterchainPotential/interchain_env_log.txt\n",
            "basic.io.database: Database file opened: scoring/score_functions/InterchainPotential/interchain_pair_log.txt\n",
            "basic.io.database: Database file opened: scoring/score_functions/EnvPairPotential/env_log.txt\n",
            "basic.io.database: Database file opened: scoring/score_functions/EnvPairPotential/cbeta_den.txt\n",
            "basic.io.database: Database file opened: scoring/score_functions/EnvPairPotential/pair_log.txt\n",
            "basic.io.database: Database file opened: scoring/score_functions/EnvPairPotential/cenpack_log.txt\n",
            "core.scoring.ramachandran: shapovalov_lib::shap_rama_smooth_level of 4( aka highest_smooth ) got activated.\n",
            "basic.io.database: Database file opened: scoring/score_functions/rama/shapovalov/kappa25/all.ramaProb\n",
            "protocols.jd2.parser.ScoreFunctionLoader: defined score function \"fa_standard\" with weights \"ref2015.wts\"\n",
            "core.select.residue_selector.util: Found residue selector chX\n",
            "core.select.residue_selector.util: Found residue selector chX\n",
            "core.pose.ref_pose: Loading PDB: store_native\n",
            "core.simple_metrics.metrics.RMSDMetric: Loaded reference pose: store_native with 0 residues\n",
            "core.pose.ref_pose: Loading PDB: store_native\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"spm\" of type SavePoseMover\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"transform\" of type Transform\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"high_res_docker\" of type HighResDocker\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Defined mover named \"final\" of type FinalMinimizer\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Defined filter named \"interfE\" of type LigInterfaceEnergy\n",
            "protocols.rosetta_scripts.RosettaScriptsParser: Defined filter named \"rmsd_chX\" of type SimpleMetricFilter\n",
            "protocols.rosetta_scripts.ParsedProtocol: ParsedProtocol mover with the following movers and filters\n",
            "protocols.rosetta_scripts.ParsedProtocol: added mover \"spm\" with filter \"true_filter\"\n",
            "protocols.rosetta_scripts.ParsedProtocol: added mover \"transform\" with filter \"true_filter\"\n",
            "protocols.rosetta_scripts.ParsedProtocol: added mover \"high_res_docker\" with filter \"true_filter\"\n",
            "protocols.rosetta_scripts.ParsedProtocol: added mover \"final\" with filter \"true_filter\"\n",
            "protocols.rosetta_scripts.ParsedProtocol: added mover \"NULL_MOVER\" with filter \"interfE\"\n",
            "protocols.rosetta_scripts.ParsedProtocol: added mover \"NULL_MOVER\" with filter \"rmsd_chX\"\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "colab_type": "text",
        "id": "A8oB_udK5gCc"
      },
      "source": [
        "Now we are ready to produce 5 global ligand docking trajectories (We could create as many as we like). Each of them will score differently."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab_type": "code",
        "id": "9DVf8VzG5gCc",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "d53f6b17-c791-4b47-da2c-8d909473beaa"
      },
      "source": [
        "scorefxn = pyrosetta.create_score_function(\"ref2015\")\n",
        "output_dir = \"output\"\n",
        "os.chdir(working_dir)\n",
        "if not os.path.exists(output_dir):\n",
        "    os.mkdir(output_dir)\n",
        "os.chdir(output_dir)\n",
        "\n",
        "df = pd.DataFrame()\n",
        "for i in range(5):\n",
        "    test_pose = pose.clone()\n",
        "    xml.apply(test_pose)\n",
        "    test_df = pd.DataFrame.from_records(dict(test_pose.scores), index=[i])\n",
        "    df = df.append(test_df)\n",
        "    test_pose.dump_pdb('{}_{}.pdb'.format(target_protein, str(i)))\n",
        "os.chdir(working_dir)"
      ],
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER SavePoseMover - spm=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Transform - transform=======================\n",
            "protocols.qsar.scoring_grid.GridManager: [ WARNING ] option -qsar:grid_dir is not set.  Use this flag to specify a directory to store scoring grids.  This will save you a huge amount of time\n",
            "protocols.qsar.scoring_grid.GridManager: No conformation matching hash: 8f08f9042fb9bdbd5e8b03bf2921d0cd6038cfa3 Updating grid and adding it to the cache\n",
            "protocols.ligand_docking.Transform: Considering 1 conformers during sampling\n",
            "protocols.ligand_docking.Transform: percent acceptance: 374 0.1496 2126\n",
            "protocols.ligand_docking.Transform: Moves rejected for being outside of grid: 2124  0.8496\n",
            "protocols.ligand_docking.Transform: [ WARNING ] A large number of moves were rejected for being outside the grid. You likely want to reexamine your settings.\n",
            "protocols.ligand_docking.Transform: [ WARNING ]     For the current settings, a grid size of at least 93.7    and a box size of at least 260.9 are recommended.\n",
            "protocols.evaluation.ChiWellRmsdEvaluatorCreator: Evaluation Creator active ...\n",
            "protocols.ligand_docking.Transform: Accepted pose with grid score: -37\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER HighResDocker - high_res_docker=======================\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 157 rotamers at 13 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 185 rotamers at 13 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 185 rotamers at 13 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -605.183 -605.544\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER FinalMinimizer - final=======================\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: Energy prior to minimizing: -605.544\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: [ WARNING ] Warning: FinalMinimizer with backbone minimization used without chainbreak scoreterm - setting chainbreak to 1\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: Energy after minimizing: -619.049\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN FILTER interfE=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================END FILTER interfE=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN FILTER rmsd_chX=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================END FILTER rmsd_chX=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "protocols.enzdes.EnzFilters: Calculated Interface Energy is before cst correction is:-0.327919\n",
            "protocols.enzdes.EnzFilters: Calculated Interface Energy is -0.327919\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============Begin report for interfE==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: \n",
            "    chain      res       AA    total  contact   fa_atr   fa_rep hb_bb_sc    hb_sc   fa_sol   fa_dun  fa_pair\n",
            "        A      127      GLU   -5.840    -1.121    -8.425     0.346    -0.874    -0.447     8.279     3.028     0.000\n",
            "        A      128      GLN    0.499     0.536    -4.476     0.380     0.000     0.000     4.633     2.411     0.000\n",
            "        A      130      ALA   -2.345    -2.777    -5.422     0.423     0.000     0.000     2.222     0.000     0.000\n",
            "        A      131      GLY   -2.123     0.077    -2.680     0.303     0.000     0.000     2.454     0.000     0.000\n",
            "        A      132      ASN   -1.526    -2.031    -5.509     0.381     0.000     0.000     3.097     1.569     0.000\n",
            "        A      133      LEU   -3.428    -3.591    -7.418     0.740     0.000     0.000     3.087     0.177     0.000\n",
            "        A      134      ARG   -5.230    -3.820    -9.980     0.805    -0.874    -2.042     8.270     3.006     0.000\n",
            "        A      135      GLU   -1.288    -1.003    -4.738     0.595     0.000     0.000     3.141     3.163     0.000\n",
            "        A      136      ASN   -2.595    -1.317    -6.319     0.468     0.000     0.000     4.534     1.873     0.000\n",
            "        A      137      ILE   -4.481    -5.245    -7.455     0.287     0.000     0.000     1.923     0.238     0.000\n",
            "        A      166      ARG   -1.832    -1.971    -6.869     0.308     0.000     0.000     4.590     2.222     0.000\n",
            "        X        1      LG1   -7.699    -5.236    -6.657     0.800     0.000     0.000     0.622     0.000     0.000\n",
            "============End report for interfE==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============Begin report for rmsd_chX==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============End report for rmsd_chX==================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER SavePoseMover - spm=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Transform - transform=======================\n",
            "protocols.qsar.scoring_grid.GridManager: Found a conformation matching hash: 8f08f9042fb9bdbd5e8b03bf2921d0cd6038cfa3 Loading from grid cache\n",
            "protocols.ligand_docking.Transform: Considering 1 conformers during sampling\n",
            "protocols.ligand_docking.Transform: percent acceptance: 378 0.1512 2122\n",
            "protocols.ligand_docking.Transform: Moves rejected for being outside of grid: 2116  0.8464\n",
            "protocols.ligand_docking.Transform: [ WARNING ] A large number of moves were rejected for being outside the grid. You likely want to reexamine your settings.\n",
            "protocols.ligand_docking.Transform: [ WARNING ]     For the current settings, a grid size of at least 93.7    and a box size of at least 260.9 are recommended.\n",
            "protocols.ligand_docking.Transform: Accepted pose with grid score: -30\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER HighResDocker - high_res_docker=======================\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 211 rotamers at 11 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 247 rotamers at 12 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 247 rotamers at 12 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -604.36 -604.999\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER FinalMinimizer - final=======================\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: Energy prior to minimizing: -604.999\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: [ WARNING ] Warning: FinalMinimizer with backbone minimization used without chainbreak scoreterm - setting chainbreak to 1\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: Energy after minimizing: -611.338\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN FILTER interfE=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================END FILTER interfE=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN FILTER rmsd_chX=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================END FILTER rmsd_chX=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "protocols.enzdes.EnzFilters: Calculated Interface Energy is before cst correction is:-0.323956\n",
            "protocols.enzdes.EnzFilters: Calculated Interface Energy is -0.323956\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============Begin report for interfE==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: \n",
            "    chain      res       AA    total  contact   fa_atr   fa_rep hb_bb_sc    hb_sc   fa_sol   fa_dun  fa_pair\n",
            "        A      123      TYR   -0.927    -2.395    -9.004     0.888     0.000    -0.811     6.533     2.663     0.000\n",
            "        A      124      ASP    0.334     0.888    -3.590     0.307     0.000     0.000     4.172     1.341     0.000\n",
            "        A      125      ARG    0.729    -0.637    -7.162     0.979     0.000     0.000     5.546     2.788     0.000\n",
            "        A      127      GLU   -5.513    -1.041    -8.842     0.586    -0.952    -0.453     8.620     2.963     0.000\n",
            "        A      128      GLN    0.046     0.405    -4.366     0.331     0.000     0.000     4.441     2.477     0.000\n",
            "        A      129      LEU   -2.530    -3.556    -7.056     1.825     0.000     0.000     1.675     0.530     0.000\n",
            "        A      130      ALA   -0.993    -3.348    -5.608     0.388    -0.373     0.000     2.246     0.000     0.000\n",
            "        A      131      GLY   -0.446     0.192    -2.650     0.268     0.000     0.000     2.574     0.000     0.000\n",
            "        A      132      ASN    0.300    -1.265    -4.951     0.207    -0.373     0.000     3.852     2.185     0.000\n",
            "        A      133      LEU   -3.347    -3.913    -7.747     1.089     0.000     0.000     2.744     0.223     0.000\n",
            "        A      134      ARG   -5.278    -3.856   -10.137     0.915    -0.952    -2.174     8.492     2.966     0.000\n",
            "        A      135      GLU   -0.793    -0.783    -4.762     0.702     0.000     0.000     3.276     3.283     0.000\n",
            "        A      136      ASN   -1.529     0.052    -4.271     0.479     0.000     0.000     3.843     1.665     0.000\n",
            "        A      162      PRO    0.258    -1.569    -5.121     0.587     0.000     0.000     2.965     1.604     0.000\n",
            "        A      166      ARG    5.401    -2.083    -8.125     0.365     0.000     0.000     5.678     9.513     0.000\n",
            "        B      118      ALA    1.833    -0.586    -1.787     0.045     0.000     0.000     1.156     0.000     0.000\n",
            "        X        1      LG1   -4.305    -2.932    -3.261     0.598     0.000     0.000    -0.269     0.000     0.000\n",
            "============End report for interfE==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============Begin report for rmsd_chX==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============End report for rmsd_chX==================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER SavePoseMover - spm=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Transform - transform=======================\n",
            "protocols.qsar.scoring_grid.GridManager: Found a conformation matching hash: 8f08f9042fb9bdbd5e8b03bf2921d0cd6038cfa3 Loading from grid cache\n",
            "protocols.ligand_docking.Transform: Considering 1 conformers during sampling\n",
            "protocols.ligand_docking.Transform: percent acceptance: 362 0.1448 2138\n",
            "protocols.ligand_docking.Transform: Moves rejected for being outside of grid: 2131  0.8524\n",
            "protocols.ligand_docking.Transform: [ WARNING ] A large number of moves were rejected for being outside the grid. You likely want to reexamine your settings.\n",
            "protocols.ligand_docking.Transform: [ WARNING ]     For the current settings, a grid size of at least 93.7    and a box size of at least 260.9 are recommended.\n",
            "protocols.ligand_docking.Transform: Accepted pose with grid score: -37\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER HighResDocker - high_res_docker=======================\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 824 rotamers at 34 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 820 rotamers at 32 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 820 rotamers at 32 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -625.558 -625.558\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER FinalMinimizer - final=======================\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: Energy prior to minimizing: -625.558\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: [ WARNING ] Warning: FinalMinimizer with backbone minimization used without chainbreak scoreterm - setting chainbreak to 1\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: Energy after minimizing: -667.68\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN FILTER interfE=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================END FILTER interfE=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN FILTER rmsd_chX=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================END FILTER rmsd_chX=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "protocols.enzdes.EnzFilters: Calculated Interface Energy is before cst correction is:2.93849\n",
            "protocols.enzdes.EnzFilters: Calculated Interface Energy is 2.93849\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============Begin report for interfE==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: \n",
            "    chain      res       AA    total  contact   fa_atr   fa_rep hb_bb_sc    hb_sc   fa_sol   fa_dun  fa_pair\n",
            "        A      123      TYR   -0.069    -1.938    -9.452     1.245     0.000    -0.725     6.994     3.322     0.000\n",
            "        A      124      ASP   -0.500     0.654    -3.840     0.877    -0.904     0.000     4.521     1.477     0.000\n",
            "        A      125      ARG    0.104    -0.435    -5.176     0.215     0.000     0.000     4.526     1.982     0.000\n",
            "        A      127      GLU   -5.505    -0.992    -8.418     0.406    -0.807    -0.444     8.271     2.810     0.000\n",
            "        A      133      LEU   -0.882    -0.909    -5.542     0.510     0.000     0.000     4.123     0.074     0.000\n",
            "        A      134      ARG   -4.636    -3.413   -10.058     0.962    -0.807    -2.070     8.561     2.982     0.000\n",
            "        A      135      GLU   -1.040    -1.004    -4.485     0.619     0.000     0.000     2.862     3.318     0.000\n",
            "        A      202      PRO   -3.208    -0.492    -5.662     1.195     0.000     0.000     3.975     0.523     0.000\n",
            "        A      205      ILE   -3.259    -3.305    -8.254     0.334     0.000     0.000     4.615     0.120     0.000\n",
            "        A      206      THR   -1.680    -1.283    -7.245     0.619    -0.367     0.000     5.710     0.319     0.000\n",
            "        A      208      GLU   -0.127     0.854    -8.690     0.798     0.000    -0.465     9.210     3.424     0.000\n",
            "        A      209      ASN   -1.310    -0.641    -5.717     0.266     0.000    -1.161     5.971     1.320     0.000\n",
            "        A      210      SER   -2.278     0.101    -6.222     0.626    -0.367    -0.131     6.195     1.112     0.000\n",
            "        A      213      ARG    0.269     0.791    -7.217     0.681     0.000    -0.131     7.458     1.736     0.000\n",
            "        A      231      GLN   -1.205    -1.380    -5.071     0.319     0.000     0.000     3.372     2.794     0.000\n",
            "        A      232      LEU   -5.253    -6.565    -7.947     0.754     0.000     0.000     0.628     0.108     0.000\n",
            "        A      233      GLN   -2.164    -2.823    -7.834     0.787    -0.750     0.000     4.974     3.235     0.000\n",
            "        A      234      ARG   -2.573    -1.329    -8.748     0.501     0.000    -0.699     7.617     1.889     0.000\n",
            "        A      237      GLY   -3.491    -0.573    -3.121     0.188    -0.750     0.000     3.111     0.000     0.000\n",
            "        A      239      LYS   -0.184    -0.734    -3.381     0.213     0.000     0.000     2.434     2.399     0.000\n",
            "        B       94      HIS   -0.715    -0.925    -6.001     0.540     0.000     0.000     4.537     1.896     0.000\n",
            "        B       95      PRO   -4.938    -2.761    -5.569     0.654     0.000     0.000     2.153     0.599     0.000\n",
            "        B       96      ASP   -1.838    -0.236    -2.578     0.151     0.000     0.000     2.190     1.643     0.000\n",
            "        B       97      ASN   -2.963    -0.576    -8.291     0.828    -0.630     0.000     7.518     1.489     0.000\n",
            "        B       98      GLN   -0.089    -1.924    -6.649     0.323     0.000     0.000     4.402     4.186     0.000\n",
            "        B       99      GLU    6.730     1.316    -4.985     0.552     0.000     0.000     5.749     9.889     0.000\n",
            "        B      100      ASP   -4.092     0.249    -6.217     0.255    -0.630    -0.461     7.302     1.299     0.000\n",
            "        B      101      ALA   -1.138    -1.006    -4.574     0.530     0.000     0.000     3.038     0.000     0.000\n",
            "        B      102      GLU    0.893     0.294    -4.860     0.403     0.000    -0.919     5.670     4.649     0.000\n",
            "        B      116      THR   -1.574    -2.413    -4.405     0.295     0.000     0.000     1.698     0.080     0.000\n",
            "        X        1      LG1   -9.942    -5.654    -8.859     0.872     0.000     0.000     2.333     0.000     0.000\n",
            "============End report for interfE==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============Begin report for rmsd_chX==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============End report for rmsd_chX==================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER SavePoseMover - spm=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Transform - transform=======================\n",
            "protocols.qsar.scoring_grid.GridManager: Found a conformation matching hash: 8f08f9042fb9bdbd5e8b03bf2921d0cd6038cfa3 Loading from grid cache\n",
            "protocols.ligand_docking.Transform: Considering 1 conformers during sampling\n",
            "protocols.ligand_docking.Transform: percent acceptance: 367 0.1468 2133\n",
            "protocols.ligand_docking.Transform: Moves rejected for being outside of grid: 2127  0.8508\n",
            "protocols.ligand_docking.Transform: [ WARNING ] A large number of moves were rejected for being outside the grid. You likely want to reexamine your settings.\n",
            "protocols.ligand_docking.Transform: [ WARNING ]     For the current settings, a grid size of at least 93.7    and a box size of at least 260.9 are recommended.\n",
            "protocols.ligand_docking.Transform: Accepted pose with grid score: -28\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER HighResDocker - high_res_docker=======================\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 503 rotamers at 27 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 473 rotamers at 24 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 571 rotamers at 23 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -629.841 -631.284\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER FinalMinimizer - final=======================\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: Energy prior to minimizing: -631.284\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: [ WARNING ] Warning: FinalMinimizer with backbone minimization used without chainbreak scoreterm - setting chainbreak to 1\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: Energy after minimizing: -641.489\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN FILTER interfE=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================END FILTER interfE=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN FILTER rmsd_chX=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================END FILTER rmsd_chX=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "protocols.enzdes.EnzFilters: Calculated Interface Energy is before cst correction is:3.74686\n",
            "protocols.enzdes.EnzFilters: Calculated Interface Energy is 3.74686\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============Begin report for interfE==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: \n",
            "    chain      res       AA    total  contact   fa_atr   fa_rep hb_bb_sc    hb_sc   fa_sol   fa_dun  fa_pair\n",
            "        A      123      TYR   -1.125    -2.313    -9.479     0.991     0.000    -0.810     6.985     2.674     0.000\n",
            "        A      124      ASP   -1.244     0.270    -3.655     0.246    -0.710     0.000     4.389     1.283     0.000\n",
            "        A      125      ARG    0.544    -0.310    -5.495     0.402     0.000     0.000     4.783     2.197     0.000\n",
            "        A      127      GLU   -5.134    -0.704    -9.236     0.791    -0.855    -0.467     9.064     3.047     0.000\n",
            "        A      128      GLN    0.162     0.370    -3.912     0.309     0.000     0.000     3.973     2.420     0.000\n",
            "        A      131      GLY   -0.761     0.528    -1.889     0.107     0.000     0.000     2.311     0.000     0.000\n",
            "        A      133      LEU   -1.610    -1.715    -5.954     0.763     0.000     0.000     3.477     0.111     0.000\n",
            "        A      134      ARG   -5.260    -3.484   -10.036     0.933    -0.855    -2.086     8.561     2.931     0.000\n",
            "        A      135      GLU   -1.046    -0.971    -4.318     0.551     0.000     0.000     2.796     3.240     0.000\n",
            "        A      205      ILE   -2.993    -2.961    -7.943     0.234     0.000     0.000     4.749     0.097     0.000\n",
            "        A      209      ASN   -1.004    -0.410    -4.687     0.132     0.000    -1.138     5.283     1.194     0.000\n",
            "        B       93      PHE   -4.771    -6.758    -8.686     0.617     0.000     0.000     1.310     2.234     0.000\n",
            "        B       94      HIS   -3.709    -2.243    -6.937     0.878    -0.685     0.000     4.501     1.403     0.000\n",
            "        B       95      PRO   -2.818    -1.829    -4.644     0.946    -0.685     0.000     2.554     0.856     0.000\n",
            "        B       96      ASP   -0.526     0.385    -1.596     0.029     0.000     0.000     1.951     1.739     0.000\n",
            "        B      117      PHE   -3.742    -6.494    -9.476     1.443     0.000     0.000     1.539     1.981     0.000\n",
            "        B      118      ALA   -0.778    -1.343    -2.750     0.273     0.000     0.000     1.134     0.000     0.000\n",
            "        B      119      PHE   -1.904    -6.378    -8.588     1.465     0.000     0.000     0.745     4.136     0.000\n",
            "        B      120      GLY   -1.043    -1.024    -3.064     0.245     0.000     0.000     1.795     0.000     0.000\n",
            "        B      125      ARG   -0.353    -1.499    -6.878     0.495     0.000    -0.279     5.163     2.450     0.000\n",
            "        X        1      LG1   -4.769    -2.636    -3.100     0.333     0.000     0.000     0.131     0.000     0.000\n",
            "============End report for interfE==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============Begin report for rmsd_chX==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============End report for rmsd_chX==================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER SavePoseMover - spm=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER Transform - transform=======================\n",
            "protocols.qsar.scoring_grid.GridManager: Found a conformation matching hash: 8f08f9042fb9bdbd5e8b03bf2921d0cd6038cfa3 Loading from grid cache\n",
            "protocols.ligand_docking.Transform: Considering 1 conformers during sampling\n",
            "protocols.ligand_docking.Transform: percent acceptance: 350 0.14 2150\n",
            "protocols.ligand_docking.Transform: Moves rejected for being outside of grid: 2145  0.858\n",
            "protocols.ligand_docking.Transform: [ WARNING ] A large number of moves were rejected for being outside the grid. You likely want to reexamine your settings.\n",
            "protocols.ligand_docking.Transform: [ WARNING ]     For the current settings, a grid size of at least 93.7    and a box size of at least 260.9 are recommended.\n",
            "protocols.ligand_docking.Transform: Accepted pose with grid score: -29\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER HighResDocker - high_res_docker=======================\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 173 rotamers at 14 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 173 rotamers at 14 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.pack_rotamers: built 173 rotamers at 14 positions.\n",
            "core.pack.interaction_graph.interaction_graph_factory: Instantiating DensePDInteractionGraph\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "core.pack.task: Packer task: initialize from command line()\n",
            "core.pack.rotamer_set.RotamerSet_: Using simple Rotamer generation logic for pdb_LG1\n",
            "protocols.moves.MonteCarlo: MonteCarlo:: last_accepted_score,lowest_score: -559.838 -561.165\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN MOVER FinalMinimizer - final=======================\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: Energy prior to minimizing: -561.165\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: [ WARNING ] Warning: FinalMinimizer with backbone minimization used without chainbreak scoreterm - setting chainbreak to 1\n",
            "protocols.ligand_docking.ligand_options.FinalMinimizer: Energy after minimizing: -574.019\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN FILTER interfE=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================END FILTER interfE=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================BEGIN FILTER rmsd_chX=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: =======================END FILTER rmsd_chX=======================\n",
            "protocols.rosetta_scripts.ParsedProtocol: setting status to success\n",
            "protocols.enzdes.EnzFilters: Calculated Interface Energy is before cst correction is:-0.375605\n",
            "protocols.enzdes.EnzFilters: Calculated Interface Energy is -0.375605\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============Begin report for interfE==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: \n",
            "    chain      res       AA    total  contact   fa_atr   fa_rep hb_bb_sc    hb_sc   fa_sol   fa_dun  fa_pair\n",
            "        A      132      ASN    3.283    -0.969    -5.490     0.296     0.000     0.000     4.224     2.255     0.000\n",
            "        A      133      LEU   -0.351    -0.834    -5.547     0.504     0.000     0.000     4.209     0.217     0.000\n",
            "        A      134      ARG   -5.260    -3.816   -10.061     0.668    -0.767    -2.171     8.515     2.979     0.000\n",
            "        A      135      GLU   -1.848    -1.683    -5.778     0.682     0.000     0.000     3.414     3.434     0.000\n",
            "        A      136      ASN   -4.017    -2.379    -8.278     0.999     0.000     0.000     4.900     1.956     0.000\n",
            "        A      137      ILE   -5.276    -5.524    -8.591     0.512     0.000     0.000     2.555     0.245     0.000\n",
            "        A      138      GLU   -4.124    -1.971    -6.939     1.015     0.000    -0.460     4.412     2.991     0.000\n",
            "        A      166      ARG    5.416    -2.161    -8.462     0.395     0.000     0.000     5.906     9.513     0.000\n",
            "        A      191      ARG   -0.416    -0.424   -10.920     0.756    -0.160    -0.490    10.390     4.515     0.000\n",
            "        A      196      ARG    9.630     0.318    -4.387     1.158     0.000     0.000     3.546    10.246     0.000\n",
            "        A      197      ARG   -0.520    -1.913    -5.895     0.439     0.000    -1.819     5.361     4.126     0.000\n",
            "        A      198      SER   -2.240     0.690    -5.046     0.500     0.000     0.000     5.235     0.186     0.000\n",
            "        A      199      ALA   -2.373    -2.431    -5.043     0.755    -0.160     0.000     2.017     0.000     0.000\n",
            "        A      200      PRO   -5.670    -3.175    -5.864     0.892     0.000     0.000     1.798     0.837     0.000\n",
            "        A      201      ASP   -2.826     2.456    -6.099     1.100     0.000    -0.546     8.001     1.689     0.000\n",
            "        A      202      PRO   -2.891    -0.594    -5.474     1.073     0.000     0.000     3.807     0.168     0.000\n",
            "        A      235      ARG   -0.798     0.977    -6.997     0.824     0.000    -1.358     8.508     3.291     0.000\n",
            "        X        1      LG1   -6.865    -4.927    -7.329     0.725     0.000     0.000     1.677     0.000     0.000\n",
            "============End report for interfE==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============Begin report for rmsd_chX==================\n",
            "protocols.rosetta_scripts.ParsedProtocol.REPORT: ============End report for rmsd_chX==================\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "colab_type": "text",
        "id": "o63uy8Ch5gCf"
      },
      "source": [
        "Now that we have sampled some global ligand docking trajectories, let's plot the ligand binding energy landscape. The less interface energy the better drug target we found:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab_type": "code",
        "id": "V1sAgnk55gCf",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 519
        },
        "outputId": "5a526ddc-4b9f-4600-c52d-01726acbe175"
      },
      "source": [
        "matplotlib.rcParams['figure.figsize'] = [12.0, 8.0]\n",
        "seaborn.scatterplot(x=\"rmsd_chX\", y=\"interfE\", data=df)"
      ],
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<matplotlib.axes._subplots.AxesSubplot at 0x7fb5d0ad5eb8>"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 15
        },
        {
          "output_type": "display_data",
          "data": {
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAs0AAAHlCAYAAAAOWMFHAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4yLjIsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy+WH4yJAAAbzUlEQVR4nO3df7DVdZ3H8dc55wqBIT+uoIz2a9U1ps0oELelMrFGM0iatTEdnbJyKnTKpjIXXFeNJNrNtML1V2XtNJa6xSTWxLZsY9kP3a3JDMYcMyxBULytoIRy7nf/SO6GgZ+rcM/5wn08/rrne869n/e9nzn69Ov3nNOoqqoKAACwU81uDwAAAHUnmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACnq6PcBg9fU9lv7+vfvd8Xp7n58NGzZ1ewx2wN7Um/2pL3tTb/anvuxN5zWbjYwfv+9O799jorm/v9rroznJsPgd91T2pt7sT33Zm3qzP/Vlb+rF5RkAAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDPs5VqtZqpWM1sbjVStZlotT3sAeLZ6uj0AMHRarWb6Hn8yl1x3e9b3bc6k8aMy/50zMn70Pmm3+7s9HgDsMZxygr3Y1mQgmJNkfd/mXHLd7dna3bEAYI8jmmEv1u6vBoJ5m/V9m9Pur7o0EQDsmUQz7MVazUYmjR+13bFJ40el1Wx0aSIA2DOJZtiL9SSZ/84ZA+G87ZpmL2YAgGfHvzthL9Zu92f86H2yaN7MtPurtJqN9Dx1HAAYPNEMe7l2uz+NPPVkb1dpd3keANgTuTwDAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAU9nVpo3rx5+f3vf59ms5nRo0fnH//xHzNlypROLQ8AAM9Zx6J58eLFGTNmTJLke9/7XubPn59vfvObnVoeAACes45dnrEtmJNk06ZNaTQanVoaAAB2SaOqqqpTiy1YsCC33XZbqqrKtddem8MOO6xTSwMAwHPW0WjeZunSpbnllltyzTXXDPp7NmzYlP7+jo/aURMnjslDD23s9hjsgL2pN/tTX/am3uxPfdmbzms2G+ntff7O7+/gLAPmzp2bn/70p+nr6+vG8gAA8Kx0JJofe+yxrF27duD2ihUrMnbs2IwbN64TywMAsAdotZqpWs1sbTRStZppterz7sgdefeMzZs354Mf/GA2b96cZrOZsWPH5sorr/RiQAAAkvwpmPsefzKXXHd71vdtzqTxozL/nTMyfvQ+abf7uz1eZ6J5//33zw033NCJpQAA2ANtTQaCOUnW923OJdfdnkXzZqYOp1nrc84bAIBhq91fDQTzNuv7NqddkzeCEM0AAHRdq9nIpPGjtjs2afyotJp1OM8smgEAqIGeJPPfOWMgnLdd09yxj68uqMscAAAMY+12f8aP3ieL5s1Mu79Kq9lIz1PH60A0AwBQC+12fxp5KlDbVdpdnufPuTwDAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAAp6OrFIX19fzj333Nx///0ZMWJEXvSiF+Xiiy/OhAkTOrE8AADsko6caW40GnnPe96T7373u7n55pvzghe8IP/yL//SiaUBAGCXdSSax40bl6OOOmrg9tSpU7NmzZpOLA0AALus49c09/f35/rrr8+sWbM6vTQAADwnjaqqqk4ueNFFF2XdunX5/Oc/n2bT6xABAKi/jrwQcJvFixdn9erVufLKK591MG/YsCn9/R3t+46bOHFMHnpoY7fHYAfsTb3Zn/qyN/Vmf+rL3nRes9lIb+/zd3p/x6L50ksvzV133ZWrr746I0aM6NSyAACwyzoSzffcc0+uuuqqvPjFL87b3/72JMnBBx+cJUuWdGJ5AADYJR2J5sMOOyx33313J5YCAIDdzivxAACgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBgl6P5kUce2R1zAABAbRWjecaMGdvdfsc73rHd7Te84Q27dyIAAKiZYjQ/+eST291etWrVdrerqtq9EwEAQM0Uo7nRaOzS/QAAsKfzQkAAACjoKT3giSeeyLnnnjtw+/HHH9/u9hNPPDE0kwEAQE0Uo/l973vfs7oNAAB7m2eM5sWLF+djH/tYkuTHP/5xXv3qV3dkKAAAqJNnvKb5hhtuGPj6rLPOGvJhAACgjp7xTPNLX/rSfOADH8ghhxySJ554IpdffvkOH/fBD35wSIYDAIA6eMZo/uxnP5uvf/3rWbNmTZLkwQcf7MhQAABQJ88Yzb29vZk3b16qqsoTTzyRhQsXptVqdWo2AACohUG9T3Oj0cjy5ct9kAkAAMPSoD/cZMqUKbnvvvuGchYAAKil4vs0bzNjxoyceeaZeetb35oDDzxwu7POJ5100pAMBwAAdTDoaP7Zz36Wgw46KLfffvt2xxuNhmgGAGCvNuho/rd/+7ehnAMAAGpr0Nc0J0lfX1+WLl2aa6+9Nkmybt06b0MHAMBeb9DRfPvtt+f444/PzTffnCVLliRJVq9enQsvvHCoZgMAgFoYdDRfcsklueyyy/KFL3whPT1/uqrjFa94Re68884hGw4AAOpg0NH8wAMP5NWvfnWSDLxzxj777JN2uz00kwEAQE0MOpoPOeSQ/OAHP9ju2I9+9KP89V//9W4fCgAA6mTQ755x3nnn5b3vfW9e//rX549//GMuuOCCrFixIldcccVQzgcAAF036DPNU6dOzbe+9a0ceuih+fu///scfPDBuemmm3LEEUcM5XwAANB1gz7T/IUvfCHvfve7c+aZZ253/Etf+lLOOOOM3T4YAADUxaDPNG97m7mn+9d//dfdNgwAANRR8Uzzj3/84yRJf39/fvKTn6SqqoH7fv/732ffffcduukAAKAGitG8YMGCJMmWLVsyf/78geONRiMTJ07M+eefP3TTAQBADRSjecWKFUmSc889N5/61KeGfCAAAKibQb8Q8M+Dub+/f7v7ms1BXxoNAAB7nEFH869+9atcfPHFufvuu7Nly5YkSVVVaTQaWbVq1ZANCAAA3fasPtzkmGOOySWXXJLnPe95QzkTAADUyqCj+YEHHsiHPvShNBqNoZwHAABqZ9AXI7/xjW/MD3/4w6GcBQAAamnQZ5q3bNmSs88+O9OmTcv++++/3X3eVQMAgL3ZoKP50EMPzaGHHjqUswAAQC09YzTfcccdOfLII5Mk06ZN68hAAABQN88YzRdddFGWLVuW5P8/GfDpGo1G/vM//3P3TwYAADXxjNG8LZiT//9kQAAAGG58lB8AABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAUdiebFixdn1qxZOfzww/PrX/+6E0sCAMBu05FoPvbYY/PVr341Bx10UCeWAwCA3aqnE4tMnz69E8sAAMCQcE0zAAAUdORM8+7Q2/v8bo/QERMnjun2COyEvak3+1Nf9qbe7E992Zt62WOiecOGTenvr7o9xpCaOHFMHnpoY7fHYAfsTb3Zn/qyN/Vmf+rL3nRes9l4xpO0Ls8AAICCjkTzwoUL87rXvS4PPvhgzjjjjLz5zW/uxLIAALBbdOTyjPPPPz/nn39+J5YCAIDdzuUZAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUdCya77vvvpx88sk57rjjcvLJJ+e3v/1tp5YGAIBd0rFo/qd/+qeceuqp+e53v5tTTz01F1xwQaeWBgCAXdKRaN6wYUNWrlyZ2bNnJ0lmz56dlStX5pFHHunE8gAAsEs6Es1r167NAQcckFarlSRptVqZNGlS1q5d24nlAQBgl/R0e4DB6u19frdH6IiJE8d0ewR2wt7Um/2pL3tTb/anvuxNvXQkmidPnpx169al3W6n1Wql3W5n/fr1mTx58qB/xoYNm9LfXw3hlN03ceKYPPTQxm6PwQ7Ym3qzP/Vlb+rN/tSXvem8ZrPxjCdpO3J5Rm9vb6ZMmZJly5YlSZYtW5YpU6ZkwoQJnVgeAAB2Sccuz7jwwgtz3nnn5Yorrsh+++2XxYsXd2ppAADYJR2L5kMOOSQ33nhjp5YDAIDdxicCAgBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEMwAAFIhmAAAoEM0AAFAgmneg1WqmajWztdFI1Wqm1frLP9NgHgMAwN6hp9sD1E2r1Uzf40/mkutuz/q+zZk0flTmv3NGxo/eJ+12/6AfAwDA3sPp0afZmgzEcJKs79ucS667PVuf5WMAANh7iOanafdXAzG8zfq+zWn3V8/qMQAA7D1E89O0mo1MGj9qu2OTxo9Kq9l4Vo8BAGDvIZqfpifJ/HfOGIjibdcr9zzLxwDArvCCc6gXnfc07XZ/xo/eJ4vmzUy7v0qr2UjPU8efzWMA4LnygnOoH//ZugPtdn8a7f70VFUa7f4d/gNqMI8BgOfCC86hfkQzANSMF5xD/YhmAKgZLziH+hHNAFAzXnAO9eP5BwA14wXnUD+iGQBqqN3uTyNP/Yu6XaXd5XlguHN5BgAAFIhmAAAoEM0AAFAgmgEAoEA0AwBAgWgGAIAC0QwAAAWiGQAACkQzAAAUiGYAACgQzQAAUCCaAQCgQDQDAEBBT7cHGKxms9HtETpiuPyeeyJ7U2/2p77sTb3Zn/qyN51V+ns3qqqqOjQLAADskVyeAQAABaIZAAAKRDMAABSIZgAAKBDNAABQIJoBAKBANAMAQIFoBgCAAtEMAAAFohkAAApEcw3813/9V+bOnZsTTzwxb3nLW7J8+fJuj8Sf+f73v5+3vvWtmTNnTk477bT87ne/6/ZIw9bixYsza9asHH744fn1r389cPy+++7LySefnOOOOy4nn3xyfvvb33ZvyGFqZ3uzs+N01o72oa+vL2eeeWaOO+64zJkzJ2effXYeeeSRLk86/OzsOTJv3ry85S1vydy5c3Pqqadm1apVXZySJElFV/X391fTp0+v7r777qqqqmrVqlXV1KlTq3a73eXJqKqq+sMf/lDNmDGj+s1vflNVVVUtXbq0ete73tXlqYavO+64o1qzZk11zDHHDDxnqqqqTj/99Grp0qVVVf1pj04//fRujThs7WxvdnacztrRPvT19VU/+clPBh7zyU9+svqHf/iHbo04bO3sOfLoo48OfP0f//Ef1dy5c7sxHn/GmeYaaDab2bhxY5Jk48aNmTRpUppNW1MHq1evzv7775+XvOQlSZKjjz46P/zhD52N6ZLp06dn8uTJ2x3bsGFDVq5cmdmzZydJZs+enZUrV9qjDtvR3jzTcTprR/swbty4HHXUUQO3p06dmjVr1nR6tGFvZ8+RMWPGDHy9adOmNBqNTo7FDvR0e4DhrtFo5LLLLsu8efMyevToPPbYY7n66qu7PRZPeclLXpKHH344d955Z4444ojcfPPNSZK1a9dmwoQJXZ6O5E97ccABB6TVaiVJWq1WJk2aZI/gWejv78/111+fWbNmdXsU/syCBQty2223paqqXHvttd0eZ9gTzV22devWXHXVVbniiisybdq0/M///E/OOeec3HLLLdl33327Pd6wN2bMmHzmM5/JokWLsmXLlrzuda/LfvvtNxBoAHuDj3/84xk9enROO+20bo/Cn/nEJz6RJFm6dGk+9alP5ZprrunyRMObawC6bNWqVVm/fn2mTZuWJJk2bVpGjRqVe++9t8uTsc3f/d3f5frrr883vvGNnHbaafnjH/+YF77whd0ei6dMnjw569atS7vdTpK02+2sX7/eJQEwSIsXL87q1atz2WWXuTSwpubOnZuf/vSn6evr6/Yow5pnR5cdeOCBefDBB/Ob3/wmSXLvvfdmw4YNoqxGHnrooSR/+t+Xl156ad7+9rdn9OjRXZ6KbXp7ezNlypQsW7YsSbJs2bJMmTLFpRkwCJdeemnuuuuuLFmyJCNGjOj2ODzlsccey9q1awdur1ixImPHjs24ceO6OBWNqqqqbg8x3H3rW9/KNddcM3CR/wc+8IG84Q1v6PJUbLNgwYL87Gc/y5NPPpmZM2dm/vz5GTlyZLfHGpYWLlyY5cuX5+GHH8748eMzbty43HLLLbn33ntz3nnn5dFHH81+++2XxYsX56/+6q+6Pe6wsrO92dlxOmtH+3DZZZdl9uzZefGLX5znPe95SZKDDz44S5Ys6fK0w8uO9ubLX/5y5s2bl82bN6fZbGbs2LH52Mc+lpe97GXdHndYE80AAFDg8gwAACgQzQAAUCCaAQCgQDQDAECBaAYAgALRDAAABaIZYC9y+umn58Ybb3zO3z9r1qz86Ec/2o0TAewdRDMAg7JixYrMnDkzf/jDHwaOfe9738trX/vabNy4sYuTAQw90QzQYVu3bu32CM/JrFmz8rd/+7dZtGhRkuTRRx/NhRdemAsvvDBjxozp8nQAQ0s0A3TArFmzcvXVV2fOnDmZOnVqDj/88Pz7v/97jj766Bx55JG5/vrrc+edd2bOnDmZPn16Lr744oHvXb16dU477bRMmzYtRx11VM4555yB+2677bYcf/zxmTZtWi6++OIM9kNeb7jhhrzpTW/KK1/5ypxwwgn51a9+NXDfqlWrMmfOnEybNi3nnHNOtmzZMnDfggULcuutt+YHP/hBFi1alBkzZuTYY4/dDX8hgHrr6fYAAMPFLbfckquvvjqPP/54TjjhhPziF7/I8uXLc8cdd+T9739/Xvva1+a6667L1q1bM3fu3Bx//PGZMWNGLr/88sycOTNf+cpX8uSTT+aXv/xlkuSRRx7J2WefnUWLFuXYY4/NV7/61Xzta1/LiSee+IxzfOc738nnPve5LFmyJC9/+ctz//33p6enZ7v7r7322owcOTKnnHJKvvGNb+SUU05JkkyYMCELFizIRz7ykTQajXz7298euj8YQI040wzQIaeffnomT56ckSNHJknOOuusjBw5Mq95zWsyevTozJ49O729vTnggAMyffr0rFy5MknS09OTNWvWZP369Rk5cmSmT5+eJLn11ltz2GGH5fjjj88+++yTd7zjHdl///2Lc9x00015z3vekyOOOCKNRiMvetGLctBBB2035wEHHJBx48blmGOOyapVq7b7/le84hXZtGlTZs6cmQkTJuyuPw9ArYlmgA6ZPHnydrd7e3sHvh45cuRf3H788ceTJB/96EdTVVVOOumkvPnNb85NN92UJFm/fn0OPPDAge9pNBp/scaOrF27Ni984Qt3ev/EiRMHvh41atTAHNtccMEFOfHEE3Prrbfm5z//eXE9gL2ByzMAOqTRaDyn75s4cWIWLlyYJPnv//7vnHHGGTnyyCMzceLEPPjggwOPq6oqa9euLf68yZMn5/77739Os9x4441Zu3ZtrrrqqvzN3/xNzj///Hzzm9/MiBEjntPPA9hTONMMUHPf+c53BuJ47NixaTQaaTabOfroo3PPPfdk+fLl2bp1a77yla/k4YcfLv68k046KV/84hdz1113paqqrF69Og888EDx+9atW5d//ud/zsKFCzNixIiccsopGTduXK688spd/h0B6k40A9TcL3/5y7ztbW/LK1/5yrz//e/PggUL8oIXvCATJkzI5Zdfnk9/+tM56qijsnr16rzqVa8q/rw3velNed/73pcPf/jDedWrXpWzzjor//u//1v8vosuuignnHDCwDXVjUYjH//4x/PlL38599xzzy7/ngB11qgG+/5EAAAwTDnTDAAABV4ICLAXuuCCC3LzzTf/xfE5c+Zs98EpAAyOyzMAAKDA5RkAAFAgmgEAoEA0AwBAgWgGAICC/wOB/hxVHK8RTgAAAABJRU5ErkJggg==\n",
            "text/plain": [
              "<Figure size 864x576 with 1 Axes>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "colab_type": "text",
        "id": "4O6WA7yj5gCj"
      },
      "source": [
        "We can check which `.pdb` file has the lowest `interfE` score:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab_type": "code",
        "id": "MmQPJe6P5gCk",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 224
        },
        "outputId": "53f594e1-674c-422a-c6b8-fd5321248a35"
      },
      "source": [
        "df.sort_values(by=\"interfE\")"
      ],
      "execution_count": 16,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>dslf_fa13</th>\n",
              "      <th>fa_atr</th>\n",
              "      <th>fa_dun</th>\n",
              "      <th>fa_elec</th>\n",
              "      <th>fa_intra_rep</th>\n",
              "      <th>fa_intra_sol_xover4</th>\n",
              "      <th>fa_rep</th>\n",
              "      <th>fa_sol</th>\n",
              "      <th>hbond_bb_sc</th>\n",
              "      <th>hbond_lr_bb</th>\n",
              "      <th>hbond_sc</th>\n",
              "      <th>hbond_sr_bb</th>\n",
              "      <th>interfE</th>\n",
              "      <th>lk_ball_wtd</th>\n",
              "      <th>omega</th>\n",
              "      <th>p_aa_pp</th>\n",
              "      <th>pro_close</th>\n",
              "      <th>rama_prepro</th>\n",
              "      <th>ref</th>\n",
              "      <th>rmsd_chX</th>\n",
              "      <th>total_score</th>\n",
              "      <th>yhh_planarity</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>0.0</td>\n",
              "      <td>-3246.610207</td>\n",
              "      <td>1585.290000</td>\n",
              "      <td>-727.945308</td>\n",
              "      <td>1301.132015</td>\n",
              "      <td>109.147282</td>\n",
              "      <td>933.234359</td>\n",
              "      <td>1872.078060</td>\n",
              "      <td>-69.590704</td>\n",
              "      <td>-100.963594</td>\n",
              "      <td>-58.705269</td>\n",
              "      <td>-196.408615</td>\n",
              "      <td>-0.375605</td>\n",
              "      <td>-74.487095</td>\n",
              "      <td>214.995496</td>\n",
              "      <td>-122.728950</td>\n",
              "      <td>50.796050</td>\n",
              "      <td>1.422722</td>\n",
              "      <td>213.48252</td>\n",
              "      <td>11.784042</td>\n",
              "      <td>-574.019256</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>0.0</td>\n",
              "      <td>-3238.146212</td>\n",
              "      <td>1562.937506</td>\n",
              "      <td>-725.588698</td>\n",
              "      <td>1298.238454</td>\n",
              "      <td>109.250323</td>\n",
              "      <td>876.252579</td>\n",
              "      <td>1862.973628</td>\n",
              "      <td>-69.821278</td>\n",
              "      <td>-100.851318</td>\n",
              "      <td>-56.501676</td>\n",
              "      <td>-196.158306</td>\n",
              "      <td>-0.327919</td>\n",
              "      <td>-75.074810</td>\n",
              "      <td>207.696298</td>\n",
              "      <td>-123.495350</td>\n",
              "      <td>51.692238</td>\n",
              "      <td>2.898161</td>\n",
              "      <td>213.48252</td>\n",
              "      <td>7.812201</td>\n",
              "      <td>-619.048682</td>\n",
              "      <td>0.000000</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>0.0</td>\n",
              "      <td>-3234.387877</td>\n",
              "      <td>1566.747124</td>\n",
              "      <td>-724.172184</td>\n",
              "      <td>1303.256749</td>\n",
              "      <td>108.326948</td>\n",
              "      <td>871.750710</td>\n",
              "      <td>1862.691410</td>\n",
              "      <td>-70.723646</td>\n",
              "      <td>-101.288453</td>\n",
              "      <td>-57.018896</td>\n",
              "      <td>-196.009005</td>\n",
              "      <td>-0.323956</td>\n",
              "      <td>-73.762114</td>\n",
              "      <td>215.075495</td>\n",
              "      <td>-122.549088</td>\n",
              "      <td>51.692238</td>\n",
              "      <td>3.783256</td>\n",
              "      <td>213.48252</td>\n",
              "      <td>7.729473</td>\n",
              "      <td>-611.338370</td>\n",
              "      <td>0.003614</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>0.0</td>\n",
              "      <td>-3237.486734</td>\n",
              "      <td>1557.880337</td>\n",
              "      <td>-734.780767</td>\n",
              "      <td>1280.309328</td>\n",
              "      <td>106.937970</td>\n",
              "      <td>825.933446</td>\n",
              "      <td>1865.408021</td>\n",
              "      <td>-74.072568</td>\n",
              "      <td>-100.779933</td>\n",
              "      <td>-56.916882</td>\n",
              "      <td>-195.872898</td>\n",
              "      <td>2.938486</td>\n",
              "      <td>-76.826340</td>\n",
              "      <td>205.349359</td>\n",
              "      <td>-124.543942</td>\n",
              "      <td>50.834541</td>\n",
              "      <td>2.416340</td>\n",
              "      <td>213.48252</td>\n",
              "      <td>13.618117</td>\n",
              "      <td>-667.679505</td>\n",
              "      <td>0.004833</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>0.0</td>\n",
              "      <td>-3226.268422</td>\n",
              "      <td>1546.487115</td>\n",
              "      <td>-732.328791</td>\n",
              "      <td>1298.859229</td>\n",
              "      <td>105.382392</td>\n",
              "      <td>864.996516</td>\n",
              "      <td>1859.182650</td>\n",
              "      <td>-72.427579</td>\n",
              "      <td>-101.155931</td>\n",
              "      <td>-56.956479</td>\n",
              "      <td>-196.974655</td>\n",
              "      <td>3.746858</td>\n",
              "      <td>-75.433060</td>\n",
              "      <td>213.669606</td>\n",
              "      <td>-123.228240</td>\n",
              "      <td>51.714470</td>\n",
              "      <td>2.334085</td>\n",
              "      <td>213.48252</td>\n",
              "      <td>9.473510</td>\n",
              "      <td>-641.488806</td>\n",
              "      <td>0.001380</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "   dslf_fa13       fa_atr       fa_dun  ...   rmsd_chX  total_score  yhh_planarity\n",
              "4        0.0 -3246.610207  1585.290000  ...  11.784042  -574.019256       0.000000\n",
              "0        0.0 -3238.146212  1562.937506  ...   7.812201  -619.048682       0.000000\n",
              "1        0.0 -3234.387877  1566.747124  ...   7.729473  -611.338370       0.003614\n",
              "2        0.0 -3237.486734  1557.880337  ...  13.618117  -667.679505       0.004833\n",
              "3        0.0 -3226.268422  1546.487115  ...   9.473510  -641.488806       0.001380\n",
              "\n",
              "[5 rows x 22 columns]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 16
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "colab_type": "text",
        "id": "TXZnqxjd5gCm"
      },
      "source": [
        "Let's take a look at the pose with the lowest `interfE` value that was generated:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "H-BUhh-fov5S",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 817
        },
        "outputId": "a44a21e7-f6bb-4ad9-baf9-9c40f777a68a"
      },
      "source": [
        "#Skip for tests\n",
        "lowest_energy_pdb_filename = './{}/{}_{}.pdb'.format(output_dir,target_protein, str(df.sort_values(by=\"interfE\").head(1).index[0])) \n",
        "# os.path.join(\"output\", str(df.sort_values(by=\"interfE\").head(1).index[0]))\n",
        "print(lowest_energy_pdb_filename)\n",
        "test_pose = pyrosetta.io.pose_from_file(filename=lowest_energy_pdb_filename)\n",
        "\n",
        "chE = pyrosetta.rosetta.core.select.residue_selector.ChainSelector(\"E\")\n",
        "\n",
        "view = viewer.init(test_pose)\n",
        "view.add(viewer.setStyle())\n",
        "view.add(viewer.setStyle(command=({\"hetflag\": True}, {\"stick\": {\"colorscheme\": \"brownCarbon\", \"radius\": 0.2}})))\n",
        "view.add(viewer.setSurface(residue_selector=chE, opacity=0.7, color='white'))\n",
        "view.add(viewer.setHydrogenBonds())\n",
        "view()"
      ],
      "execution_count": 17,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_15942021952474504\"  style=\"position: relative; width: 1200px; height: 800px\">\n        <p id=\"3dmolwarning_15942021952474504\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n      resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.csb.pitt.edu/build/3Dmol.js');\n}\n\nvar viewer_15942021952474504 = null;\nvar warn = document.getElementById(\"3dmolwarning_15942021952474504\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_15942021952474504 = $3Dmol.createViewer($(\"#3dmolviewer_15942021952474504\"),{backgroundColor:\"white\"});\n\tviewer_15942021952474504.addModels(\"HEADER                                            08-JUL-20   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 08-JUL-20                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.27+release.cd4c8de                                     \\nHETNAM     LG1 E 523  LG1                                                       \\nATOM      1  N   TYR A   1       1.933 -23.542  23.417  1.00  0.00           N  \\nATOM      2  CA  TYR A   1       1.383 -22.219  23.670  1.00  0.00           C  \\nATOM      3  C   TYR A   1      -0.123 -22.278  23.695  1.00  0.00           C  \\nATOM      4  O   TYR A   1      -0.702 -23.130  22.973  1.00  0.00           O  \\nATOM      5  CB  TYR A   1       1.813 -21.241  22.588  1.00  0.00           C  \\nATOM      6  CG  TYR A   1       3.328 -20.953  22.584  1.00  0.00           C  \\nATOM      7  CD1 TYR A   1       3.914 -20.324  23.650  1.00  0.00           C  \\nATOM      8  CD2 TYR A   1       4.121 -21.306  21.504  1.00  0.00           C  \\nATOM      9  CE1 TYR A   1       5.270 -20.103  23.661  1.00  0.00           C  \\nATOM     10  CE2 TYR A   1       5.514 -21.014  21.466  1.00  0.00           C  \\nATOM     11  CZ  TYR A   1       6.039 -20.389  22.583  1.00  0.00           C  \\nATOM     12  OH  TYR A   1       7.368 -20.114  22.655  1.00  0.00           O  \\nATOM     13 1H   TYR A   1       2.932 -23.491  23.403  1.00  0.00           H  \\nATOM     14 2H   TYR A   1       1.642 -24.168  24.140  1.00  0.00           H  \\nATOM     15 3H   TYR A   1       1.604 -23.877  22.534  1.00  0.00           H  \\nATOM     16  HA  TYR A   1       1.763 -21.864  24.628  1.00  0.00           H  \\nATOM     17 1HB  TYR A   1       1.537 -21.636  21.609  1.00  0.00           H  \\nATOM     18 2HB  TYR A   1       1.285 -20.297  22.721  1.00  0.00           H  \\nATOM     19  HD1 TYR A   1       3.304 -19.997  24.493  1.00  0.00           H  \\nATOM     20  HD2 TYR A   1       3.681 -21.822  20.650  1.00  0.00           H  \\nATOM     21  HE1 TYR A   1       5.734 -19.602  24.511  1.00  0.00           H  \\nATOM     22  HE2 TYR A   1       6.103 -21.282  20.589  1.00  0.00           H  \\nATOM     23  HH  TYR A   1       7.800 -20.416  21.853  1.00  0.00           H  \\nATOM     24  N   PRO A   2      -0.781 -21.400  24.462  1.00  0.00           N  \\nATOM     25  CA  PRO A   2      -2.232 -21.470  24.468  1.00  0.00           C  \\nATOM     26  C   PRO A   2      -2.859 -21.124  23.102  1.00  0.00           C  \\nATOM     27  O   PRO A   2      -2.334 -20.299  22.404  1.00  0.00           O  \\nATOM     28  CB  PRO A   2      -2.657 -20.419  25.487  1.00  0.00           C  \\nATOM     29  CG  PRO A   2      -1.424 -20.087  26.272  1.00  0.00           C  \\nATOM     30  CD  PRO A   2      -0.295 -20.301  25.338  1.00  0.00           C  \\nATOM     31  HA  PRO A   2      -2.544 -22.472  24.798  1.00  0.00           H  \\nATOM     32 1HB  PRO A   2      -3.066 -19.538  24.971  1.00  0.00           H  \\nATOM     33 2HB  PRO A   2      -3.458 -20.818  26.127  1.00  0.00           H  \\nATOM     34 1HG  PRO A   2      -1.472 -19.049  26.634  1.00  0.00           H  \\nATOM     35 2HG  PRO A   2      -1.355 -20.731  27.161  1.00  0.00           H  \\nATOM     36 1HD  PRO A   2      -0.111 -19.379  24.767  1.00  0.00           H  \\nATOM     37 2HD  PRO A   2       0.601 -20.591  25.906  1.00  0.00           H  \\nATOM     38  N   ILE A   3      -3.967 -21.745  22.755  1.00  0.00           N  \\nATOM     39  CA  ILE A   3      -4.615 -21.520  21.456  1.00  0.00           C  \\nATOM     40  C   ILE A   3      -5.958 -20.963  21.764  1.00  0.00           C  \\nATOM     41  O   ILE A   3      -6.690 -21.511  22.565  1.00  0.00           O  \\nATOM     42  CB  ILE A   3      -4.758 -22.781  20.539  1.00  0.00           C  \\nATOM     43  CG1 ILE A   3      -3.415 -23.455  20.278  1.00  0.00           C  \\nATOM     44  CG2 ILE A   3      -5.389 -22.400  19.143  1.00  0.00           C  \\nATOM     45  CD1 ILE A   3      -3.509 -24.784  19.530  1.00  0.00           C  \\nATOM     46  H   ILE A   3      -4.382 -22.397  23.406  1.00  0.00           H  \\nATOM     47  HA  ILE A   3      -4.019 -20.804  20.892  1.00  0.00           H  \\nATOM     48  HB  ILE A   3      -5.403 -23.513  21.025  1.00  0.00           H  \\nATOM     49 1HG1 ILE A   3      -2.780 -22.787  19.697  1.00  0.00           H  \\nATOM     50 2HG1 ILE A   3      -2.911 -23.641  21.227  1.00  0.00           H  \\nATOM     51 1HG2 ILE A   3      -5.476 -23.295  18.527  1.00  0.00           H  \\nATOM     52 2HG2 ILE A   3      -6.377 -21.968  19.296  1.00  0.00           H  \\nATOM     53 3HG2 ILE A   3      -4.749 -21.674  18.641  1.00  0.00           H  \\nATOM     54 1HD1 ILE A   3      -2.508 -25.193  19.388  1.00  0.00           H  \\nATOM     55 2HD1 ILE A   3      -4.109 -25.486  20.110  1.00  0.00           H  \\nATOM     56 3HD1 ILE A   3      -3.975 -24.623  18.559  1.00  0.00           H  \\nATOM     57  N   ILE A   4      -6.349 -19.869  21.127  1.00  0.00           N  \\nATOM     58  CA  ILE A   4      -7.675 -19.272  21.346  1.00  0.00           C  \\nATOM     59  C   ILE A   4      -8.359 -19.317  19.941  1.00  0.00           C  \\nATOM     60  O   ILE A   4      -7.746 -18.875  18.961  1.00  0.00           O  \\nATOM     61  CB  ILE A   4      -7.562 -17.794  21.756  1.00  0.00           C  \\nATOM     62  CG1 ILE A   4      -6.999 -17.680  23.152  1.00  0.00           C  \\nATOM     63  CG2 ILE A   4      -8.940 -17.205  21.920  1.00  0.00           C  \\nATOM     64  CD1 ILE A   4      -5.617 -17.266  23.145  1.00  0.00           C  \\nATOM     65  H   ILE A   4      -5.716 -19.434  20.472  1.00  0.00           H  \\nATOM     66  HA  ILE A   4      -8.169 -19.813  22.153  1.00  0.00           H  \\nATOM     67  HB  ILE A   4      -7.024 -17.243  20.986  1.00  0.00           H  \\nATOM     68 1HG1 ILE A   4      -7.586 -16.962  23.723  1.00  0.00           H  \\nATOM     69 2HG1 ILE A   4      -7.081 -18.643  23.657  1.00  0.00           H  \\nATOM     70 1HG2 ILE A   4      -8.856 -16.158  22.211  1.00  0.00           H  \\nATOM     71 2HG2 ILE A   4      -9.481 -17.277  20.977  1.00  0.00           H  \\nATOM     72 3HG2 ILE A   4      -9.481 -17.754  22.691  1.00  0.00           H  \\nATOM     73 1HD1 ILE A   4      -5.252 -17.197  24.170  1.00  0.00           H  \\nATOM     74 2HD1 ILE A   4      -5.021 -17.996  22.596  1.00  0.00           H  \\nATOM     75 3HD1 ILE A   4      -5.532 -16.292  22.665  1.00  0.00           H  \\nATOM     76  N   ASN A   5      -9.578 -19.841  19.843  1.00  0.00           N  \\nATOM     77  CA  ASN A   5     -10.230 -20.006  18.490  1.00  0.00           C  \\nATOM     78  C   ASN A   5     -11.334 -18.982  18.265  1.00  0.00           C  \\nATOM     79  O   ASN A   5     -11.959 -18.460  19.171  1.00  0.00           O  \\nATOM     80  CB  ASN A   5     -10.900 -21.371  18.462  1.00  0.00           C  \\nATOM     81  CG  ASN A   5      -9.992 -22.460  18.877  1.00  0.00           C  \\nATOM     82  OD1 ASN A   5      -9.015 -22.775  18.234  1.00  0.00           O  \\nATOM     83  ND2 ASN A   5     -10.316 -23.049  19.990  1.00  0.00           N  \\nATOM     84  H   ASN A   5     -10.079 -20.133  20.670  1.00  0.00           H  \\nATOM     85  HA  ASN A   5      -9.455 -19.956  17.724  1.00  0.00           H  \\nATOM     86 1HB  ASN A   5     -11.767 -21.365  19.124  1.00  0.00           H  \\nATOM     87 2HB  ASN A   5     -11.260 -21.579  17.454  1.00  0.00           H  \\nATOM     88 1HD2 ASN A   5      -9.753 -23.798  20.342  1.00  0.00           H  \\nATOM     89 2HD2 ASN A   5     -11.127 -22.754  20.494  1.00  0.00           H  \\nATOM     90  N   PHE A   6     -11.436 -18.562  17.004  1.00  0.00           N  \\nATOM     91  CA  PHE A   6     -12.589 -17.732  16.544  1.00  0.00           C  \\nATOM     92  C   PHE A   6     -12.942 -18.177  15.156  1.00  0.00           C  \\nATOM     93  O   PHE A   6     -12.045 -18.462  14.341  1.00  0.00           O  \\nATOM     94  CB  PHE A   6     -12.252 -16.236  16.594  1.00  0.00           C  \\nATOM     95  CG  PHE A   6     -13.388 -15.352  16.144  1.00  0.00           C  \\nATOM     96  CD1 PHE A   6     -14.518 -15.221  16.917  1.00  0.00           C  \\nATOM     97  CD2 PHE A   6     -13.275 -14.653  14.922  1.00  0.00           C  \\nATOM     98  CE1 PHE A   6     -15.538 -14.385  16.540  1.00  0.00           C  \\nATOM     99  CE2 PHE A   6     -14.280 -13.870  14.476  1.00  0.00           C  \\nATOM    100  CZ  PHE A   6     -15.465 -13.747  15.318  1.00  0.00           C  \\nATOM    101  H   PHE A   6     -10.715 -18.812  16.343  1.00  0.00           H  \\nATOM    102  HA  PHE A   6     -13.434 -17.915  17.209  1.00  0.00           H  \\nATOM    103 1HB  PHE A   6     -11.982 -15.958  17.612  1.00  0.00           H  \\nATOM    104 2HB  PHE A   6     -11.389 -16.036  15.960  1.00  0.00           H  \\nATOM    105  HD1 PHE A   6     -14.599 -15.792  17.843  1.00  0.00           H  \\nATOM    106  HD2 PHE A   6     -12.371 -14.773  14.324  1.00  0.00           H  \\nATOM    107  HE1 PHE A   6     -16.390 -14.232  17.202  1.00  0.00           H  \\nATOM    108  HE2 PHE A   6     -14.231 -13.340  13.525  1.00  0.00           H  \\nATOM    109  HZ  PHE A   6     -16.300 -13.142  14.968  1.00  0.00           H  \\nATOM    110  N   THR A   7     -14.232 -18.219  14.889  1.00  0.00           N  \\nATOM    111  CA  THR A   7     -14.636 -18.414  13.513  1.00  0.00           C  \\nATOM    112  C   THR A   7     -15.552 -17.294  13.039  1.00  0.00           C  \\nATOM    113  O   THR A   7     -16.350 -16.796  13.826  1.00  0.00           O  \\nATOM    114  CB  THR A   7     -15.330 -19.740  13.366  1.00  0.00           C  \\nATOM    115  OG1 THR A   7     -15.714 -19.850  11.981  1.00  0.00           O  \\nATOM    116  CG2 THR A   7     -16.601 -19.869  14.179  1.00  0.00           C  \\nATOM    117  H   THR A   7     -14.942 -18.121  15.601  1.00  0.00           H  \\nATOM    118  HA  THR A   7     -13.744 -18.411  12.887  1.00  0.00           H  \\nATOM    119  HB  THR A   7     -14.660 -20.539  13.684  1.00  0.00           H  \\nATOM    120  HG1 THR A   7     -15.435 -19.062  11.509  1.00  0.00           H  \\nATOM    121 1HG2 THR A   7     -17.038 -20.854  14.017  1.00  0.00           H  \\nATOM    122 2HG2 THR A   7     -16.370 -19.744  15.237  1.00  0.00           H  \\nATOM    123 3HG2 THR A   7     -17.310 -19.102  13.869  1.00  0.00           H  \\nATOM    124  N   THR A   8     -15.354 -16.860  11.813  1.00  0.00           N  \\nATOM    125  CA  THR A   8     -16.285 -15.908  11.219  1.00  0.00           C  \\nATOM    126  C   THR A   8     -17.638 -16.570  10.913  1.00  0.00           C  \\nATOM    127  O   THR A   8     -18.639 -15.854  10.743  1.00  0.00           O  \\nATOM    128  CB  THR A   8     -15.727 -15.247   9.995  1.00  0.00           C  \\nATOM    129  OG1 THR A   8     -15.345 -16.295   9.038  1.00  0.00           O  \\nATOM    130  CG2 THR A   8     -14.596 -14.458  10.361  1.00  0.00           C  \\nATOM    131  H   THR A   8     -14.562 -17.177  11.272  1.00  0.00           H  \\nATOM    132  HA  THR A   8     -16.494 -15.127  11.950  1.00  0.00           H  \\nATOM    133  HB  THR A   8     -16.490 -14.612   9.544  1.00  0.00           H  \\nATOM    134  HG1 THR A   8     -15.539 -17.157   9.414  1.00  0.00           H  \\nATOM    135 1HG2 THR A   8     -14.187 -13.975   9.474  1.00  0.00           H  \\nATOM    136 2HG2 THR A   8     -14.900 -13.698  11.081  1.00  0.00           H  \\nATOM    137 3HG2 THR A   8     -13.836 -15.097  10.808  1.00  0.00           H  \\nATOM    138  N   ALA A   9     -17.649 -17.879  10.773  1.00  0.00           N  \\nATOM    139  CA  ALA A   9     -18.906 -18.560  10.352  1.00  0.00           C  \\nATOM    140  C   ALA A   9     -19.972 -18.483  11.496  1.00  0.00           C  \\nATOM    141  O   ALA A   9     -19.783 -19.110  12.576  1.00  0.00           O  \\nATOM    142  CB  ALA A   9     -18.580 -19.942  10.010  1.00  0.00           C  \\nATOM    143  H   ALA A   9     -16.823 -18.434  10.945  1.00  0.00           H  \\nATOM    144  HA  ALA A   9     -19.295 -18.041   9.476  1.00  0.00           H  \\nATOM    145 1HB  ALA A   9     -19.485 -20.463   9.698  1.00  0.00           H  \\nATOM    146 2HB  ALA A   9     -17.855 -19.954   9.196  1.00  0.00           H  \\nATOM    147 3HB  ALA A   9     -18.156 -20.441  10.880  1.00  0.00           H  \\nATOM    148  N   GLY A  10     -21.032 -17.709  11.266  1.00  0.00           N  \\nATOM    149  CA  GLY A  10     -22.101 -17.563  12.289  1.00  0.00           C  \\nATOM    150  C   GLY A  10     -21.632 -16.647  13.433  1.00  0.00           C  \\nATOM    151  O   GLY A  10     -22.203 -16.680  14.508  1.00  0.00           O  \\nATOM    152  H   GLY A  10     -21.118 -17.213  10.390  1.00  0.00           H  \\nATOM    153 1HA  GLY A  10     -22.996 -17.151  11.823  1.00  0.00           H  \\nATOM    154 2HA  GLY A  10     -22.366 -18.544  12.681  1.00  0.00           H  \\nATOM    155  N   ALA A  11     -20.593 -15.828  13.182  1.00  0.00           N  \\nATOM    156  CA  ALA A  11     -20.126 -14.877  14.223  1.00  0.00           C  \\nATOM    157  C   ALA A  11     -21.195 -13.939  14.683  1.00  0.00           C  \\nATOM    158  O   ALA A  11     -21.954 -13.446  13.885  1.00  0.00           O  \\nATOM    159  CB  ALA A  11     -18.987 -14.063  13.662  1.00  0.00           C  \\nATOM    160  H   ALA A  11     -20.122 -15.850  12.289  1.00  0.00           H  \\nATOM    161  HA  ALA A  11     -19.776 -15.456  15.078  1.00  0.00           H  \\nATOM    162 1HB  ALA A  11     -18.635 -13.360  14.418  1.00  0.00           H  \\nATOM    163 2HB  ALA A  11     -18.171 -14.727  13.376  1.00  0.00           H  \\nATOM    164 3HB  ALA A  11     -19.330 -13.513  12.787  1.00  0.00           H  \\nATOM    165  N   THR A  12     -21.263 -13.650  16.014  1.00  0.00           N  \\nATOM    166  CA  THR A  12     -22.151 -12.682  16.502  1.00  0.00           C  \\nATOM    167  C   THR A  12     -21.387 -11.647  17.314  1.00  0.00           C  \\nATOM    168  O   THR A  12     -20.227 -11.823  17.569  1.00  0.00           O  \\nATOM    169  CB  THR A  12     -23.179 -13.386  17.387  1.00  0.00           C  \\nATOM    170  OG1 THR A  12     -22.488 -14.018  18.416  1.00  0.00           O  \\nATOM    171  CG2 THR A  12     -23.881 -14.502  16.638  1.00  0.00           C  \\nATOM    172  H   THR A  12     -20.664 -14.140  16.663  1.00  0.00           H  \\nATOM    173  HA  THR A  12     -22.652 -12.214  15.655  1.00  0.00           H  \\nATOM    174  HB  THR A  12     -23.925 -12.666  17.722  1.00  0.00           H  \\nATOM    175  HG1 THR A  12     -21.547 -13.856  18.315  1.00  0.00           H  \\nATOM    176 1HG2 THR A  12     -24.606 -14.982  17.296  1.00  0.00           H  \\nATOM    177 2HG2 THR A  12     -24.395 -14.090  15.770  1.00  0.00           H  \\nATOM    178 3HG2 THR A  12     -23.147 -15.237  16.310  1.00  0.00           H  \\nATOM    179  N   VAL A  13     -22.036 -10.578  17.769  1.00  0.00           N  \\nATOM    180  CA  VAL A  13     -21.477  -9.633  18.681  1.00  0.00           C  \\nATOM    181  C   VAL A  13     -20.964 -10.386  19.922  1.00  0.00           C  \\nATOM    182  O   VAL A  13     -19.808 -10.112  20.355  1.00  0.00           O  \\nATOM    183  CB  VAL A  13     -22.541  -8.559  19.108  1.00  0.00           C  \\nATOM    184  CG1 VAL A  13     -22.070  -7.634  20.146  1.00  0.00           C  \\nATOM    185  CG2 VAL A  13     -22.894  -7.739  17.903  1.00  0.00           C  \\nATOM    186  H   VAL A  13     -22.981 -10.443  17.438  1.00  0.00           H  \\nATOM    187  HA  VAL A  13     -20.652  -9.120  18.185  1.00  0.00           H  \\nATOM    188  HB  VAL A  13     -23.425  -9.069  19.490  1.00  0.00           H  \\nATOM    189 1HG1 VAL A  13     -22.861  -6.924  20.387  1.00  0.00           H  \\nATOM    190 2HG1 VAL A  13     -21.804  -8.196  21.041  1.00  0.00           H  \\nATOM    191 3HG1 VAL A  13     -21.196  -7.093  19.782  1.00  0.00           H  \\nATOM    192 1HG2 VAL A  13     -23.633  -6.987  18.178  1.00  0.00           H  \\nATOM    193 2HG2 VAL A  13     -21.999  -7.246  17.523  1.00  0.00           H  \\nATOM    194 3HG2 VAL A  13     -23.307  -8.387  17.130  1.00  0.00           H  \\nATOM    195  N   GLN A  14     -21.743 -11.346  20.460  1.00  0.00           N  \\nATOM    196  CA  GLN A  14     -21.296 -12.053  21.685  1.00  0.00           C  \\nATOM    197  C   GLN A  14     -20.074 -12.922  21.389  1.00  0.00           C  \\nATOM    198  O   GLN A  14     -19.163 -12.933  22.251  1.00  0.00           O  \\nATOM    199  CB  GLN A  14     -22.397 -12.923  22.239  1.00  0.00           C  \\nATOM    200  CG  GLN A  14     -22.039 -13.400  23.661  1.00  0.00           C  \\nATOM    201  CD  GLN A  14     -22.068 -12.267  24.733  1.00  0.00           C  \\nATOM    202  OE1 GLN A  14     -23.111 -11.631  24.948  1.00  0.00           O  \\nATOM    203  NE2 GLN A  14     -20.917 -12.006  25.392  1.00  0.00           N  \\nATOM    204  H   GLN A  14     -22.629 -11.595  20.044  1.00  0.00           H  \\nATOM    205  HA  GLN A  14     -21.034 -11.309  22.437  1.00  0.00           H  \\nATOM    206 1HB  GLN A  14     -23.330 -12.360  22.259  1.00  0.00           H  \\nATOM    207 2HB  GLN A  14     -22.546 -13.782  21.585  1.00  0.00           H  \\nATOM    208 1HG  GLN A  14     -22.755 -14.162  23.969  1.00  0.00           H  \\nATOM    209 2HG  GLN A  14     -21.032 -13.817  23.651  1.00  0.00           H  \\nATOM    210 1HE2 GLN A  14     -20.890 -11.287  26.088  1.00  0.00           H  \\nATOM    211 2HE2 GLN A  14     -20.091 -12.531  25.185  1.00  0.00           H  \\nATOM    212  N   SER A  15     -20.026 -13.591  20.258  1.00  0.00           N  \\nATOM    213  CA  SER A  15     -18.919 -14.497  20.063  1.00  0.00           C  \\nATOM    214  C   SER A  15     -17.629 -13.721  19.783  1.00  0.00           C  \\nATOM    215  O   SER A  15     -16.578 -14.194  20.203  1.00  0.00           O  \\nATOM    216  CB  SER A  15     -19.204 -15.517  18.992  1.00  0.00           C  \\nATOM    217  OG  SER A  15     -19.099 -14.953  17.657  1.00  0.00           O  \\nATOM    218  H   SER A  15     -20.721 -13.499  19.531  1.00  0.00           H  \\nATOM    219  HA  SER A  15     -18.742 -15.033  20.997  1.00  0.00           H  \\nATOM    220 1HB  SER A  15     -18.503 -16.346  19.084  1.00  0.00           H  \\nATOM    221 2HB  SER A  15     -20.207 -15.919  19.130  1.00  0.00           H  \\nATOM    222  HG  SER A  15     -18.864 -14.030  17.777  1.00  0.00           H  \\nATOM    223  N   TYR A  16     -17.739 -12.584  19.121  1.00  0.00           N  \\nATOM    224  CA  TYR A  16     -16.534 -11.714  18.965  1.00  0.00           C  \\nATOM    225  C   TYR A  16     -16.088 -11.136  20.268  1.00  0.00           C  \\nATOM    226  O   TYR A  16     -14.896 -11.157  20.558  1.00  0.00           O  \\nATOM    227  CB  TYR A  16     -16.831 -10.635  17.919  1.00  0.00           C  \\nATOM    228  CG  TYR A  16     -15.648  -9.723  17.676  1.00  0.00           C  \\nATOM    229  CD1 TYR A  16     -14.517 -10.150  16.930  1.00  0.00           C  \\nATOM    230  CD2 TYR A  16     -15.635  -8.473  18.256  1.00  0.00           C  \\nATOM    231  CE1 TYR A  16     -13.436  -9.303  16.797  1.00  0.00           C  \\nATOM    232  CE2 TYR A  16     -14.544  -7.676  18.103  1.00  0.00           C  \\nATOM    233  CZ  TYR A  16     -13.476  -8.099  17.357  1.00  0.00           C  \\nATOM    234  OH  TYR A  16     -12.397  -7.254  17.260  1.00  0.00           O  \\nATOM    235  H   TYR A  16     -18.616 -12.287  18.717  1.00  0.00           H  \\nATOM    236  HA  TYR A  16     -15.704 -12.332  18.622  1.00  0.00           H  \\nATOM    237 1HB  TYR A  16     -17.111 -11.108  16.977  1.00  0.00           H  \\nATOM    238 2HB  TYR A  16     -17.678 -10.034  18.247  1.00  0.00           H  \\nATOM    239  HD1 TYR A  16     -14.509 -11.138  16.470  1.00  0.00           H  \\nATOM    240  HD2 TYR A  16     -16.494  -8.131  18.833  1.00  0.00           H  \\nATOM    241  HE1 TYR A  16     -12.563  -9.619  16.227  1.00  0.00           H  \\nATOM    242  HE2 TYR A  16     -14.515  -6.694  18.576  1.00  0.00           H  \\nATOM    243  HH  TYR A  16     -12.573  -6.453  17.760  1.00  0.00           H  \\nATOM    244  N   THR A  17     -17.038 -10.599  21.005  1.00  0.00           N  \\nATOM    245  CA  THR A  17     -16.763 -10.039  22.324  1.00  0.00           C  \\nATOM    246  C   THR A  17     -16.084 -11.119  23.186  1.00  0.00           C  \\nATOM    247  O   THR A  17     -15.037 -10.837  23.803  1.00  0.00           O  \\nATOM    248  CB  THR A  17     -18.008  -9.613  22.989  1.00  0.00           C  \\nATOM    249  OG1 THR A  17     -18.584  -8.542  22.186  1.00  0.00           O  \\nATOM    250  CG2 THR A  17     -17.629  -9.092  24.382  1.00  0.00           C  \\nATOM    251  H   THR A  17     -17.982 -10.574  20.647  1.00  0.00           H  \\nATOM    252  HA  THR A  17     -16.122  -9.165  22.202  1.00  0.00           H  \\nATOM    253  HB  THR A  17     -18.688 -10.461  23.066  1.00  0.00           H  \\nATOM    254  HG1 THR A  17     -18.021  -8.374  21.427  1.00  0.00           H  \\nATOM    255 1HG2 THR A  17     -18.528  -8.767  24.906  1.00  0.00           H  \\nATOM    256 2HG2 THR A  17     -17.146  -9.888  24.949  1.00  0.00           H  \\nATOM    257 3HG2 THR A  17     -16.944  -8.251  24.282  1.00  0.00           H  \\nATOM    258  N   ASN A  18     -16.619 -12.332  23.240  1.00  0.00           N  \\nATOM    259  CA  ASN A  18     -15.972 -13.311  24.060  1.00  0.00           C  \\nATOM    260  C   ASN A  18     -14.614 -13.709  23.585  1.00  0.00           C  \\nATOM    261  O   ASN A  18     -13.721 -13.920  24.398  1.00  0.00           O  \\nATOM    262  CB  ASN A  18     -16.859 -14.569  24.163  1.00  0.00           C  \\nATOM    263  CG  ASN A  18     -18.141 -14.359  25.007  1.00  0.00           C  \\nATOM    264  OD1 ASN A  18     -18.294 -13.400  25.732  1.00  0.00           O  \\nATOM    265  ND2 ASN A  18     -19.090 -15.281  24.819  1.00  0.00           N  \\nATOM    266  H   ASN A  18     -17.453 -12.589  22.732  1.00  0.00           H  \\nATOM    267  HA  ASN A  18     -15.833 -12.890  25.057  1.00  0.00           H  \\nATOM    268 1HB  ASN A  18     -17.157 -14.887  23.163  1.00  0.00           H  \\nATOM    269 2HB  ASN A  18     -16.286 -15.382  24.609  1.00  0.00           H  \\nATOM    270 1HD2 ASN A  18     -19.952 -15.222  25.324  1.00  0.00           H  \\nATOM    271 2HD2 ASN A  18     -18.939 -16.029  24.174  1.00  0.00           H  \\nATOM    272  N   PHE A  19     -14.381 -13.733  22.283  1.00  0.00           N  \\nATOM    273  CA  PHE A  19     -13.087 -14.004  21.728  1.00  0.00           C  \\nATOM    274  C   PHE A  19     -12.096 -12.962  22.184  1.00  0.00           C  \\nATOM    275  O   PHE A  19     -11.024 -13.337  22.625  1.00  0.00           O  \\nATOM    276  CB  PHE A  19     -13.235 -14.054  20.216  1.00  0.00           C  \\nATOM    277  CG  PHE A  19     -11.950 -14.018  19.481  1.00  0.00           C  \\nATOM    278  CD1 PHE A  19     -10.985 -14.970  19.737  1.00  0.00           C  \\nATOM    279  CD2 PHE A  19     -11.722 -13.018  18.519  1.00  0.00           C  \\nATOM    280  CE1 PHE A  19      -9.788 -14.947  18.981  1.00  0.00           C  \\nATOM    281  CE2 PHE A  19     -10.470 -12.968  17.799  1.00  0.00           C  \\nATOM    282  CZ  PHE A  19      -9.521 -13.954  18.048  1.00  0.00           C  \\nATOM    283  H   PHE A  19     -15.155 -13.551  21.660  1.00  0.00           H  \\nATOM    284  HA  PHE A  19     -12.746 -14.970  22.101  1.00  0.00           H  \\nATOM    285 1HB  PHE A  19     -13.760 -14.965  19.932  1.00  0.00           H  \\nATOM    286 2HB  PHE A  19     -13.838 -13.211  19.882  1.00  0.00           H  \\nATOM    287  HD1 PHE A  19     -11.165 -15.716  20.512  1.00  0.00           H  \\nATOM    288  HD2 PHE A  19     -12.483 -12.257  18.343  1.00  0.00           H  \\nATOM    289  HE1 PHE A  19      -9.040 -15.727  19.122  1.00  0.00           H  \\nATOM    290  HE2 PHE A  19     -10.260 -12.181  17.075  1.00  0.00           H  \\nATOM    291  HZ  PHE A  19      -8.571 -13.947  17.515  1.00  0.00           H  \\nATOM    292  N   ILE A  20     -12.408 -11.687  22.009  1.00  0.00           N  \\nATOM    293  CA  ILE A  20     -11.452 -10.624  22.391  1.00  0.00           C  \\nATOM    294  C   ILE A  20     -11.164 -10.692  23.856  1.00  0.00           C  \\nATOM    295  O   ILE A  20     -10.003 -10.512  24.251  1.00  0.00           O  \\nATOM    296  CB  ILE A  20     -11.941  -9.274  21.893  1.00  0.00           C  \\nATOM    297  CG1 ILE A  20     -11.933  -9.171  20.356  1.00  0.00           C  \\nATOM    298  CG2 ILE A  20     -11.096  -8.144  22.422  1.00  0.00           C  \\nATOM    299  CD1 ILE A  20     -10.654  -9.540  19.591  1.00  0.00           C  \\nATOM    300  H   ILE A  20     -13.300 -11.427  21.614  1.00  0.00           H  \\nATOM    301  HA  ILE A  20     -10.490 -10.841  21.928  1.00  0.00           H  \\nATOM    302  HB  ILE A  20     -12.968  -9.116  22.221  1.00  0.00           H  \\nATOM    303 1HG1 ILE A  20     -12.712  -9.812  19.945  1.00  0.00           H  \\nATOM    304 2HG1 ILE A  20     -12.162  -8.147  20.059  1.00  0.00           H  \\nATOM    305 1HG2 ILE A  20     -11.477  -7.196  22.043  1.00  0.00           H  \\nATOM    306 2HG2 ILE A  20     -11.134  -8.141  23.510  1.00  0.00           H  \\nATOM    307 3HG2 ILE A  20     -10.065  -8.277  22.095  1.00  0.00           H  \\nATOM    308 1HD1 ILE A  20     -10.820  -9.411  18.521  1.00  0.00           H  \\nATOM    309 2HD1 ILE A  20      -9.838  -8.892  19.912  1.00  0.00           H  \\nATOM    310 3HD1 ILE A  20     -10.395 -10.578  19.795  1.00  0.00           H  \\nATOM    311  N   ARG A  21     -12.198 -10.967  24.668  1.00  0.00           N  \\nATOM    312  CA  ARG A  21     -11.945 -11.081  26.117  1.00  0.00           C  \\nATOM    313  C   ARG A  21     -11.065 -12.264  26.390  1.00  0.00           C  \\nATOM    314  O   ARG A  21     -10.189 -12.148  27.305  1.00  0.00           O  \\nATOM    315  CB  ARG A  21     -13.271 -11.202  26.870  1.00  0.00           C  \\nATOM    316  CG  ARG A  21     -14.037  -9.904  26.944  1.00  0.00           C  \\nATOM    317  CD  ARG A  21     -15.403 -10.215  27.484  1.00  0.00           C  \\nATOM    318  NE  ARG A  21     -16.189  -8.990  27.583  1.00  0.00           N  \\nATOM    319  CZ  ARG A  21     -17.478  -8.973  27.940  1.00  0.00           C  \\nATOM    320  NH1 ARG A  21     -18.092 -10.123  28.223  1.00  0.00           N  \\nATOM    321  NH2 ARG A  21     -18.163  -7.811  27.964  1.00  0.00           N  \\nATOM    322  H   ARG A  21     -13.137 -11.097  24.321  1.00  0.00           H  \\nATOM    323  HA  ARG A  21     -11.430 -10.179  26.450  1.00  0.00           H  \\nATOM    324 1HB  ARG A  21     -13.900 -11.945  26.382  1.00  0.00           H  \\nATOM    325 2HB  ARG A  21     -13.082 -11.549  27.886  1.00  0.00           H  \\nATOM    326 1HG  ARG A  21     -13.516  -9.211  27.605  1.00  0.00           H  \\nATOM    327 2HG  ARG A  21     -14.112  -9.468  25.947  1.00  0.00           H  \\nATOM    328 1HD  ARG A  21     -15.908 -10.912  26.816  1.00  0.00           H  \\nATOM    329 2HD  ARG A  21     -15.310 -10.663  28.473  1.00  0.00           H  \\nATOM    330  HE  ARG A  21     -15.727  -8.116  27.369  1.00  0.00           H  \\nATOM    331 1HH1 ARG A  21     -17.584 -10.994  28.167  1.00  0.00           H  \\nATOM    332 2HH1 ARG A  21     -19.065 -10.123  28.494  1.00  0.00           H  \\nATOM    333 1HH2 ARG A  21     -17.702  -6.947  27.711  1.00  0.00           H  \\nATOM    334 2HH2 ARG A  21     -19.135  -7.804  28.234  1.00  0.00           H  \\nATOM    335  N   ALA A  22     -11.178 -13.383  25.690  1.00  0.00           N  \\nATOM    336  CA  ALA A  22     -10.257 -14.535  25.907  1.00  0.00           C  \\nATOM    337  C   ALA A  22      -8.845 -14.190  25.525  1.00  0.00           C  \\nATOM    338  O   ALA A  22      -7.926 -14.601  26.216  1.00  0.00           O  \\nATOM    339  CB  ALA A  22     -10.704 -15.721  25.088  1.00  0.00           C  \\nATOM    340  H   ALA A  22     -11.905 -13.459  24.993  1.00  0.00           H  \\nATOM    341  HA  ALA A  22     -10.290 -14.801  26.964  1.00  0.00           H  \\nATOM    342 1HB  ALA A  22     -10.023 -16.556  25.255  1.00  0.00           H  \\nATOM    343 2HB  ALA A  22     -11.712 -16.010  25.386  1.00  0.00           H  \\nATOM    344 3HB  ALA A  22     -10.700 -15.456  24.032  1.00  0.00           H  \\nATOM    345  N   VAL A  23      -8.661 -13.503  24.401  1.00  0.00           N  \\nATOM    346  CA  VAL A  23      -7.301 -13.089  24.006  1.00  0.00           C  \\nATOM    347  C   VAL A  23      -6.691 -12.232  25.110  1.00  0.00           C  \\nATOM    348  O   VAL A  23      -5.563 -12.476  25.456  1.00  0.00           O  \\nATOM    349  CB  VAL A  23      -7.332 -12.354  22.666  1.00  0.00           C  \\nATOM    350  CG1 VAL A  23      -5.942 -11.762  22.368  1.00  0.00           C  \\nATOM    351  CG2 VAL A  23      -7.842 -13.297  21.566  1.00  0.00           C  \\nATOM    352  H   VAL A  23      -9.441 -13.258  23.809  1.00  0.00           H  \\nATOM    353  HA  VAL A  23      -6.685 -13.982  23.899  1.00  0.00           H  \\nATOM    354  HB  VAL A  23      -7.999 -11.495  22.748  1.00  0.00           H  \\nATOM    355 1HG1 VAL A  23      -5.967 -11.238  21.412  1.00  0.00           H  \\nATOM    356 2HG1 VAL A  23      -5.668 -11.063  23.157  1.00  0.00           H  \\nATOM    357 3HG1 VAL A  23      -5.207 -12.565  22.322  1.00  0.00           H  \\nATOM    358 1HG2 VAL A  23      -7.863 -12.768  20.614  1.00  0.00           H  \\nATOM    359 2HG2 VAL A  23      -7.178 -14.158  21.489  1.00  0.00           H  \\nATOM    360 3HG2 VAL A  23      -8.848 -13.635  21.814  1.00  0.00           H  \\nATOM    361  N   ARG A  24      -7.388 -11.180  25.548  1.00  0.00           N  \\nATOM    362  CA  ARG A  24      -6.858 -10.350  26.603  1.00  0.00           C  \\nATOM    363  C   ARG A  24      -6.524 -11.148  27.817  1.00  0.00           C  \\nATOM    364  O   ARG A  24      -5.444 -10.884  28.462  1.00  0.00           O  \\nATOM    365  CB  ARG A  24      -7.836  -9.319  27.061  1.00  0.00           C  \\nATOM    366  CG  ARG A  24      -8.095  -8.291  26.031  1.00  0.00           C  \\nATOM    367  CD  ARG A  24      -9.104  -7.256  26.583  1.00  0.00           C  \\nATOM    368  NE  ARG A  24      -9.407  -6.297  25.538  1.00  0.00           N  \\nATOM    369  CZ  ARG A  24     -10.624  -5.840  25.256  1.00  0.00           C  \\nATOM    370  NH1 ARG A  24     -11.667  -6.178  26.007  1.00  0.00           N  \\nATOM    371  NH2 ARG A  24     -10.842  -5.048  24.253  1.00  0.00           N  \\nATOM    372  H   ARG A  24      -8.288 -10.954  25.149  1.00  0.00           H  \\nATOM    373  HA  ARG A  24      -5.979  -9.828  26.224  1.00  0.00           H  \\nATOM    374 1HB  ARG A  24      -8.777  -9.801  27.323  1.00  0.00           H  \\nATOM    375 2HB  ARG A  24      -7.457  -8.832  27.959  1.00  0.00           H  \\nATOM    376 1HG  ARG A  24      -7.162  -7.789  25.775  1.00  0.00           H  \\nATOM    377 2HG  ARG A  24      -8.509  -8.765  25.140  1.00  0.00           H  \\nATOM    378 1HD  ARG A  24     -10.016  -7.766  26.892  1.00  0.00           H  \\nATOM    379 2HD  ARG A  24      -8.667  -6.743  27.439  1.00  0.00           H  \\nATOM    380  HE  ARG A  24      -8.637  -5.949  24.982  1.00  0.00           H  \\nATOM    381 1HH1 ARG A  24     -11.540  -6.788  26.802  1.00  0.00           H  \\nATOM    382 2HH1 ARG A  24     -12.586  -5.825  25.783  1.00  0.00           H  \\nATOM    383 1HH2 ARG A  24     -10.075  -4.761  23.660  1.00  0.00           H  \\nATOM    384 2HH2 ARG A  24     -11.777  -4.718  24.065  1.00  0.00           H  \\nATOM    385  N   GLY A  25      -7.411 -12.038  28.218  1.00  0.00           N  \\nATOM    386  CA  GLY A  25      -7.206 -12.848  29.442  1.00  0.00           C  \\nATOM    387  C   GLY A  25      -6.051 -13.792  29.321  1.00  0.00           C  \\nATOM    388  O   GLY A  25      -5.410 -14.102  30.359  1.00  0.00           O  \\nATOM    389  H   GLY A  25      -8.253 -12.171  27.676  1.00  0.00           H  \\nATOM    390 1HA  GLY A  25      -7.035 -12.187  30.292  1.00  0.00           H  \\nATOM    391 2HA  GLY A  25      -8.109 -13.419  29.656  1.00  0.00           H  \\nATOM    392  N   ARG A  26      -5.678 -14.213  28.126  1.00  0.00           N  \\nATOM    393  CA  ARG A  26      -4.493 -15.053  27.963  1.00  0.00           C  \\nATOM    394  C   ARG A  26      -3.163 -14.286  27.716  1.00  0.00           C  \\nATOM    395  O   ARG A  26      -2.061 -14.852  27.818  1.00  0.00           O  \\nATOM    396  CB  ARG A  26      -4.719 -16.116  26.878  1.00  0.00           C  \\nATOM    397  CG  ARG A  26      -5.888 -17.024  27.280  1.00  0.00           C  \\nATOM    398  CD  ARG A  26      -5.504 -17.956  28.516  1.00  0.00           C  \\nATOM    399  NE  ARG A  26      -5.915 -19.264  28.122  1.00  0.00           N  \\nATOM    400  CZ  ARG A  26      -5.269 -20.380  28.281  1.00  0.00           C  \\nATOM    401  NH1 ARG A  26      -4.042 -20.438  28.835  1.00  0.00           N  \\nATOM    402  NH2 ARG A  26      -5.854 -21.453  27.797  1.00  0.00           N  \\nATOM    403  H   ARG A  26      -6.214 -13.955  27.310  1.00  0.00           H  \\nATOM    404  HA  ARG A  26      -4.296 -15.561  28.908  1.00  0.00           H  \\nATOM    405 1HB  ARG A  26      -4.929 -15.627  25.928  1.00  0.00           H  \\nATOM    406 2HB  ARG A  26      -3.810 -16.704  26.749  1.00  0.00           H  \\nATOM    407 1HG  ARG A  26      -6.744 -16.411  27.563  1.00  0.00           H  \\nATOM    408 2HG  ARG A  26      -6.160 -17.662  26.438  1.00  0.00           H  \\nATOM    409 1HD  ARG A  26      -4.430 -17.904  28.691  1.00  0.00           H  \\nATOM    410 2HD  ARG A  26      -6.033 -17.616  29.406  1.00  0.00           H  \\nATOM    411  HE  ARG A  26      -6.811 -19.357  27.663  1.00  0.00           H  \\nATOM    412 1HH1 ARG A  26      -3.588 -19.591  29.146  1.00  0.00           H  \\nATOM    413 2HH1 ARG A  26      -3.577 -21.328  28.939  1.00  0.00           H  \\nATOM    414 1HH2 ARG A  26      -6.750 -21.376  27.336  1.00  0.00           H  \\nATOM    415 2HH2 ARG A  26      -5.408 -22.354  27.885  1.00  0.00           H  \\nATOM    416  N   LEU A  27      -3.290 -13.025  27.332  1.00  0.00           N  \\nATOM    417  CA  LEU A  27      -2.121 -12.204  27.105  1.00  0.00           C  \\nATOM    418  C   LEU A  27      -1.569 -11.754  28.411  1.00  0.00           C  \\nATOM    419  O   LEU A  27      -0.337 -11.692  28.494  1.00  0.00           O  \\nATOM    420  CB  LEU A  27      -2.491 -10.960  26.286  1.00  0.00           C  \\nATOM    421  CG  LEU A  27      -2.639 -11.225  24.810  1.00  0.00           C  \\nATOM    422  CD1 LEU A  27      -3.255 -10.033  24.200  1.00  0.00           C  \\nATOM    423  CD2 LEU A  27      -1.309 -11.555  24.023  1.00  0.00           C  \\nATOM    424  H   LEU A  27      -4.207 -12.624  27.194  1.00  0.00           H  \\nATOM    425  HA  LEU A  27      -1.393 -12.788  26.543  1.00  0.00           H  \\nATOM    426 1HB  LEU A  27      -3.431 -10.562  26.665  1.00  0.00           H  \\nATOM    427 2HB  LEU A  27      -1.717 -10.206  26.430  1.00  0.00           H  \\nATOM    428  HG  LEU A  27      -3.298 -12.080  24.658  1.00  0.00           H  \\nATOM    429 1HD1 LEU A  27      -3.376 -10.194  23.129  1.00  0.00           H  \\nATOM    430 2HD1 LEU A  27      -4.231  -9.856  24.652  1.00  0.00           H  \\nATOM    431 3HD1 LEU A  27      -2.615  -9.167  24.367  1.00  0.00           H  \\nATOM    432 1HD2 LEU A  27      -1.541 -11.728  22.972  1.00  0.00           H  \\nATOM    433 2HD2 LEU A  27      -0.618 -10.716  24.109  1.00  0.00           H  \\nATOM    434 3HD2 LEU A  27      -0.849 -12.449  24.444  1.00  0.00           H  \\nATOM    435  N   THR A  28      -2.410 -11.488  29.424  1.00  0.00           N  \\nATOM    436  CA  THR A  28      -1.959 -10.866  30.703  1.00  0.00           C  \\nATOM    437  C   THR A  28      -2.170 -11.827  31.871  1.00  0.00           C  \\nATOM    438  O   THR A  28      -3.002 -12.664  31.799  1.00  0.00           O  \\nATOM    439  CB  THR A  28      -2.801  -9.589  31.103  1.00  0.00           C  \\nATOM    440  OG1 THR A  28      -4.212  -9.921  31.241  1.00  0.00           O  \\nATOM    441  CG2 THR A  28      -2.693  -8.480  30.102  1.00  0.00           C  \\nATOM    442  H   THR A  28      -3.386 -11.721  29.308  1.00  0.00           H  \\nATOM    443  HA  THR A  28      -0.922 -10.549  30.587  1.00  0.00           H  \\nATOM    444  HB  THR A  28      -2.452  -9.209  32.063  1.00  0.00           H  \\nATOM    445  HG1 THR A  28      -4.340 -10.855  31.059  1.00  0.00           H  \\nATOM    446 1HG2 THR A  28      -3.292  -7.632  30.433  1.00  0.00           H  \\nATOM    447 2HG2 THR A  28      -1.651  -8.174  30.009  1.00  0.00           H  \\nATOM    448 3HG2 THR A  28      -3.057  -8.826  29.135  1.00  0.00           H  \\nATOM    449  N   THR A  29      -1.423 -11.649  32.936  1.00  0.00           N  \\nATOM    450  CA  THR A  29      -1.629 -12.426  34.156  1.00  0.00           C  \\nATOM    451  C   THR A  29      -2.754 -11.794  35.026  1.00  0.00           C  \\nATOM    452  O   THR A  29      -3.320 -12.447  35.939  1.00  0.00           O  \\nATOM    453  CB  THR A  29      -0.320 -12.406  34.947  1.00  0.00           C  \\nATOM    454  OG1 THR A  29      -0.145 -11.095  35.517  1.00  0.00           O  \\nATOM    455  CG2 THR A  29       0.866 -12.700  33.981  1.00  0.00           C  \\nATOM    456  H   THR A  29      -0.687 -10.957  32.911  1.00  0.00           H  \\nATOM    457  HA  THR A  29      -1.879 -13.450  33.876  1.00  0.00           H  \\nATOM    458  HB  THR A  29      -0.356 -13.164  35.730  1.00  0.00           H  \\nATOM    459  HG1 THR A  29      -0.887 -10.538  35.270  1.00  0.00           H  \\nATOM    460 1HG2 THR A  29       1.802 -12.687  34.539  1.00  0.00           H  \\nATOM    461 2HG2 THR A  29       0.729 -13.680  33.524  1.00  0.00           H  \\nATOM    462 3HG2 THR A  29       0.898 -11.939  33.203  1.00  0.00           H  \\nATOM    463  N   GLY A  30      -3.068 -10.518  34.782  1.00  0.00           N  \\nATOM    464  CA  GLY A  30      -3.999  -9.833  35.660  1.00  0.00           C  \\nATOM    465  C   GLY A  30      -3.421  -9.410  37.016  1.00  0.00           C  \\nATOM    466  O   GLY A  30      -4.079  -8.735  37.797  1.00  0.00           O  \\nATOM    467  H   GLY A  30      -2.672 -10.018  33.999  1.00  0.00           H  \\nATOM    468 1HA  GLY A  30      -4.373  -8.936  35.165  1.00  0.00           H  \\nATOM    469 2HA  GLY A  30      -4.857 -10.476  35.852  1.00  0.00           H  \\nATOM    470  N   ALA A  31      -2.180  -9.793  37.310  1.00  0.00           N  \\nATOM    471  CA  ALA A  31      -1.513  -9.365  38.567  1.00  0.00           C  \\nATOM    472  C   ALA A  31      -1.029  -7.945  38.466  1.00  0.00           C  \\nATOM    473  O   ALA A  31      -0.942  -7.219  39.482  1.00  0.00           O  \\nATOM    474  CB  ALA A  31      -0.384 -10.263  38.849  1.00  0.00           C  \\nATOM    475  H   ALA A  31      -1.679 -10.388  36.665  1.00  0.00           H  \\nATOM    476  HA  ALA A  31      -2.240  -9.433  39.376  1.00  0.00           H  \\nATOM    477 1HB  ALA A  31       0.107  -9.951  39.771  1.00  0.00           H  \\nATOM    478 2HB  ALA A  31      -0.750 -11.284  38.960  1.00  0.00           H  \\nATOM    479 3HB  ALA A  31       0.328 -10.221  38.027  1.00  0.00           H  \\nATOM    480  N   ASP A  32      -0.750  -7.477  37.232  1.00  0.00           N  \\nATOM    481  CA  ASP A  32      -0.113  -6.171  37.067  1.00  0.00           C  \\nATOM    482  C   ASP A  32      -1.112  -5.187  36.375  1.00  0.00           C  \\nATOM    483  O   ASP A  32      -1.425  -5.260  35.203  1.00  0.00           O  \\nATOM    484  CB  ASP A  32       1.225  -6.304  36.330  1.00  0.00           C  \\nATOM    485  CG  ASP A  32       1.996  -4.932  36.148  1.00  0.00           C  \\nATOM    486  OD1 ASP A  32       1.575  -3.907  36.750  1.00  0.00           O  \\nATOM    487  OD2 ASP A  32       3.052  -4.915  35.527  1.00  0.00           O  \\nATOM    488  H   ASP A  32      -0.977  -8.021  36.412  1.00  0.00           H  \\nATOM    489  HA  ASP A  32       0.078  -5.752  38.055  1.00  0.00           H  \\nATOM    490 1HB  ASP A  32       1.875  -6.989  36.876  1.00  0.00           H  \\nATOM    491 2HB  ASP A  32       1.055  -6.732  35.342  1.00  0.00           H  \\nATOM    492  N   VAL A  33      -1.698  -4.303  37.174  1.00  0.00           N  \\nATOM    493  CA  VAL A  33      -2.754  -3.455  36.683  1.00  0.00           C  \\nATOM    494  C   VAL A  33      -2.469  -2.089  37.225  1.00  0.00           C  \\nATOM    495  O   VAL A  33      -2.253  -1.957  38.423  1.00  0.00           O  \\nATOM    496  CB  VAL A  33      -4.125  -3.922  37.171  1.00  0.00           C  \\nATOM    497  CG1 VAL A  33      -5.229  -3.021  36.776  1.00  0.00           C  \\nATOM    498  CG2 VAL A  33      -4.424  -5.332  36.812  1.00  0.00           C  \\nATOM    499  H   VAL A  33      -1.406  -4.218  38.137  1.00  0.00           H  \\nATOM    500  HA  VAL A  33      -2.751  -3.495  35.593  1.00  0.00           H  \\nATOM    501  HB  VAL A  33      -4.162  -3.838  38.257  1.00  0.00           H  \\nATOM    502 1HG1 VAL A  33      -6.173  -3.413  37.156  1.00  0.00           H  \\nATOM    503 2HG1 VAL A  33      -5.055  -2.029  37.192  1.00  0.00           H  \\nATOM    504 3HG1 VAL A  33      -5.275  -2.957  35.689  1.00  0.00           H  \\nATOM    505 1HG2 VAL A  33      -5.411  -5.602  37.188  1.00  0.00           H  \\nATOM    506 2HG2 VAL A  33      -4.405  -5.443  35.728  1.00  0.00           H  \\nATOM    507 3HG2 VAL A  33      -3.675  -5.987  37.257  1.00  0.00           H  \\nATOM    508  N   ARG A  34      -2.428  -1.097  36.346  1.00  0.00           N  \\nATOM    509  CA  ARG A  34      -2.114   0.264  36.755  1.00  0.00           C  \\nATOM    510  C   ARG A  34      -3.182   1.178  36.250  1.00  0.00           C  \\nATOM    511  O   ARG A  34      -3.500   1.199  35.067  1.00  0.00           O  \\nATOM    512  CB  ARG A  34      -0.763   0.634  36.210  1.00  0.00           C  \\nATOM    513  CG  ARG A  34       0.202  -0.373  36.734  1.00  0.00           C  \\nATOM    514  CD  ARG A  34       1.584  -0.040  36.546  1.00  0.00           C  \\nATOM    515  NE  ARG A  34       2.378  -1.221  36.316  1.00  0.00           N  \\nATOM    516  CZ  ARG A  34       3.634  -1.158  35.911  1.00  0.00           C  \\nATOM    517  NH1 ARG A  34       4.191   0.014  35.679  1.00  0.00           N  \\nATOM    518  NH2 ARG A  34       4.284  -2.299  35.671  1.00  0.00           N  \\nATOM    519  H   ARG A  34      -2.618  -1.287  35.372  1.00  0.00           H  \\nATOM    520  HA  ARG A  34      -2.089   0.301  37.845  1.00  0.00           H  \\nATOM    521 1HB  ARG A  34      -0.797   0.633  35.122  1.00  0.00           H  \\nATOM    522 2HB  ARG A  34      -0.506   1.644  36.528  1.00  0.00           H  \\nATOM    523 1HG  ARG A  34       0.053  -0.496  37.807  1.00  0.00           H  \\nATOM    524 2HG  ARG A  34       0.039  -1.328  36.233  1.00  0.00           H  \\nATOM    525 1HD  ARG A  34       1.685   0.620  35.685  1.00  0.00           H  \\nATOM    526 2HD  ARG A  34       1.961   0.464  37.435  1.00  0.00           H  \\nATOM    527  HE  ARG A  34       1.942  -2.119  36.475  1.00  0.00           H  \\nATOM    528 1HH1 ARG A  34       3.658   0.862  35.811  1.00  0.00           H  \\nATOM    529 2HH1 ARG A  34       5.151   0.064  35.369  1.00  0.00           H  \\nATOM    530 1HH2 ARG A  34       3.815  -3.185  35.800  1.00  0.00           H  \\nATOM    531 2HH2 ARG A  34       5.244  -2.277  35.361  1.00  0.00           H  \\nATOM    532  N   HIS A  35      -3.775   1.953  37.146  1.00  0.00           N  \\nATOM    533  CA  HIS A  35      -4.920   2.763  36.783  1.00  0.00           C  \\nATOM    534  C   HIS A  35      -6.021   1.985  36.190  1.00  0.00           C  \\nATOM    535  O   HIS A  35      -6.675   2.483  35.311  1.00  0.00           O  \\nATOM    536  CB  HIS A  35      -4.560   3.982  35.952  1.00  0.00           C  \\nATOM    537  CG  HIS A  35      -3.530   4.827  36.637  1.00  0.00           C  \\nATOM    538  ND1 HIS A  35      -3.883   5.682  37.674  1.00  0.00           N  \\nATOM    539  CD2 HIS A  35      -2.181   4.856  36.573  1.00  0.00           C  \\nATOM    540  CE1 HIS A  35      -2.790   6.281  38.126  1.00  0.00           C  \\nATOM    541  NE2 HIS A  35      -1.744   5.777  37.500  1.00  0.00           N  \\nATOM    542  H   HIS A  35      -3.431   1.985  38.095  1.00  0.00           H  \\nATOM    543  HA  HIS A  35      -5.405   3.133  37.686  1.00  0.00           H  \\nATOM    544 1HB  HIS A  35      -4.179   3.662  34.982  1.00  0.00           H  \\nATOM    545 2HB  HIS A  35      -5.455   4.577  35.772  1.00  0.00           H  \\nATOM    546  HD2 HIS A  35      -1.556   4.270  35.899  1.00  0.00           H  \\nATOM    547  HE1 HIS A  35      -2.759   7.060  38.888  1.00  0.00           H  \\nATOM    548  HE2 HIS A  35      -0.779   6.023  37.669  1.00  0.00           H  \\nATOM    549  N   GLU A  36      -6.271   0.796  36.750  1.00  0.00           N  \\nATOM    550  CA  GLU A  36      -7.321  -0.062  36.287  1.00  0.00           C  \\nATOM    551  C   GLU A  36      -7.055  -0.600  34.893  1.00  0.00           C  \\nATOM    552  O   GLU A  36      -7.948  -1.164  34.278  1.00  0.00           O  \\nATOM    553  CB  GLU A  36      -8.695   0.602  36.459  1.00  0.00           C  \\nATOM    554  CG  GLU A  36      -9.024   0.734  37.986  1.00  0.00           C  \\nATOM    555  CD  GLU A  36      -8.611  -0.517  38.798  1.00  0.00           C  \\nATOM    556  OE1 GLU A  36      -9.391  -1.510  38.879  1.00  0.00           O  \\nATOM    557  OE2 GLU A  36      -7.494  -0.515  39.379  1.00  0.00           O  \\nATOM    558  H   GLU A  36      -5.697   0.497  37.526  1.00  0.00           H  \\nATOM    559  HA  GLU A  36      -7.306  -0.978  36.879  1.00  0.00           H  \\nATOM    560 1HB  GLU A  36      -8.687   1.584  35.985  1.00  0.00           H  \\nATOM    561 2HB  GLU A  36      -9.454   0.003  35.957  1.00  0.00           H  \\nATOM    562 1HG  GLU A  36      -8.502   1.603  38.386  1.00  0.00           H  \\nATOM    563 2HG  GLU A  36     -10.094   0.902  38.103  1.00  0.00           H  \\nATOM    564  N   ILE A  37      -5.837  -0.371  34.394  1.00  0.00           N  \\nATOM    565  CA  ILE A  37      -5.439  -0.843  33.045  1.00  0.00           C  \\nATOM    566  C   ILE A  37      -4.342  -1.916  33.172  1.00  0.00           C  \\nATOM    567  O   ILE A  37      -3.241  -1.689  33.658  1.00  0.00           O  \\nATOM    568  CB  ILE A  37      -4.950   0.252  32.144  1.00  0.00           C  \\nATOM    569  CG1 ILE A  37      -5.976   1.396  31.994  1.00  0.00           C  \\nATOM    570  CG2 ILE A  37      -4.650  -0.357  30.693  1.00  0.00           C  \\nATOM    571  CD1 ILE A  37      -5.259   2.660  31.746  1.00  0.00           C  \\nATOM    572  H   ILE A  37      -5.169   0.139  34.954  1.00  0.00           H  \\nATOM    573  HA  ILE A  37      -6.309  -1.289  32.564  1.00  0.00           H  \\nATOM    574  HB  ILE A  37      -4.038   0.682  32.556  1.00  0.00           H  \\nATOM    575 1HG1 ILE A  37      -6.654   1.171  31.172  1.00  0.00           H  \\nATOM    576 2HG1 ILE A  37      -6.576   1.468  32.901  1.00  0.00           H  \\nATOM    577 1HG2 ILE A  37      -4.295   0.434  30.033  1.00  0.00           H  \\nATOM    578 2HG2 ILE A  37      -3.888  -1.131  30.774  1.00  0.00           H  \\nATOM    579 3HG2 ILE A  37      -5.564  -0.788  30.283  1.00  0.00           H  \\nATOM    580 1HD1 ILE A  37      -5.979   3.472  31.640  1.00  0.00           H  \\nATOM    581 2HD1 ILE A  37      -4.594   2.872  32.584  1.00  0.00           H  \\nATOM    582 3HD1 ILE A  37      -4.674   2.573  30.832  1.00  0.00           H  \\nATOM    583  N   PRO A  38      -4.681  -3.120  32.696  1.00  0.00           N  \\nATOM    584  CA  PRO A  38      -3.711  -4.211  32.710  1.00  0.00           C  \\nATOM    585  C   PRO A  38      -2.479  -4.022  31.856  1.00  0.00           C  \\nATOM    586  O   PRO A  38      -2.518  -3.408  30.737  1.00  0.00           O  \\nATOM    587  CB  PRO A  38      -4.513  -5.390  32.095  1.00  0.00           C  \\nATOM    588  CG  PRO A  38      -5.927  -5.031  32.384  1.00  0.00           C  \\nATOM    589  CD  PRO A  38      -6.031  -3.574  32.290  1.00  0.00           C  \\nATOM    590  HA  PRO A  38      -3.424  -4.427  33.750  1.00  0.00           H  \\nATOM    591 1HB  PRO A  38      -4.297  -5.472  31.020  1.00  0.00           H  \\nATOM    592 2HB  PRO A  38      -4.202  -6.338  32.558  1.00  0.00           H  \\nATOM    593 1HG  PRO A  38      -6.599  -5.526  31.667  1.00  0.00           H  \\nATOM    594 2HG  PRO A  38      -6.212  -5.387  33.385  1.00  0.00           H  \\nATOM    595 1HD  PRO A  38      -6.263  -3.289  31.253  1.00  0.00           H  \\nATOM    596 2HD  PRO A  38      -6.814  -3.217  32.975  1.00  0.00           H  \\nATOM    597  N   VAL A  39      -1.357  -4.473  32.408  1.00  0.00           N  \\nATOM    598  CA  VAL A  39      -0.035  -4.364  31.754  1.00  0.00           C  \\nATOM    599  C   VAL A  39       0.392  -5.765  31.264  1.00  0.00           C  \\nATOM    600  O   VAL A  39       0.192  -6.783  31.916  1.00  0.00           O  \\nATOM    601  CB  VAL A  39       1.005  -3.776  32.737  1.00  0.00           C  \\nATOM    602  CG1 VAL A  39       2.234  -3.509  32.023  1.00  0.00           C  \\nATOM    603  CG2 VAL A  39       0.434  -2.451  33.330  1.00  0.00           C  \\nATOM    604  H   VAL A  39      -1.422  -4.909  33.317  1.00  0.00           H  \\nATOM    605  HA  VAL A  39      -0.126  -3.693  30.899  1.00  0.00           H  \\nATOM    606  HB  VAL A  39       1.186  -4.497  33.534  1.00  0.00           H  \\nATOM    607 1HG1 VAL A  39       2.972  -3.094  32.711  1.00  0.00           H  \\nATOM    608 2HG1 VAL A  39       2.616  -4.437  31.598  1.00  0.00           H  \\nATOM    609 3HG1 VAL A  39       2.042  -2.794  31.224  1.00  0.00           H  \\nATOM    610 1HG2 VAL A  39       1.155  -2.024  34.026  1.00  0.00           H  \\nATOM    611 2HG2 VAL A  39       0.245  -1.743  32.523  1.00  0.00           H  \\nATOM    612 3HG2 VAL A  39      -0.498  -2.660  33.856  1.00  0.00           H  \\nATOM    613  N   LEU A  40       0.931  -5.786  30.058  1.00  0.00           N  \\nATOM    614  CA  LEU A  40       1.376  -7.014  29.497  1.00  0.00           C  \\nATOM    615  C   LEU A  40       2.718  -7.440  30.136  1.00  0.00           C  \\nATOM    616  O   LEU A  40       3.366  -6.669  30.829  1.00  0.00           O  \\nATOM    617  CB  LEU A  40       1.598  -6.762  27.995  1.00  0.00           C  \\nATOM    618  CG  LEU A  40       0.351  -6.563  27.145  1.00  0.00           C  \\nATOM    619  CD1 LEU A  40       0.790  -6.057  25.790  1.00  0.00           C  \\nATOM    620  CD2 LEU A  40      -0.472  -7.884  27.026  1.00  0.00           C  \\nATOM    621  H   LEU A  40       1.033  -4.935  29.523  1.00  0.00           H  \\nATOM    622  HA  LEU A  40       0.599  -7.763  29.646  1.00  0.00           H  \\nATOM    623 1HB  LEU A  40       2.212  -5.871  27.880  1.00  0.00           H  \\nATOM    624 2HB  LEU A  40       2.141  -7.609  27.577  1.00  0.00           H  \\nATOM    625  HG  LEU A  40      -0.282  -5.801  27.600  1.00  0.00           H  \\nATOM    626 1HD1 LEU A  40      -0.085  -5.905  25.158  1.00  0.00           H  \\nATOM    627 2HD1 LEU A  40       1.320  -5.112  25.909  1.00  0.00           H  \\nATOM    628 3HD1 LEU A  40       1.450  -6.788  25.325  1.00  0.00           H  \\nATOM    629 1HD2 LEU A  40      -1.357  -7.708  26.413  1.00  0.00           H  \\nATOM    630 2HD2 LEU A  40       0.143  -8.655  26.562  1.00  0.00           H  \\nATOM    631 3HD2 LEU A  40      -0.778  -8.213  28.019  1.00  0.00           H  \\nATOM    632  N   PRO A  41       3.109  -8.690  29.961  1.00  0.00           N  \\nATOM    633  CA  PRO A  41       4.340  -9.174  30.590  1.00  0.00           C  \\nATOM    634  C   PRO A  41       5.593  -8.445  30.081  1.00  0.00           C  \\nATOM    635  O   PRO A  41       5.692  -8.031  28.965  1.00  0.00           O  \\nATOM    636  CB  PRO A  41       4.449 -10.634  30.082  1.00  0.00           C  \\nATOM    637  CG  PRO A  41       3.041 -11.081  29.853  1.00  0.00           C  \\nATOM    638  CD  PRO A  41       2.289  -9.805  29.408  1.00  0.00           C  \\nATOM    639  HA  PRO A  41       4.222  -9.148  31.683  1.00  0.00           H  \\nATOM    640 1HB  PRO A  41       5.051 -10.668  29.162  1.00  0.00           H  \\nATOM    641 2HB  PRO A  41       4.967 -11.254  30.828  1.00  0.00           H  \\nATOM    642 1HG  PRO A  41       3.013 -11.873  29.090  1.00  0.00           H  \\nATOM    643 2HG  PRO A  41       2.625 -11.514  30.775  1.00  0.00           H  \\nATOM    644 1HD  PRO A  41       2.252  -9.765  28.309  1.00  0.00           H  \\nATOM    645 2HD  PRO A  41       1.274  -9.811  29.831  1.00  0.00           H  \\nATOM    646  N   ASN A  42       6.574  -8.293  30.969  1.00  0.00           N  \\nATOM    647  CA  ASN A  42       7.807  -7.604  30.619  1.00  0.00           C  \\nATOM    648  C   ASN A  42       8.681  -8.526  29.719  1.00  0.00           C  \\nATOM    649  O   ASN A  42       8.839  -9.708  30.031  1.00  0.00           O  \\nATOM    650  CB  ASN A  42       8.525  -7.271  31.904  1.00  0.00           C  \\nATOM    651  CG  ASN A  42       9.861  -6.580  31.657  1.00  0.00           C  \\nATOM    652  OD1 ASN A  42      10.222  -6.216  30.530  1.00  0.00           O  \\nATOM    653  ND2 ASN A  42      10.583  -6.355  32.753  1.00  0.00           N  \\nATOM    654  H   ASN A  42       6.464  -8.661  31.903  1.00  0.00           H  \\nATOM    655  HA  ASN A  42       7.552  -6.687  30.086  1.00  0.00           H  \\nATOM    656 1HB  ASN A  42       7.897  -6.621  32.514  1.00  0.00           H  \\nATOM    657 2HB  ASN A  42       8.698  -8.185  32.472  1.00  0.00           H  \\nATOM    658 1HD2 ASN A  42      11.474  -5.905  32.682  1.00  0.00           H  \\nATOM    659 2HD2 ASN A  42      10.235  -6.635  33.647  1.00  0.00           H  \\nATOM    660  N   ARG A  43       9.128  -8.039  28.587  1.00  0.00           N  \\nATOM    661  CA  ARG A  43       9.966  -8.836  27.670  1.00  0.00           C  \\nATOM    662  C   ARG A  43      11.256  -9.280  28.368  1.00  0.00           C  \\nATOM    663  O   ARG A  43      11.793 -10.327  28.078  1.00  0.00           O  \\nATOM    664  CB  ARG A  43      10.318  -7.980  26.457  1.00  0.00           C  \\nATOM    665  CG  ARG A  43      10.948  -8.683  25.226  1.00  0.00           C  \\nATOM    666  CD  ARG A  43      11.134  -7.605  24.091  1.00  0.00           C  \\nATOM    667  NE  ARG A  43      11.940  -8.021  22.904  1.00  0.00           N  \\nATOM    668  CZ  ARG A  43      13.276  -7.976  22.851  1.00  0.00           C  \\nATOM    669  NH1 ARG A  43      13.976  -7.572  23.914  1.00  0.00           N  \\nATOM    670  NH2 ARG A  43      13.936  -8.329  21.739  1.00  0.00           N  \\nATOM    671  H   ARG A  43       8.891  -7.090  28.337  1.00  0.00           H  \\nATOM    672  HA  ARG A  43       9.394  -9.706  27.347  1.00  0.00           H  \\nATOM    673 1HB  ARG A  43       9.421  -7.483  26.090  1.00  0.00           H  \\nATOM    674 2HB  ARG A  43      11.025  -7.204  26.752  1.00  0.00           H  \\nATOM    675 1HG  ARG A  43      11.913  -9.107  25.504  1.00  0.00           H  \\nATOM    676 2HG  ARG A  43      10.286  -9.480  24.884  1.00  0.00           H  \\nATOM    677 1HD  ARG A  43      10.159  -7.315  23.702  1.00  0.00           H  \\nATOM    678 2HD  ARG A  43      11.636  -6.729  24.500  1.00  0.00           H  \\nATOM    679  HE  ARG A  43      11.441  -8.357  22.091  1.00  0.00           H  \\nATOM    680 1HH1 ARG A  43      13.497  -7.300  24.761  1.00  0.00           H  \\nATOM    681 2HH1 ARG A  43      14.984  -7.539  23.872  1.00  0.00           H  \\nATOM    682 1HH2 ARG A  43      13.424  -8.635  20.923  1.00  0.00           H  \\nATOM    683 2HH2 ARG A  43      14.944  -8.289  21.716  1.00  0.00           H  \\nATOM    684  N   VAL A  44      11.800  -8.424  29.226  1.00  0.00           N  \\nATOM    685  CA  VAL A  44      13.109  -8.717  29.826  1.00  0.00           C  \\nATOM    686  C   VAL A  44      12.933  -9.821  30.860  1.00  0.00           C  \\nATOM    687  O   VAL A  44      12.183  -9.699  31.805  1.00  0.00           O  \\nATOM    688  CB  VAL A  44      13.694  -7.479  30.532  1.00  0.00           C  \\nATOM    689  CG1 VAL A  44      15.056  -7.842  31.179  1.00  0.00           C  \\nATOM    690  CG2 VAL A  44      13.887  -6.423  29.518  1.00  0.00           C  \\nATOM    691  H   VAL A  44      11.325  -7.568  29.475  1.00  0.00           H  \\nATOM    692  HA  VAL A  44      13.796  -9.012  29.032  1.00  0.00           H  \\nATOM    693  HB  VAL A  44      12.995  -7.146  31.300  1.00  0.00           H  \\nATOM    694 1HG1 VAL A  44      15.467  -6.964  31.678  1.00  0.00           H  \\nATOM    695 2HG1 VAL A  44      14.911  -8.639  31.908  1.00  0.00           H  \\nATOM    696 3HG1 VAL A  44      15.748  -8.177  30.406  1.00  0.00           H  \\nATOM    697 1HG2 VAL A  44      14.300  -5.534  29.993  1.00  0.00           H  \\nATOM    698 2HG2 VAL A  44      14.575  -6.777  28.750  1.00  0.00           H  \\nATOM    699 3HG2 VAL A  44      12.928  -6.177  29.061  1.00  0.00           H  \\nATOM    700  N   GLY A  45      13.556 -10.943  30.591  1.00  0.00           N  \\nATOM    701  CA  GLY A  45      13.460 -12.068  31.447  1.00  0.00           C  \\nATOM    702  C   GLY A  45      12.304 -12.980  31.207  1.00  0.00           C  \\nATOM    703  O   GLY A  45      12.119 -13.855  31.992  1.00  0.00           O  \\nATOM    704  H   GLY A  45      14.118 -11.008  29.754  1.00  0.00           H  \\nATOM    705 1HA  GLY A  45      14.365 -12.669  31.361  1.00  0.00           H  \\nATOM    706 2HA  GLY A  45      13.394 -11.733  32.482  1.00  0.00           H  \\nATOM    707  N   LEU A  46      11.508 -12.746  30.144  1.00  0.00           N  \\nATOM    708  CA  LEU A  46      10.296 -13.555  29.952  1.00  0.00           C  \\nATOM    709  C   LEU A  46      10.694 -14.915  29.350  1.00  0.00           C  \\nATOM    710  O   LEU A  46      11.333 -14.959  28.287  1.00  0.00           O  \\nATOM    711  CB  LEU A  46       9.301 -12.838  29.024  1.00  0.00           C  \\nATOM    712  CG  LEU A  46       7.953 -13.535  28.911  1.00  0.00           C  \\nATOM    713  CD1 LEU A  46       7.158 -13.531  30.126  1.00  0.00           C  \\nATOM    714  CD2 LEU A  46       7.151 -12.660  27.838  1.00  0.00           C  \\nATOM    715  H   LEU A  46      11.728 -12.023  29.474  1.00  0.00           H  \\nATOM    716  HA  LEU A  46       9.820 -13.701  30.921  1.00  0.00           H  \\nATOM    717 1HB  LEU A  46       9.141 -11.829  29.400  1.00  0.00           H  \\nATOM    718 2HB  LEU A  46       9.742 -12.768  28.030  1.00  0.00           H  \\nATOM    719  HG  LEU A  46       8.101 -14.561  28.574  1.00  0.00           H  \\nATOM    720 1HD1 LEU A  46       6.217 -14.051  29.947  1.00  0.00           H  \\nATOM    721 2HD1 LEU A  46       7.705 -14.038  30.921  1.00  0.00           H  \\nATOM    722 3HD1 LEU A  46       6.954 -12.503  30.424  1.00  0.00           H  \\nATOM    723 1HD2 LEU A  46       6.159 -13.088  27.688  1.00  0.00           H  \\nATOM    724 2HD2 LEU A  46       7.053 -11.637  28.202  1.00  0.00           H  \\nATOM    725 3HD2 LEU A  46       7.693 -12.658  26.892  1.00  0.00           H  \\nATOM    726  N   PRO A  47      10.326 -16.016  30.049  1.00  0.00           N  \\nATOM    727  CA  PRO A  47      10.725 -17.333  29.545  1.00  0.00           C  \\nATOM    728  C   PRO A  47      10.044 -17.588  28.236  1.00  0.00           C  \\nATOM    729  O   PRO A  47       8.907 -17.132  28.015  1.00  0.00           O  \\nATOM    730  CB  PRO A  47      10.108 -18.289  30.541  1.00  0.00           C  \\nATOM    731  CG  PRO A  47      10.129 -17.524  31.789  1.00  0.00           C  \\nATOM    732  CD  PRO A  47       9.683 -16.131  31.344  1.00  0.00           C  \\nATOM    733  HA  PRO A  47      11.822 -17.408  29.559  1.00  0.00           H  \\nATOM    734 1HB  PRO A  47       9.095 -18.568  30.217  1.00  0.00           H  \\nATOM    735 2HB  PRO A  47      10.698 -19.217  30.587  1.00  0.00           H  \\nATOM    736 1HG  PRO A  47       9.455 -17.983  32.527  1.00  0.00           H  \\nATOM    737 2HG  PRO A  47      11.138 -17.540  32.227  1.00  0.00           H  \\nATOM    738 1HD  PRO A  47       8.586 -16.103  31.270  1.00  0.00           H  \\nATOM    739 2HD  PRO A  47      10.042 -15.382  32.065  1.00  0.00           H  \\nATOM    740  N   ILE A  48      10.791 -18.242  27.366  1.00  0.00           N  \\nATOM    741  CA  ILE A  48      10.316 -18.462  25.955  1.00  0.00           C  \\nATOM    742  C   ILE A  48       9.002 -19.211  25.913  1.00  0.00           C  \\nATOM    743  O   ILE A  48       8.144 -18.987  25.041  1.00  0.00           O  \\nATOM    744  CB  ILE A  48      11.456 -19.007  25.028  1.00  0.00           C  \\nATOM    745  CG1 ILE A  48      10.999 -19.203  23.598  1.00  0.00           C  \\nATOM    746  CG2 ILE A  48      11.954 -20.382  25.429  1.00  0.00           C  \\nATOM    747  CD1 ILE A  48      10.761 -17.943  22.766  1.00  0.00           C  \\nATOM    748  H   ILE A  48      11.693 -18.602  27.643  1.00  0.00           H  \\nATOM    749  HA  ILE A  48       9.983 -17.507  25.549  1.00  0.00           H  \\nATOM    750  HB  ILE A  48      12.308 -18.329  25.062  1.00  0.00           H  \\nATOM    751 1HG1 ILE A  48      11.737 -19.796  23.059  1.00  0.00           H  \\nATOM    752 2HG1 ILE A  48      10.062 -19.761  23.589  1.00  0.00           H  \\nATOM    753 1HG2 ILE A  48      12.741 -20.698  24.745  1.00  0.00           H  \\nATOM    754 2HG2 ILE A  48      12.349 -20.344  26.443  1.00  0.00           H  \\nATOM    755 3HG2 ILE A  48      11.130 -21.094  25.387  1.00  0.00           H  \\nATOM    756 1HD1 ILE A  48      10.437 -18.224  21.764  1.00  0.00           H  \\nATOM    757 2HD1 ILE A  48       9.989 -17.335  23.239  1.00  0.00           H  \\nATOM    758 3HD1 ILE A  48      11.685 -17.371  22.701  1.00  0.00           H  \\nATOM    759  N   ASN A  49       8.738 -20.011  26.902  1.00  0.00           N  \\nATOM    760  CA  ASN A  49       7.562 -20.812  26.816  1.00  0.00           C  \\nATOM    761  C   ASN A  49       6.307 -20.034  27.248  1.00  0.00           C  \\nATOM    762  O   ASN A  49       5.226 -20.577  27.171  1.00  0.00           O  \\nATOM    763  CB  ASN A  49       7.629 -22.120  27.594  1.00  0.00           C  \\nATOM    764  CG  ASN A  49       8.027 -21.973  28.996  1.00  0.00           C  \\nATOM    765  OD1 ASN A  49       8.442 -20.920  29.464  1.00  0.00           O  \\nATOM    766  ND2 ASN A  49       8.005 -23.126  29.698  1.00  0.00           N  \\nATOM    767  H   ASN A  49       9.326 -20.087  27.720  1.00  0.00           H  \\nATOM    768  HA  ASN A  49       7.400 -21.078  25.770  1.00  0.00           H  \\nATOM    769 1HB  ASN A  49       6.653 -22.606  27.573  1.00  0.00           H  \\nATOM    770 2HB  ASN A  49       8.340 -22.793  27.114  1.00  0.00           H  \\nATOM    771 1HD2 ASN A  49       8.261 -23.128  30.665  1.00  0.00           H  \\nATOM    772 2HD2 ASN A  49       7.733 -23.978  29.251  1.00  0.00           H  \\nATOM    773  N   GLN A  50       6.495 -18.802  27.723  1.00  0.00           N  \\nATOM    774  CA  GLN A  50       5.450 -17.858  28.030  1.00  0.00           C  \\nATOM    775  C   GLN A  50       5.407 -16.697  26.972  1.00  0.00           C  \\nATOM    776  O   GLN A  50       4.655 -15.757  27.234  1.00  0.00           O  \\nATOM    777  CB  GLN A  50       5.733 -17.227  29.404  1.00  0.00           C  \\nATOM    778  CG  GLN A  50       5.598 -18.185  30.609  1.00  0.00           C  \\nATOM    779  CD  GLN A  50       6.147 -17.551  31.862  1.00  0.00           C  \\nATOM    780  OE1 GLN A  50       6.726 -18.241  32.669  1.00  0.00           O  \\nATOM    781  NE2 GLN A  50       6.018 -16.227  32.006  1.00  0.00           N  \\nATOM    782  H   GLN A  50       7.458 -18.535  27.870  1.00  0.00           H  \\nATOM    783  HA  GLN A  50       4.501 -18.393  28.067  1.00  0.00           H  \\nATOM    784 1HB  GLN A  50       6.746 -16.825  29.418  1.00  0.00           H  \\nATOM    785 2HB  GLN A  50       5.049 -16.395  29.570  1.00  0.00           H  \\nATOM    786 1HG  GLN A  50       4.544 -18.417  30.759  1.00  0.00           H  \\nATOM    787 2HG  GLN A  50       6.155 -19.098  30.398  1.00  0.00           H  \\nATOM    788 1HE2 GLN A  50       6.372 -15.776  32.826  1.00  0.00           H  \\nATOM    789 2HE2 GLN A  50       5.568 -15.688  31.294  1.00  0.00           H  \\nATOM    790  N   ARG A  51       6.118 -16.781  25.857  1.00  0.00           N  \\nATOM    791  CA  ARG A  51       6.284 -15.598  25.018  1.00  0.00           C  \\nATOM    792  C   ARG A  51       5.131 -15.340  24.059  1.00  0.00           C  \\nATOM    793  O   ARG A  51       4.979 -14.185  23.620  1.00  0.00           O  \\nATOM    794  CB  ARG A  51       7.665 -15.640  24.317  1.00  0.00           C  \\nATOM    795  CG  ARG A  51       8.088 -14.525  23.312  1.00  0.00           C  \\nATOM    796  CD  ARG A  51       7.987 -13.158  24.056  1.00  0.00           C  \\nATOM    797  NE  ARG A  51       8.403 -12.130  23.129  1.00  0.00           N  \\nATOM    798  CZ  ARG A  51       9.625 -11.861  22.698  1.00  0.00           C  \\nATOM    799  NH1 ARG A  51       9.694 -10.925  21.777  1.00  0.00           N  \\nATOM    800  NH2 ARG A  51      10.741 -12.489  23.132  1.00  0.00           N  \\nATOM    801  H   ARG A  51       6.549 -17.649  25.574  1.00  0.00           H  \\nATOM    802  HA  ARG A  51       6.236 -14.713  25.654  1.00  0.00           H  \\nATOM    803 1HB  ARG A  51       8.454 -15.639  25.068  1.00  0.00           H  \\nATOM    804 2HB  ARG A  51       7.756 -16.565  23.747  1.00  0.00           H  \\nATOM    805 1HG  ARG A  51       9.111 -14.703  22.981  1.00  0.00           H  \\nATOM    806 2HG  ARG A  51       7.420 -14.539  22.450  1.00  0.00           H  \\nATOM    807 1HD  ARG A  51       6.958 -12.992  24.373  1.00  0.00           H  \\nATOM    808 2HD  ARG A  51       8.638 -13.170  24.929  1.00  0.00           H  \\nATOM    809  HE  ARG A  51       7.689 -11.525  22.746  1.00  0.00           H  \\nATOM    810 1HH1 ARG A  51       8.851 -10.472  21.453  1.00  0.00           H  \\nATOM    811 2HH1 ARG A  51      10.589 -10.659  21.394  1.00  0.00           H  \\nATOM    812 1HH2 ARG A  51      10.672 -13.216  23.831  1.00  0.00           H  \\nATOM    813 2HH2 ARG A  51      11.642 -12.232  22.758  1.00  0.00           H  \\nATOM    814  N   PHE A  52       4.418 -16.413  23.687  1.00  0.00           N  \\nATOM    815  CA  PHE A  52       3.473 -16.338  22.557  1.00  0.00           C  \\nATOM    816  C   PHE A  52       2.130 -16.931  22.891  1.00  0.00           C  \\nATOM    817  O   PHE A  52       1.984 -17.772  23.811  1.00  0.00           O  \\nATOM    818  CB  PHE A  52       4.027 -17.013  21.268  1.00  0.00           C  \\nATOM    819  CG  PHE A  52       5.268 -16.420  20.763  1.00  0.00           C  \\nATOM    820  CD1 PHE A  52       5.313 -15.180  20.233  1.00  0.00           C  \\nATOM    821  CD2 PHE A  52       6.460 -17.156  20.876  1.00  0.00           C  \\nATOM    822  CE1 PHE A  52       6.518 -14.634  19.830  1.00  0.00           C  \\nATOM    823  CE2 PHE A  52       7.612 -16.652  20.422  1.00  0.00           C  \\nATOM    824  CZ  PHE A  52       7.678 -15.399  19.893  1.00  0.00           C  \\nATOM    825  H   PHE A  52       4.524 -17.287  24.182  1.00  0.00           H  \\nATOM    826  HA  PHE A  52       3.291 -15.287  22.327  1.00  0.00           H  \\nATOM    827 1HB  PHE A  52       4.214 -18.068  21.462  1.00  0.00           H  \\nATOM    828 2HB  PHE A  52       3.281 -16.953  20.477  1.00  0.00           H  \\nATOM    829  HD1 PHE A  52       4.388 -14.613  20.127  1.00  0.00           H  \\nATOM    830  HD2 PHE A  52       6.433 -18.163  21.295  1.00  0.00           H  \\nATOM    831  HE1 PHE A  52       6.556 -13.608  19.465  1.00  0.00           H  \\nATOM    832  HE2 PHE A  52       8.510 -17.266  20.481  1.00  0.00           H  \\nATOM    833  HZ  PHE A  52       8.619 -14.995  19.522  1.00  0.00           H  \\nATOM    834  N   ILE A  53       1.106 -16.411  22.234  1.00  0.00           N  \\nATOM    835  CA  ILE A  53      -0.229 -17.114  22.257  1.00  0.00           C  \\nATOM    836  C   ILE A  53      -0.607 -17.322  20.794  1.00  0.00           C  \\nATOM    837  O   ILE A  53      -0.129 -16.591  19.917  1.00  0.00           O  \\nATOM    838  CB  ILE A  53      -1.313 -16.422  23.034  1.00  0.00           C  \\nATOM    839  CG1 ILE A  53      -1.740 -15.107  22.424  1.00  0.00           C  \\nATOM    840  CG2 ILE A  53      -0.960 -16.318  24.529  1.00  0.00           C  \\nATOM    841  CD1 ILE A  53      -3.047 -14.734  23.021  1.00  0.00           C  \\nATOM    842  H   ILE A  53       1.200 -15.550  21.715  1.00  0.00           H  \\nATOM    843  HA  ILE A  53      -0.098 -18.090  22.724  1.00  0.00           H  \\nATOM    844  HB  ILE A  53      -2.243 -16.982  22.938  1.00  0.00           H  \\nATOM    845 1HG1 ILE A  53      -0.982 -14.350  22.623  1.00  0.00           H  \\nATOM    846 2HG1 ILE A  53      -1.816 -15.215  21.342  1.00  0.00           H  \\nATOM    847 1HG2 ILE A  53      -1.766 -15.812  25.059  1.00  0.00           H  \\nATOM    848 2HG2 ILE A  53      -0.827 -17.317  24.941  1.00  0.00           H  \\nATOM    849 3HG2 ILE A  53      -0.036 -15.751  24.646  1.00  0.00           H  \\nATOM    850 1HD1 ILE A  53      -3.383 -13.786  22.599  1.00  0.00           H  \\nATOM    851 2HD1 ILE A  53      -3.782 -15.509  22.802  1.00  0.00           H  \\nATOM    852 3HD1 ILE A  53      -2.938 -14.631  24.100  1.00  0.00           H  \\nATOM    853  N   LEU A  54      -1.454 -18.315  20.544  1.00  0.00           N  \\nATOM    854  CA  LEU A  54      -1.828 -18.601  19.155  1.00  0.00           C  \\nATOM    855  C   LEU A  54      -3.326 -18.367  19.046  1.00  0.00           C  \\nATOM    856  O   LEU A  54      -4.084 -18.572  19.960  1.00  0.00           O  \\nATOM    857  CB  LEU A  54      -1.584 -20.077  18.851  1.00  0.00           C  \\nATOM    858  CG  LEU A  54      -0.135 -20.483  19.110  1.00  0.00           C  \\nATOM    859  CD1 LEU A  54      -0.089 -21.974  18.847  1.00  0.00           C  \\nATOM    860  CD2 LEU A  54       0.790 -19.743  18.216  1.00  0.00           C  \\nATOM    861  H   LEU A  54      -1.847 -18.878  21.285  1.00  0.00           H  \\nATOM    862  HA  LEU A  54      -1.205 -17.995  18.498  1.00  0.00           H  \\nATOM    863 1HB  LEU A  54      -2.248 -20.674  19.474  1.00  0.00           H  \\nATOM    864 2HB  LEU A  54      -1.835 -20.263  17.807  1.00  0.00           H  \\nATOM    865  HG  LEU A  54       0.126 -20.261  20.145  1.00  0.00           H  \\nATOM    866 1HD1 LEU A  54       0.924 -22.342  19.013  1.00  0.00           H  \\nATOM    867 2HD1 LEU A  54      -0.775 -22.484  19.524  1.00  0.00           H  \\nATOM    868 3HD1 LEU A  54      -0.382 -22.171  17.816  1.00  0.00           H  \\nATOM    869 1HD2 LEU A  54       1.816 -20.049  18.420  1.00  0.00           H  \\nATOM    870 2HD2 LEU A  54       0.546 -19.963  17.177  1.00  0.00           H  \\nATOM    871 3HD2 LEU A  54       0.689 -18.672  18.393  1.00  0.00           H  \\nATOM    872  N   VAL A  55      -3.722 -17.793  17.913  1.00  0.00           N  \\nATOM    873  CA  VAL A  55      -5.130 -17.482  17.637  1.00  0.00           C  \\nATOM    874  C   VAL A  55      -5.480 -18.272  16.339  1.00  0.00           C  \\nATOM    875  O   VAL A  55      -5.015 -17.971  15.247  1.00  0.00           O  \\nATOM    876  CB  VAL A  55      -5.242 -15.994  17.387  1.00  0.00           C  \\nATOM    877  CG1 VAL A  55      -6.599 -15.670  16.841  1.00  0.00           C  \\nATOM    878  CG2 VAL A  55      -5.083 -15.252  18.715  1.00  0.00           C  \\nATOM    879  H   VAL A  55      -3.023 -17.565  17.221  1.00  0.00           H  \\nATOM    880  HA  VAL A  55      -5.723 -17.755  18.511  1.00  0.00           H  \\nATOM    881  HB  VAL A  55      -4.456 -15.691  16.695  1.00  0.00           H  \\nATOM    882 1HG1 VAL A  55      -6.673 -14.597  16.663  1.00  0.00           H  \\nATOM    883 2HG1 VAL A  55      -6.751 -16.204  15.903  1.00  0.00           H  \\nATOM    884 3HG1 VAL A  55      -7.361 -15.971  17.559  1.00  0.00           H  \\nATOM    885 1HG2 VAL A  55      -5.162 -14.179  18.544  1.00  0.00           H  \\nATOM    886 2HG2 VAL A  55      -5.866 -15.568  19.405  1.00  0.00           H  \\nATOM    887 3HG2 VAL A  55      -4.107 -15.480  19.145  1.00  0.00           H  \\nATOM    888  N   GLU A  56      -6.323 -19.279  16.471  1.00  0.00           N  \\nATOM    889  CA  GLU A  56      -6.738 -20.117  15.384  1.00  0.00           C  \\nATOM    890  C   GLU A  56      -8.043 -19.552  14.787  1.00  0.00           C  \\nATOM    891  O   GLU A  56      -9.079 -19.532  15.408  1.00  0.00           O  \\nATOM    892  CB  GLU A  56      -6.912 -21.569  15.845  1.00  0.00           C  \\nATOM    893  CG  GLU A  56      -7.314 -22.430  14.603  1.00  0.00           C  \\nATOM    894  CD  GLU A  56      -8.109 -23.697  15.007  1.00  0.00           C  \\nATOM    895  OE1 GLU A  56      -7.561 -24.816  14.874  1.00  0.00           O  \\nATOM    896  OE2 GLU A  56      -9.293 -23.586  15.446  1.00  0.00           O  \\nATOM    897  H   GLU A  56      -6.688 -19.457  17.396  1.00  0.00           H  \\nATOM    898  HA  GLU A  56      -5.967 -20.091  14.613  1.00  0.00           H  \\nATOM    899 1HB  GLU A  56      -5.980 -21.925  16.284  1.00  0.00           H  \\nATOM    900 2HB  GLU A  56      -7.678 -21.616  16.619  1.00  0.00           H  \\nATOM    901 1HG  GLU A  56      -7.922 -21.822  13.934  1.00  0.00           H  \\nATOM    902 2HG  GLU A  56      -6.411 -22.719  14.067  1.00  0.00           H  \\nATOM    903  N   LEU A  57      -7.924 -19.038  13.565  1.00  0.00           N  \\nATOM    904  CA  LEU A  57      -9.090 -18.459  12.880  1.00  0.00           C  \\nATOM    905  C   LEU A  57      -9.629 -19.388  11.798  1.00  0.00           C  \\nATOM    906  O   LEU A  57      -8.895 -19.938  11.049  1.00  0.00           O  \\nATOM    907  CB  LEU A  57      -8.612 -17.204  12.149  1.00  0.00           C  \\nATOM    908  CG  LEU A  57      -7.986 -16.126  13.036  1.00  0.00           C  \\nATOM    909  CD1 LEU A  57      -7.454 -15.001  12.115  1.00  0.00           C  \\nATOM    910  CD2 LEU A  57      -8.986 -15.527  14.022  1.00  0.00           C  \\nATOM    911  H   LEU A  57      -7.029 -19.041  13.096  1.00  0.00           H  \\nATOM    912  HA  LEU A  57      -9.836 -18.196  13.629  1.00  0.00           H  \\nATOM    913 1HB  LEU A  57      -7.872 -17.496  11.406  1.00  0.00           H  \\nATOM    914 2HB  LEU A  57      -9.461 -16.759  11.631  1.00  0.00           H  \\nATOM    915  HG  LEU A  57      -7.166 -16.558  13.611  1.00  0.00           H  \\nATOM    916 1HD1 LEU A  57      -7.001 -14.217  12.722  1.00  0.00           H  \\nATOM    917 2HD1 LEU A  57      -6.706 -15.410  11.436  1.00  0.00           H  \\nATOM    918 3HD1 LEU A  57      -8.278 -14.582  11.539  1.00  0.00           H  \\nATOM    919 1HD2 LEU A  57      -8.490 -14.768  14.628  1.00  0.00           H  \\nATOM    920 2HD2 LEU A  57      -9.810 -15.071  13.473  1.00  0.00           H  \\nATOM    921 3HD2 LEU A  57      -9.373 -16.313  14.671  1.00  0.00           H  \\nATOM    922  N   SER A  58     -10.959 -19.499  11.770  1.00  0.00           N  \\nATOM    923  CA  SER A  58     -11.626 -20.347  10.752  1.00  0.00           C  \\nATOM    924  C   SER A  58     -12.653 -19.483  10.086  1.00  0.00           C  \\nATOM    925  O   SER A  58     -13.085 -18.470  10.665  1.00  0.00           O  \\nATOM    926  CB  SER A  58     -12.288 -21.518  11.482  1.00  0.00           C  \\nATOM    927  OG  SER A  58     -11.272 -22.409  11.973  1.00  0.00           O  \\nATOM    928  H   SER A  58     -11.524 -19.004  12.445  1.00  0.00           H  \\nATOM    929  HA  SER A  58     -10.868 -20.719  10.061  1.00  0.00           H  \\nATOM    930 1HB  SER A  58     -12.893 -21.138  12.305  1.00  0.00           H  \\nATOM    931 2HB  SER A  58     -12.956 -22.041  10.799  1.00  0.00           H  \\nATOM    932  HG  SER A  58     -10.433 -22.025  11.707  1.00  0.00           H  \\nATOM    933  N   ASN A  59     -13.174 -19.925   8.930  1.00  0.00           N  \\nATOM    934  CA  ASN A  59     -14.171 -19.090   8.249  1.00  0.00           C  \\nATOM    935  C   ASN A  59     -15.255 -19.947   7.558  1.00  0.00           C  \\nATOM    936  O   ASN A  59     -15.205 -21.156   7.634  1.00  0.00           O  \\nATOM    937  CB  ASN A  59     -13.486 -18.089   7.298  1.00  0.00           C  \\nATOM    938  CG  ASN A  59     -12.769 -18.771   6.144  1.00  0.00           C  \\nATOM    939  OD1 ASN A  59     -12.771 -19.997   6.023  1.00  0.00           O  \\nATOM    940  ND2 ASN A  59     -12.146 -17.994   5.314  1.00  0.00           N  \\nATOM    941  H   ASN A  59     -12.905 -20.806   8.516  1.00  0.00           H  \\nATOM    942  HA  ASN A  59     -14.725 -18.528   9.003  1.00  0.00           H  \\nATOM    943 1HB  ASN A  59     -14.232 -17.404   6.892  1.00  0.00           H  \\nATOM    944 2HB  ASN A  59     -12.764 -17.493   7.856  1.00  0.00           H  \\nATOM    945 1HD2 ASN A  59     -11.656 -18.385   4.534  1.00  0.00           H  \\nATOM    946 2HD2 ASN A  59     -12.155 -17.004   5.453  1.00  0.00           H  \\nATOM    947  N   HIS A  60     -16.236 -19.257   6.994  1.00  0.00           N  \\nATOM    948  CA  HIS A  60     -17.290 -19.970   6.216  1.00  0.00           C  \\nATOM    949  C   HIS A  60     -16.737 -20.854   5.113  1.00  0.00           C  \\nATOM    950  O   HIS A  60     -17.363 -21.897   4.792  1.00  0.00           O  \\nATOM    951  CB  HIS A  60     -18.204 -18.988   5.558  1.00  0.00           C  \\nATOM    952  CG  HIS A  60     -19.161 -18.298   6.446  1.00  0.00           C  \\nATOM    953  ND1 HIS A  60     -20.234 -18.946   7.010  1.00  0.00           N  \\nATOM    954  CD2 HIS A  60     -19.251 -16.992   6.822  1.00  0.00           C  \\nATOM    955  CE1 HIS A  60     -20.973 -18.076   7.669  1.00  0.00           C  \\nATOM    956  NE2 HIS A  60     -20.378 -16.894   7.609  1.00  0.00           N  \\nATOM    957  H   HIS A  60     -16.284 -18.252   7.081  1.00  0.00           H  \\nATOM    958  HA  HIS A  60     -17.883 -20.589   6.889  1.00  0.00           H  \\nATOM    959 1HB  HIS A  60     -17.614 -18.216   5.064  1.00  0.00           H  \\nATOM    960 2HB  HIS A  60     -18.789 -19.494   4.790  1.00  0.00           H  \\nATOM    961  HD2 HIS A  60     -18.555 -16.187   6.585  1.00  0.00           H  \\nATOM    962  HE1 HIS A  60     -21.913 -18.293   8.177  1.00  0.00           H  \\nATOM    963  HE2 HIS A  60     -20.694 -16.050   8.065  1.00  0.00           H  \\nATOM    964  N   ALA A  61     -15.618 -20.464   4.519  1.00  0.00           N  \\nATOM    965  CA  ALA A  61     -14.986 -21.261   3.472  1.00  0.00           C  \\nATOM    966  C   ALA A  61     -14.395 -22.539   4.004  1.00  0.00           C  \\nATOM    967  O   ALA A  61     -13.904 -23.349   3.219  1.00  0.00           O  \\nATOM    968  CB  ALA A  61     -13.980 -20.413   2.701  1.00  0.00           C  \\nATOM    969  H   ALA A  61     -15.192 -19.592   4.799  1.00  0.00           H  \\nATOM    970  HA  ALA A  61     -15.764 -21.599   2.787  1.00  0.00           H  \\nATOM    971 1HB  ALA A  61     -13.515 -21.018   1.923  1.00  0.00           H  \\nATOM    972 2HB  ALA A  61     -14.492 -19.566   2.245  1.00  0.00           H  \\nATOM    973 3HB  ALA A  61     -13.214 -20.048   3.383  1.00  0.00           H  \\nATOM    974  N   GLU A  62     -14.414 -22.744   5.320  1.00  0.00           N  \\nATOM    975  CA  GLU A  62     -13.927 -23.967   6.007  1.00  0.00           C  \\nATOM    976  C   GLU A  62     -12.381 -24.048   5.910  1.00  0.00           C  \\nATOM    977  O   GLU A  62     -11.753 -25.135   5.921  1.00  0.00           O  \\nATOM    978  CB  GLU A  62     -14.583 -25.219   5.388  1.00  0.00           C  \\nATOM    979  CG  GLU A  62     -16.119 -25.237   5.575  1.00  0.00           C  \\nATOM    980  CD  GLU A  62     -16.677 -26.568   5.121  1.00  0.00           C  \\nATOM    981  OE1 GLU A  62     -16.752 -26.800   3.888  1.00  0.00           O  \\nATOM    982  OE2 GLU A  62     -16.992 -27.397   6.003  1.00  0.00           O  \\nATOM    983  H   GLU A  62     -14.796 -21.989   5.871  1.00  0.00           H  \\nATOM    984  HA  GLU A  62     -14.206 -23.907   7.059  1.00  0.00           H  \\nATOM    985 1HB  GLU A  62     -14.357 -25.259   4.322  1.00  0.00           H  \\nATOM    986 2HB  GLU A  62     -14.162 -26.114   5.845  1.00  0.00           H  \\nATOM    987 1HG  GLU A  62     -16.349 -25.068   6.627  1.00  0.00           H  \\nATOM    988 2HG  GLU A  62     -16.554 -24.420   5.001  1.00  0.00           H  \\nATOM    989  N   LEU A  63     -11.734 -22.874   5.863  1.00  0.00           N  \\nATOM    990  CA  LEU A  63     -10.269 -22.767   5.884  1.00  0.00           C  \\nATOM    991  C   LEU A  63      -9.873 -22.218   7.249  1.00  0.00           C  \\nATOM    992  O   LEU A  63     -10.680 -21.524   7.930  1.00  0.00           O  \\nATOM    993  CB  LEU A  63      -9.785 -21.804   4.801  1.00  0.00           C  \\nATOM    994  CG  LEU A  63     -10.168 -22.135   3.372  1.00  0.00           C  \\nATOM    995  CD1 LEU A  63      -9.734 -21.069   2.403  1.00  0.00           C  \\nATOM    996  CD2 LEU A  63      -9.584 -23.480   3.024  1.00  0.00           C  \\nATOM    997  H   LEU A  63     -12.288 -22.031   5.811  1.00  0.00           H  \\nATOM    998  HA  LEU A  63      -9.848 -23.752   5.686  1.00  0.00           H  \\nATOM    999 1HB  LEU A  63     -10.179 -20.812   5.017  1.00  0.00           H  \\nATOM   1000 2HB  LEU A  63      -8.697 -21.757   4.839  1.00  0.00           H  \\nATOM   1001  HG  LEU A  63     -11.254 -22.166   3.284  1.00  0.00           H  \\nATOM   1002 1HD1 LEU A  63     -10.031 -21.352   1.393  1.00  0.00           H  \\nATOM   1003 2HD1 LEU A  63     -10.206 -20.123   2.667  1.00  0.00           H  \\nATOM   1004 3HD1 LEU A  63      -8.651 -20.959   2.445  1.00  0.00           H  \\nATOM   1005 1HD2 LEU A  63      -9.849 -23.738   1.998  1.00  0.00           H  \\nATOM   1006 2HD2 LEU A  63      -8.499 -23.441   3.119  1.00  0.00           H  \\nATOM   1007 3HD2 LEU A  63      -9.981 -24.236   3.702  1.00  0.00           H  \\nATOM   1008  N   SER A  64      -8.664 -22.537   7.662  1.00  0.00           N  \\nATOM   1009  CA  SER A  64      -8.159 -22.106   8.954  1.00  0.00           C  \\nATOM   1010  C   SER A  64      -6.720 -21.658   8.815  1.00  0.00           C  \\nATOM   1011  O   SER A  64      -5.928 -22.233   8.077  1.00  0.00           O  \\nATOM   1012  CB  SER A  64      -8.219 -23.283   9.897  1.00  0.00           C  \\nATOM   1013  OG  SER A  64      -7.778 -22.979  11.183  1.00  0.00           O  \\nATOM   1014  H   SER A  64      -8.072 -23.096   7.064  1.00  0.00           H  \\nATOM   1015  HA  SER A  64      -8.797 -21.303   9.325  1.00  0.00           H  \\nATOM   1016 1HB  SER A  64      -9.244 -23.647   9.960  1.00  0.00           H  \\nATOM   1017 2HB  SER A  64      -7.608 -24.095   9.505  1.00  0.00           H  \\nATOM   1018  HG  SER A  64      -7.529 -22.052  11.165  1.00  0.00           H  \\nATOM   1019  N   VAL A  65      -6.372 -20.614   9.577  1.00  0.00           N  \\nATOM   1020  CA  VAL A  65      -4.973 -20.182   9.709  1.00  0.00           C  \\nATOM   1021  C   VAL A  65      -4.758 -19.952  11.202  1.00  0.00           C  \\nATOM   1022  O   VAL A  65      -5.700 -19.696  11.925  1.00  0.00           O  \\nATOM   1023  CB  VAL A  65      -4.560 -18.911   8.949  1.00  0.00           C  \\nATOM   1024  CG1 VAL A  65      -4.717 -19.157   7.434  1.00  0.00           C  \\nATOM   1025  CG2 VAL A  65      -5.340 -17.707   9.410  1.00  0.00           C  \\nATOM   1026  H   VAL A  65      -7.092 -20.110  10.075  1.00  0.00           H  \\nATOM   1027  HA  VAL A  65      -4.329 -20.973   9.323  1.00  0.00           H  \\nATOM   1028  HB  VAL A  65      -3.500 -18.725   9.120  1.00  0.00           H  \\nATOM   1029 1HG1 VAL A  65      -4.426 -18.260   6.887  1.00  0.00           H  \\nATOM   1030 2HG1 VAL A  65      -4.080 -19.988   7.132  1.00  0.00           H  \\nATOM   1031 3HG1 VAL A  65      -5.756 -19.396   7.210  1.00  0.00           H  \\nATOM   1032 1HG2 VAL A  65      -5.019 -16.829   8.850  1.00  0.00           H  \\nATOM   1033 2HG2 VAL A  65      -6.404 -17.876   9.242  1.00  0.00           H  \\nATOM   1034 3HG2 VAL A  65      -5.163 -17.544  10.473  1.00  0.00           H  \\nATOM   1035  N   THR A  66      -3.534 -20.058  11.676  1.00  0.00           N  \\nATOM   1036  CA  THR A  66      -3.252 -19.782  13.089  1.00  0.00           C  \\nATOM   1037  C   THR A  66      -2.228 -18.652  13.158  1.00  0.00           C  \\nATOM   1038  O   THR A  66      -1.136 -18.740  12.682  1.00  0.00           O  \\nATOM   1039  CB  THR A  66      -2.636 -21.043  13.776  1.00  0.00           C  \\nATOM   1040  OG1 THR A  66      -3.590 -22.114  13.671  1.00  0.00           O  \\nATOM   1041  CG2 THR A  66      -2.380 -20.809  15.246  1.00  0.00           C  \\nATOM   1042  H   THR A  66      -2.776 -20.332  11.067  1.00  0.00           H  \\nATOM   1043  HA  THR A  66      -4.190 -19.535  13.587  1.00  0.00           H  \\nATOM   1044  HB  THR A  66      -1.691 -21.294  13.295  1.00  0.00           H  \\nATOM   1045  HG1 THR A  66      -4.371 -21.802  13.208  1.00  0.00           H  \\nATOM   1046 1HG2 THR A  66      -1.951 -21.708  15.688  1.00  0.00           H  \\nATOM   1047 2HG2 THR A  66      -1.685 -19.978  15.367  1.00  0.00           H  \\nATOM   1048 3HG2 THR A  66      -3.319 -20.572  15.745  1.00  0.00           H  \\nATOM   1049  N   LEU A  67      -2.631 -17.532  13.767  1.00  0.00           N  \\nATOM   1050  CA  LEU A  67      -1.712 -16.385  13.999  1.00  0.00           C  \\nATOM   1051  C   LEU A  67      -0.962 -16.602  15.276  1.00  0.00           C  \\nATOM   1052  O   LEU A  67      -1.469 -17.184  16.217  1.00  0.00           O  \\nATOM   1053  CB  LEU A  67      -2.522 -15.062  14.182  1.00  0.00           C  \\nATOM   1054  CG  LEU A  67      -3.608 -14.826  13.168  1.00  0.00           C  \\nATOM   1055  CD1 LEU A  67      -4.216 -13.452  13.508  1.00  0.00           C  \\nATOM   1056  CD2 LEU A  67      -2.880 -14.680  11.866  1.00  0.00           C  \\nATOM   1057  H   LEU A  67      -3.590 -17.465  14.078  1.00  0.00           H  \\nATOM   1058  HA  LEU A  67      -1.065 -16.279  13.129  1.00  0.00           H  \\nATOM   1059 1HB  LEU A  67      -2.982 -15.070  15.169  1.00  0.00           H  \\nATOM   1060 2HB  LEU A  67      -1.830 -14.221  14.133  1.00  0.00           H  \\nATOM   1061  HG  LEU A  67      -4.289 -15.677  13.160  1.00  0.00           H  \\nATOM   1062 1HD1 LEU A  67      -5.015 -13.223  12.803  1.00  0.00           H  \\nATOM   1063 2HD1 LEU A  67      -4.621 -13.474  14.520  1.00  0.00           H  \\nATOM   1064 3HD1 LEU A  67      -3.444 -12.686  13.442  1.00  0.00           H  \\nATOM   1065 1HD2 LEU A  67      -3.599 -14.505  11.065  1.00  0.00           H  \\nATOM   1066 2HD2 LEU A  67      -2.191 -13.837  11.927  1.00  0.00           H  \\nATOM   1067 3HD2 LEU A  67      -2.320 -15.592  11.658  1.00  0.00           H  \\nATOM   1068  N   ALA A  68       0.282 -16.176  15.300  1.00  0.00           N  \\nATOM   1069  CA  ALA A  68       1.034 -16.070  16.559  1.00  0.00           C  \\nATOM   1070  C   ALA A  68       1.187 -14.603  17.041  1.00  0.00           C  \\nATOM   1071  O   ALA A  68       1.581 -13.720  16.236  1.00  0.00           O  \\nATOM   1072  CB  ALA A  68       2.424 -16.662  16.324  1.00  0.00           C  \\nATOM   1073  H   ALA A  68       0.732 -15.915  14.434  1.00  0.00           H  \\nATOM   1074  HA  ALA A  68       0.505 -16.645  17.319  1.00  0.00           H  \\nATOM   1075 1HB  ALA A  68       3.007 -16.597  17.243  1.00  0.00           H  \\nATOM   1076 2HB  ALA A  68       2.329 -17.707  16.027  1.00  0.00           H  \\nATOM   1077 3HB  ALA A  68       2.928 -16.105  15.535  1.00  0.00           H  \\nATOM   1078  N   LEU A  69       0.819 -14.384  18.292  1.00  0.00           N  \\nATOM   1079  CA  LEU A  69       0.921 -13.054  18.864  1.00  0.00           C  \\nATOM   1080  C   LEU A  69       1.961 -13.045  19.939  1.00  0.00           C  \\nATOM   1081  O   LEU A  69       2.047 -13.990  20.753  1.00  0.00           O  \\nATOM   1082  CB  LEU A  69      -0.425 -12.655  19.509  1.00  0.00           C  \\nATOM   1083  CG  LEU A  69      -1.552 -12.126  18.606  1.00  0.00           C  \\nATOM   1084  CD1 LEU A  69      -1.933 -13.039  17.478  1.00  0.00           C  \\nATOM   1085  CD2 LEU A  69      -2.810 -11.969  19.579  1.00  0.00           C  \\nATOM   1086  H   LEU A  69       0.464 -15.139  18.862  1.00  0.00           H  \\nATOM   1087  HA  LEU A  69       1.152 -12.351  18.065  1.00  0.00           H  \\nATOM   1088 1HB  LEU A  69      -0.834 -13.523  20.024  1.00  0.00           H  \\nATOM   1089 2HB  LEU A  69      -0.239 -11.875  20.247  1.00  0.00           H  \\nATOM   1090  HG  LEU A  69      -1.253 -11.171  18.173  1.00  0.00           H  \\nATOM   1091 1HD1 LEU A  69      -2.734 -12.583  16.896  1.00  0.00           H  \\nATOM   1092 2HD1 LEU A  69      -1.068 -13.205  16.836  1.00  0.00           H  \\nATOM   1093 3HD1 LEU A  69      -2.274 -13.992  17.881  1.00  0.00           H  \\nATOM   1094 1HD2 LEU A  69      -3.663 -11.593  19.013  1.00  0.00           H  \\nATOM   1095 2HD2 LEU A  69      -3.059 -12.939  20.011  1.00  0.00           H  \\nATOM   1096 3HD2 LEU A  69      -2.567 -11.268  20.378  1.00  0.00           H  \\nATOM   1097  N   ASP A  70       2.771 -11.995  19.968  1.00  0.00           N  \\nATOM   1098  CA  ASP A  70       3.773 -11.817  21.042  1.00  0.00           C  \\nATOM   1099  C   ASP A  70       3.003 -11.280  22.258  1.00  0.00           C  \\nATOM   1100  O   ASP A  70       2.334 -10.282  22.183  1.00  0.00           O  \\nATOM   1101  CB  ASP A  70       4.801 -10.823  20.457  1.00  0.00           C  \\nATOM   1102  CG  ASP A  70       6.067 -10.634  21.310  1.00  0.00           C  \\nATOM   1103  OD1 ASP A  70       5.923 -10.752  22.538  1.00  0.00           O  \\nATOM   1104  OD2 ASP A  70       7.114 -10.358  20.663  1.00  0.00           O  \\nATOM   1105  H   ASP A  70       2.702 -11.299  19.239  1.00  0.00           H  \\nATOM   1106  HA  ASP A  70       4.231 -12.784  21.252  1.00  0.00           H  \\nATOM   1107 1HB  ASP A  70       5.114 -11.162  19.469  1.00  0.00           H  \\nATOM   1108 2HB  ASP A  70       4.333  -9.846  20.335  1.00  0.00           H  \\nATOM   1109  N   VAL A  71       3.135 -11.951  23.413  1.00  0.00           N  \\nATOM   1110  CA  VAL A  71       2.427 -11.516  24.642  1.00  0.00           C  \\nATOM   1111  C   VAL A  71       2.984 -10.166  25.136  1.00  0.00           C  \\nATOM   1112  O   VAL A  71       2.273  -9.491  25.856  1.00  0.00           O  \\nATOM   1113  CB  VAL A  71       2.474 -12.490  25.807  1.00  0.00           C  \\nATOM   1114  CG1 VAL A  71       1.886 -13.876  25.407  1.00  0.00           C  \\nATOM   1115  CG2 VAL A  71       3.943 -12.653  26.317  1.00  0.00           C  \\nATOM   1116  H   VAL A  71       3.727 -12.768  23.450  1.00  0.00           H  \\nATOM   1117  HA  VAL A  71       1.371 -11.380  24.404  1.00  0.00           H  \\nATOM   1118  HB  VAL A  71       1.852 -12.106  26.615  1.00  0.00           H  \\nATOM   1119 1HG1 VAL A  71       1.933 -14.552  26.262  1.00  0.00           H  \\nATOM   1120 2HG1 VAL A  71       0.848 -13.755  25.097  1.00  0.00           H  \\nATOM   1121 3HG1 VAL A  71       2.465 -14.293  24.584  1.00  0.00           H  \\nATOM   1122 1HG2 VAL A  71       3.963 -13.353  27.152  1.00  0.00           H  \\nATOM   1123 2HG2 VAL A  71       4.568 -13.034  25.509  1.00  0.00           H  \\nATOM   1124 3HG2 VAL A  71       4.324 -11.686  26.646  1.00  0.00           H  \\nATOM   1125  N   THR A  72       4.173  -9.790  24.664  1.00  0.00           N  \\nATOM   1126  CA  THR A  72       4.751  -8.516  25.170  1.00  0.00           C  \\nATOM   1127  C   THR A  72       4.117  -7.281  24.543  1.00  0.00           C  \\nATOM   1128  O   THR A  72       4.216  -6.174  25.081  1.00  0.00           O  \\nATOM   1129  CB  THR A  72       6.279  -8.478  25.009  1.00  0.00           C  \\nATOM   1130  OG1 THR A  72       6.654  -8.436  23.670  1.00  0.00           O  \\nATOM   1131  CG2 THR A  72       6.951  -9.653  25.730  1.00  0.00           C  \\nATOM   1132  H   THR A  72       4.690 -10.329  23.985  1.00  0.00           H  \\nATOM   1133  HA  THR A  72       4.522  -8.430  26.233  1.00  0.00           H  \\nATOM   1134  HB  THR A  72       6.666  -7.548  25.424  1.00  0.00           H  \\nATOM   1135  HG1 THR A  72       5.868  -8.440  23.118  1.00  0.00           H  \\nATOM   1136 1HG2 THR A  72       8.031  -9.593  25.595  1.00  0.00           H  \\nATOM   1137 2HG2 THR A  72       6.715  -9.610  26.793  1.00  0.00           H  \\nATOM   1138 3HG2 THR A  72       6.585 -10.591  25.315  1.00  0.00           H  \\nATOM   1139  N   ASN A  73       3.514  -7.430  23.374  1.00  0.00           N  \\nATOM   1140  CA  ASN A  73       2.913  -6.259  22.729  1.00  0.00           C  \\nATOM   1141  C   ASN A  73       1.599  -6.568  22.079  1.00  0.00           C  \\nATOM   1142  O   ASN A  73       1.023  -5.709  21.431  1.00  0.00           O  \\nATOM   1143  CB  ASN A  73       3.907  -5.624  21.733  1.00  0.00           C  \\nATOM   1144  CG  ASN A  73       4.337  -6.584  20.683  1.00  0.00           C  \\nATOM   1145  OD1 ASN A  73       3.737  -7.631  20.448  1.00  0.00           O  \\nATOM   1146  ND2 ASN A  73       5.356  -6.163  19.914  1.00  0.00           N  \\nATOM   1147  H   ASN A  73       3.457  -8.328  22.915  1.00  0.00           H  \\nATOM   1148  HA  ASN A  73       2.673  -5.525  23.499  1.00  0.00           H  \\nATOM   1149 1HB  ASN A  73       3.443  -4.759  21.257  1.00  0.00           H  \\nATOM   1150 2HB  ASN A  73       4.785  -5.269  22.272  1.00  0.00           H  \\nATOM   1151 1HD2 ASN A  73       5.707  -6.749  19.183  1.00  0.00           H  \\nATOM   1152 2HD2 ASN A  73       5.763  -5.264  20.072  1.00  0.00           H  \\nATOM   1153  N   ALA A  74       1.090  -7.782  22.253  1.00  0.00           N  \\nATOM   1154  CA  ALA A  74      -0.143  -8.274  21.622  1.00  0.00           C  \\nATOM   1155  C   ALA A  74      -0.159  -8.300  20.110  1.00  0.00           C  \\nATOM   1156  O   ALA A  74      -1.216  -8.488  19.537  1.00  0.00           O  \\nATOM   1157  CB  ALA A  74      -1.382  -7.484  22.142  1.00  0.00           C  \\nATOM   1158  H   ALA A  74       1.606  -8.394  22.869  1.00  0.00           H  \\nATOM   1159  HA  ALA A  74      -0.262  -9.325  21.886  1.00  0.00           H  \\nATOM   1160 1HB  ALA A  74      -2.284  -7.864  21.662  1.00  0.00           H  \\nATOM   1161 2HB  ALA A  74      -1.466  -7.608  23.222  1.00  0.00           H  \\nATOM   1162 3HB  ALA A  74      -1.264  -6.427  21.907  1.00  0.00           H  \\nATOM   1163  N   TYR A  75       0.991  -8.201  19.477  1.00  0.00           N  \\nATOM   1164  CA  TYR A  75       1.019  -7.972  18.016  1.00  0.00           C  \\nATOM   1165  C   TYR A  75       1.306  -9.288  17.267  1.00  0.00           C  \\nATOM   1166  O   TYR A  75       1.927 -10.189  17.829  1.00  0.00           O  \\nATOM   1167  CB  TYR A  75       2.059  -6.898  17.648  1.00  0.00           C  \\nATOM   1168  CG  TYR A  75       1.810  -6.181  16.343  1.00  0.00           C  \\nATOM   1169  CD1 TYR A  75       0.954  -5.088  16.263  1.00  0.00           C  \\nATOM   1170  CD2 TYR A  75       2.500  -6.535  15.169  1.00  0.00           C  \\nATOM   1171  CE1 TYR A  75       0.789  -4.368  15.078  1.00  0.00           C  \\nATOM   1172  CE2 TYR A  75       2.342  -5.834  13.989  1.00  0.00           C  \\nATOM   1173  CZ  TYR A  75       1.439  -4.763  13.944  1.00  0.00           C  \\nATOM   1174  OH  TYR A  75       1.307  -4.041  12.776  1.00  0.00           O  \\nATOM   1175  H   TYR A  75       1.864  -8.280  19.979  1.00  0.00           H  \\nATOM   1176  HA  TYR A  75       0.035  -7.622  17.702  1.00  0.00           H  \\nATOM   1177 1HB  TYR A  75       2.096  -6.142  18.434  1.00  0.00           H  \\nATOM   1178 2HB  TYR A  75       3.047  -7.354  17.586  1.00  0.00           H  \\nATOM   1179  HD1 TYR A  75       0.389  -4.772  17.141  1.00  0.00           H  \\nATOM   1180  HD2 TYR A  75       3.182  -7.385  15.173  1.00  0.00           H  \\nATOM   1181  HE1 TYR A  75       0.110  -3.516  15.048  1.00  0.00           H  \\nATOM   1182  HE2 TYR A  75       2.916  -6.117  13.106  1.00  0.00           H  \\nATOM   1183  HH  TYR A  75       1.897  -4.402  12.110  1.00  0.00           H  \\nATOM   1184  N   VAL A  76       0.866  -9.391  16.020  1.00  0.00           N  \\nATOM   1185  CA  VAL A  76       1.082 -10.616  15.204  1.00  0.00           C  \\nATOM   1186  C   VAL A  76       2.523 -10.653  14.723  1.00  0.00           C  \\nATOM   1187  O   VAL A  76       3.011  -9.697  14.161  1.00  0.00           O  \\nATOM   1188  CB  VAL A  76       0.122 -10.500  14.000  1.00  0.00           C  \\nATOM   1189  CG1 VAL A  76       0.372 -11.674  12.973  1.00  0.00           C  \\nATOM   1190  CG2 VAL A  76      -1.304 -10.580  14.428  1.00  0.00           C  \\nATOM   1191  H   VAL A  76       0.369  -8.611  15.614  1.00  0.00           H  \\nATOM   1192  HA  VAL A  76       0.834 -11.486  15.813  1.00  0.00           H  \\nATOM   1193  HB  VAL A  76       0.286  -9.541  13.507  1.00  0.00           H  \\nATOM   1194 1HG1 VAL A  76      -0.314 -11.574  12.132  1.00  0.00           H  \\nATOM   1195 2HG1 VAL A  76       1.399 -11.628  12.610  1.00  0.00           H  \\nATOM   1196 3HG1 VAL A  76       0.203 -12.631  13.466  1.00  0.00           H  \\nATOM   1197 1HG2 VAL A  76      -1.951 -10.494  13.555  1.00  0.00           H  \\nATOM   1198 2HG2 VAL A  76      -1.483 -11.536  14.920  1.00  0.00           H  \\nATOM   1199 3HG2 VAL A  76      -1.522  -9.768  15.122  1.00  0.00           H  \\nATOM   1200  N   VAL A  77       3.199 -11.790  14.958  1.00  0.00           N  \\nATOM   1201  CA  VAL A  77       4.581 -12.023  14.462  1.00  0.00           C  \\nATOM   1202  C   VAL A  77       4.666 -13.001  13.323  1.00  0.00           C  \\nATOM   1203  O   VAL A  77       5.631 -13.001  12.595  1.00  0.00           O  \\nATOM   1204  CB  VAL A  77       5.539 -12.478  15.539  1.00  0.00           C  \\nATOM   1205  CG1 VAL A  77       5.599 -11.427  16.643  1.00  0.00           C  \\nATOM   1206  CG2 VAL A  77       5.299 -13.844  16.175  1.00  0.00           C  \\nATOM   1207  H   VAL A  77       2.741 -12.512  15.496  1.00  0.00           H  \\nATOM   1208  HA  VAL A  77       4.968 -11.085  14.062  1.00  0.00           H  \\nATOM   1209  HB  VAL A  77       6.545 -12.525  15.122  1.00  0.00           H  \\nATOM   1210 1HG1 VAL A  77       6.290 -11.754  17.420  1.00  0.00           H  \\nATOM   1211 2HG1 VAL A  77       5.944 -10.481  16.226  1.00  0.00           H  \\nATOM   1212 3HG1 VAL A  77       4.607 -11.295  17.074  1.00  0.00           H  \\nATOM   1213 1HG2 VAL A  77       6.064 -14.034  16.928  1.00  0.00           H  \\nATOM   1214 2HG2 VAL A  77       4.315 -13.859  16.645  1.00  0.00           H  \\nATOM   1215 3HG2 VAL A  77       5.346 -14.616  15.407  1.00  0.00           H  \\nATOM   1216  N   GLY A  78       3.618 -13.693  13.091  1.00  0.00           N  \\nATOM   1217  CA  GLY A  78       3.629 -14.715  11.968  1.00  0.00           C  \\nATOM   1218  C   GLY A  78       2.377 -15.519  12.000  1.00  0.00           C  \\nATOM   1219  O   GLY A  78       1.488 -15.284  12.801  1.00  0.00           O  \\nATOM   1220  H   GLY A  78       2.779 -13.575  13.640  1.00  0.00           H  \\nATOM   1221 1HA  GLY A  78       3.726 -14.203  11.011  1.00  0.00           H  \\nATOM   1222 2HA  GLY A  78       4.498 -15.363  12.075  1.00  0.00           H  \\nATOM   1223  N   TYR A  79       2.317 -16.487  11.064  1.00  0.00           N  \\nATOM   1224  CA  TYR A  79       1.168 -17.363  11.027  1.00  0.00           C  \\nATOM   1225  C   TYR A  79       1.486 -18.684  10.382  1.00  0.00           C  \\nATOM   1226  O   TYR A  79       2.454 -18.778   9.646  1.00  0.00           O  \\nATOM   1227  CB  TYR A  79      -0.012 -16.701  10.306  1.00  0.00           C  \\nATOM   1228  CG  TYR A  79       0.020 -16.754   8.771  1.00  0.00           C  \\nATOM   1229  CD1 TYR A  79       0.788 -15.852   8.050  1.00  0.00           C  \\nATOM   1230  CD2 TYR A  79      -0.770 -17.659   8.102  1.00  0.00           C  \\nATOM   1231  CE1 TYR A  79       0.762 -15.833   6.680  1.00  0.00           C  \\nATOM   1232  CE2 TYR A  79      -0.810 -17.613   6.686  1.00  0.00           C  \\nATOM   1233  CZ  TYR A  79      -0.030 -16.689   6.015  1.00  0.00           C  \\nATOM   1234  OH  TYR A  79      -0.058 -16.633   4.648  1.00  0.00           O  \\nATOM   1235  H   TYR A  79       3.056 -16.616  10.387  1.00  0.00           H  \\nATOM   1236  HA  TYR A  79       0.865 -17.579  12.052  1.00  0.00           H  \\nATOM   1237 1HB  TYR A  79      -0.943 -17.173  10.622  1.00  0.00           H  \\nATOM   1238 2HB  TYR A  79      -0.067 -15.650  10.587  1.00  0.00           H  \\nATOM   1239  HD1 TYR A  79       1.428 -15.141   8.573  1.00  0.00           H  \\nATOM   1240  HD2 TYR A  79      -1.351 -18.393   8.661  1.00  0.00           H  \\nATOM   1241  HE1 TYR A  79       1.374 -15.116   6.133  1.00  0.00           H  \\nATOM   1242  HE2 TYR A  79      -1.452 -18.301   6.136  1.00  0.00           H  \\nATOM   1243  HH  TYR A  79      -0.665 -17.298   4.314  1.00  0.00           H  \\nATOM   1244  N   ARG A  80       0.609 -19.630  10.652  1.00  0.00           N  \\nATOM   1245  CA  ARG A  80       0.679 -20.964   9.996  1.00  0.00           C  \\nATOM   1246  C   ARG A  80      -0.553 -21.193   9.131  1.00  0.00           C  \\nATOM   1247  O   ARG A  80      -1.666 -20.916   9.520  1.00  0.00           O  \\nATOM   1248  CB  ARG A  80       0.748 -22.002  11.122  1.00  0.00           C  \\nATOM   1249  CG  ARG A  80       0.773 -23.414  10.536  1.00  0.00           C  \\nATOM   1250  CD  ARG A  80       0.875 -24.518  11.578  1.00  0.00           C  \\nATOM   1251  NE  ARG A  80      -0.333 -24.699  12.340  1.00  0.00           N  \\nATOM   1252  CZ  ARG A  80      -1.436 -25.377  11.924  1.00  0.00           C  \\nATOM   1253  NH1 ARG A  80      -1.503 -25.971  10.709  1.00  0.00           N  \\nATOM   1254  NH2 ARG A  80      -2.499 -25.496  12.743  1.00  0.00           N  \\nATOM   1255  H   ARG A  80      -0.128 -19.448  11.319  1.00  0.00           H  \\nATOM   1256  HA  ARG A  80       1.582 -21.002   9.385  1.00  0.00           H  \\nATOM   1257 1HB  ARG A  80       1.641 -21.830  11.721  1.00  0.00           H  \\nATOM   1258 2HB  ARG A  80      -0.114 -21.884  11.779  1.00  0.00           H  \\nATOM   1259 1HG  ARG A  80      -0.143 -23.590   9.972  1.00  0.00           H  \\nATOM   1260 2HG  ARG A  80       1.633 -23.518   9.874  1.00  0.00           H  \\nATOM   1261 1HD  ARG A  80       1.095 -25.464  11.085  1.00  0.00           H  \\nATOM   1262 2HD  ARG A  80       1.673 -24.282  12.281  1.00  0.00           H  \\nATOM   1263  HE  ARG A  80      -0.372 -24.290  13.264  1.00  0.00           H  \\nATOM   1264 1HH1 ARG A  80      -0.716 -25.919  10.077  1.00  0.00           H  \\nATOM   1265 2HH1 ARG A  80      -2.339 -26.466  10.434  1.00  0.00           H  \\nATOM   1266 1HH2 ARG A  80      -2.476 -25.082  13.665  1.00  0.00           H  \\nATOM   1267 2HH2 ARG A  80      -3.318 -25.999  12.436  1.00  0.00           H  \\nATOM   1268  N   ALA A  81      -0.326 -21.770   7.946  1.00  0.00           N  \\nATOM   1269  CA  ALA A  81      -1.419 -22.263   7.132  1.00  0.00           C  \\nATOM   1270  C   ALA A  81      -0.956 -23.637   6.680  1.00  0.00           C  \\nATOM   1271  O   ALA A  81       0.013 -23.734   5.973  1.00  0.00           O  \\nATOM   1272  CB  ALA A  81      -1.677 -21.329   5.978  1.00  0.00           C  \\nATOM   1273  H   ALA A  81       0.621 -21.866   7.608  1.00  0.00           H  \\nATOM   1274  HA  ALA A  81      -2.312 -22.308   7.756  1.00  0.00           H  \\nATOM   1275 1HB  ALA A  81      -2.501 -21.713   5.376  1.00  0.00           H  \\nATOM   1276 2HB  ALA A  81      -1.936 -20.342   6.360  1.00  0.00           H  \\nATOM   1277 3HB  ALA A  81      -0.782 -21.256   5.363  1.00  0.00           H  \\nATOM   1278  N   GLY A  82      -1.612 -24.631   7.223  1.00  0.00           N  \\nATOM   1279  CA  GLY A  82      -1.390 -26.048   6.803  1.00  0.00           C  \\nATOM   1280  C   GLY A  82       0.015 -26.362   7.209  1.00  0.00           C  \\nATOM   1281  O   GLY A  82       0.389 -26.281   8.382  1.00  0.00           O  \\nATOM   1282  H   GLY A  82      -2.288 -24.433   7.947  1.00  0.00           H  \\nATOM   1283 1HA  GLY A  82      -2.123 -26.691   7.290  1.00  0.00           H  \\nATOM   1284 2HA  GLY A  82      -1.548 -26.138   5.729  1.00  0.00           H  \\nATOM   1285  N   ASN A  83       0.845 -26.739   6.205  1.00  0.00           N  \\nATOM   1286  CA  ASN A  83       2.217 -27.099   6.470  1.00  0.00           C  \\nATOM   1287  C   ASN A  83       3.222 -26.048   6.150  1.00  0.00           C  \\nATOM   1288  O   ASN A  83       4.411 -26.316   5.869  1.00  0.00           O  \\nATOM   1289  CB  ASN A  83       2.544 -28.430   5.705  1.00  0.00           C  \\nATOM   1290  CG  ASN A  83       2.803 -29.489   6.706  1.00  0.00           C  \\nATOM   1291  OD1 ASN A  83       1.918 -30.248   7.062  1.00  0.00           O  \\nATOM   1292  ND2 ASN A  83       3.973 -29.404   7.334  1.00  0.00           N  \\nATOM   1293  H   ASN A  83       0.503 -26.770   5.255  1.00  0.00           H  \\nATOM   1294  HA  ASN A  83       2.332 -27.258   7.543  1.00  0.00           H  \\nATOM   1295 1HB  ASN A  83       1.704 -28.696   5.062  1.00  0.00           H  \\nATOM   1296 2HB  ASN A  83       3.412 -28.277   5.063  1.00  0.00           H  \\nATOM   1297 1HD2 ASN A  83       4.220 -30.084   8.025  1.00  0.00           H  \\nATOM   1298 2HD2 ASN A  83       4.605 -28.661   7.115  1.00  0.00           H  \\nATOM   1299  N   SER A  84       2.797 -24.794   6.254  1.00  0.00           N  \\nATOM   1300  CA  SER A  84       3.713 -23.703   5.988  1.00  0.00           C  \\nATOM   1301  C   SER A  84       3.577 -22.681   7.081  1.00  0.00           C  \\nATOM   1302  O   SER A  84       2.499 -22.462   7.581  1.00  0.00           O  \\nATOM   1303  CB  SER A  84       3.414 -23.076   4.657  1.00  0.00           C  \\nATOM   1304  OG  SER A  84       3.684 -24.095   3.618  1.00  0.00           O  \\nATOM   1305  H   SER A  84       1.844 -24.584   6.516  1.00  0.00           H  \\nATOM   1306  HA  SER A  84       4.728 -24.101   5.965  1.00  0.00           H  \\nATOM   1307 1HB  SER A  84       2.375 -22.748   4.634  1.00  0.00           H  \\nATOM   1308 2HB  SER A  84       4.038 -22.194   4.522  1.00  0.00           H  \\nATOM   1309  HG  SER A  84       3.978 -24.880   4.087  1.00  0.00           H  \\nATOM   1310  N   ALA A  85       4.689 -22.107   7.477  1.00  0.00           N  \\nATOM   1311  CA  ALA A  85       4.572 -20.960   8.462  1.00  0.00           C  \\nATOM   1312  C   ALA A  85       5.377 -19.791   7.892  1.00  0.00           C  \\nATOM   1313  O   ALA A  85       6.404 -19.926   7.141  1.00  0.00           O  \\nATOM   1314  CB  ALA A  85       5.104 -21.369   9.801  1.00  0.00           C  \\nATOM   1315  H   ALA A  85       5.605 -22.391   7.159  1.00  0.00           H  \\nATOM   1316  HA  ALA A  85       3.517 -20.707   8.563  1.00  0.00           H  \\nATOM   1317 1HB  ALA A  85       5.014 -20.536  10.498  1.00  0.00           H  \\nATOM   1318 2HB  ALA A  85       4.533 -22.219  10.175  1.00  0.00           H  \\nATOM   1319 3HB  ALA A  85       6.152 -21.650   9.706  1.00  0.00           H  \\nATOM   1320  N   TYR A  86       4.899 -18.570   8.200  1.00  0.00           N  \\nATOM   1321  CA  TYR A  86       5.390 -17.320   7.643  1.00  0.00           C  \\nATOM   1322  C   TYR A  86       5.561 -16.284   8.752  1.00  0.00           C  \\nATOM   1323  O   TYR A  86       4.619 -16.069   9.557  1.00  0.00           O  \\nATOM   1324  CB  TYR A  86       4.426 -16.722   6.587  1.00  0.00           C  \\nATOM   1325  CG  TYR A  86       4.094 -17.702   5.518  1.00  0.00           C  \\nATOM   1326  CD1 TYR A  86       2.986 -18.583   5.637  1.00  0.00           C  \\nATOM   1327  CD2 TYR A  86       4.886 -17.765   4.402  1.00  0.00           C  \\nATOM   1328  CE1 TYR A  86       2.680 -19.526   4.644  1.00  0.00           C  \\nATOM   1329  CE2 TYR A  86       4.579 -18.751   3.385  1.00  0.00           C  \\nATOM   1330  CZ  TYR A  86       3.537 -19.586   3.574  1.00  0.00           C  \\nATOM   1331  OH  TYR A  86       3.231 -20.531   2.572  1.00  0.00           O  \\nATOM   1332  H   TYR A  86       4.144 -18.547   8.870  1.00  0.00           H  \\nATOM   1333  HA  TYR A  86       6.343 -17.512   7.150  1.00  0.00           H  \\nATOM   1334 1HB  TYR A  86       3.505 -16.400   7.075  1.00  0.00           H  \\nATOM   1335 2HB  TYR A  86       4.882 -15.842   6.135  1.00  0.00           H  \\nATOM   1336  HD1 TYR A  86       2.346 -18.538   6.518  1.00  0.00           H  \\nATOM   1337  HD2 TYR A  86       5.729 -17.082   4.292  1.00  0.00           H  \\nATOM   1338  HE1 TYR A  86       1.823 -20.187   4.766  1.00  0.00           H  \\nATOM   1339  HE2 TYR A  86       5.188 -18.811   2.482  1.00  0.00           H  \\nATOM   1340  HH  TYR A  86       3.853 -20.439   1.846  1.00  0.00           H  \\nATOM   1341  N   PHE A  87       6.721 -15.649   8.789  1.00  0.00           N  \\nATOM   1342  CA  PHE A  87       7.053 -14.720   9.916  1.00  0.00           C  \\nATOM   1343  C   PHE A  87       7.441 -13.377   9.341  1.00  0.00           C  \\nATOM   1344  O   PHE A  87       8.157 -13.304   8.325  1.00  0.00           O  \\nATOM   1345  CB  PHE A  87       8.183 -15.289  10.738  1.00  0.00           C  \\nATOM   1346  CG  PHE A  87       7.860 -16.595  11.435  1.00  0.00           C  \\nATOM   1347  CD1 PHE A  87       7.177 -16.683  12.597  1.00  0.00           C  \\nATOM   1348  CD2 PHE A  87       8.211 -17.808  10.783  1.00  0.00           C  \\nATOM   1349  CE1 PHE A  87       6.933 -17.941  13.139  1.00  0.00           C  \\nATOM   1350  CE2 PHE A  87       7.928 -19.006  11.354  1.00  0.00           C  \\nATOM   1351  CZ  PHE A  87       7.271 -19.102  12.440  1.00  0.00           C  \\nATOM   1352  H   PHE A  87       7.396 -15.788   8.051  1.00  0.00           H  \\nATOM   1353  HA  PHE A  87       6.172 -14.617  10.551  1.00  0.00           H  \\nATOM   1354 1HB  PHE A  87       9.048 -15.459  10.098  1.00  0.00           H  \\nATOM   1355 2HB  PHE A  87       8.476 -14.569  11.501  1.00  0.00           H  \\nATOM   1356  HD1 PHE A  87       6.833 -15.770  13.083  1.00  0.00           H  \\nATOM   1357  HD2 PHE A  87       8.777 -17.768   9.852  1.00  0.00           H  \\nATOM   1358  HE1 PHE A  87       6.472 -18.030  14.122  1.00  0.00           H  \\nATOM   1359  HE2 PHE A  87       8.284 -19.905  10.852  1.00  0.00           H  \\nATOM   1360  HZ  PHE A  87       6.975 -20.077  12.825  1.00  0.00           H  \\nATOM   1361  N   PHE A  88       6.948 -12.274   9.936  1.00  0.00           N  \\nATOM   1362  CA  PHE A  88       7.534 -10.993   9.606  1.00  0.00           C  \\nATOM   1363  C   PHE A  88       8.976 -10.937   9.939  1.00  0.00           C  \\nATOM   1364  O   PHE A  88       9.486 -11.617  10.826  1.00  0.00           O  \\nATOM   1365  CB  PHE A  88       6.846  -9.821  10.300  1.00  0.00           C  \\nATOM   1366  CG  PHE A  88       5.418  -9.685   9.998  1.00  0.00           C  \\nATOM   1367  CD1 PHE A  88       4.997  -9.424   8.698  1.00  0.00           C  \\nATOM   1368  CD2 PHE A  88       4.418  -9.815  11.035  1.00  0.00           C  \\nATOM   1369  CE1 PHE A  88       3.663  -9.291   8.412  1.00  0.00           C  \\nATOM   1370  CE2 PHE A  88       3.049  -9.647  10.738  1.00  0.00           C  \\nATOM   1371  CZ  PHE A  88       2.656  -9.367   9.453  1.00  0.00           C  \\nATOM   1372  H   PHE A  88       6.188 -12.311  10.600  1.00  0.00           H  \\nATOM   1373  HA  PHE A  88       7.438 -10.835   8.531  1.00  0.00           H  \\nATOM   1374 1HB  PHE A  88       6.948  -9.927  11.379  1.00  0.00           H  \\nATOM   1375 2HB  PHE A  88       7.336  -8.891  10.014  1.00  0.00           H  \\nATOM   1376  HD1 PHE A  88       5.741  -9.327   7.907  1.00  0.00           H  \\nATOM   1377  HD2 PHE A  88       4.731 -10.037  12.056  1.00  0.00           H  \\nATOM   1378  HE1 PHE A  88       3.341  -9.125   7.384  1.00  0.00           H  \\nATOM   1379  HE2 PHE A  88       2.303  -9.738  11.528  1.00  0.00           H  \\nATOM   1380  HZ  PHE A  88       1.607  -9.202   9.211  1.00  0.00           H  \\nATOM   1381  N   HIS A  89       9.676 -10.077   9.184  1.00  0.00           N  \\nATOM   1382  CA  HIS A  89      11.123  -9.910   9.476  1.00  0.00           C  \\nATOM   1383  C   HIS A  89      11.405  -9.362  10.848  1.00  0.00           C  \\nATOM   1384  O   HIS A  89      11.063  -8.177  11.128  1.00  0.00           O  \\nATOM   1385  CB  HIS A  89      11.731  -8.955   8.447  1.00  0.00           C  \\nATOM   1386  CG  HIS A  89      13.248  -8.977   8.391  1.00  0.00           C  \\nATOM   1387  ND1 HIS A  89      14.021  -7.887   8.717  1.00  0.00           N  \\nATOM   1388  CD2 HIS A  89      14.116  -9.939   7.980  1.00  0.00           C  \\nATOM   1389  CE1 HIS A  89      15.302  -8.191   8.562  1.00  0.00           C  \\nATOM   1390  NE2 HIS A  89      15.387  -9.433   8.127  1.00  0.00           N  \\nATOM   1391  H   HIS A  89       9.261  -9.545   8.433  1.00  0.00           H  \\nATOM   1392  HA  HIS A  89      11.623 -10.875   9.400  1.00  0.00           H  \\nATOM   1393 1HB  HIS A  89      11.356  -9.203   7.454  1.00  0.00           H  \\nATOM   1394 2HB  HIS A  89      11.419  -7.935   8.669  1.00  0.00           H  \\nATOM   1395  HD2 HIS A  89      13.854 -10.940   7.637  1.00  0.00           H  \\nATOM   1396  HE1 HIS A  89      16.143  -7.527   8.761  1.00  0.00           H  \\nATOM   1397  HE2 HIS A  89      16.242  -9.934   7.933  1.00  0.00           H  \\nATOM   1398  N   PRO A  90      12.089 -10.185  11.652  1.00  0.00           N  \\nATOM   1399  CA  PRO A  90      12.365  -9.738  13.010  1.00  0.00           C  \\nATOM   1400  C   PRO A  90      13.248  -8.455  13.042  1.00  0.00           C  \\nATOM   1401  O   PRO A  90      14.143  -8.281  12.205  1.00  0.00           O  \\nATOM   1402  CB  PRO A  90      13.151 -10.895  13.622  1.00  0.00           C  \\nATOM   1403  CG  PRO A  90      13.020 -11.996  12.720  1.00  0.00           C  \\nATOM   1404  CD  PRO A  90      12.730 -11.474  11.347  1.00  0.00           C  \\nATOM   1405  HA  PRO A  90      11.414  -9.591  13.543  1.00  0.00           H  \\nATOM   1406 1HB  PRO A  90      14.203 -10.604  13.759  1.00  0.00           H  \\nATOM   1407 2HB  PRO A  90      12.752 -11.136  14.618  1.00  0.00           H  \\nATOM   1408 1HG  PRO A  90      13.944 -12.593  12.713  1.00  0.00           H  \\nATOM   1409 2HG  PRO A  90      12.212 -12.665  13.052  1.00  0.00           H  \\nATOM   1410 1HD  PRO A  90      13.673 -11.353  10.794  1.00  0.00           H  \\nATOM   1411 2HD  PRO A  90      12.059 -12.171  10.823  1.00  0.00           H  \\nATOM   1412  N   ASP A  91      13.011  -7.644  14.045  1.00  0.00           N  \\nATOM   1413  CA  ASP A  91      13.717  -6.424  14.329  1.00  0.00           C  \\nATOM   1414  C   ASP A  91      15.141  -6.664  14.830  1.00  0.00           C  \\nATOM   1415  O   ASP A  91      15.962  -5.772  14.653  1.00  0.00           O  \\nATOM   1416  CB  ASP A  91      12.915  -5.578  15.379  1.00  0.00           C  \\nATOM   1417  CG  ASP A  91      11.496  -4.944  14.842  1.00  0.00           C  \\nATOM   1418  OD1 ASP A  91      11.155  -4.827  13.620  1.00  0.00           O  \\nATOM   1419  OD2 ASP A  91      10.691  -4.478  15.704  1.00  0.00           O  \\nATOM   1420  H   ASP A  91      12.258  -7.930  14.655  1.00  0.00           H  \\nATOM   1421  HA  ASP A  91      13.804  -5.852  13.405  1.00  0.00           H  \\nATOM   1422 1HB  ASP A  91      12.685  -6.197  16.246  1.00  0.00           H  \\nATOM   1423 2HB  ASP A  91      13.531  -4.747  15.724  1.00  0.00           H  \\nATOM   1424  N   ASN A  92      15.435  -7.811  15.491  1.00  0.00           N  \\nATOM   1425  CA  ASN A  92      16.730  -8.144  16.088  1.00  0.00           C  \\nATOM   1426  C   ASN A  92      16.927  -9.666  16.311  1.00  0.00           C  \\nATOM   1427  O   ASN A  92      16.005 -10.477  16.152  1.00  0.00           O  \\nATOM   1428  CB  ASN A  92      16.945  -7.438  17.447  1.00  0.00           C  \\nATOM   1429  CG  ASN A  92      15.829  -7.755  18.467  1.00  0.00           C  \\nATOM   1430  OD1 ASN A  92      15.781  -8.838  19.048  1.00  0.00           O  \\nATOM   1431  ND2 ASN A  92      14.874  -6.830  18.616  1.00  0.00           N  \\nATOM   1432  H   ASN A  92      14.676  -8.473  15.562  1.00  0.00           H  \\nATOM   1433  HA  ASN A  92      17.519  -7.812  15.411  1.00  0.00           H  \\nATOM   1434 1HB  ASN A  92      17.903  -7.744  17.869  1.00  0.00           H  \\nATOM   1435 2HB  ASN A  92      16.984  -6.359  17.294  1.00  0.00           H  \\nATOM   1436 1HD2 ASN A  92      14.126  -6.983  19.263  1.00  0.00           H  \\nATOM   1437 2HD2 ASN A  92      14.907  -5.987  18.081  1.00  0.00           H  \\nATOM   1438  N   GLN A  93      18.146 -10.028  16.682  1.00  0.00           N  \\nATOM   1439  CA  GLN A  93      18.565 -11.424  16.820  1.00  0.00           C  \\nATOM   1440  C   GLN A  93      17.723 -12.164  17.847  1.00  0.00           C  \\nATOM   1441  O   GLN A  93      17.443 -13.377  17.699  1.00  0.00           O  \\nATOM   1442  CB  GLN A  93      20.061 -11.483  17.260  1.00  0.00           C  \\nATOM   1443  CG  GLN A  93      20.698 -12.864  17.151  1.00  0.00           C  \\nATOM   1444  CD  GLN A  93      20.389 -13.839  18.314  1.00  0.00           C  \\nATOM   1445  OE1 GLN A  93      20.203 -13.436  19.485  1.00  0.00           O  \\nATOM   1446  NE2 GLN A  93      20.362 -15.150  17.984  1.00  0.00           N  \\nATOM   1447  H   GLN A  93      18.811  -9.293  16.877  1.00  0.00           H  \\nATOM   1448  HA  GLN A  93      18.460 -11.914  15.852  1.00  0.00           H  \\nATOM   1449 1HB  GLN A  93      20.647 -10.796  16.649  1.00  0.00           H  \\nATOM   1450 2HB  GLN A  93      20.149 -11.156  18.296  1.00  0.00           H  \\nATOM   1451 1HG  GLN A  93      20.345 -13.345  16.239  1.00  0.00           H  \\nATOM   1452 2HG  GLN A  93      21.782 -12.751  17.118  1.00  0.00           H  \\nATOM   1453 1HE2 GLN A  93      20.169 -15.839  18.684  1.00  0.00           H  \\nATOM   1454 2HE2 GLN A  93      20.535 -15.432  17.040  1.00  0.00           H  \\nATOM   1455  N   GLU A  94      17.364 -11.480  18.927  1.00  0.00           N  \\nATOM   1456  CA  GLU A  94      16.705 -12.154  20.009  1.00  0.00           C  \\nATOM   1457  C   GLU A  94      15.274 -12.502  19.518  1.00  0.00           C  \\nATOM   1458  O   GLU A  94      14.745 -13.614  19.829  1.00  0.00           O  \\nATOM   1459  CB  GLU A  94      16.708 -11.267  21.275  1.00  0.00           C  \\nATOM   1460  CG  GLU A  94      17.980 -11.477  22.152  1.00  0.00           C  \\nATOM   1461  CD  GLU A  94      19.220 -10.798  21.524  1.00  0.00           C  \\nATOM   1462  OE1 GLU A  94      19.062  -9.623  21.045  1.00  0.00           O  \\nATOM   1463  OE2 GLU A  94      20.329 -11.416  21.470  1.00  0.00           O  \\nATOM   1464  H   GLU A  94      17.546 -10.489  19.003  1.00  0.00           H  \\nATOM   1465  HA  GLU A  94      17.248 -13.075  20.225  1.00  0.00           H  \\nATOM   1466 1HB  GLU A  94      16.651 -10.218  20.984  1.00  0.00           H  \\nATOM   1467 2HB  GLU A  94      15.826 -11.488  21.876  1.00  0.00           H  \\nATOM   1468 1HG  GLU A  94      17.799 -11.060  23.142  1.00  0.00           H  \\nATOM   1469 2HG  GLU A  94      18.158 -12.545  22.265  1.00  0.00           H  \\nATOM   1470  N   ASP A  95      14.637 -11.575  18.839  1.00  0.00           N  \\nATOM   1471  CA  ASP A  95      13.273 -11.857  18.305  1.00  0.00           C  \\nATOM   1472  C   ASP A  95      13.339 -12.897  17.203  1.00  0.00           C  \\nATOM   1473  O   ASP A  95      12.432 -13.746  17.086  1.00  0.00           O  \\nATOM   1474  CB  ASP A  95      12.665 -10.615  17.732  1.00  0.00           C  \\nATOM   1475  CG  ASP A  95      12.134  -9.624  18.854  1.00  0.00           C  \\nATOM   1476  OD1 ASP A  95      12.030  -9.976  20.073  1.00  0.00           O  \\nATOM   1477  OD2 ASP A  95      11.778  -8.488  18.434  1.00  0.00           O  \\nATOM   1478  H   ASP A  95      15.054 -10.671  18.670  1.00  0.00           H  \\nATOM   1479  HA  ASP A  95      12.645 -12.205  19.126  1.00  0.00           H  \\nATOM   1480 1HB  ASP A  95      13.405 -10.095  17.124  1.00  0.00           H  \\nATOM   1481 2HB  ASP A  95      11.835 -10.884  17.078  1.00  0.00           H  \\nATOM   1482  N   ALA A  96      14.407 -12.867  16.384  1.00  0.00           N  \\nATOM   1483  CA  ALA A  96      14.683 -13.994  15.456  1.00  0.00           C  \\nATOM   1484  C   ALA A  96      14.801 -15.350  16.109  1.00  0.00           C  \\nATOM   1485  O   ALA A  96      14.225 -16.345  15.599  1.00  0.00           O  \\nATOM   1486  CB  ALA A  96      15.971 -13.723  14.623  1.00  0.00           C  \\nATOM   1487  H   ALA A  96      15.032 -12.074  16.394  1.00  0.00           H  \\nATOM   1488  HA  ALA A  96      13.838 -14.085  14.773  1.00  0.00           H  \\nATOM   1489 1HB  ALA A  96      16.153 -14.562  13.951  1.00  0.00           H  \\nATOM   1490 2HB  ALA A  96      15.842 -12.812  14.039  1.00  0.00           H  \\nATOM   1491 3HB  ALA A  96      16.820 -13.606  15.294  1.00  0.00           H  \\nATOM   1492  N   GLU A  97      15.547 -15.442  17.210  1.00  0.00           N  \\nATOM   1493  CA  GLU A  97      15.644 -16.691  17.955  1.00  0.00           C  \\nATOM   1494  C   GLU A  97      14.282 -17.086  18.542  1.00  0.00           C  \\nATOM   1495  O   GLU A  97      13.870 -18.289  18.506  1.00  0.00           O  \\nATOM   1496  CB  GLU A  97      16.705 -16.566  19.059  1.00  0.00           C  \\nATOM   1497  CG  GLU A  97      17.156 -17.943  19.609  1.00  0.00           C  \\nATOM   1498  CD  GLU A  97      16.252 -18.586  20.665  1.00  0.00           C  \\nATOM   1499  OE1 GLU A  97      15.534 -17.812  21.347  1.00  0.00           O  \\nATOM   1500  OE2 GLU A  97      16.290 -19.837  20.868  1.00  0.00           O  \\nATOM   1501  H   GLU A  97      16.056 -14.633  17.537  1.00  0.00           H  \\nATOM   1502  HA  GLU A  97      15.944 -17.482  17.267  1.00  0.00           H  \\nATOM   1503 1HB  GLU A  97      17.577 -16.040  18.668  1.00  0.00           H  \\nATOM   1504 2HB  GLU A  97      16.306 -15.972  19.881  1.00  0.00           H  \\nATOM   1505 1HG  GLU A  97      17.228 -18.647  18.781  1.00  0.00           H  \\nATOM   1506 2HG  GLU A  97      18.147 -17.840  20.049  1.00  0.00           H  \\nATOM   1507  N   ALA A  98      13.565 -16.085  19.087  1.00  0.00           N  \\nATOM   1508  CA  ALA A  98      12.305 -16.380  19.720  1.00  0.00           C  \\nATOM   1509  C   ALA A  98      11.346 -17.083  18.727  1.00  0.00           C  \\nATOM   1510  O   ALA A  98      10.720 -18.063  19.120  1.00  0.00           O  \\nATOM   1511  CB  ALA A  98      11.678 -15.107  20.272  1.00  0.00           C  \\nATOM   1512  H   ALA A  98      13.891 -15.129  19.062  1.00  0.00           H  \\nATOM   1513  HA  ALA A  98      12.493 -17.067  20.545  1.00  0.00           H  \\nATOM   1514 1HB  ALA A  98      10.727 -15.346  20.747  1.00  0.00           H  \\nATOM   1515 2HB  ALA A  98      12.348 -14.660  21.007  1.00  0.00           H  \\nATOM   1516 3HB  ALA A  98      11.510 -14.403  19.459  1.00  0.00           H  \\nATOM   1517  N   ILE A  99      11.189 -16.571  17.509  1.00  0.00           N  \\nATOM   1518  CA  ILE A  99      10.234 -17.154  16.551  1.00  0.00           C  \\nATOM   1519  C   ILE A  99      10.593 -18.540  16.056  1.00  0.00           C  \\nATOM   1520  O   ILE A  99       9.724 -19.234  15.426  1.00  0.00           O  \\nATOM   1521  CB  ILE A  99       9.852 -16.249  15.391  1.00  0.00           C  \\nATOM   1522  CG1 ILE A  99      11.043 -16.010  14.480  1.00  0.00           C  \\nATOM   1523  CG2 ILE A  99       9.336 -15.010  16.032  1.00  0.00           C  \\nATOM   1524  CD1 ILE A  99      10.758 -15.313  13.113  1.00  0.00           C  \\nATOM   1525  H   ILE A  99      11.734 -15.767  17.233  1.00  0.00           H  \\nATOM   1526  HA  ILE A  99       9.310 -17.382  17.080  1.00  0.00           H  \\nATOM   1527  HB  ILE A  99       9.090 -16.737  14.784  1.00  0.00           H  \\nATOM   1528 1HG1 ILE A  99      11.778 -15.394  14.997  1.00  0.00           H  \\nATOM   1529 2HG1 ILE A  99      11.521 -16.962  14.249  1.00  0.00           H  \\nATOM   1530 1HG2 ILE A  99       9.037 -14.299  15.262  1.00  0.00           H  \\nATOM   1531 2HG2 ILE A  99       8.476 -15.254  16.654  1.00  0.00           H  \\nATOM   1532 3HG2 ILE A  99      10.117 -14.568  16.650  1.00  0.00           H  \\nATOM   1533 1HD1 ILE A  99      11.691 -15.202  12.559  1.00  0.00           H  \\nATOM   1534 2HD1 ILE A  99      10.062 -15.920  12.533  1.00  0.00           H  \\nATOM   1535 3HD1 ILE A  99      10.323 -14.330  13.290  1.00  0.00           H  \\nATOM   1536  N   THR A 100      11.795 -19.019  16.450  1.00  0.00           N  \\nATOM   1537  CA  THR A 100      12.115 -20.362  16.025  1.00  0.00           C  \\nATOM   1538  C   THR A 100      11.364 -21.365  16.807  1.00  0.00           C  \\nATOM   1539  O   THR A 100      11.316 -22.546  16.400  1.00  0.00           O  \\nATOM   1540  CB  THR A 100      13.688 -20.723  16.077  1.00  0.00           C  \\nATOM   1541  OG1 THR A 100      14.094 -20.805  17.450  1.00  0.00           O  \\nATOM   1542  CG2 THR A 100      14.471 -19.675  15.367  1.00  0.00           C  \\nATOM   1543  H   THR A 100      12.463 -18.507  17.008  1.00  0.00           H  \\nATOM   1544  HA  THR A 100      11.801 -20.481  14.988  1.00  0.00           H  \\nATOM   1545  HB  THR A 100      13.856 -21.688  15.600  1.00  0.00           H  \\nATOM   1546  HG1 THR A 100      13.340 -20.627  18.017  1.00  0.00           H  \\nATOM   1547 1HG2 THR A 100      15.531 -19.924  15.405  1.00  0.00           H  \\nATOM   1548 2HG2 THR A 100      14.148 -19.621  14.328  1.00  0.00           H  \\nATOM   1549 3HG2 THR A 100      14.308 -18.711  15.848  1.00  0.00           H  \\nATOM   1550  N   HIS A 101      10.702 -20.931  17.864  1.00  0.00           N  \\nATOM   1551  CA  HIS A 101       9.994 -21.768  18.814  1.00  0.00           C  \\nATOM   1552  C   HIS A 101       8.528 -21.911  18.435  1.00  0.00           C  \\nATOM   1553  O   HIS A 101       7.775 -22.594  19.100  1.00  0.00           O  \\nATOM   1554  CB  HIS A 101      10.160 -21.211  20.217  1.00  0.00           C  \\nATOM   1555  CG  HIS A 101      11.597 -21.239  20.702  1.00  0.00           C  \\nATOM   1556  ND1 HIS A 101      12.106 -22.277  21.461  1.00  0.00           N  \\nATOM   1557  CD2 HIS A 101      12.634 -20.398  20.464  1.00  0.00           C  \\nATOM   1558  CE1 HIS A 101      13.386 -22.046  21.711  1.00  0.00           C  \\nATOM   1559  NE2 HIS A 101      13.741 -20.932  21.099  1.00  0.00           N  \\nATOM   1560  H   HIS A 101      10.706 -19.930  17.997  1.00  0.00           H  \\nATOM   1561  HA  HIS A 101      10.409 -22.775  18.792  1.00  0.00           H  \\nATOM   1562 1HB  HIS A 101       9.805 -20.180  20.244  1.00  0.00           H  \\nATOM   1563 2HB  HIS A 101       9.548 -21.786  20.911  1.00  0.00           H  \\nATOM   1564  HD2 HIS A 101      12.606 -19.481  19.875  1.00  0.00           H  \\nATOM   1565  HE1 HIS A 101      14.039 -22.670  22.320  1.00  0.00           H  \\nATOM   1566  HE2 HIS A 101      14.668 -20.531  21.092  1.00  0.00           H  \\nATOM   1567  N   LEU A 102       8.112 -21.199  17.388  1.00  0.00           N  \\nATOM   1568  CA  LEU A 102       6.744 -21.293  16.965  1.00  0.00           C  \\nATOM   1569  C   LEU A 102       6.565 -22.289  15.798  1.00  0.00           C  \\nATOM   1570  O   LEU A 102       7.367 -22.322  14.934  1.00  0.00           O  \\nATOM   1571  CB  LEU A 102       6.306 -19.914  16.407  1.00  0.00           C  \\nATOM   1572  CG  LEU A 102       5.986 -18.929  17.567  1.00  0.00           C  \\nATOM   1573  CD1 LEU A 102       5.990 -17.535  16.910  1.00  0.00           C  \\nATOM   1574  CD2 LEU A 102       4.716 -19.306  18.208  1.00  0.00           C  \\nATOM   1575  H   LEU A 102       8.742 -20.592  16.884  1.00  0.00           H  \\nATOM   1576  HA  LEU A 102       6.132 -21.549  17.829  1.00  0.00           H  \\nATOM   1577 1HB  LEU A 102       7.109 -19.519  15.787  1.00  0.00           H  \\nATOM   1578 2HB  LEU A 102       5.427 -20.057  15.779  1.00  0.00           H  \\nATOM   1579  HG  LEU A 102       6.786 -18.965  18.306  1.00  0.00           H  \\nATOM   1580 1HD1 LEU A 102       5.772 -16.777  17.663  1.00  0.00           H  \\nATOM   1581 2HD1 LEU A 102       6.970 -17.341  16.475  1.00  0.00           H  \\nATOM   1582 3HD1 LEU A 102       5.232 -17.499  16.129  1.00  0.00           H  \\nATOM   1583 1HD2 LEU A 102       4.498 -18.612  19.020  1.00  0.00           H  \\nATOM   1584 2HD2 LEU A 102       3.912 -19.268  17.473  1.00  0.00           H  \\nATOM   1585 3HD2 LEU A 102       4.796 -20.318  18.606  1.00  0.00           H  \\nATOM   1586  N   PHE A 103       5.468 -23.020  15.798  1.00  0.00           N  \\nATOM   1587  CA  PHE A 103       5.046 -23.848  14.647  1.00  0.00           C  \\nATOM   1588  C   PHE A 103       6.133 -24.837  14.235  1.00  0.00           C  \\nATOM   1589  O   PHE A 103       6.447 -24.926  13.054  1.00  0.00           O  \\nATOM   1590  CB  PHE A 103       4.734 -22.953  13.457  1.00  0.00           C  \\nATOM   1591  CG  PHE A 103       3.674 -21.943  13.755  1.00  0.00           C  \\nATOM   1592  CD1 PHE A 103       2.514 -22.302  14.466  1.00  0.00           C  \\nATOM   1593  CD2 PHE A 103       3.837 -20.659  13.354  1.00  0.00           C  \\nATOM   1594  CE1 PHE A 103       1.518 -21.321  14.741  1.00  0.00           C  \\nATOM   1595  CE2 PHE A 103       2.824 -19.696  13.688  1.00  0.00           C  \\nATOM   1596  CZ  PHE A 103       1.726 -20.055  14.343  1.00  0.00           C  \\nATOM   1597  H   PHE A 103       4.896 -23.006  16.630  1.00  0.00           H  \\nATOM   1598  HA  PHE A 103       4.144 -24.395  14.926  1.00  0.00           H  \\nATOM   1599 1HB  PHE A 103       5.638 -22.430  13.148  1.00  0.00           H  \\nATOM   1600 2HB  PHE A 103       4.407 -23.565  12.617  1.00  0.00           H  \\nATOM   1601  HD1 PHE A 103       2.392 -23.333  14.800  1.00  0.00           H  \\nATOM   1602  HD2 PHE A 103       4.732 -20.344  12.815  1.00  0.00           H  \\nATOM   1603  HE1 PHE A 103       0.606 -21.604  15.266  1.00  0.00           H  \\nATOM   1604  HE2 PHE A 103       2.967 -18.656  13.394  1.00  0.00           H  \\nATOM   1605  HZ  PHE A 103       0.983 -19.287  14.554  1.00  0.00           H  \\nATOM   1606  N   THR A 104       6.739 -25.464  15.234  1.00  0.00           N  \\nATOM   1607  CA  THR A 104       8.003 -26.191  14.974  1.00  0.00           C  \\nATOM   1608  C   THR A 104       7.813 -27.503  14.185  1.00  0.00           C  \\nATOM   1609  O   THR A 104       8.792 -28.038  13.634  1.00  0.00           O  \\nATOM   1610  CB  THR A 104       8.846 -26.413  16.236  1.00  0.00           C  \\nATOM   1611  OG1 THR A 104       8.119 -27.187  17.201  1.00  0.00           O  \\nATOM   1612  CG2 THR A 104       9.217 -25.088  16.872  1.00  0.00           C  \\nATOM   1613  H   THR A 104       6.359 -25.458  16.170  1.00  0.00           H  \\nATOM   1614  HA  THR A 104       8.605 -25.601  14.282  1.00  0.00           H  \\nATOM   1615  HB  THR A 104       9.757 -26.950  15.975  1.00  0.00           H  \\nATOM   1616  HG1 THR A 104       7.252 -27.403  16.849  1.00  0.00           H  \\nATOM   1617 1HG2 THR A 104       9.815 -25.268  17.765  1.00  0.00           H  \\nATOM   1618 2HG2 THR A 104       9.793 -24.493  16.163  1.00  0.00           H  \\nATOM   1619 3HG2 THR A 104       8.311 -24.549  17.145  1.00  0.00           H  \\nATOM   1620  N   ASP A 105       6.582 -27.990  14.089  1.00  0.00           N  \\nATOM   1621  CA  ASP A 105       6.359 -29.242  13.339  1.00  0.00           C  \\nATOM   1622  C   ASP A 105       6.033 -28.997  11.857  1.00  0.00           C  \\nATOM   1623  O   ASP A 105       5.775 -29.942  11.103  1.00  0.00           O  \\nATOM   1624  CB  ASP A 105       5.320 -30.145  14.060  1.00  0.00           C  \\nATOM   1625  CG  ASP A 105       4.110 -29.371  14.583  1.00  0.00           C  \\nATOM   1626  OD1 ASP A 105       4.067 -28.125  14.438  1.00  0.00           O  \\nATOM   1627  OD2 ASP A 105       3.192 -30.000  15.175  1.00  0.00           O  \\nATOM   1628  H   ASP A 105       5.794 -27.525  14.516  1.00  0.00           H  \\nATOM   1629  HA  ASP A 105       7.304 -29.783  13.281  1.00  0.00           H  \\nATOM   1630 1HB  ASP A 105       4.968 -30.915  13.373  1.00  0.00           H  \\nATOM   1631 2HB  ASP A 105       5.798 -30.650  14.900  1.00  0.00           H  \\nATOM   1632  N   VAL A 106       6.056 -27.735  11.420  1.00  0.00           N  \\nATOM   1633  CA  VAL A 106       5.718 -27.344  10.074  1.00  0.00           C  \\nATOM   1634  C   VAL A 106       6.890 -27.686   9.132  1.00  0.00           C  \\nATOM   1635  O   VAL A 106       8.047 -27.549   9.528  1.00  0.00           O  \\nATOM   1636  CB  VAL A 106       5.355 -25.779  10.109  1.00  0.00           C  \\nATOM   1637  CG1 VAL A 106       6.149 -24.960   9.115  1.00  0.00           C  \\nATOM   1638  CG2 VAL A 106       3.905 -25.588   9.998  1.00  0.00           C  \\nATOM   1639  H   VAL A 106       6.329 -27.025  12.085  1.00  0.00           H  \\nATOM   1640  HA  VAL A 106       4.852 -27.924   9.752  1.00  0.00           H  \\nATOM   1641  HB  VAL A 106       5.702 -25.356  11.052  1.00  0.00           H  \\nATOM   1642 1HG1 VAL A 106       5.856 -23.912   9.190  1.00  0.00           H  \\nATOM   1643 2HG1 VAL A 106       7.213 -25.055   9.332  1.00  0.00           H  \\nATOM   1644 3HG1 VAL A 106       5.950 -25.320   8.106  1.00  0.00           H  \\nATOM   1645 1HG2 VAL A 106       3.675 -24.523  10.024  1.00  0.00           H  \\nATOM   1646 2HG2 VAL A 106       3.553 -26.013   9.058  1.00  0.00           H  \\nATOM   1647 3HG2 VAL A 106       3.408 -26.087  10.830  1.00  0.00           H  \\nATOM   1648  N   GLN A 107       6.611 -28.067   7.886  1.00  0.00           N  \\nATOM   1649  CA  GLN A 107       7.687 -28.391   6.928  1.00  0.00           C  \\nATOM   1650  C   GLN A 107       8.297 -27.269   6.101  1.00  0.00           C  \\nATOM   1651  O   GLN A 107       9.331 -27.445   5.470  1.00  0.00           O  \\nATOM   1652  CB  GLN A 107       7.178 -29.429   5.951  1.00  0.00           C  \\nATOM   1653  CG  GLN A 107       6.607 -30.626   6.638  1.00  0.00           C  \\nATOM   1654  CD  GLN A 107       5.900 -31.525   5.661  1.00  0.00           C  \\nATOM   1655  OE1 GLN A 107       4.831 -32.075   5.960  1.00  0.00           O  \\nATOM   1656  NE2 GLN A 107       6.483 -31.678   4.490  1.00  0.00           N  \\nATOM   1657  H   GLN A 107       5.649 -28.137   7.587  1.00  0.00           H  \\nATOM   1658  HA  GLN A 107       8.532 -28.799   7.482  1.00  0.00           H  \\nATOM   1659 1HB  GLN A 107       6.410 -28.986   5.317  1.00  0.00           H  \\nATOM   1660 2HB  GLN A 107       7.994 -29.749   5.303  1.00  0.00           H  \\nATOM   1661 1HG  GLN A 107       7.417 -31.183   7.109  1.00  0.00           H  \\nATOM   1662 2HG  GLN A 107       5.894 -30.293   7.393  1.00  0.00           H  \\nATOM   1663 1HE2 GLN A 107       6.064 -32.263   3.795  1.00  0.00           H  \\nATOM   1664 2HE2 GLN A 107       7.345 -31.209   4.295  1.00  0.00           H  \\nATOM   1665  N   ASN A 108       7.662 -26.108   6.037  1.00  0.00           N  \\nATOM   1666  CA  ASN A 108       8.250 -25.027   5.281  1.00  0.00           C  \\nATOM   1667  C   ASN A 108       8.095 -23.813   6.111  1.00  0.00           C  \\nATOM   1668  O   ASN A 108       6.959 -23.494   6.503  1.00  0.00           O  \\nATOM   1669  CB  ASN A 108       7.500 -24.805   3.984  1.00  0.00           C  \\nATOM   1670  CG  ASN A 108       7.203 -26.092   3.291  1.00  0.00           C  \\nATOM   1671  OD1 ASN A 108       6.180 -26.757   3.560  1.00  0.00           O  \\nATOM   1672  ND2 ASN A 108       8.119 -26.478   2.420  1.00  0.00           N  \\nATOM   1673  H   ASN A 108       6.778 -25.959   6.503  1.00  0.00           H  \\nATOM   1674  HA  ASN A 108       9.281 -25.292   5.043  1.00  0.00           H  \\nATOM   1675 1HB  ASN A 108       6.565 -24.282   4.189  1.00  0.00           H  \\nATOM   1676 2HB  ASN A 108       8.092 -24.170   3.325  1.00  0.00           H  \\nATOM   1677 1HD2 ASN A 108       7.998 -27.331   1.911  1.00  0.00           H  \\nATOM   1678 2HD2 ASN A 108       8.933 -25.918   2.270  1.00  0.00           H  \\nATOM   1679  N   ARG A 109       9.214 -23.178   6.380  1.00  0.00           N  \\nATOM   1680  CA  ARG A 109       9.257 -21.892   7.191  1.00  0.00           C  \\nATOM   1681  C   ARG A 109       9.803 -20.776   6.354  1.00  0.00           C  \\nATOM   1682  O   ARG A 109      10.767 -20.910   5.624  1.00  0.00           O  \\nATOM   1683  CB  ARG A 109      10.136 -21.993   8.449  1.00  0.00           C  \\nATOM   1684  CG  ARG A 109       9.743 -23.083   9.362  1.00  0.00           C  \\nATOM   1685  CD  ARG A 109      10.768 -23.273  10.446  1.00  0.00           C  \\nATOM   1686  NE  ARG A 109      10.689 -22.193  11.432  1.00  0.00           N  \\nATOM   1687  CZ  ARG A 109       9.821 -22.215  12.449  1.00  0.00           C  \\nATOM   1688  NH1 ARG A 109       8.940 -23.205  12.620  1.00  0.00           N  \\nATOM   1689  NH2 ARG A 109       9.819 -21.217  13.326  1.00  0.00           N  \\nATOM   1690  H   ARG A 109      10.077 -23.568   6.029  1.00  0.00           H  \\nATOM   1691  HA  ARG A 109       8.245 -21.658   7.522  1.00  0.00           H  \\nATOM   1692 1HB  ARG A 109      11.173 -22.151   8.156  1.00  0.00           H  \\nATOM   1693 2HB  ARG A 109      10.091 -21.054   9.000  1.00  0.00           H  \\nATOM   1694 1HG  ARG A 109       8.785 -22.843   9.825  1.00  0.00           H  \\nATOM   1695 2HG  ARG A 109       9.652 -24.013   8.800  1.00  0.00           H  \\nATOM   1696 1HD  ARG A 109      10.592 -24.222  10.951  1.00  0.00           H  \\nATOM   1697 2HD  ARG A 109      11.765 -23.275  10.007  1.00  0.00           H  \\nATOM   1698  HE  ARG A 109      11.321 -21.411  11.324  1.00  0.00           H  \\nATOM   1699 1HH1 ARG A 109       8.913 -23.975  11.967  1.00  0.00           H  \\nATOM   1700 2HH1 ARG A 109       8.302 -23.183  13.402  1.00  0.00           H  \\nATOM   1701 1HH2 ARG A 109      10.470 -20.450  13.221  1.00  0.00           H  \\nATOM   1702 2HH2 ARG A 109       9.168 -21.224  14.097  1.00  0.00           H  \\nATOM   1703  N   TYR A 110       9.160 -19.642   6.403  1.00  0.00           N  \\nATOM   1704  CA  TYR A 110       9.684 -18.525   5.683  1.00  0.00           C  \\nATOM   1705  C   TYR A 110       9.585 -17.284   6.532  1.00  0.00           C  \\nATOM   1706  O   TYR A 110       8.561 -17.042   7.158  1.00  0.00           O  \\nATOM   1707  CB  TYR A 110       8.851 -18.223   4.433  1.00  0.00           C  \\nATOM   1708  CG  TYR A 110       8.569 -19.394   3.513  1.00  0.00           C  \\nATOM   1709  CD1 TYR A 110       7.620 -20.355   3.830  1.00  0.00           C  \\nATOM   1710  CD2 TYR A 110       9.274 -19.514   2.334  1.00  0.00           C  \\nATOM   1711  CE1 TYR A 110       7.386 -21.442   2.958  1.00  0.00           C  \\nATOM   1712  CE2 TYR A 110       9.050 -20.585   1.463  1.00  0.00           C  \\nATOM   1713  CZ  TYR A 110       8.116 -21.528   1.787  1.00  0.00           C  \\nATOM   1714  OH  TYR A 110       7.936 -22.566   0.908  1.00  0.00           O  \\nATOM   1715  H   TYR A 110       8.307 -19.530   6.932  1.00  0.00           H  \\nATOM   1716  HA  TYR A 110      10.699 -18.764   5.365  1.00  0.00           H  \\nATOM   1717 1HB  TYR A 110       7.884 -17.815   4.730  1.00  0.00           H  \\nATOM   1718 2HB  TYR A 110       9.356 -17.466   3.834  1.00  0.00           H  \\nATOM   1719  HD1 TYR A 110       7.050 -20.274   4.756  1.00  0.00           H  \\nATOM   1720  HD2 TYR A 110      10.022 -18.767   2.068  1.00  0.00           H  \\nATOM   1721  HE1 TYR A 110       6.639 -22.194   3.212  1.00  0.00           H  \\nATOM   1722  HE2 TYR A 110       9.619 -20.662   0.536  1.00  0.00           H  \\nATOM   1723  HH  TYR A 110       8.530 -22.458   0.161  1.00  0.00           H  \\nATOM   1724  N   THR A 111      10.621 -16.460   6.426  1.00  0.00           N  \\nATOM   1725  CA  THR A 111      10.679 -15.131   7.011  1.00  0.00           C  \\nATOM   1726  C   THR A 111      10.577 -14.148   5.871  1.00  0.00           C  \\nATOM   1727  O   THR A 111      11.372 -14.194   4.943  1.00  0.00           O  \\nATOM   1728  CB  THR A 111      11.997 -14.962   7.827  1.00  0.00           C  \\nATOM   1729  OG1 THR A 111      12.034 -15.888   8.945  1.00  0.00           O  \\nATOM   1730  CG2 THR A 111      12.089 -13.508   8.293  1.00  0.00           C  \\nATOM   1731  H   THR A 111      11.412 -16.800   5.898  1.00  0.00           H  \\nATOM   1732  HA  THR A 111       9.830 -15.012   7.685  1.00  0.00           H  \\nATOM   1733  HB  THR A 111      12.850 -15.212   7.196  1.00  0.00           H  \\nATOM   1734  HG1 THR A 111      11.232 -16.416   8.949  1.00  0.00           H  \\nATOM   1735 1HG2 THR A 111      13.005 -13.367   8.866  1.00  0.00           H  \\nATOM   1736 2HG2 THR A 111      12.098 -12.848   7.426  1.00  0.00           H  \\nATOM   1737 3HG2 THR A 111      11.230 -13.272   8.919  1.00  0.00           H  \\nATOM   1738  N   PHE A 112       9.559 -13.312   5.873  1.00  0.00           N  \\nATOM   1739  CA  PHE A 112       9.412 -12.275   4.824  1.00  0.00           C  \\nATOM   1740  C   PHE A 112      10.553 -11.244   4.921  1.00  0.00           C  \\nATOM   1741  O   PHE A 112      11.180 -11.132   5.969  1.00  0.00           O  \\nATOM   1742  CB  PHE A 112       8.098 -11.486   5.043  1.00  0.00           C  \\nATOM   1743  CG  PHE A 112       6.861 -12.321   4.833  1.00  0.00           C  \\nATOM   1744  CD1 PHE A 112       6.657 -13.034   3.654  1.00  0.00           C  \\nATOM   1745  CD2 PHE A 112       5.877 -12.365   5.815  1.00  0.00           C  \\nATOM   1746  CE1 PHE A 112       5.523 -13.798   3.489  1.00  0.00           C  \\nATOM   1747  CE2 PHE A 112       4.767 -13.131   5.637  1.00  0.00           C  \\nATOM   1748  CZ  PHE A 112       4.594 -13.828   4.485  1.00  0.00           C  \\nATOM   1749  H   PHE A 112       8.863 -13.376   6.602  1.00  0.00           H  \\nATOM   1750  HA  PHE A 112       9.370 -12.769   3.852  1.00  0.00           H  \\nATOM   1751 1HB  PHE A 112       8.078 -11.088   6.057  1.00  0.00           H  \\nATOM   1752 2HB  PHE A 112       8.064 -10.640   4.358  1.00  0.00           H  \\nATOM   1753  HD1 PHE A 112       7.406 -12.981   2.863  1.00  0.00           H  \\nATOM   1754  HD2 PHE A 112       6.011 -11.807   6.742  1.00  0.00           H  \\nATOM   1755  HE1 PHE A 112       5.379 -14.368   2.571  1.00  0.00           H  \\nATOM   1756  HE2 PHE A 112       4.014 -13.184   6.423  1.00  0.00           H  \\nATOM   1757  HZ  PHE A 112       3.687 -14.419   4.368  1.00  0.00           H  \\nATOM   1758  N   ALA A 113      10.793 -10.506   3.856  1.00  0.00           N  \\nATOM   1759  CA  ALA A 113      11.804  -9.442   3.867  1.00  0.00           C  \\nATOM   1760  C   ALA A 113      11.342  -8.228   4.638  1.00  0.00           C  \\nATOM   1761  O   ALA A 113      12.160  -7.420   5.118  1.00  0.00           O  \\nATOM   1762  CB  ALA A 113      12.148  -9.070   2.408  1.00  0.00           C  \\nATOM   1763  H   ALA A 113      10.268 -10.677   3.010  1.00  0.00           H  \\nATOM   1764  HA  ALA A 113      12.693  -9.825   4.368  1.00  0.00           H  \\nATOM   1765 1HB  ALA A 113      12.899  -8.280   2.401  1.00  0.00           H  \\nATOM   1766 2HB  ALA A 113      12.539  -9.947   1.892  1.00  0.00           H  \\nATOM   1767 3HB  ALA A 113      11.250  -8.721   1.901  1.00  0.00           H  \\nATOM   1768  N   PHE A 114      10.019  -8.138   4.874  1.00  0.00           N  \\nATOM   1769  CA  PHE A 114       9.392  -6.940   5.382  1.00  0.00           C  \\nATOM   1770  C   PHE A 114       8.909  -7.304   6.759  1.00  0.00           C  \\nATOM   1771  O   PHE A 114       8.596  -8.490   7.052  1.00  0.00           O  \\nATOM   1772  CB  PHE A 114       8.233  -6.492   4.456  1.00  0.00           C  \\nATOM   1773  CG  PHE A 114       7.228  -7.579   4.128  1.00  0.00           C  \\nATOM   1774  CD1 PHE A 114       6.190  -7.910   5.035  1.00  0.00           C  \\nATOM   1775  CD2 PHE A 114       7.255  -8.223   2.886  1.00  0.00           C  \\nATOM   1776  CE1 PHE A 114       5.227  -8.933   4.707  1.00  0.00           C  \\nATOM   1777  CE2 PHE A 114       6.382  -9.185   2.604  1.00  0.00           C  \\nATOM   1778  CZ  PHE A 114       5.345  -9.559   3.474  1.00  0.00           C  \\nATOM   1779  H   PHE A 114       9.448  -8.949   4.684  1.00  0.00           H  \\nATOM   1780  HA  PHE A 114      10.138  -6.145   5.412  1.00  0.00           H  \\nATOM   1781 1HB  PHE A 114       7.692  -5.670   4.923  1.00  0.00           H  \\nATOM   1782 2HB  PHE A 114       8.641  -6.123   3.516  1.00  0.00           H  \\nATOM   1783  HD1 PHE A 114       6.126  -7.381   5.986  1.00  0.00           H  \\nATOM   1784  HD2 PHE A 114       8.041  -7.974   2.172  1.00  0.00           H  \\nATOM   1785  HE1 PHE A 114       4.442  -9.189   5.418  1.00  0.00           H  \\nATOM   1786  HE2 PHE A 114       6.499  -9.692   1.647  1.00  0.00           H  \\nATOM   1787  HZ  PHE A 114       4.638 -10.336   3.186  1.00  0.00           H  \\nATOM   1788  N   GLY A 115       8.962  -6.286   7.625  1.00  0.00           N  \\nATOM   1789  CA  GLY A 115       8.342  -6.314   8.981  1.00  0.00           C  \\nATOM   1790  C   GLY A 115       6.825  -6.192   8.893  1.00  0.00           C  \\nATOM   1791  O   GLY A 115       6.245  -6.055   7.797  1.00  0.00           O  \\nATOM   1792  H   GLY A 115       9.455  -5.457   7.327  1.00  0.00           H  \\nATOM   1793 1HA  GLY A 115       8.609  -7.244   9.484  1.00  0.00           H  \\nATOM   1794 2HA  GLY A 115       8.743  -5.498   9.581  1.00  0.00           H  \\nATOM   1795  N   GLY A 116       6.201  -6.225  10.065  1.00  0.00           N  \\nATOM   1796  CA  GLY A 116       4.725  -6.232  10.089  1.00  0.00           C  \\nATOM   1797  C   GLY A 116       4.187  -4.881  10.548  1.00  0.00           C  \\nATOM   1798  O   GLY A 116       3.038  -4.789  10.934  1.00  0.00           O  \\nATOM   1799  H   GLY A 116       6.707  -6.245  10.939  1.00  0.00           H  \\nATOM   1800 1HA  GLY A 116       4.346  -6.465   9.094  1.00  0.00           H  \\nATOM   1801 2HA  GLY A 116       4.376  -7.018  10.758  1.00  0.00           H  \\nATOM   1802  N   ASN A 117       5.022  -3.818  10.513  1.00  0.00           N  \\nATOM   1803  CA  ASN A 117       4.538  -2.477  10.864  1.00  0.00           C  \\nATOM   1804  C   ASN A 117       3.642  -1.962   9.786  1.00  0.00           C  \\nATOM   1805  O   ASN A 117       3.793  -2.299   8.580  1.00  0.00           O  \\nATOM   1806  CB  ASN A 117       5.731  -1.519  11.112  1.00  0.00           C  \\nATOM   1807  CG  ASN A 117       6.594  -1.390   9.889  1.00  0.00           C  \\nATOM   1808  OD1 ASN A 117       7.434  -2.291   9.603  1.00  0.00           O  \\nATOM   1809  ND2 ASN A 117       6.403  -0.297   9.121  1.00  0.00           N  \\nATOM   1810  H   ASN A 117       5.988  -3.938  10.244  1.00  0.00           H  \\nATOM   1811  HA  ASN A 117       3.952  -2.550  11.781  1.00  0.00           H  \\nATOM   1812 1HB  ASN A 117       5.355  -0.535  11.396  1.00  0.00           H  \\nATOM   1813 2HB  ASN A 117       6.331  -1.892  11.942  1.00  0.00           H  \\nATOM   1814 1HD2 ASN A 117       6.952  -0.164   8.295  1.00  0.00           H  \\nATOM   1815 2HD2 ASN A 117       5.713   0.379   9.377  1.00  0.00           H  \\nATOM   1816  N   TYR A 118       2.644  -1.156  10.162  1.00  0.00           N  \\nATOM   1817  CA  TYR A 118       1.697  -0.633   9.216  1.00  0.00           C  \\nATOM   1818  C   TYR A 118       2.247   0.126   8.008  1.00  0.00           C  \\nATOM   1819  O   TYR A 118       1.679  -0.027   6.908  1.00  0.00           O  \\nATOM   1820  CB  TYR A 118       0.662   0.282   9.962  1.00  0.00           C  \\nATOM   1821  CG  TYR A 118      -0.326  -0.425  10.904  1.00  0.00           C  \\nATOM   1822  CD1 TYR A 118      -1.081  -1.516  10.460  1.00  0.00           C  \\nATOM   1823  CD2 TYR A 118      -0.387  -0.088  12.253  1.00  0.00           C  \\nATOM   1824  CE1 TYR A 118      -1.928  -2.200  11.332  1.00  0.00           C  \\nATOM   1825  CE2 TYR A 118      -1.235  -0.772  13.133  1.00  0.00           C  \\nATOM   1826  CZ  TYR A 118      -1.986  -1.736  12.668  1.00  0.00           C  \\nATOM   1827  OH  TYR A 118      -2.865  -2.382  13.494  1.00  0.00           O  \\nATOM   1828  H   TYR A 118       2.553  -0.908  11.137  1.00  0.00           H  \\nATOM   1829  HA  TYR A 118       1.169  -1.469   8.757  1.00  0.00           H  \\nATOM   1830 1HB  TYR A 118       1.195   1.020  10.563  1.00  0.00           H  \\nATOM   1831 2HB  TYR A 118       0.064   0.826   9.231  1.00  0.00           H  \\nATOM   1832  HD1 TYR A 118      -1.011  -1.841   9.422  1.00  0.00           H  \\nATOM   1833  HD2 TYR A 118       0.231   0.722  12.641  1.00  0.00           H  \\nATOM   1834  HE1 TYR A 118      -2.510  -3.048  10.973  1.00  0.00           H  \\nATOM   1835  HE2 TYR A 118      -1.262  -0.497  14.188  1.00  0.00           H  \\nATOM   1836  HH  TYR A 118      -2.806  -2.009  14.377  1.00  0.00           H  \\nATOM   1837  N   ASP A 119       3.303   0.918   8.184  1.00  0.00           N  \\nATOM   1838  CA  ASP A 119       3.775   1.728   7.028  1.00  0.00           C  \\nATOM   1839  C   ASP A 119       4.208   0.806   5.896  1.00  0.00           C  \\nATOM   1840  O   ASP A 119       3.815   1.042   4.723  1.00  0.00           O  \\nATOM   1841  CB  ASP A 119       4.962   2.583   7.481  1.00  0.00           C  \\nATOM   1842  CG  ASP A 119       4.559   3.744   8.416  1.00  0.00           C  \\nATOM   1843  OD1 ASP A 119       5.499   4.483   8.802  1.00  0.00           O  \\nATOM   1844  OD2 ASP A 119       3.352   3.925   8.758  1.00  0.00           O  \\nATOM   1845  H   ASP A 119       3.791   0.988   9.066  1.00  0.00           H  \\nATOM   1846  HA  ASP A 119       2.961   2.377   6.705  1.00  0.00           H  \\nATOM   1847 1HB  ASP A 119       5.684   1.955   8.003  1.00  0.00           H  \\nATOM   1848 2HB  ASP A 119       5.463   3.002   6.608  1.00  0.00           H  \\nATOM   1849  N   ARG A 120       4.919  -0.262   6.270  1.00  0.00           N  \\nATOM   1850  CA  ARG A 120       5.404  -1.272   5.324  1.00  0.00           C  \\nATOM   1851  C   ARG A 120       4.226  -2.095   4.662  1.00  0.00           C  \\nATOM   1852  O   ARG A 120       4.109  -2.279   3.410  1.00  0.00           O  \\nATOM   1853  CB  ARG A 120       6.345  -2.207   5.984  1.00  0.00           C  \\nATOM   1854  CG  ARG A 120       7.022  -3.130   4.975  1.00  0.00           C  \\nATOM   1855  CD  ARG A 120       8.257  -2.415   4.371  1.00  0.00           C  \\nATOM   1856  NE  ARG A 120       8.633  -2.938   3.052  1.00  0.00           N  \\nATOM   1857  CZ  ARG A 120       7.998  -2.661   1.907  1.00  0.00           C  \\nATOM   1858  NH1 ARG A 120       6.924  -1.883   1.891  1.00  0.00           N  \\nATOM   1859  NH2 ARG A 120       8.423  -3.175   0.761  1.00  0.00           N  \\nATOM   1860  H   ARG A 120       5.126  -0.369   7.253  1.00  0.00           H  \\nATOM   1861  HA  ARG A 120       5.932  -0.764   4.516  1.00  0.00           H  \\nATOM   1862 1HB  ARG A 120       7.105  -1.640   6.520  1.00  0.00           H  \\nATOM   1863 2HB  ARG A 120       5.807  -2.808   6.717  1.00  0.00           H  \\nATOM   1864 1HG  ARG A 120       7.340  -4.045   5.475  1.00  0.00           H  \\nATOM   1865 2HG  ARG A 120       6.319  -3.377   4.179  1.00  0.00           H  \\nATOM   1866 1HD  ARG A 120       8.043  -1.353   4.257  1.00  0.00           H  \\nATOM   1867 2HD  ARG A 120       9.112  -2.543   5.034  1.00  0.00           H  \\nATOM   1868  HE  ARG A 120       9.433  -3.555   3.001  1.00  0.00           H  \\nATOM   1869 1HH1 ARG A 120       6.573  -1.489   2.752  1.00  0.00           H  \\nATOM   1870 2HH1 ARG A 120       6.457  -1.685   1.018  1.00  0.00           H  \\nATOM   1871 1HH2 ARG A 120       9.232  -3.781   0.748  1.00  0.00           H  \\nATOM   1872 2HH2 ARG A 120       7.939  -2.961  -0.098  1.00  0.00           H  \\nATOM   1873  N   LEU A 121       3.269  -2.539   5.476  1.00  0.00           N  \\nATOM   1874  CA  LEU A 121       2.125  -3.276   4.971  1.00  0.00           C  \\nATOM   1875  C   LEU A 121       1.250  -2.467   4.035  1.00  0.00           C  \\nATOM   1876  O   LEU A 121       0.747  -2.995   3.079  1.00  0.00           O  \\nATOM   1877  CB  LEU A 121       1.304  -3.880   6.170  1.00  0.00           C  \\nATOM   1878  CG  LEU A 121       2.053  -4.910   7.016  1.00  0.00           C  \\nATOM   1879  CD1 LEU A 121       1.064  -5.355   8.092  1.00  0.00           C  \\nATOM   1880  CD2 LEU A 121       2.581  -6.078   6.218  1.00  0.00           C  \\nATOM   1881  H   LEU A 121       3.340  -2.359   6.467  1.00  0.00           H  \\nATOM   1882  HA  LEU A 121       2.488  -4.091   4.345  1.00  0.00           H  \\nATOM   1883 1HB  LEU A 121       0.996  -3.066   6.825  1.00  0.00           H  \\nATOM   1884 2HB  LEU A 121       0.409  -4.358   5.773  1.00  0.00           H  \\nATOM   1885  HG  LEU A 121       2.907  -4.433   7.499  1.00  0.00           H  \\nATOM   1886 1HD1 LEU A 121       1.537  -6.096   8.737  1.00  0.00           H  \\nATOM   1887 2HD1 LEU A 121       0.764  -4.494   8.689  1.00  0.00           H  \\nATOM   1888 3HD1 LEU A 121       0.186  -5.793   7.620  1.00  0.00           H  \\nATOM   1889 1HD2 LEU A 121       3.102  -6.768   6.883  1.00  0.00           H  \\nATOM   1890 2HD2 LEU A 121       1.751  -6.595   5.737  1.00  0.00           H  \\nATOM   1891 3HD2 LEU A 121       3.273  -5.716   5.457  1.00  0.00           H  \\nATOM   1892  N   GLU A 122       1.006  -1.194   4.388  1.00  0.00           N  \\nATOM   1893  CA  GLU A 122       0.226  -0.292   3.579  1.00  0.00           C  \\nATOM   1894  C   GLU A 122       0.874  -0.013   2.194  1.00  0.00           C  \\nATOM   1895  O   GLU A 122       0.138  -0.033   1.209  1.00  0.00           O  \\nATOM   1896  CB  GLU A 122       0.093   1.007   4.315  1.00  0.00           C  \\nATOM   1897  CG  GLU A 122      -0.825   0.859   5.591  1.00  0.00           C  \\nATOM   1898  CD  GLU A 122      -0.946   2.192   6.294  1.00  0.00           C  \\nATOM   1899  OE1 GLU A 122      -0.002   3.038   6.248  1.00  0.00           O  \\nATOM   1900  OE2 GLU A 122      -2.004   2.446   6.973  1.00  0.00           O  \\nATOM   1901  H   GLU A 122       1.391  -0.864   5.261  1.00  0.00           H  \\nATOM   1902  HA  GLU A 122      -0.761  -0.730   3.426  1.00  0.00           H  \\nATOM   1903 1HB  GLU A 122       1.080   1.356   4.618  1.00  0.00           H  \\nATOM   1904 2HB  GLU A 122      -0.330   1.760   3.650  1.00  0.00           H  \\nATOM   1905 1HG  GLU A 122      -1.807   0.503   5.280  1.00  0.00           H  \\nATOM   1906 2HG  GLU A 122      -0.391   0.111   6.253  1.00  0.00           H  \\nATOM   1907  N   GLN A 123       2.178   0.045   2.201  1.00  0.00           N  \\nATOM   1908  CA  GLN A 123       2.940   0.223   0.901  1.00  0.00           C  \\nATOM   1909  C   GLN A 123       2.711  -1.079   0.063  1.00  0.00           C  \\nATOM   1910  O   GLN A 123       2.258  -0.992  -1.108  1.00  0.00           O  \\nATOM   1911  CB  GLN A 123       4.449   0.401   1.130  1.00  0.00           C  \\nATOM   1912  CG  GLN A 123       4.964   1.158   2.383  1.00  0.00           C  \\nATOM   1913  CD  GLN A 123       6.538   1.145   2.631  1.00  0.00           C  \\nATOM   1914  OE1 GLN A 123       7.323   0.553   1.860  1.00  0.00           O  \\nATOM   1915  NE2 GLN A 123       6.965   1.770   3.748  1.00  0.00           N  \\nATOM   1916  H   GLN A 123       2.693  -0.027   3.067  1.00  0.00           H  \\nATOM   1917  HA  GLN A 123       2.571   1.120   0.404  1.00  0.00           H  \\nATOM   1918 1HB  GLN A 123       4.927  -0.577   1.181  1.00  0.00           H  \\nATOM   1919 2HB  GLN A 123       4.885   0.934   0.285  1.00  0.00           H  \\nATOM   1920 1HG  GLN A 123       4.680   2.207   2.303  1.00  0.00           H  \\nATOM   1921 2HG  GLN A 123       4.515   0.716   3.272  1.00  0.00           H  \\nATOM   1922 1HE2 GLN A 123       7.942   1.799   3.963  1.00  0.00           H  \\nATOM   1923 2HE2 GLN A 123       6.306   2.205   4.361  1.00  0.00           H  \\nATOM   1924  N   LEU A 124       2.994  -2.262   0.672  1.00  0.00           N  \\nATOM   1925  CA  LEU A 124       2.810  -3.531  -0.042  1.00  0.00           C  \\nATOM   1926  C   LEU A 124       1.364  -3.673  -0.536  1.00  0.00           C  \\nATOM   1927  O   LEU A 124       1.085  -4.103  -1.645  1.00  0.00           O  \\nATOM   1928  CB  LEU A 124       3.080  -4.717   0.830  1.00  0.00           C  \\nATOM   1929  CG  LEU A 124       4.549  -4.898   1.151  1.00  0.00           C  \\nATOM   1930  CD1 LEU A 124       4.850  -5.959   2.188  1.00  0.00           C  \\nATOM   1931  CD2 LEU A 124       5.356  -5.154  -0.204  1.00  0.00           C  \\nATOM   1932  H   LEU A 124       3.334  -2.277   1.623  1.00  0.00           H  \\nATOM   1933  HA  LEU A 124       3.512  -3.564  -0.875  1.00  0.00           H  \\nATOM   1934 1HB  LEU A 124       2.526  -4.598   1.759  1.00  0.00           H  \\nATOM   1935 2HB  LEU A 124       2.714  -5.611   0.325  1.00  0.00           H  \\nATOM   1936  HG  LEU A 124       4.929  -3.999   1.638  1.00  0.00           H  \\nATOM   1937 1HD1 LEU A 124       5.927  -6.014   2.350  1.00  0.00           H  \\nATOM   1938 2HD1 LEU A 124       4.356  -5.703   3.125  1.00  0.00           H  \\nATOM   1939 3HD1 LEU A 124       4.486  -6.924   1.837  1.00  0.00           H  \\nATOM   1940 1HD2 LEU A 124       6.415  -5.284   0.022  1.00  0.00           H  \\nATOM   1941 2HD2 LEU A 124       4.975  -6.053  -0.690  1.00  0.00           H  \\nATOM   1942 3HD2 LEU A 124       5.229  -4.300  -0.870  1.00  0.00           H  \\nATOM   1943  N   ALA A 125       0.364  -3.324   0.281  1.00  0.00           N  \\nATOM   1944  CA  ALA A 125      -1.021  -3.496  -0.065  1.00  0.00           C  \\nATOM   1945  C   ALA A 125      -1.529  -2.534  -1.147  1.00  0.00           C  \\nATOM   1946  O   ALA A 125      -2.688  -2.678  -1.683  1.00  0.00           O  \\nATOM   1947  CB  ALA A 125      -1.858  -3.239   1.284  1.00  0.00           C  \\nATOM   1948  H   ALA A 125       0.601  -2.923   1.177  1.00  0.00           H  \\nATOM   1949  HA  ALA A 125      -1.158  -4.519  -0.417  1.00  0.00           H  \\nATOM   1950 1HB  ALA A 125      -2.922  -3.357   1.080  1.00  0.00           H  \\nATOM   1951 2HB  ALA A 125      -1.553  -3.958   2.044  1.00  0.00           H  \\nATOM   1952 3HB  ALA A 125      -1.668  -2.229   1.643  1.00  0.00           H  \\nATOM   1953  N   GLY A 126      -0.774  -1.452  -1.263  1.00  0.00           N  \\nATOM   1954  CA  GLY A 126      -1.253  -0.272  -2.059  1.00  0.00           C  \\nATOM   1955  C   GLY A 126      -2.530   0.437  -1.563  1.00  0.00           C  \\nATOM   1956  O   GLY A 126      -3.275   1.046  -2.321  1.00  0.00           O  \\nATOM   1957  H   GLY A 126       0.131  -1.410  -0.817  1.00  0.00           H  \\nATOM   1958 1HA  GLY A 126      -0.470   0.486  -2.094  1.00  0.00           H  \\nATOM   1959 2HA  GLY A 126      -1.447  -0.581  -3.086  1.00  0.00           H  \\nATOM   1960  N   ASN A 127      -2.797   0.353  -0.250  1.00  0.00           N  \\nATOM   1961  CA  ASN A 127      -3.940   0.994   0.352  1.00  0.00           C  \\nATOM   1962  C   ASN A 127      -3.453   1.350   1.780  1.00  0.00           C  \\nATOM   1963  O   ASN A 127      -2.766   0.537   2.476  1.00  0.00           O  \\nATOM   1964  CB  ASN A 127      -5.166   0.046   0.469  1.00  0.00           C  \\nATOM   1965  CG  ASN A 127      -5.845  -0.242  -0.888  1.00  0.00           C  \\nATOM   1966  OD1 ASN A 127      -6.649   0.547  -1.368  1.00  0.00           O  \\nATOM   1967  ND2 ASN A 127      -5.581  -1.447  -1.440  1.00  0.00           N  \\nATOM   1968  H   ASN A 127      -2.169  -0.181   0.333  1.00  0.00           H  \\nATOM   1969  HA  ASN A 127      -4.234   1.837  -0.275  1.00  0.00           H  \\nATOM   1970 1HB  ASN A 127      -4.852  -0.903   0.907  1.00  0.00           H  \\nATOM   1971 2HB  ASN A 127      -5.905   0.487   1.139  1.00  0.00           H  \\nATOM   1972 1HD2 ASN A 127      -5.989  -1.695  -2.319  1.00  0.00           H  \\nATOM   1973 2HD2 ASN A 127      -4.977  -2.090  -0.970  1.00  0.00           H  \\nATOM   1974  N   LEU A 128      -3.590   2.330   2.304  1.00  0.00           N  \\nATOM   1975  CA  LEU A 128      -3.626   2.785   3.696  1.00  0.00           C  \\nATOM   1976  C   LEU A 128      -4.771   2.170   4.471  1.00  0.00           C  \\nATOM   1977  O   LEU A 128      -5.795   1.817   3.940  1.00  0.00           O  \\nATOM   1978  CB  LEU A 128      -3.825   4.298   3.731  1.00  0.00           C  \\nATOM   1979  CG  LEU A 128      -2.692   5.123   3.054  1.00  0.00           C  \\nATOM   1980  CD1 LEU A 128      -2.984   6.603   3.070  1.00  0.00           C  \\nATOM   1981  CD2 LEU A 128      -1.434   4.938   3.597  1.00  0.00           C  \\nATOM   1982  H   LEU A 128      -3.963   2.929   1.581  1.00  0.00           H  \\nATOM   1983  HA  LEU A 128      -2.674   2.538   4.165  1.00  0.00           H  \\nATOM   1984 1HB  LEU A 128      -4.763   4.536   3.231  1.00  0.00           H  \\nATOM   1985 2HB  LEU A 128      -3.901   4.615   4.771  1.00  0.00           H  \\nATOM   1986  HG  LEU A 128      -2.621   4.850   2.001  1.00  0.00           H  \\nATOM   1987 1HD1 LEU A 128      -2.167   7.140   2.588  1.00  0.00           H  \\nATOM   1988 2HD1 LEU A 128      -3.912   6.796   2.532  1.00  0.00           H  \\nATOM   1989 3HD1 LEU A 128      -3.083   6.942   4.100  1.00  0.00           H  \\nATOM   1990 1HD2 LEU A 128      -0.711   5.554   3.060  1.00  0.00           H  \\nATOM   1991 2HD2 LEU A 128      -1.445   5.226   4.648  1.00  0.00           H  \\nATOM   1992 3HD2 LEU A 128      -1.151   3.889   3.511  1.00  0.00           H  \\nATOM   1993  N   ARG A 129      -4.601   2.075   5.787  1.00  0.00           N  \\nATOM   1994  CA  ARG A 129      -5.672   1.619   6.666  1.00  0.00           C  \\nATOM   1995  C   ARG A 129      -6.995   2.336   6.373  1.00  0.00           C  \\nATOM   1996  O   ARG A 129      -8.082   1.739   6.357  1.00  0.00           O  \\nATOM   1997  CB  ARG A 129      -5.239   1.836   8.138  1.00  0.00           C  \\nATOM   1998  CG  ARG A 129      -4.302   0.750   8.638  1.00  0.00           C  \\nATOM   1999  CD  ARG A 129      -3.749   1.132  10.025  1.00  0.00           C  \\nATOM   2000  NE  ARG A 129      -2.642   2.092   9.834  1.00  0.00           N  \\nATOM   2001  CZ  ARG A 129      -2.062   2.729  10.829  1.00  0.00           C  \\nATOM   2002  NH1 ARG A 129      -2.537   2.611  12.082  1.00  0.00           N  \\nATOM   2003  NH2 ARG A 129      -1.024   3.532  10.628  1.00  0.00           N  \\nATOM   2004  H   ARG A 129      -3.708   2.324   6.188  1.00  0.00           H  \\nATOM   2005  HA  ARG A 129      -5.835   0.555   6.490  1.00  0.00           H  \\nATOM   2006 1HB  ARG A 129      -4.742   2.800   8.232  1.00  0.00           H  \\nATOM   2007 2HB  ARG A 129      -6.122   1.859   8.777  1.00  0.00           H  \\nATOM   2008 1HG  ARG A 129      -4.844  -0.192   8.714  1.00  0.00           H  \\nATOM   2009 2HG  ARG A 129      -3.472   0.636   7.939  1.00  0.00           H  \\nATOM   2010 1HD  ARG A 129      -4.540   1.587  10.620  1.00  0.00           H  \\nATOM   2011 2HD  ARG A 129      -3.385   0.238  10.530  1.00  0.00           H  \\nATOM   2012  HE  ARG A 129      -2.323   2.260   8.889  1.00  0.00           H  \\nATOM   2013 1HH1 ARG A 129      -3.344   2.030  12.263  1.00  0.00           H  \\nATOM   2014 2HH1 ARG A 129      -2.088   3.102  12.841  1.00  0.00           H  \\nATOM   2015 1HH2 ARG A 129      -0.661   3.665   9.694  1.00  0.00           H  \\nATOM   2016 2HH2 ARG A 129      -0.598   4.009  11.408  1.00  0.00           H  \\nATOM   2017  N   GLU A 130      -6.946   3.640   6.100  1.00  0.00           N  \\nATOM   2018  CA  GLU A 130      -8.233   4.324   5.872  1.00  0.00           C  \\nATOM   2019  C   GLU A 130      -8.958   3.837   4.604  1.00  0.00           C  \\nATOM   2020  O   GLU A 130     -10.182   4.098   4.444  1.00  0.00           O  \\nATOM   2021  CB  GLU A 130      -8.010   5.835   5.781  1.00  0.00           C  \\nATOM   2022  CG  GLU A 130      -7.284   6.289   4.523  1.00  0.00           C  \\nATOM   2023  CD  GLU A 130      -7.049   7.774   4.486  1.00  0.00           C  \\nATOM   2024  OE1 GLU A 130      -7.395   8.434   5.436  1.00  0.00           O  \\nATOM   2025  OE2 GLU A 130      -6.524   8.248   3.507  1.00  0.00           O  \\nATOM   2026  H   GLU A 130      -6.089   4.171   6.042  1.00  0.00           H  \\nATOM   2027  HA  GLU A 130      -8.884   4.129   6.725  1.00  0.00           H  \\nATOM   2028 1HB  GLU A 130      -8.972   6.348   5.816  1.00  0.00           H  \\nATOM   2029 2HB  GLU A 130      -7.430   6.170   6.642  1.00  0.00           H  \\nATOM   2030 1HG  GLU A 130      -6.322   5.780   4.468  1.00  0.00           H  \\nATOM   2031 2HG  GLU A 130      -7.869   5.996   3.653  1.00  0.00           H  \\nATOM   2032  N   ASN A 131      -8.240   3.156   3.710  1.00  0.00           N  \\nATOM   2033  CA  ASN A 131      -8.848   2.638   2.506  1.00  0.00           C  \\nATOM   2034  C   ASN A 131      -9.247   1.152   2.570  1.00  0.00           C  \\nATOM   2035  O   ASN A 131      -9.728   0.608   1.602  1.00  0.00           O  \\nATOM   2036  CB  ASN A 131      -7.912   2.879   1.336  1.00  0.00           C  \\nATOM   2037  CG  ASN A 131      -7.680   4.341   1.073  1.00  0.00           C  \\nATOM   2038  OD1 ASN A 131      -8.463   5.195   1.507  1.00  0.00           O  \\nATOM   2039  ND2 ASN A 131      -6.620   4.647   0.370  1.00  0.00           N  \\nATOM   2040  H   ASN A 131      -7.255   2.997   3.871  1.00  0.00           H  \\nATOM   2041  HA  ASN A 131      -9.777   3.182   2.327  1.00  0.00           H  \\nATOM   2042 1HB  ASN A 131      -6.952   2.400   1.533  1.00  0.00           H  \\nATOM   2043 2HB  ASN A 131      -8.328   2.422   0.438  1.00  0.00           H  \\nATOM   2044 1HD2 ASN A 131      -6.415   5.604   0.164  1.00  0.00           H  \\nATOM   2045 2HD2 ASN A 131      -6.014   3.924   0.039  1.00  0.00           H  \\nATOM   2046  N   ILE A 132      -9.029   0.505   3.718  1.00  0.00           N  \\nATOM   2047  CA  ILE A 132      -9.258  -0.944   3.856  1.00  0.00           C  \\nATOM   2048  C   ILE A 132     -10.412  -1.137   4.829  1.00  0.00           C  \\nATOM   2049  O   ILE A 132     -10.348  -0.765   6.010  1.00  0.00           O  \\nATOM   2050  CB  ILE A 132      -8.007  -1.684   4.365  1.00  0.00           C  \\nATOM   2051  CG1 ILE A 132      -6.853  -1.528   3.370  1.00  0.00           C  \\nATOM   2052  CG2 ILE A 132      -8.316  -3.154   4.600  1.00  0.00           C  \\nATOM   2053  CD1 ILE A 132      -5.520  -1.999   3.903  1.00  0.00           C  \\nATOM   2054  H   ILE A 132      -8.697   1.029   4.514  1.00  0.00           H  \\nATOM   2055  HA  ILE A 132      -9.548  -1.343   2.885  1.00  0.00           H  \\nATOM   2056  HB  ILE A 132      -7.677  -1.237   5.303  1.00  0.00           H  \\nATOM   2057 1HG1 ILE A 132      -7.077  -2.089   2.464  1.00  0.00           H  \\nATOM   2058 2HG1 ILE A 132      -6.755  -0.479   3.088  1.00  0.00           H  \\nATOM   2059 1HG2 ILE A 132      -7.421  -3.661   4.958  1.00  0.00           H  \\nATOM   2060 2HG2 ILE A 132      -9.107  -3.245   5.343  1.00  0.00           H  \\nATOM   2061 3HG2 ILE A 132      -8.643  -3.611   3.665  1.00  0.00           H  \\nATOM   2062 1HD1 ILE A 132      -4.753  -1.856   3.142  1.00  0.00           H  \\nATOM   2063 2HD1 ILE A 132      -5.259  -1.424   4.793  1.00  0.00           H  \\nATOM   2064 3HD1 ILE A 132      -5.583  -3.056   4.159  1.00  0.00           H  \\nATOM   2065  N   GLU A 133     -11.458  -1.802   4.340  1.00  0.00           N  \\nATOM   2066  CA  GLU A 133     -12.687  -2.029   5.163  1.00  0.00           C  \\nATOM   2067  C   GLU A 133     -12.447  -3.199   6.135  1.00  0.00           C  \\nATOM   2068  O   GLU A 133     -11.802  -4.203   5.809  1.00  0.00           O  \\nATOM   2069  CB  GLU A 133     -13.893  -2.306   4.289  1.00  0.00           C  \\nATOM   2070  CG  GLU A 133     -14.209  -1.154   3.340  1.00  0.00           C  \\nATOM   2071  CD  GLU A 133     -15.410  -1.391   2.420  1.00  0.00           C  \\nATOM   2072  OE1 GLU A 133     -15.799  -2.545   2.238  1.00  0.00           O  \\nATOM   2073  OE2 GLU A 133     -15.881  -0.382   1.781  1.00  0.00           O  \\nATOM   2074  H   GLU A 133     -11.424  -2.162   3.397  1.00  0.00           H  \\nATOM   2075  HA  GLU A 133     -12.887  -1.128   5.743  1.00  0.00           H  \\nATOM   2076 1HB  GLU A 133     -13.716  -3.206   3.700  1.00  0.00           H  \\nATOM   2077 2HB  GLU A 133     -14.763  -2.491   4.918  1.00  0.00           H  \\nATOM   2078 1HG  GLU A 133     -14.410  -0.258   3.928  1.00  0.00           H  \\nATOM   2079 2HG  GLU A 133     -13.336  -0.958   2.719  1.00  0.00           H  \\nATOM   2080  N   LEU A 134     -12.986  -3.023   7.296  1.00  0.00           N  \\nATOM   2081  CA  LEU A 134     -12.976  -4.022   8.416  1.00  0.00           C  \\nATOM   2082  C   LEU A 134     -14.379  -4.512   8.686  1.00  0.00           C  \\nATOM   2083  O   LEU A 134     -15.396  -3.811   8.462  1.00  0.00           O  \\nATOM   2084  CB  LEU A 134     -12.393  -3.388   9.674  1.00  0.00           C  \\nATOM   2085  CG  LEU A 134     -10.971  -2.777   9.528  1.00  0.00           C  \\nATOM   2086  CD1 LEU A 134     -10.491  -2.036  10.850  1.00  0.00           C  \\nATOM   2087  CD2 LEU A 134      -9.872  -3.768   9.071  1.00  0.00           C  \\nATOM   2088  H   LEU A 134     -13.444  -2.133   7.434  1.00  0.00           H  \\nATOM   2089  HA  LEU A 134     -12.350  -4.864   8.123  1.00  0.00           H  \\nATOM   2090 1HB  LEU A 134     -13.061  -2.594  10.004  1.00  0.00           H  \\nATOM   2091 2HB  LEU A 134     -12.348  -4.146  10.456  1.00  0.00           H  \\nATOM   2092  HG  LEU A 134     -10.993  -1.976   8.789  1.00  0.00           H  \\nATOM   2093 1HD1 LEU A 134      -9.492  -1.627  10.696  1.00  0.00           H  \\nATOM   2094 2HD1 LEU A 134     -11.182  -1.226  11.084  1.00  0.00           H  \\nATOM   2095 3HD1 LEU A 134     -10.468  -2.745  11.677  1.00  0.00           H  \\nATOM   2096 1HD2 LEU A 134      -8.918  -3.245   8.999  1.00  0.00           H  \\nATOM   2097 2HD2 LEU A 134      -9.788  -4.578   9.796  1.00  0.00           H  \\nATOM   2098 3HD2 LEU A 134     -10.135  -4.179   8.096  1.00  0.00           H  \\nATOM   2099  N   GLY A 135     -14.523  -5.632   9.338  1.00  0.00           N  \\nATOM   2100  CA  GLY A 135     -15.773  -6.277   9.646  1.00  0.00           C  \\nATOM   2101  C   GLY A 135     -15.720  -7.798   9.408  1.00  0.00           C  \\nATOM   2102  O   GLY A 135     -14.656  -8.323   9.075  1.00  0.00           O  \\nATOM   2103  H   GLY A 135     -13.660  -6.061   9.641  1.00  0.00           H  \\nATOM   2104 1HA  GLY A 135     -16.034  -6.087  10.687  1.00  0.00           H  \\nATOM   2105 2HA  GLY A 135     -16.565  -5.847   9.033  1.00  0.00           H  \\nATOM   2106  N   ASN A 136     -16.818  -8.479   9.618  1.00  0.00           N  \\nATOM   2107  CA  ASN A 136     -16.776  -9.923   9.491  1.00  0.00           C  \\nATOM   2108  C   ASN A 136     -16.567 -10.336   8.042  1.00  0.00           C  \\nATOM   2109  O   ASN A 136     -15.886 -11.380   7.796  1.00  0.00           O  \\nATOM   2110  CB  ASN A 136     -18.037 -10.547  10.019  1.00  0.00           C  \\nATOM   2111  CG  ASN A 136     -17.833 -11.974  10.457  1.00  0.00           C  \\nATOM   2112  OD1 ASN A 136     -16.878 -12.295  11.195  1.00  0.00           O  \\nATOM   2113  ND2 ASN A 136     -18.758 -12.855  10.057  1.00  0.00           N  \\nATOM   2114  H   ASN A 136     -17.689  -8.032   9.865  1.00  0.00           H  \\nATOM   2115  HA  ASN A 136     -15.935 -10.298  10.077  1.00  0.00           H  \\nATOM   2116 1HB  ASN A 136     -18.403  -9.966  10.867  1.00  0.00           H  \\nATOM   2117 2HB  ASN A 136     -18.807 -10.521   9.247  1.00  0.00           H  \\nATOM   2118 1HD2 ASN A 136     -18.677 -13.818  10.316  1.00  0.00           H  \\nATOM   2119 2HD2 ASN A 136     -19.530 -12.551   9.499  1.00  0.00           H  \\nATOM   2120  N   GLY A 137     -17.035  -9.580   7.070  1.00  0.00           N  \\nATOM   2121  CA  GLY A 137     -16.793  -9.926   5.661  1.00  0.00           C  \\nATOM   2122  C   GLY A 137     -15.317  -9.813   5.344  1.00  0.00           C  \\nATOM   2123  O   GLY A 137     -14.694 -10.752   4.814  1.00  0.00           O  \\nATOM   2124  H   GLY A 137     -17.568  -8.750   7.288  1.00  0.00           H  \\nATOM   2125 1HA  GLY A 137     -17.144 -10.940   5.470  1.00  0.00           H  \\nATOM   2126 2HA  GLY A 137     -17.369  -9.261   5.019  1.00  0.00           H  \\nATOM   2127  N   PRO A 138     -14.697  -8.670   5.649  1.00  0.00           N  \\nATOM   2128  CA  PRO A 138     -13.237  -8.535   5.459  1.00  0.00           C  \\nATOM   2129  C   PRO A 138     -12.461  -9.594   6.148  1.00  0.00           C  \\nATOM   2130  O   PRO A 138     -11.454 -10.086   5.637  1.00  0.00           O  \\nATOM   2131  CB  PRO A 138     -12.935  -7.111   6.050  1.00  0.00           C  \\nATOM   2132  CG  PRO A 138     -14.169  -6.371   5.660  1.00  0.00           C  \\nATOM   2133  CD  PRO A 138     -15.350  -7.373   5.928  1.00  0.00           C  \\nATOM   2134  HA  PRO A 138     -13.007  -8.572   4.384  1.00  0.00           H  \\nATOM   2135 1HB  PRO A 138     -12.773  -7.181   7.136  1.00  0.00           H  \\nATOM   2136 2HB  PRO A 138     -12.009  -6.710   5.612  1.00  0.00           H  \\nATOM   2137 1HG  PRO A 138     -14.262  -5.449   6.252  1.00  0.00           H  \\nATOM   2138 2HG  PRO A 138     -14.112  -6.066   4.605  1.00  0.00           H  \\nATOM   2139 1HD  PRO A 138     -15.678  -7.283   6.974  1.00  0.00           H  \\nATOM   2140 2HD  PRO A 138     -16.180  -7.154   5.241  1.00  0.00           H  \\nATOM   2141  N   LEU A 139     -12.866  -9.966   7.374  1.00  0.00           N  \\nATOM   2142  CA  LEU A 139     -12.059 -10.970   8.116  1.00  0.00           C  \\nATOM   2143  C   LEU A 139     -12.200 -12.332   7.435  1.00  0.00           C  \\nATOM   2144  O   LEU A 139     -11.241 -13.055   7.363  1.00  0.00           O  \\nATOM   2145  CB  LEU A 139     -12.489 -10.994   9.600  1.00  0.00           C  \\nATOM   2146  CG  LEU A 139     -11.640 -11.916  10.525  1.00  0.00           C  \\nATOM   2147  CD1 LEU A 139     -10.194 -11.559  10.549  1.00  0.00           C  \\nATOM   2148  CD2 LEU A 139     -12.223 -11.709  11.978  1.00  0.00           C  \\nATOM   2149  H   LEU A 139     -13.697  -9.587   7.805  1.00  0.00           H  \\nATOM   2150  HA  LEU A 139     -11.010 -10.683   8.056  1.00  0.00           H  \\nATOM   2151 1HB  LEU A 139     -12.432  -9.981   9.994  1.00  0.00           H  \\nATOM   2152 2HB  LEU A 139     -13.526 -11.326   9.656  1.00  0.00           H  \\nATOM   2153  HG  LEU A 139     -11.742 -12.952  10.199  1.00  0.00           H  \\nATOM   2154 1HD1 LEU A 139      -9.662 -12.241  11.212  1.00  0.00           H  \\nATOM   2155 2HD1 LEU A 139      -9.783 -11.638   9.543  1.00  0.00           H  \\nATOM   2156 3HD1 LEU A 139     -10.078 -10.538  10.910  1.00  0.00           H  \\nATOM   2157 1HD2 LEU A 139     -11.671 -12.331  12.684  1.00  0.00           H  \\nATOM   2158 2HD2 LEU A 139     -12.125 -10.661  12.264  1.00  0.00           H  \\nATOM   2159 3HD2 LEU A 139     -13.276 -11.992  11.991  1.00  0.00           H  \\nATOM   2160  N   GLU A 140     -13.403 -12.665   6.986  1.00  0.00           N  \\nATOM   2161  CA  GLU A 140     -13.578 -13.907   6.250  1.00  0.00           C  \\nATOM   2162  C   GLU A 140     -12.678 -13.921   5.053  1.00  0.00           C  \\nATOM   2163  O   GLU A 140     -12.008 -14.916   4.802  1.00  0.00           O  \\nATOM   2164  CB  GLU A 140     -15.067 -14.003   5.831  1.00  0.00           C  \\nATOM   2165  CG  GLU A 140     -15.337 -15.054   4.788  1.00  0.00           C  \\nATOM   2166  CD  GLU A 140     -15.546 -16.428   5.317  1.00  0.00           C  \\nATOM   2167  OE1 GLU A 140     -16.076 -16.514   6.454  1.00  0.00           O  \\nATOM   2168  OE2 GLU A 140     -15.222 -17.430   4.566  1.00  0.00           O  \\nATOM   2169  H   GLU A 140     -14.204 -12.071   7.146  1.00  0.00           H  \\nATOM   2170  HA  GLU A 140     -13.323 -14.738   6.908  1.00  0.00           H  \\nATOM   2171 1HB  GLU A 140     -15.678 -14.227   6.706  1.00  0.00           H  \\nATOM   2172 2HB  GLU A 140     -15.397 -13.041   5.439  1.00  0.00           H  \\nATOM   2173 1HG  GLU A 140     -16.230 -14.775   4.229  1.00  0.00           H  \\nATOM   2174 2HG  GLU A 140     -14.500 -15.083   4.092  1.00  0.00           H  \\nATOM   2175  N   GLU A 141     -12.642 -12.827   4.289  1.00  0.00           N  \\nATOM   2176  CA  GLU A 141     -11.768 -12.711   3.111  1.00  0.00           C  \\nATOM   2177  C   GLU A 141     -10.321 -12.849   3.492  1.00  0.00           C  \\nATOM   2178  O   GLU A 141      -9.523 -13.499   2.745  1.00  0.00           O  \\nATOM   2179  CB  GLU A 141     -12.025 -11.371   2.394  1.00  0.00           C  \\nATOM   2180  CG  GLU A 141     -13.423 -11.338   1.723  1.00  0.00           C  \\nATOM   2181  CD  GLU A 141     -13.857  -9.947   1.345  1.00  0.00           C  \\nATOM   2182  OE1 GLU A 141     -13.120  -8.975   1.582  1.00  0.00           O  \\nATOM   2183  OE2 GLU A 141     -14.987  -9.799   0.836  1.00  0.00           O  \\nATOM   2184  H   GLU A 141     -13.241 -12.052   4.535  1.00  0.00           H  \\nATOM   2185  HA  GLU A 141     -12.000 -13.528   2.426  1.00  0.00           H  \\nATOM   2186 1HB  GLU A 141     -11.951 -10.554   3.112  1.00  0.00           H  \\nATOM   2187 2HB  GLU A 141     -11.258 -11.211   1.636  1.00  0.00           H  \\nATOM   2188 1HG  GLU A 141     -13.398 -11.956   0.826  1.00  0.00           H  \\nATOM   2189 2HG  GLU A 141     -14.153 -11.769   2.407  1.00  0.00           H  \\nATOM   2190  N   ALA A 142      -9.908 -12.200   4.582  1.00  0.00           N  \\nATOM   2191  CA  ALA A 142      -8.525 -12.144   4.977  1.00  0.00           C  \\nATOM   2192  C   ALA A 142      -8.042 -13.537   5.388  1.00  0.00           C  \\nATOM   2193  O   ALA A 142      -6.896 -13.888   5.102  1.00  0.00           O  \\nATOM   2194  CB  ALA A 142      -8.350 -11.151   6.131  1.00  0.00           C  \\nATOM   2195  H   ALA A 142     -10.601 -11.731   5.148  1.00  0.00           H  \\nATOM   2196  HA  ALA A 142      -7.942 -11.803   4.122  1.00  0.00           H  \\nATOM   2197 1HB  ALA A 142      -7.301 -11.115   6.424  1.00  0.00           H  \\nATOM   2198 2HB  ALA A 142      -8.672 -10.160   5.810  1.00  0.00           H  \\nATOM   2199 3HB  ALA A 142      -8.952 -11.471   6.980  1.00  0.00           H  \\nATOM   2200  N   ILE A 143      -8.866 -14.310   6.036  1.00  0.00           N  \\nATOM   2201  CA  ILE A 143      -8.465 -15.674   6.451  1.00  0.00           C  \\nATOM   2202  C   ILE A 143      -8.269 -16.529   5.201  1.00  0.00           C  \\nATOM   2203  O   ILE A 143      -7.302 -17.278   5.139  1.00  0.00           O  \\nATOM   2204  CB  ILE A 143      -9.527 -16.282   7.411  1.00  0.00           C  \\nATOM   2205  CG1 ILE A 143      -9.513 -15.547   8.723  1.00  0.00           C  \\nATOM   2206  CG2 ILE A 143      -9.337 -17.803   7.575  1.00  0.00           C  \\nATOM   2207  CD1 ILE A 143     -10.842 -15.831   9.491  1.00  0.00           C  \\nATOM   2208  H   ILE A 143      -9.793 -13.980   6.262  1.00  0.00           H  \\nATOM   2209  HA  ILE A 143      -7.514 -15.609   6.978  1.00  0.00           H  \\nATOM   2210  HB  ILE A 143     -10.524 -16.101   7.012  1.00  0.00           H  \\nATOM   2211 1HG1 ILE A 143      -8.655 -15.870   9.311  1.00  0.00           H  \\nATOM   2212 2HG1 ILE A 143      -9.400 -14.478   8.540  1.00  0.00           H  \\nATOM   2213 1HG2 ILE A 143     -10.096 -18.193   8.253  1.00  0.00           H  \\nATOM   2214 2HG2 ILE A 143      -9.433 -18.288   6.605  1.00  0.00           H  \\nATOM   2215 3HG2 ILE A 143      -8.347 -18.004   7.985  1.00  0.00           H  \\nATOM   2216 1HD1 ILE A 143     -10.834 -15.298  10.442  1.00  0.00           H  \\nATOM   2217 2HD1 ILE A 143     -11.688 -15.491   8.892  1.00  0.00           H  \\nATOM   2218 3HD1 ILE A 143     -10.934 -16.900   9.676  1.00  0.00           H  \\nATOM   2219  N   SER A 144      -9.156 -16.384   4.232  1.00  0.00           N  \\nATOM   2220  CA  SER A 144      -8.895 -17.119   2.958  1.00  0.00           C  \\nATOM   2221  C   SER A 144      -7.627 -16.667   2.305  1.00  0.00           C  \\nATOM   2222  O   SER A 144      -6.852 -17.543   1.804  1.00  0.00           O  \\nATOM   2223  CB  SER A 144     -10.035 -17.023   1.987  1.00  0.00           C  \\nATOM   2224  OG  SER A 144     -11.143 -17.701   2.535  1.00  0.00           O  \\nATOM   2225  H   SER A 144      -9.985 -15.812   4.309  1.00  0.00           H  \\nATOM   2226  HA  SER A 144      -8.750 -18.175   3.191  1.00  0.00           H  \\nATOM   2227 1HB  SER A 144     -10.270 -15.975   1.804  1.00  0.00           H  \\nATOM   2228 2HB  SER A 144      -9.740 -17.463   1.035  1.00  0.00           H  \\nATOM   2229  HG  SER A 144     -10.853 -18.038   3.386  1.00  0.00           H  \\nATOM   2230  N   ALA A 145      -7.392 -15.365   2.274  1.00  0.00           N  \\nATOM   2231  CA  ALA A 145      -6.164 -14.886   1.638  1.00  0.00           C  \\nATOM   2232  C   ALA A 145      -4.915 -15.406   2.278  1.00  0.00           C  \\nATOM   2233  O   ALA A 145      -3.926 -15.820   1.580  1.00  0.00           O  \\nATOM   2234  CB  ALA A 145      -6.119 -13.387   1.601  1.00  0.00           C  \\nATOM   2235  H   ALA A 145      -8.032 -14.695   2.675  1.00  0.00           H  \\nATOM   2236  HA  ALA A 145      -6.146 -15.257   0.613  1.00  0.00           H  \\nATOM   2237 1HB  ALA A 145      -5.195 -13.062   1.123  1.00  0.00           H  \\nATOM   2238 2HB  ALA A 145      -6.972 -13.011   1.035  1.00  0.00           H  \\nATOM   2239 3HB  ALA A 145      -6.158 -12.997   2.617  1.00  0.00           H  \\nATOM   2240  N   LEU A 146      -4.854 -15.395   3.620  1.00  0.00           N  \\nATOM   2241  CA  LEU A 146      -3.625 -15.895   4.251  1.00  0.00           C  \\nATOM   2242  C   LEU A 146      -3.497 -17.389   3.998  1.00  0.00           C  \\nATOM   2243  O   LEU A 146      -2.395 -17.859   3.793  1.00  0.00           O  \\nATOM   2244  CB  LEU A 146      -3.696 -15.692   5.780  1.00  0.00           C  \\nATOM   2245  CG  LEU A 146      -3.093 -14.380   6.326  1.00  0.00           C  \\nATOM   2246  CD1 LEU A 146      -3.672 -13.206   5.783  1.00  0.00           C  \\nATOM   2247  CD2 LEU A 146      -3.158 -14.362   7.876  1.00  0.00           C  \\nATOM   2248  H   LEU A 146      -5.607 -15.062   4.205  1.00  0.00           H  \\nATOM   2249  HA  LEU A 146      -2.780 -15.329   3.859  1.00  0.00           H  \\nATOM   2250 1HB  LEU A 146      -4.740 -15.721   6.086  1.00  0.00           H  \\nATOM   2251 2HB  LEU A 146      -3.173 -16.516   6.265  1.00  0.00           H  \\nATOM   2252  HG  LEU A 146      -2.052 -14.300   6.013  1.00  0.00           H  \\nATOM   2253 1HD1 LEU A 146      -3.193 -12.329   6.218  1.00  0.00           H  \\nATOM   2254 2HD1 LEU A 146      -3.531 -13.197   4.702  1.00  0.00           H  \\nATOM   2255 3HD1 LEU A 146      -4.737 -13.187   6.011  1.00  0.00           H  \\nATOM   2256 1HD2 LEU A 146      -2.729 -13.431   8.248  1.00  0.00           H  \\nATOM   2257 2HD2 LEU A 146      -4.197 -14.436   8.198  1.00  0.00           H  \\nATOM   2258 3HD2 LEU A 146      -2.594 -15.206   8.274  1.00  0.00           H  \\nATOM   2259  N   TYR A 147      -4.619 -18.118   3.992  1.00  0.00           N  \\nATOM   2260  CA  TYR A 147      -4.530 -19.566   3.664  1.00  0.00           C  \\nATOM   2261  C   TYR A 147      -4.023 -19.782   2.270  1.00  0.00           C  \\nATOM   2262  O   TYR A 147      -3.081 -20.554   2.103  1.00  0.00           O  \\nATOM   2263  CB  TYR A 147      -5.879 -20.288   3.893  1.00  0.00           C  \\nATOM   2264  CG  TYR A 147      -5.733 -21.740   3.514  1.00  0.00           C  \\nATOM   2265  CD1 TYR A 147      -5.988 -22.160   2.241  1.00  0.00           C  \\nATOM   2266  CD2 TYR A 147      -5.337 -22.615   4.472  1.00  0.00           C  \\nATOM   2267  CE1 TYR A 147      -5.799 -23.517   1.907  1.00  0.00           C  \\nATOM   2268  CE2 TYR A 147      -5.150 -23.950   4.158  1.00  0.00           C  \\nATOM   2269  CZ  TYR A 147      -5.393 -24.368   2.883  1.00  0.00           C  \\nATOM   2270  OH  TYR A 147      -5.212 -25.720   2.614  1.00  0.00           O  \\nATOM   2271  H   TYR A 147      -5.521 -17.716   4.203  1.00  0.00           H  \\nATOM   2272  HA  TYR A 147      -3.787 -20.024   4.317  1.00  0.00           H  \\nATOM   2273 1HB  TYR A 147      -6.170 -20.192   4.940  1.00  0.00           H  \\nATOM   2274 2HB  TYR A 147      -6.653 -19.811   3.293  1.00  0.00           H  \\nATOM   2275  HD1 TYR A 147      -6.335 -21.448   1.491  1.00  0.00           H  \\nATOM   2276  HD2 TYR A 147      -5.167 -22.267   5.491  1.00  0.00           H  \\nATOM   2277  HE1 TYR A 147      -6.000 -23.862   0.893  1.00  0.00           H  \\nATOM   2278  HE2 TYR A 147      -4.814 -24.649   4.925  1.00  0.00           H  \\nATOM   2279  HH  TYR A 147      -4.929 -26.169   3.414  1.00  0.00           H  \\nATOM   2280  N   TYR A 148      -4.527 -19.038   1.317  1.00  0.00           N  \\nATOM   2281  CA  TYR A 148      -4.143 -19.343  -0.098  1.00  0.00           C  \\nATOM   2282  C   TYR A 148      -2.765 -18.760  -0.429  1.00  0.00           C  \\nATOM   2283  O   TYR A 148      -2.211 -19.072  -1.503  1.00  0.00           O  \\nATOM   2284  CB  TYR A 148      -5.248 -18.902  -1.041  1.00  0.00           C  \\nATOM   2285  CG  TYR A 148      -6.440 -19.811  -1.057  1.00  0.00           C  \\nATOM   2286  CD1 TYR A 148      -6.314 -21.185  -1.304  1.00  0.00           C  \\nATOM   2287  CD2 TYR A 148      -7.693 -19.310  -0.834  1.00  0.00           C  \\nATOM   2288  CE1 TYR A 148      -7.403 -21.981  -1.432  1.00  0.00           C  \\nATOM   2289  CE2 TYR A 148      -8.766 -20.101  -0.931  1.00  0.00           C  \\nATOM   2290  CZ  TYR A 148      -8.645 -21.406  -1.195  1.00  0.00           C  \\nATOM   2291  OH  TYR A 148      -9.748 -22.187  -1.295  1.00  0.00           O  \\nATOM   2292  H   TYR A 148      -5.160 -18.273   1.501  1.00  0.00           H  \\nATOM   2293  HA  TYR A 148      -4.005 -20.420  -0.195  1.00  0.00           H  \\nATOM   2294 1HB  TYR A 148      -5.589 -17.904  -0.761  1.00  0.00           H  \\nATOM   2295 2HB  TYR A 148      -4.857 -18.842  -2.056  1.00  0.00           H  \\nATOM   2296  HD1 TYR A 148      -5.325 -21.634  -1.397  1.00  0.00           H  \\nATOM   2297  HD2 TYR A 148      -7.822 -18.259  -0.576  1.00  0.00           H  \\nATOM   2298  HE1 TYR A 148      -7.280 -23.047  -1.624  1.00  0.00           H  \\nATOM   2299  HE2 TYR A 148      -9.764 -19.685  -0.795  1.00  0.00           H  \\nATOM   2300  HH  TYR A 148     -10.532 -21.652  -1.150  1.00  0.00           H  \\nATOM   2301  N   TYR A 149      -2.141 -17.998   0.463  1.00  0.00           N  \\nATOM   2302  CA  TYR A 149      -0.721 -17.664   0.190  1.00  0.00           C  \\nATOM   2303  C   TYR A 149       0.097 -18.939   0.238  1.00  0.00           C  \\nATOM   2304  O   TYR A 149       1.071 -19.050  -0.525  1.00  0.00           O  \\nATOM   2305  CB  TYR A 149      -0.227 -16.632   1.193  1.00  0.00           C  \\nATOM   2306  CG  TYR A 149       1.131 -16.077   0.815  1.00  0.00           C  \\nATOM   2307  CD1 TYR A 149       1.270 -15.207  -0.273  1.00  0.00           C  \\nATOM   2308  CD2 TYR A 149       2.218 -16.447   1.540  1.00  0.00           C  \\nATOM   2309  CE1 TYR A 149       2.515 -14.713  -0.606  1.00  0.00           C  \\nATOM   2310  CE2 TYR A 149       3.490 -15.967   1.215  1.00  0.00           C  \\nATOM   2311  CZ  TYR A 149       3.598 -15.157   0.125  1.00  0.00           C  \\nATOM   2312  OH  TYR A 149       4.866 -14.703  -0.202  1.00  0.00           O  \\nATOM   2313  H   TYR A 149      -2.578 -17.642   1.301  1.00  0.00           H  \\nATOM   2314  HA  TYR A 149      -0.651 -17.242  -0.813  1.00  0.00           H  \\nATOM   2315 1HB  TYR A 149      -0.944 -15.812   1.254  1.00  0.00           H  \\nATOM   2316 2HB  TYR A 149      -0.164 -17.086   2.181  1.00  0.00           H  \\nATOM   2317  HD1 TYR A 149       0.395 -14.920  -0.857  1.00  0.00           H  \\nATOM   2318  HD2 TYR A 149       2.104 -17.126   2.385  1.00  0.00           H  \\nATOM   2319  HE1 TYR A 149       2.624 -14.036  -1.453  1.00  0.00           H  \\nATOM   2320  HE2 TYR A 149       4.350 -16.245   1.825  1.00  0.00           H  \\nATOM   2321  HH  TYR A 149       5.502 -15.052   0.428  1.00  0.00           H  \\nATOM   2322  N   SER A 150      -0.277 -19.911   1.054  1.00  0.00           N  \\nATOM   2323  CA  SER A 150       0.501 -21.172   1.177  1.00  0.00           C  \\nATOM   2324  C   SER A 150       0.362 -22.061  -0.062  1.00  0.00           C  \\nATOM   2325  O   SER A 150       1.204 -22.942  -0.261  1.00  0.00           O  \\nATOM   2326  CB  SER A 150       0.090 -21.978   2.429  1.00  0.00           C  \\nATOM   2327  OG  SER A 150      -1.183 -22.541   2.376  1.00  0.00           O  \\nATOM   2328  H   SER A 150      -1.113 -19.793   1.608  1.00  0.00           H  \\nATOM   2329  HA  SER A 150       1.558 -20.919   1.270  1.00  0.00           H  \\nATOM   2330 1HB  SER A 150       0.801 -22.788   2.589  1.00  0.00           H  \\nATOM   2331 2HB  SER A 150       0.127 -21.332   3.305  1.00  0.00           H  \\nATOM   2332  HG  SER A 150      -1.548 -22.291   1.524  1.00  0.00           H  \\nATOM   2333  N   THR A 151      -0.669 -21.805  -0.877  1.00  0.00           N  \\nATOM   2334  CA  THR A 151      -0.900 -22.614  -2.070  1.00  0.00           C  \\nATOM   2335  C   THR A 151      -0.435 -21.918  -3.351  1.00  0.00           C  \\nATOM   2336  O   THR A 151      -0.558 -22.475  -4.489  1.00  0.00           O  \\nATOM   2337  CB  THR A 151      -2.378 -22.978  -2.223  1.00  0.00           C  \\nATOM   2338  OG1 THR A 151      -3.108 -21.769  -2.531  1.00  0.00           O  \\nATOM   2339  CG2 THR A 151      -2.859 -23.586  -0.949  1.00  0.00           C  \\nATOM   2340  H   THR A 151      -1.299 -21.043  -0.669  1.00  0.00           H  \\nATOM   2341  HA  THR A 151      -0.330 -23.539  -1.976  1.00  0.00           H  \\nATOM   2342  HB  THR A 151      -2.495 -23.687  -3.042  1.00  0.00           H  \\nATOM   2343  HG1 THR A 151      -2.498 -21.028  -2.567  1.00  0.00           H  \\nATOM   2344 1HG2 THR A 151      -3.912 -23.850  -1.047  1.00  0.00           H  \\nATOM   2345 2HG2 THR A 151      -2.278 -24.482  -0.732  1.00  0.00           H  \\nATOM   2346 3HG2 THR A 151      -2.739 -22.871  -0.137  1.00  0.00           H  \\nATOM   2347  N   GLY A 152       0.086 -20.705  -3.209  1.00  0.00           N  \\nATOM   2348  CA  GLY A 152       0.352 -19.865  -4.399  1.00  0.00           C  \\nATOM   2349  C   GLY A 152      -0.714 -19.059  -4.964  1.00  0.00           C  \\nATOM   2350  O   GLY A 152      -0.463 -18.347  -5.949  1.00  0.00           O  \\nATOM   2351  H   GLY A 152       0.306 -20.343  -2.292  1.00  0.00           H  \\nATOM   2352 1HA  GLY A 152       1.153 -19.160  -4.175  1.00  0.00           H  \\nATOM   2353 2HA  GLY A 152       0.697 -20.494  -5.218  1.00  0.00           H  \\nATOM   2354  N   GLY A 153      -1.903 -19.110  -4.379  1.00  0.00           N  \\nATOM   2355  CA  GLY A 153      -3.073 -18.438  -4.829  1.00  0.00           C  \\nATOM   2356  C   GLY A 153      -3.235 -16.970  -4.464  1.00  0.00           C  \\nATOM   2357  O   GLY A 153      -3.961 -16.270  -5.118  1.00  0.00           O  \\nATOM   2358  H   GLY A 153      -1.953 -19.681  -3.547  1.00  0.00           H  \\nATOM   2359 1HA  GLY A 153      -3.125 -18.487  -5.917  1.00  0.00           H  \\nATOM   2360 2HA  GLY A 153      -3.956 -18.946  -4.442  1.00  0.00           H  \\nATOM   2361  N   THR A 154      -2.557 -16.514  -3.443  1.00  0.00           N  \\nATOM   2362  CA  THR A 154      -2.591 -15.121  -2.977  1.00  0.00           C  \\nATOM   2363  C   THR A 154      -1.355 -14.344  -3.353  1.00  0.00           C  \\nATOM   2364  O   THR A 154      -0.220 -14.809  -3.057  1.00  0.00           O  \\nATOM   2365  CB  THR A 154      -2.773 -15.105  -1.448  1.00  0.00           C  \\nATOM   2366  OG1 THR A 154      -3.970 -15.779  -1.088  1.00  0.00           O  \\nATOM   2367  CG2 THR A 154      -2.880 -13.622  -0.922  1.00  0.00           C  \\nATOM   2368  H   THR A 154      -1.977 -17.182  -2.956  1.00  0.00           H  \\nATOM   2369  HA  THR A 154      -3.438 -14.620  -3.446  1.00  0.00           H  \\nATOM   2370  HB  THR A 154      -1.921 -15.593  -0.975  1.00  0.00           H  \\nATOM   2371  HG1 THR A 154      -4.407 -16.099  -1.881  1.00  0.00           H  \\nATOM   2372 1HG2 THR A 154      -3.008 -13.628   0.160  1.00  0.00           H  \\nATOM   2373 2HG2 THR A 154      -1.970 -13.079  -1.177  1.00  0.00           H  \\nATOM   2374 3HG2 THR A 154      -3.736 -13.133  -1.385  1.00  0.00           H  \\nATOM   2375  N   GLN A 155      -1.565 -13.174  -3.908  1.00  0.00           N  \\nATOM   2376  CA  GLN A 155      -0.458 -12.340  -4.246  1.00  0.00           C  \\nATOM   2377  C   GLN A 155      -0.141 -11.399  -3.114  1.00  0.00           C  \\nATOM   2378  O   GLN A 155      -0.988 -11.173  -2.234  1.00  0.00           O  \\nATOM   2379  CB  GLN A 155      -0.831 -11.519  -5.492  1.00  0.00           C  \\nATOM   2380  CG  GLN A 155      -0.974 -12.424  -6.751  1.00  0.00           C  \\nATOM   2381  CD  GLN A 155       0.121 -13.535  -6.826  1.00  0.00           C  \\nATOM   2382  OE1 GLN A 155       1.339 -13.246  -6.908  1.00  0.00           O  \\nATOM   2383  NE2 GLN A 155      -0.321 -14.811  -6.815  1.00  0.00           N  \\nATOM   2384  H   GLN A 155      -2.502 -12.850  -4.102  1.00  0.00           H  \\nATOM   2385  HA  GLN A 155       0.399 -12.976  -4.469  1.00  0.00           H  \\nATOM   2386 1HB  GLN A 155      -1.770 -10.995  -5.315  1.00  0.00           H  \\nATOM   2387 2HB  GLN A 155      -0.065 -10.765  -5.674  1.00  0.00           H  \\nATOM   2388 1HG  GLN A 155      -1.949 -12.911  -6.727  1.00  0.00           H  \\nATOM   2389 2HG  GLN A 155      -0.888 -11.804  -7.643  1.00  0.00           H  \\nATOM   2390 1HE2 GLN A 155       0.332 -15.568  -6.861  1.00  0.00           H  \\nATOM   2391 2HE2 GLN A 155      -1.301 -15.001  -6.761  1.00  0.00           H  \\nATOM   2392  N   LEU A 156       1.073 -10.875  -3.095  1.00  0.00           N  \\nATOM   2393  CA  LEU A 156       1.559 -10.057  -1.998  1.00  0.00           C  \\nATOM   2394  C   LEU A 156       0.663  -8.885  -1.582  1.00  0.00           C  \\nATOM   2395  O   LEU A 156       0.435  -8.651  -0.370  1.00  0.00           O  \\nATOM   2396  CB  LEU A 156       2.926  -9.498  -2.400  1.00  0.00           C  \\nATOM   2397  CG  LEU A 156       4.009  -9.683  -1.358  1.00  0.00           C  \\nATOM   2398  CD1 LEU A 156       5.162  -8.622  -1.516  1.00  0.00           C  \\nATOM   2399  CD2 LEU A 156       3.569  -9.882   0.151  1.00  0.00           C  \\nATOM   2400  H   LEU A 156       1.681 -11.056  -3.881  1.00  0.00           H  \\nATOM   2401  HA  LEU A 156       1.661 -10.688  -1.115  1.00  0.00           H  \\nATOM   2402 1HB  LEU A 156       3.246  -9.989  -3.318  1.00  0.00           H  \\nATOM   2403 2HB  LEU A 156       2.820  -8.432  -2.600  1.00  0.00           H  \\nATOM   2404  HG  LEU A 156       4.598 -10.569  -1.598  1.00  0.00           H  \\nATOM   2405 1HD1 LEU A 156       5.919  -8.790  -0.749  1.00  0.00           H  \\nATOM   2406 2HD1 LEU A 156       5.617  -8.723  -2.502  1.00  0.00           H  \\nATOM   2407 3HD1 LEU A 156       4.751  -7.619  -1.406  1.00  0.00           H  \\nATOM   2408 1HD2 LEU A 156       4.455 -10.002   0.776  1.00  0.00           H  \\nATOM   2409 2HD2 LEU A 156       3.006  -9.010   0.484  1.00  0.00           H  \\nATOM   2410 3HD2 LEU A 156       2.944 -10.771   0.234  1.00  0.00           H  \\nATOM   2411  N   PRO A 157       0.164  -8.062  -2.523  1.00  0.00           N  \\nATOM   2412  CA  PRO A 157      -0.663  -6.949  -2.038  1.00  0.00           C  \\nATOM   2413  C   PRO A 157      -1.915  -7.394  -1.297  1.00  0.00           C  \\nATOM   2414  O   PRO A 157      -2.265  -6.751  -0.229  1.00  0.00           O  \\nATOM   2415  CB  PRO A 157      -1.036  -6.136  -3.301  1.00  0.00           C  \\nATOM   2416  CG  PRO A 157       0.195  -6.473  -4.224  1.00  0.00           C  \\nATOM   2417  CD  PRO A 157       0.303  -7.978  -4.011  1.00  0.00           C  \\nATOM   2418  HA  PRO A 157      -0.067  -6.327  -1.354  1.00  0.00           H  \\nATOM   2419 1HB  PRO A 157      -2.009  -6.471  -3.690  1.00  0.00           H  \\nATOM   2420 2HB  PRO A 157      -1.142  -5.071  -3.045  1.00  0.00           H  \\nATOM   2421 1HG  PRO A 157      -0.015  -6.174  -5.261  1.00  0.00           H  \\nATOM   2422 2HG  PRO A 157       1.079  -5.903  -3.902  1.00  0.00           H  \\nATOM   2423 1HD  PRO A 157      -0.516  -8.484  -4.543  1.00  0.00           H  \\nATOM   2424 2HD  PRO A 157       1.279  -8.331  -4.375  1.00  0.00           H  \\nATOM   2425  N   THR A 158      -2.533  -8.485  -1.726  1.00  0.00           N  \\nATOM   2426  CA  THR A 158      -3.693  -9.030  -0.973  1.00  0.00           C  \\nATOM   2427  C   THR A 158      -3.297  -9.620   0.351  1.00  0.00           C  \\nATOM   2428  O   THR A 158      -4.056  -9.493   1.321  1.00  0.00           O  \\nATOM   2429  CB  THR A 158      -4.449 -10.058  -1.848  1.00  0.00           C  \\nATOM   2430  OG1 THR A 158      -5.004  -9.376  -3.004  1.00  0.00           O  \\nATOM   2431  CG2 THR A 158      -5.570 -10.642  -1.123  1.00  0.00           C  \\nATOM   2432  H   THR A 158      -2.225  -8.957  -2.565  1.00  0.00           H  \\nATOM   2433  HA  THR A 158      -4.366  -8.207  -0.731  1.00  0.00           H  \\nATOM   2434  HB  THR A 158      -3.767 -10.855  -2.144  1.00  0.00           H  \\nATOM   2435  HG1 THR A 158      -4.777  -8.444  -2.964  1.00  0.00           H  \\nATOM   2436 1HG2 THR A 158      -6.082 -11.361  -1.763  1.00  0.00           H  \\nATOM   2437 2HG2 THR A 158      -5.202 -11.148  -0.230  1.00  0.00           H  \\nATOM   2438 3HG2 THR A 158      -6.265  -9.855  -0.833  1.00  0.00           H  \\nATOM   2439  N   LEU A 159      -2.150 -10.266   0.432  1.00  0.00           N  \\nATOM   2440  CA  LEU A 159      -1.671 -10.796   1.690  1.00  0.00           C  \\nATOM   2441  C   LEU A 159      -1.459  -9.642   2.648  1.00  0.00           C  \\nATOM   2442  O   LEU A 159      -1.821  -9.749   3.831  1.00  0.00           O  \\nATOM   2443  CB  LEU A 159      -0.333 -11.539   1.498  1.00  0.00           C  \\nATOM   2444  CG  LEU A 159       0.277 -12.105   2.783  1.00  0.00           C  \\nATOM   2445  CD1 LEU A 159      -0.635 -13.224   3.263  1.00  0.00           C  \\nATOM   2446  CD2 LEU A 159       1.685 -12.642   2.515  1.00  0.00           C  \\nATOM   2447  H   LEU A 159      -1.592 -10.395  -0.400  1.00  0.00           H  \\nATOM   2448  HA  LEU A 159      -2.407 -11.503   2.070  1.00  0.00           H  \\nATOM   2449 1HB  LEU A 159      -0.490 -12.364   0.805  1.00  0.00           H  \\nATOM   2450 2HB  LEU A 159       0.386 -10.851   1.055  1.00  0.00           H  \\nATOM   2451  HG  LEU A 159       0.335 -11.318   3.536  1.00  0.00           H  \\nATOM   2452 1HD1 LEU A 159      -0.232 -13.654   4.180  1.00  0.00           H  \\nATOM   2453 2HD1 LEU A 159      -1.630 -12.824   3.457  1.00  0.00           H  \\nATOM   2454 3HD1 LEU A 159      -0.697 -13.997   2.497  1.00  0.00           H  \\nATOM   2455 1HD2 LEU A 159       2.105 -13.040   3.439  1.00  0.00           H  \\nATOM   2456 2HD2 LEU A 159       1.636 -13.434   1.768  1.00  0.00           H  \\nATOM   2457 3HD2 LEU A 159       2.318 -11.835   2.147  1.00  0.00           H  \\nATOM   2458  N   ALA A 160      -0.821  -8.564   2.179  1.00  0.00           N  \\nATOM   2459  CA  ALA A 160      -0.568  -7.454   3.110  1.00  0.00           C  \\nATOM   2460  C   ALA A 160      -1.845  -6.787   3.595  1.00  0.00           C  \\nATOM   2461  O   ALA A 160      -1.984  -6.457   4.790  1.00  0.00           O  \\nATOM   2462  CB  ALA A 160       0.402  -6.413   2.444  1.00  0.00           C  \\nATOM   2463  H   ALA A 160      -0.510  -8.484   1.221  1.00  0.00           H  \\nATOM   2464  HA  ALA A 160      -0.096  -7.862   4.004  1.00  0.00           H  \\nATOM   2465 1HB  ALA A 160       0.587  -5.592   3.137  1.00  0.00           H  \\nATOM   2466 2HB  ALA A 160       1.346  -6.900   2.199  1.00  0.00           H  \\nATOM   2467 3HB  ALA A 160      -0.052  -6.024   1.534  1.00  0.00           H  \\nATOM   2468  N   ARG A 161      -2.827  -6.659   2.722  1.00  0.00           N  \\nATOM   2469  CA  ARG A 161      -4.135  -6.080   3.080  1.00  0.00           C  \\nATOM   2470  C   ARG A 161      -4.757  -6.993   4.150  1.00  0.00           C  \\nATOM   2471  O   ARG A 161      -5.357  -6.529   5.084  1.00  0.00           O  \\nATOM   2472  CB  ARG A 161      -4.971  -6.082   1.820  1.00  0.00           C  \\nATOM   2473  CG  ARG A 161      -6.367  -5.616   2.107  1.00  0.00           C  \\nATOM   2474  CD  ARG A 161      -7.259  -5.898   0.901  1.00  0.00           C  \\nATOM   2475  NE  ARG A 161      -7.470  -4.574   0.366  1.00  0.00           N  \\nATOM   2476  CZ  ARG A 161      -8.561  -3.857   0.585  1.00  0.00           C  \\nATOM   2477  NH1 ARG A 161      -9.572  -4.383   1.251  1.00  0.00           N  \\nATOM   2478  NH2 ARG A 161      -8.676  -2.644   0.092  1.00  0.00           N  \\nATOM   2479  H   ARG A 161      -2.670  -6.971   1.774  1.00  0.00           H  \\nATOM   2480  HA  ARG A 161      -3.977  -5.060   3.431  1.00  0.00           H  \\nATOM   2481 1HB  ARG A 161      -4.511  -5.432   1.078  1.00  0.00           H  \\nATOM   2482 2HB  ARG A 161      -4.997  -7.088   1.402  1.00  0.00           H  \\nATOM   2483 1HG  ARG A 161      -6.756  -6.146   2.977  1.00  0.00           H  \\nATOM   2484 2HG  ARG A 161      -6.359  -4.544   2.309  1.00  0.00           H  \\nATOM   2485 1HD  ARG A 161      -6.737  -6.559   0.210  1.00  0.00           H  \\nATOM   2486 2HD  ARG A 161      -8.180  -6.375   1.234  1.00  0.00           H  \\nATOM   2487  HE  ARG A 161      -6.738  -4.177  -0.208  1.00  0.00           H  \\nATOM   2488 1HH1 ARG A 161      -9.513  -5.332   1.593  1.00  0.00           H  \\nATOM   2489 2HH1 ARG A 161     -10.405  -3.837   1.419  1.00  0.00           H  \\nATOM   2490 1HH2 ARG A 161      -7.927  -2.250  -0.461  1.00  0.00           H  \\nATOM   2491 2HH2 ARG A 161      -9.512  -2.106   0.265  1.00  0.00           H  \\nATOM   2492  N   SER A 162      -4.598  -8.300   3.978  1.00  0.00           N  \\nATOM   2493  CA  SER A 162      -5.226  -9.278   4.869  1.00  0.00           C  \\nATOM   2494  C   SER A 162      -4.529  -9.242   6.203  1.00  0.00           C  \\nATOM   2495  O   SER A 162      -5.197  -9.411   7.212  1.00  0.00           O  \\nATOM   2496  CB  SER A 162      -5.193 -10.640   4.216  1.00  0.00           C  \\nATOM   2497  OG  SER A 162      -5.968 -10.522   2.999  1.00  0.00           O  \\nATOM   2498  H   SER A 162      -4.028  -8.628   3.211  1.00  0.00           H  \\nATOM   2499  HA  SER A 162      -6.263  -8.982   5.031  1.00  0.00           H  \\nATOM   2500 1HB  SER A 162      -4.160 -10.924   4.018  1.00  0.00           H  \\nATOM   2501 2HB  SER A 162      -5.610 -11.380   4.897  1.00  0.00           H  \\nATOM   2502  HG  SER A 162      -6.274  -9.612   2.967  1.00  0.00           H  \\nATOM   2503  N   PHE A 163      -3.213  -9.002   6.267  1.00  0.00           N  \\nATOM   2504  CA  PHE A 163      -2.555  -8.821   7.563  1.00  0.00           C  \\nATOM   2505  C   PHE A 163      -3.134  -7.589   8.236  1.00  0.00           C  \\nATOM   2506  O   PHE A 163      -3.347  -7.576   9.493  1.00  0.00           O  \\nATOM   2507  CB  PHE A 163      -1.057  -8.627   7.374  1.00  0.00           C  \\nATOM   2508  CG  PHE A 163      -0.304  -9.883   7.031  1.00  0.00           C  \\nATOM   2509  CD1 PHE A 163      -0.567 -11.029   7.713  1.00  0.00           C  \\nATOM   2510  CD2 PHE A 163       0.761  -9.833   6.142  1.00  0.00           C  \\nATOM   2511  CE1 PHE A 163       0.208 -12.215   7.474  1.00  0.00           C  \\nATOM   2512  CE2 PHE A 163       1.531 -11.023   5.909  1.00  0.00           C  \\nATOM   2513  CZ  PHE A 163       1.234 -12.166   6.553  1.00  0.00           C  \\nATOM   2514  H   PHE A 163      -2.662  -8.943   5.423  1.00  0.00           H  \\nATOM   2515  HA  PHE A 163      -2.719  -9.718   8.163  1.00  0.00           H  \\nATOM   2516 1HB  PHE A 163      -0.882  -7.905   6.577  1.00  0.00           H  \\nATOM   2517 2HB  PHE A 163      -0.625  -8.218   8.287  1.00  0.00           H  \\nATOM   2518  HD1 PHE A 163      -1.375 -11.039   8.445  1.00  0.00           H  \\nATOM   2519  HD2 PHE A 163       1.010  -8.911   5.615  1.00  0.00           H  \\nATOM   2520  HE1 PHE A 163      -0.019 -13.132   8.016  1.00  0.00           H  \\nATOM   2521  HE2 PHE A 163       2.362 -10.995   5.205  1.00  0.00           H  \\nATOM   2522  HZ  PHE A 163       1.818 -13.060   6.339  1.00  0.00           H  \\nATOM   2523  N   ILE A 164      -3.330  -6.522   7.494  1.00  0.00           N  \\nATOM   2524  CA  ILE A 164      -3.762  -5.287   8.091  1.00  0.00           C  \\nATOM   2525  C   ILE A 164      -5.143  -5.505   8.729  1.00  0.00           C  \\nATOM   2526  O   ILE A 164      -5.452  -4.966   9.777  1.00  0.00           O  \\nATOM   2527  CB  ILE A 164      -3.794  -4.141   7.062  1.00  0.00           C  \\nATOM   2528  CG1 ILE A 164      -2.373  -3.746   6.744  1.00  0.00           C  \\nATOM   2529  CG2 ILE A 164      -4.577  -2.917   7.554  1.00  0.00           C  \\nATOM   2530  CD1 ILE A 164      -2.273  -2.820   5.508  1.00  0.00           C  \\nATOM   2531  H   ILE A 164      -3.179  -6.560   6.496  1.00  0.00           H  \\nATOM   2532  HA  ILE A 164      -3.057  -5.017   8.876  1.00  0.00           H  \\nATOM   2533  HB  ILE A 164      -4.267  -4.489   6.144  1.00  0.00           H  \\nATOM   2534 1HG1 ILE A 164      -1.937  -3.236   7.602  1.00  0.00           H  \\nATOM   2535 2HG1 ILE A 164      -1.779  -4.642   6.560  1.00  0.00           H  \\nATOM   2536 1HG2 ILE A 164      -4.563  -2.144   6.786  1.00  0.00           H  \\nATOM   2537 2HG2 ILE A 164      -5.607  -3.204   7.762  1.00  0.00           H  \\nATOM   2538 3HG2 ILE A 164      -4.117  -2.533   8.464  1.00  0.00           H  \\nATOM   2539 1HD1 ILE A 164      -1.228  -2.568   5.326  1.00  0.00           H  \\nATOM   2540 2HD1 ILE A 164      -2.680  -3.332   4.636  1.00  0.00           H  \\nATOM   2541 3HD1 ILE A 164      -2.839  -1.908   5.691  1.00  0.00           H  \\nATOM   2542  N   ILE A 165      -6.026  -6.219   8.067  1.00  0.00           N  \\nATOM   2543  CA  ILE A 165      -7.371  -6.541   8.574  1.00  0.00           C  \\nATOM   2544  C   ILE A 165      -7.211  -7.387   9.855  1.00  0.00           C  \\nATOM   2545  O   ILE A 165      -7.871  -7.084  10.874  1.00  0.00           O  \\nATOM   2546  CB  ILE A 165      -8.212  -7.324   7.547  1.00  0.00           C  \\nATOM   2547  CG1 ILE A 165      -8.488  -6.364   6.351  1.00  0.00           C  \\nATOM   2548  CG2 ILE A 165      -9.470  -7.917   8.189  1.00  0.00           C  \\nATOM   2549  CD1 ILE A 165      -8.977  -7.041   5.072  1.00  0.00           C  \\nATOM   2550  H   ILE A 165      -5.749  -6.559   7.157  1.00  0.00           H  \\nATOM   2551  HA  ILE A 165      -7.892  -5.609   8.788  1.00  0.00           H  \\nATOM   2552  HB  ILE A 165      -7.617  -8.139   7.136  1.00  0.00           H  \\nATOM   2553 1HG1 ILE A 165      -9.238  -5.629   6.640  1.00  0.00           H  \\nATOM   2554 2HG1 ILE A 165      -7.576  -5.819   6.104  1.00  0.00           H  \\nATOM   2555 1HG2 ILE A 165     -10.040  -8.462   7.437  1.00  0.00           H  \\nATOM   2556 2HG2 ILE A 165      -9.183  -8.597   8.990  1.00  0.00           H  \\nATOM   2557 3HG2 ILE A 165     -10.083  -7.114   8.597  1.00  0.00           H  \\nATOM   2558 1HD1 ILE A 165      -9.139  -6.288   4.301  1.00  0.00           H  \\nATOM   2559 2HD1 ILE A 165      -8.228  -7.756   4.729  1.00  0.00           H  \\nATOM   2560 3HD1 ILE A 165      -9.912  -7.562   5.271  1.00  0.00           H  \\nATOM   2561  N   CYS A 166      -6.439  -8.462   9.809  1.00  0.00           N  \\nATOM   2562  CA  CYS A 166      -6.307  -9.279  11.005  1.00  0.00           C  \\nATOM   2563  C   CYS A 166      -5.683  -8.611  12.170  1.00  0.00           C  \\nATOM   2564  O   CYS A 166      -6.117  -8.797  13.286  1.00  0.00           O  \\nATOM   2565  CB  CYS A 166      -5.509 -10.543  10.708  1.00  0.00           C  \\nATOM   2566  SG  CYS A 166      -6.271 -11.761   9.591  1.00  0.00           S  \\nATOM   2567  H   CYS A 166      -5.941  -8.731   8.972  1.00  0.00           H  \\nATOM   2568  HA  CYS A 166      -7.304  -9.568  11.338  1.00  0.00           H  \\nATOM   2569 1HB  CYS A 166      -4.552 -10.273  10.262  1.00  0.00           H  \\nATOM   2570 2HB  CYS A 166      -5.299 -11.068  11.640  1.00  0.00           H  \\nATOM   2571  HG  CYS A 166      -5.268 -12.633   9.632  1.00  0.00           H  \\nATOM   2572  N   ILE A 167      -4.625  -7.856  11.989  1.00  0.00           N  \\nATOM   2573  CA  ILE A 167      -3.973  -7.160  13.099  1.00  0.00           C  \\nATOM   2574  C   ILE A 167      -4.930  -6.165  13.764  1.00  0.00           C  \\nATOM   2575  O   ILE A 167      -4.966  -6.071  14.962  1.00  0.00           O  \\nATOM   2576  CB  ILE A 167      -2.693  -6.450  12.550  1.00  0.00           C  \\nATOM   2577  CG1 ILE A 167      -1.654  -7.516  12.173  1.00  0.00           C  \\nATOM   2578  CG2 ILE A 167      -2.175  -5.408  13.541  1.00  0.00           C  \\nATOM   2579  CD1 ILE A 167      -0.573  -6.935  11.310  1.00  0.00           C  \\nATOM   2580  H   ILE A 167      -4.251  -7.753  11.056  1.00  0.00           H  \\nATOM   2581  HA  ILE A 167      -3.690  -7.896  13.850  1.00  0.00           H  \\nATOM   2582  HB  ILE A 167      -2.929  -5.951  11.611  1.00  0.00           H  \\nATOM   2583 1HG1 ILE A 167      -1.216  -7.933  13.079  1.00  0.00           H  \\nATOM   2584 2HG1 ILE A 167      -2.146  -8.332  11.643  1.00  0.00           H  \\nATOM   2585 1HG2 ILE A 167      -1.284  -4.930  13.134  1.00  0.00           H  \\nATOM   2586 2HG2 ILE A 167      -2.944  -4.656  13.713  1.00  0.00           H  \\nATOM   2587 3HG2 ILE A 167      -1.927  -5.895  14.484  1.00  0.00           H  \\nATOM   2588 1HD1 ILE A 167       0.149  -7.712  11.058  1.00  0.00           H  \\nATOM   2589 2HD1 ILE A 167      -1.012  -6.537  10.395  1.00  0.00           H  \\nATOM   2590 3HD1 ILE A 167      -0.069  -6.134  11.849  1.00  0.00           H  \\nATOM   2591  N   GLN A 168      -5.743  -5.464  12.990  1.00  0.00           N  \\nATOM   2592  CA  GLN A 168      -6.649  -4.482  13.596  1.00  0.00           C  \\nATOM   2593  C   GLN A 168      -7.785  -5.229  14.252  1.00  0.00           C  \\nATOM   2594  O   GLN A 168      -8.142  -4.779  15.321  1.00  0.00           O  \\nATOM   2595  CB  GLN A 168      -7.240  -3.505  12.645  1.00  0.00           C  \\nATOM   2596  CG  GLN A 168      -6.095  -2.601  12.070  1.00  0.00           C  \\nATOM   2597  CD  GLN A 168      -6.638  -1.635  11.130  1.00  0.00           C  \\nATOM   2598  OE1 GLN A 168      -7.091  -0.517  11.553  1.00  0.00           O  \\nATOM   2599  NE2 GLN A 168      -6.785  -2.047   9.881  1.00  0.00           N  \\nATOM   2600  H   GLN A 168      -5.754  -5.590  11.988  1.00  0.00           H  \\nATOM   2601  HA  GLN A 168      -6.089  -3.900  14.328  1.00  0.00           H  \\nATOM   2602 1HB  GLN A 168      -7.749  -4.040  11.843  1.00  0.00           H  \\nATOM   2603 2HB  GLN A 168      -7.986  -2.901  13.161  1.00  0.00           H  \\nATOM   2604 1HG  GLN A 168      -5.615  -2.072  12.894  1.00  0.00           H  \\nATOM   2605 2HG  GLN A 168      -5.369  -3.233  11.559  1.00  0.00           H  \\nATOM   2606 1HE2 GLN A 168      -7.154  -1.425   9.189  1.00  0.00           H  \\nATOM   2607 2HE2 GLN A 168      -6.527  -2.980   9.628  1.00  0.00           H  \\nATOM   2608  N   MET A 169      -8.334  -6.272  13.680  1.00  0.00           N  \\nATOM   2609  CA  MET A 169      -9.509  -6.874  14.294  1.00  0.00           C  \\nATOM   2610  C   MET A 169      -9.169  -7.809  15.459  1.00  0.00           C  \\nATOM   2611  O   MET A 169     -10.048  -8.182  16.165  1.00  0.00           O  \\nATOM   2612  CB  MET A 169     -10.360  -7.609  13.253  1.00  0.00           C  \\nATOM   2613  CG  MET A 169     -10.868  -6.676  12.190  1.00  0.00           C  \\nATOM   2614  SD  MET A 169     -12.033  -7.473  11.078  1.00  0.00           S  \\nATOM   2615  CE  MET A 169     -13.421  -7.665  12.160  1.00  0.00           C  \\nATOM   2616  H   MET A 169      -7.966  -6.669  12.827  1.00  0.00           H  \\nATOM   2617  HA  MET A 169     -10.114  -6.081  14.734  1.00  0.00           H  \\nATOM   2618 1HB  MET A 169      -9.767  -8.395  12.788  1.00  0.00           H  \\nATOM   2619 2HB  MET A 169     -11.207  -8.087  13.747  1.00  0.00           H  \\nATOM   2620 1HG  MET A 169     -11.362  -5.825  12.658  1.00  0.00           H  \\nATOM   2621 2HG  MET A 169     -10.029  -6.302  11.603  1.00  0.00           H  \\nATOM   2622 1HE  MET A 169     -14.238  -8.147  11.622  1.00  0.00           H  \\nATOM   2623 2HE  MET A 169     -13.134  -8.280  13.014  1.00  0.00           H  \\nATOM   2624 3HE  MET A 169     -13.747  -6.685  12.511  1.00  0.00           H  \\nATOM   2625  N   ILE A 170      -7.901  -8.102  15.649  1.00  0.00           N  \\nATOM   2626  CA  ILE A 170      -7.477  -9.011  16.707  1.00  0.00           C  \\nATOM   2627  C   ILE A 170      -6.536  -8.320  17.658  1.00  0.00           C  \\nATOM   2628  O   ILE A 170      -6.887  -8.082  18.812  1.00  0.00           O  \\nATOM   2629  CB  ILE A 170      -6.860 -10.280  16.127  1.00  0.00           C  \\nATOM   2630  CG1 ILE A 170      -7.942 -11.066  15.360  1.00  0.00           C  \\nATOM   2631  CG2 ILE A 170      -6.366 -11.163  17.288  1.00  0.00           C  \\nATOM   2632  CD1 ILE A 170      -7.384 -12.167  14.433  1.00  0.00           C  \\nATOM   2633  H   ILE A 170      -7.203  -7.688  15.048  1.00  0.00           H  \\nATOM   2634  HA  ILE A 170      -8.352  -9.292  17.292  1.00  0.00           H  \\nATOM   2635  HB  ILE A 170      -6.023 -10.015  15.481  1.00  0.00           H  \\nATOM   2636 1HG1 ILE A 170      -8.623 -11.534  16.070  1.00  0.00           H  \\nATOM   2637 2HG1 ILE A 170      -8.529 -10.378  14.751  1.00  0.00           H  \\nATOM   2638 1HG2 ILE A 170      -5.923 -12.075  16.888  1.00  0.00           H  \\nATOM   2639 2HG2 ILE A 170      -5.619 -10.620  17.866  1.00  0.00           H  \\nATOM   2640 3HG2 ILE A 170      -7.207 -11.421  17.932  1.00  0.00           H  \\nATOM   2641 1HD1 ILE A 170      -8.209 -12.672  13.930  1.00  0.00           H  \\nATOM   2642 2HD1 ILE A 170      -6.726 -11.717  13.688  1.00  0.00           H  \\nATOM   2643 3HD1 ILE A 170      -6.823 -12.890  15.024  1.00  0.00           H  \\nATOM   2644  N   SER A 171      -5.336  -7.908  17.213  1.00  0.00           N  \\nATOM   2645  CA  SER A 171      -4.369  -7.270  18.121  1.00  0.00           C  \\nATOM   2646  C   SER A 171      -4.864  -5.929  18.636  1.00  0.00           C  \\nATOM   2647  O   SER A 171      -4.811  -5.653  19.843  1.00  0.00           O  \\nATOM   2648  CB  SER A 171      -3.030  -6.993  17.442  1.00  0.00           C  \\nATOM   2649  OG  SER A 171      -2.439  -8.264  17.071  1.00  0.00           O  \\nATOM   2650  H   SER A 171      -5.089  -8.036  16.242  1.00  0.00           H  \\nATOM   2651  HA  SER A 171      -4.184  -7.944  18.958  1.00  0.00           H  \\nATOM   2652 1HB  SER A 171      -3.189  -6.364  16.566  1.00  0.00           H  \\nATOM   2653 2HB  SER A 171      -2.383  -6.444  18.125  1.00  0.00           H  \\nATOM   2654  HG  SER A 171      -3.060  -8.938  17.358  1.00  0.00           H  \\nATOM   2655  N   GLU A 172      -5.283  -5.041  17.714  1.00  0.00           N  \\nATOM   2656  CA  GLU A 172      -5.690  -3.699  18.164  1.00  0.00           C  \\nATOM   2657  C   GLU A 172      -6.989  -3.723  19.018  1.00  0.00           C  \\nATOM   2658  O   GLU A 172      -7.130  -2.972  19.976  1.00  0.00           O  \\nATOM   2659  CB  GLU A 172      -5.786  -2.711  17.002  1.00  0.00           C  \\nATOM   2660  CG  GLU A 172      -4.566  -2.606  16.139  1.00  0.00           C  \\nATOM   2661  CD  GLU A 172      -3.417  -1.868  16.737  1.00  0.00           C  \\nATOM   2662  OE1 GLU A 172      -3.352  -1.552  17.958  1.00  0.00           O  \\nATOM   2663  OE2 GLU A 172      -2.446  -1.582  15.975  1.00  0.00           O  \\nATOM   2664  H   GLU A 172      -5.327  -5.264  16.730  1.00  0.00           H  \\nATOM   2665  HA  GLU A 172      -4.939  -3.325  18.861  1.00  0.00           H  \\nATOM   2666 1HB  GLU A 172      -6.617  -2.991  16.354  1.00  0.00           H  \\nATOM   2667 2HB  GLU A 172      -5.995  -1.713  17.389  1.00  0.00           H  \\nATOM   2668 1HG  GLU A 172      -4.219  -3.610  15.895  1.00  0.00           H  \\nATOM   2669 2HG  GLU A 172      -4.834  -2.108  15.208  1.00  0.00           H  \\nATOM   2670  N   ALA A 173      -7.932  -4.591  18.636  1.00  0.00           N  \\nATOM   2671  CA  ALA A 173      -9.121  -4.811  19.393  1.00  0.00           C  \\nATOM   2672  C   ALA A 173      -8.782  -5.342  20.799  1.00  0.00           C  \\nATOM   2673  O   ALA A 173      -9.416  -4.917  21.754  1.00  0.00           O  \\nATOM   2674  CB  ALA A 173     -10.087  -5.821  18.657  1.00  0.00           C  \\nATOM   2675  H   ALA A 173      -7.792  -5.107  17.779  1.00  0.00           H  \\nATOM   2676  HA  ALA A 173      -9.635  -3.856  19.501  1.00  0.00           H  \\nATOM   2677 1HB  ALA A 173     -10.985  -5.969  19.258  1.00  0.00           H  \\nATOM   2678 2HB  ALA A 173     -10.364  -5.416  17.684  1.00  0.00           H  \\nATOM   2679 3HB  ALA A 173      -9.580  -6.775  18.521  1.00  0.00           H  \\nATOM   2680  N   ALA A 174      -7.820  -6.279  20.931  1.00  0.00           N  \\nATOM   2681  CA  ALA A 174      -7.382  -6.653  22.289  1.00  0.00           C  \\nATOM   2682  C   ALA A 174      -6.766  -5.492  23.036  1.00  0.00           C  \\nATOM   2683  O   ALA A 174      -7.049  -5.334  24.214  1.00  0.00           O  \\nATOM   2684  CB  ALA A 174      -6.384  -7.797  22.166  1.00  0.00           C  \\nATOM   2685  H   ALA A 174      -7.393  -6.730  20.135  1.00  0.00           H  \\nATOM   2686  HA  ALA A 174      -8.258  -6.982  22.849  1.00  0.00           H  \\nATOM   2687 1HB  ALA A 174      -6.045  -8.092  23.159  1.00  0.00           H  \\nATOM   2688 2HB  ALA A 174      -6.863  -8.647  21.679  1.00  0.00           H  \\nATOM   2689 3HB  ALA A 174      -5.531  -7.473  21.573  1.00  0.00           H  \\nATOM   2690  N   ARG A 175      -5.966  -4.663  22.367  1.00  0.00           N  \\nATOM   2691  CA  ARG A 175      -5.350  -3.488  23.044  1.00  0.00           C  \\nATOM   2692  C   ARG A 175      -6.348  -2.450  23.496  1.00  0.00           C  \\nATOM   2693  O   ARG A 175      -6.100  -1.778  24.458  1.00  0.00           O  \\nATOM   2694  CB  ARG A 175      -4.322  -2.837  22.076  1.00  0.00           C  \\nATOM   2695  CG  ARG A 175      -3.126  -3.724  21.839  1.00  0.00           C  \\nATOM   2696  CD  ARG A 175      -2.333  -2.992  20.718  1.00  0.00           C  \\nATOM   2697  NE  ARG A 175      -0.999  -3.586  20.489  1.00  0.00           N  \\nATOM   2698  CZ  ARG A 175      -0.161  -3.053  19.599  1.00  0.00           C  \\nATOM   2699  NH1 ARG A 175      -0.518  -2.015  18.841  1.00  0.00           N  \\nATOM   2700  NH2 ARG A 175       1.028  -3.555  19.507  1.00  0.00           N  \\nATOM   2701  H   ARG A 175      -5.770  -4.826  21.390  1.00  0.00           H  \\nATOM   2702  HA  ARG A 175      -4.836  -3.836  23.941  1.00  0.00           H  \\nATOM   2703 1HB  ARG A 175      -4.804  -2.626  21.122  1.00  0.00           H  \\nATOM   2704 2HB  ARG A 175      -3.984  -1.886  22.489  1.00  0.00           H  \\nATOM   2705 1HG  ARG A 175      -2.550  -3.812  22.760  1.00  0.00           H  \\nATOM   2706 2HG  ARG A 175      -3.463  -4.713  21.526  1.00  0.00           H  \\nATOM   2707 1HD  ARG A 175      -2.890  -3.047  19.783  1.00  0.00           H  \\nATOM   2708 2HD  ARG A 175      -2.192  -1.948  20.995  1.00  0.00           H  \\nATOM   2709  HE  ARG A 175      -0.750  -4.404  21.029  1.00  0.00           H  \\nATOM   2710 1HH1 ARG A 175      -1.441  -1.615  18.934  1.00  0.00           H  \\nATOM   2711 2HH1 ARG A 175       0.134  -1.629  18.174  1.00  0.00           H  \\nATOM   2712 1HH2 ARG A 175       1.297  -4.328  20.100  1.00  0.00           H  \\nATOM   2713 2HH2 ARG A 175       1.686  -3.175  18.844  1.00  0.00           H  \\nATOM   2714  N   PHE A 176      -7.427  -2.238  22.762  1.00  0.00           N  \\nATOM   2715  CA  PHE A 176      -8.319  -1.089  23.018  1.00  0.00           C  \\nATOM   2716  C   PHE A 176      -9.752  -1.500  23.067  1.00  0.00           C  \\nATOM   2717  O   PHE A 176     -10.288  -1.952  22.071  1.00  0.00           O  \\nATOM   2718  CB  PHE A 176      -8.205  -0.110  21.860  1.00  0.00           C  \\nATOM   2719  CG  PHE A 176      -6.871   0.592  21.765  1.00  0.00           C  \\nATOM   2720  CD1 PHE A 176      -6.596   1.698  22.548  1.00  0.00           C  \\nATOM   2721  CD2 PHE A 176      -5.896   0.206  20.866  1.00  0.00           C  \\nATOM   2722  CE1 PHE A 176      -5.394   2.407  22.421  1.00  0.00           C  \\nATOM   2723  CE2 PHE A 176      -4.693   0.900  20.758  1.00  0.00           C  \\nATOM   2724  CZ  PHE A 176      -4.421   1.996  21.520  1.00  0.00           C  \\nATOM   2725  H   PHE A 176      -7.648  -2.874  22.009  1.00  0.00           H  \\nATOM   2726  HA  PHE A 176      -7.998  -0.600  23.939  1.00  0.00           H  \\nATOM   2727 1HB  PHE A 176      -8.373  -0.636  20.921  1.00  0.00           H  \\nATOM   2728 2HB  PHE A 176      -8.978   0.652  21.951  1.00  0.00           H  \\nATOM   2729  HD1 PHE A 176      -7.332   2.026  23.283  1.00  0.00           H  \\nATOM   2730  HD2 PHE A 176      -6.079  -0.667  20.239  1.00  0.00           H  \\nATOM   2731  HE1 PHE A 176      -5.216   3.291  23.033  1.00  0.00           H  \\nATOM   2732  HE2 PHE A 176      -3.958   0.543  20.037  1.00  0.00           H  \\nATOM   2733  HZ  PHE A 176      -3.484   2.545  21.441  1.00  0.00           H  \\nATOM   2734  N   GLN A 177     -10.434  -1.317  24.217  1.00  0.00           N  \\nATOM   2735  CA  GLN A 177     -11.938  -1.553  24.203  1.00  0.00           C  \\nATOM   2736  C   GLN A 177     -12.624  -0.719  23.154  1.00  0.00           C  \\nATOM   2737  O   GLN A 177     -13.636  -1.159  22.603  1.00  0.00           O  \\nATOM   2738  CB  GLN A 177     -12.478  -1.109  25.570  1.00  0.00           C  \\nATOM   2739  CG  GLN A 177     -12.035  -1.972  26.672  1.00  0.00           C  \\nATOM   2740  CD  GLN A 177     -10.989  -1.224  27.411  1.00  0.00           C  \\nATOM   2741  OE1 GLN A 177     -10.235  -0.467  26.795  1.00  0.00           O  \\nATOM   2742  NE2 GLN A 177     -11.013  -1.293  28.703  1.00  0.00           N  \\nATOM   2743  H   GLN A 177      -9.989  -1.032  25.078  1.00  0.00           H  \\nATOM   2744  HA  GLN A 177     -12.120  -2.617  24.053  1.00  0.00           H  \\nATOM   2745 1HB  GLN A 177     -12.153  -0.089  25.776  1.00  0.00           H  \\nATOM   2746 2HB  GLN A 177     -13.568  -1.107  25.548  1.00  0.00           H  \\nATOM   2747 1HG  GLN A 177     -12.888  -2.190  27.315  1.00  0.00           H  \\nATOM   2748 2HG  GLN A 177     -11.635  -2.897  26.257  1.00  0.00           H  \\nATOM   2749 1HE2 GLN A 177     -10.327  -0.804  29.243  1.00  0.00           H  \\nATOM   2750 2HE2 GLN A 177     -11.717  -1.834  29.162  1.00  0.00           H  \\nATOM   2751  N   TYR A 178     -12.088   0.504  22.829  1.00  0.00           N  \\nATOM   2752  CA  TYR A 178     -12.712   1.352  21.790  1.00  0.00           C  \\nATOM   2753  C   TYR A 178     -12.691   0.681  20.424  1.00  0.00           C  \\nATOM   2754  O   TYR A 178     -13.690   0.719  19.743  1.00  0.00           O  \\nATOM   2755  CB  TYR A 178     -11.935   2.677  21.717  1.00  0.00           C  \\nATOM   2756  CG  TYR A 178     -12.446   3.652  20.721  1.00  0.00           C  \\nATOM   2757  CD1 TYR A 178     -13.547   4.440  21.070  1.00  0.00           C  \\nATOM   2758  CD2 TYR A 178     -11.828   3.863  19.492  1.00  0.00           C  \\nATOM   2759  CE1 TYR A 178     -14.066   5.371  20.135  1.00  0.00           C  \\nATOM   2760  CE2 TYR A 178     -12.327   4.734  18.598  1.00  0.00           C  \\nATOM   2761  CZ  TYR A 178     -13.409   5.527  18.945  1.00  0.00           C  \\nATOM   2762  OH  TYR A 178     -13.860   6.454  18.067  1.00  0.00           O  \\nATOM   2763  H   TYR A 178     -11.258   0.834  23.301  1.00  0.00           H  \\nATOM   2764  HA  TYR A 178     -13.745   1.548  22.078  1.00  0.00           H  \\nATOM   2765 1HB  TYR A 178     -11.954   3.164  22.693  1.00  0.00           H  \\nATOM   2766 2HB  TYR A 178     -10.893   2.474  21.472  1.00  0.00           H  \\nATOM   2767  HD1 TYR A 178     -14.001   4.336  22.056  1.00  0.00           H  \\nATOM   2768  HD2 TYR A 178     -10.920   3.316  19.237  1.00  0.00           H  \\nATOM   2769  HE1 TYR A 178     -14.924   5.989  20.400  1.00  0.00           H  \\nATOM   2770  HE2 TYR A 178     -11.881   4.813  17.606  1.00  0.00           H  \\nATOM   2771  HH  TYR A 178     -13.325   6.426  17.270  1.00  0.00           H  \\nATOM   2772  N   ILE A 179     -11.582  -0.010  20.119  1.00  0.00           N  \\nATOM   2773  CA  ILE A 179     -11.465  -0.671  18.818  1.00  0.00           C  \\nATOM   2774  C   ILE A 179     -12.252  -1.995  18.770  1.00  0.00           C  \\nATOM   2775  O   ILE A 179     -12.891  -2.293  17.733  1.00  0.00           O  \\nATOM   2776  CB  ILE A 179      -9.969  -0.860  18.488  1.00  0.00           C  \\nATOM   2777  CG1 ILE A 179      -9.368   0.562  18.255  1.00  0.00           C  \\nATOM   2778  CG2 ILE A 179      -9.865  -1.622  17.117  1.00  0.00           C  \\nATOM   2779  CD1 ILE A 179      -7.938   0.576  17.768  1.00  0.00           C  \\nATOM   2780  H   ILE A 179     -10.820  -0.082  20.778  1.00  0.00           H  \\nATOM   2781  HA  ILE A 179     -11.925  -0.034  18.064  1.00  0.00           H  \\nATOM   2782  HB  ILE A 179      -9.493  -1.441  19.277  1.00  0.00           H  \\nATOM   2783 1HG1 ILE A 179      -9.972   1.097  17.523  1.00  0.00           H  \\nATOM   2784 2HG1 ILE A 179      -9.405   1.130  19.185  1.00  0.00           H  \\nATOM   2785 1HG2 ILE A 179      -8.816  -1.768  16.860  1.00  0.00           H  \\nATOM   2786 2HG2 ILE A 179     -10.356  -2.591  17.201  1.00  0.00           H  \\nATOM   2787 3HG2 ILE A 179     -10.351  -1.036  16.337  1.00  0.00           H  \\nATOM   2788 1HD1 ILE A 179      -7.609   1.607  17.636  1.00  0.00           H  \\nATOM   2789 2HD1 ILE A 179      -7.299   0.082  18.501  1.00  0.00           H  \\nATOM   2790 3HD1 ILE A 179      -7.873   0.050  16.816  1.00  0.00           H  \\nATOM   2791  N   GLU A 180     -12.289  -2.725  19.872  1.00  0.00           N  \\nATOM   2792  CA  GLU A 180     -13.187  -3.859  20.013  1.00  0.00           C  \\nATOM   2793  C   GLU A 180     -14.613  -3.364  19.801  1.00  0.00           C  \\nATOM   2794  O   GLU A 180     -15.356  -3.963  19.016  1.00  0.00           O  \\nATOM   2795  CB  GLU A 180     -13.051  -4.429  21.447  1.00  0.00           C  \\nATOM   2796  CG  GLU A 180     -14.111  -5.547  21.767  1.00  0.00           C  \\nATOM   2797  CD  GLU A 180     -14.336  -5.713  23.249  1.00  0.00           C  \\nATOM   2798  OE1 GLU A 180     -13.590  -5.122  24.075  1.00  0.00           O  \\nATOM   2799  OE2 GLU A 180     -15.299  -6.491  23.559  1.00  0.00           O  \\nATOM   2800  H   GLU A 180     -11.674  -2.486  20.637  1.00  0.00           H  \\nATOM   2801  HA  GLU A 180     -12.897  -4.622  19.290  1.00  0.00           H  \\nATOM   2802 1HB  GLU A 180     -12.053  -4.848  21.579  1.00  0.00           H  \\nATOM   2803 2HB  GLU A 180     -13.166  -3.623  22.171  1.00  0.00           H  \\nATOM   2804 1HG  GLU A 180     -15.055  -5.286  21.289  1.00  0.00           H  \\nATOM   2805 2HG  GLU A 180     -13.769  -6.490  21.342  1.00  0.00           H  \\nATOM   2806  N   GLY A 181     -14.898  -2.186  20.337  1.00  0.00           N  \\nATOM   2807  CA  GLY A 181     -16.199  -1.618  20.069  1.00  0.00           C  \\nATOM   2808  C   GLY A 181     -16.542  -1.309  18.655  1.00  0.00           C  \\nATOM   2809  O   GLY A 181     -17.686  -1.491  18.255  1.00  0.00           O  \\nATOM   2810  H   GLY A 181     -14.249  -1.676  20.919  1.00  0.00           H  \\nATOM   2811 1HA  GLY A 181     -16.973  -2.296  20.429  1.00  0.00           H  \\nATOM   2812 2HA  GLY A 181     -16.309  -0.684  20.620  1.00  0.00           H  \\nATOM   2813  N   GLU A 182     -15.585  -0.790  17.903  1.00  0.00           N  \\nATOM   2814  CA  GLU A 182     -15.781  -0.509  16.454  1.00  0.00           C  \\nATOM   2815  C   GLU A 182     -16.080  -1.764  15.723  1.00  0.00           C  \\nATOM   2816  O   GLU A 182     -16.870  -1.799  14.688  1.00  0.00           O  \\nATOM   2817  CB  GLU A 182     -14.499   0.044  15.894  1.00  0.00           C  \\nATOM   2818  CG  GLU A 182     -14.179   1.455  16.350  1.00  0.00           C  \\nATOM   2819  CD  GLU A 182     -15.223   2.414  15.803  1.00  0.00           C  \\nATOM   2820  OE1 GLU A 182     -15.290   2.555  14.529  1.00  0.00           O  \\nATOM   2821  OE2 GLU A 182     -16.002   2.928  16.624  1.00  0.00           O  \\nATOM   2822  H   GLU A 182     -14.694  -0.579  18.329  1.00  0.00           H  \\nATOM   2823  HA  GLU A 182     -16.574   0.232  16.349  1.00  0.00           H  \\nATOM   2824 1HB  GLU A 182     -13.668  -0.600  16.183  1.00  0.00           H  \\nATOM   2825 2HB  GLU A 182     -14.548   0.045  14.805  1.00  0.00           H  \\nATOM   2826 1HG  GLU A 182     -14.170   1.480  17.439  1.00  0.00           H  \\nATOM   2827 2HG  GLU A 182     -13.183   1.722  15.999  1.00  0.00           H  \\nATOM   2828  N   MET A 183     -15.412  -2.855  16.101  1.00  0.00           N  \\nATOM   2829  CA  MET A 183     -15.716  -4.115  15.420  1.00  0.00           C  \\nATOM   2830  C   MET A 183     -17.031  -4.707  15.807  1.00  0.00           C  \\nATOM   2831  O   MET A 183     -17.665  -5.338  14.975  1.00  0.00           O  \\nATOM   2832  CB  MET A 183     -14.530  -5.120  15.656  1.00  0.00           C  \\nATOM   2833  CG  MET A 183     -13.153  -4.561  15.277  1.00  0.00           C  \\nATOM   2834  SD  MET A 183     -13.112  -3.968  13.525  1.00  0.00           S  \\nATOM   2835  CE  MET A 183     -12.756  -2.210  13.605  1.00  0.00           C  \\nATOM   2836  H   MET A 183     -14.713  -2.846  16.830  1.00  0.00           H  \\nATOM   2837  HA  MET A 183     -15.815  -3.916  14.353  1.00  0.00           H  \\nATOM   2838 1HB  MET A 183     -14.500  -5.407  16.706  1.00  0.00           H  \\nATOM   2839 2HB  MET A 183     -14.699  -6.026  15.073  1.00  0.00           H  \\nATOM   2840 1HG  MET A 183     -12.903  -3.731  15.937  1.00  0.00           H  \\nATOM   2841 2HG  MET A 183     -12.398  -5.336  15.405  1.00  0.00           H  \\nATOM   2842 1HE  MET A 183     -12.719  -1.800  12.595  1.00  0.00           H  \\nATOM   2843 2HE  MET A 183     -13.538  -1.706  14.174  1.00  0.00           H  \\nATOM   2844 3HE  MET A 183     -11.794  -2.055  14.095  1.00  0.00           H  \\nATOM   2845  N   ARG A 184     -17.393  -4.559  17.081  1.00  0.00           N  \\nATOM   2846  CA  ARG A 184     -18.718  -5.100  17.573  1.00  0.00           C  \\nATOM   2847  C   ARG A 184     -19.829  -4.371  16.748  1.00  0.00           C  \\nATOM   2848  O   ARG A 184     -20.790  -4.985  16.381  1.00  0.00           O  \\nATOM   2849  CB  ARG A 184     -18.925  -4.743  19.078  1.00  0.00           C  \\nATOM   2850  CG  ARG A 184     -18.237  -5.562  20.066  1.00  0.00           C  \\nATOM   2851  CD  ARG A 184     -19.008  -5.407  21.519  1.00  0.00           C  \\nATOM   2852  NE  ARG A 184     -19.273  -4.026  21.885  1.00  0.00           N  \\nATOM   2853  CZ  ARG A 184     -18.407  -3.287  22.600  1.00  0.00           C  \\nATOM   2854  NH1 ARG A 184     -17.286  -3.869  23.007  1.00  0.00           N  \\nATOM   2855  NH2 ARG A 184     -18.619  -1.988  22.913  1.00  0.00           N  \\nATOM   2856  H   ARG A 184     -16.784  -4.083  17.731  1.00  0.00           H  \\nATOM   2857  HA  ARG A 184     -18.709  -6.185  17.469  1.00  0.00           H  \\nATOM   2858 1HB  ARG A 184     -18.602  -3.718  19.256  1.00  0.00           H  \\nATOM   2859 2HB  ARG A 184     -19.986  -4.799  19.323  1.00  0.00           H  \\nATOM   2860 1HG  ARG A 184     -18.254  -6.605  19.749  1.00  0.00           H  \\nATOM   2861 2HG  ARG A 184     -17.203  -5.228  20.160  1.00  0.00           H  \\nATOM   2862 1HD  ARG A 184     -19.966  -5.925  21.480  1.00  0.00           H  \\nATOM   2863 2HD  ARG A 184     -18.392  -5.842  22.306  1.00  0.00           H  \\nATOM   2864  HE  ARG A 184     -20.148  -3.619  21.582  1.00  0.00           H  \\nATOM   2865 1HH1 ARG A 184     -17.107  -4.836  22.777  1.00  0.00           H  \\nATOM   2866 2HH1 ARG A 184     -16.612  -3.345  23.546  1.00  0.00           H  \\nATOM   2867 1HH2 ARG A 184     -19.464  -1.525  22.606  1.00  0.00           H  \\nATOM   2868 2HH2 ARG A 184     -17.934  -1.481  23.453  1.00  0.00           H  \\nATOM   2869  N   THR A 185     -19.688  -3.038  16.582  1.00  0.00           N  \\nATOM   2870  CA  THR A 185     -20.675  -2.236  15.800  1.00  0.00           C  \\nATOM   2871  C   THR A 185     -20.779  -2.713  14.341  1.00  0.00           C  \\nATOM   2872  O   THR A 185     -21.865  -2.847  13.818  1.00  0.00           O  \\nATOM   2873  CB  THR A 185     -20.269  -0.732  15.861  1.00  0.00           C  \\nATOM   2874  OG1 THR A 185     -20.281  -0.259  17.191  1.00  0.00           O  \\nATOM   2875  CG2 THR A 185     -21.254   0.156  15.080  1.00  0.00           C  \\nATOM   2876  H   THR A 185     -18.894  -2.572  16.998  1.00  0.00           H  \\nATOM   2877  HA  THR A 185     -21.659  -2.363  16.251  1.00  0.00           H  \\nATOM   2878  HB  THR A 185     -19.275  -0.606  15.433  1.00  0.00           H  \\nATOM   2879  HG1 THR A 185     -20.539  -0.971  17.781  1.00  0.00           H  \\nATOM   2880 1HG2 THR A 185     -20.937   1.197  15.146  1.00  0.00           H  \\nATOM   2881 2HG2 THR A 185     -21.272  -0.153  14.035  1.00  0.00           H  \\nATOM   2882 3HG2 THR A 185     -22.252   0.054  15.505  1.00  0.00           H  \\nATOM   2883  N   ARG A 186     -19.646  -3.017  13.714  1.00  0.00           N  \\nATOM   2884  CA  ARG A 186     -19.702  -3.539  12.357  1.00  0.00           C  \\nATOM   2885  C   ARG A 186     -20.415  -4.880  12.341  1.00  0.00           C  \\nATOM   2886  O   ARG A 186     -21.187  -5.158  11.393  1.00  0.00           O  \\nATOM   2887  CB  ARG A 186     -18.293  -3.604  11.781  1.00  0.00           C  \\nATOM   2888  CG  ARG A 186     -17.705  -2.185  11.400  1.00  0.00           C  \\nATOM   2889  CD  ARG A 186     -16.284  -2.256  11.221  1.00  0.00           C  \\nATOM   2890  NE  ARG A 186     -15.644  -0.993  10.739  1.00  0.00           N  \\nATOM   2891  CZ  ARG A 186     -15.581   0.111  11.468  1.00  0.00           C  \\nATOM   2892  NH1 ARG A 186     -15.979   0.104  12.745  1.00  0.00           N  \\nATOM   2893  NH2 ARG A 186     -15.021   1.214  11.021  1.00  0.00           N  \\nATOM   2894  H   ARG A 186     -18.747  -2.894  14.157  1.00  0.00           H  \\nATOM   2895  HA  ARG A 186     -20.306  -2.863  11.751  1.00  0.00           H  \\nATOM   2896 1HB  ARG A 186     -17.626  -4.070  12.504  1.00  0.00           H  \\nATOM   2897 2HB  ARG A 186     -18.294  -4.228  10.887  1.00  0.00           H  \\nATOM   2898 1HG  ARG A 186     -18.160  -1.840  10.471  1.00  0.00           H  \\nATOM   2899 2HG  ARG A 186     -17.924  -1.474  12.197  1.00  0.00           H  \\nATOM   2900 1HD  ARG A 186     -15.810  -2.503  12.170  1.00  0.00           H  \\nATOM   2901 2HD  ARG A 186     -16.048  -3.025  10.487  1.00  0.00           H  \\nATOM   2902  HE  ARG A 186     -15.243  -0.994   9.810  1.00  0.00           H  \\nATOM   2903 1HH1 ARG A 186     -16.331  -0.747  13.160  1.00  0.00           H  \\nATOM   2904 2HH1 ARG A 186     -15.928   0.950  13.294  1.00  0.00           H  \\nATOM   2905 1HH2 ARG A 186     -14.621   1.237  10.093  1.00  0.00           H  \\nATOM   2906 2HH2 ARG A 186     -14.991   2.037  11.604  1.00  0.00           H  \\nATOM   2907  N   ILE A 187     -20.124  -5.754  13.295  1.00  0.00           N  \\nATOM   2908  CA  ILE A 187     -20.822  -7.055  13.299  1.00  0.00           C  \\nATOM   2909  C   ILE A 187     -22.306  -6.852  13.610  1.00  0.00           C  \\nATOM   2910  O   ILE A 187     -23.165  -7.482  12.992  1.00  0.00           O  \\nATOM   2911  CB  ILE A 187     -20.170  -7.995  14.283  1.00  0.00           C  \\nATOM   2912  CG1 ILE A 187     -18.801  -8.274  13.715  1.00  0.00           C  \\nATOM   2913  CG2 ILE A 187     -20.960  -9.267  14.416  1.00  0.00           C  \\nATOM   2914  CD1 ILE A 187     -17.776  -8.733  14.793  1.00  0.00           C  \\nATOM   2915  H   ILE A 187     -19.443  -5.559  14.015  1.00  0.00           H  \\nATOM   2916  HA  ILE A 187     -20.752  -7.488  12.302  1.00  0.00           H  \\nATOM   2917  HB  ILE A 187     -20.112  -7.516  15.260  1.00  0.00           H  \\nATOM   2918 1HG1 ILE A 187     -18.875  -9.048  12.952  1.00  0.00           H  \\nATOM   2919 2HG1 ILE A 187     -18.417  -7.375  13.232  1.00  0.00           H  \\nATOM   2920 1HG2 ILE A 187     -20.469  -9.927  15.130  1.00  0.00           H  \\nATOM   2921 2HG2 ILE A 187     -21.965  -9.036  14.767  1.00  0.00           H  \\nATOM   2922 3HG2 ILE A 187     -21.020  -9.761  13.446  1.00  0.00           H  \\nATOM   2923 1HD1 ILE A 187     -16.810  -8.918  14.322  1.00  0.00           H  \\nATOM   2924 2HD1 ILE A 187     -17.668  -7.954  15.548  1.00  0.00           H  \\nATOM   2925 3HD1 ILE A 187     -18.130  -9.649  15.264  1.00  0.00           H  \\nATOM   2926  N   ARG A 188     -22.641  -5.949  14.547  1.00  0.00           N  \\nATOM   2927  CA  ARG A 188     -24.057  -5.707  14.921  1.00  0.00           C  \\nATOM   2928  C   ARG A 188     -24.878  -5.320  13.753  1.00  0.00           C  \\nATOM   2929  O   ARG A 188     -25.980  -5.828  13.610  1.00  0.00           O  \\nATOM   2930  CB  ARG A 188     -24.133  -4.602  15.969  1.00  0.00           C  \\nATOM   2931  CG  ARG A 188     -25.562  -4.167  16.323  1.00  0.00           C  \\nATOM   2932  CD  ARG A 188     -25.699  -3.549  17.714  1.00  0.00           C  \\nATOM   2933  NE  ARG A 188     -24.478  -3.002  18.172  1.00  0.00           N  \\nATOM   2934  CZ  ARG A 188     -23.883  -3.263  19.330  1.00  0.00           C  \\nATOM   2935  NH1 ARG A 188     -24.401  -4.085  20.260  1.00  0.00           N  \\nATOM   2936  NH2 ARG A 188     -22.733  -2.663  19.548  1.00  0.00           N  \\nATOM   2937  H   ARG A 188     -21.913  -5.422  15.008  1.00  0.00           H  \\nATOM   2938  HA  ARG A 188     -24.465  -6.625  15.344  1.00  0.00           H  \\nATOM   2939 1HB  ARG A 188     -23.647  -4.935  16.885  1.00  0.00           H  \\nATOM   2940 2HB  ARG A 188     -23.593  -3.725  15.613  1.00  0.00           H  \\nATOM   2941 1HG  ARG A 188     -25.904  -3.421  15.605  1.00  0.00           H  \\nATOM   2942 2HG  ARG A 188     -26.225  -5.032  16.290  1.00  0.00           H  \\nATOM   2943 1HD  ARG A 188     -26.440  -2.751  17.686  1.00  0.00           H  \\nATOM   2944 2HD  ARG A 188     -26.016  -4.314  18.422  1.00  0.00           H  \\nATOM   2945  HE  ARG A 188     -24.001  -2.346  17.568  1.00  0.00           H  \\nATOM   2946 1HH1 ARG A 188     -25.287  -4.540  20.094  1.00  0.00           H  \\nATOM   2947 2HH1 ARG A 188     -23.904  -4.247  21.124  1.00  0.00           H  \\nATOM   2948 1HH2 ARG A 188     -22.347  -2.041  18.851  1.00  0.00           H  \\nATOM   2949 2HH2 ARG A 188     -22.236  -2.824  20.411  1.00  0.00           H  \\nATOM   2950  N   TYR A 189     -24.358  -4.447  12.907  1.00  0.00           N  \\nATOM   2951  CA  TYR A 189     -25.095  -3.903  11.795  1.00  0.00           C  \\nATOM   2952  C   TYR A 189     -24.796  -4.614  10.493  1.00  0.00           C  \\nATOM   2953  O   TYR A 189     -25.295  -4.228   9.475  1.00  0.00           O  \\nATOM   2954  CB  TYR A 189     -24.831  -2.373  11.710  1.00  0.00           C  \\nATOM   2955  CG  TYR A 189     -25.470  -1.613  12.801  1.00  0.00           C  \\nATOM   2956  CD1 TYR A 189     -26.863  -1.477  12.882  1.00  0.00           C  \\nATOM   2957  CD2 TYR A 189     -24.700  -1.074  13.790  1.00  0.00           C  \\nATOM   2958  CE1 TYR A 189     -27.443  -0.727  13.958  1.00  0.00           C  \\nATOM   2959  CE2 TYR A 189     -25.260  -0.364  14.897  1.00  0.00           C  \\nATOM   2960  CZ  TYR A 189     -26.642  -0.226  14.944  1.00  0.00           C  \\nATOM   2961  OH  TYR A 189     -27.189   0.453  16.017  1.00  0.00           O  \\nATOM   2962  H   TYR A 189     -23.403  -4.155  13.054  1.00  0.00           H  \\nATOM   2963  HA  TYR A 189     -26.158  -4.077  11.966  1.00  0.00           H  \\nATOM   2964 1HB  TYR A 189     -23.757  -2.187  11.741  1.00  0.00           H  \\nATOM   2965 2HB  TYR A 189     -25.202  -1.992  10.759  1.00  0.00           H  \\nATOM   2966  HD1 TYR A 189     -27.501  -1.940  12.129  1.00  0.00           H  \\nATOM   2967  HD2 TYR A 189     -23.615  -1.176  13.753  1.00  0.00           H  \\nATOM   2968  HE1 TYR A 189     -28.525  -0.618  14.025  1.00  0.00           H  \\nATOM   2969  HE2 TYR A 189     -24.607   0.048  15.667  1.00  0.00           H  \\nATOM   2970  HH  TYR A 189     -26.486   0.744  16.603  1.00  0.00           H  \\nATOM   2971  N   ASN A 190     -24.037  -5.709  10.552  1.00  0.00           N  \\nATOM   2972  CA  ASN A 190     -23.642  -6.441   9.323  1.00  0.00           C  \\nATOM   2973  C   ASN A 190     -23.027  -5.571   8.260  1.00  0.00           C  \\nATOM   2974  O   ASN A 190     -23.512  -5.513   7.123  1.00  0.00           O  \\nATOM   2975  CB  ASN A 190     -24.832  -7.215   8.811  1.00  0.00           C  \\nATOM   2976  CG  ASN A 190     -25.071  -8.422   9.618  1.00  0.00           C  \\nATOM   2977  OD1 ASN A 190     -24.154  -9.240   9.858  1.00  0.00           O  \\nATOM   2978  ND2 ASN A 190     -26.278  -8.543  10.097  1.00  0.00           N  \\nATOM   2979  H   ASN A 190     -23.723  -6.049  11.450  1.00  0.00           H  \\nATOM   2980  HA  ASN A 190     -22.840  -7.136   9.577  1.00  0.00           H  \\nATOM   2981 1HB  ASN A 190     -25.718  -6.579   8.835  1.00  0.00           H  \\nATOM   2982 2HB  ASN A 190     -24.662  -7.499   7.772  1.00  0.00           H  \\nATOM   2983 1HD2 ASN A 190     -26.517  -9.338  10.656  1.00  0.00           H  \\nATOM   2984 2HD2 ASN A 190     -26.964  -7.842   9.906  1.00  0.00           H  \\nATOM   2985  N   ARG A 191     -22.102  -4.719   8.708  1.00  0.00           N  \\nATOM   2986  CA  ARG A 191     -21.497  -3.723   7.871  1.00  0.00           C  \\nATOM   2987  C   ARG A 191     -20.041  -3.990   7.759  1.00  0.00           C  \\nATOM   2988  O   ARG A 191     -19.400  -4.588   8.663  1.00  0.00           O  \\nATOM   2989  CB  ARG A 191     -21.701  -2.285   8.410  1.00  0.00           C  \\nATOM   2990  CG  ARG A 191     -23.084  -2.049   8.758  1.00  0.00           C  \\nATOM   2991  CD  ARG A 191     -23.497  -0.584   8.629  1.00  0.00           C  \\nATOM   2992  NE  ARG A 191     -22.417   0.364   8.942  1.00  0.00           N  \\nATOM   2993  CZ  ARG A 191     -22.507   1.415   9.783  1.00  0.00           C  \\nATOM   2994  NH1 ARG A 191     -23.656   1.702  10.427  1.00  0.00           N  \\nATOM   2995  NH2 ARG A 191     -21.445   2.202   9.927  1.00  0.00           N  \\nATOM   2996  H   ARG A 191     -21.822  -4.784   9.676  1.00  0.00           H  \\nATOM   2997  HA  ARG A 191     -21.959  -3.773   6.884  1.00  0.00           H  \\nATOM   2998 1HB  ARG A 191     -21.073  -2.133   9.287  1.00  0.00           H  \\nATOM   2999 2HB  ARG A 191     -21.387  -1.565   7.654  1.00  0.00           H  \\nATOM   3000 1HG  ARG A 191     -23.729  -2.631   8.100  1.00  0.00           H  \\nATOM   3001 2HG  ARG A 191     -23.255  -2.349   9.793  1.00  0.00           H  \\nATOM   3002 1HD  ARG A 191     -23.817  -0.385   7.607  1.00  0.00           H  \\nATOM   3003 2HD  ARG A 191     -24.319  -0.376   9.313  1.00  0.00           H  \\nATOM   3004  HE  ARG A 191     -21.523   0.223   8.491  1.00  0.00           H  \\nATOM   3005 1HH1 ARG A 191     -24.473   1.125  10.284  1.00  0.00           H  \\nATOM   3006 2HH1 ARG A 191     -23.700   2.494  11.052  1.00  0.00           H  \\nATOM   3007 1HH2 ARG A 191     -20.597   2.006   9.413  1.00  0.00           H  \\nATOM   3008 2HH2 ARG A 191     -21.485   2.995  10.550  1.00  0.00           H  \\nATOM   3009  N   ARG A 192     -19.469  -3.544   6.679  1.00  0.00           N  \\nATOM   3010  CA  ARG A 192     -18.039  -3.448   6.544  1.00  0.00           C  \\nATOM   3011  C   ARG A 192     -17.688  -2.023   6.226  1.00  0.00           C  \\nATOM   3012  O   ARG A 192     -18.384  -1.299   5.448  1.00  0.00           O  \\nATOM   3013  CB  ARG A 192     -17.547  -4.426   5.474  1.00  0.00           C  \\nATOM   3014  CG  ARG A 192     -18.155  -4.222   4.078  1.00  0.00           C  \\nATOM   3015  CD  ARG A 192     -17.828  -5.389   3.137  1.00  0.00           C  \\nATOM   3016  NE  ARG A 192     -16.444  -5.364   2.739  1.00  0.00           N  \\nATOM   3017  CZ  ARG A 192     -15.791  -6.420   2.228  1.00  0.00           C  \\nATOM   3018  NH1 ARG A 192     -16.406  -7.607   2.100  1.00  0.00           N  \\nATOM   3019  NH2 ARG A 192     -14.563  -6.294   1.910  1.00  0.00           N  \\nATOM   3020  H   ARG A 192     -20.056  -3.256   5.910  1.00  0.00           H  \\nATOM   3021  HA  ARG A 192     -17.583  -3.712   7.499  1.00  0.00           H  \\nATOM   3022 1HB  ARG A 192     -16.466  -4.343   5.375  1.00  0.00           H  \\nATOM   3023 2HB  ARG A 192     -17.770  -5.447   5.783  1.00  0.00           H  \\nATOM   3024 1HG  ARG A 192     -19.240  -4.145   4.161  1.00  0.00           H  \\nATOM   3025 2HG  ARG A 192     -17.759  -3.307   3.638  1.00  0.00           H  \\nATOM   3026 1HD  ARG A 192     -18.026  -6.332   3.645  1.00  0.00           H  \\nATOM   3027 2HD  ARG A 192     -18.447  -5.321   2.244  1.00  0.00           H  \\nATOM   3028  HE  ARG A 192     -15.937  -4.497   2.854  1.00  0.00           H  \\nATOM   3029 1HH1 ARG A 192     -17.368  -7.712   2.390  1.00  0.00           H  \\nATOM   3030 2HH1 ARG A 192     -15.906  -8.396   1.714  1.00  0.00           H  \\nATOM   3031 1HH2 ARG A 192     -14.094  -5.409   2.045  1.00  0.00           H  \\nATOM   3032 2HH2 ARG A 192     -14.059  -7.079   1.524  1.00  0.00           H  \\nATOM   3033  N   SER A 193     -16.627  -1.568   6.831  1.00  0.00           N  \\nATOM   3034  CA  SER A 193     -16.140  -0.212   6.599  1.00  0.00           C  \\nATOM   3035  C   SER A 193     -14.757   0.037   7.079  1.00  0.00           C  \\nATOM   3036  O   SER A 193     -14.301  -0.660   7.978  1.00  0.00           O  \\nATOM   3037  CB  SER A 193     -17.064   0.787   7.266  1.00  0.00           C  \\nATOM   3038  OG  SER A 193     -17.010   0.668   8.661  1.00  0.00           O  \\nATOM   3039  H   SER A 193     -16.129  -2.166   7.476  1.00  0.00           H  \\nATOM   3040  HA  SER A 193     -16.099  -0.039   5.523  1.00  0.00           H  \\nATOM   3041 1HB  SER A 193     -16.780   1.798   6.973  1.00  0.00           H  \\nATOM   3042 2HB  SER A 193     -18.085   0.623   6.924  1.00  0.00           H  \\nATOM   3043  HG  SER A 193     -16.795   1.543   8.993  1.00  0.00           H  \\nATOM   3044  N   ALA A 194     -14.107   1.041   6.544  1.00  0.00           N  \\nATOM   3045  CA  ALA A 194     -12.776   1.356   6.970  1.00  0.00           C  \\nATOM   3046  C   ALA A 194     -12.773   1.933   8.395  1.00  0.00           C  \\nATOM   3047  O   ALA A 194     -13.816   2.505   8.751  1.00  0.00           O  \\nATOM   3048  CB  ALA A 194     -12.156   2.339   5.962  1.00  0.00           C  \\nATOM   3049  H   ALA A 194     -14.539   1.602   5.824  1.00  0.00           H  \\nATOM   3050  HA  ALA A 194     -12.198   0.432   6.986  1.00  0.00           H  \\nATOM   3051 1HB  ALA A 194     -11.142   2.588   6.274  1.00  0.00           H  \\nATOM   3052 2HB  ALA A 194     -12.129   1.878   4.974  1.00  0.00           H  \\nATOM   3053 3HB  ALA A 194     -12.757   3.246   5.923  1.00  0.00           H  \\nATOM   3054  N   PRO A 195     -11.558   1.878   9.191  1.00  0.00           N  \\nATOM   3055  CA  PRO A 195     -11.440   2.600  10.459  1.00  0.00           C  \\nATOM   3056  C   PRO A 195     -11.449   4.065  10.265  1.00  0.00           C  \\nATOM   3057  O   PRO A 195     -10.931   4.526   9.269  1.00  0.00           O  \\nATOM   3058  CB  PRO A 195     -10.032   2.247  10.925  1.00  0.00           C  \\nATOM   3059  CG  PRO A 195      -9.333   1.749   9.774  1.00  0.00           C  \\nATOM   3060  CD  PRO A 195     -10.331   1.156   8.877  1.00  0.00           C  \\nATOM   3061  HA  PRO A 195     -12.195   2.220  11.163  1.00  0.00           H  \\nATOM   3062 1HB  PRO A 195      -9.541   3.136  11.346  1.00  0.00           H  \\nATOM   3063 2HB  PRO A 195     -10.079   1.495  11.727  1.00  0.00           H  \\nATOM   3064 1HG  PRO A 195      -8.789   2.567   9.277  1.00  0.00           H  \\nATOM   3065 2HG  PRO A 195      -8.580   1.006  10.078  1.00  0.00           H  \\nATOM   3066 1HD  PRO A 195     -10.028   1.318   7.832  1.00  0.00           H  \\nATOM   3067 2HD  PRO A 195     -10.424   0.081   9.092  1.00  0.00           H  \\nATOM   3068  N   ASP A 196     -11.956   4.807  11.256  1.00  0.00           N  \\nATOM   3069  CA  ASP A 196     -11.955   6.268  11.130  1.00  0.00           C  \\nATOM   3070  C   ASP A 196     -10.672   6.856  11.714  1.00  0.00           C  \\nATOM   3071  O   ASP A 196      -9.906   6.117  12.315  1.00  0.00           O  \\nATOM   3072  CB  ASP A 196     -13.269   6.866  11.692  1.00  0.00           C  \\nATOM   3073  CG  ASP A 196     -13.435   6.688  13.191  1.00  0.00           C  \\nATOM   3074  OD1 ASP A 196     -12.484   6.509  13.912  1.00  0.00           O  \\nATOM   3075  OD2 ASP A 196     -14.596   6.761  13.650  1.00  0.00           O  \\nATOM   3076  H   ASP A 196     -12.339   4.386  12.090  1.00  0.00           H  \\nATOM   3077  HA  ASP A 196     -11.883   6.523  10.072  1.00  0.00           H  \\nATOM   3078 1HB  ASP A 196     -13.306   7.933  11.471  1.00  0.00           H  \\nATOM   3079 2HB  ASP A 196     -14.122   6.400  11.198  1.00  0.00           H  \\nATOM   3080  N   PRO A 197     -10.409   8.181  11.561  1.00  0.00           N  \\nATOM   3081  CA  PRO A 197      -9.186   8.701  12.112  1.00  0.00           C  \\nATOM   3082  C   PRO A 197      -9.011   8.425  13.564  1.00  0.00           C  \\nATOM   3083  O   PRO A 197      -7.865   8.483  14.037  1.00  0.00           O  \\nATOM   3084  CB  PRO A 197      -9.319  10.205  11.854  1.00  0.00           C  \\nATOM   3085  CG  PRO A 197     -10.167  10.292  10.632  1.00  0.00           C  \\nATOM   3086  CD  PRO A 197     -11.173   9.183  10.794  1.00  0.00           C  \\nATOM   3087  HA  PRO A 197      -8.336   8.326  11.524  1.00  0.00           H  \\nATOM   3088 1HB  PRO A 197      -9.774  10.698  12.726  1.00  0.00           H  \\nATOM   3089 2HB  PRO A 197      -8.324  10.652  11.716  1.00  0.00           H  \\nATOM   3090 1HG  PRO A 197     -10.638  11.284  10.566  1.00  0.00           H  \\nATOM   3091 2HG  PRO A 197      -9.548  10.174   9.730  1.00  0.00           H  \\nATOM   3092 1HD  PRO A 197     -12.044   9.556  11.352  1.00  0.00           H  \\nATOM   3093 2HD  PRO A 197     -11.476   8.815   9.802  1.00  0.00           H  \\nATOM   3094  N   SER A 198     -10.103   8.380  14.357  1.00  0.00           N  \\nATOM   3095  CA  SER A 198      -9.953   8.129  15.741  1.00  0.00           C  \\nATOM   3096  C   SER A 198      -9.289   6.807  15.997  1.00  0.00           C  \\nATOM   3097  O   SER A 198      -8.423   6.687  16.852  1.00  0.00           O  \\nATOM   3098  CB  SER A 198     -11.300   8.253  16.524  1.00  0.00           C  \\nATOM   3099  OG  SER A 198     -12.134   7.187  16.168  1.00  0.00           O  \\nATOM   3100  H   SER A 198     -11.029   8.523  13.980  1.00  0.00           H  \\nATOM   3101  HA  SER A 198      -9.263   8.866  16.154  1.00  0.00           H  \\nATOM   3102 1HB  SER A 198     -11.100   8.244  17.595  1.00  0.00           H  \\nATOM   3103 2HB  SER A 198     -11.772   9.206  16.287  1.00  0.00           H  \\nATOM   3104  HG  SER A 198     -11.640   6.666  15.531  1.00  0.00           H  \\nATOM   3105  N   VAL A 199      -9.690   5.823  15.180  1.00  0.00           N  \\nATOM   3106  CA  VAL A 199      -9.118   4.462  15.352  1.00  0.00           C  \\nATOM   3107  C   VAL A 199      -7.691   4.388  14.958  1.00  0.00           C  \\nATOM   3108  O   VAL A 199      -6.842   3.880  15.688  1.00  0.00           O  \\nATOM   3109  CB  VAL A 199      -9.894   3.490  14.500  1.00  0.00           C  \\nATOM   3110  CG1 VAL A 199      -9.254   2.121  14.579  1.00  0.00           C  \\nATOM   3111  CG2 VAL A 199     -11.374   3.493  14.941  1.00  0.00           C  \\nATOM   3112  H   VAL A 199     -10.370   5.982  14.451  1.00  0.00           H  \\nATOM   3113  HA  VAL A 199      -9.212   4.177  16.401  1.00  0.00           H  \\nATOM   3114  HB  VAL A 199      -9.819   3.796  13.456  1.00  0.00           H  \\nATOM   3115 1HG1 VAL A 199      -9.817   1.420  13.962  1.00  0.00           H  \\nATOM   3116 2HG1 VAL A 199      -8.227   2.178  14.218  1.00  0.00           H  \\nATOM   3117 3HG1 VAL A 199      -9.258   1.777  15.613  1.00  0.00           H  \\nATOM   3118 1HG2 VAL A 199     -11.938   2.792  14.327  1.00  0.00           H  \\nATOM   3119 2HG2 VAL A 199     -11.442   3.196  15.988  1.00  0.00           H  \\nATOM   3120 3HG2 VAL A 199     -11.788   4.494  14.821  1.00  0.00           H  \\nATOM   3121  N   ILE A 200      -7.362   5.019  13.810  1.00  0.00           N  \\nATOM   3122  CA  ILE A 200      -6.050   5.007  13.280  1.00  0.00           C  \\nATOM   3123  C   ILE A 200      -5.057   5.710  14.220  1.00  0.00           C  \\nATOM   3124  O   ILE A 200      -3.963   5.190  14.499  1.00  0.00           O  \\nATOM   3125  CB  ILE A 200      -6.057   5.601  11.822  1.00  0.00           C  \\nATOM   3126  CG1 ILE A 200      -6.799   4.583  10.954  1.00  0.00           C  \\nATOM   3127  CG2 ILE A 200      -4.639   5.828  11.371  1.00  0.00           C  \\nATOM   3128  CD1 ILE A 200      -7.417   5.244   9.675  1.00  0.00           C  \\nATOM   3129  H   ILE A 200      -8.088   5.518  13.315  1.00  0.00           H  \\nATOM   3130  HA  ILE A 200      -5.703   3.975  13.240  1.00  0.00           H  \\nATOM   3131  HB  ILE A 200      -6.597   6.547  11.817  1.00  0.00           H  \\nATOM   3132 1HG1 ILE A 200      -6.112   3.792  10.654  1.00  0.00           H  \\nATOM   3133 2HG1 ILE A 200      -7.595   4.119  11.537  1.00  0.00           H  \\nATOM   3134 1HG2 ILE A 200      -4.640   6.239  10.362  1.00  0.00           H  \\nATOM   3135 2HG2 ILE A 200      -4.150   6.528  12.047  1.00  0.00           H  \\nATOM   3136 3HG2 ILE A 200      -4.099   4.881  11.377  1.00  0.00           H  \\nATOM   3137 1HD1 ILE A 200      -7.935   4.486   9.088  1.00  0.00           H  \\nATOM   3138 2HD1 ILE A 200      -8.125   6.019   9.972  1.00  0.00           H  \\nATOM   3139 3HD1 ILE A 200      -6.623   5.687   9.075  1.00  0.00           H  \\nATOM   3140  N   THR A 201      -5.420   6.909  14.677  1.00  0.00           N  \\nATOM   3141  CA  THR A 201      -4.617   7.625  15.632  1.00  0.00           C  \\nATOM   3142  C   THR A 201      -4.393   6.889  17.010  1.00  0.00           C  \\nATOM   3143  O   THR A 201      -3.272   6.870  17.573  1.00  0.00           O  \\nATOM   3144  CB  THR A 201      -5.121   9.060  15.834  1.00  0.00           C  \\nATOM   3145  OG1 THR A 201      -4.865   9.712  14.577  1.00  0.00           O  \\nATOM   3146  CG2 THR A 201      -4.355   9.711  16.860  1.00  0.00           C  \\nATOM   3147  H   THR A 201      -6.279   7.325  14.346  1.00  0.00           H  \\nATOM   3148  HA  THR A 201      -3.595   7.676  15.256  1.00  0.00           H  \\nATOM   3149  HB  THR A 201      -6.171   9.039  16.124  1.00  0.00           H  \\nATOM   3150  HG1 THR A 201      -4.459   9.088  13.970  1.00  0.00           H  \\nATOM   3151 1HG2 THR A 201      -4.720  10.729  16.996  1.00  0.00           H  \\nATOM   3152 2HG2 THR A 201      -4.457   9.160  17.795  1.00  0.00           H  \\nATOM   3153 3HG2 THR A 201      -3.306   9.739  16.568  1.00  0.00           H  \\nATOM   3154  N   LEU A 202      -5.408   6.189  17.533  1.00  0.00           N  \\nATOM   3155  CA  LEU A 202      -5.160   5.366  18.694  1.00  0.00           C  \\nATOM   3156  C   LEU A 202      -4.181   4.289  18.411  1.00  0.00           C  \\nATOM   3157  O   LEU A 202      -3.282   4.059  19.181  1.00  0.00           O  \\nATOM   3158  CB  LEU A 202      -6.467   4.748  19.193  1.00  0.00           C  \\nATOM   3159  CG  LEU A 202      -7.338   5.638  20.089  1.00  0.00           C  \\nATOM   3160  CD1 LEU A 202      -8.631   4.951  20.266  1.00  0.00           C  \\nATOM   3161  CD2 LEU A 202      -6.708   5.761  21.453  1.00  0.00           C  \\nATOM   3162  H   LEU A 202      -6.338   6.219  17.141  1.00  0.00           H  \\nATOM   3163  HA  LEU A 202      -4.747   5.995  19.482  1.00  0.00           H  \\nATOM   3164 1HB  LEU A 202      -7.069   4.465  18.331  1.00  0.00           H  \\nATOM   3165 2HB  LEU A 202      -6.232   3.846  19.758  1.00  0.00           H  \\nATOM   3166  HG  LEU A 202      -7.426   6.629  19.642  1.00  0.00           H  \\nATOM   3167 1HD1 LEU A 202      -9.280   5.555  20.900  1.00  0.00           H  \\nATOM   3168 2HD1 LEU A 202      -9.104   4.809  19.294  1.00  0.00           H  \\nATOM   3169 3HD1 LEU A 202      -8.467   3.982  20.735  1.00  0.00           H  \\nATOM   3170 1HD2 LEU A 202      -7.330   6.395  22.086  1.00  0.00           H  \\nATOM   3171 2HD2 LEU A 202      -6.621   4.772  21.904  1.00  0.00           H  \\nATOM   3172 3HD2 LEU A 202      -5.717   6.205  21.357  1.00  0.00           H  \\nATOM   3173  N   GLU A 203      -4.364   3.516  17.318  1.00  0.00           N  \\nATOM   3174  CA  GLU A 203      -3.372   2.493  16.977  1.00  0.00           C  \\nATOM   3175  C   GLU A 203      -1.962   3.066  16.930  1.00  0.00           C  \\nATOM   3176  O   GLU A 203      -1.030   2.446  17.386  1.00  0.00           O  \\nATOM   3177  CB  GLU A 203      -3.704   1.871  15.578  1.00  0.00           C  \\nATOM   3178  CG  GLU A 203      -5.090   1.198  15.526  1.00  0.00           C  \\nATOM   3179  CD  GLU A 203      -5.532   0.785  14.153  1.00  0.00           C  \\nATOM   3180  OE1 GLU A 203      -4.925   1.221  13.173  1.00  0.00           O  \\nATOM   3181  OE2 GLU A 203      -6.521   0.076  14.033  1.00  0.00           O  \\nATOM   3182  H   GLU A 203      -5.176   3.630  16.728  1.00  0.00           H  \\nATOM   3183  HA  GLU A 203      -3.412   1.707  17.732  1.00  0.00           H  \\nATOM   3184 1HB  GLU A 203      -3.669   2.650  14.816  1.00  0.00           H  \\nATOM   3185 2HB  GLU A 203      -2.949   1.128  15.322  1.00  0.00           H  \\nATOM   3186 1HG  GLU A 203      -5.074   0.309  16.156  1.00  0.00           H  \\nATOM   3187 2HG  GLU A 203      -5.831   1.885  15.933  1.00  0.00           H  \\nATOM   3188  N   ASN A 204      -1.815   4.191  16.254  1.00  0.00           N  \\nATOM   3189  CA  ASN A 204      -0.501   4.788  16.166  1.00  0.00           C  \\nATOM   3190  C   ASN A 204       0.082   5.240  17.496  1.00  0.00           C  \\nATOM   3191  O   ASN A 204       1.333   5.348  17.580  1.00  0.00           O  \\nATOM   3192  CB  ASN A 204      -0.544   6.014  15.231  1.00  0.00           C  \\nATOM   3193  CG  ASN A 204      -0.772   5.630  13.738  1.00  0.00           C  \\nATOM   3194  OD1 ASN A 204      -0.599   4.459  13.361  1.00  0.00           O  \\nATOM   3195  ND2 ASN A 204      -1.184   6.574  12.918  1.00  0.00           N  \\nATOM   3196  H   ASN A 204      -2.593   4.645  15.798  1.00  0.00           H  \\nATOM   3197  HA  ASN A 204       0.187   4.050  15.751  1.00  0.00           H  \\nATOM   3198 1HB  ASN A 204      -1.345   6.684  15.545  1.00  0.00           H  \\nATOM   3199 2HB  ASN A 204       0.393   6.565  15.310  1.00  0.00           H  \\nATOM   3200 1HD2 ASN A 204      -1.341   6.364  11.952  1.00  0.00           H  \\nATOM   3201 2HD2 ASN A 204      -1.340   7.500  13.260  1.00  0.00           H  \\nATOM   3202  N   SER A 205      -0.748   5.438  18.516  1.00  0.00           N  \\nATOM   3203  CA  SER A 205      -0.313   6.073  19.740  1.00  0.00           C  \\nATOM   3204  C   SER A 205      -0.235   5.075  20.899  1.00  0.00           C  \\nATOM   3205  O   SER A 205       0.110   5.475  22.029  1.00  0.00           O  \\nATOM   3206  CB  SER A 205      -1.303   7.130  20.155  1.00  0.00           C  \\nATOM   3207  OG  SER A 205      -1.504   8.132  19.115  1.00  0.00           O  \\nATOM   3208  H   SER A 205      -1.708   5.136  18.433  1.00  0.00           H  \\nATOM   3209  HA  SER A 205       0.653   6.547  19.560  1.00  0.00           H  \\nATOM   3210 1HB  SER A 205      -2.259   6.662  20.388  1.00  0.00           H  \\nATOM   3211 2HB  SER A 205      -0.951   7.622  21.060  1.00  0.00           H  \\nATOM   3212  HG  SER A 205      -0.938   7.873  18.384  1.00  0.00           H  \\nATOM   3213  N   TRP A 206      -0.465   3.803  20.615  1.00  0.00           N  \\nATOM   3214  CA  TRP A 206      -0.550   2.848  21.726  1.00  0.00           C  \\nATOM   3215  C   TRP A 206       0.694   2.801  22.528  1.00  0.00           C  \\nATOM   3216  O   TRP A 206       0.659   2.753  23.741  1.00  0.00           O  \\nATOM   3217  CB  TRP A 206      -0.773   1.452  21.130  1.00  0.00           C  \\nATOM   3218  CG  TRP A 206      -0.969   0.391  22.196  1.00  0.00           C  \\nATOM   3219  CD1 TRP A 206      -2.042   0.279  23.047  1.00  0.00           C  \\nATOM   3220  CD2 TRP A 206      -0.089  -0.653  22.521  1.00  0.00           C  \\nATOM   3221  NE1 TRP A 206      -1.876  -0.782  23.901  1.00  0.00           N  \\nATOM   3222  CE2 TRP A 206      -0.681  -1.392  23.558  1.00  0.00           C  \\nATOM   3223  CE3 TRP A 206       1.134  -1.061  22.017  1.00  0.00           C  \\nATOM   3224  CZ2 TRP A 206      -0.072  -2.513  24.110  1.00  0.00           C  \\nATOM   3225  CZ3 TRP A 206       1.744  -2.214  22.606  1.00  0.00           C  \\nATOM   3226  CH2 TRP A 206       1.128  -2.916  23.593  1.00  0.00           C  \\nATOM   3227  H   TRP A 206      -0.583   3.469  19.669  1.00  0.00           H  \\nATOM   3228  HA  TRP A 206      -1.397   3.123  22.354  1.00  0.00           H  \\nATOM   3229 1HB  TRP A 206      -1.650   1.468  20.483  1.00  0.00           H  \\nATOM   3230 2HB  TRP A 206       0.083   1.179  20.514  1.00  0.00           H  \\nATOM   3231  HD1 TRP A 206      -2.908   0.939  23.048  1.00  0.00           H  \\nATOM   3232  HE1 TRP A 206      -2.515  -1.052  24.635  1.00  0.00           H  \\nATOM   3233  HE3 TRP A 206       1.621  -0.531  21.199  1.00  0.00           H  \\nATOM   3234  HZ2 TRP A 206      -0.522  -3.071  24.931  1.00  0.00           H  \\nATOM   3235  HZ3 TRP A 206       2.725  -2.515  22.237  1.00  0.00           H  \\nATOM   3236  HH2 TRP A 206       1.602  -3.818  23.980  1.00  0.00           H  \\nATOM   3237  N   GLY A 207       1.861   2.752  21.896  1.00  0.00           N  \\nATOM   3238  CA  GLY A 207       3.094   2.719  22.673  1.00  0.00           C  \\nATOM   3239  C   GLY A 207       3.293   3.977  23.523  1.00  0.00           C  \\nATOM   3240  O   GLY A 207       3.689   3.901  24.681  1.00  0.00           O  \\nATOM   3241  H   GLY A 207       1.914   2.737  20.888  1.00  0.00           H  \\nATOM   3242 1HA  GLY A 207       3.088   1.848  23.328  1.00  0.00           H  \\nATOM   3243 2HA  GLY A 207       3.944   2.609  22.001  1.00  0.00           H  \\nATOM   3244  N   ARG A 208       3.015   5.082  22.912  1.00  0.00           N  \\nATOM   3245  CA  ARG A 208       3.173   6.406  23.616  1.00  0.00           C  \\nATOM   3246  C   ARG A 208       2.164   6.537  24.730  1.00  0.00           C  \\nATOM   3247  O   ARG A 208       2.486   7.092  25.757  1.00  0.00           O  \\nATOM   3248  CB  ARG A 208       3.077   7.555  22.646  1.00  0.00           C  \\nATOM   3249  CG  ARG A 208       4.345   7.702  21.869  1.00  0.00           C  \\nATOM   3250  CD  ARG A 208       4.283   8.893  20.809  1.00  0.00           C  \\nATOM   3251  NE  ARG A 208       3.114   8.961  19.907  1.00  0.00           N  \\nATOM   3252  CZ  ARG A 208       2.919   8.257  18.793  1.00  0.00           C  \\nATOM   3253  NH1 ARG A 208       3.794   7.331  18.387  1.00  0.00           N  \\nATOM   3254  NH2 ARG A 208       1.828   8.504  18.025  1.00  0.00           N  \\nATOM   3255  H   ARG A 208       2.689   5.069  21.956  1.00  0.00           H  \\nATOM   3256  HA  ARG A 208       4.158   6.434  24.083  1.00  0.00           H  \\nATOM   3257 1HB  ARG A 208       2.245   7.386  21.964  1.00  0.00           H  \\nATOM   3258 2HB  ARG A 208       2.871   8.476  23.192  1.00  0.00           H  \\nATOM   3259 1HG  ARG A 208       5.169   7.906  22.553  1.00  0.00           H  \\nATOM   3260 2HG  ARG A 208       4.547   6.780  21.323  1.00  0.00           H  \\nATOM   3261 1HD  ARG A 208       4.290   9.848  21.334  1.00  0.00           H  \\nATOM   3262 2HD  ARG A 208       5.147   8.836  20.148  1.00  0.00           H  \\nATOM   3263  HE  ARG A 208       2.373   9.606  20.146  1.00  0.00           H  \\nATOM   3264 1HH1 ARG A 208       4.627   7.151  18.929  1.00  0.00           H  \\nATOM   3265 2HH1 ARG A 208       3.622   6.811  17.539  1.00  0.00           H  \\nATOM   3266 1HH2 ARG A 208       1.168   9.219  18.299  1.00  0.00           H  \\nATOM   3267 2HH2 ARG A 208       1.675   7.974  17.180  1.00  0.00           H  \\nATOM   3268  N   LEU A 209       0.966   6.000  24.570  1.00  0.00           N  \\nATOM   3269  CA  LEU A 209      -0.045   5.975  25.647  1.00  0.00           C  \\nATOM   3270  C   LEU A 209       0.424   5.149  26.775  1.00  0.00           C  \\nATOM   3271  O   LEU A 209       0.221   5.519  27.967  1.00  0.00           O  \\nATOM   3272  CB  LEU A 209      -1.438   5.528  25.112  1.00  0.00           C  \\nATOM   3273  CG  LEU A 209      -2.094   6.517  24.171  1.00  0.00           C  \\nATOM   3274  CD1 LEU A 209      -3.348   5.945  23.367  1.00  0.00           C  \\nATOM   3275  CD2 LEU A 209      -2.520   7.733  24.942  1.00  0.00           C  \\nATOM   3276  H   LEU A 209       0.740   5.593  23.674  1.00  0.00           H  \\nATOM   3277  HA  LEU A 209      -0.144   6.982  26.052  1.00  0.00           H  \\nATOM   3278 1HB  LEU A 209      -1.321   4.581  24.588  1.00  0.00           H  \\nATOM   3279 2HB  LEU A 209      -2.102   5.370  25.962  1.00  0.00           H  \\nATOM   3280  HG  LEU A 209      -1.384   6.808  23.396  1.00  0.00           H  \\nATOM   3281 1HD1 LEU A 209      -3.751   6.724  22.719  1.00  0.00           H  \\nATOM   3282 2HD1 LEU A 209      -3.034   5.096  22.760  1.00  0.00           H  \\nATOM   3283 3HD1 LEU A 209      -4.116   5.625  24.070  1.00  0.00           H  \\nATOM   3284 1HD2 LEU A 209      -2.992   8.446  24.265  1.00  0.00           H  \\nATOM   3285 2HD2 LEU A 209      -3.231   7.443  25.716  1.00  0.00           H  \\nATOM   3286 3HD2 LEU A 209      -1.648   8.194  25.405  1.00  0.00           H  \\nATOM   3287  N   SER A 210       0.968   4.003  26.482  1.00  0.00           N  \\nATOM   3288  CA  SER A 210       1.488   3.099  27.493  1.00  0.00           C  \\nATOM   3289  C   SER A 210       2.551   3.786  28.331  1.00  0.00           C  \\nATOM   3290  O   SER A 210       2.539   3.681  29.534  1.00  0.00           O  \\nATOM   3291  CB  SER A 210       2.070   1.814  26.801  1.00  0.00           C  \\nATOM   3292  OG  SER A 210       1.030   1.061  26.186  1.00  0.00           O  \\nATOM   3293  H   SER A 210       1.027   3.739  25.509  1.00  0.00           H  \\nATOM   3294  HA  SER A 210       0.669   2.809  28.152  1.00  0.00           H  \\nATOM   3295 1HB  SER A 210       2.807   2.106  26.053  1.00  0.00           H  \\nATOM   3296 2HB  SER A 210       2.581   1.202  27.543  1.00  0.00           H  \\nATOM   3297  HG  SER A 210       0.218   1.544  26.356  1.00  0.00           H  \\nATOM   3298  N   THR A 211       3.431   4.500  27.678  1.00  0.00           N  \\nATOM   3299  CA  THR A 211       4.469   5.189  28.392  1.00  0.00           C  \\nATOM   3300  C   THR A 211       3.854   6.389  29.197  1.00  0.00           C  \\nATOM   3301  O   THR A 211       4.174   6.483  30.435  1.00  0.00           O  \\nATOM   3302  CB  THR A 211       5.495   5.754  27.425  1.00  0.00           C  \\nATOM   3303  OG1 THR A 211       6.101   4.672  26.717  1.00  0.00           O  \\nATOM   3304  CG2 THR A 211       6.534   6.474  28.184  1.00  0.00           C  \\nATOM   3305  H   THR A 211       3.393   4.576  26.672  1.00  0.00           H  \\nATOM   3306  HA  THR A 211       4.967   4.477  29.051  1.00  0.00           H  \\nATOM   3307  HB  THR A 211       5.004   6.437  26.732  1.00  0.00           H  \\nATOM   3308  HG1 THR A 211       5.721   3.842  27.015  1.00  0.00           H  \\nATOM   3309 1HG2 THR A 211       7.272   6.881  27.494  1.00  0.00           H  \\nATOM   3310 2HG2 THR A 211       6.076   7.287  28.747  1.00  0.00           H  \\nATOM   3311 3HG2 THR A 211       7.023   5.787  28.873  1.00  0.00           H  \\nATOM   3312  N   ALA A 212       2.988   7.171  28.570  1.00  0.00           N  \\nATOM   3313  CA  ALA A 212       2.441   8.336  29.311  1.00  0.00           C  \\nATOM   3314  C   ALA A 212       1.685   7.856  30.555  1.00  0.00           C  \\nATOM   3315  O   ALA A 212       1.827   8.451  31.645  1.00  0.00           O  \\nATOM   3316  CB  ALA A 212       1.536   9.128  28.441  1.00  0.00           C  \\nATOM   3317  H   ALA A 212       2.689   7.008  27.619  1.00  0.00           H  \\nATOM   3318  HA  ALA A 212       3.277   8.966  29.616  1.00  0.00           H  \\nATOM   3319 1HB  ALA A 212       1.145   9.978  29.001  1.00  0.00           H  \\nATOM   3320 2HB  ALA A 212       2.089   9.489  27.573  1.00  0.00           H  \\nATOM   3321 3HB  ALA A 212       0.710   8.501  28.109  1.00  0.00           H  \\nATOM   3322  N   ILE A 213       0.920   6.774  30.497  1.00  0.00           N  \\nATOM   3323  CA  ILE A 213       0.206   6.329  31.660  1.00  0.00           C  \\nATOM   3324  C   ILE A 213       1.104   5.790  32.684  1.00  0.00           C  \\nATOM   3325  O   ILE A 213       0.983   6.142  33.882  1.00  0.00           O  \\nATOM   3326  CB  ILE A 213      -0.829   5.237  31.241  1.00  0.00           C  \\nATOM   3327  CG1 ILE A 213      -1.928   5.918  30.416  1.00  0.00           C  \\nATOM   3328  CG2 ILE A 213      -1.417   4.485  32.442  1.00  0.00           C  \\nATOM   3329  CD1 ILE A 213      -2.787   4.921  29.593  1.00  0.00           C  \\nATOM   3330  H   ILE A 213       0.832   6.253  29.636  1.00  0.00           H  \\nATOM   3331  HA  ILE A 213      -0.323   7.180  32.088  1.00  0.00           H  \\nATOM   3332  HB  ILE A 213      -0.345   4.506  30.595  1.00  0.00           H  \\nATOM   3333 1HG1 ILE A 213      -2.587   6.477  31.079  1.00  0.00           H  \\nATOM   3334 2HG1 ILE A 213      -1.476   6.633  29.728  1.00  0.00           H  \\nATOM   3335 1HG2 ILE A 213      -2.130   3.739  32.091  1.00  0.00           H  \\nATOM   3336 2HG2 ILE A 213      -0.616   3.991  32.989  1.00  0.00           H  \\nATOM   3337 3HG2 ILE A 213      -1.924   5.191  33.100  1.00  0.00           H  \\nATOM   3338 1HD1 ILE A 213      -3.545   5.470  29.033  1.00  0.00           H  \\nATOM   3339 2HD1 ILE A 213      -2.147   4.375  28.899  1.00  0.00           H  \\nATOM   3340 3HD1 ILE A 213      -3.274   4.218  30.267  1.00  0.00           H  \\nATOM   3341  N   GLN A 214       2.011   4.890  32.333  1.00  0.00           N  \\nATOM   3342  CA  GLN A 214       2.940   4.326  33.324  1.00  0.00           C  \\nATOM   3343  C   GLN A 214       3.948   5.302  33.921  1.00  0.00           C  \\nATOM   3344  O   GLN A 214       4.302   5.079  35.101  1.00  0.00           O  \\nATOM   3345  CB  GLN A 214       3.655   3.099  32.676  1.00  0.00           C  \\nATOM   3346  CG  GLN A 214       2.737   1.944  32.650  1.00  0.00           C  \\nATOM   3347  CD  GLN A 214       3.379   0.833  31.816  1.00  0.00           C  \\nATOM   3348  OE1 GLN A 214       4.263   0.155  32.270  1.00  0.00           O  \\nATOM   3349  NE2 GLN A 214       2.877   0.635  30.609  1.00  0.00           N  \\nATOM   3350  H   GLN A 214       2.070   4.583  31.373  1.00  0.00           H  \\nATOM   3351  HA  GLN A 214       2.365   4.002  34.192  1.00  0.00           H  \\nATOM   3352 1HB  GLN A 214       3.970   3.354  31.664  1.00  0.00           H  \\nATOM   3353 2HB  GLN A 214       4.551   2.857  33.247  1.00  0.00           H  \\nATOM   3354 1HG  GLN A 214       2.572   1.601  33.671  1.00  0.00           H  \\nATOM   3355 2HG  GLN A 214       1.793   2.254  32.203  1.00  0.00           H  \\nATOM   3356 1HE2 GLN A 214       3.258  -0.078  30.019  1.00  0.00           H  \\nATOM   3357 2HE2 GLN A 214       2.117   1.198  30.285  1.00  0.00           H  \\nATOM   3358  N   GLU A 215       4.329   6.374  33.229  1.00  0.00           N  \\nATOM   3359  CA  GLU A 215       5.269   7.335  33.838  1.00  0.00           C  \\nATOM   3360  C   GLU A 215       4.540   8.523  34.355  1.00  0.00           C  \\nATOM   3361  O   GLU A 215       5.182   9.510  34.790  1.00  0.00           O  \\nATOM   3362  CB  GLU A 215       6.302   7.789  32.803  1.00  0.00           C  \\nATOM   3363  CG  GLU A 215       6.906   6.628  32.090  1.00  0.00           C  \\nATOM   3364  CD  GLU A 215       8.036   6.026  32.816  1.00  0.00           C  \\nATOM   3365  OE1 GLU A 215       9.042   6.771  33.104  1.00  0.00           O  \\nATOM   3366  OE2 GLU A 215       7.939   4.795  33.038  1.00  0.00           O  \\nATOM   3367  H   GLU A 215       3.988   6.545  32.294  1.00  0.00           H  \\nATOM   3368  HA  GLU A 215       5.788   6.839  34.659  1.00  0.00           H  \\nATOM   3369 1HB  GLU A 215       5.826   8.451  32.079  1.00  0.00           H  \\nATOM   3370 2HB  GLU A 215       7.088   8.358  33.299  1.00  0.00           H  \\nATOM   3371 1HG  GLU A 215       6.140   5.867  31.943  1.00  0.00           H  \\nATOM   3372 2HG  GLU A 215       7.247   6.955  31.109  1.00  0.00           H  \\nATOM   3373  N   SER A 216       3.251   8.439  34.415  1.00  0.00           N  \\nATOM   3374  CA  SER A 216       2.457   9.569  34.947  1.00  0.00           C  \\nATOM   3375  C   SER A 216       2.671   9.779  36.456  1.00  0.00           C  \\nATOM   3376  O   SER A 216       3.124   8.920  37.178  1.00  0.00           O  \\nATOM   3377  CB  SER A 216       0.969   9.352  34.687  1.00  0.00           C  \\nATOM   3378  OG  SER A 216       0.425   8.375  35.426  1.00  0.00           O  \\nATOM   3379  H   SER A 216       2.773   7.606  34.102  1.00  0.00           H  \\nATOM   3380  HA  SER A 216       2.769  10.482  34.437  1.00  0.00           H  \\nATOM   3381 1HB  SER A 216       0.426  10.274  34.892  1.00  0.00           H  \\nATOM   3382 2HB  SER A 216       0.815   9.108  33.637  1.00  0.00           H  \\nATOM   3383  HG  SER A 216       1.136   8.027  35.970  1.00  0.00           H  \\nATOM   3384  N   ASN A 217       2.271  11.031  36.804  1.00  0.00           N  \\nATOM   3385  CA  ASN A 217       2.351  11.455  38.222  1.00  0.00           C  \\nATOM   3386  C   ASN A 217       0.999  11.194  38.849  1.00  0.00           C  \\nATOM   3387  O   ASN A 217       0.087  12.023  38.762  1.00  0.00           O  \\nATOM   3388  CB  ASN A 217       2.630  12.936  38.169  1.00  0.00           C  \\nATOM   3389  CG  ASN A 217       2.668  13.587  39.498  1.00  0.00           C  \\nATOM   3390  OD1 ASN A 217       2.729  12.938  40.522  1.00  0.00           O  \\nATOM   3391  ND2 ASN A 217       2.516  14.900  39.469  1.00  0.00           N  \\nATOM   3392  H   ASN A 217       1.919  11.681  36.115  1.00  0.00           H  \\nATOM   3393  HA  ASN A 217       3.170  10.915  38.699  1.00  0.00           H  \\nATOM   3394 1HB  ASN A 217       3.589  13.109  37.679  1.00  0.00           H  \\nATOM   3395 2HB  ASN A 217       1.864  13.430  37.571  1.00  0.00           H  \\nATOM   3396 1HD2 ASN A 217       2.528  15.424  40.321  1.00  0.00           H  \\nATOM   3397 2HD2 ASN A 217       2.389  15.369  38.596  1.00  0.00           H  \\nATOM   3398  N   GLN A 218       0.832   9.996  39.356  1.00  0.00           N  \\nATOM   3399  CA  GLN A 218      -0.428   9.534  39.885  1.00  0.00           C  \\nATOM   3400  C   GLN A 218      -1.557   9.811  38.975  1.00  0.00           C  \\nATOM   3401  O   GLN A 218      -2.630  10.212  39.329  1.00  0.00           O  \\nATOM   3402  CB  GLN A 218      -0.717  10.185  41.281  1.00  0.00           C  \\nATOM   3403  CG  GLN A 218       0.483  10.070  42.241  1.00  0.00           C  \\nATOM   3404  CD  GLN A 218       0.037  10.181  43.685  1.00  0.00           C  \\nATOM   3405  OE1 GLN A 218      -0.817   9.434  44.156  1.00  0.00           O  \\nATOM   3406  NE2 GLN A 218       0.590  11.149  44.355  1.00  0.00           N  \\nATOM   3407  H   GLN A 218       1.629   9.376  39.372  1.00  0.00           H  \\nATOM   3408  HA  GLN A 218      -0.374   8.453  40.012  1.00  0.00           H  \\nATOM   3409 1HB  GLN A 218      -0.964  11.238  41.147  1.00  0.00           H  \\nATOM   3410 2HB  GLN A 218      -1.582   9.702  41.736  1.00  0.00           H  \\nATOM   3411 1HG  GLN A 218       0.963   9.103  42.092  1.00  0.00           H  \\nATOM   3412 2HG  GLN A 218       1.187  10.874  42.025  1.00  0.00           H  \\nATOM   3413 1HE2 GLN A 218       0.351  11.293  45.316  1.00  0.00           H  \\nATOM   3414 2HE2 GLN A 218       1.254  11.750  43.910  1.00  0.00           H  \\nATOM   3415  N   GLY A 219      -1.351   9.504  37.681  1.00  0.00           N  \\nATOM   3416  CA  GLY A 219      -2.411   9.577  36.670  1.00  0.00           C  \\nATOM   3417  C   GLY A 219      -2.366  10.804  35.838  1.00  0.00           C  \\nATOM   3418  O   GLY A 219      -3.001  10.839  34.791  1.00  0.00           O  \\nATOM   3419  H   GLY A 219      -0.425   9.212  37.403  1.00  0.00           H  \\nATOM   3420 1HA  GLY A 219      -2.343   8.713  36.009  1.00  0.00           H  \\nATOM   3421 2HA  GLY A 219      -3.383   9.532  37.159  1.00  0.00           H  \\nATOM   3422  N   ALA A 220      -1.612  11.845  36.277  1.00  0.00           N  \\nATOM   3423  CA  ALA A 220      -1.552  13.065  35.579  1.00  0.00           C  \\nATOM   3424  C   ALA A 220      -0.361  13.062  34.612  1.00  0.00           C  \\nATOM   3425  O   ALA A 220       0.753  12.754  34.984  1.00  0.00           O  \\nATOM   3426  CB  ALA A 220      -1.435  14.258  36.620  1.00  0.00           C  \\nATOM   3427  H   ALA A 220      -1.080  11.739  37.129  1.00  0.00           H  \\nATOM   3428  HA  ALA A 220      -2.473  13.173  35.006  1.00  0.00           H  \\nATOM   3429 1HB  ALA A 220      -1.388  15.206  36.084  1.00  0.00           H  \\nATOM   3430 2HB  ALA A 220      -2.305  14.255  37.276  1.00  0.00           H  \\nATOM   3431 3HB  ALA A 220      -0.532  14.134  37.215  1.00  0.00           H  \\nATOM   3432  N   PHE A 221      -0.590  13.388  33.345  1.00  0.00           N  \\nATOM   3433  CA  PHE A 221       0.531  13.255  32.385  1.00  0.00           C  \\nATOM   3434  C   PHE A 221       1.485  14.390  32.482  1.00  0.00           C  \\nATOM   3435  O   PHE A 221       1.037  15.522  32.609  1.00  0.00           O  \\nATOM   3436  CB  PHE A 221       0.080  13.166  30.926  1.00  0.00           C  \\nATOM   3437  CG  PHE A 221      -0.738  11.957  30.617  1.00  0.00           C  \\nATOM   3438  CD1 PHE A 221      -0.872  10.915  31.527  1.00  0.00           C  \\nATOM   3439  CD2 PHE A 221      -1.454  11.926  29.412  1.00  0.00           C  \\nATOM   3440  CE1 PHE A 221      -1.674   9.816  31.195  1.00  0.00           C  \\nATOM   3441  CE2 PHE A 221      -2.256  10.826  29.092  1.00  0.00           C  \\nATOM   3442  CZ  PHE A 221      -2.304   9.768  29.946  1.00  0.00           C  \\nATOM   3443  H   PHE A 221      -1.485  13.718  33.014  1.00  0.00           H  \\nATOM   3444  HA  PHE A 221       1.073  12.335  32.610  1.00  0.00           H  \\nATOM   3445 1HB  PHE A 221      -0.509  14.047  30.673  1.00  0.00           H  \\nATOM   3446 2HB  PHE A 221       0.953  13.159  30.275  1.00  0.00           H  \\nATOM   3447  HD1 PHE A 221      -0.351  10.973  32.483  1.00  0.00           H  \\nATOM   3448  HD2 PHE A 221      -1.406  12.780  28.736  1.00  0.00           H  \\nATOM   3449  HE1 PHE A 221      -1.806   9.003  31.908  1.00  0.00           H  \\nATOM   3450  HE2 PHE A 221      -2.838  10.809  28.171  1.00  0.00           H  \\nATOM   3451  HZ  PHE A 221      -2.844   8.873  29.641  1.00  0.00           H  \\nATOM   3452  N   ALA A 222       2.780  14.035  32.356  1.00  0.00           N  \\nATOM   3453  CA  ALA A 222       3.857  15.003  32.367  1.00  0.00           C  \\nATOM   3454  C   ALA A 222       3.666  15.953  31.175  1.00  0.00           C  \\nATOM   3455  O   ALA A 222       3.846  17.168  31.304  1.00  0.00           O  \\nATOM   3456  CB  ALA A 222       5.164  14.280  32.285  1.00  0.00           C  \\nATOM   3457  H   ALA A 222       3.000  13.055  32.250  1.00  0.00           H  \\nATOM   3458  HA  ALA A 222       3.805  15.557  33.305  1.00  0.00           H  \\nATOM   3459 1HB  ALA A 222       5.980  15.002  32.293  1.00  0.00           H  \\nATOM   3460 2HB  ALA A 222       5.264  13.610  33.139  1.00  0.00           H  \\nATOM   3461 3HB  ALA A 222       5.202  13.700  31.364  1.00  0.00           H  \\nATOM   3462  N   SER A 223       3.254  15.421  30.023  1.00  0.00           N  \\nATOM   3463  CA  SER A 223       2.781  16.278  28.928  1.00  0.00           C  \\nATOM   3464  C   SER A 223       1.655  15.621  28.169  1.00  0.00           C  \\nATOM   3465  O   SER A 223       1.482  14.408  28.300  1.00  0.00           O  \\nATOM   3466  CB  SER A 223       3.922  16.617  27.983  1.00  0.00           C  \\nATOM   3467  OG  SER A 223       4.750  17.525  28.677  1.00  0.00           O  \\nATOM   3468  H   SER A 223       3.264  14.419  29.894  1.00  0.00           H  \\nATOM   3469  HA  SER A 223       2.396  17.205  29.355  1.00  0.00           H  \\nATOM   3470 1HB  SER A 223       4.452  15.705  27.711  1.00  0.00           H  \\nATOM   3471 2HB  SER A 223       3.520  17.047  27.067  1.00  0.00           H  \\nATOM   3472  HG  SER A 223       4.341  17.653  29.536  1.00  0.00           H  \\nATOM   3473  N   PRO A 224       0.883  16.436  27.419  1.00  0.00           N  \\nATOM   3474  CA  PRO A 224      -0.260  15.893  26.713  1.00  0.00           C  \\nATOM   3475  C   PRO A 224       0.098  14.947  25.552  1.00  0.00           C  \\nATOM   3476  O   PRO A 224       1.215  14.936  25.033  1.00  0.00           O  \\nATOM   3477  CB  PRO A 224      -1.022  17.099  26.203  1.00  0.00           C  \\nATOM   3478  CG  PRO A 224      -0.210  18.334  26.596  1.00  0.00           C  \\nATOM   3479  CD  PRO A 224       1.116  17.864  27.131  1.00  0.00           C  \\nATOM   3480  HA  PRO A 224      -0.884  15.325  27.418  1.00  0.00           H  \\nATOM   3481 1HB  PRO A 224      -1.150  17.030  25.113  1.00  0.00           H  \\nATOM   3482 2HB  PRO A 224      -2.030  17.121  26.644  1.00  0.00           H  \\nATOM   3483 1HG  PRO A 224      -0.071  18.990  25.724  1.00  0.00           H  \\nATOM   3484 2HG  PRO A 224      -0.753  18.921  27.352  1.00  0.00           H  \\nATOM   3485 1HD  PRO A 224       1.891  17.999  26.362  1.00  0.00           H  \\nATOM   3486 2HD  PRO A 224       1.367  18.431  28.039  1.00  0.00           H  \\nATOM   3487  N   ILE A 225      -0.852  14.091  25.239  1.00  0.00           N  \\nATOM   3488  CA  ILE A 225      -0.740  13.233  24.079  1.00  0.00           C  \\nATOM   3489  C   ILE A 225      -1.910  13.621  23.184  1.00  0.00           C  \\nATOM   3490  O   ILE A 225      -3.025  13.772  23.646  1.00  0.00           O  \\nATOM   3491  CB  ILE A 225      -0.790  11.711  24.501  1.00  0.00           C  \\nATOM   3492  CG1 ILE A 225       0.658  11.232  24.818  1.00  0.00           C  \\nATOM   3493  CG2 ILE A 225      -1.505  10.792  23.475  1.00  0.00           C  \\nATOM   3494  CD1 ILE A 225       0.956  11.051  26.179  1.00  0.00           C  \\nATOM   3495  H   ILE A 225      -1.677  14.030  25.818  1.00  0.00           H  \\nATOM   3496  HA  ILE A 225       0.217  13.427  23.597  1.00  0.00           H  \\nATOM   3497  HB  ILE A 225      -1.324  11.613  25.446  1.00  0.00           H  \\nATOM   3498 1HG1 ILE A 225       0.845  10.283  24.317  1.00  0.00           H  \\nATOM   3499 2HG1 ILE A 225       1.374  11.954  24.425  1.00  0.00           H  \\nATOM   3500 1HG2 ILE A 225      -1.496   9.765  23.839  1.00  0.00           H  \\nATOM   3501 2HG2 ILE A 225      -2.535  11.122  23.347  1.00  0.00           H  \\nATOM   3502 3HG2 ILE A 225      -0.986  10.843  22.518  1.00  0.00           H  \\nATOM   3503 1HD1 ILE A 225       1.989  10.718  26.284  1.00  0.00           H  \\nATOM   3504 2HD1 ILE A 225       0.824  11.995  26.709  1.00  0.00           H  \\nATOM   3505 3HD1 ILE A 225       0.289  10.301  26.601  1.00  0.00           H  \\nATOM   3506  N   GLN A 226      -1.600  13.845  21.922  1.00  0.00           N  \\nATOM   3507  CA  GLN A 226      -2.577  14.237  20.938  1.00  0.00           C  \\nATOM   3508  C   GLN A 226      -3.238  13.053  20.284  1.00  0.00           C  \\nATOM   3509  O   GLN A 226      -2.565  12.123  19.776  1.00  0.00           O  \\nATOM   3510  CB  GLN A 226      -1.944  15.140  19.891  1.00  0.00           C  \\nATOM   3511  CG  GLN A 226      -2.967  15.813  19.024  1.00  0.00           C  \\nATOM   3512  CD  GLN A 226      -2.263  16.564  18.036  1.00  0.00           C  \\nATOM   3513  OE1 GLN A 226      -2.056  17.762  18.180  1.00  0.00           O  \\nATOM   3514  NE2 GLN A 226      -1.803  15.864  17.032  1.00  0.00           N  \\nATOM   3515  H   GLN A 226      -0.636  13.735  21.642  1.00  0.00           H  \\nATOM   3516  HA  GLN A 226      -3.371  14.789  21.440  1.00  0.00           H  \\nATOM   3517 1HB  GLN A 226      -1.342  15.903  20.385  1.00  0.00           H  \\nATOM   3518 2HB  GLN A 226      -1.276  14.553  19.260  1.00  0.00           H  \\nATOM   3519 1HG  GLN A 226      -3.591  15.051  18.557  1.00  0.00           H  \\nATOM   3520 2HG  GLN A 226      -3.579  16.468  19.644  1.00  0.00           H  \\nATOM   3521 1HE2 GLN A 226      -1.297  16.316  16.296  1.00  0.00           H  \\nATOM   3522 2HE2 GLN A 226      -1.956  14.876  16.999  1.00  0.00           H  \\nATOM   3523  N   LEU A 227      -4.564  13.130  20.271  1.00  0.00           N  \\nATOM   3524  CA  LEU A 227      -5.340  12.131  19.560  1.00  0.00           C  \\nATOM   3525  C   LEU A 227      -6.237  12.774  18.556  1.00  0.00           C  \\nATOM   3526  O   LEU A 227      -6.149  13.997  18.375  1.00  0.00           O  \\nATOM   3527  CB  LEU A 227      -6.119  11.290  20.525  1.00  0.00           C  \\nATOM   3528  CG  LEU A 227      -5.347  10.390  21.481  1.00  0.00           C  \\nATOM   3529  CD1 LEU A 227      -6.339   9.812  22.555  1.00  0.00           C  \\nATOM   3530  CD2 LEU A 227      -4.444   9.372  20.750  1.00  0.00           C  \\nATOM   3531  H   LEU A 227      -5.045  13.877  20.751  1.00  0.00           H  \\nATOM   3532  HA  LEU A 227      -4.654  11.487  19.011  1.00  0.00           H  \\nATOM   3533 1HB  LEU A 227      -6.728  11.946  21.145  1.00  0.00           H  \\nATOM   3534 2HB  LEU A 227      -6.785  10.638  19.959  1.00  0.00           H  \\nATOM   3535  HG  LEU A 227      -4.709  11.001  22.120  1.00  0.00           H  \\nATOM   3536 1HD1 LEU A 227      -5.794   9.166  23.244  1.00  0.00           H  \\nATOM   3537 2HD1 LEU A 227      -6.793  10.633  23.111  1.00  0.00           H  \\nATOM   3538 3HD1 LEU A 227      -7.118   9.236  22.058  1.00  0.00           H  \\nATOM   3539 1HD2 LEU A 227      -3.919   8.759  21.483  1.00  0.00           H  \\nATOM   3540 2HD2 LEU A 227      -5.057   8.732  20.115  1.00  0.00           H  \\nATOM   3541 3HD2 LEU A 227      -3.718   9.905  20.136  1.00  0.00           H  \\nATOM   3542  N   GLN A 228      -7.077  12.019  17.911  1.00  0.00           N  \\nATOM   3543  CA  GLN A 228      -7.976  12.609  16.881  1.00  0.00           C  \\nATOM   3544  C   GLN A 228      -9.353  12.090  17.087  1.00  0.00           C  \\nATOM   3545  O   GLN A 228      -9.573  10.937  17.410  1.00  0.00           O  \\nATOM   3546  CB  GLN A 228      -7.505  12.251  15.461  1.00  0.00           C  \\nATOM   3547  CG  GLN A 228      -6.373  13.104  14.979  1.00  0.00           C  \\nATOM   3548  CD  GLN A 228      -5.877  12.781  13.575  1.00  0.00           C  \\nATOM   3549  OE1 GLN A 228      -4.674  12.844  13.314  1.00  0.00           O  \\nATOM   3550  NE2 GLN A 228      -6.762  12.446  12.703  1.00  0.00           N  \\nATOM   3551  H   GLN A 228      -7.129  11.028  18.100  1.00  0.00           H  \\nATOM   3552  HA  GLN A 228      -7.956  13.694  16.985  1.00  0.00           H  \\nATOM   3553 1HB  GLN A 228      -7.186  11.209  15.435  1.00  0.00           H  \\nATOM   3554 2HB  GLN A 228      -8.337  12.356  14.764  1.00  0.00           H  \\nATOM   3555 1HG  GLN A 228      -6.694  14.146  14.972  1.00  0.00           H  \\nATOM   3556 2HG  GLN A 228      -5.525  12.979  15.653  1.00  0.00           H  \\nATOM   3557 1HE2 GLN A 228      -6.484  12.225  11.767  1.00  0.00           H  \\nATOM   3558 2HE2 GLN A 228      -7.728  12.406  12.960  1.00  0.00           H  \\nATOM   3559  N   ARG A 229     -10.347  12.986  16.896  1.00  0.00           N  \\nATOM   3560  CA  ARG A 229     -11.742  12.597  16.948  1.00  0.00           C  \\nATOM   3561  C   ARG A 229     -12.116  11.911  15.650  1.00  0.00           C  \\nATOM   3562  O   ARG A 229     -11.332  11.982  14.629  1.00  0.00           O  \\nATOM   3563  CB  ARG A 229     -12.627  13.868  17.132  1.00  0.00           C  \\nATOM   3564  CG  ARG A 229     -12.326  14.646  18.391  1.00  0.00           C  \\nATOM   3565  CD  ARG A 229     -13.467  15.667  18.590  1.00  0.00           C  \\nATOM   3566  NE  ARG A 229     -13.151  16.472  19.779  1.00  0.00           N  \\nATOM   3567  CZ  ARG A 229     -12.418  17.579  19.776  1.00  0.00           C  \\nATOM   3568  NH1 ARG A 229     -11.907  18.051  18.663  1.00  0.00           N  \\nATOM   3569  NH2 ARG A 229     -12.196  18.189  20.937  1.00  0.00           N  \\nATOM   3570  H   ARG A 229     -10.112  13.951  16.712  1.00  0.00           H  \\nATOM   3571  HA  ARG A 229     -11.885  11.934  17.802  1.00  0.00           H  \\nATOM   3572 1HB  ARG A 229     -12.489  14.533  16.281  1.00  0.00           H  \\nATOM   3573 2HB  ARG A 229     -13.678  13.579  17.155  1.00  0.00           H  \\nATOM   3574 1HG  ARG A 229     -12.275  13.962  19.238  1.00  0.00           H  \\nATOM   3575 2HG  ARG A 229     -11.370  15.160  18.282  1.00  0.00           H  \\nATOM   3576 1HD  ARG A 229     -13.540  16.306  17.711  1.00  0.00           H  \\nATOM   3577 2HD  ARG A 229     -14.408  15.137  18.733  1.00  0.00           H  \\nATOM   3578  HE  ARG A 229     -13.519  16.161  20.668  1.00  0.00           H  \\nATOM   3579 1HH1 ARG A 229     -12.069  17.570  17.789  1.00  0.00           H  \\nATOM   3580 2HH1 ARG A 229     -11.352  18.894  18.680  1.00  0.00           H  \\nATOM   3581 1HH2 ARG A 229     -12.582  17.808  21.790  1.00  0.00           H  \\nATOM   3582 2HH2 ARG A 229     -11.643  19.032  20.966  1.00  0.00           H  \\nATOM   3583  N   ARG A 230     -13.346  11.401  15.525  1.00  0.00           N  \\nATOM   3584  CA  ARG A 230     -13.767  10.771  14.291  1.00  0.00           C  \\nATOM   3585  C   ARG A 230     -13.742  11.658  13.064  1.00  0.00           C  \\nATOM   3586  O   ARG A 230     -13.690  11.112  11.967  1.00  0.00           O  \\nATOM   3587  CB  ARG A 230     -15.169  10.247  14.444  1.00  0.00           C  \\nATOM   3588  CG  ARG A 230     -15.278   9.207  15.499  1.00  0.00           C  \\nATOM   3589  CD  ARG A 230     -16.630   8.608  15.556  1.00  0.00           C  \\nATOM   3590  NE  ARG A 230     -16.895   7.932  16.862  1.00  0.00           N  \\nATOM   3591  CZ  ARG A 230     -16.585   6.626  17.071  1.00  0.00           C  \\nATOM   3592  NH1 ARG A 230     -16.088   5.877  16.069  1.00  0.00           N  \\nATOM   3593  NH2 ARG A 230     -16.867   6.050  18.224  1.00  0.00           N  \\nATOM   3594  H   ARG A 230     -13.995  11.453  16.297  1.00  0.00           H  \\nATOM   3595  HA  ARG A 230     -13.096   9.937  14.082  1.00  0.00           H  \\nATOM   3596 1HB  ARG A 230     -15.839  11.069  14.691  1.00  0.00           H  \\nATOM   3597 2HB  ARG A 230     -15.505   9.824  13.497  1.00  0.00           H  \\nATOM   3598 1HG  ARG A 230     -14.563   8.409  15.299  1.00  0.00           H  \\nATOM   3599 2HG  ARG A 230     -15.064   9.651  16.472  1.00  0.00           H  \\nATOM   3600 1HD  ARG A 230     -17.379   9.388  15.422  1.00  0.00           H  \\nATOM   3601 2HD  ARG A 230     -16.733   7.867  14.765  1.00  0.00           H  \\nATOM   3602  HE  ARG A 230     -17.318   8.481  17.599  1.00  0.00           H  \\nATOM   3603 1HH1 ARG A 230     -15.944   6.285  15.156  1.00  0.00           H  \\nATOM   3604 2HH1 ARG A 230     -15.859   4.907  16.230  1.00  0.00           H  \\nATOM   3605 1HH2 ARG A 230     -17.316   6.581  18.958  1.00  0.00           H  \\nATOM   3606 2HH2 ARG A 230     -16.634   5.080  18.373  1.00  0.00           H  \\nATOM   3607  N   ASN A 231     -13.849  12.977  13.245  1.00  0.00           N  \\nATOM   3608  CA  ASN A 231     -13.894  13.874  12.102  1.00  0.00           C  \\nATOM   3609  C   ASN A 231     -12.501  14.378  11.776  1.00  0.00           C  \\nATOM   3610  O   ASN A 231     -12.283  15.162  10.778  1.00  0.00           O  \\nATOM   3611  CB  ASN A 231     -14.908  14.966  12.412  1.00  0.00           C  \\nATOM   3612  CG  ASN A 231     -14.402  15.982  13.450  1.00  0.00           C  \\nATOM   3613  OD1 ASN A 231     -13.457  15.789  14.256  1.00  0.00           O  \\nATOM   3614  ND2 ASN A 231     -15.081  17.131  13.451  1.00  0.00           N  \\nATOM   3615  H   ASN A 231     -13.900  13.364  14.176  1.00  0.00           H  \\nATOM   3616  HA  ASN A 231     -14.214  13.305  11.228  1.00  0.00           H  \\nATOM   3617 1HB  ASN A 231     -15.158  15.502  11.495  1.00  0.00           H  \\nATOM   3618 2HB  ASN A 231     -15.826  14.514  12.788  1.00  0.00           H  \\nATOM   3619 1HD2 ASN A 231     -14.831  17.859  14.090  1.00  0.00           H  \\nATOM   3620 2HD2 ASN A 231     -15.839  17.264  12.814  1.00  0.00           H  \\nATOM   3621  N   GLY A 232     -11.510  13.903  12.515  1.00  0.00           N  \\nATOM   3622  CA  GLY A 232     -10.133  14.208  12.152  1.00  0.00           C  \\nATOM   3623  C   GLY A 232      -9.581  15.355  12.984  1.00  0.00           C  \\nATOM   3624  O   GLY A 232      -8.348  15.534  13.005  1.00  0.00           O  \\nATOM   3625  H   GLY A 232     -11.689  13.331  13.328  1.00  0.00           H  \\nATOM   3626 1HA  GLY A 232      -9.514  13.322  12.295  1.00  0.00           H  \\nATOM   3627 2HA  GLY A 232     -10.084  14.467  11.095  1.00  0.00           H  \\nATOM   3628  N   SER A 233     -10.414  16.028  13.772  1.00  0.00           N  \\nATOM   3629  CA  SER A 233      -9.921  17.168  14.579  1.00  0.00           C  \\nATOM   3630  C   SER A 233      -9.045  16.673  15.710  1.00  0.00           C  \\nATOM   3631  O   SER A 233      -9.228  15.554  16.197  1.00  0.00           O  \\nATOM   3632  CB  SER A 233     -11.079  18.049  15.108  1.00  0.00           C  \\nATOM   3633  OG  SER A 233     -12.014  17.233  15.790  1.00  0.00           O  \\nATOM   3634  H   SER A 233     -11.389  15.770  13.829  1.00  0.00           H  \\nATOM   3635  HA  SER A 233      -9.288  17.792  13.947  1.00  0.00           H  \\nATOM   3636 1HB  SER A 233     -10.680  18.812  15.775  1.00  0.00           H  \\nATOM   3637 2HB  SER A 233     -11.556  18.561  14.274  1.00  0.00           H  \\nATOM   3638  HG  SER A 233     -11.671  16.338  15.739  1.00  0.00           H  \\nATOM   3639  N   LYS A 234      -8.073  17.484  16.095  1.00  0.00           N  \\nATOM   3640  CA  LYS A 234      -7.122  17.055  17.061  1.00  0.00           C  \\nATOM   3641  C   LYS A 234      -7.518  17.513  18.444  1.00  0.00           C  \\nATOM   3642  O   LYS A 234      -8.100  18.614  18.631  1.00  0.00           O  \\nATOM   3643  CB  LYS A 234      -5.755  17.592  16.728  1.00  0.00           C  \\nATOM   3644  CG  LYS A 234      -5.323  17.252  15.320  1.00  0.00           C  \\nATOM   3645  CD  LYS A 234      -4.730  15.949  15.070  1.00  0.00           C  \\nATOM   3646  CE  LYS A 234      -4.100  16.035  13.667  1.00  0.00           C  \\nATOM   3647  NZ  LYS A 234      -3.299  17.306  13.581  1.00  0.00           N  \\nATOM   3648  H   LYS A 234      -7.996  18.414  15.708  1.00  0.00           H  \\nATOM   3649  HA  LYS A 234      -7.082  15.965  17.047  1.00  0.00           H  \\nATOM   3650 1HB  LYS A 234      -5.752  18.676  16.844  1.00  0.00           H  \\nATOM   3651 2HB  LYS A 234      -5.024  17.186  17.427  1.00  0.00           H  \\nATOM   3652 1HG  LYS A 234      -6.183  17.306  14.652  1.00  0.00           H  \\nATOM   3653 2HG  LYS A 234      -4.579  17.973  14.984  1.00  0.00           H  \\nATOM   3654 1HD  LYS A 234      -3.983  15.736  15.836  1.00  0.00           H  \\nATOM   3655 2HD  LYS A 234      -5.503  15.182  15.118  1.00  0.00           H  \\nATOM   3656 1HE  LYS A 234      -3.462  15.168  13.501  1.00  0.00           H  \\nATOM   3657 2HE  LYS A 234      -4.887  16.028  12.914  1.00  0.00           H  \\nATOM   3658 1HZ  LYS A 234      -2.879  17.378  12.665  1.00  0.00           H  \\nATOM   3659 2HZ  LYS A 234      -3.906  18.099  13.736  1.00  0.00           H  \\nATOM   3660 3HZ  LYS A 234      -2.574  17.296  14.284  1.00  0.00           H  \\nATOM   3661  N   PHE A 235      -7.115  16.743  19.436  1.00  0.00           N  \\nATOM   3662  CA  PHE A 235      -7.379  17.151  20.804  1.00  0.00           C  \\nATOM   3663  C   PHE A 235      -6.327  16.498  21.702  1.00  0.00           C  \\nATOM   3664  O   PHE A 235      -5.609  15.548  21.261  1.00  0.00           O  \\nATOM   3665  CB  PHE A 235      -8.847  16.838  21.182  1.00  0.00           C  \\nATOM   3666  CG  PHE A 235      -9.128  15.365  21.493  1.00  0.00           C  \\nATOM   3667  CD1 PHE A 235      -9.407  14.443  20.463  1.00  0.00           C  \\nATOM   3668  CD2 PHE A 235      -9.072  14.924  22.742  1.00  0.00           C  \\nATOM   3669  CE1 PHE A 235      -9.717  13.110  20.731  1.00  0.00           C  \\nATOM   3670  CE2 PHE A 235      -9.324  13.588  23.039  1.00  0.00           C  \\nATOM   3671  CZ  PHE A 235      -9.688  12.700  22.062  1.00  0.00           C  \\nATOM   3672  H   PHE A 235      -6.627  15.875  19.267  1.00  0.00           H  \\nATOM   3673  HA  PHE A 235      -7.219  18.227  20.881  1.00  0.00           H  \\nATOM   3674 1HB  PHE A 235      -9.129  17.421  22.058  1.00  0.00           H  \\nATOM   3675 2HB  PHE A 235      -9.504  17.135  20.366  1.00  0.00           H  \\nATOM   3676  HD1 PHE A 235      -9.380  14.779  19.426  1.00  0.00           H  \\nATOM   3677  HD2 PHE A 235      -8.784  15.614  23.536  1.00  0.00           H  \\nATOM   3678  HE1 PHE A 235      -9.969  12.435  19.914  1.00  0.00           H  \\nATOM   3679  HE2 PHE A 235      -9.228  13.252  24.071  1.00  0.00           H  \\nATOM   3680  HZ  PHE A 235      -9.955  11.677  22.324  1.00  0.00           H  \\nATOM   3681  N   SER A 236      -6.217  17.031  22.923  1.00  0.00           N  \\nATOM   3682  CA  SER A 236      -5.165  16.656  23.827  1.00  0.00           C  \\nATOM   3683  C   SER A 236      -5.742  15.920  25.026  1.00  0.00           C  \\nATOM   3684  O   SER A 236      -6.795  16.304  25.537  1.00  0.00           O  \\nATOM   3685  CB  SER A 236      -4.513  17.947  24.297  1.00  0.00           C  \\nATOM   3686  OG  SER A 236      -3.636  18.402  23.281  1.00  0.00           O  \\nATOM   3687  H   SER A 236      -6.896  17.719  23.217  1.00  0.00           H  \\nATOM   3688  HA  SER A 236      -4.445  16.041  23.285  1.00  0.00           H  \\nATOM   3689 1HB  SER A 236      -5.284  18.688  24.507  1.00  0.00           H  \\nATOM   3690 2HB  SER A 236      -3.973  17.765  25.225  1.00  0.00           H  \\nATOM   3691  HG  SER A 236      -3.699  17.760  22.570  1.00  0.00           H  \\nATOM   3692  N   VAL A 237      -5.024  14.868  25.463  1.00  0.00           N  \\nATOM   3693  CA  VAL A 237      -5.413  14.157  26.720  1.00  0.00           C  \\nATOM   3694  C   VAL A 237      -4.355  14.409  27.760  1.00  0.00           C  \\nATOM   3695  O   VAL A 237      -3.180  14.362  27.454  1.00  0.00           O  \\nATOM   3696  CB  VAL A 237      -5.675  12.641  26.508  1.00  0.00           C  \\nATOM   3697  CG1 VAL A 237      -6.705  12.414  25.416  1.00  0.00           C  \\nATOM   3698  CG2 VAL A 237      -4.454  11.842  26.147  1.00  0.00           C  \\nATOM   3699  H   VAL A 237      -4.217  14.549  24.947  1.00  0.00           H  \\nATOM   3700  HA  VAL A 237      -6.338  14.597  27.095  1.00  0.00           H  \\nATOM   3701  HB  VAL A 237      -6.076  12.219  27.430  1.00  0.00           H  \\nATOM   3702 1HG1 VAL A 237      -6.870  11.344  25.288  1.00  0.00           H  \\nATOM   3703 2HG1 VAL A 237      -7.643  12.894  25.695  1.00  0.00           H  \\nATOM   3704 3HG1 VAL A 237      -6.343  12.839  24.480  1.00  0.00           H  \\nATOM   3705 1HG2 VAL A 237      -4.729  10.795  26.019  1.00  0.00           H  \\nATOM   3706 2HG2 VAL A 237      -4.031  12.223  25.217  1.00  0.00           H  \\nATOM   3707 3HG2 VAL A 237      -3.715  11.928  26.943  1.00  0.00           H  \\nATOM   3708  N   TYR A 238      -4.772  14.775  28.962  1.00  0.00           N  \\nATOM   3709  CA  TYR A 238      -3.856  15.211  29.988  1.00  0.00           C  \\nATOM   3710  C   TYR A 238      -3.762  14.258  31.175  1.00  0.00           C  \\nATOM   3711  O   TYR A 238      -2.923  14.470  32.052  1.00  0.00           O  \\nATOM   3712  CB  TYR A 238      -4.216  16.655  30.463  1.00  0.00           C  \\nATOM   3713  CG  TYR A 238      -4.230  17.715  29.369  1.00  0.00           C  \\nATOM   3714  CD1 TYR A 238      -5.382  17.905  28.603  1.00  0.00           C  \\nATOM   3715  CD2 TYR A 238      -3.140  18.524  29.141  1.00  0.00           C  \\nATOM   3716  CE1 TYR A 238      -5.458  18.914  27.621  1.00  0.00           C  \\nATOM   3717  CE2 TYR A 238      -3.187  19.513  28.167  1.00  0.00           C  \\nATOM   3718  CZ  TYR A 238      -4.337  19.694  27.409  1.00  0.00           C  \\nATOM   3719  OH  TYR A 238      -4.395  20.676  26.422  1.00  0.00           O  \\nATOM   3720  H   TYR A 238      -5.761  14.747  29.165  1.00  0.00           H  \\nATOM   3721  HA  TYR A 238      -2.849  15.222  29.571  1.00  0.00           H  \\nATOM   3722 1HB  TYR A 238      -5.204  16.650  30.926  1.00  0.00           H  \\nATOM   3723 2HB  TYR A 238      -3.502  16.977  31.220  1.00  0.00           H  \\nATOM   3724  HD1 TYR A 238      -6.251  17.265  28.757  1.00  0.00           H  \\nATOM   3725  HD2 TYR A 238      -2.230  18.390  29.726  1.00  0.00           H  \\nATOM   3726  HE1 TYR A 238      -6.370  19.039  27.038  1.00  0.00           H  \\nATOM   3727  HE2 TYR A 238      -2.319  20.150  27.995  1.00  0.00           H  \\nATOM   3728  HH  TYR A 238      -3.557  21.144  26.389  1.00  0.00           H  \\nATOM   3729  N   ASP A 239      -4.567  13.200  31.189  1.00  0.00           N  \\nATOM   3730  CA  ASP A 239      -4.566  12.295  32.322  1.00  0.00           C  \\nATOM   3731  C   ASP A 239      -5.214  11.024  32.035  1.00  0.00           C  \\nATOM   3732  O   ASP A 239      -5.984  10.915  31.072  1.00  0.00           O  \\nATOM   3733  CB  ASP A 239      -5.137  12.911  33.614  1.00  0.00           C  \\nATOM   3734  CG  ASP A 239      -6.610  13.260  33.522  1.00  0.00           C  \\nATOM   3735  OD1 ASP A 239      -6.923  14.462  33.266  1.00  0.00           O  \\nATOM   3736  OD2 ASP A 239      -7.445  12.375  33.824  1.00  0.00           O  \\nATOM   3737  H   ASP A 239      -5.186  13.016  30.413  1.00  0.00           H  \\nATOM   3738  HA  ASP A 239      -3.536  12.008  32.535  1.00  0.00           H  \\nATOM   3739 1HB  ASP A 239      -5.002  12.213  34.441  1.00  0.00           H  \\nATOM   3740 2HB  ASP A 239      -4.585  13.819  33.859  1.00  0.00           H  \\nATOM   3741  N   VAL A 240      -4.962  10.047  32.914  1.00  0.00           N  \\nATOM   3742  CA  VAL A 240      -5.376   8.688  32.585  1.00  0.00           C  \\nATOM   3743  C   VAL A 240      -6.886   8.481  32.635  1.00  0.00           C  \\nATOM   3744  O   VAL A 240      -7.412   7.629  31.961  1.00  0.00           O  \\nATOM   3745  CB  VAL A 240      -4.676   7.701  33.501  1.00  0.00           C  \\nATOM   3746  CG1 VAL A 240      -4.863   6.189  33.052  1.00  0.00           C  \\nATOM   3747  CG2 VAL A 240      -3.218   7.998  33.552  1.00  0.00           C  \\nATOM   3748  H   VAL A 240      -4.497  10.220  33.793  1.00  0.00           H  \\nATOM   3749  HA  VAL A 240      -5.089   8.479  31.554  1.00  0.00           H  \\nATOM   3750  HB  VAL A 240      -5.099   7.786  34.502  1.00  0.00           H  \\nATOM   3751 1HG1 VAL A 240      -4.338   5.535  33.749  1.00  0.00           H  \\nATOM   3752 2HG1 VAL A 240      -5.924   5.939  33.049  1.00  0.00           H  \\nATOM   3753 3HG1 VAL A 240      -4.454   6.054  32.051  1.00  0.00           H  \\nATOM   3754 1HG2 VAL A 240      -2.724   7.285  34.212  1.00  0.00           H  \\nATOM   3755 2HG2 VAL A 240      -2.795   7.918  32.550  1.00  0.00           H  \\nATOM   3756 3HG2 VAL A 240      -3.066   9.009  33.931  1.00  0.00           H  \\nATOM   3757  N   SER A 241      -7.623   9.266  33.409  1.00  0.00           N  \\nATOM   3758  CA  SER A 241      -9.001   8.878  33.688  1.00  0.00           C  \\nATOM   3759  C   SER A 241      -9.820   8.726  32.429  1.00  0.00           C  \\nATOM   3760  O   SER A 241     -10.629   7.787  32.324  1.00  0.00           O  \\nATOM   3761  CB  SER A 241      -9.677   9.888  34.640  1.00  0.00           C  \\nATOM   3762  OG  SER A 241      -9.288  11.216  34.388  1.00  0.00           O  \\nATOM   3763  H   SER A 241      -7.262  10.120  33.809  1.00  0.00           H  \\nATOM   3764  HA  SER A 241      -8.994   7.900  34.172  1.00  0.00           H  \\nATOM   3765 1HB  SER A 241     -10.760   9.815  34.539  1.00  0.00           H  \\nATOM   3766 2HB  SER A 241      -9.427   9.639  35.671  1.00  0.00           H  \\nATOM   3767  HG  SER A 241      -8.670  11.176  33.654  1.00  0.00           H  \\nATOM   3768  N   ILE A 242      -9.646   9.636  31.481  1.00  0.00           N  \\nATOM   3769  CA  ILE A 242     -10.483   9.623  30.306  1.00  0.00           C  \\nATOM   3770  C   ILE A 242     -10.118   8.462  29.353  1.00  0.00           C  \\nATOM   3771  O   ILE A 242     -10.876   8.113  28.466  1.00  0.00           O  \\nATOM   3772  CB  ILE A 242     -10.359  10.990  29.574  1.00  0.00           C  \\nATOM   3773  CG1 ILE A 242     -11.432  11.132  28.528  1.00  0.00           C  \\nATOM   3774  CG2 ILE A 242      -8.984  11.237  29.023  1.00  0.00           C  \\nATOM   3775  CD1 ILE A 242     -11.249  12.431  27.667  1.00  0.00           C  \\nATOM   3776  H   ILE A 242      -8.931  10.344  31.568  1.00  0.00           H  \\nATOM   3777  HA  ILE A 242     -11.516   9.476  30.619  1.00  0.00           H  \\nATOM   3778  HB  ILE A 242     -10.587  11.797  30.270  1.00  0.00           H  \\nATOM   3779 1HG1 ILE A 242     -11.417  10.263  27.871  1.00  0.00           H  \\nATOM   3780 2HG1 ILE A 242     -12.409  11.159  29.011  1.00  0.00           H  \\nATOM   3781 1HG2 ILE A 242      -8.961  12.205  28.524  1.00  0.00           H  \\nATOM   3782 2HG2 ILE A 242      -8.260  11.231  29.837  1.00  0.00           H  \\nATOM   3783 3HG2 ILE A 242      -8.733  10.454  28.308  1.00  0.00           H  \\nATOM   3784 1HD1 ILE A 242     -12.046  12.492  26.926  1.00  0.00           H  \\nATOM   3785 2HD1 ILE A 242     -11.290  13.307  28.316  1.00  0.00           H  \\nATOM   3786 3HD1 ILE A 242     -10.285  12.399  27.160  1.00  0.00           H  \\nATOM   3787  N   LEU A 243      -8.981   7.843  29.592  1.00  0.00           N  \\nATOM   3788  CA  LEU A 243      -8.532   6.711  28.741  1.00  0.00           C  \\nATOM   3789  C   LEU A 243      -8.831   5.384  29.332  1.00  0.00           C  \\nATOM   3790  O   LEU A 243      -8.698   4.382  28.650  1.00  0.00           O  \\nATOM   3791  CB  LEU A 243      -7.044   6.809  28.407  1.00  0.00           C  \\nATOM   3792  CG  LEU A 243      -6.571   8.094  27.709  1.00  0.00           C  \\nATOM   3793  CD1 LEU A 243      -5.070   8.103  27.818  1.00  0.00           C  \\nATOM   3794  CD2 LEU A 243      -7.042   7.976  26.280  1.00  0.00           C  \\nATOM   3795  H   LEU A 243      -8.401   8.138  30.364  1.00  0.00           H  \\nATOM   3796  HA  LEU A 243      -9.090   6.739  27.806  1.00  0.00           H  \\nATOM   3797 1HB  LEU A 243      -6.474   6.719  29.330  1.00  0.00           H  \\nATOM   3798 2HB  LEU A 243      -6.778   5.975  27.757  1.00  0.00           H  \\nATOM   3799  HG  LEU A 243      -7.013   8.960  28.203  1.00  0.00           H  \\nATOM   3800 1HD1 LEU A 243      -4.676   8.998  27.337  1.00  0.00           H  \\nATOM   3801 2HD1 LEU A 243      -4.781   8.100  28.869  1.00  0.00           H  \\nATOM   3802 3HD1 LEU A 243      -4.664   7.219  27.327  1.00  0.00           H  \\nATOM   3803 1HD2 LEU A 243      -6.738   8.863  25.722  1.00  0.00           H  \\nATOM   3804 2HD2 LEU A 243      -6.600   7.091  25.823  1.00  0.00           H  \\nATOM   3805 3HD2 LEU A 243      -8.129   7.890  26.261  1.00  0.00           H  \\nATOM   3806  N   ILE A 244      -9.220   5.268  30.619  1.00  0.00           N  \\nATOM   3807  CA  ILE A 244      -9.508   3.970  31.195  1.00  0.00           C  \\nATOM   3808  C   ILE A 244     -10.632   3.187  30.418  1.00  0.00           C  \\nATOM   3809  O   ILE A 244     -10.459   1.996  30.279  1.00  0.00           O  \\nATOM   3810  CB  ILE A 244      -9.799   4.119  32.702  1.00  0.00           C  \\nATOM   3811  CG1 ILE A 244      -8.547   4.530  33.401  1.00  0.00           C  \\nATOM   3812  CG2 ILE A 244     -10.402   2.821  33.292  1.00  0.00           C  \\nATOM   3813  CD1 ILE A 244      -8.764   5.024  34.889  1.00  0.00           C  \\nATOM   3814  H   ILE A 244      -9.314   6.093  31.194  1.00  0.00           H  \\nATOM   3815  HA  ILE A 244      -8.634   3.332  31.065  1.00  0.00           H  \\nATOM   3816  HB  ILE A 244     -10.509   4.931  32.857  1.00  0.00           H  \\nATOM   3817 1HG1 ILE A 244      -7.851   3.692  33.422  1.00  0.00           H  \\nATOM   3818 2HG1 ILE A 244      -8.069   5.337  32.846  1.00  0.00           H  \\nATOM   3819 1HG2 ILE A 244     -10.594   2.960  34.356  1.00  0.00           H  \\nATOM   3820 2HG2 ILE A 244     -11.336   2.589  32.782  1.00  0.00           H  \\nATOM   3821 3HG2 ILE A 244      -9.700   1.998  33.155  1.00  0.00           H  \\nATOM   3822 1HD1 ILE A 244      -7.804   5.304  35.323  1.00  0.00           H  \\nATOM   3823 2HD1 ILE A 244      -9.429   5.888  34.892  1.00  0.00           H  \\nATOM   3824 3HD1 ILE A 244      -9.207   4.222  35.477  1.00  0.00           H  \\nATOM   3825  N   PRO A 245     -11.659   3.857  29.851  1.00  0.00           N  \\nATOM   3826  CA  PRO A 245     -12.628   3.045  29.055  1.00  0.00           C  \\nATOM   3827  C   PRO A 245     -12.164   2.649  27.663  1.00  0.00           C  \\nATOM   3828  O   PRO A 245     -12.840   1.881  26.990  1.00  0.00           O  \\nATOM   3829  CB  PRO A 245     -13.873   3.962  28.900  1.00  0.00           C  \\nATOM   3830  CG  PRO A 245     -13.704   4.973  29.950  1.00  0.00           C  \\nATOM   3831  CD  PRO A 245     -12.215   5.226  30.087  1.00  0.00           C  \\nATOM   3832  HA  PRO A 245     -12.891   2.139  29.620  1.00  0.00           H  \\nATOM   3833 1HB  PRO A 245     -13.898   4.395  27.889  1.00  0.00           H  \\nATOM   3834 2HB  PRO A 245     -14.793   3.371  29.019  1.00  0.00           H  \\nATOM   3835 1HG  PRO A 245     -14.243   5.893  29.681  1.00  0.00           H  \\nATOM   3836 2HG  PRO A 245     -14.138   4.615  30.895  1.00  0.00           H  \\nATOM   3837 1HD  PRO A 245     -11.893   5.948  29.322  1.00  0.00           H  \\nATOM   3838 2HD  PRO A 245     -11.999   5.608  31.096  1.00  0.00           H  \\nATOM   3839  N   ILE A 246     -10.993   3.157  27.251  1.00  0.00           N  \\nATOM   3840  CA  ILE A 246     -10.548   3.207  25.860  1.00  0.00           C  \\nATOM   3841  C   ILE A 246      -9.445   2.152  25.651  1.00  0.00           C  \\nATOM   3842  O   ILE A 246      -9.436   1.454  24.665  1.00  0.00           O  \\nATOM   3843  CB  ILE A 246      -9.937   4.612  25.552  1.00  0.00           C  \\nATOM   3844  CG1 ILE A 246     -11.010   5.718  25.636  1.00  0.00           C  \\nATOM   3845  CG2 ILE A 246      -9.498   4.721  24.096  1.00  0.00           C  \\nATOM   3846  CD1 ILE A 246     -12.331   5.432  24.953  1.00  0.00           C  \\nATOM   3847  H   ILE A 246     -10.392   3.526  27.974  1.00  0.00           H  \\nATOM   3848  HA  ILE A 246     -11.411   3.046  25.215  1.00  0.00           H  \\nATOM   3849  HB  ILE A 246      -9.068   4.778  26.188  1.00  0.00           H  \\nATOM   3850 1HG1 ILE A 246     -11.235   5.928  26.681  1.00  0.00           H  \\nATOM   3851 2HG1 ILE A 246     -10.621   6.636  25.195  1.00  0.00           H  \\nATOM   3852 1HG2 ILE A 246      -9.078   5.710  23.916  1.00  0.00           H  \\nATOM   3853 2HG2 ILE A 246      -8.744   3.964  23.886  1.00  0.00           H  \\nATOM   3854 3HG2 ILE A 246     -10.358   4.567  23.444  1.00  0.00           H  \\nATOM   3855 1HD1 ILE A 246     -12.999   6.284  25.082  1.00  0.00           H  \\nATOM   3856 2HD1 ILE A 246     -12.162   5.262  23.889  1.00  0.00           H  \\nATOM   3857 3HD1 ILE A 246     -12.784   4.546  25.395  1.00  0.00           H  \\nATOM   3858  N   ILE A 247      -8.493   2.108  26.607  1.00  0.00           N  \\nATOM   3859  CA  ILE A 247      -7.323   1.235  26.493  1.00  0.00           C  \\nATOM   3860  C   ILE A 247      -7.397   0.038  27.442  1.00  0.00           C  \\nATOM   3861  O   ILE A 247      -7.642   0.194  28.621  1.00  0.00           O  \\nATOM   3862  CB  ILE A 247      -5.989   2.041  26.634  1.00  0.00           C  \\nATOM   3863  CG1 ILE A 247      -4.765   1.116  26.421  1.00  0.00           C  \\nATOM   3864  CG2 ILE A 247      -5.894   2.679  27.947  1.00  0.00           C  \\nATOM   3865  CD1 ILE A 247      -3.415   1.930  26.148  1.00  0.00           C  \\nATOM   3866  H   ILE A 247      -8.592   2.694  27.423  1.00  0.00           H  \\nATOM   3867  HA  ILE A 247      -7.335   0.768  25.509  1.00  0.00           H  \\nATOM   3868  HB  ILE A 247      -5.949   2.815  25.868  1.00  0.00           H  \\nATOM   3869 1HG1 ILE A 247      -4.626   0.489  27.301  1.00  0.00           H  \\nATOM   3870 2HG1 ILE A 247      -4.952   0.454  25.575  1.00  0.00           H  \\nATOM   3871 1HG2 ILE A 247      -4.957   3.231  28.016  1.00  0.00           H  \\nATOM   3872 2HG2 ILE A 247      -6.730   3.365  28.079  1.00  0.00           H  \\nATOM   3873 3HG2 ILE A 247      -5.924   1.916  28.725  1.00  0.00           H  \\nATOM   3874 1HD1 ILE A 247      -2.591   1.231  26.006  1.00  0.00           H  \\nATOM   3875 2HD1 ILE A 247      -3.532   2.540  25.251  1.00  0.00           H  \\nATOM   3876 3HD1 ILE A 247      -3.200   2.574  27.000  1.00  0.00           H  \\nATOM   3877  N   ALA A 248      -7.216  -1.155  26.890  1.00  0.00           N  \\nATOM   3878  CA  ALA A 248      -7.411  -2.402  27.614  1.00  0.00           C  \\nATOM   3879  C   ALA A 248      -6.127  -3.039  28.076  1.00  0.00           C  \\nATOM   3880  O   ALA A 248      -6.135  -3.818  29.061  1.00  0.00           O  \\nATOM   3881  CB  ALA A 248      -8.239  -3.423  26.731  1.00  0.00           C  \\nATOM   3882  H   ALA A 248      -6.930  -1.188  25.922  1.00  0.00           H  \\nATOM   3883  HA  ALA A 248      -7.972  -2.182  28.522  1.00  0.00           H  \\nATOM   3884 1HB  ALA A 248      -8.378  -4.352  27.284  1.00  0.00           H  \\nATOM   3885 2HB  ALA A 248      -9.212  -2.993  26.493  1.00  0.00           H  \\nATOM   3886 3HB  ALA A 248      -7.698  -3.628  25.808  1.00  0.00           H  \\nATOM   3887  N   LEU A 249      -5.039  -2.801  27.373  1.00  0.00           N  \\nATOM   3888  CA  LEU A 249      -3.733  -3.374  27.661  1.00  0.00           C  \\nATOM   3889  C   LEU A 249      -2.632  -2.354  27.383  1.00  0.00           C  \\nATOM   3890  O   LEU A 249      -2.756  -1.570  26.565  1.00  0.00           O  \\nATOM   3891  CB  LEU A 249      -3.404  -4.520  26.682  1.00  0.00           C  \\nATOM   3892  CG  LEU A 249      -4.285  -5.719  26.616  1.00  0.00           C  \\nATOM   3893  CD1 LEU A 249      -4.050  -6.572  25.419  1.00  0.00           C  \\nATOM   3894  CD2 LEU A 249      -4.282  -6.503  27.802  1.00  0.00           C  \\nATOM   3895  H   LEU A 249      -5.139  -2.176  26.586  1.00  0.00           H  \\nATOM   3896  HA  LEU A 249      -3.749  -3.780  28.672  1.00  0.00           H  \\nATOM   3897 1HB  LEU A 249      -3.386  -4.118  25.670  1.00  0.00           H  \\nATOM   3898 2HB  LEU A 249      -2.411  -4.903  26.916  1.00  0.00           H  \\nATOM   3899  HG  LEU A 249      -5.314  -5.405  26.440  1.00  0.00           H  \\nATOM   3900 1HD1 LEU A 249      -4.729  -7.425  25.441  1.00  0.00           H  \\nATOM   3901 2HD1 LEU A 249      -4.230  -5.989  24.516  1.00  0.00           H  \\nATOM   3902 3HD1 LEU A 249      -3.020  -6.928  25.423  1.00  0.00           H  \\nATOM   3903 1HD2 LEU A 249      -4.948  -7.357  27.675  1.00  0.00           H  \\nATOM   3904 2HD2 LEU A 249      -3.271  -6.858  28.001  1.00  0.00           H  \\nATOM   3905 3HD2 LEU A 249      -4.625  -5.896  28.640  1.00  0.00           H  \\nATOM   3906  N   MET A 250      -1.520  -2.468  28.092  1.00  0.00           N  \\nATOM   3907  CA  MET A 250      -0.362  -1.632  27.848  1.00  0.00           C  \\nATOM   3908  C   MET A 250       0.905  -2.474  27.752  1.00  0.00           C  \\nATOM   3909  O   MET A 250       1.041  -3.403  28.444  1.00  0.00           O  \\nATOM   3910  CB  MET A 250      -0.161  -0.647  29.005  1.00  0.00           C  \\nATOM   3911  CG  MET A 250      -1.233   0.286  29.106  1.00  0.00           C  \\nATOM   3912  SD  MET A 250      -0.976   1.582  30.336  1.00  0.00           S  \\nATOM   3913  CE  MET A 250      -1.037   0.597  31.797  1.00  0.00           C  \\nATOM   3914  H   MET A 250      -1.481  -3.161  28.826  1.00  0.00           H  \\nATOM   3915  HA  MET A 250      -0.531  -1.065  26.933  1.00  0.00           H  \\nATOM   3916 1HB  MET A 250      -0.079  -1.198  29.941  1.00  0.00           H  \\nATOM   3917 2HB  MET A 250       0.773  -0.103  28.862  1.00  0.00           H  \\nATOM   3918 1HG  MET A 250      -1.385   0.772  28.143  1.00  0.00           H  \\nATOM   3919 2HG  MET A 250      -2.150  -0.241  29.370  1.00  0.00           H  \\nATOM   3920 1HE  MET A 250      -0.896   1.233  32.671  1.00  0.00           H  \\nATOM   3921 2HE  MET A 250      -2.006   0.101  31.861  1.00  0.00           H  \\nATOM   3922 3HE  MET A 250      -0.247  -0.154  31.763  1.00  0.00           H  \\nATOM   3923  N   VAL A 251       1.829  -2.029  26.917  1.00  0.00           N  \\nATOM   3924  CA  VAL A 251       3.111  -2.615  26.911  1.00  0.00           C  \\nATOM   3925  C   VAL A 251       3.850  -2.190  28.179  1.00  0.00           C  \\nATOM   3926  O   VAL A 251       3.754  -1.083  28.581  1.00  0.00           O  \\nATOM   3927  CB  VAL A 251       3.881  -2.247  25.664  1.00  0.00           C  \\nATOM   3928  CG1 VAL A 251       3.934  -0.846  25.474  1.00  0.00           C  \\nATOM   3929  CG2 VAL A 251       5.265  -2.854  25.681  1.00  0.00           C  \\nATOM   3930  H   VAL A 251       1.631  -1.271  26.279  1.00  0.00           H  \\nATOM   3931  HA  VAL A 251       2.996  -3.699  26.932  1.00  0.00           H  \\nATOM   3932  HB  VAL A 251       3.342  -2.619  24.792  1.00  0.00           H  \\nATOM   3933 1HG1 VAL A 251       4.496  -0.623  24.567  1.00  0.00           H  \\nATOM   3934 2HG1 VAL A 251       2.922  -0.453  25.379  1.00  0.00           H  \\nATOM   3935 3HG1 VAL A 251       4.426  -0.381  26.328  1.00  0.00           H  \\nATOM   3936 1HG2 VAL A 251       5.796  -2.574  24.772  1.00  0.00           H  \\nATOM   3937 2HG2 VAL A 251       5.813  -2.487  26.550  1.00  0.00           H  \\nATOM   3938 3HG2 VAL A 251       5.185  -3.940  25.734  1.00  0.00           H  \\nATOM   3939  N   TYR A 252       4.579  -3.123  28.768  1.00  0.00           N  \\nATOM   3940  CA  TYR A 252       5.309  -2.901  30.012  1.00  0.00           C  \\nATOM   3941  C   TYR A 252       6.391  -1.888  29.820  1.00  0.00           C  \\nATOM   3942  O   TYR A 252       7.198  -2.025  28.955  1.00  0.00           O  \\nATOM   3943  CB  TYR A 252       5.964  -4.190  30.465  1.00  0.00           C  \\nATOM   3944  CG  TYR A 252       6.696  -4.125  31.785  1.00  0.00           C  \\nATOM   3945  CD1 TYR A 252       7.976  -3.681  31.878  1.00  0.00           C  \\nATOM   3946  CD2 TYR A 252       6.096  -4.540  32.908  1.00  0.00           C  \\nATOM   3947  CE1 TYR A 252       8.585  -3.622  33.098  1.00  0.00           C  \\nATOM   3948  CE2 TYR A 252       6.712  -4.502  34.109  1.00  0.00           C  \\nATOM   3949  CZ  TYR A 252       7.934  -4.055  34.198  1.00  0.00           C  \\nATOM   3950  OH  TYR A 252       8.475  -4.076  35.451  1.00  0.00           O  \\nATOM   3951  H   TYR A 252       4.625  -4.030  28.326  1.00  0.00           H  \\nATOM   3952  HA  TYR A 252       4.601  -2.578  30.776  1.00  0.00           H  \\nATOM   3953 1HB  TYR A 252       5.208  -4.971  30.555  1.00  0.00           H  \\nATOM   3954 2HB  TYR A 252       6.683  -4.516  29.714  1.00  0.00           H  \\nATOM   3955  HD1 TYR A 252       8.513  -3.373  30.980  1.00  0.00           H  \\nATOM   3956  HD2 TYR A 252       5.077  -4.925  32.871  1.00  0.00           H  \\nATOM   3957  HE1 TYR A 252       9.612  -3.266  33.178  1.00  0.00           H  \\nATOM   3958  HE2 TYR A 252       6.188  -4.845  35.001  1.00  0.00           H  \\nATOM   3959  HH  TYR A 252       7.832  -4.431  36.070  1.00  0.00           H  \\nATOM   3960  N   ARG A 253       6.421  -0.876  30.681  1.00  0.00           N  \\nATOM   3961  CA  ARG A 253       7.480   0.094  30.637  1.00  0.00           C  \\nATOM   3962  C   ARG A 253       8.390   0.078  31.872  1.00  0.00           C  \\nATOM   3963  O   ARG A 253       9.609   0.085  31.760  1.00  0.00           O  \\nATOM   3964  CB  ARG A 253       6.921   1.506  30.436  1.00  0.00           C  \\nATOM   3965  CG  ARG A 253       6.321   1.825  29.092  1.00  0.00           C  \\nATOM   3966  CD  ARG A 253       7.107   1.374  27.938  1.00  0.00           C  \\nATOM   3967  NE  ARG A 253       6.574   1.923  26.724  1.00  0.00           N  \\nATOM   3968  CZ  ARG A 253       6.806   1.457  25.535  1.00  0.00           C  \\nATOM   3969  NH1 ARG A 253       7.555   0.419  25.365  1.00  0.00           N  \\nATOM   3970  NH2 ARG A 253       6.274   2.046  24.498  1.00  0.00           N  \\nATOM   3971  H   ARG A 253       5.695  -0.781  31.377  1.00  0.00           H  \\nATOM   3972  HA  ARG A 253       8.131  -0.142  29.794  1.00  0.00           H  \\nATOM   3973 1HB  ARG A 253       6.142   1.697  31.172  1.00  0.00           H  \\nATOM   3974 2HB  ARG A 253       7.712   2.237  30.600  1.00  0.00           H  \\nATOM   3975 1HG  ARG A 253       5.343   1.350  29.008  1.00  0.00           H  \\nATOM   3976 2HG  ARG A 253       6.210   2.905  28.990  1.00  0.00           H  \\nATOM   3977 1HD  ARG A 253       8.140   1.701  28.051  1.00  0.00           H  \\nATOM   3978 2HD  ARG A 253       7.075   0.287  27.879  1.00  0.00           H  \\nATOM   3979  HE  ARG A 253       5.973   2.733  26.798  1.00  0.00           H  \\nATOM   3980 1HH1 ARG A 253       7.969  -0.041  26.163  1.00  0.00           H  \\nATOM   3981 2HH1 ARG A 253       7.727   0.067  24.434  1.00  0.00           H  \\nATOM   3982 1HH2 ARG A 253       5.686   2.859  24.625  1.00  0.00           H  \\nATOM   3983 2HH2 ARG A 253       6.450   1.690  23.570  1.00  0.00           H  \\nATOM   3984  N   CYS A 254       7.793   0.073  33.028  1.00  0.00           N  \\nATOM   3985  CA  CYS A 254       8.563   0.171  34.249  1.00  0.00           C  \\nATOM   3986  C   CYS A 254       7.880  -0.659  35.315  1.00  0.00           C  \\nATOM   3987  O   CYS A 254       6.740  -1.044  35.188  1.00  0.00           O  \\nATOM   3988  CB  CYS A 254       8.779   1.619  34.726  1.00  0.00           C  \\nATOM   3989  SG  CYS A 254       7.444   2.885  34.418  1.00  0.00           S  \\nATOM   3990  H   CYS A 254       6.787   0.001  33.085  1.00  0.00           H  \\nATOM   3991  HA  CYS A 254       9.549  -0.259  34.072  1.00  0.00           H  \\nATOM   3992 1HB  CYS A 254       8.940   1.627  35.804  1.00  0.00           H  \\nATOM   3993 2HB  CYS A 254       9.675   2.026  34.258  1.00  0.00           H  \\nATOM   3994  HG  CYS A 254       8.084   3.902  34.986  1.00  0.00           H  \\nATOM   3995  N   ALA A 255       8.637  -0.960  36.347  1.00  0.00           N  \\nATOM   3996  CA  ALA A 255       8.153  -1.771  37.425  1.00  0.00           C  \\nATOM   3997  C   ALA A 255       7.132  -0.958  38.146  1.00  0.00           C  \\nATOM   3998  O   ALA A 255       7.087   0.214  37.930  1.00  0.00           O  \\nATOM   3999  OXT ALA A 255       6.383  -1.486  38.920  1.00  0.00           O  \\nATOM   4000  CB  ALA A 255       9.274  -2.127  38.331  1.00  0.00           C  \\nATOM   4001  H   ALA A 255       9.584  -0.611  36.381  1.00  0.00           H  \\nATOM   4002  HA  ALA A 255       7.733  -2.683  37.001  1.00  0.00           H  \\nATOM   4003 1HB  ALA A 255       8.900  -2.743  39.148  1.00  0.00           H  \\nATOM   4004 2HB  ALA A 255      10.029  -2.682  37.775  1.00  0.00           H  \\nATOM   4005 3HB  ALA A 255       9.717  -1.218  38.736  1.00  0.00           H  \\nTER                                                                             \\nATOM   4007  N   TYR B 256      20.163  27.071  23.881  1.00  0.00           N  \\nATOM   4008  CA  TYR B 256      20.779  25.704  23.868  1.00  0.00           C  \\nATOM   4009  C   TYR B 256      22.287  25.786  23.845  1.00  0.00           C  \\nATOM   4010  O   TYR B 256      22.840  26.596  23.112  1.00  0.00           O  \\nATOM   4011  CB  TYR B 256      20.268  24.831  22.672  1.00  0.00           C  \\nATOM   4012  CG  TYR B 256      18.792  24.511  22.692  1.00  0.00           C  \\nATOM   4013  CD1 TYR B 256      18.223  23.933  23.826  1.00  0.00           C  \\nATOM   4014  CD2 TYR B 256      17.956  24.793  21.587  1.00  0.00           C  \\nATOM   4015  CE1 TYR B 256      16.848  23.646  23.884  1.00  0.00           C  \\nATOM   4016  CE2 TYR B 256      16.581  24.486  21.599  1.00  0.00           C  \\nATOM   4017  CZ  TYR B 256      16.067  23.948  22.811  1.00  0.00           C  \\nATOM   4018  OH  TYR B 256      14.731  23.636  22.786  1.00  0.00           O  \\nATOM   4019 1H   TYR B 256      19.167  26.988  23.896  1.00  0.00           H  \\nATOM   4020 2H   TYR B 256      20.468  27.567  24.694  1.00  0.00           H  \\nATOM   4021 3H   TYR B 256      20.443  27.571  23.062  1.00  0.00           H  \\nATOM   4022  HA  TYR B 256      20.507  25.193  24.792  1.00  0.00           H  \\nATOM   4023 1HB  TYR B 256      20.478  25.343  21.732  1.00  0.00           H  \\nATOM   4024 2HB  TYR B 256      20.807  23.884  22.657  1.00  0.00           H  \\nATOM   4025  HD1 TYR B 256      18.847  23.696  24.688  1.00  0.00           H  \\nATOM   4026  HD2 TYR B 256      18.372  25.259  20.694  1.00  0.00           H  \\nATOM   4027  HE1 TYR B 256      16.423  23.194  24.780  1.00  0.00           H  \\nATOM   4028  HE2 TYR B 256      15.975  24.663  20.710  1.00  0.00           H  \\nATOM   4029  HH  TYR B 256      14.371  23.846  21.921  1.00  0.00           H  \\nATOM   4030  N   PRO B 257      22.972  24.918  24.592  1.00  0.00           N  \\nATOM   4031  CA  PRO B 257      24.421  24.990  24.590  1.00  0.00           C  \\nATOM   4032  C   PRO B 257      24.989  24.604  23.211  1.00  0.00           C  \\nATOM   4033  O   PRO B 257      24.434  23.709  22.556  1.00  0.00           O  \\nATOM   4034  CB  PRO B 257      24.831  23.918  25.614  1.00  0.00           C  \\nATOM   4035  CG  PRO B 257      23.606  23.538  26.321  1.00  0.00           C  \\nATOM   4036  CD  PRO B 257      22.496  23.788  25.385  1.00  0.00           C  \\nATOM   4037  HA  PRO B 257      24.734  25.990  24.924  1.00  0.00           H  \\nATOM   4038 1HB  PRO B 257      25.288  23.061  25.098  1.00  0.00           H  \\nATOM   4039 2HB  PRO B 257      25.590  24.323  26.299  1.00  0.00           H  \\nATOM   4040 1HG  PRO B 257      23.654  22.481  26.623  1.00  0.00           H  \\nATOM   4041 2HG  PRO B 257      23.501  24.128  27.244  1.00  0.00           H  \\nATOM   4042 1HD  PRO B 257      22.333  22.896  24.762  1.00  0.00           H  \\nATOM   4043 2HD  PRO B 257      21.587  24.036  25.952  1.00  0.00           H  \\nATOM   4044  N   ILE B 258      26.048  25.284  22.778  1.00  0.00           N  \\nATOM   4045  CA  ILE B 258      26.786  24.989  21.531  1.00  0.00           C  \\nATOM   4046  C   ILE B 258      28.186  24.449  21.849  1.00  0.00           C  \\nATOM   4047  O   ILE B 258      28.981  25.052  22.624  1.00  0.00           O  \\nATOM   4048  CB  ILE B 258      26.936  26.246  20.594  1.00  0.00           C  \\nATOM   4049  CG1 ILE B 258      25.551  26.855  20.249  1.00  0.00           C  \\nATOM   4050  CG2 ILE B 258      27.669  25.883  19.341  1.00  0.00           C  \\nATOM   4051  CD1 ILE B 258      25.649  28.273  19.519  1.00  0.00           C  \\nATOM   4052  H   ILE B 258      26.351  26.052  23.359  1.00  0.00           H  \\nATOM   4053  HA  ILE B 258      26.236  24.231  20.975  1.00  0.00           H  \\nATOM   4054  HB  ILE B 258      27.492  27.024  21.116  1.00  0.00           H  \\nATOM   4055 1HG1 ILE B 258      25.003  26.170  19.603  1.00  0.00           H  \\nATOM   4056 2HG1 ILE B 258      24.969  26.977  21.163  1.00  0.00           H  \\nATOM   4057 1HG2 ILE B 258      27.762  26.764  18.706  1.00  0.00           H  \\nATOM   4058 2HG2 ILE B 258      28.661  25.512  19.595  1.00  0.00           H  \\nATOM   4059 3HG2 ILE B 258      27.117  25.108  18.808  1.00  0.00           H  \\nATOM   4060 1HD1 ILE B 258      24.646  28.643  19.306  1.00  0.00           H  \\nATOM   4061 2HD1 ILE B 258      26.167  28.981  20.167  1.00  0.00           H  \\nATOM   4062 3HD1 ILE B 258      26.200  28.164  18.586  1.00  0.00           H  \\nATOM   4063  N   ILE B 259      28.531  23.324  21.238  1.00  0.00           N  \\nATOM   4064  CA  ILE B 259      29.807  22.702  21.371  1.00  0.00           C  \\nATOM   4065  C   ILE B 259      30.473  22.735  19.985  1.00  0.00           C  \\nATOM   4066  O   ILE B 259      29.839  22.294  19.017  1.00  0.00           O  \\nATOM   4067  CB  ILE B 259      29.724  21.184  21.748  1.00  0.00           C  \\nATOM   4068  CG1 ILE B 259      29.077  20.938  23.065  1.00  0.00           C  \\nATOM   4069  CG2 ILE B 259      31.178  20.638  22.013  1.00  0.00           C  \\nATOM   4070  CD1 ILE B 259      27.632  20.865  23.025  1.00  0.00           C  \\nATOM   4071  H   ILE B 259      27.834  22.895  20.646  1.00  0.00           H  \\nATOM   4072  HA  ILE B 259      30.354  23.203  22.169  1.00  0.00           H  \\nATOM   4073  HB  ILE B 259      29.269  20.630  20.928  1.00  0.00           H  \\nATOM   4074 1HG1 ILE B 259      29.446  20.002  23.482  1.00  0.00           H  \\nATOM   4075 2HG1 ILE B 259      29.349  21.734  23.759  1.00  0.00           H  \\nATOM   4076 1HG2 ILE B 259      31.128  19.582  22.277  1.00  0.00           H  \\nATOM   4077 2HG2 ILE B 259      31.781  20.759  21.114  1.00  0.00           H  \\nATOM   4078 3HG2 ILE B 259      31.632  21.195  22.833  1.00  0.00           H  \\nATOM   4079 1HD1 ILE B 259      27.249  20.684  24.030  1.00  0.00           H  \\nATOM   4080 2HD1 ILE B 259      27.230  21.806  22.648  1.00  0.00           H  \\nATOM   4081 3HD1 ILE B 259      27.327  20.051  22.369  1.00  0.00           H  \\nATOM   4082  N   ASN B 260      31.728  23.183  19.883  1.00  0.00           N  \\nATOM   4083  CA  ASN B 260      32.328  23.386  18.557  1.00  0.00           C  \\nATOM   4084  C   ASN B 260      33.450  22.342  18.266  1.00  0.00           C  \\nATOM   4085  O   ASN B 260      34.133  21.819  19.182  1.00  0.00           O  \\nATOM   4086  CB  ASN B 260      33.050  24.758  18.501  1.00  0.00           C  \\nATOM   4087  CG  ASN B 260      32.227  25.901  18.966  1.00  0.00           C  \\nATOM   4088  OD1 ASN B 260      32.518  26.507  20.027  1.00  0.00           O  \\nATOM   4089  ND2 ASN B 260      31.250  26.257  18.188  1.00  0.00           N  \\nATOM   4090  H   ASN B 260      32.276  23.386  20.707  1.00  0.00           H  \\nATOM   4091  HA  ASN B 260      31.531  23.377  17.811  1.00  0.00           H  \\nATOM   4092 1HB  ASN B 260      33.950  24.721  19.116  1.00  0.00           H  \\nATOM   4093 2HB  ASN B 260      33.361  24.964  17.477  1.00  0.00           H  \\nATOM   4094 1HD2 ASN B 260      30.658  27.021  18.446  1.00  0.00           H  \\nATOM   4095 2HD2 ASN B 260      31.089  25.767  17.332  1.00  0.00           H  \\nATOM   4096  N   PHE B 261      33.605  22.029  16.984  1.00  0.00           N  \\nATOM   4097  CA  PHE B 261      34.721  21.149  16.533  1.00  0.00           C  \\nATOM   4098  C   PHE B 261      35.032  21.565  15.122  1.00  0.00           C  \\nATOM   4099  O   PHE B 261      34.098  21.846  14.328  1.00  0.00           O  \\nATOM   4100  CB  PHE B 261      34.315  19.669  16.588  1.00  0.00           C  \\nATOM   4101  CG  PHE B 261      35.403  18.758  16.057  1.00  0.00           C  \\nATOM   4102  CD1 PHE B 261      36.558  18.538  16.815  1.00  0.00           C  \\nATOM   4103  CD2 PHE B 261      35.224  18.056  14.840  1.00  0.00           C  \\nATOM   4104  CE1 PHE B 261      37.556  17.693  16.399  1.00  0.00           C  \\nATOM   4105  CE2 PHE B 261      36.200  17.186  14.434  1.00  0.00           C  \\nATOM   4106  CZ  PHE B 261      37.377  17.075  15.171  1.00  0.00           C  \\nATOM   4107  H   PHE B 261      32.957  22.392  16.299  1.00  0.00           H  \\nATOM   4108  HA  PHE B 261      35.570  21.297  17.201  1.00  0.00           H  \\nATOM   4109 1HB  PHE B 261      34.090  19.392  17.617  1.00  0.00           H  \\nATOM   4110 2HB  PHE B 261      33.409  19.519  16.003  1.00  0.00           H  \\nATOM   4111  HD1 PHE B 261      36.674  19.052  17.770  1.00  0.00           H  \\nATOM   4112  HD2 PHE B 261      34.301  18.146  14.266  1.00  0.00           H  \\nATOM   4113  HE1 PHE B 261      38.433  17.536  17.026  1.00  0.00           H  \\nATOM   4114  HE2 PHE B 261      36.060  16.582  13.538  1.00  0.00           H  \\nATOM   4115  HZ  PHE B 261      38.176  16.472  14.742  1.00  0.00           H  \\nATOM   4116  N   THR B 262      36.330  21.558  14.815  1.00  0.00           N  \\nATOM   4117  CA  THR B 262      36.727  21.708  13.455  1.00  0.00           C  \\nATOM   4118  C   THR B 262      37.588  20.577  12.951  1.00  0.00           C  \\nATOM   4119  O   THR B 262      38.427  20.041  13.717  1.00  0.00           O  \\nATOM   4120  CB  THR B 262      37.512  23.049  13.226  1.00  0.00           C  \\nATOM   4121  OG1 THR B 262      37.798  23.131  11.850  1.00  0.00           O  \\nATOM   4122  CG2 THR B 262      38.825  23.191  14.055  1.00  0.00           C  \\nATOM   4123  H   THR B 262      37.040  21.449  15.525  1.00  0.00           H  \\nATOM   4124  HA  THR B 262      35.831  21.730  12.834  1.00  0.00           H  \\nATOM   4125  HB  THR B 262      36.876  23.893  13.493  1.00  0.00           H  \\nATOM   4126  HG1 THR B 262      37.446  22.357  11.404  1.00  0.00           H  \\nATOM   4127 1HG2 THR B 262      39.296  24.148  13.830  1.00  0.00           H  \\nATOM   4128 2HG2 THR B 262      38.591  23.142  15.118  1.00  0.00           H  \\nATOM   4129 3HG2 THR B 262      39.507  22.382  13.797  1.00  0.00           H  \\nATOM   4130  N   THR B 263      37.353  20.161  11.713  1.00  0.00           N  \\nATOM   4131  CA  THR B 263      38.202  19.091  11.130  1.00  0.00           C  \\nATOM   4132  C   THR B 263      39.564  19.669  10.732  1.00  0.00           C  \\nATOM   4133  O   THR B 263      40.518  18.929  10.492  1.00  0.00           O  \\nATOM   4134  CB  THR B 263      37.591  18.376   9.970  1.00  0.00           C  \\nATOM   4135  OG1 THR B 263      37.181  19.413   9.005  1.00  0.00           O  \\nATOM   4136  CG2 THR B 263      36.366  17.599  10.373  1.00  0.00           C  \\nATOM   4137  H   THR B 263      36.608  20.556  11.157  1.00  0.00           H  \\nATOM   4138  HA  THR B 263      38.387  18.339  11.898  1.00  0.00           H  \\nATOM   4139  HB  THR B 263      38.317  17.682   9.547  1.00  0.00           H  \\nATOM   4140  HG1 THR B 263      37.404  20.280   9.354  1.00  0.00           H  \\nATOM   4141 1HG2 THR B 263      35.952  17.094   9.501  1.00  0.00           H  \\nATOM   4142 2HG2 THR B 263      36.636  16.860  11.127  1.00  0.00           H  \\nATOM   4143 3HG2 THR B 263      35.622  18.280  10.783  1.00  0.00           H  \\nATOM   4144  N   ALA B 264      39.649  21.006  10.640  1.00  0.00           N  \\nATOM   4145  CA  ALA B 264      40.932  21.678  10.229  1.00  0.00           C  \\nATOM   4146  C   ALA B 264      42.070  21.467  11.236  1.00  0.00           C  \\nATOM   4147  O   ALA B 264      41.995  21.989  12.304  1.00  0.00           O  \\nATOM   4148  CB  ALA B 264      40.691  23.194  10.005  1.00  0.00           C  \\nATOM   4149  H   ALA B 264      38.841  21.575  10.849  1.00  0.00           H  \\nATOM   4150  HA  ALA B 264      41.265  21.229   9.293  1.00  0.00           H  \\nATOM   4151 1HB  ALA B 264      41.625  23.671   9.708  1.00  0.00           H  \\nATOM   4152 2HB  ALA B 264      39.947  23.332   9.220  1.00  0.00           H  \\nATOM   4153 3HB  ALA B 264      40.331  23.645  10.928  1.00  0.00           H  \\nATOM   4154  N   GLY B 265      43.053  20.591  10.926  1.00  0.00           N  \\nATOM   4155  CA  GLY B 265      44.182  20.394  11.770  1.00  0.00           C  \\nATOM   4156  C   GLY B 265      43.711  19.527  12.956  1.00  0.00           C  \\nATOM   4157  O   GLY B 265      44.231  19.612  14.040  1.00  0.00           O  \\nATOM   4158  H   GLY B 265      42.981  20.062  10.069  1.00  0.00           H  \\nATOM   4159 1HA  GLY B 265      44.979  19.910  11.205  1.00  0.00           H  \\nATOM   4160 2HA  GLY B 265      44.563  21.360  12.100  1.00  0.00           H  \\nATOM   4161  N   ALA B 266      42.640  18.772  12.766  1.00  0.00           N  \\nATOM   4162  CA  ALA B 266      42.113  18.009  13.948  1.00  0.00           C  \\nATOM   4163  C   ALA B 266      43.158  17.067  14.493  1.00  0.00           C  \\nATOM   4164  O   ALA B 266      43.873  16.446  13.697  1.00  0.00           O  \\nATOM   4165  CB  ALA B 266      40.941  17.151  13.489  1.00  0.00           C  \\nATOM   4166  H   ALA B 266      42.169  18.688  11.876  1.00  0.00           H  \\nATOM   4167  HA  ALA B 266      41.773  18.730  14.692  1.00  0.00           H  \\nATOM   4168 1HB  ALA B 266      40.547  16.590  14.336  1.00  0.00           H  \\nATOM   4169 2HB  ALA B 266      40.158  17.792  13.082  1.00  0.00           H  \\nATOM   4170 3HB  ALA B 266      41.278  16.457  12.720  1.00  0.00           H  \\nATOM   4171  N   THR B 267      43.229  16.883  15.825  1.00  0.00           N  \\nATOM   4172  CA  THR B 267      44.104  15.921  16.364  1.00  0.00           C  \\nATOM   4173  C   THR B 267      43.362  14.849  17.163  1.00  0.00           C  \\nATOM   4174  O   THR B 267      42.179  15.072  17.459  1.00  0.00           O  \\nATOM   4175  CB  THR B 267      45.153  16.661  17.255  1.00  0.00           C  \\nATOM   4176  OG1 THR B 267      44.503  17.302  18.364  1.00  0.00           O  \\nATOM   4177  CG2 THR B 267      45.905  17.663  16.358  1.00  0.00           C  \\nATOM   4178  H   THR B 267      42.656  17.432  16.450  1.00  0.00           H  \\nATOM   4179  HA  THR B 267      44.611  15.415  15.542  1.00  0.00           H  \\nATOM   4180  HB  THR B 267      45.844  15.935  17.682  1.00  0.00           H  \\nATOM   4181  HG1 THR B 267      43.558  17.137  18.319  1.00  0.00           H  \\nATOM   4182 1HG2 THR B 267      46.647  18.197  16.952  1.00  0.00           H  \\nATOM   4183 2HG2 THR B 267      46.403  17.126  15.551  1.00  0.00           H  \\nATOM   4184 3HG2 THR B 267      45.197  18.376  15.936  1.00  0.00           H  \\nATOM   4185  N   VAL B 268      44.025  13.779  17.658  1.00  0.00           N  \\nATOM   4186  CA  VAL B 268      43.414  12.868  18.579  1.00  0.00           C  \\nATOM   4187  C   VAL B 268      42.916  13.661  19.837  1.00  0.00           C  \\nATOM   4188  O   VAL B 268      41.772  13.437  20.248  1.00  0.00           O  \\nATOM   4189  CB  VAL B 268      44.449  11.772  19.075  1.00  0.00           C  \\nATOM   4190  CG1 VAL B 268      43.913  10.930  20.154  1.00  0.00           C  \\nATOM   4191  CG2 VAL B 268      44.719  10.906  17.883  1.00  0.00           C  \\nATOM   4192  H   VAL B 268      44.979  13.619  17.368  1.00  0.00           H  \\nATOM   4193  HA  VAL B 268      42.594  12.359  18.070  1.00  0.00           H  \\nATOM   4194  HB  VAL B 268      45.354  12.271  19.422  1.00  0.00           H  \\nATOM   4195 1HG1 VAL B 268      44.665  10.199  20.453  1.00  0.00           H  \\nATOM   4196 2HG1 VAL B 268      43.655  11.555  21.009  1.00  0.00           H  \\nATOM   4197 3HG1 VAL B 268      43.023  10.410  19.801  1.00  0.00           H  \\nATOM   4198 1HG2 VAL B 268      45.429  10.125  18.154  1.00  0.00           H  \\nATOM   4199 2HG2 VAL B 268      43.788  10.450  17.546  1.00  0.00           H  \\nATOM   4200 3HG2 VAL B 268      45.137  11.513  17.079  1.00  0.00           H  \\nATOM   4201  N   GLN B 269      43.709  14.607  20.364  1.00  0.00           N  \\nATOM   4202  CA  GLN B 269      43.309  15.336  21.591  1.00  0.00           C  \\nATOM   4203  C   GLN B 269      42.060  16.184  21.288  1.00  0.00           C  \\nATOM   4204  O   GLN B 269      41.162  16.183  22.115  1.00  0.00           O  \\nATOM   4205  CB  GLN B 269      44.452  16.231  22.141  1.00  0.00           C  \\nATOM   4206  CG  GLN B 269      44.178  16.617  23.711  1.00  0.00           C  \\nATOM   4207  CD  GLN B 269      43.906  15.359  24.734  1.00  0.00           C  \\nATOM   4208  OE1 GLN B 269      44.605  14.321  24.725  1.00  0.00           O  \\nATOM   4209  NE2 GLN B 269      42.936  15.522  25.624  1.00  0.00           N  \\nATOM   4210  H   GLN B 269      44.591  14.830  19.924  1.00  0.00           H  \\nATOM   4211  HA  GLN B 269      43.058  14.606  22.360  1.00  0.00           H  \\nATOM   4212 1HB  GLN B 269      45.402  15.705  22.050  1.00  0.00           H  \\nATOM   4213 2HB  GLN B 269      44.522  17.140  21.543  1.00  0.00           H  \\nATOM   4214 1HG  GLN B 269      45.047  17.146  24.101  1.00  0.00           H  \\nATOM   4215 2HG  GLN B 269      43.296  17.254  23.769  1.00  0.00           H  \\nATOM   4216 1HE2 GLN B 269      42.724  14.790  26.274  1.00  0.00           H  \\nATOM   4217 2HE2 GLN B 269      42.416  16.376  25.647  1.00  0.00           H  \\nATOM   4218  N   SER B 270      42.070  16.848  20.129  1.00  0.00           N  \\nATOM   4219  CA  SER B 270      40.987  17.835  19.901  1.00  0.00           C  \\nATOM   4220  C   SER B 270      39.702  17.059  19.691  1.00  0.00           C  \\nATOM   4221  O   SER B 270      38.647  17.543  20.151  1.00  0.00           O  \\nATOM   4222  CB  SER B 270      41.204  18.852  18.803  1.00  0.00           C  \\nATOM   4223  OG  SER B 270      41.164  18.257  17.532  1.00  0.00           O  \\nATOM   4224  H   SER B 270      42.769  16.711  19.413  1.00  0.00           H  \\nATOM   4225  HA  SER B 270      40.852  18.418  20.813  1.00  0.00           H  \\nATOM   4226 1HB  SER B 270      40.437  19.623  18.865  1.00  0.00           H  \\nATOM   4227 2HB  SER B 270      42.168  19.338  18.943  1.00  0.00           H  \\nATOM   4228  HG  SER B 270      41.004  17.322  17.682  1.00  0.00           H  \\nATOM   4229  N   TYR B 271      39.733  15.933  18.984  1.00  0.00           N  \\nATOM   4230  CA  TYR B 271      38.486  15.129  18.896  1.00  0.00           C  \\nATOM   4231  C   TYR B 271      38.072  14.516  20.197  1.00  0.00           C  \\nATOM   4232  O   TYR B 271      36.910  14.522  20.478  1.00  0.00           O  \\nATOM   4233  CB  TYR B 271      38.688  14.042  17.822  1.00  0.00           C  \\nATOM   4234  CG  TYR B 271      37.544  13.121  17.650  1.00  0.00           C  \\nATOM   4235  CD1 TYR B 271      36.414  13.503  16.884  1.00  0.00           C  \\nATOM   4236  CD2 TYR B 271      37.582  11.877  18.200  1.00  0.00           C  \\nATOM   4237  CE1 TYR B 271      35.329  12.654  16.751  1.00  0.00           C  \\nATOM   4238  CE2 TYR B 271      36.519  11.024  18.070  1.00  0.00           C  \\nATOM   4239  CZ  TYR B 271      35.416  11.447  17.320  1.00  0.00           C  \\nATOM   4240  OH  TYR B 271      34.299  10.627  17.179  1.00  0.00           O  \\nATOM   4241  H   TYR B 271      40.563  15.607  18.510  1.00  0.00           H  \\nATOM   4242  HA  TYR B 271      37.669  15.789  18.603  1.00  0.00           H  \\nATOM   4243 1HB  TYR B 271      38.881  14.514  16.858  1.00  0.00           H  \\nATOM   4244 2HB  TYR B 271      39.562  13.441  18.073  1.00  0.00           H  \\nATOM   4245  HD1 TYR B 271      36.393  14.475  16.391  1.00  0.00           H  \\nATOM   4246  HD2 TYR B 271      38.464  11.552  18.751  1.00  0.00           H  \\nATOM   4247  HE1 TYR B 271      34.467  12.960  16.159  1.00  0.00           H  \\nATOM   4248  HE2 TYR B 271      36.546  10.043  18.546  1.00  0.00           H  \\nATOM   4249  HH  TYR B 271      34.443   9.806  17.656  1.00  0.00           H  \\nATOM   4250  N   THR B 272      38.973  13.895  20.938  1.00  0.00           N  \\nATOM   4251  CA  THR B 272      38.692  13.348  22.259  1.00  0.00           C  \\nATOM   4252  C   THR B 272      37.968  14.468  23.184  1.00  0.00           C  \\nATOM   4253  O   THR B 272      36.957  14.208  23.826  1.00  0.00           O  \\nATOM   4254  CB  THR B 272      39.958  12.898  22.880  1.00  0.00           C  \\nATOM   4255  OG1 THR B 272      40.455  11.821  22.041  1.00  0.00           O  \\nATOM   4256  CG2 THR B 272      39.560  12.345  24.210  1.00  0.00           C  \\nATOM   4257  H   THR B 272      39.903  13.802  20.555  1.00  0.00           H  \\nATOM   4258  HA  THR B 272      38.024  12.494  22.145  1.00  0.00           H  \\nATOM   4259  HB  THR B 272      40.636  13.745  22.978  1.00  0.00           H  \\nATOM   4260  HG1 THR B 272      39.847  11.678  21.311  1.00  0.00           H  \\nATOM   4261 1HG2 THR B 272      40.446  11.989  24.736  1.00  0.00           H  \\nATOM   4262 2HG2 THR B 272      39.077  13.125  24.799  1.00  0.00           H  \\nATOM   4263 3HG2 THR B 272      38.867  11.517  24.067  1.00  0.00           H  \\nATOM   4264  N   ASN B 273      38.583  15.666  23.232  1.00  0.00           N  \\nATOM   4265  CA  ASN B 273      38.055  16.692  24.115  1.00  0.00           C  \\nATOM   4266  C   ASN B 273      36.679  17.096  23.610  1.00  0.00           C  \\nATOM   4267  O   ASN B 273      35.824  17.276  24.419  1.00  0.00           O  \\nATOM   4268  CB  ASN B 273      38.974  17.886  24.076  1.00  0.00           C  \\nATOM   4269  CG  ASN B 273      40.245  17.673  24.847  1.00  0.00           C  \\nATOM   4270  OD1 ASN B 273      40.416  16.704  25.603  1.00  0.00           O  \\nATOM   4271  ND2 ASN B 273      41.143  18.631  24.695  1.00  0.00           N  \\nATOM   4272  H   ASN B 273      39.399  15.870  22.673  1.00  0.00           H  \\nATOM   4273  HA  ASN B 273      38.022  16.292  25.130  1.00  0.00           H  \\nATOM   4274 1HB  ASN B 273      39.229  18.115  23.041  1.00  0.00           H  \\nATOM   4275 2HB  ASN B 273      38.459  18.755  24.487  1.00  0.00           H  \\nATOM   4276 1HD2 ASN B 273      42.021  18.574  25.171  1.00  0.00           H  \\nATOM   4277 2HD2 ASN B 273      40.945  19.413  24.104  1.00  0.00           H  \\nATOM   4278  N   PHE B 274      36.445  17.144  22.293  1.00  0.00           N  \\nATOM   4279  CA  PHE B 274      35.133  17.458  21.691  1.00  0.00           C  \\nATOM   4280  C   PHE B 274      34.168  16.407  22.156  1.00  0.00           C  \\nATOM   4281  O   PHE B 274      33.142  16.760  22.646  1.00  0.00           O  \\nATOM   4282  CB  PHE B 274      35.271  17.510  20.192  1.00  0.00           C  \\nATOM   4283  CG  PHE B 274      33.982  17.461  19.442  1.00  0.00           C  \\nATOM   4284  CD1 PHE B 274      33.036  18.404  19.682  1.00  0.00           C  \\nATOM   4285  CD2 PHE B 274      33.734  16.418  18.533  1.00  0.00           C  \\nATOM   4286  CE1 PHE B 274      31.801  18.419  18.995  1.00  0.00           C  \\nATOM   4287  CE2 PHE B 274      32.499  16.441  17.762  1.00  0.00           C  \\nATOM   4288  CZ  PHE B 274      31.537  17.402  18.012  1.00  0.00           C  \\nATOM   4289  H   PHE B 274      37.230  16.950  21.688  1.00  0.00           H  \\nATOM   4290  HA  PHE B 274      34.810  18.434  22.055  1.00  0.00           H  \\nATOM   4291 1HB  PHE B 274      35.785  18.427  19.906  1.00  0.00           H  \\nATOM   4292 2HB  PHE B 274      35.882  16.674  19.854  1.00  0.00           H  \\nATOM   4293  HD1 PHE B 274      33.231  19.176  20.427  1.00  0.00           H  \\nATOM   4294  HD2 PHE B 274      34.488  15.657  18.331  1.00  0.00           H  \\nATOM   4295  HE1 PHE B 274      31.066  19.193  19.213  1.00  0.00           H  \\nATOM   4296  HE2 PHE B 274      32.334  15.695  16.985  1.00  0.00           H  \\nATOM   4297  HZ  PHE B 274      30.589  17.394  17.476  1.00  0.00           H  \\nATOM   4298  N   ILE B 275      34.458  15.119  21.953  1.00  0.00           N  \\nATOM   4299  CA  ILE B 275      33.462  14.059  22.350  1.00  0.00           C  \\nATOM   4300  C   ILE B 275      33.131  14.116  23.841  1.00  0.00           C  \\nATOM   4301  O   ILE B 275      31.977  13.994  24.242  1.00  0.00           O  \\nATOM   4302  CB  ILE B 275      33.948  12.695  21.915  1.00  0.00           C  \\nATOM   4303  CG1 ILE B 275      33.942  12.678  20.359  1.00  0.00           C  \\nATOM   4304  CG2 ILE B 275      33.027  11.629  22.546  1.00  0.00           C  \\nATOM   4305  CD1 ILE B 275      32.570  12.956  19.569  1.00  0.00           C  \\nATOM   4306  H   ILE B 275      35.335  14.839  21.537  1.00  0.00           H  \\nATOM   4307  HA  ILE B 275      32.516  14.269  21.852  1.00  0.00           H  \\nATOM   4308  HB  ILE B 275      34.974  12.550  22.253  1.00  0.00           H  \\nATOM   4309 1HG1 ILE B 275      34.643  13.423  19.984  1.00  0.00           H  \\nATOM   4310 2HG1 ILE B 275      34.281  11.704  20.004  1.00  0.00           H  \\nATOM   4311 1HG2 ILE B 275      33.360  10.636  22.244  1.00  0.00           H  \\nATOM   4312 2HG2 ILE B 275      33.067  11.711  23.631  1.00  0.00           H  \\nATOM   4313 3HG2 ILE B 275      32.003  11.787  22.208  1.00  0.00           H  \\nATOM   4314 1HD1 ILE B 275      32.752  12.905  18.495  1.00  0.00           H  \\nATOM   4315 2HD1 ILE B 275      31.831  12.205  19.848  1.00  0.00           H  \\nATOM   4316 3HD1 ILE B 275      32.197  13.946  19.826  1.00  0.00           H  \\nATOM   4317  N   ARG B 276      34.182  14.325  24.651  1.00  0.00           N  \\nATOM   4318  CA  ARG B 276      33.971  14.472  26.111  1.00  0.00           C  \\nATOM   4319  C   ARG B 276      33.109  15.646  26.416  1.00  0.00           C  \\nATOM   4320  O   ARG B 276      32.283  15.529  27.371  1.00  0.00           O  \\nATOM   4321  CB  ARG B 276      35.326  14.622  26.794  1.00  0.00           C  \\nATOM   4322  CG  ARG B 276      36.020  13.223  26.923  1.00  0.00           C  \\nATOM   4323  CD  ARG B 276      37.405  13.450  27.455  1.00  0.00           C  \\nATOM   4324  NE  ARG B 276      38.122  12.201  27.528  1.00  0.00           N  \\nATOM   4325  CZ  ARG B 276      39.390  12.136  27.921  1.00  0.00           C  \\nATOM   4326  NH1 ARG B 276      40.024  13.252  28.295  1.00  0.00           N  \\nATOM   4327  NH2 ARG B 276      39.987  10.954  27.957  1.00  0.00           N  \\nATOM   4328  H   ARG B 276      35.120  14.384  24.281  1.00  0.00           H  \\nATOM   4329  HA  ARG B 276      33.476  13.573  26.481  1.00  0.00           H  \\nATOM   4330 1HB  ARG B 276      35.952  15.299  26.215  1.00  0.00           H  \\nATOM   4331 2HB  ARG B 276      35.189  15.066  27.780  1.00  0.00           H  \\nATOM   4332 1HG  ARG B 276      35.449  12.595  27.607  1.00  0.00           H  \\nATOM   4333 2HG  ARG B 276      36.063  12.746  25.943  1.00  0.00           H  \\nATOM   4334 1HD  ARG B 276      37.941  14.130  26.794  1.00  0.00           H  \\nATOM   4335 2HD  ARG B 276      37.345  13.885  28.452  1.00  0.00           H  \\nATOM   4336  HE  ARG B 276      37.627  11.359  27.268  1.00  0.00           H  \\nATOM   4337 1HH1 ARG B 276      39.539  14.138  28.278  1.00  0.00           H  \\nATOM   4338 2HH1 ARG B 276      40.987  13.209  28.595  1.00  0.00           H  \\nATOM   4339 1HH2 ARG B 276      39.479  10.123  27.688  1.00  0.00           H  \\nATOM   4340 2HH2 ARG B 276      40.949  10.884  28.253  1.00  0.00           H  \\nATOM   4341  N   ALA B 277      33.238  16.747  25.649  1.00  0.00           N  \\nATOM   4342  CA  ALA B 277      32.407  17.945  25.923  1.00  0.00           C  \\nATOM   4343  C   ALA B 277      30.951  17.616  25.570  1.00  0.00           C  \\nATOM   4344  O   ALA B 277      30.041  18.048  26.289  1.00  0.00           O  \\nATOM   4345  CB  ALA B 277      32.902  19.084  25.135  1.00  0.00           C  \\nATOM   4346  H   ALA B 277      33.897  16.770  24.884  1.00  0.00           H  \\nATOM   4347  HA  ALA B 277      32.491  18.180  26.984  1.00  0.00           H  \\nATOM   4348 1HB  ALA B 277      32.289  19.962  25.339  1.00  0.00           H  \\nATOM   4349 2HB  ALA B 277      33.937  19.293  25.408  1.00  0.00           H  \\nATOM   4350 3HB  ALA B 277      32.848  18.843  24.075  1.00  0.00           H  \\nATOM   4351  N   VAL B 278      30.762  16.933  24.445  1.00  0.00           N  \\nATOM   4352  CA  VAL B 278      29.401  16.588  24.030  1.00  0.00           C  \\nATOM   4353  C   VAL B 278      28.756  15.686  25.116  1.00  0.00           C  \\nATOM   4354  O   VAL B 278      27.648  15.935  25.526  1.00  0.00           O  \\nATOM   4355  CB  VAL B 278      29.426  15.850  22.685  1.00  0.00           C  \\nATOM   4356  CG1 VAL B 278      28.027  15.251  22.410  1.00  0.00           C  \\nATOM   4357  CG2 VAL B 278      29.883  16.834  21.520  1.00  0.00           C  \\nATOM   4358  H   VAL B 278      31.540  16.646  23.868  1.00  0.00           H  \\nATOM   4359  HA  VAL B 278      28.831  17.510  23.913  1.00  0.00           H  \\nATOM   4360  HB  VAL B 278      30.129  15.019  22.750  1.00  0.00           H  \\nATOM   4361 1HG1 VAL B 278      28.038  14.725  21.455  1.00  0.00           H  \\nATOM   4362 2HG1 VAL B 278      27.767  14.553  23.206  1.00  0.00           H  \\nATOM   4363 3HG1 VAL B 278      27.289  16.052  22.373  1.00  0.00           H  \\nATOM   4364 1HG2 VAL B 278      29.896  16.295  20.573  1.00  0.00           H  \\nATOM   4365 2HG2 VAL B 278      29.185  17.668  21.453  1.00  0.00           H  \\nATOM   4366 3HG2 VAL B 278      30.882  17.213  21.735  1.00  0.00           H  \\nATOM   4367  N   ARG B 279      29.445  14.657  25.581  1.00  0.00           N  \\nATOM   4368  CA  ARG B 279      28.935  13.810  26.677  1.00  0.00           C  \\nATOM   4369  C   ARG B 279      28.597  14.617  27.917  1.00  0.00           C  \\nATOM   4370  O   ARG B 279      27.569  14.355  28.536  1.00  0.00           O  \\nATOM   4371  CB  ARG B 279      29.925  12.743  27.095  1.00  0.00           C  \\nATOM   4372  CG  ARG B 279      30.167  11.684  26.002  1.00  0.00           C  \\nATOM   4373  CD  ARG B 279      31.143  10.667  26.560  1.00  0.00           C  \\nATOM   4374  NE  ARG B 279      31.384   9.713  25.506  1.00  0.00           N  \\nATOM   4375  CZ  ARG B 279      32.575   9.267  25.196  1.00  0.00           C  \\nATOM   4376  NH1 ARG B 279      33.652   9.621  25.932  1.00  0.00           N  \\nATOM   4377  NH2 ARG B 279      32.765   8.511  24.167  1.00  0.00           N  \\nATOM   4378  H   ARG B 279      30.345  14.446  25.174  1.00  0.00           H  \\nATOM   4379  HA  ARG B 279      28.031  13.306  26.333  1.00  0.00           H  \\nATOM   4380 1HB  ARG B 279      30.877  13.208  27.344  1.00  0.00           H  \\nATOM   4381 2HB  ARG B 279      29.562  12.241  27.991  1.00  0.00           H  \\nATOM   4382 1HG  ARG B 279      29.223  11.204  25.745  1.00  0.00           H  \\nATOM   4383 2HG  ARG B 279      30.583  12.164  25.116  1.00  0.00           H  \\nATOM   4384 1HD  ARG B 279      32.065  11.169  26.850  1.00  0.00           H  \\nATOM   4385 2HD  ARG B 279      30.705  10.182  27.432  1.00  0.00           H  \\nATOM   4386  HE  ARG B 279      30.585   9.377  24.985  1.00  0.00           H  \\nATOM   4387 1HH1 ARG B 279      33.541  10.234  26.727  1.00  0.00           H  \\nATOM   4388 2HH1 ARG B 279      34.568   9.273  25.687  1.00  0.00           H  \\nATOM   4389 1HH2 ARG B 279      31.984   8.251  23.580  1.00  0.00           H  \\nATOM   4390 2HH2 ARG B 279      33.693   8.180  23.949  1.00  0.00           H  \\nATOM   4391  N   GLY B 280      29.492  15.546  28.278  1.00  0.00           N  \\nATOM   4392  CA  GLY B 280      29.312  16.344  29.500  1.00  0.00           C  \\nATOM   4393  C   GLY B 280      28.108  17.254  29.400  1.00  0.00           C  \\nATOM   4394  O   GLY B 280      27.488  17.590  30.461  1.00  0.00           O  \\nATOM   4395  H   GLY B 280      30.307  15.704  27.703  1.00  0.00           H  \\nATOM   4396 1HA  GLY B 280      29.194  15.678  30.355  1.00  0.00           H  \\nATOM   4397 2HA  GLY B 280      30.205  16.942  29.679  1.00  0.00           H  \\nATOM   4398  N   ARG B 281      27.695  17.634  28.213  1.00  0.00           N  \\nATOM   4399  CA  ARG B 281      26.551  18.496  28.032  1.00  0.00           C  \\nATOM   4400  C   ARG B 281      25.260  17.755  27.758  1.00  0.00           C  \\nATOM   4401  O   ARG B 281      24.151  18.302  27.922  1.00  0.00           O  \\nATOM   4402  CB  ARG B 281      26.804  19.621  26.987  1.00  0.00           C  \\nATOM   4403  CG  ARG B 281      28.030  20.410  27.375  1.00  0.00           C  \\nATOM   4404  CD  ARG B 281      27.736  21.281  28.624  1.00  0.00           C  \\nATOM   4405  NE  ARG B 281      28.191  22.597  28.284  1.00  0.00           N  \\nATOM   4406  CZ  ARG B 281      27.529  23.730  28.354  1.00  0.00           C  \\nATOM   4407  NH1 ARG B 281      26.283  23.803  28.840  1.00  0.00           N  \\nATOM   4408  NH2 ARG B 281      28.166  24.794  27.913  1.00  0.00           N  \\nATOM   4409  H   ARG B 281      28.199  17.309  27.401  1.00  0.00           H  \\nATOM   4410  HA  ARG B 281      26.326  18.977  28.984  1.00  0.00           H  \\nATOM   4411 1HB  ARG B 281      26.936  19.178  26.001  1.00  0.00           H  \\nATOM   4412 2HB  ARG B 281      25.933  20.275  26.938  1.00  0.00           H  \\nATOM   4413 1HG  ARG B 281      28.847  19.725  27.603  1.00  0.00           H  \\nATOM   4414 2HG  ARG B 281      28.321  21.060  26.549  1.00  0.00           H  \\nATOM   4415 1HD  ARG B 281      26.666  21.270  28.832  1.00  0.00           H  \\nATOM   4416 2HD  ARG B 281      28.276  20.881  29.481  1.00  0.00           H  \\nATOM   4417  HE  ARG B 281      29.139  22.694  27.944  1.00  0.00           H  \\nATOM   4418 1HH1 ARG B 281      25.819  22.968  29.169  1.00  0.00           H  \\nATOM   4419 2HH1 ARG B 281      25.807  24.693  28.877  1.00  0.00           H  \\nATOM   4420 1HH2 ARG B 281      29.105  24.706  27.550  1.00  0.00           H  \\nATOM   4421 2HH2 ARG B 281      27.717  25.697  27.937  1.00  0.00           H  \\nATOM   4422  N   LEU B 282      25.405  16.499  27.369  1.00  0.00           N  \\nATOM   4423  CA  LEU B 282      24.175  15.699  27.165  1.00  0.00           C  \\nATOM   4424  C   LEU B 282      23.624  15.244  28.455  1.00  0.00           C  \\nATOM   4425  O   LEU B 282      22.400  15.132  28.563  1.00  0.00           O  \\nATOM   4426  CB  LEU B 282      24.462  14.461  26.314  1.00  0.00           C  \\nATOM   4427  CG  LEU B 282      24.741  14.715  24.863  1.00  0.00           C  \\nATOM   4428  CD1 LEU B 282      25.299  13.515  24.208  1.00  0.00           C  \\nATOM   4429  CD2 LEU B 282      23.415  15.136  24.100  1.00  0.00           C  \\nATOM   4430  H   LEU B 282      26.304  16.069  27.208  1.00  0.00           H  \\nATOM   4431  HA  LEU B 282      23.445  16.315  26.641  1.00  0.00           H  \\nATOM   4432 1HB  LEU B 282      25.326  13.946  26.730  1.00  0.00           H  \\nATOM   4433 2HB  LEU B 282      23.603  13.792  26.373  1.00  0.00           H  \\nATOM   4434  HG  LEU B 282      25.471  15.519  24.768  1.00  0.00           H  \\nATOM   4435 1HD1 LEU B 282      25.491  13.730  23.157  1.00  0.00           H  \\nATOM   4436 2HD1 LEU B 282      26.232  13.235  24.696  1.00  0.00           H  \\nATOM   4437 3HD1 LEU B 282      24.587  12.694  24.286  1.00  0.00           H  \\nATOM   4438 1HD2 LEU B 282      23.642  15.316  23.049  1.00  0.00           H  \\nATOM   4439 2HD2 LEU B 282      22.679  14.336  24.181  1.00  0.00           H  \\nATOM   4440 3HD2 LEU B 282      23.012  16.046  24.545  1.00  0.00           H  \\nATOM   4441  N   THR B 283      24.477  14.930  29.432  1.00  0.00           N  \\nATOM   4442  CA  THR B 283      23.982  14.408  30.731  1.00  0.00           C  \\nATOM   4443  C   THR B 283      24.412  15.250  31.918  1.00  0.00           C  \\nATOM   4444  O   THR B 283      25.448  15.896  31.882  1.00  0.00           O  \\nATOM   4445  CB  THR B 283      24.433  12.952  31.071  1.00  0.00           C  \\nATOM   4446  OG1 THR B 283      25.857  12.847  31.117  1.00  0.00           O  \\nATOM   4447  CG2 THR B 283      23.889  11.939  30.162  1.00  0.00           C  \\nATOM   4448  H   THR B 283      25.471  15.046  29.295  1.00  0.00           H  \\nATOM   4449  HA  THR B 283      22.892  14.395  30.706  1.00  0.00           H  \\nATOM   4450  HB  THR B 283      24.108  12.697  32.079  1.00  0.00           H  \\nATOM   4451  HG1 THR B 283      26.247  13.704  30.927  1.00  0.00           H  \\nATOM   4452 1HG2 THR B 283      24.245  10.953  30.461  1.00  0.00           H  \\nATOM   4453 2HG2 THR B 283      22.800  11.960  30.204  1.00  0.00           H  \\nATOM   4454 3HG2 THR B 283      24.217  12.150  29.145  1.00  0.00           H  \\nATOM   4455  N   THR B 284      23.623  15.177  32.982  1.00  0.00           N  \\nATOM   4456  CA  THR B 284      23.907  15.848  34.240  1.00  0.00           C  \\nATOM   4457  C   THR B 284      25.108  15.222  34.967  1.00  0.00           C  \\nATOM   4458  O   THR B 284      25.754  15.878  35.793  1.00  0.00           O  \\nATOM   4459  CB  THR B 284      22.695  15.700  35.176  1.00  0.00           C  \\nATOM   4460  OG1 THR B 284      23.000  16.396  36.387  1.00  0.00           O  \\nATOM   4461  CG2 THR B 284      22.354  14.206  35.471  1.00  0.00           C  \\nATOM   4462  H   THR B 284      22.783  14.622  32.899  1.00  0.00           H  \\nATOM   4463  HA  THR B 284      24.078  16.905  34.037  1.00  0.00           H  \\nATOM   4464  HB  THR B 284      21.823  16.165  34.718  1.00  0.00           H  \\nATOM   4465  HG1 THR B 284      23.877  16.782  36.322  1.00  0.00           H  \\nATOM   4466 1HG2 THR B 284      21.492  14.154  36.135  1.00  0.00           H  \\nATOM   4467 2HG2 THR B 284      22.124  13.694  34.537  1.00  0.00           H  \\nATOM   4468 3HG2 THR B 284      23.208  13.725  35.947  1.00  0.00           H  \\nATOM   4469  N   GLY B 285      25.369  13.943  34.699  1.00  0.00           N  \\nATOM   4470  CA  GLY B 285      26.281  13.183  35.572  1.00  0.00           C  \\nATOM   4471  C   GLY B 285      25.725  12.794  36.952  1.00  0.00           C  \\nATOM   4472  O   GLY B 285      26.423  12.137  37.748  1.00  0.00           O  \\nATOM   4473  H   GLY B 285      24.951  13.482  33.904  1.00  0.00           H  \\nATOM   4474 1HA  GLY B 285      26.578  12.260  35.073  1.00  0.00           H  \\nATOM   4475 2HA  GLY B 285      27.188  13.762  35.742  1.00  0.00           H  \\nATOM   4476  N   ALA B 286      24.502  13.225  37.262  1.00  0.00           N  \\nATOM   4477  CA  ALA B 286      23.805  12.825  38.523  1.00  0.00           C  \\nATOM   4478  C   ALA B 286      23.482  11.336  38.483  1.00  0.00           C  \\nATOM   4479  O   ALA B 286      23.760  10.548  39.464  1.00  0.00           O  \\nATOM   4480  CB  ALA B 286      22.554  13.650  38.730  1.00  0.00           C  \\nATOM   4481  H   ALA B 286      24.034  13.845  36.617  1.00  0.00           H  \\nATOM   4482  HA  ALA B 286      24.484  13.005  39.357  1.00  0.00           H  \\nATOM   4483 1HB  ALA B 286      22.063  13.342  39.653  1.00  0.00           H  \\nATOM   4484 2HB  ALA B 286      22.821  14.705  38.796  1.00  0.00           H  \\nATOM   4485 3HB  ALA B 286      21.876  13.499  37.891  1.00  0.00           H  \\nATOM   4486  N   ASP B 287      23.003  10.907  37.286  1.00  0.00           N  \\nATOM   4487  CA  ASP B 287      22.310   9.622  37.168  1.00  0.00           C  \\nATOM   4488  C   ASP B 287      23.280   8.649  36.455  1.00  0.00           C  \\nATOM   4489  O   ASP B 287      23.534   8.698  35.230  1.00  0.00           O  \\nATOM   4490  CB  ASP B 287      20.985   9.781  36.462  1.00  0.00           C  \\nATOM   4491  CG  ASP B 287      20.206   8.438  36.303  1.00  0.00           C  \\nATOM   4492  OD1 ASP B 287      20.722   7.374  36.849  1.00  0.00           O  \\nATOM   4493  OD2 ASP B 287      19.119   8.419  35.690  1.00  0.00           O  \\nATOM   4494  H   ASP B 287      23.124  11.478  36.462  1.00  0.00           H  \\nATOM   4495  HA  ASP B 287      22.120   9.238  38.171  1.00  0.00           H  \\nATOM   4496 1HB  ASP B 287      20.359  10.480  37.017  1.00  0.00           H  \\nATOM   4497 2HB  ASP B 287      21.149  10.204  35.471  1.00  0.00           H  \\nATOM   4498  N   VAL B 288      23.914   7.807  37.257  1.00  0.00           N  \\nATOM   4499  CA  VAL B 288      24.914   6.906  36.754  1.00  0.00           C  \\nATOM   4500  C   VAL B 288      24.551   5.556  37.301  1.00  0.00           C  \\nATOM   4501  O   VAL B 288      24.234   5.449  38.513  1.00  0.00           O  \\nATOM   4502  CB  VAL B 288      26.342   7.346  37.178  1.00  0.00           C  \\nATOM   4503  CG1 VAL B 288      27.406   6.610  36.455  1.00  0.00           C  \\nATOM   4504  CG2 VAL B 288      26.515   8.836  36.936  1.00  0.00           C  \\nATOM   4505  H   VAL B 288      23.692   7.796  38.242  1.00  0.00           H  \\nATOM   4506  HA  VAL B 288      24.867   6.910  35.664  1.00  0.00           H  \\nATOM   4507  HB  VAL B 288      26.479   7.130  38.238  1.00  0.00           H  \\nATOM   4508 1HG1 VAL B 288      28.383   6.958  36.792  1.00  0.00           H  \\nATOM   4509 2HG1 VAL B 288      27.312   5.543  36.657  1.00  0.00           H  \\nATOM   4510 3HG1 VAL B 288      27.308   6.787  35.384  1.00  0.00           H  \\nATOM   4511 1HG2 VAL B 288      27.518   9.139  37.236  1.00  0.00           H  \\nATOM   4512 2HG2 VAL B 288      26.372   9.052  35.877  1.00  0.00           H  \\nATOM   4513 3HG2 VAL B 288      25.779   9.388  37.522  1.00  0.00           H  \\nATOM   4514  N   ARG B 289      24.593   4.552  36.429  1.00  0.00           N  \\nATOM   4515  CA  ARG B 289      24.277   3.165  36.805  1.00  0.00           C  \\nATOM   4516  C   ARG B 289      25.340   2.229  36.265  1.00  0.00           C  \\nATOM   4517  O   ARG B 289      25.658   2.249  35.072  1.00  0.00           O  \\nATOM   4518  CB  ARG B 289      22.913   2.803  36.252  1.00  0.00           C  \\nATOM   4519  CG  ARG B 289      21.969   3.894  36.674  1.00  0.00           C  \\nATOM   4520  CD  ARG B 289      20.559   3.533  36.674  1.00  0.00           C  \\nATOM   4521  NE  ARG B 289      19.794   4.726  36.466  1.00  0.00           N  \\nATOM   4522  CZ  ARG B 289      18.542   4.698  36.007  1.00  0.00           C  \\nATOM   4523  NH1 ARG B 289      17.952   3.544  35.735  1.00  0.00           N  \\nATOM   4524  NH2 ARG B 289      17.910   5.835  35.769  1.00  0.00           N  \\nATOM   4525  H   ARG B 289      24.852   4.756  35.474  1.00  0.00           H  \\nATOM   4526  HA  ARG B 289      24.255   3.098  37.893  1.00  0.00           H  \\nATOM   4527 1HB  ARG B 289      22.971   2.717  35.168  1.00  0.00           H  \\nATOM   4528 2HB  ARG B 289      22.608   1.832  36.642  1.00  0.00           H  \\nATOM   4529 1HG  ARG B 289      22.207   4.208  37.691  1.00  0.00           H  \\nATOM   4530 2HG  ARG B 289      22.070   4.744  35.998  1.00  0.00           H  \\nATOM   4531 1HD  ARG B 289      20.366   2.819  35.873  1.00  0.00           H  \\nATOM   4532 2HD  ARG B 289      20.299   3.083  37.631  1.00  0.00           H  \\nATOM   4533  HE  ARG B 289      20.235   5.611  36.680  1.00  0.00           H  \\nATOM   4534 1HH1 ARG B 289      18.450   2.676  35.875  1.00  0.00           H  \\nATOM   4535 2HH1 ARG B 289      17.004   3.531  35.388  1.00  0.00           H  \\nATOM   4536 1HH2 ARG B 289      18.376   6.717  35.935  1.00  0.00           H  \\nATOM   4537 2HH2 ARG B 289      16.963   5.823  35.422  1.00  0.00           H  \\nATOM   4538  N   HIS B 290      25.940   1.421  37.145  1.00  0.00           N  \\nATOM   4539  CA  HIS B 290      27.057   0.622  36.712  1.00  0.00           C  \\nATOM   4540  C   HIS B 290      28.173   1.458  36.153  1.00  0.00           C  \\nATOM   4541  O   HIS B 290      28.875   1.016  35.259  1.00  0.00           O  \\nATOM   4542  CB  HIS B 290      26.639  -0.581  35.861  1.00  0.00           C  \\nATOM   4543  CG  HIS B 290      25.643  -1.450  36.582  1.00  0.00           C  \\nATOM   4544  ND1 HIS B 290      26.053  -2.303  37.588  1.00  0.00           N  \\nATOM   4545  CD2 HIS B 290      24.284  -1.511  36.562  1.00  0.00           C  \\nATOM   4546  CE1 HIS B 290      24.995  -2.922  38.093  1.00  0.00           C  \\nATOM   4547  NE2 HIS B 290      23.905  -2.431  37.525  1.00  0.00           N  \\nATOM   4548  H   HIS B 290      25.631   1.356  38.104  1.00  0.00           H  \\nATOM   4549  HA  HIS B 290      27.581   0.227  37.582  1.00  0.00           H  \\nATOM   4550 1HB  HIS B 290      26.202  -0.231  34.925  1.00  0.00           H  \\nATOM   4551 2HB  HIS B 290      27.519  -1.172  35.610  1.00  0.00           H  \\nATOM   4552  HD2 HIS B 290      23.617  -0.934  35.920  1.00  0.00           H  \\nATOM   4553  HE1 HIS B 290      25.016  -3.705  38.850  1.00  0.00           H  \\nATOM   4554  HE2 HIS B 290      22.954  -2.684  37.754  1.00  0.00           H  \\nATOM   4555  N   GLU B 291      28.363   2.661  36.696  1.00  0.00           N  \\nATOM   4556  CA  GLU B 291      29.468   3.525  36.217  1.00  0.00           C  \\nATOM   4557  C   GLU B 291      29.120   4.075  34.854  1.00  0.00           C  \\nATOM   4558  O   GLU B 291      30.019   4.640  34.185  1.00  0.00           O  \\nATOM   4559  CB  GLU B 291      30.847   2.785  36.083  1.00  0.00           C  \\nATOM   4560  CG  GLU B 291      31.463   2.107  37.307  1.00  0.00           C  \\nATOM   4561  CD  GLU B 291      31.606   3.052  38.463  1.00  0.00           C  \\nATOM   4562  OE1 GLU B 291      31.355   4.261  38.260  1.00  0.00           O  \\nATOM   4563  OE2 GLU B 291      31.970   2.593  39.587  1.00  0.00           O  \\nATOM   4564  H   GLU B 291      27.761   2.996  37.435  1.00  0.00           H  \\nATOM   4565  HA  GLU B 291      29.611   4.334  36.934  1.00  0.00           H  \\nATOM   4566 1HB  GLU B 291      30.766   1.997  35.334  1.00  0.00           H  \\nATOM   4567 2HB  GLU B 291      31.605   3.488  35.738  1.00  0.00           H  \\nATOM   4568 1HG  GLU B 291      30.830   1.271  37.603  1.00  0.00           H  \\nATOM   4569 2HG  GLU B 291      32.440   1.710  37.036  1.00  0.00           H  \\nATOM   4570  N   ILE B 292      27.882   3.840  34.399  1.00  0.00           N  \\nATOM   4571  CA  ILE B 292      27.461   4.290  33.040  1.00  0.00           C  \\nATOM   4572  C   ILE B 292      26.424   5.405  33.181  1.00  0.00           C  \\nATOM   4573  O   ILE B 292      25.323   5.198  33.647  1.00  0.00           O  \\nATOM   4574  CB  ILE B 292      26.940   3.164  32.145  1.00  0.00           C  \\nATOM   4575  CG1 ILE B 292      27.990   2.056  31.916  1.00  0.00           C  \\nATOM   4576  CG2 ILE B 292      26.614   3.783  30.660  1.00  0.00           C  \\nATOM   4577  CD1 ILE B 292      27.335   0.718  31.693  1.00  0.00           C  \\nATOM   4578  H   ILE B 292      27.219   3.350  34.982  1.00  0.00           H  \\nATOM   4579  HA  ILE B 292      28.325   4.722  32.536  1.00  0.00           H  \\nATOM   4580  HB  ILE B 292      26.033   2.745  32.579  1.00  0.00           H  \\nATOM   4581 1HG1 ILE B 292      28.604   2.310  31.053  1.00  0.00           H  \\nATOM   4582 2HG1 ILE B 292      28.651   1.997  32.781  1.00  0.00           H  \\nATOM   4583 1HG2 ILE B 292      26.242   2.992  30.010  1.00  0.00           H  \\nATOM   4584 2HG2 ILE B 292      25.860   4.564  30.749  1.00  0.00           H  \\nATOM   4585 3HG2 ILE B 292      27.524   4.205  30.234  1.00  0.00           H  \\nATOM   4586 1HD1 ILE B 292      28.102  -0.041  31.535  1.00  0.00           H  \\nATOM   4587 2HD1 ILE B 292      26.738   0.455  32.567  1.00  0.00           H  \\nATOM   4588 3HD1 ILE B 292      26.691   0.770  30.816  1.00  0.00           H  \\nATOM   4589  N   PRO B 293      26.797   6.617  32.730  1.00  0.00           N  \\nATOM   4590  CA  PRO B 293      25.805   7.717  32.780  1.00  0.00           C  \\nATOM   4591  C   PRO B 293      24.559   7.488  31.907  1.00  0.00           C  \\nATOM   4592  O   PRO B 293      24.635   6.904  30.764  1.00  0.00           O  \\nATOM   4593  CB  PRO B 293      26.566   8.918  32.228  1.00  0.00           C  \\nATOM   4594  CG  PRO B 293      27.993   8.594  32.436  1.00  0.00           C  \\nATOM   4595  CD  PRO B 293      28.129   7.097  32.349  1.00  0.00           C  \\nATOM   4596  HA  PRO B 293      25.511   7.892  33.825  1.00  0.00           H  \\nATOM   4597 1HB  PRO B 293      26.319   9.064  31.166  1.00  0.00           H  \\nATOM   4598 2HB  PRO B 293      26.261   9.832  32.758  1.00  0.00           H  \\nATOM   4599 1HG  PRO B 293      28.611   9.094  31.676  1.00  0.00           H  \\nATOM   4600 2HG  PRO B 293      28.329   8.968  33.415  1.00  0.00           H  \\nATOM   4601 1HD  PRO B 293      28.384   6.811  31.318  1.00  0.00           H  \\nATOM   4602 2HD  PRO B 293      28.906   6.758  33.049  1.00  0.00           H  \\nATOM   4603  N   VAL B 294      23.440   7.944  32.467  1.00  0.00           N  \\nATOM   4604  CA  VAL B 294      22.137   7.825  31.814  1.00  0.00           C  \\nATOM   4605  C   VAL B 294      21.701   9.253  31.330  1.00  0.00           C  \\nATOM   4606  O   VAL B 294      21.924  10.270  32.001  1.00  0.00           O  \\nATOM   4607  CB  VAL B 294      21.106   7.261  32.779  1.00  0.00           C  \\nATOM   4608  CG1 VAL B 294      19.811   6.999  32.064  1.00  0.00           C  \\nATOM   4609  CG2 VAL B 294      21.674   5.981  33.465  1.00  0.00           C  \\nATOM   4610  H   VAL B 294      23.498   8.386  33.373  1.00  0.00           H  \\nATOM   4611  HA  VAL B 294      22.234   7.141  30.970  1.00  0.00           H  \\nATOM   4612  HB  VAL B 294      20.881   8.013  33.536  1.00  0.00           H  \\nATOM   4613 1HG1 VAL B 294      19.082   6.596  32.767  1.00  0.00           H  \\nATOM   4614 2HG1 VAL B 294      19.432   7.931  31.644  1.00  0.00           H  \\nATOM   4615 3HG1 VAL B 294      19.977   6.280  31.262  1.00  0.00           H  \\nATOM   4616 1HG2 VAL B 294      20.933   5.580  34.156  1.00  0.00           H  \\nATOM   4617 2HG2 VAL B 294      21.905   5.233  32.706  1.00  0.00           H  \\nATOM   4618 3HG2 VAL B 294      22.582   6.234  34.013  1.00  0.00           H  \\nATOM   4619  N   LEU B 295      21.160   9.297  30.098  1.00  0.00           N  \\nATOM   4620  CA  LEU B 295      20.657  10.543  29.539  1.00  0.00           C  \\nATOM   4621  C   LEU B 295      19.375  10.956  30.257  1.00  0.00           C  \\nATOM   4622  O   LEU B 295      18.713  10.149  30.929  1.00  0.00           O  \\nATOM   4623  CB  LEU B 295      20.329  10.308  28.061  1.00  0.00           C  \\nATOM   4624  CG  LEU B 295      21.606  10.023  27.275  1.00  0.00           C  \\nATOM   4625  CD1 LEU B 295      21.279   9.568  25.920  1.00  0.00           C  \\nATOM   4626  CD2 LEU B 295      22.501  11.264  27.147  1.00  0.00           C  \\nATOM   4627  H   LEU B 295      21.100   8.452  29.548  1.00  0.00           H  \\nATOM   4628  HA  LEU B 295      21.435  11.300  29.627  1.00  0.00           H  \\nATOM   4629 1HB  LEU B 295      19.641   9.468  27.987  1.00  0.00           H  \\nATOM   4630 2HB  LEU B 295      19.830  11.195  27.670  1.00  0.00           H  \\nATOM   4631  HG  LEU B 295      22.179   9.245  27.781  1.00  0.00           H  \\nATOM   4632 1HD1 LEU B 295      22.199   9.368  25.371  1.00  0.00           H  \\nATOM   4633 2HD1 LEU B 295      20.686   8.655  25.976  1.00  0.00           H  \\nATOM   4634 3HD1 LEU B 295      20.710  10.341  25.404  1.00  0.00           H  \\nATOM   4635 1HD2 LEU B 295      23.397  11.010  26.580  1.00  0.00           H  \\nATOM   4636 2HD2 LEU B 295      21.956  12.054  26.629  1.00  0.00           H  \\nATOM   4637 3HD2 LEU B 295      22.786  11.611  28.140  1.00  0.00           H  \\nATOM   4638  N   PRO B 296      18.982  12.218  30.127  1.00  0.00           N  \\nATOM   4639  CA  PRO B 296      17.677  12.711  30.651  1.00  0.00           C  \\nATOM   4640  C   PRO B 296      16.446  11.951  30.199  1.00  0.00           C  \\nATOM   4641  O   PRO B 296      16.353  11.578  29.069  1.00  0.00           O  \\nATOM   4642  CB  PRO B 296      17.620  14.147  30.106  1.00  0.00           C  \\nATOM   4643  CG  PRO B 296      19.118  14.622  29.982  1.00  0.00           C  \\nATOM   4644  CD  PRO B 296      19.743  13.281  29.423  1.00  0.00           C  \\nATOM   4645  HA  PRO B 296      17.703  12.704  31.751  1.00  0.00           H  \\nATOM   4646 1HB  PRO B 296      17.102  14.161  29.136  1.00  0.00           H  \\nATOM   4647 2HB  PRO B 296      17.041  14.786  30.789  1.00  0.00           H  \\nATOM   4648 1HG  PRO B 296      19.187  15.491  29.311  1.00  0.00           H  \\nATOM   4649 2HG  PRO B 296      19.496  14.947  30.963  1.00  0.00           H  \\nATOM   4650 1HD  PRO B 296      19.590  13.230  28.335  1.00  0.00           H  \\nATOM   4651 2HD  PRO B 296      20.815  13.246  29.667  1.00  0.00           H  \\nATOM   4652  N   ASN B 297      15.449  11.779  31.110  1.00  0.00           N  \\nATOM   4653  CA  ASN B 297      14.218  11.112  30.708  1.00  0.00           C  \\nATOM   4654  C   ASN B 297      13.381  11.997  29.769  1.00  0.00           C  \\nATOM   4655  O   ASN B 297      13.240  13.161  30.094  1.00  0.00           O  \\nATOM   4656  CB  ASN B 297      13.443  10.745  31.974  1.00  0.00           C  \\nATOM   4657  CG  ASN B 297      12.103  10.038  31.708  1.00  0.00           C  \\nATOM   4658  OD1 ASN B 297      11.729   9.651  30.580  1.00  0.00           O  \\nATOM   4659  ND2 ASN B 297      11.420   9.692  32.866  1.00  0.00           N  \\nATOM   4660  H   ASN B 297      15.544  12.105  32.061  1.00  0.00           H  \\nATOM   4661  HA  ASN B 297      14.479  10.206  30.158  1.00  0.00           H  \\nATOM   4662 1HB  ASN B 297      14.053  10.089  32.597  1.00  0.00           H  \\nATOM   4663 2HB  ASN B 297      13.239  11.648  32.550  1.00  0.00           H  \\nATOM   4664 1HD2 ASN B 297      10.536   9.227  32.805  1.00  0.00           H  \\nATOM   4665 2HD2 ASN B 297      11.811   9.907  33.760  1.00  0.00           H  \\nATOM   4666  N   ARG B 298      12.940  11.514  28.611  1.00  0.00           N  \\nATOM   4667  CA  ARG B 298      12.093  12.281  27.692  1.00  0.00           C  \\nATOM   4668  C   ARG B 298      10.827  12.756  28.417  1.00  0.00           C  \\nATOM   4669  O   ARG B 298      10.361  13.885  28.212  1.00  0.00           O  \\nATOM   4670  CB  ARG B 298      11.735  11.436  26.397  1.00  0.00           C  \\nATOM   4671  CG  ARG B 298      11.112  12.218  25.129  1.00  0.00           C  \\nATOM   4672  CD  ARG B 298      11.424  11.442  23.754  1.00  0.00           C  \\nATOM   4673  NE  ARG B 298      10.531  11.635  22.561  1.00  0.00           N  \\nATOM   4674  CZ  ARG B 298       9.204  11.436  22.527  1.00  0.00           C  \\nATOM   4675  NH1 ARG B 298       8.523  11.082  23.637  1.00  0.00           N  \\nATOM   4676  NH2 ARG B 298       8.524  11.596  21.389  1.00  0.00           N  \\nATOM   4677  H   ARG B 298      13.208  10.572  28.363  1.00  0.00           H  \\nATOM   4678  HA  ARG B 298      12.640  13.171  27.379  1.00  0.00           H  \\nATOM   4679 1HB  ARG B 298      12.630  10.937  26.030  1.00  0.00           H  \\nATOM   4680 2HB  ARG B 298      11.012  10.662  26.656  1.00  0.00           H  \\nATOM   4681 1HG  ARG B 298      10.031  12.297  25.245  1.00  0.00           H  \\nATOM   4682 2HG  ARG B 298      11.544  13.217  25.069  1.00  0.00           H  \\nATOM   4683 1HD  ARG B 298      12.411  11.727  23.391  1.00  0.00           H  \\nATOM   4684 2HD  ARG B 298      11.399  10.367  23.929  1.00  0.00           H  \\nATOM   4685  HE  ARG B 298      10.959  11.943  21.698  1.00  0.00           H  \\nATOM   4686 1HH1 ARG B 298       9.012  10.962  24.513  1.00  0.00           H  \\nATOM   4687 2HH1 ARG B 298       7.525  10.937  23.592  1.00  0.00           H  \\nATOM   4688 1HH2 ARG B 298       9.008  11.869  20.545  1.00  0.00           H  \\nATOM   4689 2HH2 ARG B 298       7.526  11.444  21.372  1.00  0.00           H  \\nATOM   4690  N   VAL B 299      10.268  11.874  29.269  1.00  0.00           N  \\nATOM   4691  CA  VAL B 299       8.977  12.210  29.917  1.00  0.00           C  \\nATOM   4692  C   VAL B 299       9.092  13.366  30.929  1.00  0.00           C  \\nATOM   4693  O   VAL B 299       9.792  13.340  31.933  1.00  0.00           O  \\nATOM   4694  CB  VAL B 299       8.333  10.950  30.555  1.00  0.00           C  \\nATOM   4695  CG1 VAL B 299       6.924  11.251  31.084  1.00  0.00           C  \\nATOM   4696  CG2 VAL B 299       8.273   9.863  29.536  1.00  0.00           C  \\nATOM   4697  H   VAL B 299      10.707  10.989  29.477  1.00  0.00           H  \\nATOM   4698  HA  VAL B 299       8.298  12.597  29.156  1.00  0.00           H  \\nATOM   4699  HB  VAL B 299       8.941  10.632  31.402  1.00  0.00           H  \\nATOM   4700 1HG1 VAL B 299       6.500  10.348  31.525  1.00  0.00           H  \\nATOM   4701 2HG1 VAL B 299       6.979  12.033  31.841  1.00  0.00           H  \\nATOM   4702 3HG1 VAL B 299       6.290  11.584  30.262  1.00  0.00           H  \\nATOM   4703 1HG2 VAL B 299       7.822   8.975  29.978  1.00  0.00           H  \\nATOM   4704 2HG2 VAL B 299       7.672  10.192  28.688  1.00  0.00           H  \\nATOM   4705 3HG2 VAL B 299       9.282   9.627  29.196  1.00  0.00           H  \\nATOM   4706  N   GLY B 300       8.421  14.461  30.599  1.00  0.00           N  \\nATOM   4707  CA  GLY B 300       8.455  15.614  31.467  1.00  0.00           C  \\nATOM   4708  C   GLY B 300       9.692  16.518  31.265  1.00  0.00           C  \\nATOM   4709  O   GLY B 300       9.880  17.473  32.030  1.00  0.00           O  \\nATOM   4710  H   GLY B 300       7.883  14.503  29.745  1.00  0.00           H  \\nATOM   4711 1HA  GLY B 300       7.562  16.218  31.305  1.00  0.00           H  \\nATOM   4712 2HA  GLY B 300       8.438  15.288  32.506  1.00  0.00           H  \\nATOM   4713  N   LEU B 301      10.569  16.199  30.297  1.00  0.00           N  \\nATOM   4714  CA  LEU B 301      11.752  17.024  30.063  1.00  0.00           C  \\nATOM   4715  C   LEU B 301      11.350  18.383  29.451  1.00  0.00           C  \\nATOM   4716  O   LEU B 301      10.676  18.439  28.412  1.00  0.00           O  \\nATOM   4717  CB  LEU B 301      12.754  16.290  29.161  1.00  0.00           C  \\nATOM   4718  CG  LEU B 301      14.086  17.011  29.020  1.00  0.00           C  \\nATOM   4719  CD1 LEU B 301      14.862  17.098  30.278  1.00  0.00           C  \\nATOM   4720  CD2 LEU B 301      14.943  16.281  27.932  1.00  0.00           C  \\nATOM   4721  H   LEU B 301      10.416  15.383  29.721  1.00  0.00           H  \\nATOM   4722  HA  LEU B 301      12.230  17.223  31.022  1.00  0.00           H  \\nATOM   4723 1HB  LEU B 301      12.933  15.299  29.575  1.00  0.00           H  \\nATOM   4724 2HB  LEU B 301      12.311  16.173  28.172  1.00  0.00           H  \\nATOM   4725  HG  LEU B 301      13.911  18.044  28.719  1.00  0.00           H  \\nATOM   4726 1HD1 LEU B 301      15.797  17.626  30.093  1.00  0.00           H  \\nATOM   4727 2HD1 LEU B 301      14.283  17.639  31.026  1.00  0.00           H  \\nATOM   4728 3HD1 LEU B 301      15.078  16.094  30.642  1.00  0.00           H  \\nATOM   4729 1HD2 LEU B 301      15.901  16.790  27.821  1.00  0.00           H  \\nATOM   4730 2HD2 LEU B 301      15.114  15.248  28.236  1.00  0.00           H  \\nATOM   4731 3HD2 LEU B 301      14.412  16.296  26.980  1.00  0.00           H  \\nATOM   4732  N   PRO B 302      11.747  19.485  30.120  1.00  0.00           N  \\nATOM   4733  CA  PRO B 302      11.386  20.814  29.606  1.00  0.00           C  \\nATOM   4734  C   PRO B 302      12.060  21.062  28.274  1.00  0.00           C  \\nATOM   4735  O   PRO B 302      13.194  20.654  28.044  1.00  0.00           O  \\nATOM   4736  CB  PRO B 302      11.992  21.760  30.637  1.00  0.00           C  \\nATOM   4737  CG  PRO B 302      12.005  20.996  31.877  1.00  0.00           C  \\nATOM   4738  CD  PRO B 302      12.184  19.542  31.509  1.00  0.00           C  \\nATOM   4739  HA  PRO B 302      10.290  20.908  29.583  1.00  0.00           H  \\nATOM   4740 1HB  PRO B 302      12.999  22.067  30.319  1.00  0.00           H  \\nATOM   4741 2HB  PRO B 302      11.385  22.675  30.711  1.00  0.00           H  \\nATOM   4742 1HG  PRO B 302      12.820  21.344  32.529  1.00  0.00           H  \\nATOM   4743 2HG  PRO B 302      11.067  21.153  32.430  1.00  0.00           H  \\nATOM   4744 1HD  PRO B 302      13.244  19.266  31.609  1.00  0.00           H  \\nATOM   4745 2HD  PRO B 302      11.557  18.918  32.162  1.00  0.00           H  \\nATOM   4746  N   ILE B 303      11.309  21.720  27.424  1.00  0.00           N  \\nATOM   4747  CA  ILE B 303      11.772  21.948  26.045  1.00  0.00           C  \\nATOM   4748  C   ILE B 303      13.135  22.720  25.993  1.00  0.00           C  \\nATOM   4749  O   ILE B 303      13.978  22.481  25.102  1.00  0.00           O  \\nATOM   4750  CB  ILE B 303      10.625  22.566  25.249  1.00  0.00           C  \\nATOM   4751  CG1 ILE B 303      11.012  22.722  23.790  1.00  0.00           C  \\nATOM   4752  CG2 ILE B 303      10.188  23.946  25.914  1.00  0.00           C  \\nATOM   4753  CD1 ILE B 303      11.357  21.370  22.986  1.00  0.00           C  \\nATOM   4754  H   ILE B 303      10.407  22.077  27.705  1.00  0.00           H  \\nATOM   4755  HA  ILE B 303      12.045  20.988  25.609  1.00  0.00           H  \\nATOM   4756  HB  ILE B 303       9.776  21.884  25.245  1.00  0.00           H  \\nATOM   4757 1HG1 ILE B 303      10.202  23.211  23.249  1.00  0.00           H  \\nATOM   4758 2HG1 ILE B 303      11.890  23.364  23.713  1.00  0.00           H  \\nATOM   4759 1HG2 ILE B 303       9.370  24.383  25.342  1.00  0.00           H  \\nATOM   4760 2HG2 ILE B 303       9.860  23.769  26.937  1.00  0.00           H  \\nATOM   4761 3HG2 ILE B 303      11.035  24.632  25.918  1.00  0.00           H  \\nATOM   4762 1HD1 ILE B 303      11.618  21.615  21.956  1.00  0.00           H  \\nATOM   4763 2HD1 ILE B 303      12.198  20.867  23.464  1.00  0.00           H  \\nATOM   4764 3HD1 ILE B 303      10.489  20.712  22.992  1.00  0.00           H  \\nATOM   4765  N   ASN B 304      13.415  23.540  26.996  1.00  0.00           N  \\nATOM   4766  CA  ASN B 304      14.618  24.312  26.967  1.00  0.00           C  \\nATOM   4767  C   ASN B 304      15.844  23.498  27.410  1.00  0.00           C  \\nATOM   4768  O   ASN B 304      16.972  23.977  27.341  1.00  0.00           O  \\nATOM   4769  CB  ASN B 304      14.491  25.634  27.707  1.00  0.00           C  \\nATOM   4770  CG  ASN B 304      14.020  25.479  29.116  1.00  0.00           C  \\nATOM   4771  OD1 ASN B 304      13.252  24.554  29.477  1.00  0.00           O  \\nATOM   4772  ND2 ASN B 304      14.407  26.442  29.929  1.00  0.00           N  \\nATOM   4773  H   ASN B 304      12.791  23.631  27.785  1.00  0.00           H  \\nATOM   4774  HA  ASN B 304      14.858  24.541  25.927  1.00  0.00           H  \\nATOM   4775 1HB  ASN B 304      15.458  26.139  27.720  1.00  0.00           H  \\nATOM   4776 2HB  ASN B 304      13.791  26.281  27.178  1.00  0.00           H  \\nATOM   4777 1HD2 ASN B 304      14.138  26.421  30.893  1.00  0.00           H  \\nATOM   4778 2HD2 ASN B 304      14.969  27.193  29.583  1.00  0.00           H  \\nATOM   4779  N   GLN B 305      15.613  22.259  27.870  1.00  0.00           N  \\nATOM   4780  CA  GLN B 305      16.677  21.304  28.197  1.00  0.00           C  \\nATOM   4781  C   GLN B 305      16.737  20.167  27.170  1.00  0.00           C  \\nATOM   4782  O   GLN B 305      17.486  19.184  27.459  1.00  0.00           O  \\nATOM   4783  CB  GLN B 305      16.408  20.650  29.553  1.00  0.00           C  \\nATOM   4784  CG  GLN B 305      16.480  21.614  30.710  1.00  0.00           C  \\nATOM   4785  CD  GLN B 305      15.856  21.071  31.978  1.00  0.00           C  \\nATOM   4786  OE1 GLN B 305      15.217  21.829  32.699  1.00  0.00           O  \\nATOM   4787  NE2 GLN B 305      16.047  19.772  32.271  1.00  0.00           N  \\nATOM   4788  H   GLN B 305      14.649  21.983  27.993  1.00  0.00           H  \\nATOM   4789  HA  GLN B 305      17.622  21.844  28.251  1.00  0.00           H  \\nATOM   4790 1HB  GLN B 305      15.418  20.194  29.547  1.00  0.00           H  \\nATOM   4791 2HB  GLN B 305      17.133  19.855  29.724  1.00  0.00           H  \\nATOM   4792 1HG  GLN B 305      17.526  21.836  30.920  1.00  0.00           H  \\nATOM   4793 2HG  GLN B 305      15.951  22.528  30.441  1.00  0.00           H  \\nATOM   4794 1HE2 GLN B 305      15.648  19.382  33.102  1.00  0.00           H  \\nATOM   4795 2HE2 GLN B 305      16.588  19.195  31.659  1.00  0.00           H  \\nATOM   4796  N   ARG B 306      16.016  20.274  26.030  1.00  0.00           N  \\nATOM   4797  CA  ARG B 306      15.814  19.089  25.182  1.00  0.00           C  \\nATOM   4798  C   ARG B 306      16.964  18.830  24.205  1.00  0.00           C  \\nATOM   4799  O   ARG B 306      17.109  17.717  23.734  1.00  0.00           O  \\nATOM   4800  CB  ARG B 306      14.414  19.099  24.512  1.00  0.00           C  \\nATOM   4801  CG  ARG B 306      14.047  18.010  23.497  1.00  0.00           C  \\nATOM   4802  CD  ARG B 306      13.987  16.696  24.239  1.00  0.00           C  \\nATOM   4803  NE  ARG B 306      13.643  15.703  23.266  1.00  0.00           N  \\nATOM   4804  CZ  ARG B 306      12.435  15.434  22.810  1.00  0.00           C  \\nATOM   4805  NH1 ARG B 306      12.315  14.526  21.855  1.00  0.00           N  \\nATOM   4806  NH2 ARG B 306      11.361  16.068  23.307  1.00  0.00           N  \\nATOM   4807  H   ARG B 306      15.612  21.155  25.748  1.00  0.00           H  \\nATOM   4808  HA  ARG B 306      15.884  18.199  25.808  1.00  0.00           H  \\nATOM   4809 1HB  ARG B 306      13.643  19.039  25.279  1.00  0.00           H  \\nATOM   4810 2HB  ARG B 306      14.272  20.039  23.979  1.00  0.00           H  \\nATOM   4811 1HG  ARG B 306      13.079  18.239  23.051  1.00  0.00           H  \\nATOM   4812 2HG  ARG B 306      14.806  17.970  22.715  1.00  0.00           H  \\nATOM   4813 1HD  ARG B 306      14.958  16.486  24.686  1.00  0.00           H  \\nATOM   4814 2HD  ARG B 306      13.232  16.755  25.022  1.00  0.00           H  \\nATOM   4815  HE  ARG B 306      14.392  15.143  22.880  1.00  0.00           H  \\nATOM   4816 1HH1 ARG B 306      13.136  14.060  21.495  1.00  0.00           H  \\nATOM   4817 2HH1 ARG B 306      11.403  14.299  21.486  1.00  0.00           H  \\nATOM   4818 1HH2 ARG B 306      11.474  16.758  24.037  1.00  0.00           H  \\nATOM   4819 2HH2 ARG B 306      10.441  15.856  22.952  1.00  0.00           H  \\nATOM   4820  N   PHE B 307      17.624  19.921  23.810  1.00  0.00           N  \\nATOM   4821  CA  PHE B 307      18.607  19.850  22.683  1.00  0.00           C  \\nATOM   4822  C   PHE B 307      19.971  20.473  23.061  1.00  0.00           C  \\nATOM   4823  O   PHE B 307      20.103  21.264  24.030  1.00  0.00           O  \\nATOM   4824  CB  PHE B 307      18.066  20.559  21.412  1.00  0.00           C  \\nATOM   4825  CG  PHE B 307      16.782  19.991  20.919  1.00  0.00           C  \\nATOM   4826  CD1 PHE B 307      16.764  18.744  20.358  1.00  0.00           C  \\nATOM   4827  CD2 PHE B 307      15.552  20.688  21.099  1.00  0.00           C  \\nATOM   4828  CE1 PHE B 307      15.549  18.159  19.835  1.00  0.00           C  \\nATOM   4829  CE2 PHE B 307      14.402  20.175  20.666  1.00  0.00           C  \\nATOM   4830  CZ  PHE B 307      14.342  18.902  19.972  1.00  0.00           C  \\nATOM   4831  H   PHE B 307      17.465  20.806  24.269  1.00  0.00           H  \\nATOM   4832  HA  PHE B 307      18.780  18.801  22.440  1.00  0.00           H  \\nATOM   4833 1HB  PHE B 307      17.915  21.617  21.622  1.00  0.00           H  \\nATOM   4834 2HB  PHE B 307      18.803  20.486  20.613  1.00  0.00           H  \\nATOM   4835  HD1 PHE B 307      17.690  18.172  20.303  1.00  0.00           H  \\nATOM   4836  HD2 PHE B 307      15.560  21.648  21.617  1.00  0.00           H  \\nATOM   4837  HE1 PHE B 307      15.578  17.181  19.356  1.00  0.00           H  \\nATOM   4838  HE2 PHE B 307      13.488  20.741  20.845  1.00  0.00           H  \\nATOM   4839  HZ  PHE B 307      13.412  18.513  19.561  1.00  0.00           H  \\nATOM   4840  N   ILE B 308      21.023  19.939  22.423  1.00  0.00           N  \\nATOM   4841  CA  ILE B 308      22.325  20.584  22.370  1.00  0.00           C  \\nATOM   4842  C   ILE B 308      22.673  20.779  20.902  1.00  0.00           C  \\nATOM   4843  O   ILE B 308      22.152  20.108  19.993  1.00  0.00           O  \\nATOM   4844  CB  ILE B 308      23.453  19.914  23.177  1.00  0.00           C  \\nATOM   4845  CG1 ILE B 308      23.852  18.573  22.606  1.00  0.00           C  \\nATOM   4846  CG2 ILE B 308      23.055  19.783  24.715  1.00  0.00           C  \\nATOM   4847  CD1 ILE B 308      25.175  18.091  23.215  1.00  0.00           C  \\nATOM   4848  H   ILE B 308      20.896  19.049  21.962  1.00  0.00           H  \\nATOM   4849  HA  ILE B 308      22.229  21.590  22.776  1.00  0.00           H  \\nATOM   4850  HB  ILE B 308      24.360  20.513  23.101  1.00  0.00           H  \\nATOM   4851 1HG1 ILE B 308      23.067  17.845  22.806  1.00  0.00           H  \\nATOM   4852 2HG1 ILE B 308      23.954  18.654  21.523  1.00  0.00           H  \\nATOM   4853 1HG2 ILE B 308      23.868  19.306  25.262  1.00  0.00           H  \\nATOM   4854 2HG2 ILE B 308      22.872  20.774  25.129  1.00  0.00           H  \\nATOM   4855 3HG2 ILE B 308      22.153  19.178  24.807  1.00  0.00           H  \\nATOM   4856 1HD1 ILE B 308      25.441  17.124  22.788  1.00  0.00           H  \\nATOM   4857 2HD1 ILE B 308      25.962  18.814  22.995  1.00  0.00           H  \\nATOM   4858 3HD1 ILE B 308      25.064  17.992  24.294  1.00  0.00           H  \\nATOM   4859  N   LEU B 309      23.511  21.767  20.666  1.00  0.00           N  \\nATOM   4860  CA  LEU B 309      23.921  22.025  19.294  1.00  0.00           C  \\nATOM   4861  C   LEU B 309      25.437  21.807  19.153  1.00  0.00           C  \\nATOM   4862  O   LEU B 309      26.230  22.039  20.071  1.00  0.00           O  \\nATOM   4863  CB  LEU B 309      23.647  23.465  18.979  1.00  0.00           C  \\nATOM   4864  CG  LEU B 309      22.293  24.047  19.268  1.00  0.00           C  \\nATOM   4865  CD1 LEU B 309      22.365  25.589  18.984  1.00  0.00           C  \\nATOM   4866  CD2 LEU B 309      21.262  23.325  18.485  1.00  0.00           C  \\nATOM   4867  H   LEU B 309      23.876  22.349  21.406  1.00  0.00           H  \\nATOM   4868  HA  LEU B 309      23.335  21.388  18.632  1.00  0.00           H  \\nATOM   4869 1HB  LEU B 309      24.354  24.082  19.532  1.00  0.00           H  \\nATOM   4870 2HB  LEU B 309      23.812  23.625  17.914  1.00  0.00           H  \\nATOM   4871  HG  LEU B 309      22.074  23.948  20.332  1.00  0.00           H  \\nATOM   4872 1HD1 LEU B 309      21.394  26.042  19.185  1.00  0.00           H  \\nATOM   4873 2HD1 LEU B 309      23.117  26.043  19.629  1.00  0.00           H  \\nATOM   4874 3HD1 LEU B 309      22.633  25.755  17.941  1.00  0.00           H  \\nATOM   4875 1HD2 LEU B 309      20.280  23.749  18.698  1.00  0.00           H  \\nATOM   4876 2HD2 LEU B 309      21.478  23.424  17.421  1.00  0.00           H  \\nATOM   4877 3HD2 LEU B 309      21.269  22.270  18.760  1.00  0.00           H  \\nATOM   4878  N   VAL B 310      25.810  21.252  18.011  1.00  0.00           N  \\nATOM   4879  CA  VAL B 310      27.192  20.951  17.698  1.00  0.00           C  \\nATOM   4880  C   VAL B 310      27.547  21.762  16.405  1.00  0.00           C  \\nATOM   4881  O   VAL B 310      27.097  21.416  15.275  1.00  0.00           O  \\nATOM   4882  CB  VAL B 310      27.351  19.450  17.431  1.00  0.00           C  \\nATOM   4883  CG1 VAL B 310      28.776  19.108  16.924  1.00  0.00           C  \\nATOM   4884  CG2 VAL B 310      27.141  18.666  18.731  1.00  0.00           C  \\nATOM   4885  H   VAL B 310      25.093  21.032  17.334  1.00  0.00           H  \\nATOM   4886  HA  VAL B 310      27.809  21.225  18.555  1.00  0.00           H  \\nATOM   4887  HB  VAL B 310      26.609  19.141  16.694  1.00  0.00           H  \\nATOM   4888 1HG1 VAL B 310      28.851  18.035  16.745  1.00  0.00           H  \\nATOM   4889 2HG1 VAL B 310      28.970  19.644  15.995  1.00  0.00           H  \\nATOM   4890 3HG1 VAL B 310      29.509  19.402  17.675  1.00  0.00           H  \\nATOM   4891 1HG2 VAL B 310      27.254  17.600  18.535  1.00  0.00           H  \\nATOM   4892 2HG2 VAL B 310      27.879  18.980  19.469  1.00  0.00           H  \\nATOM   4893 3HG2 VAL B 310      26.139  18.860  19.115  1.00  0.00           H  \\nATOM   4894  N   GLU B 311      28.453  22.764  16.542  1.00  0.00           N  \\nATOM   4895  CA  GLU B 311      28.866  23.573  15.419  1.00  0.00           C  \\nATOM   4896  C   GLU B 311      30.124  22.981  14.844  1.00  0.00           C  \\nATOM   4897  O   GLU B 311      31.173  22.966  15.497  1.00  0.00           O  \\nATOM   4898  CB  GLU B 311      29.048  25.035  15.837  1.00  0.00           C  \\nATOM   4899  CG  GLU B 311      29.491  25.887  14.618  1.00  0.00           C  \\nATOM   4900  CD  GLU B 311      30.180  27.188  15.024  1.00  0.00           C  \\nATOM   4901  OE1 GLU B 311      29.480  28.217  15.063  1.00  0.00           O  \\nATOM   4902  OE2 GLU B 311      31.413  27.182  15.350  1.00  0.00           O  \\nATOM   4903  H   GLU B 311      28.848  22.948  17.453  1.00  0.00           H  \\nATOM   4904  HA  GLU B 311      28.089  23.527  14.655  1.00  0.00           H  \\nATOM   4905 1HB  GLU B 311      28.110  25.417  16.241  1.00  0.00           H  \\nATOM   4906 2HB  GLU B 311      29.795  25.097  16.629  1.00  0.00           H  \\nATOM   4907 1HG  GLU B 311      30.177  25.300  14.008  1.00  0.00           H  \\nATOM   4908 2HG  GLU B 311      28.616  26.118  14.012  1.00  0.00           H  \\nATOM   4909  N   LEU B 312      30.000  22.464  13.630  1.00  0.00           N  \\nATOM   4910  CA  LEU B 312      31.154  21.870  12.931  1.00  0.00           C  \\nATOM   4911  C   LEU B 312      31.681  22.767  11.845  1.00  0.00           C  \\nATOM   4912  O   LEU B 312      30.914  23.264  10.982  1.00  0.00           O  \\nATOM   4913  CB  LEU B 312      30.620  20.587  12.184  1.00  0.00           C  \\nATOM   4914  CG  LEU B 312      29.984  19.569  13.137  1.00  0.00           C  \\nATOM   4915  CD1 LEU B 312      29.324  18.498  12.174  1.00  0.00           C  \\nATOM   4916  CD2 LEU B 312      31.073  18.894  14.113  1.00  0.00           C  \\nATOM   4917  H   LEU B 312      29.100  22.475  13.172  1.00  0.00           H  \\nATOM   4918  HA  LEU B 312      31.903  21.594  13.673  1.00  0.00           H  \\nATOM   4919 1HB  LEU B 312      29.882  20.897  11.445  1.00  0.00           H  \\nATOM   4920 2HB  LEU B 312      31.453  20.118  11.661  1.00  0.00           H  \\nATOM   4921  HG  LEU B 312      29.236  20.067  13.755  1.00  0.00           H  \\nATOM   4922 1HD1 LEU B 312      28.840  17.723  12.769  1.00  0.00           H  \\nATOM   4923 2HD1 LEU B 312      28.582  18.986  11.542  1.00  0.00           H  \\nATOM   4924 3HD1 LEU B 312      30.094  18.047  11.549  1.00  0.00           H  \\nATOM   4925 1HD2 LEU B 312      30.578  18.180  14.772  1.00  0.00           H  \\nATOM   4926 2HD2 LEU B 312      31.825  18.377  13.517  1.00  0.00           H  \\nATOM   4927 3HD2 LEU B 312      31.554  19.668  14.712  1.00  0.00           H  \\nATOM   4928  N   SER B 313      32.999  22.877  11.810  1.00  0.00           N  \\nATOM   4929  CA  SER B 313      33.673  23.697  10.761  1.00  0.00           C  \\nATOM   4930  C   SER B 313      34.741  22.831  10.119  1.00  0.00           C  \\nATOM   4931  O   SER B 313      35.149  21.808  10.687  1.00  0.00           O  \\nATOM   4932  CB  SER B 313      34.353  24.931  11.370  1.00  0.00           C  \\nATOM   4933  OG  SER B 313      33.388  25.773  11.998  1.00  0.00           O  \\nATOM   4934  H   SER B 313      33.563  22.402  12.500  1.00  0.00           H  \\nATOM   4935  HA  SER B 313      32.919  24.038  10.051  1.00  0.00           H  \\nATOM   4936 1HB  SER B 313      35.099  24.613  12.098  1.00  0.00           H  \\nATOM   4937 2HB  SER B 313      34.874  25.482  10.588  1.00  0.00           H  \\nATOM   4938  HG  SER B 313      32.540  25.341  11.875  1.00  0.00           H  \\nATOM   4939  N   ASN B 314      35.204  23.225   8.909  1.00  0.00           N  \\nATOM   4940  CA  ASN B 314      36.186  22.438   8.248  1.00  0.00           C  \\nATOM   4941  C   ASN B 314      37.261  23.288   7.598  1.00  0.00           C  \\nATOM   4942  O   ASN B 314      37.231  24.515   7.696  1.00  0.00           O  \\nATOM   4943  CB  ASN B 314      35.520  21.484   7.224  1.00  0.00           C  \\nATOM   4944  CG  ASN B 314      34.823  22.216   6.085  1.00  0.00           C  \\nATOM   4945  OD1 ASN B 314      34.841  23.472   5.961  1.00  0.00           O  \\nATOM   4946  ND2 ASN B 314      34.141  21.456   5.306  1.00  0.00           N  \\nATOM   4947  H   ASN B 314      34.867  24.067   8.464  1.00  0.00           H  \\nATOM   4948  HA  ASN B 314      36.708  21.838   8.994  1.00  0.00           H  \\nATOM   4949 1HB  ASN B 314      36.275  20.822   6.798  1.00  0.00           H  \\nATOM   4950 2HB  ASN B 314      34.786  20.859   7.733  1.00  0.00           H  \\nATOM   4951 1HD2 ASN B 314      33.648  21.849   4.529  1.00  0.00           H  \\nATOM   4952 2HD2 ASN B 314      34.103  20.472   5.476  1.00  0.00           H  \\nATOM   4953  N   HIS B 315      38.212  22.592   6.968  1.00  0.00           N  \\nATOM   4954  CA  HIS B 315      39.301  23.341   6.302  1.00  0.00           C  \\nATOM   4955  C   HIS B 315      38.791  24.278   5.184  1.00  0.00           C  \\nATOM   4956  O   HIS B 315      39.403  25.352   4.949  1.00  0.00           O  \\nATOM   4957  CB  HIS B 315      40.337  22.331   5.804  1.00  0.00           C  \\nATOM   4958  CG  HIS B 315      41.423  22.905   4.935  1.00  0.00           C  \\nATOM   4959  ND1 HIS B 315      42.665  23.223   5.422  1.00  0.00           N  \\nATOM   4960  CD2 HIS B 315      41.467  23.141   3.608  1.00  0.00           C  \\nATOM   4961  CE1 HIS B 315      43.413  23.695   4.438  1.00  0.00           C  \\nATOM   4962  NE2 HIS B 315      42.698  23.673   3.328  1.00  0.00           N  \\nATOM   4963  H   HIS B 315      38.214  21.583   6.930  1.00  0.00           H  \\nATOM   4964  HA  HIS B 315      39.770  24.015   7.018  1.00  0.00           H  \\nATOM   4965 1HB  HIS B 315      40.819  21.853   6.658  1.00  0.00           H  \\nATOM   4966 2HB  HIS B 315      39.838  21.550   5.231  1.00  0.00           H  \\nATOM   4967  HD2 HIS B 315      40.651  22.990   2.901  1.00  0.00           H  \\nATOM   4968  HE1 HIS B 315      44.442  24.042   4.529  1.00  0.00           H  \\nATOM   4969  HE2 HIS B 315      43.005  23.994   2.421  1.00  0.00           H  \\nATOM   4970  N   ALA B 316      37.684  23.906   4.518  1.00  0.00           N  \\nATOM   4971  CA  ALA B 316      37.057  24.707   3.464  1.00  0.00           C  \\nATOM   4972  C   ALA B 316      36.424  25.997   4.059  1.00  0.00           C  \\nATOM   4973  O   ALA B 316      35.863  26.781   3.338  1.00  0.00           O  \\nATOM   4974  CB  ALA B 316      36.023  23.900   2.737  1.00  0.00           C  \\nATOM   4975  H   ALA B 316      37.273  23.019   4.772  1.00  0.00           H  \\nATOM   4976  HA  ALA B 316      37.832  25.004   2.758  1.00  0.00           H  \\nATOM   4977 1HB  ALA B 316      35.567  24.510   1.957  1.00  0.00           H  \\nATOM   4978 2HB  ALA B 316      36.494  23.027   2.285  1.00  0.00           H  \\nATOM   4979 3HB  ALA B 316      35.256  23.576   3.438  1.00  0.00           H  \\nATOM   4980  N   GLU B 317      36.527  26.185   5.384  1.00  0.00           N  \\nATOM   4981  CA  GLU B 317      35.963  27.317   6.122  1.00  0.00           C  \\nATOM   4982  C   GLU B 317      34.402  27.431   6.041  1.00  0.00           C  \\nATOM   4983  O   GLU B 317      33.818  28.536   6.104  1.00  0.00           O  \\nATOM   4984  CB  GLU B 317      36.663  28.600   5.613  1.00  0.00           C  \\nATOM   4985  CG  GLU B 317      38.111  28.607   6.044  1.00  0.00           C  \\nATOM   4986  CD  GLU B 317      38.907  29.777   5.518  1.00  0.00           C  \\nATOM   4987  OE1 GLU B 317      38.403  30.601   4.707  1.00  0.00           O  \\nATOM   4988  OE2 GLU B 317      40.091  29.813   5.906  1.00  0.00           O  \\nATOM   4989  H   GLU B 317      37.039  25.475   5.888  1.00  0.00           H  \\nATOM   4990  HA  GLU B 317      36.168  27.175   7.184  1.00  0.00           H  \\nATOM   4991 1HB  GLU B 317      36.595  28.645   4.526  1.00  0.00           H  \\nATOM   4992 2HB  GLU B 317      36.150  29.476   6.009  1.00  0.00           H  \\nATOM   4993 1HG  GLU B 317      38.155  28.628   7.133  1.00  0.00           H  \\nATOM   4994 2HG  GLU B 317      38.584  27.686   5.706  1.00  0.00           H  \\nATOM   4995  N   LEU B 318      33.727  26.274   5.970  1.00  0.00           N  \\nATOM   4996  CA  LEU B 318      32.281  26.196   5.945  1.00  0.00           C  \\nATOM   4997  C   LEU B 318      31.901  25.669   7.339  1.00  0.00           C  \\nATOM   4998  O   LEU B 318      32.663  24.950   7.979  1.00  0.00           O  \\nATOM   4999  CB  LEU B 318      31.831  25.238   4.873  1.00  0.00           C  \\nATOM   5000  CG  LEU B 318      32.201  25.532   3.423  1.00  0.00           C  \\nATOM   5001  CD1 LEU B 318      31.525  24.560   2.568  1.00  0.00           C  \\nATOM   5002  CD2 LEU B 318      31.617  26.918   3.175  1.00  0.00           C  \\nATOM   5003  H   LEU B 318      34.266  25.421   5.933  1.00  0.00           H  \\nATOM   5004  HA  LEU B 318      31.885  27.186   5.720  1.00  0.00           H  \\nATOM   5005 1HB  LEU B 318      32.239  24.253   5.094  1.00  0.00           H  \\nATOM   5006 2HB  LEU B 318      30.743  25.173   4.899  1.00  0.00           H  \\nATOM   5007  HG  LEU B 318      33.286  25.515   3.314  1.00  0.00           H  \\nATOM   5008 1HD1 LEU B 318      31.778  24.754   1.526  1.00  0.00           H  \\nATOM   5009 2HD1 LEU B 318      31.846  23.554   2.837  1.00  0.00           H  \\nATOM   5010 3HD1 LEU B 318      30.447  24.646   2.702  1.00  0.00           H  \\nATOM   5011 1HD2 LEU B 318      31.831  27.225   2.151  1.00  0.00           H  \\nATOM   5012 2HD2 LEU B 318      30.538  26.891   3.328  1.00  0.00           H  \\nATOM   5013 3HD2 LEU B 318      32.064  27.631   3.868  1.00  0.00           H  \\nATOM   5014  N   SER B 319      30.746  26.072   7.829  1.00  0.00           N  \\nATOM   5015  CA  SER B 319      30.294  25.589   9.111  1.00  0.00           C  \\nATOM   5016  C   SER B 319      28.840  25.168   8.992  1.00  0.00           C  \\nATOM   5017  O   SER B 319      28.014  25.779   8.273  1.00  0.00           O  \\nATOM   5018  CB  SER B 319      30.376  26.751  10.074  1.00  0.00           C  \\nATOM   5019  OG  SER B 319      29.989  26.320  11.345  1.00  0.00           O  \\nATOM   5020  H   SER B 319      30.167  26.720   7.315  1.00  0.00           H  \\nATOM   5021  HA  SER B 319      30.957  24.784   9.431  1.00  0.00           H  \\nATOM   5022 1HB  SER B 319      31.395  27.136  10.093  1.00  0.00           H  \\nATOM   5023 2HB  SER B 319      29.729  27.556   9.728  1.00  0.00           H  \\nATOM   5024  HG  SER B 319      29.774  25.389  11.255  1.00  0.00           H  \\nATOM   5025  N   VAL B 320      28.481  24.090   9.694  1.00  0.00           N  \\nATOM   5026  CA  VAL B 320      27.060  23.718   9.886  1.00  0.00           C  \\nATOM   5027  C   VAL B 320      26.787  23.442  11.359  1.00  0.00           C  \\nATOM   5028  O   VAL B 320      27.722  23.063  12.072  1.00  0.00           O  \\nATOM   5029  CB  VAL B 320      26.599  22.464   9.082  1.00  0.00           C  \\nATOM   5030  CG1 VAL B 320      26.766  22.782   7.549  1.00  0.00           C  \\nATOM   5031  CG2 VAL B 320      27.377  21.157   9.498  1.00  0.00           C  \\nATOM   5032  H   VAL B 320      29.201  23.513  10.105  1.00  0.00           H  \\nATOM   5033  HA  VAL B 320      26.437  24.548   9.551  1.00  0.00           H  \\nATOM   5034  HB  VAL B 320      25.537  22.299   9.262  1.00  0.00           H  \\nATOM   5035 1HG1 VAL B 320      26.450  21.919   6.962  1.00  0.00           H  \\nATOM   5036 2HG1 VAL B 320      26.152  23.643   7.287  1.00  0.00           H  \\nATOM   5037 3HG1 VAL B 320      27.812  23.002   7.335  1.00  0.00           H  \\nATOM   5038 1HG2 VAL B 320      27.016  20.315   8.908  1.00  0.00           H  \\nATOM   5039 2HG2 VAL B 320      28.443  21.296   9.318  1.00  0.00           H  \\nATOM   5040 3HG2 VAL B 320      27.210  20.956  10.556  1.00  0.00           H  \\nATOM   5041  N   THR B 321      25.548  23.594  11.846  1.00  0.00           N  \\nATOM   5042  CA  THR B 321      25.328  23.293  13.263  1.00  0.00           C  \\nATOM   5043  C   THR B 321      24.308  22.180  13.371  1.00  0.00           C  \\nATOM   5044  O   THR B 321      23.160  22.291  12.880  1.00  0.00           O  \\nATOM   5045  CB  THR B 321      24.741  24.554  13.980  1.00  0.00           C  \\nATOM   5046  OG1 THR B 321      25.729  25.601  13.859  1.00  0.00           O  \\nATOM   5047  CG2 THR B 321      24.527  24.308  15.466  1.00  0.00           C  \\nATOM   5048  H   THR B 321      24.769  23.904  11.282  1.00  0.00           H  \\nATOM   5049  HA  THR B 321      26.287  23.039  13.716  1.00  0.00           H  \\nATOM   5050  HB  THR B 321      23.783  24.815  13.530  1.00  0.00           H  \\nATOM   5051  HG1 THR B 321      26.489  25.268  13.376  1.00  0.00           H  \\nATOM   5052 1HG2 THR B 321      24.118  25.206  15.928  1.00  0.00           H  \\nATOM   5053 2HG2 THR B 321      23.830  23.481  15.601  1.00  0.00           H  \\nATOM   5054 3HG2 THR B 321      25.479  24.061  15.934  1.00  0.00           H  \\nATOM   5055  N   LEU B 322      24.713  21.050  13.940  1.00  0.00           N  \\nATOM   5056  CA  LEU B 322      23.777  19.929  14.147  1.00  0.00           C  \\nATOM   5057  C   LEU B 322      23.050  20.138  15.430  1.00  0.00           C  \\nATOM   5058  O   LEU B 322      23.550  20.736  16.398  1.00  0.00           O  \\nATOM   5059  CB  LEU B 322      24.560  18.596  14.300  1.00  0.00           C  \\nATOM   5060  CG  LEU B 322      25.531  18.265  13.216  1.00  0.00           C  \\nATOM   5061  CD1 LEU B 322      26.177  16.925  13.618  1.00  0.00           C  \\nATOM   5062  CD2 LEU B 322      24.896  18.202  11.911  1.00  0.00           C  \\nATOM   5063  H   LEU B 322      25.673  20.950  14.237  1.00  0.00           H  \\nATOM   5064  HA  LEU B 322      23.127  19.856  13.275  1.00  0.00           H  \\nATOM   5065 1HB  LEU B 322      25.117  18.626  15.235  1.00  0.00           H  \\nATOM   5066 2HB  LEU B 322      23.844  17.776  14.354  1.00  0.00           H  \\nATOM   5067  HG  LEU B 322      26.308  19.029  13.178  1.00  0.00           H  \\nATOM   5068 1HD1 LEU B 322      26.902  16.628  12.859  1.00  0.00           H  \\nATOM   5069 2HD1 LEU B 322      26.683  17.038  14.577  1.00  0.00           H  \\nATOM   5070 3HD1 LEU B 322      25.406  16.160  13.702  1.00  0.00           H  \\nATOM   5071 1HD2 LEU B 322      25.643  17.959  11.154  1.00  0.00           H  \\nATOM   5072 2HD2 LEU B 322      24.124  17.433  11.919  1.00  0.00           H  \\nATOM   5073 3HD2 LEU B 322      24.445  19.167  11.679  1.00  0.00           H  \\nATOM   5074  N   ALA B 323      21.814  19.652  15.449  1.00  0.00           N  \\nATOM   5075  CA  ALA B 323      21.022  19.630  16.690  1.00  0.00           C  \\nATOM   5076  C   ALA B 323      20.844  18.189  17.170  1.00  0.00           C  \\nATOM   5077  O   ALA B 323      20.428  17.289  16.325  1.00  0.00           O  \\nATOM   5078  CB  ALA B 323      19.607  20.209  16.438  1.00  0.00           C  \\nATOM   5079  H   ALA B 323      21.406  19.290  14.599  1.00  0.00           H  \\nATOM   5080  HA  ALA B 323      21.531  20.251  17.428  1.00  0.00           H  \\nATOM   5081 1HB  ALA B 323      19.034  20.186  17.365  1.00  0.00           H  \\nATOM   5082 2HB  ALA B 323      19.692  21.238  16.090  1.00  0.00           H  \\nATOM   5083 3HB  ALA B 323      19.099  19.611  15.683  1.00  0.00           H  \\nATOM   5084  N   LEU B 324      21.281  17.932  18.412  1.00  0.00           N  \\nATOM   5085  CA  LEU B 324      21.119  16.584  18.959  1.00  0.00           C  \\nATOM   5086  C   LEU B 324      20.040  16.595  20.032  1.00  0.00           C  \\nATOM   5087  O   LEU B 324      19.994  17.511  20.877  1.00  0.00           O  \\nATOM   5088  CB  LEU B 324      22.465  16.118  19.599  1.00  0.00           C  \\nATOM   5089  CG  LEU B 324      23.551  15.611  18.577  1.00  0.00           C  \\nATOM   5090  CD1 LEU B 324      23.960  16.533  17.581  1.00  0.00           C  \\nATOM   5091  CD2 LEU B 324      24.760  15.394  19.586  1.00  0.00           C  \\nATOM   5092  H   LEU B 324      21.719  18.644  18.979  1.00  0.00           H  \\nATOM   5093  HA  LEU B 324      20.858  15.909  18.144  1.00  0.00           H  \\nATOM   5094 1HB  LEU B 324      22.891  16.950  20.157  1.00  0.00           H  \\nATOM   5095 2HB  LEU B 324      22.256  15.308  20.298  1.00  0.00           H  \\nATOM   5096  HG  LEU B 324      23.198  14.700  18.094  1.00  0.00           H  \\nATOM   5097 1HD1 LEU B 324      24.709  16.068  16.940  1.00  0.00           H  \\nATOM   5098 2HD1 LEU B 324      23.099  16.825  16.979  1.00  0.00           H  \\nATOM   5099 3HD1 LEU B 324      24.387  17.415  18.056  1.00  0.00           H  \\nATOM   5100 1HD2 LEU B 324      25.630  15.031  19.039  1.00  0.00           H  \\nATOM   5101 2HD2 LEU B 324      25.004  16.340  20.070  1.00  0.00           H  \\nATOM   5102 3HD2 LEU B 324      24.476  14.662  20.342  1.00  0.00           H  \\nATOM   5103  N   ASP B 325      19.222  15.548  20.087  1.00  0.00           N  \\nATOM   5104  CA  ASP B 325      18.201  15.345  21.169  1.00  0.00           C  \\nATOM   5105  C   ASP B 325      18.975  14.782  22.356  1.00  0.00           C  \\nATOM   5106  O   ASP B 325      19.643  13.757  22.222  1.00  0.00           O  \\nATOM   5107  CB  ASP B 325      17.170  14.358  20.538  1.00  0.00           C  \\nATOM   5108  CG  ASP B 325      15.940  14.140  21.401  1.00  0.00           C  \\nATOM   5109  OD1 ASP B 325      16.141  14.266  22.627  1.00  0.00           O  \\nATOM   5110  OD2 ASP B 325      14.864  13.796  20.782  1.00  0.00           O  \\nATOM   5111  H   ASP B 325      19.304  14.859  19.353  1.00  0.00           H  \\nATOM   5112  HA  ASP B 325      17.746  16.308  21.400  1.00  0.00           H  \\nATOM   5113 1HB  ASP B 325      16.848  14.739  19.569  1.00  0.00           H  \\nATOM   5114 2HB  ASP B 325      17.647  13.392  20.368  1.00  0.00           H  \\nATOM   5115  N   VAL B 326      18.867  15.456  23.514  1.00  0.00           N  \\nATOM   5116  CA  VAL B 326      19.590  14.994  24.719  1.00  0.00           C  \\nATOM   5117  C   VAL B 326      19.026  13.647  25.188  1.00  0.00           C  \\nATOM   5118  O   VAL B 326      19.773  12.928  25.918  1.00  0.00           O  \\nATOM   5119  CB  VAL B 326      19.613  15.973  25.877  1.00  0.00           C  \\nATOM   5120  CG1 VAL B 326      20.149  17.325  25.424  1.00  0.00           C  \\nATOM   5121  CG2 VAL B 326      18.209  16.101  26.487  1.00  0.00           C  \\nATOM   5122  H   VAL B 326      18.292  16.284  23.573  1.00  0.00           H  \\nATOM   5123  HA  VAL B 326      20.632  14.815  24.450  1.00  0.00           H  \\nATOM   5124  HB  VAL B 326      20.306  15.608  26.635  1.00  0.00           H  \\nATOM   5125 1HG1 VAL B 326      20.158  18.014  26.269  1.00  0.00           H  \\nATOM   5126 2HG1 VAL B 326      21.163  17.205  25.043  1.00  0.00           H  \\nATOM   5127 3HG1 VAL B 326      19.509  17.725  24.637  1.00  0.00           H  \\nATOM   5128 1HG2 VAL B 326      18.236  16.805  27.318  1.00  0.00           H  \\nATOM   5129 2HG2 VAL B 326      17.514  16.461  25.728  1.00  0.00           H  \\nATOM   5130 3HG2 VAL B 326      17.879  15.127  26.848  1.00  0.00           H  \\nATOM   5131  N   THR B 327      17.847  13.241  24.724  1.00  0.00           N  \\nATOM   5132  CA  THR B 327      17.313  12.003  25.298  1.00  0.00           C  \\nATOM   5133  C   THR B 327      17.933  10.764  24.630  1.00  0.00           C  \\nATOM   5134  O   THR B 327      17.776   9.663  25.155  1.00  0.00           O  \\nATOM   5135  CB  THR B 327      15.770  11.944  25.168  1.00  0.00           C  \\nATOM   5136  OG1 THR B 327      15.383  11.915  23.828  1.00  0.00           O  \\nATOM   5137  CG2 THR B 327      15.122  13.117  25.848  1.00  0.00           C  \\nATOM   5138  H   THR B 327      17.313  13.726  24.017  1.00  0.00           H  \\nATOM   5139  HA  THR B 327      17.568  11.975  26.358  1.00  0.00           H  \\nATOM   5140  HB  THR B 327      15.401  11.025  25.624  1.00  0.00           H  \\nATOM   5141  HG1 THR B 327      16.164  11.938  23.269  1.00  0.00           H  \\nATOM   5142 1HG2 THR B 327      14.040  13.048  25.740  1.00  0.00           H  \\nATOM   5143 2HG2 THR B 327      15.383  13.114  26.906  1.00  0.00           H  \\nATOM   5144 3HG2 THR B 327      15.473  14.041  25.391  1.00  0.00           H  \\nATOM   5145  N   ASN B 328      18.467  10.916  23.424  1.00  0.00           N  \\nATOM   5146  CA  ASN B 328      19.082   9.745  22.768  1.00  0.00           C  \\nATOM   5147  C   ASN B 328      20.424  10.062  22.124  1.00  0.00           C  \\nATOM   5148  O   ASN B 328      20.979   9.165  21.489  1.00  0.00           O  \\nATOM   5149  CB  ASN B 328      18.112   9.131  21.791  1.00  0.00           C  \\nATOM   5150  CG  ASN B 328      17.687  10.106  20.709  1.00  0.00           C  \\nATOM   5151  OD1 ASN B 328      18.265  11.125  20.503  1.00  0.00           O  \\nATOM   5152  ND2 ASN B 328      16.640   9.707  19.943  1.00  0.00           N  \\nATOM   5153  H   ASN B 328      18.465  11.804  22.944  1.00  0.00           H  \\nATOM   5154  HA  ASN B 328      19.325   9.007  23.534  1.00  0.00           H  \\nATOM   5155 1HB  ASN B 328      18.570   8.259  21.322  1.00  0.00           H  \\nATOM   5156 2HB  ASN B 328      17.226   8.787  22.325  1.00  0.00           H  \\nATOM   5157 1HD2 ASN B 328      16.303  10.298  19.209  1.00  0.00           H  \\nATOM   5158 2HD2 ASN B 328      16.205   8.823  20.112  1.00  0.00           H  \\nATOM   5159  N   ALA B 329      20.931  11.289  22.298  1.00  0.00           N  \\nATOM   5160  CA  ALA B 329      22.177  11.765  21.678  1.00  0.00           C  \\nATOM   5161  C   ALA B 329      22.161  11.847  20.156  1.00  0.00           C  \\nATOM   5162  O   ALA B 329      23.202  12.046  19.546  1.00  0.00           O  \\nATOM   5163  CB  ALA B 329      23.447  10.928  22.222  1.00  0.00           C  \\nATOM   5164  H   ALA B 329      20.410  11.913  22.897  1.00  0.00           H  \\nATOM   5165  HA  ALA B 329      22.310  12.813  21.946  1.00  0.00           H  \\nATOM   5166 1HB  ALA B 329      24.357  11.298  21.749  1.00  0.00           H  \\nATOM   5167 2HB  ALA B 329      23.526  11.047  23.303  1.00  0.00           H  \\nATOM   5168 3HB  ALA B 329      23.317   9.874  21.982  1.00  0.00           H  \\nATOM   5169  N   TYR B 330      21.004  11.663  19.516  1.00  0.00           N  \\nATOM   5170  CA  TYR B 330      20.928  11.477  18.033  1.00  0.00           C  \\nATOM   5171  C   TYR B 330      20.625  12.830  17.332  1.00  0.00           C  \\nATOM   5172  O   TYR B 330      19.984  13.710  17.928  1.00  0.00           O  \\nATOM   5173  CB  TYR B 330      19.805  10.452  17.716  1.00  0.00           C  \\nATOM   5174  CG  TYR B 330      20.046   9.686  16.401  1.00  0.00           C  \\nATOM   5175  CD1 TYR B 330      20.882   8.581  16.345  1.00  0.00           C  \\nATOM   5176  CD2 TYR B 330      19.389  10.071  15.207  1.00  0.00           C  \\nATOM   5177  CE1 TYR B 330      21.044   7.832  15.185  1.00  0.00           C  \\nATOM   5178  CE2 TYR B 330      19.567   9.371  14.061  1.00  0.00           C  \\nATOM   5179  CZ  TYR B 330      20.394   8.267  14.003  1.00  0.00           C  \\nATOM   5180  OH  TYR B 330      20.480   7.569  12.794  1.00  0.00           O  \\nATOM   5181  H   TYR B 330      20.152  11.650  20.058  1.00  0.00           H  \\nATOM   5182  HA  TYR B 330      21.885  11.088  17.684  1.00  0.00           H  \\nATOM   5183 1HB  TYR B 330      19.729   9.731  18.531  1.00  0.00           H  \\nATOM   5184 2HB  TYR B 330      18.849  10.970  17.648  1.00  0.00           H  \\nATOM   5185  HD1 TYR B 330      21.439   8.274  17.230  1.00  0.00           H  \\nATOM   5186  HD2 TYR B 330      18.732  10.940  15.203  1.00  0.00           H  \\nATOM   5187  HE1 TYR B 330      21.711   6.970  15.182  1.00  0.00           H  \\nATOM   5188  HE2 TYR B 330      19.051   9.676  13.150  1.00  0.00           H  \\nATOM   5189  HH  TYR B 330      19.914   7.989  12.142  1.00  0.00           H  \\nATOM   5190  N   VAL B 331      21.103  12.938  16.090  1.00  0.00           N  \\nATOM   5191  CA  VAL B 331      20.883  14.146  15.311  1.00  0.00           C  \\nATOM   5192  C   VAL B 331      19.398  14.205  14.882  1.00  0.00           C  \\nATOM   5193  O   VAL B 331      18.874  13.248  14.265  1.00  0.00           O  \\nATOM   5194  CB  VAL B 331      21.797  14.037  14.057  1.00  0.00           C  \\nATOM   5195  CG1 VAL B 331      21.511  15.243  13.117  1.00  0.00           C  \\nATOM   5196  CG2 VAL B 331      23.273  14.101  14.417  1.00  0.00           C  \\nATOM   5197  H   VAL B 331      21.624  12.176  15.680  1.00  0.00           H  \\nATOM   5198  HA  VAL B 331      21.168  15.006  15.918  1.00  0.00           H  \\nATOM   5199  HB  VAL B 331      21.605  13.086  13.560  1.00  0.00           H  \\nATOM   5200 1HG1 VAL B 331      22.147  15.175  12.234  1.00  0.00           H  \\nATOM   5201 2HG1 VAL B 331      20.465  15.226  12.812  1.00  0.00           H  \\nATOM   5202 3HG1 VAL B 331      21.722  16.173  13.644  1.00  0.00           H  \\nATOM   5203 1HG2 VAL B 331      23.872  14.020  13.511  1.00  0.00           H  \\nATOM   5204 2HG2 VAL B 331      23.486  15.049  14.910  1.00  0.00           H  \\nATOM   5205 3HG2 VAL B 331      23.520  13.279  15.089  1.00  0.00           H  \\nATOM   5206  N   VAL B 332      18.776  15.371  15.092  1.00  0.00           N  \\nATOM   5207  CA  VAL B 332      17.394  15.606  14.593  1.00  0.00           C  \\nATOM   5208  C   VAL B 332      17.359  16.614  13.434  1.00  0.00           C  \\nATOM   5209  O   VAL B 332      16.350  16.649  12.754  1.00  0.00           O  \\nATOM   5210  CB  VAL B 332      16.436  16.087  15.695  1.00  0.00           C  \\nATOM   5211  CG1 VAL B 332      16.455  15.051  16.873  1.00  0.00           C  \\nATOM   5212  CG2 VAL B 332      16.784  17.497  16.302  1.00  0.00           C  \\nATOM   5213  H   VAL B 332      19.248  16.107  15.597  1.00  0.00           H  \\nATOM   5214  HA  VAL B 332      17.002  14.665  14.205  1.00  0.00           H  \\nATOM   5215  HB  VAL B 332      15.429  16.156  15.284  1.00  0.00           H  \\nATOM   5216 1HG1 VAL B 332      15.778  15.383  17.661  1.00  0.00           H  \\nATOM   5217 2HG1 VAL B 332      16.134  14.077  16.506  1.00  0.00           H  \\nATOM   5218 3HG1 VAL B 332      17.466  14.974  17.273  1.00  0.00           H  \\nATOM   5219 1HG2 VAL B 332      16.054  17.753  17.070  1.00  0.00           H  \\nATOM   5220 2HG2 VAL B 332      17.780  17.467  16.743  1.00  0.00           H  \\nATOM   5221 3HG2 VAL B 332      16.759  18.249  15.513  1.00  0.00           H  \\nATOM   5222  N   GLY B 333      18.398  17.350  13.227  1.00  0.00           N  \\nATOM   5223  CA  GLY B 333      18.389  18.332  12.105  1.00  0.00           C  \\nATOM   5224  C   GLY B 333      19.647  19.128  12.120  1.00  0.00           C  \\nATOM   5225  O   GLY B 333      20.528  18.873  12.955  1.00  0.00           O  \\nATOM   5226  H   GLY B 333      19.220  17.276  13.809  1.00  0.00           H  \\nATOM   5227 1HA  GLY B 333      18.286  17.802  11.158  1.00  0.00           H  \\nATOM   5228 2HA  GLY B 333      17.524  18.987  12.202  1.00  0.00           H  \\nATOM   5229  N   TYR B 334      19.733  20.140  11.213  1.00  0.00           N  \\nATOM   5230  CA  TYR B 334      20.882  20.982  11.255  1.00  0.00           C  \\nATOM   5231  C   TYR B 334      20.580  22.320  10.613  1.00  0.00           C  \\nATOM   5232  O   TYR B 334      19.599  22.451   9.882  1.00  0.00           O  \\nATOM   5233  CB  TYR B 334      22.042  20.304  10.449  1.00  0.00           C  \\nATOM   5234  CG  TYR B 334      21.962  20.331   8.936  1.00  0.00           C  \\nATOM   5235  CD1 TYR B 334      21.218  19.438   8.256  1.00  0.00           C  \\nATOM   5236  CD2 TYR B 334      22.712  21.285   8.220  1.00  0.00           C  \\nATOM   5237  CE1 TYR B 334      21.231  19.464   6.798  1.00  0.00           C  \\nATOM   5238  CE2 TYR B 334      22.747  21.292   6.788  1.00  0.00           C  \\nATOM   5239  CZ  TYR B 334      21.990  20.373   6.122  1.00  0.00           C  \\nATOM   5240  OH  TYR B 334      21.957  20.294   4.770  1.00  0.00           O  \\nATOM   5241  H   TYR B 334      19.020  20.314  10.519  1.00  0.00           H  \\nATOM   5242  HA  TYR B 334      21.186  21.100  12.295  1.00  0.00           H  \\nATOM   5243 1HB  TYR B 334      22.990  20.776  10.709  1.00  0.00           H  \\nATOM   5244 2HB  TYR B 334      22.113  19.253  10.728  1.00  0.00           H  \\nATOM   5245  HD1 TYR B 334      20.615  18.707   8.796  1.00  0.00           H  \\nATOM   5246  HD2 TYR B 334      23.285  22.042   8.756  1.00  0.00           H  \\nATOM   5247  HE1 TYR B 334      20.634  18.747   6.235  1.00  0.00           H  \\nATOM   5248  HE2 TYR B 334      23.367  22.018   6.261  1.00  0.00           H  \\nATOM   5249  HH  TYR B 334      22.532  20.966   4.397  1.00  0.00           H  \\nATOM   5250  N   ARG B 335      21.477  23.238  10.852  1.00  0.00           N  \\nATOM   5251  CA  ARG B 335      21.384  24.640  10.262  1.00  0.00           C  \\nATOM   5252  C   ARG B 335      22.611  24.817   9.389  1.00  0.00           C  \\nATOM   5253  O   ARG B 335      23.717  24.466   9.810  1.00  0.00           O  \\nATOM   5254  CB  ARG B 335      21.361  25.666  11.382  1.00  0.00           C  \\nATOM   5255  CG  ARG B 335      21.342  27.104  10.852  1.00  0.00           C  \\nATOM   5256  CD  ARG B 335      21.306  28.114  11.918  1.00  0.00           C  \\nATOM   5257  NE  ARG B 335      22.587  28.357  12.578  1.00  0.00           N  \\nATOM   5258  CZ  ARG B 335      23.633  29.071  12.104  1.00  0.00           C  \\nATOM   5259  NH1 ARG B 335      23.648  29.628  10.895  1.00  0.00           N  \\nATOM   5260  NH2 ARG B 335      24.708  29.224  12.882  1.00  0.00           N  \\nATOM   5261  H   ARG B 335      22.258  23.006  11.449  1.00  0.00           H  \\nATOM   5262  HA  ARG B 335      20.457  24.715   9.692  1.00  0.00           H  \\nATOM   5263 1HB  ARG B 335      20.481  25.506  12.004  1.00  0.00           H  \\nATOM   5264 2HB  ARG B 335      22.238  25.533  12.015  1.00  0.00           H  \\nATOM   5265 1HG  ARG B 335      22.238  27.285  10.258  1.00  0.00           H  \\nATOM   5266 2HG  ARG B 335      20.458  27.251  10.230  1.00  0.00           H  \\nATOM   5267 1HD  ARG B 335      20.981  29.068  11.504  1.00  0.00           H  \\nATOM   5268 2HD  ARG B 335      20.609  27.799  12.694  1.00  0.00           H  \\nATOM   5269  HE  ARG B 335      22.727  27.954  13.495  1.00  0.00           H  \\nATOM   5270 1HH1 ARG B 335      22.851  29.527  10.282  1.00  0.00           H  \\nATOM   5271 2HH1 ARG B 335      24.456  30.152  10.590  1.00  0.00           H  \\nATOM   5272 1HH2 ARG B 335      24.728  28.810  13.804  1.00  0.00           H  \\nATOM   5273 2HH2 ARG B 335      25.500  29.753  12.549  1.00  0.00           H  \\nATOM   5274  N   ALA B 336      22.422  25.434   8.199  1.00  0.00           N  \\nATOM   5275  CA  ALA B 336      23.558  25.893   7.340  1.00  0.00           C  \\nATOM   5276  C   ALA B 336      23.142  27.308   6.904  1.00  0.00           C  \\nATOM   5277  O   ALA B 336      22.146  27.422   6.210  1.00  0.00           O  \\nATOM   5278  CB  ALA B 336      23.751  25.045   6.191  1.00  0.00           C  \\nATOM   5279  H   ALA B 336      21.474  25.587   7.885  1.00  0.00           H  \\nATOM   5280  HA  ALA B 336      24.469  25.862   7.938  1.00  0.00           H  \\nATOM   5281 1HB  ALA B 336      24.586  25.419   5.599  1.00  0.00           H  \\nATOM   5282 2HB  ALA B 336      23.966  24.029   6.521  1.00  0.00           H  \\nATOM   5283 3HB  ALA B 336      22.847  25.045   5.583  1.00  0.00           H  \\nATOM   5284  N   GLY B 337      23.879  28.279   7.473  1.00  0.00           N  \\nATOM   5285  CA  GLY B 337      23.591  29.740   7.294  1.00  0.00           C  \\nATOM   5286  C   GLY B 337      22.146  30.047   7.662  1.00  0.00           C  \\nATOM   5287  O   GLY B 337      21.761  29.896   8.791  1.00  0.00           O  \\nATOM   5288  H   GLY B 337      24.662  27.998   8.046  1.00  0.00           H  \\nATOM   5289 1HA  GLY B 337      24.268  30.324   7.918  1.00  0.00           H  \\nATOM   5290 2HA  GLY B 337      23.782  30.024   6.260  1.00  0.00           H  \\nATOM   5291  N   ASN B 338      21.331  30.483   6.681  1.00  0.00           N  \\nATOM   5292  CA  ASN B 338      19.970  30.980   6.974  1.00  0.00           C  \\nATOM   5293  C   ASN B 338      18.957  29.906   6.600  1.00  0.00           C  \\nATOM   5294  O   ASN B 338      17.778  30.183   6.408  1.00  0.00           O  \\nATOM   5295  CB  ASN B 338      19.683  32.333   6.229  1.00  0.00           C  \\nATOM   5296  CG  ASN B 338      20.723  33.426   6.538  1.00  0.00           C  \\nATOM   5297  OD1 ASN B 338      21.349  33.998   5.620  1.00  0.00           O  \\nATOM   5298  ND2 ASN B 338      20.910  33.726   7.814  1.00  0.00           N  \\nATOM   5299  H   ASN B 338      21.655  30.471   5.724  1.00  0.00           H  \\nATOM   5300  HA  ASN B 338      19.890  31.158   8.048  1.00  0.00           H  \\nATOM   5301 1HB  ASN B 338      19.671  32.160   5.152  1.00  0.00           H  \\nATOM   5302 2HB  ASN B 338      18.697  32.702   6.512  1.00  0.00           H  \\nATOM   5303 1HD2 ASN B 338      21.574  34.429   8.070  1.00  0.00           H  \\nATOM   5304 2HD2 ASN B 338      20.388  33.250   8.521  1.00  0.00           H  \\nATOM   5305  N   SER B 339      19.383  28.637   6.624  1.00  0.00           N  \\nATOM   5306  CA  SER B 339      18.468  27.568   6.368  1.00  0.00           C  \\nATOM   5307  C   SER B 339      18.577  26.448   7.418  1.00  0.00           C  \\nATOM   5308  O   SER B 339      19.715  26.133   7.838  1.00  0.00           O  \\nATOM   5309  CB  SER B 339      18.726  26.966   4.990  1.00  0.00           C  \\nATOM   5310  OG  SER B 339      18.487  27.976   3.975  1.00  0.00           O  \\nATOM   5311  H   SER B 339      20.349  28.421   6.822  1.00  0.00           H  \\nATOM   5312  HA  SER B 339      17.453  27.968   6.390  1.00  0.00           H  \\nATOM   5313 1HB  SER B 339      19.753  26.606   4.937  1.00  0.00           H  \\nATOM   5314 2HB  SER B 339      18.071  26.109   4.839  1.00  0.00           H  \\nATOM   5315  HG  SER B 339      18.225  28.770   4.448  1.00  0.00           H  \\nATOM   5316  N   ALA B 340      17.446  25.903   7.850  1.00  0.00           N  \\nATOM   5317  CA  ALA B 340      17.520  24.752   8.796  1.00  0.00           C  \\nATOM   5318  C   ALA B 340      16.753  23.610   8.225  1.00  0.00           C  \\nATOM   5319  O   ALA B 340      15.762  23.796   7.537  1.00  0.00           O  \\nATOM   5320  CB  ALA B 340      16.928  25.110  10.148  1.00  0.00           C  \\nATOM   5321  H   ALA B 340      16.541  26.243   7.559  1.00  0.00           H  \\nATOM   5322  HA  ALA B 340      18.570  24.496   8.936  1.00  0.00           H  \\nATOM   5323 1HB  ALA B 340      16.997  24.250  10.815  1.00  0.00           H  \\nATOM   5324 2HB  ALA B 340      17.480  25.947  10.577  1.00  0.00           H  \\nATOM   5325 3HB  ALA B 340      15.883  25.390  10.025  1.00  0.00           H  \\nATOM   5326  N   TYR B 341      17.195  22.391   8.566  1.00  0.00           N  \\nATOM   5327  CA  TYR B 341      16.733  21.152   7.991  1.00  0.00           C  \\nATOM   5328  C   TYR B 341      16.456  20.111   9.089  1.00  0.00           C  \\nATOM   5329  O   TYR B 341      17.296  19.943   9.950  1.00  0.00           O  \\nATOM   5330  CB  TYR B 341      17.746  20.532   6.982  1.00  0.00           C  \\nATOM   5331  CG  TYR B 341      17.966  21.474   5.855  1.00  0.00           C  \\nATOM   5332  CD1 TYR B 341      17.108  21.440   4.757  1.00  0.00           C  \\nATOM   5333  CD2 TYR B 341      18.994  22.403   5.908  1.00  0.00           C  \\nATOM   5334  CE1 TYR B 341      17.341  22.383   3.702  1.00  0.00           C  \\nATOM   5335  CE2 TYR B 341      19.245  23.256   4.915  1.00  0.00           C  \\nATOM   5336  CZ  TYR B 341      18.389  23.256   3.825  1.00  0.00           C  \\nATOM   5337  OH  TYR B 341      18.629  24.240   2.858  1.00  0.00           O  \\nATOM   5338  H   TYR B 341      17.906  22.366   9.283  1.00  0.00           H  \\nATOM   5339  HA  TYR B 341      15.809  21.347   7.446  1.00  0.00           H  \\nATOM   5340 1HB  TYR B 341      18.687  20.324   7.493  1.00  0.00           H  \\nATOM   5341 2HB  TYR B 341      17.357  19.583   6.613  1.00  0.00           H  \\nATOM   5342  HD1 TYR B 341      16.295  20.714   4.722  1.00  0.00           H  \\nATOM   5343  HD2 TYR B 341      19.636  22.455   6.787  1.00  0.00           H  \\nATOM   5344  HE1 TYR B 341      16.694  22.391   2.825  1.00  0.00           H  \\nATOM   5345  HE2 TYR B 341      20.103  23.926   4.975  1.00  0.00           H  \\nATOM   5346  HH  TYR B 341      19.381  24.774   3.126  1.00  0.00           H  \\nATOM   5347  N   PHE B 342      15.322  19.392   9.024  1.00  0.00           N  \\nATOM   5348  CA  PHE B 342      14.943  18.459  10.086  1.00  0.00           C  \\nATOM   5349  C   PHE B 342      14.526  17.154   9.466  1.00  0.00           C  \\nATOM   5350  O   PHE B 342      13.846  17.084   8.450  1.00  0.00           O  \\nATOM   5351  CB  PHE B 342      13.832  19.029  10.971  1.00  0.00           C  \\nATOM   5352  CG  PHE B 342      14.160  20.268  11.600  1.00  0.00           C  \\nATOM   5353  CD1 PHE B 342      14.822  20.323  12.807  1.00  0.00           C  \\nATOM   5354  CD2 PHE B 342      13.796  21.493  10.961  1.00  0.00           C  \\nATOM   5355  CE1 PHE B 342      15.136  21.558  13.333  1.00  0.00           C  \\nATOM   5356  CE2 PHE B 342      14.145  22.708  11.589  1.00  0.00           C  \\nATOM   5357  CZ  PHE B 342      14.739  22.728  12.736  1.00  0.00           C  \\nATOM   5358  H   PHE B 342      14.716  19.498   8.223  1.00  0.00           H  \\nATOM   5359  HA  PHE B 342      15.816  18.277  10.715  1.00  0.00           H  \\nATOM   5360 1HB  PHE B 342      12.934  19.184  10.374  1.00  0.00           H  \\nATOM   5361 2HB  PHE B 342      13.585  18.311  11.752  1.00  0.00           H  \\nATOM   5362  HD1 PHE B 342      15.081  19.394  13.315  1.00  0.00           H  \\nATOM   5363  HD2 PHE B 342      13.295  21.505   9.993  1.00  0.00           H  \\nATOM   5364  HE1 PHE B 342      15.718  21.626  14.252  1.00  0.00           H  \\nATOM   5365  HE2 PHE B 342      13.902  23.644  11.086  1.00  0.00           H  \\nATOM   5366  HZ  PHE B 342      14.922  23.682  13.229  1.00  0.00           H  \\nATOM   5367  N   PHE B 343      14.984  16.039  10.069  1.00  0.00           N  \\nATOM   5368  CA  PHE B 343      14.395  14.739   9.748  1.00  0.00           C  \\nATOM   5369  C   PHE B 343      12.920  14.655  10.033  1.00  0.00           C  \\nATOM   5370  O   PHE B 343      12.418  15.337  10.871  1.00  0.00           O  \\nATOM   5371  CB  PHE B 343      15.097  13.597  10.501  1.00  0.00           C  \\nATOM   5372  CG  PHE B 343      16.525  13.401  10.092  1.00  0.00           C  \\nATOM   5373  CD1 PHE B 343      16.924  13.175   8.748  1.00  0.00           C  \\nATOM   5374  CD2 PHE B 343      17.510  13.407  11.091  1.00  0.00           C  \\nATOM   5375  CE1 PHE B 343      18.264  12.972   8.421  1.00  0.00           C  \\nATOM   5376  CE2 PHE B 343      18.846  13.168  10.789  1.00  0.00           C  \\nATOM   5377  CZ  PHE B 343      19.265  12.984   9.467  1.00  0.00           C  \\nATOM   5378  H   PHE B 343      15.733  16.086  10.744  1.00  0.00           H  \\nATOM   5379  HA  PHE B 343      14.511  14.561   8.678  1.00  0.00           H  \\nATOM   5380 1HB  PHE B 343      15.071  13.797  11.571  1.00  0.00           H  \\nATOM   5381 2HB  PHE B 343      14.560  12.665  10.330  1.00  0.00           H  \\nATOM   5382  HD1 PHE B 343      16.167  13.162   7.963  1.00  0.00           H  \\nATOM   5383  HD2 PHE B 343      17.202  13.548  12.128  1.00  0.00           H  \\nATOM   5384  HE1 PHE B 343      18.552  12.806   7.383  1.00  0.00           H  \\nATOM   5385  HE2 PHE B 343      19.572  13.124  11.601  1.00  0.00           H  \\nATOM   5386  HZ  PHE B 343      20.317  12.850   9.218  1.00  0.00           H  \\nATOM   5387  N   HIS B 344      12.212  13.853   9.213  1.00  0.00           N  \\nATOM   5388  CA  HIS B 344      10.761  13.683   9.456  1.00  0.00           C  \\nATOM   5389  C   HIS B 344      10.489  13.114  10.856  1.00  0.00           C  \\nATOM   5390  O   HIS B 344      10.947  11.978  11.150  1.00  0.00           O  \\nATOM   5391  CB  HIS B 344      10.160  12.721   8.437  1.00  0.00           C  \\nATOM   5392  CG  HIS B 344       8.662  12.772   8.395  1.00  0.00           C  \\nATOM   5393  ND1 HIS B 344       7.869  11.956   9.177  1.00  0.00           N  \\nATOM   5394  CD2 HIS B 344       7.819  13.635   7.775  1.00  0.00           C  \\nATOM   5395  CE1 HIS B 344       6.600  12.285   9.005  1.00  0.00           C  \\nATOM   5396  NE2 HIS B 344       6.544  13.300   8.160  1.00  0.00           N  \\nATOM   5397  H   HIS B 344      12.642  13.366   8.440  1.00  0.00           H  \\nATOM   5398  HA  HIS B 344      10.260  14.645   9.352  1.00  0.00           H  \\nATOM   5399 1HB  HIS B 344      10.544  12.956   7.444  1.00  0.00           H  \\nATOM   5400 2HB  HIS B 344      10.467  11.702   8.674  1.00  0.00           H  \\nATOM   5401  HD2 HIS B 344       8.098  14.432   7.085  1.00  0.00           H  \\nATOM   5402  HE1 HIS B 344       5.746  11.802   9.479  1.00  0.00           H  \\nATOM   5403  HE2 HIS B 344       5.702  13.760   7.844  1.00  0.00           H  \\nATOM   5404  N   PRO B 345       9.821  13.890  11.715  1.00  0.00           N  \\nATOM   5405  CA  PRO B 345       9.583  13.403  13.101  1.00  0.00           C  \\nATOM   5406  C   PRO B 345       8.634  12.185  13.129  1.00  0.00           C  \\nATOM   5407  O   PRO B 345       7.667  12.097  12.298  1.00  0.00           O  \\nATOM   5408  CB  PRO B 345       8.950  14.607  13.799  1.00  0.00           C  \\nATOM   5409  CG  PRO B 345       8.492  15.490  12.692  1.00  0.00           C  \\nATOM   5410  CD  PRO B 345       9.332  15.277  11.518  1.00  0.00           C  \\nATOM   5411  HA  PRO B 345      10.547  13.146  13.565  1.00  0.00           H  \\nATOM   5412 1HB  PRO B 345       8.123  14.276  14.444  1.00  0.00           H  \\nATOM   5413 2HB  PRO B 345       9.689  15.096  14.450  1.00  0.00           H  \\nATOM   5414 1HG  PRO B 345       7.441  15.278  12.447  1.00  0.00           H  \\nATOM   5415 2HG  PRO B 345       8.539  16.544  13.003  1.00  0.00           H  \\nATOM   5416 1HD  PRO B 345       8.722  15.371  10.608  1.00  0.00           H  \\nATOM   5417 2HD  PRO B 345      10.151  16.012  11.514  1.00  0.00           H  \\nATOM   5418  N   ASP B 346       8.912  11.273  14.048  1.00  0.00           N  \\nATOM   5419  CA  ASP B 346       8.137  10.065  14.296  1.00  0.00           C  \\nATOM   5420  C   ASP B 346       6.741  10.246  14.885  1.00  0.00           C  \\nATOM   5421  O   ASP B 346       5.927   9.318  14.850  1.00  0.00           O  \\nATOM   5422  CB  ASP B 346       8.961   9.107  15.173  1.00  0.00           C  \\nATOM   5423  CG  ASP B 346      10.129   8.447  14.385  1.00  0.00           C  \\nATOM   5424  OD1 ASP B 346       9.895   7.420  13.645  1.00  0.00           O  \\nATOM   5425  OD2 ASP B 346      11.278   8.969  14.501  1.00  0.00           O  \\nATOM   5426  H   ASP B 346       9.733  11.455  14.608  1.00  0.00           H  \\nATOM   5427  HA  ASP B 346       7.927   9.586  13.339  1.00  0.00           H  \\nATOM   5428 1HB  ASP B 346       9.370   9.653  16.024  1.00  0.00           H  \\nATOM   5429 2HB  ASP B 346       8.311   8.325  15.567  1.00  0.00           H  \\nATOM   5430  N   ASN B 347       6.452  11.417  15.461  1.00  0.00           N  \\nATOM   5431  CA  ASN B 347       5.158  11.712  16.081  1.00  0.00           C  \\nATOM   5432  C   ASN B 347       5.042  13.221  16.307  1.00  0.00           C  \\nATOM   5433  O   ASN B 347       6.022  13.996  16.141  1.00  0.00           O  \\nATOM   5434  CB  ASN B 347       4.951  11.003  17.443  1.00  0.00           C  \\nATOM   5435  CG  ASN B 347       6.022  11.373  18.453  1.00  0.00           C  \\nATOM   5436  OD1 ASN B 347       6.053  12.513  18.959  1.00  0.00           O  \\nATOM   5437  ND2 ASN B 347       6.990  10.482  18.650  1.00  0.00           N  \\nATOM   5438  H   ASN B 347       7.171  12.126  15.462  1.00  0.00           H  \\nATOM   5439  HA  ASN B 347       4.367  11.366  15.414  1.00  0.00           H  \\nATOM   5440 1HB  ASN B 347       3.974  11.270  17.848  1.00  0.00           H  \\nATOM   5441 2HB  ASN B 347       4.960   9.923  17.296  1.00  0.00           H  \\nATOM   5442 1HD2 ASN B 347       7.722  10.675  19.305  1.00  0.00           H  \\nATOM   5443 2HD2 ASN B 347       6.986   9.620  18.144  1.00  0.00           H  \\nATOM   5444  N   GLN B 348       3.847  13.615  16.729  1.00  0.00           N  \\nATOM   5445  CA  GLN B 348       3.463  14.993  16.836  1.00  0.00           C  \\nATOM   5446  C   GLN B 348       4.275  15.717  17.907  1.00  0.00           C  \\nATOM   5447  O   GLN B 348       4.593  16.926  17.726  1.00  0.00           O  \\nATOM   5448  CB  GLN B 348       1.967  15.098  17.142  1.00  0.00           C  \\nATOM   5449  CG  GLN B 348       1.470  16.519  17.345  1.00  0.00           C  \\nATOM   5450  CD  GLN B 348       1.615  17.366  16.095  1.00  0.00           C  \\nATOM   5451  OE1 GLN B 348       1.137  16.996  15.019  1.00  0.00           O  \\nATOM   5452  NE2 GLN B 348       2.276  18.510  16.229  1.00  0.00           N  \\nATOM   5453  H   GLN B 348       3.183  12.898  16.985  1.00  0.00           H  \\nATOM   5454  HA  GLN B 348       3.617  15.471  15.869  1.00  0.00           H  \\nATOM   5455 1HB  GLN B 348       1.396  14.655  16.325  1.00  0.00           H  \\nATOM   5456 2HB  GLN B 348       1.739  14.530  18.045  1.00  0.00           H  \\nATOM   5457 1HG  GLN B 348       0.415  16.489  17.617  1.00  0.00           H  \\nATOM   5458 2HG  GLN B 348       2.049  16.985  18.142  1.00  0.00           H  \\nATOM   5459 1HE2 GLN B 348       2.403  19.111  15.439  1.00  0.00           H  \\nATOM   5460 2HE2 GLN B 348       2.647  18.772  17.121  1.00  0.00           H  \\nATOM   5461  N   GLU B 349       4.567  15.021  19.020  1.00  0.00           N  \\nATOM   5462  CA  GLU B 349       5.251  15.723  20.133  1.00  0.00           C  \\nATOM   5463  C   GLU B 349       6.711  16.060  19.672  1.00  0.00           C  \\nATOM   5464  O   GLU B 349       7.244  17.130  20.009  1.00  0.00           O  \\nATOM   5465  CB  GLU B 349       5.054  14.970  21.502  1.00  0.00           C  \\nATOM   5466  CG  GLU B 349       5.241  13.429  21.481  1.00  0.00           C  \\nATOM   5467  CD  GLU B 349       4.244  12.529  22.363  1.00  0.00           C  \\nATOM   5468  OE1 GLU B 349       4.526  12.230  23.574  1.00  0.00           O  \\nATOM   5469  OE2 GLU B 349       3.205  12.044  21.823  1.00  0.00           O  \\nATOM   5470  H   GLU B 349       4.338  14.043  19.122  1.00  0.00           H  \\nATOM   5471  HA  GLU B 349       4.820  16.720  20.230  1.00  0.00           H  \\nATOM   5472 1HB  GLU B 349       5.758  15.361  22.237  1.00  0.00           H  \\nATOM   5473 2HB  GLU B 349       4.049  15.158  21.879  1.00  0.00           H  \\nATOM   5474 1HG  GLU B 349       5.138  13.075  20.456  1.00  0.00           H  \\nATOM   5475 2HG  GLU B 349       6.249  13.191  21.818  1.00  0.00           H  \\nATOM   5476  N   ASP B 350       7.358  15.150  18.957  1.00  0.00           N  \\nATOM   5477  CA  ASP B 350       8.695  15.474  18.382  1.00  0.00           C  \\nATOM   5478  C   ASP B 350       8.611  16.574  17.312  1.00  0.00           C  \\nATOM   5479  O   ASP B 350       9.560  17.360  17.183  1.00  0.00           O  \\nATOM   5480  CB  ASP B 350       9.287  14.248  17.738  1.00  0.00           C  \\nATOM   5481  CG  ASP B 350       9.791  13.208  18.776  1.00  0.00           C  \\nATOM   5482  OD1 ASP B 350       9.866  13.537  19.976  1.00  0.00           O  \\nATOM   5483  OD2 ASP B 350      10.211  12.107  18.319  1.00  0.00           O  \\nATOM   5484  H   ASP B 350       6.960  14.236  18.793  1.00  0.00           H  \\nATOM   5485  HA  ASP B 350       9.350  15.800  19.191  1.00  0.00           H  \\nATOM   5486 1HB  ASP B 350       8.539  13.774  17.102  1.00  0.00           H  \\nATOM   5487 2HB  ASP B 350      10.122  14.539  17.100  1.00  0.00           H  \\nATOM   5488  N   ALA B 351       7.569  16.542  16.456  1.00  0.00           N  \\nATOM   5489  CA  ALA B 351       7.316  17.719  15.586  1.00  0.00           C  \\nATOM   5490  C   ALA B 351       7.191  19.006  16.275  1.00  0.00           C  \\nATOM   5491  O   ALA B 351       7.714  20.024  15.782  1.00  0.00           O  \\nATOM   5492  CB  ALA B 351       6.136  17.473  14.642  1.00  0.00           C  \\nATOM   5493  H   ALA B 351       6.957  15.741  16.390  1.00  0.00           H  \\nATOM   5494  HA  ALA B 351       8.208  17.893  14.984  1.00  0.00           H  \\nATOM   5495 1HB  ALA B 351       5.978  18.354  14.020  1.00  0.00           H  \\nATOM   5496 2HB  ALA B 351       6.351  16.614  14.007  1.00  0.00           H  \\nATOM   5497 3HB  ALA B 351       5.238  17.276  15.226  1.00  0.00           H  \\nATOM   5498  N   GLU B 352       6.499  19.037  17.405  1.00  0.00           N  \\nATOM   5499  CA  GLU B 352       6.381  20.248  18.133  1.00  0.00           C  \\nATOM   5500  C   GLU B 352       7.768  20.612  18.715  1.00  0.00           C  \\nATOM   5501  O   GLU B 352       8.248  21.791  18.620  1.00  0.00           O  \\nATOM   5502  CB  GLU B 352       5.311  20.111  19.208  1.00  0.00           C  \\nATOM   5503  CG  GLU B 352       4.970  21.500  19.779  1.00  0.00           C  \\nATOM   5504  CD  GLU B 352       5.908  22.051  20.879  1.00  0.00           C  \\nATOM   5505  OE1 GLU B 352       6.669  21.238  21.456  1.00  0.00           O  \\nATOM   5506  OE2 GLU B 352       5.868  23.283  21.183  1.00  0.00           O  \\nATOM   5507  H   GLU B 352       6.050  18.204  17.759  1.00  0.00           H  \\nATOM   5508  HA  GLU B 352       6.087  21.039  17.443  1.00  0.00           H  \\nATOM   5509 1HB  GLU B 352       4.422  19.649  18.779  1.00  0.00           H  \\nATOM   5510 2HB  GLU B 352       5.672  19.454  19.999  1.00  0.00           H  \\nATOM   5511 1HG  GLU B 352       4.976  22.227  18.967  1.00  0.00           H  \\nATOM   5512 2HG  GLU B 352       3.965  21.472  20.198  1.00  0.00           H  \\nATOM   5513  N   ALA B 353       8.533  19.593  19.162  1.00  0.00           N  \\nATOM   5514  CA  ALA B 353       9.748  20.003  19.847  1.00  0.00           C  \\nATOM   5515  C   ALA B 353      10.704  20.738  18.929  1.00  0.00           C  \\nATOM   5516  O   ALA B 353      11.347  21.684  19.334  1.00  0.00           O  \\nATOM   5517  CB  ALA B 353      10.452  18.774  20.444  1.00  0.00           C  \\nATOM   5518  H   ALA B 353       8.334  18.608  19.061  1.00  0.00           H  \\nATOM   5519  HA  ALA B 353       9.470  20.682  20.653  1.00  0.00           H  \\nATOM   5520 1HB  ALA B 353      11.362  19.089  20.956  1.00  0.00           H  \\nATOM   5521 2HB  ALA B 353       9.787  18.283  21.155  1.00  0.00           H  \\nATOM   5522 3HB  ALA B 353      10.707  18.078  19.646  1.00  0.00           H  \\nATOM   5523  N   ILE B 354      10.819  20.248  17.699  1.00  0.00           N  \\nATOM   5524  CA  ILE B 354      11.798  20.803  16.740  1.00  0.00           C  \\nATOM   5525  C   ILE B 354      11.469  22.221  16.314  1.00  0.00           C  \\nATOM   5526  O   ILE B 354      12.339  22.929  15.755  1.00  0.00           O  \\nATOM   5527  CB  ILE B 354      12.165  19.863  15.613  1.00  0.00           C  \\nATOM   5528  CG1 ILE B 354      10.954  19.593  14.710  1.00  0.00           C  \\nATOM   5529  CG2 ILE B 354      12.801  18.609  16.223  1.00  0.00           C  \\nATOM   5530  CD1 ILE B 354      11.297  18.916  13.403  1.00  0.00           C  \\nATOM   5531  H   ILE B 354      10.228  19.482  17.410  1.00  0.00           H  \\nATOM   5532  HA  ILE B 354      12.719  21.026  17.277  1.00  0.00           H  \\nATOM   5533  HB  ILE B 354      12.872  20.354  14.945  1.00  0.00           H  \\nATOM   5534 1HG1 ILE B 354      10.239  18.965  15.239  1.00  0.00           H  \\nATOM   5535 2HG1 ILE B 354      10.454  20.534  14.481  1.00  0.00           H  \\nATOM   5536 1HG2 ILE B 354      13.073  17.916  15.428  1.00  0.00           H  \\nATOM   5537 2HG2 ILE B 354      13.693  18.889  16.782  1.00  0.00           H  \\nATOM   5538 3HG2 ILE B 354      12.088  18.129  16.894  1.00  0.00           H  \\nATOM   5539 1HD1 ILE B 354      10.387  18.761  12.823  1.00  0.00           H  \\nATOM   5540 2HD1 ILE B 354      11.985  19.545  12.837  1.00  0.00           H  \\nATOM   5541 3HD1 ILE B 354      11.766  17.954  13.604  1.00  0.00           H  \\nATOM   5542  N   THR B 355      10.222  22.637  16.577  1.00  0.00           N  \\nATOM   5543  CA  THR B 355       9.894  24.031  16.277  1.00  0.00           C  \\nATOM   5544  C   THR B 355      10.671  24.997  17.139  1.00  0.00           C  \\nATOM   5545  O   THR B 355      10.699  26.227  16.835  1.00  0.00           O  \\nATOM   5546  CB  THR B 355       8.338  24.345  16.336  1.00  0.00           C  \\nATOM   5547  OG1 THR B 355       7.913  24.341  17.713  1.00  0.00           O  \\nATOM   5548  CG2 THR B 355       7.542  23.322  15.475  1.00  0.00           C  \\nATOM   5549  H   THR B 355       9.510  22.037  16.968  1.00  0.00           H  \\nATOM   5550  HA  THR B 355      10.228  24.255  15.264  1.00  0.00           H  \\nATOM   5551  HB  THR B 355       8.155  25.350  15.957  1.00  0.00           H  \\nATOM   5552  HG1 THR B 355       8.665  24.150  18.279  1.00  0.00           H  \\nATOM   5553 1HG2 THR B 355       6.478  23.553  15.527  1.00  0.00           H  \\nATOM   5554 2HG2 THR B 355       7.876  23.380  14.439  1.00  0.00           H  \\nATOM   5555 3HG2 THR B 355       7.713  22.316  15.855  1.00  0.00           H  \\nATOM   5556  N   HIS B 356      11.323  24.516  18.210  1.00  0.00           N  \\nATOM   5557  CA  HIS B 356      12.069  25.351  19.132  1.00  0.00           C  \\nATOM   5558  C   HIS B 356      13.543  25.489  18.767  1.00  0.00           C  \\nATOM   5559  O   HIS B 356      14.286  26.163  19.479  1.00  0.00           O  \\nATOM   5560  CB  HIS B 356      11.920  24.799  20.550  1.00  0.00           C  \\nATOM   5561  CG  HIS B 356      10.492  24.831  21.038  1.00  0.00           C  \\nATOM   5562  ND1 HIS B 356      10.043  25.791  21.912  1.00  0.00           N  \\nATOM   5563  CD2 HIS B 356       9.438  24.015  20.785  1.00  0.00           C  \\nATOM   5564  CE1 HIS B 356       8.764  25.574  22.172  1.00  0.00           C  \\nATOM   5565  NE2 HIS B 356       8.373  24.500  21.506  1.00  0.00           N  \\nATOM   5566  H   HIS B 356      11.284  23.520  18.370  1.00  0.00           H  \\nATOM   5567  HA  HIS B 356      11.671  26.365  19.108  1.00  0.00           H  \\nATOM   5568 1HB  HIS B 356      12.279  23.770  20.579  1.00  0.00           H  \\nATOM   5569 2HB  HIS B 356      12.538  25.381  21.234  1.00  0.00           H  \\nATOM   5570  HD2 HIS B 356       9.435  23.139  20.136  1.00  0.00           H  \\nATOM   5571  HE1 HIS B 356       8.135  26.178  22.825  1.00  0.00           H  \\nATOM   5572  HE2 HIS B 356       7.446  24.100  21.522  1.00  0.00           H  \\nATOM   5573  N   LEU B 357      13.957  24.779  17.712  1.00  0.00           N  \\nATOM   5574  CA  LEU B 357      15.316  24.810  17.196  1.00  0.00           C  \\nATOM   5575  C   LEU B 357      15.492  25.876  16.106  1.00  0.00           C  \\nATOM   5576  O   LEU B 357      14.622  25.943  15.191  1.00  0.00           O  \\nATOM   5577  CB  LEU B 357      15.645  23.402  16.594  1.00  0.00           C  \\nATOM   5578  CG  LEU B 357      16.033  22.471  17.729  1.00  0.00           C  \\nATOM   5579  CD1 LEU B 357      16.025  21.076  17.171  1.00  0.00           C  \\nATOM   5580  CD2 LEU B 357      17.434  22.761  18.295  1.00  0.00           C  \\nATOM   5581  H   LEU B 357      13.272  24.192  17.259  1.00  0.00           H  \\nATOM   5582  HA  LEU B 357      15.994  25.024  18.021  1.00  0.00           H  \\nATOM   5583 1HB  LEU B 357      14.767  23.037  16.064  1.00  0.00           H  \\nATOM   5584 2HB  LEU B 357      16.457  23.512  15.876  1.00  0.00           H  \\nATOM   5585  HG  LEU B 357      15.319  22.576  18.546  1.00  0.00           H  \\nATOM   5586 1HD1 LEU B 357      16.299  20.369  17.954  1.00  0.00           H  \\nATOM   5587 2HD1 LEU B 357      15.028  20.838  16.801  1.00  0.00           H  \\nATOM   5588 3HD1 LEU B 357      16.742  21.007  16.354  1.00  0.00           H  \\nATOM   5589 1HD2 LEU B 357      17.652  22.062  19.104  1.00  0.00           H  \\nATOM   5590 2HD2 LEU B 357      18.177  22.645  17.505  1.00  0.00           H  \\nATOM   5591 3HD2 LEU B 357      17.468  23.781  18.678  1.00  0.00           H  \\nATOM   5592  N   PHE B 358      16.613  26.598  16.184  1.00  0.00           N  \\nATOM   5593  CA  PHE B 358      17.032  27.534  15.100  1.00  0.00           C  \\nATOM   5594  C   PHE B 358      15.849  28.510  14.737  1.00  0.00           C  \\nATOM   5595  O   PHE B 358      15.511  28.672  13.549  1.00  0.00           O  \\nATOM   5596  CB  PHE B 358      17.435  26.760  13.839  1.00  0.00           C  \\nATOM   5597  CG  PHE B 358      18.505  25.654  14.080  1.00  0.00           C  \\nATOM   5598  CD1 PHE B 358      19.714  25.957  14.722  1.00  0.00           C  \\nATOM   5599  CD2 PHE B 358      18.241  24.384  13.703  1.00  0.00           C  \\nATOM   5600  CE1 PHE B 358      20.686  24.926  14.962  1.00  0.00           C  \\nATOM   5601  CE2 PHE B 358      19.221  23.352  13.970  1.00  0.00           C  \\nATOM   5602  CZ  PHE B 358      20.418  23.643  14.547  1.00  0.00           C  \\nATOM   5603  H   PHE B 358      17.194  26.505  17.005  1.00  0.00           H  \\nATOM   5604  HA  PHE B 358      17.896  28.102  15.448  1.00  0.00           H  \\nATOM   5605 1HB  PHE B 358      16.555  26.286  13.406  1.00  0.00           H  \\nATOM   5606 2HB  PHE B 358      17.831  27.453  13.098  1.00  0.00           H  \\nATOM   5607  HD1 PHE B 358      19.907  26.983  15.037  1.00  0.00           H  \\nATOM   5608  HD2 PHE B 358      17.289  24.124  13.239  1.00  0.00           H  \\nATOM   5609  HE1 PHE B 358      21.621  25.166  15.467  1.00  0.00           H  \\nATOM   5610  HE2 PHE B 358      18.984  22.324  13.698  1.00  0.00           H  \\nATOM   5611  HZ  PHE B 358      21.163  22.860  14.678  1.00  0.00           H  \\nATOM   5612  N   THR B 359      15.217  29.065  15.764  1.00  0.00           N  \\nATOM   5613  CA  THR B 359      13.902  29.778  15.551  1.00  0.00           C  \\nATOM   5614  C   THR B 359      14.019  31.099  14.779  1.00  0.00           C  \\nATOM   5615  O   THR B 359      13.013  31.603  14.248  1.00  0.00           O  \\nATOM   5616  CB  THR B 359      13.110  30.025  16.835  1.00  0.00           C  \\nATOM   5617  OG1 THR B 359      13.894  30.813  17.727  1.00  0.00           O  \\nATOM   5618  CG2 THR B 359      12.761  28.702  17.522  1.00  0.00           C  \\nATOM   5619  H   THR B 359      15.606  29.017  16.695  1.00  0.00           H  \\nATOM   5620  HA  THR B 359      13.275  29.162  14.906  1.00  0.00           H  \\nATOM   5621  HB  THR B 359      12.188  30.556  16.597  1.00  0.00           H  \\nATOM   5622  HG1 THR B 359      14.741  31.010  17.319  1.00  0.00           H  \\nATOM   5623 1HG2 THR B 359      12.197  28.903  18.433  1.00  0.00           H  \\nATOM   5624 2HG2 THR B 359      12.159  28.091  16.850  1.00  0.00           H  \\nATOM   5625 3HG2 THR B 359      13.678  28.170  17.773  1.00  0.00           H  \\nATOM   5626  N   ASP B 360      15.224  31.630  14.724  1.00  0.00           N  \\nATOM   5627  CA  ASP B 360      15.560  32.882  13.981  1.00  0.00           C  \\nATOM   5628  C   ASP B 360      15.960  32.622  12.529  1.00  0.00           C  \\nATOM   5629  O   ASP B 360      16.191  33.560  11.742  1.00  0.00           O  \\nATOM   5630  CB  ASP B 360      16.750  33.563  14.648  1.00  0.00           C  \\nATOM   5631  CG  ASP B 360      17.979  32.687  14.640  1.00  0.00           C  \\nATOM   5632  OD1 ASP B 360      17.949  31.564  15.208  1.00  0.00           O  \\nATOM   5633  OD2 ASP B 360      18.980  33.083  14.023  1.00  0.00           O  \\nATOM   5634  H   ASP B 360      15.955  31.143  15.224  1.00  0.00           H  \\nATOM   5635  HA  ASP B 360      14.698  33.549  14.019  1.00  0.00           H  \\nATOM   5636 1HB  ASP B 360      16.973  34.496  14.130  1.00  0.00           H  \\nATOM   5637 2HB  ASP B 360      16.496  33.813  15.678  1.00  0.00           H  \\nATOM   5638  N   VAL B 361      16.080  31.363  12.146  1.00  0.00           N  \\nATOM   5639  CA  VAL B 361      16.496  31.050  10.788  1.00  0.00           C  \\nATOM   5640  C   VAL B 361      15.433  31.461   9.760  1.00  0.00           C  \\nATOM   5641  O   VAL B 361      14.213  31.359  10.024  1.00  0.00           O  \\nATOM   5642  CB  VAL B 361      16.844  29.529  10.694  1.00  0.00           C  \\nATOM   5643  CG1 VAL B 361      15.671  28.703  10.151  1.00  0.00           C  \\nATOM   5644  CG2 VAL B 361      18.073  29.310   9.864  1.00  0.00           C  \\nATOM   5645  H   VAL B 361      15.886  30.607  12.787  1.00  0.00           H  \\nATOM   5646  HA  VAL B 361      17.385  31.637  10.554  1.00  0.00           H  \\nATOM   5647  HB  VAL B 361      17.021  29.143  11.698  1.00  0.00           H  \\nATOM   5648 1HG1 VAL B 361      15.957  27.652  10.103  1.00  0.00           H  \\nATOM   5649 2HG1 VAL B 361      14.811  28.814  10.811  1.00  0.00           H  \\nATOM   5650 3HG1 VAL B 361      15.411  29.053   9.152  1.00  0.00           H  \\nATOM   5651 1HG2 VAL B 361      18.294  28.244   9.815  1.00  0.00           H  \\nATOM   5652 2HG2 VAL B 361      17.905  29.693   8.857  1.00  0.00           H  \\nATOM   5653 3HG2 VAL B 361      18.915  29.834  10.316  1.00  0.00           H  \\nATOM   5654  N   GLN B 362      15.878  31.919   8.585  1.00  0.00           N  \\nATOM   5655  CA  GLN B 362      14.944  32.459   7.610  1.00  0.00           C  \\nATOM   5656  C   GLN B 362      14.060  31.414   6.981  1.00  0.00           C  \\nATOM   5657  O   GLN B 362      12.878  31.651   6.703  1.00  0.00           O  \\nATOM   5658  CB  GLN B 362      15.690  33.286   6.538  1.00  0.00           C  \\nATOM   5659  CG  GLN B 362      16.581  34.358   7.157  1.00  0.00           C  \\nATOM   5660  CD  GLN B 362      16.790  35.532   6.241  1.00  0.00           C  \\nATOM   5661  OE1 GLN B 362      17.255  35.381   5.111  1.00  0.00           O  \\nATOM   5662  NE2 GLN B 362      16.421  36.721   6.717  1.00  0.00           N  \\nATOM   5663  H   GLN B 362      16.863  31.895   8.362  1.00  0.00           H  \\nATOM   5664  HA  GLN B 362      14.244  33.114   8.128  1.00  0.00           H  \\nATOM   5665 1HB  GLN B 362      16.303  32.622   5.928  1.00  0.00           H  \\nATOM   5666 2HB  GLN B 362      14.966  33.763   5.877  1.00  0.00           H  \\nATOM   5667 1HG  GLN B 362      16.114  34.719   8.073  1.00  0.00           H  \\nATOM   5668 2HG  GLN B 362      17.555  33.922   7.381  1.00  0.00           H  \\nATOM   5669 1HE2 GLN B 362      16.534  37.542   6.156  1.00  0.00           H  \\nATOM   5670 2HE2 GLN B 362      16.031  36.793   7.635  1.00  0.00           H  \\nATOM   5671  N   ASN B 363      14.637  30.221   6.773  1.00  0.00           N  \\nATOM   5672  CA  ASN B 363      14.013  29.137   6.015  1.00  0.00           C  \\nATOM   5673  C   ASN B 363      14.098  27.814   6.778  1.00  0.00           C  \\nATOM   5674  O   ASN B 363      15.187  27.492   7.322  1.00  0.00           O  \\nATOM   5675  CB  ASN B 363      14.772  28.949   4.752  1.00  0.00           C  \\nATOM   5676  CG  ASN B 363      15.035  30.242   4.053  1.00  0.00           C  \\nATOM   5677  OD1 ASN B 363      16.190  30.673   3.864  1.00  0.00           O  \\nATOM   5678  ND2 ASN B 363      13.954  30.883   3.652  1.00  0.00           N  \\nATOM   5679  H   ASN B 363      15.555  30.079   7.169  1.00  0.00           H  \\nATOM   5680  HA  ASN B 363      12.985  29.422   5.786  1.00  0.00           H  \\nATOM   5681 1HB  ASN B 363      15.724  28.462   4.968  1.00  0.00           H  \\nATOM   5682 2HB  ASN B 363      14.212  28.293   4.085  1.00  0.00           H  \\nATOM   5683 1HD2 ASN B 363      14.040  31.758   3.175  1.00  0.00           H  \\nATOM   5684 2HD2 ASN B 363      13.050  30.494   3.825  1.00  0.00           H  \\nATOM   5685  N   ARG B 364      12.978  27.084   6.795  1.00  0.00           N  \\nATOM   5686  CA  ARG B 364      12.905  25.763   7.538  1.00  0.00           C  \\nATOM   5687  C   ARG B 364      12.314  24.758   6.628  1.00  0.00           C  \\nATOM   5688  O   ARG B 364      11.333  25.003   5.906  1.00  0.00           O  \\nATOM   5689  CB  ARG B 364      12.029  25.869   8.790  1.00  0.00           C  \\nATOM   5690  CG  ARG B 364      12.423  26.933   9.767  1.00  0.00           C  \\nATOM   5691  CD  ARG B 364      11.373  27.029  10.771  1.00  0.00           C  \\nATOM   5692  NE  ARG B 364      11.462  25.903  11.681  1.00  0.00           N  \\nATOM   5693  CZ  ARG B 364      12.287  25.889  12.721  1.00  0.00           C  \\nATOM   5694  NH1 ARG B 364      13.123  26.907  13.006  1.00  0.00           N  \\nATOM   5695  NH2 ARG B 364      12.270  24.839  13.489  1.00  0.00           N  \\nATOM   5696  H   ARG B 364      12.161  27.418   6.303  1.00  0.00           H  \\nATOM   5697  HA  ARG B 364      13.912  25.486   7.851  1.00  0.00           H  \\nATOM   5698 1HB  ARG B 364      10.999  26.066   8.497  1.00  0.00           H  \\nATOM   5699 2HB  ARG B 364      12.042  24.919   9.325  1.00  0.00           H  \\nATOM   5700 1HG  ARG B 364      13.370  26.664  10.236  1.00  0.00           H  \\nATOM   5701 2HG  ARG B 364      12.534  27.884   9.245  1.00  0.00           H  \\nATOM   5702 1HD  ARG B 364      11.489  27.956  11.332  1.00  0.00           H  \\nATOM   5703 2HD  ARG B 364      10.400  27.021  10.281  1.00  0.00           H  \\nATOM   5704  HE  ARG B 364      10.866  25.106  11.502  1.00  0.00           H  \\nATOM   5705 1HH1 ARG B 364      13.140  27.727  12.416  1.00  0.00           H  \\nATOM   5706 2HH1 ARG B 364      13.732  26.850  13.809  1.00  0.00           H  \\nATOM   5707 1HH2 ARG B 364      11.644  24.073  13.284  1.00  0.00           H  \\nATOM   5708 2HH2 ARG B 364      12.882  24.790  14.290  1.00  0.00           H  \\nATOM   5709  N   TYR B 365      12.861  23.560   6.686  1.00  0.00           N  \\nATOM   5710  CA  TYR B 365      12.403  22.502   5.908  1.00  0.00           C  \\nATOM   5711  C   TYR B 365      12.495  21.192   6.654  1.00  0.00           C  \\nATOM   5712  O   TYR B 365      13.546  20.920   7.239  1.00  0.00           O  \\nATOM   5713  CB  TYR B 365      13.302  22.404   4.684  1.00  0.00           C  \\nATOM   5714  CG  TYR B 365      12.837  21.383   3.705  1.00  0.00           C  \\nATOM   5715  CD1 TYR B 365      11.737  21.612   2.884  1.00  0.00           C  \\nATOM   5716  CD2 TYR B 365      13.489  20.155   3.603  1.00  0.00           C  \\nATOM   5717  CE1 TYR B 365      11.280  20.665   2.031  1.00  0.00           C  \\nATOM   5718  CE2 TYR B 365      13.067  19.199   2.687  1.00  0.00           C  \\nATOM   5719  CZ  TYR B 365      11.943  19.470   1.932  1.00  0.00           C  \\nATOM   5720  OH  TYR B 365      11.549  18.525   1.058  1.00  0.00           O  \\nATOM   5721  H   TYR B 365      13.636  23.408   7.315  1.00  0.00           H  \\nATOM   5722  HA  TYR B 365      11.379  22.716   5.601  1.00  0.00           H  \\nATOM   5723 1HB  TYR B 365      13.344  23.373   4.185  1.00  0.00           H  \\nATOM   5724 2HB  TYR B 365      14.315  22.152   4.996  1.00  0.00           H  \\nATOM   5725  HD1 TYR B 365      11.222  22.572   2.918  1.00  0.00           H  \\nATOM   5726  HD2 TYR B 365      14.340  19.935   4.247  1.00  0.00           H  \\nATOM   5727  HE1 TYR B 365      10.416  20.872   1.400  1.00  0.00           H  \\nATOM   5728  HE2 TYR B 365      13.619  18.265   2.580  1.00  0.00           H  \\nATOM   5729  HH  TYR B 365      12.148  17.776   1.106  1.00  0.00           H  \\nATOM   5730  N   THR B 366      11.440  20.376   6.577  1.00  0.00           N  \\nATOM   5731  CA  THR B 366      11.378  19.010   7.119  1.00  0.00           C  \\nATOM   5732  C   THR B 366      11.429  18.023   5.947  1.00  0.00           C  \\nATOM   5733  O   THR B 366      10.600  18.033   5.027  1.00  0.00           O  \\nATOM   5734  CB  THR B 366      10.119  18.809   8.012  1.00  0.00           C  \\nATOM   5735  OG1 THR B 366      10.121  19.741   9.102  1.00  0.00           O  \\nATOM   5736  CG2 THR B 366      10.055  17.336   8.534  1.00  0.00           C  \\nATOM   5737  H   THR B 366      10.633  20.754   6.102  1.00  0.00           H  \\nATOM   5738  HA  THR B 366      12.262  18.842   7.734  1.00  0.00           H  \\nATOM   5739  HB  THR B 366       9.223  19.024   7.431  1.00  0.00           H  \\nATOM   5740  HG1 THR B 366      10.904  20.294   9.048  1.00  0.00           H  \\nATOM   5741 1HG2 THR B 366       9.170  17.209   9.157  1.00  0.00           H  \\nATOM   5742 2HG2 THR B 366      10.005  16.652   7.687  1.00  0.00           H  \\nATOM   5743 3HG2 THR B 366      10.946  17.120   9.122  1.00  0.00           H  \\nATOM   5744  N   PHE B 367      12.402  17.126   5.988  1.00  0.00           N  \\nATOM   5745  CA  PHE B 367      12.580  16.138   4.935  1.00  0.00           C  \\nATOM   5746  C   PHE B 367      11.460  15.146   5.023  1.00  0.00           C  \\nATOM   5747  O   PHE B 367      10.813  14.982   6.082  1.00  0.00           O  \\nATOM   5748  CB  PHE B 367      13.898  15.348   5.145  1.00  0.00           C  \\nATOM   5749  CG  PHE B 367      15.118  16.177   4.931  1.00  0.00           C  \\nATOM   5750  CD1 PHE B 367      15.393  16.781   3.700  1.00  0.00           C  \\nATOM   5751  CD2 PHE B 367      16.063  16.309   5.961  1.00  0.00           C  \\nATOM   5752  CE1 PHE B 367      16.546  17.543   3.554  1.00  0.00           C  \\nATOM   5753  CE2 PHE B 367      17.184  17.033   5.789  1.00  0.00           C  \\nATOM   5754  CZ  PHE B 367      17.426  17.677   4.601  1.00  0.00           C  \\nATOM   5755  H   PHE B 367      13.038  17.129   6.773  1.00  0.00           H  \\nATOM   5756  HA  PHE B 367      12.635  16.658   3.978  1.00  0.00           H  \\nATOM   5757 1HB  PHE B 367      13.925  14.948   6.158  1.00  0.00           H  \\nATOM   5758 2HB  PHE B 367      13.929  14.503   4.458  1.00  0.00           H  \\nATOM   5759  HD1 PHE B 367      14.696  16.645   2.873  1.00  0.00           H  \\nATOM   5760  HD2 PHE B 367      15.882  15.802   6.909  1.00  0.00           H  \\nATOM   5761  HE1 PHE B 367      16.755  18.038   2.606  1.00  0.00           H  \\nATOM   5762  HE2 PHE B 367      17.904  17.105   6.604  1.00  0.00           H  \\nATOM   5763  HZ  PHE B 367      18.317  18.293   4.490  1.00  0.00           H  \\nATOM   5764  N   ALA B 368      11.179  14.559   3.858  1.00  0.00           N  \\nATOM   5765  CA  ALA B 368      10.181  13.523   3.803  1.00  0.00           C  \\nATOM   5766  C   ALA B 368      10.565  12.234   4.545  1.00  0.00           C  \\nATOM   5767  O   ALA B 368       9.707  11.485   4.944  1.00  0.00           O  \\nATOM   5768  CB  ALA B 368       9.773  13.230   2.347  1.00  0.00           C  \\nATOM   5769  H   ALA B 368      11.655  14.829   3.009  1.00  0.00           H  \\nATOM   5770  HA  ALA B 368       9.303  13.869   4.348  1.00  0.00           H  \\nATOM   5771 1HB  ALA B 368       9.018  12.444   2.332  1.00  0.00           H  \\nATOM   5772 2HB  ALA B 368       9.365  14.134   1.895  1.00  0.00           H  \\nATOM   5773 3HB  ALA B 368      10.646  12.905   1.783  1.00  0.00           H  \\nATOM   5774  N   PHE B 369      11.858  12.071   4.875  1.00  0.00           N  \\nATOM   5775  CA  PHE B 369      12.460  10.830   5.318  1.00  0.00           C  \\nATOM   5776  C   PHE B 369      12.953  11.108   6.755  1.00  0.00           C  \\nATOM   5777  O   PHE B 369      13.311  12.252   7.088  1.00  0.00           O  \\nATOM   5778  CB  PHE B 369      13.610  10.420   4.334  1.00  0.00           C  \\nATOM   5779  CG  PHE B 369      14.658  11.486   4.118  1.00  0.00           C  \\nATOM   5780  CD1 PHE B 369      15.705  11.710   5.054  1.00  0.00           C  \\nATOM   5781  CD2 PHE B 369      14.662  12.227   2.963  1.00  0.00           C  \\nATOM   5782  CE1 PHE B 369      16.693  12.671   4.774  1.00  0.00           C  \\nATOM   5783  CE2 PHE B 369      15.602  13.205   2.742  1.00  0.00           C  \\nATOM   5784  CZ  PHE B 369      16.599  13.440   3.613  1.00  0.00           C  \\nATOM   5785  H   PHE B 369      12.434  12.897   4.799  1.00  0.00           H  \\nATOM   5786  HA  PHE B 369      11.694  10.053   5.314  1.00  0.00           H  \\nATOM   5787 1HB  PHE B 369      14.112   9.530   4.712  1.00  0.00           H  \\nATOM   5788 2HB  PHE B 369      13.185  10.168   3.363  1.00  0.00           H  \\nATOM   5789  HD1 PHE B 369      15.727  11.131   5.978  1.00  0.00           H  \\nATOM   5790  HD2 PHE B 369      13.866  12.062   2.236  1.00  0.00           H  \\nATOM   5791  HE1 PHE B 369      17.526  12.811   5.462  1.00  0.00           H  \\nATOM   5792  HE2 PHE B 369      15.523  13.799   1.832  1.00  0.00           H  \\nATOM   5793  HZ  PHE B 369      17.328  14.225   3.417  1.00  0.00           H  \\nATOM   5794  N   GLY B 370      12.921  10.064   7.577  1.00  0.00           N  \\nATOM   5795  CA  GLY B 370      13.456  10.052   8.939  1.00  0.00           C  \\nATOM   5796  C   GLY B 370      14.952   9.855   8.855  1.00  0.00           C  \\nATOM   5797  O   GLY B 370      15.519   9.627   7.774  1.00  0.00           O  \\nATOM   5798  H   GLY B 370      12.490   9.228   7.211  1.00  0.00           H  \\nATOM   5799 1HA  GLY B 370      13.208  10.990   9.436  1.00  0.00           H  \\nATOM   5800 2HA  GLY B 370      12.983   9.252   9.508  1.00  0.00           H  \\nATOM   5801  N   GLY B 371      15.591   9.893   9.999  1.00  0.00           N  \\nATOM   5802  CA  GLY B 371      17.055   9.818  10.011  1.00  0.00           C  \\nATOM   5803  C   GLY B 371      17.557   8.506  10.591  1.00  0.00           C  \\nATOM   5804  O   GLY B 371      18.709   8.444  11.009  1.00  0.00           O  \\nATOM   5805  H   GLY B 371      15.095   9.973  10.875  1.00  0.00           H  \\nATOM   5806 1HA  GLY B 371      17.433   9.929   8.995  1.00  0.00           H  \\nATOM   5807 2HA  GLY B 371      17.457  10.645  10.595  1.00  0.00           H  \\nATOM   5808  N   ASN B 372      16.682   7.457  10.579  1.00  0.00           N  \\nATOM   5809  CA  ASN B 372      17.121   6.113  10.952  1.00  0.00           C  \\nATOM   5810  C   ASN B 372      18.038   5.588   9.879  1.00  0.00           C  \\nATOM   5811  O   ASN B 372      17.892   5.870   8.623  1.00  0.00           O  \\nATOM   5812  CB  ASN B 372      15.925   5.170  11.096  1.00  0.00           C  \\nATOM   5813  CG  ASN B 372      15.069   5.160   9.823  1.00  0.00           C  \\nATOM   5814  OD1 ASN B 372      14.307   6.121   9.539  1.00  0.00           O  \\nATOM   5815  ND2 ASN B 372      15.150   4.051   9.063  1.00  0.00           N  \\nATOM   5816  H   ASN B 372      15.719   7.603  10.311  1.00  0.00           H  \\nATOM   5817  HA  ASN B 372      17.633   6.171  11.914  1.00  0.00           H  \\nATOM   5818 1HB  ASN B 372      16.281   4.160  11.304  1.00  0.00           H  \\nATOM   5819 2HB  ASN B 372      15.315   5.483  11.943  1.00  0.00           H  \\nATOM   5820 1HD2 ASN B 372      14.616   3.982   8.220  1.00  0.00           H  \\nATOM   5821 2HD2 ASN B 372      15.743   3.297   9.342  1.00  0.00           H  \\nATOM   5822  N   TYR B 373      19.046   4.793  10.261  1.00  0.00           N  \\nATOM   5823  CA  TYR B 373      20.005   4.268   9.298  1.00  0.00           C  \\nATOM   5824  C   TYR B 373      19.423   3.535   8.036  1.00  0.00           C  \\nATOM   5825  O   TYR B 373      19.965   3.686   6.949  1.00  0.00           O  \\nATOM   5826  CB  TYR B 373      21.023   3.319  10.008  1.00  0.00           C  \\nATOM   5827  CG  TYR B 373      22.028   3.989  10.957  1.00  0.00           C  \\nATOM   5828  CD1 TYR B 373      22.767   5.096  10.559  1.00  0.00           C  \\nATOM   5829  CD2 TYR B 373      22.140   3.546  12.282  1.00  0.00           C  \\nATOM   5830  CE1 TYR B 373      23.643   5.726  11.457  1.00  0.00           C  \\nATOM   5831  CE2 TYR B 373      23.015   4.184  13.176  1.00  0.00           C  \\nATOM   5832  CZ  TYR B 373      23.763   5.173  12.729  1.00  0.00           C  \\nATOM   5833  OH  TYR B 373      24.689   5.781  13.529  1.00  0.00           O  \\nATOM   5834  H   TYR B 373      19.146   4.551  11.236  1.00  0.00           H  \\nATOM   5835  HA  TYR B 373      20.552   5.105   8.863  1.00  0.00           H  \\nATOM   5836 1HB  TYR B 373      20.480   2.576  10.594  1.00  0.00           H  \\nATOM   5837 2HB  TYR B 373      21.604   2.783   9.258  1.00  0.00           H  \\nATOM   5838  HD1 TYR B 373      22.669   5.481   9.544  1.00  0.00           H  \\nATOM   5839  HD2 TYR B 373      21.546   2.699  12.626  1.00  0.00           H  \\nATOM   5840  HE1 TYR B 373      24.219   6.594  11.138  1.00  0.00           H  \\nATOM   5841  HE2 TYR B 373      23.066   3.858  14.215  1.00  0.00           H  \\nATOM   5842  HH  TYR B 373      24.677   5.373  14.398  1.00  0.00           H  \\nATOM   5843  N   ASP B 374      18.399   2.705   8.241  1.00  0.00           N  \\nATOM   5844  CA  ASP B 374      17.849   1.971   7.094  1.00  0.00           C  \\nATOM   5845  C   ASP B 374      17.440   2.889   5.962  1.00  0.00           C  \\nATOM   5846  O   ASP B 374      17.872   2.579   4.769  1.00  0.00           O  \\nATOM   5847  CB  ASP B 374      16.653   1.136   7.580  1.00  0.00           C  \\nATOM   5848  CG  ASP B 374      17.061  -0.015   8.529  1.00  0.00           C  \\nATOM   5849  OD1 ASP B 374      18.262  -0.318   8.641  1.00  0.00           O  \\nATOM   5850  OD2 ASP B 374      16.141  -0.638   9.129  1.00  0.00           O  \\nATOM   5851  H   ASP B 374      17.992   2.566   9.155  1.00  0.00           H  \\nATOM   5852  HA  ASP B 374      18.623   1.310   6.703  1.00  0.00           H  \\nATOM   5853 1HB  ASP B 374      15.947   1.782   8.102  1.00  0.00           H  \\nATOM   5854 2HB  ASP B 374      16.134   0.710   6.721  1.00  0.00           H  \\nATOM   5855  N   ARG B 375      16.736   3.975   6.297  1.00  0.00           N  \\nATOM   5856  CA  ARG B 375      16.296   4.928   5.310  1.00  0.00           C  \\nATOM   5857  C   ARG B 375      17.490   5.700   4.682  1.00  0.00           C  \\nATOM   5858  O   ARG B 375      17.606   5.881   3.432  1.00  0.00           O  \\nATOM   5859  CB  ARG B 375      15.323   5.901   5.951  1.00  0.00           C  \\nATOM   5860  CG  ARG B 375      14.986   7.146   5.076  1.00  0.00           C  \\nATOM   5861  CD  ARG B 375      14.255   6.832   3.722  1.00  0.00           C  \\nATOM   5862  NE  ARG B 375      13.000   6.075   3.865  1.00  0.00           N  \\nATOM   5863  CZ  ARG B 375      12.815   4.851   3.360  1.00  0.00           C  \\nATOM   5864  NH1 ARG B 375      13.795   4.206   2.669  1.00  0.00           N  \\nATOM   5865  NH2 ARG B 375      11.633   4.240   3.568  1.00  0.00           N  \\nATOM   5866  H   ARG B 375      16.507   4.131   7.268  1.00  0.00           H  \\nATOM   5867  HA  ARG B 375      15.789   4.388   4.510  1.00  0.00           H  \\nATOM   5868 1HB  ARG B 375      14.389   5.387   6.174  1.00  0.00           H  \\nATOM   5869 2HB  ARG B 375      15.734   6.258   6.895  1.00  0.00           H  \\nATOM   5870 1HG  ARG B 375      14.332   7.817   5.634  1.00  0.00           H  \\nATOM   5871 2HG  ARG B 375      15.907   7.669   4.818  1.00  0.00           H  \\nATOM   5872 1HD  ARG B 375      14.005   7.765   3.219  1.00  0.00           H  \\nATOM   5873 2HD  ARG B 375      14.910   6.240   3.084  1.00  0.00           H  \\nATOM   5874  HE  ARG B 375      12.243   6.512   4.374  1.00  0.00           H  \\nATOM   5875 1HH1 ARG B 375      14.690   4.652   2.527  1.00  0.00           H  \\nATOM   5876 2HH1 ARG B 375      13.627   3.282   2.299  1.00  0.00           H  \\nATOM   5877 1HH2 ARG B 375      10.909   4.707   4.096  1.00  0.00           H  \\nATOM   5878 2HH2 ARG B 375      11.470   3.316   3.196  1.00  0.00           H  \\nATOM   5879  N   LEU B 376      18.436   6.160   5.524  1.00  0.00           N  \\nATOM   5880  CA  LEU B 376      19.527   6.945   5.029  1.00  0.00           C  \\nATOM   5881  C   LEU B 376      20.427   6.132   4.108  1.00  0.00           C  \\nATOM   5882  O   LEU B 376      20.973   6.643   3.153  1.00  0.00           O  \\nATOM   5883  CB  LEU B 376      20.280   7.552   6.258  1.00  0.00           C  \\nATOM   5884  CG  LEU B 376      19.507   8.563   7.050  1.00  0.00           C  \\nATOM   5885  CD1 LEU B 376      20.457   8.901   8.260  1.00  0.00           C  \\nATOM   5886  CD2 LEU B 376      19.168   9.850   6.278  1.00  0.00           C  \\nATOM   5887  H   LEU B 376      18.385   5.956   6.512  1.00  0.00           H  \\nATOM   5888  HA  LEU B 376      19.124   7.745   4.410  1.00  0.00           H  \\nATOM   5889 1HB  LEU B 376      20.557   6.742   6.930  1.00  0.00           H  \\nATOM   5890 2HB  LEU B 376      21.193   8.031   5.904  1.00  0.00           H  \\nATOM   5891  HG  LEU B 376      18.561   8.127   7.372  1.00  0.00           H  \\nATOM   5892 1HD1 LEU B 376      19.978   9.638   8.904  1.00  0.00           H  \\nATOM   5893 2HD1 LEU B 376      20.654   7.994   8.832  1.00  0.00           H  \\nATOM   5894 3HD1 LEU B 376      21.396   9.305   7.883  1.00  0.00           H  \\nATOM   5895 1HD2 LEU B 376      18.608  10.526   6.925  1.00  0.00           H  \\nATOM   5896 2HD2 LEU B 376      20.090  10.336   5.957  1.00  0.00           H  \\nATOM   5897 3HD2 LEU B 376      18.566   9.602   5.404  1.00  0.00           H  \\nATOM   5898  N   GLU B 377      20.574   4.832   4.434  1.00  0.00           N  \\nATOM   5899  CA  GLU B 377      21.410   3.968   3.654  1.00  0.00           C  \\nATOM   5900  C   GLU B 377      20.728   3.747   2.265  1.00  0.00           C  \\nATOM   5901  O   GLU B 377      21.484   3.783   1.287  1.00  0.00           O  \\nATOM   5902  CB  GLU B 377      21.566   2.617   4.334  1.00  0.00           C  \\nATOM   5903  CG  GLU B 377      22.549   2.630   5.555  1.00  0.00           C  \\nATOM   5904  CD  GLU B 377      22.528   1.313   6.285  1.00  0.00           C  \\nATOM   5905  OE1 GLU B 377      21.537   0.571   6.185  1.00  0.00           O  \\nATOM   5906  OE2 GLU B 377      23.539   1.016   7.048  1.00  0.00           O  \\nATOM   5907  H   GLU B 377      20.091   4.456   5.237  1.00  0.00           H  \\nATOM   5908  HA  GLU B 377      22.395   4.428   3.565  1.00  0.00           H  \\nATOM   5909 1HB  GLU B 377      20.593   2.272   4.685  1.00  0.00           H  \\nATOM   5910 2HB  GLU B 377      21.931   1.886   3.612  1.00  0.00           H  \\nATOM   5911 1HG  GLU B 377      23.557   2.834   5.195  1.00  0.00           H  \\nATOM   5912 2HG  GLU B 377      22.264   3.436   6.229  1.00  0.00           H  \\nATOM   5913  N   GLN B 378      19.383   3.668   2.275  1.00  0.00           N  \\nATOM   5914  CA  GLN B 378      18.673   3.517   0.953  1.00  0.00           C  \\nATOM   5915  C   GLN B 378      18.941   4.748   0.125  1.00  0.00           C  \\nATOM   5916  O   GLN B 378      19.286   4.661  -1.084  1.00  0.00           O  \\nATOM   5917  CB  GLN B 378      17.207   3.197   1.227  1.00  0.00           C  \\nATOM   5918  CG  GLN B 378      16.390   3.083  -0.121  1.00  0.00           C  \\nATOM   5919  CD  GLN B 378      15.701   4.356  -0.463  1.00  0.00           C  \\nATOM   5920  OE1 GLN B 378      15.154   5.049   0.403  1.00  0.00           O  \\nATOM   5921  NE2 GLN B 378      15.704   4.685  -1.777  1.00  0.00           N  \\nATOM   5922  H   GLN B 378      18.838   3.706   3.125  1.00  0.00           H  \\nATOM   5923  HA  GLN B 378      19.132   2.693   0.406  1.00  0.00           H  \\nATOM   5924 1HB  GLN B 378      17.136   2.260   1.779  1.00  0.00           H  \\nATOM   5925 2HB  GLN B 378      16.775   3.978   1.852  1.00  0.00           H  \\nATOM   5926 1HG  GLN B 378      17.076   2.833  -0.930  1.00  0.00           H  \\nATOM   5927 2HG  GLN B 378      15.639   2.301  -0.013  1.00  0.00           H  \\nATOM   5928 1HE2 GLN B 378      15.260   5.528  -2.083  1.00  0.00           H  \\nATOM   5929 2HE2 GLN B 378      16.150   4.086  -2.442  1.00  0.00           H  \\nATOM   5930  N   LEU B 379      18.776   5.921   0.735  1.00  0.00           N  \\nATOM   5931  CA  LEU B 379      18.962   7.162  -0.011  1.00  0.00           C  \\nATOM   5932  C   LEU B 379      20.396   7.338  -0.457  1.00  0.00           C  \\nATOM   5933  O   LEU B 379      20.674   7.917  -1.523  1.00  0.00           O  \\nATOM   5934  CB  LEU B 379      18.545   8.341   0.854  1.00  0.00           C  \\nATOM   5935  CG  LEU B 379      17.096   8.478   1.279  1.00  0.00           C  \\nATOM   5936  CD1 LEU B 379      16.862   9.648   2.195  1.00  0.00           C  \\nATOM   5937  CD2 LEU B 379      16.180   8.593   0.065  1.00  0.00           C  \\nATOM   5938  H   LEU B 379      18.523   5.967   1.712  1.00  0.00           H  \\nATOM   5939  HA  LEU B 379      18.331   7.130  -0.899  1.00  0.00           H  \\nATOM   5940 1HB  LEU B 379      19.122   8.316   1.777  1.00  0.00           H  \\nATOM   5941 2HB  LEU B 379      18.786   9.263   0.325  1.00  0.00           H  \\nATOM   5942  HG  LEU B 379      16.802   7.602   1.857  1.00  0.00           H  \\nATOM   5943 1HD1 LEU B 379      15.807   9.694   2.465  1.00  0.00           H  \\nATOM   5944 2HD1 LEU B 379      17.462   9.529   3.098  1.00  0.00           H  \\nATOM   5945 3HD1 LEU B 379      17.147  10.569   1.688  1.00  0.00           H  \\nATOM   5946 1HD2 LEU B 379      15.146   8.689   0.397  1.00  0.00           H  \\nATOM   5947 2HD2 LEU B 379      16.457   9.471  -0.518  1.00  0.00           H  \\nATOM   5948 3HD2 LEU B 379      16.281   7.701  -0.553  1.00  0.00           H  \\nATOM   5949  N   ALA B 380      21.400   6.951   0.377  1.00  0.00           N  \\nATOM   5950  CA  ALA B 380      22.776   7.145   0.047  1.00  0.00           C  \\nATOM   5951  C   ALA B 380      23.235   6.127  -1.007  1.00  0.00           C  \\nATOM   5952  O   ALA B 380      24.294   6.233  -1.643  1.00  0.00           O  \\nATOM   5953  CB  ALA B 380      23.641   6.944   1.337  1.00  0.00           C  \\nATOM   5954  H   ALA B 380      21.162   6.513   1.256  1.00  0.00           H  \\nATOM   5955  HA  ALA B 380      22.896   8.165  -0.318  1.00  0.00           H  \\nATOM   5956 1HB  ALA B 380      24.694   7.090   1.096  1.00  0.00           H  \\nATOM   5957 2HB  ALA B 380      23.339   7.668   2.094  1.00  0.00           H  \\nATOM   5958 3HB  ALA B 380      23.494   5.936   1.720  1.00  0.00           H  \\nATOM   5959  N   GLY B 381      22.377   5.127  -1.178  1.00  0.00           N  \\nATOM   5960  CA  GLY B 381      22.803   3.950  -1.969  1.00  0.00           C  \\nATOM   5961  C   GLY B 381      23.960   3.094  -1.473  1.00  0.00           C  \\nATOM   5962  O   GLY B 381      24.678   2.462  -2.261  1.00  0.00           O  \\nATOM   5963  H   GLY B 381      21.446   5.146  -0.786  1.00  0.00           H  \\nATOM   5964 1HA  GLY B 381      21.964   3.263  -2.080  1.00  0.00           H  \\nATOM   5965 2HA  GLY B 381      23.091   4.269  -2.970  1.00  0.00           H  \\nATOM   5966  N   ASN B 382      24.192   3.111  -0.173  1.00  0.00           N  \\nATOM   5967  CA  ASN B 382      25.306   2.395   0.416  1.00  0.00           C  \\nATOM   5968  C   ASN B 382      24.968   2.080   1.875  1.00  0.00           C  \\nATOM   5969  O   ASN B 382      24.394   2.968   2.553  1.00  0.00           O  \\nATOM   5970  CB  ASN B 382      26.611   3.211   0.352  1.00  0.00           C  \\nATOM   5971  CG  ASN B 382      27.302   3.165  -1.031  1.00  0.00           C  \\nATOM   5972  OD1 ASN B 382      27.512   2.099  -1.541  1.00  0.00           O  \\nATOM   5973  ND2 ASN B 382      27.641   4.313  -1.621  1.00  0.00           N  \\nATOM   5974  H   ASN B 382      23.574   3.639   0.427  1.00  0.00           H  \\nATOM   5975  HA  ASN B 382      25.459   1.471  -0.144  1.00  0.00           H  \\nATOM   5976 1HB  ASN B 382      26.401   4.253   0.596  1.00  0.00           H  \\nATOM   5977 2HB  ASN B 382      27.312   2.835   1.098  1.00  0.00           H  \\nATOM   5978 1HD2 ASN B 382      28.089   4.303  -2.516  1.00  0.00           H  \\nATOM   5979 2HD2 ASN B 382      27.448   5.183  -1.169  1.00  0.00           H  \\nATOM   5980  N   LEU B 383      25.281   0.865   2.339  1.00  0.00           N  \\nATOM   5981  CA  LEU B 383      25.209   0.533   3.781  1.00  0.00           C  \\nATOM   5982  C   LEU B 383      26.385   1.169   4.581  1.00  0.00           C  \\nATOM   5983  O   LEU B 383      27.407   1.546   4.023  1.00  0.00           O  \\nATOM   5984  CB  LEU B 383      25.167  -0.999   4.080  1.00  0.00           C  \\nATOM   5985  CG  LEU B 383      24.123  -1.729   3.247  1.00  0.00           C  \\nATOM   5986  CD1 LEU B 383      24.510  -3.207   3.312  1.00  0.00           C  \\nATOM   5987  CD2 LEU B 383      22.767  -1.453   3.681  1.00  0.00           C  \\nATOM   5988  H   LEU B 383      25.575   0.154   1.685  1.00  0.00           H  \\nATOM   5989  HA  LEU B 383      24.294   0.960   4.189  1.00  0.00           H  \\nATOM   5990 1HB  LEU B 383      26.150  -1.420   3.875  1.00  0.00           H  \\nATOM   5991 2HB  LEU B 383      24.947  -1.141   5.138  1.00  0.00           H  \\nATOM   5992  HG  LEU B 383      24.209  -1.421   2.205  1.00  0.00           H  \\nATOM   5993 1HD1 LEU B 383      23.799  -3.795   2.732  1.00  0.00           H  \\nATOM   5994 2HD1 LEU B 383      25.511  -3.340   2.900  1.00  0.00           H  \\nATOM   5995 3HD1 LEU B 383      24.496  -3.541   4.349  1.00  0.00           H  \\nATOM   5996 1HD2 LEU B 383      22.065  -2.000   3.051  1.00  0.00           H  \\nATOM   5997 2HD2 LEU B 383      22.645  -1.768   4.718  1.00  0.00           H  \\nATOM   5998 3HD2 LEU B 383      22.570  -0.384   3.601  1.00  0.00           H  \\nATOM   5999  N   ARG B 384      26.115   1.380   5.889  1.00  0.00           N  \\nATOM   6000  CA  ARG B 384      27.215   1.786   6.771  1.00  0.00           C  \\nATOM   6001  C   ARG B 384      28.501   1.032   6.504  1.00  0.00           C  \\nATOM   6002  O   ARG B 384      29.675   1.598   6.511  1.00  0.00           O  \\nATOM   6003  CB  ARG B 384      26.810   1.628   8.196  1.00  0.00           C  \\nATOM   6004  CG  ARG B 384      25.916   2.787   8.679  1.00  0.00           C  \\nATOM   6005  CD  ARG B 384      25.367   2.463  10.058  1.00  0.00           C  \\nATOM   6006  NE  ARG B 384      24.286   1.455   9.910  1.00  0.00           N  \\nATOM   6007  CZ  ARG B 384      23.778   0.741  10.899  1.00  0.00           C  \\nATOM   6008  NH1 ARG B 384      24.285   0.886  12.177  1.00  0.00           N  \\nATOM   6009  NH2 ARG B 384      22.673  -0.069  10.735  1.00  0.00           N  \\nATOM   6010  H   ARG B 384      25.189   1.269   6.277  1.00  0.00           H  \\nATOM   6011  HA  ARG B 384      27.440   2.836   6.582  1.00  0.00           H  \\nATOM   6012 1HB  ARG B 384      26.272   0.689   8.319  1.00  0.00           H  \\nATOM   6013 2HB  ARG B 384      27.700   1.580   8.824  1.00  0.00           H  \\nATOM   6014 1HG  ARG B 384      26.504   3.704   8.729  1.00  0.00           H  \\nATOM   6015 2HG  ARG B 384      25.089   2.924   7.982  1.00  0.00           H  \\nATOM   6016 1HD  ARG B 384      26.165   2.063  10.682  1.00  0.00           H  \\nATOM   6017 2HD  ARG B 384      24.970   3.369  10.514  1.00  0.00           H  \\nATOM   6018  HE  ARG B 384      23.909   1.302   8.984  1.00  0.00           H  \\nATOM   6019 1HH1 ARG B 384      25.040   1.535  12.350  1.00  0.00           H  \\nATOM   6020 2HH1 ARG B 384      23.901   0.343  12.937  1.00  0.00           H  \\nATOM   6021 1HH2 ARG B 384      22.222  -0.132   9.833  1.00  0.00           H  \\nATOM   6022 2HH2 ARG B 384      22.315  -0.599  11.516  1.00  0.00           H  \\nATOM   6023  N   GLU B 385      28.346  -0.272   6.262  1.00  0.00           N  \\nATOM   6024  CA  GLU B 385      29.581  -1.035   5.954  1.00  0.00           C  \\nATOM   6025  C   GLU B 385      30.431  -0.629   4.789  1.00  0.00           C  \\nATOM   6026  O   GLU B 385      31.602  -1.075   4.638  1.00  0.00           O  \\nATOM   6027  CB  GLU B 385      29.214  -2.506   5.743  1.00  0.00           C  \\nATOM   6028  CG  GLU B 385      30.408  -3.430   5.549  1.00  0.00           C  \\nATOM   6029  CD  GLU B 385      30.017  -4.878   5.447  1.00  0.00           C  \\nATOM   6030  OE1 GLU B 385      28.847  -5.165   5.536  1.00  0.00           O  \\nATOM   6031  OE2 GLU B 385      30.888  -5.698   5.279  1.00  0.00           O  \\nATOM   6032  H   GLU B 385      27.460  -0.756   6.276  1.00  0.00           H  \\nATOM   6033  HA  GLU B 385      30.311  -0.844   6.741  1.00  0.00           H  \\nATOM   6034 1HB  GLU B 385      28.648  -2.866   6.601  1.00  0.00           H  \\nATOM   6035 2HB  GLU B 385      28.574  -2.598   4.865  1.00  0.00           H  \\nATOM   6036 1HG  GLU B 385      30.933  -3.142   4.638  1.00  0.00           H  \\nATOM   6037 2HG  GLU B 385      31.093  -3.302   6.386  1.00  0.00           H  \\nATOM   6038  N   ASN B 386      29.837   0.209   3.915  1.00  0.00           N  \\nATOM   6039  CA  ASN B 386      30.550   0.679   2.737  1.00  0.00           C  \\nATOM   6040  C   ASN B 386      30.694   2.181   2.674  1.00  0.00           C  \\nATOM   6041  O   ASN B 386      30.955   2.743   1.615  1.00  0.00           O  \\nATOM   6042  CB  ASN B 386      29.786   0.191   1.477  1.00  0.00           C  \\nATOM   6043  CG  ASN B 386      29.837  -1.343   1.305  1.00  0.00           C  \\nATOM   6044  OD1 ASN B 386      28.784  -2.001   1.173  1.00  0.00           O  \\nATOM   6045  ND2 ASN B 386      31.032  -1.917   1.424  1.00  0.00           N  \\nATOM   6046  H   ASN B 386      28.888   0.518   4.072  1.00  0.00           H  \\nATOM   6047  HA  ASN B 386      31.554   0.252   2.746  1.00  0.00           H  \\nATOM   6048 1HB  ASN B 386      28.742   0.502   1.541  1.00  0.00           H  \\nATOM   6049 2HB  ASN B 386      30.213   0.657   0.589  1.00  0.00           H  \\nATOM   6050 1HD2 ASN B 386      31.123  -2.908   1.322  1.00  0.00           H  \\nATOM   6051 2HD2 ASN B 386      31.838  -1.358   1.615  1.00  0.00           H  \\nATOM   6052  N   ILE B 387      30.588   2.871   3.824  1.00  0.00           N  \\nATOM   6053  CA  ILE B 387      30.798   4.319   3.841  1.00  0.00           C  \\nATOM   6054  C   ILE B 387      32.007   4.609   4.770  1.00  0.00           C  \\nATOM   6055  O   ILE B 387      31.970   4.336   6.014  1.00  0.00           O  \\nATOM   6056  CB  ILE B 387      29.521   5.011   4.359  1.00  0.00           C  \\nATOM   6057  CG1 ILE B 387      28.401   4.877   3.323  1.00  0.00           C  \\nATOM   6058  CG2 ILE B 387      29.727   6.531   4.680  1.00  0.00           C  \\nATOM   6059  CD1 ILE B 387      26.983   5.218   3.853  1.00  0.00           C  \\nATOM   6060  H   ILE B 387      30.363   2.396   4.687  1.00  0.00           H  \\nATOM   6061  HA  ILE B 387      30.999   4.651   2.823  1.00  0.00           H  \\nATOM   6062  HB  ILE B 387      29.190   4.524   5.276  1.00  0.00           H  \\nATOM   6063 1HG1 ILE B 387      28.607   5.533   2.478  1.00  0.00           H  \\nATOM   6064 2HG1 ILE B 387      28.376   3.855   2.944  1.00  0.00           H  \\nATOM   6065 1HG2 ILE B 387      28.790   6.957   5.039  1.00  0.00           H  \\nATOM   6066 2HG2 ILE B 387      30.493   6.640   5.447  1.00  0.00           H  \\nATOM   6067 3HG2 ILE B 387      30.041   7.054   3.777  1.00  0.00           H  \\nATOM   6068 1HD1 ILE B 387      26.255   5.095   3.051  1.00  0.00           H  \\nATOM   6069 2HD1 ILE B 387      26.732   4.549   4.677  1.00  0.00           H  \\nATOM   6070 3HD1 ILE B 387      26.964   6.249   4.204  1.00  0.00           H  \\nATOM   6071  N   GLU B 388      33.061   5.161   4.188  1.00  0.00           N  \\nATOM   6072  CA  GLU B 388      34.265   5.438   5.001  1.00  0.00           C  \\nATOM   6073  C   GLU B 388      34.023   6.557   6.031  1.00  0.00           C  \\nATOM   6074  O   GLU B 388      33.384   7.597   5.780  1.00  0.00           O  \\nATOM   6075  CB  GLU B 388      35.476   5.825   4.163  1.00  0.00           C  \\nATOM   6076  CG  GLU B 388      36.054   4.633   3.329  1.00  0.00           C  \\nATOM   6077  CD  GLU B 388      37.068   5.043   2.251  1.00  0.00           C  \\nATOM   6078  OE1 GLU B 388      37.097   6.228   1.860  1.00  0.00           O  \\nATOM   6079  OE2 GLU B 388      37.826   4.146   1.762  1.00  0.00           O  \\nATOM   6080  H   GLU B 388      33.065   5.398   3.206  1.00  0.00           H  \\nATOM   6081  HA  GLU B 388      34.526   4.534   5.552  1.00  0.00           H  \\nATOM   6082 1HB  GLU B 388      35.203   6.627   3.477  1.00  0.00           H  \\nATOM   6083 2HB  GLU B 388      36.263   6.205   4.815  1.00  0.00           H  \\nATOM   6084 1HG  GLU B 388      36.546   3.934   4.006  1.00  0.00           H  \\nATOM   6085 2HG  GLU B 388      35.231   4.109   2.845  1.00  0.00           H  \\nATOM   6086  N   LEU B 389      34.606   6.317   7.207  1.00  0.00           N  \\nATOM   6087  CA  LEU B 389      34.670   7.345   8.306  1.00  0.00           C  \\nATOM   6088  C   LEU B 389      36.095   7.774   8.560  1.00  0.00           C  \\nATOM   6089  O   LEU B 389      37.004   7.021   8.285  1.00  0.00           O  \\nATOM   6090  CB  LEU B 389      34.108   6.705   9.588  1.00  0.00           C  \\nATOM   6091  CG  LEU B 389      32.680   6.108   9.439  1.00  0.00           C  \\nATOM   6092  CD1 LEU B 389      32.160   5.449  10.798  1.00  0.00           C  \\nATOM   6093  CD2 LEU B 389      31.626   7.183   8.996  1.00  0.00           C  \\nATOM   6094  H   LEU B 389      35.019   5.408   7.359  1.00  0.00           H  \\nATOM   6095  HA  LEU B 389      34.056   8.197   8.018  1.00  0.00           H  \\nATOM   6096 1HB  LEU B 389      34.781   5.909   9.902  1.00  0.00           H  \\nATOM   6097 2HB  LEU B 389      34.083   7.462  10.372  1.00  0.00           H  \\nATOM   6098  HG  LEU B 389      32.692   5.319   8.687  1.00  0.00           H  \\nATOM   6099 1HD1 LEU B 389      31.159   5.045  10.647  1.00  0.00           H  \\nATOM   6100 2HD1 LEU B 389      32.834   4.645  11.094  1.00  0.00           H  \\nATOM   6101 3HD1 LEU B 389      32.131   6.206  11.581  1.00  0.00           H  \\nATOM   6102 1HD2 LEU B 389      30.645   6.715   8.905  1.00  0.00           H  \\nATOM   6103 2HD2 LEU B 389      31.582   7.978   9.741  1.00  0.00           H  \\nATOM   6104 3HD2 LEU B 389      31.918   7.604   8.034  1.00  0.00           H  \\nATOM   6105  N   GLY B 390      36.289   8.939   9.180  1.00  0.00           N  \\nATOM   6106  CA  GLY B 390      37.513   9.594   9.421  1.00  0.00           C  \\nATOM   6107  C   GLY B 390      37.451  11.062   9.276  1.00  0.00           C  \\nATOM   6108  O   GLY B 390      36.353  11.585   9.010  1.00  0.00           O  \\nATOM   6109  H   GLY B 390      35.431   9.367   9.497  1.00  0.00           H  \\nATOM   6110 1HA  GLY B 390      37.855   9.366  10.431  1.00  0.00           H  \\nATOM   6111 2HA  GLY B 390      38.269   9.217   8.733  1.00  0.00           H  \\nATOM   6112  N   ASN B 391      38.567  11.729   9.515  1.00  0.00           N  \\nATOM   6113  CA  ASN B 391      38.530  13.151   9.448  1.00  0.00           C  \\nATOM   6114  C   ASN B 391      38.233  13.653   8.034  1.00  0.00           C  \\nATOM   6115  O   ASN B 391      37.590  14.737   7.848  1.00  0.00           O  \\nATOM   6116  CB  ASN B 391      39.836  13.729   9.981  1.00  0.00           C  \\nATOM   6117  CG  ASN B 391      39.659  15.177  10.422  1.00  0.00           C  \\nATOM   6118  OD1 ASN B 391      38.830  15.486  11.256  1.00  0.00           O  \\nATOM   6119  ND2 ASN B 391      40.481  16.040   9.910  1.00  0.00           N  \\nATOM   6120  H   ASN B 391      39.438  11.270   9.742  1.00  0.00           H  \\nATOM   6121  HA  ASN B 391      37.706  13.506  10.069  1.00  0.00           H  \\nATOM   6122 1HB  ASN B 391      40.182  13.130  10.825  1.00  0.00           H  \\nATOM   6123 2HB  ASN B 391      40.601  13.675   9.206  1.00  0.00           H  \\nATOM   6124 1HD2 ASN B 391      40.413  17.006  10.162  1.00  0.00           H  \\nATOM   6125 2HD2 ASN B 391      41.182  15.740   9.264  1.00  0.00           H  \\nATOM   6126  N   GLY B 392      38.711  12.955   7.017  1.00  0.00           N  \\nATOM   6127  CA  GLY B 392      38.458  13.315   5.593  1.00  0.00           C  \\nATOM   6128  C   GLY B 392      36.999  13.191   5.265  1.00  0.00           C  \\nATOM   6129  O   GLY B 392      36.434  14.139   4.761  1.00  0.00           O  \\nATOM   6130  H   GLY B 392      39.272  12.142   7.227  1.00  0.00           H  \\nATOM   6131 1HA  GLY B 392      38.795  14.336   5.410  1.00  0.00           H  \\nATOM   6132 2HA  GLY B 392      39.042  12.663   4.944  1.00  0.00           H  \\nATOM   6133  N   PRO B 393      36.370  12.044   5.544  1.00  0.00           N  \\nATOM   6134  CA  PRO B 393      34.929  11.929   5.380  1.00  0.00           C  \\nATOM   6135  C   PRO B 393      34.161  13.009   6.131  1.00  0.00           C  \\nATOM   6136  O   PRO B 393      33.145  13.497   5.615  1.00  0.00           O  \\nATOM   6137  CB  PRO B 393      34.588  10.525   5.897  1.00  0.00           C  \\nATOM   6138  CG  PRO B 393      35.857   9.777   5.453  1.00  0.00           C  \\nATOM   6139  CD  PRO B 393      36.959  10.707   5.788  1.00  0.00           C  \\nATOM   6140  HA  PRO B 393      34.679  12.011   4.312  1.00  0.00           H  \\nATOM   6141 1HB  PRO B 393      34.420  10.555   6.984  1.00  0.00           H  \\nATOM   6142 2HB  PRO B 393      33.653  10.171   5.437  1.00  0.00           H  \\nATOM   6143 1HG  PRO B 393      35.935   8.815   5.981  1.00  0.00           H  \\nATOM   6144 2HG  PRO B 393      35.806   9.546   4.379  1.00  0.00           H  \\nATOM   6145 1HD  PRO B 393      37.241  10.577   6.843  1.00  0.00           H  \\nATOM   6146 2HD  PRO B 393      37.818  10.510   5.130  1.00  0.00           H  \\nATOM   6147  N   LEU B 394      34.594  13.346   7.372  1.00  0.00           N  \\nATOM   6148  CA  LEU B 394      33.833  14.333   8.103  1.00  0.00           C  \\nATOM   6149  C   LEU B 394      34.059  15.738   7.464  1.00  0.00           C  \\nATOM   6150  O   LEU B 394      33.091  16.465   7.330  1.00  0.00           O  \\nATOM   6151  CB  LEU B 394      34.235  14.253   9.591  1.00  0.00           C  \\nATOM   6152  CG  LEU B 394      33.465  15.257  10.463  1.00  0.00           C  \\nATOM   6153  CD1 LEU B 394      31.973  14.892  10.462  1.00  0.00           C  \\nATOM   6154  CD2 LEU B 394      33.961  15.117  11.924  1.00  0.00           C  \\nATOM   6155  H   LEU B 394      35.414  12.941   7.800  1.00  0.00           H  \\nATOM   6156  HA  LEU B 394      32.774  14.101   7.997  1.00  0.00           H  \\nATOM   6157 1HB  LEU B 394      34.043  13.243   9.949  1.00  0.00           H  \\nATOM   6158 2HB  LEU B 394      35.304  14.448   9.673  1.00  0.00           H  \\nATOM   6159  HG  LEU B 394      33.650  16.269  10.102  1.00  0.00           H  \\nATOM   6160 1HD1 LEU B 394      31.424  15.603  11.080  1.00  0.00           H  \\nATOM   6161 2HD1 LEU B 394      31.590  14.928   9.442  1.00  0.00           H  \\nATOM   6162 3HD1 LEU B 394      31.844  13.888  10.864  1.00  0.00           H  \\nATOM   6163 1HD2 LEU B 394      33.426  15.823  12.559  1.00  0.00           H  \\nATOM   6164 2HD2 LEU B 394      33.777  14.101  12.275  1.00  0.00           H  \\nATOM   6165 3HD2 LEU B 394      35.030  15.328  11.967  1.00  0.00           H  \\nATOM   6166  N   GLU B 395      35.282  16.053   7.006  1.00  0.00           N  \\nATOM   6167  CA  GLU B 395      35.436  17.306   6.292  1.00  0.00           C  \\nATOM   6168  C   GLU B 395      34.494  17.398   5.120  1.00  0.00           C  \\nATOM   6169  O   GLU B 395      33.870  18.443   4.902  1.00  0.00           O  \\nATOM   6170  CB  GLU B 395      36.905  17.414   5.892  1.00  0.00           C  \\nATOM   6171  CG  GLU B 395      37.199  18.381   4.834  1.00  0.00           C  \\nATOM   6172  CD  GLU B 395      37.496  19.804   5.372  1.00  0.00           C  \\nATOM   6173  OE1 GLU B 395      38.089  19.966   6.516  1.00  0.00           O  \\nATOM   6174  OE2 GLU B 395      37.217  20.771   4.538  1.00  0.00           O  \\nATOM   6175  H   GLU B 395      36.088  15.459   7.137  1.00  0.00           H  \\nATOM   6176  HA  GLU B 395      35.166  18.122   6.963  1.00  0.00           H  \\nATOM   6177 1HB  GLU B 395      37.499  17.694   6.762  1.00  0.00           H  \\nATOM   6178 2HB  GLU B 395      37.262  16.442   5.551  1.00  0.00           H  \\nATOM   6179 1HG  GLU B 395      38.064  18.034   4.269  1.00  0.00           H  \\nATOM   6180 2HG  GLU B 395      36.350  18.428   4.153  1.00  0.00           H  \\nATOM   6181  N   GLU B 396      34.401  16.307   4.368  1.00  0.00           N  \\nATOM   6182  CA  GLU B 396      33.580  16.211   3.131  1.00  0.00           C  \\nATOM   6183  C   GLU B 396      32.127  16.363   3.492  1.00  0.00           C  \\nATOM   6184  O   GLU B 396      31.369  17.020   2.793  1.00  0.00           O  \\nATOM   6185  CB  GLU B 396      33.856  14.909   2.328  1.00  0.00           C  \\nATOM   6186  CG  GLU B 396      35.330  14.907   1.780  1.00  0.00           C  \\nATOM   6187  CD  GLU B 396      35.800  13.496   1.428  1.00  0.00           C  \\nATOM   6188  OE1 GLU B 396      35.028  12.492   1.601  1.00  0.00           O  \\nATOM   6189  OE2 GLU B 396      36.994  13.412   1.034  1.00  0.00           O  \\nATOM   6190  H   GLU B 396      34.929  15.501   4.672  1.00  0.00           H  \\nATOM   6191  HA  GLU B 396      33.827  17.054   2.485  1.00  0.00           H  \\nATOM   6192 1HB  GLU B 396      33.697  14.044   2.972  1.00  0.00           H  \\nATOM   6193 2HB  GLU B 396      33.151  14.836   1.500  1.00  0.00           H  \\nATOM   6194 1HG  GLU B 396      35.377  15.537   0.892  1.00  0.00           H  \\nATOM   6195 2HG  GLU B 396      35.987  15.338   2.534  1.00  0.00           H  \\nATOM   6196  N   ALA B 397      31.731  15.765   4.628  1.00  0.00           N  \\nATOM   6197  CA  ALA B 397      30.334  15.719   5.030  1.00  0.00           C  \\nATOM   6198  C   ALA B 397      29.856  17.111   5.413  1.00  0.00           C  \\nATOM   6199  O   ALA B 397      28.723  17.484   5.149  1.00  0.00           O  \\nATOM   6200  CB  ALA B 397      30.112  14.712   6.236  1.00  0.00           C  \\nATOM   6201  H   ALA B 397      32.428  15.335   5.219  1.00  0.00           H  \\nATOM   6202  HA  ALA B 397      29.748  15.372   4.179  1.00  0.00           H  \\nATOM   6203 1HB  ALA B 397      29.057  14.701   6.512  1.00  0.00           H  \\nATOM   6204 2HB  ALA B 397      30.418  13.710   5.935  1.00  0.00           H  \\nATOM   6205 3HB  ALA B 397      30.707  15.031   7.090  1.00  0.00           H  \\nATOM   6206  N   ILE B 398      30.737  17.849   6.088  1.00  0.00           N  \\nATOM   6207  CA  ILE B 398      30.413  19.257   6.509  1.00  0.00           C  \\nATOM   6208  C   ILE B 398      30.141  20.106   5.258  1.00  0.00           C  \\nATOM   6209  O   ILE B 398      29.166  20.856   5.249  1.00  0.00           O  \\nATOM   6210  CB  ILE B 398      31.514  19.829   7.462  1.00  0.00           C  \\nATOM   6211  CG1 ILE B 398      31.519  19.002   8.761  1.00  0.00           C  \\nATOM   6212  CG2 ILE B 398      31.394  21.381   7.692  1.00  0.00           C  \\nATOM   6213  CD1 ILE B 398      32.898  19.266   9.444  1.00  0.00           C  \\nATOM   6214  H   ILE B 398      31.639  17.460   6.323  1.00  0.00           H  \\nATOM   6215  HA  ILE B 398      29.466  19.248   7.047  1.00  0.00           H  \\nATOM   6216  HB  ILE B 398      32.499  19.636   7.037  1.00  0.00           H  \\nATOM   6217 1HG1 ILE B 398      30.684  19.309   9.390  1.00  0.00           H  \\nATOM   6218 2HG1 ILE B 398      31.377  17.948   8.522  1.00  0.00           H  \\nATOM   6219 1HG2 ILE B 398      32.187  21.712   8.362  1.00  0.00           H  \\nATOM   6220 2HG2 ILE B 398      31.487  21.898   6.738  1.00  0.00           H  \\nATOM   6221 3HG2 ILE B 398      30.425  21.609   8.136  1.00  0.00           H  \\nATOM   6222 1HD1 ILE B 398      32.957  18.701  10.375  1.00  0.00           H  \\nATOM   6223 2HD1 ILE B 398      33.701  18.951   8.777  1.00  0.00           H  \\nATOM   6224 3HD1 ILE B 398      33.000  20.329   9.658  1.00  0.00           H  \\nATOM   6225  N   SER B 399      31.039  20.011   4.287  1.00  0.00           N  \\nATOM   6226  CA  SER B 399      30.800  20.733   3.032  1.00  0.00           C  \\nATOM   6227  C   SER B 399      29.505  20.276   2.357  1.00  0.00           C  \\nATOM   6228  O   SER B 399      28.708  21.164   1.913  1.00  0.00           O  \\nATOM   6229  CB  SER B 399      31.956  20.546   2.044  1.00  0.00           C  \\nATOM   6230  OG  SER B 399      33.091  21.217   2.538  1.00  0.00           O  \\nATOM   6231  H   SER B 399      31.879  19.459   4.385  1.00  0.00           H  \\nATOM   6232  HA  SER B 399      30.715  21.797   3.256  1.00  0.00           H  \\nATOM   6233 1HB  SER B 399      32.160  19.483   1.920  1.00  0.00           H  \\nATOM   6234 2HB  SER B 399      31.669  20.940   1.070  1.00  0.00           H  \\nATOM   6235  HG  SER B 399      32.823  21.613   3.371  1.00  0.00           H  \\nATOM   6236  N   ALA B 400      29.253  18.970   2.323  1.00  0.00           N  \\nATOM   6237  CA  ALA B 400      28.021  18.461   1.673  1.00  0.00           C  \\nATOM   6238  C   ALA B 400      26.799  19.012   2.342  1.00  0.00           C  \\nATOM   6239  O   ALA B 400      25.776  19.482   1.693  1.00  0.00           O  \\nATOM   6240  CB  ALA B 400      28.025  16.975   1.668  1.00  0.00           C  \\nATOM   6241  H   ALA B 400      29.897  18.313   2.739  1.00  0.00           H  \\nATOM   6242  HA  ALA B 400      28.008  18.820   0.644  1.00  0.00           H  \\nATOM   6243 1HB  ALA B 400      27.116  16.610   1.189  1.00  0.00           H  \\nATOM   6244 2HB  ALA B 400      28.895  16.615   1.118  1.00  0.00           H  \\nATOM   6245 3HB  ALA B 400      28.067  16.608   2.692  1.00  0.00           H  \\nATOM   6246  N   LEU B 401      26.733  18.978   3.693  1.00  0.00           N  \\nATOM   6247  CA  LEU B 401      25.563  19.489   4.395  1.00  0.00           C  \\nATOM   6248  C   LEU B 401      25.458  20.981   4.096  1.00  0.00           C  \\nATOM   6249  O   LEU B 401      24.330  21.453   3.889  1.00  0.00           O  \\nATOM   6250  CB  LEU B 401      25.698  19.226   5.905  1.00  0.00           C  \\nATOM   6251  CG  LEU B 401      24.961  17.966   6.370  1.00  0.00           C  \\nATOM   6252  CD1 LEU B 401      25.508  16.774   5.829  1.00  0.00           C  \\nATOM   6253  CD2 LEU B 401      25.177  18.000   7.935  1.00  0.00           C  \\nATOM   6254  H   LEU B 401      27.500  18.596   4.228  1.00  0.00           H  \\nATOM   6255  HA  LEU B 401      24.682  18.967   4.024  1.00  0.00           H  \\nATOM   6256 1HB  LEU B 401      26.755  19.127   6.147  1.00  0.00           H  \\nATOM   6257 2HB  LEU B 401      25.302  20.086   6.444  1.00  0.00           H  \\nATOM   6258  HG  LEU B 401      23.910  18.034   6.087  1.00  0.00           H  \\nATOM   6259 1HD1 LEU B 401      24.944  15.916   6.195  1.00  0.00           H  \\nATOM   6260 2HD1 LEU B 401      25.450  16.808   4.741  1.00  0.00           H  \\nATOM   6261 3HD1 LEU B 401      26.549  16.680   6.135  1.00  0.00           H  \\nATOM   6262 1HD2 LEU B 401      24.688  17.137   8.389  1.00  0.00           H  \\nATOM   6263 2HD2 LEU B 401      26.244  17.970   8.156  1.00  0.00           H  \\nATOM   6264 3HD2 LEU B 401      24.748  18.916   8.342  1.00  0.00           H  \\nATOM   6265  N   TYR B 402      26.571  21.713   4.103  1.00  0.00           N  \\nATOM   6266  CA  TYR B 402      26.458  23.170   3.827  1.00  0.00           C  \\nATOM   6267  C   TYR B 402      25.870  23.459   2.438  1.00  0.00           C  \\nATOM   6268  O   TYR B 402      24.968  24.326   2.365  1.00  0.00           O  \\nATOM   6269  CB  TYR B 402      27.807  23.829   3.892  1.00  0.00           C  \\nATOM   6270  CG  TYR B 402      27.675  25.368   3.726  1.00  0.00           C  \\nATOM   6271  CD1 TYR B 402      27.323  26.170   4.798  1.00  0.00           C  \\nATOM   6272  CD2 TYR B 402      27.933  25.961   2.493  1.00  0.00           C  \\nATOM   6273  CE1 TYR B 402      27.163  27.538   4.654  1.00  0.00           C  \\nATOM   6274  CE2 TYR B 402      27.794  27.317   2.329  1.00  0.00           C  \\nATOM   6275  CZ  TYR B 402      27.402  28.093   3.403  1.00  0.00           C  \\nATOM   6276  OH  TYR B 402      27.286  29.470   3.183  1.00  0.00           O  \\nATOM   6277  H   TYR B 402      27.481  21.316   4.288  1.00  0.00           H  \\nATOM   6278  HA  TYR B 402      25.817  23.617   4.587  1.00  0.00           H  \\nATOM   6279 1HB  TYR B 402      28.279  23.600   4.849  1.00  0.00           H  \\nATOM   6280 2HB  TYR B 402      28.447  23.427   3.107  1.00  0.00           H  \\nATOM   6281  HD1 TYR B 402      27.166  25.727   5.782  1.00  0.00           H  \\nATOM   6282  HD2 TYR B 402      28.247  25.348   1.648  1.00  0.00           H  \\nATOM   6283  HE1 TYR B 402      26.885  28.148   5.514  1.00  0.00           H  \\nATOM   6284  HE2 TYR B 402      27.992  27.772   1.358  1.00  0.00           H  \\nATOM   6285  HH  TYR B 402      27.510  29.667   2.270  1.00  0.00           H  \\nATOM   6286  N   TYR B 403      26.268  22.708   1.464  1.00  0.00           N  \\nATOM   6287  CA  TYR B 403      25.864  23.121   0.073  1.00  0.00           C  \\nATOM   6288  C   TYR B 403      24.517  22.521  -0.297  1.00  0.00           C  \\nATOM   6289  O   TYR B 403      24.041  22.742  -1.396  1.00  0.00           O  \\nATOM   6290  CB  TYR B 403      26.912  22.681  -0.969  1.00  0.00           C  \\nATOM   6291  CG  TYR B 403      28.151  23.565  -0.997  1.00  0.00           C  \\nATOM   6292  CD1 TYR B 403      28.032  24.952  -1.113  1.00  0.00           C  \\nATOM   6293  CD2 TYR B 403      29.416  23.035  -0.711  1.00  0.00           C  \\nATOM   6294  CE1 TYR B 403      29.183  25.791  -1.105  1.00  0.00           C  \\nATOM   6295  CE2 TYR B 403      30.564  23.863  -0.695  1.00  0.00           C  \\nATOM   6296  CZ  TYR B 403      30.430  25.229  -0.883  1.00  0.00           C  \\nATOM   6297  OH  TYR B 403      31.520  26.088  -0.829  1.00  0.00           O  \\nATOM   6298  H   TYR B 403      26.825  21.875   1.587  1.00  0.00           H  \\nATOM   6299  HA  TYR B 403      25.789  24.208   0.043  1.00  0.00           H  \\nATOM   6300 1HB  TYR B 403      27.226  21.657  -0.760  1.00  0.00           H  \\nATOM   6301 2HB  TYR B 403      26.463  22.689  -1.962  1.00  0.00           H  \\nATOM   6302  HD1 TYR B 403      27.047  25.408  -1.212  1.00  0.00           H  \\nATOM   6303  HD2 TYR B 403      29.526  21.972  -0.498  1.00  0.00           H  \\nATOM   6304  HE1 TYR B 403      29.071  26.871  -1.197  1.00  0.00           H  \\nATOM   6305  HE2 TYR B 403      31.550  23.425  -0.536  1.00  0.00           H  \\nATOM   6306  HH  TYR B 403      32.316  25.582  -0.650  1.00  0.00           H  \\nATOM   6307  N   TYR B 404      23.871  21.746   0.563  1.00  0.00           N  \\nATOM   6308  CA  TYR B 404      22.461  21.426   0.309  1.00  0.00           C  \\nATOM   6309  C   TYR B 404      21.643  22.694   0.223  1.00  0.00           C  \\nATOM   6310  O   TYR B 404      20.713  22.777  -0.595  1.00  0.00           O  \\nATOM   6311  CB  TYR B 404      21.890  20.450   1.352  1.00  0.00           C  \\nATOM   6312  CG  TYR B 404      20.541  19.892   0.875  1.00  0.00           C  \\nATOM   6313  CD1 TYR B 404      20.513  18.913  -0.076  1.00  0.00           C  \\nATOM   6314  CD2 TYR B 404      19.348  20.383   1.390  1.00  0.00           C  \\nATOM   6315  CE1 TYR B 404      19.282  18.395  -0.565  1.00  0.00           C  \\nATOM   6316  CE2 TYR B 404      18.080  19.909   0.905  1.00  0.00           C  \\nATOM   6317  CZ  TYR B 404      18.075  18.888  -0.036  1.00  0.00           C  \\nATOM   6318  OH  TYR B 404      16.867  18.391  -0.523  1.00  0.00           O  \\nATOM   6319  H   TYR B 404      24.320  21.370   1.386  1.00  0.00           H  \\nATOM   6320  HA  TYR B 404      22.385  20.951  -0.669  1.00  0.00           H  \\nATOM   6321 1HB  TYR B 404      22.596  19.634   1.512  1.00  0.00           H  \\nATOM   6322 2HB  TYR B 404      21.765  20.966   2.303  1.00  0.00           H  \\nATOM   6323  HD1 TYR B 404      21.446  18.514  -0.475  1.00  0.00           H  \\nATOM   6324  HD2 TYR B 404      19.363  21.140   2.174  1.00  0.00           H  \\nATOM   6325  HE1 TYR B 404      19.283  17.614  -1.324  1.00  0.00           H  \\nATOM   6326  HE2 TYR B 404      17.155  20.351   1.276  1.00  0.00           H  \\nATOM   6327  HH  TYR B 404      16.139  18.860  -0.108  1.00  0.00           H  \\nATOM   6328  N   SER B 405      21.956  23.691   1.024  1.00  0.00           N  \\nATOM   6329  CA  SER B 405      21.147  24.842   1.055  1.00  0.00           C  \\nATOM   6330  C   SER B 405      21.233  25.644  -0.251  1.00  0.00           C  \\nATOM   6331  O   SER B 405      20.316  26.371  -0.542  1.00  0.00           O  \\nATOM   6332  CB  SER B 405      21.645  25.701   2.266  1.00  0.00           C  \\nATOM   6333  OG  SER B 405      23.006  26.136   2.058  1.00  0.00           O  \\nATOM   6334  H   SER B 405      22.770  23.646   1.620  1.00  0.00           H  \\nATOM   6335  HA  SER B 405      20.111  24.533   1.204  1.00  0.00           H  \\nATOM   6336 1HB  SER B 405      20.995  26.567   2.389  1.00  0.00           H  \\nATOM   6337 2HB  SER B 405      21.582  25.112   3.180  1.00  0.00           H  \\nATOM   6338  HG  SER B 405      23.271  25.780   1.207  1.00  0.00           H  \\nATOM   6339  N   THR B 406      22.304  25.473  -1.035  1.00  0.00           N  \\nATOM   6340  CA  THR B 406      22.506  26.254  -2.256  1.00  0.00           C  \\nATOM   6341  C   THR B 406      22.076  25.484  -3.462  1.00  0.00           C  \\nATOM   6342  O   THR B 406      21.980  26.049  -4.576  1.00  0.00           O  \\nATOM   6343  CB  THR B 406      24.022  26.569  -2.449  1.00  0.00           C  \\nATOM   6344  OG1 THR B 406      24.750  25.361  -2.758  1.00  0.00           O  \\nATOM   6345  CG2 THR B 406      24.647  27.255  -1.225  1.00  0.00           C  \\nATOM   6346  H   THR B 406      22.993  24.782  -0.773  1.00  0.00           H  \\nATOM   6347  HA  THR B 406      21.962  27.194  -2.160  1.00  0.00           H  \\nATOM   6348  HB  THR B 406      24.150  27.229  -3.307  1.00  0.00           H  \\nATOM   6349  HG1 THR B 406      24.142  24.618  -2.779  1.00  0.00           H  \\nATOM   6350 1HG2 THR B 406      25.702  27.450  -1.417  1.00  0.00           H  \\nATOM   6351 2HG2 THR B 406      24.132  28.196  -1.033  1.00  0.00           H  \\nATOM   6352 3HG2 THR B 406      24.551  26.606  -0.356  1.00  0.00           H  \\nATOM   6353  N   GLY B 407      21.726  24.229  -3.257  1.00  0.00           N  \\nATOM   6354  CA  GLY B 407      21.494  23.298  -4.416  1.00  0.00           C  \\nATOM   6355  C   GLY B 407      22.698  22.508  -4.909  1.00  0.00           C  \\nATOM   6356  O   GLY B 407      22.560  21.704  -5.843  1.00  0.00           O  \\nATOM   6357  H   GLY B 407      21.610  23.881  -2.316  1.00  0.00           H  \\nATOM   6358 1HA  GLY B 407      20.727  22.570  -4.150  1.00  0.00           H  \\nATOM   6359 2HA  GLY B 407      21.119  23.864  -5.268  1.00  0.00           H  \\nATOM   6360  N   GLY B 408      23.887  22.719  -4.330  1.00  0.00           N  \\nATOM   6361  CA  GLY B 408      25.105  22.054  -4.747  1.00  0.00           C  \\nATOM   6362  C   GLY B 408      25.270  20.608  -4.352  1.00  0.00           C  \\nATOM   6363  O   GLY B 408      26.119  19.885  -4.949  1.00  0.00           O  \\nATOM   6364  H   GLY B 408      23.923  23.377  -3.565  1.00  0.00           H  \\nATOM   6365 1HA  GLY B 408      25.185  22.090  -5.834  1.00  0.00           H  \\nATOM   6366 2HA  GLY B 408      25.966  22.586  -4.344  1.00  0.00           H  \\nATOM   6367  N   THR B 409      24.491  20.162  -3.378  1.00  0.00           N  \\nATOM   6368  CA  THR B 409      24.596  18.776  -2.874  1.00  0.00           C  \\nATOM   6369  C   THR B 409      23.318  18.034  -3.213  1.00  0.00           C  \\nATOM   6370  O   THR B 409      22.180  18.537  -2.973  1.00  0.00           O  \\nATOM   6371  CB  THR B 409      24.706  18.786  -1.338  1.00  0.00           C  \\nATOM   6372  OG1 THR B 409      25.879  19.494  -0.961  1.00  0.00           O  \\nATOM   6373  CG2 THR B 409      24.730  17.275  -0.742  1.00  0.00           C  \\nATOM   6374  H   THR B 409      23.808  20.784  -2.969  1.00  0.00           H  \\nATOM   6375  HA  THR B 409      25.496  18.324  -3.291  1.00  0.00           H  \\nATOM   6376  HB  THR B 409      23.853  19.317  -0.916  1.00  0.00           H  \\nATOM   6377  HG1 THR B 409      26.333  19.803  -1.749  1.00  0.00           H  \\nATOM   6378 1HG2 THR B 409      24.808  17.314   0.344  1.00  0.00           H  \\nATOM   6379 2HG2 THR B 409      23.812  16.759  -1.022  1.00  0.00           H  \\nATOM   6380 3HG2 THR B 409      25.586  16.738  -1.148  1.00  0.00           H  \\nATOM   6381  N   GLN B 410      23.519  16.884  -3.852  1.00  0.00           N  \\nATOM   6382  CA  GLN B 410      22.422  15.985  -4.259  1.00  0.00           C  \\nATOM   6383  C   GLN B 410      22.009  15.091  -3.127  1.00  0.00           C  \\nATOM   6384  O   GLN B 410      22.854  14.846  -2.253  1.00  0.00           O  \\nATOM   6385  CB  GLN B 410      22.850  15.135  -5.511  1.00  0.00           C  \\nATOM   6386  CG  GLN B 410      23.191  15.995  -6.825  1.00  0.00           C  \\nATOM   6387  CD  GLN B 410      22.069  16.924  -7.197  1.00  0.00           C  \\nATOM   6388  OE1 GLN B 410      21.011  16.486  -7.705  1.00  0.00           O  \\nATOM   6389  NE2 GLN B 410      22.273  18.229  -6.943  1.00  0.00           N  \\nATOM   6390  H   GLN B 410      24.472  16.624  -4.063  1.00  0.00           H  \\nATOM   6391  HA  GLN B 410      21.558  16.593  -4.527  1.00  0.00           H  \\nATOM   6392 1HB  GLN B 410      23.732  14.544  -5.265  1.00  0.00           H  \\nATOM   6393 2HB  GLN B 410      22.051  14.440  -5.769  1.00  0.00           H  \\nATOM   6394 1HG  GLN B 410      24.084  16.590  -6.635  1.00  0.00           H  \\nATOM   6395 2HG  GLN B 410      23.366  15.313  -7.657  1.00  0.00           H  \\nATOM   6396 1HE2 GLN B 410      21.565  18.900  -7.167  1.00  0.00           H  \\nATOM   6397 2HE2 GLN B 410      23.133  18.530  -6.530  1.00  0.00           H  \\nATOM   6398  N   LEU B 411      20.778  14.622  -3.117  1.00  0.00           N  \\nATOM   6399  CA  LEU B 411      20.330  13.774  -1.970  1.00  0.00           C  \\nATOM   6400  C   LEU B 411      21.218  12.563  -1.547  1.00  0.00           C  \\nATOM   6401  O   LEU B 411      21.456  12.364  -0.339  1.00  0.00           O  \\nATOM   6402  CB  LEU B 411      18.913  13.318  -2.240  1.00  0.00           C  \\nATOM   6403  CG  LEU B 411      18.271  12.591  -1.067  1.00  0.00           C  \\nATOM   6404  CD1 LEU B 411      18.192  13.440   0.179  1.00  0.00           C  \\nATOM   6405  CD2 LEU B 411      16.833  12.182  -1.528  1.00  0.00           C  \\nATOM   6406  H   LEU B 411      20.129  14.821  -3.865  1.00  0.00           H  \\nATOM   6407  HA  LEU B 411      20.354  14.378  -1.063  1.00  0.00           H  \\nATOM   6408 1HB  LEU B 411      18.308  14.190  -2.485  1.00  0.00           H  \\nATOM   6409 2HB  LEU B 411      18.920  12.653  -3.103  1.00  0.00           H  \\nATOM   6410  HG  LEU B 411      18.864  11.711  -0.816  1.00  0.00           H  \\nATOM   6411 1HD1 LEU B 411      17.725  12.867   0.981  1.00  0.00           H  \\nATOM   6412 2HD1 LEU B 411      19.197  13.735   0.483  1.00  0.00           H  \\nATOM   6413 3HD1 LEU B 411      17.597  14.330  -0.024  1.00  0.00           H  \\nATOM   6414 1HD2 LEU B 411      16.328  11.654  -0.719  1.00  0.00           H  \\nATOM   6415 2HD2 LEU B 411      16.266  13.076  -1.787  1.00  0.00           H  \\nATOM   6416 3HD2 LEU B 411      16.902  11.530  -2.399  1.00  0.00           H  \\nATOM   6417  N   PRO B 412      21.710  11.724  -2.458  1.00  0.00           N  \\nATOM   6418  CA  PRO B 412      22.556  10.622  -1.994  1.00  0.00           C  \\nATOM   6419  C   PRO B 412      23.779  11.066  -1.239  1.00  0.00           C  \\nATOM   6420  O   PRO B 412      24.062  10.398  -0.187  1.00  0.00           O  \\nATOM   6421  CB  PRO B 412      22.956   9.874  -3.245  1.00  0.00           C  \\nATOM   6422  CG  PRO B 412      21.733  10.241  -4.176  1.00  0.00           C  \\nATOM   6423  CD  PRO B 412      21.635  11.725  -3.957  1.00  0.00           C  \\nATOM   6424  HA  PRO B 412      21.967   9.966  -1.336  1.00  0.00           H  \\nATOM   6425 1HB  PRO B 412      23.934  10.231  -3.600  1.00  0.00           H  \\nATOM   6426 2HB  PRO B 412      23.066   8.802  -3.023  1.00  0.00           H  \\nATOM   6427 1HG  PRO B 412      21.947   9.952  -5.216  1.00  0.00           H  \\nATOM   6428 2HG  PRO B 412      20.839   9.680  -3.867  1.00  0.00           H  \\nATOM   6429 1HD  PRO B 412      22.484  12.225  -4.445  1.00  0.00           H  \\nATOM   6430 2HD  PRO B 412      20.682  12.094  -4.363  1.00  0.00           H  \\nATOM   6431  N   THR B 413      24.426  12.169  -1.646  1.00  0.00           N  \\nATOM   6432  CA  THR B 413      25.555  12.655  -0.902  1.00  0.00           C  \\nATOM   6433  C   THR B 413      25.121  13.237   0.429  1.00  0.00           C  \\nATOM   6434  O   THR B 413      25.867  13.087   1.416  1.00  0.00           O  \\nATOM   6435  CB  THR B 413      26.289  13.655  -1.784  1.00  0.00           C  \\nATOM   6436  OG1 THR B 413      26.812  12.925  -2.901  1.00  0.00           O  \\nATOM   6437  CG2 THR B 413      27.358  14.245  -1.075  1.00  0.00           C  \\nATOM   6438  H   THR B 413      24.130  12.667  -2.473  1.00  0.00           H  \\nATOM   6439  HA  THR B 413      26.206  11.812  -0.671  1.00  0.00           H  \\nATOM   6440  HB  THR B 413      25.597  14.434  -2.104  1.00  0.00           H  \\nATOM   6441  HG1 THR B 413      26.573  11.999  -2.817  1.00  0.00           H  \\nATOM   6442 1HG2 THR B 413      27.872  14.958  -1.719  1.00  0.00           H  \\nATOM   6443 2HG2 THR B 413      26.971  14.762  -0.197  1.00  0.00           H  \\nATOM   6444 3HG2 THR B 413      28.057  13.471  -0.760  1.00  0.00           H  \\nATOM   6445  N   LEU B 414      23.966  13.900   0.500  1.00  0.00           N  \\nATOM   6446  CA  LEU B 414      23.454  14.436   1.743  1.00  0.00           C  \\nATOM   6447  C   LEU B 414      23.217  13.286   2.734  1.00  0.00           C  \\nATOM   6448  O   LEU B 414      23.687  13.335   3.854  1.00  0.00           O  \\nATOM   6449  CB  LEU B 414      22.134  15.161   1.531  1.00  0.00           C  \\nATOM   6450  CG  LEU B 414      21.555  15.732   2.815  1.00  0.00           C  \\nATOM   6451  CD1 LEU B 414      22.453  16.848   3.258  1.00  0.00           C  \\nATOM   6452  CD2 LEU B 414      20.093  16.162   2.578  1.00  0.00           C  \\nATOM   6453  H   LEU B 414      23.433  14.030  -0.348  1.00  0.00           H  \\nATOM   6454  HA  LEU B 414      24.176  15.151   2.135  1.00  0.00           H  \\nATOM   6455 1HB  LEU B 414      22.293  15.971   0.821  1.00  0.00           H  \\nATOM   6456 2HB  LEU B 414      21.419  14.462   1.098  1.00  0.00           H  \\nATOM   6457  HG  LEU B 414      21.587  14.973   3.597  1.00  0.00           H  \\nATOM   6458 1HD1 LEU B 414      22.066  17.282   4.180  1.00  0.00           H  \\nATOM   6459 2HD1 LEU B 414      23.456  16.460   3.434  1.00  0.00           H  \\nATOM   6460 3HD1 LEU B 414      22.490  17.614   2.484  1.00  0.00           H  \\nATOM   6461 1HD2 LEU B 414      19.680  16.571   3.501  1.00  0.00           H  \\nATOM   6462 2HD2 LEU B 414      20.059  16.922   1.797  1.00  0.00           H  \\nATOM   6463 3HD2 LEU B 414      19.505  15.298   2.268  1.00  0.00           H  \\nATOM   6464  N   ALA B 415      22.595  12.214   2.250  1.00  0.00           N  \\nATOM   6465  CA  ALA B 415      22.281  11.075   3.126  1.00  0.00           C  \\nATOM   6466  C   ALA B 415      23.541  10.390   3.618  1.00  0.00           C  \\nATOM   6467  O   ALA B 415      23.654  10.062   4.826  1.00  0.00           O  \\nATOM   6468  CB  ALA B 415      21.281  10.096   2.433  1.00  0.00           C  \\nATOM   6469  H   ALA B 415      22.332  12.172   1.276  1.00  0.00           H  \\nATOM   6470  HA  ALA B 415      21.816  11.463   4.032  1.00  0.00           H  \\nATOM   6471 1HB  ALA B 415      21.065   9.263   3.102  1.00  0.00           H  \\nATOM   6472 2HB  ALA B 415      20.356  10.624   2.201  1.00  0.00           H  \\nATOM   6473 3HB  ALA B 415      21.722   9.717   1.512  1.00  0.00           H  \\nATOM   6474  N   ARG B 416      24.537  10.237   2.757  1.00  0.00           N  \\nATOM   6475  CA  ARG B 416      25.826   9.660   3.131  1.00  0.00           C  \\nATOM   6476  C   ARG B 416      26.465  10.555   4.222  1.00  0.00           C  \\nATOM   6477  O   ARG B 416      27.087  10.046   5.181  1.00  0.00           O  \\nATOM   6478  CB  ARG B 416      26.741   9.633   1.918  1.00  0.00           C  \\nATOM   6479  CG  ARG B 416      28.099   8.981   2.158  1.00  0.00           C  \\nATOM   6480  CD  ARG B 416      29.131   9.288   0.948  1.00  0.00           C  \\nATOM   6481  NE  ARG B 416      30.297   8.440   1.098  1.00  0.00           N  \\nATOM   6482  CZ  ARG B 416      30.392   7.208   0.600  1.00  0.00           C  \\nATOM   6483  NH1 ARG B 416      29.367   6.683  -0.069  1.00  0.00           N  \\nATOM   6484  NH2 ARG B 416      31.495   6.488   0.784  1.00  0.00           N  \\nATOM   6485  H   ARG B 416      24.390  10.535   1.803  1.00  0.00           H  \\nATOM   6486  HA  ARG B 416      25.661   8.639   3.476  1.00  0.00           H  \\nATOM   6487 1HB  ARG B 416      26.254   9.094   1.107  1.00  0.00           H  \\nATOM   6488 2HB  ARG B 416      26.919  10.652   1.574  1.00  0.00           H  \\nATOM   6489 1HG  ARG B 416      28.529   9.366   3.083  1.00  0.00           H  \\nATOM   6490 2HG  ARG B 416      27.975   7.900   2.238  1.00  0.00           H  \\nATOM   6491 1HD  ARG B 416      28.646   9.081  -0.005  1.00  0.00           H  \\nATOM   6492 2HD  ARG B 416      29.431  10.335   0.982  1.00  0.00           H  \\nATOM   6493  HE  ARG B 416      31.085   8.810   1.613  1.00  0.00           H  \\nATOM   6494 1HH1 ARG B 416      28.520   7.219  -0.198  1.00  0.00           H  \\nATOM   6495 2HH1 ARG B 416      29.436   5.750  -0.448  1.00  0.00           H  \\nATOM   6496 1HH2 ARG B 416      32.270   6.875   1.305  1.00  0.00           H  \\nATOM   6497 2HH2 ARG B 416      31.558   5.556   0.403  1.00  0.00           H  \\nATOM   6498  N   SER B 417      26.304  11.862   4.084  1.00  0.00           N  \\nATOM   6499  CA  SER B 417      26.929  12.851   4.948  1.00  0.00           C  \\nATOM   6500  C   SER B 417      26.325  12.788   6.300  1.00  0.00           C  \\nATOM   6501  O   SER B 417      27.019  12.903   7.317  1.00  0.00           O  \\nATOM   6502  CB  SER B 417      26.959  14.250   4.321  1.00  0.00           C  \\nATOM   6503  OG  SER B 417      27.738  14.172   3.135  1.00  0.00           O  \\nATOM   6504  H   SER B 417      25.709  12.174   3.330  1.00  0.00           H  \\nATOM   6505  HA  SER B 417      27.962  12.549   5.129  1.00  0.00           H  \\nATOM   6506 1HB  SER B 417      25.941  14.575   4.107  1.00  0.00           H  \\nATOM   6507 2HB  SER B 417      27.386  14.957   5.031  1.00  0.00           H  \\nATOM   6508  HG  SER B 417      28.024  13.258   3.066  1.00  0.00           H  \\nATOM   6509  N   PHE B 418      25.021  12.587   6.365  1.00  0.00           N  \\nATOM   6510  CA  PHE B 418      24.331  12.407   7.653  1.00  0.00           C  \\nATOM   6511  C   PHE B 418      24.824  11.152   8.300  1.00  0.00           C  \\nATOM   6512  O   PHE B 418      25.036  11.119   9.543  1.00  0.00           O  \\nATOM   6513  CB  PHE B 418      22.830  12.255   7.475  1.00  0.00           C  \\nATOM   6514  CG  PHE B 418      22.125  13.548   7.167  1.00  0.00           C  \\nATOM   6515  CD1 PHE B 418      22.413  14.711   7.874  1.00  0.00           C  \\nATOM   6516  CD2 PHE B 418      21.143  13.580   6.203  1.00  0.00           C  \\nATOM   6517  CE1 PHE B 418      21.651  15.902   7.592  1.00  0.00           C  \\nATOM   6518  CE2 PHE B 418      20.411  14.752   5.952  1.00  0.00           C  \\nATOM   6519  CZ  PHE B 418      20.693  15.882   6.636  1.00  0.00           C  \\nATOM   6520  H   PHE B 418      24.484  12.556   5.510  1.00  0.00           H  \\nATOM   6521  HA  PHE B 418      24.509  13.291   8.268  1.00  0.00           H  \\nATOM   6522 1HB  PHE B 418      22.629  11.555   6.665  1.00  0.00           H  \\nATOM   6523 2HB  PHE B 418      22.397  11.838   8.383  1.00  0.00           H  \\nATOM   6524  HD1 PHE B 418      23.206  14.697   8.622  1.00  0.00           H  \\nATOM   6525  HD2 PHE B 418      20.916  12.667   5.652  1.00  0.00           H  \\nATOM   6526  HE1 PHE B 418      21.847  16.818   8.149  1.00  0.00           H  \\nATOM   6527  HE2 PHE B 418      19.616  14.751   5.207  1.00  0.00           H  \\nATOM   6528  HZ  PHE B 418      20.132  16.787   6.406  1.00  0.00           H  \\nATOM   6529  N   ILE B 419      25.046  10.100   7.527  1.00  0.00           N  \\nATOM   6530  CA  ILE B 419      25.487   8.834   8.117  1.00  0.00           C  \\nATOM   6531  C   ILE B 419      26.863   8.987   8.743  1.00  0.00           C  \\nATOM   6532  O   ILE B 419      27.140   8.430   9.823  1.00  0.00           O  \\nATOM   6533  CB  ILE B 419      25.482   7.695   6.997  1.00  0.00           C  \\nATOM   6534  CG1 ILE B 419      24.040   7.252   6.834  1.00  0.00           C  \\nATOM   6535  CG2 ILE B 419      26.397   6.449   7.401  1.00  0.00           C  \\nATOM   6536  CD1 ILE B 419      23.904   6.385   5.477  1.00  0.00           C  \\nATOM   6537  H   ILE B 419      24.915  10.158   6.527  1.00  0.00           H  \\nATOM   6538  HA  ILE B 419      24.790   8.560   8.908  1.00  0.00           H  \\nATOM   6539  HB  ILE B 419      25.861   8.104   6.061  1.00  0.00           H  \\nATOM   6540 1HG1 ILE B 419      23.741   6.662   7.700  1.00  0.00           H  \\nATOM   6541 2HG1 ILE B 419      23.392   8.128   6.796  1.00  0.00           H  \\nATOM   6542 1HG2 ILE B 419      26.362   5.700   6.610  1.00  0.00           H  \\nATOM   6543 2HG2 ILE B 419      27.425   6.782   7.540  1.00  0.00           H  \\nATOM   6544 3HG2 ILE B 419      26.029   6.013   8.330  1.00  0.00           H  \\nATOM   6545 1HD1 ILE B 419      22.870   6.063   5.353  1.00  0.00           H  \\nATOM   6546 2HD1 ILE B 419      24.199   6.996   4.623  1.00  0.00           H  \\nATOM   6547 3HD1 ILE B 419      24.550   5.511   5.538  1.00  0.00           H  \\nATOM   6548  N   ILE B 420      27.740   9.748   8.086  1.00  0.00           N  \\nATOM   6549  CA  ILE B 420      29.141  10.017   8.585  1.00  0.00           C  \\nATOM   6550  C   ILE B 420      29.040  10.847   9.843  1.00  0.00           C  \\nATOM   6551  O   ILE B 420      29.604  10.466  10.868  1.00  0.00           O  \\nATOM   6552  CB  ILE B 420      29.983  10.758   7.548  1.00  0.00           C  \\nATOM   6553  CG1 ILE B 420      30.261   9.764   6.366  1.00  0.00           C  \\nATOM   6554  CG2 ILE B 420      31.286  11.330   8.185  1.00  0.00           C  \\nATOM   6555  CD1 ILE B 420      30.760  10.539   5.111  1.00  0.00           C  \\nATOM   6556  H   ILE B 420      27.441  10.159   7.213  1.00  0.00           H  \\nATOM   6557  HA  ILE B 420      29.624   9.062   8.790  1.00  0.00           H  \\nATOM   6558  HB  ILE B 420      29.406  11.585   7.135  1.00  0.00           H  \\nATOM   6559 1HG1 ILE B 420      31.007   9.032   6.674  1.00  0.00           H  \\nATOM   6560 2HG1 ILE B 420      29.348   9.218   6.127  1.00  0.00           H  \\nATOM   6561 1HG2 ILE B 420      31.865  11.851   7.423  1.00  0.00           H  \\nATOM   6562 2HG2 ILE B 420      31.026  12.025   8.982  1.00  0.00           H  \\nATOM   6563 3HG2 ILE B 420      31.879  10.513   8.596  1.00  0.00           H  \\nATOM   6564 1HD1 ILE B 420      30.948   9.835   4.300  1.00  0.00           H  \\nATOM   6565 2HD1 ILE B 420      30.000  11.257   4.800  1.00  0.00           H  \\nATOM   6566 3HD1 ILE B 420      31.681  11.067   5.353  1.00  0.00           H  \\nATOM   6567  N   CYS B 421      28.232  11.918   9.834  1.00  0.00           N  \\nATOM   6568  CA  CYS B 421      28.070  12.753  11.047  1.00  0.00           C  \\nATOM   6569  C   CYS B 421      27.471  12.087  12.236  1.00  0.00           C  \\nATOM   6570  O   CYS B 421      27.989  12.241  13.372  1.00  0.00           O  \\nATOM   6571  CB  CYS B 421      27.253  14.021  10.737  1.00  0.00           C  \\nATOM   6572  SG  CYS B 421      28.078  15.247   9.662  1.00  0.00           S  \\nATOM   6573  H   CYS B 421      27.725  12.164   8.996  1.00  0.00           H  \\nATOM   6574  HA  CYS B 421      29.058  13.056  11.393  1.00  0.00           H  \\nATOM   6575 1HB  CYS B 421      26.318  13.742  10.251  1.00  0.00           H  \\nATOM   6576 2HB  CYS B 421      26.999  14.527  11.668  1.00  0.00           H  \\nATOM   6577  HG  CYS B 421      27.078  16.123   9.657  1.00  0.00           H  \\nATOM   6578  N   ILE B 422      26.422  11.288  12.015  1.00  0.00           N  \\nATOM   6579  CA  ILE B 422      25.730  10.636  13.087  1.00  0.00           C  \\nATOM   6580  C   ILE B 422      26.732   9.651  13.722  1.00  0.00           C  \\nATOM   6581  O   ILE B 422      26.812   9.545  14.976  1.00  0.00           O  \\nATOM   6582  CB  ILE B 422      24.530   9.913  12.543  1.00  0.00           C  \\nATOM   6583  CG1 ILE B 422      23.468  10.918  12.254  1.00  0.00           C  \\nATOM   6584  CG2 ILE B 422      24.027   8.880  13.526  1.00  0.00           C  \\nATOM   6585  CD1 ILE B 422      22.393  10.334  11.362  1.00  0.00           C  \\nATOM   6586  H   ILE B 422      26.109  11.141  11.066  1.00  0.00           H  \\nATOM   6587  HA  ILE B 422      25.399  11.394  13.796  1.00  0.00           H  \\nATOM   6588  HB  ILE B 422      24.799   9.409  11.615  1.00  0.00           H  \\nATOM   6589 1HG1 ILE B 422      23.024  11.257  13.189  1.00  0.00           H  \\nATOM   6590 2HG1 ILE B 422      23.912  11.788  11.769  1.00  0.00           H  \\nATOM   6591 1HG2 ILE B 422      23.158   8.372  13.108  1.00  0.00           H  \\nATOM   6592 2HG2 ILE B 422      24.813   8.152  13.722  1.00  0.00           H  \\nATOM   6593 3HG2 ILE B 422      23.746   9.371  14.458  1.00  0.00           H  \\nATOM   6594 1HD1 ILE B 422      21.631  11.090  11.167  1.00  0.00           H  \\nATOM   6595 2HD1 ILE B 422      22.837  10.015  10.418  1.00  0.00           H  \\nATOM   6596 3HD1 ILE B 422      21.936   9.478  11.856  1.00  0.00           H  \\nATOM   6597  N   GLN B 423      27.520   8.909  12.952  1.00  0.00           N  \\nATOM   6598  CA  GLN B 423      28.418   7.950  13.596  1.00  0.00           C  \\nATOM   6599  C   GLN B 423      29.605   8.621  14.269  1.00  0.00           C  \\nATOM   6600  O   GLN B 423      29.934   8.203  15.387  1.00  0.00           O  \\nATOM   6601  CB  GLN B 423      28.917   6.905  12.626  1.00  0.00           C  \\nATOM   6602  CG  GLN B 423      27.753   5.991  12.062  1.00  0.00           C  \\nATOM   6603  CD  GLN B 423      28.360   5.046  11.121  1.00  0.00           C  \\nATOM   6604  OE1 GLN B 423      28.788   3.900  11.498  1.00  0.00           O  \\nATOM   6605  NE2 GLN B 423      28.438   5.469   9.890  1.00  0.00           N  \\nATOM   6606  H   GLN B 423      27.520   8.984  11.945  1.00  0.00           H  \\nATOM   6607  HA  GLN B 423      27.869   7.439  14.387  1.00  0.00           H  \\nATOM   6608 1HB  GLN B 423      29.416   7.394  11.789  1.00  0.00           H  \\nATOM   6609 2HB  GLN B 423      29.653   6.271  13.121  1.00  0.00           H  \\nATOM   6610 1HG  GLN B 423      27.284   5.464  12.893  1.00  0.00           H  \\nATOM   6611 2HG  GLN B 423      27.020   6.622  11.559  1.00  0.00           H  \\nATOM   6612 1HE2 GLN B 423      28.840   4.884   9.184  1.00  0.00           H  \\nATOM   6613 2HE2 GLN B 423      28.096   6.378   9.650  1.00  0.00           H  \\nATOM   6614  N   MET B 424      30.123   9.702  13.696  1.00  0.00           N  \\nATOM   6615  CA  MET B 424      31.301  10.290  14.290  1.00  0.00           C  \\nATOM   6616  C   MET B 424      30.997  11.195  15.437  1.00  0.00           C  \\nATOM   6617  O   MET B 424      31.906  11.531  16.174  1.00  0.00           O  \\nATOM   6618  CB  MET B 424      32.096  11.052  13.251  1.00  0.00           C  \\nATOM   6619  CG  MET B 424      32.639  10.080  12.129  1.00  0.00           C  \\nATOM   6620  SD  MET B 424      33.785  10.814  10.995  1.00  0.00           S  \\nATOM   6621  CE  MET B 424      35.240  11.072  12.009  1.00  0.00           C  \\nATOM   6622  H   MET B 424      29.727  10.124  12.868  1.00  0.00           H  \\nATOM   6623  HA  MET B 424      31.925   9.489  14.687  1.00  0.00           H  \\nATOM   6624 1HB  MET B 424      31.465  11.817  12.800  1.00  0.00           H  \\nATOM   6625 2HB  MET B 424      32.932  11.560  13.733  1.00  0.00           H  \\nATOM   6626 1HG  MET B 424      33.142   9.233  12.594  1.00  0.00           H  \\nATOM   6627 2HG  MET B 424      31.803   9.699  11.542  1.00  0.00           H  \\nATOM   6628 1HE  MET B 424      36.027  11.527  11.407  1.00  0.00           H  \\nATOM   6629 2HE  MET B 424      34.991  11.731  12.842  1.00  0.00           H  \\nATOM   6630 3HE  MET B 424      35.588  10.114  12.397  1.00  0.00           H  \\nATOM   6631  N   ILE B 425      29.753  11.549  15.602  1.00  0.00           N  \\nATOM   6632  CA  ILE B 425      29.346  12.438  16.680  1.00  0.00           C  \\nATOM   6633  C   ILE B 425      28.435  11.718  17.683  1.00  0.00           C  \\nATOM   6634  O   ILE B 425      28.848  11.483  18.823  1.00  0.00           O  \\nATOM   6635  CB  ILE B 425      28.733  13.748  16.142  1.00  0.00           C  \\nATOM   6636  CG1 ILE B 425      29.814  14.490  15.317  1.00  0.00           C  \\nATOM   6637  CG2 ILE B 425      28.300  14.613  17.324  1.00  0.00           C  \\nATOM   6638  CD1 ILE B 425      29.220  15.571  14.358  1.00  0.00           C  \\nATOM   6639  H   ILE B 425      29.053  11.197  14.964  1.00  0.00           H  \\nATOM   6640  HA  ILE B 425      30.227  12.695  17.267  1.00  0.00           H  \\nATOM   6641  HB  ILE B 425      27.869  13.517  15.519  1.00  0.00           H  \\nATOM   6642 1HG1 ILE B 425      30.519  14.974  15.993  1.00  0.00           H  \\nATOM   6643 2HG1 ILE B 425      30.376  13.770  14.721  1.00  0.00           H  \\nATOM   6644 1HG2 ILE B 425      27.866  15.542  16.955  1.00  0.00           H  \\nATOM   6645 2HG2 ILE B 425      27.559  14.076  17.915  1.00  0.00           H  \\nATOM   6646 3HG2 ILE B 425      29.166  14.839  17.946  1.00  0.00           H  \\nATOM   6647 1HD1 ILE B 425      30.029  16.054  13.809  1.00  0.00           H  \\nATOM   6648 2HD1 ILE B 425      28.536  15.096  13.654  1.00  0.00           H  \\nATOM   6649 3HD1 ILE B 425      28.682  16.317  14.940  1.00  0.00           H  \\nATOM   6650  N   SER B 426      27.223  11.313  17.258  1.00  0.00           N  \\nATOM   6651  CA  SER B 426      26.245  10.751  18.149  1.00  0.00           C  \\nATOM   6652  C   SER B 426      26.753   9.358  18.632  1.00  0.00           C  \\nATOM   6653  O   SER B 426      26.721   9.067  19.876  1.00  0.00           O  \\nATOM   6654  CB  SER B 426      24.972  10.463  17.434  1.00  0.00           C  \\nATOM   6655  OG  SER B 426      24.337  11.747  17.102  1.00  0.00           O  \\nATOM   6656  H   SER B 426      26.998  11.410  16.278  1.00  0.00           H  \\nATOM   6657  HA  SER B 426      26.039  11.475  18.938  1.00  0.00           H  \\nATOM   6658 1HB  SER B 426      25.183   9.884  16.535  1.00  0.00           H  \\nATOM   6659 2HB  SER B 426      24.327   9.858  18.069  1.00  0.00           H  \\nATOM   6660  HG  SER B 426      24.923  12.428  17.441  1.00  0.00           H  \\nATOM   6661  N   GLU B 427      27.210   8.490  17.685  1.00  0.00           N  \\nATOM   6662  CA  GLU B 427      27.574   7.156  18.172  1.00  0.00           C  \\nATOM   6663  C   GLU B 427      28.870   7.184  18.995  1.00  0.00           C  \\nATOM   6664  O   GLU B 427      29.043   6.408  19.932  1.00  0.00           O  \\nATOM   6665  CB  GLU B 427      27.761   6.141  17.033  1.00  0.00           C  \\nATOM   6666  CG  GLU B 427      26.462   6.049  16.166  1.00  0.00           C  \\nATOM   6667  CD  GLU B 427      25.290   5.286  16.773  1.00  0.00           C  \\nATOM   6668  OE1 GLU B 427      25.190   5.046  17.987  1.00  0.00           O  \\nATOM   6669  OE2 GLU B 427      24.343   4.987  15.981  1.00  0.00           O  \\nATOM   6670  H   GLU B 427      27.311   8.699  16.702  1.00  0.00           H  \\nATOM   6671  HA  GLU B 427      26.770   6.790  18.811  1.00  0.00           H  \\nATOM   6672 1HB  GLU B 427      28.603   6.444  16.411  1.00  0.00           H  \\nATOM   6673 2HB  GLU B 427      27.999   5.163  17.452  1.00  0.00           H  \\nATOM   6674 1HG  GLU B 427      26.108   7.057  15.953  1.00  0.00           H  \\nATOM   6675 2HG  GLU B 427      26.704   5.571  15.218  1.00  0.00           H  \\nATOM   6676  N   ALA B 428      29.784   8.049  18.599  1.00  0.00           N  \\nATOM   6677  CA  ALA B 428      30.999   8.196  19.367  1.00  0.00           C  \\nATOM   6678  C   ALA B 428      30.707   8.737  20.772  1.00  0.00           C  \\nATOM   6679  O   ALA B 428      31.365   8.299  21.710  1.00  0.00           O  \\nATOM   6680  CB  ALA B 428      31.965   9.166  18.654  1.00  0.00           C  \\nATOM   6681  H   ALA B 428      29.655   8.612  17.770  1.00  0.00           H  \\nATOM   6682  HA  ALA B 428      31.472   7.217  19.444  1.00  0.00           H  \\nATOM   6683 1HB  ALA B 428      32.877   9.269  19.242  1.00  0.00           H  \\nATOM   6684 2HB  ALA B 428      32.212   8.774  17.667  1.00  0.00           H  \\nATOM   6685 3HB  ALA B 428      31.490  10.140  18.549  1.00  0.00           H  \\nATOM   6686  N   ALA B 429      29.782   9.681  20.924  1.00  0.00           N  \\nATOM   6687  CA  ALA B 429      29.317  10.056  22.286  1.00  0.00           C  \\nATOM   6688  C   ALA B 429      28.734   8.891  23.024  1.00  0.00           C  \\nATOM   6689  O   ALA B 429      29.035   8.741  24.226  1.00  0.00           O  \\nATOM   6690  CB  ALA B 429      28.320  11.254  22.189  1.00  0.00           C  \\nATOM   6691  H   ALA B 429      29.387  10.151  20.122  1.00  0.00           H  \\nATOM   6692  HA  ALA B 429      30.187  10.359  22.869  1.00  0.00           H  \\nATOM   6693 1HB  ALA B 429      27.981  11.527  23.188  1.00  0.00           H  \\nATOM   6694 2HB  ALA B 429      28.820  12.107  21.730  1.00  0.00           H  \\nATOM   6695 3HB  ALA B 429      27.464  10.966  21.582  1.00  0.00           H  \\nATOM   6696  N   ARG B 430      27.950   8.051  22.342  1.00  0.00           N  \\nATOM   6697  CA  ARG B 430      27.279   6.964  23.003  1.00  0.00           C  \\nATOM   6698  C   ARG B 430      28.315   5.916  23.464  1.00  0.00           C  \\nATOM   6699  O   ARG B 430      27.990   5.195  24.411  1.00  0.00           O  \\nATOM   6700  CB  ARG B 430      26.231   6.311  22.040  1.00  0.00           C  \\nATOM   6701  CG  ARG B 430      25.005   7.199  21.906  1.00  0.00           C  \\nATOM   6702  CD  ARG B 430      24.243   6.469  20.713  1.00  0.00           C  \\nATOM   6703  NE  ARG B 430      22.898   7.095  20.537  1.00  0.00           N  \\nATOM   6704  CZ  ARG B 430      22.055   6.563  19.640  1.00  0.00           C  \\nATOM   6705  NH1 ARG B 430      22.455   5.479  18.893  1.00  0.00           N  \\nATOM   6706  NH2 ARG B 430      20.834   7.082  19.578  1.00  0.00           N  \\nATOM   6707  H   ARG B 430      27.823   8.176  21.348  1.00  0.00           H  \\nATOM   6708  HA  ARG B 430      26.757   7.359  23.875  1.00  0.00           H  \\nATOM   6709 1HB  ARG B 430      26.684   6.154  21.062  1.00  0.00           H  \\nATOM   6710 2HB  ARG B 430      25.941   5.334  22.427  1.00  0.00           H  \\nATOM   6711 1HG  ARG B 430      24.451   7.199  22.845  1.00  0.00           H  \\nATOM   6712 2HG  ARG B 430      25.317   8.217  21.669  1.00  0.00           H  \\nATOM   6713 1HD  ARG B 430      24.819   6.575  19.794  1.00  0.00           H  \\nATOM   6714 2HD  ARG B 430      24.127   5.412  20.949  1.00  0.00           H  \\nATOM   6715  HE  ARG B 430      22.676   7.901  21.106  1.00  0.00           H  \\nATOM   6716 1HH1 ARG B 430      23.378   5.089  19.022  1.00  0.00           H  \\nATOM   6717 2HH1 ARG B 430      21.826   5.073  18.216  1.00  0.00           H  \\nATOM   6718 1HH2 ARG B 430      20.577   7.844  20.191  1.00  0.00           H  \\nATOM   6719 2HH2 ARG B 430      20.163   6.716  18.919  1.00  0.00           H  \\nATOM   6720  N   PHE B 431      29.410   5.660  22.737  1.00  0.00           N  \\nATOM   6721  CA  PHE B 431      30.282   4.483  22.987  1.00  0.00           C  \\nATOM   6722  C   PHE B 431      31.706   4.905  23.004  1.00  0.00           C  \\nATOM   6723  O   PHE B 431      32.216   5.401  22.014  1.00  0.00           O  \\nATOM   6724  CB  PHE B 431      30.133   3.503  21.840  1.00  0.00           C  \\nATOM   6725  CG  PHE B 431      28.787   2.800  21.777  1.00  0.00           C  \\nATOM   6726  CD1 PHE B 431      28.554   1.687  22.555  1.00  0.00           C  \\nATOM   6727  CD2 PHE B 431      27.835   3.209  20.883  1.00  0.00           C  \\nATOM   6728  CE1 PHE B 431      27.375   0.983  22.431  1.00  0.00           C  \\nATOM   6729  CE2 PHE B 431      26.637   2.520  20.751  1.00  0.00           C  \\nATOM   6730  CZ  PHE B 431      26.397   1.405  21.501  1.00  0.00           C  \\nATOM   6731  H   PHE B 431      29.646   6.297  21.990  1.00  0.00           H  \\nATOM   6732  HA  PHE B 431      29.961   4.005  23.914  1.00  0.00           H  \\nATOM   6733 1HB  PHE B 431      30.279   4.024  20.895  1.00  0.00           H  \\nATOM   6734 2HB  PHE B 431      30.904   2.737  21.914  1.00  0.00           H  \\nATOM   6735  HD1 PHE B 431      29.313   1.369  23.271  1.00  0.00           H  \\nATOM   6736  HD2 PHE B 431      28.021   4.099  20.281  1.00  0.00           H  \\nATOM   6737  HE1 PHE B 431      27.199   0.103  23.050  1.00  0.00           H  \\nATOM   6738  HE2 PHE B 431      25.886   2.872  20.044  1.00  0.00           H  \\nATOM   6739  HZ  PHE B 431      25.471   0.843  21.390  1.00  0.00           H  \\nATOM   6740  N   GLN B 432      32.404   4.736  24.146  1.00  0.00           N  \\nATOM   6741  CA  GLN B 432      33.896   4.970  24.125  1.00  0.00           C  \\nATOM   6742  C   GLN B 432      34.541   4.101  23.085  1.00  0.00           C  \\nATOM   6743  O   GLN B 432      35.545   4.537  22.462  1.00  0.00           O  \\nATOM   6744  CB  GLN B 432      34.536   4.530  25.472  1.00  0.00           C  \\nATOM   6745  CG  GLN B 432      34.096   5.401  26.643  1.00  0.00           C  \\nATOM   6746  CD  GLN B 432      33.031   4.680  27.394  1.00  0.00           C  \\nATOM   6747  OE1 GLN B 432      32.205   3.962  26.769  1.00  0.00           O  \\nATOM   6748  NE2 GLN B 432      33.084   4.731  28.711  1.00  0.00           N  \\nATOM   6749  H   GLN B 432      31.962   4.459  25.011  1.00  0.00           H  \\nATOM   6750  HA  GLN B 432      34.077   6.036  23.985  1.00  0.00           H  \\nATOM   6751 1HB  GLN B 432      34.265   3.495  25.683  1.00  0.00           H  \\nATOM   6752 2HB  GLN B 432      35.622   4.574  25.390  1.00  0.00           H  \\nATOM   6753 1HG  GLN B 432      34.955   5.584  27.289  1.00  0.00           H  \\nATOM   6754 2HG  GLN B 432      33.712   6.345  26.256  1.00  0.00           H  \\nATOM   6755 1HE2 GLN B 432      32.390   4.262  29.259  1.00  0.00           H  \\nATOM   6756 2HE2 GLN B 432      33.818   5.238  29.163  1.00  0.00           H  \\nATOM   6757  N   TYR B 433      33.967   2.907  22.796  1.00  0.00           N  \\nATOM   6758  CA  TYR B 433      34.542   2.040  21.712  1.00  0.00           C  \\nATOM   6759  C   TYR B 433      34.514   2.762  20.364  1.00  0.00           C  \\nATOM   6760  O   TYR B 433      35.455   2.667  19.609  1.00  0.00           O  \\nATOM   6761  CB  TYR B 433      33.774   0.696  21.637  1.00  0.00           C  \\nATOM   6762  CG  TYR B 433      34.296  -0.257  20.647  1.00  0.00           C  \\nATOM   6763  CD1 TYR B 433      35.362  -1.086  21.008  1.00  0.00           C  \\nATOM   6764  CD2 TYR B 433      33.661  -0.469  19.447  1.00  0.00           C  \\nATOM   6765  CE1 TYR B 433      35.776  -2.055  20.145  1.00  0.00           C  \\nATOM   6766  CE2 TYR B 433      34.116  -1.371  18.542  1.00  0.00           C  \\nATOM   6767  CZ  TYR B 433      35.173  -2.190  18.894  1.00  0.00           C  \\nATOM   6768  OH  TYR B 433      35.638  -3.217  18.100  1.00  0.00           O  \\nATOM   6769  H   TYR B 433      33.153   2.588  23.301  1.00  0.00           H  \\nATOM   6770  HA  TYR B 433      35.586   1.835  21.950  1.00  0.00           H  \\nATOM   6771 1HB  TYR B 433      33.799   0.207  22.612  1.00  0.00           H  \\nATOM   6772 2HB  TYR B 433      32.729   0.887  21.393  1.00  0.00           H  \\nATOM   6773  HD1 TYR B 433      35.853  -0.954  21.972  1.00  0.00           H  \\nATOM   6774  HD2 TYR B 433      32.763   0.097  19.198  1.00  0.00           H  \\nATOM   6775  HE1 TYR B 433      36.606  -2.707  20.416  1.00  0.00           H  \\nATOM   6776  HE2 TYR B 433      33.650  -1.439  17.559  1.00  0.00           H  \\nATOM   6777  HH  TYR B 433      35.117  -3.258  17.294  1.00  0.00           H  \\nATOM   6778  N   ILE B 434      33.386   3.438  20.045  1.00  0.00           N  \\nATOM   6779  CA  ILE B 434      33.268   4.102  18.776  1.00  0.00           C  \\nATOM   6780  C   ILE B 434      34.111   5.395  18.740  1.00  0.00           C  \\nATOM   6781  O   ILE B 434      34.733   5.692  17.713  1.00  0.00           O  \\nATOM   6782  CB  ILE B 434      31.778   4.249  18.351  1.00  0.00           C  \\nATOM   6783  CG1 ILE B 434      31.235   2.800  18.187  1.00  0.00           C  \\nATOM   6784  CG2 ILE B 434      31.701   5.027  17.000  1.00  0.00           C  \\nATOM   6785  CD1 ILE B 434      29.790   2.759  17.666  1.00  0.00           C  \\nATOM   6786  H   ILE B 434      32.616   3.482  20.697  1.00  0.00           H  \\nATOM   6787  HA  ILE B 434      33.780   3.503  18.024  1.00  0.00           H  \\nATOM   6788  HB  ILE B 434      31.233   4.797  19.119  1.00  0.00           H  \\nATOM   6789 1HG1 ILE B 434      31.872   2.249  17.496  1.00  0.00           H  \\nATOM   6790 2HG1 ILE B 434      31.274   2.286  19.148  1.00  0.00           H  \\nATOM   6791 1HG2 ILE B 434      30.659   5.131  16.699  1.00  0.00           H  \\nATOM   6792 2HG2 ILE B 434      32.143   6.015  17.123  1.00  0.00           H  \\nATOM   6793 3HG2 ILE B 434      32.247   4.478  16.232  1.00  0.00           H  \\nATOM   6794 1HD1 ILE B 434      29.466   1.722  17.572  1.00  0.00           H  \\nATOM   6795 2HD1 ILE B 434      29.136   3.282  18.365  1.00  0.00           H  \\nATOM   6796 3HD1 ILE B 434      29.741   3.243  16.692  1.00  0.00           H  \\nATOM   6797  N   GLU B 435      34.189   6.161  19.847  1.00  0.00           N  \\nATOM   6798  CA  GLU B 435      35.130   7.278  19.949  1.00  0.00           C  \\nATOM   6799  C   GLU B 435      36.558   6.771  19.658  1.00  0.00           C  \\nATOM   6800  O   GLU B 435      37.296   7.386  18.882  1.00  0.00           O  \\nATOM   6801  CB  GLU B 435      35.028   7.848  21.405  1.00  0.00           C  \\nATOM   6802  CG  GLU B 435      36.040   8.988  21.654  1.00  0.00           C  \\nATOM   6803  CD  GLU B 435      36.266   9.134  23.114  1.00  0.00           C  \\nATOM   6804  OE1 GLU B 435      35.521   8.526  23.943  1.00  0.00           O  \\nATOM   6805  OE2 GLU B 435      37.291   9.808  23.394  1.00  0.00           O  \\nATOM   6806  H   GLU B 435      33.581   5.957  20.627  1.00  0.00           H  \\nATOM   6807  HA  GLU B 435      34.840   8.042  19.227  1.00  0.00           H  \\nATOM   6808 1HB  GLU B 435      34.019   8.222  21.579  1.00  0.00           H  \\nATOM   6809 2HB  GLU B 435      35.209   7.048  22.123  1.00  0.00           H  \\nATOM   6810 1HG  GLU B 435      36.972   8.750  21.143  1.00  0.00           H  \\nATOM   6811 2HG  GLU B 435      35.647   9.909  21.225  1.00  0.00           H  \\nATOM   6812  N   GLY B 436      36.837   5.582  20.198  1.00  0.00           N  \\nATOM   6813  CA  GLY B 436      38.115   4.929  19.977  1.00  0.00           C  \\nATOM   6814  C   GLY B 436      38.405   4.659  18.521  1.00  0.00           C  \\nATOM   6815  O   GLY B 436      39.562   4.877  18.074  1.00  0.00           O  \\nATOM   6816  H   GLY B 436      36.144   5.125  20.774  1.00  0.00           H  \\nATOM   6817 1HA  GLY B 436      38.915   5.549  20.381  1.00  0.00           H  \\nATOM   6818 2HA  GLY B 436      38.138   3.983  20.517  1.00  0.00           H  \\nATOM   6819  N   GLU B 437      37.405   4.161  17.769  1.00  0.00           N  \\nATOM   6820  CA  GLU B 437      37.555   3.856  16.353  1.00  0.00           C  \\nATOM   6821  C   GLU B 437      37.898   5.127  15.625  1.00  0.00           C  \\nATOM   6822  O   GLU B 437      38.667   5.185  14.645  1.00  0.00           O  \\nATOM   6823  CB  GLU B 437      36.225   3.302  15.830  1.00  0.00           C  \\nATOM   6824  CG  GLU B 437      35.935   1.912  16.266  1.00  0.00           C  \\nATOM   6825  CD  GLU B 437      36.929   0.923  15.641  1.00  0.00           C  \\nATOM   6826  OE1 GLU B 437      36.971   0.858  14.362  1.00  0.00           O  \\nATOM   6827  OE2 GLU B 437      37.708   0.375  16.436  1.00  0.00           O  \\nATOM   6828  H   GLU B 437      36.515   3.995  18.216  1.00  0.00           H  \\nATOM   6829  HA  GLU B 437      38.334   3.101  16.242  1.00  0.00           H  \\nATOM   6830 1HB  GLU B 437      35.407   3.939  16.167  1.00  0.00           H  \\nATOM   6831 2HB  GLU B 437      36.226   3.321  14.740  1.00  0.00           H  \\nATOM   6832 1HG  GLU B 437      35.998   1.863  17.353  1.00  0.00           H  \\nATOM   6833 2HG  GLU B 437      34.917   1.657  15.976  1.00  0.00           H  \\nATOM   6834  N   MET B 438      37.282   6.272  16.041  1.00  0.00           N  \\nATOM   6835  CA  MET B 438      37.562   7.507  15.370  1.00  0.00           C  \\nATOM   6836  C   MET B 438      38.907   8.101  15.697  1.00  0.00           C  \\nATOM   6837  O   MET B 438      39.534   8.662  14.789  1.00  0.00           O  \\nATOM   6838  CB  MET B 438      36.429   8.554  15.571  1.00  0.00           C  \\nATOM   6839  CG  MET B 438      34.990   8.030  15.197  1.00  0.00           C  \\nATOM   6840  SD  MET B 438      34.950   7.360  13.469  1.00  0.00           S  \\nATOM   6841  CE  MET B 438      33.699   6.209  13.960  1.00  0.00           C  \\nATOM   6842  H   MET B 438      36.631   6.268  16.813  1.00  0.00           H  \\nATOM   6843  HA  MET B 438      37.647   7.309  14.302  1.00  0.00           H  \\nATOM   6844 1HB  MET B 438      36.410   8.874  16.612  1.00  0.00           H  \\nATOM   6845 2HB  MET B 438      36.635   9.435  14.962  1.00  0.00           H  \\nATOM   6846 1HG  MET B 438      34.697   7.245  15.893  1.00  0.00           H  \\nATOM   6847 2HG  MET B 438      34.272   8.846  15.282  1.00  0.00           H  \\nATOM   6848 1HE  MET B 438      33.413   5.595  13.105  1.00  0.00           H  \\nATOM   6849 2HE  MET B 438      34.085   5.569  14.755  1.00  0.00           H  \\nATOM   6850 3HE  MET B 438      32.826   6.753  14.324  1.00  0.00           H  \\nATOM   6851  N   ARG B 439      39.312   7.896  16.932  1.00  0.00           N  \\nATOM   6852  CA  ARG B 439      40.654   8.353  17.390  1.00  0.00           C  \\nATOM   6853  C   ARG B 439      41.689   7.614  16.544  1.00  0.00           C  \\nATOM   6854  O   ARG B 439      42.655   8.199  16.098  1.00  0.00           O  \\nATOM   6855  CB  ARG B 439      40.908   8.054  18.873  1.00  0.00           C  \\nATOM   6856  CG  ARG B 439      40.104   8.802  19.897  1.00  0.00           C  \\nATOM   6857  CD  ARG B 439      40.446   8.126  21.282  1.00  0.00           C  \\nATOM   6858  NE  ARG B 439      41.684   8.645  21.795  1.00  0.00           N  \\nATOM   6859  CZ  ARG B 439      41.715   9.362  22.923  1.00  0.00           C  \\nATOM   6860  NH1 ARG B 439      40.586   9.517  23.627  1.00  0.00           N  \\nATOM   6861  NH2 ARG B 439      42.818   9.910  23.383  1.00  0.00           N  \\nATOM   6862  H   ARG B 439      38.705   7.421  17.584  1.00  0.00           H  \\nATOM   6863  HA  ARG B 439      40.715   9.434  17.257  1.00  0.00           H  \\nATOM   6864 1HB  ARG B 439      40.726   6.998  19.067  1.00  0.00           H  \\nATOM   6865 2HB  ARG B 439      41.952   8.255  19.111  1.00  0.00           H  \\nATOM   6866 1HG  ARG B 439      40.390   9.854  19.887  1.00  0.00           H  \\nATOM   6867 2HG  ARG B 439      39.043   8.713  19.663  1.00  0.00           H  \\nATOM   6868 1HD  ARG B 439      39.649   8.335  21.995  1.00  0.00           H  \\nATOM   6869 2HD  ARG B 439      40.539   7.049  21.148  1.00  0.00           H  \\nATOM   6870  HE  ARG B 439      42.528   8.449  21.274  1.00  0.00           H  \\nATOM   6871 1HH1 ARG B 439      39.727   9.095  23.304  1.00  0.00           H  \\nATOM   6872 2HH1 ARG B 439      40.593  10.056  24.481  1.00  0.00           H  \\nATOM   6873 1HH2 ARG B 439      43.687   9.797  22.879  1.00  0.00           H  \\nATOM   6874 2HH2 ARG B 439      42.797  10.444  24.239  1.00  0.00           H  \\nATOM   6875  N   THR B 440      41.461   6.319  16.263  1.00  0.00           N  \\nATOM   6876  CA  THR B 440      42.456   5.519  15.578  1.00  0.00           C  \\nATOM   6877  C   THR B 440      42.548   6.007  14.123  1.00  0.00           C  \\nATOM   6878  O   THR B 440      43.598   6.119  13.520  1.00  0.00           O  \\nATOM   6879  CB  THR B 440      42.146   4.032  15.664  1.00  0.00           C  \\nATOM   6880  OG1 THR B 440      42.146   3.604  16.986  1.00  0.00           O  \\nATOM   6881  CG2 THR B 440      43.256   3.157  14.996  1.00  0.00           C  \\nATOM   6882  H   THR B 440      40.586   5.892  16.531  1.00  0.00           H  \\nATOM   6883  HA  THR B 440      43.423   5.689  16.052  1.00  0.00           H  \\nATOM   6884  HB  THR B 440      41.201   3.828  15.161  1.00  0.00           H  \\nATOM   6885  HG1 THR B 440      42.343   4.347  17.562  1.00  0.00           H  \\nATOM   6886 1HG2 THR B 440      42.990   2.104  15.082  1.00  0.00           H  \\nATOM   6887 2HG2 THR B 440      43.345   3.424  13.943  1.00  0.00           H  \\nATOM   6888 3HG2 THR B 440      44.208   3.332  15.496  1.00  0.00           H  \\nATOM   6889  N   ARG B 441      41.416   6.306  13.550  1.00  0.00           N  \\nATOM   6890  CA  ARG B 441      41.412   6.850  12.178  1.00  0.00           C  \\nATOM   6891  C   ARG B 441      42.153   8.158  12.061  1.00  0.00           C  \\nATOM   6892  O   ARG B 441      42.862   8.376  11.086  1.00  0.00           O  \\nATOM   6893  CB  ARG B 441      39.997   6.872  11.597  1.00  0.00           C  \\nATOM   6894  CG  ARG B 441      39.465   5.431  11.317  1.00  0.00           C  \\nATOM   6895  CD  ARG B 441      38.003   5.591  11.235  1.00  0.00           C  \\nATOM   6896  NE  ARG B 441      37.309   4.427  10.707  1.00  0.00           N  \\nATOM   6897  CZ  ARG B 441      37.100   3.300  11.364  1.00  0.00           C  \\nATOM   6898  NH1 ARG B 441      37.551   3.209  12.640  1.00  0.00           N  \\nATOM   6899  NH2 ARG B 441      36.446   2.281  10.824  1.00  0.00           N  \\nATOM   6900  H   ARG B 441      40.537   6.172  14.029  1.00  0.00           H  \\nATOM   6901  HA  ARG B 441      42.031   6.208  11.550  1.00  0.00           H  \\nATOM   6902 1HB  ARG B 441      39.326   7.374  12.293  1.00  0.00           H  \\nATOM   6903 2HB  ARG B 441      39.994   7.444  10.669  1.00  0.00           H  \\nATOM   6904 1HG  ARG B 441      39.888   5.062  10.382  1.00  0.00           H  \\nATOM   6905 2HG  ARG B 441      39.758   4.770  12.133  1.00  0.00           H  \\nATOM   6906 1HD  ARG B 441      37.601   5.780  12.230  1.00  0.00           H  \\nATOM   6907 2HD  ARG B 441      37.765   6.430  10.582  1.00  0.00           H  \\nATOM   6908  HE  ARG B 441      36.956   4.479   9.761  1.00  0.00           H  \\nATOM   6909 1HH1 ARG B 441      38.032   3.990  13.063  1.00  0.00           H  \\nATOM   6910 2HH1 ARG B 441      37.406   2.360  13.167  1.00  0.00           H  \\nATOM   6911 1HH2 ARG B 441      36.089   2.352   9.881  1.00  0.00           H  \\nATOM   6912 2HH2 ARG B 441      36.304   1.434  11.355  1.00  0.00           H  \\nATOM   6913  N   ILE B 442      42.042   9.030  13.098  1.00  0.00           N  \\nATOM   6914  CA  ILE B 442      42.748  10.264  13.086  1.00  0.00           C  \\nATOM   6915  C   ILE B 442      44.237  10.019  13.274  1.00  0.00           C  \\nATOM   6916  O   ILE B 442      45.076  10.573  12.587  1.00  0.00           O  \\nATOM   6917  CB  ILE B 442      42.177  11.228  14.134  1.00  0.00           C  \\nATOM   6918  CG1 ILE B 442      40.727  11.566  13.773  1.00  0.00           C  \\nATOM   6919  CG2 ILE B 442      43.110  12.569  14.326  1.00  0.00           C  \\nATOM   6920  CD1 ILE B 442      40.015  12.221  14.941  1.00  0.00           C  \\nATOM   6921  H   ILE B 442      41.454   8.807  13.889  1.00  0.00           H  \\nATOM   6922  HA  ILE B 442      42.635  10.717  12.102  1.00  0.00           H  \\nATOM   6923  HB  ILE B 442      42.113  10.722  15.097  1.00  0.00           H  \\nATOM   6924 1HG1 ILE B 442      40.713  12.233  12.912  1.00  0.00           H  \\nATOM   6925 2HG1 ILE B 442      40.201  10.655  13.488  1.00  0.00           H  \\nATOM   6926 1HG2 ILE B 442      42.662  13.219  15.078  1.00  0.00           H  \\nATOM   6927 2HG2 ILE B 442      44.108  12.275  14.649  1.00  0.00           H  \\nATOM   6928 3HG2 ILE B 442      43.178  13.104  13.378  1.00  0.00           H  \\nATOM   6929 1HD1 ILE B 442      38.987  12.450  14.658  1.00  0.00           H  \\nATOM   6930 2HD1 ILE B 442      40.014  11.541  15.794  1.00  0.00           H  \\nATOM   6931 3HD1 ILE B 442      40.530  13.142  15.211  1.00  0.00           H  \\nATOM   6932  N   ARG B 443      44.534   9.059  14.154  1.00  0.00           N  \\nATOM   6933  CA  ARG B 443      45.944   8.819  14.517  1.00  0.00           C  \\nATOM   6934  C   ARG B 443      46.772   8.360  13.335  1.00  0.00           C  \\nATOM   6935  O   ARG B 443      47.887   8.802  13.211  1.00  0.00           O  \\nATOM   6936  CB  ARG B 443      45.936   7.788  15.649  1.00  0.00           C  \\nATOM   6937  CG  ARG B 443      47.300   7.369  16.118  1.00  0.00           C  \\nATOM   6938  CD  ARG B 443      47.132   6.432  17.266  1.00  0.00           C  \\nATOM   6939  NE  ARG B 443      46.923   7.161  18.510  1.00  0.00           N  \\nATOM   6940  CZ  ARG B 443      45.933   6.929  19.387  1.00  0.00           C  \\nATOM   6941  NH1 ARG B 443      45.008   5.997  19.259  1.00  0.00           N  \\nATOM   6942  NH2 ARG B 443      45.963   7.646  20.506  1.00  0.00           N  \\nATOM   6943  H   ARG B 443      43.814   8.492  14.578  1.00  0.00           H  \\nATOM   6944  HA  ARG B 443      46.378   9.759  14.860  1.00  0.00           H  \\nATOM   6945 1HB  ARG B 443      45.398   8.193  16.505  1.00  0.00           H  \\nATOM   6946 2HB  ARG B 443      45.406   6.893  15.323  1.00  0.00           H  \\nATOM   6947 1HG  ARG B 443      47.828   6.872  15.304  1.00  0.00           H  \\nATOM   6948 2HG  ARG B 443      47.863   8.249  16.432  1.00  0.00           H  \\nATOM   6949 1HD  ARG B 443      46.269   5.790  17.089  1.00  0.00           H  \\nATOM   6950 2HD  ARG B 443      48.026   5.818  17.368  1.00  0.00           H  \\nATOM   6951  HE  ARG B 443      47.573   7.902  18.735  1.00  0.00           H  \\nATOM   6952 1HH1 ARG B 443      45.011   5.390  18.451  1.00  0.00           H  \\nATOM   6953 2HH1 ARG B 443      44.297   5.890  19.968  1.00  0.00           H  \\nATOM   6954 1HH2 ARG B 443      46.703   8.318  20.656  1.00  0.00           H  \\nATOM   6955 2HH2 ARG B 443      45.247   7.519  21.206  1.00  0.00           H  \\nATOM   6956  N   TYR B 444      46.217   7.475  12.515  1.00  0.00           N  \\nATOM   6957  CA  TYR B 444      46.847   6.892  11.283  1.00  0.00           C  \\nATOM   6958  C   TYR B 444      46.436   7.597   9.989  1.00  0.00           C  \\nATOM   6959  O   TYR B 444      46.716   7.080   8.880  1.00  0.00           O  \\nATOM   6960  CB  TYR B 444      46.595   5.365  11.215  1.00  0.00           C  \\nATOM   6961  CG  TYR B 444      47.210   4.697  12.415  1.00  0.00           C  \\nATOM   6962  CD1 TYR B 444      48.594   4.498  12.514  1.00  0.00           C  \\nATOM   6963  CD2 TYR B 444      46.428   4.231  13.408  1.00  0.00           C  \\nATOM   6964  CE1 TYR B 444      49.163   3.915  13.645  1.00  0.00           C  \\nATOM   6965  CE2 TYR B 444      46.995   3.672  14.631  1.00  0.00           C  \\nATOM   6966  CZ  TYR B 444      48.372   3.500  14.695  1.00  0.00           C  \\nATOM   6967  OH  TYR B 444      48.963   2.872  15.770  1.00  0.00           O  \\nATOM   6968  H   TYR B 444      45.284   7.188  12.773  1.00  0.00           H  \\nATOM   6969  HA  TYR B 444      47.922   7.066  11.329  1.00  0.00           H  \\nATOM   6970 1HB  TYR B 444      45.521   5.174  11.183  1.00  0.00           H  \\nATOM   6971 2HB  TYR B 444      47.025   4.966  10.297  1.00  0.00           H  \\nATOM   6972  HD1 TYR B 444      49.248   4.801  11.696  1.00  0.00           H  \\nATOM   6973  HD2 TYR B 444      45.343   4.265  13.309  1.00  0.00           H  \\nATOM   6974  HE1 TYR B 444      50.242   3.770  13.699  1.00  0.00           H  \\nATOM   6975  HE2 TYR B 444      46.342   3.402  15.461  1.00  0.00           H  \\nATOM   6976  HH  TYR B 444      48.285   2.608  16.397  1.00  0.00           H  \\nATOM   6977  N   ASN B 445      45.759   8.757  10.120  1.00  0.00           N  \\nATOM   6978  CA  ASN B 445      45.331   9.591   9.020  1.00  0.00           C  \\nATOM   6979  C   ASN B 445      44.669   8.687   7.977  1.00  0.00           C  \\nATOM   6980  O   ASN B 445      45.047   8.646   6.798  1.00  0.00           O  \\nATOM   6981  CB  ASN B 445      46.540  10.401   8.504  1.00  0.00           C  \\nATOM   6982  CG  ASN B 445      46.146  11.520   7.523  1.00  0.00           C  \\nATOM   6983  OD1 ASN B 445      45.271  12.352   7.805  1.00  0.00           O  \\nATOM   6984  ND2 ASN B 445      46.813  11.544   6.356  1.00  0.00           N  \\nATOM   6985  H   ASN B 445      45.547   9.043  11.065  1.00  0.00           H  \\nATOM   6986  HA  ASN B 445      44.564  10.277   9.384  1.00  0.00           H  \\nATOM   6987 1HB  ASN B 445      47.065  10.850   9.348  1.00  0.00           H  \\nATOM   6988 2HB  ASN B 445      47.239   9.732   8.003  1.00  0.00           H  \\nATOM   6989 1HD2 ASN B 445      46.604  12.246   5.675  1.00  0.00           H  \\nATOM   6990 2HD2 ASN B 445      47.517  10.860   6.169  1.00  0.00           H  \\nATOM   6991  N   ARG B 446      43.736   7.844   8.431  1.00  0.00           N  \\nATOM   6992  CA  ARG B 446      43.100   6.850   7.585  1.00  0.00           C  \\nATOM   6993  C   ARG B 446      41.598   7.067   7.535  1.00  0.00           C  \\nATOM   6994  O   ARG B 446      41.014   7.531   8.510  1.00  0.00           O  \\nATOM   6995  CB  ARG B 446      43.368   5.412   8.091  1.00  0.00           C  \\nATOM   6996  CG  ARG B 446      43.196   4.458   6.905  1.00  0.00           C  \\nATOM   6997  CD  ARG B 446      43.091   3.047   7.306  1.00  0.00           C  \\nATOM   6998  NE  ARG B 446      44.423   2.467   7.265  1.00  0.00           N  \\nATOM   6999  CZ  ARG B 446      44.774   1.415   6.497  1.00  0.00           C  \\nATOM   7000  NH1 ARG B 446      43.859   0.803   5.718  1.00  0.00           N  \\nATOM   7001  NH2 ARG B 446      46.021   0.934   6.549  1.00  0.00           N  \\nATOM   7002  H   ARG B 446      43.468   7.907   9.403  1.00  0.00           H  \\nATOM   7003  HA  ARG B 446      43.512   6.937   6.579  1.00  0.00           H  \\nATOM   7004 1HB  ARG B 446      44.375   5.353   8.503  1.00  0.00           H  \\nATOM   7005 2HB  ARG B 446      42.671   5.173   8.895  1.00  0.00           H  \\nATOM   7006 1HG  ARG B 446      42.286   4.714   6.361  1.00  0.00           H  \\nATOM   7007 2HG  ARG B 446      44.055   4.547   6.238  1.00  0.00           H  \\nATOM   7008 1HD  ARG B 446      42.685   2.985   8.315  1.00  0.00           H  \\nATOM   7009 2HD  ARG B 446      42.431   2.521   6.617  1.00  0.00           H  \\nATOM   7010  HE  ARG B 446      45.134   2.881   7.853  1.00  0.00           H  \\nATOM   7011 1HH1 ARG B 446      42.904   1.132   5.707  1.00  0.00           H  \\nATOM   7012 2HH1 ARG B 446      44.129   0.016   5.146  1.00  0.00           H  \\nATOM   7013 1HH2 ARG B 446      46.701   1.359   7.164  1.00  0.00           H  \\nATOM   7014 2HH2 ARG B 446      46.283   0.147   5.974  1.00  0.00           H  \\nATOM   7015  N   ARG B 447      40.983   6.803   6.408  1.00  0.00           N  \\nATOM   7016  CA  ARG B 447      39.537   6.652   6.308  1.00  0.00           C  \\nATOM   7017  C   ARG B 447      39.243   5.181   6.079  1.00  0.00           C  \\nATOM   7018  O   ARG B 447      39.981   4.458   5.356  1.00  0.00           O  \\nATOM   7019  CB  ARG B 447      38.921   7.551   5.204  1.00  0.00           C  \\nATOM   7020  CG  ARG B 447      39.786   7.496   3.881  1.00  0.00           C  \\nATOM   7021  CD  ARG B 447      39.778   8.874   3.184  1.00  0.00           C  \\nATOM   7022  NE  ARG B 447      38.440   9.037   2.671  1.00  0.00           N  \\nATOM   7023  CZ  ARG B 447      37.863  10.209   2.333  1.00  0.00           C  \\nATOM   7024  NH1 ARG B 447      38.445  11.388   2.498  1.00  0.00           N  \\nATOM   7025  NH2 ARG B 447      36.651  10.144   1.861  1.00  0.00           N  \\nATOM   7026  H   ARG B 447      41.547   6.701   5.576  1.00  0.00           H  \\nATOM   7027  HA  ARG B 447      39.092   6.945   7.259  1.00  0.00           H  \\nATOM   7028 1HB  ARG B 447      37.905   7.221   4.992  1.00  0.00           H  \\nATOM   7029 2HB  ARG B 447      38.863   8.579   5.562  1.00  0.00           H  \\nATOM   7030 1HG  ARG B 447      40.813   7.227   4.129  1.00  0.00           H  \\nATOM   7031 2HG  ARG B 447      39.368   6.750   3.204  1.00  0.00           H  \\nATOM   7032 1HD  ARG B 447      40.024   9.650   3.909  1.00  0.00           H  \\nATOM   7033 2HD  ARG B 447      40.515   8.880   2.382  1.00  0.00           H  \\nATOM   7034  HE  ARG B 447      37.878   8.205   2.551  1.00  0.00           H  \\nATOM   7035 1HH1 ARG B 447      39.372  11.442   2.897  1.00  0.00           H  \\nATOM   7036 2HH1 ARG B 447      37.962  12.232   2.225  1.00  0.00           H  \\nATOM   7037 1HH2 ARG B 447      36.193   9.248   1.765  1.00  0.00           H  \\nATOM   7038 2HH2 ARG B 447      36.169  10.988   1.591  1.00  0.00           H  \\nATOM   7039  N   SER B 448      38.167   4.722   6.665  1.00  0.00           N  \\nATOM   7040  CA  SER B 448      37.740   3.347   6.472  1.00  0.00           C  \\nATOM   7041  C   SER B 448      36.379   3.140   7.031  1.00  0.00           C  \\nATOM   7042  O   SER B 448      35.925   3.850   7.986  1.00  0.00           O  \\nATOM   7043  CB  SER B 448      38.744   2.430   7.100  1.00  0.00           C  \\nATOM   7044  OG  SER B 448      38.746   2.640   8.482  1.00  0.00           O  \\nATOM   7045  H   SER B 448      37.621   5.328   7.261  1.00  0.00           H  \\nATOM   7046  HA  SER B 448      37.689   3.147   5.401  1.00  0.00           H  \\nATOM   7047 1HB  SER B 448      38.488   1.397   6.867  1.00  0.00           H  \\nATOM   7048 2HB  SER B 448      39.729   2.626   6.679  1.00  0.00           H  \\nATOM   7049  HG  SER B 448      38.093   3.324   8.647  1.00  0.00           H  \\nATOM   7050  N   ALA B 449      35.663   2.187   6.455  1.00  0.00           N  \\nATOM   7051  CA  ALA B 449      34.323   1.958   6.861  1.00  0.00           C  \\nATOM   7052  C   ALA B 449      34.268   1.213   8.211  1.00  0.00           C  \\nATOM   7053  O   ALA B 449      35.226   0.509   8.556  1.00  0.00           O  \\nATOM   7054  CB  ALA B 449      33.571   1.198   5.692  1.00  0.00           C  \\nATOM   7055  H   ALA B 449      36.060   1.615   5.724  1.00  0.00           H  \\nATOM   7056  HA  ALA B 449      33.852   2.925   7.037  1.00  0.00           H  \\nATOM   7057 1HB  ALA B 449      32.538   1.012   5.985  1.00  0.00           H  \\nATOM   7058 2HB  ALA B 449      33.588   1.810   4.790  1.00  0.00           H  \\nATOM   7059 3HB  ALA B 449      34.068   0.249   5.496  1.00  0.00           H  \\nATOM   7060  N   PRO B 450      33.178   1.364   8.972  1.00  0.00           N  \\nATOM   7061  CA  PRO B 450      33.103   0.651  10.249  1.00  0.00           C  \\nATOM   7062  C   PRO B 450      32.971  -0.865  10.074  1.00  0.00           C  \\nATOM   7063  O   PRO B 450      32.197  -1.368   9.213  1.00  0.00           O  \\nATOM   7064  CB  PRO B 450      31.801   1.145  10.861  1.00  0.00           C  \\nATOM   7065  CG  PRO B 450      30.981   1.629   9.671  1.00  0.00           C  \\nATOM   7066  CD  PRO B 450      31.990   2.204   8.725  1.00  0.00           C  \\nATOM   7067  HA  PRO B 450      33.960   0.939  10.875  1.00  0.00           H  \\nATOM   7068 1HB  PRO B 450      31.310   0.329  11.411  1.00  0.00           H  \\nATOM   7069 2HB  PRO B 450      32.007   1.944  11.588  1.00  0.00           H  \\nATOM   7070 1HG  PRO B 450      30.420   0.792   9.231  1.00  0.00           H  \\nATOM   7071 2HG  PRO B 450      30.239   2.372   9.998  1.00  0.00           H  \\nATOM   7072 1HD  PRO B 450      31.624   2.104   7.692  1.00  0.00           H  \\nATOM   7073 2HD  PRO B 450      32.166   3.261   8.974  1.00  0.00           H  \\nATOM   7074  N   ASP B 451      33.666  -1.593  10.947  1.00  0.00           N  \\nATOM   7075  CA  ASP B 451      33.591  -3.041  10.935  1.00  0.00           C  \\nATOM   7076  C   ASP B 451      32.336  -3.548  11.649  1.00  0.00           C  \\nATOM   7077  O   ASP B 451      31.596  -2.749  12.244  1.00  0.00           O  \\nATOM   7078  CB  ASP B 451      34.960  -3.666  11.395  1.00  0.00           C  \\nATOM   7079  CG  ASP B 451      35.178  -3.629  12.865  1.00  0.00           C  \\nATOM   7080  OD1 ASP B 451      34.203  -3.779  13.654  1.00  0.00           O  \\nATOM   7081  OD2 ASP B 451      36.395  -3.476  13.179  1.00  0.00           O  \\nATOM   7082  H   ASP B 451      34.255  -1.137  11.629  1.00  0.00           H  \\nATOM   7083  HA  ASP B 451      33.383  -3.368   9.916  1.00  0.00           H  \\nATOM   7084 1HB  ASP B 451      35.012  -4.706  11.071  1.00  0.00           H  \\nATOM   7085 2HB  ASP B 451      35.782  -3.133  10.917  1.00  0.00           H  \\nATOM   7086  N   PRO B 452      31.997  -4.878  11.567  1.00  0.00           N  \\nATOM   7087  CA  PRO B 452      30.777  -5.418  12.140  1.00  0.00           C  \\nATOM   7088  C   PRO B 452      30.613  -5.168  13.611  1.00  0.00           C  \\nATOM   7089  O   PRO B 452      29.489  -5.129  14.103  1.00  0.00           O  \\nATOM   7090  CB  PRO B 452      30.847  -6.973  11.921  1.00  0.00           C  \\nATOM   7091  CG  PRO B 452      31.909  -7.105  10.867  1.00  0.00           C  \\nATOM   7092  CD  PRO B 452      32.758  -5.882  10.784  1.00  0.00           C  \\nATOM   7093  HA  PRO B 452      29.911  -5.008  11.600  1.00  0.00           H  \\nATOM   7094 1HB  PRO B 452      31.099  -7.475  12.867  1.00  0.00           H  \\nATOM   7095 2HB  PRO B 452      29.864  -7.353  11.607  1.00  0.00           H  \\nATOM   7096 1HG  PRO B 452      32.552  -7.971  11.082  1.00  0.00           H  \\nATOM   7097 2HG  PRO B 452      31.449  -7.287   9.885  1.00  0.00           H  \\nATOM   7098 1HD  PRO B 452      33.740  -6.085  11.236  1.00  0.00           H  \\nATOM   7099 2HD  PRO B 452      32.870  -5.583   9.731  1.00  0.00           H  \\nATOM   7100  N   SER B 453      31.702  -5.047  14.395  1.00  0.00           N  \\nATOM   7101  CA  SER B 453      31.574  -4.787  15.795  1.00  0.00           C  \\nATOM   7102  C   SER B 453      30.925  -3.414  16.024  1.00  0.00           C  \\nATOM   7103  O   SER B 453      30.097  -3.245  16.978  1.00  0.00           O  \\nATOM   7104  CB  SER B 453      32.955  -4.944  16.543  1.00  0.00           C  \\nATOM   7105  OG  SER B 453      33.871  -3.946  16.140  1.00  0.00           O  \\nATOM   7106  H   SER B 453      32.624  -5.140  13.994  1.00  0.00           H  \\nATOM   7107  HA  SER B 453      30.875  -5.509  16.219  1.00  0.00           H  \\nATOM   7108 1HB  SER B 453      32.795  -4.879  17.619  1.00  0.00           H  \\nATOM   7109 2HB  SER B 453      33.372  -5.928  16.333  1.00  0.00           H  \\nATOM   7110  HG  SER B 453      33.408  -3.408  15.493  1.00  0.00           H  \\nATOM   7111  N   VAL B 454      31.341  -2.437  15.198  1.00  0.00           N  \\nATOM   7112  CA  VAL B 454      30.805  -1.057  15.368  1.00  0.00           C  \\nATOM   7113  C   VAL B 454      29.367  -0.999  14.947  1.00  0.00           C  \\nATOM   7114  O   VAL B 454      28.486  -0.452  15.668  1.00  0.00           O  \\nATOM   7115  CB  VAL B 454      31.608  -0.146  14.501  1.00  0.00           C  \\nATOM   7116  CG1 VAL B 454      31.037   1.274  14.501  1.00  0.00           C  \\nATOM   7117  CG2 VAL B 454      33.114  -0.174  14.931  1.00  0.00           C  \\nATOM   7118  H   VAL B 454      32.010  -2.618  14.463  1.00  0.00           H  \\nATOM   7119  HA  VAL B 454      30.917  -0.769  16.414  1.00  0.00           H  \\nATOM   7120  HB  VAL B 454      31.522  -0.476  13.466  1.00  0.00           H  \\nATOM   7121 1HG1 VAL B 454      31.646   1.911  13.859  1.00  0.00           H  \\nATOM   7122 2HG1 VAL B 454      30.014   1.255  14.126  1.00  0.00           H  \\nATOM   7123 3HG1 VAL B 454      31.045   1.670  15.516  1.00  0.00           H  \\nATOM   7124 1HG2 VAL B 454      33.688   0.494  14.290  1.00  0.00           H  \\nATOM   7125 2HG2 VAL B 454      33.203   0.152  15.967  1.00  0.00           H  \\nATOM   7126 3HG2 VAL B 454      33.501  -1.189  14.836  1.00  0.00           H  \\nATOM   7127  N   ILE B 455      29.086  -1.619  13.788  1.00  0.00           N  \\nATOM   7128  CA  ILE B 455      27.741  -1.622  13.247  1.00  0.00           C  \\nATOM   7129  C   ILE B 455      26.761  -2.304  14.252  1.00  0.00           C  \\nATOM   7130  O   ILE B 455      25.648  -1.828  14.604  1.00  0.00           O  \\nATOM   7131  CB  ILE B 455      27.658  -2.215  11.791  1.00  0.00           C  \\nATOM   7132  CG1 ILE B 455      28.489  -1.302  10.868  1.00  0.00           C  \\nATOM   7133  CG2 ILE B 455      26.120  -2.503  11.436  1.00  0.00           C  \\nATOM   7134  CD1 ILE B 455      28.837  -1.801   9.445  1.00  0.00           C  \\nATOM   7135  H   ILE B 455      29.821  -2.092  13.282  1.00  0.00           H  \\nATOM   7136  HA  ILE B 455      27.386  -0.593  13.198  1.00  0.00           H  \\nATOM   7137  HB  ILE B 455      28.227  -3.143  11.743  1.00  0.00           H  \\nATOM   7138 1HG1 ILE B 455      27.968  -0.356  10.727  1.00  0.00           H  \\nATOM   7139 2HG1 ILE B 455      29.446  -1.080  11.341  1.00  0.00           H  \\nATOM   7140 1HG2 ILE B 455      26.050  -2.914  10.429  1.00  0.00           H  \\nATOM   7141 2HG2 ILE B 455      25.710  -3.218  12.148  1.00  0.00           H  \\nATOM   7142 3HG2 ILE B 455      25.554  -1.573  11.489  1.00  0.00           H  \\nATOM   7143 1HD1 ILE B 455      29.424  -1.043   8.927  1.00  0.00           H  \\nATOM   7144 2HD1 ILE B 455      29.415  -2.723   9.514  1.00  0.00           H  \\nATOM   7145 3HD1 ILE B 455      27.918  -1.989   8.891  1.00  0.00           H  \\nATOM   7146  N   THR B 456      27.147  -3.503  14.763  1.00  0.00           N  \\nATOM   7147  CA  THR B 456      26.323  -4.159  15.740  1.00  0.00           C  \\nATOM   7148  C   THR B 456      26.088  -3.421  17.066  1.00  0.00           C  \\nATOM   7149  O   THR B 456      24.971  -3.367  17.622  1.00  0.00           O  \\nATOM   7150  CB  THR B 456      26.743  -5.624  15.976  1.00  0.00           C  \\nATOM   7151  OG1 THR B 456      26.483  -6.309  14.745  1.00  0.00           O  \\nATOM   7152  CG2 THR B 456      25.931  -6.261  17.002  1.00  0.00           C  \\nATOM   7153  H   THR B 456      28.006  -3.944  14.468  1.00  0.00           H  \\nATOM   7154  HA  THR B 456      25.294  -4.163  15.378  1.00  0.00           H  \\nATOM   7155  HB  THR B 456      27.786  -5.657  16.290  1.00  0.00           H  \\nATOM   7156  HG1 THR B 456      26.122  -5.690  14.105  1.00  0.00           H  \\nATOM   7157 1HG2 THR B 456      26.259  -7.291  17.138  1.00  0.00           H  \\nATOM   7158 2HG2 THR B 456      26.039  -5.718  17.941  1.00  0.00           H  \\nATOM   7159 3HG2 THR B 456      24.886  -6.251  16.696  1.00  0.00           H  \\nATOM   7160  N   LEU B 457      27.147  -2.760  17.568  1.00  0.00           N  \\nATOM   7161  CA  LEU B 457      26.952  -1.873  18.659  1.00  0.00           C  \\nATOM   7162  C   LEU B 457      25.956  -0.792  18.418  1.00  0.00           C  \\nATOM   7163  O   LEU B 457      25.059  -0.588  19.228  1.00  0.00           O  \\nATOM   7164  CB  LEU B 457      28.251  -1.282  19.226  1.00  0.00           C  \\nATOM   7165  CG  LEU B 457      29.065  -2.229  20.156  1.00  0.00           C  \\nATOM   7166  CD1 LEU B 457      30.462  -1.511  20.199  1.00  0.00           C  \\nATOM   7167  CD2 LEU B 457      28.496  -2.358  21.517  1.00  0.00           C  \\nATOM   7168  H   LEU B 457      28.076  -2.878  17.190  1.00  0.00           H  \\nATOM   7169  HA  LEU B 457      26.474  -2.422  19.470  1.00  0.00           H  \\nATOM   7170 1HB  LEU B 457      28.894  -0.996  18.395  1.00  0.00           H  \\nATOM   7171 2HB  LEU B 457      28.006  -0.385  19.794  1.00  0.00           H  \\nATOM   7172  HG  LEU B 457      29.096  -3.228  19.721  1.00  0.00           H  \\nATOM   7173 1HD1 LEU B 457      31.145  -2.081  20.830  1.00  0.00           H  \\nATOM   7174 2HD1 LEU B 457      30.870  -1.447  19.190  1.00  0.00           H  \\nATOM   7175 3HD1 LEU B 457      30.343  -0.508  20.607  1.00  0.00           H  \\nATOM   7176 1HD2 LEU B 457      29.115  -3.034  22.108  1.00  0.00           H  \\nATOM   7177 2HD2 LEU B 457      28.470  -1.378  21.995  1.00  0.00           H  \\nATOM   7178 3HD2 LEU B 457      27.484  -2.757  21.452  1.00  0.00           H  \\nATOM   7179  N   GLU B 458      26.109  -0.067  17.304  1.00  0.00           N  \\nATOM   7180  CA  GLU B 458      25.131   0.969  16.948  1.00  0.00           C  \\nATOM   7181  C   GLU B 458      23.688   0.373  16.971  1.00  0.00           C  \\nATOM   7182  O   GLU B 458      22.765   1.025  17.468  1.00  0.00           O  \\nATOM   7183  CB  GLU B 458      25.467   1.556  15.562  1.00  0.00           C  \\nATOM   7184  CG  GLU B 458      26.823   2.258  15.501  1.00  0.00           C  \\nATOM   7185  CD  GLU B 458      27.283   2.687  14.142  1.00  0.00           C  \\nATOM   7186  OE1 GLU B 458      26.600   2.319  13.158  1.00  0.00           O  \\nATOM   7187  OE2 GLU B 458      28.318   3.313  14.022  1.00  0.00           O  \\nATOM   7188  H   GLU B 458      26.900  -0.226  16.697  1.00  0.00           H  \\nATOM   7189  HA  GLU B 458      25.184   1.765  17.691  1.00  0.00           H  \\nATOM   7190 1HB  GLU B 458      25.463   0.759  14.819  1.00  0.00           H  \\nATOM   7191 2HB  GLU B 458      24.699   2.275  15.275  1.00  0.00           H  \\nATOM   7192 1HG  GLU B 458      26.787   3.151  16.125  1.00  0.00           H  \\nATOM   7193 2HG  GLU B 458      27.583   1.594  15.910  1.00  0.00           H  \\nATOM   7194  N   ASN B 459      23.541  -0.764  16.325  1.00  0.00           N  \\nATOM   7195  CA  ASN B 459      22.212  -1.307  16.168  1.00  0.00           C  \\nATOM   7196  C   ASN B 459      21.687  -1.796  17.539  1.00  0.00           C  \\nATOM   7197  O   ASN B 459      20.456  -1.858  17.694  1.00  0.00           O  \\nATOM   7198  CB  ASN B 459      22.269  -2.502  15.213  1.00  0.00           C  \\nATOM   7199  CG  ASN B 459      22.494  -2.106  13.752  1.00  0.00           C  \\nATOM   7200  OD1 ASN B 459      22.321  -0.930  13.376  1.00  0.00           O  \\nATOM   7201  ND2 ASN B 459      22.844  -3.102  12.942  1.00  0.00           N  \\nATOM   7202  H   ASN B 459      24.327  -1.266  15.937  1.00  0.00           H  \\nATOM   7203  HA  ASN B 459      21.570  -0.534  15.741  1.00  0.00           H  \\nATOM   7204 1HB  ASN B 459      23.075  -3.171  15.515  1.00  0.00           H  \\nATOM   7205 2HB  ASN B 459      21.337  -3.063  15.276  1.00  0.00           H  \\nATOM   7206 1HD2 ASN B 459      23.009  -2.923  11.972  1.00  0.00           H  \\nATOM   7207 2HD2 ASN B 459      22.942  -4.029  13.303  1.00  0.00           H  \\nATOM   7208  N   SER B 460      22.598  -1.999  18.546  1.00  0.00           N  \\nATOM   7209  CA  SER B 460      22.152  -2.571  19.812  1.00  0.00           C  \\nATOM   7210  C   SER B 460      22.089  -1.579  20.933  1.00  0.00           C  \\nATOM   7211  O   SER B 460      21.704  -1.933  22.061  1.00  0.00           O  \\nATOM   7212  CB  SER B 460      23.045  -3.703  20.257  1.00  0.00           C  \\nATOM   7213  OG  SER B 460      23.148  -4.743  19.185  1.00  0.00           O  \\nATOM   7214  H   SER B 460      23.573  -1.761  18.430  1.00  0.00           H  \\nATOM   7215  HA  SER B 460      21.145  -2.968  19.677  1.00  0.00           H  \\nATOM   7216 1HB  SER B 460      24.035  -3.314  20.493  1.00  0.00           H  \\nATOM   7217 2HB  SER B 460      22.643  -4.147  21.167  1.00  0.00           H  \\nATOM   7218  HG  SER B 460      22.607  -4.428  18.457  1.00  0.00           H  \\nATOM   7219  N   TRP B 461      22.328  -0.304  20.638  1.00  0.00           N  \\nATOM   7220  CA  TRP B 461      22.434   0.610  21.788  1.00  0.00           C  \\nATOM   7221  C   TRP B 461      21.216   0.668  22.655  1.00  0.00           C  \\nATOM   7222  O   TRP B 461      21.282   0.722  23.889  1.00  0.00           O  \\nATOM   7223  CB  TRP B 461      22.720   1.980  21.232  1.00  0.00           C  \\nATOM   7224  CG  TRP B 461      22.945   3.057  22.249  1.00  0.00           C  \\nATOM   7225  CD1 TRP B 461      24.048   3.175  23.121  1.00  0.00           C  \\nATOM   7226  CD2 TRP B 461      22.048   4.127  22.621  1.00  0.00           C  \\nATOM   7227  NE1 TRP B 461      23.833   4.239  23.992  1.00  0.00           N  \\nATOM   7228  CE2 TRP B 461      22.668   4.891  23.646  1.00  0.00           C  \\nATOM   7229  CE3 TRP B 461      20.819   4.580  22.076  1.00  0.00           C  \\nATOM   7230  CZ2 TRP B 461      22.044   6.040  24.224  1.00  0.00           C  \\nATOM   7231  CZ3 TRP B 461      20.191   5.742  22.673  1.00  0.00           C  \\nATOM   7232  CH2 TRP B 461      20.798   6.430  23.704  1.00  0.00           C  \\nATOM   7233  H   TRP B 461      22.438   0.065  19.704  1.00  0.00           H  \\nATOM   7234  HA  TRP B 461      23.258   0.277  22.419  1.00  0.00           H  \\nATOM   7235 1HB  TRP B 461      23.609   1.939  20.603  1.00  0.00           H  \\nATOM   7236 2HB  TRP B 461      21.889   2.298  20.603  1.00  0.00           H  \\nATOM   7237  HD1 TRP B 461      24.925   2.530  23.107  1.00  0.00           H  \\nATOM   7238  HE1 TRP B 461      24.433   4.500  24.762  1.00  0.00           H  \\nATOM   7239  HE3 TRP B 461      20.358   4.072  21.230  1.00  0.00           H  \\nATOM   7240  HZ2 TRP B 461      22.505   6.603  25.036  1.00  0.00           H  \\nATOM   7241  HZ3 TRP B 461      19.223   6.060  22.285  1.00  0.00           H  \\nATOM   7242  HH2 TRP B 461      20.291   7.300  24.123  1.00  0.00           H  \\nATOM   7243  N   GLY B 462      20.015   0.779  22.028  1.00  0.00           N  \\nATOM   7244  CA  GLY B 462      18.711   0.797  22.731  1.00  0.00           C  \\nATOM   7245  C   GLY B 462      18.542  -0.452  23.594  1.00  0.00           C  \\nATOM   7246  O   GLY B 462      18.174  -0.320  24.765  1.00  0.00           O  \\nATOM   7247  H   GLY B 462      20.032   0.853  21.021  1.00  0.00           H  \\nATOM   7248 1HA  GLY B 462      18.645   1.690  23.353  1.00  0.00           H  \\nATOM   7249 2HA  GLY B 462      17.905   0.855  22.001  1.00  0.00           H  \\nATOM   7250  N   ARG B 463      18.822  -1.576  22.985  1.00  0.00           N  \\nATOM   7251  CA  ARG B 463      18.654  -2.872  23.679  1.00  0.00           C  \\nATOM   7252  C   ARG B 463      19.670  -3.007  24.794  1.00  0.00           C  \\nATOM   7253  O   ARG B 463      19.341  -3.595  25.863  1.00  0.00           O  \\nATOM   7254  CB  ARG B 463      18.682  -4.055  22.725  1.00  0.00           C  \\nATOM   7255  CG  ARG B 463      17.313  -4.345  22.242  1.00  0.00           C  \\nATOM   7256  CD  ARG B 463      17.166  -4.867  20.815  1.00  0.00           C  \\nATOM   7257  NE  ARG B 463      18.423  -5.209  20.129  1.00  0.00           N  \\nATOM   7258  CZ  ARG B 463      19.053  -6.347  20.305  1.00  0.00           C  \\nATOM   7259  NH1 ARG B 463      18.566  -7.274  21.164  1.00  0.00           N  \\nATOM   7260  NH2 ARG B 463      20.184  -6.543  19.645  1.00  0.00           N  \\nATOM   7261  H   ARG B 463      19.157  -1.568  22.032  1.00  0.00           H  \\nATOM   7262  HA  ARG B 463      17.684  -2.874  24.178  1.00  0.00           H  \\nATOM   7263 1HB  ARG B 463      19.338  -3.829  21.885  1.00  0.00           H  \\nATOM   7264 2HB  ARG B 463      19.093  -4.925  23.237  1.00  0.00           H  \\nATOM   7265 1HG  ARG B 463      16.857  -5.106  22.876  1.00  0.00           H  \\nATOM   7266 2HG  ARG B 463      16.713  -3.435  22.279  1.00  0.00           H  \\nATOM   7267 1HD  ARG B 463      16.564  -5.775  20.820  1.00  0.00           H  \\nATOM   7268 2HD  ARG B 463      16.678  -4.112  20.200  1.00  0.00           H  \\nATOM   7269  HE  ARG B 463      18.812  -4.526  19.492  1.00  0.00           H  \\nATOM   7270 1HH1 ARG B 463      17.713  -7.092  21.674  1.00  0.00           H  \\nATOM   7271 2HH1 ARG B 463      19.055  -8.147  21.295  1.00  0.00           H  \\nATOM   7272 1HH2 ARG B 463      20.539  -5.826  19.027  1.00  0.00           H  \\nATOM   7273 2HH2 ARG B 463      20.692  -7.408  19.759  1.00  0.00           H  \\nATOM   7274  N   LEU B 464      20.899  -2.529  24.590  1.00  0.00           N  \\nATOM   7275  CA  LEU B 464      21.869  -2.573  25.675  1.00  0.00           C  \\nATOM   7276  C   LEU B 464      21.427  -1.703  26.830  1.00  0.00           C  \\nATOM   7277  O   LEU B 464      21.697  -2.048  28.029  1.00  0.00           O  \\nATOM   7278  CB  LEU B 464      23.252  -2.144  25.220  1.00  0.00           C  \\nATOM   7279  CG  LEU B 464      23.894  -3.062  24.202  1.00  0.00           C  \\nATOM   7280  CD1 LEU B 464      25.115  -2.550  23.409  1.00  0.00           C  \\nATOM   7281  CD2 LEU B 464      24.249  -4.404  24.943  1.00  0.00           C  \\nATOM   7282  H   LEU B 464      21.173  -2.137  23.701  1.00  0.00           H  \\nATOM   7283  HA  LEU B 464      21.938  -3.599  26.035  1.00  0.00           H  \\nATOM   7284 1HB  LEU B 464      23.181  -1.148  24.785  1.00  0.00           H  \\nATOM   7285 2HB  LEU B 464      23.904  -2.091  26.092  1.00  0.00           H  \\nATOM   7286  HG  LEU B 464      23.194  -3.248  23.387  1.00  0.00           H  \\nATOM   7287 1HD1 LEU B 464      25.455  -3.326  22.723  1.00  0.00           H  \\nATOM   7288 2HD1 LEU B 464      24.834  -1.663  22.841  1.00  0.00           H  \\nATOM   7289 3HD1 LEU B 464      25.919  -2.300  24.100  1.00  0.00           H  \\nATOM   7290 1HD2 LEU B 464      24.715  -5.096  24.240  1.00  0.00           H  \\nATOM   7291 2HD2 LEU B 464      24.940  -4.197  25.760  1.00  0.00           H  \\nATOM   7292 3HD2 LEU B 464      23.338  -4.851  25.342  1.00  0.00           H  \\nATOM   7293  N   SER B 465      20.970  -0.505  26.542  1.00  0.00           N  \\nATOM   7294  CA  SER B 465      20.489   0.350  27.548  1.00  0.00           C  \\nATOM   7295  C   SER B 465      19.436  -0.324  28.451  1.00  0.00           C  \\nATOM   7296  O   SER B 465      19.442  -0.170  29.679  1.00  0.00           O  \\nATOM   7297  CB  SER B 465      19.898   1.638  26.885  1.00  0.00           C  \\nATOM   7298  OG  SER B 465      20.943   2.426  26.260  1.00  0.00           O  \\nATOM   7299  H   SER B 465      20.962  -0.189  25.583  1.00  0.00           H  \\nATOM   7300  HA  SER B 465      21.323   0.627  28.195  1.00  0.00           H  \\nATOM   7301 1HB  SER B 465      19.156   1.355  26.139  1.00  0.00           H  \\nATOM   7302 2HB  SER B 465      19.390   2.236  27.641  1.00  0.00           H  \\nATOM   7303  HG  SER B 465      21.762   1.948  26.412  1.00  0.00           H  \\nATOM   7304  N   THR B 466      18.463  -0.967  27.815  1.00  0.00           N  \\nATOM   7305  CA  THR B 466      17.430  -1.746  28.544  1.00  0.00           C  \\nATOM   7306  C   THR B 466      18.063  -2.909  29.316  1.00  0.00           C  \\nATOM   7307  O   THR B 466      17.719  -2.954  30.557  1.00  0.00           O  \\nATOM   7308  CB  THR B 466      16.403  -2.273  27.550  1.00  0.00           C  \\nATOM   7309  OG1 THR B 466      15.832  -1.134  26.886  1.00  0.00           O  \\nATOM   7310  CG2 THR B 466      15.385  -3.076  28.207  1.00  0.00           C  \\nATOM   7311  H   THR B 466      18.426  -0.924  26.807  1.00  0.00           H  \\nATOM   7312  HA  THR B 466      16.935  -1.084  29.255  1.00  0.00           H  \\nATOM   7313  HB  THR B 466      16.903  -2.889  26.803  1.00  0.00           H  \\nATOM   7314  HG1 THR B 466      16.233  -0.331  27.228  1.00  0.00           H  \\nATOM   7315 1HG2 THR B 466      14.671  -3.435  27.466  1.00  0.00           H  \\nATOM   7316 2HG2 THR B 466      15.855  -3.927  28.700  1.00  0.00           H  \\nATOM   7317 3HG2 THR B 466      14.865  -2.469  28.948  1.00  0.00           H  \\nATOM   7318  N   ALA B 467      18.953  -3.687  28.651  1.00  0.00           N  \\nATOM   7319  CA  ALA B 467      19.540  -4.902  29.313  1.00  0.00           C  \\nATOM   7320  C   ALA B 467      20.262  -4.403  30.579  1.00  0.00           C  \\nATOM   7321  O   ALA B 467      20.103  -5.009  31.672  1.00  0.00           O  \\nATOM   7322  CB  ALA B 467      20.458  -5.598  28.480  1.00  0.00           C  \\nATOM   7323  H   ALA B 467      19.230  -3.459  27.707  1.00  0.00           H  \\nATOM   7324  HA  ALA B 467      18.725  -5.586  29.550  1.00  0.00           H  \\nATOM   7325 1HB  ALA B 467      20.848  -6.465  29.013  1.00  0.00           H  \\nATOM   7326 2HB  ALA B 467      19.949  -5.928  27.574  1.00  0.00           H  \\nATOM   7327 3HB  ALA B 467      21.281  -4.936  28.214  1.00  0.00           H  \\nATOM   7328  N   ILE B 468      20.979  -3.270  30.520  1.00  0.00           N  \\nATOM   7329  CA  ILE B 468      21.725  -2.852  31.716  1.00  0.00           C  \\nATOM   7330  C   ILE B 468      20.858  -2.299  32.792  1.00  0.00           C  \\nATOM   7331  O   ILE B 468      20.985  -2.670  33.953  1.00  0.00           O  \\nATOM   7332  CB  ILE B 468      22.780  -1.786  31.248  1.00  0.00           C  \\nATOM   7333  CG1 ILE B 468      23.929  -2.437  30.435  1.00  0.00           C  \\nATOM   7334  CG2 ILE B 468      23.527  -1.116  32.419  1.00  0.00           C  \\nATOM   7335  CD1 ILE B 468      24.783  -1.488  29.614  1.00  0.00           C  \\nATOM   7336  H   ILE B 468      21.023  -2.699  29.688  1.00  0.00           H  \\nATOM   7337  HA  ILE B 468      22.227  -3.724  32.132  1.00  0.00           H  \\nATOM   7338  HB  ILE B 468      22.278  -1.000  30.686  1.00  0.00           H  \\nATOM   7339 1HG1 ILE B 468      24.599  -2.967  31.112  1.00  0.00           H  \\nATOM   7340 2HG1 ILE B 468      23.515  -3.172  29.744  1.00  0.00           H  \\nATOM   7341 1HG2 ILE B 468      24.241  -0.391  32.028  1.00  0.00           H  \\nATOM   7342 2HG2 ILE B 468      22.811  -0.608  33.064  1.00  0.00           H  \\nATOM   7343 3HG2 ILE B 468      24.058  -1.875  32.994  1.00  0.00           H  \\nATOM   7344 1HD1 ILE B 468      25.553  -2.053  29.088  1.00  0.00           H  \\nATOM   7345 2HD1 ILE B 468      24.156  -0.968  28.889  1.00  0.00           H  \\nATOM   7346 3HD1 ILE B 468      25.255  -0.761  30.273  1.00  0.00           H  \\nATOM   7347  N   GLN B 469      19.893  -1.400  32.499  1.00  0.00           N  \\nATOM   7348  CA  GLN B 469      18.997  -0.876  33.472  1.00  0.00           C  \\nATOM   7349  C   GLN B 469      17.992  -1.822  34.078  1.00  0.00           C  \\nATOM   7350  O   GLN B 469      17.630  -1.638  35.250  1.00  0.00           O  \\nATOM   7351  CB  GLN B 469      18.312   0.351  32.812  1.00  0.00           C  \\nATOM   7352  CG  GLN B 469      19.269   1.585  32.745  1.00  0.00           C  \\nATOM   7353  CD  GLN B 469      18.652   2.679  31.937  1.00  0.00           C  \\nATOM   7354  OE1 GLN B 469      17.711   3.336  32.385  1.00  0.00           O  \\nATOM   7355  NE2 GLN B 469      19.235   2.914  30.770  1.00  0.00           N  \\nATOM   7356  H   GLN B 469      19.813  -1.092  31.540  1.00  0.00           H  \\nATOM   7357  HA  GLN B 469      19.574  -0.572  34.345  1.00  0.00           H  \\nATOM   7358 1HB  GLN B 469      17.992   0.090  31.803  1.00  0.00           H  \\nATOM   7359 2HB  GLN B 469      17.421   0.619  33.379  1.00  0.00           H  \\nATOM   7360 1HG  GLN B 469      19.453   1.945  33.757  1.00  0.00           H  \\nATOM   7361 2HG  GLN B 469      20.207   1.280  32.281  1.00  0.00           H  \\nATOM   7362 1HE2 GLN B 469      18.881   3.632  30.169  1.00  0.00           H  \\nATOM   7363 2HE2 GLN B 469      20.029   2.374  30.489  1.00  0.00           H  \\nATOM   7364  N   GLU B 470      17.575  -2.835  33.339  1.00  0.00           N  \\nATOM   7365  CA  GLU B 470      16.682  -3.855  33.968  1.00  0.00           C  \\nATOM   7366  C   GLU B 470      17.461  -5.039  34.501  1.00  0.00           C  \\nATOM   7367  O   GLU B 470      16.846  -6.036  34.910  1.00  0.00           O  \\nATOM   7368  CB  GLU B 470      15.665  -4.302  32.984  1.00  0.00           C  \\nATOM   7369  CG  GLU B 470      14.826  -3.084  32.507  1.00  0.00           C  \\nATOM   7370  CD  GLU B 470      13.404  -3.455  32.289  1.00  0.00           C  \\nATOM   7371  OE1 GLU B 470      12.891  -4.354  33.022  1.00  0.00           O  \\nATOM   7372  OE2 GLU B 470      12.798  -2.841  31.376  1.00  0.00           O  \\nATOM   7373  H   GLU B 470      17.838  -2.942  32.370  1.00  0.00           H  \\nATOM   7374  HA  GLU B 470      16.177  -3.395  34.818  1.00  0.00           H  \\nATOM   7375 1HB  GLU B 470      16.162  -4.774  32.136  1.00  0.00           H  \\nATOM   7376 2HB  GLU B 470      15.019  -5.050  33.443  1.00  0.00           H  \\nATOM   7377 1HG  GLU B 470      14.887  -2.297  33.259  1.00  0.00           H  \\nATOM   7378 2HG  GLU B 470      15.256  -2.700  31.583  1.00  0.00           H  \\nATOM   7379  N   SER B 471      18.782  -4.990  34.472  1.00  0.00           N  \\nATOM   7380  CA  SER B 471      19.592  -6.135  34.963  1.00  0.00           C  \\nATOM   7381  C   SER B 471      19.389  -6.398  36.437  1.00  0.00           C  \\nATOM   7382  O   SER B 471      18.915  -5.575  37.186  1.00  0.00           O  \\nATOM   7383  CB  SER B 471      21.068  -5.903  34.631  1.00  0.00           C  \\nATOM   7384  OG  SER B 471      21.654  -4.976  35.459  1.00  0.00           O  \\nATOM   7385  H   SER B 471      19.253  -4.171  34.115  1.00  0.00           H  \\nATOM   7386  HA  SER B 471      19.252  -7.041  34.459  1.00  0.00           H  \\nATOM   7387 1HB  SER B 471      21.612  -6.844  34.714  1.00  0.00           H  \\nATOM   7388 2HB  SER B 471      21.160  -5.563  33.600  1.00  0.00           H  \\nATOM   7389  HG  SER B 471      20.966  -4.691  36.066  1.00  0.00           H  \\nATOM   7390  N   ASN B 472      19.766  -7.645  36.767  1.00  0.00           N  \\nATOM   7391  CA  ASN B 472      19.707  -8.085  38.188  1.00  0.00           C  \\nATOM   7392  C   ASN B 472      21.050  -7.818  38.899  1.00  0.00           C  \\nATOM   7393  O   ASN B 472      21.958  -8.621  38.755  1.00  0.00           O  \\nATOM   7394  CB  ASN B 472      19.372  -9.550  38.208  1.00  0.00           C  \\nATOM   7395  CG  ASN B 472      19.458 -10.150  39.572  1.00  0.00           C  \\nATOM   7396  OD1 ASN B 472      19.330  -9.480  40.579  1.00  0.00           O  \\nATOM   7397  ND2 ASN B 472      19.639 -11.482  39.553  1.00  0.00           N  \\nATOM   7398  H   ASN B 472      20.091  -8.294  36.064  1.00  0.00           H  \\nATOM   7399  HA  ASN B 472      18.923  -7.519  38.693  1.00  0.00           H  \\nATOM   7400 1HB  ASN B 472      18.361  -9.698  37.826  1.00  0.00           H  \\nATOM   7401 2HB  ASN B 472      20.053 -10.088  37.548  1.00  0.00           H  \\nATOM   7402 1HD2 ASN B 472      19.711 -11.989  40.412  1.00  0.00           H  \\nATOM   7403 2HD2 ASN B 472      19.702 -11.965  38.680  1.00  0.00           H  \\nATOM   7404  N   GLN B 473      21.217  -6.591  39.387  1.00  0.00           N  \\nATOM   7405  CA  GLN B 473      22.479  -6.156  39.945  1.00  0.00           C  \\nATOM   7406  C   GLN B 473      23.620  -6.366  38.984  1.00  0.00           C  \\nATOM   7407  O   GLN B 473      24.697  -6.807  39.278  1.00  0.00           O  \\nATOM   7408  CB  GLN B 473      22.753  -6.773  41.351  1.00  0.00           C  \\nATOM   7409  CG  GLN B 473      21.605  -6.582  42.293  1.00  0.00           C  \\nATOM   7410  CD  GLN B 473      21.974  -6.750  43.736  1.00  0.00           C  \\nATOM   7411  OE1 GLN B 473      21.442  -7.695  44.416  1.00  0.00           O  \\nATOM   7412  NE2 GLN B 473      22.967  -6.009  44.175  1.00  0.00           N  \\nATOM   7413  H   GLN B 473      20.440  -5.946  39.367  1.00  0.00           H  \\nATOM   7414  HA  GLN B 473      22.452  -5.072  40.059  1.00  0.00           H  \\nATOM   7415 1HB  GLN B 473      22.951  -7.840  41.248  1.00  0.00           H  \\nATOM   7416 2HB  GLN B 473      23.643  -6.315  41.782  1.00  0.00           H  \\nATOM   7417 1HG  GLN B 473      21.210  -5.574  42.169  1.00  0.00           H  \\nATOM   7418 2HG  GLN B 473      20.833  -7.316  42.062  1.00  0.00           H  \\nATOM   7419 1HE2 GLN B 473      23.259  -6.077  45.130  1.00  0.00           H  \\nATOM   7420 2HE2 GLN B 473      23.432  -5.376  43.555  1.00  0.00           H  \\nATOM   7421  N   GLY B 474      23.395  -6.059  37.684  1.00  0.00           N  \\nATOM   7422  CA  GLY B 474      24.420  -6.181  36.666  1.00  0.00           C  \\nATOM   7423  C   GLY B 474      24.351  -7.402  35.821  1.00  0.00           C  \\nATOM   7424  O   GLY B 474      24.914  -7.454  34.702  1.00  0.00           O  \\nATOM   7425  H   GLY B 474      22.476  -5.733  37.420  1.00  0.00           H  \\nATOM   7426 1HA  GLY B 474      24.373  -5.322  35.997  1.00  0.00           H  \\nATOM   7427 2HA  GLY B 474      25.403  -6.168  37.136  1.00  0.00           H  \\nATOM   7428  N   ALA B 475      23.601  -8.427  36.273  1.00  0.00           N  \\nATOM   7429  CA  ALA B 475      23.477  -9.658  35.560  1.00  0.00           C  \\nATOM   7430  C   ALA B 475      22.254  -9.611  34.653  1.00  0.00           C  \\nATOM   7431  O   ALA B 475      21.165  -9.274  35.061  1.00  0.00           O  \\nATOM   7432  CB  ALA B 475      23.357 -10.829  36.602  1.00  0.00           C  \\nATOM   7433  H   ALA B 475      23.111  -8.314  37.149  1.00  0.00           H  \\nATOM   7434  HA  ALA B 475      24.375  -9.792  34.957  1.00  0.00           H  \\nATOM   7435 1HB  ALA B 475      23.261 -11.778  36.074  1.00  0.00           H  \\nATOM   7436 2HB  ALA B 475      24.248 -10.851  37.229  1.00  0.00           H  \\nATOM   7437 3HB  ALA B 475      22.479 -10.672  37.226  1.00  0.00           H  \\nATOM   7438  N   PHE B 476      22.473  -9.923  33.393  1.00  0.00           N  \\nATOM   7439  CA  PHE B 476      21.389  -9.736  32.426  1.00  0.00           C  \\nATOM   7440  C   PHE B 476      20.422 -10.872  32.540  1.00  0.00           C  \\nATOM   7441  O   PHE B 476      20.837 -12.027  32.622  1.00  0.00           O  \\nATOM   7442  CB  PHE B 476      21.919  -9.697  30.992  1.00  0.00           C  \\nATOM   7443  CG  PHE B 476      22.767  -8.496  30.672  1.00  0.00           C  \\nATOM   7444  CD1 PHE B 476      22.866  -7.382  31.563  1.00  0.00           C  \\nATOM   7445  CD2 PHE B 476      23.447  -8.489  29.451  1.00  0.00           C  \\nATOM   7446  CE1 PHE B 476      23.664  -6.268  31.181  1.00  0.00           C  \\nATOM   7447  CE2 PHE B 476      24.244  -7.391  29.101  1.00  0.00           C  \\nATOM   7448  CZ  PHE B 476      24.296  -6.286  29.986  1.00  0.00           C  \\nATOM   7449  H   PHE B 476      23.360 -10.285  33.074  1.00  0.00           H  \\nATOM   7450  HA  PHE B 476      20.900  -8.783  32.634  1.00  0.00           H  \\nATOM   7451 1HB  PHE B 476      22.517 -10.587  30.801  1.00  0.00           H  \\nATOM   7452 2HB  PHE B 476      21.082  -9.709  30.295  1.00  0.00           H  \\nATOM   7453  HD1 PHE B 476      22.331  -7.410  32.512  1.00  0.00           H  \\nATOM   7454  HD2 PHE B 476      23.388  -9.353  28.788  1.00  0.00           H  \\nATOM   7455  HE1 PHE B 476      23.761  -5.413  31.849  1.00  0.00           H  \\nATOM   7456  HE2 PHE B 476      24.815  -7.373  28.173  1.00  0.00           H  \\nATOM   7457  HZ  PHE B 476      24.878  -5.424  29.663  1.00  0.00           H  \\nATOM   7458  N   ALA B 477      19.144 -10.522  32.432  1.00  0.00           N  \\nATOM   7459  CA  ALA B 477      18.092 -11.548  32.415  1.00  0.00           C  \\nATOM   7460  C   ALA B 477      18.195 -12.470  31.181  1.00  0.00           C  \\nATOM   7461  O   ALA B 477      17.947 -13.668  31.257  1.00  0.00           O  \\nATOM   7462  CB  ALA B 477      16.773 -10.853  32.444  1.00  0.00           C  \\nATOM   7463  H   ALA B 477      18.883  -9.549  32.361  1.00  0.00           H  \\nATOM   7464  HA  ALA B 477      18.208 -12.167  33.305  1.00  0.00           H  \\nATOM   7465 1HB  ALA B 477      15.972 -11.592  32.432  1.00  0.00           H  \\nATOM   7466 2HB  ALA B 477      16.698 -10.252  33.350  1.00  0.00           H  \\nATOM   7467 3HB  ALA B 477      16.683 -10.207  31.572  1.00  0.00           H  \\nATOM   7468  N   SER B 478      18.598 -11.923  30.040  1.00  0.00           N  \\nATOM   7469  CA  SER B 478      18.890 -12.758  28.874  1.00  0.00           C  \\nATOM   7470  C   SER B 478      20.090 -12.148  28.157  1.00  0.00           C  \\nATOM   7471  O   SER B 478      20.315 -10.935  28.278  1.00  0.00           O  \\nATOM   7472  CB  SER B 478      17.657 -12.817  27.942  1.00  0.00           C  \\nATOM   7473  OG  SER B 478      16.869 -11.579  28.033  1.00  0.00           O  \\nATOM   7474  H   SER B 478      18.707 -10.921  29.968  1.00  0.00           H  \\nATOM   7475  HA  SER B 478      19.119 -13.767  29.220  1.00  0.00           H  \\nATOM   7476 1HB  SER B 478      17.985 -12.968  26.914  1.00  0.00           H  \\nATOM   7477 2HB  SER B 478      17.035 -13.668  28.215  1.00  0.00           H  \\nATOM   7478  HG  SER B 478      17.326 -11.021  28.667  1.00  0.00           H  \\nATOM   7479  N   PRO B 479      20.875 -12.987  27.453  1.00  0.00           N  \\nATOM   7480  CA  PRO B 479      22.074 -12.540  26.772  1.00  0.00           C  \\nATOM   7481  C   PRO B 479      21.758 -11.573  25.617  1.00  0.00           C  \\nATOM   7482  O   PRO B 479      20.656 -11.565  25.103  1.00  0.00           O  \\nATOM   7483  CB  PRO B 479      22.713 -13.829  26.256  1.00  0.00           C  \\nATOM   7484  CG  PRO B 479      21.621 -14.932  26.470  1.00  0.00           C  \\nATOM   7485  CD  PRO B 479      20.796 -14.465  27.581  1.00  0.00           C  \\nATOM   7486  HA  PRO B 479      22.738 -12.045  27.496  1.00  0.00           H  \\nATOM   7487 1HB  PRO B 479      22.993 -13.713  25.199  1.00  0.00           H  \\nATOM   7488 2HB  PRO B 479      23.638 -14.039  26.812  1.00  0.00           H  \\nATOM   7489 1HG  PRO B 479      21.033 -15.068  25.550  1.00  0.00           H  \\nATOM   7490 2HG  PRO B 479      22.096 -15.900  26.689  1.00  0.00           H  \\nATOM   7491 1HD  PRO B 479      19.767 -14.832  27.453  1.00  0.00           H  \\nATOM   7492 2HD  PRO B 479      21.221 -14.824  28.530  1.00  0.00           H  \\nATOM   7493  N   ILE B 480      22.705 -10.724  25.240  1.00  0.00           N  \\nATOM   7494  CA  ILE B 480      22.485  -9.921  24.044  1.00  0.00           C  \\nATOM   7495  C   ILE B 480      23.660 -10.267  23.185  1.00  0.00           C  \\nATOM   7496  O   ILE B 480      24.771 -10.372  23.656  1.00  0.00           O  \\nATOM   7497  CB  ILE B 480      22.360  -8.397  24.308  1.00  0.00           C  \\nATOM   7498  CG1 ILE B 480      21.023  -8.160  25.041  1.00  0.00           C  \\nATOM   7499  CG2 ILE B 480      22.513  -7.572  22.949  1.00  0.00           C  \\nATOM   7500  CD1 ILE B 480      20.519  -6.737  25.171  1.00  0.00           C  \\nATOM   7501  H   ILE B 480      23.569 -10.617  25.752  1.00  0.00           H  \\nATOM   7502  HA  ILE B 480      21.550 -10.237  23.583  1.00  0.00           H  \\nATOM   7503  HB  ILE B 480      23.141  -8.082  24.999  1.00  0.00           H  \\nATOM   7504 1HG1 ILE B 480      20.230  -8.712  24.538  1.00  0.00           H  \\nATOM   7505 2HG1 ILE B 480      21.093  -8.544  26.059  1.00  0.00           H  \\nATOM   7506 1HG2 ILE B 480      22.421  -6.506  23.158  1.00  0.00           H  \\nATOM   7507 2HG2 ILE B 480      23.489  -7.773  22.510  1.00  0.00           H  \\nATOM   7508 3HG2 ILE B 480      21.732  -7.873  22.251  1.00  0.00           H  \\nATOM   7509 1HD1 ILE B 480      19.571  -6.734  25.709  1.00  0.00           H  \\nATOM   7510 2HD1 ILE B 480      21.249  -6.141  25.719  1.00  0.00           H  \\nATOM   7511 3HD1 ILE B 480      20.373  -6.311  24.179  1.00  0.00           H  \\nATOM   7512  N   GLN B 481      23.339 -10.577  21.946  1.00  0.00           N  \\nATOM   7513  CA  GLN B 481      24.317 -10.920  20.943  1.00  0.00           C  \\nATOM   7514  C   GLN B 481      24.931  -9.726  20.309  1.00  0.00           C  \\nATOM   7515  O   GLN B 481      24.257  -8.760  19.885  1.00  0.00           O  \\nATOM   7516  CB  GLN B 481      23.669 -11.773  19.878  1.00  0.00           C  \\nATOM   7517  CG  GLN B 481      24.675 -12.445  19.038  1.00  0.00           C  \\nATOM   7518  CD  GLN B 481      23.978 -13.174  18.017  1.00  0.00           C  \\nATOM   7519  OE1 GLN B 481      23.588 -12.600  16.978  1.00  0.00           O  \\nATOM   7520  NE2 GLN B 481      23.812 -14.462  18.265  1.00  0.00           N  \\nATOM   7521  H   GLN B 481      22.360 -10.571  21.697  1.00  0.00           H  \\nATOM   7522  HA  GLN B 481      25.116 -11.489  21.419  1.00  0.00           H  \\nATOM   7523 1HB  GLN B 481      23.032 -12.522  20.349  1.00  0.00           H  \\nATOM   7524 2HB  GLN B 481      23.032 -11.150  19.251  1.00  0.00           H  \\nATOM   7525 1HG  GLN B 481      25.325 -11.691  18.594  1.00  0.00           H  \\nATOM   7526 2HG  GLN B 481      25.260 -13.122  19.661  1.00  0.00           H  \\nATOM   7527 1HE2 GLN B 481      23.335 -15.042  17.603  1.00  0.00           H  \\nATOM   7528 2HE2 GLN B 481      24.162 -14.857  19.114  1.00  0.00           H  \\nATOM   7529  N   LEU B 482      26.250  -9.785  20.313  1.00  0.00           N  \\nATOM   7530  CA  LEU B 482      27.029  -8.768  19.591  1.00  0.00           C  \\nATOM   7531  C   LEU B 482      27.906  -9.475  18.588  1.00  0.00           C  \\nATOM   7532  O   LEU B 482      27.859 -10.743  18.493  1.00  0.00           O  \\nATOM   7533  CB  LEU B 482      27.817  -7.865  20.550  1.00  0.00           C  \\nATOM   7534  CG  LEU B 482      27.053  -6.981  21.527  1.00  0.00           C  \\nATOM   7535  CD1 LEU B 482      28.126  -6.434  22.519  1.00  0.00           C  \\nATOM   7536  CD2 LEU B 482      26.216  -5.931  20.857  1.00  0.00           C  \\nATOM   7537  H   LEU B 482      26.737 -10.520  20.806  1.00  0.00           H  \\nATOM   7538  HA  LEU B 482      26.340  -8.140  19.028  1.00  0.00           H  \\nATOM   7539 1HB  LEU B 482      28.469  -8.490  21.159  1.00  0.00           H  \\nATOM   7540 2HB  LEU B 482      28.440  -7.192  19.961  1.00  0.00           H  \\nATOM   7541  HG  LEU B 482      26.383  -7.597  22.128  1.00  0.00           H  \\nATOM   7542 1HD1 LEU B 482      27.646  -5.787  23.254  1.00  0.00           H  \\nATOM   7543 2HD1 LEU B 482      28.608  -7.268  23.030  1.00  0.00           H  \\nATOM   7544 3HD1 LEU B 482      28.874  -5.864  21.968  1.00  0.00           H  \\nATOM   7545 1HD2 LEU B 482      25.700  -5.339  21.614  1.00  0.00           H  \\nATOM   7546 2HD2 LEU B 482      26.857  -5.280  20.262  1.00  0.00           H  \\nATOM   7547 3HD2 LEU B 482      25.483  -6.410  20.208  1.00  0.00           H  \\nATOM   7548  N   GLN B 483      28.626  -8.713  17.780  1.00  0.00           N  \\nATOM   7549  CA  GLN B 483      29.621  -9.225  16.761  1.00  0.00           C  \\nATOM   7550  C   GLN B 483      31.036  -8.822  17.017  1.00  0.00           C  \\nATOM   7551  O   GLN B 483      31.367  -7.622  17.317  1.00  0.00           O  \\nATOM   7552  CB  GLN B 483      29.274  -8.771  15.312  1.00  0.00           C  \\nATOM   7553  CG  GLN B 483      28.119  -9.455  14.681  1.00  0.00           C  \\nATOM   7554  CD  GLN B 483      28.122  -8.996  13.197  1.00  0.00           C  \\nATOM   7555  OE1 GLN B 483      27.733  -7.888  12.893  1.00  0.00           O  \\nATOM   7556  NE2 GLN B 483      28.708  -9.738  12.372  1.00  0.00           N  \\nATOM   7557  H   GLN B 483      28.482  -7.717  17.870  1.00  0.00           H  \\nATOM   7558  HA  GLN B 483      29.605 -10.315  16.780  1.00  0.00           H  \\nATOM   7559 1HB  GLN B 483      29.056  -7.703  15.308  1.00  0.00           H  \\nATOM   7560 2HB  GLN B 483      30.136  -8.932  14.664  1.00  0.00           H  \\nATOM   7561 1HG  GLN B 483      28.256 -10.533  14.769  1.00  0.00           H  \\nATOM   7562 2HG  GLN B 483      27.206  -9.152  15.194  1.00  0.00           H  \\nATOM   7563 1HE2 GLN B 483      28.737  -9.485  11.404  1.00  0.00           H  \\nATOM   7564 2HE2 GLN B 483      29.146 -10.582  12.681  1.00  0.00           H  \\nATOM   7565  N   ARG B 484      31.985  -9.762  16.782  1.00  0.00           N  \\nATOM   7566  CA  ARG B 484      33.401  -9.333  16.873  1.00  0.00           C  \\nATOM   7567  C   ARG B 484      33.798  -8.623  15.609  1.00  0.00           C  \\nATOM   7568  O   ARG B 484      32.997  -8.599  14.620  1.00  0.00           O  \\nATOM   7569  CB  ARG B 484      34.348 -10.524  17.126  1.00  0.00           C  \\nATOM   7570  CG  ARG B 484      34.016 -11.343  18.349  1.00  0.00           C  \\nATOM   7571  CD  ARG B 484      34.842 -12.710  18.257  1.00  0.00           C  \\nATOM   7572  NE  ARG B 484      34.189 -13.883  18.903  1.00  0.00           N  \\nATOM   7573  CZ  ARG B 484      34.218 -14.034  20.214  1.00  0.00           C  \\nATOM   7574  NH1 ARG B 484      34.863 -13.121  20.930  1.00  0.00           N  \\nATOM   7575  NH2 ARG B 484      33.633 -15.043  20.820  1.00  0.00           N  \\nATOM   7576  H   ARG B 484      31.776 -10.723  16.552  1.00  0.00           H  \\nATOM   7577  HA  ARG B 484      33.500  -8.641  17.710  1.00  0.00           H  \\nATOM   7578 1HB  ARG B 484      34.331 -11.191  16.265  1.00  0.00           H  \\nATOM   7579 2HB  ARG B 484      35.369 -10.160  17.238  1.00  0.00           H  \\nATOM   7580 1HG  ARG B 484      34.298 -10.789  19.245  1.00  0.00           H  \\nATOM   7581 2HG  ARG B 484      32.945 -11.548  18.370  1.00  0.00           H  \\nATOM   7582 1HD  ARG B 484      34.993 -12.974  17.211  1.00  0.00           H  \\nATOM   7583 2HD  ARG B 484      35.809 -12.584  18.743  1.00  0.00           H  \\nATOM   7584  HE  ARG B 484      33.727 -14.553  18.303  1.00  0.00           H  \\nATOM   7585 1HH1 ARG B 484      35.312 -12.342  20.469  1.00  0.00           H  \\nATOM   7586 2HH1 ARG B 484      34.905 -13.205  21.935  1.00  0.00           H  \\nATOM   7587 1HH2 ARG B 484      33.138 -15.739  20.279  1.00  0.00           H  \\nATOM   7588 2HH2 ARG B 484      33.678 -15.123  21.825  1.00  0.00           H  \\nATOM   7589  N   ARG B 485      35.014  -8.091  15.509  1.00  0.00           N  \\nATOM   7590  CA  ARG B 485      35.459  -7.440  14.265  1.00  0.00           C  \\nATOM   7591  C   ARG B 485      35.425  -8.328  13.008  1.00  0.00           C  \\nATOM   7592  O   ARG B 485      35.196  -7.848  11.928  1.00  0.00           O  \\nATOM   7593  CB  ARG B 485      36.873  -6.874  14.375  1.00  0.00           C  \\nATOM   7594  CG  ARG B 485      36.948  -5.830  15.495  1.00  0.00           C  \\nATOM   7595  CD  ARG B 485      38.272  -5.145  15.519  1.00  0.00           C  \\nATOM   7596  NE  ARG B 485      38.500  -4.281  16.690  1.00  0.00           N  \\nATOM   7597  CZ  ARG B 485      38.243  -2.968  16.649  1.00  0.00           C  \\nATOM   7598  NH1 ARG B 485      37.675  -2.438  15.584  1.00  0.00           N  \\nATOM   7599  NH2 ARG B 485      38.523  -2.199  17.685  1.00  0.00           N  \\nATOM   7600  H   ARG B 485      35.646  -8.132  16.296  1.00  0.00           H  \\nATOM   7601  HA  ARG B 485      34.786  -6.609  14.049  1.00  0.00           H  \\nATOM   7602 1HB  ARG B 485      37.574  -7.683  14.575  1.00  0.00           H  \\nATOM   7603 2HB  ARG B 485      37.158  -6.421  13.426  1.00  0.00           H  \\nATOM   7604 1HG  ARG B 485      36.174  -5.077  15.343  1.00  0.00           H  \\nATOM   7605 2HG  ARG B 485      36.796  -6.318  16.458  1.00  0.00           H  \\nATOM   7606 1HD  ARG B 485      39.067  -5.890  15.520  1.00  0.00           H  \\nATOM   7607 2HD  ARG B 485      38.370  -4.511  14.638  1.00  0.00           H  \\nATOM   7608  HE  ARG B 485      38.859  -4.714  17.531  1.00  0.00           H  \\nATOM   7609 1HH1 ARG B 485      37.433  -3.023  14.797  1.00  0.00           H  \\nATOM   7610 2HH1 ARG B 485      37.482  -1.447  15.555  1.00  0.00           H  \\nATOM   7611 1HH2 ARG B 485      38.934  -2.601  18.516  1.00  0.00           H  \\nATOM   7612 2HH2 ARG B 485      38.326  -1.210  17.647  1.00  0.00           H  \\nATOM   7613  N   ASN B 486      35.628  -9.621  13.190  1.00  0.00           N  \\nATOM   7614  CA  ASN B 486      35.574 -10.509  12.044  1.00  0.00           C  \\nATOM   7615  C   ASN B 486      34.163 -10.977  11.712  1.00  0.00           C  \\nATOM   7616  O   ASN B 486      33.954 -11.809  10.786  1.00  0.00           O  \\nATOM   7617  CB  ASN B 486      36.497 -11.684  12.296  1.00  0.00           C  \\nATOM   7618  CG  ASN B 486      36.124 -12.548  13.498  1.00  0.00           C  \\nATOM   7619  OD1 ASN B 486      35.033 -12.482  14.108  1.00  0.00           O  \\nATOM   7620  ND2 ASN B 486      37.071 -13.427  13.850  1.00  0.00           N  \\nATOM   7621  H   ASN B 486      35.819 -10.008  14.103  1.00  0.00           H  \\nATOM   7622  HA  ASN B 486      35.915  -9.960  11.164  1.00  0.00           H  \\nATOM   7623 1HB  ASN B 486      36.512 -12.330  11.417  1.00  0.00           H  \\nATOM   7624 2HB  ASN B 486      37.513 -11.321  12.453  1.00  0.00           H  \\nATOM   7625 1HD2 ASN B 486      36.921 -14.039  14.627  1.00  0.00           H  \\nATOM   7626 2HD2 ASN B 486      37.927 -13.471  13.337  1.00  0.00           H  \\nATOM   7627  N   GLY B 487      33.145 -10.482  12.395  1.00  0.00           N  \\nATOM   7628  CA  GLY B 487      31.773 -10.872  12.069  1.00  0.00           C  \\nATOM   7629  C   GLY B 487      31.259 -11.998  12.967  1.00  0.00           C  \\nATOM   7630  O   GLY B 487      30.018 -12.205  13.034  1.00  0.00           O  \\nATOM   7631  H   GLY B 487      33.305  -9.829  13.149  1.00  0.00           H  \\nATOM   7632 1HA  GLY B 487      31.116 -10.008  12.169  1.00  0.00           H  \\nATOM   7633 2HA  GLY B 487      31.724 -11.195  11.030  1.00  0.00           H  \\nATOM   7634  N   SER B 488      32.142 -12.705  13.663  1.00  0.00           N  \\nATOM   7635  CA  SER B 488      31.699 -13.831  14.562  1.00  0.00           C  \\nATOM   7636  C   SER B 488      30.765 -13.376  15.691  1.00  0.00           C  \\nATOM   7637  O   SER B 488      30.881 -12.226  16.130  1.00  0.00           O  \\nATOM   7638  CB  SER B 488      32.861 -14.656  15.115  1.00  0.00           C  \\nATOM   7639  OG  SER B 488      33.718 -13.848  15.894  1.00  0.00           O  \\nATOM   7640  H   SER B 488      33.126 -12.490  13.593  1.00  0.00           H  \\nATOM   7641  HA  SER B 488      31.068 -14.508  13.984  1.00  0.00           H  \\nATOM   7642 1HB  SER B 488      32.471 -15.473  15.721  1.00  0.00           H  \\nATOM   7643 2HB  SER B 488      33.418 -15.098  14.290  1.00  0.00           H  \\nATOM   7644  HG  SER B 488      33.340 -12.966  15.875  1.00  0.00           H  \\nATOM   7645  N   LYS B 489      29.769 -14.195  16.064  1.00  0.00           N  \\nATOM   7646  CA  LYS B 489      28.754 -13.778  17.005  1.00  0.00           C  \\nATOM   7647  C   LYS B 489      29.132 -14.256  18.423  1.00  0.00           C  \\nATOM   7648  O   LYS B 489      29.743 -15.348  18.600  1.00  0.00           O  \\nATOM   7649  CB  LYS B 489      27.385 -14.321  16.591  1.00  0.00           C  \\nATOM   7650  CG  LYS B 489      26.863 -13.769  15.271  1.00  0.00           C  \\nATOM   7651  CD  LYS B 489      25.485 -14.327  14.947  1.00  0.00           C  \\nATOM   7652  CE  LYS B 489      24.964 -13.783  13.624  1.00  0.00           C  \\nATOM   7653  NZ  LYS B 489      23.623 -14.332  13.286  1.00  0.00           N  \\nATOM   7654  H   LYS B 489      29.728 -15.127  15.677  1.00  0.00           H  \\nATOM   7655  HA  LYS B 489      28.710 -12.688  17.006  1.00  0.00           H  \\nATOM   7656 1HB  LYS B 489      27.436 -15.407  16.503  1.00  0.00           H  \\nATOM   7657 2HB  LYS B 489      26.653 -14.089  17.364  1.00  0.00           H  \\nATOM   7658 1HG  LYS B 489      26.801 -12.682  15.330  1.00  0.00           H  \\nATOM   7659 2HG  LYS B 489      27.550 -14.034  14.469  1.00  0.00           H  \\nATOM   7660 1HD  LYS B 489      25.537 -15.415  14.887  1.00  0.00           H  \\nATOM   7661 2HD  LYS B 489      24.787 -14.056  15.739  1.00  0.00           H  \\nATOM   7662 1HE  LYS B 489      24.896 -12.697  13.679  1.00  0.00           H  \\nATOM   7663 2HE  LYS B 489      25.660 -14.040  12.826  1.00  0.00           H  \\nATOM   7664 1HZ  LYS B 489      23.315 -13.948  12.404  1.00  0.00           H  \\nATOM   7665 2HZ  LYS B 489      23.679 -15.338  13.213  1.00  0.00           H  \\nATOM   7666 3HZ  LYS B 489      22.965 -14.083  14.009  1.00  0.00           H  \\nATOM   7667  N   PHE B 490      28.883 -13.430  19.427  1.00  0.00           N  \\nATOM   7668  CA  PHE B 490      29.092 -13.820  20.825  1.00  0.00           C  \\nATOM   7669  C   PHE B 490      28.025 -13.136  21.695  1.00  0.00           C  \\nATOM   7670  O   PHE B 490      27.369 -12.150  21.261  1.00  0.00           O  \\nATOM   7671  CB  PHE B 490      30.538 -13.502  21.373  1.00  0.00           C  \\nATOM   7672  CG  PHE B 490      30.825 -12.028  21.570  1.00  0.00           C  \\nATOM   7673  CD1 PHE B 490      31.062 -11.199  20.472  1.00  0.00           C  \\nATOM   7674  CD2 PHE B 490      30.886 -11.505  22.845  1.00  0.00           C  \\nATOM   7675  CE1 PHE B 490      31.286  -9.874  20.617  1.00  0.00           C  \\nATOM   7676  CE2 PHE B 490      31.147 -10.169  23.048  1.00  0.00           C  \\nATOM   7677  CZ  PHE B 490      31.372  -9.335  21.924  1.00  0.00           C  \\nATOM   7678  H   PHE B 490      28.539 -12.503  19.221  1.00  0.00           H  \\nATOM   7679  HA  PHE B 490      28.952 -14.899  20.908  1.00  0.00           H  \\nATOM   7680 1HB  PHE B 490      30.682 -14.000  22.331  1.00  0.00           H  \\nATOM   7681 2HB  PHE B 490      31.283 -13.897  20.684  1.00  0.00           H  \\nATOM   7682  HD1 PHE B 490      31.066 -11.630  19.470  1.00  0.00           H  \\nATOM   7683  HD2 PHE B 490      30.711 -12.162  23.698  1.00  0.00           H  \\nATOM   7684  HE1 PHE B 490      31.397  -9.241  19.737  1.00  0.00           H  \\nATOM   7685  HE2 PHE B 490      31.183  -9.749  24.053  1.00  0.00           H  \\nATOM   7686  HZ  PHE B 490      31.612  -8.282  22.066  1.00  0.00           H  \\nATOM   7687  N   SER B 491      27.919 -13.630  22.947  1.00  0.00           N  \\nATOM   7688  CA  SER B 491      26.919 -13.165  23.858  1.00  0.00           C  \\nATOM   7689  C   SER B 491      27.599 -12.453  25.005  1.00  0.00           C  \\nATOM   7690  O   SER B 491      28.635 -12.906  25.476  1.00  0.00           O  \\nATOM   7691  CB  SER B 491      26.146 -14.364  24.398  1.00  0.00           C  \\nATOM   7692  OG  SER B 491      25.481 -14.976  23.308  1.00  0.00           O  \\nATOM   7693  H   SER B 491      28.563 -14.347  23.248  1.00  0.00           H  \\nATOM   7694  HA  SER B 491      26.237 -12.508  23.317  1.00  0.00           H  \\nATOM   7695 1HB  SER B 491      26.837 -15.056  24.879  1.00  0.00           H  \\nATOM   7696 2HB  SER B 491      25.439 -14.029  25.155  1.00  0.00           H  \\nATOM   7697  HG  SER B 491      25.709 -14.456  22.533  1.00  0.00           H  \\nATOM   7698  N   VAL B 492      26.965 -11.371  25.443  1.00  0.00           N  \\nATOM   7699  CA  VAL B 492      27.307 -10.679  26.720  1.00  0.00           C  \\nATOM   7700  C   VAL B 492      26.221 -10.971  27.754  1.00  0.00           C  \\nATOM   7701  O   VAL B 492      25.012 -10.907  27.445  1.00  0.00           O  \\nATOM   7702  CB  VAL B 492      27.488  -9.117  26.583  1.00  0.00           C  \\nATOM   7703  CG1 VAL B 492      28.705  -8.752  25.660  1.00  0.00           C  \\nATOM   7704  CG2 VAL B 492      26.202  -8.444  26.106  1.00  0.00           C  \\nATOM   7705  H   VAL B 492      26.215 -11.008  24.873  1.00  0.00           H  \\nATOM   7706  HA  VAL B 492      28.255 -11.075  27.085  1.00  0.00           H  \\nATOM   7707  HB  VAL B 492      27.758  -8.705  27.555  1.00  0.00           H  \\nATOM   7708 1HG1 VAL B 492      28.797  -7.668  25.591  1.00  0.00           H  \\nATOM   7709 2HG1 VAL B 492      29.620  -9.167  26.083  1.00  0.00           H  \\nATOM   7710 3HG1 VAL B 492      28.544  -9.166  24.665  1.00  0.00           H  \\nATOM   7711 1HG2 VAL B 492      26.363  -7.369  26.024  1.00  0.00           H  \\nATOM   7712 2HG2 VAL B 492      25.921  -8.845  25.132  1.00  0.00           H  \\nATOM   7713 3HG2 VAL B 492      25.403  -8.637  26.822  1.00  0.00           H  \\nATOM   7714  N   TYR B 493      26.653 -11.330  28.978  1.00  0.00           N  \\nATOM   7715  CA  TYR B 493      25.772 -11.827  29.991  1.00  0.00           C  \\nATOM   7716  C   TYR B 493      25.647 -10.865  31.159  1.00  0.00           C  \\nATOM   7717  O   TYR B 493      24.804 -11.043  31.996  1.00  0.00           O  \\nATOM   7718  CB  TYR B 493      26.191 -13.249  30.511  1.00  0.00           C  \\nATOM   7719  CG  TYR B 493      26.165 -14.348  29.433  1.00  0.00           C  \\nATOM   7720  CD1 TYR B 493      27.266 -14.556  28.591  1.00  0.00           C  \\nATOM   7721  CD2 TYR B 493      25.031 -15.128  29.247  1.00  0.00           C  \\nATOM   7722  CE1 TYR B 493      27.255 -15.545  27.600  1.00  0.00           C  \\nATOM   7723  CE2 TYR B 493      24.984 -16.089  28.251  1.00  0.00           C  \\nATOM   7724  CZ  TYR B 493      26.091 -16.306  27.447  1.00  0.00           C  \\nATOM   7725  OH  TYR B 493      26.026 -17.298  26.477  1.00  0.00           O  \\nATOM   7726  H   TYR B 493      27.638 -11.243  29.182  1.00  0.00           H  \\nATOM   7727  HA  TYR B 493      24.771 -11.912  29.567  1.00  0.00           H  \\nATOM   7728 1HB  TYR B 493      27.201 -13.204  30.921  1.00  0.00           H  \\nATOM   7729 2HB  TYR B 493      25.524 -13.553  31.317  1.00  0.00           H  \\nATOM   7730  HD1 TYR B 493      28.159 -13.940  28.699  1.00  0.00           H  \\nATOM   7731  HD2 TYR B 493      24.161 -14.987  29.889  1.00  0.00           H  \\nATOM   7732  HE1 TYR B 493      28.126 -15.684  26.960  1.00  0.00           H  \\nATOM   7733  HE2 TYR B 493      24.076 -16.673  28.100  1.00  0.00           H  \\nATOM   7734  HH  TYR B 493      25.163 -17.718  26.507  1.00  0.00           H  \\nATOM   7735  N   ASP B 494      26.521  -9.876  31.225  1.00  0.00           N  \\nATOM   7736  CA  ASP B 494      26.554  -8.958  32.367  1.00  0.00           C  \\nATOM   7737  C   ASP B 494      27.135  -7.626  32.031  1.00  0.00           C  \\nATOM   7738  O   ASP B 494      27.814  -7.500  31.047  1.00  0.00           O  \\nATOM   7739  CB  ASP B 494      27.310  -9.551  33.578  1.00  0.00           C  \\nATOM   7740  CG  ASP B 494      28.651 -10.127  33.196  1.00  0.00           C  \\nATOM   7741  OD1 ASP B 494      29.528  -9.428  32.595  1.00  0.00           O  \\nATOM   7742  OD2 ASP B 494      28.855 -11.322  33.524  1.00  0.00           O  \\nATOM   7743  H   ASP B 494      27.182  -9.749  30.472  1.00  0.00           H  \\nATOM   7744  HA  ASP B 494      25.529  -8.759  32.681  1.00  0.00           H  \\nATOM   7745 1HB  ASP B 494      27.461  -8.775  34.329  1.00  0.00           H  \\nATOM   7746 2HB  ASP B 494      26.707 -10.336  34.035  1.00  0.00           H  \\nATOM   7747  N   VAL B 495      26.908  -6.632  32.901  1.00  0.00           N  \\nATOM   7748  CA  VAL B 495      27.319  -5.265  32.564  1.00  0.00           C  \\nATOM   7749  C   VAL B 495      28.837  -5.078  32.584  1.00  0.00           C  \\nATOM   7750  O   VAL B 495      29.377  -4.227  31.847  1.00  0.00           O  \\nATOM   7751  CB  VAL B 495      26.631  -4.254  33.517  1.00  0.00           C  \\nATOM   7752  CG1 VAL B 495      27.227  -4.358  34.931  1.00  0.00           C  \\nATOM   7753  CG2 VAL B 495      26.722  -2.800  32.939  1.00  0.00           C  \\nATOM   7754  H   VAL B 495      26.458  -6.806  33.788  1.00  0.00           H  \\nATOM   7755  HA  VAL B 495      27.008  -5.053  31.540  1.00  0.00           H  \\nATOM   7756  HB  VAL B 495      25.582  -4.530  33.628  1.00  0.00           H  \\nATOM   7757 1HG1 VAL B 495      26.733  -3.642  35.588  1.00  0.00           H  \\nATOM   7758 2HG1 VAL B 495      27.076  -5.366  35.316  1.00  0.00           H  \\nATOM   7759 3HG1 VAL B 495      28.294  -4.138  34.893  1.00  0.00           H  \\nATOM   7760 1HG2 VAL B 495      26.233  -2.106  33.622  1.00  0.00           H  \\nATOM   7761 2HG2 VAL B 495      27.769  -2.519  32.824  1.00  0.00           H  \\nATOM   7762 3HG2 VAL B 495      26.227  -2.763  31.968  1.00  0.00           H  \\nATOM   7763  N   SER B 496      29.563  -5.847  33.391  1.00  0.00           N  \\nATOM   7764  CA  SER B 496      30.975  -5.523  33.629  1.00  0.00           C  \\nATOM   7765  C   SER B 496      31.768  -5.341  32.361  1.00  0.00           C  \\nATOM   7766  O   SER B 496      32.474  -4.347  32.227  1.00  0.00           O  \\nATOM   7767  CB  SER B 496      31.676  -6.579  34.540  1.00  0.00           C  \\nATOM   7768  OG  SER B 496      31.337  -7.884  34.161  1.00  0.00           O  \\nATOM   7769  H   SER B 496      29.160  -6.654  33.846  1.00  0.00           H  \\nATOM   7770  HA  SER B 496      31.028  -4.558  34.135  1.00  0.00           H  \\nATOM   7771 1HB  SER B 496      32.757  -6.452  34.479  1.00  0.00           H  \\nATOM   7772 2HB  SER B 496      31.385  -6.415  35.577  1.00  0.00           H  \\nATOM   7773  HG  SER B 496      30.741  -7.796  33.414  1.00  0.00           H  \\nATOM   7774  N   ILE B 497      31.689  -6.306  31.432  1.00  0.00           N  \\nATOM   7775  CA  ILE B 497      32.426  -6.245  30.184  1.00  0.00           C  \\nATOM   7776  C   ILE B 497      32.103  -5.016  29.300  1.00  0.00           C  \\nATOM   7777  O   ILE B 497      32.887  -4.663  28.430  1.00  0.00           O  \\nATOM   7778  CB  ILE B 497      32.270  -7.604  29.361  1.00  0.00           C  \\nATOM   7779  CG1 ILE B 497      33.443  -7.760  28.411  1.00  0.00           C  \\nATOM   7780  CG2 ILE B 497      30.947  -7.716  28.684  1.00  0.00           C  \\nATOM   7781  CD1 ILE B 497      34.681  -8.478  28.951  1.00  0.00           C  \\nATOM   7782  H   ILE B 497      31.093  -7.101  31.613  1.00  0.00           H  \\nATOM   7783  HA  ILE B 497      33.481  -6.101  30.412  1.00  0.00           H  \\nATOM   7784  HB  ILE B 497      32.372  -8.452  30.037  1.00  0.00           H  \\nATOM   7785 1HG1 ILE B 497      33.124  -8.311  27.527  1.00  0.00           H  \\nATOM   7786 2HG1 ILE B 497      33.776  -6.776  28.079  1.00  0.00           H  \\nATOM   7787 1HG2 ILE B 497      30.896  -8.658  28.139  1.00  0.00           H  \\nATOM   7788 2HG2 ILE B 497      30.153  -7.685  29.429  1.00  0.00           H  \\nATOM   7789 3HG2 ILE B 497      30.823  -6.887  27.987  1.00  0.00           H  \\nATOM   7790 1HD1 ILE B 497      35.444  -8.522  28.174  1.00  0.00           H  \\nATOM   7791 2HD1 ILE B 497      35.071  -7.934  29.812  1.00  0.00           H  \\nATOM   7792 3HD1 ILE B 497      34.412  -9.490  29.252  1.00  0.00           H  \\nATOM   7793  N   LEU B 498      30.918  -4.423  29.545  1.00  0.00           N  \\nATOM   7794  CA  LEU B 498      30.430  -3.319  28.712  1.00  0.00           C  \\nATOM   7795  C   LEU B 498      30.796  -1.994  29.257  1.00  0.00           C  \\nATOM   7796  O   LEU B 498      30.724  -1.005  28.563  1.00  0.00           O  \\nATOM   7797  CB  LEU B 498      28.916  -3.403  28.412  1.00  0.00           C  \\nATOM   7798  CG  LEU B 498      28.440  -4.680  27.699  1.00  0.00           C  \\nATOM   7799  CD1 LEU B 498      26.967  -4.567  27.498  1.00  0.00           C  \\nATOM   7800  CD2 LEU B 498      29.093  -4.745  26.346  1.00  0.00           C  \\nATOM   7801  H   LEU B 498      30.352  -4.744  30.318  1.00  0.00           H  \\nATOM   7802  HA  LEU B 498      30.951  -3.351  27.756  1.00  0.00           H  \\nATOM   7803 1HB  LEU B 498      28.371  -3.329  29.352  1.00  0.00           H  \\nATOM   7804 2HB  LEU B 498      28.638  -2.555  27.787  1.00  0.00           H  \\nATOM   7805  HG  LEU B 498      28.721  -5.553  28.289  1.00  0.00           H  \\nATOM   7806 1HD1 LEU B 498      26.599  -5.460  26.993  1.00  0.00           H  \\nATOM   7807 2HD1 LEU B 498      26.474  -4.469  28.465  1.00  0.00           H  \\nATOM   7808 3HD1 LEU B 498      26.749  -3.691  26.889  1.00  0.00           H  \\nATOM   7809 1HD2 LEU B 498      28.766  -5.647  25.827  1.00  0.00           H  \\nATOM   7810 2HD2 LEU B 498      28.811  -3.868  25.763  1.00  0.00           H  \\nATOM   7811 3HD2 LEU B 498      30.176  -4.768  26.465  1.00  0.00           H  \\nATOM   7812  N   ILE B 499      31.244  -1.876  30.508  1.00  0.00           N  \\nATOM   7813  CA  ILE B 499      31.511  -0.593  31.129  1.00  0.00           C  \\nATOM   7814  C   ILE B 499      32.570   0.218  30.327  1.00  0.00           C  \\nATOM   7815  O   ILE B 499      32.372   1.388  30.176  1.00  0.00           O  \\nATOM   7816  CB  ILE B 499      31.828  -0.770  32.626  1.00  0.00           C  \\nATOM   7817  CG1 ILE B 499      30.584  -1.155  33.344  1.00  0.00           C  \\nATOM   7818  CG2 ILE B 499      32.181   0.553  33.239  1.00  0.00           C  \\nATOM   7819  CD1 ILE B 499      30.793  -1.685  34.841  1.00  0.00           C  \\nATOM   7820  H   ILE B 499      31.401  -2.721  31.038  1.00  0.00           H  \\nATOM   7821  HA  ILE B 499      30.622   0.029  31.034  1.00  0.00           H  \\nATOM   7822  HB  ILE B 499      32.669  -1.453  32.741  1.00  0.00           H  \\nATOM   7823 1HG1 ILE B 499      29.912  -0.299  33.391  1.00  0.00           H  \\nATOM   7824 2HG1 ILE B 499      30.072  -1.941  32.789  1.00  0.00           H  \\nATOM   7825 1HG2 ILE B 499      32.403   0.416  34.297  1.00  0.00           H  \\nATOM   7826 2HG2 ILE B 499      33.055   0.965  32.736  1.00  0.00           H  \\nATOM   7827 3HG2 ILE B 499      31.342   1.240  33.130  1.00  0.00           H  \\nATOM   7828 1HD1 ILE B 499      29.826  -1.939  35.275  1.00  0.00           H  \\nATOM   7829 2HD1 ILE B 499      31.429  -2.571  34.828  1.00  0.00           H  \\nATOM   7830 3HD1 ILE B 499      31.265  -0.907  35.439  1.00  0.00           H  \\nATOM   7831  N   PRO B 500      33.622  -0.411  29.761  1.00  0.00           N  \\nATOM   7832  CA  PRO B 500      34.563   0.376  28.984  1.00  0.00           C  \\nATOM   7833  C   PRO B 500      34.041   0.791  27.610  1.00  0.00           C  \\nATOM   7834  O   PRO B 500      34.715   1.531  26.928  1.00  0.00           O  \\nATOM   7835  CB  PRO B 500      35.789  -0.574  28.784  1.00  0.00           C  \\nATOM   7836  CG  PRO B 500      35.619  -1.559  29.907  1.00  0.00           C  \\nATOM   7837  CD  PRO B 500      34.148  -1.789  29.976  1.00  0.00           C  \\nATOM   7838  HA  PRO B 500      34.857   1.264  29.563  1.00  0.00           H  \\nATOM   7839 1HB  PRO B 500      35.751  -1.032  27.784  1.00  0.00           H  \\nATOM   7840 2HB  PRO B 500      36.724   0.002  28.839  1.00  0.00           H  \\nATOM   7841 1HG  PRO B 500      36.183  -2.479  29.694  1.00  0.00           H  \\nATOM   7842 2HG  PRO B 500      36.028  -1.145  30.840  1.00  0.00           H  \\nATOM   7843 1HD  PRO B 500      33.847  -2.482  29.177  1.00  0.00           H  \\nATOM   7844 2HD  PRO B 500      33.886  -2.197  30.963  1.00  0.00           H  \\nATOM   7845  N   ILE B 501      32.963   0.143  27.163  1.00  0.00           N  \\nATOM   7846  CA  ILE B 501      32.442   0.216  25.827  1.00  0.00           C  \\nATOM   7847  C   ILE B 501      31.350   1.264  25.609  1.00  0.00           C  \\nATOM   7848  O   ILE B 501      31.381   2.001  24.611  1.00  0.00           O  \\nATOM   7849  CB  ILE B 501      31.873  -1.149  25.452  1.00  0.00           C  \\nATOM   7850  CG1 ILE B 501      32.900  -2.267  25.562  1.00  0.00           C  \\nATOM   7851  CG2 ILE B 501      31.322  -1.150  23.978  1.00  0.00           C  \\nATOM   7852  CD1 ILE B 501      34.223  -1.929  24.906  1.00  0.00           C  \\nATOM   7853  H   ILE B 501      32.495  -0.440  27.842  1.00  0.00           H  \\nATOM   7854  HA  ILE B 501      33.258   0.469  25.151  1.00  0.00           H  \\nATOM   7855  HB  ILE B 501      31.057  -1.401  26.128  1.00  0.00           H  \\nATOM   7856 1HG1 ILE B 501      33.081  -2.493  26.612  1.00  0.00           H  \\nATOM   7857 2HG1 ILE B 501      32.505  -3.171  25.098  1.00  0.00           H  \\nATOM   7858 1HG2 ILE B 501      30.924  -2.136  23.739  1.00  0.00           H  \\nATOM   7859 2HG2 ILE B 501      30.531  -0.407  23.885  1.00  0.00           H  \\nATOM   7860 3HG2 ILE B 501      32.130  -0.908  23.287  1.00  0.00           H  \\nATOM   7861 1HD1 ILE B 501      34.910  -2.768  25.020  1.00  0.00           H  \\nATOM   7862 2HD1 ILE B 501      34.064  -1.732  23.845  1.00  0.00           H  \\nATOM   7863 3HD1 ILE B 501      34.648  -1.045  25.380  1.00  0.00           H  \\nATOM   7864  N   ILE B 502      30.383   1.269  26.509  1.00  0.00           N  \\nATOM   7865  CA  ILE B 502      29.262   2.165  26.391  1.00  0.00           C  \\nATOM   7866  C   ILE B 502      29.371   3.320  27.360  1.00  0.00           C  \\nATOM   7867  O   ILE B 502      29.684   3.143  28.614  1.00  0.00           O  \\nATOM   7868  CB  ILE B 502      27.960   1.415  26.610  1.00  0.00           C  \\nATOM   7869  CG1 ILE B 502      26.670   2.279  26.409  1.00  0.00           C  \\nATOM   7870  CG2 ILE B 502      27.911   0.818  27.995  1.00  0.00           C  \\nATOM   7871  CD1 ILE B 502      25.382   1.298  26.182  1.00  0.00           C  \\nATOM   7872  H   ILE B 502      30.426   0.636  27.295  1.00  0.00           H  \\nATOM   7873  HA  ILE B 502      29.259   2.584  25.385  1.00  0.00           H  \\nATOM   7874  HB  ILE B 502      27.876   0.612  25.879  1.00  0.00           H  \\nATOM   7875 1HG1 ILE B 502      26.517   2.912  27.282  1.00  0.00           H  \\nATOM   7876 2HG1 ILE B 502      26.802   2.936  25.549  1.00  0.00           H  \\nATOM   7877 1HG2 ILE B 502      26.969   0.286  28.129  1.00  0.00           H  \\nATOM   7878 2HG2 ILE B 502      28.741   0.124  28.121  1.00  0.00           H  \\nATOM   7879 3HG2 ILE B 502      27.987   1.613  28.737  1.00  0.00           H  \\nATOM   7880 1HD1 ILE B 502      24.485   1.902  26.042  1.00  0.00           H  \\nATOM   7881 2HD1 ILE B 502      25.546   0.681  25.298  1.00  0.00           H  \\nATOM   7882 3HD1 ILE B 502      25.255   0.658  27.054  1.00  0.00           H  \\nATOM   7883  N   ALA B 503      29.168   4.548  26.847  1.00  0.00           N  \\nATOM   7884  CA  ALA B 503      29.350   5.779  27.597  1.00  0.00           C  \\nATOM   7885  C   ALA B 503      28.127   6.440  28.077  1.00  0.00           C  \\nATOM   7886  O   ALA B 503      28.263   7.250  29.058  1.00  0.00           O  \\nATOM   7887  CB  ALA B 503      30.159   6.842  26.672  1.00  0.00           C  \\nATOM   7888  H   ALA B 503      28.872   4.599  25.883  1.00  0.00           H  \\nATOM   7889  HA  ALA B 503      29.935   5.549  28.488  1.00  0.00           H  \\nATOM   7890 1HB  ALA B 503      30.302   7.771  27.225  1.00  0.00           H  \\nATOM   7891 2HB  ALA B 503      31.130   6.426  26.404  1.00  0.00           H  \\nATOM   7892 3HB  ALA B 503      29.589   7.043  25.766  1.00  0.00           H  \\nATOM   7893  N   LEU B 504      26.992   6.175  27.400  1.00  0.00           N  \\nATOM   7894  CA  LEU B 504      25.734   6.916  27.659  1.00  0.00           C  \\nATOM   7895  C   LEU B 504      24.626   5.871  27.392  1.00  0.00           C  \\nATOM   7896  O   LEU B 504      24.716   5.013  26.538  1.00  0.00           O  \\nATOM   7897  CB  LEU B 504      25.429   8.042  26.563  1.00  0.00           C  \\nATOM   7898  CG  LEU B 504      26.321   9.231  26.678  1.00  0.00           C  \\nATOM   7899  CD1 LEU B 504      25.967  10.074  25.381  1.00  0.00           C  \\nATOM   7900  CD2 LEU B 504      26.330  10.047  27.881  1.00  0.00           C  \\nATOM   7901  H   LEU B 504      27.000   5.451  26.696  1.00  0.00           H  \\nATOM   7902  HA  LEU B 504      25.820   7.416  28.623  1.00  0.00           H  \\nATOM   7903 1HB  LEU B 504      25.548   7.604  25.573  1.00  0.00           H  \\nATOM   7904 2HB  LEU B 504      24.393   8.363  26.674  1.00  0.00           H  \\nATOM   7905  HG  LEU B 504      27.360   8.917  26.577  1.00  0.00           H  \\nATOM   7906 1HD1 LEU B 504      26.571  10.981  25.361  1.00  0.00           H  \\nATOM   7907 2HD1 LEU B 504      26.177   9.479  24.492  1.00  0.00           H  \\nATOM   7908 3HD1 LEU B 504      24.911  10.341  25.398  1.00  0.00           H  \\nATOM   7909 1HD2 LEU B 504      27.047  10.861  27.764  1.00  0.00           H  \\nATOM   7910 2HD2 LEU B 504      25.335  10.461  28.047  1.00  0.00           H  \\nATOM   7911 3HD2 LEU B 504      26.615   9.433  28.735  1.00  0.00           H  \\nATOM   7912  N   MET B 505      23.528   5.942  28.127  1.00  0.00           N  \\nATOM   7913  CA  MET B 505      22.343   5.108  27.851  1.00  0.00           C  \\nATOM   7914  C   MET B 505      21.094   5.930  27.826  1.00  0.00           C  \\nATOM   7915  O   MET B 505      20.985   6.930  28.587  1.00  0.00           O  \\nATOM   7916  CB  MET B 505      22.210   4.124  28.977  1.00  0.00           C  \\nATOM   7917  CG  MET B 505      23.286   3.098  29.085  1.00  0.00           C  \\nATOM   7918  SD  MET B 505      22.896   1.833  30.395  1.00  0.00           S  \\nATOM   7919  CE  MET B 505      22.964   2.774  31.934  1.00  0.00           C  \\nATOM   7920  H   MET B 505      23.504   6.590  28.902  1.00  0.00           H  \\nATOM   7921  HA  MET B 505      22.503   4.580  26.911  1.00  0.00           H  \\nATOM   7922 1HB  MET B 505      22.186   4.658  29.926  1.00  0.00           H  \\nATOM   7923 2HB  MET B 505      21.267   3.585  28.880  1.00  0.00           H  \\nATOM   7924 1HG  MET B 505      23.404   2.593  28.127  1.00  0.00           H  \\nATOM   7925 2HG  MET B 505      24.229   3.585  29.332  1.00  0.00           H  \\nATOM   7926 1HE  MET B 505      22.747   2.114  32.774  1.00  0.00           H  \\nATOM   7927 2HE  MET B 505      23.960   3.202  32.056  1.00  0.00           H  \\nATOM   7928 3HE  MET B 505      22.226   3.577  31.903  1.00  0.00           H  \\nATOM   7929  N   VAL B 506      20.180   5.548  26.975  1.00  0.00           N  \\nATOM   7930  CA  VAL B 506      18.859   6.139  26.978  1.00  0.00           C  \\nATOM   7931  C   VAL B 506      18.147   5.714  28.264  1.00  0.00           C  \\nATOM   7932  O   VAL B 506      18.273   4.604  28.686  1.00  0.00           O  \\nATOM   7933  CB  VAL B 506      18.054   5.749  25.717  1.00  0.00           C  \\nATOM   7934  CG1 VAL B 506      17.999   4.324  25.600  1.00  0.00           C  \\nATOM   7935  CG2 VAL B 506      16.664   6.389  25.698  1.00  0.00           C  \\nATOM   7936  H   VAL B 506      20.395   4.827  26.301  1.00  0.00           H  \\nATOM   7937  HA  VAL B 506      18.965   7.224  26.988  1.00  0.00           H  \\nATOM   7938  HB  VAL B 506      18.598   6.082  24.833  1.00  0.00           H  \\nATOM   7939 1HG1 VAL B 506      17.431   4.054  24.710  1.00  0.00           H  \\nATOM   7940 2HG1 VAL B 506      19.010   3.926  25.519  1.00  0.00           H  \\nATOM   7941 3HG1 VAL B 506      17.513   3.905  26.481  1.00  0.00           H  \\nATOM   7942 1HG2 VAL B 506      16.137   6.087  24.794  1.00  0.00           H  \\nATOM   7943 2HG2 VAL B 506      16.101   6.063  26.573  1.00  0.00           H  \\nATOM   7944 3HG2 VAL B 506      16.763   7.475  25.714  1.00  0.00           H  \\nATOM   7945  N   TYR B 507      17.458   6.641  28.878  1.00  0.00           N  \\nATOM   7946  CA  TYR B 507      16.734   6.385  30.098  1.00  0.00           C  \\nATOM   7947  C   TYR B 507      15.623   5.395  29.889  1.00  0.00           C  \\nATOM   7948  O   TYR B 507      14.852   5.514  29.008  1.00  0.00           O  \\nATOM   7949  CB  TYR B 507      16.157   7.678  30.626  1.00  0.00           C  \\nATOM   7950  CG  TYR B 507      15.403   7.598  31.920  1.00  0.00           C  \\nATOM   7951  CD1 TYR B 507      14.084   7.196  31.955  1.00  0.00           C  \\nATOM   7952  CD2 TYR B 507      15.980   7.994  33.058  1.00  0.00           C  \\nATOM   7953  CE1 TYR B 507      13.409   7.154  33.148  1.00  0.00           C  \\nATOM   7954  CE2 TYR B 507      15.307   7.964  34.230  1.00  0.00           C  \\nATOM   7955  CZ  TYR B 507      14.031   7.537  34.269  1.00  0.00           C  \\nATOM   7956  OH  TYR B 507      13.375   7.528  35.456  1.00  0.00           O  \\nATOM   7957  H   TYR B 507      17.436   7.568  28.478  1.00  0.00           H  \\nATOM   7958  HA  TYR B 507      17.428   5.978  30.834  1.00  0.00           H  \\nATOM   7959 1HB  TYR B 507      16.960   8.401  30.776  1.00  0.00           H  \\nATOM   7960 2HB  TYR B 507      15.472   8.098  29.890  1.00  0.00           H  \\nATOM   7961  HD1 TYR B 507      13.576   6.911  31.033  1.00  0.00           H  \\nATOM   7962  HD2 TYR B 507      17.010   8.350  33.052  1.00  0.00           H  \\nATOM   7963  HE1 TYR B 507      12.367   6.836  33.174  1.00  0.00           H  \\nATOM   7964  HE2 TYR B 507      15.797   8.287  35.149  1.00  0.00           H  \\nATOM   7965  HH  TYR B 507      13.959   7.851  36.146  1.00  0.00           H  \\nATOM   7966  N   ARG B 508      15.577   4.409  30.759  1.00  0.00           N  \\nATOM   7967  CA  ARG B 508      14.501   3.447  30.770  1.00  0.00           C  \\nATOM   7968  C   ARG B 508      13.661   3.462  32.057  1.00  0.00           C  \\nATOM   7969  O   ARG B 508      12.455   3.466  31.978  1.00  0.00           O  \\nATOM   7970  CB  ARG B 508      15.020   2.026  30.482  1.00  0.00           C  \\nATOM   7971  CG  ARG B 508      15.692   1.839  29.147  1.00  0.00           C  \\nATOM   7972  CD  ARG B 508      14.848   2.286  27.945  1.00  0.00           C  \\nATOM   7973  NE  ARG B 508      15.367   1.709  26.722  1.00  0.00           N  \\nATOM   7974  CZ  ARG B 508      15.139   2.147  25.514  1.00  0.00           C  \\nATOM   7975  NH1 ARG B 508      14.402   3.183  25.332  1.00  0.00           N  \\nATOM   7976  NH2 ARG B 508      15.650   1.526  24.492  1.00  0.00           N  \\nATOM   7977  H   ARG B 508      16.319   4.325  31.439  1.00  0.00           H  \\nATOM   7978  HA  ARG B 508      13.789   3.716  29.989  1.00  0.00           H  \\nATOM   7979 1HB  ARG B 508      15.738   1.739  31.249  1.00  0.00           H  \\nATOM   7980 2HB  ARG B 508      14.191   1.319  30.529  1.00  0.00           H  \\nATOM   7981 1HG  ARG B 508      16.615   2.418  29.120  1.00  0.00           H  \\nATOM   7982 2HG  ARG B 508      15.921   0.783  29.000  1.00  0.00           H  \\nATOM   7983 1HD  ARG B 508      13.818   1.959  28.082  1.00  0.00           H  \\nATOM   7984 2HD  ARG B 508      14.876   3.372  27.865  1.00  0.00           H  \\nATOM   7985  HE  ARG B 508      15.960   0.894  26.802  1.00  0.00           H  \\nATOM   7986 1HH1 ARG B 508      13.997   3.661  26.125  1.00  0.00           H  \\nATOM   7987 2HH1 ARG B 508      14.227   3.520  24.396  1.00  0.00           H  \\nATOM   7988 1HH2 ARG B 508      16.223   0.705  24.634  1.00  0.00           H  \\nATOM   7989 2HH2 ARG B 508      15.475   1.863  23.557  1.00  0.00           H  \\nATOM   7990  N   CYS B 509      14.295   3.447  33.196  1.00  0.00           N  \\nATOM   7991  CA  CYS B 509      13.550   3.374  34.430  1.00  0.00           C  \\nATOM   7992  C   CYS B 509      14.196   4.210  35.475  1.00  0.00           C  \\nATOM   7993  O   CYS B 509      15.269   4.695  35.279  1.00  0.00           O  \\nATOM   7994  CB  CYS B 509      13.359   1.939  34.876  1.00  0.00           C  \\nATOM   7995  SG  CYS B 509      14.727   0.929  34.603  1.00  0.00           S  \\nATOM   7996  H   CYS B 509      15.304   3.486  33.232  1.00  0.00           H  \\nATOM   7997  HA  CYS B 509      12.566   3.814  34.266  1.00  0.00           H  \\nATOM   7998 1HB  CYS B 509      13.131   1.916  35.942  1.00  0.00           H  \\nATOM   7999 2HB  CYS B 509      12.509   1.505  34.350  1.00  0.00           H  \\nATOM   8000  HG  CYS B 509      14.184  -0.177  35.101  1.00  0.00           H  \\nATOM   8001  N   ALA B 510      13.484   4.394  36.564  1.00  0.00           N  \\nATOM   8002  CA  ALA B 510      13.919   5.255  37.627  1.00  0.00           C  \\nATOM   8003  C   ALA B 510      15.092   4.583  38.336  1.00  0.00           C  \\nATOM   8004  O   ALA B 510      15.278   3.398  38.215  1.00  0.00           O  \\nATOM   8005  OXT ALA B 510      15.834   5.235  39.018  1.00  0.00           O  \\nATOM   8006  CB  ALA B 510      12.767   5.499  38.582  1.00  0.00           C  \\nATOM   8007  H   ALA B 510      12.601   3.911  36.652  1.00  0.00           H  \\nATOM   8008  HA  ALA B 510      14.232   6.203  37.189  1.00  0.00           H  \\nATOM   8009 1HB  ALA B 510      13.096   6.153  39.390  1.00  0.00           H  \\nATOM   8010 2HB  ALA B 510      11.944   5.971  38.046  1.00  0.00           H  \\nATOM   8011 3HB  ALA B 510      12.432   4.550  38.998  1.00  0.00           H  \\nTER                                                                             \\nATOM   8013  N   GLY C 511     -17.984   5.581  27.591  1.00  0.00           N  \\nATOM   8014  CA  GLY C 511     -16.735   6.403  27.701  1.00  0.00           C  \\nATOM   8015  C   GLY C 511     -16.107   6.703  26.341  1.00  0.00           C  \\nATOM   8016  O   GLY C 511     -15.043   7.369  26.241  1.00  0.00           O  \\nATOM   8017 1H   GLY C 511     -18.355   5.413  28.504  1.00  0.00           H  \\nATOM   8018 2H   GLY C 511     -18.660   6.073  27.043  1.00  0.00           H  \\nATOM   8019 3H   GLY C 511     -17.771   4.708  27.152  1.00  0.00           H  \\nATOM   8020 1HA  GLY C 511     -16.962   7.344  28.202  1.00  0.00           H  \\nATOM   8021 2HA  GLY C 511     -16.007   5.878  28.319  1.00  0.00           H  \\nATOM   8022  N   PHE C 512     -16.804   6.300  25.279  1.00  0.00           N  \\nATOM   8023  CA  PHE C 512     -16.252   6.464  23.927  1.00  0.00           C  \\nATOM   8024  C   PHE C 512     -16.621   7.833  23.358  1.00  0.00           C  \\nATOM   8025  O   PHE C 512     -16.106   8.237  22.298  1.00  0.00           O  \\nATOM   8026  CB  PHE C 512     -16.805   5.394  23.007  1.00  0.00           C  \\nATOM   8027  CG  PHE C 512     -16.371   4.036  23.357  1.00  0.00           C  \\nATOM   8028  CD1 PHE C 512     -16.962   2.951  22.730  1.00  0.00           C  \\nATOM   8029  CD2 PHE C 512     -15.412   3.819  24.372  1.00  0.00           C  \\nATOM   8030  CE1 PHE C 512     -16.551   1.641  23.011  1.00  0.00           C  \\nATOM   8031  CE2 PHE C 512     -14.980   2.506  24.682  1.00  0.00           C  \\nATOM   8032  CZ  PHE C 512     -15.585   1.408  24.043  1.00  0.00           C  \\nATOM   8033  H   PHE C 512     -17.715   5.879  25.391  1.00  0.00           H  \\nATOM   8034  HA  PHE C 512     -15.168   6.357  23.980  1.00  0.00           H  \\nATOM   8035 1HB  PHE C 512     -17.894   5.422  23.030  1.00  0.00           H  \\nATOM   8036 2HB  PHE C 512     -16.495   5.599  21.983  1.00  0.00           H  \\nATOM   8037  HD1 PHE C 512     -17.759   3.123  22.006  1.00  0.00           H  \\nATOM   8038  HD2 PHE C 512     -14.949   4.675  24.863  1.00  0.00           H  \\nATOM   8039  HE1 PHE C 512     -16.966   0.807  22.445  1.00  0.00           H  \\nATOM   8040  HE2 PHE C 512     -14.186   2.342  25.410  1.00  0.00           H  \\nATOM   8041  HZ  PHE C 512     -15.326   0.389  24.328  1.00  0.00           H  \\nATOM   8042  N   GLY C 513     -17.412   8.616  24.133  1.00  0.00           N  \\nATOM   8043  CA  GLY C 513     -17.780   9.984  23.768  1.00  0.00           C  \\nATOM   8044  C   GLY C 513     -16.681  10.939  23.452  1.00  0.00           C  \\nATOM   8045  O   GLY C 513     -16.885  11.857  22.649  1.00  0.00           O  \\nATOM   8046  H   GLY C 513     -17.758   8.228  24.999  1.00  0.00           H  \\nATOM   8047 1HA  GLY C 513     -18.427   9.965  22.891  1.00  0.00           H  \\nATOM   8048 2HA  GLY C 513     -18.351  10.436  24.579  1.00  0.00           H  \\nATOM   8049  N   LEU C 514     -15.458  10.789  24.037  1.00  0.00           N  \\nATOM   8050  CA  LEU C 514     -14.323  11.732  23.683  1.00  0.00           C  \\nATOM   8051  C   LEU C 514     -14.013  11.870  22.186  1.00  0.00           C  \\nATOM   8052  O   LEU C 514     -13.525  12.937  21.748  1.00  0.00           O  \\nATOM   8053  CB  LEU C 514     -12.961  11.519  24.445  1.00  0.00           C  \\nATOM   8054  CG  LEU C 514     -12.180  10.287  24.127  1.00  0.00           C  \\nATOM   8055  CD1 LEU C 514     -10.877  10.189  24.930  1.00  0.00           C  \\nATOM   8056  CD2 LEU C 514     -13.134   9.066  24.395  1.00  0.00           C  \\nATOM   8057  H   LEU C 514     -15.295  10.053  24.709  1.00  0.00           H  \\nATOM   8058  HA  LEU C 514     -14.633  12.750  23.918  1.00  0.00           H  \\nATOM   8059 1HB  LEU C 514     -12.310  12.366  24.236  1.00  0.00           H  \\nATOM   8060 2HB  LEU C 514     -13.162  11.497  25.516  1.00  0.00           H  \\nATOM   8061  HG  LEU C 514     -11.868  10.313  23.083  1.00  0.00           H  \\nATOM   8062 1HD1 LEU C 514     -10.350   9.274  24.658  1.00  0.00           H  \\nATOM   8063 2HD1 LEU C 514     -10.246  11.050  24.707  1.00  0.00           H  \\nATOM   8064 3HD1 LEU C 514     -11.106  10.173  25.995  1.00  0.00           H  \\nATOM   8065 1HD2 LEU C 514     -12.607   8.137  24.175  1.00  0.00           H  \\nATOM   8066 2HD2 LEU C 514     -13.444   9.069  25.440  1.00  0.00           H  \\nATOM   8067 3HD2 LEU C 514     -14.013   9.144  23.756  1.00  0.00           H  \\nATOM   8068  N   PHE C 515     -14.401  10.872  21.388  1.00  0.00           N  \\nATOM   8069  CA  PHE C 515     -14.087  10.889  19.937  1.00  0.00           C  \\nATOM   8070  C   PHE C 515     -15.187  11.420  19.022  1.00  0.00           C  \\nATOM   8071  O   PHE C 515     -15.032  11.567  17.846  1.00  0.00           O  \\nATOM   8072  CB  PHE C 515     -13.557   9.518  19.451  1.00  0.00           C  \\nATOM   8073  CG  PHE C 515     -12.395   9.020  20.234  1.00  0.00           C  \\nATOM   8074  CD1 PHE C 515     -11.140   9.544  19.996  1.00  0.00           C  \\nATOM   8075  CD2 PHE C 515     -12.542   8.049  21.236  1.00  0.00           C  \\nATOM   8076  CE1 PHE C 515     -10.001   9.065  20.692  1.00  0.00           C  \\nATOM   8077  CE2 PHE C 515     -11.418   7.563  21.899  1.00  0.00           C  \\nATOM   8078  CZ  PHE C 515     -10.147   8.095  21.647  1.00  0.00           C  \\nATOM   8079  H   PHE C 515     -14.916  10.092  21.771  1.00  0.00           H  \\nATOM   8080  HA  PHE C 515     -13.310  11.634  19.760  1.00  0.00           H  \\nATOM   8081 1HB  PHE C 515     -14.354   8.778  19.512  1.00  0.00           H  \\nATOM   8082 2HB  PHE C 515     -13.259   9.593  18.406  1.00  0.00           H  \\nATOM   8083  HD1 PHE C 515     -11.026  10.340  19.260  1.00  0.00           H  \\nATOM   8084  HD2 PHE C 515     -13.524   7.623  21.444  1.00  0.00           H  \\nATOM   8085  HE1 PHE C 515      -9.015   9.470  20.466  1.00  0.00           H  \\nATOM   8086  HE2 PHE C 515     -11.537   6.756  22.622  1.00  0.00           H  \\nATOM   8087  HZ  PHE C 515      -9.280   7.741  22.203  1.00  0.00           H  \\nATOM   8088  N   ASP C 516     -16.342  11.739  19.617  1.00  0.00           N  \\nATOM   8089  CA  ASP C 516     -17.472  12.287  18.852  1.00  0.00           C  \\nATOM   8090  C   ASP C 516     -17.462  13.787  18.725  1.00  0.00           C  \\nATOM   8091  O   ASP C 516     -16.724  14.500  19.447  1.00  0.00           O  \\nATOM   8092  OXT ASP C 516     -18.170  14.326  17.920  1.00  0.00           O  \\nATOM   8093  CB  ASP C 516     -18.786  11.877  19.531  1.00  0.00           C  \\nATOM   8094  CG  ASP C 516     -19.067  10.394  19.445  1.00  0.00           C  \\nATOM   8095  OD1 ASP C 516     -19.631   9.843  20.454  1.00  0.00           O  \\nATOM   8096  OD2 ASP C 516     -18.765   9.805  18.360  1.00  0.00           O  \\nATOM   8097  H   ASP C 516     -16.442  11.601  20.613  1.00  0.00           H  \\nATOM   8098  HA  ASP C 516     -17.442  11.871  17.845  1.00  0.00           H  \\nATOM   8099 1HB  ASP C 516     -18.756  12.162  20.583  1.00  0.00           H  \\nATOM   8100 2HB  ASP C 516     -19.616  12.413  19.070  1.00  0.00           H  \\nTER                                                                             \\nATOM   8102  N   GLY D 517      39.878  -2.302  27.465  1.00  0.00           N  \\nATOM   8103  CA  GLY D 517      38.527  -2.946  27.563  1.00  0.00           C  \\nATOM   8104  C   GLY D 517      37.960  -3.299  26.197  1.00  0.00           C  \\nATOM   8105  O   GLY D 517      36.916  -3.941  26.105  1.00  0.00           O  \\nATOM   8106 1H   GLY D 517      40.213  -2.087  28.382  1.00  0.00           H  \\nATOM   8107 2H   GLY D 517      40.515  -2.929  27.017  1.00  0.00           H  \\nATOM   8108 3H   GLY D 517      39.808  -1.460  26.930  1.00  0.00           H  \\nATOM   8109 1HA  GLY D 517      38.598  -3.851  28.167  1.00  0.00           H  \\nATOM   8110 2HA  GLY D 517      37.839  -2.273  28.074  1.00  0.00           H  \\nATOM   8111  N   PHE D 518      38.658  -2.930  25.123  1.00  0.00           N  \\nATOM   8112  CA  PHE D 518      38.118  -3.148  23.781  1.00  0.00           C  \\nATOM   8113  C   PHE D 518      38.488  -4.524  23.230  1.00  0.00           C  \\nATOM   8114  O   PHE D 518      38.064  -4.905  22.150  1.00  0.00           O  \\nATOM   8115  CB  PHE D 518      38.656  -2.102  22.853  1.00  0.00           C  \\nATOM   8116  CG  PHE D 518      38.253  -0.712  23.217  1.00  0.00           C  \\nATOM   8117  CD1 PHE D 518      38.782   0.344  22.518  1.00  0.00           C  \\nATOM   8118  CD2 PHE D 518      37.338  -0.466  24.264  1.00  0.00           C  \\nATOM   8119  CE1 PHE D 518      38.411   1.689  22.831  1.00  0.00           C  \\nATOM   8120  CE2 PHE D 518      36.944   0.884  24.569  1.00  0.00           C  \\nATOM   8121  CZ  PHE D 518      37.526   1.948  23.870  1.00  0.00           C  \\nATOM   8122  H   PHE D 518      39.565  -2.497  25.224  1.00  0.00           H  \\nATOM   8123  HA  PHE D 518      37.031  -3.062  23.827  1.00  0.00           H  \\nATOM   8124 1HB  PHE D 518      39.744  -2.150  22.843  1.00  0.00           H  \\nATOM   8125 2HB  PHE D 518      38.311  -2.303  21.840  1.00  0.00           H  \\nATOM   8126  HD1 PHE D 518      39.493   0.148  21.715  1.00  0.00           H  \\nATOM   8127  HD2 PHE D 518      36.889  -1.300  24.805  1.00  0.00           H  \\nATOM   8128  HE1 PHE D 518      38.823   2.516  22.253  1.00  0.00           H  \\nATOM   8129  HE2 PHE D 518      36.198   1.082  25.339  1.00  0.00           H  \\nATOM   8130  HZ  PHE D 518      37.287   2.976  24.140  1.00  0.00           H  \\nATOM   8131  N   GLY D 519      39.247  -5.318  23.996  1.00  0.00           N  \\nATOM   8132  CA  GLY D 519      39.654  -6.642  23.513  1.00  0.00           C  \\nATOM   8133  C   GLY D 519      38.568  -7.650  23.303  1.00  0.00           C  \\nATOM   8134  O   GLY D 519      38.701  -8.563  22.522  1.00  0.00           O  \\nATOM   8135  H   GLY D 519      39.547  -5.015  24.912  1.00  0.00           H  \\nATOM   8136 1HA  GLY D 519      40.173  -6.538  22.560  1.00  0.00           H  \\nATOM   8137 2HA  GLY D 519      40.359  -7.084  24.216  1.00  0.00           H  \\nATOM   8138  N   LEU D 520      37.424  -7.419  23.941  1.00  0.00           N  \\nATOM   8139  CA  LEU D 520      36.193  -8.141  23.678  1.00  0.00           C  \\nATOM   8140  C   LEU D 520      35.882  -8.413  22.195  1.00  0.00           C  \\nATOM   8141  O   LEU D 520      35.382  -9.485  21.876  1.00  0.00           O  \\nATOM   8142  CB  LEU D 520      35.105  -7.251  24.229  1.00  0.00           C  \\nATOM   8143  CG  LEU D 520      33.828  -7.971  24.403  1.00  0.00           C  \\nATOM   8144  CD1 LEU D 520      34.050  -9.182  25.384  1.00  0.00           C  \\nATOM   8145  CD2 LEU D 520      32.687  -7.040  24.899  1.00  0.00           C  \\nATOM   8146  H   LEU D 520      37.429  -6.696  24.646  1.00  0.00           H  \\nATOM   8147  HA  LEU D 520      36.232  -9.094  24.205  1.00  0.00           H  \\nATOM   8148 1HB  LEU D 520      35.433  -6.856  25.189  1.00  0.00           H  \\nATOM   8149 2HB  LEU D 520      34.961  -6.415  23.545  1.00  0.00           H  \\nATOM   8150  HG  LEU D 520      33.520  -8.402  23.450  1.00  0.00           H  \\nATOM   8151 1HD1 LEU D 520      33.111  -9.719  25.517  1.00  0.00           H  \\nATOM   8152 2HD1 LEU D 520      34.797  -9.857  24.966  1.00  0.00           H  \\nATOM   8153 3HD1 LEU D 520      34.395  -8.810  26.348  1.00  0.00           H  \\nATOM   8154 1HD2 LEU D 520      31.769  -7.617  25.010  1.00  0.00           H  \\nATOM   8155 2HD2 LEU D 520      32.963  -6.607  25.861  1.00  0.00           H  \\nATOM   8156 3HD2 LEU D 520      32.528  -6.241  24.174  1.00  0.00           H  \\nATOM   8157  N   PHE D 521      36.249  -7.499  21.305  1.00  0.00           N  \\nATOM   8158  CA  PHE D 521      35.868  -7.614  19.861  1.00  0.00           C  \\nATOM   8159  C   PHE D 521      36.974  -8.126  18.930  1.00  0.00           C  \\nATOM   8160  O   PHE D 521      36.755  -8.306  17.754  1.00  0.00           O  \\nATOM   8161  CB  PHE D 521      35.375  -6.244  19.347  1.00  0.00           C  \\nATOM   8162  CG  PHE D 521      34.193  -5.696  20.149  1.00  0.00           C  \\nATOM   8163  CD1 PHE D 521      32.899  -6.269  19.969  1.00  0.00           C  \\nATOM   8164  CD2 PHE D 521      34.394  -4.747  21.216  1.00  0.00           C  \\nATOM   8165  CE1 PHE D 521      31.831  -5.807  20.677  1.00  0.00           C  \\nATOM   8166  CE2 PHE D 521      33.295  -4.286  21.898  1.00  0.00           C  \\nATOM   8167  CZ  PHE D 521      32.012  -4.821  21.638  1.00  0.00           C  \\nATOM   8168  H   PHE D 521      36.798  -6.707  21.607  1.00  0.00           H  \\nATOM   8169  HA  PHE D 521      35.059  -8.341  19.773  1.00  0.00           H  \\nATOM   8170 1HB  PHE D 521      36.191  -5.524  19.393  1.00  0.00           H  \\nATOM   8171 2HB  PHE D 521      35.076  -6.333  18.303  1.00  0.00           H  \\nATOM   8172  HD1 PHE D 521      32.772  -7.085  19.257  1.00  0.00           H  \\nATOM   8173  HD2 PHE D 521      35.379  -4.328  21.423  1.00  0.00           H  \\nATOM   8174  HE1 PHE D 521      30.837  -6.212  20.489  1.00  0.00           H  \\nATOM   8175  HE2 PHE D 521      33.416  -3.501  22.644  1.00  0.00           H  \\nATOM   8176  HZ  PHE D 521      31.160  -4.447  22.204  1.00  0.00           H  \\nATOM   8177  N   ASP D 522      38.149  -8.466  19.509  1.00  0.00           N  \\nATOM   8178  CA  ASP D 522      39.284  -9.002  18.745  1.00  0.00           C  \\nATOM   8179  C   ASP D 522      39.299 -10.525  18.593  1.00  0.00           C  \\nATOM   8180  O   ASP D 522      38.530 -11.239  19.239  1.00  0.00           O  \\nATOM   8181  OXT ASP D 522      40.069 -11.045  17.834  1.00  0.00           O  \\nATOM   8182  CB  ASP D 522      40.575  -8.624  19.432  1.00  0.00           C  \\nATOM   8183  CG  ASP D 522      40.845  -7.173  19.396  1.00  0.00           C  \\nATOM   8184  OD1 ASP D 522      40.405  -6.532  18.442  1.00  0.00           O  \\nATOM   8185  OD2 ASP D 522      41.488  -6.665  20.360  1.00  0.00           O  \\nATOM   8186  H   ASP D 522      38.242  -8.343  20.507  1.00  0.00           H  \\nATOM   8187  HA  ASP D 522      39.268  -8.563  17.747  1.00  0.00           H  \\nATOM   8188 1HB  ASP D 522      40.541  -8.945  20.473  1.00  0.00           H  \\nATOM   8189 2HB  ASP D 522      41.407  -9.144  18.956  1.00  0.00           H  \\nTER                                                                             \\nHETATM 8191 C05  LG1 E 523     -14.825   3.833   3.998  1.00  0.00           C  \\nHETATM 8192 C06  LG1 E 523     -14.177   3.099   3.023  1.00  0.00           C  \\nHETATM 8193 C01  LG1 E 523     -13.256   3.701   2.173  1.00  0.00           C  \\nHETATM 8194 N14  LG1 E 523     -12.638   2.960   1.175  1.00  0.00           N  \\nHETATM 8195 C13  LG1 E 523     -11.863   3.560   0.196  1.00  0.00           C  \\nHETATM 8196 C18  LG1 E 523     -11.488   2.889  -0.957  1.00  0.00           C  \\nHETATM 8197 C17  LG1 E 523     -10.687   3.508  -1.907  1.00  0.00           C  \\nHETATM 8198 C16  LG1 E 523     -10.236   4.799  -1.732  1.00  0.00           C  \\nHETATM 8199 C15  LG1 E 523     -10.601   5.498  -0.586  1.00  0.00           C  \\nHETATM 8200 C19  LG1 E 523     -10.141   6.863  -0.307  1.00  0.00           C  \\nHETATM 8201 C20  LG1 E 523      -9.372   7.572  -1.229  1.00  0.00           C  \\nHETATM 8202 C21  LG1 E 523      -8.964   8.855  -0.934  1.00  0.00           C  \\nHETATM 8203 F30  LG1 E 523      -8.217   9.545  -1.824  1.00  0.00           F  \\nHETATM 8204 C22  LG1 E 523      -9.320   9.440   0.276  1.00  0.00           C  \\nHETATM 8205 C23  LG1 E 523     -10.067   8.760   1.195  1.00  0.00           C  \\nHETATM 8206 C24  LG1 E 523     -10.486   7.448   0.937  1.00  0.00           C  \\nHETATM 8207 NP5  LG1 E 523     -11.185   6.738   1.843  1.00  0.00           N  \\nHETATM 8208 C11  LG1 E 523     -11.865   5.657   1.548  1.00  0.00           C  \\nHETATM 8209 C02  LG1 E 523     -12.948   5.066   2.353  1.00  0.00           C  \\nHETATM 8210 C03  LG1 E 523     -13.654   5.801   3.301  1.00  0.00           C  \\nHETATM 8211 C04  LG1 E 523     -14.576   5.189   4.127  1.00  0.00           C  \\nHETATM 8212 F08  LG1 E 523     -15.226   5.912   5.065  1.00  0.00           F  \\nHETATM 8213 C12  LG1 E 523     -11.440   4.888   0.366  1.00  0.00           C  \\nHETATM 8214 C34  LG1 E 523     -11.041   7.131   3.250  1.00  0.00           C  \\nHETATM 8215 C27  LG1 E 523     -10.302   2.749  -3.152  1.00  0.00           C  \\nHETATM 8216 C33  LG1 E 523     -12.833   1.509   1.130  1.00  0.00           C  \\nHETATM 8217 H09  LG1 E 523     -15.523   3.345   4.660  1.00  0.00           H  \\nHETATM 8218 H10  LG1 E 523     -14.423   2.049   2.951  1.00  0.00           H  \\nHETATM 8219 H26  LG1 E 523     -11.817   1.880  -1.154  1.00  0.00           H  \\nHETATM 8220 H28  LG1 E 523      -9.607   5.260  -2.478  1.00  0.00           H  \\nHETATM 8221 H29  LG1 E 523      -9.095   7.121  -2.171  1.00  0.00           H  \\nHETATM 8222 H31  LG1 E 523      -9.005  10.450   0.494  1.00  0.00           H  \\nHETATM 8223 H32  LG1 E 523     -10.333   9.272   2.107  1.00  0.00           H  \\nHETATM 8224 H07  LG1 E 523     -13.535   6.873   3.376  1.00  0.00           H  \\nHETATM 8225 H35  LG1 E 523     -11.795   7.879   3.498  1.00  0.00           H  \\nHETATM 8226 H36  LG1 E 523     -11.173   6.257   3.887  1.00  0.00           H  \\nHETATM 8227 H37  LG1 E 523     -10.048   7.551   3.410  1.00  0.00           H  \\nHETATM 8228 H38  LG1 E 523     -11.048   2.922  -3.927  1.00  0.00           H  \\nHETATM 8229 H39  LG1 E 523      -9.329   3.093  -3.500  1.00  0.00           H  \\nHETATM 8230 H40  LG1 E 523     -10.252   1.683  -2.927  1.00  0.00           H  \\nHETATM 8231 H41  LG1 E 523     -13.752   1.283   0.588  1.00  0.00           H  \\nHETATM 8232 H42  LG1 E 523     -11.988   1.045   0.622  1.00  0.00           H  \\nHETATM 8233 H43  LG1 E 523     -12.904   1.120   2.145  1.00  0.00           H  \\nTER                                                                             \\nCONECT 8191 8192 8211 8217                                                      \\nCONECT 8192 8191 8193 8218                                                      \\nCONECT 8193 8192 8194 8209                                                      \\nCONECT 8194 8193 8195 8216                                                      \\nCONECT 8195 8194 8196 8213                                                      \\nCONECT 8196 8195 8197 8219                                                      \\nCONECT 8197 8196 8198 8215                                                      \\nCONECT 8198 8197 8199 8220                                                      \\nCONECT 8199 8198 8200 8213                                                      \\nCONECT 8200 8199 8201 8206                                                      \\nCONECT 8201 8200 8202 8221                                                      \\nCONECT 8202 8201 8203 8204                                                      \\nCONECT 8203 8202                                                                \\nCONECT 8204 8202 8205 8222                                                      \\nCONECT 8205 8204 8206 8223                                                      \\nCONECT 8206 8200 8205 8207                                                      \\nCONECT 8207 8206 8208 8214                                                      \\nCONECT 8208 8207 8209 8213                                                      \\nCONECT 8209 8193 8208 8210                                                      \\nCONECT 8210 8209 8211 8224                                                      \\nCONECT 8211 8191 8210 8212                                                      \\nCONECT 8212 8211                                                                \\nCONECT 8213 8195 8199 8208                                                      \\nCONECT 8214 8207 8225 8226 8227                                                 \\nCONECT 8215 8197 8228 8229 8230                                                 \\nCONECT 8216 8194 8231 8232 8233                                                 \\nCONECT 8217 8191                                                                \\nCONECT 8218 8192                                                                \\nCONECT 8219 8196                                                                \\nCONECT 8220 8198                                                                \\nCONECT 8221 8201                                                                \\nCONECT 8222 8204                                                                \\nCONECT 8223 8205                                                                \\nCONECT 8224 8210                                                                \\nCONECT 8225 8214                                                                \\nCONECT 8226 8214                                                                \\nCONECT 8227 8214                                                                \\nCONECT 8228 8215                                                                \\nCONECT 8229 8215                                                                \\nCONECT 8230 8215                                                                \\nCONECT 8231 8216                                                                \\nCONECT 8232 8216                                                                \\nCONECT 8233 8216                                                                \\n\",\"pdb\");\n\tviewer_15942021952474504.zoomTo();\n\tviewer_15942021952474504.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"blackCarbon\", \"radius\": \"0.1\"}});\n\tviewer_15942021952474504.setStyle({\"hetflag\": true},{\"stick\": {\"colorscheme\": \"brownCarbon\", \"radius\": 0.2}});\n\tviewer_15942021952474504.addSurface(1,{\"opacity\": 0.7, \"color\": \"white\"},{\"resi\": [523], \"chain\": [\"E\"]});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.847, \"y\": -18.878, \"z\": 21.285}, \"end\": {\"x\": -2.334, \"y\": -20.299, \"z\": 22.404}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.688, \"y\": -19.457, \"z\": 17.396}, \"end\": {\"x\": -7.746, \"y\": -18.875, \"z\": 18.961}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.715, \"y\": -18.812, \"z\": 16.343}, \"end\": {\"x\": -9.079, \"y\": -19.532, \"z\": 15.408}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.524, \"y\": -19.004, \"z\": 12.445}, \"end\": {\"x\": -12.045, \"y\": -18.462, \"z\": 14.341}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.823, \"y\": -18.434, \"z\": 10.945}, \"end\": {\"x\": -15.714, \"y\": -19.85, \"z\": 11.981}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.122, \"y\": -15.85, \"z\": 12.289}, \"end\": {\"x\": -18.639, \"y\": -15.854, \"z\": 10.743}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.677, \"y\": -13.818, \"z\": 10.316}, \"end\": {\"x\": -18.639, \"y\": -15.854, \"z\": 10.743}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.823, \"y\": -18.434, \"z\": 10.945}, \"end\": {\"x\": -15.714, \"y\": -19.85, \"z\": 11.981}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.122, \"y\": -15.85, \"z\": 12.289}, \"end\": {\"x\": -18.639, \"y\": -15.854, \"z\": 10.743}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.616, \"y\": -12.287, \"z\": 18.717}, \"end\": {\"x\": -20.227, \"y\": -11.823, \"z\": 17.569}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.664, \"y\": -14.14, \"z\": 16.663}, \"end\": {\"x\": -19.099, \"y\": -14.953, \"z\": 17.657}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.721, \"y\": -13.499, \"z\": 19.531}, \"end\": {\"x\": -22.488, \"y\": -14.018, \"z\": 18.416}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.982, \"y\": -10.574, \"z\": 20.647}, \"end\": {\"x\": -19.808, \"y\": -10.112, \"z\": 20.355}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.091, \"y\": -12.531, \"z\": 25.185}, \"end\": {\"x\": -18.294, \"y\": -13.4, \"z\": 25.732}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.453, \"y\": -12.589, \"z\": 22.732}, \"end\": {\"x\": -19.163, \"y\": -12.933, \"z\": 22.251}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.664, \"y\": -14.14, \"z\": 16.663}, \"end\": {\"x\": -19.099, \"y\": -14.953, \"z\": 17.657}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.721, \"y\": -13.499, \"z\": 19.531}, \"end\": {\"x\": -22.488, \"y\": -14.018, \"z\": 18.416}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.155, \"y\": -13.551, \"z\": 21.66}, \"end\": {\"x\": -16.578, \"y\": -14.194, \"z\": 20.203}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.616, \"y\": -12.287, \"z\": 18.717}, \"end\": {\"x\": -20.227, \"y\": -11.823, \"z\": 17.569}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.3, \"y\": -11.427, \"z\": 21.614}, \"end\": {\"x\": -14.896, \"y\": -11.157, \"z\": 20.558}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.982, \"y\": -10.574, \"z\": 20.647}, \"end\": {\"x\": -19.808, \"y\": -10.112, \"z\": 20.355}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.137, \"y\": -11.097, \"z\": 24.321}, \"end\": {\"x\": -15.037, \"y\": -10.837, \"z\": 23.803}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.091, \"y\": -12.531, \"z\": 25.185}, \"end\": {\"x\": -18.294, \"y\": -13.4, \"z\": 25.732}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.453, \"y\": -12.589, \"z\": 22.732}, \"end\": {\"x\": -19.163, \"y\": -12.933, \"z\": 22.251}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.905, \"y\": -13.459, \"z\": 24.993}, \"end\": {\"x\": -13.721, \"y\": -13.92, \"z\": 24.398}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.155, \"y\": -13.551, \"z\": 21.66}, \"end\": {\"x\": -16.578, \"y\": -14.194, \"z\": 20.203}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.441, \"y\": -13.258, \"z\": 23.809}, \"end\": {\"x\": -11.024, \"y\": -13.337, \"z\": 22.625}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.3, \"y\": -11.427, \"z\": 21.614}, \"end\": {\"x\": -14.896, \"y\": -11.157, \"z\": 20.558}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.288, \"y\": -10.954, \"z\": 25.149}, \"end\": {\"x\": -10.003, \"y\": -10.512, \"z\": 24.251}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.137, \"y\": -11.097, \"z\": 24.321}, \"end\": {\"x\": -15.037, \"y\": -10.837, \"z\": 23.803}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.253, \"y\": -12.171, \"z\": 27.676}, \"end\": {\"x\": -10.189, \"y\": -12.148, \"z\": 27.305}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.905, \"y\": -13.459, \"z\": 24.993}, \"end\": {\"x\": -13.721, \"y\": -13.92, \"z\": 24.398}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.214, \"y\": -13.955, \"z\": 27.31}, \"end\": {\"x\": -7.926, \"y\": -14.601, \"z\": 26.216}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.441, \"y\": -13.258, \"z\": 23.809}, \"end\": {\"x\": -11.024, \"y\": -13.337, \"z\": 22.625}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.207, \"y\": -12.624, \"z\": 27.194}, \"end\": {\"x\": -5.563, \"y\": -12.476, \"z\": 25.456}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.288, \"y\": -10.954, \"z\": 25.149}, \"end\": {\"x\": -10.003, \"y\": -10.512, \"z\": 24.251}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.586, \"y\": -5.825, \"z\": 25.783}, \"end\": {\"x\": -13.59, \"y\": -5.122, \"z\": 24.075}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.777, \"y\": -4.718, \"z\": 24.065}, \"end\": {\"x\": -13.59, \"y\": -5.122, \"z\": 24.075}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.075, \"y\": -4.761, \"z\": 23.66}, \"end\": {\"x\": -9.416, \"y\": -4.917, \"z\": 21.754}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.386, \"y\": -11.721, \"z\": 29.308}, \"end\": {\"x\": -5.444, \"y\": -10.884, \"z\": 28.462}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.637, \"y\": -5.949, \"z\": 24.982}, \"end\": {\"x\": -7.049, \"y\": -5.334, \"z\": 24.214}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.253, \"y\": -12.171, \"z\": 27.676}, \"end\": {\"x\": -10.189, \"y\": -12.148, \"z\": 27.305}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.214, \"y\": -13.955, \"z\": 27.31}, \"end\": {\"x\": -7.926, \"y\": -14.601, \"z\": 26.216}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.207, \"y\": -12.624, \"z\": 27.194}, \"end\": {\"x\": -5.563, \"y\": -12.476, \"z\": 25.456}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.386, \"y\": -11.721, \"z\": 29.308}, \"end\": {\"x\": -5.444, \"y\": -10.884, \"z\": 28.462}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.679, \"y\": -10.388, \"z\": 36.665}, \"end\": {\"x\": -0.145, \"y\": -11.095, \"z\": 35.517}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.679, \"y\": -10.388, \"z\": 36.665}, \"end\": {\"x\": -0.145, \"y\": -11.095, \"z\": 35.517}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.942, \"y\": -2.119, \"z\": 36.475}, \"end\": {\"x\": 1.575, \"y\": -3.907, \"z\": 36.75}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.815, \"y\": -3.185, \"z\": 35.8}, \"end\": {\"x\": 3.052, \"y\": -4.915, \"z\": 35.527}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.422, \"y\": -4.909, \"z\": 33.317}, \"end\": {\"x\": -1.425, \"y\": -5.26, \"z\": 35.203}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.942, \"y\": -2.119, \"z\": 36.475}, \"end\": {\"x\": 1.575, \"y\": -3.907, \"z\": 36.75}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.815, \"y\": -3.185, \"z\": 35.8}, \"end\": {\"x\": 3.052, \"y\": -4.915, \"z\": 35.527}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.618, \"y\": -1.287, \"z\": 35.372}, \"end\": {\"x\": -3.241, \"y\": -1.689, \"z\": 33.658}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.169, \"y\": 0.139, \"z\": 34.954}, \"end\": {\"x\": -3.5, \"y\": 1.199, \"z\": 35.067}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.618, \"y\": -1.287, \"z\": 35.372}, \"end\": {\"x\": -3.241, \"y\": -1.689, \"z\": 33.658}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.169, \"y\": 0.139, \"z\": 34.954}, \"end\": {\"x\": -3.5, \"y\": 1.199, \"z\": 35.067}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.481, \"y\": -3.161, \"z\": 28.826}, \"end\": {\"x\": -2.518, \"y\": -3.408, \"z\": 30.737}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.422, \"y\": -4.909, \"z\": 33.317}, \"end\": {\"x\": -1.425, \"y\": -5.26, \"z\": 35.203}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.033, \"y\": -4.935, \"z\": 29.523}, \"end\": {\"x\": 1.041, \"y\": -3.403, \"z\": 28.444}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.325, \"y\": -7.568, \"z\": 29.475}, \"end\": {\"x\": 10.222, \"y\": -6.216, \"z\": 30.53}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.728, \"y\": -12.023, \"z\": 29.474}, \"end\": {\"x\": 11.793, \"y\": -10.327, \"z\": 28.078}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.441, \"y\": -8.357, \"z\": 22.091}, \"end\": {\"x\": 12.03, \"y\": -9.976, \"z\": 20.073}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.424, \"y\": -8.635, \"z\": 20.923}, \"end\": {\"x\": 12.03, \"y\": -9.976, \"z\": 20.073}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.325, \"y\": -7.568, \"z\": 29.475}, \"end\": {\"x\": 10.222, \"y\": -6.216, \"z\": 30.53}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.728, \"y\": -12.023, \"z\": 29.474}, \"end\": {\"x\": 11.793, \"y\": -10.327, \"z\": 28.078}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.458, \"y\": -18.535, \"z\": 27.87}, \"end\": {\"x\": 8.907, \"y\": -17.132, \"z\": 28.015}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.549, \"y\": -17.649, \"z\": 25.574}, \"end\": {\"x\": 8.144, \"y\": -18.987, \"z\": 25.041}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.458, \"y\": -18.535, \"z\": 27.87}, \"end\": {\"x\": 8.907, \"y\": -17.132, \"z\": 28.015}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.549, \"y\": -17.649, \"z\": 25.574}, \"end\": {\"x\": 8.144, \"y\": -18.987, \"z\": 25.041}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.689, \"y\": -11.525, \"z\": 22.746}, \"end\": {\"x\": 5.923, \"y\": -10.752, \"z\": 22.538}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.851, \"y\": -10.472, \"z\": 21.453}, \"end\": {\"x\": 7.114, \"y\": -10.358, \"z\": 20.663}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.727, \"y\": -12.768, \"z\": 23.45}, \"end\": {\"x\": 4.979, \"y\": -14.185, \"z\": 23.62}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.2, \"y\": -15.55, \"z\": 21.715}, \"end\": {\"x\": 2.047, \"y\": -13.99, \"z\": 20.753}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.464, \"y\": -15.139, \"z\": 18.862}, \"end\": {\"x\": -0.129, \"y\": -16.591, \"z\": 19.917}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.847, \"y\": -18.878, \"z\": 21.285}, \"end\": {\"x\": -2.334, \"y\": -20.299, \"z\": 22.404}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.023, \"y\": -17.565, \"z\": 17.221}, \"end\": {\"x\": -1.469, \"y\": -17.184, \"z\": 16.217}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.59, \"y\": -17.465, \"z\": 14.078}, \"end\": {\"x\": -5.015, \"y\": -17.971, \"z\": 15.247}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.688, \"y\": -19.457, \"z\": 17.396}, \"end\": {\"x\": -7.746, \"y\": -18.875, \"z\": 18.961}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.715, \"y\": -18.812, \"z\": 16.343}, \"end\": {\"x\": -9.079, \"y\": -19.532, \"z\": 15.408}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.029, \"y\": -19.041, \"z\": 13.096}, \"end\": {\"x\": -5.7, \"y\": -19.696, \"z\": 11.925}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.092, \"y\": -20.11, \"z\": 10.075}, \"end\": {\"x\": -8.895, \"y\": -19.938, \"z\": 11.049}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.524, \"y\": -19.004, \"z\": 12.445}, \"end\": {\"x\": -12.045, \"y\": -18.462, \"z\": 14.341}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.155, \"y\": -17.004, \"z\": 5.453}, \"end\": {\"x\": -12.008, \"y\": -14.916, \"z\": 4.802}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.905, \"y\": -20.806, \"z\": 8.516}, \"end\": {\"x\": -10.68, \"y\": -21.524, \"z\": 7.93}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.288, \"y\": -22.031, \"z\": 5.811}, \"end\": {\"x\": -12.771, \"y\": -19.997, \"z\": 6.023}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.656, \"y\": -18.385, \"z\": 4.534}, \"end\": {\"x\": -11.143, \"y\": -17.701, \"z\": 2.535}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.284, \"y\": -18.252, \"z\": 7.081}, \"end\": {\"x\": -16.076, \"y\": -16.514, \"z\": 6.454}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.905, \"y\": -20.806, \"z\": 8.516}, \"end\": {\"x\": -10.68, \"y\": -21.524, \"z\": 7.93}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.288, \"y\": -22.031, \"z\": 5.811}, \"end\": {\"x\": -12.771, \"y\": -19.997, \"z\": 6.023}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.029, \"y\": -19.041, \"z\": 13.096}, \"end\": {\"x\": -5.7, \"y\": -19.696, \"z\": 11.925}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.092, \"y\": -20.11, \"z\": 10.075}, \"end\": {\"x\": -8.895, \"y\": -19.938, \"z\": 11.049}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.776, \"y\": -20.332, \"z\": 11.067}, \"end\": {\"x\": -1.666, \"y\": -20.916, \"z\": 9.52}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.128, \"y\": -19.448, \"z\": 11.319}, \"end\": {\"x\": -1.136, \"y\": -18.74, \"z\": 12.682}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.023, \"y\": -17.565, \"z\": 17.221}, \"end\": {\"x\": -1.469, \"y\": -17.184, \"z\": 16.217}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.59, \"y\": -17.465, \"z\": 14.078}, \"end\": {\"x\": -5.015, \"y\": -17.971, \"z\": 15.247}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.741, \"y\": -12.512, \"z\": 15.496}, \"end\": {\"x\": 1.581, \"y\": -13.72, \"z\": 16.236}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.732, \"y\": -15.915, \"z\": 14.434}, \"end\": {\"x\": 1.488, \"y\": -15.284, \"z\": 12.801}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.2, \"y\": -15.55, \"z\": 21.715}, \"end\": {\"x\": 2.047, \"y\": -13.99, \"z\": 20.753}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.464, \"y\": -15.139, \"z\": 18.862}, \"end\": {\"x\": -0.129, \"y\": -16.591, \"z\": 19.917}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.689, \"y\": -11.525, \"z\": 22.746}, \"end\": {\"x\": 5.923, \"y\": -10.752, \"z\": 22.538}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.851, \"y\": -10.472, \"z\": 21.453}, \"end\": {\"x\": 7.114, \"y\": -10.358, \"z\": 20.663}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.702, \"y\": -11.299, \"z\": 19.239}, \"end\": {\"x\": 1.927, \"y\": -10.189, \"z\": 17.829}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.69, \"y\": -10.329, \"z\": 23.985}, \"end\": {\"x\": 5.923, \"y\": -10.752, \"z\": 22.538}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.457, \"y\": -8.328, \"z\": 22.915}, \"end\": {\"x\": 2.334, \"y\": -10.282, \"z\": 22.183}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.727, \"y\": -12.768, \"z\": 23.45}, \"end\": {\"x\": 4.979, \"y\": -14.185, \"z\": 23.62}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.69, \"y\": -10.329, \"z\": 23.985}, \"end\": {\"x\": 5.923, \"y\": -10.752, \"z\": 22.538}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.457, \"y\": -8.328, \"z\": 22.915}, \"end\": {\"x\": 2.334, \"y\": -10.282, \"z\": 22.183}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.75, \"y\": -4.404, \"z\": 21.029}, \"end\": {\"x\": 1.023, \"y\": -5.709, \"z\": 21.431}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.297, \"y\": -4.328, \"z\": 20.1}, \"end\": {\"x\": 1.023, \"y\": -5.709, \"z\": 21.431}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.864, \"y\": -8.28, \"z\": 19.979}, \"end\": {\"x\": 3.737, \"y\": -7.631, \"z\": 20.448}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.702, \"y\": -11.299, \"z\": 19.239}, \"end\": {\"x\": 1.927, \"y\": -10.189, \"z\": 17.829}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.864, \"y\": -8.28, \"z\": 19.979}, \"end\": {\"x\": 3.737, \"y\": -7.631, \"z\": 20.448}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.897, \"y\": -4.402, \"z\": 12.11}, \"end\": {\"x\": 3.038, \"y\": -4.789, \"z\": 10.934}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.741, \"y\": -12.512, \"z\": 15.496}, \"end\": {\"x\": 1.581, \"y\": -13.72, \"z\": 16.236}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.188, \"y\": -12.311, \"z\": 10.6}, \"end\": {\"x\": 5.631, \"y\": -13.001, \"z\": 12.595}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.732, \"y\": -15.915, \"z\": 14.434}, \"end\": {\"x\": 1.488, \"y\": -15.284, \"z\": 12.801}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.665, \"y\": -17.298, \"z\": 4.314}, \"end\": {\"x\": -2.395, \"y\": -17.859, \"z\": 3.793}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.056, \"y\": -16.616, \"z\": 10.387}, \"end\": {\"x\": 4.619, \"y\": -16.069, \"z\": 9.557}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.144, \"y\": -18.547, \"z\": 8.87}, \"end\": {\"x\": 2.454, \"y\": -18.778, \"z\": 9.646}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.776, \"y\": -20.332, \"z\": 11.067}, \"end\": {\"x\": -1.666, \"y\": -20.916, \"z\": 9.52}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.128, \"y\": -19.448, \"z\": 11.319}, \"end\": {\"x\": -1.136, \"y\": -18.74, \"z\": 12.682}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.716, \"y\": -25.919, \"z\": 10.077}, \"end\": {\"x\": 0.389, \"y\": -26.281, \"z\": 8.382}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.621, \"y\": -21.866, \"z\": 7.608}, \"end\": {\"x\": 2.499, \"y\": -22.462, \"z\": 7.581}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.844, \"y\": -24.584, \"z\": 6.516}, \"end\": {\"x\": 0.013, \"y\": -23.734, \"z\": 5.973}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.716, \"y\": -25.919, \"z\": 10.077}, \"end\": {\"x\": 0.389, \"y\": -26.281, \"z\": 8.382}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.778, \"y\": -25.959, \"z\": 6.503}, \"end\": {\"x\": 4.411, \"y\": -26.316, \"z\": 5.869}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.621, \"y\": -21.866, \"z\": 7.608}, \"end\": {\"x\": 2.499, \"y\": -22.462, \"z\": 7.581}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.844, \"y\": -24.584, \"z\": 6.516}, \"end\": {\"x\": 0.013, \"y\": -23.734, \"z\": 5.973}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.307, \"y\": -19.53, \"z\": 6.932}, \"end\": {\"x\": 6.404, \"y\": -19.926, \"z\": 7.141}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.056, \"y\": -16.616, \"z\": 10.387}, \"end\": {\"x\": 4.619, \"y\": -16.069, \"z\": 9.557}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.144, \"y\": -18.547, \"z\": 8.87}, \"end\": {\"x\": 2.454, \"y\": -18.778, \"z\": 9.646}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.396, \"y\": -15.788, \"z\": 8.051}, \"end\": {\"x\": 8.561, \"y\": -17.042, \"z\": 7.158}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.863, \"y\": -13.376, \"z\": 6.602}, \"end\": {\"x\": 8.157, \"y\": -13.304, \"z\": 8.325}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.188, \"y\": -12.311, \"z\": 10.6}, \"end\": {\"x\": 5.631, \"y\": -13.001, \"z\": 12.595}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.261, \"y\": -9.545, \"z\": 8.433}, \"end\": {\"x\": 8.596, \"y\": -8.49, \"z\": 7.052}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.054, \"y\": -10.671, \"z\": 18.67}, \"end\": {\"x\": 15.781, \"y\": -8.838, \"z\": 19.048}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.056, \"y\": -14.633, \"z\": 17.537}, \"end\": {\"x\": 17.443, \"y\": -13.377, \"z\": 17.699}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.891, \"y\": -15.129, \"z\": 19.062}, \"end\": {\"x\": 14.745, \"y\": -13.614, \"z\": 19.829}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.36, \"y\": -10.571, \"z\": 21.697}, \"end\": {\"x\": 20.329, \"y\": -11.416, \"z\": 21.47}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.055, \"y\": -8.147, \"z\": 21.295}, \"end\": {\"x\": 19.062, \"y\": -9.623, \"z\": 21.045}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.441, \"y\": -8.357, \"z\": 22.091}, \"end\": {\"x\": 12.03, \"y\": -9.976, \"z\": 20.073}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.424, \"y\": -8.635, \"z\": 20.923}, \"end\": {\"x\": 12.03, \"y\": -9.976, \"z\": 20.073}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.054, \"y\": -10.671, \"z\": 18.67}, \"end\": {\"x\": 15.781, \"y\": -8.838, \"z\": 19.048}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.734, \"y\": -15.767, \"z\": 17.233}, \"end\": {\"x\": 12.432, \"y\": -13.746, \"z\": 17.086}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.056, \"y\": -14.633, \"z\": 17.537}, \"end\": {\"x\": 17.443, \"y\": -13.377, \"z\": 17.699}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.463, \"y\": -18.507, \"z\": 17.008}, \"end\": {\"x\": 13.87, \"y\": -18.289, \"z\": 18.506}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.891, \"y\": -15.129, \"z\": 19.062}, \"end\": {\"x\": 14.745, \"y\": -13.614, \"z\": 19.829}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.706, \"y\": -19.93, \"z\": 17.997}, \"end\": {\"x\": 10.72, \"y\": -18.063, \"z\": 19.12}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.734, \"y\": -15.767, \"z\": 17.233}, \"end\": {\"x\": 12.432, \"y\": -13.746, \"z\": 17.086}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.742, \"y\": -20.592, \"z\": 16.884}, \"end\": {\"x\": 9.724, \"y\": -19.234, \"z\": 15.426}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.463, \"y\": -18.507, \"z\": 17.008}, \"end\": {\"x\": 13.87, \"y\": -18.289, \"z\": 18.506}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.706, \"y\": -19.93, \"z\": 17.997}, \"end\": {\"x\": 10.72, \"y\": -18.063, \"z\": 19.12}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.742, \"y\": -20.592, \"z\": 16.884}, \"end\": {\"x\": 9.724, \"y\": -19.234, \"z\": 15.426}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.302, \"y\": -23.183, \"z\": 13.402}, \"end\": {\"x\": 7.367, \"y\": -22.322, \"z\": 14.934}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.168, \"y\": -21.224, \"z\": 14.097}, \"end\": {\"x\": 7.367, \"y\": -22.322, \"z\": 14.934}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.329, \"y\": -27.025, \"z\": 12.085}, \"end\": {\"x\": 6.447, \"y\": -24.926, \"z\": 13.054}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.329, \"y\": -27.025, \"z\": 12.085}, \"end\": {\"x\": 6.447, \"y\": -24.926, \"z\": 13.054}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.778, \"y\": -25.959, \"z\": 6.503}, \"end\": {\"x\": 4.411, \"y\": -26.316, \"z\": 5.869}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.302, \"y\": -23.183, \"z\": 13.402}, \"end\": {\"x\": 7.367, \"y\": -22.322, \"z\": 14.934}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.168, \"y\": -21.224, \"z\": 14.097}, \"end\": {\"x\": 7.367, \"y\": -22.322, \"z\": 14.934}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.307, \"y\": -19.53, \"z\": 6.932}, \"end\": {\"x\": 6.404, \"y\": -19.926, \"z\": 7.141}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.396, \"y\": -15.788, \"z\": 8.051}, \"end\": {\"x\": 8.561, \"y\": -17.042, \"z\": 7.158}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.863, \"y\": -13.376, \"z\": 6.602}, \"end\": {\"x\": 8.157, \"y\": -13.304, \"z\": 8.325}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.261, \"y\": -9.545, \"z\": 8.433}, \"end\": {\"x\": 8.596, \"y\": -8.49, \"z\": 7.052}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.897, \"y\": -4.402, \"z\": 12.11}, \"end\": {\"x\": 3.038, \"y\": -4.789, \"z\": 10.934}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.391, \"y\": -0.864, \"z\": 5.261}, \"end\": {\"x\": 1.679, \"y\": -0.027, \"z\": 6.908}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.693, \"y\": -0.027, \"z\": 3.067}, \"end\": {\"x\": 3.815, \"y\": 1.042, \"z\": 4.723}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.334, \"y\": -2.277, \"z\": 1.623}, \"end\": {\"x\": 4.109, \"y\": -2.279, \"z\": 3.41}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.601, \"y\": -2.923, \"z\": 1.177}, \"end\": {\"x\": 0.747, \"y\": -2.995, \"z\": 3.079}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.391, \"y\": -0.864, \"z\": 5.261}, \"end\": {\"x\": 1.679, \"y\": -0.027, \"z\": 6.908}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.169, \"y\": -0.181, \"z\": 0.333}, \"end\": {\"x\": 0.138, \"y\": -0.033, \"z\": 1.209}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.708, \"y\": 2.324, \"z\": 6.188}, \"end\": {\"x\": -2.004, \"y\": 2.446, \"z\": 6.973}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.323, \"y\": 2.26, \"z\": 8.889}, \"end\": {\"x\": -2.004, \"y\": 2.446, \"z\": 6.973}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.693, \"y\": -0.027, \"z\": 3.067}, \"end\": {\"x\": 3.815, \"y\": 1.042, \"z\": 4.723}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.131, \"y\": -1.41, \"z\": -0.817}, \"end\": {\"x\": 2.258, \"y\": -0.992, \"z\": -1.108}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.334, \"y\": -2.277, \"z\": 1.623}, \"end\": {\"x\": 4.109, \"y\": -2.279, \"z\": 3.41}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.601, \"y\": -2.923, \"z\": 1.177}, \"end\": {\"x\": 0.747, \"y\": -2.995, \"z\": 3.079}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.131, \"y\": -1.41, \"z\": -0.817}, \"end\": {\"x\": 2.258, \"y\": -0.992, \"z\": -1.108}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.169, \"y\": -0.181, \"z\": 0.333}, \"end\": {\"x\": 0.138, \"y\": -0.033, \"z\": 1.209}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.255, \"y\": 2.997, \"z\": 3.871}, \"end\": {\"x\": -5.795, \"y\": 1.817, \"z\": 3.94}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.708, \"y\": 2.324, \"z\": 6.188}, \"end\": {\"x\": -2.004, \"y\": 2.446, \"z\": 6.973}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.323, \"y\": 2.26, \"z\": 8.889}, \"end\": {\"x\": -2.004, \"y\": 2.446, \"z\": 6.973}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.697, \"y\": 1.029, \"z\": 4.514}, \"end\": {\"x\": -8.082, \"y\": 1.739, \"z\": 6.357}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.344, \"y\": 2.03, \"z\": 12.263}, \"end\": {\"x\": -4.925, \"y\": 1.221, \"z\": 13.173}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.088, \"y\": 3.102, \"z\": 12.841}, \"end\": {\"x\": -0.599, \"y\": 4.459, \"z\": 13.361}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.598, \"y\": 4.009, \"z\": 11.408}, \"end\": {\"x\": -0.599, \"y\": 4.459, \"z\": 13.361}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.255, \"y\": 2.997, \"z\": 3.871}, \"end\": {\"x\": -5.795, \"y\": 1.817, \"z\": 3.94}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.697, \"y\": 1.029, \"z\": 4.514}, \"end\": {\"x\": -8.082, \"y\": 1.739, \"z\": 6.357}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.937, \"y\": -4.497, \"z\": 2.854}, \"end\": {\"x\": -15.799, \"y\": -2.545, \"z\": 2.238}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.444, \"y\": -2.133, \"z\": 7.434}, \"end\": {\"x\": -14.301, \"y\": -0.66, \"z\": 7.978}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.129, \"y\": -2.166, \"z\": 7.476}, \"end\": {\"x\": -15.396, \"y\": -3.811, \"z\": 8.462}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.697, \"y\": -9.587, \"z\": 7.805}, \"end\": {\"x\": -14.656, \"y\": -8.323, \"z\": 9.075}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.677, \"y\": -13.818, \"z\": 10.316}, \"end\": {\"x\": -18.639, \"y\": -15.854, \"z\": 10.743}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.204, \"y\": -12.071, \"z\": 7.146}, \"end\": {\"x\": -15.886, \"y\": -11.38, \"z\": 7.796}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.241, \"y\": -12.052, \"z\": 4.535}, \"end\": {\"x\": -14.694, \"y\": -10.752, \"z\": 4.814}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.601, \"y\": -11.731, \"z\": 5.148}, \"end\": {\"x\": -11.454, \"y\": -10.086, \"z\": 5.637}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.697, \"y\": -9.587, \"z\": 7.805}, \"end\": {\"x\": -14.656, \"y\": -8.323, \"z\": 9.075}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.793, \"y\": -13.98, \"z\": 6.262}, \"end\": {\"x\": -11.241, \"y\": -13.055, \"z\": 7.363}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.155, \"y\": -17.004, \"z\": 5.453}, \"end\": {\"x\": -12.008, \"y\": -14.916, \"z\": 4.802}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.284, \"y\": -18.252, \"z\": 7.081}, \"end\": {\"x\": -16.076, \"y\": -16.514, \"z\": 6.454}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.204, \"y\": -12.071, \"z\": 7.146}, \"end\": {\"x\": -15.886, \"y\": -11.38, \"z\": 7.796}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.985, \"y\": -15.812, \"z\": 4.309}, \"end\": {\"x\": -12.008, \"y\": -14.916, \"z\": 4.802}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.241, \"y\": -12.052, \"z\": 4.535}, \"end\": {\"x\": -14.694, \"y\": -10.752, \"z\": 4.814}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.906, \"y\": -8.396, \"z\": 1.714}, \"end\": {\"x\": -14.987, \"y\": -9.799, \"z\": 0.836}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.059, \"y\": -7.079, \"z\": 1.524}, \"end\": {\"x\": -13.12, \"y\": -8.975, \"z\": 1.582}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.032, \"y\": -14.695, \"z\": 2.675}, \"end\": {\"x\": -9.523, \"y\": -13.499, \"z\": 2.745}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.601, \"y\": -11.731, \"z\": 5.148}, \"end\": {\"x\": -11.454, \"y\": -10.086, \"z\": 5.637}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.607, \"y\": -15.062, \"z\": 4.205}, \"end\": {\"x\": -6.896, \"y\": -13.888, \"z\": 5.102}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.793, \"y\": -13.98, \"z\": 6.262}, \"end\": {\"x\": -11.241, \"y\": -13.055, \"z\": 7.363}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.521, \"y\": -17.716, \"z\": 4.203}, \"end\": {\"x\": -7.302, \"y\": -17.278, \"z\": 5.139}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.656, \"y\": -18.385, \"z\": 4.534}, \"end\": {\"x\": -11.143, \"y\": -17.701, \"z\": 2.535}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.985, \"y\": -15.812, \"z\": 4.309}, \"end\": {\"x\": -12.008, \"y\": -14.916, \"z\": 4.802}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.16, \"y\": -18.273, \"z\": 1.501}, \"end\": {\"x\": -6.852, \"y\": -17.543, \"z\": 1.804}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.032, \"y\": -14.695, \"z\": 2.675}, \"end\": {\"x\": -9.523, \"y\": -13.499, \"z\": 2.745}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.578, \"y\": -17.642, \"z\": 1.301}, \"end\": {\"x\": -3.926, \"y\": -15.82, \"z\": 1.58}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.665, \"y\": -17.298, \"z\": 4.314}, \"end\": {\"x\": -2.395, \"y\": -17.859, \"z\": 3.793}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.607, \"y\": -15.062, \"z\": 4.205}, \"end\": {\"x\": -6.896, \"y\": -13.888, \"z\": 5.102}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.521, \"y\": -17.716, \"z\": 4.203}, \"end\": {\"x\": -7.302, \"y\": -17.278, \"z\": 5.139}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.113, \"y\": -19.793, \"z\": 1.608}, \"end\": {\"x\": -3.081, \"y\": -20.554, \"z\": 2.103}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.16, \"y\": -18.273, \"z\": 1.501}, \"end\": {\"x\": -6.852, \"y\": -17.543, \"z\": 1.804}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.953, \"y\": -19.681, \"z\": -3.547}, \"end\": {\"x\": -2.211, \"y\": -19.072, \"z\": -1.503}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.977, \"y\": -17.182, \"z\": -2.956}, \"end\": {\"x\": -2.211, \"y\": -19.072, \"z\": -1.503}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.578, \"y\": -17.642, \"z\": 1.301}, \"end\": {\"x\": -3.926, \"y\": -15.82, \"z\": 1.58}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.306, \"y\": -20.343, \"z\": -2.292}, \"end\": {\"x\": 1.071, \"y\": -19.05, \"z\": -0.525}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.113, \"y\": -19.793, \"z\": 1.608}, \"end\": {\"x\": -3.081, \"y\": -20.554, \"z\": 2.103}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.306, \"y\": -20.343, \"z\": -2.292}, \"end\": {\"x\": 1.071, \"y\": -19.05, \"z\": -0.525}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.953, \"y\": -19.681, \"z\": -3.547}, \"end\": {\"x\": -2.211, \"y\": -19.072, \"z\": -1.503}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.977, \"y\": -17.182, \"z\": -2.956}, \"end\": {\"x\": -2.211, \"y\": -19.072, \"z\": -1.503}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.592, \"y\": -10.395, \"z\": -0.4}, \"end\": {\"x\": -0.988, \"y\": -11.173, \"z\": -2.234}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.51, \"y\": -8.484, \"z\": 1.221}, \"end\": {\"x\": 0.435, \"y\": -8.651, \"z\": -0.37}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.67, \"y\": -6.971, \"z\": 1.774}, \"end\": {\"x\": -2.265, \"y\": -6.751, \"z\": -0.229}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.028, \"y\": -8.628, \"z\": 3.211}, \"end\": {\"x\": -4.056, \"y\": -9.493, \"z\": 1.321}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.592, \"y\": -10.395, \"z\": -0.4}, \"end\": {\"x\": -0.988, \"y\": -11.173, \"z\": -2.234}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.662, \"y\": -8.943, \"z\": 5.423}, \"end\": {\"x\": -1.821, \"y\": -9.749, \"z\": 3.831}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.51, \"y\": -8.484, \"z\": 1.221}, \"end\": {\"x\": 0.435, \"y\": -8.651, \"z\": -0.37}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.179, \"y\": -6.56, \"z\": 6.496}, \"end\": {\"x\": -1.984, \"y\": -6.457, \"z\": 4.79}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.67, \"y\": -6.971, \"z\": 1.774}, \"end\": {\"x\": -2.265, \"y\": -6.751, \"z\": -0.229}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.749, \"y\": -6.559, \"z\": 7.157}, \"end\": {\"x\": -5.357, \"y\": -6.529, \"z\": 5.084}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.028, \"y\": -8.628, \"z\": 3.211}, \"end\": {\"x\": -4.056, \"y\": -9.493, \"z\": 1.321}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.941, \"y\": -8.731, \"z\": 8.972}, \"end\": {\"x\": -5.197, \"y\": -9.411, \"z\": 7.212}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.662, \"y\": -8.943, \"z\": 5.423}, \"end\": {\"x\": -1.821, \"y\": -9.749, \"z\": 3.831}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.251, \"y\": -7.753, \"z\": 11.056}, \"end\": {\"x\": -3.347, \"y\": -7.576, \"z\": 9.493}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.179, \"y\": -6.56, \"z\": 6.496}, \"end\": {\"x\": -1.984, \"y\": -6.457, \"z\": 4.79}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.754, \"y\": -5.59, \"z\": 11.988}, \"end\": {\"x\": -5.452, \"y\": -4.966, \"z\": 9.777}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.527, \"y\": -2.98, \"z\": 9.628}, \"end\": {\"x\": -5.452, \"y\": -4.966, \"z\": 9.777}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.749, \"y\": -6.559, \"z\": 7.157}, \"end\": {\"x\": -5.357, \"y\": -6.529, \"z\": 5.084}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.966, \"y\": -6.669, \"z\": 12.827}, \"end\": {\"x\": -7.871, \"y\": -7.084, \"z\": 10.874}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.941, \"y\": -8.731, \"z\": 8.972}, \"end\": {\"x\": -5.197, \"y\": -9.411, \"z\": 7.212}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.203, \"y\": -7.688, \"z\": 15.048}, \"end\": {\"x\": -6.117, \"y\": -8.797, \"z\": 13.286}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.251, \"y\": -7.753, \"z\": 11.056}, \"end\": {\"x\": -3.347, \"y\": -7.576, \"z\": 9.493}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.089, \"y\": -8.036, \"z\": 16.242}, \"end\": {\"x\": -4.966, \"y\": -6.071, \"z\": 14.962}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.327, \"y\": -5.264, \"z\": 16.73}, \"end\": {\"x\": -4.966, \"y\": -6.071, \"z\": 14.962}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.754, \"y\": -5.59, \"z\": 11.988}, \"end\": {\"x\": -5.452, \"y\": -4.966, \"z\": 9.777}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.527, \"y\": -2.98, \"z\": 9.628}, \"end\": {\"x\": -5.452, \"y\": -4.966, \"z\": 9.777}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.966, \"y\": -6.669, \"z\": 12.827}, \"end\": {\"x\": -7.871, \"y\": -7.084, \"z\": 10.874}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.203, \"y\": -7.688, \"z\": 15.048}, \"end\": {\"x\": -6.117, \"y\": -8.797, \"z\": 13.286}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.393, \"y\": -6.73, \"z\": 20.135}, \"end\": {\"x\": -6.887, \"y\": -8.082, \"z\": 18.812}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.089, \"y\": -8.036, \"z\": 16.242}, \"end\": {\"x\": -4.966, \"y\": -6.071, \"z\": 14.962}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.77, \"y\": -4.826, \"z\": 21.39}, \"end\": {\"x\": -4.811, \"y\": -5.653, \"z\": 19.843}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.327, \"y\": -5.264, \"z\": 16.73}, \"end\": {\"x\": -4.966, \"y\": -6.071, \"z\": 14.962}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.441, \"y\": -1.615, \"z\": 18.934}, \"end\": {\"x\": -3.352, \"y\": -1.552, \"z\": 17.958}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.648, \"y\": -2.874, \"z\": 22.009}, \"end\": {\"x\": -7.13, \"y\": -2.972, \"z\": 19.976}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.075, \"y\": -4.761, \"z\": 23.66}, \"end\": {\"x\": -9.416, \"y\": -4.917, \"z\": 21.754}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.637, \"y\": -5.949, \"z\": 24.982}, \"end\": {\"x\": -7.049, \"y\": -5.334, \"z\": 24.214}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.393, \"y\": -6.73, \"z\": 20.135}, \"end\": {\"x\": -6.887, \"y\": -8.082, \"z\": 18.812}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.75, \"y\": -4.404, \"z\": 21.029}, \"end\": {\"x\": 1.023, \"y\": -5.709, \"z\": 21.431}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.297, \"y\": -4.328, \"z\": 20.1}, \"end\": {\"x\": 1.023, \"y\": -5.709, \"z\": 21.431}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.77, \"y\": -4.826, \"z\": 21.39}, \"end\": {\"x\": -4.811, \"y\": -5.653, \"z\": 19.843}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.441, \"y\": -1.615, \"z\": 18.934}, \"end\": {\"x\": -3.352, \"y\": -1.552, \"z\": 17.958}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.93, \"y\": -1.188, \"z\": 25.922}, \"end\": {\"x\": -6.1, \"y\": -1.778, \"z\": 24.458}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.139, \"y\": -2.176, \"z\": 26.586}, \"end\": {\"x\": -6.1, \"y\": -1.778, \"z\": 24.458}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.648, \"y\": -2.874, \"z\": 22.009}, \"end\": {\"x\": -7.13, \"y\": -2.972, \"z\": 19.976}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.674, \"y\": -2.486, \"z\": 20.637}, \"end\": {\"x\": -10.288, \"y\": -1.952, \"z\": 22.071}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.249, \"y\": -1.676, \"z\": 20.919}, \"end\": {\"x\": -13.636, \"y\": -1.159, \"z\": 22.603}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.694, \"y\": -0.579, \"z\": 18.329}, \"end\": {\"x\": -13.69, \"y\": 0.719, \"z\": 19.743}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.258, \"y\": 0.834, \"z\": 23.301}, \"end\": {\"x\": -9.436, \"y\": 1.454, \"z\": 24.665}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.325, \"y\": 6.426, \"z\": 17.27}, \"end\": {\"x\": -12.134, \"y\": 7.187, \"z\": 16.168}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.713, \"y\": -2.846, \"z\": 16.83}, \"end\": {\"x\": -12.891, \"y\": -2.293, \"z\": 17.733}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.586, \"y\": -5.825, \"z\": 25.783}, \"end\": {\"x\": -13.59, \"y\": -5.122, \"z\": 24.075}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.777, \"y\": -4.718, \"z\": 24.065}, \"end\": {\"x\": -13.59, \"y\": -5.122, \"z\": 24.075}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.674, \"y\": -2.486, \"z\": 20.637}, \"end\": {\"x\": -10.288, \"y\": -1.952, \"z\": 22.071}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.784, \"y\": -4.083, \"z\": 17.731}, \"end\": {\"x\": -15.356, \"y\": -3.963, \"z\": 19.016}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.249, \"y\": -1.676, \"z\": 20.919}, \"end\": {\"x\": -13.636, \"y\": -1.159, \"z\": 22.603}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.894, \"y\": -2.572, \"z\": 16.998}, \"end\": {\"x\": -17.686, \"y\": -1.491, \"z\": 18.255}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.694, \"y\": -0.579, \"z\": 18.329}, \"end\": {\"x\": -13.69, \"y\": 0.719, \"z\": 19.743}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.747, \"y\": -2.894, \"z\": 14.157}, \"end\": {\"x\": -16.87, \"y\": -1.799, \"z\": 14.688}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.331, \"y\": -0.747, \"z\": 13.16}, \"end\": {\"x\": -16.87, \"y\": -1.799, \"z\": 14.688}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.928, \"y\": 0.95, \"z\": 13.294}, \"end\": {\"x\": -15.29, \"y\": 2.555, \"z\": 14.529}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.713, \"y\": -2.846, \"z\": 16.83}, \"end\": {\"x\": -12.891, \"y\": -2.293, \"z\": 17.733}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -19.443, \"y\": -5.559, \"z\": 14.015}, \"end\": {\"x\": -17.665, \"y\": -5.338, \"z\": 14.975}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.784, \"y\": -4.083, \"z\": 17.731}, \"end\": {\"x\": -15.356, \"y\": -3.963, \"z\": 19.016}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -21.913, \"y\": -5.422, \"z\": 15.008}, \"end\": {\"x\": -20.79, \"y\": -4.985, \"z\": 16.381}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.894, \"y\": -2.572, \"z\": 16.998}, \"end\": {\"x\": -17.686, \"y\": -1.491, \"z\": 18.255}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -23.403, \"y\": -4.155, \"z\": 13.054}, \"end\": {\"x\": -21.865, \"y\": -2.847, \"z\": 13.818}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.747, \"y\": -2.894, \"z\": 14.157}, \"end\": {\"x\": -16.87, \"y\": -1.799, \"z\": 14.688}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.331, \"y\": -0.747, \"z\": 13.16}, \"end\": {\"x\": -16.87, \"y\": -1.799, \"z\": 14.688}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.928, \"y\": 0.95, \"z\": 13.294}, \"end\": {\"x\": -15.29, \"y\": 2.555, \"z\": 14.529}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -21.822, \"y\": -4.784, \"z\": 9.676}, \"end\": {\"x\": -21.187, \"y\": -5.158, \"z\": 11.393}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.243, \"y\": -0.994, \"z\": 9.81}, \"end\": {\"x\": -14.301, \"y\": -0.66, \"z\": 7.978}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -19.443, \"y\": -5.559, \"z\": 14.015}, \"end\": {\"x\": -17.665, \"y\": -5.338, \"z\": 14.975}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -21.913, \"y\": -5.422, \"z\": 15.008}, \"end\": {\"x\": -20.79, \"y\": -4.985, \"z\": 16.381}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -23.403, \"y\": -4.155, \"z\": 13.054}, \"end\": {\"x\": -21.865, \"y\": -2.847, \"z\": 13.818}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -21.822, \"y\": -4.784, \"z\": 9.676}, \"end\": {\"x\": -21.187, \"y\": -5.158, \"z\": 11.393}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.937, \"y\": -4.497, \"z\": 2.854}, \"end\": {\"x\": -15.799, \"y\": -2.545, \"z\": 2.238}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.906, \"y\": -8.396, \"z\": 1.714}, \"end\": {\"x\": -14.987, \"y\": -9.799, \"z\": 0.836}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.059, \"y\": -7.079, \"z\": 1.524}, \"end\": {\"x\": -13.12, \"y\": -8.975, \"z\": 1.582}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.444, \"y\": -2.133, \"z\": 7.434}, \"end\": {\"x\": -14.301, \"y\": -0.66, \"z\": 7.978}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.129, \"y\": -2.166, \"z\": 7.476}, \"end\": {\"x\": -15.396, \"y\": -3.811, \"z\": 8.462}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.243, \"y\": -0.994, \"z\": 9.81}, \"end\": {\"x\": -14.301, \"y\": -0.66, \"z\": 7.978}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.944, \"y\": 6.285, \"z\": 15.156}, \"end\": {\"x\": -14.596, \"y\": 6.761, \"z\": 13.65}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.37, \"y\": 5.982, \"z\": 14.451}, \"end\": {\"x\": -9.906, \"y\": 6.117, \"z\": 12.315}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.37, \"y\": 5.982, \"z\": 14.451}, \"end\": {\"x\": -12.484, \"y\": 6.509, \"z\": 13.912}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.64, \"y\": 6.666, \"z\": 15.531}, \"end\": {\"x\": -12.484, \"y\": 6.509, \"z\": 13.912}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.279, \"y\": 7.325, \"z\": 14.346}, \"end\": {\"x\": -7.865, \"y\": 8.483, \"z\": 14.037}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.325, \"y\": 6.426, \"z\": 17.27}, \"end\": {\"x\": -12.134, \"y\": 7.187, \"z\": 16.168}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.64, \"y\": 6.666, \"z\": 15.531}, \"end\": {\"x\": -12.484, \"y\": 6.509, \"z\": 13.912}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.338, \"y\": 6.219, \"z\": 17.141}, \"end\": {\"x\": -8.423, \"y\": 6.687, \"z\": 16.852}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.37, \"y\": 5.982, \"z\": 14.451}, \"end\": {\"x\": -9.906, \"y\": 6.117, \"z\": 12.315}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.37, \"y\": 5.982, \"z\": 14.451}, \"end\": {\"x\": -12.484, \"y\": 6.509, \"z\": 13.912}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.176, \"y\": 3.63, \"z\": 16.728}, \"end\": {\"x\": -6.842, \"y\": 3.88, \"z\": 15.688}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.593, \"y\": 4.645, \"z\": 15.798}, \"end\": {\"x\": -3.963, \"y\": 5.19, \"z\": 14.499}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.279, \"y\": 7.325, \"z\": 14.346}, \"end\": {\"x\": -7.865, \"y\": 8.483, \"z\": 14.037}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.338, \"y\": 6.219, \"z\": 17.141}, \"end\": {\"x\": -8.423, \"y\": 6.687, \"z\": 16.852}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.708, \"y\": 5.136, \"z\": 18.433}, \"end\": {\"x\": -3.282, \"y\": 4.059, \"z\": 19.181}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.344, \"y\": 2.03, \"z\": 12.263}, \"end\": {\"x\": -4.925, \"y\": 1.221, \"z\": 13.173}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.176, \"y\": 3.63, \"z\": 16.728}, \"end\": {\"x\": -6.842, \"y\": 3.88, \"z\": 15.688}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.583, \"y\": 3.469, \"z\": 19.669}, \"end\": {\"x\": -1.03, \"y\": 2.446, \"z\": 17.386}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.088, \"y\": 3.102, \"z\": 12.841}, \"end\": {\"x\": -0.599, \"y\": 4.459, \"z\": 13.361}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.598, \"y\": 4.009, \"z\": 11.408}, \"end\": {\"x\": -0.599, \"y\": 4.459, \"z\": 13.361}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.593, \"y\": 4.645, \"z\": 15.798}, \"end\": {\"x\": -3.963, \"y\": 5.19, \"z\": 14.499}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.708, \"y\": 5.136, \"z\": 18.433}, \"end\": {\"x\": -3.282, \"y\": 4.059, \"z\": 19.181}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.74, \"y\": 5.593, \"z\": 23.674}, \"end\": {\"x\": 0.11, \"y\": 5.475, \"z\": 22.029}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.583, \"y\": 3.469, \"z\": 19.669}, \"end\": {\"x\": -1.03, \"y\": 2.446, \"z\": 17.386}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.515, \"y\": -1.052, \"z\": 24.635}, \"end\": {\"x\": -2.756, \"y\": -1.57, \"z\": 26.565}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.027, \"y\": 3.739, \"z\": 25.509}, \"end\": {\"x\": 0.659, \"y\": 2.753, \"z\": 23.741}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.393, \"y\": 4.576, \"z\": 26.672}, \"end\": {\"x\": 3.689, \"y\": 3.901, \"z\": 24.681}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.686, \"y\": 2.859, \"z\": 24.625}, \"end\": {\"x\": 3.689, \"y\": 3.901, \"z\": 24.681}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.689, \"y\": 7.008, \"z\": 27.619}, \"end\": {\"x\": 2.486, \"y\": 7.092, \"z\": 25.757}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.74, \"y\": 5.593, \"z\": 23.674}, \"end\": {\"x\": 0.11, \"y\": 5.475, \"z\": 22.029}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.832, \"y\": 6.253, \"z\": 29.636}, \"end\": {\"x\": 0.221, \"y\": 5.519, \"z\": 27.967}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.027, \"y\": 3.739, \"z\": 25.509}, \"end\": {\"x\": 0.659, \"y\": 2.753, \"z\": 23.741}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.631, \"y\": -1.271, \"z\": 26.279}, \"end\": {\"x\": 1.03, \"y\": 1.061, \"z\": 26.186}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.07, \"y\": 4.583, \"z\": 31.373}, \"end\": {\"x\": 2.539, \"y\": 3.681, \"z\": 29.534}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.393, \"y\": 4.576, \"z\": 26.672}, \"end\": {\"x\": 3.689, \"y\": 3.901, \"z\": 24.681}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.973, \"y\": 2.733, \"z\": 26.798}, \"end\": {\"x\": 6.101, \"y\": 4.672, \"z\": 26.717}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.988, \"y\": 6.545, \"z\": 32.294}, \"end\": {\"x\": 4.174, \"y\": 6.483, \"z\": 30.435}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.689, \"y\": 7.008, \"z\": 27.619}, \"end\": {\"x\": 2.486, \"y\": 7.092, \"z\": 25.757}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.832, \"y\": 6.253, \"z\": 29.636}, \"end\": {\"x\": 0.221, \"y\": 5.519, \"z\": 27.967}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.773, \"y\": 7.606, \"z\": 34.102}, \"end\": {\"x\": 0.983, \"y\": 6.142, \"z\": 33.882}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.07, \"y\": 4.583, \"z\": 31.373}, \"end\": {\"x\": 2.539, \"y\": 3.681, \"z\": 29.534}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.695, \"y\": -0.781, \"z\": 31.377}, \"end\": {\"x\": 4.263, \"y\": 0.155, \"z\": 32.27}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.988, \"y\": 6.545, \"z\": 32.294}, \"end\": {\"x\": 4.174, \"y\": 6.483, \"z\": 30.435}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.773, \"y\": 7.606, \"z\": 34.102}, \"end\": {\"x\": 0.983, \"y\": 6.142, \"z\": 33.882}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.425, \"y\": 9.212, \"z\": 37.403}, \"end\": {\"x\": 0.425, \"y\": 8.375, \"z\": 35.426}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.919, \"y\": 11.681, \"z\": 36.115}, \"end\": {\"x\": 0.753, \"y\": 12.754, \"z\": 34.984}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.08, \"y\": 11.739, \"z\": 37.129}, \"end\": {\"x\": 0.087, \"y\": 12.023, \"z\": 38.762}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.425, \"y\": 9.212, \"z\": 37.403}, \"end\": {\"x\": 0.425, \"y\": 8.375, \"z\": 35.426}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.497, \"y\": 10.22, \"z\": 33.793}, \"end\": {\"x\": -3.001, \"y\": 10.839, \"z\": 34.791}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.919, \"y\": 11.681, \"z\": 36.115}, \"end\": {\"x\": 0.753, \"y\": 12.754, \"z\": 34.984}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.08, \"y\": 11.739, \"z\": 37.129}, \"end\": {\"x\": 0.087, \"y\": 12.023, \"z\": 38.762}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.485, \"y\": 13.718, \"z\": 33.014}, \"end\": {\"x\": -2.923, \"y\": 14.47, \"z\": 32.052}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.341, \"y\": 17.653, \"z\": 29.536}, \"end\": {\"x\": 3.846, \"y\": 17.168, \"z\": 31.304}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.341, \"y\": 17.653, \"z\": 29.536}, \"end\": {\"x\": 3.846, \"y\": 17.168, \"z\": 31.304}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.677, \"y\": 14.03, \"z\": 25.818}, \"end\": {\"x\": -3.18, \"y\": 14.362, \"z\": 27.454}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.217, \"y\": 14.549, \"z\": 24.947}, \"end\": {\"x\": -3.025, \"y\": 13.772, \"z\": 23.646}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.045, \"y\": 13.877, \"z\": 20.751}, \"end\": {\"x\": -5.609, \"y\": 15.548, \"z\": 21.261}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.627, \"y\": 15.875, \"z\": 19.267}, \"end\": {\"x\": -6.149, \"y\": 13.997, \"z\": 18.375}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.689, \"y\": 13.331, \"z\": 13.328}, \"end\": {\"x\": -11.332, \"y\": 11.982, \"z\": 14.629}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.112, \"y\": 13.951, \"z\": 16.712}, \"end\": {\"x\": -9.228, \"y\": 15.554, \"z\": 16.197}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.069, \"y\": 17.57, \"z\": 17.789}, \"end\": {\"x\": -12.014, \"y\": 17.233, \"z\": 15.79}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.944, \"y\": 6.285, \"z\": 15.156}, \"end\": {\"x\": -14.596, \"y\": 6.761, \"z\": 13.65}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.995, \"y\": 11.453, \"z\": 16.297}, \"end\": {\"x\": -15.032, \"y\": 11.567, \"z\": 17.846}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.318, \"y\": 8.481, \"z\": 17.599}, \"end\": {\"x\": -18.765, \"y\": 9.805, \"z\": 18.36}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.389, \"y\": 15.77, \"z\": 13.829}, \"end\": {\"x\": -13.457, \"y\": 15.789, \"z\": 14.256}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.689, \"y\": 13.331, \"z\": 13.328}, \"end\": {\"x\": -11.332, \"y\": 11.982, \"z\": 14.629}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.112, \"y\": 13.951, \"z\": 16.712}, \"end\": {\"x\": -9.228, \"y\": 15.554, \"z\": 16.197}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.069, \"y\": 17.57, \"z\": 17.789}, \"end\": {\"x\": -12.014, \"y\": 17.233, \"z\": 15.79}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.389, \"y\": 15.77, \"z\": 13.829}, \"end\": {\"x\": -13.457, \"y\": 15.789, \"z\": 14.256}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.045, \"y\": 13.877, \"z\": 20.751}, \"end\": {\"x\": -5.609, \"y\": 15.548, \"z\": 21.261}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.627, \"y\": 15.875, \"z\": 19.267}, \"end\": {\"x\": -6.149, \"y\": 13.997, \"z\": 18.375}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.677, \"y\": 14.03, \"z\": 25.818}, \"end\": {\"x\": -3.18, \"y\": 14.362, \"z\": 27.454}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.217, \"y\": 14.549, \"z\": 24.947}, \"end\": {\"x\": -3.025, \"y\": 13.772, \"z\": 23.646}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.485, \"y\": 13.718, \"z\": 33.014}, \"end\": {\"x\": -2.923, \"y\": 14.47, \"z\": 32.052}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.262, \"y\": 10.12, \"z\": 33.809}, \"end\": {\"x\": -7.445, \"y\": 12.375, \"z\": 33.824}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.497, \"y\": 10.22, \"z\": 33.793}, \"end\": {\"x\": -3.001, \"y\": 10.839, \"z\": 34.791}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.401, \"y\": 8.138, \"z\": 30.364}, \"end\": {\"x\": -7.412, \"y\": 7.629, \"z\": 31.961}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.262, \"y\": 10.12, \"z\": 33.809}, \"end\": {\"x\": -7.445, \"y\": 12.375, \"z\": 33.824}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.314, \"y\": 6.093, \"z\": 31.194}, \"end\": {\"x\": -10.629, \"y\": 7.787, \"z\": 32.324}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.401, \"y\": 8.138, \"z\": 30.364}, \"end\": {\"x\": -7.412, \"y\": 7.629, \"z\": 31.961}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.392, \"y\": 3.526, \"z\": 27.974}, \"end\": {\"x\": -8.698, \"y\": 4.382, \"z\": 28.65}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.592, \"y\": 2.694, \"z\": 27.423}, \"end\": {\"x\": -8.698, \"y\": 4.382, \"z\": 28.65}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.314, \"y\": 6.093, \"z\": 31.194}, \"end\": {\"x\": -10.629, \"y\": 7.787, \"z\": 32.324}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.258, \"y\": 0.834, \"z\": 23.301}, \"end\": {\"x\": -9.436, \"y\": 1.454, \"z\": 24.665}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.392, \"y\": 3.526, \"z\": 27.974}, \"end\": {\"x\": -8.698, \"y\": 4.382, \"z\": 28.65}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.592, \"y\": 2.694, \"z\": 27.423}, \"end\": {\"x\": -8.698, \"y\": 4.382, \"z\": 28.65}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.93, \"y\": -1.188, \"z\": 25.922}, \"end\": {\"x\": -6.1, \"y\": -1.778, \"z\": 24.458}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.139, \"y\": -2.176, \"z\": 26.586}, \"end\": {\"x\": -6.1, \"y\": -1.778, \"z\": 24.458}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.515, \"y\": -1.052, \"z\": 24.635}, \"end\": {\"x\": -2.756, \"y\": -1.57, \"z\": 26.565}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.481, \"y\": -3.161, \"z\": 28.826}, \"end\": {\"x\": -2.518, \"y\": -3.408, \"z\": 30.737}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.033, \"y\": -4.935, \"z\": 29.523}, \"end\": {\"x\": 1.041, \"y\": -3.403, \"z\": 28.444}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.631, \"y\": -1.271, \"z\": 26.279}, \"end\": {\"x\": 1.03, \"y\": 1.061, \"z\": 26.186}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.686, \"y\": 2.859, \"z\": 24.625}, \"end\": {\"x\": 3.689, \"y\": 3.901, \"z\": 24.681}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.973, \"y\": 2.733, \"z\": 26.798}, \"end\": {\"x\": 6.101, \"y\": 4.672, \"z\": 26.717}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.695, \"y\": -0.781, \"z\": 31.377}, \"end\": {\"x\": 4.263, \"y\": 0.155, \"z\": 32.27}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.876, \"y\": 22.349, \"z\": 21.406}, \"end\": {\"x\": 24.434, \"y\": 23.709, \"z\": 22.556}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.957, \"y\": 22.392, \"z\": 16.299}, \"end\": {\"x\": 31.173, \"y\": 22.966, \"z\": 15.497}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.563, \"y\": 22.402, \"z\": 12.5}, \"end\": {\"x\": 34.098, \"y\": 21.846, \"z\": 14.328}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.841, \"y\": 21.575, \"z\": 10.849}, \"end\": {\"x\": 37.798, \"y\": 23.131, \"z\": 11.85}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.169, \"y\": 18.688, \"z\": 11.876}, \"end\": {\"x\": 40.518, \"y\": 18.929, \"z\": 10.492}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.413, \"y\": 17.006, \"z\": 10.162}, \"end\": {\"x\": 40.518, \"y\": 18.929, \"z\": 10.492}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.841, \"y\": 21.575, \"z\": 10.849}, \"end\": {\"x\": 37.798, \"y\": 23.131, \"z\": 11.85}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.169, \"y\": 18.688, \"z\": 11.876}, \"end\": {\"x\": 40.518, \"y\": 18.929, \"z\": 10.492}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.563, \"y\": 15.607, \"z\": 18.51}, \"end\": {\"x\": 42.179, \"y\": 15.072, \"z\": 17.459}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.656, \"y\": 17.432, \"z\": 16.45}, \"end\": {\"x\": 41.164, \"y\": 18.257, \"z\": 17.532}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.769, \"y\": 16.711, \"z\": 19.413}, \"end\": {\"x\": 44.503, \"y\": 17.302, \"z\": 18.364}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.903, \"y\": 13.802, \"z\": 20.555}, \"end\": {\"x\": 41.772, \"y\": 13.437, \"z\": 20.248}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.416, \"y\": 16.376, \"z\": 25.647}, \"end\": {\"x\": 40.416, \"y\": 16.704, \"z\": 25.603}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.399, \"y\": 15.87, \"z\": 22.673}, \"end\": {\"x\": 41.162, \"y\": 16.183, \"z\": 22.115}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.656, \"y\": 17.432, \"z\": 16.45}, \"end\": {\"x\": 41.164, \"y\": 18.257, \"z\": 17.532}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.769, \"y\": 16.711, \"z\": 19.413}, \"end\": {\"x\": 44.503, \"y\": 17.302, \"z\": 18.364}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.23, \"y\": 16.95, \"z\": 21.688}, \"end\": {\"x\": 38.647, \"y\": 17.543, \"z\": 20.151}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.563, \"y\": 15.607, \"z\": 18.51}, \"end\": {\"x\": 42.179, \"y\": 15.072, \"z\": 17.459}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.335, \"y\": 14.839, \"z\": 21.537}, \"end\": {\"x\": 36.91, \"y\": 14.522, \"z\": 20.478}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.903, \"y\": 13.802, \"z\": 20.555}, \"end\": {\"x\": 41.772, \"y\": 13.437, \"z\": 20.248}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.12, \"y\": 14.384, \"z\": 24.281}, \"end\": {\"x\": 36.957, \"y\": 14.208, \"z\": 23.826}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.416, \"y\": 16.376, \"z\": 25.647}, \"end\": {\"x\": 40.416, \"y\": 16.704, \"z\": 25.603}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.399, \"y\": 15.87, \"z\": 22.673}, \"end\": {\"x\": 41.162, \"y\": 16.183, \"z\": 22.115}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.897, \"y\": 16.77, \"z\": 24.884}, \"end\": {\"x\": 35.824, \"y\": 17.276, \"z\": 24.419}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.23, \"y\": 16.95, \"z\": 21.688}, \"end\": {\"x\": 38.647, \"y\": 17.543, \"z\": 20.151}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.54, \"y\": 16.646, \"z\": 23.868}, \"end\": {\"x\": 33.142, \"y\": 16.76, \"z\": 22.646}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.335, \"y\": 14.839, \"z\": 21.537}, \"end\": {\"x\": 36.91, \"y\": 14.522, \"z\": 20.478}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.345, \"y\": 14.446, \"z\": 25.174}, \"end\": {\"x\": 31.977, \"y\": 13.994, \"z\": 24.242}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.12, \"y\": 14.384, \"z\": 24.281}, \"end\": {\"x\": 36.957, \"y\": 14.208, \"z\": 23.826}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.307, \"y\": 15.704, \"z\": 27.703}, \"end\": {\"x\": 32.283, \"y\": 15.529, \"z\": 27.371}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.897, \"y\": 16.77, \"z\": 24.884}, \"end\": {\"x\": 35.824, \"y\": 17.276, \"z\": 24.419}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.199, \"y\": 17.309, \"z\": 27.401}, \"end\": {\"x\": 30.041, \"y\": 18.048, \"z\": 26.289}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.54, \"y\": 16.646, \"z\": 23.868}, \"end\": {\"x\": 33.142, \"y\": 16.76, \"z\": 22.646}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.304, \"y\": 16.069, \"z\": 27.208}, \"end\": {\"x\": 27.648, \"y\": 15.935, \"z\": 25.526}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.345, \"y\": 14.446, \"z\": 25.174}, \"end\": {\"x\": 31.977, \"y\": 13.994, \"z\": 24.242}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.984, \"y\": 8.251, \"z\": 23.58}, \"end\": {\"x\": 31.365, \"y\": 8.299, \"z\": 21.71}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.471, \"y\": 15.046, \"z\": 29.295}, \"end\": {\"x\": 27.569, \"y\": 14.355, \"z\": 28.536}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.585, \"y\": 9.377, \"z\": 24.985}, \"end\": {\"x\": 29.035, \"y\": 8.741, \"z\": 24.226}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.568, \"y\": 9.273, \"z\": 25.687}, \"end\": {\"x\": 35.521, \"y\": 8.526, \"z\": 23.943}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.693, \"y\": 8.18, \"z\": 23.949}, \"end\": {\"x\": 35.521, \"y\": 8.526, \"z\": 23.943}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.307, \"y\": 15.704, \"z\": 27.703}, \"end\": {\"x\": 32.283, \"y\": 15.529, \"z\": 27.371}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.199, \"y\": 17.309, \"z\": 27.401}, \"end\": {\"x\": 30.041, \"y\": 18.048, \"z\": 26.289}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.304, \"y\": 16.069, \"z\": 27.208}, \"end\": {\"x\": 27.648, \"y\": 15.935, \"z\": 25.526}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.471, \"y\": 15.046, \"z\": 29.295}, \"end\": {\"x\": 27.569, \"y\": 14.355, \"z\": 28.536}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.235, \"y\": 5.611, \"z\": 36.68}, \"end\": {\"x\": 20.722, \"y\": 7.374, \"z\": 36.849}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.376, \"y\": 6.717, \"z\": 35.935}, \"end\": {\"x\": 19.119, \"y\": 8.419, \"z\": 35.69}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.498, \"y\": 8.386, \"z\": 33.373}, \"end\": {\"x\": 23.534, \"y\": 8.698, \"z\": 35.23}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.235, \"y\": 5.611, \"z\": 36.68}, \"end\": {\"x\": 20.722, \"y\": 7.374, \"z\": 36.849}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.376, \"y\": 6.717, \"z\": 35.935}, \"end\": {\"x\": 19.119, \"y\": 8.419, \"z\": 35.69}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.852, \"y\": 4.756, \"z\": 35.474}, \"end\": {\"x\": 25.323, \"y\": 5.198, \"z\": 33.647}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.219, \"y\": 3.35, \"z\": 34.982}, \"end\": {\"x\": 25.658, \"y\": 2.249, \"z\": 35.072}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.852, \"y\": 4.756, \"z\": 35.474}, \"end\": {\"x\": 25.323, \"y\": 5.198, \"z\": 33.647}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.219, \"y\": 3.35, \"z\": 34.982}, \"end\": {\"x\": 25.658, \"y\": 2.249, \"z\": 35.072}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.504, \"y\": 6.59, \"z\": 28.902}, \"end\": {\"x\": 24.635, \"y\": 6.904, \"z\": 30.764}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.498, \"y\": 8.386, \"z\": 33.373}, \"end\": {\"x\": 23.534, \"y\": 8.698, \"z\": 35.23}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.1, \"y\": 8.452, \"z\": 29.548}, \"end\": {\"x\": 20.985, \"y\": 6.93, \"z\": 28.587}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.707, \"y\": 10.989, \"z\": 29.477}, \"end\": {\"x\": 11.729, \"y\": 9.651, \"z\": 30.58}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.959, \"y\": 11.943, \"z\": 21.698}, \"end\": {\"x\": 9.866, \"y\": 13.537, \"z\": 19.976}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.008, \"y\": 11.869, \"z\": 20.545}, \"end\": {\"x\": 9.866, \"y\": 13.537, \"z\": 19.976}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.416, \"y\": 15.383, \"z\": 29.721}, \"end\": {\"x\": 10.361, \"y\": 13.885, \"z\": 28.212}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.707, \"y\": 10.989, \"z\": 29.477}, \"end\": {\"x\": 11.729, \"y\": 9.651, \"z\": 30.58}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.416, \"y\": 15.383, \"z\": 29.721}, \"end\": {\"x\": 10.361, \"y\": 13.885, \"z\": 28.212}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.649, \"y\": 21.983, \"z\": 27.993}, \"end\": {\"x\": 13.194, \"y\": 20.654, \"z\": 28.044}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.612, \"y\": 21.155, \"z\": 25.748}, \"end\": {\"x\": 13.978, \"y\": 22.481, \"z\": 25.102}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.649, \"y\": 21.983, \"z\": 27.993}, \"end\": {\"x\": 13.194, \"y\": 20.654, \"z\": 28.044}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.612, \"y\": 21.155, \"z\": 25.748}, \"end\": {\"x\": 13.978, \"y\": 22.481, \"z\": 25.102}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.392, \"y\": 15.143, \"z\": 22.88}, \"end\": {\"x\": 16.141, \"y\": 14.266, \"z\": 22.627}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.136, \"y\": 14.06, \"z\": 21.495}, \"end\": {\"x\": 14.864, \"y\": 13.796, \"z\": 20.782}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.292, \"y\": 16.284, \"z\": 23.573}, \"end\": {\"x\": 17.109, \"y\": 17.717, \"z\": 23.734}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.896, \"y\": 19.049, \"z\": 21.962}, \"end\": {\"x\": 19.994, \"y\": 17.511, \"z\": 20.877}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.719, \"y\": 18.644, \"z\": 18.979}, \"end\": {\"x\": 22.152, \"y\": 20.108, \"z\": 19.993}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.876, \"y\": 22.349, \"z\": 21.406}, \"end\": {\"x\": 24.434, \"y\": 23.709, \"z\": 22.556}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.093, \"y\": 21.032, \"z\": 17.334}, \"end\": {\"x\": 23.55, \"y\": 20.736, \"z\": 16.398}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.673, \"y\": 20.95, \"z\": 14.237}, \"end\": {\"x\": 27.097, \"y\": 21.416, \"z\": 15.275}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.957, \"y\": 22.392, \"z\": 16.299}, \"end\": {\"x\": 31.173, \"y\": 22.966, \"z\": 15.497}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.1, \"y\": 22.475, \"z\": 13.172}, \"end\": {\"x\": 27.722, \"y\": 23.063, \"z\": 12.072}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.201, \"y\": 23.513, \"z\": 10.105}, \"end\": {\"x\": 30.914, \"y\": 23.264, \"z\": 10.982}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.563, \"y\": 22.402, \"z\": 12.5}, \"end\": {\"x\": 34.098, \"y\": 21.846, \"z\": 14.328}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.867, \"y\": 24.067, \"z\": 8.464}, \"end\": {\"x\": 32.663, \"y\": 24.95, \"z\": 7.979}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.266, \"y\": 25.421, \"z\": 5.933}, \"end\": {\"x\": 34.841, \"y\": 23.472, \"z\": 5.961}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.648, \"y\": 21.849, \"z\": 4.529}, \"end\": {\"x\": 33.091, \"y\": 21.217, \"z\": 2.538}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.103, \"y\": 20.472, \"z\": 5.476}, \"end\": {\"x\": 33.87, \"y\": 18.443, \"z\": 4.902}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.214, \"y\": 21.583, \"z\": 6.93}, \"end\": {\"x\": 38.089, \"y\": 19.966, \"z\": 6.516}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.867, \"y\": 24.067, \"z\": 8.464}, \"end\": {\"x\": 32.663, \"y\": 24.95, \"z\": 7.979}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.266, \"y\": 25.421, \"z\": 5.933}, \"end\": {\"x\": 34.841, \"y\": 23.472, \"z\": 5.961}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.1, \"y\": 22.475, \"z\": 13.172}, \"end\": {\"x\": 27.722, \"y\": 23.063, \"z\": 12.072}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.201, \"y\": 23.513, \"z\": 10.105}, \"end\": {\"x\": 30.914, \"y\": 23.264, \"z\": 10.982}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.769, \"y\": 23.904, \"z\": 11.282}, \"end\": {\"x\": 23.717, \"y\": 24.466, \"z\": 9.81}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.258, \"y\": 23.006, \"z\": 11.449}, \"end\": {\"x\": 23.16, \"y\": 22.291, \"z\": 12.88}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.093, \"y\": 21.032, \"z\": 17.334}, \"end\": {\"x\": 23.55, \"y\": 20.736, \"z\": 16.398}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.673, \"y\": 20.95, \"z\": 14.237}, \"end\": {\"x\": 27.097, \"y\": 21.416, \"z\": 15.275}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.248, \"y\": 16.107, \"z\": 15.597}, \"end\": {\"x\": 20.428, \"y\": 17.289, \"z\": 16.325}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.406, \"y\": 19.29, \"z\": 14.599}, \"end\": {\"x\": 20.528, \"y\": 18.873, \"z\": 12.955}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.22, \"y\": 17.276, \"z\": 13.809}, \"end\": {\"x\": 20.428, \"y\": 17.289, \"z\": 16.325}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.896, \"y\": 19.049, \"z\": 21.962}, \"end\": {\"x\": 19.994, \"y\": 17.511, \"z\": 20.877}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.719, \"y\": 18.644, \"z\": 18.979}, \"end\": {\"x\": 22.152, \"y\": 20.108, \"z\": 19.993}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.392, \"y\": 15.143, \"z\": 22.88}, \"end\": {\"x\": 16.141, \"y\": 14.266, \"z\": 22.627}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.136, \"y\": 14.06, \"z\": 21.495}, \"end\": {\"x\": 14.864, \"y\": 13.796, \"z\": 20.782}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.304, \"y\": 14.859, \"z\": 19.353}, \"end\": {\"x\": 19.984, \"y\": 13.71, \"z\": 17.928}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.313, \"y\": 13.726, \"z\": 24.017}, \"end\": {\"x\": 16.141, \"y\": 14.266, \"z\": 22.627}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.465, \"y\": 11.804, \"z\": 22.944}, \"end\": {\"x\": 19.643, \"y\": 13.757, \"z\": 22.222}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.292, \"y\": 16.284, \"z\": 23.573}, \"end\": {\"x\": 17.109, \"y\": 17.717, \"z\": 23.734}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.313, \"y\": 13.726, \"z\": 24.017}, \"end\": {\"x\": 16.141, \"y\": 14.266, \"z\": 22.627}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.465, \"y\": 11.804, \"z\": 22.944}, \"end\": {\"x\": 19.643, \"y\": 13.757, \"z\": 22.222}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.152, \"y\": 11.65, \"z\": 20.058}, \"end\": {\"x\": 18.265, \"y\": 11.125, \"z\": 20.503}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.676, \"y\": 7.901, \"z\": 21.106}, \"end\": {\"x\": 20.979, \"y\": 9.165, \"z\": 21.489}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.577, \"y\": 7.844, \"z\": 20.191}, \"end\": {\"x\": 20.979, \"y\": 9.165, \"z\": 21.489}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.304, \"y\": 14.859, \"z\": 19.353}, \"end\": {\"x\": 19.984, \"y\": 13.71, \"z\": 17.928}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.152, \"y\": 11.65, \"z\": 20.058}, \"end\": {\"x\": 18.265, \"y\": 11.125, \"z\": 20.503}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.914, \"y\": 7.989, \"z\": 12.142}, \"end\": {\"x\": 18.709, \"y\": 8.444, \"z\": 11.009}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.248, \"y\": 16.107, \"z\": 15.597}, \"end\": {\"x\": 20.428, \"y\": 17.289, \"z\": 16.325}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.733, \"y\": 16.086, \"z\": 10.744}, \"end\": {\"x\": 16.35, \"y\": 16.649, \"z\": 12.754}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.406, \"y\": 19.29, \"z\": 14.599}, \"end\": {\"x\": 20.528, \"y\": 18.873, \"z\": 12.955}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.22, \"y\": 17.276, \"z\": 13.809}, \"end\": {\"x\": 20.428, \"y\": 17.289, \"z\": 16.325}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.02, \"y\": 20.314, \"z\": 10.519}, \"end\": {\"x\": 17.296, \"y\": 19.943, \"z\": 9.95}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.906, \"y\": 22.366, \"z\": 9.283}, \"end\": {\"x\": 19.599, \"y\": 22.451, \"z\": 9.882}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.532, \"y\": 20.966, \"z\": 4.397}, \"end\": {\"x\": 24.33, \"y\": 21.453, \"z\": 3.889}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.769, \"y\": 23.904, \"z\": 11.282}, \"end\": {\"x\": 23.717, \"y\": 24.466, \"z\": 9.81}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.258, \"y\": 23.006, \"z\": 11.449}, \"end\": {\"x\": 23.16, \"y\": 22.291, \"z\": 12.88}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.851, \"y\": 29.527, \"z\": 10.282}, \"end\": {\"x\": 21.761, \"y\": 29.896, \"z\": 8.791}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.474, \"y\": 25.587, \"z\": 7.885}, \"end\": {\"x\": 19.715, \"y\": 26.133, \"z\": 7.838}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.349, \"y\": 28.421, \"z\": 6.822}, \"end\": {\"x\": 22.146, \"y\": 27.422, \"z\": 6.21}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.851, \"y\": 29.527, \"z\": 10.282}, \"end\": {\"x\": 21.761, \"y\": 29.896, \"z\": 8.791}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.555, \"y\": 30.079, \"z\": 7.169}, \"end\": {\"x\": 17.778, \"y\": 30.183, \"z\": 6.408}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.474, \"y\": 25.587, \"z\": 7.885}, \"end\": {\"x\": 19.715, \"y\": 26.133, \"z\": 7.838}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.349, \"y\": 28.421, \"z\": 6.822}, \"end\": {\"x\": 22.146, \"y\": 27.422, \"z\": 6.21}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.636, \"y\": 23.408, \"z\": 7.315}, \"end\": {\"x\": 15.762, \"y\": 23.796, \"z\": 7.537}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.02, \"y\": 20.314, \"z\": 10.519}, \"end\": {\"x\": 17.296, \"y\": 19.943, \"z\": 9.95}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.906, \"y\": 22.366, \"z\": 9.283}, \"end\": {\"x\": 19.599, \"y\": 22.451, \"z\": 9.882}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.038, \"y\": 17.129, \"z\": 6.773}, \"end\": {\"x\": 13.846, \"y\": 17.084, \"z\": 8.45}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.716, \"y\": 19.498, \"z\": 8.223}, \"end\": {\"x\": 13.546, \"y\": 20.92, \"z\": 7.239}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.733, \"y\": 16.086, \"z\": 10.744}, \"end\": {\"x\": 16.35, \"y\": 16.649, \"z\": 12.754}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.642, \"y\": 13.366, \"z\": 8.44}, \"end\": {\"x\": 13.311, \"y\": 12.252, \"z\": 7.088}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.96, \"y\": 14.236, \"z\": 18.793}, \"end\": {\"x\": 6.053, \"y\": 12.513, \"z\": 18.959}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.338, \"y\": 14.043, \"z\": 19.122}, \"end\": {\"x\": 6.053, \"y\": 12.513, \"z\": 18.959}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.05, \"y\": 18.204, \"z\": 17.759}, \"end\": {\"x\": 4.593, \"y\": 16.926, \"z\": 17.726}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.338, \"y\": 14.043, \"z\": 19.122}, \"end\": {\"x\": 6.053, \"y\": 12.513, \"z\": 18.959}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.334, \"y\": 18.608, \"z\": 19.061}, \"end\": {\"x\": 7.244, \"y\": 17.13, \"z\": 20.009}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.959, \"y\": 11.943, \"z\": 21.698}, \"end\": {\"x\": 9.866, \"y\": 13.537, \"z\": 19.976}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.008, \"y\": 11.869, \"z\": 20.545}, \"end\": {\"x\": 9.866, \"y\": 13.537, \"z\": 19.976}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.96, \"y\": 14.236, \"z\": 18.793}, \"end\": {\"x\": 6.053, \"y\": 12.513, \"z\": 18.959}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.228, \"y\": 19.482, \"z\": 17.41}, \"end\": {\"x\": 9.56, \"y\": 17.36, \"z\": 17.183}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.05, \"y\": 18.204, \"z\": 17.759}, \"end\": {\"x\": 4.593, \"y\": 16.926, \"z\": 17.726}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.51, \"y\": 22.037, \"z\": 16.968}, \"end\": {\"x\": 8.248, \"y\": 21.791, \"z\": 18.62}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.334, \"y\": 18.608, \"z\": 19.061}, \"end\": {\"x\": 7.244, \"y\": 17.13, \"z\": 20.009}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.284, \"y\": 23.52, \"z\": 18.37}, \"end\": {\"x\": 11.347, \"y\": 21.684, \"z\": 19.334}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.228, \"y\": 19.482, \"z\": 17.41}, \"end\": {\"x\": 9.56, \"y\": 17.36, \"z\": 17.183}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.272, \"y\": 24.192, \"z\": 17.259}, \"end\": {\"x\": 12.339, \"y\": 22.929, \"z\": 15.755}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.51, \"y\": 22.037, \"z\": 16.968}, \"end\": {\"x\": 8.248, \"y\": 21.791, \"z\": 18.62}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.284, \"y\": 23.52, \"z\": 18.37}, \"end\": {\"x\": 11.347, \"y\": 21.684, \"z\": 19.334}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.272, \"y\": 24.192, \"z\": 17.259}, \"end\": {\"x\": 12.339, \"y\": 22.929, \"z\": 15.755}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.732, \"y\": 26.85, \"z\": 13.809}, \"end\": {\"x\": 14.622, \"y\": 25.943, \"z\": 15.191}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.882, \"y\": 24.79, \"z\": 14.29}, \"end\": {\"x\": 14.622, \"y\": 25.943, \"z\": 15.191}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.886, \"y\": 30.607, \"z\": 12.787}, \"end\": {\"x\": 15.511, \"y\": 28.672, \"z\": 13.549}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.886, \"y\": 30.607, \"z\": 12.787}, \"end\": {\"x\": 15.511, \"y\": 28.672, \"z\": 13.549}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.555, \"y\": 30.079, \"z\": 7.169}, \"end\": {\"x\": 17.778, \"y\": 30.183, \"z\": 6.408}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.732, \"y\": 26.85, \"z\": 13.809}, \"end\": {\"x\": 14.622, \"y\": 25.943, \"z\": 15.191}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.882, \"y\": 24.79, \"z\": 14.29}, \"end\": {\"x\": 14.622, \"y\": 25.943, \"z\": 15.191}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.636, \"y\": 23.408, \"z\": 7.315}, \"end\": {\"x\": 15.762, \"y\": 23.796, \"z\": 7.537}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.716, \"y\": 19.498, \"z\": 8.223}, \"end\": {\"x\": 13.546, \"y\": 20.92, \"z\": 7.239}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.038, \"y\": 17.129, \"z\": 6.773}, \"end\": {\"x\": 13.846, \"y\": 17.084, \"z\": 8.45}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.642, \"y\": 13.366, \"z\": 8.44}, \"end\": {\"x\": 13.311, \"y\": 12.252, \"z\": 7.088}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.914, \"y\": 7.989, \"z\": 12.142}, \"end\": {\"x\": 18.709, \"y\": 8.444, \"z\": 11.009}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.091, \"y\": 4.456, \"z\": 5.237}, \"end\": {\"x\": 19.965, \"y\": 3.686, \"z\": 6.949}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.838, \"y\": 3.706, \"z\": 3.125}, \"end\": {\"x\": 17.872, \"y\": 2.579, \"z\": 4.769}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.69, \"y\": 4.652, \"z\": 2.527}, \"end\": {\"x\": 15.154, \"y\": 5.049, \"z\": 0.403}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.523, \"y\": 5.967, \"z\": 1.712}, \"end\": {\"x\": 17.606, \"y\": 5.881, \"z\": 3.432}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.162, \"y\": 6.513, \"z\": 1.256}, \"end\": {\"x\": 20.973, \"y\": 6.643, \"z\": 3.153}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.091, \"y\": 4.456, \"z\": 5.237}, \"end\": {\"x\": 19.965, \"y\": 3.686, \"z\": 6.949}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.574, \"y\": 3.639, \"z\": 0.427}, \"end\": {\"x\": 21.484, \"y\": 3.783, \"z\": 1.287}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.189, \"y\": 1.269, \"z\": 6.277}, \"end\": {\"x\": 23.539, \"y\": 1.016, \"z\": 7.048}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.909, \"y\": 1.302, \"z\": 8.984}, \"end\": {\"x\": 23.539, \"y\": 1.016, \"z\": 7.048}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.838, \"y\": 3.706, \"z\": 3.125}, \"end\": {\"x\": 17.872, \"y\": 2.579, \"z\": 4.769}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.69, \"y\": 4.652, \"z\": 2.527}, \"end\": {\"x\": 15.154, \"y\": 5.049, \"z\": 0.403}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.446, \"y\": 5.146, \"z\": -0.786}, \"end\": {\"x\": 19.286, \"y\": 4.661, \"z\": -1.084}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.523, \"y\": 5.967, \"z\": 1.712}, \"end\": {\"x\": 17.606, \"y\": 5.881, \"z\": 3.432}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.162, \"y\": 6.513, \"z\": 1.256}, \"end\": {\"x\": 20.973, \"y\": 6.643, \"z\": 3.153}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.446, \"y\": 5.146, \"z\": -0.786}, \"end\": {\"x\": 19.286, \"y\": 4.661, \"z\": -1.084}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.574, \"y\": 3.639, \"z\": 0.427}, \"end\": {\"x\": 21.484, \"y\": 3.783, \"z\": 1.287}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.888, \"y\": 0.518, \"z\": 4.072}, \"end\": {\"x\": 27.407, \"y\": 1.546, \"z\": 4.023}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.189, \"y\": 1.269, \"z\": 6.277}, \"end\": {\"x\": 23.539, \"y\": 1.016, \"z\": 7.048}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.909, \"y\": 1.302, \"z\": 8.984}, \"end\": {\"x\": 23.539, \"y\": 1.016, \"z\": 7.048}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.901, \"y\": 0.343, \"z\": 12.937}, \"end\": {\"x\": 22.321, \"y\": -0.93, \"z\": 13.376}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.315, \"y\": -0.599, \"z\": 11.516}, \"end\": {\"x\": 22.321, \"y\": -0.93, \"z\": 13.376}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.363, \"y\": 2.396, \"z\": 4.687}, \"end\": {\"x\": 29.675, \"y\": 1.598, \"z\": 6.511}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.04, \"y\": 1.535, \"z\": 12.35}, \"end\": {\"x\": 26.6, \"y\": 2.319, \"z\": 13.158}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.888, \"y\": 0.518, \"z\": 4.072}, \"end\": {\"x\": 27.407, \"y\": 1.546, \"z\": 4.023}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.363, \"y\": 2.396, \"z\": 4.687}, \"end\": {\"x\": 29.675, \"y\": 1.598, \"z\": 6.511}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.878, \"y\": 8.205, \"z\": 2.551}, \"end\": {\"x\": 37.097, \"y\": 6.228, \"z\": 1.86}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.019, \"y\": 5.408, \"z\": 7.359}, \"end\": {\"x\": 35.925, \"y\": 3.85, \"z\": 7.986}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.621, \"y\": 5.328, \"z\": 7.261}, \"end\": {\"x\": 37.004, \"y\": 7.021, \"z\": 8.285}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.414, \"y\": 12.941, \"z\": 7.8}, \"end\": {\"x\": 36.353, \"y\": 11.585, \"z\": 9.01}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.413, \"y\": 17.006, \"z\": 10.162}, \"end\": {\"x\": 40.518, \"y\": 18.929, \"z\": 10.492}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.088, \"y\": 15.459, \"z\": 7.137}, \"end\": {\"x\": 37.59, \"y\": 14.737, \"z\": 7.848}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.929, \"y\": 15.501, \"z\": 4.672}, \"end\": {\"x\": 36.434, \"y\": 14.139, \"z\": 4.761}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.428, \"y\": 15.335, \"z\": 5.219}, \"end\": {\"x\": 33.145, \"y\": 13.497, \"z\": 5.615}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.414, \"y\": 12.941, \"z\": 7.8}, \"end\": {\"x\": 36.353, \"y\": 11.585, \"z\": 9.01}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.639, \"y\": 17.46, \"z\": 6.323}, \"end\": {\"x\": 33.091, \"y\": 16.465, \"z\": 7.33}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.103, \"y\": 20.472, \"z\": 5.476}, \"end\": {\"x\": 33.87, \"y\": 18.443, \"z\": 4.902}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.214, \"y\": 21.583, \"z\": 6.93}, \"end\": {\"x\": 38.089, \"y\": 19.966, \"z\": 6.516}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.088, \"y\": 15.459, \"z\": 7.137}, \"end\": {\"x\": 37.59, \"y\": 14.737, \"z\": 7.848}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.879, \"y\": 19.459, \"z\": 4.385}, \"end\": {\"x\": 33.87, \"y\": 18.443, \"z\": 4.902}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.929, \"y\": 15.501, \"z\": 4.672}, \"end\": {\"x\": 36.434, \"y\": 14.139, \"z\": 4.761}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.897, \"y\": 18.313, \"z\": 2.739}, \"end\": {\"x\": 31.369, \"y\": 17.02, \"z\": 2.793}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.962, \"y\": 12.232, \"z\": 2.225}, \"end\": {\"x\": 36.994, \"y\": 13.412, \"z\": 1.034}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.169, \"y\": 10.988, \"z\": 1.591}, \"end\": {\"x\": 35.028, \"y\": 12.492, \"z\": 1.601}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.428, \"y\": 15.335, \"z\": 5.219}, \"end\": {\"x\": 33.145, \"y\": 13.497, \"z\": 5.615}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.5, \"y\": 18.596, \"z\": 4.228}, \"end\": {\"x\": 28.723, \"y\": 17.484, \"z\": 5.149}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.639, \"y\": 17.46, \"z\": 6.323}, \"end\": {\"x\": 33.091, \"y\": 16.465, \"z\": 7.33}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.481, \"y\": 21.316, \"z\": 4.288}, \"end\": {\"x\": 29.166, \"y\": 20.856, \"z\": 5.249}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.648, \"y\": 21.849, \"z\": 4.529}, \"end\": {\"x\": 33.091, \"y\": 21.217, \"z\": 2.538}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.879, \"y\": 19.459, \"z\": 4.385}, \"end\": {\"x\": 33.87, \"y\": 18.443, \"z\": 4.902}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.825, \"y\": 21.875, \"z\": 1.587}, \"end\": {\"x\": 28.708, \"y\": 21.164, \"z\": 1.913}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.897, \"y\": 18.313, \"z\": 2.739}, \"end\": {\"x\": 31.369, \"y\": 17.02, \"z\": 2.793}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.32, \"y\": 21.37, \"z\": 1.386}, \"end\": {\"x\": 25.776, \"y\": 19.482, \"z\": 1.693}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.532, \"y\": 20.966, \"z\": 4.397}, \"end\": {\"x\": 24.33, \"y\": 21.453, \"z\": 3.889}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.5, \"y\": 18.596, \"z\": 4.228}, \"end\": {\"x\": 28.723, \"y\": 17.484, \"z\": 5.149}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.481, \"y\": 21.316, \"z\": 4.288}, \"end\": {\"x\": 29.166, \"y\": 20.856, \"z\": 5.249}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.77, \"y\": 23.646, \"z\": 1.62}, \"end\": {\"x\": 24.968, \"y\": 24.326, \"z\": 2.365}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.825, \"y\": 21.875, \"z\": 1.587}, \"end\": {\"x\": 28.708, \"y\": 21.164, \"z\": 1.913}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.923, \"y\": 23.377, \"z\": -3.565}, \"end\": {\"x\": 24.041, \"y\": 22.742, \"z\": -1.396}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.32, \"y\": 21.37, \"z\": 1.386}, \"end\": {\"x\": 25.776, \"y\": 19.482, \"z\": 1.693}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.61, \"y\": 23.881, \"z\": -2.316}, \"end\": {\"x\": 20.713, \"y\": 22.777, \"z\": -0.595}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.77, \"y\": 23.646, \"z\": 1.62}, \"end\": {\"x\": 24.968, \"y\": 24.326, \"z\": 2.365}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.923, \"y\": 23.377, \"z\": -3.565}, \"end\": {\"x\": 24.75, \"y\": 25.361, \"z\": -2.758}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.61, \"y\": 23.881, \"z\": -2.316}, \"end\": {\"x\": 20.713, \"y\": 22.777, \"z\": -0.595}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.923, \"y\": 23.377, \"z\": -3.565}, \"end\": {\"x\": 24.041, \"y\": 22.742, \"z\": -1.396}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.923, \"y\": 23.377, \"z\": -3.565}, \"end\": {\"x\": 24.75, \"y\": 25.361, \"z\": -2.758}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.433, \"y\": 14.03, \"z\": -0.348}, \"end\": {\"x\": 22.854, \"y\": 14.846, \"z\": -2.253}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.332, \"y\": 12.172, \"z\": 1.276}, \"end\": {\"x\": 21.456, \"y\": 12.364, \"z\": -0.339}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.39, \"y\": 10.535, \"z\": 1.803}, \"end\": {\"x\": 24.062, \"y\": 10.398, \"z\": -0.187}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.709, \"y\": 12.174, \"z\": 3.33}, \"end\": {\"x\": 25.867, \"y\": 13.087, \"z\": 1.416}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.433, \"y\": 14.03, \"z\": -0.348}, \"end\": {\"x\": 22.854, \"y\": 14.846, \"z\": -2.253}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.484, \"y\": 12.556, \"z\": 5.51}, \"end\": {\"x\": 23.687, \"y\": 13.335, \"z\": 3.854}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.332, \"y\": 12.172, \"z\": 1.276}, \"end\": {\"x\": 21.456, \"y\": 12.364, \"z\": -0.339}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.915, \"y\": 10.158, \"z\": 6.527}, \"end\": {\"x\": 23.654, \"y\": 10.062, \"z\": 4.826}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.39, \"y\": 10.535, \"z\": 1.803}, \"end\": {\"x\": 24.062, \"y\": 10.398, \"z\": -0.187}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.441, \"y\": 10.159, \"z\": 7.213}, \"end\": {\"x\": 27.087, \"y\": 10.046, \"z\": 5.181}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.709, \"y\": 12.174, \"z\": 3.33}, \"end\": {\"x\": 25.867, \"y\": 13.087, \"z\": 1.416}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.725, \"y\": 12.164, \"z\": 8.996}, \"end\": {\"x\": 27.019, \"y\": 12.903, \"z\": 7.317}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.484, \"y\": 12.556, \"z\": 5.51}, \"end\": {\"x\": 23.687, \"y\": 13.335, \"z\": 3.854}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.109, \"y\": 11.141, \"z\": 11.066}, \"end\": {\"x\": 25.036, \"y\": 11.119, \"z\": 9.543}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.915, \"y\": 10.158, \"z\": 6.527}, \"end\": {\"x\": 23.654, \"y\": 10.062, \"z\": 4.826}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.52, \"y\": 8.984, \"z\": 11.945}, \"end\": {\"x\": 27.14, \"y\": 8.43, \"z\": 9.823}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.096, \"y\": 6.378, \"z\": 9.65}, \"end\": {\"x\": 27.14, \"y\": 8.43, \"z\": 9.823}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.441, \"y\": 10.159, \"z\": 7.213}, \"end\": {\"x\": 27.087, \"y\": 10.046, \"z\": 5.181}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.727, \"y\": 10.124, \"z\": 12.868}, \"end\": {\"x\": 29.604, \"y\": 10.466, \"z\": 10.868}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.725, \"y\": 12.164, \"z\": 8.996}, \"end\": {\"x\": 27.019, \"y\": 12.903, \"z\": 7.317}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.053, \"y\": 11.197, \"z\": 14.964}, \"end\": {\"x\": 27.989, \"y\": 12.241, \"z\": 13.372}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.109, \"y\": 11.141, \"z\": 11.066}, \"end\": {\"x\": 25.036, \"y\": 11.119, \"z\": 9.543}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.311, \"y\": 8.699, \"z\": 16.702}, \"end\": {\"x\": 26.812, \"y\": 9.545, \"z\": 14.976}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.998, \"y\": 11.41, \"z\": 16.278}, \"end\": {\"x\": 26.812, \"y\": 9.545, \"z\": 14.976}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.52, \"y\": 8.984, \"z\": 11.945}, \"end\": {\"x\": 27.14, \"y\": 8.43, \"z\": 9.823}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.096, \"y\": 6.378, \"z\": 9.65}, \"end\": {\"x\": 27.14, \"y\": 8.43, \"z\": 9.823}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.727, \"y\": 10.124, \"z\": 12.868}, \"end\": {\"x\": 29.604, \"y\": 10.466, \"z\": 10.868}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.053, \"y\": 11.197, \"z\": 14.964}, \"end\": {\"x\": 27.989, \"y\": 12.241, \"z\": 13.372}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.387, \"y\": 10.151, \"z\": 20.122}, \"end\": {\"x\": 28.848, \"y\": 11.483, \"z\": 18.823}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.998, \"y\": 11.41, \"z\": 16.278}, \"end\": {\"x\": 26.812, \"y\": 9.545, \"z\": 14.976}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.823, \"y\": 8.176, \"z\": 21.348}, \"end\": {\"x\": 26.721, \"y\": 9.067, \"z\": 19.876}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.311, \"y\": 8.699, \"z\": 16.702}, \"end\": {\"x\": 26.812, \"y\": 9.545, \"z\": 14.976}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.378, \"y\": 5.089, \"z\": 19.022}, \"end\": {\"x\": 25.19, \"y\": 5.046, \"z\": 17.987}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.646, \"y\": 6.297, \"z\": 21.99}, \"end\": {\"x\": 29.043, \"y\": 6.408, \"z\": 19.932}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.984, \"y\": 8.251, \"z\": 23.58}, \"end\": {\"x\": 31.365, \"y\": 8.299, \"z\": 21.71}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.585, \"y\": 9.377, \"z\": 24.985}, \"end\": {\"x\": 29.035, \"y\": 8.741, \"z\": 24.226}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.387, \"y\": 10.151, \"z\": 20.122}, \"end\": {\"x\": 28.848, \"y\": 11.483, \"z\": 18.823}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.676, \"y\": 7.901, \"z\": 21.106}, \"end\": {\"x\": 20.979, \"y\": 9.165, \"z\": 21.489}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.577, \"y\": 7.844, \"z\": 20.191}, \"end\": {\"x\": 20.979, \"y\": 9.165, \"z\": 21.489}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.823, \"y\": 8.176, \"z\": 21.348}, \"end\": {\"x\": 26.721, \"y\": 9.067, \"z\": 19.876}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.378, \"y\": 5.089, \"z\": 19.022}, \"end\": {\"x\": 25.19, \"y\": 5.046, \"z\": 17.987}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.872, \"y\": 4.599, \"z\": 25.883}, \"end\": {\"x\": 27.99, \"y\": 5.195, \"z\": 24.411}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.0, \"y\": 5.451, \"z\": 26.696}, \"end\": {\"x\": 27.99, \"y\": 5.195, \"z\": 24.411}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.646, \"y\": 6.297, \"z\": 21.99}, \"end\": {\"x\": 29.043, \"y\": 6.408, \"z\": 19.932}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.581, \"y\": 5.957, \"z\": 20.627}, \"end\": {\"x\": 32.216, \"y\": 5.401, \"z\": 22.014}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.144, \"y\": 5.125, \"z\": 20.774}, \"end\": {\"x\": 35.545, \"y\": 4.537, \"z\": 22.462}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.515, \"y\": 3.995, \"z\": 18.216}, \"end\": {\"x\": 35.455, \"y\": 2.667, \"z\": 19.609}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.117, \"y\": -3.258, \"z\": 17.294}, \"end\": {\"x\": 33.871, \"y\": -3.946, \"z\": 16.14}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.153, \"y\": 2.588, \"z\": 23.301}, \"end\": {\"x\": 31.381, \"y\": 2.001, \"z\": 24.611}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.631, \"y\": 6.268, \"z\": 16.813}, \"end\": {\"x\": 34.733, \"y\": 5.692, \"z\": 17.713}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.568, \"y\": 9.273, \"z\": 25.687}, \"end\": {\"x\": 35.521, \"y\": 8.526, \"z\": 23.943}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.693, \"y\": 8.18, \"z\": 23.949}, \"end\": {\"x\": 35.521, \"y\": 8.526, \"z\": 23.943}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.581, \"y\": 5.957, \"z\": 20.627}, \"end\": {\"x\": 32.216, \"y\": 5.401, \"z\": 22.014}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.705, \"y\": 7.421, \"z\": 17.584}, \"end\": {\"x\": 37.296, \"y\": 7.386, \"z\": 18.882}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.144, \"y\": 5.125, \"z\": 20.774}, \"end\": {\"x\": 35.545, \"y\": 4.537, \"z\": 22.462}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.586, \"y\": 5.892, \"z\": 16.531}, \"end\": {\"x\": 39.562, \"y\": 4.877, \"z\": 18.074}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.515, \"y\": 3.995, \"z\": 18.216}, \"end\": {\"x\": 35.455, \"y\": 2.667, \"z\": 19.609}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.537, \"y\": 6.172, \"z\": 14.029}, \"end\": {\"x\": 38.667, \"y\": 5.185, \"z\": 14.645}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.032, \"y\": 3.99, \"z\": 13.063}, \"end\": {\"x\": 38.667, \"y\": 5.185, \"z\": 14.645}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.406, \"y\": 2.36, \"z\": 13.167}, \"end\": {\"x\": 36.971, \"y\": 0.858, \"z\": 14.362}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.631, \"y\": 6.268, \"z\": 16.813}, \"end\": {\"x\": 34.733, \"y\": 5.692, \"z\": 17.713}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.454, \"y\": 8.807, \"z\": 13.889}, \"end\": {\"x\": 39.534, \"y\": 8.662, \"z\": 14.789}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.705, \"y\": 7.421, \"z\": 17.584}, \"end\": {\"x\": 37.296, \"y\": 7.386, \"z\": 18.882}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.814, \"y\": 8.492, \"z\": 14.578}, \"end\": {\"x\": 42.655, \"y\": 8.199, \"z\": 16.098}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.586, \"y\": 5.892, \"z\": 16.531}, \"end\": {\"x\": 39.562, \"y\": 4.877, \"z\": 18.074}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.284, \"y\": 7.188, \"z\": 12.773}, \"end\": {\"x\": 43.598, \"y\": 6.119, \"z\": 13.52}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.537, \"y\": 6.172, \"z\": 14.029}, \"end\": {\"x\": 38.667, \"y\": 5.185, \"z\": 14.645}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.032, \"y\": 3.99, \"z\": 13.063}, \"end\": {\"x\": 38.667, \"y\": 5.185, \"z\": 14.645}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.406, \"y\": 2.36, \"z\": 13.167}, \"end\": {\"x\": 36.971, \"y\": 0.858, \"z\": 14.362}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.468, \"y\": 7.907, \"z\": 9.403}, \"end\": {\"x\": 42.862, \"y\": 8.376, \"z\": 11.086}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.956, \"y\": 4.479, \"z\": 9.761}, \"end\": {\"x\": 35.925, \"y\": 3.85, \"z\": 7.986}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.454, \"y\": 8.807, \"z\": 13.889}, \"end\": {\"x\": 39.534, \"y\": 8.662, \"z\": 14.789}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.814, \"y\": 8.492, \"z\": 14.578}, \"end\": {\"x\": 42.655, \"y\": 8.199, \"z\": 16.098}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.284, \"y\": 7.188, \"z\": 12.773}, \"end\": {\"x\": 43.598, \"y\": 6.119, \"z\": 13.52}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.468, \"y\": 7.907, \"z\": 9.403}, \"end\": {\"x\": 42.862, \"y\": 8.376, \"z\": 11.086}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.878, \"y\": 8.205, \"z\": 2.551}, \"end\": {\"x\": 37.097, \"y\": 6.228, \"z\": 1.86}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.962, \"y\": 12.232, \"z\": 2.225}, \"end\": {\"x\": 36.994, \"y\": 13.412, \"z\": 1.034}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.169, \"y\": 10.988, \"z\": 1.591}, \"end\": {\"x\": 35.028, \"y\": 12.492, \"z\": 1.601}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.019, \"y\": 5.408, \"z\": 7.359}, \"end\": {\"x\": 35.925, \"y\": 3.85, \"z\": 7.986}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.621, \"y\": 5.328, \"z\": 7.261}, \"end\": {\"x\": 37.004, \"y\": 7.021, \"z\": 8.285}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.956, \"y\": 4.479, \"z\": 9.761}, \"end\": {\"x\": 35.925, \"y\": 3.85, \"z\": 7.986}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.624, \"y\": -5.14, \"z\": 13.994}, \"end\": {\"x\": 34.203, \"y\": -3.779, \"z\": 13.654}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.01, \"y\": -2.618, \"z\": 14.463}, \"end\": {\"x\": 31.596, \"y\": -2.749, \"z\": 12.244}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.433, \"y\": -3.023, \"z\": 14.797}, \"end\": {\"x\": 36.395, \"y\": -3.476, \"z\": 13.179}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.006, \"y\": -3.944, \"z\": 14.468}, \"end\": {\"x\": 29.489, \"y\": -5.129, \"z\": 14.103}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.117, \"y\": -3.258, \"z\": 17.294}, \"end\": {\"x\": 33.871, \"y\": -3.946, \"z\": 16.14}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.624, \"y\": -5.14, \"z\": 13.994}, \"end\": {\"x\": 34.203, \"y\": -3.779, \"z\": 13.654}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.076, \"y\": -2.878, \"z\": 17.19}, \"end\": {\"x\": 30.097, \"y\": -3.245, \"z\": 16.978}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.01, \"y\": -2.618, \"z\": 14.463}, \"end\": {\"x\": 31.596, \"y\": -2.749, \"z\": 12.244}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.9, \"y\": -0.226, \"z\": 16.697}, \"end\": {\"x\": 28.486, \"y\": -0.452, \"z\": 15.668}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.327, \"y\": -1.266, \"z\": 15.937}, \"end\": {\"x\": 25.648, \"y\": -1.828, \"z\": 14.604}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.006, \"y\": -3.944, \"z\": 14.468}, \"end\": {\"x\": 29.489, \"y\": -5.129, \"z\": 14.103}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.573, \"y\": -1.761, \"z\": 18.43}, \"end\": {\"x\": 24.971, \"y\": -3.367, \"z\": 17.622}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.076, \"y\": -2.878, \"z\": 17.19}, \"end\": {\"x\": 30.097, \"y\": -3.245, \"z\": 16.978}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.573, \"y\": -1.761, \"z\": 18.43}, \"end\": {\"x\": 25.059, \"y\": -0.588, \"z\": 19.228}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.04, \"y\": 1.535, \"z\": 12.35}, \"end\": {\"x\": 26.6, \"y\": 2.319, \"z\": 13.158}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.9, \"y\": -0.226, \"z\": 16.697}, \"end\": {\"x\": 28.486, \"y\": -0.452, \"z\": 15.668}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.438, \"y\": 0.065, \"z\": 19.704}, \"end\": {\"x\": 22.765, \"y\": 1.025, \"z\": 17.468}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.901, \"y\": 0.343, \"z\": 12.937}, \"end\": {\"x\": 22.321, \"y\": -0.93, \"z\": 13.376}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.315, \"y\": -0.599, \"z\": 11.516}, \"end\": {\"x\": 22.321, \"y\": -0.93, \"z\": 13.376}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.327, \"y\": -1.266, \"z\": 15.937}, \"end\": {\"x\": 25.648, \"y\": -1.828, \"z\": 14.604}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.573, \"y\": -1.761, \"z\": 18.43}, \"end\": {\"x\": 24.971, \"y\": -3.367, \"z\": 17.622}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.573, \"y\": -1.761, \"z\": 18.43}, \"end\": {\"x\": 25.059, \"y\": -0.588, \"z\": 19.228}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.173, \"y\": -2.137, \"z\": 23.701}, \"end\": {\"x\": 21.704, \"y\": -1.933, \"z\": 22.061}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.438, \"y\": 0.065, \"z\": 19.704}, \"end\": {\"x\": 22.765, \"y\": 1.025, \"z\": 17.468}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.962, \"y\": -0.189, \"z\": 25.583}, \"end\": {\"x\": 21.282, \"y\": 0.722, \"z\": 23.889}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.433, \"y\": 4.5, \"z\": 24.762}, \"end\": {\"x\": 24.716, \"y\": 5.013, \"z\": 26.538}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.426, \"y\": -0.924, \"z\": 26.807}, \"end\": {\"x\": 18.174, \"y\": -0.32, \"z\": 24.765}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.223, \"y\": 0.705, \"z\": 24.634}, \"end\": {\"x\": 18.174, \"y\": -0.32, \"z\": 24.765}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.055, \"y\": -8.147, \"z\": 21.295}, \"end\": {\"x\": 19.062, \"y\": -9.623, \"z\": 21.045}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.23, \"y\": -3.459, \"z\": 27.707}, \"end\": {\"x\": 19.341, \"y\": -3.595, \"z\": 25.863}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.173, \"y\": -2.137, \"z\": 23.701}, \"end\": {\"x\": 21.704, \"y\": -1.933, \"z\": 22.061}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.023, \"y\": -2.699, \"z\": 29.688}, \"end\": {\"x\": 21.697, \"y\": -2.048, \"z\": 28.029}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.962, \"y\": -0.189, \"z\": 25.583}, \"end\": {\"x\": 21.282, \"y\": 0.722, \"z\": 23.889}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.395, \"y\": 4.827, \"z\": 26.301}, \"end\": {\"x\": 20.943, \"y\": 2.426, \"z\": 26.26}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.813, \"y\": -1.092, \"z\": 31.54}, \"end\": {\"x\": 19.442, \"y\": -0.17, \"z\": 29.679}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.426, \"y\": -0.924, \"z\": 26.807}, \"end\": {\"x\": 18.174, \"y\": -0.32, \"z\": 24.765}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.96, \"y\": 0.894, \"z\": 26.802}, \"end\": {\"x\": 15.832, \"y\": -1.134, \"z\": 26.886}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.838, \"y\": -2.942, \"z\": 32.37}, \"end\": {\"x\": 17.719, \"y\": -2.954, \"z\": 30.557}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.23, \"y\": -3.459, \"z\": 27.707}, \"end\": {\"x\": 19.341, \"y\": -3.595, \"z\": 25.863}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.023, \"y\": -2.699, \"z\": 29.688}, \"end\": {\"x\": 21.697, \"y\": -2.048, \"z\": 28.029}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.253, \"y\": -4.171, \"z\": 34.115}, \"end\": {\"x\": 20.985, \"y\": -2.67, \"z\": 33.953}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.813, \"y\": -1.092, \"z\": 31.54}, \"end\": {\"x\": 19.442, \"y\": -0.17, \"z\": 29.679}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.319, \"y\": 4.325, \"z\": 31.439}, \"end\": {\"x\": 17.711, \"y\": 3.336, \"z\": 32.385}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.838, \"y\": -2.942, \"z\": 32.37}, \"end\": {\"x\": 17.719, \"y\": -2.954, \"z\": 30.557}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.253, \"y\": -4.171, \"z\": 34.115}, \"end\": {\"x\": 20.985, \"y\": -2.67, \"z\": 33.953}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.476, \"y\": -5.733, \"z\": 37.42}, \"end\": {\"x\": 21.654, \"y\": -4.976, \"z\": 35.459}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.091, \"y\": -8.294, \"z\": 36.064}, \"end\": {\"x\": 21.165, \"y\": -9.274, \"z\": 35.061}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.111, \"y\": -8.314, \"z\": 37.149}, \"end\": {\"x\": 21.958, \"y\": -8.621, \"z\": 38.755}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.476, \"y\": -5.733, \"z\": 37.42}, \"end\": {\"x\": 21.654, \"y\": -4.976, \"z\": 35.459}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.458, \"y\": -6.806, \"z\": 33.788}, \"end\": {\"x\": 24.914, \"y\": -7.454, \"z\": 34.702}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.091, \"y\": -8.294, \"z\": 36.064}, \"end\": {\"x\": 21.165, \"y\": -9.274, \"z\": 35.061}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.111, \"y\": -8.314, \"z\": 37.149}, \"end\": {\"x\": 21.958, \"y\": -8.621, \"z\": 38.755}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.36, \"y\": -10.285, \"z\": 33.074}, \"end\": {\"x\": 24.804, \"y\": -11.043, \"z\": 31.996}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.569, \"y\": -10.617, \"z\": 25.752}, \"end\": {\"x\": 25.012, \"y\": -10.907, \"z\": 27.445}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.215, \"y\": -11.008, \"z\": 24.873}, \"end\": {\"x\": 24.771, \"y\": -10.372, \"z\": 23.656}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.36, \"y\": -10.571, \"z\": 21.697}, \"end\": {\"x\": 20.329, \"y\": -11.416, \"z\": 21.47}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.737, \"y\": -10.52, \"z\": 20.806}, \"end\": {\"x\": 27.369, \"y\": -12.15, \"z\": 21.261}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.539, \"y\": -12.503, \"z\": 19.221}, \"end\": {\"x\": 27.859, \"y\": -10.743, \"z\": 18.493}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.146, \"y\": -10.582, \"z\": 12.681}, \"end\": {\"x\": 30.018, \"y\": -12.205, \"z\": 13.034}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.305, \"y\": -9.829, \"z\": 13.149}, \"end\": {\"x\": 32.997, \"y\": -8.599, \"z\": 14.62}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.776, \"y\": -10.723, \"z\": 16.552}, \"end\": {\"x\": 30.881, \"y\": -12.226, \"z\": 16.13}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.433, \"y\": -3.023, \"z\": 14.797}, \"end\": {\"x\": 36.395, \"y\": -3.476, \"z\": 13.179}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.646, \"y\": -8.132, \"z\": 16.296}, \"end\": {\"x\": 36.755, \"y\": -8.306, \"z\": 17.754}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.859, \"y\": -4.714, \"z\": 17.531}, \"end\": {\"x\": 40.405, \"y\": -6.532, \"z\": 18.442}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.126, \"y\": -12.49, \"z\": 13.593}, \"end\": {\"x\": 35.033, \"y\": -12.482, \"z\": 14.108}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.146, \"y\": -10.582, \"z\": 12.681}, \"end\": {\"x\": 30.018, \"y\": -12.205, \"z\": 13.034}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.305, \"y\": -9.829, \"z\": 13.149}, \"end\": {\"x\": 32.997, \"y\": -8.599, \"z\": 14.62}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.776, \"y\": -10.723, \"z\": 16.552}, \"end\": {\"x\": 30.881, \"y\": -12.226, \"z\": 16.13}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.126, \"y\": -12.49, \"z\": 13.593}, \"end\": {\"x\": 35.033, \"y\": -12.482, \"z\": 14.108}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.737, \"y\": -10.52, \"z\": 20.806}, \"end\": {\"x\": 27.369, \"y\": -12.15, \"z\": 21.261}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.539, \"y\": -12.503, \"z\": 19.221}, \"end\": {\"x\": 27.859, \"y\": -10.743, \"z\": 18.493}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.569, \"y\": -10.617, \"z\": 25.752}, \"end\": {\"x\": 25.012, \"y\": -10.907, \"z\": 27.445}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.215, \"y\": -11.008, \"z\": 24.873}, \"end\": {\"x\": 24.771, \"y\": -10.372, \"z\": 23.656}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.36, \"y\": -10.285, \"z\": 33.074}, \"end\": {\"x\": 24.804, \"y\": -11.043, \"z\": 31.996}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.458, \"y\": -6.806, \"z\": 33.788}, \"end\": {\"x\": 24.914, \"y\": -7.454, \"z\": 34.702}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.352, \"y\": -4.744, \"z\": 30.318}, \"end\": {\"x\": 29.377, \"y\": -4.227, \"z\": 31.847}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.401, \"y\": -2.721, \"z\": 31.038}, \"end\": {\"x\": 32.474, \"y\": -4.347, \"z\": 32.227}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.352, \"y\": -4.744, \"z\": 30.318}, \"end\": {\"x\": 29.377, \"y\": -4.227, \"z\": 31.847}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.495, \"y\": -0.44, \"z\": 27.842}, \"end\": {\"x\": 30.724, \"y\": -1.005, \"z\": 28.563}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.426, \"y\": 0.636, \"z\": 27.295}, \"end\": {\"x\": 30.724, \"y\": -1.005, \"z\": 28.563}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.401, \"y\": -2.721, \"z\": 31.038}, \"end\": {\"x\": 32.474, \"y\": -4.347, \"z\": 32.227}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.153, \"y\": 2.588, \"z\": 23.301}, \"end\": {\"x\": 31.381, \"y\": 2.001, \"z\": 24.611}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.495, \"y\": -0.44, \"z\": 27.842}, \"end\": {\"x\": 30.724, \"y\": -1.005, \"z\": 28.563}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.426, \"y\": 0.636, \"z\": 27.295}, \"end\": {\"x\": 30.724, \"y\": -1.005, \"z\": 28.563}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.872, \"y\": 4.599, \"z\": 25.883}, \"end\": {\"x\": 27.99, \"y\": 5.195, \"z\": 24.411}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.0, \"y\": 5.451, \"z\": 26.696}, \"end\": {\"x\": 27.99, \"y\": 5.195, \"z\": 24.411}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.433, \"y\": 4.5, \"z\": 24.762}, \"end\": {\"x\": 24.716, \"y\": 5.013, \"z\": 26.538}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.504, \"y\": 6.59, \"z\": 28.902}, \"end\": {\"x\": 24.635, \"y\": 6.904, \"z\": 30.764}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.1, \"y\": 8.452, \"z\": 29.548}, \"end\": {\"x\": 20.985, \"y\": 6.93, \"z\": 28.587}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.395, \"y\": 4.827, \"z\": 26.301}, \"end\": {\"x\": 20.943, \"y\": 2.426, \"z\": 26.26}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.223, \"y\": 0.705, \"z\": 24.634}, \"end\": {\"x\": 18.174, \"y\": -0.32, \"z\": 24.765}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.96, \"y\": 0.894, \"z\": 26.802}, \"end\": {\"x\": 15.832, \"y\": -1.134, \"z\": 26.886}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.319, \"y\": 4.325, \"z\": 31.439}, \"end\": {\"x\": 17.711, \"y\": 3.336, \"z\": 32.385}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.916, \"y\": 10.092, \"z\": 21.771}, \"end\": {\"x\": -16.106, \"y\": 8.237, \"z\": 22.298}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.442, \"y\": 11.601, \"z\": 20.613}, \"end\": {\"x\": -16.885, \"y\": 11.857, \"z\": 22.649}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.995, \"y\": 11.453, \"z\": 16.297}, \"end\": {\"x\": -15.032, \"y\": 11.567, \"z\": 17.846}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.916, \"y\": 10.092, \"z\": 21.771}, \"end\": {\"x\": -16.106, \"y\": 8.237, \"z\": 22.298}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.318, \"y\": 8.481, \"z\": 17.599}, \"end\": {\"x\": -18.765, \"y\": 9.805, \"z\": 18.36}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.442, \"y\": 11.601, \"z\": 20.613}, \"end\": {\"x\": -16.885, \"y\": 11.857, \"z\": 22.649}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.798, \"y\": -6.707, \"z\": 21.607}, \"end\": {\"x\": 38.064, \"y\": -4.905, \"z\": 22.15}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.242, \"y\": -8.343, \"z\": 20.507}, \"end\": {\"x\": 38.701, \"y\": -8.563, \"z\": 22.522}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.646, \"y\": -8.132, \"z\": 16.296}, \"end\": {\"x\": 36.755, \"y\": -8.306, \"z\": 17.754}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.798, \"y\": -6.707, \"z\": 21.607}, \"end\": {\"x\": 38.064, \"y\": -4.905, \"z\": 22.15}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.859, \"y\": -4.714, \"z\": 17.531}, \"end\": {\"x\": 40.405, \"y\": -6.532, \"z\": 18.442}});\n\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.242, \"y\": -8.343, \"z\": 20.507}, \"end\": {\"x\": 38.701, \"y\": -8.563, \"z\": 22.522}});\nviewer_15942021952474504.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_15942021952474504\"  style=\"position: relative; width: 1200px; height: 800px\">\n",
              "        <p id=\"3dmolwarning_15942021952474504\" style=\"background-color:#ffcccc;color:black\">You appear to be running in JupyterLab (or JavaScript failed to load for some other reason).  You need to install the 3dmol extension: <br>\n",
              "        <tt>jupyter labextension install jupyterlab_3dmol</tt></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "      resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.csb.pitt.edu/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_15942021952474504 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_15942021952474504\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_15942021952474504 = $3Dmol.createViewer($(\"#3dmolviewer_15942021952474504\"),{backgroundColor:\"white\"});\n",
              "\tviewer_15942021952474504.addModels(\"HEADER                                            08-JUL-20   XXXX              \\nEXPDTA    THEORETICAL MODEL                                                     \\nREMARK 220                                                                      \\nREMARK 220 EXPERIMENTAL DETAILS                                                 \\nREMARK 220  EXPERIMENT TYPE                : THEORETICAL MODELLING              \\nREMARK 220  DATE OF DATA COLLECTION        : 08-JUL-20                          \\nREMARK 220                                                                      \\nREMARK 220 REMARK: MODEL GENERATED BY ROSETTA                                   \\nREMARK 220  VERSION 2020.27+release.cd4c8de                                     \\nHETNAM     LG1 E 523  LG1                                                       \\nATOM      1  N   TYR A   1       1.933 -23.542  23.417  1.00  0.00           N  \\nATOM      2  CA  TYR A   1       1.383 -22.219  23.670  1.00  0.00           C  \\nATOM      3  C   TYR A   1      -0.123 -22.278  23.695  1.00  0.00           C  \\nATOM      4  O   TYR A   1      -0.702 -23.130  22.973  1.00  0.00           O  \\nATOM      5  CB  TYR A   1       1.813 -21.241  22.588  1.00  0.00           C  \\nATOM      6  CG  TYR A   1       3.328 -20.953  22.584  1.00  0.00           C  \\nATOM      7  CD1 TYR A   1       3.914 -20.324  23.650  1.00  0.00           C  \\nATOM      8  CD2 TYR A   1       4.121 -21.306  21.504  1.00  0.00           C  \\nATOM      9  CE1 TYR A   1       5.270 -20.103  23.661  1.00  0.00           C  \\nATOM     10  CE2 TYR A   1       5.514 -21.014  21.466  1.00  0.00           C  \\nATOM     11  CZ  TYR A   1       6.039 -20.389  22.583  1.00  0.00           C  \\nATOM     12  OH  TYR A   1       7.368 -20.114  22.655  1.00  0.00           O  \\nATOM     13 1H   TYR A   1       2.932 -23.491  23.403  1.00  0.00           H  \\nATOM     14 2H   TYR A   1       1.642 -24.168  24.140  1.00  0.00           H  \\nATOM     15 3H   TYR A   1       1.604 -23.877  22.534  1.00  0.00           H  \\nATOM     16  HA  TYR A   1       1.763 -21.864  24.628  1.00  0.00           H  \\nATOM     17 1HB  TYR A   1       1.537 -21.636  21.609  1.00  0.00           H  \\nATOM     18 2HB  TYR A   1       1.285 -20.297  22.721  1.00  0.00           H  \\nATOM     19  HD1 TYR A   1       3.304 -19.997  24.493  1.00  0.00           H  \\nATOM     20  HD2 TYR A   1       3.681 -21.822  20.650  1.00  0.00           H  \\nATOM     21  HE1 TYR A   1       5.734 -19.602  24.511  1.00  0.00           H  \\nATOM     22  HE2 TYR A   1       6.103 -21.282  20.589  1.00  0.00           H  \\nATOM     23  HH  TYR A   1       7.800 -20.416  21.853  1.00  0.00           H  \\nATOM     24  N   PRO A   2      -0.781 -21.400  24.462  1.00  0.00           N  \\nATOM     25  CA  PRO A   2      -2.232 -21.470  24.468  1.00  0.00           C  \\nATOM     26  C   PRO A   2      -2.859 -21.124  23.102  1.00  0.00           C  \\nATOM     27  O   PRO A   2      -2.334 -20.299  22.404  1.00  0.00           O  \\nATOM     28  CB  PRO A   2      -2.657 -20.419  25.487  1.00  0.00           C  \\nATOM     29  CG  PRO A   2      -1.424 -20.087  26.272  1.00  0.00           C  \\nATOM     30  CD  PRO A   2      -0.295 -20.301  25.338  1.00  0.00           C  \\nATOM     31  HA  PRO A   2      -2.544 -22.472  24.798  1.00  0.00           H  \\nATOM     32 1HB  PRO A   2      -3.066 -19.538  24.971  1.00  0.00           H  \\nATOM     33 2HB  PRO A   2      -3.458 -20.818  26.127  1.00  0.00           H  \\nATOM     34 1HG  PRO A   2      -1.472 -19.049  26.634  1.00  0.00           H  \\nATOM     35 2HG  PRO A   2      -1.355 -20.731  27.161  1.00  0.00           H  \\nATOM     36 1HD  PRO A   2      -0.111 -19.379  24.767  1.00  0.00           H  \\nATOM     37 2HD  PRO A   2       0.601 -20.591  25.906  1.00  0.00           H  \\nATOM     38  N   ILE A   3      -3.967 -21.745  22.755  1.00  0.00           N  \\nATOM     39  CA  ILE A   3      -4.615 -21.520  21.456  1.00  0.00           C  \\nATOM     40  C   ILE A   3      -5.958 -20.963  21.764  1.00  0.00           C  \\nATOM     41  O   ILE A   3      -6.690 -21.511  22.565  1.00  0.00           O  \\nATOM     42  CB  ILE A   3      -4.758 -22.781  20.539  1.00  0.00           C  \\nATOM     43  CG1 ILE A   3      -3.415 -23.455  20.278  1.00  0.00           C  \\nATOM     44  CG2 ILE A   3      -5.389 -22.400  19.143  1.00  0.00           C  \\nATOM     45  CD1 ILE A   3      -3.509 -24.784  19.530  1.00  0.00           C  \\nATOM     46  H   ILE A   3      -4.382 -22.397  23.406  1.00  0.00           H  \\nATOM     47  HA  ILE A   3      -4.019 -20.804  20.892  1.00  0.00           H  \\nATOM     48  HB  ILE A   3      -5.403 -23.513  21.025  1.00  0.00           H  \\nATOM     49 1HG1 ILE A   3      -2.780 -22.787  19.697  1.00  0.00           H  \\nATOM     50 2HG1 ILE A   3      -2.911 -23.641  21.227  1.00  0.00           H  \\nATOM     51 1HG2 ILE A   3      -5.476 -23.295  18.527  1.00  0.00           H  \\nATOM     52 2HG2 ILE A   3      -6.377 -21.968  19.296  1.00  0.00           H  \\nATOM     53 3HG2 ILE A   3      -4.749 -21.674  18.641  1.00  0.00           H  \\nATOM     54 1HD1 ILE A   3      -2.508 -25.193  19.388  1.00  0.00           H  \\nATOM     55 2HD1 ILE A   3      -4.109 -25.486  20.110  1.00  0.00           H  \\nATOM     56 3HD1 ILE A   3      -3.975 -24.623  18.559  1.00  0.00           H  \\nATOM     57  N   ILE A   4      -6.349 -19.869  21.127  1.00  0.00           N  \\nATOM     58  CA  ILE A   4      -7.675 -19.272  21.346  1.00  0.00           C  \\nATOM     59  C   ILE A   4      -8.359 -19.317  19.941  1.00  0.00           C  \\nATOM     60  O   ILE A   4      -7.746 -18.875  18.961  1.00  0.00           O  \\nATOM     61  CB  ILE A   4      -7.562 -17.794  21.756  1.00  0.00           C  \\nATOM     62  CG1 ILE A   4      -6.999 -17.680  23.152  1.00  0.00           C  \\nATOM     63  CG2 ILE A   4      -8.940 -17.205  21.920  1.00  0.00           C  \\nATOM     64  CD1 ILE A   4      -5.617 -17.266  23.145  1.00  0.00           C  \\nATOM     65  H   ILE A   4      -5.716 -19.434  20.472  1.00  0.00           H  \\nATOM     66  HA  ILE A   4      -8.169 -19.813  22.153  1.00  0.00           H  \\nATOM     67  HB  ILE A   4      -7.024 -17.243  20.986  1.00  0.00           H  \\nATOM     68 1HG1 ILE A   4      -7.586 -16.962  23.723  1.00  0.00           H  \\nATOM     69 2HG1 ILE A   4      -7.081 -18.643  23.657  1.00  0.00           H  \\nATOM     70 1HG2 ILE A   4      -8.856 -16.158  22.211  1.00  0.00           H  \\nATOM     71 2HG2 ILE A   4      -9.481 -17.277  20.977  1.00  0.00           H  \\nATOM     72 3HG2 ILE A   4      -9.481 -17.754  22.691  1.00  0.00           H  \\nATOM     73 1HD1 ILE A   4      -5.252 -17.197  24.170  1.00  0.00           H  \\nATOM     74 2HD1 ILE A   4      -5.021 -17.996  22.596  1.00  0.00           H  \\nATOM     75 3HD1 ILE A   4      -5.532 -16.292  22.665  1.00  0.00           H  \\nATOM     76  N   ASN A   5      -9.578 -19.841  19.843  1.00  0.00           N  \\nATOM     77  CA  ASN A   5     -10.230 -20.006  18.490  1.00  0.00           C  \\nATOM     78  C   ASN A   5     -11.334 -18.982  18.265  1.00  0.00           C  \\nATOM     79  O   ASN A   5     -11.959 -18.460  19.171  1.00  0.00           O  \\nATOM     80  CB  ASN A   5     -10.900 -21.371  18.462  1.00  0.00           C  \\nATOM     81  CG  ASN A   5      -9.992 -22.460  18.877  1.00  0.00           C  \\nATOM     82  OD1 ASN A   5      -9.015 -22.775  18.234  1.00  0.00           O  \\nATOM     83  ND2 ASN A   5     -10.316 -23.049  19.990  1.00  0.00           N  \\nATOM     84  H   ASN A   5     -10.079 -20.133  20.670  1.00  0.00           H  \\nATOM     85  HA  ASN A   5      -9.455 -19.956  17.724  1.00  0.00           H  \\nATOM     86 1HB  ASN A   5     -11.767 -21.365  19.124  1.00  0.00           H  \\nATOM     87 2HB  ASN A   5     -11.260 -21.579  17.454  1.00  0.00           H  \\nATOM     88 1HD2 ASN A   5      -9.753 -23.798  20.342  1.00  0.00           H  \\nATOM     89 2HD2 ASN A   5     -11.127 -22.754  20.494  1.00  0.00           H  \\nATOM     90  N   PHE A   6     -11.436 -18.562  17.004  1.00  0.00           N  \\nATOM     91  CA  PHE A   6     -12.589 -17.732  16.544  1.00  0.00           C  \\nATOM     92  C   PHE A   6     -12.942 -18.177  15.156  1.00  0.00           C  \\nATOM     93  O   PHE A   6     -12.045 -18.462  14.341  1.00  0.00           O  \\nATOM     94  CB  PHE A   6     -12.252 -16.236  16.594  1.00  0.00           C  \\nATOM     95  CG  PHE A   6     -13.388 -15.352  16.144  1.00  0.00           C  \\nATOM     96  CD1 PHE A   6     -14.518 -15.221  16.917  1.00  0.00           C  \\nATOM     97  CD2 PHE A   6     -13.275 -14.653  14.922  1.00  0.00           C  \\nATOM     98  CE1 PHE A   6     -15.538 -14.385  16.540  1.00  0.00           C  \\nATOM     99  CE2 PHE A   6     -14.280 -13.870  14.476  1.00  0.00           C  \\nATOM    100  CZ  PHE A   6     -15.465 -13.747  15.318  1.00  0.00           C  \\nATOM    101  H   PHE A   6     -10.715 -18.812  16.343  1.00  0.00           H  \\nATOM    102  HA  PHE A   6     -13.434 -17.915  17.209  1.00  0.00           H  \\nATOM    103 1HB  PHE A   6     -11.982 -15.958  17.612  1.00  0.00           H  \\nATOM    104 2HB  PHE A   6     -11.389 -16.036  15.960  1.00  0.00           H  \\nATOM    105  HD1 PHE A   6     -14.599 -15.792  17.843  1.00  0.00           H  \\nATOM    106  HD2 PHE A   6     -12.371 -14.773  14.324  1.00  0.00           H  \\nATOM    107  HE1 PHE A   6     -16.390 -14.232  17.202  1.00  0.00           H  \\nATOM    108  HE2 PHE A   6     -14.231 -13.340  13.525  1.00  0.00           H  \\nATOM    109  HZ  PHE A   6     -16.300 -13.142  14.968  1.00  0.00           H  \\nATOM    110  N   THR A   7     -14.232 -18.219  14.889  1.00  0.00           N  \\nATOM    111  CA  THR A   7     -14.636 -18.414  13.513  1.00  0.00           C  \\nATOM    112  C   THR A   7     -15.552 -17.294  13.039  1.00  0.00           C  \\nATOM    113  O   THR A   7     -16.350 -16.796  13.826  1.00  0.00           O  \\nATOM    114  CB  THR A   7     -15.330 -19.740  13.366  1.00  0.00           C  \\nATOM    115  OG1 THR A   7     -15.714 -19.850  11.981  1.00  0.00           O  \\nATOM    116  CG2 THR A   7     -16.601 -19.869  14.179  1.00  0.00           C  \\nATOM    117  H   THR A   7     -14.942 -18.121  15.601  1.00  0.00           H  \\nATOM    118  HA  THR A   7     -13.744 -18.411  12.887  1.00  0.00           H  \\nATOM    119  HB  THR A   7     -14.660 -20.539  13.684  1.00  0.00           H  \\nATOM    120  HG1 THR A   7     -15.435 -19.062  11.509  1.00  0.00           H  \\nATOM    121 1HG2 THR A   7     -17.038 -20.854  14.017  1.00  0.00           H  \\nATOM    122 2HG2 THR A   7     -16.370 -19.744  15.237  1.00  0.00           H  \\nATOM    123 3HG2 THR A   7     -17.310 -19.102  13.869  1.00  0.00           H  \\nATOM    124  N   THR A   8     -15.354 -16.860  11.813  1.00  0.00           N  \\nATOM    125  CA  THR A   8     -16.285 -15.908  11.219  1.00  0.00           C  \\nATOM    126  C   THR A   8     -17.638 -16.570  10.913  1.00  0.00           C  \\nATOM    127  O   THR A   8     -18.639 -15.854  10.743  1.00  0.00           O  \\nATOM    128  CB  THR A   8     -15.727 -15.247   9.995  1.00  0.00           C  \\nATOM    129  OG1 THR A   8     -15.345 -16.295   9.038  1.00  0.00           O  \\nATOM    130  CG2 THR A   8     -14.596 -14.458  10.361  1.00  0.00           C  \\nATOM    131  H   THR A   8     -14.562 -17.177  11.272  1.00  0.00           H  \\nATOM    132  HA  THR A   8     -16.494 -15.127  11.950  1.00  0.00           H  \\nATOM    133  HB  THR A   8     -16.490 -14.612   9.544  1.00  0.00           H  \\nATOM    134  HG1 THR A   8     -15.539 -17.157   9.414  1.00  0.00           H  \\nATOM    135 1HG2 THR A   8     -14.187 -13.975   9.474  1.00  0.00           H  \\nATOM    136 2HG2 THR A   8     -14.900 -13.698  11.081  1.00  0.00           H  \\nATOM    137 3HG2 THR A   8     -13.836 -15.097  10.808  1.00  0.00           H  \\nATOM    138  N   ALA A   9     -17.649 -17.879  10.773  1.00  0.00           N  \\nATOM    139  CA  ALA A   9     -18.906 -18.560  10.352  1.00  0.00           C  \\nATOM    140  C   ALA A   9     -19.972 -18.483  11.496  1.00  0.00           C  \\nATOM    141  O   ALA A   9     -19.783 -19.110  12.576  1.00  0.00           O  \\nATOM    142  CB  ALA A   9     -18.580 -19.942  10.010  1.00  0.00           C  \\nATOM    143  H   ALA A   9     -16.823 -18.434  10.945  1.00  0.00           H  \\nATOM    144  HA  ALA A   9     -19.295 -18.041   9.476  1.00  0.00           H  \\nATOM    145 1HB  ALA A   9     -19.485 -20.463   9.698  1.00  0.00           H  \\nATOM    146 2HB  ALA A   9     -17.855 -19.954   9.196  1.00  0.00           H  \\nATOM    147 3HB  ALA A   9     -18.156 -20.441  10.880  1.00  0.00           H  \\nATOM    148  N   GLY A  10     -21.032 -17.709  11.266  1.00  0.00           N  \\nATOM    149  CA  GLY A  10     -22.101 -17.563  12.289  1.00  0.00           C  \\nATOM    150  C   GLY A  10     -21.632 -16.647  13.433  1.00  0.00           C  \\nATOM    151  O   GLY A  10     -22.203 -16.680  14.508  1.00  0.00           O  \\nATOM    152  H   GLY A  10     -21.118 -17.213  10.390  1.00  0.00           H  \\nATOM    153 1HA  GLY A  10     -22.996 -17.151  11.823  1.00  0.00           H  \\nATOM    154 2HA  GLY A  10     -22.366 -18.544  12.681  1.00  0.00           H  \\nATOM    155  N   ALA A  11     -20.593 -15.828  13.182  1.00  0.00           N  \\nATOM    156  CA  ALA A  11     -20.126 -14.877  14.223  1.00  0.00           C  \\nATOM    157  C   ALA A  11     -21.195 -13.939  14.683  1.00  0.00           C  \\nATOM    158  O   ALA A  11     -21.954 -13.446  13.885  1.00  0.00           O  \\nATOM    159  CB  ALA A  11     -18.987 -14.063  13.662  1.00  0.00           C  \\nATOM    160  H   ALA A  11     -20.122 -15.850  12.289  1.00  0.00           H  \\nATOM    161  HA  ALA A  11     -19.776 -15.456  15.078  1.00  0.00           H  \\nATOM    162 1HB  ALA A  11     -18.635 -13.360  14.418  1.00  0.00           H  \\nATOM    163 2HB  ALA A  11     -18.171 -14.727  13.376  1.00  0.00           H  \\nATOM    164 3HB  ALA A  11     -19.330 -13.513  12.787  1.00  0.00           H  \\nATOM    165  N   THR A  12     -21.263 -13.650  16.014  1.00  0.00           N  \\nATOM    166  CA  THR A  12     -22.151 -12.682  16.502  1.00  0.00           C  \\nATOM    167  C   THR A  12     -21.387 -11.647  17.314  1.00  0.00           C  \\nATOM    168  O   THR A  12     -20.227 -11.823  17.569  1.00  0.00           O  \\nATOM    169  CB  THR A  12     -23.179 -13.386  17.387  1.00  0.00           C  \\nATOM    170  OG1 THR A  12     -22.488 -14.018  18.416  1.00  0.00           O  \\nATOM    171  CG2 THR A  12     -23.881 -14.502  16.638  1.00  0.00           C  \\nATOM    172  H   THR A  12     -20.664 -14.140  16.663  1.00  0.00           H  \\nATOM    173  HA  THR A  12     -22.652 -12.214  15.655  1.00  0.00           H  \\nATOM    174  HB  THR A  12     -23.925 -12.666  17.722  1.00  0.00           H  \\nATOM    175  HG1 THR A  12     -21.547 -13.856  18.315  1.00  0.00           H  \\nATOM    176 1HG2 THR A  12     -24.606 -14.982  17.296  1.00  0.00           H  \\nATOM    177 2HG2 THR A  12     -24.395 -14.090  15.770  1.00  0.00           H  \\nATOM    178 3HG2 THR A  12     -23.147 -15.237  16.310  1.00  0.00           H  \\nATOM    179  N   VAL A  13     -22.036 -10.578  17.769  1.00  0.00           N  \\nATOM    180  CA  VAL A  13     -21.477  -9.633  18.681  1.00  0.00           C  \\nATOM    181  C   VAL A  13     -20.964 -10.386  19.922  1.00  0.00           C  \\nATOM    182  O   VAL A  13     -19.808 -10.112  20.355  1.00  0.00           O  \\nATOM    183  CB  VAL A  13     -22.541  -8.559  19.108  1.00  0.00           C  \\nATOM    184  CG1 VAL A  13     -22.070  -7.634  20.146  1.00  0.00           C  \\nATOM    185  CG2 VAL A  13     -22.894  -7.739  17.903  1.00  0.00           C  \\nATOM    186  H   VAL A  13     -22.981 -10.443  17.438  1.00  0.00           H  \\nATOM    187  HA  VAL A  13     -20.652  -9.120  18.185  1.00  0.00           H  \\nATOM    188  HB  VAL A  13     -23.425  -9.069  19.490  1.00  0.00           H  \\nATOM    189 1HG1 VAL A  13     -22.861  -6.924  20.387  1.00  0.00           H  \\nATOM    190 2HG1 VAL A  13     -21.804  -8.196  21.041  1.00  0.00           H  \\nATOM    191 3HG1 VAL A  13     -21.196  -7.093  19.782  1.00  0.00           H  \\nATOM    192 1HG2 VAL A  13     -23.633  -6.987  18.178  1.00  0.00           H  \\nATOM    193 2HG2 VAL A  13     -21.999  -7.246  17.523  1.00  0.00           H  \\nATOM    194 3HG2 VAL A  13     -23.307  -8.387  17.130  1.00  0.00           H  \\nATOM    195  N   GLN A  14     -21.743 -11.346  20.460  1.00  0.00           N  \\nATOM    196  CA  GLN A  14     -21.296 -12.053  21.685  1.00  0.00           C  \\nATOM    197  C   GLN A  14     -20.074 -12.922  21.389  1.00  0.00           C  \\nATOM    198  O   GLN A  14     -19.163 -12.933  22.251  1.00  0.00           O  \\nATOM    199  CB  GLN A  14     -22.397 -12.923  22.239  1.00  0.00           C  \\nATOM    200  CG  GLN A  14     -22.039 -13.400  23.661  1.00  0.00           C  \\nATOM    201  CD  GLN A  14     -22.068 -12.267  24.733  1.00  0.00           C  \\nATOM    202  OE1 GLN A  14     -23.111 -11.631  24.948  1.00  0.00           O  \\nATOM    203  NE2 GLN A  14     -20.917 -12.006  25.392  1.00  0.00           N  \\nATOM    204  H   GLN A  14     -22.629 -11.595  20.044  1.00  0.00           H  \\nATOM    205  HA  GLN A  14     -21.034 -11.309  22.437  1.00  0.00           H  \\nATOM    206 1HB  GLN A  14     -23.330 -12.360  22.259  1.00  0.00           H  \\nATOM    207 2HB  GLN A  14     -22.546 -13.782  21.585  1.00  0.00           H  \\nATOM    208 1HG  GLN A  14     -22.755 -14.162  23.969  1.00  0.00           H  \\nATOM    209 2HG  GLN A  14     -21.032 -13.817  23.651  1.00  0.00           H  \\nATOM    210 1HE2 GLN A  14     -20.890 -11.287  26.088  1.00  0.00           H  \\nATOM    211 2HE2 GLN A  14     -20.091 -12.531  25.185  1.00  0.00           H  \\nATOM    212  N   SER A  15     -20.026 -13.591  20.258  1.00  0.00           N  \\nATOM    213  CA  SER A  15     -18.919 -14.497  20.063  1.00  0.00           C  \\nATOM    214  C   SER A  15     -17.629 -13.721  19.783  1.00  0.00           C  \\nATOM    215  O   SER A  15     -16.578 -14.194  20.203  1.00  0.00           O  \\nATOM    216  CB  SER A  15     -19.204 -15.517  18.992  1.00  0.00           C  \\nATOM    217  OG  SER A  15     -19.099 -14.953  17.657  1.00  0.00           O  \\nATOM    218  H   SER A  15     -20.721 -13.499  19.531  1.00  0.00           H  \\nATOM    219  HA  SER A  15     -18.742 -15.033  20.997  1.00  0.00           H  \\nATOM    220 1HB  SER A  15     -18.503 -16.346  19.084  1.00  0.00           H  \\nATOM    221 2HB  SER A  15     -20.207 -15.919  19.130  1.00  0.00           H  \\nATOM    222  HG  SER A  15     -18.864 -14.030  17.777  1.00  0.00           H  \\nATOM    223  N   TYR A  16     -17.739 -12.584  19.121  1.00  0.00           N  \\nATOM    224  CA  TYR A  16     -16.534 -11.714  18.965  1.00  0.00           C  \\nATOM    225  C   TYR A  16     -16.088 -11.136  20.268  1.00  0.00           C  \\nATOM    226  O   TYR A  16     -14.896 -11.157  20.558  1.00  0.00           O  \\nATOM    227  CB  TYR A  16     -16.831 -10.635  17.919  1.00  0.00           C  \\nATOM    228  CG  TYR A  16     -15.648  -9.723  17.676  1.00  0.00           C  \\nATOM    229  CD1 TYR A  16     -14.517 -10.150  16.930  1.00  0.00           C  \\nATOM    230  CD2 TYR A  16     -15.635  -8.473  18.256  1.00  0.00           C  \\nATOM    231  CE1 TYR A  16     -13.436  -9.303  16.797  1.00  0.00           C  \\nATOM    232  CE2 TYR A  16     -14.544  -7.676  18.103  1.00  0.00           C  \\nATOM    233  CZ  TYR A  16     -13.476  -8.099  17.357  1.00  0.00           C  \\nATOM    234  OH  TYR A  16     -12.397  -7.254  17.260  1.00  0.00           O  \\nATOM    235  H   TYR A  16     -18.616 -12.287  18.717  1.00  0.00           H  \\nATOM    236  HA  TYR A  16     -15.704 -12.332  18.622  1.00  0.00           H  \\nATOM    237 1HB  TYR A  16     -17.111 -11.108  16.977  1.00  0.00           H  \\nATOM    238 2HB  TYR A  16     -17.678 -10.034  18.247  1.00  0.00           H  \\nATOM    239  HD1 TYR A  16     -14.509 -11.138  16.470  1.00  0.00           H  \\nATOM    240  HD2 TYR A  16     -16.494  -8.131  18.833  1.00  0.00           H  \\nATOM    241  HE1 TYR A  16     -12.563  -9.619  16.227  1.00  0.00           H  \\nATOM    242  HE2 TYR A  16     -14.515  -6.694  18.576  1.00  0.00           H  \\nATOM    243  HH  TYR A  16     -12.573  -6.453  17.760  1.00  0.00           H  \\nATOM    244  N   THR A  17     -17.038 -10.599  21.005  1.00  0.00           N  \\nATOM    245  CA  THR A  17     -16.763 -10.039  22.324  1.00  0.00           C  \\nATOM    246  C   THR A  17     -16.084 -11.119  23.186  1.00  0.00           C  \\nATOM    247  O   THR A  17     -15.037 -10.837  23.803  1.00  0.00           O  \\nATOM    248  CB  THR A  17     -18.008  -9.613  22.989  1.00  0.00           C  \\nATOM    249  OG1 THR A  17     -18.584  -8.542  22.186  1.00  0.00           O  \\nATOM    250  CG2 THR A  17     -17.629  -9.092  24.382  1.00  0.00           C  \\nATOM    251  H   THR A  17     -17.982 -10.574  20.647  1.00  0.00           H  \\nATOM    252  HA  THR A  17     -16.122  -9.165  22.202  1.00  0.00           H  \\nATOM    253  HB  THR A  17     -18.688 -10.461  23.066  1.00  0.00           H  \\nATOM    254  HG1 THR A  17     -18.021  -8.374  21.427  1.00  0.00           H  \\nATOM    255 1HG2 THR A  17     -18.528  -8.767  24.906  1.00  0.00           H  \\nATOM    256 2HG2 THR A  17     -17.146  -9.888  24.949  1.00  0.00           H  \\nATOM    257 3HG2 THR A  17     -16.944  -8.251  24.282  1.00  0.00           H  \\nATOM    258  N   ASN A  18     -16.619 -12.332  23.240  1.00  0.00           N  \\nATOM    259  CA  ASN A  18     -15.972 -13.311  24.060  1.00  0.00           C  \\nATOM    260  C   ASN A  18     -14.614 -13.709  23.585  1.00  0.00           C  \\nATOM    261  O   ASN A  18     -13.721 -13.920  24.398  1.00  0.00           O  \\nATOM    262  CB  ASN A  18     -16.859 -14.569  24.163  1.00  0.00           C  \\nATOM    263  CG  ASN A  18     -18.141 -14.359  25.007  1.00  0.00           C  \\nATOM    264  OD1 ASN A  18     -18.294 -13.400  25.732  1.00  0.00           O  \\nATOM    265  ND2 ASN A  18     -19.090 -15.281  24.819  1.00  0.00           N  \\nATOM    266  H   ASN A  18     -17.453 -12.589  22.732  1.00  0.00           H  \\nATOM    267  HA  ASN A  18     -15.833 -12.890  25.057  1.00  0.00           H  \\nATOM    268 1HB  ASN A  18     -17.157 -14.887  23.163  1.00  0.00           H  \\nATOM    269 2HB  ASN A  18     -16.286 -15.382  24.609  1.00  0.00           H  \\nATOM    270 1HD2 ASN A  18     -19.952 -15.222  25.324  1.00  0.00           H  \\nATOM    271 2HD2 ASN A  18     -18.939 -16.029  24.174  1.00  0.00           H  \\nATOM    272  N   PHE A  19     -14.381 -13.733  22.283  1.00  0.00           N  \\nATOM    273  CA  PHE A  19     -13.087 -14.004  21.728  1.00  0.00           C  \\nATOM    274  C   PHE A  19     -12.096 -12.962  22.184  1.00  0.00           C  \\nATOM    275  O   PHE A  19     -11.024 -13.337  22.625  1.00  0.00           O  \\nATOM    276  CB  PHE A  19     -13.235 -14.054  20.216  1.00  0.00           C  \\nATOM    277  CG  PHE A  19     -11.950 -14.018  19.481  1.00  0.00           C  \\nATOM    278  CD1 PHE A  19     -10.985 -14.970  19.737  1.00  0.00           C  \\nATOM    279  CD2 PHE A  19     -11.722 -13.018  18.519  1.00  0.00           C  \\nATOM    280  CE1 PHE A  19      -9.788 -14.947  18.981  1.00  0.00           C  \\nATOM    281  CE2 PHE A  19     -10.470 -12.968  17.799  1.00  0.00           C  \\nATOM    282  CZ  PHE A  19      -9.521 -13.954  18.048  1.00  0.00           C  \\nATOM    283  H   PHE A  19     -15.155 -13.551  21.660  1.00  0.00           H  \\nATOM    284  HA  PHE A  19     -12.746 -14.970  22.101  1.00  0.00           H  \\nATOM    285 1HB  PHE A  19     -13.760 -14.965  19.932  1.00  0.00           H  \\nATOM    286 2HB  PHE A  19     -13.838 -13.211  19.882  1.00  0.00           H  \\nATOM    287  HD1 PHE A  19     -11.165 -15.716  20.512  1.00  0.00           H  \\nATOM    288  HD2 PHE A  19     -12.483 -12.257  18.343  1.00  0.00           H  \\nATOM    289  HE1 PHE A  19      -9.040 -15.727  19.122  1.00  0.00           H  \\nATOM    290  HE2 PHE A  19     -10.260 -12.181  17.075  1.00  0.00           H  \\nATOM    291  HZ  PHE A  19      -8.571 -13.947  17.515  1.00  0.00           H  \\nATOM    292  N   ILE A  20     -12.408 -11.687  22.009  1.00  0.00           N  \\nATOM    293  CA  ILE A  20     -11.452 -10.624  22.391  1.00  0.00           C  \\nATOM    294  C   ILE A  20     -11.164 -10.692  23.856  1.00  0.00           C  \\nATOM    295  O   ILE A  20     -10.003 -10.512  24.251  1.00  0.00           O  \\nATOM    296  CB  ILE A  20     -11.941  -9.274  21.893  1.00  0.00           C  \\nATOM    297  CG1 ILE A  20     -11.933  -9.171  20.356  1.00  0.00           C  \\nATOM    298  CG2 ILE A  20     -11.096  -8.144  22.422  1.00  0.00           C  \\nATOM    299  CD1 ILE A  20     -10.654  -9.540  19.591  1.00  0.00           C  \\nATOM    300  H   ILE A  20     -13.300 -11.427  21.614  1.00  0.00           H  \\nATOM    301  HA  ILE A  20     -10.490 -10.841  21.928  1.00  0.00           H  \\nATOM    302  HB  ILE A  20     -12.968  -9.116  22.221  1.00  0.00           H  \\nATOM    303 1HG1 ILE A  20     -12.712  -9.812  19.945  1.00  0.00           H  \\nATOM    304 2HG1 ILE A  20     -12.162  -8.147  20.059  1.00  0.00           H  \\nATOM    305 1HG2 ILE A  20     -11.477  -7.196  22.043  1.00  0.00           H  \\nATOM    306 2HG2 ILE A  20     -11.134  -8.141  23.510  1.00  0.00           H  \\nATOM    307 3HG2 ILE A  20     -10.065  -8.277  22.095  1.00  0.00           H  \\nATOM    308 1HD1 ILE A  20     -10.820  -9.411  18.521  1.00  0.00           H  \\nATOM    309 2HD1 ILE A  20      -9.838  -8.892  19.912  1.00  0.00           H  \\nATOM    310 3HD1 ILE A  20     -10.395 -10.578  19.795  1.00  0.00           H  \\nATOM    311  N   ARG A  21     -12.198 -10.967  24.668  1.00  0.00           N  \\nATOM    312  CA  ARG A  21     -11.945 -11.081  26.117  1.00  0.00           C  \\nATOM    313  C   ARG A  21     -11.065 -12.264  26.390  1.00  0.00           C  \\nATOM    314  O   ARG A  21     -10.189 -12.148  27.305  1.00  0.00           O  \\nATOM    315  CB  ARG A  21     -13.271 -11.202  26.870  1.00  0.00           C  \\nATOM    316  CG  ARG A  21     -14.037  -9.904  26.944  1.00  0.00           C  \\nATOM    317  CD  ARG A  21     -15.403 -10.215  27.484  1.00  0.00           C  \\nATOM    318  NE  ARG A  21     -16.189  -8.990  27.583  1.00  0.00           N  \\nATOM    319  CZ  ARG A  21     -17.478  -8.973  27.940  1.00  0.00           C  \\nATOM    320  NH1 ARG A  21     -18.092 -10.123  28.223  1.00  0.00           N  \\nATOM    321  NH2 ARG A  21     -18.163  -7.811  27.964  1.00  0.00           N  \\nATOM    322  H   ARG A  21     -13.137 -11.097  24.321  1.00  0.00           H  \\nATOM    323  HA  ARG A  21     -11.430 -10.179  26.450  1.00  0.00           H  \\nATOM    324 1HB  ARG A  21     -13.900 -11.945  26.382  1.00  0.00           H  \\nATOM    325 2HB  ARG A  21     -13.082 -11.549  27.886  1.00  0.00           H  \\nATOM    326 1HG  ARG A  21     -13.516  -9.211  27.605  1.00  0.00           H  \\nATOM    327 2HG  ARG A  21     -14.112  -9.468  25.947  1.00  0.00           H  \\nATOM    328 1HD  ARG A  21     -15.908 -10.912  26.816  1.00  0.00           H  \\nATOM    329 2HD  ARG A  21     -15.310 -10.663  28.473  1.00  0.00           H  \\nATOM    330  HE  ARG A  21     -15.727  -8.116  27.369  1.00  0.00           H  \\nATOM    331 1HH1 ARG A  21     -17.584 -10.994  28.167  1.00  0.00           H  \\nATOM    332 2HH1 ARG A  21     -19.065 -10.123  28.494  1.00  0.00           H  \\nATOM    333 1HH2 ARG A  21     -17.702  -6.947  27.711  1.00  0.00           H  \\nATOM    334 2HH2 ARG A  21     -19.135  -7.804  28.234  1.00  0.00           H  \\nATOM    335  N   ALA A  22     -11.178 -13.383  25.690  1.00  0.00           N  \\nATOM    336  CA  ALA A  22     -10.257 -14.535  25.907  1.00  0.00           C  \\nATOM    337  C   ALA A  22      -8.845 -14.190  25.525  1.00  0.00           C  \\nATOM    338  O   ALA A  22      -7.926 -14.601  26.216  1.00  0.00           O  \\nATOM    339  CB  ALA A  22     -10.704 -15.721  25.088  1.00  0.00           C  \\nATOM    340  H   ALA A  22     -11.905 -13.459  24.993  1.00  0.00           H  \\nATOM    341  HA  ALA A  22     -10.290 -14.801  26.964  1.00  0.00           H  \\nATOM    342 1HB  ALA A  22     -10.023 -16.556  25.255  1.00  0.00           H  \\nATOM    343 2HB  ALA A  22     -11.712 -16.010  25.386  1.00  0.00           H  \\nATOM    344 3HB  ALA A  22     -10.700 -15.456  24.032  1.00  0.00           H  \\nATOM    345  N   VAL A  23      -8.661 -13.503  24.401  1.00  0.00           N  \\nATOM    346  CA  VAL A  23      -7.301 -13.089  24.006  1.00  0.00           C  \\nATOM    347  C   VAL A  23      -6.691 -12.232  25.110  1.00  0.00           C  \\nATOM    348  O   VAL A  23      -5.563 -12.476  25.456  1.00  0.00           O  \\nATOM    349  CB  VAL A  23      -7.332 -12.354  22.666  1.00  0.00           C  \\nATOM    350  CG1 VAL A  23      -5.942 -11.762  22.368  1.00  0.00           C  \\nATOM    351  CG2 VAL A  23      -7.842 -13.297  21.566  1.00  0.00           C  \\nATOM    352  H   VAL A  23      -9.441 -13.258  23.809  1.00  0.00           H  \\nATOM    353  HA  VAL A  23      -6.685 -13.982  23.899  1.00  0.00           H  \\nATOM    354  HB  VAL A  23      -7.999 -11.495  22.748  1.00  0.00           H  \\nATOM    355 1HG1 VAL A  23      -5.967 -11.238  21.412  1.00  0.00           H  \\nATOM    356 2HG1 VAL A  23      -5.668 -11.063  23.157  1.00  0.00           H  \\nATOM    357 3HG1 VAL A  23      -5.207 -12.565  22.322  1.00  0.00           H  \\nATOM    358 1HG2 VAL A  23      -7.863 -12.768  20.614  1.00  0.00           H  \\nATOM    359 2HG2 VAL A  23      -7.178 -14.158  21.489  1.00  0.00           H  \\nATOM    360 3HG2 VAL A  23      -8.848 -13.635  21.814  1.00  0.00           H  \\nATOM    361  N   ARG A  24      -7.388 -11.180  25.548  1.00  0.00           N  \\nATOM    362  CA  ARG A  24      -6.858 -10.350  26.603  1.00  0.00           C  \\nATOM    363  C   ARG A  24      -6.524 -11.148  27.817  1.00  0.00           C  \\nATOM    364  O   ARG A  24      -5.444 -10.884  28.462  1.00  0.00           O  \\nATOM    365  CB  ARG A  24      -7.836  -9.319  27.061  1.00  0.00           C  \\nATOM    366  CG  ARG A  24      -8.095  -8.291  26.031  1.00  0.00           C  \\nATOM    367  CD  ARG A  24      -9.104  -7.256  26.583  1.00  0.00           C  \\nATOM    368  NE  ARG A  24      -9.407  -6.297  25.538  1.00  0.00           N  \\nATOM    369  CZ  ARG A  24     -10.624  -5.840  25.256  1.00  0.00           C  \\nATOM    370  NH1 ARG A  24     -11.667  -6.178  26.007  1.00  0.00           N  \\nATOM    371  NH2 ARG A  24     -10.842  -5.048  24.253  1.00  0.00           N  \\nATOM    372  H   ARG A  24      -8.288 -10.954  25.149  1.00  0.00           H  \\nATOM    373  HA  ARG A  24      -5.979  -9.828  26.224  1.00  0.00           H  \\nATOM    374 1HB  ARG A  24      -8.777  -9.801  27.323  1.00  0.00           H  \\nATOM    375 2HB  ARG A  24      -7.457  -8.832  27.959  1.00  0.00           H  \\nATOM    376 1HG  ARG A  24      -7.162  -7.789  25.775  1.00  0.00           H  \\nATOM    377 2HG  ARG A  24      -8.509  -8.765  25.140  1.00  0.00           H  \\nATOM    378 1HD  ARG A  24     -10.016  -7.766  26.892  1.00  0.00           H  \\nATOM    379 2HD  ARG A  24      -8.667  -6.743  27.439  1.00  0.00           H  \\nATOM    380  HE  ARG A  24      -8.637  -5.949  24.982  1.00  0.00           H  \\nATOM    381 1HH1 ARG A  24     -11.540  -6.788  26.802  1.00  0.00           H  \\nATOM    382 2HH1 ARG A  24     -12.586  -5.825  25.783  1.00  0.00           H  \\nATOM    383 1HH2 ARG A  24     -10.075  -4.761  23.660  1.00  0.00           H  \\nATOM    384 2HH2 ARG A  24     -11.777  -4.718  24.065  1.00  0.00           H  \\nATOM    385  N   GLY A  25      -7.411 -12.038  28.218  1.00  0.00           N  \\nATOM    386  CA  GLY A  25      -7.206 -12.848  29.442  1.00  0.00           C  \\nATOM    387  C   GLY A  25      -6.051 -13.792  29.321  1.00  0.00           C  \\nATOM    388  O   GLY A  25      -5.410 -14.102  30.359  1.00  0.00           O  \\nATOM    389  H   GLY A  25      -8.253 -12.171  27.676  1.00  0.00           H  \\nATOM    390 1HA  GLY A  25      -7.035 -12.187  30.292  1.00  0.00           H  \\nATOM    391 2HA  GLY A  25      -8.109 -13.419  29.656  1.00  0.00           H  \\nATOM    392  N   ARG A  26      -5.678 -14.213  28.126  1.00  0.00           N  \\nATOM    393  CA  ARG A  26      -4.493 -15.053  27.963  1.00  0.00           C  \\nATOM    394  C   ARG A  26      -3.163 -14.286  27.716  1.00  0.00           C  \\nATOM    395  O   ARG A  26      -2.061 -14.852  27.818  1.00  0.00           O  \\nATOM    396  CB  ARG A  26      -4.719 -16.116  26.878  1.00  0.00           C  \\nATOM    397  CG  ARG A  26      -5.888 -17.024  27.280  1.00  0.00           C  \\nATOM    398  CD  ARG A  26      -5.504 -17.956  28.516  1.00  0.00           C  \\nATOM    399  NE  ARG A  26      -5.915 -19.264  28.122  1.00  0.00           N  \\nATOM    400  CZ  ARG A  26      -5.269 -20.380  28.281  1.00  0.00           C  \\nATOM    401  NH1 ARG A  26      -4.042 -20.438  28.835  1.00  0.00           N  \\nATOM    402  NH2 ARG A  26      -5.854 -21.453  27.797  1.00  0.00           N  \\nATOM    403  H   ARG A  26      -6.214 -13.955  27.310  1.00  0.00           H  \\nATOM    404  HA  ARG A  26      -4.296 -15.561  28.908  1.00  0.00           H  \\nATOM    405 1HB  ARG A  26      -4.929 -15.627  25.928  1.00  0.00           H  \\nATOM    406 2HB  ARG A  26      -3.810 -16.704  26.749  1.00  0.00           H  \\nATOM    407 1HG  ARG A  26      -6.744 -16.411  27.563  1.00  0.00           H  \\nATOM    408 2HG  ARG A  26      -6.160 -17.662  26.438  1.00  0.00           H  \\nATOM    409 1HD  ARG A  26      -4.430 -17.904  28.691  1.00  0.00           H  \\nATOM    410 2HD  ARG A  26      -6.033 -17.616  29.406  1.00  0.00           H  \\nATOM    411  HE  ARG A  26      -6.811 -19.357  27.663  1.00  0.00           H  \\nATOM    412 1HH1 ARG A  26      -3.588 -19.591  29.146  1.00  0.00           H  \\nATOM    413 2HH1 ARG A  26      -3.577 -21.328  28.939  1.00  0.00           H  \\nATOM    414 1HH2 ARG A  26      -6.750 -21.376  27.336  1.00  0.00           H  \\nATOM    415 2HH2 ARG A  26      -5.408 -22.354  27.885  1.00  0.00           H  \\nATOM    416  N   LEU A  27      -3.290 -13.025  27.332  1.00  0.00           N  \\nATOM    417  CA  LEU A  27      -2.121 -12.204  27.105  1.00  0.00           C  \\nATOM    418  C   LEU A  27      -1.569 -11.754  28.411  1.00  0.00           C  \\nATOM    419  O   LEU A  27      -0.337 -11.692  28.494  1.00  0.00           O  \\nATOM    420  CB  LEU A  27      -2.491 -10.960  26.286  1.00  0.00           C  \\nATOM    421  CG  LEU A  27      -2.639 -11.225  24.810  1.00  0.00           C  \\nATOM    422  CD1 LEU A  27      -3.255 -10.033  24.200  1.00  0.00           C  \\nATOM    423  CD2 LEU A  27      -1.309 -11.555  24.023  1.00  0.00           C  \\nATOM    424  H   LEU A  27      -4.207 -12.624  27.194  1.00  0.00           H  \\nATOM    425  HA  LEU A  27      -1.393 -12.788  26.543  1.00  0.00           H  \\nATOM    426 1HB  LEU A  27      -3.431 -10.562  26.665  1.00  0.00           H  \\nATOM    427 2HB  LEU A  27      -1.717 -10.206  26.430  1.00  0.00           H  \\nATOM    428  HG  LEU A  27      -3.298 -12.080  24.658  1.00  0.00           H  \\nATOM    429 1HD1 LEU A  27      -3.376 -10.194  23.129  1.00  0.00           H  \\nATOM    430 2HD1 LEU A  27      -4.231  -9.856  24.652  1.00  0.00           H  \\nATOM    431 3HD1 LEU A  27      -2.615  -9.167  24.367  1.00  0.00           H  \\nATOM    432 1HD2 LEU A  27      -1.541 -11.728  22.972  1.00  0.00           H  \\nATOM    433 2HD2 LEU A  27      -0.618 -10.716  24.109  1.00  0.00           H  \\nATOM    434 3HD2 LEU A  27      -0.849 -12.449  24.444  1.00  0.00           H  \\nATOM    435  N   THR A  28      -2.410 -11.488  29.424  1.00  0.00           N  \\nATOM    436  CA  THR A  28      -1.959 -10.866  30.703  1.00  0.00           C  \\nATOM    437  C   THR A  28      -2.170 -11.827  31.871  1.00  0.00           C  \\nATOM    438  O   THR A  28      -3.002 -12.664  31.799  1.00  0.00           O  \\nATOM    439  CB  THR A  28      -2.801  -9.589  31.103  1.00  0.00           C  \\nATOM    440  OG1 THR A  28      -4.212  -9.921  31.241  1.00  0.00           O  \\nATOM    441  CG2 THR A  28      -2.693  -8.480  30.102  1.00  0.00           C  \\nATOM    442  H   THR A  28      -3.386 -11.721  29.308  1.00  0.00           H  \\nATOM    443  HA  THR A  28      -0.922 -10.549  30.587  1.00  0.00           H  \\nATOM    444  HB  THR A  28      -2.452  -9.209  32.063  1.00  0.00           H  \\nATOM    445  HG1 THR A  28      -4.340 -10.855  31.059  1.00  0.00           H  \\nATOM    446 1HG2 THR A  28      -3.292  -7.632  30.433  1.00  0.00           H  \\nATOM    447 2HG2 THR A  28      -1.651  -8.174  30.009  1.00  0.00           H  \\nATOM    448 3HG2 THR A  28      -3.057  -8.826  29.135  1.00  0.00           H  \\nATOM    449  N   THR A  29      -1.423 -11.649  32.936  1.00  0.00           N  \\nATOM    450  CA  THR A  29      -1.629 -12.426  34.156  1.00  0.00           C  \\nATOM    451  C   THR A  29      -2.754 -11.794  35.026  1.00  0.00           C  \\nATOM    452  O   THR A  29      -3.320 -12.447  35.939  1.00  0.00           O  \\nATOM    453  CB  THR A  29      -0.320 -12.406  34.947  1.00  0.00           C  \\nATOM    454  OG1 THR A  29      -0.145 -11.095  35.517  1.00  0.00           O  \\nATOM    455  CG2 THR A  29       0.866 -12.700  33.981  1.00  0.00           C  \\nATOM    456  H   THR A  29      -0.687 -10.957  32.911  1.00  0.00           H  \\nATOM    457  HA  THR A  29      -1.879 -13.450  33.876  1.00  0.00           H  \\nATOM    458  HB  THR A  29      -0.356 -13.164  35.730  1.00  0.00           H  \\nATOM    459  HG1 THR A  29      -0.887 -10.538  35.270  1.00  0.00           H  \\nATOM    460 1HG2 THR A  29       1.802 -12.687  34.539  1.00  0.00           H  \\nATOM    461 2HG2 THR A  29       0.729 -13.680  33.524  1.00  0.00           H  \\nATOM    462 3HG2 THR A  29       0.898 -11.939  33.203  1.00  0.00           H  \\nATOM    463  N   GLY A  30      -3.068 -10.518  34.782  1.00  0.00           N  \\nATOM    464  CA  GLY A  30      -3.999  -9.833  35.660  1.00  0.00           C  \\nATOM    465  C   GLY A  30      -3.421  -9.410  37.016  1.00  0.00           C  \\nATOM    466  O   GLY A  30      -4.079  -8.735  37.797  1.00  0.00           O  \\nATOM    467  H   GLY A  30      -2.672 -10.018  33.999  1.00  0.00           H  \\nATOM    468 1HA  GLY A  30      -4.373  -8.936  35.165  1.00  0.00           H  \\nATOM    469 2HA  GLY A  30      -4.857 -10.476  35.852  1.00  0.00           H  \\nATOM    470  N   ALA A  31      -2.180  -9.793  37.310  1.00  0.00           N  \\nATOM    471  CA  ALA A  31      -1.513  -9.365  38.567  1.00  0.00           C  \\nATOM    472  C   ALA A  31      -1.029  -7.945  38.466  1.00  0.00           C  \\nATOM    473  O   ALA A  31      -0.942  -7.219  39.482  1.00  0.00           O  \\nATOM    474  CB  ALA A  31      -0.384 -10.263  38.849  1.00  0.00           C  \\nATOM    475  H   ALA A  31      -1.679 -10.388  36.665  1.00  0.00           H  \\nATOM    476  HA  ALA A  31      -2.240  -9.433  39.376  1.00  0.00           H  \\nATOM    477 1HB  ALA A  31       0.107  -9.951  39.771  1.00  0.00           H  \\nATOM    478 2HB  ALA A  31      -0.750 -11.284  38.960  1.00  0.00           H  \\nATOM    479 3HB  ALA A  31       0.328 -10.221  38.027  1.00  0.00           H  \\nATOM    480  N   ASP A  32      -0.750  -7.477  37.232  1.00  0.00           N  \\nATOM    481  CA  ASP A  32      -0.113  -6.171  37.067  1.00  0.00           C  \\nATOM    482  C   ASP A  32      -1.112  -5.187  36.375  1.00  0.00           C  \\nATOM    483  O   ASP A  32      -1.425  -5.260  35.203  1.00  0.00           O  \\nATOM    484  CB  ASP A  32       1.225  -6.304  36.330  1.00  0.00           C  \\nATOM    485  CG  ASP A  32       1.996  -4.932  36.148  1.00  0.00           C  \\nATOM    486  OD1 ASP A  32       1.575  -3.907  36.750  1.00  0.00           O  \\nATOM    487  OD2 ASP A  32       3.052  -4.915  35.527  1.00  0.00           O  \\nATOM    488  H   ASP A  32      -0.977  -8.021  36.412  1.00  0.00           H  \\nATOM    489  HA  ASP A  32       0.078  -5.752  38.055  1.00  0.00           H  \\nATOM    490 1HB  ASP A  32       1.875  -6.989  36.876  1.00  0.00           H  \\nATOM    491 2HB  ASP A  32       1.055  -6.732  35.342  1.00  0.00           H  \\nATOM    492  N   VAL A  33      -1.698  -4.303  37.174  1.00  0.00           N  \\nATOM    493  CA  VAL A  33      -2.754  -3.455  36.683  1.00  0.00           C  \\nATOM    494  C   VAL A  33      -2.469  -2.089  37.225  1.00  0.00           C  \\nATOM    495  O   VAL A  33      -2.253  -1.957  38.423  1.00  0.00           O  \\nATOM    496  CB  VAL A  33      -4.125  -3.922  37.171  1.00  0.00           C  \\nATOM    497  CG1 VAL A  33      -5.229  -3.021  36.776  1.00  0.00           C  \\nATOM    498  CG2 VAL A  33      -4.424  -5.332  36.812  1.00  0.00           C  \\nATOM    499  H   VAL A  33      -1.406  -4.218  38.137  1.00  0.00           H  \\nATOM    500  HA  VAL A  33      -2.751  -3.495  35.593  1.00  0.00           H  \\nATOM    501  HB  VAL A  33      -4.162  -3.838  38.257  1.00  0.00           H  \\nATOM    502 1HG1 VAL A  33      -6.173  -3.413  37.156  1.00  0.00           H  \\nATOM    503 2HG1 VAL A  33      -5.055  -2.029  37.192  1.00  0.00           H  \\nATOM    504 3HG1 VAL A  33      -5.275  -2.957  35.689  1.00  0.00           H  \\nATOM    505 1HG2 VAL A  33      -5.411  -5.602  37.188  1.00  0.00           H  \\nATOM    506 2HG2 VAL A  33      -4.405  -5.443  35.728  1.00  0.00           H  \\nATOM    507 3HG2 VAL A  33      -3.675  -5.987  37.257  1.00  0.00           H  \\nATOM    508  N   ARG A  34      -2.428  -1.097  36.346  1.00  0.00           N  \\nATOM    509  CA  ARG A  34      -2.114   0.264  36.755  1.00  0.00           C  \\nATOM    510  C   ARG A  34      -3.182   1.178  36.250  1.00  0.00           C  \\nATOM    511  O   ARG A  34      -3.500   1.199  35.067  1.00  0.00           O  \\nATOM    512  CB  ARG A  34      -0.763   0.634  36.210  1.00  0.00           C  \\nATOM    513  CG  ARG A  34       0.202  -0.373  36.734  1.00  0.00           C  \\nATOM    514  CD  ARG A  34       1.584  -0.040  36.546  1.00  0.00           C  \\nATOM    515  NE  ARG A  34       2.378  -1.221  36.316  1.00  0.00           N  \\nATOM    516  CZ  ARG A  34       3.634  -1.158  35.911  1.00  0.00           C  \\nATOM    517  NH1 ARG A  34       4.191   0.014  35.679  1.00  0.00           N  \\nATOM    518  NH2 ARG A  34       4.284  -2.299  35.671  1.00  0.00           N  \\nATOM    519  H   ARG A  34      -2.618  -1.287  35.372  1.00  0.00           H  \\nATOM    520  HA  ARG A  34      -2.089   0.301  37.845  1.00  0.00           H  \\nATOM    521 1HB  ARG A  34      -0.797   0.633  35.122  1.00  0.00           H  \\nATOM    522 2HB  ARG A  34      -0.506   1.644  36.528  1.00  0.00           H  \\nATOM    523 1HG  ARG A  34       0.053  -0.496  37.807  1.00  0.00           H  \\nATOM    524 2HG  ARG A  34       0.039  -1.328  36.233  1.00  0.00           H  \\nATOM    525 1HD  ARG A  34       1.685   0.620  35.685  1.00  0.00           H  \\nATOM    526 2HD  ARG A  34       1.961   0.464  37.435  1.00  0.00           H  \\nATOM    527  HE  ARG A  34       1.942  -2.119  36.475  1.00  0.00           H  \\nATOM    528 1HH1 ARG A  34       3.658   0.862  35.811  1.00  0.00           H  \\nATOM    529 2HH1 ARG A  34       5.151   0.064  35.369  1.00  0.00           H  \\nATOM    530 1HH2 ARG A  34       3.815  -3.185  35.800  1.00  0.00           H  \\nATOM    531 2HH2 ARG A  34       5.244  -2.277  35.361  1.00  0.00           H  \\nATOM    532  N   HIS A  35      -3.775   1.953  37.146  1.00  0.00           N  \\nATOM    533  CA  HIS A  35      -4.920   2.763  36.783  1.00  0.00           C  \\nATOM    534  C   HIS A  35      -6.021   1.985  36.190  1.00  0.00           C  \\nATOM    535  O   HIS A  35      -6.675   2.483  35.311  1.00  0.00           O  \\nATOM    536  CB  HIS A  35      -4.560   3.982  35.952  1.00  0.00           C  \\nATOM    537  CG  HIS A  35      -3.530   4.827  36.637  1.00  0.00           C  \\nATOM    538  ND1 HIS A  35      -3.883   5.682  37.674  1.00  0.00           N  \\nATOM    539  CD2 HIS A  35      -2.181   4.856  36.573  1.00  0.00           C  \\nATOM    540  CE1 HIS A  35      -2.790   6.281  38.126  1.00  0.00           C  \\nATOM    541  NE2 HIS A  35      -1.744   5.777  37.500  1.00  0.00           N  \\nATOM    542  H   HIS A  35      -3.431   1.985  38.095  1.00  0.00           H  \\nATOM    543  HA  HIS A  35      -5.405   3.133  37.686  1.00  0.00           H  \\nATOM    544 1HB  HIS A  35      -4.179   3.662  34.982  1.00  0.00           H  \\nATOM    545 2HB  HIS A  35      -5.455   4.577  35.772  1.00  0.00           H  \\nATOM    546  HD2 HIS A  35      -1.556   4.270  35.899  1.00  0.00           H  \\nATOM    547  HE1 HIS A  35      -2.759   7.060  38.888  1.00  0.00           H  \\nATOM    548  HE2 HIS A  35      -0.779   6.023  37.669  1.00  0.00           H  \\nATOM    549  N   GLU A  36      -6.271   0.796  36.750  1.00  0.00           N  \\nATOM    550  CA  GLU A  36      -7.321  -0.062  36.287  1.00  0.00           C  \\nATOM    551  C   GLU A  36      -7.055  -0.600  34.893  1.00  0.00           C  \\nATOM    552  O   GLU A  36      -7.948  -1.164  34.278  1.00  0.00           O  \\nATOM    553  CB  GLU A  36      -8.695   0.602  36.459  1.00  0.00           C  \\nATOM    554  CG  GLU A  36      -9.024   0.734  37.986  1.00  0.00           C  \\nATOM    555  CD  GLU A  36      -8.611  -0.517  38.798  1.00  0.00           C  \\nATOM    556  OE1 GLU A  36      -9.391  -1.510  38.879  1.00  0.00           O  \\nATOM    557  OE2 GLU A  36      -7.494  -0.515  39.379  1.00  0.00           O  \\nATOM    558  H   GLU A  36      -5.697   0.497  37.526  1.00  0.00           H  \\nATOM    559  HA  GLU A  36      -7.306  -0.978  36.879  1.00  0.00           H  \\nATOM    560 1HB  GLU A  36      -8.687   1.584  35.985  1.00  0.00           H  \\nATOM    561 2HB  GLU A  36      -9.454   0.003  35.957  1.00  0.00           H  \\nATOM    562 1HG  GLU A  36      -8.502   1.603  38.386  1.00  0.00           H  \\nATOM    563 2HG  GLU A  36     -10.094   0.902  38.103  1.00  0.00           H  \\nATOM    564  N   ILE A  37      -5.837  -0.371  34.394  1.00  0.00           N  \\nATOM    565  CA  ILE A  37      -5.439  -0.843  33.045  1.00  0.00           C  \\nATOM    566  C   ILE A  37      -4.342  -1.916  33.172  1.00  0.00           C  \\nATOM    567  O   ILE A  37      -3.241  -1.689  33.658  1.00  0.00           O  \\nATOM    568  CB  ILE A  37      -4.950   0.252  32.144  1.00  0.00           C  \\nATOM    569  CG1 ILE A  37      -5.976   1.396  31.994  1.00  0.00           C  \\nATOM    570  CG2 ILE A  37      -4.650  -0.357  30.693  1.00  0.00           C  \\nATOM    571  CD1 ILE A  37      -5.259   2.660  31.746  1.00  0.00           C  \\nATOM    572  H   ILE A  37      -5.169   0.139  34.954  1.00  0.00           H  \\nATOM    573  HA  ILE A  37      -6.309  -1.289  32.564  1.00  0.00           H  \\nATOM    574  HB  ILE A  37      -4.038   0.682  32.556  1.00  0.00           H  \\nATOM    575 1HG1 ILE A  37      -6.654   1.171  31.172  1.00  0.00           H  \\nATOM    576 2HG1 ILE A  37      -6.576   1.468  32.901  1.00  0.00           H  \\nATOM    577 1HG2 ILE A  37      -4.295   0.434  30.033  1.00  0.00           H  \\nATOM    578 2HG2 ILE A  37      -3.888  -1.131  30.774  1.00  0.00           H  \\nATOM    579 3HG2 ILE A  37      -5.564  -0.788  30.283  1.00  0.00           H  \\nATOM    580 1HD1 ILE A  37      -5.979   3.472  31.640  1.00  0.00           H  \\nATOM    581 2HD1 ILE A  37      -4.594   2.872  32.584  1.00  0.00           H  \\nATOM    582 3HD1 ILE A  37      -4.674   2.573  30.832  1.00  0.00           H  \\nATOM    583  N   PRO A  38      -4.681  -3.120  32.696  1.00  0.00           N  \\nATOM    584  CA  PRO A  38      -3.711  -4.211  32.710  1.00  0.00           C  \\nATOM    585  C   PRO A  38      -2.479  -4.022  31.856  1.00  0.00           C  \\nATOM    586  O   PRO A  38      -2.518  -3.408  30.737  1.00  0.00           O  \\nATOM    587  CB  PRO A  38      -4.513  -5.390  32.095  1.00  0.00           C  \\nATOM    588  CG  PRO A  38      -5.927  -5.031  32.384  1.00  0.00           C  \\nATOM    589  CD  PRO A  38      -6.031  -3.574  32.290  1.00  0.00           C  \\nATOM    590  HA  PRO A  38      -3.424  -4.427  33.750  1.00  0.00           H  \\nATOM    591 1HB  PRO A  38      -4.297  -5.472  31.020  1.00  0.00           H  \\nATOM    592 2HB  PRO A  38      -4.202  -6.338  32.558  1.00  0.00           H  \\nATOM    593 1HG  PRO A  38      -6.599  -5.526  31.667  1.00  0.00           H  \\nATOM    594 2HG  PRO A  38      -6.212  -5.387  33.385  1.00  0.00           H  \\nATOM    595 1HD  PRO A  38      -6.263  -3.289  31.253  1.00  0.00           H  \\nATOM    596 2HD  PRO A  38      -6.814  -3.217  32.975  1.00  0.00           H  \\nATOM    597  N   VAL A  39      -1.357  -4.473  32.408  1.00  0.00           N  \\nATOM    598  CA  VAL A  39      -0.035  -4.364  31.754  1.00  0.00           C  \\nATOM    599  C   VAL A  39       0.392  -5.765  31.264  1.00  0.00           C  \\nATOM    600  O   VAL A  39       0.192  -6.783  31.916  1.00  0.00           O  \\nATOM    601  CB  VAL A  39       1.005  -3.776  32.737  1.00  0.00           C  \\nATOM    602  CG1 VAL A  39       2.234  -3.509  32.023  1.00  0.00           C  \\nATOM    603  CG2 VAL A  39       0.434  -2.451  33.330  1.00  0.00           C  \\nATOM    604  H   VAL A  39      -1.422  -4.909  33.317  1.00  0.00           H  \\nATOM    605  HA  VAL A  39      -0.126  -3.693  30.899  1.00  0.00           H  \\nATOM    606  HB  VAL A  39       1.186  -4.497  33.534  1.00  0.00           H  \\nATOM    607 1HG1 VAL A  39       2.972  -3.094  32.711  1.00  0.00           H  \\nATOM    608 2HG1 VAL A  39       2.616  -4.437  31.598  1.00  0.00           H  \\nATOM    609 3HG1 VAL A  39       2.042  -2.794  31.224  1.00  0.00           H  \\nATOM    610 1HG2 VAL A  39       1.155  -2.024  34.026  1.00  0.00           H  \\nATOM    611 2HG2 VAL A  39       0.245  -1.743  32.523  1.00  0.00           H  \\nATOM    612 3HG2 VAL A  39      -0.498  -2.660  33.856  1.00  0.00           H  \\nATOM    613  N   LEU A  40       0.931  -5.786  30.058  1.00  0.00           N  \\nATOM    614  CA  LEU A  40       1.376  -7.014  29.497  1.00  0.00           C  \\nATOM    615  C   LEU A  40       2.718  -7.440  30.136  1.00  0.00           C  \\nATOM    616  O   LEU A  40       3.366  -6.669  30.829  1.00  0.00           O  \\nATOM    617  CB  LEU A  40       1.598  -6.762  27.995  1.00  0.00           C  \\nATOM    618  CG  LEU A  40       0.351  -6.563  27.145  1.00  0.00           C  \\nATOM    619  CD1 LEU A  40       0.790  -6.057  25.790  1.00  0.00           C  \\nATOM    620  CD2 LEU A  40      -0.472  -7.884  27.026  1.00  0.00           C  \\nATOM    621  H   LEU A  40       1.033  -4.935  29.523  1.00  0.00           H  \\nATOM    622  HA  LEU A  40       0.599  -7.763  29.646  1.00  0.00           H  \\nATOM    623 1HB  LEU A  40       2.212  -5.871  27.880  1.00  0.00           H  \\nATOM    624 2HB  LEU A  40       2.141  -7.609  27.577  1.00  0.00           H  \\nATOM    625  HG  LEU A  40      -0.282  -5.801  27.600  1.00  0.00           H  \\nATOM    626 1HD1 LEU A  40      -0.085  -5.905  25.158  1.00  0.00           H  \\nATOM    627 2HD1 LEU A  40       1.320  -5.112  25.909  1.00  0.00           H  \\nATOM    628 3HD1 LEU A  40       1.450  -6.788  25.325  1.00  0.00           H  \\nATOM    629 1HD2 LEU A  40      -1.357  -7.708  26.413  1.00  0.00           H  \\nATOM    630 2HD2 LEU A  40       0.143  -8.655  26.562  1.00  0.00           H  \\nATOM    631 3HD2 LEU A  40      -0.778  -8.213  28.019  1.00  0.00           H  \\nATOM    632  N   PRO A  41       3.109  -8.690  29.961  1.00  0.00           N  \\nATOM    633  CA  PRO A  41       4.340  -9.174  30.590  1.00  0.00           C  \\nATOM    634  C   PRO A  41       5.593  -8.445  30.081  1.00  0.00           C  \\nATOM    635  O   PRO A  41       5.692  -8.031  28.965  1.00  0.00           O  \\nATOM    636  CB  PRO A  41       4.449 -10.634  30.082  1.00  0.00           C  \\nATOM    637  CG  PRO A  41       3.041 -11.081  29.853  1.00  0.00           C  \\nATOM    638  CD  PRO A  41       2.289  -9.805  29.408  1.00  0.00           C  \\nATOM    639  HA  PRO A  41       4.222  -9.148  31.683  1.00  0.00           H  \\nATOM    640 1HB  PRO A  41       5.051 -10.668  29.162  1.00  0.00           H  \\nATOM    641 2HB  PRO A  41       4.967 -11.254  30.828  1.00  0.00           H  \\nATOM    642 1HG  PRO A  41       3.013 -11.873  29.090  1.00  0.00           H  \\nATOM    643 2HG  PRO A  41       2.625 -11.514  30.775  1.00  0.00           H  \\nATOM    644 1HD  PRO A  41       2.252  -9.765  28.309  1.00  0.00           H  \\nATOM    645 2HD  PRO A  41       1.274  -9.811  29.831  1.00  0.00           H  \\nATOM    646  N   ASN A  42       6.574  -8.293  30.969  1.00  0.00           N  \\nATOM    647  CA  ASN A  42       7.807  -7.604  30.619  1.00  0.00           C  \\nATOM    648  C   ASN A  42       8.681  -8.526  29.719  1.00  0.00           C  \\nATOM    649  O   ASN A  42       8.839  -9.708  30.031  1.00  0.00           O  \\nATOM    650  CB  ASN A  42       8.525  -7.271  31.904  1.00  0.00           C  \\nATOM    651  CG  ASN A  42       9.861  -6.580  31.657  1.00  0.00           C  \\nATOM    652  OD1 ASN A  42      10.222  -6.216  30.530  1.00  0.00           O  \\nATOM    653  ND2 ASN A  42      10.583  -6.355  32.753  1.00  0.00           N  \\nATOM    654  H   ASN A  42       6.464  -8.661  31.903  1.00  0.00           H  \\nATOM    655  HA  ASN A  42       7.552  -6.687  30.086  1.00  0.00           H  \\nATOM    656 1HB  ASN A  42       7.897  -6.621  32.514  1.00  0.00           H  \\nATOM    657 2HB  ASN A  42       8.698  -8.185  32.472  1.00  0.00           H  \\nATOM    658 1HD2 ASN A  42      11.474  -5.905  32.682  1.00  0.00           H  \\nATOM    659 2HD2 ASN A  42      10.235  -6.635  33.647  1.00  0.00           H  \\nATOM    660  N   ARG A  43       9.128  -8.039  28.587  1.00  0.00           N  \\nATOM    661  CA  ARG A  43       9.966  -8.836  27.670  1.00  0.00           C  \\nATOM    662  C   ARG A  43      11.256  -9.280  28.368  1.00  0.00           C  \\nATOM    663  O   ARG A  43      11.793 -10.327  28.078  1.00  0.00           O  \\nATOM    664  CB  ARG A  43      10.318  -7.980  26.457  1.00  0.00           C  \\nATOM    665  CG  ARG A  43      10.948  -8.683  25.226  1.00  0.00           C  \\nATOM    666  CD  ARG A  43      11.134  -7.605  24.091  1.00  0.00           C  \\nATOM    667  NE  ARG A  43      11.940  -8.021  22.904  1.00  0.00           N  \\nATOM    668  CZ  ARG A  43      13.276  -7.976  22.851  1.00  0.00           C  \\nATOM    669  NH1 ARG A  43      13.976  -7.572  23.914  1.00  0.00           N  \\nATOM    670  NH2 ARG A  43      13.936  -8.329  21.739  1.00  0.00           N  \\nATOM    671  H   ARG A  43       8.891  -7.090  28.337  1.00  0.00           H  \\nATOM    672  HA  ARG A  43       9.394  -9.706  27.347  1.00  0.00           H  \\nATOM    673 1HB  ARG A  43       9.421  -7.483  26.090  1.00  0.00           H  \\nATOM    674 2HB  ARG A  43      11.025  -7.204  26.752  1.00  0.00           H  \\nATOM    675 1HG  ARG A  43      11.913  -9.107  25.504  1.00  0.00           H  \\nATOM    676 2HG  ARG A  43      10.286  -9.480  24.884  1.00  0.00           H  \\nATOM    677 1HD  ARG A  43      10.159  -7.315  23.702  1.00  0.00           H  \\nATOM    678 2HD  ARG A  43      11.636  -6.729  24.500  1.00  0.00           H  \\nATOM    679  HE  ARG A  43      11.441  -8.357  22.091  1.00  0.00           H  \\nATOM    680 1HH1 ARG A  43      13.497  -7.300  24.761  1.00  0.00           H  \\nATOM    681 2HH1 ARG A  43      14.984  -7.539  23.872  1.00  0.00           H  \\nATOM    682 1HH2 ARG A  43      13.424  -8.635  20.923  1.00  0.00           H  \\nATOM    683 2HH2 ARG A  43      14.944  -8.289  21.716  1.00  0.00           H  \\nATOM    684  N   VAL A  44      11.800  -8.424  29.226  1.00  0.00           N  \\nATOM    685  CA  VAL A  44      13.109  -8.717  29.826  1.00  0.00           C  \\nATOM    686  C   VAL A  44      12.933  -9.821  30.860  1.00  0.00           C  \\nATOM    687  O   VAL A  44      12.183  -9.699  31.805  1.00  0.00           O  \\nATOM    688  CB  VAL A  44      13.694  -7.479  30.532  1.00  0.00           C  \\nATOM    689  CG1 VAL A  44      15.056  -7.842  31.179  1.00  0.00           C  \\nATOM    690  CG2 VAL A  44      13.887  -6.423  29.518  1.00  0.00           C  \\nATOM    691  H   VAL A  44      11.325  -7.568  29.475  1.00  0.00           H  \\nATOM    692  HA  VAL A  44      13.796  -9.012  29.032  1.00  0.00           H  \\nATOM    693  HB  VAL A  44      12.995  -7.146  31.300  1.00  0.00           H  \\nATOM    694 1HG1 VAL A  44      15.467  -6.964  31.678  1.00  0.00           H  \\nATOM    695 2HG1 VAL A  44      14.911  -8.639  31.908  1.00  0.00           H  \\nATOM    696 3HG1 VAL A  44      15.748  -8.177  30.406  1.00  0.00           H  \\nATOM    697 1HG2 VAL A  44      14.300  -5.534  29.993  1.00  0.00           H  \\nATOM    698 2HG2 VAL A  44      14.575  -6.777  28.750  1.00  0.00           H  \\nATOM    699 3HG2 VAL A  44      12.928  -6.177  29.061  1.00  0.00           H  \\nATOM    700  N   GLY A  45      13.556 -10.943  30.591  1.00  0.00           N  \\nATOM    701  CA  GLY A  45      13.460 -12.068  31.447  1.00  0.00           C  \\nATOM    702  C   GLY A  45      12.304 -12.980  31.207  1.00  0.00           C  \\nATOM    703  O   GLY A  45      12.119 -13.855  31.992  1.00  0.00           O  \\nATOM    704  H   GLY A  45      14.118 -11.008  29.754  1.00  0.00           H  \\nATOM    705 1HA  GLY A  45      14.365 -12.669  31.361  1.00  0.00           H  \\nATOM    706 2HA  GLY A  45      13.394 -11.733  32.482  1.00  0.00           H  \\nATOM    707  N   LEU A  46      11.508 -12.746  30.144  1.00  0.00           N  \\nATOM    708  CA  LEU A  46      10.296 -13.555  29.952  1.00  0.00           C  \\nATOM    709  C   LEU A  46      10.694 -14.915  29.350  1.00  0.00           C  \\nATOM    710  O   LEU A  46      11.333 -14.959  28.287  1.00  0.00           O  \\nATOM    711  CB  LEU A  46       9.301 -12.838  29.024  1.00  0.00           C  \\nATOM    712  CG  LEU A  46       7.953 -13.535  28.911  1.00  0.00           C  \\nATOM    713  CD1 LEU A  46       7.158 -13.531  30.126  1.00  0.00           C  \\nATOM    714  CD2 LEU A  46       7.151 -12.660  27.838  1.00  0.00           C  \\nATOM    715  H   LEU A  46      11.728 -12.023  29.474  1.00  0.00           H  \\nATOM    716  HA  LEU A  46       9.820 -13.701  30.921  1.00  0.00           H  \\nATOM    717 1HB  LEU A  46       9.141 -11.829  29.400  1.00  0.00           H  \\nATOM    718 2HB  LEU A  46       9.742 -12.768  28.030  1.00  0.00           H  \\nATOM    719  HG  LEU A  46       8.101 -14.561  28.574  1.00  0.00           H  \\nATOM    720 1HD1 LEU A  46       6.217 -14.051  29.947  1.00  0.00           H  \\nATOM    721 2HD1 LEU A  46       7.705 -14.038  30.921  1.00  0.00           H  \\nATOM    722 3HD1 LEU A  46       6.954 -12.503  30.424  1.00  0.00           H  \\nATOM    723 1HD2 LEU A  46       6.159 -13.088  27.688  1.00  0.00           H  \\nATOM    724 2HD2 LEU A  46       7.053 -11.637  28.202  1.00  0.00           H  \\nATOM    725 3HD2 LEU A  46       7.693 -12.658  26.892  1.00  0.00           H  \\nATOM    726  N   PRO A  47      10.326 -16.016  30.049  1.00  0.00           N  \\nATOM    727  CA  PRO A  47      10.725 -17.333  29.545  1.00  0.00           C  \\nATOM    728  C   PRO A  47      10.044 -17.588  28.236  1.00  0.00           C  \\nATOM    729  O   PRO A  47       8.907 -17.132  28.015  1.00  0.00           O  \\nATOM    730  CB  PRO A  47      10.108 -18.289  30.541  1.00  0.00           C  \\nATOM    731  CG  PRO A  47      10.129 -17.524  31.789  1.00  0.00           C  \\nATOM    732  CD  PRO A  47       9.683 -16.131  31.344  1.00  0.00           C  \\nATOM    733  HA  PRO A  47      11.822 -17.408  29.559  1.00  0.00           H  \\nATOM    734 1HB  PRO A  47       9.095 -18.568  30.217  1.00  0.00           H  \\nATOM    735 2HB  PRO A  47      10.698 -19.217  30.587  1.00  0.00           H  \\nATOM    736 1HG  PRO A  47       9.455 -17.983  32.527  1.00  0.00           H  \\nATOM    737 2HG  PRO A  47      11.138 -17.540  32.227  1.00  0.00           H  \\nATOM    738 1HD  PRO A  47       8.586 -16.103  31.270  1.00  0.00           H  \\nATOM    739 2HD  PRO A  47      10.042 -15.382  32.065  1.00  0.00           H  \\nATOM    740  N   ILE A  48      10.791 -18.242  27.366  1.00  0.00           N  \\nATOM    741  CA  ILE A  48      10.316 -18.462  25.955  1.00  0.00           C  \\nATOM    742  C   ILE A  48       9.002 -19.211  25.913  1.00  0.00           C  \\nATOM    743  O   ILE A  48       8.144 -18.987  25.041  1.00  0.00           O  \\nATOM    744  CB  ILE A  48      11.456 -19.007  25.028  1.00  0.00           C  \\nATOM    745  CG1 ILE A  48      10.999 -19.203  23.598  1.00  0.00           C  \\nATOM    746  CG2 ILE A  48      11.954 -20.382  25.429  1.00  0.00           C  \\nATOM    747  CD1 ILE A  48      10.761 -17.943  22.766  1.00  0.00           C  \\nATOM    748  H   ILE A  48      11.693 -18.602  27.643  1.00  0.00           H  \\nATOM    749  HA  ILE A  48       9.983 -17.507  25.549  1.00  0.00           H  \\nATOM    750  HB  ILE A  48      12.308 -18.329  25.062  1.00  0.00           H  \\nATOM    751 1HG1 ILE A  48      11.737 -19.796  23.059  1.00  0.00           H  \\nATOM    752 2HG1 ILE A  48      10.062 -19.761  23.589  1.00  0.00           H  \\nATOM    753 1HG2 ILE A  48      12.741 -20.698  24.745  1.00  0.00           H  \\nATOM    754 2HG2 ILE A  48      12.349 -20.344  26.443  1.00  0.00           H  \\nATOM    755 3HG2 ILE A  48      11.130 -21.094  25.387  1.00  0.00           H  \\nATOM    756 1HD1 ILE A  48      10.437 -18.224  21.764  1.00  0.00           H  \\nATOM    757 2HD1 ILE A  48       9.989 -17.335  23.239  1.00  0.00           H  \\nATOM    758 3HD1 ILE A  48      11.685 -17.371  22.701  1.00  0.00           H  \\nATOM    759  N   ASN A  49       8.738 -20.011  26.902  1.00  0.00           N  \\nATOM    760  CA  ASN A  49       7.562 -20.812  26.816  1.00  0.00           C  \\nATOM    761  C   ASN A  49       6.307 -20.034  27.248  1.00  0.00           C  \\nATOM    762  O   ASN A  49       5.226 -20.577  27.171  1.00  0.00           O  \\nATOM    763  CB  ASN A  49       7.629 -22.120  27.594  1.00  0.00           C  \\nATOM    764  CG  ASN A  49       8.027 -21.973  28.996  1.00  0.00           C  \\nATOM    765  OD1 ASN A  49       8.442 -20.920  29.464  1.00  0.00           O  \\nATOM    766  ND2 ASN A  49       8.005 -23.126  29.698  1.00  0.00           N  \\nATOM    767  H   ASN A  49       9.326 -20.087  27.720  1.00  0.00           H  \\nATOM    768  HA  ASN A  49       7.400 -21.078  25.770  1.00  0.00           H  \\nATOM    769 1HB  ASN A  49       6.653 -22.606  27.573  1.00  0.00           H  \\nATOM    770 2HB  ASN A  49       8.340 -22.793  27.114  1.00  0.00           H  \\nATOM    771 1HD2 ASN A  49       8.261 -23.128  30.665  1.00  0.00           H  \\nATOM    772 2HD2 ASN A  49       7.733 -23.978  29.251  1.00  0.00           H  \\nATOM    773  N   GLN A  50       6.495 -18.802  27.723  1.00  0.00           N  \\nATOM    774  CA  GLN A  50       5.450 -17.858  28.030  1.00  0.00           C  \\nATOM    775  C   GLN A  50       5.407 -16.697  26.972  1.00  0.00           C  \\nATOM    776  O   GLN A  50       4.655 -15.757  27.234  1.00  0.00           O  \\nATOM    777  CB  GLN A  50       5.733 -17.227  29.404  1.00  0.00           C  \\nATOM    778  CG  GLN A  50       5.598 -18.185  30.609  1.00  0.00           C  \\nATOM    779  CD  GLN A  50       6.147 -17.551  31.862  1.00  0.00           C  \\nATOM    780  OE1 GLN A  50       6.726 -18.241  32.669  1.00  0.00           O  \\nATOM    781  NE2 GLN A  50       6.018 -16.227  32.006  1.00  0.00           N  \\nATOM    782  H   GLN A  50       7.458 -18.535  27.870  1.00  0.00           H  \\nATOM    783  HA  GLN A  50       4.501 -18.393  28.067  1.00  0.00           H  \\nATOM    784 1HB  GLN A  50       6.746 -16.825  29.418  1.00  0.00           H  \\nATOM    785 2HB  GLN A  50       5.049 -16.395  29.570  1.00  0.00           H  \\nATOM    786 1HG  GLN A  50       4.544 -18.417  30.759  1.00  0.00           H  \\nATOM    787 2HG  GLN A  50       6.155 -19.098  30.398  1.00  0.00           H  \\nATOM    788 1HE2 GLN A  50       6.372 -15.776  32.826  1.00  0.00           H  \\nATOM    789 2HE2 GLN A  50       5.568 -15.688  31.294  1.00  0.00           H  \\nATOM    790  N   ARG A  51       6.118 -16.781  25.857  1.00  0.00           N  \\nATOM    791  CA  ARG A  51       6.284 -15.598  25.018  1.00  0.00           C  \\nATOM    792  C   ARG A  51       5.131 -15.340  24.059  1.00  0.00           C  \\nATOM    793  O   ARG A  51       4.979 -14.185  23.620  1.00  0.00           O  \\nATOM    794  CB  ARG A  51       7.665 -15.640  24.317  1.00  0.00           C  \\nATOM    795  CG  ARG A  51       8.088 -14.525  23.312  1.00  0.00           C  \\nATOM    796  CD  ARG A  51       7.987 -13.158  24.056  1.00  0.00           C  \\nATOM    797  NE  ARG A  51       8.403 -12.130  23.129  1.00  0.00           N  \\nATOM    798  CZ  ARG A  51       9.625 -11.861  22.698  1.00  0.00           C  \\nATOM    799  NH1 ARG A  51       9.694 -10.925  21.777  1.00  0.00           N  \\nATOM    800  NH2 ARG A  51      10.741 -12.489  23.132  1.00  0.00           N  \\nATOM    801  H   ARG A  51       6.549 -17.649  25.574  1.00  0.00           H  \\nATOM    802  HA  ARG A  51       6.236 -14.713  25.654  1.00  0.00           H  \\nATOM    803 1HB  ARG A  51       8.454 -15.639  25.068  1.00  0.00           H  \\nATOM    804 2HB  ARG A  51       7.756 -16.565  23.747  1.00  0.00           H  \\nATOM    805 1HG  ARG A  51       9.111 -14.703  22.981  1.00  0.00           H  \\nATOM    806 2HG  ARG A  51       7.420 -14.539  22.450  1.00  0.00           H  \\nATOM    807 1HD  ARG A  51       6.958 -12.992  24.373  1.00  0.00           H  \\nATOM    808 2HD  ARG A  51       8.638 -13.170  24.929  1.00  0.00           H  \\nATOM    809  HE  ARG A  51       7.689 -11.525  22.746  1.00  0.00           H  \\nATOM    810 1HH1 ARG A  51       8.851 -10.472  21.453  1.00  0.00           H  \\nATOM    811 2HH1 ARG A  51      10.589 -10.659  21.394  1.00  0.00           H  \\nATOM    812 1HH2 ARG A  51      10.672 -13.216  23.831  1.00  0.00           H  \\nATOM    813 2HH2 ARG A  51      11.642 -12.232  22.758  1.00  0.00           H  \\nATOM    814  N   PHE A  52       4.418 -16.413  23.687  1.00  0.00           N  \\nATOM    815  CA  PHE A  52       3.473 -16.338  22.557  1.00  0.00           C  \\nATOM    816  C   PHE A  52       2.130 -16.931  22.891  1.00  0.00           C  \\nATOM    817  O   PHE A  52       1.984 -17.772  23.811  1.00  0.00           O  \\nATOM    818  CB  PHE A  52       4.027 -17.013  21.268  1.00  0.00           C  \\nATOM    819  CG  PHE A  52       5.268 -16.420  20.763  1.00  0.00           C  \\nATOM    820  CD1 PHE A  52       5.313 -15.180  20.233  1.00  0.00           C  \\nATOM    821  CD2 PHE A  52       6.460 -17.156  20.876  1.00  0.00           C  \\nATOM    822  CE1 PHE A  52       6.518 -14.634  19.830  1.00  0.00           C  \\nATOM    823  CE2 PHE A  52       7.612 -16.652  20.422  1.00  0.00           C  \\nATOM    824  CZ  PHE A  52       7.678 -15.399  19.893  1.00  0.00           C  \\nATOM    825  H   PHE A  52       4.524 -17.287  24.182  1.00  0.00           H  \\nATOM    826  HA  PHE A  52       3.291 -15.287  22.327  1.00  0.00           H  \\nATOM    827 1HB  PHE A  52       4.214 -18.068  21.462  1.00  0.00           H  \\nATOM    828 2HB  PHE A  52       3.281 -16.953  20.477  1.00  0.00           H  \\nATOM    829  HD1 PHE A  52       4.388 -14.613  20.127  1.00  0.00           H  \\nATOM    830  HD2 PHE A  52       6.433 -18.163  21.295  1.00  0.00           H  \\nATOM    831  HE1 PHE A  52       6.556 -13.608  19.465  1.00  0.00           H  \\nATOM    832  HE2 PHE A  52       8.510 -17.266  20.481  1.00  0.00           H  \\nATOM    833  HZ  PHE A  52       8.619 -14.995  19.522  1.00  0.00           H  \\nATOM    834  N   ILE A  53       1.106 -16.411  22.234  1.00  0.00           N  \\nATOM    835  CA  ILE A  53      -0.229 -17.114  22.257  1.00  0.00           C  \\nATOM    836  C   ILE A  53      -0.607 -17.322  20.794  1.00  0.00           C  \\nATOM    837  O   ILE A  53      -0.129 -16.591  19.917  1.00  0.00           O  \\nATOM    838  CB  ILE A  53      -1.313 -16.422  23.034  1.00  0.00           C  \\nATOM    839  CG1 ILE A  53      -1.740 -15.107  22.424  1.00  0.00           C  \\nATOM    840  CG2 ILE A  53      -0.960 -16.318  24.529  1.00  0.00           C  \\nATOM    841  CD1 ILE A  53      -3.047 -14.734  23.021  1.00  0.00           C  \\nATOM    842  H   ILE A  53       1.200 -15.550  21.715  1.00  0.00           H  \\nATOM    843  HA  ILE A  53      -0.098 -18.090  22.724  1.00  0.00           H  \\nATOM    844  HB  ILE A  53      -2.243 -16.982  22.938  1.00  0.00           H  \\nATOM    845 1HG1 ILE A  53      -0.982 -14.350  22.623  1.00  0.00           H  \\nATOM    846 2HG1 ILE A  53      -1.816 -15.215  21.342  1.00  0.00           H  \\nATOM    847 1HG2 ILE A  53      -1.766 -15.812  25.059  1.00  0.00           H  \\nATOM    848 2HG2 ILE A  53      -0.827 -17.317  24.941  1.00  0.00           H  \\nATOM    849 3HG2 ILE A  53      -0.036 -15.751  24.646  1.00  0.00           H  \\nATOM    850 1HD1 ILE A  53      -3.383 -13.786  22.599  1.00  0.00           H  \\nATOM    851 2HD1 ILE A  53      -3.782 -15.509  22.802  1.00  0.00           H  \\nATOM    852 3HD1 ILE A  53      -2.938 -14.631  24.100  1.00  0.00           H  \\nATOM    853  N   LEU A  54      -1.454 -18.315  20.544  1.00  0.00           N  \\nATOM    854  CA  LEU A  54      -1.828 -18.601  19.155  1.00  0.00           C  \\nATOM    855  C   LEU A  54      -3.326 -18.367  19.046  1.00  0.00           C  \\nATOM    856  O   LEU A  54      -4.084 -18.572  19.960  1.00  0.00           O  \\nATOM    857  CB  LEU A  54      -1.584 -20.077  18.851  1.00  0.00           C  \\nATOM    858  CG  LEU A  54      -0.135 -20.483  19.110  1.00  0.00           C  \\nATOM    859  CD1 LEU A  54      -0.089 -21.974  18.847  1.00  0.00           C  \\nATOM    860  CD2 LEU A  54       0.790 -19.743  18.216  1.00  0.00           C  \\nATOM    861  H   LEU A  54      -1.847 -18.878  21.285  1.00  0.00           H  \\nATOM    862  HA  LEU A  54      -1.205 -17.995  18.498  1.00  0.00           H  \\nATOM    863 1HB  LEU A  54      -2.248 -20.674  19.474  1.00  0.00           H  \\nATOM    864 2HB  LEU A  54      -1.835 -20.263  17.807  1.00  0.00           H  \\nATOM    865  HG  LEU A  54       0.126 -20.261  20.145  1.00  0.00           H  \\nATOM    866 1HD1 LEU A  54       0.924 -22.342  19.013  1.00  0.00           H  \\nATOM    867 2HD1 LEU A  54      -0.775 -22.484  19.524  1.00  0.00           H  \\nATOM    868 3HD1 LEU A  54      -0.382 -22.171  17.816  1.00  0.00           H  \\nATOM    869 1HD2 LEU A  54       1.816 -20.049  18.420  1.00  0.00           H  \\nATOM    870 2HD2 LEU A  54       0.546 -19.963  17.177  1.00  0.00           H  \\nATOM    871 3HD2 LEU A  54       0.689 -18.672  18.393  1.00  0.00           H  \\nATOM    872  N   VAL A  55      -3.722 -17.793  17.913  1.00  0.00           N  \\nATOM    873  CA  VAL A  55      -5.130 -17.482  17.637  1.00  0.00           C  \\nATOM    874  C   VAL A  55      -5.480 -18.272  16.339  1.00  0.00           C  \\nATOM    875  O   VAL A  55      -5.015 -17.971  15.247  1.00  0.00           O  \\nATOM    876  CB  VAL A  55      -5.242 -15.994  17.387  1.00  0.00           C  \\nATOM    877  CG1 VAL A  55      -6.599 -15.670  16.841  1.00  0.00           C  \\nATOM    878  CG2 VAL A  55      -5.083 -15.252  18.715  1.00  0.00           C  \\nATOM    879  H   VAL A  55      -3.023 -17.565  17.221  1.00  0.00           H  \\nATOM    880  HA  VAL A  55      -5.723 -17.755  18.511  1.00  0.00           H  \\nATOM    881  HB  VAL A  55      -4.456 -15.691  16.695  1.00  0.00           H  \\nATOM    882 1HG1 VAL A  55      -6.673 -14.597  16.663  1.00  0.00           H  \\nATOM    883 2HG1 VAL A  55      -6.751 -16.204  15.903  1.00  0.00           H  \\nATOM    884 3HG1 VAL A  55      -7.361 -15.971  17.559  1.00  0.00           H  \\nATOM    885 1HG2 VAL A  55      -5.162 -14.179  18.544  1.00  0.00           H  \\nATOM    886 2HG2 VAL A  55      -5.866 -15.568  19.405  1.00  0.00           H  \\nATOM    887 3HG2 VAL A  55      -4.107 -15.480  19.145  1.00  0.00           H  \\nATOM    888  N   GLU A  56      -6.323 -19.279  16.471  1.00  0.00           N  \\nATOM    889  CA  GLU A  56      -6.738 -20.117  15.384  1.00  0.00           C  \\nATOM    890  C   GLU A  56      -8.043 -19.552  14.787  1.00  0.00           C  \\nATOM    891  O   GLU A  56      -9.079 -19.532  15.408  1.00  0.00           O  \\nATOM    892  CB  GLU A  56      -6.912 -21.569  15.845  1.00  0.00           C  \\nATOM    893  CG  GLU A  56      -7.314 -22.430  14.603  1.00  0.00           C  \\nATOM    894  CD  GLU A  56      -8.109 -23.697  15.007  1.00  0.00           C  \\nATOM    895  OE1 GLU A  56      -7.561 -24.816  14.874  1.00  0.00           O  \\nATOM    896  OE2 GLU A  56      -9.293 -23.586  15.446  1.00  0.00           O  \\nATOM    897  H   GLU A  56      -6.688 -19.457  17.396  1.00  0.00           H  \\nATOM    898  HA  GLU A  56      -5.967 -20.091  14.613  1.00  0.00           H  \\nATOM    899 1HB  GLU A  56      -5.980 -21.925  16.284  1.00  0.00           H  \\nATOM    900 2HB  GLU A  56      -7.678 -21.616  16.619  1.00  0.00           H  \\nATOM    901 1HG  GLU A  56      -7.922 -21.822  13.934  1.00  0.00           H  \\nATOM    902 2HG  GLU A  56      -6.411 -22.719  14.067  1.00  0.00           H  \\nATOM    903  N   LEU A  57      -7.924 -19.038  13.565  1.00  0.00           N  \\nATOM    904  CA  LEU A  57      -9.090 -18.459  12.880  1.00  0.00           C  \\nATOM    905  C   LEU A  57      -9.629 -19.388  11.798  1.00  0.00           C  \\nATOM    906  O   LEU A  57      -8.895 -19.938  11.049  1.00  0.00           O  \\nATOM    907  CB  LEU A  57      -8.612 -17.204  12.149  1.00  0.00           C  \\nATOM    908  CG  LEU A  57      -7.986 -16.126  13.036  1.00  0.00           C  \\nATOM    909  CD1 LEU A  57      -7.454 -15.001  12.115  1.00  0.00           C  \\nATOM    910  CD2 LEU A  57      -8.986 -15.527  14.022  1.00  0.00           C  \\nATOM    911  H   LEU A  57      -7.029 -19.041  13.096  1.00  0.00           H  \\nATOM    912  HA  LEU A  57      -9.836 -18.196  13.629  1.00  0.00           H  \\nATOM    913 1HB  LEU A  57      -7.872 -17.496  11.406  1.00  0.00           H  \\nATOM    914 2HB  LEU A  57      -9.461 -16.759  11.631  1.00  0.00           H  \\nATOM    915  HG  LEU A  57      -7.166 -16.558  13.611  1.00  0.00           H  \\nATOM    916 1HD1 LEU A  57      -7.001 -14.217  12.722  1.00  0.00           H  \\nATOM    917 2HD1 LEU A  57      -6.706 -15.410  11.436  1.00  0.00           H  \\nATOM    918 3HD1 LEU A  57      -8.278 -14.582  11.539  1.00  0.00           H  \\nATOM    919 1HD2 LEU A  57      -8.490 -14.768  14.628  1.00  0.00           H  \\nATOM    920 2HD2 LEU A  57      -9.810 -15.071  13.473  1.00  0.00           H  \\nATOM    921 3HD2 LEU A  57      -9.373 -16.313  14.671  1.00  0.00           H  \\nATOM    922  N   SER A  58     -10.959 -19.499  11.770  1.00  0.00           N  \\nATOM    923  CA  SER A  58     -11.626 -20.347  10.752  1.00  0.00           C  \\nATOM    924  C   SER A  58     -12.653 -19.483  10.086  1.00  0.00           C  \\nATOM    925  O   SER A  58     -13.085 -18.470  10.665  1.00  0.00           O  \\nATOM    926  CB  SER A  58     -12.288 -21.518  11.482  1.00  0.00           C  \\nATOM    927  OG  SER A  58     -11.272 -22.409  11.973  1.00  0.00           O  \\nATOM    928  H   SER A  58     -11.524 -19.004  12.445  1.00  0.00           H  \\nATOM    929  HA  SER A  58     -10.868 -20.719  10.061  1.00  0.00           H  \\nATOM    930 1HB  SER A  58     -12.893 -21.138  12.305  1.00  0.00           H  \\nATOM    931 2HB  SER A  58     -12.956 -22.041  10.799  1.00  0.00           H  \\nATOM    932  HG  SER A  58     -10.433 -22.025  11.707  1.00  0.00           H  \\nATOM    933  N   ASN A  59     -13.174 -19.925   8.930  1.00  0.00           N  \\nATOM    934  CA  ASN A  59     -14.171 -19.090   8.249  1.00  0.00           C  \\nATOM    935  C   ASN A  59     -15.255 -19.947   7.558  1.00  0.00           C  \\nATOM    936  O   ASN A  59     -15.205 -21.156   7.634  1.00  0.00           O  \\nATOM    937  CB  ASN A  59     -13.486 -18.089   7.298  1.00  0.00           C  \\nATOM    938  CG  ASN A  59     -12.769 -18.771   6.144  1.00  0.00           C  \\nATOM    939  OD1 ASN A  59     -12.771 -19.997   6.023  1.00  0.00           O  \\nATOM    940  ND2 ASN A  59     -12.146 -17.994   5.314  1.00  0.00           N  \\nATOM    941  H   ASN A  59     -12.905 -20.806   8.516  1.00  0.00           H  \\nATOM    942  HA  ASN A  59     -14.725 -18.528   9.003  1.00  0.00           H  \\nATOM    943 1HB  ASN A  59     -14.232 -17.404   6.892  1.00  0.00           H  \\nATOM    944 2HB  ASN A  59     -12.764 -17.493   7.856  1.00  0.00           H  \\nATOM    945 1HD2 ASN A  59     -11.656 -18.385   4.534  1.00  0.00           H  \\nATOM    946 2HD2 ASN A  59     -12.155 -17.004   5.453  1.00  0.00           H  \\nATOM    947  N   HIS A  60     -16.236 -19.257   6.994  1.00  0.00           N  \\nATOM    948  CA  HIS A  60     -17.290 -19.970   6.216  1.00  0.00           C  \\nATOM    949  C   HIS A  60     -16.737 -20.854   5.113  1.00  0.00           C  \\nATOM    950  O   HIS A  60     -17.363 -21.897   4.792  1.00  0.00           O  \\nATOM    951  CB  HIS A  60     -18.204 -18.988   5.558  1.00  0.00           C  \\nATOM    952  CG  HIS A  60     -19.161 -18.298   6.446  1.00  0.00           C  \\nATOM    953  ND1 HIS A  60     -20.234 -18.946   7.010  1.00  0.00           N  \\nATOM    954  CD2 HIS A  60     -19.251 -16.992   6.822  1.00  0.00           C  \\nATOM    955  CE1 HIS A  60     -20.973 -18.076   7.669  1.00  0.00           C  \\nATOM    956  NE2 HIS A  60     -20.378 -16.894   7.609  1.00  0.00           N  \\nATOM    957  H   HIS A  60     -16.284 -18.252   7.081  1.00  0.00           H  \\nATOM    958  HA  HIS A  60     -17.883 -20.589   6.889  1.00  0.00           H  \\nATOM    959 1HB  HIS A  60     -17.614 -18.216   5.064  1.00  0.00           H  \\nATOM    960 2HB  HIS A  60     -18.789 -19.494   4.790  1.00  0.00           H  \\nATOM    961  HD2 HIS A  60     -18.555 -16.187   6.585  1.00  0.00           H  \\nATOM    962  HE1 HIS A  60     -21.913 -18.293   8.177  1.00  0.00           H  \\nATOM    963  HE2 HIS A  60     -20.694 -16.050   8.065  1.00  0.00           H  \\nATOM    964  N   ALA A  61     -15.618 -20.464   4.519  1.00  0.00           N  \\nATOM    965  CA  ALA A  61     -14.986 -21.261   3.472  1.00  0.00           C  \\nATOM    966  C   ALA A  61     -14.395 -22.539   4.004  1.00  0.00           C  \\nATOM    967  O   ALA A  61     -13.904 -23.349   3.219  1.00  0.00           O  \\nATOM    968  CB  ALA A  61     -13.980 -20.413   2.701  1.00  0.00           C  \\nATOM    969  H   ALA A  61     -15.192 -19.592   4.799  1.00  0.00           H  \\nATOM    970  HA  ALA A  61     -15.764 -21.599   2.787  1.00  0.00           H  \\nATOM    971 1HB  ALA A  61     -13.515 -21.018   1.923  1.00  0.00           H  \\nATOM    972 2HB  ALA A  61     -14.492 -19.566   2.245  1.00  0.00           H  \\nATOM    973 3HB  ALA A  61     -13.214 -20.048   3.383  1.00  0.00           H  \\nATOM    974  N   GLU A  62     -14.414 -22.744   5.320  1.00  0.00           N  \\nATOM    975  CA  GLU A  62     -13.927 -23.967   6.007  1.00  0.00           C  \\nATOM    976  C   GLU A  62     -12.381 -24.048   5.910  1.00  0.00           C  \\nATOM    977  O   GLU A  62     -11.753 -25.135   5.921  1.00  0.00           O  \\nATOM    978  CB  GLU A  62     -14.583 -25.219   5.388  1.00  0.00           C  \\nATOM    979  CG  GLU A  62     -16.119 -25.237   5.575  1.00  0.00           C  \\nATOM    980  CD  GLU A  62     -16.677 -26.568   5.121  1.00  0.00           C  \\nATOM    981  OE1 GLU A  62     -16.752 -26.800   3.888  1.00  0.00           O  \\nATOM    982  OE2 GLU A  62     -16.992 -27.397   6.003  1.00  0.00           O  \\nATOM    983  H   GLU A  62     -14.796 -21.989   5.871  1.00  0.00           H  \\nATOM    984  HA  GLU A  62     -14.206 -23.907   7.059  1.00  0.00           H  \\nATOM    985 1HB  GLU A  62     -14.357 -25.259   4.322  1.00  0.00           H  \\nATOM    986 2HB  GLU A  62     -14.162 -26.114   5.845  1.00  0.00           H  \\nATOM    987 1HG  GLU A  62     -16.349 -25.068   6.627  1.00  0.00           H  \\nATOM    988 2HG  GLU A  62     -16.554 -24.420   5.001  1.00  0.00           H  \\nATOM    989  N   LEU A  63     -11.734 -22.874   5.863  1.00  0.00           N  \\nATOM    990  CA  LEU A  63     -10.269 -22.767   5.884  1.00  0.00           C  \\nATOM    991  C   LEU A  63      -9.873 -22.218   7.249  1.00  0.00           C  \\nATOM    992  O   LEU A  63     -10.680 -21.524   7.930  1.00  0.00           O  \\nATOM    993  CB  LEU A  63      -9.785 -21.804   4.801  1.00  0.00           C  \\nATOM    994  CG  LEU A  63     -10.168 -22.135   3.372  1.00  0.00           C  \\nATOM    995  CD1 LEU A  63      -9.734 -21.069   2.403  1.00  0.00           C  \\nATOM    996  CD2 LEU A  63      -9.584 -23.480   3.024  1.00  0.00           C  \\nATOM    997  H   LEU A  63     -12.288 -22.031   5.811  1.00  0.00           H  \\nATOM    998  HA  LEU A  63      -9.848 -23.752   5.686  1.00  0.00           H  \\nATOM    999 1HB  LEU A  63     -10.179 -20.812   5.017  1.00  0.00           H  \\nATOM   1000 2HB  LEU A  63      -8.697 -21.757   4.839  1.00  0.00           H  \\nATOM   1001  HG  LEU A  63     -11.254 -22.166   3.284  1.00  0.00           H  \\nATOM   1002 1HD1 LEU A  63     -10.031 -21.352   1.393  1.00  0.00           H  \\nATOM   1003 2HD1 LEU A  63     -10.206 -20.123   2.667  1.00  0.00           H  \\nATOM   1004 3HD1 LEU A  63      -8.651 -20.959   2.445  1.00  0.00           H  \\nATOM   1005 1HD2 LEU A  63      -9.849 -23.738   1.998  1.00  0.00           H  \\nATOM   1006 2HD2 LEU A  63      -8.499 -23.441   3.119  1.00  0.00           H  \\nATOM   1007 3HD2 LEU A  63      -9.981 -24.236   3.702  1.00  0.00           H  \\nATOM   1008  N   SER A  64      -8.664 -22.537   7.662  1.00  0.00           N  \\nATOM   1009  CA  SER A  64      -8.159 -22.106   8.954  1.00  0.00           C  \\nATOM   1010  C   SER A  64      -6.720 -21.658   8.815  1.00  0.00           C  \\nATOM   1011  O   SER A  64      -5.928 -22.233   8.077  1.00  0.00           O  \\nATOM   1012  CB  SER A  64      -8.219 -23.283   9.897  1.00  0.00           C  \\nATOM   1013  OG  SER A  64      -7.778 -22.979  11.183  1.00  0.00           O  \\nATOM   1014  H   SER A  64      -8.072 -23.096   7.064  1.00  0.00           H  \\nATOM   1015  HA  SER A  64      -8.797 -21.303   9.325  1.00  0.00           H  \\nATOM   1016 1HB  SER A  64      -9.244 -23.647   9.960  1.00  0.00           H  \\nATOM   1017 2HB  SER A  64      -7.608 -24.095   9.505  1.00  0.00           H  \\nATOM   1018  HG  SER A  64      -7.529 -22.052  11.165  1.00  0.00           H  \\nATOM   1019  N   VAL A  65      -6.372 -20.614   9.577  1.00  0.00           N  \\nATOM   1020  CA  VAL A  65      -4.973 -20.182   9.709  1.00  0.00           C  \\nATOM   1021  C   VAL A  65      -4.758 -19.952  11.202  1.00  0.00           C  \\nATOM   1022  O   VAL A  65      -5.700 -19.696  11.925  1.00  0.00           O  \\nATOM   1023  CB  VAL A  65      -4.560 -18.911   8.949  1.00  0.00           C  \\nATOM   1024  CG1 VAL A  65      -4.717 -19.157   7.434  1.00  0.00           C  \\nATOM   1025  CG2 VAL A  65      -5.340 -17.707   9.410  1.00  0.00           C  \\nATOM   1026  H   VAL A  65      -7.092 -20.110  10.075  1.00  0.00           H  \\nATOM   1027  HA  VAL A  65      -4.329 -20.973   9.323  1.00  0.00           H  \\nATOM   1028  HB  VAL A  65      -3.500 -18.725   9.120  1.00  0.00           H  \\nATOM   1029 1HG1 VAL A  65      -4.426 -18.260   6.887  1.00  0.00           H  \\nATOM   1030 2HG1 VAL A  65      -4.080 -19.988   7.132  1.00  0.00           H  \\nATOM   1031 3HG1 VAL A  65      -5.756 -19.396   7.210  1.00  0.00           H  \\nATOM   1032 1HG2 VAL A  65      -5.019 -16.829   8.850  1.00  0.00           H  \\nATOM   1033 2HG2 VAL A  65      -6.404 -17.876   9.242  1.00  0.00           H  \\nATOM   1034 3HG2 VAL A  65      -5.163 -17.544  10.473  1.00  0.00           H  \\nATOM   1035  N   THR A  66      -3.534 -20.058  11.676  1.00  0.00           N  \\nATOM   1036  CA  THR A  66      -3.252 -19.782  13.089  1.00  0.00           C  \\nATOM   1037  C   THR A  66      -2.228 -18.652  13.158  1.00  0.00           C  \\nATOM   1038  O   THR A  66      -1.136 -18.740  12.682  1.00  0.00           O  \\nATOM   1039  CB  THR A  66      -2.636 -21.043  13.776  1.00  0.00           C  \\nATOM   1040  OG1 THR A  66      -3.590 -22.114  13.671  1.00  0.00           O  \\nATOM   1041  CG2 THR A  66      -2.380 -20.809  15.246  1.00  0.00           C  \\nATOM   1042  H   THR A  66      -2.776 -20.332  11.067  1.00  0.00           H  \\nATOM   1043  HA  THR A  66      -4.190 -19.535  13.587  1.00  0.00           H  \\nATOM   1044  HB  THR A  66      -1.691 -21.294  13.295  1.00  0.00           H  \\nATOM   1045  HG1 THR A  66      -4.371 -21.802  13.208  1.00  0.00           H  \\nATOM   1046 1HG2 THR A  66      -1.951 -21.708  15.688  1.00  0.00           H  \\nATOM   1047 2HG2 THR A  66      -1.685 -19.978  15.367  1.00  0.00           H  \\nATOM   1048 3HG2 THR A  66      -3.319 -20.572  15.745  1.00  0.00           H  \\nATOM   1049  N   LEU A  67      -2.631 -17.532  13.767  1.00  0.00           N  \\nATOM   1050  CA  LEU A  67      -1.712 -16.385  13.999  1.00  0.00           C  \\nATOM   1051  C   LEU A  67      -0.962 -16.602  15.276  1.00  0.00           C  \\nATOM   1052  O   LEU A  67      -1.469 -17.184  16.217  1.00  0.00           O  \\nATOM   1053  CB  LEU A  67      -2.522 -15.062  14.182  1.00  0.00           C  \\nATOM   1054  CG  LEU A  67      -3.608 -14.826  13.168  1.00  0.00           C  \\nATOM   1055  CD1 LEU A  67      -4.216 -13.452  13.508  1.00  0.00           C  \\nATOM   1056  CD2 LEU A  67      -2.880 -14.680  11.866  1.00  0.00           C  \\nATOM   1057  H   LEU A  67      -3.590 -17.465  14.078  1.00  0.00           H  \\nATOM   1058  HA  LEU A  67      -1.065 -16.279  13.129  1.00  0.00           H  \\nATOM   1059 1HB  LEU A  67      -2.982 -15.070  15.169  1.00  0.00           H  \\nATOM   1060 2HB  LEU A  67      -1.830 -14.221  14.133  1.00  0.00           H  \\nATOM   1061  HG  LEU A  67      -4.289 -15.677  13.160  1.00  0.00           H  \\nATOM   1062 1HD1 LEU A  67      -5.015 -13.223  12.803  1.00  0.00           H  \\nATOM   1063 2HD1 LEU A  67      -4.621 -13.474  14.520  1.00  0.00           H  \\nATOM   1064 3HD1 LEU A  67      -3.444 -12.686  13.442  1.00  0.00           H  \\nATOM   1065 1HD2 LEU A  67      -3.599 -14.505  11.065  1.00  0.00           H  \\nATOM   1066 2HD2 LEU A  67      -2.191 -13.837  11.927  1.00  0.00           H  \\nATOM   1067 3HD2 LEU A  67      -2.320 -15.592  11.658  1.00  0.00           H  \\nATOM   1068  N   ALA A  68       0.282 -16.176  15.300  1.00  0.00           N  \\nATOM   1069  CA  ALA A  68       1.034 -16.070  16.559  1.00  0.00           C  \\nATOM   1070  C   ALA A  68       1.187 -14.603  17.041  1.00  0.00           C  \\nATOM   1071  O   ALA A  68       1.581 -13.720  16.236  1.00  0.00           O  \\nATOM   1072  CB  ALA A  68       2.424 -16.662  16.324  1.00  0.00           C  \\nATOM   1073  H   ALA A  68       0.732 -15.915  14.434  1.00  0.00           H  \\nATOM   1074  HA  ALA A  68       0.505 -16.645  17.319  1.00  0.00           H  \\nATOM   1075 1HB  ALA A  68       3.007 -16.597  17.243  1.00  0.00           H  \\nATOM   1076 2HB  ALA A  68       2.329 -17.707  16.027  1.00  0.00           H  \\nATOM   1077 3HB  ALA A  68       2.928 -16.105  15.535  1.00  0.00           H  \\nATOM   1078  N   LEU A  69       0.819 -14.384  18.292  1.00  0.00           N  \\nATOM   1079  CA  LEU A  69       0.921 -13.054  18.864  1.00  0.00           C  \\nATOM   1080  C   LEU A  69       1.961 -13.045  19.939  1.00  0.00           C  \\nATOM   1081  O   LEU A  69       2.047 -13.990  20.753  1.00  0.00           O  \\nATOM   1082  CB  LEU A  69      -0.425 -12.655  19.509  1.00  0.00           C  \\nATOM   1083  CG  LEU A  69      -1.552 -12.126  18.606  1.00  0.00           C  \\nATOM   1084  CD1 LEU A  69      -1.933 -13.039  17.478  1.00  0.00           C  \\nATOM   1085  CD2 LEU A  69      -2.810 -11.969  19.579  1.00  0.00           C  \\nATOM   1086  H   LEU A  69       0.464 -15.139  18.862  1.00  0.00           H  \\nATOM   1087  HA  LEU A  69       1.152 -12.351  18.065  1.00  0.00           H  \\nATOM   1088 1HB  LEU A  69      -0.834 -13.523  20.024  1.00  0.00           H  \\nATOM   1089 2HB  LEU A  69      -0.239 -11.875  20.247  1.00  0.00           H  \\nATOM   1090  HG  LEU A  69      -1.253 -11.171  18.173  1.00  0.00           H  \\nATOM   1091 1HD1 LEU A  69      -2.734 -12.583  16.896  1.00  0.00           H  \\nATOM   1092 2HD1 LEU A  69      -1.068 -13.205  16.836  1.00  0.00           H  \\nATOM   1093 3HD1 LEU A  69      -2.274 -13.992  17.881  1.00  0.00           H  \\nATOM   1094 1HD2 LEU A  69      -3.663 -11.593  19.013  1.00  0.00           H  \\nATOM   1095 2HD2 LEU A  69      -3.059 -12.939  20.011  1.00  0.00           H  \\nATOM   1096 3HD2 LEU A  69      -2.567 -11.268  20.378  1.00  0.00           H  \\nATOM   1097  N   ASP A  70       2.771 -11.995  19.968  1.00  0.00           N  \\nATOM   1098  CA  ASP A  70       3.773 -11.817  21.042  1.00  0.00           C  \\nATOM   1099  C   ASP A  70       3.003 -11.280  22.258  1.00  0.00           C  \\nATOM   1100  O   ASP A  70       2.334 -10.282  22.183  1.00  0.00           O  \\nATOM   1101  CB  ASP A  70       4.801 -10.823  20.457  1.00  0.00           C  \\nATOM   1102  CG  ASP A  70       6.067 -10.634  21.310  1.00  0.00           C  \\nATOM   1103  OD1 ASP A  70       5.923 -10.752  22.538  1.00  0.00           O  \\nATOM   1104  OD2 ASP A  70       7.114 -10.358  20.663  1.00  0.00           O  \\nATOM   1105  H   ASP A  70       2.702 -11.299  19.239  1.00  0.00           H  \\nATOM   1106  HA  ASP A  70       4.231 -12.784  21.252  1.00  0.00           H  \\nATOM   1107 1HB  ASP A  70       5.114 -11.162  19.469  1.00  0.00           H  \\nATOM   1108 2HB  ASP A  70       4.333  -9.846  20.335  1.00  0.00           H  \\nATOM   1109  N   VAL A  71       3.135 -11.951  23.413  1.00  0.00           N  \\nATOM   1110  CA  VAL A  71       2.427 -11.516  24.642  1.00  0.00           C  \\nATOM   1111  C   VAL A  71       2.984 -10.166  25.136  1.00  0.00           C  \\nATOM   1112  O   VAL A  71       2.273  -9.491  25.856  1.00  0.00           O  \\nATOM   1113  CB  VAL A  71       2.474 -12.490  25.807  1.00  0.00           C  \\nATOM   1114  CG1 VAL A  71       1.886 -13.876  25.407  1.00  0.00           C  \\nATOM   1115  CG2 VAL A  71       3.943 -12.653  26.317  1.00  0.00           C  \\nATOM   1116  H   VAL A  71       3.727 -12.768  23.450  1.00  0.00           H  \\nATOM   1117  HA  VAL A  71       1.371 -11.380  24.404  1.00  0.00           H  \\nATOM   1118  HB  VAL A  71       1.852 -12.106  26.615  1.00  0.00           H  \\nATOM   1119 1HG1 VAL A  71       1.933 -14.552  26.262  1.00  0.00           H  \\nATOM   1120 2HG1 VAL A  71       0.848 -13.755  25.097  1.00  0.00           H  \\nATOM   1121 3HG1 VAL A  71       2.465 -14.293  24.584  1.00  0.00           H  \\nATOM   1122 1HG2 VAL A  71       3.963 -13.353  27.152  1.00  0.00           H  \\nATOM   1123 2HG2 VAL A  71       4.568 -13.034  25.509  1.00  0.00           H  \\nATOM   1124 3HG2 VAL A  71       4.324 -11.686  26.646  1.00  0.00           H  \\nATOM   1125  N   THR A  72       4.173  -9.790  24.664  1.00  0.00           N  \\nATOM   1126  CA  THR A  72       4.751  -8.516  25.170  1.00  0.00           C  \\nATOM   1127  C   THR A  72       4.117  -7.281  24.543  1.00  0.00           C  \\nATOM   1128  O   THR A  72       4.216  -6.174  25.081  1.00  0.00           O  \\nATOM   1129  CB  THR A  72       6.279  -8.478  25.009  1.00  0.00           C  \\nATOM   1130  OG1 THR A  72       6.654  -8.436  23.670  1.00  0.00           O  \\nATOM   1131  CG2 THR A  72       6.951  -9.653  25.730  1.00  0.00           C  \\nATOM   1132  H   THR A  72       4.690 -10.329  23.985  1.00  0.00           H  \\nATOM   1133  HA  THR A  72       4.522  -8.430  26.233  1.00  0.00           H  \\nATOM   1134  HB  THR A  72       6.666  -7.548  25.424  1.00  0.00           H  \\nATOM   1135  HG1 THR A  72       5.868  -8.440  23.118  1.00  0.00           H  \\nATOM   1136 1HG2 THR A  72       8.031  -9.593  25.595  1.00  0.00           H  \\nATOM   1137 2HG2 THR A  72       6.715  -9.610  26.793  1.00  0.00           H  \\nATOM   1138 3HG2 THR A  72       6.585 -10.591  25.315  1.00  0.00           H  \\nATOM   1139  N   ASN A  73       3.514  -7.430  23.374  1.00  0.00           N  \\nATOM   1140  CA  ASN A  73       2.913  -6.259  22.729  1.00  0.00           C  \\nATOM   1141  C   ASN A  73       1.599  -6.568  22.079  1.00  0.00           C  \\nATOM   1142  O   ASN A  73       1.023  -5.709  21.431  1.00  0.00           O  \\nATOM   1143  CB  ASN A  73       3.907  -5.624  21.733  1.00  0.00           C  \\nATOM   1144  CG  ASN A  73       4.337  -6.584  20.683  1.00  0.00           C  \\nATOM   1145  OD1 ASN A  73       3.737  -7.631  20.448  1.00  0.00           O  \\nATOM   1146  ND2 ASN A  73       5.356  -6.163  19.914  1.00  0.00           N  \\nATOM   1147  H   ASN A  73       3.457  -8.328  22.915  1.00  0.00           H  \\nATOM   1148  HA  ASN A  73       2.673  -5.525  23.499  1.00  0.00           H  \\nATOM   1149 1HB  ASN A  73       3.443  -4.759  21.257  1.00  0.00           H  \\nATOM   1150 2HB  ASN A  73       4.785  -5.269  22.272  1.00  0.00           H  \\nATOM   1151 1HD2 ASN A  73       5.707  -6.749  19.183  1.00  0.00           H  \\nATOM   1152 2HD2 ASN A  73       5.763  -5.264  20.072  1.00  0.00           H  \\nATOM   1153  N   ALA A  74       1.090  -7.782  22.253  1.00  0.00           N  \\nATOM   1154  CA  ALA A  74      -0.143  -8.274  21.622  1.00  0.00           C  \\nATOM   1155  C   ALA A  74      -0.159  -8.300  20.110  1.00  0.00           C  \\nATOM   1156  O   ALA A  74      -1.216  -8.488  19.537  1.00  0.00           O  \\nATOM   1157  CB  ALA A  74      -1.382  -7.484  22.142  1.00  0.00           C  \\nATOM   1158  H   ALA A  74       1.606  -8.394  22.869  1.00  0.00           H  \\nATOM   1159  HA  ALA A  74      -0.262  -9.325  21.886  1.00  0.00           H  \\nATOM   1160 1HB  ALA A  74      -2.284  -7.864  21.662  1.00  0.00           H  \\nATOM   1161 2HB  ALA A  74      -1.466  -7.608  23.222  1.00  0.00           H  \\nATOM   1162 3HB  ALA A  74      -1.264  -6.427  21.907  1.00  0.00           H  \\nATOM   1163  N   TYR A  75       0.991  -8.201  19.477  1.00  0.00           N  \\nATOM   1164  CA  TYR A  75       1.019  -7.972  18.016  1.00  0.00           C  \\nATOM   1165  C   TYR A  75       1.306  -9.288  17.267  1.00  0.00           C  \\nATOM   1166  O   TYR A  75       1.927 -10.189  17.829  1.00  0.00           O  \\nATOM   1167  CB  TYR A  75       2.059  -6.898  17.648  1.00  0.00           C  \\nATOM   1168  CG  TYR A  75       1.810  -6.181  16.343  1.00  0.00           C  \\nATOM   1169  CD1 TYR A  75       0.954  -5.088  16.263  1.00  0.00           C  \\nATOM   1170  CD2 TYR A  75       2.500  -6.535  15.169  1.00  0.00           C  \\nATOM   1171  CE1 TYR A  75       0.789  -4.368  15.078  1.00  0.00           C  \\nATOM   1172  CE2 TYR A  75       2.342  -5.834  13.989  1.00  0.00           C  \\nATOM   1173  CZ  TYR A  75       1.439  -4.763  13.944  1.00  0.00           C  \\nATOM   1174  OH  TYR A  75       1.307  -4.041  12.776  1.00  0.00           O  \\nATOM   1175  H   TYR A  75       1.864  -8.280  19.979  1.00  0.00           H  \\nATOM   1176  HA  TYR A  75       0.035  -7.622  17.702  1.00  0.00           H  \\nATOM   1177 1HB  TYR A  75       2.096  -6.142  18.434  1.00  0.00           H  \\nATOM   1178 2HB  TYR A  75       3.047  -7.354  17.586  1.00  0.00           H  \\nATOM   1179  HD1 TYR A  75       0.389  -4.772  17.141  1.00  0.00           H  \\nATOM   1180  HD2 TYR A  75       3.182  -7.385  15.173  1.00  0.00           H  \\nATOM   1181  HE1 TYR A  75       0.110  -3.516  15.048  1.00  0.00           H  \\nATOM   1182  HE2 TYR A  75       2.916  -6.117  13.106  1.00  0.00           H  \\nATOM   1183  HH  TYR A  75       1.897  -4.402  12.110  1.00  0.00           H  \\nATOM   1184  N   VAL A  76       0.866  -9.391  16.020  1.00  0.00           N  \\nATOM   1185  CA  VAL A  76       1.082 -10.616  15.204  1.00  0.00           C  \\nATOM   1186  C   VAL A  76       2.523 -10.653  14.723  1.00  0.00           C  \\nATOM   1187  O   VAL A  76       3.011  -9.697  14.161  1.00  0.00           O  \\nATOM   1188  CB  VAL A  76       0.122 -10.500  14.000  1.00  0.00           C  \\nATOM   1189  CG1 VAL A  76       0.372 -11.674  12.973  1.00  0.00           C  \\nATOM   1190  CG2 VAL A  76      -1.304 -10.580  14.428  1.00  0.00           C  \\nATOM   1191  H   VAL A  76       0.369  -8.611  15.614  1.00  0.00           H  \\nATOM   1192  HA  VAL A  76       0.834 -11.486  15.813  1.00  0.00           H  \\nATOM   1193  HB  VAL A  76       0.286  -9.541  13.507  1.00  0.00           H  \\nATOM   1194 1HG1 VAL A  76      -0.314 -11.574  12.132  1.00  0.00           H  \\nATOM   1195 2HG1 VAL A  76       1.399 -11.628  12.610  1.00  0.00           H  \\nATOM   1196 3HG1 VAL A  76       0.203 -12.631  13.466  1.00  0.00           H  \\nATOM   1197 1HG2 VAL A  76      -1.951 -10.494  13.555  1.00  0.00           H  \\nATOM   1198 2HG2 VAL A  76      -1.483 -11.536  14.920  1.00  0.00           H  \\nATOM   1199 3HG2 VAL A  76      -1.522  -9.768  15.122  1.00  0.00           H  \\nATOM   1200  N   VAL A  77       3.199 -11.790  14.958  1.00  0.00           N  \\nATOM   1201  CA  VAL A  77       4.581 -12.023  14.462  1.00  0.00           C  \\nATOM   1202  C   VAL A  77       4.666 -13.001  13.323  1.00  0.00           C  \\nATOM   1203  O   VAL A  77       5.631 -13.001  12.595  1.00  0.00           O  \\nATOM   1204  CB  VAL A  77       5.539 -12.478  15.539  1.00  0.00           C  \\nATOM   1205  CG1 VAL A  77       5.599 -11.427  16.643  1.00  0.00           C  \\nATOM   1206  CG2 VAL A  77       5.299 -13.844  16.175  1.00  0.00           C  \\nATOM   1207  H   VAL A  77       2.741 -12.512  15.496  1.00  0.00           H  \\nATOM   1208  HA  VAL A  77       4.968 -11.085  14.062  1.00  0.00           H  \\nATOM   1209  HB  VAL A  77       6.545 -12.525  15.122  1.00  0.00           H  \\nATOM   1210 1HG1 VAL A  77       6.290 -11.754  17.420  1.00  0.00           H  \\nATOM   1211 2HG1 VAL A  77       5.944 -10.481  16.226  1.00  0.00           H  \\nATOM   1212 3HG1 VAL A  77       4.607 -11.295  17.074  1.00  0.00           H  \\nATOM   1213 1HG2 VAL A  77       6.064 -14.034  16.928  1.00  0.00           H  \\nATOM   1214 2HG2 VAL A  77       4.315 -13.859  16.645  1.00  0.00           H  \\nATOM   1215 3HG2 VAL A  77       5.346 -14.616  15.407  1.00  0.00           H  \\nATOM   1216  N   GLY A  78       3.618 -13.693  13.091  1.00  0.00           N  \\nATOM   1217  CA  GLY A  78       3.629 -14.715  11.968  1.00  0.00           C  \\nATOM   1218  C   GLY A  78       2.377 -15.519  12.000  1.00  0.00           C  \\nATOM   1219  O   GLY A  78       1.488 -15.284  12.801  1.00  0.00           O  \\nATOM   1220  H   GLY A  78       2.779 -13.575  13.640  1.00  0.00           H  \\nATOM   1221 1HA  GLY A  78       3.726 -14.203  11.011  1.00  0.00           H  \\nATOM   1222 2HA  GLY A  78       4.498 -15.363  12.075  1.00  0.00           H  \\nATOM   1223  N   TYR A  79       2.317 -16.487  11.064  1.00  0.00           N  \\nATOM   1224  CA  TYR A  79       1.168 -17.363  11.027  1.00  0.00           C  \\nATOM   1225  C   TYR A  79       1.486 -18.684  10.382  1.00  0.00           C  \\nATOM   1226  O   TYR A  79       2.454 -18.778   9.646  1.00  0.00           O  \\nATOM   1227  CB  TYR A  79      -0.012 -16.701  10.306  1.00  0.00           C  \\nATOM   1228  CG  TYR A  79       0.020 -16.754   8.771  1.00  0.00           C  \\nATOM   1229  CD1 TYR A  79       0.788 -15.852   8.050  1.00  0.00           C  \\nATOM   1230  CD2 TYR A  79      -0.770 -17.659   8.102  1.00  0.00           C  \\nATOM   1231  CE1 TYR A  79       0.762 -15.833   6.680  1.00  0.00           C  \\nATOM   1232  CE2 TYR A  79      -0.810 -17.613   6.686  1.00  0.00           C  \\nATOM   1233  CZ  TYR A  79      -0.030 -16.689   6.015  1.00  0.00           C  \\nATOM   1234  OH  TYR A  79      -0.058 -16.633   4.648  1.00  0.00           O  \\nATOM   1235  H   TYR A  79       3.056 -16.616  10.387  1.00  0.00           H  \\nATOM   1236  HA  TYR A  79       0.865 -17.579  12.052  1.00  0.00           H  \\nATOM   1237 1HB  TYR A  79      -0.943 -17.173  10.622  1.00  0.00           H  \\nATOM   1238 2HB  TYR A  79      -0.067 -15.650  10.587  1.00  0.00           H  \\nATOM   1239  HD1 TYR A  79       1.428 -15.141   8.573  1.00  0.00           H  \\nATOM   1240  HD2 TYR A  79      -1.351 -18.393   8.661  1.00  0.00           H  \\nATOM   1241  HE1 TYR A  79       1.374 -15.116   6.133  1.00  0.00           H  \\nATOM   1242  HE2 TYR A  79      -1.452 -18.301   6.136  1.00  0.00           H  \\nATOM   1243  HH  TYR A  79      -0.665 -17.298   4.314  1.00  0.00           H  \\nATOM   1244  N   ARG A  80       0.609 -19.630  10.652  1.00  0.00           N  \\nATOM   1245  CA  ARG A  80       0.679 -20.964   9.996  1.00  0.00           C  \\nATOM   1246  C   ARG A  80      -0.553 -21.193   9.131  1.00  0.00           C  \\nATOM   1247  O   ARG A  80      -1.666 -20.916   9.520  1.00  0.00           O  \\nATOM   1248  CB  ARG A  80       0.748 -22.002  11.122  1.00  0.00           C  \\nATOM   1249  CG  ARG A  80       0.773 -23.414  10.536  1.00  0.00           C  \\nATOM   1250  CD  ARG A  80       0.875 -24.518  11.578  1.00  0.00           C  \\nATOM   1251  NE  ARG A  80      -0.333 -24.699  12.340  1.00  0.00           N  \\nATOM   1252  CZ  ARG A  80      -1.436 -25.377  11.924  1.00  0.00           C  \\nATOM   1253  NH1 ARG A  80      -1.503 -25.971  10.709  1.00  0.00           N  \\nATOM   1254  NH2 ARG A  80      -2.499 -25.496  12.743  1.00  0.00           N  \\nATOM   1255  H   ARG A  80      -0.128 -19.448  11.319  1.00  0.00           H  \\nATOM   1256  HA  ARG A  80       1.582 -21.002   9.385  1.00  0.00           H  \\nATOM   1257 1HB  ARG A  80       1.641 -21.830  11.721  1.00  0.00           H  \\nATOM   1258 2HB  ARG A  80      -0.114 -21.884  11.779  1.00  0.00           H  \\nATOM   1259 1HG  ARG A  80      -0.143 -23.590   9.972  1.00  0.00           H  \\nATOM   1260 2HG  ARG A  80       1.633 -23.518   9.874  1.00  0.00           H  \\nATOM   1261 1HD  ARG A  80       1.095 -25.464  11.085  1.00  0.00           H  \\nATOM   1262 2HD  ARG A  80       1.673 -24.282  12.281  1.00  0.00           H  \\nATOM   1263  HE  ARG A  80      -0.372 -24.290  13.264  1.00  0.00           H  \\nATOM   1264 1HH1 ARG A  80      -0.716 -25.919  10.077  1.00  0.00           H  \\nATOM   1265 2HH1 ARG A  80      -2.339 -26.466  10.434  1.00  0.00           H  \\nATOM   1266 1HH2 ARG A  80      -2.476 -25.082  13.665  1.00  0.00           H  \\nATOM   1267 2HH2 ARG A  80      -3.318 -25.999  12.436  1.00  0.00           H  \\nATOM   1268  N   ALA A  81      -0.326 -21.770   7.946  1.00  0.00           N  \\nATOM   1269  CA  ALA A  81      -1.419 -22.263   7.132  1.00  0.00           C  \\nATOM   1270  C   ALA A  81      -0.956 -23.637   6.680  1.00  0.00           C  \\nATOM   1271  O   ALA A  81       0.013 -23.734   5.973  1.00  0.00           O  \\nATOM   1272  CB  ALA A  81      -1.677 -21.329   5.978  1.00  0.00           C  \\nATOM   1273  H   ALA A  81       0.621 -21.866   7.608  1.00  0.00           H  \\nATOM   1274  HA  ALA A  81      -2.312 -22.308   7.756  1.00  0.00           H  \\nATOM   1275 1HB  ALA A  81      -2.501 -21.713   5.376  1.00  0.00           H  \\nATOM   1276 2HB  ALA A  81      -1.936 -20.342   6.360  1.00  0.00           H  \\nATOM   1277 3HB  ALA A  81      -0.782 -21.256   5.363  1.00  0.00           H  \\nATOM   1278  N   GLY A  82      -1.612 -24.631   7.223  1.00  0.00           N  \\nATOM   1279  CA  GLY A  82      -1.390 -26.048   6.803  1.00  0.00           C  \\nATOM   1280  C   GLY A  82       0.015 -26.362   7.209  1.00  0.00           C  \\nATOM   1281  O   GLY A  82       0.389 -26.281   8.382  1.00  0.00           O  \\nATOM   1282  H   GLY A  82      -2.288 -24.433   7.947  1.00  0.00           H  \\nATOM   1283 1HA  GLY A  82      -2.123 -26.691   7.290  1.00  0.00           H  \\nATOM   1284 2HA  GLY A  82      -1.548 -26.138   5.729  1.00  0.00           H  \\nATOM   1285  N   ASN A  83       0.845 -26.739   6.205  1.00  0.00           N  \\nATOM   1286  CA  ASN A  83       2.217 -27.099   6.470  1.00  0.00           C  \\nATOM   1287  C   ASN A  83       3.222 -26.048   6.150  1.00  0.00           C  \\nATOM   1288  O   ASN A  83       4.411 -26.316   5.869  1.00  0.00           O  \\nATOM   1289  CB  ASN A  83       2.544 -28.430   5.705  1.00  0.00           C  \\nATOM   1290  CG  ASN A  83       2.803 -29.489   6.706  1.00  0.00           C  \\nATOM   1291  OD1 ASN A  83       1.918 -30.248   7.062  1.00  0.00           O  \\nATOM   1292  ND2 ASN A  83       3.973 -29.404   7.334  1.00  0.00           N  \\nATOM   1293  H   ASN A  83       0.503 -26.770   5.255  1.00  0.00           H  \\nATOM   1294  HA  ASN A  83       2.332 -27.258   7.543  1.00  0.00           H  \\nATOM   1295 1HB  ASN A  83       1.704 -28.696   5.062  1.00  0.00           H  \\nATOM   1296 2HB  ASN A  83       3.412 -28.277   5.063  1.00  0.00           H  \\nATOM   1297 1HD2 ASN A  83       4.220 -30.084   8.025  1.00  0.00           H  \\nATOM   1298 2HD2 ASN A  83       4.605 -28.661   7.115  1.00  0.00           H  \\nATOM   1299  N   SER A  84       2.797 -24.794   6.254  1.00  0.00           N  \\nATOM   1300  CA  SER A  84       3.713 -23.703   5.988  1.00  0.00           C  \\nATOM   1301  C   SER A  84       3.577 -22.681   7.081  1.00  0.00           C  \\nATOM   1302  O   SER A  84       2.499 -22.462   7.581  1.00  0.00           O  \\nATOM   1303  CB  SER A  84       3.414 -23.076   4.657  1.00  0.00           C  \\nATOM   1304  OG  SER A  84       3.684 -24.095   3.618  1.00  0.00           O  \\nATOM   1305  H   SER A  84       1.844 -24.584   6.516  1.00  0.00           H  \\nATOM   1306  HA  SER A  84       4.728 -24.101   5.965  1.00  0.00           H  \\nATOM   1307 1HB  SER A  84       2.375 -22.748   4.634  1.00  0.00           H  \\nATOM   1308 2HB  SER A  84       4.038 -22.194   4.522  1.00  0.00           H  \\nATOM   1309  HG  SER A  84       3.978 -24.880   4.087  1.00  0.00           H  \\nATOM   1310  N   ALA A  85       4.689 -22.107   7.477  1.00  0.00           N  \\nATOM   1311  CA  ALA A  85       4.572 -20.960   8.462  1.00  0.00           C  \\nATOM   1312  C   ALA A  85       5.377 -19.791   7.892  1.00  0.00           C  \\nATOM   1313  O   ALA A  85       6.404 -19.926   7.141  1.00  0.00           O  \\nATOM   1314  CB  ALA A  85       5.104 -21.369   9.801  1.00  0.00           C  \\nATOM   1315  H   ALA A  85       5.605 -22.391   7.159  1.00  0.00           H  \\nATOM   1316  HA  ALA A  85       3.517 -20.707   8.563  1.00  0.00           H  \\nATOM   1317 1HB  ALA A  85       5.014 -20.536  10.498  1.00  0.00           H  \\nATOM   1318 2HB  ALA A  85       4.533 -22.219  10.175  1.00  0.00           H  \\nATOM   1319 3HB  ALA A  85       6.152 -21.650   9.706  1.00  0.00           H  \\nATOM   1320  N   TYR A  86       4.899 -18.570   8.200  1.00  0.00           N  \\nATOM   1321  CA  TYR A  86       5.390 -17.320   7.643  1.00  0.00           C  \\nATOM   1322  C   TYR A  86       5.561 -16.284   8.752  1.00  0.00           C  \\nATOM   1323  O   TYR A  86       4.619 -16.069   9.557  1.00  0.00           O  \\nATOM   1324  CB  TYR A  86       4.426 -16.722   6.587  1.00  0.00           C  \\nATOM   1325  CG  TYR A  86       4.094 -17.702   5.518  1.00  0.00           C  \\nATOM   1326  CD1 TYR A  86       2.986 -18.583   5.637  1.00  0.00           C  \\nATOM   1327  CD2 TYR A  86       4.886 -17.765   4.402  1.00  0.00           C  \\nATOM   1328  CE1 TYR A  86       2.680 -19.526   4.644  1.00  0.00           C  \\nATOM   1329  CE2 TYR A  86       4.579 -18.751   3.385  1.00  0.00           C  \\nATOM   1330  CZ  TYR A  86       3.537 -19.586   3.574  1.00  0.00           C  \\nATOM   1331  OH  TYR A  86       3.231 -20.531   2.572  1.00  0.00           O  \\nATOM   1332  H   TYR A  86       4.144 -18.547   8.870  1.00  0.00           H  \\nATOM   1333  HA  TYR A  86       6.343 -17.512   7.150  1.00  0.00           H  \\nATOM   1334 1HB  TYR A  86       3.505 -16.400   7.075  1.00  0.00           H  \\nATOM   1335 2HB  TYR A  86       4.882 -15.842   6.135  1.00  0.00           H  \\nATOM   1336  HD1 TYR A  86       2.346 -18.538   6.518  1.00  0.00           H  \\nATOM   1337  HD2 TYR A  86       5.729 -17.082   4.292  1.00  0.00           H  \\nATOM   1338  HE1 TYR A  86       1.823 -20.187   4.766  1.00  0.00           H  \\nATOM   1339  HE2 TYR A  86       5.188 -18.811   2.482  1.00  0.00           H  \\nATOM   1340  HH  TYR A  86       3.853 -20.439   1.846  1.00  0.00           H  \\nATOM   1341  N   PHE A  87       6.721 -15.649   8.789  1.00  0.00           N  \\nATOM   1342  CA  PHE A  87       7.053 -14.720   9.916  1.00  0.00           C  \\nATOM   1343  C   PHE A  87       7.441 -13.377   9.341  1.00  0.00           C  \\nATOM   1344  O   PHE A  87       8.157 -13.304   8.325  1.00  0.00           O  \\nATOM   1345  CB  PHE A  87       8.183 -15.289  10.738  1.00  0.00           C  \\nATOM   1346  CG  PHE A  87       7.860 -16.595  11.435  1.00  0.00           C  \\nATOM   1347  CD1 PHE A  87       7.177 -16.683  12.597  1.00  0.00           C  \\nATOM   1348  CD2 PHE A  87       8.211 -17.808  10.783  1.00  0.00           C  \\nATOM   1349  CE1 PHE A  87       6.933 -17.941  13.139  1.00  0.00           C  \\nATOM   1350  CE2 PHE A  87       7.928 -19.006  11.354  1.00  0.00           C  \\nATOM   1351  CZ  PHE A  87       7.271 -19.102  12.440  1.00  0.00           C  \\nATOM   1352  H   PHE A  87       7.396 -15.788   8.051  1.00  0.00           H  \\nATOM   1353  HA  PHE A  87       6.172 -14.617  10.551  1.00  0.00           H  \\nATOM   1354 1HB  PHE A  87       9.048 -15.459  10.098  1.00  0.00           H  \\nATOM   1355 2HB  PHE A  87       8.476 -14.569  11.501  1.00  0.00           H  \\nATOM   1356  HD1 PHE A  87       6.833 -15.770  13.083  1.00  0.00           H  \\nATOM   1357  HD2 PHE A  87       8.777 -17.768   9.852  1.00  0.00           H  \\nATOM   1358  HE1 PHE A  87       6.472 -18.030  14.122  1.00  0.00           H  \\nATOM   1359  HE2 PHE A  87       8.284 -19.905  10.852  1.00  0.00           H  \\nATOM   1360  HZ  PHE A  87       6.975 -20.077  12.825  1.00  0.00           H  \\nATOM   1361  N   PHE A  88       6.948 -12.274   9.936  1.00  0.00           N  \\nATOM   1362  CA  PHE A  88       7.534 -10.993   9.606  1.00  0.00           C  \\nATOM   1363  C   PHE A  88       8.976 -10.937   9.939  1.00  0.00           C  \\nATOM   1364  O   PHE A  88       9.486 -11.617  10.826  1.00  0.00           O  \\nATOM   1365  CB  PHE A  88       6.846  -9.821  10.300  1.00  0.00           C  \\nATOM   1366  CG  PHE A  88       5.418  -9.685   9.998  1.00  0.00           C  \\nATOM   1367  CD1 PHE A  88       4.997  -9.424   8.698  1.00  0.00           C  \\nATOM   1368  CD2 PHE A  88       4.418  -9.815  11.035  1.00  0.00           C  \\nATOM   1369  CE1 PHE A  88       3.663  -9.291   8.412  1.00  0.00           C  \\nATOM   1370  CE2 PHE A  88       3.049  -9.647  10.738  1.00  0.00           C  \\nATOM   1371  CZ  PHE A  88       2.656  -9.367   9.453  1.00  0.00           C  \\nATOM   1372  H   PHE A  88       6.188 -12.311  10.600  1.00  0.00           H  \\nATOM   1373  HA  PHE A  88       7.438 -10.835   8.531  1.00  0.00           H  \\nATOM   1374 1HB  PHE A  88       6.948  -9.927  11.379  1.00  0.00           H  \\nATOM   1375 2HB  PHE A  88       7.336  -8.891  10.014  1.00  0.00           H  \\nATOM   1376  HD1 PHE A  88       5.741  -9.327   7.907  1.00  0.00           H  \\nATOM   1377  HD2 PHE A  88       4.731 -10.037  12.056  1.00  0.00           H  \\nATOM   1378  HE1 PHE A  88       3.341  -9.125   7.384  1.00  0.00           H  \\nATOM   1379  HE2 PHE A  88       2.303  -9.738  11.528  1.00  0.00           H  \\nATOM   1380  HZ  PHE A  88       1.607  -9.202   9.211  1.00  0.00           H  \\nATOM   1381  N   HIS A  89       9.676 -10.077   9.184  1.00  0.00           N  \\nATOM   1382  CA  HIS A  89      11.123  -9.910   9.476  1.00  0.00           C  \\nATOM   1383  C   HIS A  89      11.405  -9.362  10.848  1.00  0.00           C  \\nATOM   1384  O   HIS A  89      11.063  -8.177  11.128  1.00  0.00           O  \\nATOM   1385  CB  HIS A  89      11.731  -8.955   8.447  1.00  0.00           C  \\nATOM   1386  CG  HIS A  89      13.248  -8.977   8.391  1.00  0.00           C  \\nATOM   1387  ND1 HIS A  89      14.021  -7.887   8.717  1.00  0.00           N  \\nATOM   1388  CD2 HIS A  89      14.116  -9.939   7.980  1.00  0.00           C  \\nATOM   1389  CE1 HIS A  89      15.302  -8.191   8.562  1.00  0.00           C  \\nATOM   1390  NE2 HIS A  89      15.387  -9.433   8.127  1.00  0.00           N  \\nATOM   1391  H   HIS A  89       9.261  -9.545   8.433  1.00  0.00           H  \\nATOM   1392  HA  HIS A  89      11.623 -10.875   9.400  1.00  0.00           H  \\nATOM   1393 1HB  HIS A  89      11.356  -9.203   7.454  1.00  0.00           H  \\nATOM   1394 2HB  HIS A  89      11.419  -7.935   8.669  1.00  0.00           H  \\nATOM   1395  HD2 HIS A  89      13.854 -10.940   7.637  1.00  0.00           H  \\nATOM   1396  HE1 HIS A  89      16.143  -7.527   8.761  1.00  0.00           H  \\nATOM   1397  HE2 HIS A  89      16.242  -9.934   7.933  1.00  0.00           H  \\nATOM   1398  N   PRO A  90      12.089 -10.185  11.652  1.00  0.00           N  \\nATOM   1399  CA  PRO A  90      12.365  -9.738  13.010  1.00  0.00           C  \\nATOM   1400  C   PRO A  90      13.248  -8.455  13.042  1.00  0.00           C  \\nATOM   1401  O   PRO A  90      14.143  -8.281  12.205  1.00  0.00           O  \\nATOM   1402  CB  PRO A  90      13.151 -10.895  13.622  1.00  0.00           C  \\nATOM   1403  CG  PRO A  90      13.020 -11.996  12.720  1.00  0.00           C  \\nATOM   1404  CD  PRO A  90      12.730 -11.474  11.347  1.00  0.00           C  \\nATOM   1405  HA  PRO A  90      11.414  -9.591  13.543  1.00  0.00           H  \\nATOM   1406 1HB  PRO A  90      14.203 -10.604  13.759  1.00  0.00           H  \\nATOM   1407 2HB  PRO A  90      12.752 -11.136  14.618  1.00  0.00           H  \\nATOM   1408 1HG  PRO A  90      13.944 -12.593  12.713  1.00  0.00           H  \\nATOM   1409 2HG  PRO A  90      12.212 -12.665  13.052  1.00  0.00           H  \\nATOM   1410 1HD  PRO A  90      13.673 -11.353  10.794  1.00  0.00           H  \\nATOM   1411 2HD  PRO A  90      12.059 -12.171  10.823  1.00  0.00           H  \\nATOM   1412  N   ASP A  91      13.011  -7.644  14.045  1.00  0.00           N  \\nATOM   1413  CA  ASP A  91      13.717  -6.424  14.329  1.00  0.00           C  \\nATOM   1414  C   ASP A  91      15.141  -6.664  14.830  1.00  0.00           C  \\nATOM   1415  O   ASP A  91      15.962  -5.772  14.653  1.00  0.00           O  \\nATOM   1416  CB  ASP A  91      12.915  -5.578  15.379  1.00  0.00           C  \\nATOM   1417  CG  ASP A  91      11.496  -4.944  14.842  1.00  0.00           C  \\nATOM   1418  OD1 ASP A  91      11.155  -4.827  13.620  1.00  0.00           O  \\nATOM   1419  OD2 ASP A  91      10.691  -4.478  15.704  1.00  0.00           O  \\nATOM   1420  H   ASP A  91      12.258  -7.930  14.655  1.00  0.00           H  \\nATOM   1421  HA  ASP A  91      13.804  -5.852  13.405  1.00  0.00           H  \\nATOM   1422 1HB  ASP A  91      12.685  -6.197  16.246  1.00  0.00           H  \\nATOM   1423 2HB  ASP A  91      13.531  -4.747  15.724  1.00  0.00           H  \\nATOM   1424  N   ASN A  92      15.435  -7.811  15.491  1.00  0.00           N  \\nATOM   1425  CA  ASN A  92      16.730  -8.144  16.088  1.00  0.00           C  \\nATOM   1426  C   ASN A  92      16.927  -9.666  16.311  1.00  0.00           C  \\nATOM   1427  O   ASN A  92      16.005 -10.477  16.152  1.00  0.00           O  \\nATOM   1428  CB  ASN A  92      16.945  -7.438  17.447  1.00  0.00           C  \\nATOM   1429  CG  ASN A  92      15.829  -7.755  18.467  1.00  0.00           C  \\nATOM   1430  OD1 ASN A  92      15.781  -8.838  19.048  1.00  0.00           O  \\nATOM   1431  ND2 ASN A  92      14.874  -6.830  18.616  1.00  0.00           N  \\nATOM   1432  H   ASN A  92      14.676  -8.473  15.562  1.00  0.00           H  \\nATOM   1433  HA  ASN A  92      17.519  -7.812  15.411  1.00  0.00           H  \\nATOM   1434 1HB  ASN A  92      17.903  -7.744  17.869  1.00  0.00           H  \\nATOM   1435 2HB  ASN A  92      16.984  -6.359  17.294  1.00  0.00           H  \\nATOM   1436 1HD2 ASN A  92      14.126  -6.983  19.263  1.00  0.00           H  \\nATOM   1437 2HD2 ASN A  92      14.907  -5.987  18.081  1.00  0.00           H  \\nATOM   1438  N   GLN A  93      18.146 -10.028  16.682  1.00  0.00           N  \\nATOM   1439  CA  GLN A  93      18.565 -11.424  16.820  1.00  0.00           C  \\nATOM   1440  C   GLN A  93      17.723 -12.164  17.847  1.00  0.00           C  \\nATOM   1441  O   GLN A  93      17.443 -13.377  17.699  1.00  0.00           O  \\nATOM   1442  CB  GLN A  93      20.061 -11.483  17.260  1.00  0.00           C  \\nATOM   1443  CG  GLN A  93      20.698 -12.864  17.151  1.00  0.00           C  \\nATOM   1444  CD  GLN A  93      20.389 -13.839  18.314  1.00  0.00           C  \\nATOM   1445  OE1 GLN A  93      20.203 -13.436  19.485  1.00  0.00           O  \\nATOM   1446  NE2 GLN A  93      20.362 -15.150  17.984  1.00  0.00           N  \\nATOM   1447  H   GLN A  93      18.811  -9.293  16.877  1.00  0.00           H  \\nATOM   1448  HA  GLN A  93      18.460 -11.914  15.852  1.00  0.00           H  \\nATOM   1449 1HB  GLN A  93      20.647 -10.796  16.649  1.00  0.00           H  \\nATOM   1450 2HB  GLN A  93      20.149 -11.156  18.296  1.00  0.00           H  \\nATOM   1451 1HG  GLN A  93      20.345 -13.345  16.239  1.00  0.00           H  \\nATOM   1452 2HG  GLN A  93      21.782 -12.751  17.118  1.00  0.00           H  \\nATOM   1453 1HE2 GLN A  93      20.169 -15.839  18.684  1.00  0.00           H  \\nATOM   1454 2HE2 GLN A  93      20.535 -15.432  17.040  1.00  0.00           H  \\nATOM   1455  N   GLU A  94      17.364 -11.480  18.927  1.00  0.00           N  \\nATOM   1456  CA  GLU A  94      16.705 -12.154  20.009  1.00  0.00           C  \\nATOM   1457  C   GLU A  94      15.274 -12.502  19.518  1.00  0.00           C  \\nATOM   1458  O   GLU A  94      14.745 -13.614  19.829  1.00  0.00           O  \\nATOM   1459  CB  GLU A  94      16.708 -11.267  21.275  1.00  0.00           C  \\nATOM   1460  CG  GLU A  94      17.980 -11.477  22.152  1.00  0.00           C  \\nATOM   1461  CD  GLU A  94      19.220 -10.798  21.524  1.00  0.00           C  \\nATOM   1462  OE1 GLU A  94      19.062  -9.623  21.045  1.00  0.00           O  \\nATOM   1463  OE2 GLU A  94      20.329 -11.416  21.470  1.00  0.00           O  \\nATOM   1464  H   GLU A  94      17.546 -10.489  19.003  1.00  0.00           H  \\nATOM   1465  HA  GLU A  94      17.248 -13.075  20.225  1.00  0.00           H  \\nATOM   1466 1HB  GLU A  94      16.651 -10.218  20.984  1.00  0.00           H  \\nATOM   1467 2HB  GLU A  94      15.826 -11.488  21.876  1.00  0.00           H  \\nATOM   1468 1HG  GLU A  94      17.799 -11.060  23.142  1.00  0.00           H  \\nATOM   1469 2HG  GLU A  94      18.158 -12.545  22.265  1.00  0.00           H  \\nATOM   1470  N   ASP A  95      14.637 -11.575  18.839  1.00  0.00           N  \\nATOM   1471  CA  ASP A  95      13.273 -11.857  18.305  1.00  0.00           C  \\nATOM   1472  C   ASP A  95      13.339 -12.897  17.203  1.00  0.00           C  \\nATOM   1473  O   ASP A  95      12.432 -13.746  17.086  1.00  0.00           O  \\nATOM   1474  CB  ASP A  95      12.665 -10.615  17.732  1.00  0.00           C  \\nATOM   1475  CG  ASP A  95      12.134  -9.624  18.854  1.00  0.00           C  \\nATOM   1476  OD1 ASP A  95      12.030  -9.976  20.073  1.00  0.00           O  \\nATOM   1477  OD2 ASP A  95      11.778  -8.488  18.434  1.00  0.00           O  \\nATOM   1478  H   ASP A  95      15.054 -10.671  18.670  1.00  0.00           H  \\nATOM   1479  HA  ASP A  95      12.645 -12.205  19.126  1.00  0.00           H  \\nATOM   1480 1HB  ASP A  95      13.405 -10.095  17.124  1.00  0.00           H  \\nATOM   1481 2HB  ASP A  95      11.835 -10.884  17.078  1.00  0.00           H  \\nATOM   1482  N   ALA A  96      14.407 -12.867  16.384  1.00  0.00           N  \\nATOM   1483  CA  ALA A  96      14.683 -13.994  15.456  1.00  0.00           C  \\nATOM   1484  C   ALA A  96      14.801 -15.350  16.109  1.00  0.00           C  \\nATOM   1485  O   ALA A  96      14.225 -16.345  15.599  1.00  0.00           O  \\nATOM   1486  CB  ALA A  96      15.971 -13.723  14.623  1.00  0.00           C  \\nATOM   1487  H   ALA A  96      15.032 -12.074  16.394  1.00  0.00           H  \\nATOM   1488  HA  ALA A  96      13.838 -14.085  14.773  1.00  0.00           H  \\nATOM   1489 1HB  ALA A  96      16.153 -14.562  13.951  1.00  0.00           H  \\nATOM   1490 2HB  ALA A  96      15.842 -12.812  14.039  1.00  0.00           H  \\nATOM   1491 3HB  ALA A  96      16.820 -13.606  15.294  1.00  0.00           H  \\nATOM   1492  N   GLU A  97      15.547 -15.442  17.210  1.00  0.00           N  \\nATOM   1493  CA  GLU A  97      15.644 -16.691  17.955  1.00  0.00           C  \\nATOM   1494  C   GLU A  97      14.282 -17.086  18.542  1.00  0.00           C  \\nATOM   1495  O   GLU A  97      13.870 -18.289  18.506  1.00  0.00           O  \\nATOM   1496  CB  GLU A  97      16.705 -16.566  19.059  1.00  0.00           C  \\nATOM   1497  CG  GLU A  97      17.156 -17.943  19.609  1.00  0.00           C  \\nATOM   1498  CD  GLU A  97      16.252 -18.586  20.665  1.00  0.00           C  \\nATOM   1499  OE1 GLU A  97      15.534 -17.812  21.347  1.00  0.00           O  \\nATOM   1500  OE2 GLU A  97      16.290 -19.837  20.868  1.00  0.00           O  \\nATOM   1501  H   GLU A  97      16.056 -14.633  17.537  1.00  0.00           H  \\nATOM   1502  HA  GLU A  97      15.944 -17.482  17.267  1.00  0.00           H  \\nATOM   1503 1HB  GLU A  97      17.577 -16.040  18.668  1.00  0.00           H  \\nATOM   1504 2HB  GLU A  97      16.306 -15.972  19.881  1.00  0.00           H  \\nATOM   1505 1HG  GLU A  97      17.228 -18.647  18.781  1.00  0.00           H  \\nATOM   1506 2HG  GLU A  97      18.147 -17.840  20.049  1.00  0.00           H  \\nATOM   1507  N   ALA A  98      13.565 -16.085  19.087  1.00  0.00           N  \\nATOM   1508  CA  ALA A  98      12.305 -16.380  19.720  1.00  0.00           C  \\nATOM   1509  C   ALA A  98      11.346 -17.083  18.727  1.00  0.00           C  \\nATOM   1510  O   ALA A  98      10.720 -18.063  19.120  1.00  0.00           O  \\nATOM   1511  CB  ALA A  98      11.678 -15.107  20.272  1.00  0.00           C  \\nATOM   1512  H   ALA A  98      13.891 -15.129  19.062  1.00  0.00           H  \\nATOM   1513  HA  ALA A  98      12.493 -17.067  20.545  1.00  0.00           H  \\nATOM   1514 1HB  ALA A  98      10.727 -15.346  20.747  1.00  0.00           H  \\nATOM   1515 2HB  ALA A  98      12.348 -14.660  21.007  1.00  0.00           H  \\nATOM   1516 3HB  ALA A  98      11.510 -14.403  19.459  1.00  0.00           H  \\nATOM   1517  N   ILE A  99      11.189 -16.571  17.509  1.00  0.00           N  \\nATOM   1518  CA  ILE A  99      10.234 -17.154  16.551  1.00  0.00           C  \\nATOM   1519  C   ILE A  99      10.593 -18.540  16.056  1.00  0.00           C  \\nATOM   1520  O   ILE A  99       9.724 -19.234  15.426  1.00  0.00           O  \\nATOM   1521  CB  ILE A  99       9.852 -16.249  15.391  1.00  0.00           C  \\nATOM   1522  CG1 ILE A  99      11.043 -16.010  14.480  1.00  0.00           C  \\nATOM   1523  CG2 ILE A  99       9.336 -15.010  16.032  1.00  0.00           C  \\nATOM   1524  CD1 ILE A  99      10.758 -15.313  13.113  1.00  0.00           C  \\nATOM   1525  H   ILE A  99      11.734 -15.767  17.233  1.00  0.00           H  \\nATOM   1526  HA  ILE A  99       9.310 -17.382  17.080  1.00  0.00           H  \\nATOM   1527  HB  ILE A  99       9.090 -16.737  14.784  1.00  0.00           H  \\nATOM   1528 1HG1 ILE A  99      11.778 -15.394  14.997  1.00  0.00           H  \\nATOM   1529 2HG1 ILE A  99      11.521 -16.962  14.249  1.00  0.00           H  \\nATOM   1530 1HG2 ILE A  99       9.037 -14.299  15.262  1.00  0.00           H  \\nATOM   1531 2HG2 ILE A  99       8.476 -15.254  16.654  1.00  0.00           H  \\nATOM   1532 3HG2 ILE A  99      10.117 -14.568  16.650  1.00  0.00           H  \\nATOM   1533 1HD1 ILE A  99      11.691 -15.202  12.559  1.00  0.00           H  \\nATOM   1534 2HD1 ILE A  99      10.062 -15.920  12.533  1.00  0.00           H  \\nATOM   1535 3HD1 ILE A  99      10.323 -14.330  13.290  1.00  0.00           H  \\nATOM   1536  N   THR A 100      11.795 -19.019  16.450  1.00  0.00           N  \\nATOM   1537  CA  THR A 100      12.115 -20.362  16.025  1.00  0.00           C  \\nATOM   1538  C   THR A 100      11.364 -21.365  16.807  1.00  0.00           C  \\nATOM   1539  O   THR A 100      11.316 -22.546  16.400  1.00  0.00           O  \\nATOM   1540  CB  THR A 100      13.688 -20.723  16.077  1.00  0.00           C  \\nATOM   1541  OG1 THR A 100      14.094 -20.805  17.450  1.00  0.00           O  \\nATOM   1542  CG2 THR A 100      14.471 -19.675  15.367  1.00  0.00           C  \\nATOM   1543  H   THR A 100      12.463 -18.507  17.008  1.00  0.00           H  \\nATOM   1544  HA  THR A 100      11.801 -20.481  14.988  1.00  0.00           H  \\nATOM   1545  HB  THR A 100      13.856 -21.688  15.600  1.00  0.00           H  \\nATOM   1546  HG1 THR A 100      13.340 -20.627  18.017  1.00  0.00           H  \\nATOM   1547 1HG2 THR A 100      15.531 -19.924  15.405  1.00  0.00           H  \\nATOM   1548 2HG2 THR A 100      14.148 -19.621  14.328  1.00  0.00           H  \\nATOM   1549 3HG2 THR A 100      14.308 -18.711  15.848  1.00  0.00           H  \\nATOM   1550  N   HIS A 101      10.702 -20.931  17.864  1.00  0.00           N  \\nATOM   1551  CA  HIS A 101       9.994 -21.768  18.814  1.00  0.00           C  \\nATOM   1552  C   HIS A 101       8.528 -21.911  18.435  1.00  0.00           C  \\nATOM   1553  O   HIS A 101       7.775 -22.594  19.100  1.00  0.00           O  \\nATOM   1554  CB  HIS A 101      10.160 -21.211  20.217  1.00  0.00           C  \\nATOM   1555  CG  HIS A 101      11.597 -21.239  20.702  1.00  0.00           C  \\nATOM   1556  ND1 HIS A 101      12.106 -22.277  21.461  1.00  0.00           N  \\nATOM   1557  CD2 HIS A 101      12.634 -20.398  20.464  1.00  0.00           C  \\nATOM   1558  CE1 HIS A 101      13.386 -22.046  21.711  1.00  0.00           C  \\nATOM   1559  NE2 HIS A 101      13.741 -20.932  21.099  1.00  0.00           N  \\nATOM   1560  H   HIS A 101      10.706 -19.930  17.997  1.00  0.00           H  \\nATOM   1561  HA  HIS A 101      10.409 -22.775  18.792  1.00  0.00           H  \\nATOM   1562 1HB  HIS A 101       9.805 -20.180  20.244  1.00  0.00           H  \\nATOM   1563 2HB  HIS A 101       9.548 -21.786  20.911  1.00  0.00           H  \\nATOM   1564  HD2 HIS A 101      12.606 -19.481  19.875  1.00  0.00           H  \\nATOM   1565  HE1 HIS A 101      14.039 -22.670  22.320  1.00  0.00           H  \\nATOM   1566  HE2 HIS A 101      14.668 -20.531  21.092  1.00  0.00           H  \\nATOM   1567  N   LEU A 102       8.112 -21.199  17.388  1.00  0.00           N  \\nATOM   1568  CA  LEU A 102       6.744 -21.293  16.965  1.00  0.00           C  \\nATOM   1569  C   LEU A 102       6.565 -22.289  15.798  1.00  0.00           C  \\nATOM   1570  O   LEU A 102       7.367 -22.322  14.934  1.00  0.00           O  \\nATOM   1571  CB  LEU A 102       6.306 -19.914  16.407  1.00  0.00           C  \\nATOM   1572  CG  LEU A 102       5.986 -18.929  17.567  1.00  0.00           C  \\nATOM   1573  CD1 LEU A 102       5.990 -17.535  16.910  1.00  0.00           C  \\nATOM   1574  CD2 LEU A 102       4.716 -19.306  18.208  1.00  0.00           C  \\nATOM   1575  H   LEU A 102       8.742 -20.592  16.884  1.00  0.00           H  \\nATOM   1576  HA  LEU A 102       6.132 -21.549  17.829  1.00  0.00           H  \\nATOM   1577 1HB  LEU A 102       7.109 -19.519  15.787  1.00  0.00           H  \\nATOM   1578 2HB  LEU A 102       5.427 -20.057  15.779  1.00  0.00           H  \\nATOM   1579  HG  LEU A 102       6.786 -18.965  18.306  1.00  0.00           H  \\nATOM   1580 1HD1 LEU A 102       5.772 -16.777  17.663  1.00  0.00           H  \\nATOM   1581 2HD1 LEU A 102       6.970 -17.341  16.475  1.00  0.00           H  \\nATOM   1582 3HD1 LEU A 102       5.232 -17.499  16.129  1.00  0.00           H  \\nATOM   1583 1HD2 LEU A 102       4.498 -18.612  19.020  1.00  0.00           H  \\nATOM   1584 2HD2 LEU A 102       3.912 -19.268  17.473  1.00  0.00           H  \\nATOM   1585 3HD2 LEU A 102       4.796 -20.318  18.606  1.00  0.00           H  \\nATOM   1586  N   PHE A 103       5.468 -23.020  15.798  1.00  0.00           N  \\nATOM   1587  CA  PHE A 103       5.046 -23.848  14.647  1.00  0.00           C  \\nATOM   1588  C   PHE A 103       6.133 -24.837  14.235  1.00  0.00           C  \\nATOM   1589  O   PHE A 103       6.447 -24.926  13.054  1.00  0.00           O  \\nATOM   1590  CB  PHE A 103       4.734 -22.953  13.457  1.00  0.00           C  \\nATOM   1591  CG  PHE A 103       3.674 -21.943  13.755  1.00  0.00           C  \\nATOM   1592  CD1 PHE A 103       2.514 -22.302  14.466  1.00  0.00           C  \\nATOM   1593  CD2 PHE A 103       3.837 -20.659  13.354  1.00  0.00           C  \\nATOM   1594  CE1 PHE A 103       1.518 -21.321  14.741  1.00  0.00           C  \\nATOM   1595  CE2 PHE A 103       2.824 -19.696  13.688  1.00  0.00           C  \\nATOM   1596  CZ  PHE A 103       1.726 -20.055  14.343  1.00  0.00           C  \\nATOM   1597  H   PHE A 103       4.896 -23.006  16.630  1.00  0.00           H  \\nATOM   1598  HA  PHE A 103       4.144 -24.395  14.926  1.00  0.00           H  \\nATOM   1599 1HB  PHE A 103       5.638 -22.430  13.148  1.00  0.00           H  \\nATOM   1600 2HB  PHE A 103       4.407 -23.565  12.617  1.00  0.00           H  \\nATOM   1601  HD1 PHE A 103       2.392 -23.333  14.800  1.00  0.00           H  \\nATOM   1602  HD2 PHE A 103       4.732 -20.344  12.815  1.00  0.00           H  \\nATOM   1603  HE1 PHE A 103       0.606 -21.604  15.266  1.00  0.00           H  \\nATOM   1604  HE2 PHE A 103       2.967 -18.656  13.394  1.00  0.00           H  \\nATOM   1605  HZ  PHE A 103       0.983 -19.287  14.554  1.00  0.00           H  \\nATOM   1606  N   THR A 104       6.739 -25.464  15.234  1.00  0.00           N  \\nATOM   1607  CA  THR A 104       8.003 -26.191  14.974  1.00  0.00           C  \\nATOM   1608  C   THR A 104       7.813 -27.503  14.185  1.00  0.00           C  \\nATOM   1609  O   THR A 104       8.792 -28.038  13.634  1.00  0.00           O  \\nATOM   1610  CB  THR A 104       8.846 -26.413  16.236  1.00  0.00           C  \\nATOM   1611  OG1 THR A 104       8.119 -27.187  17.201  1.00  0.00           O  \\nATOM   1612  CG2 THR A 104       9.217 -25.088  16.872  1.00  0.00           C  \\nATOM   1613  H   THR A 104       6.359 -25.458  16.170  1.00  0.00           H  \\nATOM   1614  HA  THR A 104       8.605 -25.601  14.282  1.00  0.00           H  \\nATOM   1615  HB  THR A 104       9.757 -26.950  15.975  1.00  0.00           H  \\nATOM   1616  HG1 THR A 104       7.252 -27.403  16.849  1.00  0.00           H  \\nATOM   1617 1HG2 THR A 104       9.815 -25.268  17.765  1.00  0.00           H  \\nATOM   1618 2HG2 THR A 104       9.793 -24.493  16.163  1.00  0.00           H  \\nATOM   1619 3HG2 THR A 104       8.311 -24.549  17.145  1.00  0.00           H  \\nATOM   1620  N   ASP A 105       6.582 -27.990  14.089  1.00  0.00           N  \\nATOM   1621  CA  ASP A 105       6.359 -29.242  13.339  1.00  0.00           C  \\nATOM   1622  C   ASP A 105       6.033 -28.997  11.857  1.00  0.00           C  \\nATOM   1623  O   ASP A 105       5.775 -29.942  11.103  1.00  0.00           O  \\nATOM   1624  CB  ASP A 105       5.320 -30.145  14.060  1.00  0.00           C  \\nATOM   1625  CG  ASP A 105       4.110 -29.371  14.583  1.00  0.00           C  \\nATOM   1626  OD1 ASP A 105       4.067 -28.125  14.438  1.00  0.00           O  \\nATOM   1627  OD2 ASP A 105       3.192 -30.000  15.175  1.00  0.00           O  \\nATOM   1628  H   ASP A 105       5.794 -27.525  14.516  1.00  0.00           H  \\nATOM   1629  HA  ASP A 105       7.304 -29.783  13.281  1.00  0.00           H  \\nATOM   1630 1HB  ASP A 105       4.968 -30.915  13.373  1.00  0.00           H  \\nATOM   1631 2HB  ASP A 105       5.798 -30.650  14.900  1.00  0.00           H  \\nATOM   1632  N   VAL A 106       6.056 -27.735  11.420  1.00  0.00           N  \\nATOM   1633  CA  VAL A 106       5.718 -27.344  10.074  1.00  0.00           C  \\nATOM   1634  C   VAL A 106       6.890 -27.686   9.132  1.00  0.00           C  \\nATOM   1635  O   VAL A 106       8.047 -27.549   9.528  1.00  0.00           O  \\nATOM   1636  CB  VAL A 106       5.355 -25.779  10.109  1.00  0.00           C  \\nATOM   1637  CG1 VAL A 106       6.149 -24.960   9.115  1.00  0.00           C  \\nATOM   1638  CG2 VAL A 106       3.905 -25.588   9.998  1.00  0.00           C  \\nATOM   1639  H   VAL A 106       6.329 -27.025  12.085  1.00  0.00           H  \\nATOM   1640  HA  VAL A 106       4.852 -27.924   9.752  1.00  0.00           H  \\nATOM   1641  HB  VAL A 106       5.702 -25.356  11.052  1.00  0.00           H  \\nATOM   1642 1HG1 VAL A 106       5.856 -23.912   9.190  1.00  0.00           H  \\nATOM   1643 2HG1 VAL A 106       7.213 -25.055   9.332  1.00  0.00           H  \\nATOM   1644 3HG1 VAL A 106       5.950 -25.320   8.106  1.00  0.00           H  \\nATOM   1645 1HG2 VAL A 106       3.675 -24.523  10.024  1.00  0.00           H  \\nATOM   1646 2HG2 VAL A 106       3.553 -26.013   9.058  1.00  0.00           H  \\nATOM   1647 3HG2 VAL A 106       3.408 -26.087  10.830  1.00  0.00           H  \\nATOM   1648  N   GLN A 107       6.611 -28.067   7.886  1.00  0.00           N  \\nATOM   1649  CA  GLN A 107       7.687 -28.391   6.928  1.00  0.00           C  \\nATOM   1650  C   GLN A 107       8.297 -27.269   6.101  1.00  0.00           C  \\nATOM   1651  O   GLN A 107       9.331 -27.445   5.470  1.00  0.00           O  \\nATOM   1652  CB  GLN A 107       7.178 -29.429   5.951  1.00  0.00           C  \\nATOM   1653  CG  GLN A 107       6.607 -30.626   6.638  1.00  0.00           C  \\nATOM   1654  CD  GLN A 107       5.900 -31.525   5.661  1.00  0.00           C  \\nATOM   1655  OE1 GLN A 107       4.831 -32.075   5.960  1.00  0.00           O  \\nATOM   1656  NE2 GLN A 107       6.483 -31.678   4.490  1.00  0.00           N  \\nATOM   1657  H   GLN A 107       5.649 -28.137   7.587  1.00  0.00           H  \\nATOM   1658  HA  GLN A 107       8.532 -28.799   7.482  1.00  0.00           H  \\nATOM   1659 1HB  GLN A 107       6.410 -28.986   5.317  1.00  0.00           H  \\nATOM   1660 2HB  GLN A 107       7.994 -29.749   5.303  1.00  0.00           H  \\nATOM   1661 1HG  GLN A 107       7.417 -31.183   7.109  1.00  0.00           H  \\nATOM   1662 2HG  GLN A 107       5.894 -30.293   7.393  1.00  0.00           H  \\nATOM   1663 1HE2 GLN A 107       6.064 -32.263   3.795  1.00  0.00           H  \\nATOM   1664 2HE2 GLN A 107       7.345 -31.209   4.295  1.00  0.00           H  \\nATOM   1665  N   ASN A 108       7.662 -26.108   6.037  1.00  0.00           N  \\nATOM   1666  CA  ASN A 108       8.250 -25.027   5.281  1.00  0.00           C  \\nATOM   1667  C   ASN A 108       8.095 -23.813   6.111  1.00  0.00           C  \\nATOM   1668  O   ASN A 108       6.959 -23.494   6.503  1.00  0.00           O  \\nATOM   1669  CB  ASN A 108       7.500 -24.805   3.984  1.00  0.00           C  \\nATOM   1670  CG  ASN A 108       7.203 -26.092   3.291  1.00  0.00           C  \\nATOM   1671  OD1 ASN A 108       6.180 -26.757   3.560  1.00  0.00           O  \\nATOM   1672  ND2 ASN A 108       8.119 -26.478   2.420  1.00  0.00           N  \\nATOM   1673  H   ASN A 108       6.778 -25.959   6.503  1.00  0.00           H  \\nATOM   1674  HA  ASN A 108       9.281 -25.292   5.043  1.00  0.00           H  \\nATOM   1675 1HB  ASN A 108       6.565 -24.282   4.189  1.00  0.00           H  \\nATOM   1676 2HB  ASN A 108       8.092 -24.170   3.325  1.00  0.00           H  \\nATOM   1677 1HD2 ASN A 108       7.998 -27.331   1.911  1.00  0.00           H  \\nATOM   1678 2HD2 ASN A 108       8.933 -25.918   2.270  1.00  0.00           H  \\nATOM   1679  N   ARG A 109       9.214 -23.178   6.380  1.00  0.00           N  \\nATOM   1680  CA  ARG A 109       9.257 -21.892   7.191  1.00  0.00           C  \\nATOM   1681  C   ARG A 109       9.803 -20.776   6.354  1.00  0.00           C  \\nATOM   1682  O   ARG A 109      10.767 -20.910   5.624  1.00  0.00           O  \\nATOM   1683  CB  ARG A 109      10.136 -21.993   8.449  1.00  0.00           C  \\nATOM   1684  CG  ARG A 109       9.743 -23.083   9.362  1.00  0.00           C  \\nATOM   1685  CD  ARG A 109      10.768 -23.273  10.446  1.00  0.00           C  \\nATOM   1686  NE  ARG A 109      10.689 -22.193  11.432  1.00  0.00           N  \\nATOM   1687  CZ  ARG A 109       9.821 -22.215  12.449  1.00  0.00           C  \\nATOM   1688  NH1 ARG A 109       8.940 -23.205  12.620  1.00  0.00           N  \\nATOM   1689  NH2 ARG A 109       9.819 -21.217  13.326  1.00  0.00           N  \\nATOM   1690  H   ARG A 109      10.077 -23.568   6.029  1.00  0.00           H  \\nATOM   1691  HA  ARG A 109       8.245 -21.658   7.522  1.00  0.00           H  \\nATOM   1692 1HB  ARG A 109      11.173 -22.151   8.156  1.00  0.00           H  \\nATOM   1693 2HB  ARG A 109      10.091 -21.054   9.000  1.00  0.00           H  \\nATOM   1694 1HG  ARG A 109       8.785 -22.843   9.825  1.00  0.00           H  \\nATOM   1695 2HG  ARG A 109       9.652 -24.013   8.800  1.00  0.00           H  \\nATOM   1696 1HD  ARG A 109      10.592 -24.222  10.951  1.00  0.00           H  \\nATOM   1697 2HD  ARG A 109      11.765 -23.275  10.007  1.00  0.00           H  \\nATOM   1698  HE  ARG A 109      11.321 -21.411  11.324  1.00  0.00           H  \\nATOM   1699 1HH1 ARG A 109       8.913 -23.975  11.967  1.00  0.00           H  \\nATOM   1700 2HH1 ARG A 109       8.302 -23.183  13.402  1.00  0.00           H  \\nATOM   1701 1HH2 ARG A 109      10.470 -20.450  13.221  1.00  0.00           H  \\nATOM   1702 2HH2 ARG A 109       9.168 -21.224  14.097  1.00  0.00           H  \\nATOM   1703  N   TYR A 110       9.160 -19.642   6.403  1.00  0.00           N  \\nATOM   1704  CA  TYR A 110       9.684 -18.525   5.683  1.00  0.00           C  \\nATOM   1705  C   TYR A 110       9.585 -17.284   6.532  1.00  0.00           C  \\nATOM   1706  O   TYR A 110       8.561 -17.042   7.158  1.00  0.00           O  \\nATOM   1707  CB  TYR A 110       8.851 -18.223   4.433  1.00  0.00           C  \\nATOM   1708  CG  TYR A 110       8.569 -19.394   3.513  1.00  0.00           C  \\nATOM   1709  CD1 TYR A 110       7.620 -20.355   3.830  1.00  0.00           C  \\nATOM   1710  CD2 TYR A 110       9.274 -19.514   2.334  1.00  0.00           C  \\nATOM   1711  CE1 TYR A 110       7.386 -21.442   2.958  1.00  0.00           C  \\nATOM   1712  CE2 TYR A 110       9.050 -20.585   1.463  1.00  0.00           C  \\nATOM   1713  CZ  TYR A 110       8.116 -21.528   1.787  1.00  0.00           C  \\nATOM   1714  OH  TYR A 110       7.936 -22.566   0.908  1.00  0.00           O  \\nATOM   1715  H   TYR A 110       8.307 -19.530   6.932  1.00  0.00           H  \\nATOM   1716  HA  TYR A 110      10.699 -18.764   5.365  1.00  0.00           H  \\nATOM   1717 1HB  TYR A 110       7.884 -17.815   4.730  1.00  0.00           H  \\nATOM   1718 2HB  TYR A 110       9.356 -17.466   3.834  1.00  0.00           H  \\nATOM   1719  HD1 TYR A 110       7.050 -20.274   4.756  1.00  0.00           H  \\nATOM   1720  HD2 TYR A 110      10.022 -18.767   2.068  1.00  0.00           H  \\nATOM   1721  HE1 TYR A 110       6.639 -22.194   3.212  1.00  0.00           H  \\nATOM   1722  HE2 TYR A 110       9.619 -20.662   0.536  1.00  0.00           H  \\nATOM   1723  HH  TYR A 110       8.530 -22.458   0.161  1.00  0.00           H  \\nATOM   1724  N   THR A 111      10.621 -16.460   6.426  1.00  0.00           N  \\nATOM   1725  CA  THR A 111      10.679 -15.131   7.011  1.00  0.00           C  \\nATOM   1726  C   THR A 111      10.577 -14.148   5.871  1.00  0.00           C  \\nATOM   1727  O   THR A 111      11.372 -14.194   4.943  1.00  0.00           O  \\nATOM   1728  CB  THR A 111      11.997 -14.962   7.827  1.00  0.00           C  \\nATOM   1729  OG1 THR A 111      12.034 -15.888   8.945  1.00  0.00           O  \\nATOM   1730  CG2 THR A 111      12.089 -13.508   8.293  1.00  0.00           C  \\nATOM   1731  H   THR A 111      11.412 -16.800   5.898  1.00  0.00           H  \\nATOM   1732  HA  THR A 111       9.830 -15.012   7.685  1.00  0.00           H  \\nATOM   1733  HB  THR A 111      12.850 -15.212   7.196  1.00  0.00           H  \\nATOM   1734  HG1 THR A 111      11.232 -16.416   8.949  1.00  0.00           H  \\nATOM   1735 1HG2 THR A 111      13.005 -13.367   8.866  1.00  0.00           H  \\nATOM   1736 2HG2 THR A 111      12.098 -12.848   7.426  1.00  0.00           H  \\nATOM   1737 3HG2 THR A 111      11.230 -13.272   8.919  1.00  0.00           H  \\nATOM   1738  N   PHE A 112       9.559 -13.312   5.873  1.00  0.00           N  \\nATOM   1739  CA  PHE A 112       9.412 -12.275   4.824  1.00  0.00           C  \\nATOM   1740  C   PHE A 112      10.553 -11.244   4.921  1.00  0.00           C  \\nATOM   1741  O   PHE A 112      11.180 -11.132   5.969  1.00  0.00           O  \\nATOM   1742  CB  PHE A 112       8.098 -11.486   5.043  1.00  0.00           C  \\nATOM   1743  CG  PHE A 112       6.861 -12.321   4.833  1.00  0.00           C  \\nATOM   1744  CD1 PHE A 112       6.657 -13.034   3.654  1.00  0.00           C  \\nATOM   1745  CD2 PHE A 112       5.877 -12.365   5.815  1.00  0.00           C  \\nATOM   1746  CE1 PHE A 112       5.523 -13.798   3.489  1.00  0.00           C  \\nATOM   1747  CE2 PHE A 112       4.767 -13.131   5.637  1.00  0.00           C  \\nATOM   1748  CZ  PHE A 112       4.594 -13.828   4.485  1.00  0.00           C  \\nATOM   1749  H   PHE A 112       8.863 -13.376   6.602  1.00  0.00           H  \\nATOM   1750  HA  PHE A 112       9.370 -12.769   3.852  1.00  0.00           H  \\nATOM   1751 1HB  PHE A 112       8.078 -11.088   6.057  1.00  0.00           H  \\nATOM   1752 2HB  PHE A 112       8.064 -10.640   4.358  1.00  0.00           H  \\nATOM   1753  HD1 PHE A 112       7.406 -12.981   2.863  1.00  0.00           H  \\nATOM   1754  HD2 PHE A 112       6.011 -11.807   6.742  1.00  0.00           H  \\nATOM   1755  HE1 PHE A 112       5.379 -14.368   2.571  1.00  0.00           H  \\nATOM   1756  HE2 PHE A 112       4.014 -13.184   6.423  1.00  0.00           H  \\nATOM   1757  HZ  PHE A 112       3.687 -14.419   4.368  1.00  0.00           H  \\nATOM   1758  N   ALA A 113      10.793 -10.506   3.856  1.00  0.00           N  \\nATOM   1759  CA  ALA A 113      11.804  -9.442   3.867  1.00  0.00           C  \\nATOM   1760  C   ALA A 113      11.342  -8.228   4.638  1.00  0.00           C  \\nATOM   1761  O   ALA A 113      12.160  -7.420   5.118  1.00  0.00           O  \\nATOM   1762  CB  ALA A 113      12.148  -9.070   2.408  1.00  0.00           C  \\nATOM   1763  H   ALA A 113      10.268 -10.677   3.010  1.00  0.00           H  \\nATOM   1764  HA  ALA A 113      12.693  -9.825   4.368  1.00  0.00           H  \\nATOM   1765 1HB  ALA A 113      12.899  -8.280   2.401  1.00  0.00           H  \\nATOM   1766 2HB  ALA A 113      12.539  -9.947   1.892  1.00  0.00           H  \\nATOM   1767 3HB  ALA A 113      11.250  -8.721   1.901  1.00  0.00           H  \\nATOM   1768  N   PHE A 114      10.019  -8.138   4.874  1.00  0.00           N  \\nATOM   1769  CA  PHE A 114       9.392  -6.940   5.382  1.00  0.00           C  \\nATOM   1770  C   PHE A 114       8.909  -7.304   6.759  1.00  0.00           C  \\nATOM   1771  O   PHE A 114       8.596  -8.490   7.052  1.00  0.00           O  \\nATOM   1772  CB  PHE A 114       8.233  -6.492   4.456  1.00  0.00           C  \\nATOM   1773  CG  PHE A 114       7.228  -7.579   4.128  1.00  0.00           C  \\nATOM   1774  CD1 PHE A 114       6.190  -7.910   5.035  1.00  0.00           C  \\nATOM   1775  CD2 PHE A 114       7.255  -8.223   2.886  1.00  0.00           C  \\nATOM   1776  CE1 PHE A 114       5.227  -8.933   4.707  1.00  0.00           C  \\nATOM   1777  CE2 PHE A 114       6.382  -9.185   2.604  1.00  0.00           C  \\nATOM   1778  CZ  PHE A 114       5.345  -9.559   3.474  1.00  0.00           C  \\nATOM   1779  H   PHE A 114       9.448  -8.949   4.684  1.00  0.00           H  \\nATOM   1780  HA  PHE A 114      10.138  -6.145   5.412  1.00  0.00           H  \\nATOM   1781 1HB  PHE A 114       7.692  -5.670   4.923  1.00  0.00           H  \\nATOM   1782 2HB  PHE A 114       8.641  -6.123   3.516  1.00  0.00           H  \\nATOM   1783  HD1 PHE A 114       6.126  -7.381   5.986  1.00  0.00           H  \\nATOM   1784  HD2 PHE A 114       8.041  -7.974   2.172  1.00  0.00           H  \\nATOM   1785  HE1 PHE A 114       4.442  -9.189   5.418  1.00  0.00           H  \\nATOM   1786  HE2 PHE A 114       6.499  -9.692   1.647  1.00  0.00           H  \\nATOM   1787  HZ  PHE A 114       4.638 -10.336   3.186  1.00  0.00           H  \\nATOM   1788  N   GLY A 115       8.962  -6.286   7.625  1.00  0.00           N  \\nATOM   1789  CA  GLY A 115       8.342  -6.314   8.981  1.00  0.00           C  \\nATOM   1790  C   GLY A 115       6.825  -6.192   8.893  1.00  0.00           C  \\nATOM   1791  O   GLY A 115       6.245  -6.055   7.797  1.00  0.00           O  \\nATOM   1792  H   GLY A 115       9.455  -5.457   7.327  1.00  0.00           H  \\nATOM   1793 1HA  GLY A 115       8.609  -7.244   9.484  1.00  0.00           H  \\nATOM   1794 2HA  GLY A 115       8.743  -5.498   9.581  1.00  0.00           H  \\nATOM   1795  N   GLY A 116       6.201  -6.225  10.065  1.00  0.00           N  \\nATOM   1796  CA  GLY A 116       4.725  -6.232  10.089  1.00  0.00           C  \\nATOM   1797  C   GLY A 116       4.187  -4.881  10.548  1.00  0.00           C  \\nATOM   1798  O   GLY A 116       3.038  -4.789  10.934  1.00  0.00           O  \\nATOM   1799  H   GLY A 116       6.707  -6.245  10.939  1.00  0.00           H  \\nATOM   1800 1HA  GLY A 116       4.346  -6.465   9.094  1.00  0.00           H  \\nATOM   1801 2HA  GLY A 116       4.376  -7.018  10.758  1.00  0.00           H  \\nATOM   1802  N   ASN A 117       5.022  -3.818  10.513  1.00  0.00           N  \\nATOM   1803  CA  ASN A 117       4.538  -2.477  10.864  1.00  0.00           C  \\nATOM   1804  C   ASN A 117       3.642  -1.962   9.786  1.00  0.00           C  \\nATOM   1805  O   ASN A 117       3.793  -2.299   8.580  1.00  0.00           O  \\nATOM   1806  CB  ASN A 117       5.731  -1.519  11.112  1.00  0.00           C  \\nATOM   1807  CG  ASN A 117       6.594  -1.390   9.889  1.00  0.00           C  \\nATOM   1808  OD1 ASN A 117       7.434  -2.291   9.603  1.00  0.00           O  \\nATOM   1809  ND2 ASN A 117       6.403  -0.297   9.121  1.00  0.00           N  \\nATOM   1810  H   ASN A 117       5.988  -3.938  10.244  1.00  0.00           H  \\nATOM   1811  HA  ASN A 117       3.952  -2.550  11.781  1.00  0.00           H  \\nATOM   1812 1HB  ASN A 117       5.355  -0.535  11.396  1.00  0.00           H  \\nATOM   1813 2HB  ASN A 117       6.331  -1.892  11.942  1.00  0.00           H  \\nATOM   1814 1HD2 ASN A 117       6.952  -0.164   8.295  1.00  0.00           H  \\nATOM   1815 2HD2 ASN A 117       5.713   0.379   9.377  1.00  0.00           H  \\nATOM   1816  N   TYR A 118       2.644  -1.156  10.162  1.00  0.00           N  \\nATOM   1817  CA  TYR A 118       1.697  -0.633   9.216  1.00  0.00           C  \\nATOM   1818  C   TYR A 118       2.247   0.126   8.008  1.00  0.00           C  \\nATOM   1819  O   TYR A 118       1.679  -0.027   6.908  1.00  0.00           O  \\nATOM   1820  CB  TYR A 118       0.662   0.282   9.962  1.00  0.00           C  \\nATOM   1821  CG  TYR A 118      -0.326  -0.425  10.904  1.00  0.00           C  \\nATOM   1822  CD1 TYR A 118      -1.081  -1.516  10.460  1.00  0.00           C  \\nATOM   1823  CD2 TYR A 118      -0.387  -0.088  12.253  1.00  0.00           C  \\nATOM   1824  CE1 TYR A 118      -1.928  -2.200  11.332  1.00  0.00           C  \\nATOM   1825  CE2 TYR A 118      -1.235  -0.772  13.133  1.00  0.00           C  \\nATOM   1826  CZ  TYR A 118      -1.986  -1.736  12.668  1.00  0.00           C  \\nATOM   1827  OH  TYR A 118      -2.865  -2.382  13.494  1.00  0.00           O  \\nATOM   1828  H   TYR A 118       2.553  -0.908  11.137  1.00  0.00           H  \\nATOM   1829  HA  TYR A 118       1.169  -1.469   8.757  1.00  0.00           H  \\nATOM   1830 1HB  TYR A 118       1.195   1.020  10.563  1.00  0.00           H  \\nATOM   1831 2HB  TYR A 118       0.064   0.826   9.231  1.00  0.00           H  \\nATOM   1832  HD1 TYR A 118      -1.011  -1.841   9.422  1.00  0.00           H  \\nATOM   1833  HD2 TYR A 118       0.231   0.722  12.641  1.00  0.00           H  \\nATOM   1834  HE1 TYR A 118      -2.510  -3.048  10.973  1.00  0.00           H  \\nATOM   1835  HE2 TYR A 118      -1.262  -0.497  14.188  1.00  0.00           H  \\nATOM   1836  HH  TYR A 118      -2.806  -2.009  14.377  1.00  0.00           H  \\nATOM   1837  N   ASP A 119       3.303   0.918   8.184  1.00  0.00           N  \\nATOM   1838  CA  ASP A 119       3.775   1.728   7.028  1.00  0.00           C  \\nATOM   1839  C   ASP A 119       4.208   0.806   5.896  1.00  0.00           C  \\nATOM   1840  O   ASP A 119       3.815   1.042   4.723  1.00  0.00           O  \\nATOM   1841  CB  ASP A 119       4.962   2.583   7.481  1.00  0.00           C  \\nATOM   1842  CG  ASP A 119       4.559   3.744   8.416  1.00  0.00           C  \\nATOM   1843  OD1 ASP A 119       5.499   4.483   8.802  1.00  0.00           O  \\nATOM   1844  OD2 ASP A 119       3.352   3.925   8.758  1.00  0.00           O  \\nATOM   1845  H   ASP A 119       3.791   0.988   9.066  1.00  0.00           H  \\nATOM   1846  HA  ASP A 119       2.961   2.377   6.705  1.00  0.00           H  \\nATOM   1847 1HB  ASP A 119       5.684   1.955   8.003  1.00  0.00           H  \\nATOM   1848 2HB  ASP A 119       5.463   3.002   6.608  1.00  0.00           H  \\nATOM   1849  N   ARG A 120       4.919  -0.262   6.270  1.00  0.00           N  \\nATOM   1850  CA  ARG A 120       5.404  -1.272   5.324  1.00  0.00           C  \\nATOM   1851  C   ARG A 120       4.226  -2.095   4.662  1.00  0.00           C  \\nATOM   1852  O   ARG A 120       4.109  -2.279   3.410  1.00  0.00           O  \\nATOM   1853  CB  ARG A 120       6.345  -2.207   5.984  1.00  0.00           C  \\nATOM   1854  CG  ARG A 120       7.022  -3.130   4.975  1.00  0.00           C  \\nATOM   1855  CD  ARG A 120       8.257  -2.415   4.371  1.00  0.00           C  \\nATOM   1856  NE  ARG A 120       8.633  -2.938   3.052  1.00  0.00           N  \\nATOM   1857  CZ  ARG A 120       7.998  -2.661   1.907  1.00  0.00           C  \\nATOM   1858  NH1 ARG A 120       6.924  -1.883   1.891  1.00  0.00           N  \\nATOM   1859  NH2 ARG A 120       8.423  -3.175   0.761  1.00  0.00           N  \\nATOM   1860  H   ARG A 120       5.126  -0.369   7.253  1.00  0.00           H  \\nATOM   1861  HA  ARG A 120       5.932  -0.764   4.516  1.00  0.00           H  \\nATOM   1862 1HB  ARG A 120       7.105  -1.640   6.520  1.00  0.00           H  \\nATOM   1863 2HB  ARG A 120       5.807  -2.808   6.717  1.00  0.00           H  \\nATOM   1864 1HG  ARG A 120       7.340  -4.045   5.475  1.00  0.00           H  \\nATOM   1865 2HG  ARG A 120       6.319  -3.377   4.179  1.00  0.00           H  \\nATOM   1866 1HD  ARG A 120       8.043  -1.353   4.257  1.00  0.00           H  \\nATOM   1867 2HD  ARG A 120       9.112  -2.543   5.034  1.00  0.00           H  \\nATOM   1868  HE  ARG A 120       9.433  -3.555   3.001  1.00  0.00           H  \\nATOM   1869 1HH1 ARG A 120       6.573  -1.489   2.752  1.00  0.00           H  \\nATOM   1870 2HH1 ARG A 120       6.457  -1.685   1.018  1.00  0.00           H  \\nATOM   1871 1HH2 ARG A 120       9.232  -3.781   0.748  1.00  0.00           H  \\nATOM   1872 2HH2 ARG A 120       7.939  -2.961  -0.098  1.00  0.00           H  \\nATOM   1873  N   LEU A 121       3.269  -2.539   5.476  1.00  0.00           N  \\nATOM   1874  CA  LEU A 121       2.125  -3.276   4.971  1.00  0.00           C  \\nATOM   1875  C   LEU A 121       1.250  -2.467   4.035  1.00  0.00           C  \\nATOM   1876  O   LEU A 121       0.747  -2.995   3.079  1.00  0.00           O  \\nATOM   1877  CB  LEU A 121       1.304  -3.880   6.170  1.00  0.00           C  \\nATOM   1878  CG  LEU A 121       2.053  -4.910   7.016  1.00  0.00           C  \\nATOM   1879  CD1 LEU A 121       1.064  -5.355   8.092  1.00  0.00           C  \\nATOM   1880  CD2 LEU A 121       2.581  -6.078   6.218  1.00  0.00           C  \\nATOM   1881  H   LEU A 121       3.340  -2.359   6.467  1.00  0.00           H  \\nATOM   1882  HA  LEU A 121       2.488  -4.091   4.345  1.00  0.00           H  \\nATOM   1883 1HB  LEU A 121       0.996  -3.066   6.825  1.00  0.00           H  \\nATOM   1884 2HB  LEU A 121       0.409  -4.358   5.773  1.00  0.00           H  \\nATOM   1885  HG  LEU A 121       2.907  -4.433   7.499  1.00  0.00           H  \\nATOM   1886 1HD1 LEU A 121       1.537  -6.096   8.737  1.00  0.00           H  \\nATOM   1887 2HD1 LEU A 121       0.764  -4.494   8.689  1.00  0.00           H  \\nATOM   1888 3HD1 LEU A 121       0.186  -5.793   7.620  1.00  0.00           H  \\nATOM   1889 1HD2 LEU A 121       3.102  -6.768   6.883  1.00  0.00           H  \\nATOM   1890 2HD2 LEU A 121       1.751  -6.595   5.737  1.00  0.00           H  \\nATOM   1891 3HD2 LEU A 121       3.273  -5.716   5.457  1.00  0.00           H  \\nATOM   1892  N   GLU A 122       1.006  -1.194   4.388  1.00  0.00           N  \\nATOM   1893  CA  GLU A 122       0.226  -0.292   3.579  1.00  0.00           C  \\nATOM   1894  C   GLU A 122       0.874  -0.013   2.194  1.00  0.00           C  \\nATOM   1895  O   GLU A 122       0.138  -0.033   1.209  1.00  0.00           O  \\nATOM   1896  CB  GLU A 122       0.093   1.007   4.315  1.00  0.00           C  \\nATOM   1897  CG  GLU A 122      -0.825   0.859   5.591  1.00  0.00           C  \\nATOM   1898  CD  GLU A 122      -0.946   2.192   6.294  1.00  0.00           C  \\nATOM   1899  OE1 GLU A 122      -0.002   3.038   6.248  1.00  0.00           O  \\nATOM   1900  OE2 GLU A 122      -2.004   2.446   6.973  1.00  0.00           O  \\nATOM   1901  H   GLU A 122       1.391  -0.864   5.261  1.00  0.00           H  \\nATOM   1902  HA  GLU A 122      -0.761  -0.730   3.426  1.00  0.00           H  \\nATOM   1903 1HB  GLU A 122       1.080   1.356   4.618  1.00  0.00           H  \\nATOM   1904 2HB  GLU A 122      -0.330   1.760   3.650  1.00  0.00           H  \\nATOM   1905 1HG  GLU A 122      -1.807   0.503   5.280  1.00  0.00           H  \\nATOM   1906 2HG  GLU A 122      -0.391   0.111   6.253  1.00  0.00           H  \\nATOM   1907  N   GLN A 123       2.178   0.045   2.201  1.00  0.00           N  \\nATOM   1908  CA  GLN A 123       2.940   0.223   0.901  1.00  0.00           C  \\nATOM   1909  C   GLN A 123       2.711  -1.079   0.063  1.00  0.00           C  \\nATOM   1910  O   GLN A 123       2.258  -0.992  -1.108  1.00  0.00           O  \\nATOM   1911  CB  GLN A 123       4.449   0.401   1.130  1.00  0.00           C  \\nATOM   1912  CG  GLN A 123       4.964   1.158   2.383  1.00  0.00           C  \\nATOM   1913  CD  GLN A 123       6.538   1.145   2.631  1.00  0.00           C  \\nATOM   1914  OE1 GLN A 123       7.323   0.553   1.860  1.00  0.00           O  \\nATOM   1915  NE2 GLN A 123       6.965   1.770   3.748  1.00  0.00           N  \\nATOM   1916  H   GLN A 123       2.693  -0.027   3.067  1.00  0.00           H  \\nATOM   1917  HA  GLN A 123       2.571   1.120   0.404  1.00  0.00           H  \\nATOM   1918 1HB  GLN A 123       4.927  -0.577   1.181  1.00  0.00           H  \\nATOM   1919 2HB  GLN A 123       4.885   0.934   0.285  1.00  0.00           H  \\nATOM   1920 1HG  GLN A 123       4.680   2.207   2.303  1.00  0.00           H  \\nATOM   1921 2HG  GLN A 123       4.515   0.716   3.272  1.00  0.00           H  \\nATOM   1922 1HE2 GLN A 123       7.942   1.799   3.963  1.00  0.00           H  \\nATOM   1923 2HE2 GLN A 123       6.306   2.205   4.361  1.00  0.00           H  \\nATOM   1924  N   LEU A 124       2.994  -2.262   0.672  1.00  0.00           N  \\nATOM   1925  CA  LEU A 124       2.810  -3.531  -0.042  1.00  0.00           C  \\nATOM   1926  C   LEU A 124       1.364  -3.673  -0.536  1.00  0.00           C  \\nATOM   1927  O   LEU A 124       1.085  -4.103  -1.645  1.00  0.00           O  \\nATOM   1928  CB  LEU A 124       3.080  -4.717   0.830  1.00  0.00           C  \\nATOM   1929  CG  LEU A 124       4.549  -4.898   1.151  1.00  0.00           C  \\nATOM   1930  CD1 LEU A 124       4.850  -5.959   2.188  1.00  0.00           C  \\nATOM   1931  CD2 LEU A 124       5.356  -5.154  -0.204  1.00  0.00           C  \\nATOM   1932  H   LEU A 124       3.334  -2.277   1.623  1.00  0.00           H  \\nATOM   1933  HA  LEU A 124       3.512  -3.564  -0.875  1.00  0.00           H  \\nATOM   1934 1HB  LEU A 124       2.526  -4.598   1.759  1.00  0.00           H  \\nATOM   1935 2HB  LEU A 124       2.714  -5.611   0.325  1.00  0.00           H  \\nATOM   1936  HG  LEU A 124       4.929  -3.999   1.638  1.00  0.00           H  \\nATOM   1937 1HD1 LEU A 124       5.927  -6.014   2.350  1.00  0.00           H  \\nATOM   1938 2HD1 LEU A 124       4.356  -5.703   3.125  1.00  0.00           H  \\nATOM   1939 3HD1 LEU A 124       4.486  -6.924   1.837  1.00  0.00           H  \\nATOM   1940 1HD2 LEU A 124       6.415  -5.284   0.022  1.00  0.00           H  \\nATOM   1941 2HD2 LEU A 124       4.975  -6.053  -0.690  1.00  0.00           H  \\nATOM   1942 3HD2 LEU A 124       5.229  -4.300  -0.870  1.00  0.00           H  \\nATOM   1943  N   ALA A 125       0.364  -3.324   0.281  1.00  0.00           N  \\nATOM   1944  CA  ALA A 125      -1.021  -3.496  -0.065  1.00  0.00           C  \\nATOM   1945  C   ALA A 125      -1.529  -2.534  -1.147  1.00  0.00           C  \\nATOM   1946  O   ALA A 125      -2.688  -2.678  -1.683  1.00  0.00           O  \\nATOM   1947  CB  ALA A 125      -1.858  -3.239   1.284  1.00  0.00           C  \\nATOM   1948  H   ALA A 125       0.601  -2.923   1.177  1.00  0.00           H  \\nATOM   1949  HA  ALA A 125      -1.158  -4.519  -0.417  1.00  0.00           H  \\nATOM   1950 1HB  ALA A 125      -2.922  -3.357   1.080  1.00  0.00           H  \\nATOM   1951 2HB  ALA A 125      -1.553  -3.958   2.044  1.00  0.00           H  \\nATOM   1952 3HB  ALA A 125      -1.668  -2.229   1.643  1.00  0.00           H  \\nATOM   1953  N   GLY A 126      -0.774  -1.452  -1.263  1.00  0.00           N  \\nATOM   1954  CA  GLY A 126      -1.253  -0.272  -2.059  1.00  0.00           C  \\nATOM   1955  C   GLY A 126      -2.530   0.437  -1.563  1.00  0.00           C  \\nATOM   1956  O   GLY A 126      -3.275   1.046  -2.321  1.00  0.00           O  \\nATOM   1957  H   GLY A 126       0.131  -1.410  -0.817  1.00  0.00           H  \\nATOM   1958 1HA  GLY A 126      -0.470   0.486  -2.094  1.00  0.00           H  \\nATOM   1959 2HA  GLY A 126      -1.447  -0.581  -3.086  1.00  0.00           H  \\nATOM   1960  N   ASN A 127      -2.797   0.353  -0.250  1.00  0.00           N  \\nATOM   1961  CA  ASN A 127      -3.940   0.994   0.352  1.00  0.00           C  \\nATOM   1962  C   ASN A 127      -3.453   1.350   1.780  1.00  0.00           C  \\nATOM   1963  O   ASN A 127      -2.766   0.537   2.476  1.00  0.00           O  \\nATOM   1964  CB  ASN A 127      -5.166   0.046   0.469  1.00  0.00           C  \\nATOM   1965  CG  ASN A 127      -5.845  -0.242  -0.888  1.00  0.00           C  \\nATOM   1966  OD1 ASN A 127      -6.649   0.547  -1.368  1.00  0.00           O  \\nATOM   1967  ND2 ASN A 127      -5.581  -1.447  -1.440  1.00  0.00           N  \\nATOM   1968  H   ASN A 127      -2.169  -0.181   0.333  1.00  0.00           H  \\nATOM   1969  HA  ASN A 127      -4.234   1.837  -0.275  1.00  0.00           H  \\nATOM   1970 1HB  ASN A 127      -4.852  -0.903   0.907  1.00  0.00           H  \\nATOM   1971 2HB  ASN A 127      -5.905   0.487   1.139  1.00  0.00           H  \\nATOM   1972 1HD2 ASN A 127      -5.989  -1.695  -2.319  1.00  0.00           H  \\nATOM   1973 2HD2 ASN A 127      -4.977  -2.090  -0.970  1.00  0.00           H  \\nATOM   1974  N   LEU A 128      -3.590   2.330   2.304  1.00  0.00           N  \\nATOM   1975  CA  LEU A 128      -3.626   2.785   3.696  1.00  0.00           C  \\nATOM   1976  C   LEU A 128      -4.771   2.170   4.471  1.00  0.00           C  \\nATOM   1977  O   LEU A 128      -5.795   1.817   3.940  1.00  0.00           O  \\nATOM   1978  CB  LEU A 128      -3.825   4.298   3.731  1.00  0.00           C  \\nATOM   1979  CG  LEU A 128      -2.692   5.123   3.054  1.00  0.00           C  \\nATOM   1980  CD1 LEU A 128      -2.984   6.603   3.070  1.00  0.00           C  \\nATOM   1981  CD2 LEU A 128      -1.434   4.938   3.597  1.00  0.00           C  \\nATOM   1982  H   LEU A 128      -3.963   2.929   1.581  1.00  0.00           H  \\nATOM   1983  HA  LEU A 128      -2.674   2.538   4.165  1.00  0.00           H  \\nATOM   1984 1HB  LEU A 128      -4.763   4.536   3.231  1.00  0.00           H  \\nATOM   1985 2HB  LEU A 128      -3.901   4.615   4.771  1.00  0.00           H  \\nATOM   1986  HG  LEU A 128      -2.621   4.850   2.001  1.00  0.00           H  \\nATOM   1987 1HD1 LEU A 128      -2.167   7.140   2.588  1.00  0.00           H  \\nATOM   1988 2HD1 LEU A 128      -3.912   6.796   2.532  1.00  0.00           H  \\nATOM   1989 3HD1 LEU A 128      -3.083   6.942   4.100  1.00  0.00           H  \\nATOM   1990 1HD2 LEU A 128      -0.711   5.554   3.060  1.00  0.00           H  \\nATOM   1991 2HD2 LEU A 128      -1.445   5.226   4.648  1.00  0.00           H  \\nATOM   1992 3HD2 LEU A 128      -1.151   3.889   3.511  1.00  0.00           H  \\nATOM   1993  N   ARG A 129      -4.601   2.075   5.787  1.00  0.00           N  \\nATOM   1994  CA  ARG A 129      -5.672   1.619   6.666  1.00  0.00           C  \\nATOM   1995  C   ARG A 129      -6.995   2.336   6.373  1.00  0.00           C  \\nATOM   1996  O   ARG A 129      -8.082   1.739   6.357  1.00  0.00           O  \\nATOM   1997  CB  ARG A 129      -5.239   1.836   8.138  1.00  0.00           C  \\nATOM   1998  CG  ARG A 129      -4.302   0.750   8.638  1.00  0.00           C  \\nATOM   1999  CD  ARG A 129      -3.749   1.132  10.025  1.00  0.00           C  \\nATOM   2000  NE  ARG A 129      -2.642   2.092   9.834  1.00  0.00           N  \\nATOM   2001  CZ  ARG A 129      -2.062   2.729  10.829  1.00  0.00           C  \\nATOM   2002  NH1 ARG A 129      -2.537   2.611  12.082  1.00  0.00           N  \\nATOM   2003  NH2 ARG A 129      -1.024   3.532  10.628  1.00  0.00           N  \\nATOM   2004  H   ARG A 129      -3.708   2.324   6.188  1.00  0.00           H  \\nATOM   2005  HA  ARG A 129      -5.835   0.555   6.490  1.00  0.00           H  \\nATOM   2006 1HB  ARG A 129      -4.742   2.800   8.232  1.00  0.00           H  \\nATOM   2007 2HB  ARG A 129      -6.122   1.859   8.777  1.00  0.00           H  \\nATOM   2008 1HG  ARG A 129      -4.844  -0.192   8.714  1.00  0.00           H  \\nATOM   2009 2HG  ARG A 129      -3.472   0.636   7.939  1.00  0.00           H  \\nATOM   2010 1HD  ARG A 129      -4.540   1.587  10.620  1.00  0.00           H  \\nATOM   2011 2HD  ARG A 129      -3.385   0.238  10.530  1.00  0.00           H  \\nATOM   2012  HE  ARG A 129      -2.323   2.260   8.889  1.00  0.00           H  \\nATOM   2013 1HH1 ARG A 129      -3.344   2.030  12.263  1.00  0.00           H  \\nATOM   2014 2HH1 ARG A 129      -2.088   3.102  12.841  1.00  0.00           H  \\nATOM   2015 1HH2 ARG A 129      -0.661   3.665   9.694  1.00  0.00           H  \\nATOM   2016 2HH2 ARG A 129      -0.598   4.009  11.408  1.00  0.00           H  \\nATOM   2017  N   GLU A 130      -6.946   3.640   6.100  1.00  0.00           N  \\nATOM   2018  CA  GLU A 130      -8.233   4.324   5.872  1.00  0.00           C  \\nATOM   2019  C   GLU A 130      -8.958   3.837   4.604  1.00  0.00           C  \\nATOM   2020  O   GLU A 130     -10.182   4.098   4.444  1.00  0.00           O  \\nATOM   2021  CB  GLU A 130      -8.010   5.835   5.781  1.00  0.00           C  \\nATOM   2022  CG  GLU A 130      -7.284   6.289   4.523  1.00  0.00           C  \\nATOM   2023  CD  GLU A 130      -7.049   7.774   4.486  1.00  0.00           C  \\nATOM   2024  OE1 GLU A 130      -7.395   8.434   5.436  1.00  0.00           O  \\nATOM   2025  OE2 GLU A 130      -6.524   8.248   3.507  1.00  0.00           O  \\nATOM   2026  H   GLU A 130      -6.089   4.171   6.042  1.00  0.00           H  \\nATOM   2027  HA  GLU A 130      -8.884   4.129   6.725  1.00  0.00           H  \\nATOM   2028 1HB  GLU A 130      -8.972   6.348   5.816  1.00  0.00           H  \\nATOM   2029 2HB  GLU A 130      -7.430   6.170   6.642  1.00  0.00           H  \\nATOM   2030 1HG  GLU A 130      -6.322   5.780   4.468  1.00  0.00           H  \\nATOM   2031 2HG  GLU A 130      -7.869   5.996   3.653  1.00  0.00           H  \\nATOM   2032  N   ASN A 131      -8.240   3.156   3.710  1.00  0.00           N  \\nATOM   2033  CA  ASN A 131      -8.848   2.638   2.506  1.00  0.00           C  \\nATOM   2034  C   ASN A 131      -9.247   1.152   2.570  1.00  0.00           C  \\nATOM   2035  O   ASN A 131      -9.728   0.608   1.602  1.00  0.00           O  \\nATOM   2036  CB  ASN A 131      -7.912   2.879   1.336  1.00  0.00           C  \\nATOM   2037  CG  ASN A 131      -7.680   4.341   1.073  1.00  0.00           C  \\nATOM   2038  OD1 ASN A 131      -8.463   5.195   1.507  1.00  0.00           O  \\nATOM   2039  ND2 ASN A 131      -6.620   4.647   0.370  1.00  0.00           N  \\nATOM   2040  H   ASN A 131      -7.255   2.997   3.871  1.00  0.00           H  \\nATOM   2041  HA  ASN A 131      -9.777   3.182   2.327  1.00  0.00           H  \\nATOM   2042 1HB  ASN A 131      -6.952   2.400   1.533  1.00  0.00           H  \\nATOM   2043 2HB  ASN A 131      -8.328   2.422   0.438  1.00  0.00           H  \\nATOM   2044 1HD2 ASN A 131      -6.415   5.604   0.164  1.00  0.00           H  \\nATOM   2045 2HD2 ASN A 131      -6.014   3.924   0.039  1.00  0.00           H  \\nATOM   2046  N   ILE A 132      -9.029   0.505   3.718  1.00  0.00           N  \\nATOM   2047  CA  ILE A 132      -9.258  -0.944   3.856  1.00  0.00           C  \\nATOM   2048  C   ILE A 132     -10.412  -1.137   4.829  1.00  0.00           C  \\nATOM   2049  O   ILE A 132     -10.348  -0.765   6.010  1.00  0.00           O  \\nATOM   2050  CB  ILE A 132      -8.007  -1.684   4.365  1.00  0.00           C  \\nATOM   2051  CG1 ILE A 132      -6.853  -1.528   3.370  1.00  0.00           C  \\nATOM   2052  CG2 ILE A 132      -8.316  -3.154   4.600  1.00  0.00           C  \\nATOM   2053  CD1 ILE A 132      -5.520  -1.999   3.903  1.00  0.00           C  \\nATOM   2054  H   ILE A 132      -8.697   1.029   4.514  1.00  0.00           H  \\nATOM   2055  HA  ILE A 132      -9.548  -1.343   2.885  1.00  0.00           H  \\nATOM   2056  HB  ILE A 132      -7.677  -1.237   5.303  1.00  0.00           H  \\nATOM   2057 1HG1 ILE A 132      -7.077  -2.089   2.464  1.00  0.00           H  \\nATOM   2058 2HG1 ILE A 132      -6.755  -0.479   3.088  1.00  0.00           H  \\nATOM   2059 1HG2 ILE A 132      -7.421  -3.661   4.958  1.00  0.00           H  \\nATOM   2060 2HG2 ILE A 132      -9.107  -3.245   5.343  1.00  0.00           H  \\nATOM   2061 3HG2 ILE A 132      -8.643  -3.611   3.665  1.00  0.00           H  \\nATOM   2062 1HD1 ILE A 132      -4.753  -1.856   3.142  1.00  0.00           H  \\nATOM   2063 2HD1 ILE A 132      -5.259  -1.424   4.793  1.00  0.00           H  \\nATOM   2064 3HD1 ILE A 132      -5.583  -3.056   4.159  1.00  0.00           H  \\nATOM   2065  N   GLU A 133     -11.458  -1.802   4.340  1.00  0.00           N  \\nATOM   2066  CA  GLU A 133     -12.687  -2.029   5.163  1.00  0.00           C  \\nATOM   2067  C   GLU A 133     -12.447  -3.199   6.135  1.00  0.00           C  \\nATOM   2068  O   GLU A 133     -11.802  -4.203   5.809  1.00  0.00           O  \\nATOM   2069  CB  GLU A 133     -13.893  -2.306   4.289  1.00  0.00           C  \\nATOM   2070  CG  GLU A 133     -14.209  -1.154   3.340  1.00  0.00           C  \\nATOM   2071  CD  GLU A 133     -15.410  -1.391   2.420  1.00  0.00           C  \\nATOM   2072  OE1 GLU A 133     -15.799  -2.545   2.238  1.00  0.00           O  \\nATOM   2073  OE2 GLU A 133     -15.881  -0.382   1.781  1.00  0.00           O  \\nATOM   2074  H   GLU A 133     -11.424  -2.162   3.397  1.00  0.00           H  \\nATOM   2075  HA  GLU A 133     -12.887  -1.128   5.743  1.00  0.00           H  \\nATOM   2076 1HB  GLU A 133     -13.716  -3.206   3.700  1.00  0.00           H  \\nATOM   2077 2HB  GLU A 133     -14.763  -2.491   4.918  1.00  0.00           H  \\nATOM   2078 1HG  GLU A 133     -14.410  -0.258   3.928  1.00  0.00           H  \\nATOM   2079 2HG  GLU A 133     -13.336  -0.958   2.719  1.00  0.00           H  \\nATOM   2080  N   LEU A 134     -12.986  -3.023   7.296  1.00  0.00           N  \\nATOM   2081  CA  LEU A 134     -12.976  -4.022   8.416  1.00  0.00           C  \\nATOM   2082  C   LEU A 134     -14.379  -4.512   8.686  1.00  0.00           C  \\nATOM   2083  O   LEU A 134     -15.396  -3.811   8.462  1.00  0.00           O  \\nATOM   2084  CB  LEU A 134     -12.393  -3.388   9.674  1.00  0.00           C  \\nATOM   2085  CG  LEU A 134     -10.971  -2.777   9.528  1.00  0.00           C  \\nATOM   2086  CD1 LEU A 134     -10.491  -2.036  10.850  1.00  0.00           C  \\nATOM   2087  CD2 LEU A 134      -9.872  -3.768   9.071  1.00  0.00           C  \\nATOM   2088  H   LEU A 134     -13.444  -2.133   7.434  1.00  0.00           H  \\nATOM   2089  HA  LEU A 134     -12.350  -4.864   8.123  1.00  0.00           H  \\nATOM   2090 1HB  LEU A 134     -13.061  -2.594  10.004  1.00  0.00           H  \\nATOM   2091 2HB  LEU A 134     -12.348  -4.146  10.456  1.00  0.00           H  \\nATOM   2092  HG  LEU A 134     -10.993  -1.976   8.789  1.00  0.00           H  \\nATOM   2093 1HD1 LEU A 134      -9.492  -1.627  10.696  1.00  0.00           H  \\nATOM   2094 2HD1 LEU A 134     -11.182  -1.226  11.084  1.00  0.00           H  \\nATOM   2095 3HD1 LEU A 134     -10.468  -2.745  11.677  1.00  0.00           H  \\nATOM   2096 1HD2 LEU A 134      -8.918  -3.245   8.999  1.00  0.00           H  \\nATOM   2097 2HD2 LEU A 134      -9.788  -4.578   9.796  1.00  0.00           H  \\nATOM   2098 3HD2 LEU A 134     -10.135  -4.179   8.096  1.00  0.00           H  \\nATOM   2099  N   GLY A 135     -14.523  -5.632   9.338  1.00  0.00           N  \\nATOM   2100  CA  GLY A 135     -15.773  -6.277   9.646  1.00  0.00           C  \\nATOM   2101  C   GLY A 135     -15.720  -7.798   9.408  1.00  0.00           C  \\nATOM   2102  O   GLY A 135     -14.656  -8.323   9.075  1.00  0.00           O  \\nATOM   2103  H   GLY A 135     -13.660  -6.061   9.641  1.00  0.00           H  \\nATOM   2104 1HA  GLY A 135     -16.034  -6.087  10.687  1.00  0.00           H  \\nATOM   2105 2HA  GLY A 135     -16.565  -5.847   9.033  1.00  0.00           H  \\nATOM   2106  N   ASN A 136     -16.818  -8.479   9.618  1.00  0.00           N  \\nATOM   2107  CA  ASN A 136     -16.776  -9.923   9.491  1.00  0.00           C  \\nATOM   2108  C   ASN A 136     -16.567 -10.336   8.042  1.00  0.00           C  \\nATOM   2109  O   ASN A 136     -15.886 -11.380   7.796  1.00  0.00           O  \\nATOM   2110  CB  ASN A 136     -18.037 -10.547  10.019  1.00  0.00           C  \\nATOM   2111  CG  ASN A 136     -17.833 -11.974  10.457  1.00  0.00           C  \\nATOM   2112  OD1 ASN A 136     -16.878 -12.295  11.195  1.00  0.00           O  \\nATOM   2113  ND2 ASN A 136     -18.758 -12.855  10.057  1.00  0.00           N  \\nATOM   2114  H   ASN A 136     -17.689  -8.032   9.865  1.00  0.00           H  \\nATOM   2115  HA  ASN A 136     -15.935 -10.298  10.077  1.00  0.00           H  \\nATOM   2116 1HB  ASN A 136     -18.403  -9.966  10.867  1.00  0.00           H  \\nATOM   2117 2HB  ASN A 136     -18.807 -10.521   9.247  1.00  0.00           H  \\nATOM   2118 1HD2 ASN A 136     -18.677 -13.818  10.316  1.00  0.00           H  \\nATOM   2119 2HD2 ASN A 136     -19.530 -12.551   9.499  1.00  0.00           H  \\nATOM   2120  N   GLY A 137     -17.035  -9.580   7.070  1.00  0.00           N  \\nATOM   2121  CA  GLY A 137     -16.793  -9.926   5.661  1.00  0.00           C  \\nATOM   2122  C   GLY A 137     -15.317  -9.813   5.344  1.00  0.00           C  \\nATOM   2123  O   GLY A 137     -14.694 -10.752   4.814  1.00  0.00           O  \\nATOM   2124  H   GLY A 137     -17.568  -8.750   7.288  1.00  0.00           H  \\nATOM   2125 1HA  GLY A 137     -17.144 -10.940   5.470  1.00  0.00           H  \\nATOM   2126 2HA  GLY A 137     -17.369  -9.261   5.019  1.00  0.00           H  \\nATOM   2127  N   PRO A 138     -14.697  -8.670   5.649  1.00  0.00           N  \\nATOM   2128  CA  PRO A 138     -13.237  -8.535   5.459  1.00  0.00           C  \\nATOM   2129  C   PRO A 138     -12.461  -9.594   6.148  1.00  0.00           C  \\nATOM   2130  O   PRO A 138     -11.454 -10.086   5.637  1.00  0.00           O  \\nATOM   2131  CB  PRO A 138     -12.935  -7.111   6.050  1.00  0.00           C  \\nATOM   2132  CG  PRO A 138     -14.169  -6.371   5.660  1.00  0.00           C  \\nATOM   2133  CD  PRO A 138     -15.350  -7.373   5.928  1.00  0.00           C  \\nATOM   2134  HA  PRO A 138     -13.007  -8.572   4.384  1.00  0.00           H  \\nATOM   2135 1HB  PRO A 138     -12.773  -7.181   7.136  1.00  0.00           H  \\nATOM   2136 2HB  PRO A 138     -12.009  -6.710   5.612  1.00  0.00           H  \\nATOM   2137 1HG  PRO A 138     -14.262  -5.449   6.252  1.00  0.00           H  \\nATOM   2138 2HG  PRO A 138     -14.112  -6.066   4.605  1.00  0.00           H  \\nATOM   2139 1HD  PRO A 138     -15.678  -7.283   6.974  1.00  0.00           H  \\nATOM   2140 2HD  PRO A 138     -16.180  -7.154   5.241  1.00  0.00           H  \\nATOM   2141  N   LEU A 139     -12.866  -9.966   7.374  1.00  0.00           N  \\nATOM   2142  CA  LEU A 139     -12.059 -10.970   8.116  1.00  0.00           C  \\nATOM   2143  C   LEU A 139     -12.200 -12.332   7.435  1.00  0.00           C  \\nATOM   2144  O   LEU A 139     -11.241 -13.055   7.363  1.00  0.00           O  \\nATOM   2145  CB  LEU A 139     -12.489 -10.994   9.600  1.00  0.00           C  \\nATOM   2146  CG  LEU A 139     -11.640 -11.916  10.525  1.00  0.00           C  \\nATOM   2147  CD1 LEU A 139     -10.194 -11.559  10.549  1.00  0.00           C  \\nATOM   2148  CD2 LEU A 139     -12.223 -11.709  11.978  1.00  0.00           C  \\nATOM   2149  H   LEU A 139     -13.697  -9.587   7.805  1.00  0.00           H  \\nATOM   2150  HA  LEU A 139     -11.010 -10.683   8.056  1.00  0.00           H  \\nATOM   2151 1HB  LEU A 139     -12.432  -9.981   9.994  1.00  0.00           H  \\nATOM   2152 2HB  LEU A 139     -13.526 -11.326   9.656  1.00  0.00           H  \\nATOM   2153  HG  LEU A 139     -11.742 -12.952  10.199  1.00  0.00           H  \\nATOM   2154 1HD1 LEU A 139      -9.662 -12.241  11.212  1.00  0.00           H  \\nATOM   2155 2HD1 LEU A 139      -9.783 -11.638   9.543  1.00  0.00           H  \\nATOM   2156 3HD1 LEU A 139     -10.078 -10.538  10.910  1.00  0.00           H  \\nATOM   2157 1HD2 LEU A 139     -11.671 -12.331  12.684  1.00  0.00           H  \\nATOM   2158 2HD2 LEU A 139     -12.125 -10.661  12.264  1.00  0.00           H  \\nATOM   2159 3HD2 LEU A 139     -13.276 -11.992  11.991  1.00  0.00           H  \\nATOM   2160  N   GLU A 140     -13.403 -12.665   6.986  1.00  0.00           N  \\nATOM   2161  CA  GLU A 140     -13.578 -13.907   6.250  1.00  0.00           C  \\nATOM   2162  C   GLU A 140     -12.678 -13.921   5.053  1.00  0.00           C  \\nATOM   2163  O   GLU A 140     -12.008 -14.916   4.802  1.00  0.00           O  \\nATOM   2164  CB  GLU A 140     -15.067 -14.003   5.831  1.00  0.00           C  \\nATOM   2165  CG  GLU A 140     -15.337 -15.054   4.788  1.00  0.00           C  \\nATOM   2166  CD  GLU A 140     -15.546 -16.428   5.317  1.00  0.00           C  \\nATOM   2167  OE1 GLU A 140     -16.076 -16.514   6.454  1.00  0.00           O  \\nATOM   2168  OE2 GLU A 140     -15.222 -17.430   4.566  1.00  0.00           O  \\nATOM   2169  H   GLU A 140     -14.204 -12.071   7.146  1.00  0.00           H  \\nATOM   2170  HA  GLU A 140     -13.323 -14.738   6.908  1.00  0.00           H  \\nATOM   2171 1HB  GLU A 140     -15.678 -14.227   6.706  1.00  0.00           H  \\nATOM   2172 2HB  GLU A 140     -15.397 -13.041   5.439  1.00  0.00           H  \\nATOM   2173 1HG  GLU A 140     -16.230 -14.775   4.229  1.00  0.00           H  \\nATOM   2174 2HG  GLU A 140     -14.500 -15.083   4.092  1.00  0.00           H  \\nATOM   2175  N   GLU A 141     -12.642 -12.827   4.289  1.00  0.00           N  \\nATOM   2176  CA  GLU A 141     -11.768 -12.711   3.111  1.00  0.00           C  \\nATOM   2177  C   GLU A 141     -10.321 -12.849   3.492  1.00  0.00           C  \\nATOM   2178  O   GLU A 141      -9.523 -13.499   2.745  1.00  0.00           O  \\nATOM   2179  CB  GLU A 141     -12.025 -11.371   2.394  1.00  0.00           C  \\nATOM   2180  CG  GLU A 141     -13.423 -11.338   1.723  1.00  0.00           C  \\nATOM   2181  CD  GLU A 141     -13.857  -9.947   1.345  1.00  0.00           C  \\nATOM   2182  OE1 GLU A 141     -13.120  -8.975   1.582  1.00  0.00           O  \\nATOM   2183  OE2 GLU A 141     -14.987  -9.799   0.836  1.00  0.00           O  \\nATOM   2184  H   GLU A 141     -13.241 -12.052   4.535  1.00  0.00           H  \\nATOM   2185  HA  GLU A 141     -12.000 -13.528   2.426  1.00  0.00           H  \\nATOM   2186 1HB  GLU A 141     -11.951 -10.554   3.112  1.00  0.00           H  \\nATOM   2187 2HB  GLU A 141     -11.258 -11.211   1.636  1.00  0.00           H  \\nATOM   2188 1HG  GLU A 141     -13.398 -11.956   0.826  1.00  0.00           H  \\nATOM   2189 2HG  GLU A 141     -14.153 -11.769   2.407  1.00  0.00           H  \\nATOM   2190  N   ALA A 142      -9.908 -12.200   4.582  1.00  0.00           N  \\nATOM   2191  CA  ALA A 142      -8.525 -12.144   4.977  1.00  0.00           C  \\nATOM   2192  C   ALA A 142      -8.042 -13.537   5.388  1.00  0.00           C  \\nATOM   2193  O   ALA A 142      -6.896 -13.888   5.102  1.00  0.00           O  \\nATOM   2194  CB  ALA A 142      -8.350 -11.151   6.131  1.00  0.00           C  \\nATOM   2195  H   ALA A 142     -10.601 -11.731   5.148  1.00  0.00           H  \\nATOM   2196  HA  ALA A 142      -7.942 -11.803   4.122  1.00  0.00           H  \\nATOM   2197 1HB  ALA A 142      -7.301 -11.115   6.424  1.00  0.00           H  \\nATOM   2198 2HB  ALA A 142      -8.672 -10.160   5.810  1.00  0.00           H  \\nATOM   2199 3HB  ALA A 142      -8.952 -11.471   6.980  1.00  0.00           H  \\nATOM   2200  N   ILE A 143      -8.866 -14.310   6.036  1.00  0.00           N  \\nATOM   2201  CA  ILE A 143      -8.465 -15.674   6.451  1.00  0.00           C  \\nATOM   2202  C   ILE A 143      -8.269 -16.529   5.201  1.00  0.00           C  \\nATOM   2203  O   ILE A 143      -7.302 -17.278   5.139  1.00  0.00           O  \\nATOM   2204  CB  ILE A 143      -9.527 -16.282   7.411  1.00  0.00           C  \\nATOM   2205  CG1 ILE A 143      -9.513 -15.547   8.723  1.00  0.00           C  \\nATOM   2206  CG2 ILE A 143      -9.337 -17.803   7.575  1.00  0.00           C  \\nATOM   2207  CD1 ILE A 143     -10.842 -15.831   9.491  1.00  0.00           C  \\nATOM   2208  H   ILE A 143      -9.793 -13.980   6.262  1.00  0.00           H  \\nATOM   2209  HA  ILE A 143      -7.514 -15.609   6.978  1.00  0.00           H  \\nATOM   2210  HB  ILE A 143     -10.524 -16.101   7.012  1.00  0.00           H  \\nATOM   2211 1HG1 ILE A 143      -8.655 -15.870   9.311  1.00  0.00           H  \\nATOM   2212 2HG1 ILE A 143      -9.400 -14.478   8.540  1.00  0.00           H  \\nATOM   2213 1HG2 ILE A 143     -10.096 -18.193   8.253  1.00  0.00           H  \\nATOM   2214 2HG2 ILE A 143      -9.433 -18.288   6.605  1.00  0.00           H  \\nATOM   2215 3HG2 ILE A 143      -8.347 -18.004   7.985  1.00  0.00           H  \\nATOM   2216 1HD1 ILE A 143     -10.834 -15.298  10.442  1.00  0.00           H  \\nATOM   2217 2HD1 ILE A 143     -11.688 -15.491   8.892  1.00  0.00           H  \\nATOM   2218 3HD1 ILE A 143     -10.934 -16.900   9.676  1.00  0.00           H  \\nATOM   2219  N   SER A 144      -9.156 -16.384   4.232  1.00  0.00           N  \\nATOM   2220  CA  SER A 144      -8.895 -17.119   2.958  1.00  0.00           C  \\nATOM   2221  C   SER A 144      -7.627 -16.667   2.305  1.00  0.00           C  \\nATOM   2222  O   SER A 144      -6.852 -17.543   1.804  1.00  0.00           O  \\nATOM   2223  CB  SER A 144     -10.035 -17.023   1.987  1.00  0.00           C  \\nATOM   2224  OG  SER A 144     -11.143 -17.701   2.535  1.00  0.00           O  \\nATOM   2225  H   SER A 144      -9.985 -15.812   4.309  1.00  0.00           H  \\nATOM   2226  HA  SER A 144      -8.750 -18.175   3.191  1.00  0.00           H  \\nATOM   2227 1HB  SER A 144     -10.270 -15.975   1.804  1.00  0.00           H  \\nATOM   2228 2HB  SER A 144      -9.740 -17.463   1.035  1.00  0.00           H  \\nATOM   2229  HG  SER A 144     -10.853 -18.038   3.386  1.00  0.00           H  \\nATOM   2230  N   ALA A 145      -7.392 -15.365   2.274  1.00  0.00           N  \\nATOM   2231  CA  ALA A 145      -6.164 -14.886   1.638  1.00  0.00           C  \\nATOM   2232  C   ALA A 145      -4.915 -15.406   2.278  1.00  0.00           C  \\nATOM   2233  O   ALA A 145      -3.926 -15.820   1.580  1.00  0.00           O  \\nATOM   2234  CB  ALA A 145      -6.119 -13.387   1.601  1.00  0.00           C  \\nATOM   2235  H   ALA A 145      -8.032 -14.695   2.675  1.00  0.00           H  \\nATOM   2236  HA  ALA A 145      -6.146 -15.257   0.613  1.00  0.00           H  \\nATOM   2237 1HB  ALA A 145      -5.195 -13.062   1.123  1.00  0.00           H  \\nATOM   2238 2HB  ALA A 145      -6.972 -13.011   1.035  1.00  0.00           H  \\nATOM   2239 3HB  ALA A 145      -6.158 -12.997   2.617  1.00  0.00           H  \\nATOM   2240  N   LEU A 146      -4.854 -15.395   3.620  1.00  0.00           N  \\nATOM   2241  CA  LEU A 146      -3.625 -15.895   4.251  1.00  0.00           C  \\nATOM   2242  C   LEU A 146      -3.497 -17.389   3.998  1.00  0.00           C  \\nATOM   2243  O   LEU A 146      -2.395 -17.859   3.793  1.00  0.00           O  \\nATOM   2244  CB  LEU A 146      -3.696 -15.692   5.780  1.00  0.00           C  \\nATOM   2245  CG  LEU A 146      -3.093 -14.380   6.326  1.00  0.00           C  \\nATOM   2246  CD1 LEU A 146      -3.672 -13.206   5.783  1.00  0.00           C  \\nATOM   2247  CD2 LEU A 146      -3.158 -14.362   7.876  1.00  0.00           C  \\nATOM   2248  H   LEU A 146      -5.607 -15.062   4.205  1.00  0.00           H  \\nATOM   2249  HA  LEU A 146      -2.780 -15.329   3.859  1.00  0.00           H  \\nATOM   2250 1HB  LEU A 146      -4.740 -15.721   6.086  1.00  0.00           H  \\nATOM   2251 2HB  LEU A 146      -3.173 -16.516   6.265  1.00  0.00           H  \\nATOM   2252  HG  LEU A 146      -2.052 -14.300   6.013  1.00  0.00           H  \\nATOM   2253 1HD1 LEU A 146      -3.193 -12.329   6.218  1.00  0.00           H  \\nATOM   2254 2HD1 LEU A 146      -3.531 -13.197   4.702  1.00  0.00           H  \\nATOM   2255 3HD1 LEU A 146      -4.737 -13.187   6.011  1.00  0.00           H  \\nATOM   2256 1HD2 LEU A 146      -2.729 -13.431   8.248  1.00  0.00           H  \\nATOM   2257 2HD2 LEU A 146      -4.197 -14.436   8.198  1.00  0.00           H  \\nATOM   2258 3HD2 LEU A 146      -2.594 -15.206   8.274  1.00  0.00           H  \\nATOM   2259  N   TYR A 147      -4.619 -18.118   3.992  1.00  0.00           N  \\nATOM   2260  CA  TYR A 147      -4.530 -19.566   3.664  1.00  0.00           C  \\nATOM   2261  C   TYR A 147      -4.023 -19.782   2.270  1.00  0.00           C  \\nATOM   2262  O   TYR A 147      -3.081 -20.554   2.103  1.00  0.00           O  \\nATOM   2263  CB  TYR A 147      -5.879 -20.288   3.893  1.00  0.00           C  \\nATOM   2264  CG  TYR A 147      -5.733 -21.740   3.514  1.00  0.00           C  \\nATOM   2265  CD1 TYR A 147      -5.988 -22.160   2.241  1.00  0.00           C  \\nATOM   2266  CD2 TYR A 147      -5.337 -22.615   4.472  1.00  0.00           C  \\nATOM   2267  CE1 TYR A 147      -5.799 -23.517   1.907  1.00  0.00           C  \\nATOM   2268  CE2 TYR A 147      -5.150 -23.950   4.158  1.00  0.00           C  \\nATOM   2269  CZ  TYR A 147      -5.393 -24.368   2.883  1.00  0.00           C  \\nATOM   2270  OH  TYR A 147      -5.212 -25.720   2.614  1.00  0.00           O  \\nATOM   2271  H   TYR A 147      -5.521 -17.716   4.203  1.00  0.00           H  \\nATOM   2272  HA  TYR A 147      -3.787 -20.024   4.317  1.00  0.00           H  \\nATOM   2273 1HB  TYR A 147      -6.170 -20.192   4.940  1.00  0.00           H  \\nATOM   2274 2HB  TYR A 147      -6.653 -19.811   3.293  1.00  0.00           H  \\nATOM   2275  HD1 TYR A 147      -6.335 -21.448   1.491  1.00  0.00           H  \\nATOM   2276  HD2 TYR A 147      -5.167 -22.267   5.491  1.00  0.00           H  \\nATOM   2277  HE1 TYR A 147      -6.000 -23.862   0.893  1.00  0.00           H  \\nATOM   2278  HE2 TYR A 147      -4.814 -24.649   4.925  1.00  0.00           H  \\nATOM   2279  HH  TYR A 147      -4.929 -26.169   3.414  1.00  0.00           H  \\nATOM   2280  N   TYR A 148      -4.527 -19.038   1.317  1.00  0.00           N  \\nATOM   2281  CA  TYR A 148      -4.143 -19.343  -0.098  1.00  0.00           C  \\nATOM   2282  C   TYR A 148      -2.765 -18.760  -0.429  1.00  0.00           C  \\nATOM   2283  O   TYR A 148      -2.211 -19.072  -1.503  1.00  0.00           O  \\nATOM   2284  CB  TYR A 148      -5.248 -18.902  -1.041  1.00  0.00           C  \\nATOM   2285  CG  TYR A 148      -6.440 -19.811  -1.057  1.00  0.00           C  \\nATOM   2286  CD1 TYR A 148      -6.314 -21.185  -1.304  1.00  0.00           C  \\nATOM   2287  CD2 TYR A 148      -7.693 -19.310  -0.834  1.00  0.00           C  \\nATOM   2288  CE1 TYR A 148      -7.403 -21.981  -1.432  1.00  0.00           C  \\nATOM   2289  CE2 TYR A 148      -8.766 -20.101  -0.931  1.00  0.00           C  \\nATOM   2290  CZ  TYR A 148      -8.645 -21.406  -1.195  1.00  0.00           C  \\nATOM   2291  OH  TYR A 148      -9.748 -22.187  -1.295  1.00  0.00           O  \\nATOM   2292  H   TYR A 148      -5.160 -18.273   1.501  1.00  0.00           H  \\nATOM   2293  HA  TYR A 148      -4.005 -20.420  -0.195  1.00  0.00           H  \\nATOM   2294 1HB  TYR A 148      -5.589 -17.904  -0.761  1.00  0.00           H  \\nATOM   2295 2HB  TYR A 148      -4.857 -18.842  -2.056  1.00  0.00           H  \\nATOM   2296  HD1 TYR A 148      -5.325 -21.634  -1.397  1.00  0.00           H  \\nATOM   2297  HD2 TYR A 148      -7.822 -18.259  -0.576  1.00  0.00           H  \\nATOM   2298  HE1 TYR A 148      -7.280 -23.047  -1.624  1.00  0.00           H  \\nATOM   2299  HE2 TYR A 148      -9.764 -19.685  -0.795  1.00  0.00           H  \\nATOM   2300  HH  TYR A 148     -10.532 -21.652  -1.150  1.00  0.00           H  \\nATOM   2301  N   TYR A 149      -2.141 -17.998   0.463  1.00  0.00           N  \\nATOM   2302  CA  TYR A 149      -0.721 -17.664   0.190  1.00  0.00           C  \\nATOM   2303  C   TYR A 149       0.097 -18.939   0.238  1.00  0.00           C  \\nATOM   2304  O   TYR A 149       1.071 -19.050  -0.525  1.00  0.00           O  \\nATOM   2305  CB  TYR A 149      -0.227 -16.632   1.193  1.00  0.00           C  \\nATOM   2306  CG  TYR A 149       1.131 -16.077   0.815  1.00  0.00           C  \\nATOM   2307  CD1 TYR A 149       1.270 -15.207  -0.273  1.00  0.00           C  \\nATOM   2308  CD2 TYR A 149       2.218 -16.447   1.540  1.00  0.00           C  \\nATOM   2309  CE1 TYR A 149       2.515 -14.713  -0.606  1.00  0.00           C  \\nATOM   2310  CE2 TYR A 149       3.490 -15.967   1.215  1.00  0.00           C  \\nATOM   2311  CZ  TYR A 149       3.598 -15.157   0.125  1.00  0.00           C  \\nATOM   2312  OH  TYR A 149       4.866 -14.703  -0.202  1.00  0.00           O  \\nATOM   2313  H   TYR A 149      -2.578 -17.642   1.301  1.00  0.00           H  \\nATOM   2314  HA  TYR A 149      -0.651 -17.242  -0.813  1.00  0.00           H  \\nATOM   2315 1HB  TYR A 149      -0.944 -15.812   1.254  1.00  0.00           H  \\nATOM   2316 2HB  TYR A 149      -0.164 -17.086   2.181  1.00  0.00           H  \\nATOM   2317  HD1 TYR A 149       0.395 -14.920  -0.857  1.00  0.00           H  \\nATOM   2318  HD2 TYR A 149       2.104 -17.126   2.385  1.00  0.00           H  \\nATOM   2319  HE1 TYR A 149       2.624 -14.036  -1.453  1.00  0.00           H  \\nATOM   2320  HE2 TYR A 149       4.350 -16.245   1.825  1.00  0.00           H  \\nATOM   2321  HH  TYR A 149       5.502 -15.052   0.428  1.00  0.00           H  \\nATOM   2322  N   SER A 150      -0.277 -19.911   1.054  1.00  0.00           N  \\nATOM   2323  CA  SER A 150       0.501 -21.172   1.177  1.00  0.00           C  \\nATOM   2324  C   SER A 150       0.362 -22.061  -0.062  1.00  0.00           C  \\nATOM   2325  O   SER A 150       1.204 -22.942  -0.261  1.00  0.00           O  \\nATOM   2326  CB  SER A 150       0.090 -21.978   2.429  1.00  0.00           C  \\nATOM   2327  OG  SER A 150      -1.183 -22.541   2.376  1.00  0.00           O  \\nATOM   2328  H   SER A 150      -1.113 -19.793   1.608  1.00  0.00           H  \\nATOM   2329  HA  SER A 150       1.558 -20.919   1.270  1.00  0.00           H  \\nATOM   2330 1HB  SER A 150       0.801 -22.788   2.589  1.00  0.00           H  \\nATOM   2331 2HB  SER A 150       0.127 -21.332   3.305  1.00  0.00           H  \\nATOM   2332  HG  SER A 150      -1.548 -22.291   1.524  1.00  0.00           H  \\nATOM   2333  N   THR A 151      -0.669 -21.805  -0.877  1.00  0.00           N  \\nATOM   2334  CA  THR A 151      -0.900 -22.614  -2.070  1.00  0.00           C  \\nATOM   2335  C   THR A 151      -0.435 -21.918  -3.351  1.00  0.00           C  \\nATOM   2336  O   THR A 151      -0.558 -22.475  -4.489  1.00  0.00           O  \\nATOM   2337  CB  THR A 151      -2.378 -22.978  -2.223  1.00  0.00           C  \\nATOM   2338  OG1 THR A 151      -3.108 -21.769  -2.531  1.00  0.00           O  \\nATOM   2339  CG2 THR A 151      -2.859 -23.586  -0.949  1.00  0.00           C  \\nATOM   2340  H   THR A 151      -1.299 -21.043  -0.669  1.00  0.00           H  \\nATOM   2341  HA  THR A 151      -0.330 -23.539  -1.976  1.00  0.00           H  \\nATOM   2342  HB  THR A 151      -2.495 -23.687  -3.042  1.00  0.00           H  \\nATOM   2343  HG1 THR A 151      -2.498 -21.028  -2.567  1.00  0.00           H  \\nATOM   2344 1HG2 THR A 151      -3.912 -23.850  -1.047  1.00  0.00           H  \\nATOM   2345 2HG2 THR A 151      -2.278 -24.482  -0.732  1.00  0.00           H  \\nATOM   2346 3HG2 THR A 151      -2.739 -22.871  -0.137  1.00  0.00           H  \\nATOM   2347  N   GLY A 152       0.086 -20.705  -3.209  1.00  0.00           N  \\nATOM   2348  CA  GLY A 152       0.352 -19.865  -4.399  1.00  0.00           C  \\nATOM   2349  C   GLY A 152      -0.714 -19.059  -4.964  1.00  0.00           C  \\nATOM   2350  O   GLY A 152      -0.463 -18.347  -5.949  1.00  0.00           O  \\nATOM   2351  H   GLY A 152       0.306 -20.343  -2.292  1.00  0.00           H  \\nATOM   2352 1HA  GLY A 152       1.153 -19.160  -4.175  1.00  0.00           H  \\nATOM   2353 2HA  GLY A 152       0.697 -20.494  -5.218  1.00  0.00           H  \\nATOM   2354  N   GLY A 153      -1.903 -19.110  -4.379  1.00  0.00           N  \\nATOM   2355  CA  GLY A 153      -3.073 -18.438  -4.829  1.00  0.00           C  \\nATOM   2356  C   GLY A 153      -3.235 -16.970  -4.464  1.00  0.00           C  \\nATOM   2357  O   GLY A 153      -3.961 -16.270  -5.118  1.00  0.00           O  \\nATOM   2358  H   GLY A 153      -1.953 -19.681  -3.547  1.00  0.00           H  \\nATOM   2359 1HA  GLY A 153      -3.125 -18.487  -5.917  1.00  0.00           H  \\nATOM   2360 2HA  GLY A 153      -3.956 -18.946  -4.442  1.00  0.00           H  \\nATOM   2361  N   THR A 154      -2.557 -16.514  -3.443  1.00  0.00           N  \\nATOM   2362  CA  THR A 154      -2.591 -15.121  -2.977  1.00  0.00           C  \\nATOM   2363  C   THR A 154      -1.355 -14.344  -3.353  1.00  0.00           C  \\nATOM   2364  O   THR A 154      -0.220 -14.809  -3.057  1.00  0.00           O  \\nATOM   2365  CB  THR A 154      -2.773 -15.105  -1.448  1.00  0.00           C  \\nATOM   2366  OG1 THR A 154      -3.970 -15.779  -1.088  1.00  0.00           O  \\nATOM   2367  CG2 THR A 154      -2.880 -13.622  -0.922  1.00  0.00           C  \\nATOM   2368  H   THR A 154      -1.977 -17.182  -2.956  1.00  0.00           H  \\nATOM   2369  HA  THR A 154      -3.438 -14.620  -3.446  1.00  0.00           H  \\nATOM   2370  HB  THR A 154      -1.921 -15.593  -0.975  1.00  0.00           H  \\nATOM   2371  HG1 THR A 154      -4.407 -16.099  -1.881  1.00  0.00           H  \\nATOM   2372 1HG2 THR A 154      -3.008 -13.628   0.160  1.00  0.00           H  \\nATOM   2373 2HG2 THR A 154      -1.970 -13.079  -1.177  1.00  0.00           H  \\nATOM   2374 3HG2 THR A 154      -3.736 -13.133  -1.385  1.00  0.00           H  \\nATOM   2375  N   GLN A 155      -1.565 -13.174  -3.908  1.00  0.00           N  \\nATOM   2376  CA  GLN A 155      -0.458 -12.340  -4.246  1.00  0.00           C  \\nATOM   2377  C   GLN A 155      -0.141 -11.399  -3.114  1.00  0.00           C  \\nATOM   2378  O   GLN A 155      -0.988 -11.173  -2.234  1.00  0.00           O  \\nATOM   2379  CB  GLN A 155      -0.831 -11.519  -5.492  1.00  0.00           C  \\nATOM   2380  CG  GLN A 155      -0.974 -12.424  -6.751  1.00  0.00           C  \\nATOM   2381  CD  GLN A 155       0.121 -13.535  -6.826  1.00  0.00           C  \\nATOM   2382  OE1 GLN A 155       1.339 -13.246  -6.908  1.00  0.00           O  \\nATOM   2383  NE2 GLN A 155      -0.321 -14.811  -6.815  1.00  0.00           N  \\nATOM   2384  H   GLN A 155      -2.502 -12.850  -4.102  1.00  0.00           H  \\nATOM   2385  HA  GLN A 155       0.399 -12.976  -4.469  1.00  0.00           H  \\nATOM   2386 1HB  GLN A 155      -1.770 -10.995  -5.315  1.00  0.00           H  \\nATOM   2387 2HB  GLN A 155      -0.065 -10.765  -5.674  1.00  0.00           H  \\nATOM   2388 1HG  GLN A 155      -1.949 -12.911  -6.727  1.00  0.00           H  \\nATOM   2389 2HG  GLN A 155      -0.888 -11.804  -7.643  1.00  0.00           H  \\nATOM   2390 1HE2 GLN A 155       0.332 -15.568  -6.861  1.00  0.00           H  \\nATOM   2391 2HE2 GLN A 155      -1.301 -15.001  -6.761  1.00  0.00           H  \\nATOM   2392  N   LEU A 156       1.073 -10.875  -3.095  1.00  0.00           N  \\nATOM   2393  CA  LEU A 156       1.559 -10.057  -1.998  1.00  0.00           C  \\nATOM   2394  C   LEU A 156       0.663  -8.885  -1.582  1.00  0.00           C  \\nATOM   2395  O   LEU A 156       0.435  -8.651  -0.370  1.00  0.00           O  \\nATOM   2396  CB  LEU A 156       2.926  -9.498  -2.400  1.00  0.00           C  \\nATOM   2397  CG  LEU A 156       4.009  -9.683  -1.358  1.00  0.00           C  \\nATOM   2398  CD1 LEU A 156       5.162  -8.622  -1.516  1.00  0.00           C  \\nATOM   2399  CD2 LEU A 156       3.569  -9.882   0.151  1.00  0.00           C  \\nATOM   2400  H   LEU A 156       1.681 -11.056  -3.881  1.00  0.00           H  \\nATOM   2401  HA  LEU A 156       1.661 -10.688  -1.115  1.00  0.00           H  \\nATOM   2402 1HB  LEU A 156       3.246  -9.989  -3.318  1.00  0.00           H  \\nATOM   2403 2HB  LEU A 156       2.820  -8.432  -2.600  1.00  0.00           H  \\nATOM   2404  HG  LEU A 156       4.598 -10.569  -1.598  1.00  0.00           H  \\nATOM   2405 1HD1 LEU A 156       5.919  -8.790  -0.749  1.00  0.00           H  \\nATOM   2406 2HD1 LEU A 156       5.617  -8.723  -2.502  1.00  0.00           H  \\nATOM   2407 3HD1 LEU A 156       4.751  -7.619  -1.406  1.00  0.00           H  \\nATOM   2408 1HD2 LEU A 156       4.455 -10.002   0.776  1.00  0.00           H  \\nATOM   2409 2HD2 LEU A 156       3.006  -9.010   0.484  1.00  0.00           H  \\nATOM   2410 3HD2 LEU A 156       2.944 -10.771   0.234  1.00  0.00           H  \\nATOM   2411  N   PRO A 157       0.164  -8.062  -2.523  1.00  0.00           N  \\nATOM   2412  CA  PRO A 157      -0.663  -6.949  -2.038  1.00  0.00           C  \\nATOM   2413  C   PRO A 157      -1.915  -7.394  -1.297  1.00  0.00           C  \\nATOM   2414  O   PRO A 157      -2.265  -6.751  -0.229  1.00  0.00           O  \\nATOM   2415  CB  PRO A 157      -1.036  -6.136  -3.301  1.00  0.00           C  \\nATOM   2416  CG  PRO A 157       0.195  -6.473  -4.224  1.00  0.00           C  \\nATOM   2417  CD  PRO A 157       0.303  -7.978  -4.011  1.00  0.00           C  \\nATOM   2418  HA  PRO A 157      -0.067  -6.327  -1.354  1.00  0.00           H  \\nATOM   2419 1HB  PRO A 157      -2.009  -6.471  -3.690  1.00  0.00           H  \\nATOM   2420 2HB  PRO A 157      -1.142  -5.071  -3.045  1.00  0.00           H  \\nATOM   2421 1HG  PRO A 157      -0.015  -6.174  -5.261  1.00  0.00           H  \\nATOM   2422 2HG  PRO A 157       1.079  -5.903  -3.902  1.00  0.00           H  \\nATOM   2423 1HD  PRO A 157      -0.516  -8.484  -4.543  1.00  0.00           H  \\nATOM   2424 2HD  PRO A 157       1.279  -8.331  -4.375  1.00  0.00           H  \\nATOM   2425  N   THR A 158      -2.533  -8.485  -1.726  1.00  0.00           N  \\nATOM   2426  CA  THR A 158      -3.693  -9.030  -0.973  1.00  0.00           C  \\nATOM   2427  C   THR A 158      -3.297  -9.620   0.351  1.00  0.00           C  \\nATOM   2428  O   THR A 158      -4.056  -9.493   1.321  1.00  0.00           O  \\nATOM   2429  CB  THR A 158      -4.449 -10.058  -1.848  1.00  0.00           C  \\nATOM   2430  OG1 THR A 158      -5.004  -9.376  -3.004  1.00  0.00           O  \\nATOM   2431  CG2 THR A 158      -5.570 -10.642  -1.123  1.00  0.00           C  \\nATOM   2432  H   THR A 158      -2.225  -8.957  -2.565  1.00  0.00           H  \\nATOM   2433  HA  THR A 158      -4.366  -8.207  -0.731  1.00  0.00           H  \\nATOM   2434  HB  THR A 158      -3.767 -10.855  -2.144  1.00  0.00           H  \\nATOM   2435  HG1 THR A 158      -4.777  -8.444  -2.964  1.00  0.00           H  \\nATOM   2436 1HG2 THR A 158      -6.082 -11.361  -1.763  1.00  0.00           H  \\nATOM   2437 2HG2 THR A 158      -5.202 -11.148  -0.230  1.00  0.00           H  \\nATOM   2438 3HG2 THR A 158      -6.265  -9.855  -0.833  1.00  0.00           H  \\nATOM   2439  N   LEU A 159      -2.150 -10.266   0.432  1.00  0.00           N  \\nATOM   2440  CA  LEU A 159      -1.671 -10.796   1.690  1.00  0.00           C  \\nATOM   2441  C   LEU A 159      -1.459  -9.642   2.648  1.00  0.00           C  \\nATOM   2442  O   LEU A 159      -1.821  -9.749   3.831  1.00  0.00           O  \\nATOM   2443  CB  LEU A 159      -0.333 -11.539   1.498  1.00  0.00           C  \\nATOM   2444  CG  LEU A 159       0.277 -12.105   2.783  1.00  0.00           C  \\nATOM   2445  CD1 LEU A 159      -0.635 -13.224   3.263  1.00  0.00           C  \\nATOM   2446  CD2 LEU A 159       1.685 -12.642   2.515  1.00  0.00           C  \\nATOM   2447  H   LEU A 159      -1.592 -10.395  -0.400  1.00  0.00           H  \\nATOM   2448  HA  LEU A 159      -2.407 -11.503   2.070  1.00  0.00           H  \\nATOM   2449 1HB  LEU A 159      -0.490 -12.364   0.805  1.00  0.00           H  \\nATOM   2450 2HB  LEU A 159       0.386 -10.851   1.055  1.00  0.00           H  \\nATOM   2451  HG  LEU A 159       0.335 -11.318   3.536  1.00  0.00           H  \\nATOM   2452 1HD1 LEU A 159      -0.232 -13.654   4.180  1.00  0.00           H  \\nATOM   2453 2HD1 LEU A 159      -1.630 -12.824   3.457  1.00  0.00           H  \\nATOM   2454 3HD1 LEU A 159      -0.697 -13.997   2.497  1.00  0.00           H  \\nATOM   2455 1HD2 LEU A 159       2.105 -13.040   3.439  1.00  0.00           H  \\nATOM   2456 2HD2 LEU A 159       1.636 -13.434   1.768  1.00  0.00           H  \\nATOM   2457 3HD2 LEU A 159       2.318 -11.835   2.147  1.00  0.00           H  \\nATOM   2458  N   ALA A 160      -0.821  -8.564   2.179  1.00  0.00           N  \\nATOM   2459  CA  ALA A 160      -0.568  -7.454   3.110  1.00  0.00           C  \\nATOM   2460  C   ALA A 160      -1.845  -6.787   3.595  1.00  0.00           C  \\nATOM   2461  O   ALA A 160      -1.984  -6.457   4.790  1.00  0.00           O  \\nATOM   2462  CB  ALA A 160       0.402  -6.413   2.444  1.00  0.00           C  \\nATOM   2463  H   ALA A 160      -0.510  -8.484   1.221  1.00  0.00           H  \\nATOM   2464  HA  ALA A 160      -0.096  -7.862   4.004  1.00  0.00           H  \\nATOM   2465 1HB  ALA A 160       0.587  -5.592   3.137  1.00  0.00           H  \\nATOM   2466 2HB  ALA A 160       1.346  -6.900   2.199  1.00  0.00           H  \\nATOM   2467 3HB  ALA A 160      -0.052  -6.024   1.534  1.00  0.00           H  \\nATOM   2468  N   ARG A 161      -2.827  -6.659   2.722  1.00  0.00           N  \\nATOM   2469  CA  ARG A 161      -4.135  -6.080   3.080  1.00  0.00           C  \\nATOM   2470  C   ARG A 161      -4.757  -6.993   4.150  1.00  0.00           C  \\nATOM   2471  O   ARG A 161      -5.357  -6.529   5.084  1.00  0.00           O  \\nATOM   2472  CB  ARG A 161      -4.971  -6.082   1.820  1.00  0.00           C  \\nATOM   2473  CG  ARG A 161      -6.367  -5.616   2.107  1.00  0.00           C  \\nATOM   2474  CD  ARG A 161      -7.259  -5.898   0.901  1.00  0.00           C  \\nATOM   2475  NE  ARG A 161      -7.470  -4.574   0.366  1.00  0.00           N  \\nATOM   2476  CZ  ARG A 161      -8.561  -3.857   0.585  1.00  0.00           C  \\nATOM   2477  NH1 ARG A 161      -9.572  -4.383   1.251  1.00  0.00           N  \\nATOM   2478  NH2 ARG A 161      -8.676  -2.644   0.092  1.00  0.00           N  \\nATOM   2479  H   ARG A 161      -2.670  -6.971   1.774  1.00  0.00           H  \\nATOM   2480  HA  ARG A 161      -3.977  -5.060   3.431  1.00  0.00           H  \\nATOM   2481 1HB  ARG A 161      -4.511  -5.432   1.078  1.00  0.00           H  \\nATOM   2482 2HB  ARG A 161      -4.997  -7.088   1.402  1.00  0.00           H  \\nATOM   2483 1HG  ARG A 161      -6.756  -6.146   2.977  1.00  0.00           H  \\nATOM   2484 2HG  ARG A 161      -6.359  -4.544   2.309  1.00  0.00           H  \\nATOM   2485 1HD  ARG A 161      -6.737  -6.559   0.210  1.00  0.00           H  \\nATOM   2486 2HD  ARG A 161      -8.180  -6.375   1.234  1.00  0.00           H  \\nATOM   2487  HE  ARG A 161      -6.738  -4.177  -0.208  1.00  0.00           H  \\nATOM   2488 1HH1 ARG A 161      -9.513  -5.332   1.593  1.00  0.00           H  \\nATOM   2489 2HH1 ARG A 161     -10.405  -3.837   1.419  1.00  0.00           H  \\nATOM   2490 1HH2 ARG A 161      -7.927  -2.250  -0.461  1.00  0.00           H  \\nATOM   2491 2HH2 ARG A 161      -9.512  -2.106   0.265  1.00  0.00           H  \\nATOM   2492  N   SER A 162      -4.598  -8.300   3.978  1.00  0.00           N  \\nATOM   2493  CA  SER A 162      -5.226  -9.278   4.869  1.00  0.00           C  \\nATOM   2494  C   SER A 162      -4.529  -9.242   6.203  1.00  0.00           C  \\nATOM   2495  O   SER A 162      -5.197  -9.411   7.212  1.00  0.00           O  \\nATOM   2496  CB  SER A 162      -5.193 -10.640   4.216  1.00  0.00           C  \\nATOM   2497  OG  SER A 162      -5.968 -10.522   2.999  1.00  0.00           O  \\nATOM   2498  H   SER A 162      -4.028  -8.628   3.211  1.00  0.00           H  \\nATOM   2499  HA  SER A 162      -6.263  -8.982   5.031  1.00  0.00           H  \\nATOM   2500 1HB  SER A 162      -4.160 -10.924   4.018  1.00  0.00           H  \\nATOM   2501 2HB  SER A 162      -5.610 -11.380   4.897  1.00  0.00           H  \\nATOM   2502  HG  SER A 162      -6.274  -9.612   2.967  1.00  0.00           H  \\nATOM   2503  N   PHE A 163      -3.213  -9.002   6.267  1.00  0.00           N  \\nATOM   2504  CA  PHE A 163      -2.555  -8.821   7.563  1.00  0.00           C  \\nATOM   2505  C   PHE A 163      -3.134  -7.589   8.236  1.00  0.00           C  \\nATOM   2506  O   PHE A 163      -3.347  -7.576   9.493  1.00  0.00           O  \\nATOM   2507  CB  PHE A 163      -1.057  -8.627   7.374  1.00  0.00           C  \\nATOM   2508  CG  PHE A 163      -0.304  -9.883   7.031  1.00  0.00           C  \\nATOM   2509  CD1 PHE A 163      -0.567 -11.029   7.713  1.00  0.00           C  \\nATOM   2510  CD2 PHE A 163       0.761  -9.833   6.142  1.00  0.00           C  \\nATOM   2511  CE1 PHE A 163       0.208 -12.215   7.474  1.00  0.00           C  \\nATOM   2512  CE2 PHE A 163       1.531 -11.023   5.909  1.00  0.00           C  \\nATOM   2513  CZ  PHE A 163       1.234 -12.166   6.553  1.00  0.00           C  \\nATOM   2514  H   PHE A 163      -2.662  -8.943   5.423  1.00  0.00           H  \\nATOM   2515  HA  PHE A 163      -2.719  -9.718   8.163  1.00  0.00           H  \\nATOM   2516 1HB  PHE A 163      -0.882  -7.905   6.577  1.00  0.00           H  \\nATOM   2517 2HB  PHE A 163      -0.625  -8.218   8.287  1.00  0.00           H  \\nATOM   2518  HD1 PHE A 163      -1.375 -11.039   8.445  1.00  0.00           H  \\nATOM   2519  HD2 PHE A 163       1.010  -8.911   5.615  1.00  0.00           H  \\nATOM   2520  HE1 PHE A 163      -0.019 -13.132   8.016  1.00  0.00           H  \\nATOM   2521  HE2 PHE A 163       2.362 -10.995   5.205  1.00  0.00           H  \\nATOM   2522  HZ  PHE A 163       1.818 -13.060   6.339  1.00  0.00           H  \\nATOM   2523  N   ILE A 164      -3.330  -6.522   7.494  1.00  0.00           N  \\nATOM   2524  CA  ILE A 164      -3.762  -5.287   8.091  1.00  0.00           C  \\nATOM   2525  C   ILE A 164      -5.143  -5.505   8.729  1.00  0.00           C  \\nATOM   2526  O   ILE A 164      -5.452  -4.966   9.777  1.00  0.00           O  \\nATOM   2527  CB  ILE A 164      -3.794  -4.141   7.062  1.00  0.00           C  \\nATOM   2528  CG1 ILE A 164      -2.373  -3.746   6.744  1.00  0.00           C  \\nATOM   2529  CG2 ILE A 164      -4.577  -2.917   7.554  1.00  0.00           C  \\nATOM   2530  CD1 ILE A 164      -2.273  -2.820   5.508  1.00  0.00           C  \\nATOM   2531  H   ILE A 164      -3.179  -6.560   6.496  1.00  0.00           H  \\nATOM   2532  HA  ILE A 164      -3.057  -5.017   8.876  1.00  0.00           H  \\nATOM   2533  HB  ILE A 164      -4.267  -4.489   6.144  1.00  0.00           H  \\nATOM   2534 1HG1 ILE A 164      -1.937  -3.236   7.602  1.00  0.00           H  \\nATOM   2535 2HG1 ILE A 164      -1.779  -4.642   6.560  1.00  0.00           H  \\nATOM   2536 1HG2 ILE A 164      -4.563  -2.144   6.786  1.00  0.00           H  \\nATOM   2537 2HG2 ILE A 164      -5.607  -3.204   7.762  1.00  0.00           H  \\nATOM   2538 3HG2 ILE A 164      -4.117  -2.533   8.464  1.00  0.00           H  \\nATOM   2539 1HD1 ILE A 164      -1.228  -2.568   5.326  1.00  0.00           H  \\nATOM   2540 2HD1 ILE A 164      -2.680  -3.332   4.636  1.00  0.00           H  \\nATOM   2541 3HD1 ILE A 164      -2.839  -1.908   5.691  1.00  0.00           H  \\nATOM   2542  N   ILE A 165      -6.026  -6.219   8.067  1.00  0.00           N  \\nATOM   2543  CA  ILE A 165      -7.371  -6.541   8.574  1.00  0.00           C  \\nATOM   2544  C   ILE A 165      -7.211  -7.387   9.855  1.00  0.00           C  \\nATOM   2545  O   ILE A 165      -7.871  -7.084  10.874  1.00  0.00           O  \\nATOM   2546  CB  ILE A 165      -8.212  -7.324   7.547  1.00  0.00           C  \\nATOM   2547  CG1 ILE A 165      -8.488  -6.364   6.351  1.00  0.00           C  \\nATOM   2548  CG2 ILE A 165      -9.470  -7.917   8.189  1.00  0.00           C  \\nATOM   2549  CD1 ILE A 165      -8.977  -7.041   5.072  1.00  0.00           C  \\nATOM   2550  H   ILE A 165      -5.749  -6.559   7.157  1.00  0.00           H  \\nATOM   2551  HA  ILE A 165      -7.892  -5.609   8.788  1.00  0.00           H  \\nATOM   2552  HB  ILE A 165      -7.617  -8.139   7.136  1.00  0.00           H  \\nATOM   2553 1HG1 ILE A 165      -9.238  -5.629   6.640  1.00  0.00           H  \\nATOM   2554 2HG1 ILE A 165      -7.576  -5.819   6.104  1.00  0.00           H  \\nATOM   2555 1HG2 ILE A 165     -10.040  -8.462   7.437  1.00  0.00           H  \\nATOM   2556 2HG2 ILE A 165      -9.183  -8.597   8.990  1.00  0.00           H  \\nATOM   2557 3HG2 ILE A 165     -10.083  -7.114   8.597  1.00  0.00           H  \\nATOM   2558 1HD1 ILE A 165      -9.139  -6.288   4.301  1.00  0.00           H  \\nATOM   2559 2HD1 ILE A 165      -8.228  -7.756   4.729  1.00  0.00           H  \\nATOM   2560 3HD1 ILE A 165      -9.912  -7.562   5.271  1.00  0.00           H  \\nATOM   2561  N   CYS A 166      -6.439  -8.462   9.809  1.00  0.00           N  \\nATOM   2562  CA  CYS A 166      -6.307  -9.279  11.005  1.00  0.00           C  \\nATOM   2563  C   CYS A 166      -5.683  -8.611  12.170  1.00  0.00           C  \\nATOM   2564  O   CYS A 166      -6.117  -8.797  13.286  1.00  0.00           O  \\nATOM   2565  CB  CYS A 166      -5.509 -10.543  10.708  1.00  0.00           C  \\nATOM   2566  SG  CYS A 166      -6.271 -11.761   9.591  1.00  0.00           S  \\nATOM   2567  H   CYS A 166      -5.941  -8.731   8.972  1.00  0.00           H  \\nATOM   2568  HA  CYS A 166      -7.304  -9.568  11.338  1.00  0.00           H  \\nATOM   2569 1HB  CYS A 166      -4.552 -10.273  10.262  1.00  0.00           H  \\nATOM   2570 2HB  CYS A 166      -5.299 -11.068  11.640  1.00  0.00           H  \\nATOM   2571  HG  CYS A 166      -5.268 -12.633   9.632  1.00  0.00           H  \\nATOM   2572  N   ILE A 167      -4.625  -7.856  11.989  1.00  0.00           N  \\nATOM   2573  CA  ILE A 167      -3.973  -7.160  13.099  1.00  0.00           C  \\nATOM   2574  C   ILE A 167      -4.930  -6.165  13.764  1.00  0.00           C  \\nATOM   2575  O   ILE A 167      -4.966  -6.071  14.962  1.00  0.00           O  \\nATOM   2576  CB  ILE A 167      -2.693  -6.450  12.550  1.00  0.00           C  \\nATOM   2577  CG1 ILE A 167      -1.654  -7.516  12.173  1.00  0.00           C  \\nATOM   2578  CG2 ILE A 167      -2.175  -5.408  13.541  1.00  0.00           C  \\nATOM   2579  CD1 ILE A 167      -0.573  -6.935  11.310  1.00  0.00           C  \\nATOM   2580  H   ILE A 167      -4.251  -7.753  11.056  1.00  0.00           H  \\nATOM   2581  HA  ILE A 167      -3.690  -7.896  13.850  1.00  0.00           H  \\nATOM   2582  HB  ILE A 167      -2.929  -5.951  11.611  1.00  0.00           H  \\nATOM   2583 1HG1 ILE A 167      -1.216  -7.933  13.079  1.00  0.00           H  \\nATOM   2584 2HG1 ILE A 167      -2.146  -8.332  11.643  1.00  0.00           H  \\nATOM   2585 1HG2 ILE A 167      -1.284  -4.930  13.134  1.00  0.00           H  \\nATOM   2586 2HG2 ILE A 167      -2.944  -4.656  13.713  1.00  0.00           H  \\nATOM   2587 3HG2 ILE A 167      -1.927  -5.895  14.484  1.00  0.00           H  \\nATOM   2588 1HD1 ILE A 167       0.149  -7.712  11.058  1.00  0.00           H  \\nATOM   2589 2HD1 ILE A 167      -1.012  -6.537  10.395  1.00  0.00           H  \\nATOM   2590 3HD1 ILE A 167      -0.069  -6.134  11.849  1.00  0.00           H  \\nATOM   2591  N   GLN A 168      -5.743  -5.464  12.990  1.00  0.00           N  \\nATOM   2592  CA  GLN A 168      -6.649  -4.482  13.596  1.00  0.00           C  \\nATOM   2593  C   GLN A 168      -7.785  -5.229  14.252  1.00  0.00           C  \\nATOM   2594  O   GLN A 168      -8.142  -4.779  15.321  1.00  0.00           O  \\nATOM   2595  CB  GLN A 168      -7.240  -3.505  12.645  1.00  0.00           C  \\nATOM   2596  CG  GLN A 168      -6.095  -2.601  12.070  1.00  0.00           C  \\nATOM   2597  CD  GLN A 168      -6.638  -1.635  11.130  1.00  0.00           C  \\nATOM   2598  OE1 GLN A 168      -7.091  -0.517  11.553  1.00  0.00           O  \\nATOM   2599  NE2 GLN A 168      -6.785  -2.047   9.881  1.00  0.00           N  \\nATOM   2600  H   GLN A 168      -5.754  -5.590  11.988  1.00  0.00           H  \\nATOM   2601  HA  GLN A 168      -6.089  -3.900  14.328  1.00  0.00           H  \\nATOM   2602 1HB  GLN A 168      -7.749  -4.040  11.843  1.00  0.00           H  \\nATOM   2603 2HB  GLN A 168      -7.986  -2.901  13.161  1.00  0.00           H  \\nATOM   2604 1HG  GLN A 168      -5.615  -2.072  12.894  1.00  0.00           H  \\nATOM   2605 2HG  GLN A 168      -5.369  -3.233  11.559  1.00  0.00           H  \\nATOM   2606 1HE2 GLN A 168      -7.154  -1.425   9.189  1.00  0.00           H  \\nATOM   2607 2HE2 GLN A 168      -6.527  -2.980   9.628  1.00  0.00           H  \\nATOM   2608  N   MET A 169      -8.334  -6.272  13.680  1.00  0.00           N  \\nATOM   2609  CA  MET A 169      -9.509  -6.874  14.294  1.00  0.00           C  \\nATOM   2610  C   MET A 169      -9.169  -7.809  15.459  1.00  0.00           C  \\nATOM   2611  O   MET A 169     -10.048  -8.182  16.165  1.00  0.00           O  \\nATOM   2612  CB  MET A 169     -10.360  -7.609  13.253  1.00  0.00           C  \\nATOM   2613  CG  MET A 169     -10.868  -6.676  12.190  1.00  0.00           C  \\nATOM   2614  SD  MET A 169     -12.033  -7.473  11.078  1.00  0.00           S  \\nATOM   2615  CE  MET A 169     -13.421  -7.665  12.160  1.00  0.00           C  \\nATOM   2616  H   MET A 169      -7.966  -6.669  12.827  1.00  0.00           H  \\nATOM   2617  HA  MET A 169     -10.114  -6.081  14.734  1.00  0.00           H  \\nATOM   2618 1HB  MET A 169      -9.767  -8.395  12.788  1.00  0.00           H  \\nATOM   2619 2HB  MET A 169     -11.207  -8.087  13.747  1.00  0.00           H  \\nATOM   2620 1HG  MET A 169     -11.362  -5.825  12.658  1.00  0.00           H  \\nATOM   2621 2HG  MET A 169     -10.029  -6.302  11.603  1.00  0.00           H  \\nATOM   2622 1HE  MET A 169     -14.238  -8.147  11.622  1.00  0.00           H  \\nATOM   2623 2HE  MET A 169     -13.134  -8.280  13.014  1.00  0.00           H  \\nATOM   2624 3HE  MET A 169     -13.747  -6.685  12.511  1.00  0.00           H  \\nATOM   2625  N   ILE A 170      -7.901  -8.102  15.649  1.00  0.00           N  \\nATOM   2626  CA  ILE A 170      -7.477  -9.011  16.707  1.00  0.00           C  \\nATOM   2627  C   ILE A 170      -6.536  -8.320  17.658  1.00  0.00           C  \\nATOM   2628  O   ILE A 170      -6.887  -8.082  18.812  1.00  0.00           O  \\nATOM   2629  CB  ILE A 170      -6.860 -10.280  16.127  1.00  0.00           C  \\nATOM   2630  CG1 ILE A 170      -7.942 -11.066  15.360  1.00  0.00           C  \\nATOM   2631  CG2 ILE A 170      -6.366 -11.163  17.288  1.00  0.00           C  \\nATOM   2632  CD1 ILE A 170      -7.384 -12.167  14.433  1.00  0.00           C  \\nATOM   2633  H   ILE A 170      -7.203  -7.688  15.048  1.00  0.00           H  \\nATOM   2634  HA  ILE A 170      -8.352  -9.292  17.292  1.00  0.00           H  \\nATOM   2635  HB  ILE A 170      -6.023 -10.015  15.481  1.00  0.00           H  \\nATOM   2636 1HG1 ILE A 170      -8.623 -11.534  16.070  1.00  0.00           H  \\nATOM   2637 2HG1 ILE A 170      -8.529 -10.378  14.751  1.00  0.00           H  \\nATOM   2638 1HG2 ILE A 170      -5.923 -12.075  16.888  1.00  0.00           H  \\nATOM   2639 2HG2 ILE A 170      -5.619 -10.620  17.866  1.00  0.00           H  \\nATOM   2640 3HG2 ILE A 170      -7.207 -11.421  17.932  1.00  0.00           H  \\nATOM   2641 1HD1 ILE A 170      -8.209 -12.672  13.930  1.00  0.00           H  \\nATOM   2642 2HD1 ILE A 170      -6.726 -11.717  13.688  1.00  0.00           H  \\nATOM   2643 3HD1 ILE A 170      -6.823 -12.890  15.024  1.00  0.00           H  \\nATOM   2644  N   SER A 171      -5.336  -7.908  17.213  1.00  0.00           N  \\nATOM   2645  CA  SER A 171      -4.369  -7.270  18.121  1.00  0.00           C  \\nATOM   2646  C   SER A 171      -4.864  -5.929  18.636  1.00  0.00           C  \\nATOM   2647  O   SER A 171      -4.811  -5.653  19.843  1.00  0.00           O  \\nATOM   2648  CB  SER A 171      -3.030  -6.993  17.442  1.00  0.00           C  \\nATOM   2649  OG  SER A 171      -2.439  -8.264  17.071  1.00  0.00           O  \\nATOM   2650  H   SER A 171      -5.089  -8.036  16.242  1.00  0.00           H  \\nATOM   2651  HA  SER A 171      -4.184  -7.944  18.958  1.00  0.00           H  \\nATOM   2652 1HB  SER A 171      -3.189  -6.364  16.566  1.00  0.00           H  \\nATOM   2653 2HB  SER A 171      -2.383  -6.444  18.125  1.00  0.00           H  \\nATOM   2654  HG  SER A 171      -3.060  -8.938  17.358  1.00  0.00           H  \\nATOM   2655  N   GLU A 172      -5.283  -5.041  17.714  1.00  0.00           N  \\nATOM   2656  CA  GLU A 172      -5.690  -3.699  18.164  1.00  0.00           C  \\nATOM   2657  C   GLU A 172      -6.989  -3.723  19.018  1.00  0.00           C  \\nATOM   2658  O   GLU A 172      -7.130  -2.972  19.976  1.00  0.00           O  \\nATOM   2659  CB  GLU A 172      -5.786  -2.711  17.002  1.00  0.00           C  \\nATOM   2660  CG  GLU A 172      -4.566  -2.606  16.139  1.00  0.00           C  \\nATOM   2661  CD  GLU A 172      -3.417  -1.868  16.737  1.00  0.00           C  \\nATOM   2662  OE1 GLU A 172      -3.352  -1.552  17.958  1.00  0.00           O  \\nATOM   2663  OE2 GLU A 172      -2.446  -1.582  15.975  1.00  0.00           O  \\nATOM   2664  H   GLU A 172      -5.327  -5.264  16.730  1.00  0.00           H  \\nATOM   2665  HA  GLU A 172      -4.939  -3.325  18.861  1.00  0.00           H  \\nATOM   2666 1HB  GLU A 172      -6.617  -2.991  16.354  1.00  0.00           H  \\nATOM   2667 2HB  GLU A 172      -5.995  -1.713  17.389  1.00  0.00           H  \\nATOM   2668 1HG  GLU A 172      -4.219  -3.610  15.895  1.00  0.00           H  \\nATOM   2669 2HG  GLU A 172      -4.834  -2.108  15.208  1.00  0.00           H  \\nATOM   2670  N   ALA A 173      -7.932  -4.591  18.636  1.00  0.00           N  \\nATOM   2671  CA  ALA A 173      -9.121  -4.811  19.393  1.00  0.00           C  \\nATOM   2672  C   ALA A 173      -8.782  -5.342  20.799  1.00  0.00           C  \\nATOM   2673  O   ALA A 173      -9.416  -4.917  21.754  1.00  0.00           O  \\nATOM   2674  CB  ALA A 173     -10.087  -5.821  18.657  1.00  0.00           C  \\nATOM   2675  H   ALA A 173      -7.792  -5.107  17.779  1.00  0.00           H  \\nATOM   2676  HA  ALA A 173      -9.635  -3.856  19.501  1.00  0.00           H  \\nATOM   2677 1HB  ALA A 173     -10.985  -5.969  19.258  1.00  0.00           H  \\nATOM   2678 2HB  ALA A 173     -10.364  -5.416  17.684  1.00  0.00           H  \\nATOM   2679 3HB  ALA A 173      -9.580  -6.775  18.521  1.00  0.00           H  \\nATOM   2680  N   ALA A 174      -7.820  -6.279  20.931  1.00  0.00           N  \\nATOM   2681  CA  ALA A 174      -7.382  -6.653  22.289  1.00  0.00           C  \\nATOM   2682  C   ALA A 174      -6.766  -5.492  23.036  1.00  0.00           C  \\nATOM   2683  O   ALA A 174      -7.049  -5.334  24.214  1.00  0.00           O  \\nATOM   2684  CB  ALA A 174      -6.384  -7.797  22.166  1.00  0.00           C  \\nATOM   2685  H   ALA A 174      -7.393  -6.730  20.135  1.00  0.00           H  \\nATOM   2686  HA  ALA A 174      -8.258  -6.982  22.849  1.00  0.00           H  \\nATOM   2687 1HB  ALA A 174      -6.045  -8.092  23.159  1.00  0.00           H  \\nATOM   2688 2HB  ALA A 174      -6.863  -8.647  21.679  1.00  0.00           H  \\nATOM   2689 3HB  ALA A 174      -5.531  -7.473  21.573  1.00  0.00           H  \\nATOM   2690  N   ARG A 175      -5.966  -4.663  22.367  1.00  0.00           N  \\nATOM   2691  CA  ARG A 175      -5.350  -3.488  23.044  1.00  0.00           C  \\nATOM   2692  C   ARG A 175      -6.348  -2.450  23.496  1.00  0.00           C  \\nATOM   2693  O   ARG A 175      -6.100  -1.778  24.458  1.00  0.00           O  \\nATOM   2694  CB  ARG A 175      -4.322  -2.837  22.076  1.00  0.00           C  \\nATOM   2695  CG  ARG A 175      -3.126  -3.724  21.839  1.00  0.00           C  \\nATOM   2696  CD  ARG A 175      -2.333  -2.992  20.718  1.00  0.00           C  \\nATOM   2697  NE  ARG A 175      -0.999  -3.586  20.489  1.00  0.00           N  \\nATOM   2698  CZ  ARG A 175      -0.161  -3.053  19.599  1.00  0.00           C  \\nATOM   2699  NH1 ARG A 175      -0.518  -2.015  18.841  1.00  0.00           N  \\nATOM   2700  NH2 ARG A 175       1.028  -3.555  19.507  1.00  0.00           N  \\nATOM   2701  H   ARG A 175      -5.770  -4.826  21.390  1.00  0.00           H  \\nATOM   2702  HA  ARG A 175      -4.836  -3.836  23.941  1.00  0.00           H  \\nATOM   2703 1HB  ARG A 175      -4.804  -2.626  21.122  1.00  0.00           H  \\nATOM   2704 2HB  ARG A 175      -3.984  -1.886  22.489  1.00  0.00           H  \\nATOM   2705 1HG  ARG A 175      -2.550  -3.812  22.760  1.00  0.00           H  \\nATOM   2706 2HG  ARG A 175      -3.463  -4.713  21.526  1.00  0.00           H  \\nATOM   2707 1HD  ARG A 175      -2.890  -3.047  19.783  1.00  0.00           H  \\nATOM   2708 2HD  ARG A 175      -2.192  -1.948  20.995  1.00  0.00           H  \\nATOM   2709  HE  ARG A 175      -0.750  -4.404  21.029  1.00  0.00           H  \\nATOM   2710 1HH1 ARG A 175      -1.441  -1.615  18.934  1.00  0.00           H  \\nATOM   2711 2HH1 ARG A 175       0.134  -1.629  18.174  1.00  0.00           H  \\nATOM   2712 1HH2 ARG A 175       1.297  -4.328  20.100  1.00  0.00           H  \\nATOM   2713 2HH2 ARG A 175       1.686  -3.175  18.844  1.00  0.00           H  \\nATOM   2714  N   PHE A 176      -7.427  -2.238  22.762  1.00  0.00           N  \\nATOM   2715  CA  PHE A 176      -8.319  -1.089  23.018  1.00  0.00           C  \\nATOM   2716  C   PHE A 176      -9.752  -1.500  23.067  1.00  0.00           C  \\nATOM   2717  O   PHE A 176     -10.288  -1.952  22.071  1.00  0.00           O  \\nATOM   2718  CB  PHE A 176      -8.205  -0.110  21.860  1.00  0.00           C  \\nATOM   2719  CG  PHE A 176      -6.871   0.592  21.765  1.00  0.00           C  \\nATOM   2720  CD1 PHE A 176      -6.596   1.698  22.548  1.00  0.00           C  \\nATOM   2721  CD2 PHE A 176      -5.896   0.206  20.866  1.00  0.00           C  \\nATOM   2722  CE1 PHE A 176      -5.394   2.407  22.421  1.00  0.00           C  \\nATOM   2723  CE2 PHE A 176      -4.693   0.900  20.758  1.00  0.00           C  \\nATOM   2724  CZ  PHE A 176      -4.421   1.996  21.520  1.00  0.00           C  \\nATOM   2725  H   PHE A 176      -7.648  -2.874  22.009  1.00  0.00           H  \\nATOM   2726  HA  PHE A 176      -7.998  -0.600  23.939  1.00  0.00           H  \\nATOM   2727 1HB  PHE A 176      -8.373  -0.636  20.921  1.00  0.00           H  \\nATOM   2728 2HB  PHE A 176      -8.978   0.652  21.951  1.00  0.00           H  \\nATOM   2729  HD1 PHE A 176      -7.332   2.026  23.283  1.00  0.00           H  \\nATOM   2730  HD2 PHE A 176      -6.079  -0.667  20.239  1.00  0.00           H  \\nATOM   2731  HE1 PHE A 176      -5.216   3.291  23.033  1.00  0.00           H  \\nATOM   2732  HE2 PHE A 176      -3.958   0.543  20.037  1.00  0.00           H  \\nATOM   2733  HZ  PHE A 176      -3.484   2.545  21.441  1.00  0.00           H  \\nATOM   2734  N   GLN A 177     -10.434  -1.317  24.217  1.00  0.00           N  \\nATOM   2735  CA  GLN A 177     -11.938  -1.553  24.203  1.00  0.00           C  \\nATOM   2736  C   GLN A 177     -12.624  -0.719  23.154  1.00  0.00           C  \\nATOM   2737  O   GLN A 177     -13.636  -1.159  22.603  1.00  0.00           O  \\nATOM   2738  CB  GLN A 177     -12.478  -1.109  25.570  1.00  0.00           C  \\nATOM   2739  CG  GLN A 177     -12.035  -1.972  26.672  1.00  0.00           C  \\nATOM   2740  CD  GLN A 177     -10.989  -1.224  27.411  1.00  0.00           C  \\nATOM   2741  OE1 GLN A 177     -10.235  -0.467  26.795  1.00  0.00           O  \\nATOM   2742  NE2 GLN A 177     -11.013  -1.293  28.703  1.00  0.00           N  \\nATOM   2743  H   GLN A 177      -9.989  -1.032  25.078  1.00  0.00           H  \\nATOM   2744  HA  GLN A 177     -12.120  -2.617  24.053  1.00  0.00           H  \\nATOM   2745 1HB  GLN A 177     -12.153  -0.089  25.776  1.00  0.00           H  \\nATOM   2746 2HB  GLN A 177     -13.568  -1.107  25.548  1.00  0.00           H  \\nATOM   2747 1HG  GLN A 177     -12.888  -2.190  27.315  1.00  0.00           H  \\nATOM   2748 2HG  GLN A 177     -11.635  -2.897  26.257  1.00  0.00           H  \\nATOM   2749 1HE2 GLN A 177     -10.327  -0.804  29.243  1.00  0.00           H  \\nATOM   2750 2HE2 GLN A 177     -11.717  -1.834  29.162  1.00  0.00           H  \\nATOM   2751  N   TYR A 178     -12.088   0.504  22.829  1.00  0.00           N  \\nATOM   2752  CA  TYR A 178     -12.712   1.352  21.790  1.00  0.00           C  \\nATOM   2753  C   TYR A 178     -12.691   0.681  20.424  1.00  0.00           C  \\nATOM   2754  O   TYR A 178     -13.690   0.719  19.743  1.00  0.00           O  \\nATOM   2755  CB  TYR A 178     -11.935   2.677  21.717  1.00  0.00           C  \\nATOM   2756  CG  TYR A 178     -12.446   3.652  20.721  1.00  0.00           C  \\nATOM   2757  CD1 TYR A 178     -13.547   4.440  21.070  1.00  0.00           C  \\nATOM   2758  CD2 TYR A 178     -11.828   3.863  19.492  1.00  0.00           C  \\nATOM   2759  CE1 TYR A 178     -14.066   5.371  20.135  1.00  0.00           C  \\nATOM   2760  CE2 TYR A 178     -12.327   4.734  18.598  1.00  0.00           C  \\nATOM   2761  CZ  TYR A 178     -13.409   5.527  18.945  1.00  0.00           C  \\nATOM   2762  OH  TYR A 178     -13.860   6.454  18.067  1.00  0.00           O  \\nATOM   2763  H   TYR A 178     -11.258   0.834  23.301  1.00  0.00           H  \\nATOM   2764  HA  TYR A 178     -13.745   1.548  22.078  1.00  0.00           H  \\nATOM   2765 1HB  TYR A 178     -11.954   3.164  22.693  1.00  0.00           H  \\nATOM   2766 2HB  TYR A 178     -10.893   2.474  21.472  1.00  0.00           H  \\nATOM   2767  HD1 TYR A 178     -14.001   4.336  22.056  1.00  0.00           H  \\nATOM   2768  HD2 TYR A 178     -10.920   3.316  19.237  1.00  0.00           H  \\nATOM   2769  HE1 TYR A 178     -14.924   5.989  20.400  1.00  0.00           H  \\nATOM   2770  HE2 TYR A 178     -11.881   4.813  17.606  1.00  0.00           H  \\nATOM   2771  HH  TYR A 178     -13.325   6.426  17.270  1.00  0.00           H  \\nATOM   2772  N   ILE A 179     -11.582  -0.010  20.119  1.00  0.00           N  \\nATOM   2773  CA  ILE A 179     -11.465  -0.671  18.818  1.00  0.00           C  \\nATOM   2774  C   ILE A 179     -12.252  -1.995  18.770  1.00  0.00           C  \\nATOM   2775  O   ILE A 179     -12.891  -2.293  17.733  1.00  0.00           O  \\nATOM   2776  CB  ILE A 179      -9.969  -0.860  18.488  1.00  0.00           C  \\nATOM   2777  CG1 ILE A 179      -9.368   0.562  18.255  1.00  0.00           C  \\nATOM   2778  CG2 ILE A 179      -9.865  -1.622  17.117  1.00  0.00           C  \\nATOM   2779  CD1 ILE A 179      -7.938   0.576  17.768  1.00  0.00           C  \\nATOM   2780  H   ILE A 179     -10.820  -0.082  20.778  1.00  0.00           H  \\nATOM   2781  HA  ILE A 179     -11.925  -0.034  18.064  1.00  0.00           H  \\nATOM   2782  HB  ILE A 179      -9.493  -1.441  19.277  1.00  0.00           H  \\nATOM   2783 1HG1 ILE A 179      -9.972   1.097  17.523  1.00  0.00           H  \\nATOM   2784 2HG1 ILE A 179      -9.405   1.130  19.185  1.00  0.00           H  \\nATOM   2785 1HG2 ILE A 179      -8.816  -1.768  16.860  1.00  0.00           H  \\nATOM   2786 2HG2 ILE A 179     -10.356  -2.591  17.201  1.00  0.00           H  \\nATOM   2787 3HG2 ILE A 179     -10.351  -1.036  16.337  1.00  0.00           H  \\nATOM   2788 1HD1 ILE A 179      -7.609   1.607  17.636  1.00  0.00           H  \\nATOM   2789 2HD1 ILE A 179      -7.299   0.082  18.501  1.00  0.00           H  \\nATOM   2790 3HD1 ILE A 179      -7.873   0.050  16.816  1.00  0.00           H  \\nATOM   2791  N   GLU A 180     -12.289  -2.725  19.872  1.00  0.00           N  \\nATOM   2792  CA  GLU A 180     -13.187  -3.859  20.013  1.00  0.00           C  \\nATOM   2793  C   GLU A 180     -14.613  -3.364  19.801  1.00  0.00           C  \\nATOM   2794  O   GLU A 180     -15.356  -3.963  19.016  1.00  0.00           O  \\nATOM   2795  CB  GLU A 180     -13.051  -4.429  21.447  1.00  0.00           C  \\nATOM   2796  CG  GLU A 180     -14.111  -5.547  21.767  1.00  0.00           C  \\nATOM   2797  CD  GLU A 180     -14.336  -5.713  23.249  1.00  0.00           C  \\nATOM   2798  OE1 GLU A 180     -13.590  -5.122  24.075  1.00  0.00           O  \\nATOM   2799  OE2 GLU A 180     -15.299  -6.491  23.559  1.00  0.00           O  \\nATOM   2800  H   GLU A 180     -11.674  -2.486  20.637  1.00  0.00           H  \\nATOM   2801  HA  GLU A 180     -12.897  -4.622  19.290  1.00  0.00           H  \\nATOM   2802 1HB  GLU A 180     -12.053  -4.848  21.579  1.00  0.00           H  \\nATOM   2803 2HB  GLU A 180     -13.166  -3.623  22.171  1.00  0.00           H  \\nATOM   2804 1HG  GLU A 180     -15.055  -5.286  21.289  1.00  0.00           H  \\nATOM   2805 2HG  GLU A 180     -13.769  -6.490  21.342  1.00  0.00           H  \\nATOM   2806  N   GLY A 181     -14.898  -2.186  20.337  1.00  0.00           N  \\nATOM   2807  CA  GLY A 181     -16.199  -1.618  20.069  1.00  0.00           C  \\nATOM   2808  C   GLY A 181     -16.542  -1.309  18.655  1.00  0.00           C  \\nATOM   2809  O   GLY A 181     -17.686  -1.491  18.255  1.00  0.00           O  \\nATOM   2810  H   GLY A 181     -14.249  -1.676  20.919  1.00  0.00           H  \\nATOM   2811 1HA  GLY A 181     -16.973  -2.296  20.429  1.00  0.00           H  \\nATOM   2812 2HA  GLY A 181     -16.309  -0.684  20.620  1.00  0.00           H  \\nATOM   2813  N   GLU A 182     -15.585  -0.790  17.903  1.00  0.00           N  \\nATOM   2814  CA  GLU A 182     -15.781  -0.509  16.454  1.00  0.00           C  \\nATOM   2815  C   GLU A 182     -16.080  -1.764  15.723  1.00  0.00           C  \\nATOM   2816  O   GLU A 182     -16.870  -1.799  14.688  1.00  0.00           O  \\nATOM   2817  CB  GLU A 182     -14.499   0.044  15.894  1.00  0.00           C  \\nATOM   2818  CG  GLU A 182     -14.179   1.455  16.350  1.00  0.00           C  \\nATOM   2819  CD  GLU A 182     -15.223   2.414  15.803  1.00  0.00           C  \\nATOM   2820  OE1 GLU A 182     -15.290   2.555  14.529  1.00  0.00           O  \\nATOM   2821  OE2 GLU A 182     -16.002   2.928  16.624  1.00  0.00           O  \\nATOM   2822  H   GLU A 182     -14.694  -0.579  18.329  1.00  0.00           H  \\nATOM   2823  HA  GLU A 182     -16.574   0.232  16.349  1.00  0.00           H  \\nATOM   2824 1HB  GLU A 182     -13.668  -0.600  16.183  1.00  0.00           H  \\nATOM   2825 2HB  GLU A 182     -14.548   0.045  14.805  1.00  0.00           H  \\nATOM   2826 1HG  GLU A 182     -14.170   1.480  17.439  1.00  0.00           H  \\nATOM   2827 2HG  GLU A 182     -13.183   1.722  15.999  1.00  0.00           H  \\nATOM   2828  N   MET A 183     -15.412  -2.855  16.101  1.00  0.00           N  \\nATOM   2829  CA  MET A 183     -15.716  -4.115  15.420  1.00  0.00           C  \\nATOM   2830  C   MET A 183     -17.031  -4.707  15.807  1.00  0.00           C  \\nATOM   2831  O   MET A 183     -17.665  -5.338  14.975  1.00  0.00           O  \\nATOM   2832  CB  MET A 183     -14.530  -5.120  15.656  1.00  0.00           C  \\nATOM   2833  CG  MET A 183     -13.153  -4.561  15.277  1.00  0.00           C  \\nATOM   2834  SD  MET A 183     -13.112  -3.968  13.525  1.00  0.00           S  \\nATOM   2835  CE  MET A 183     -12.756  -2.210  13.605  1.00  0.00           C  \\nATOM   2836  H   MET A 183     -14.713  -2.846  16.830  1.00  0.00           H  \\nATOM   2837  HA  MET A 183     -15.815  -3.916  14.353  1.00  0.00           H  \\nATOM   2838 1HB  MET A 183     -14.500  -5.407  16.706  1.00  0.00           H  \\nATOM   2839 2HB  MET A 183     -14.699  -6.026  15.073  1.00  0.00           H  \\nATOM   2840 1HG  MET A 183     -12.903  -3.731  15.937  1.00  0.00           H  \\nATOM   2841 2HG  MET A 183     -12.398  -5.336  15.405  1.00  0.00           H  \\nATOM   2842 1HE  MET A 183     -12.719  -1.800  12.595  1.00  0.00           H  \\nATOM   2843 2HE  MET A 183     -13.538  -1.706  14.174  1.00  0.00           H  \\nATOM   2844 3HE  MET A 183     -11.794  -2.055  14.095  1.00  0.00           H  \\nATOM   2845  N   ARG A 184     -17.393  -4.559  17.081  1.00  0.00           N  \\nATOM   2846  CA  ARG A 184     -18.718  -5.100  17.573  1.00  0.00           C  \\nATOM   2847  C   ARG A 184     -19.829  -4.371  16.748  1.00  0.00           C  \\nATOM   2848  O   ARG A 184     -20.790  -4.985  16.381  1.00  0.00           O  \\nATOM   2849  CB  ARG A 184     -18.925  -4.743  19.078  1.00  0.00           C  \\nATOM   2850  CG  ARG A 184     -18.237  -5.562  20.066  1.00  0.00           C  \\nATOM   2851  CD  ARG A 184     -19.008  -5.407  21.519  1.00  0.00           C  \\nATOM   2852  NE  ARG A 184     -19.273  -4.026  21.885  1.00  0.00           N  \\nATOM   2853  CZ  ARG A 184     -18.407  -3.287  22.600  1.00  0.00           C  \\nATOM   2854  NH1 ARG A 184     -17.286  -3.869  23.007  1.00  0.00           N  \\nATOM   2855  NH2 ARG A 184     -18.619  -1.988  22.913  1.00  0.00           N  \\nATOM   2856  H   ARG A 184     -16.784  -4.083  17.731  1.00  0.00           H  \\nATOM   2857  HA  ARG A 184     -18.709  -6.185  17.469  1.00  0.00           H  \\nATOM   2858 1HB  ARG A 184     -18.602  -3.718  19.256  1.00  0.00           H  \\nATOM   2859 2HB  ARG A 184     -19.986  -4.799  19.323  1.00  0.00           H  \\nATOM   2860 1HG  ARG A 184     -18.254  -6.605  19.749  1.00  0.00           H  \\nATOM   2861 2HG  ARG A 184     -17.203  -5.228  20.160  1.00  0.00           H  \\nATOM   2862 1HD  ARG A 184     -19.966  -5.925  21.480  1.00  0.00           H  \\nATOM   2863 2HD  ARG A 184     -18.392  -5.842  22.306  1.00  0.00           H  \\nATOM   2864  HE  ARG A 184     -20.148  -3.619  21.582  1.00  0.00           H  \\nATOM   2865 1HH1 ARG A 184     -17.107  -4.836  22.777  1.00  0.00           H  \\nATOM   2866 2HH1 ARG A 184     -16.612  -3.345  23.546  1.00  0.00           H  \\nATOM   2867 1HH2 ARG A 184     -19.464  -1.525  22.606  1.00  0.00           H  \\nATOM   2868 2HH2 ARG A 184     -17.934  -1.481  23.453  1.00  0.00           H  \\nATOM   2869  N   THR A 185     -19.688  -3.038  16.582  1.00  0.00           N  \\nATOM   2870  CA  THR A 185     -20.675  -2.236  15.800  1.00  0.00           C  \\nATOM   2871  C   THR A 185     -20.779  -2.713  14.341  1.00  0.00           C  \\nATOM   2872  O   THR A 185     -21.865  -2.847  13.818  1.00  0.00           O  \\nATOM   2873  CB  THR A 185     -20.269  -0.732  15.861  1.00  0.00           C  \\nATOM   2874  OG1 THR A 185     -20.281  -0.259  17.191  1.00  0.00           O  \\nATOM   2875  CG2 THR A 185     -21.254   0.156  15.080  1.00  0.00           C  \\nATOM   2876  H   THR A 185     -18.894  -2.572  16.998  1.00  0.00           H  \\nATOM   2877  HA  THR A 185     -21.659  -2.363  16.251  1.00  0.00           H  \\nATOM   2878  HB  THR A 185     -19.275  -0.606  15.433  1.00  0.00           H  \\nATOM   2879  HG1 THR A 185     -20.539  -0.971  17.781  1.00  0.00           H  \\nATOM   2880 1HG2 THR A 185     -20.937   1.197  15.146  1.00  0.00           H  \\nATOM   2881 2HG2 THR A 185     -21.272  -0.153  14.035  1.00  0.00           H  \\nATOM   2882 3HG2 THR A 185     -22.252   0.054  15.505  1.00  0.00           H  \\nATOM   2883  N   ARG A 186     -19.646  -3.017  13.714  1.00  0.00           N  \\nATOM   2884  CA  ARG A 186     -19.702  -3.539  12.357  1.00  0.00           C  \\nATOM   2885  C   ARG A 186     -20.415  -4.880  12.341  1.00  0.00           C  \\nATOM   2886  O   ARG A 186     -21.187  -5.158  11.393  1.00  0.00           O  \\nATOM   2887  CB  ARG A 186     -18.293  -3.604  11.781  1.00  0.00           C  \\nATOM   2888  CG  ARG A 186     -17.705  -2.185  11.400  1.00  0.00           C  \\nATOM   2889  CD  ARG A 186     -16.284  -2.256  11.221  1.00  0.00           C  \\nATOM   2890  NE  ARG A 186     -15.644  -0.993  10.739  1.00  0.00           N  \\nATOM   2891  CZ  ARG A 186     -15.581   0.111  11.468  1.00  0.00           C  \\nATOM   2892  NH1 ARG A 186     -15.979   0.104  12.745  1.00  0.00           N  \\nATOM   2893  NH2 ARG A 186     -15.021   1.214  11.021  1.00  0.00           N  \\nATOM   2894  H   ARG A 186     -18.747  -2.894  14.157  1.00  0.00           H  \\nATOM   2895  HA  ARG A 186     -20.306  -2.863  11.751  1.00  0.00           H  \\nATOM   2896 1HB  ARG A 186     -17.626  -4.070  12.504  1.00  0.00           H  \\nATOM   2897 2HB  ARG A 186     -18.294  -4.228  10.887  1.00  0.00           H  \\nATOM   2898 1HG  ARG A 186     -18.160  -1.840  10.471  1.00  0.00           H  \\nATOM   2899 2HG  ARG A 186     -17.924  -1.474  12.197  1.00  0.00           H  \\nATOM   2900 1HD  ARG A 186     -15.810  -2.503  12.170  1.00  0.00           H  \\nATOM   2901 2HD  ARG A 186     -16.048  -3.025  10.487  1.00  0.00           H  \\nATOM   2902  HE  ARG A 186     -15.243  -0.994   9.810  1.00  0.00           H  \\nATOM   2903 1HH1 ARG A 186     -16.331  -0.747  13.160  1.00  0.00           H  \\nATOM   2904 2HH1 ARG A 186     -15.928   0.950  13.294  1.00  0.00           H  \\nATOM   2905 1HH2 ARG A 186     -14.621   1.237  10.093  1.00  0.00           H  \\nATOM   2906 2HH2 ARG A 186     -14.991   2.037  11.604  1.00  0.00           H  \\nATOM   2907  N   ILE A 187     -20.124  -5.754  13.295  1.00  0.00           N  \\nATOM   2908  CA  ILE A 187     -20.822  -7.055  13.299  1.00  0.00           C  \\nATOM   2909  C   ILE A 187     -22.306  -6.852  13.610  1.00  0.00           C  \\nATOM   2910  O   ILE A 187     -23.165  -7.482  12.992  1.00  0.00           O  \\nATOM   2911  CB  ILE A 187     -20.170  -7.995  14.283  1.00  0.00           C  \\nATOM   2912  CG1 ILE A 187     -18.801  -8.274  13.715  1.00  0.00           C  \\nATOM   2913  CG2 ILE A 187     -20.960  -9.267  14.416  1.00  0.00           C  \\nATOM   2914  CD1 ILE A 187     -17.776  -8.733  14.793  1.00  0.00           C  \\nATOM   2915  H   ILE A 187     -19.443  -5.559  14.015  1.00  0.00           H  \\nATOM   2916  HA  ILE A 187     -20.752  -7.488  12.302  1.00  0.00           H  \\nATOM   2917  HB  ILE A 187     -20.112  -7.516  15.260  1.00  0.00           H  \\nATOM   2918 1HG1 ILE A 187     -18.875  -9.048  12.952  1.00  0.00           H  \\nATOM   2919 2HG1 ILE A 187     -18.417  -7.375  13.232  1.00  0.00           H  \\nATOM   2920 1HG2 ILE A 187     -20.469  -9.927  15.130  1.00  0.00           H  \\nATOM   2921 2HG2 ILE A 187     -21.965  -9.036  14.767  1.00  0.00           H  \\nATOM   2922 3HG2 ILE A 187     -21.020  -9.761  13.446  1.00  0.00           H  \\nATOM   2923 1HD1 ILE A 187     -16.810  -8.918  14.322  1.00  0.00           H  \\nATOM   2924 2HD1 ILE A 187     -17.668  -7.954  15.548  1.00  0.00           H  \\nATOM   2925 3HD1 ILE A 187     -18.130  -9.649  15.264  1.00  0.00           H  \\nATOM   2926  N   ARG A 188     -22.641  -5.949  14.547  1.00  0.00           N  \\nATOM   2927  CA  ARG A 188     -24.057  -5.707  14.921  1.00  0.00           C  \\nATOM   2928  C   ARG A 188     -24.878  -5.320  13.753  1.00  0.00           C  \\nATOM   2929  O   ARG A 188     -25.980  -5.828  13.610  1.00  0.00           O  \\nATOM   2930  CB  ARG A 188     -24.133  -4.602  15.969  1.00  0.00           C  \\nATOM   2931  CG  ARG A 188     -25.562  -4.167  16.323  1.00  0.00           C  \\nATOM   2932  CD  ARG A 188     -25.699  -3.549  17.714  1.00  0.00           C  \\nATOM   2933  NE  ARG A 188     -24.478  -3.002  18.172  1.00  0.00           N  \\nATOM   2934  CZ  ARG A 188     -23.883  -3.263  19.330  1.00  0.00           C  \\nATOM   2935  NH1 ARG A 188     -24.401  -4.085  20.260  1.00  0.00           N  \\nATOM   2936  NH2 ARG A 188     -22.733  -2.663  19.548  1.00  0.00           N  \\nATOM   2937  H   ARG A 188     -21.913  -5.422  15.008  1.00  0.00           H  \\nATOM   2938  HA  ARG A 188     -24.465  -6.625  15.344  1.00  0.00           H  \\nATOM   2939 1HB  ARG A 188     -23.647  -4.935  16.885  1.00  0.00           H  \\nATOM   2940 2HB  ARG A 188     -23.593  -3.725  15.613  1.00  0.00           H  \\nATOM   2941 1HG  ARG A 188     -25.904  -3.421  15.605  1.00  0.00           H  \\nATOM   2942 2HG  ARG A 188     -26.225  -5.032  16.290  1.00  0.00           H  \\nATOM   2943 1HD  ARG A 188     -26.440  -2.751  17.686  1.00  0.00           H  \\nATOM   2944 2HD  ARG A 188     -26.016  -4.314  18.422  1.00  0.00           H  \\nATOM   2945  HE  ARG A 188     -24.001  -2.346  17.568  1.00  0.00           H  \\nATOM   2946 1HH1 ARG A 188     -25.287  -4.540  20.094  1.00  0.00           H  \\nATOM   2947 2HH1 ARG A 188     -23.904  -4.247  21.124  1.00  0.00           H  \\nATOM   2948 1HH2 ARG A 188     -22.347  -2.041  18.851  1.00  0.00           H  \\nATOM   2949 2HH2 ARG A 188     -22.236  -2.824  20.411  1.00  0.00           H  \\nATOM   2950  N   TYR A 189     -24.358  -4.447  12.907  1.00  0.00           N  \\nATOM   2951  CA  TYR A 189     -25.095  -3.903  11.795  1.00  0.00           C  \\nATOM   2952  C   TYR A 189     -24.796  -4.614  10.493  1.00  0.00           C  \\nATOM   2953  O   TYR A 189     -25.295  -4.228   9.475  1.00  0.00           O  \\nATOM   2954  CB  TYR A 189     -24.831  -2.373  11.710  1.00  0.00           C  \\nATOM   2955  CG  TYR A 189     -25.470  -1.613  12.801  1.00  0.00           C  \\nATOM   2956  CD1 TYR A 189     -26.863  -1.477  12.882  1.00  0.00           C  \\nATOM   2957  CD2 TYR A 189     -24.700  -1.074  13.790  1.00  0.00           C  \\nATOM   2958  CE1 TYR A 189     -27.443  -0.727  13.958  1.00  0.00           C  \\nATOM   2959  CE2 TYR A 189     -25.260  -0.364  14.897  1.00  0.00           C  \\nATOM   2960  CZ  TYR A 189     -26.642  -0.226  14.944  1.00  0.00           C  \\nATOM   2961  OH  TYR A 189     -27.189   0.453  16.017  1.00  0.00           O  \\nATOM   2962  H   TYR A 189     -23.403  -4.155  13.054  1.00  0.00           H  \\nATOM   2963  HA  TYR A 189     -26.158  -4.077  11.966  1.00  0.00           H  \\nATOM   2964 1HB  TYR A 189     -23.757  -2.187  11.741  1.00  0.00           H  \\nATOM   2965 2HB  TYR A 189     -25.202  -1.992  10.759  1.00  0.00           H  \\nATOM   2966  HD1 TYR A 189     -27.501  -1.940  12.129  1.00  0.00           H  \\nATOM   2967  HD2 TYR A 189     -23.615  -1.176  13.753  1.00  0.00           H  \\nATOM   2968  HE1 TYR A 189     -28.525  -0.618  14.025  1.00  0.00           H  \\nATOM   2969  HE2 TYR A 189     -24.607   0.048  15.667  1.00  0.00           H  \\nATOM   2970  HH  TYR A 189     -26.486   0.744  16.603  1.00  0.00           H  \\nATOM   2971  N   ASN A 190     -24.037  -5.709  10.552  1.00  0.00           N  \\nATOM   2972  CA  ASN A 190     -23.642  -6.441   9.323  1.00  0.00           C  \\nATOM   2973  C   ASN A 190     -23.027  -5.571   8.260  1.00  0.00           C  \\nATOM   2974  O   ASN A 190     -23.512  -5.513   7.123  1.00  0.00           O  \\nATOM   2975  CB  ASN A 190     -24.832  -7.215   8.811  1.00  0.00           C  \\nATOM   2976  CG  ASN A 190     -25.071  -8.422   9.618  1.00  0.00           C  \\nATOM   2977  OD1 ASN A 190     -24.154  -9.240   9.858  1.00  0.00           O  \\nATOM   2978  ND2 ASN A 190     -26.278  -8.543  10.097  1.00  0.00           N  \\nATOM   2979  H   ASN A 190     -23.723  -6.049  11.450  1.00  0.00           H  \\nATOM   2980  HA  ASN A 190     -22.840  -7.136   9.577  1.00  0.00           H  \\nATOM   2981 1HB  ASN A 190     -25.718  -6.579   8.835  1.00  0.00           H  \\nATOM   2982 2HB  ASN A 190     -24.662  -7.499   7.772  1.00  0.00           H  \\nATOM   2983 1HD2 ASN A 190     -26.517  -9.338  10.656  1.00  0.00           H  \\nATOM   2984 2HD2 ASN A 190     -26.964  -7.842   9.906  1.00  0.00           H  \\nATOM   2985  N   ARG A 191     -22.102  -4.719   8.708  1.00  0.00           N  \\nATOM   2986  CA  ARG A 191     -21.497  -3.723   7.871  1.00  0.00           C  \\nATOM   2987  C   ARG A 191     -20.041  -3.990   7.759  1.00  0.00           C  \\nATOM   2988  O   ARG A 191     -19.400  -4.588   8.663  1.00  0.00           O  \\nATOM   2989  CB  ARG A 191     -21.701  -2.285   8.410  1.00  0.00           C  \\nATOM   2990  CG  ARG A 191     -23.084  -2.049   8.758  1.00  0.00           C  \\nATOM   2991  CD  ARG A 191     -23.497  -0.584   8.629  1.00  0.00           C  \\nATOM   2992  NE  ARG A 191     -22.417   0.364   8.942  1.00  0.00           N  \\nATOM   2993  CZ  ARG A 191     -22.507   1.415   9.783  1.00  0.00           C  \\nATOM   2994  NH1 ARG A 191     -23.656   1.702  10.427  1.00  0.00           N  \\nATOM   2995  NH2 ARG A 191     -21.445   2.202   9.927  1.00  0.00           N  \\nATOM   2996  H   ARG A 191     -21.822  -4.784   9.676  1.00  0.00           H  \\nATOM   2997  HA  ARG A 191     -21.959  -3.773   6.884  1.00  0.00           H  \\nATOM   2998 1HB  ARG A 191     -21.073  -2.133   9.287  1.00  0.00           H  \\nATOM   2999 2HB  ARG A 191     -21.387  -1.565   7.654  1.00  0.00           H  \\nATOM   3000 1HG  ARG A 191     -23.729  -2.631   8.100  1.00  0.00           H  \\nATOM   3001 2HG  ARG A 191     -23.255  -2.349   9.793  1.00  0.00           H  \\nATOM   3002 1HD  ARG A 191     -23.817  -0.385   7.607  1.00  0.00           H  \\nATOM   3003 2HD  ARG A 191     -24.319  -0.376   9.313  1.00  0.00           H  \\nATOM   3004  HE  ARG A 191     -21.523   0.223   8.491  1.00  0.00           H  \\nATOM   3005 1HH1 ARG A 191     -24.473   1.125  10.284  1.00  0.00           H  \\nATOM   3006 2HH1 ARG A 191     -23.700   2.494  11.052  1.00  0.00           H  \\nATOM   3007 1HH2 ARG A 191     -20.597   2.006   9.413  1.00  0.00           H  \\nATOM   3008 2HH2 ARG A 191     -21.485   2.995  10.550  1.00  0.00           H  \\nATOM   3009  N   ARG A 192     -19.469  -3.544   6.679  1.00  0.00           N  \\nATOM   3010  CA  ARG A 192     -18.039  -3.448   6.544  1.00  0.00           C  \\nATOM   3011  C   ARG A 192     -17.688  -2.023   6.226  1.00  0.00           C  \\nATOM   3012  O   ARG A 192     -18.384  -1.299   5.448  1.00  0.00           O  \\nATOM   3013  CB  ARG A 192     -17.547  -4.426   5.474  1.00  0.00           C  \\nATOM   3014  CG  ARG A 192     -18.155  -4.222   4.078  1.00  0.00           C  \\nATOM   3015  CD  ARG A 192     -17.828  -5.389   3.137  1.00  0.00           C  \\nATOM   3016  NE  ARG A 192     -16.444  -5.364   2.739  1.00  0.00           N  \\nATOM   3017  CZ  ARG A 192     -15.791  -6.420   2.228  1.00  0.00           C  \\nATOM   3018  NH1 ARG A 192     -16.406  -7.607   2.100  1.00  0.00           N  \\nATOM   3019  NH2 ARG A 192     -14.563  -6.294   1.910  1.00  0.00           N  \\nATOM   3020  H   ARG A 192     -20.056  -3.256   5.910  1.00  0.00           H  \\nATOM   3021  HA  ARG A 192     -17.583  -3.712   7.499  1.00  0.00           H  \\nATOM   3022 1HB  ARG A 192     -16.466  -4.343   5.375  1.00  0.00           H  \\nATOM   3023 2HB  ARG A 192     -17.770  -5.447   5.783  1.00  0.00           H  \\nATOM   3024 1HG  ARG A 192     -19.240  -4.145   4.161  1.00  0.00           H  \\nATOM   3025 2HG  ARG A 192     -17.759  -3.307   3.638  1.00  0.00           H  \\nATOM   3026 1HD  ARG A 192     -18.026  -6.332   3.645  1.00  0.00           H  \\nATOM   3027 2HD  ARG A 192     -18.447  -5.321   2.244  1.00  0.00           H  \\nATOM   3028  HE  ARG A 192     -15.937  -4.497   2.854  1.00  0.00           H  \\nATOM   3029 1HH1 ARG A 192     -17.368  -7.712   2.390  1.00  0.00           H  \\nATOM   3030 2HH1 ARG A 192     -15.906  -8.396   1.714  1.00  0.00           H  \\nATOM   3031 1HH2 ARG A 192     -14.094  -5.409   2.045  1.00  0.00           H  \\nATOM   3032 2HH2 ARG A 192     -14.059  -7.079   1.524  1.00  0.00           H  \\nATOM   3033  N   SER A 193     -16.627  -1.568   6.831  1.00  0.00           N  \\nATOM   3034  CA  SER A 193     -16.140  -0.212   6.599  1.00  0.00           C  \\nATOM   3035  C   SER A 193     -14.757   0.037   7.079  1.00  0.00           C  \\nATOM   3036  O   SER A 193     -14.301  -0.660   7.978  1.00  0.00           O  \\nATOM   3037  CB  SER A 193     -17.064   0.787   7.266  1.00  0.00           C  \\nATOM   3038  OG  SER A 193     -17.010   0.668   8.661  1.00  0.00           O  \\nATOM   3039  H   SER A 193     -16.129  -2.166   7.476  1.00  0.00           H  \\nATOM   3040  HA  SER A 193     -16.099  -0.039   5.523  1.00  0.00           H  \\nATOM   3041 1HB  SER A 193     -16.780   1.798   6.973  1.00  0.00           H  \\nATOM   3042 2HB  SER A 193     -18.085   0.623   6.924  1.00  0.00           H  \\nATOM   3043  HG  SER A 193     -16.795   1.543   8.993  1.00  0.00           H  \\nATOM   3044  N   ALA A 194     -14.107   1.041   6.544  1.00  0.00           N  \\nATOM   3045  CA  ALA A 194     -12.776   1.356   6.970  1.00  0.00           C  \\nATOM   3046  C   ALA A 194     -12.773   1.933   8.395  1.00  0.00           C  \\nATOM   3047  O   ALA A 194     -13.816   2.505   8.751  1.00  0.00           O  \\nATOM   3048  CB  ALA A 194     -12.156   2.339   5.962  1.00  0.00           C  \\nATOM   3049  H   ALA A 194     -14.539   1.602   5.824  1.00  0.00           H  \\nATOM   3050  HA  ALA A 194     -12.198   0.432   6.986  1.00  0.00           H  \\nATOM   3051 1HB  ALA A 194     -11.142   2.588   6.274  1.00  0.00           H  \\nATOM   3052 2HB  ALA A 194     -12.129   1.878   4.974  1.00  0.00           H  \\nATOM   3053 3HB  ALA A 194     -12.757   3.246   5.923  1.00  0.00           H  \\nATOM   3054  N   PRO A 195     -11.558   1.878   9.191  1.00  0.00           N  \\nATOM   3055  CA  PRO A 195     -11.440   2.600  10.459  1.00  0.00           C  \\nATOM   3056  C   PRO A 195     -11.449   4.065  10.265  1.00  0.00           C  \\nATOM   3057  O   PRO A 195     -10.931   4.526   9.269  1.00  0.00           O  \\nATOM   3058  CB  PRO A 195     -10.032   2.247  10.925  1.00  0.00           C  \\nATOM   3059  CG  PRO A 195      -9.333   1.749   9.774  1.00  0.00           C  \\nATOM   3060  CD  PRO A 195     -10.331   1.156   8.877  1.00  0.00           C  \\nATOM   3061  HA  PRO A 195     -12.195   2.220  11.163  1.00  0.00           H  \\nATOM   3062 1HB  PRO A 195      -9.541   3.136  11.346  1.00  0.00           H  \\nATOM   3063 2HB  PRO A 195     -10.079   1.495  11.727  1.00  0.00           H  \\nATOM   3064 1HG  PRO A 195      -8.789   2.567   9.277  1.00  0.00           H  \\nATOM   3065 2HG  PRO A 195      -8.580   1.006  10.078  1.00  0.00           H  \\nATOM   3066 1HD  PRO A 195     -10.028   1.318   7.832  1.00  0.00           H  \\nATOM   3067 2HD  PRO A 195     -10.424   0.081   9.092  1.00  0.00           H  \\nATOM   3068  N   ASP A 196     -11.956   4.807  11.256  1.00  0.00           N  \\nATOM   3069  CA  ASP A 196     -11.955   6.268  11.130  1.00  0.00           C  \\nATOM   3070  C   ASP A 196     -10.672   6.856  11.714  1.00  0.00           C  \\nATOM   3071  O   ASP A 196      -9.906   6.117  12.315  1.00  0.00           O  \\nATOM   3072  CB  ASP A 196     -13.269   6.866  11.692  1.00  0.00           C  \\nATOM   3073  CG  ASP A 196     -13.435   6.688  13.191  1.00  0.00           C  \\nATOM   3074  OD1 ASP A 196     -12.484   6.509  13.912  1.00  0.00           O  \\nATOM   3075  OD2 ASP A 196     -14.596   6.761  13.650  1.00  0.00           O  \\nATOM   3076  H   ASP A 196     -12.339   4.386  12.090  1.00  0.00           H  \\nATOM   3077  HA  ASP A 196     -11.883   6.523  10.072  1.00  0.00           H  \\nATOM   3078 1HB  ASP A 196     -13.306   7.933  11.471  1.00  0.00           H  \\nATOM   3079 2HB  ASP A 196     -14.122   6.400  11.198  1.00  0.00           H  \\nATOM   3080  N   PRO A 197     -10.409   8.181  11.561  1.00  0.00           N  \\nATOM   3081  CA  PRO A 197      -9.186   8.701  12.112  1.00  0.00           C  \\nATOM   3082  C   PRO A 197      -9.011   8.425  13.564  1.00  0.00           C  \\nATOM   3083  O   PRO A 197      -7.865   8.483  14.037  1.00  0.00           O  \\nATOM   3084  CB  PRO A 197      -9.319  10.205  11.854  1.00  0.00           C  \\nATOM   3085  CG  PRO A 197     -10.167  10.292  10.632  1.00  0.00           C  \\nATOM   3086  CD  PRO A 197     -11.173   9.183  10.794  1.00  0.00           C  \\nATOM   3087  HA  PRO A 197      -8.336   8.326  11.524  1.00  0.00           H  \\nATOM   3088 1HB  PRO A 197      -9.774  10.698  12.726  1.00  0.00           H  \\nATOM   3089 2HB  PRO A 197      -8.324  10.652  11.716  1.00  0.00           H  \\nATOM   3090 1HG  PRO A 197     -10.638  11.284  10.566  1.00  0.00           H  \\nATOM   3091 2HG  PRO A 197      -9.548  10.174   9.730  1.00  0.00           H  \\nATOM   3092 1HD  PRO A 197     -12.044   9.556  11.352  1.00  0.00           H  \\nATOM   3093 2HD  PRO A 197     -11.476   8.815   9.802  1.00  0.00           H  \\nATOM   3094  N   SER A 198     -10.103   8.380  14.357  1.00  0.00           N  \\nATOM   3095  CA  SER A 198      -9.953   8.129  15.741  1.00  0.00           C  \\nATOM   3096  C   SER A 198      -9.289   6.807  15.997  1.00  0.00           C  \\nATOM   3097  O   SER A 198      -8.423   6.687  16.852  1.00  0.00           O  \\nATOM   3098  CB  SER A 198     -11.300   8.253  16.524  1.00  0.00           C  \\nATOM   3099  OG  SER A 198     -12.134   7.187  16.168  1.00  0.00           O  \\nATOM   3100  H   SER A 198     -11.029   8.523  13.980  1.00  0.00           H  \\nATOM   3101  HA  SER A 198      -9.263   8.866  16.154  1.00  0.00           H  \\nATOM   3102 1HB  SER A 198     -11.100   8.244  17.595  1.00  0.00           H  \\nATOM   3103 2HB  SER A 198     -11.772   9.206  16.287  1.00  0.00           H  \\nATOM   3104  HG  SER A 198     -11.640   6.666  15.531  1.00  0.00           H  \\nATOM   3105  N   VAL A 199      -9.690   5.823  15.180  1.00  0.00           N  \\nATOM   3106  CA  VAL A 199      -9.118   4.462  15.352  1.00  0.00           C  \\nATOM   3107  C   VAL A 199      -7.691   4.388  14.958  1.00  0.00           C  \\nATOM   3108  O   VAL A 199      -6.842   3.880  15.688  1.00  0.00           O  \\nATOM   3109  CB  VAL A 199      -9.894   3.490  14.500  1.00  0.00           C  \\nATOM   3110  CG1 VAL A 199      -9.254   2.121  14.579  1.00  0.00           C  \\nATOM   3111  CG2 VAL A 199     -11.374   3.493  14.941  1.00  0.00           C  \\nATOM   3112  H   VAL A 199     -10.370   5.982  14.451  1.00  0.00           H  \\nATOM   3113  HA  VAL A 199      -9.212   4.177  16.401  1.00  0.00           H  \\nATOM   3114  HB  VAL A 199      -9.819   3.796  13.456  1.00  0.00           H  \\nATOM   3115 1HG1 VAL A 199      -9.817   1.420  13.962  1.00  0.00           H  \\nATOM   3116 2HG1 VAL A 199      -8.227   2.178  14.218  1.00  0.00           H  \\nATOM   3117 3HG1 VAL A 199      -9.258   1.777  15.613  1.00  0.00           H  \\nATOM   3118 1HG2 VAL A 199     -11.938   2.792  14.327  1.00  0.00           H  \\nATOM   3119 2HG2 VAL A 199     -11.442   3.196  15.988  1.00  0.00           H  \\nATOM   3120 3HG2 VAL A 199     -11.788   4.494  14.821  1.00  0.00           H  \\nATOM   3121  N   ILE A 200      -7.362   5.019  13.810  1.00  0.00           N  \\nATOM   3122  CA  ILE A 200      -6.050   5.007  13.280  1.00  0.00           C  \\nATOM   3123  C   ILE A 200      -5.057   5.710  14.220  1.00  0.00           C  \\nATOM   3124  O   ILE A 200      -3.963   5.190  14.499  1.00  0.00           O  \\nATOM   3125  CB  ILE A 200      -6.057   5.601  11.822  1.00  0.00           C  \\nATOM   3126  CG1 ILE A 200      -6.799   4.583  10.954  1.00  0.00           C  \\nATOM   3127  CG2 ILE A 200      -4.639   5.828  11.371  1.00  0.00           C  \\nATOM   3128  CD1 ILE A 200      -7.417   5.244   9.675  1.00  0.00           C  \\nATOM   3129  H   ILE A 200      -8.088   5.518  13.315  1.00  0.00           H  \\nATOM   3130  HA  ILE A 200      -5.703   3.975  13.240  1.00  0.00           H  \\nATOM   3131  HB  ILE A 200      -6.597   6.547  11.817  1.00  0.00           H  \\nATOM   3132 1HG1 ILE A 200      -6.112   3.792  10.654  1.00  0.00           H  \\nATOM   3133 2HG1 ILE A 200      -7.595   4.119  11.537  1.00  0.00           H  \\nATOM   3134 1HG2 ILE A 200      -4.640   6.239  10.362  1.00  0.00           H  \\nATOM   3135 2HG2 ILE A 200      -4.150   6.528  12.047  1.00  0.00           H  \\nATOM   3136 3HG2 ILE A 200      -4.099   4.881  11.377  1.00  0.00           H  \\nATOM   3137 1HD1 ILE A 200      -7.935   4.486   9.088  1.00  0.00           H  \\nATOM   3138 2HD1 ILE A 200      -8.125   6.019   9.972  1.00  0.00           H  \\nATOM   3139 3HD1 ILE A 200      -6.623   5.687   9.075  1.00  0.00           H  \\nATOM   3140  N   THR A 201      -5.420   6.909  14.677  1.00  0.00           N  \\nATOM   3141  CA  THR A 201      -4.617   7.625  15.632  1.00  0.00           C  \\nATOM   3142  C   THR A 201      -4.393   6.889  17.010  1.00  0.00           C  \\nATOM   3143  O   THR A 201      -3.272   6.870  17.573  1.00  0.00           O  \\nATOM   3144  CB  THR A 201      -5.121   9.060  15.834  1.00  0.00           C  \\nATOM   3145  OG1 THR A 201      -4.865   9.712  14.577  1.00  0.00           O  \\nATOM   3146  CG2 THR A 201      -4.355   9.711  16.860  1.00  0.00           C  \\nATOM   3147  H   THR A 201      -6.279   7.325  14.346  1.00  0.00           H  \\nATOM   3148  HA  THR A 201      -3.595   7.676  15.256  1.00  0.00           H  \\nATOM   3149  HB  THR A 201      -6.171   9.039  16.124  1.00  0.00           H  \\nATOM   3150  HG1 THR A 201      -4.459   9.088  13.970  1.00  0.00           H  \\nATOM   3151 1HG2 THR A 201      -4.720  10.729  16.996  1.00  0.00           H  \\nATOM   3152 2HG2 THR A 201      -4.457   9.160  17.795  1.00  0.00           H  \\nATOM   3153 3HG2 THR A 201      -3.306   9.739  16.568  1.00  0.00           H  \\nATOM   3154  N   LEU A 202      -5.408   6.189  17.533  1.00  0.00           N  \\nATOM   3155  CA  LEU A 202      -5.160   5.366  18.694  1.00  0.00           C  \\nATOM   3156  C   LEU A 202      -4.181   4.289  18.411  1.00  0.00           C  \\nATOM   3157  O   LEU A 202      -3.282   4.059  19.181  1.00  0.00           O  \\nATOM   3158  CB  LEU A 202      -6.467   4.748  19.193  1.00  0.00           C  \\nATOM   3159  CG  LEU A 202      -7.338   5.638  20.089  1.00  0.00           C  \\nATOM   3160  CD1 LEU A 202      -8.631   4.951  20.266  1.00  0.00           C  \\nATOM   3161  CD2 LEU A 202      -6.708   5.761  21.453  1.00  0.00           C  \\nATOM   3162  H   LEU A 202      -6.338   6.219  17.141  1.00  0.00           H  \\nATOM   3163  HA  LEU A 202      -4.747   5.995  19.482  1.00  0.00           H  \\nATOM   3164 1HB  LEU A 202      -7.069   4.465  18.331  1.00  0.00           H  \\nATOM   3165 2HB  LEU A 202      -6.232   3.846  19.758  1.00  0.00           H  \\nATOM   3166  HG  LEU A 202      -7.426   6.629  19.642  1.00  0.00           H  \\nATOM   3167 1HD1 LEU A 202      -9.280   5.555  20.900  1.00  0.00           H  \\nATOM   3168 2HD1 LEU A 202      -9.104   4.809  19.294  1.00  0.00           H  \\nATOM   3169 3HD1 LEU A 202      -8.467   3.982  20.735  1.00  0.00           H  \\nATOM   3170 1HD2 LEU A 202      -7.330   6.395  22.086  1.00  0.00           H  \\nATOM   3171 2HD2 LEU A 202      -6.621   4.772  21.904  1.00  0.00           H  \\nATOM   3172 3HD2 LEU A 202      -5.717   6.205  21.357  1.00  0.00           H  \\nATOM   3173  N   GLU A 203      -4.364   3.516  17.318  1.00  0.00           N  \\nATOM   3174  CA  GLU A 203      -3.372   2.493  16.977  1.00  0.00           C  \\nATOM   3175  C   GLU A 203      -1.962   3.066  16.930  1.00  0.00           C  \\nATOM   3176  O   GLU A 203      -1.030   2.446  17.386  1.00  0.00           O  \\nATOM   3177  CB  GLU A 203      -3.704   1.871  15.578  1.00  0.00           C  \\nATOM   3178  CG  GLU A 203      -5.090   1.198  15.526  1.00  0.00           C  \\nATOM   3179  CD  GLU A 203      -5.532   0.785  14.153  1.00  0.00           C  \\nATOM   3180  OE1 GLU A 203      -4.925   1.221  13.173  1.00  0.00           O  \\nATOM   3181  OE2 GLU A 203      -6.521   0.076  14.033  1.00  0.00           O  \\nATOM   3182  H   GLU A 203      -5.176   3.630  16.728  1.00  0.00           H  \\nATOM   3183  HA  GLU A 203      -3.412   1.707  17.732  1.00  0.00           H  \\nATOM   3184 1HB  GLU A 203      -3.669   2.650  14.816  1.00  0.00           H  \\nATOM   3185 2HB  GLU A 203      -2.949   1.128  15.322  1.00  0.00           H  \\nATOM   3186 1HG  GLU A 203      -5.074   0.309  16.156  1.00  0.00           H  \\nATOM   3187 2HG  GLU A 203      -5.831   1.885  15.933  1.00  0.00           H  \\nATOM   3188  N   ASN A 204      -1.815   4.191  16.254  1.00  0.00           N  \\nATOM   3189  CA  ASN A 204      -0.501   4.788  16.166  1.00  0.00           C  \\nATOM   3190  C   ASN A 204       0.082   5.240  17.496  1.00  0.00           C  \\nATOM   3191  O   ASN A 204       1.333   5.348  17.580  1.00  0.00           O  \\nATOM   3192  CB  ASN A 204      -0.544   6.014  15.231  1.00  0.00           C  \\nATOM   3193  CG  ASN A 204      -0.772   5.630  13.738  1.00  0.00           C  \\nATOM   3194  OD1 ASN A 204      -0.599   4.459  13.361  1.00  0.00           O  \\nATOM   3195  ND2 ASN A 204      -1.184   6.574  12.918  1.00  0.00           N  \\nATOM   3196  H   ASN A 204      -2.593   4.645  15.798  1.00  0.00           H  \\nATOM   3197  HA  ASN A 204       0.187   4.050  15.751  1.00  0.00           H  \\nATOM   3198 1HB  ASN A 204      -1.345   6.684  15.545  1.00  0.00           H  \\nATOM   3199 2HB  ASN A 204       0.393   6.565  15.310  1.00  0.00           H  \\nATOM   3200 1HD2 ASN A 204      -1.341   6.364  11.952  1.00  0.00           H  \\nATOM   3201 2HD2 ASN A 204      -1.340   7.500  13.260  1.00  0.00           H  \\nATOM   3202  N   SER A 205      -0.748   5.438  18.516  1.00  0.00           N  \\nATOM   3203  CA  SER A 205      -0.313   6.073  19.740  1.00  0.00           C  \\nATOM   3204  C   SER A 205      -0.235   5.075  20.899  1.00  0.00           C  \\nATOM   3205  O   SER A 205       0.110   5.475  22.029  1.00  0.00           O  \\nATOM   3206  CB  SER A 205      -1.303   7.130  20.155  1.00  0.00           C  \\nATOM   3207  OG  SER A 205      -1.504   8.132  19.115  1.00  0.00           O  \\nATOM   3208  H   SER A 205      -1.708   5.136  18.433  1.00  0.00           H  \\nATOM   3209  HA  SER A 205       0.653   6.547  19.560  1.00  0.00           H  \\nATOM   3210 1HB  SER A 205      -2.259   6.662  20.388  1.00  0.00           H  \\nATOM   3211 2HB  SER A 205      -0.951   7.622  21.060  1.00  0.00           H  \\nATOM   3212  HG  SER A 205      -0.938   7.873  18.384  1.00  0.00           H  \\nATOM   3213  N   TRP A 206      -0.465   3.803  20.615  1.00  0.00           N  \\nATOM   3214  CA  TRP A 206      -0.550   2.848  21.726  1.00  0.00           C  \\nATOM   3215  C   TRP A 206       0.694   2.801  22.528  1.00  0.00           C  \\nATOM   3216  O   TRP A 206       0.659   2.753  23.741  1.00  0.00           O  \\nATOM   3217  CB  TRP A 206      -0.773   1.452  21.130  1.00  0.00           C  \\nATOM   3218  CG  TRP A 206      -0.969   0.391  22.196  1.00  0.00           C  \\nATOM   3219  CD1 TRP A 206      -2.042   0.279  23.047  1.00  0.00           C  \\nATOM   3220  CD2 TRP A 206      -0.089  -0.653  22.521  1.00  0.00           C  \\nATOM   3221  NE1 TRP A 206      -1.876  -0.782  23.901  1.00  0.00           N  \\nATOM   3222  CE2 TRP A 206      -0.681  -1.392  23.558  1.00  0.00           C  \\nATOM   3223  CE3 TRP A 206       1.134  -1.061  22.017  1.00  0.00           C  \\nATOM   3224  CZ2 TRP A 206      -0.072  -2.513  24.110  1.00  0.00           C  \\nATOM   3225  CZ3 TRP A 206       1.744  -2.214  22.606  1.00  0.00           C  \\nATOM   3226  CH2 TRP A 206       1.128  -2.916  23.593  1.00  0.00           C  \\nATOM   3227  H   TRP A 206      -0.583   3.469  19.669  1.00  0.00           H  \\nATOM   3228  HA  TRP A 206      -1.397   3.123  22.354  1.00  0.00           H  \\nATOM   3229 1HB  TRP A 206      -1.650   1.468  20.483  1.00  0.00           H  \\nATOM   3230 2HB  TRP A 206       0.083   1.179  20.514  1.00  0.00           H  \\nATOM   3231  HD1 TRP A 206      -2.908   0.939  23.048  1.00  0.00           H  \\nATOM   3232  HE1 TRP A 206      -2.515  -1.052  24.635  1.00  0.00           H  \\nATOM   3233  HE3 TRP A 206       1.621  -0.531  21.199  1.00  0.00           H  \\nATOM   3234  HZ2 TRP A 206      -0.522  -3.071  24.931  1.00  0.00           H  \\nATOM   3235  HZ3 TRP A 206       2.725  -2.515  22.237  1.00  0.00           H  \\nATOM   3236  HH2 TRP A 206       1.602  -3.818  23.980  1.00  0.00           H  \\nATOM   3237  N   GLY A 207       1.861   2.752  21.896  1.00  0.00           N  \\nATOM   3238  CA  GLY A 207       3.094   2.719  22.673  1.00  0.00           C  \\nATOM   3239  C   GLY A 207       3.293   3.977  23.523  1.00  0.00           C  \\nATOM   3240  O   GLY A 207       3.689   3.901  24.681  1.00  0.00           O  \\nATOM   3241  H   GLY A 207       1.914   2.737  20.888  1.00  0.00           H  \\nATOM   3242 1HA  GLY A 207       3.088   1.848  23.328  1.00  0.00           H  \\nATOM   3243 2HA  GLY A 207       3.944   2.609  22.001  1.00  0.00           H  \\nATOM   3244  N   ARG A 208       3.015   5.082  22.912  1.00  0.00           N  \\nATOM   3245  CA  ARG A 208       3.173   6.406  23.616  1.00  0.00           C  \\nATOM   3246  C   ARG A 208       2.164   6.537  24.730  1.00  0.00           C  \\nATOM   3247  O   ARG A 208       2.486   7.092  25.757  1.00  0.00           O  \\nATOM   3248  CB  ARG A 208       3.077   7.555  22.646  1.00  0.00           C  \\nATOM   3249  CG  ARG A 208       4.345   7.702  21.869  1.00  0.00           C  \\nATOM   3250  CD  ARG A 208       4.283   8.893  20.809  1.00  0.00           C  \\nATOM   3251  NE  ARG A 208       3.114   8.961  19.907  1.00  0.00           N  \\nATOM   3252  CZ  ARG A 208       2.919   8.257  18.793  1.00  0.00           C  \\nATOM   3253  NH1 ARG A 208       3.794   7.331  18.387  1.00  0.00           N  \\nATOM   3254  NH2 ARG A 208       1.828   8.504  18.025  1.00  0.00           N  \\nATOM   3255  H   ARG A 208       2.689   5.069  21.956  1.00  0.00           H  \\nATOM   3256  HA  ARG A 208       4.158   6.434  24.083  1.00  0.00           H  \\nATOM   3257 1HB  ARG A 208       2.245   7.386  21.964  1.00  0.00           H  \\nATOM   3258 2HB  ARG A 208       2.871   8.476  23.192  1.00  0.00           H  \\nATOM   3259 1HG  ARG A 208       5.169   7.906  22.553  1.00  0.00           H  \\nATOM   3260 2HG  ARG A 208       4.547   6.780  21.323  1.00  0.00           H  \\nATOM   3261 1HD  ARG A 208       4.290   9.848  21.334  1.00  0.00           H  \\nATOM   3262 2HD  ARG A 208       5.147   8.836  20.148  1.00  0.00           H  \\nATOM   3263  HE  ARG A 208       2.373   9.606  20.146  1.00  0.00           H  \\nATOM   3264 1HH1 ARG A 208       4.627   7.151  18.929  1.00  0.00           H  \\nATOM   3265 2HH1 ARG A 208       3.622   6.811  17.539  1.00  0.00           H  \\nATOM   3266 1HH2 ARG A 208       1.168   9.219  18.299  1.00  0.00           H  \\nATOM   3267 2HH2 ARG A 208       1.675   7.974  17.180  1.00  0.00           H  \\nATOM   3268  N   LEU A 209       0.966   6.000  24.570  1.00  0.00           N  \\nATOM   3269  CA  LEU A 209      -0.045   5.975  25.647  1.00  0.00           C  \\nATOM   3270  C   LEU A 209       0.424   5.149  26.775  1.00  0.00           C  \\nATOM   3271  O   LEU A 209       0.221   5.519  27.967  1.00  0.00           O  \\nATOM   3272  CB  LEU A 209      -1.438   5.528  25.112  1.00  0.00           C  \\nATOM   3273  CG  LEU A 209      -2.094   6.517  24.171  1.00  0.00           C  \\nATOM   3274  CD1 LEU A 209      -3.348   5.945  23.367  1.00  0.00           C  \\nATOM   3275  CD2 LEU A 209      -2.520   7.733  24.942  1.00  0.00           C  \\nATOM   3276  H   LEU A 209       0.740   5.593  23.674  1.00  0.00           H  \\nATOM   3277  HA  LEU A 209      -0.144   6.982  26.052  1.00  0.00           H  \\nATOM   3278 1HB  LEU A 209      -1.321   4.581  24.588  1.00  0.00           H  \\nATOM   3279 2HB  LEU A 209      -2.102   5.370  25.962  1.00  0.00           H  \\nATOM   3280  HG  LEU A 209      -1.384   6.808  23.396  1.00  0.00           H  \\nATOM   3281 1HD1 LEU A 209      -3.751   6.724  22.719  1.00  0.00           H  \\nATOM   3282 2HD1 LEU A 209      -3.034   5.096  22.760  1.00  0.00           H  \\nATOM   3283 3HD1 LEU A 209      -4.116   5.625  24.070  1.00  0.00           H  \\nATOM   3284 1HD2 LEU A 209      -2.992   8.446  24.265  1.00  0.00           H  \\nATOM   3285 2HD2 LEU A 209      -3.231   7.443  25.716  1.00  0.00           H  \\nATOM   3286 3HD2 LEU A 209      -1.648   8.194  25.405  1.00  0.00           H  \\nATOM   3287  N   SER A 210       0.968   4.003  26.482  1.00  0.00           N  \\nATOM   3288  CA  SER A 210       1.488   3.099  27.493  1.00  0.00           C  \\nATOM   3289  C   SER A 210       2.551   3.786  28.331  1.00  0.00           C  \\nATOM   3290  O   SER A 210       2.539   3.681  29.534  1.00  0.00           O  \\nATOM   3291  CB  SER A 210       2.070   1.814  26.801  1.00  0.00           C  \\nATOM   3292  OG  SER A 210       1.030   1.061  26.186  1.00  0.00           O  \\nATOM   3293  H   SER A 210       1.027   3.739  25.509  1.00  0.00           H  \\nATOM   3294  HA  SER A 210       0.669   2.809  28.152  1.00  0.00           H  \\nATOM   3295 1HB  SER A 210       2.807   2.106  26.053  1.00  0.00           H  \\nATOM   3296 2HB  SER A 210       2.581   1.202  27.543  1.00  0.00           H  \\nATOM   3297  HG  SER A 210       0.218   1.544  26.356  1.00  0.00           H  \\nATOM   3298  N   THR A 211       3.431   4.500  27.678  1.00  0.00           N  \\nATOM   3299  CA  THR A 211       4.469   5.189  28.392  1.00  0.00           C  \\nATOM   3300  C   THR A 211       3.854   6.389  29.197  1.00  0.00           C  \\nATOM   3301  O   THR A 211       4.174   6.483  30.435  1.00  0.00           O  \\nATOM   3302  CB  THR A 211       5.495   5.754  27.425  1.00  0.00           C  \\nATOM   3303  OG1 THR A 211       6.101   4.672  26.717  1.00  0.00           O  \\nATOM   3304  CG2 THR A 211       6.534   6.474  28.184  1.00  0.00           C  \\nATOM   3305  H   THR A 211       3.393   4.576  26.672  1.00  0.00           H  \\nATOM   3306  HA  THR A 211       4.967   4.477  29.051  1.00  0.00           H  \\nATOM   3307  HB  THR A 211       5.004   6.437  26.732  1.00  0.00           H  \\nATOM   3308  HG1 THR A 211       5.721   3.842  27.015  1.00  0.00           H  \\nATOM   3309 1HG2 THR A 211       7.272   6.881  27.494  1.00  0.00           H  \\nATOM   3310 2HG2 THR A 211       6.076   7.287  28.747  1.00  0.00           H  \\nATOM   3311 3HG2 THR A 211       7.023   5.787  28.873  1.00  0.00           H  \\nATOM   3312  N   ALA A 212       2.988   7.171  28.570  1.00  0.00           N  \\nATOM   3313  CA  ALA A 212       2.441   8.336  29.311  1.00  0.00           C  \\nATOM   3314  C   ALA A 212       1.685   7.856  30.555  1.00  0.00           C  \\nATOM   3315  O   ALA A 212       1.827   8.451  31.645  1.00  0.00           O  \\nATOM   3316  CB  ALA A 212       1.536   9.128  28.441  1.00  0.00           C  \\nATOM   3317  H   ALA A 212       2.689   7.008  27.619  1.00  0.00           H  \\nATOM   3318  HA  ALA A 212       3.277   8.966  29.616  1.00  0.00           H  \\nATOM   3319 1HB  ALA A 212       1.145   9.978  29.001  1.00  0.00           H  \\nATOM   3320 2HB  ALA A 212       2.089   9.489  27.573  1.00  0.00           H  \\nATOM   3321 3HB  ALA A 212       0.710   8.501  28.109  1.00  0.00           H  \\nATOM   3322  N   ILE A 213       0.920   6.774  30.497  1.00  0.00           N  \\nATOM   3323  CA  ILE A 213       0.206   6.329  31.660  1.00  0.00           C  \\nATOM   3324  C   ILE A 213       1.104   5.790  32.684  1.00  0.00           C  \\nATOM   3325  O   ILE A 213       0.983   6.142  33.882  1.00  0.00           O  \\nATOM   3326  CB  ILE A 213      -0.829   5.237  31.241  1.00  0.00           C  \\nATOM   3327  CG1 ILE A 213      -1.928   5.918  30.416  1.00  0.00           C  \\nATOM   3328  CG2 ILE A 213      -1.417   4.485  32.442  1.00  0.00           C  \\nATOM   3329  CD1 ILE A 213      -2.787   4.921  29.593  1.00  0.00           C  \\nATOM   3330  H   ILE A 213       0.832   6.253  29.636  1.00  0.00           H  \\nATOM   3331  HA  ILE A 213      -0.323   7.180  32.088  1.00  0.00           H  \\nATOM   3332  HB  ILE A 213      -0.345   4.506  30.595  1.00  0.00           H  \\nATOM   3333 1HG1 ILE A 213      -2.587   6.477  31.079  1.00  0.00           H  \\nATOM   3334 2HG1 ILE A 213      -1.476   6.633  29.728  1.00  0.00           H  \\nATOM   3335 1HG2 ILE A 213      -2.130   3.739  32.091  1.00  0.00           H  \\nATOM   3336 2HG2 ILE A 213      -0.616   3.991  32.989  1.00  0.00           H  \\nATOM   3337 3HG2 ILE A 213      -1.924   5.191  33.100  1.00  0.00           H  \\nATOM   3338 1HD1 ILE A 213      -3.545   5.470  29.033  1.00  0.00           H  \\nATOM   3339 2HD1 ILE A 213      -2.147   4.375  28.899  1.00  0.00           H  \\nATOM   3340 3HD1 ILE A 213      -3.274   4.218  30.267  1.00  0.00           H  \\nATOM   3341  N   GLN A 214       2.011   4.890  32.333  1.00  0.00           N  \\nATOM   3342  CA  GLN A 214       2.940   4.326  33.324  1.00  0.00           C  \\nATOM   3343  C   GLN A 214       3.948   5.302  33.921  1.00  0.00           C  \\nATOM   3344  O   GLN A 214       4.302   5.079  35.101  1.00  0.00           O  \\nATOM   3345  CB  GLN A 214       3.655   3.099  32.676  1.00  0.00           C  \\nATOM   3346  CG  GLN A 214       2.737   1.944  32.650  1.00  0.00           C  \\nATOM   3347  CD  GLN A 214       3.379   0.833  31.816  1.00  0.00           C  \\nATOM   3348  OE1 GLN A 214       4.263   0.155  32.270  1.00  0.00           O  \\nATOM   3349  NE2 GLN A 214       2.877   0.635  30.609  1.00  0.00           N  \\nATOM   3350  H   GLN A 214       2.070   4.583  31.373  1.00  0.00           H  \\nATOM   3351  HA  GLN A 214       2.365   4.002  34.192  1.00  0.00           H  \\nATOM   3352 1HB  GLN A 214       3.970   3.354  31.664  1.00  0.00           H  \\nATOM   3353 2HB  GLN A 214       4.551   2.857  33.247  1.00  0.00           H  \\nATOM   3354 1HG  GLN A 214       2.572   1.601  33.671  1.00  0.00           H  \\nATOM   3355 2HG  GLN A 214       1.793   2.254  32.203  1.00  0.00           H  \\nATOM   3356 1HE2 GLN A 214       3.258  -0.078  30.019  1.00  0.00           H  \\nATOM   3357 2HE2 GLN A 214       2.117   1.198  30.285  1.00  0.00           H  \\nATOM   3358  N   GLU A 215       4.329   6.374  33.229  1.00  0.00           N  \\nATOM   3359  CA  GLU A 215       5.269   7.335  33.838  1.00  0.00           C  \\nATOM   3360  C   GLU A 215       4.540   8.523  34.355  1.00  0.00           C  \\nATOM   3361  O   GLU A 215       5.182   9.510  34.790  1.00  0.00           O  \\nATOM   3362  CB  GLU A 215       6.302   7.789  32.803  1.00  0.00           C  \\nATOM   3363  CG  GLU A 215       6.906   6.628  32.090  1.00  0.00           C  \\nATOM   3364  CD  GLU A 215       8.036   6.026  32.816  1.00  0.00           C  \\nATOM   3365  OE1 GLU A 215       9.042   6.771  33.104  1.00  0.00           O  \\nATOM   3366  OE2 GLU A 215       7.939   4.795  33.038  1.00  0.00           O  \\nATOM   3367  H   GLU A 215       3.988   6.545  32.294  1.00  0.00           H  \\nATOM   3368  HA  GLU A 215       5.788   6.839  34.659  1.00  0.00           H  \\nATOM   3369 1HB  GLU A 215       5.826   8.451  32.079  1.00  0.00           H  \\nATOM   3370 2HB  GLU A 215       7.088   8.358  33.299  1.00  0.00           H  \\nATOM   3371 1HG  GLU A 215       6.140   5.867  31.943  1.00  0.00           H  \\nATOM   3372 2HG  GLU A 215       7.247   6.955  31.109  1.00  0.00           H  \\nATOM   3373  N   SER A 216       3.251   8.439  34.415  1.00  0.00           N  \\nATOM   3374  CA  SER A 216       2.457   9.569  34.947  1.00  0.00           C  \\nATOM   3375  C   SER A 216       2.671   9.779  36.456  1.00  0.00           C  \\nATOM   3376  O   SER A 216       3.124   8.920  37.178  1.00  0.00           O  \\nATOM   3377  CB  SER A 216       0.969   9.352  34.687  1.00  0.00           C  \\nATOM   3378  OG  SER A 216       0.425   8.375  35.426  1.00  0.00           O  \\nATOM   3379  H   SER A 216       2.773   7.606  34.102  1.00  0.00           H  \\nATOM   3380  HA  SER A 216       2.769  10.482  34.437  1.00  0.00           H  \\nATOM   3381 1HB  SER A 216       0.426  10.274  34.892  1.00  0.00           H  \\nATOM   3382 2HB  SER A 216       0.815   9.108  33.637  1.00  0.00           H  \\nATOM   3383  HG  SER A 216       1.136   8.027  35.970  1.00  0.00           H  \\nATOM   3384  N   ASN A 217       2.271  11.031  36.804  1.00  0.00           N  \\nATOM   3385  CA  ASN A 217       2.351  11.455  38.222  1.00  0.00           C  \\nATOM   3386  C   ASN A 217       0.999  11.194  38.849  1.00  0.00           C  \\nATOM   3387  O   ASN A 217       0.087  12.023  38.762  1.00  0.00           O  \\nATOM   3388  CB  ASN A 217       2.630  12.936  38.169  1.00  0.00           C  \\nATOM   3389  CG  ASN A 217       2.668  13.587  39.498  1.00  0.00           C  \\nATOM   3390  OD1 ASN A 217       2.729  12.938  40.522  1.00  0.00           O  \\nATOM   3391  ND2 ASN A 217       2.516  14.900  39.469  1.00  0.00           N  \\nATOM   3392  H   ASN A 217       1.919  11.681  36.115  1.00  0.00           H  \\nATOM   3393  HA  ASN A 217       3.170  10.915  38.699  1.00  0.00           H  \\nATOM   3394 1HB  ASN A 217       3.589  13.109  37.679  1.00  0.00           H  \\nATOM   3395 2HB  ASN A 217       1.864  13.430  37.571  1.00  0.00           H  \\nATOM   3396 1HD2 ASN A 217       2.528  15.424  40.321  1.00  0.00           H  \\nATOM   3397 2HD2 ASN A 217       2.389  15.369  38.596  1.00  0.00           H  \\nATOM   3398  N   GLN A 218       0.832   9.996  39.356  1.00  0.00           N  \\nATOM   3399  CA  GLN A 218      -0.428   9.534  39.885  1.00  0.00           C  \\nATOM   3400  C   GLN A 218      -1.557   9.811  38.975  1.00  0.00           C  \\nATOM   3401  O   GLN A 218      -2.630  10.212  39.329  1.00  0.00           O  \\nATOM   3402  CB  GLN A 218      -0.717  10.185  41.281  1.00  0.00           C  \\nATOM   3403  CG  GLN A 218       0.483  10.070  42.241  1.00  0.00           C  \\nATOM   3404  CD  GLN A 218       0.037  10.181  43.685  1.00  0.00           C  \\nATOM   3405  OE1 GLN A 218      -0.817   9.434  44.156  1.00  0.00           O  \\nATOM   3406  NE2 GLN A 218       0.590  11.149  44.355  1.00  0.00           N  \\nATOM   3407  H   GLN A 218       1.629   9.376  39.372  1.00  0.00           H  \\nATOM   3408  HA  GLN A 218      -0.374   8.453  40.012  1.00  0.00           H  \\nATOM   3409 1HB  GLN A 218      -0.964  11.238  41.147  1.00  0.00           H  \\nATOM   3410 2HB  GLN A 218      -1.582   9.702  41.736  1.00  0.00           H  \\nATOM   3411 1HG  GLN A 218       0.963   9.103  42.092  1.00  0.00           H  \\nATOM   3412 2HG  GLN A 218       1.187  10.874  42.025  1.00  0.00           H  \\nATOM   3413 1HE2 GLN A 218       0.351  11.293  45.316  1.00  0.00           H  \\nATOM   3414 2HE2 GLN A 218       1.254  11.750  43.910  1.00  0.00           H  \\nATOM   3415  N   GLY A 219      -1.351   9.504  37.681  1.00  0.00           N  \\nATOM   3416  CA  GLY A 219      -2.411   9.577  36.670  1.00  0.00           C  \\nATOM   3417  C   GLY A 219      -2.366  10.804  35.838  1.00  0.00           C  \\nATOM   3418  O   GLY A 219      -3.001  10.839  34.791  1.00  0.00           O  \\nATOM   3419  H   GLY A 219      -0.425   9.212  37.403  1.00  0.00           H  \\nATOM   3420 1HA  GLY A 219      -2.343   8.713  36.009  1.00  0.00           H  \\nATOM   3421 2HA  GLY A 219      -3.383   9.532  37.159  1.00  0.00           H  \\nATOM   3422  N   ALA A 220      -1.612  11.845  36.277  1.00  0.00           N  \\nATOM   3423  CA  ALA A 220      -1.552  13.065  35.579  1.00  0.00           C  \\nATOM   3424  C   ALA A 220      -0.361  13.062  34.612  1.00  0.00           C  \\nATOM   3425  O   ALA A 220       0.753  12.754  34.984  1.00  0.00           O  \\nATOM   3426  CB  ALA A 220      -1.435  14.258  36.620  1.00  0.00           C  \\nATOM   3427  H   ALA A 220      -1.080  11.739  37.129  1.00  0.00           H  \\nATOM   3428  HA  ALA A 220      -2.473  13.173  35.006  1.00  0.00           H  \\nATOM   3429 1HB  ALA A 220      -1.388  15.206  36.084  1.00  0.00           H  \\nATOM   3430 2HB  ALA A 220      -2.305  14.255  37.276  1.00  0.00           H  \\nATOM   3431 3HB  ALA A 220      -0.532  14.134  37.215  1.00  0.00           H  \\nATOM   3432  N   PHE A 221      -0.590  13.388  33.345  1.00  0.00           N  \\nATOM   3433  CA  PHE A 221       0.531  13.255  32.385  1.00  0.00           C  \\nATOM   3434  C   PHE A 221       1.485  14.390  32.482  1.00  0.00           C  \\nATOM   3435  O   PHE A 221       1.037  15.522  32.609  1.00  0.00           O  \\nATOM   3436  CB  PHE A 221       0.080  13.166  30.926  1.00  0.00           C  \\nATOM   3437  CG  PHE A 221      -0.738  11.957  30.617  1.00  0.00           C  \\nATOM   3438  CD1 PHE A 221      -0.872  10.915  31.527  1.00  0.00           C  \\nATOM   3439  CD2 PHE A 221      -1.454  11.926  29.412  1.00  0.00           C  \\nATOM   3440  CE1 PHE A 221      -1.674   9.816  31.195  1.00  0.00           C  \\nATOM   3441  CE2 PHE A 221      -2.256  10.826  29.092  1.00  0.00           C  \\nATOM   3442  CZ  PHE A 221      -2.304   9.768  29.946  1.00  0.00           C  \\nATOM   3443  H   PHE A 221      -1.485  13.718  33.014  1.00  0.00           H  \\nATOM   3444  HA  PHE A 221       1.073  12.335  32.610  1.00  0.00           H  \\nATOM   3445 1HB  PHE A 221      -0.509  14.047  30.673  1.00  0.00           H  \\nATOM   3446 2HB  PHE A 221       0.953  13.159  30.275  1.00  0.00           H  \\nATOM   3447  HD1 PHE A 221      -0.351  10.973  32.483  1.00  0.00           H  \\nATOM   3448  HD2 PHE A 221      -1.406  12.780  28.736  1.00  0.00           H  \\nATOM   3449  HE1 PHE A 221      -1.806   9.003  31.908  1.00  0.00           H  \\nATOM   3450  HE2 PHE A 221      -2.838  10.809  28.171  1.00  0.00           H  \\nATOM   3451  HZ  PHE A 221      -2.844   8.873  29.641  1.00  0.00           H  \\nATOM   3452  N   ALA A 222       2.780  14.035  32.356  1.00  0.00           N  \\nATOM   3453  CA  ALA A 222       3.857  15.003  32.367  1.00  0.00           C  \\nATOM   3454  C   ALA A 222       3.666  15.953  31.175  1.00  0.00           C  \\nATOM   3455  O   ALA A 222       3.846  17.168  31.304  1.00  0.00           O  \\nATOM   3456  CB  ALA A 222       5.164  14.280  32.285  1.00  0.00           C  \\nATOM   3457  H   ALA A 222       3.000  13.055  32.250  1.00  0.00           H  \\nATOM   3458  HA  ALA A 222       3.805  15.557  33.305  1.00  0.00           H  \\nATOM   3459 1HB  ALA A 222       5.980  15.002  32.293  1.00  0.00           H  \\nATOM   3460 2HB  ALA A 222       5.264  13.610  33.139  1.00  0.00           H  \\nATOM   3461 3HB  ALA A 222       5.202  13.700  31.364  1.00  0.00           H  \\nATOM   3462  N   SER A 223       3.254  15.421  30.023  1.00  0.00           N  \\nATOM   3463  CA  SER A 223       2.781  16.278  28.928  1.00  0.00           C  \\nATOM   3464  C   SER A 223       1.655  15.621  28.169  1.00  0.00           C  \\nATOM   3465  O   SER A 223       1.482  14.408  28.300  1.00  0.00           O  \\nATOM   3466  CB  SER A 223       3.922  16.617  27.983  1.00  0.00           C  \\nATOM   3467  OG  SER A 223       4.750  17.525  28.677  1.00  0.00           O  \\nATOM   3468  H   SER A 223       3.264  14.419  29.894  1.00  0.00           H  \\nATOM   3469  HA  SER A 223       2.396  17.205  29.355  1.00  0.00           H  \\nATOM   3470 1HB  SER A 223       4.452  15.705  27.711  1.00  0.00           H  \\nATOM   3471 2HB  SER A 223       3.520  17.047  27.067  1.00  0.00           H  \\nATOM   3472  HG  SER A 223       4.341  17.653  29.536  1.00  0.00           H  \\nATOM   3473  N   PRO A 224       0.883  16.436  27.419  1.00  0.00           N  \\nATOM   3474  CA  PRO A 224      -0.260  15.893  26.713  1.00  0.00           C  \\nATOM   3475  C   PRO A 224       0.098  14.947  25.552  1.00  0.00           C  \\nATOM   3476  O   PRO A 224       1.215  14.936  25.033  1.00  0.00           O  \\nATOM   3477  CB  PRO A 224      -1.022  17.099  26.203  1.00  0.00           C  \\nATOM   3478  CG  PRO A 224      -0.210  18.334  26.596  1.00  0.00           C  \\nATOM   3479  CD  PRO A 224       1.116  17.864  27.131  1.00  0.00           C  \\nATOM   3480  HA  PRO A 224      -0.884  15.325  27.418  1.00  0.00           H  \\nATOM   3481 1HB  PRO A 224      -1.150  17.030  25.113  1.00  0.00           H  \\nATOM   3482 2HB  PRO A 224      -2.030  17.121  26.644  1.00  0.00           H  \\nATOM   3483 1HG  PRO A 224      -0.071  18.990  25.724  1.00  0.00           H  \\nATOM   3484 2HG  PRO A 224      -0.753  18.921  27.352  1.00  0.00           H  \\nATOM   3485 1HD  PRO A 224       1.891  17.999  26.362  1.00  0.00           H  \\nATOM   3486 2HD  PRO A 224       1.367  18.431  28.039  1.00  0.00           H  \\nATOM   3487  N   ILE A 225      -0.852  14.091  25.239  1.00  0.00           N  \\nATOM   3488  CA  ILE A 225      -0.740  13.233  24.079  1.00  0.00           C  \\nATOM   3489  C   ILE A 225      -1.910  13.621  23.184  1.00  0.00           C  \\nATOM   3490  O   ILE A 225      -3.025  13.772  23.646  1.00  0.00           O  \\nATOM   3491  CB  ILE A 225      -0.790  11.711  24.501  1.00  0.00           C  \\nATOM   3492  CG1 ILE A 225       0.658  11.232  24.818  1.00  0.00           C  \\nATOM   3493  CG2 ILE A 225      -1.505  10.792  23.475  1.00  0.00           C  \\nATOM   3494  CD1 ILE A 225       0.956  11.051  26.179  1.00  0.00           C  \\nATOM   3495  H   ILE A 225      -1.677  14.030  25.818  1.00  0.00           H  \\nATOM   3496  HA  ILE A 225       0.217  13.427  23.597  1.00  0.00           H  \\nATOM   3497  HB  ILE A 225      -1.324  11.613  25.446  1.00  0.00           H  \\nATOM   3498 1HG1 ILE A 225       0.845  10.283  24.317  1.00  0.00           H  \\nATOM   3499 2HG1 ILE A 225       1.374  11.954  24.425  1.00  0.00           H  \\nATOM   3500 1HG2 ILE A 225      -1.496   9.765  23.839  1.00  0.00           H  \\nATOM   3501 2HG2 ILE A 225      -2.535  11.122  23.347  1.00  0.00           H  \\nATOM   3502 3HG2 ILE A 225      -0.986  10.843  22.518  1.00  0.00           H  \\nATOM   3503 1HD1 ILE A 225       1.989  10.718  26.284  1.00  0.00           H  \\nATOM   3504 2HD1 ILE A 225       0.824  11.995  26.709  1.00  0.00           H  \\nATOM   3505 3HD1 ILE A 225       0.289  10.301  26.601  1.00  0.00           H  \\nATOM   3506  N   GLN A 226      -1.600  13.845  21.922  1.00  0.00           N  \\nATOM   3507  CA  GLN A 226      -2.577  14.237  20.938  1.00  0.00           C  \\nATOM   3508  C   GLN A 226      -3.238  13.053  20.284  1.00  0.00           C  \\nATOM   3509  O   GLN A 226      -2.565  12.123  19.776  1.00  0.00           O  \\nATOM   3510  CB  GLN A 226      -1.944  15.140  19.891  1.00  0.00           C  \\nATOM   3511  CG  GLN A 226      -2.967  15.813  19.024  1.00  0.00           C  \\nATOM   3512  CD  GLN A 226      -2.263  16.564  18.036  1.00  0.00           C  \\nATOM   3513  OE1 GLN A 226      -2.056  17.762  18.180  1.00  0.00           O  \\nATOM   3514  NE2 GLN A 226      -1.803  15.864  17.032  1.00  0.00           N  \\nATOM   3515  H   GLN A 226      -0.636  13.735  21.642  1.00  0.00           H  \\nATOM   3516  HA  GLN A 226      -3.371  14.789  21.440  1.00  0.00           H  \\nATOM   3517 1HB  GLN A 226      -1.342  15.903  20.385  1.00  0.00           H  \\nATOM   3518 2HB  GLN A 226      -1.276  14.553  19.260  1.00  0.00           H  \\nATOM   3519 1HG  GLN A 226      -3.591  15.051  18.557  1.00  0.00           H  \\nATOM   3520 2HG  GLN A 226      -3.579  16.468  19.644  1.00  0.00           H  \\nATOM   3521 1HE2 GLN A 226      -1.297  16.316  16.296  1.00  0.00           H  \\nATOM   3522 2HE2 GLN A 226      -1.956  14.876  16.999  1.00  0.00           H  \\nATOM   3523  N   LEU A 227      -4.564  13.130  20.271  1.00  0.00           N  \\nATOM   3524  CA  LEU A 227      -5.340  12.131  19.560  1.00  0.00           C  \\nATOM   3525  C   LEU A 227      -6.237  12.774  18.556  1.00  0.00           C  \\nATOM   3526  O   LEU A 227      -6.149  13.997  18.375  1.00  0.00           O  \\nATOM   3527  CB  LEU A 227      -6.119  11.290  20.525  1.00  0.00           C  \\nATOM   3528  CG  LEU A 227      -5.347  10.390  21.481  1.00  0.00           C  \\nATOM   3529  CD1 LEU A 227      -6.339   9.812  22.555  1.00  0.00           C  \\nATOM   3530  CD2 LEU A 227      -4.444   9.372  20.750  1.00  0.00           C  \\nATOM   3531  H   LEU A 227      -5.045  13.877  20.751  1.00  0.00           H  \\nATOM   3532  HA  LEU A 227      -4.654  11.487  19.011  1.00  0.00           H  \\nATOM   3533 1HB  LEU A 227      -6.728  11.946  21.145  1.00  0.00           H  \\nATOM   3534 2HB  LEU A 227      -6.785  10.638  19.959  1.00  0.00           H  \\nATOM   3535  HG  LEU A 227      -4.709  11.001  22.120  1.00  0.00           H  \\nATOM   3536 1HD1 LEU A 227      -5.794   9.166  23.244  1.00  0.00           H  \\nATOM   3537 2HD1 LEU A 227      -6.793  10.633  23.111  1.00  0.00           H  \\nATOM   3538 3HD1 LEU A 227      -7.118   9.236  22.058  1.00  0.00           H  \\nATOM   3539 1HD2 LEU A 227      -3.919   8.759  21.483  1.00  0.00           H  \\nATOM   3540 2HD2 LEU A 227      -5.057   8.732  20.115  1.00  0.00           H  \\nATOM   3541 3HD2 LEU A 227      -3.718   9.905  20.136  1.00  0.00           H  \\nATOM   3542  N   GLN A 228      -7.077  12.019  17.911  1.00  0.00           N  \\nATOM   3543  CA  GLN A 228      -7.976  12.609  16.881  1.00  0.00           C  \\nATOM   3544  C   GLN A 228      -9.353  12.090  17.087  1.00  0.00           C  \\nATOM   3545  O   GLN A 228      -9.573  10.937  17.410  1.00  0.00           O  \\nATOM   3546  CB  GLN A 228      -7.505  12.251  15.461  1.00  0.00           C  \\nATOM   3547  CG  GLN A 228      -6.373  13.104  14.979  1.00  0.00           C  \\nATOM   3548  CD  GLN A 228      -5.877  12.781  13.575  1.00  0.00           C  \\nATOM   3549  OE1 GLN A 228      -4.674  12.844  13.314  1.00  0.00           O  \\nATOM   3550  NE2 GLN A 228      -6.762  12.446  12.703  1.00  0.00           N  \\nATOM   3551  H   GLN A 228      -7.129  11.028  18.100  1.00  0.00           H  \\nATOM   3552  HA  GLN A 228      -7.956  13.694  16.985  1.00  0.00           H  \\nATOM   3553 1HB  GLN A 228      -7.186  11.209  15.435  1.00  0.00           H  \\nATOM   3554 2HB  GLN A 228      -8.337  12.356  14.764  1.00  0.00           H  \\nATOM   3555 1HG  GLN A 228      -6.694  14.146  14.972  1.00  0.00           H  \\nATOM   3556 2HG  GLN A 228      -5.525  12.979  15.653  1.00  0.00           H  \\nATOM   3557 1HE2 GLN A 228      -6.484  12.225  11.767  1.00  0.00           H  \\nATOM   3558 2HE2 GLN A 228      -7.728  12.406  12.960  1.00  0.00           H  \\nATOM   3559  N   ARG A 229     -10.347  12.986  16.896  1.00  0.00           N  \\nATOM   3560  CA  ARG A 229     -11.742  12.597  16.948  1.00  0.00           C  \\nATOM   3561  C   ARG A 229     -12.116  11.911  15.650  1.00  0.00           C  \\nATOM   3562  O   ARG A 229     -11.332  11.982  14.629  1.00  0.00           O  \\nATOM   3563  CB  ARG A 229     -12.627  13.868  17.132  1.00  0.00           C  \\nATOM   3564  CG  ARG A 229     -12.326  14.646  18.391  1.00  0.00           C  \\nATOM   3565  CD  ARG A 229     -13.467  15.667  18.590  1.00  0.00           C  \\nATOM   3566  NE  ARG A 229     -13.151  16.472  19.779  1.00  0.00           N  \\nATOM   3567  CZ  ARG A 229     -12.418  17.579  19.776  1.00  0.00           C  \\nATOM   3568  NH1 ARG A 229     -11.907  18.051  18.663  1.00  0.00           N  \\nATOM   3569  NH2 ARG A 229     -12.196  18.189  20.937  1.00  0.00           N  \\nATOM   3570  H   ARG A 229     -10.112  13.951  16.712  1.00  0.00           H  \\nATOM   3571  HA  ARG A 229     -11.885  11.934  17.802  1.00  0.00           H  \\nATOM   3572 1HB  ARG A 229     -12.489  14.533  16.281  1.00  0.00           H  \\nATOM   3573 2HB  ARG A 229     -13.678  13.579  17.155  1.00  0.00           H  \\nATOM   3574 1HG  ARG A 229     -12.275  13.962  19.238  1.00  0.00           H  \\nATOM   3575 2HG  ARG A 229     -11.370  15.160  18.282  1.00  0.00           H  \\nATOM   3576 1HD  ARG A 229     -13.540  16.306  17.711  1.00  0.00           H  \\nATOM   3577 2HD  ARG A 229     -14.408  15.137  18.733  1.00  0.00           H  \\nATOM   3578  HE  ARG A 229     -13.519  16.161  20.668  1.00  0.00           H  \\nATOM   3579 1HH1 ARG A 229     -12.069  17.570  17.789  1.00  0.00           H  \\nATOM   3580 2HH1 ARG A 229     -11.352  18.894  18.680  1.00  0.00           H  \\nATOM   3581 1HH2 ARG A 229     -12.582  17.808  21.790  1.00  0.00           H  \\nATOM   3582 2HH2 ARG A 229     -11.643  19.032  20.966  1.00  0.00           H  \\nATOM   3583  N   ARG A 230     -13.346  11.401  15.525  1.00  0.00           N  \\nATOM   3584  CA  ARG A 230     -13.767  10.771  14.291  1.00  0.00           C  \\nATOM   3585  C   ARG A 230     -13.742  11.658  13.064  1.00  0.00           C  \\nATOM   3586  O   ARG A 230     -13.690  11.112  11.967  1.00  0.00           O  \\nATOM   3587  CB  ARG A 230     -15.169  10.247  14.444  1.00  0.00           C  \\nATOM   3588  CG  ARG A 230     -15.278   9.207  15.499  1.00  0.00           C  \\nATOM   3589  CD  ARG A 230     -16.630   8.608  15.556  1.00  0.00           C  \\nATOM   3590  NE  ARG A 230     -16.895   7.932  16.862  1.00  0.00           N  \\nATOM   3591  CZ  ARG A 230     -16.585   6.626  17.071  1.00  0.00           C  \\nATOM   3592  NH1 ARG A 230     -16.088   5.877  16.069  1.00  0.00           N  \\nATOM   3593  NH2 ARG A 230     -16.867   6.050  18.224  1.00  0.00           N  \\nATOM   3594  H   ARG A 230     -13.995  11.453  16.297  1.00  0.00           H  \\nATOM   3595  HA  ARG A 230     -13.096   9.937  14.082  1.00  0.00           H  \\nATOM   3596 1HB  ARG A 230     -15.839  11.069  14.691  1.00  0.00           H  \\nATOM   3597 2HB  ARG A 230     -15.505   9.824  13.497  1.00  0.00           H  \\nATOM   3598 1HG  ARG A 230     -14.563   8.409  15.299  1.00  0.00           H  \\nATOM   3599 2HG  ARG A 230     -15.064   9.651  16.472  1.00  0.00           H  \\nATOM   3600 1HD  ARG A 230     -17.379   9.388  15.422  1.00  0.00           H  \\nATOM   3601 2HD  ARG A 230     -16.733   7.867  14.765  1.00  0.00           H  \\nATOM   3602  HE  ARG A 230     -17.318   8.481  17.599  1.00  0.00           H  \\nATOM   3603 1HH1 ARG A 230     -15.944   6.285  15.156  1.00  0.00           H  \\nATOM   3604 2HH1 ARG A 230     -15.859   4.907  16.230  1.00  0.00           H  \\nATOM   3605 1HH2 ARG A 230     -17.316   6.581  18.958  1.00  0.00           H  \\nATOM   3606 2HH2 ARG A 230     -16.634   5.080  18.373  1.00  0.00           H  \\nATOM   3607  N   ASN A 231     -13.849  12.977  13.245  1.00  0.00           N  \\nATOM   3608  CA  ASN A 231     -13.894  13.874  12.102  1.00  0.00           C  \\nATOM   3609  C   ASN A 231     -12.501  14.378  11.776  1.00  0.00           C  \\nATOM   3610  O   ASN A 231     -12.283  15.162  10.778  1.00  0.00           O  \\nATOM   3611  CB  ASN A 231     -14.908  14.966  12.412  1.00  0.00           C  \\nATOM   3612  CG  ASN A 231     -14.402  15.982  13.450  1.00  0.00           C  \\nATOM   3613  OD1 ASN A 231     -13.457  15.789  14.256  1.00  0.00           O  \\nATOM   3614  ND2 ASN A 231     -15.081  17.131  13.451  1.00  0.00           N  \\nATOM   3615  H   ASN A 231     -13.900  13.364  14.176  1.00  0.00           H  \\nATOM   3616  HA  ASN A 231     -14.214  13.305  11.228  1.00  0.00           H  \\nATOM   3617 1HB  ASN A 231     -15.158  15.502  11.495  1.00  0.00           H  \\nATOM   3618 2HB  ASN A 231     -15.826  14.514  12.788  1.00  0.00           H  \\nATOM   3619 1HD2 ASN A 231     -14.831  17.859  14.090  1.00  0.00           H  \\nATOM   3620 2HD2 ASN A 231     -15.839  17.264  12.814  1.00  0.00           H  \\nATOM   3621  N   GLY A 232     -11.510  13.903  12.515  1.00  0.00           N  \\nATOM   3622  CA  GLY A 232     -10.133  14.208  12.152  1.00  0.00           C  \\nATOM   3623  C   GLY A 232      -9.581  15.355  12.984  1.00  0.00           C  \\nATOM   3624  O   GLY A 232      -8.348  15.534  13.005  1.00  0.00           O  \\nATOM   3625  H   GLY A 232     -11.689  13.331  13.328  1.00  0.00           H  \\nATOM   3626 1HA  GLY A 232      -9.514  13.322  12.295  1.00  0.00           H  \\nATOM   3627 2HA  GLY A 232     -10.084  14.467  11.095  1.00  0.00           H  \\nATOM   3628  N   SER A 233     -10.414  16.028  13.772  1.00  0.00           N  \\nATOM   3629  CA  SER A 233      -9.921  17.168  14.579  1.00  0.00           C  \\nATOM   3630  C   SER A 233      -9.045  16.673  15.710  1.00  0.00           C  \\nATOM   3631  O   SER A 233      -9.228  15.554  16.197  1.00  0.00           O  \\nATOM   3632  CB  SER A 233     -11.079  18.049  15.108  1.00  0.00           C  \\nATOM   3633  OG  SER A 233     -12.014  17.233  15.790  1.00  0.00           O  \\nATOM   3634  H   SER A 233     -11.389  15.770  13.829  1.00  0.00           H  \\nATOM   3635  HA  SER A 233      -9.288  17.792  13.947  1.00  0.00           H  \\nATOM   3636 1HB  SER A 233     -10.680  18.812  15.775  1.00  0.00           H  \\nATOM   3637 2HB  SER A 233     -11.556  18.561  14.274  1.00  0.00           H  \\nATOM   3638  HG  SER A 233     -11.671  16.338  15.739  1.00  0.00           H  \\nATOM   3639  N   LYS A 234      -8.073  17.484  16.095  1.00  0.00           N  \\nATOM   3640  CA  LYS A 234      -7.122  17.055  17.061  1.00  0.00           C  \\nATOM   3641  C   LYS A 234      -7.518  17.513  18.444  1.00  0.00           C  \\nATOM   3642  O   LYS A 234      -8.100  18.614  18.631  1.00  0.00           O  \\nATOM   3643  CB  LYS A 234      -5.755  17.592  16.728  1.00  0.00           C  \\nATOM   3644  CG  LYS A 234      -5.323  17.252  15.320  1.00  0.00           C  \\nATOM   3645  CD  LYS A 234      -4.730  15.949  15.070  1.00  0.00           C  \\nATOM   3646  CE  LYS A 234      -4.100  16.035  13.667  1.00  0.00           C  \\nATOM   3647  NZ  LYS A 234      -3.299  17.306  13.581  1.00  0.00           N  \\nATOM   3648  H   LYS A 234      -7.996  18.414  15.708  1.00  0.00           H  \\nATOM   3649  HA  LYS A 234      -7.082  15.965  17.047  1.00  0.00           H  \\nATOM   3650 1HB  LYS A 234      -5.752  18.676  16.844  1.00  0.00           H  \\nATOM   3651 2HB  LYS A 234      -5.024  17.186  17.427  1.00  0.00           H  \\nATOM   3652 1HG  LYS A 234      -6.183  17.306  14.652  1.00  0.00           H  \\nATOM   3653 2HG  LYS A 234      -4.579  17.973  14.984  1.00  0.00           H  \\nATOM   3654 1HD  LYS A 234      -3.983  15.736  15.836  1.00  0.00           H  \\nATOM   3655 2HD  LYS A 234      -5.503  15.182  15.118  1.00  0.00           H  \\nATOM   3656 1HE  LYS A 234      -3.462  15.168  13.501  1.00  0.00           H  \\nATOM   3657 2HE  LYS A 234      -4.887  16.028  12.914  1.00  0.00           H  \\nATOM   3658 1HZ  LYS A 234      -2.879  17.378  12.665  1.00  0.00           H  \\nATOM   3659 2HZ  LYS A 234      -3.906  18.099  13.736  1.00  0.00           H  \\nATOM   3660 3HZ  LYS A 234      -2.574  17.296  14.284  1.00  0.00           H  \\nATOM   3661  N   PHE A 235      -7.115  16.743  19.436  1.00  0.00           N  \\nATOM   3662  CA  PHE A 235      -7.379  17.151  20.804  1.00  0.00           C  \\nATOM   3663  C   PHE A 235      -6.327  16.498  21.702  1.00  0.00           C  \\nATOM   3664  O   PHE A 235      -5.609  15.548  21.261  1.00  0.00           O  \\nATOM   3665  CB  PHE A 235      -8.847  16.838  21.182  1.00  0.00           C  \\nATOM   3666  CG  PHE A 235      -9.128  15.365  21.493  1.00  0.00           C  \\nATOM   3667  CD1 PHE A 235      -9.407  14.443  20.463  1.00  0.00           C  \\nATOM   3668  CD2 PHE A 235      -9.072  14.924  22.742  1.00  0.00           C  \\nATOM   3669  CE1 PHE A 235      -9.717  13.110  20.731  1.00  0.00           C  \\nATOM   3670  CE2 PHE A 235      -9.324  13.588  23.039  1.00  0.00           C  \\nATOM   3671  CZ  PHE A 235      -9.688  12.700  22.062  1.00  0.00           C  \\nATOM   3672  H   PHE A 235      -6.627  15.875  19.267  1.00  0.00           H  \\nATOM   3673  HA  PHE A 235      -7.219  18.227  20.881  1.00  0.00           H  \\nATOM   3674 1HB  PHE A 235      -9.129  17.421  22.058  1.00  0.00           H  \\nATOM   3675 2HB  PHE A 235      -9.504  17.135  20.366  1.00  0.00           H  \\nATOM   3676  HD1 PHE A 235      -9.380  14.779  19.426  1.00  0.00           H  \\nATOM   3677  HD2 PHE A 235      -8.784  15.614  23.536  1.00  0.00           H  \\nATOM   3678  HE1 PHE A 235      -9.969  12.435  19.914  1.00  0.00           H  \\nATOM   3679  HE2 PHE A 235      -9.228  13.252  24.071  1.00  0.00           H  \\nATOM   3680  HZ  PHE A 235      -9.955  11.677  22.324  1.00  0.00           H  \\nATOM   3681  N   SER A 236      -6.217  17.031  22.923  1.00  0.00           N  \\nATOM   3682  CA  SER A 236      -5.165  16.656  23.827  1.00  0.00           C  \\nATOM   3683  C   SER A 236      -5.742  15.920  25.026  1.00  0.00           C  \\nATOM   3684  O   SER A 236      -6.795  16.304  25.537  1.00  0.00           O  \\nATOM   3685  CB  SER A 236      -4.513  17.947  24.297  1.00  0.00           C  \\nATOM   3686  OG  SER A 236      -3.636  18.402  23.281  1.00  0.00           O  \\nATOM   3687  H   SER A 236      -6.896  17.719  23.217  1.00  0.00           H  \\nATOM   3688  HA  SER A 236      -4.445  16.041  23.285  1.00  0.00           H  \\nATOM   3689 1HB  SER A 236      -5.284  18.688  24.507  1.00  0.00           H  \\nATOM   3690 2HB  SER A 236      -3.973  17.765  25.225  1.00  0.00           H  \\nATOM   3691  HG  SER A 236      -3.699  17.760  22.570  1.00  0.00           H  \\nATOM   3692  N   VAL A 237      -5.024  14.868  25.463  1.00  0.00           N  \\nATOM   3693  CA  VAL A 237      -5.413  14.157  26.720  1.00  0.00           C  \\nATOM   3694  C   VAL A 237      -4.355  14.409  27.760  1.00  0.00           C  \\nATOM   3695  O   VAL A 237      -3.180  14.362  27.454  1.00  0.00           O  \\nATOM   3696  CB  VAL A 237      -5.675  12.641  26.508  1.00  0.00           C  \\nATOM   3697  CG1 VAL A 237      -6.705  12.414  25.416  1.00  0.00           C  \\nATOM   3698  CG2 VAL A 237      -4.454  11.842  26.147  1.00  0.00           C  \\nATOM   3699  H   VAL A 237      -4.217  14.549  24.947  1.00  0.00           H  \\nATOM   3700  HA  VAL A 237      -6.338  14.597  27.095  1.00  0.00           H  \\nATOM   3701  HB  VAL A 237      -6.076  12.219  27.430  1.00  0.00           H  \\nATOM   3702 1HG1 VAL A 237      -6.870  11.344  25.288  1.00  0.00           H  \\nATOM   3703 2HG1 VAL A 237      -7.643  12.894  25.695  1.00  0.00           H  \\nATOM   3704 3HG1 VAL A 237      -6.343  12.839  24.480  1.00  0.00           H  \\nATOM   3705 1HG2 VAL A 237      -4.729  10.795  26.019  1.00  0.00           H  \\nATOM   3706 2HG2 VAL A 237      -4.031  12.223  25.217  1.00  0.00           H  \\nATOM   3707 3HG2 VAL A 237      -3.715  11.928  26.943  1.00  0.00           H  \\nATOM   3708  N   TYR A 238      -4.772  14.775  28.962  1.00  0.00           N  \\nATOM   3709  CA  TYR A 238      -3.856  15.211  29.988  1.00  0.00           C  \\nATOM   3710  C   TYR A 238      -3.762  14.258  31.175  1.00  0.00           C  \\nATOM   3711  O   TYR A 238      -2.923  14.470  32.052  1.00  0.00           O  \\nATOM   3712  CB  TYR A 238      -4.216  16.655  30.463  1.00  0.00           C  \\nATOM   3713  CG  TYR A 238      -4.230  17.715  29.369  1.00  0.00           C  \\nATOM   3714  CD1 TYR A 238      -5.382  17.905  28.603  1.00  0.00           C  \\nATOM   3715  CD2 TYR A 238      -3.140  18.524  29.141  1.00  0.00           C  \\nATOM   3716  CE1 TYR A 238      -5.458  18.914  27.621  1.00  0.00           C  \\nATOM   3717  CE2 TYR A 238      -3.187  19.513  28.167  1.00  0.00           C  \\nATOM   3718  CZ  TYR A 238      -4.337  19.694  27.409  1.00  0.00           C  \\nATOM   3719  OH  TYR A 238      -4.395  20.676  26.422  1.00  0.00           O  \\nATOM   3720  H   TYR A 238      -5.761  14.747  29.165  1.00  0.00           H  \\nATOM   3721  HA  TYR A 238      -2.849  15.222  29.571  1.00  0.00           H  \\nATOM   3722 1HB  TYR A 238      -5.204  16.650  30.926  1.00  0.00           H  \\nATOM   3723 2HB  TYR A 238      -3.502  16.977  31.220  1.00  0.00           H  \\nATOM   3724  HD1 TYR A 238      -6.251  17.265  28.757  1.00  0.00           H  \\nATOM   3725  HD2 TYR A 238      -2.230  18.390  29.726  1.00  0.00           H  \\nATOM   3726  HE1 TYR A 238      -6.370  19.039  27.038  1.00  0.00           H  \\nATOM   3727  HE2 TYR A 238      -2.319  20.150  27.995  1.00  0.00           H  \\nATOM   3728  HH  TYR A 238      -3.557  21.144  26.389  1.00  0.00           H  \\nATOM   3729  N   ASP A 239      -4.567  13.200  31.189  1.00  0.00           N  \\nATOM   3730  CA  ASP A 239      -4.566  12.295  32.322  1.00  0.00           C  \\nATOM   3731  C   ASP A 239      -5.214  11.024  32.035  1.00  0.00           C  \\nATOM   3732  O   ASP A 239      -5.984  10.915  31.072  1.00  0.00           O  \\nATOM   3733  CB  ASP A 239      -5.137  12.911  33.614  1.00  0.00           C  \\nATOM   3734  CG  ASP A 239      -6.610  13.260  33.522  1.00  0.00           C  \\nATOM   3735  OD1 ASP A 239      -6.923  14.462  33.266  1.00  0.00           O  \\nATOM   3736  OD2 ASP A 239      -7.445  12.375  33.824  1.00  0.00           O  \\nATOM   3737  H   ASP A 239      -5.186  13.016  30.413  1.00  0.00           H  \\nATOM   3738  HA  ASP A 239      -3.536  12.008  32.535  1.00  0.00           H  \\nATOM   3739 1HB  ASP A 239      -5.002  12.213  34.441  1.00  0.00           H  \\nATOM   3740 2HB  ASP A 239      -4.585  13.819  33.859  1.00  0.00           H  \\nATOM   3741  N   VAL A 240      -4.962  10.047  32.914  1.00  0.00           N  \\nATOM   3742  CA  VAL A 240      -5.376   8.688  32.585  1.00  0.00           C  \\nATOM   3743  C   VAL A 240      -6.886   8.481  32.635  1.00  0.00           C  \\nATOM   3744  O   VAL A 240      -7.412   7.629  31.961  1.00  0.00           O  \\nATOM   3745  CB  VAL A 240      -4.676   7.701  33.501  1.00  0.00           C  \\nATOM   3746  CG1 VAL A 240      -4.863   6.189  33.052  1.00  0.00           C  \\nATOM   3747  CG2 VAL A 240      -3.218   7.998  33.552  1.00  0.00           C  \\nATOM   3748  H   VAL A 240      -4.497  10.220  33.793  1.00  0.00           H  \\nATOM   3749  HA  VAL A 240      -5.089   8.479  31.554  1.00  0.00           H  \\nATOM   3750  HB  VAL A 240      -5.099   7.786  34.502  1.00  0.00           H  \\nATOM   3751 1HG1 VAL A 240      -4.338   5.535  33.749  1.00  0.00           H  \\nATOM   3752 2HG1 VAL A 240      -5.924   5.939  33.049  1.00  0.00           H  \\nATOM   3753 3HG1 VAL A 240      -4.454   6.054  32.051  1.00  0.00           H  \\nATOM   3754 1HG2 VAL A 240      -2.724   7.285  34.212  1.00  0.00           H  \\nATOM   3755 2HG2 VAL A 240      -2.795   7.918  32.550  1.00  0.00           H  \\nATOM   3756 3HG2 VAL A 240      -3.066   9.009  33.931  1.00  0.00           H  \\nATOM   3757  N   SER A 241      -7.623   9.266  33.409  1.00  0.00           N  \\nATOM   3758  CA  SER A 241      -9.001   8.878  33.688  1.00  0.00           C  \\nATOM   3759  C   SER A 241      -9.820   8.726  32.429  1.00  0.00           C  \\nATOM   3760  O   SER A 241     -10.629   7.787  32.324  1.00  0.00           O  \\nATOM   3761  CB  SER A 241      -9.677   9.888  34.640  1.00  0.00           C  \\nATOM   3762  OG  SER A 241      -9.288  11.216  34.388  1.00  0.00           O  \\nATOM   3763  H   SER A 241      -7.262  10.120  33.809  1.00  0.00           H  \\nATOM   3764  HA  SER A 241      -8.994   7.900  34.172  1.00  0.00           H  \\nATOM   3765 1HB  SER A 241     -10.760   9.815  34.539  1.00  0.00           H  \\nATOM   3766 2HB  SER A 241      -9.427   9.639  35.671  1.00  0.00           H  \\nATOM   3767  HG  SER A 241      -8.670  11.176  33.654  1.00  0.00           H  \\nATOM   3768  N   ILE A 242      -9.646   9.636  31.481  1.00  0.00           N  \\nATOM   3769  CA  ILE A 242     -10.483   9.623  30.306  1.00  0.00           C  \\nATOM   3770  C   ILE A 242     -10.118   8.462  29.353  1.00  0.00           C  \\nATOM   3771  O   ILE A 242     -10.876   8.113  28.466  1.00  0.00           O  \\nATOM   3772  CB  ILE A 242     -10.359  10.990  29.574  1.00  0.00           C  \\nATOM   3773  CG1 ILE A 242     -11.432  11.132  28.528  1.00  0.00           C  \\nATOM   3774  CG2 ILE A 242      -8.984  11.237  29.023  1.00  0.00           C  \\nATOM   3775  CD1 ILE A 242     -11.249  12.431  27.667  1.00  0.00           C  \\nATOM   3776  H   ILE A 242      -8.931  10.344  31.568  1.00  0.00           H  \\nATOM   3777  HA  ILE A 242     -11.516   9.476  30.619  1.00  0.00           H  \\nATOM   3778  HB  ILE A 242     -10.587  11.797  30.270  1.00  0.00           H  \\nATOM   3779 1HG1 ILE A 242     -11.417  10.263  27.871  1.00  0.00           H  \\nATOM   3780 2HG1 ILE A 242     -12.409  11.159  29.011  1.00  0.00           H  \\nATOM   3781 1HG2 ILE A 242      -8.961  12.205  28.524  1.00  0.00           H  \\nATOM   3782 2HG2 ILE A 242      -8.260  11.231  29.837  1.00  0.00           H  \\nATOM   3783 3HG2 ILE A 242      -8.733  10.454  28.308  1.00  0.00           H  \\nATOM   3784 1HD1 ILE A 242     -12.046  12.492  26.926  1.00  0.00           H  \\nATOM   3785 2HD1 ILE A 242     -11.290  13.307  28.316  1.00  0.00           H  \\nATOM   3786 3HD1 ILE A 242     -10.285  12.399  27.160  1.00  0.00           H  \\nATOM   3787  N   LEU A 243      -8.981   7.843  29.592  1.00  0.00           N  \\nATOM   3788  CA  LEU A 243      -8.532   6.711  28.741  1.00  0.00           C  \\nATOM   3789  C   LEU A 243      -8.831   5.384  29.332  1.00  0.00           C  \\nATOM   3790  O   LEU A 243      -8.698   4.382  28.650  1.00  0.00           O  \\nATOM   3791  CB  LEU A 243      -7.044   6.809  28.407  1.00  0.00           C  \\nATOM   3792  CG  LEU A 243      -6.571   8.094  27.709  1.00  0.00           C  \\nATOM   3793  CD1 LEU A 243      -5.070   8.103  27.818  1.00  0.00           C  \\nATOM   3794  CD2 LEU A 243      -7.042   7.976  26.280  1.00  0.00           C  \\nATOM   3795  H   LEU A 243      -8.401   8.138  30.364  1.00  0.00           H  \\nATOM   3796  HA  LEU A 243      -9.090   6.739  27.806  1.00  0.00           H  \\nATOM   3797 1HB  LEU A 243      -6.474   6.719  29.330  1.00  0.00           H  \\nATOM   3798 2HB  LEU A 243      -6.778   5.975  27.757  1.00  0.00           H  \\nATOM   3799  HG  LEU A 243      -7.013   8.960  28.203  1.00  0.00           H  \\nATOM   3800 1HD1 LEU A 243      -4.676   8.998  27.337  1.00  0.00           H  \\nATOM   3801 2HD1 LEU A 243      -4.781   8.100  28.869  1.00  0.00           H  \\nATOM   3802 3HD1 LEU A 243      -4.664   7.219  27.327  1.00  0.00           H  \\nATOM   3803 1HD2 LEU A 243      -6.738   8.863  25.722  1.00  0.00           H  \\nATOM   3804 2HD2 LEU A 243      -6.600   7.091  25.823  1.00  0.00           H  \\nATOM   3805 3HD2 LEU A 243      -8.129   7.890  26.261  1.00  0.00           H  \\nATOM   3806  N   ILE A 244      -9.220   5.268  30.619  1.00  0.00           N  \\nATOM   3807  CA  ILE A 244      -9.508   3.970  31.195  1.00  0.00           C  \\nATOM   3808  C   ILE A 244     -10.632   3.187  30.418  1.00  0.00           C  \\nATOM   3809  O   ILE A 244     -10.459   1.996  30.279  1.00  0.00           O  \\nATOM   3810  CB  ILE A 244      -9.799   4.119  32.702  1.00  0.00           C  \\nATOM   3811  CG1 ILE A 244      -8.547   4.530  33.401  1.00  0.00           C  \\nATOM   3812  CG2 ILE A 244     -10.402   2.821  33.292  1.00  0.00           C  \\nATOM   3813  CD1 ILE A 244      -8.764   5.024  34.889  1.00  0.00           C  \\nATOM   3814  H   ILE A 244      -9.314   6.093  31.194  1.00  0.00           H  \\nATOM   3815  HA  ILE A 244      -8.634   3.332  31.065  1.00  0.00           H  \\nATOM   3816  HB  ILE A 244     -10.509   4.931  32.857  1.00  0.00           H  \\nATOM   3817 1HG1 ILE A 244      -7.851   3.692  33.422  1.00  0.00           H  \\nATOM   3818 2HG1 ILE A 244      -8.069   5.337  32.846  1.00  0.00           H  \\nATOM   3819 1HG2 ILE A 244     -10.594   2.960  34.356  1.00  0.00           H  \\nATOM   3820 2HG2 ILE A 244     -11.336   2.589  32.782  1.00  0.00           H  \\nATOM   3821 3HG2 ILE A 244      -9.700   1.998  33.155  1.00  0.00           H  \\nATOM   3822 1HD1 ILE A 244      -7.804   5.304  35.323  1.00  0.00           H  \\nATOM   3823 2HD1 ILE A 244      -9.429   5.888  34.892  1.00  0.00           H  \\nATOM   3824 3HD1 ILE A 244      -9.207   4.222  35.477  1.00  0.00           H  \\nATOM   3825  N   PRO A 245     -11.659   3.857  29.851  1.00  0.00           N  \\nATOM   3826  CA  PRO A 245     -12.628   3.045  29.055  1.00  0.00           C  \\nATOM   3827  C   PRO A 245     -12.164   2.649  27.663  1.00  0.00           C  \\nATOM   3828  O   PRO A 245     -12.840   1.881  26.990  1.00  0.00           O  \\nATOM   3829  CB  PRO A 245     -13.873   3.962  28.900  1.00  0.00           C  \\nATOM   3830  CG  PRO A 245     -13.704   4.973  29.950  1.00  0.00           C  \\nATOM   3831  CD  PRO A 245     -12.215   5.226  30.087  1.00  0.00           C  \\nATOM   3832  HA  PRO A 245     -12.891   2.139  29.620  1.00  0.00           H  \\nATOM   3833 1HB  PRO A 245     -13.898   4.395  27.889  1.00  0.00           H  \\nATOM   3834 2HB  PRO A 245     -14.793   3.371  29.019  1.00  0.00           H  \\nATOM   3835 1HG  PRO A 245     -14.243   5.893  29.681  1.00  0.00           H  \\nATOM   3836 2HG  PRO A 245     -14.138   4.615  30.895  1.00  0.00           H  \\nATOM   3837 1HD  PRO A 245     -11.893   5.948  29.322  1.00  0.00           H  \\nATOM   3838 2HD  PRO A 245     -11.999   5.608  31.096  1.00  0.00           H  \\nATOM   3839  N   ILE A 246     -10.993   3.157  27.251  1.00  0.00           N  \\nATOM   3840  CA  ILE A 246     -10.548   3.207  25.860  1.00  0.00           C  \\nATOM   3841  C   ILE A 246      -9.445   2.152  25.651  1.00  0.00           C  \\nATOM   3842  O   ILE A 246      -9.436   1.454  24.665  1.00  0.00           O  \\nATOM   3843  CB  ILE A 246      -9.937   4.612  25.552  1.00  0.00           C  \\nATOM   3844  CG1 ILE A 246     -11.010   5.718  25.636  1.00  0.00           C  \\nATOM   3845  CG2 ILE A 246      -9.498   4.721  24.096  1.00  0.00           C  \\nATOM   3846  CD1 ILE A 246     -12.331   5.432  24.953  1.00  0.00           C  \\nATOM   3847  H   ILE A 246     -10.392   3.526  27.974  1.00  0.00           H  \\nATOM   3848  HA  ILE A 246     -11.411   3.046  25.215  1.00  0.00           H  \\nATOM   3849  HB  ILE A 246      -9.068   4.778  26.188  1.00  0.00           H  \\nATOM   3850 1HG1 ILE A 246     -11.235   5.928  26.681  1.00  0.00           H  \\nATOM   3851 2HG1 ILE A 246     -10.621   6.636  25.195  1.00  0.00           H  \\nATOM   3852 1HG2 ILE A 246      -9.078   5.710  23.916  1.00  0.00           H  \\nATOM   3853 2HG2 ILE A 246      -8.744   3.964  23.886  1.00  0.00           H  \\nATOM   3854 3HG2 ILE A 246     -10.358   4.567  23.444  1.00  0.00           H  \\nATOM   3855 1HD1 ILE A 246     -12.999   6.284  25.082  1.00  0.00           H  \\nATOM   3856 2HD1 ILE A 246     -12.162   5.262  23.889  1.00  0.00           H  \\nATOM   3857 3HD1 ILE A 246     -12.784   4.546  25.395  1.00  0.00           H  \\nATOM   3858  N   ILE A 247      -8.493   2.108  26.607  1.00  0.00           N  \\nATOM   3859  CA  ILE A 247      -7.323   1.235  26.493  1.00  0.00           C  \\nATOM   3860  C   ILE A 247      -7.397   0.038  27.442  1.00  0.00           C  \\nATOM   3861  O   ILE A 247      -7.642   0.194  28.621  1.00  0.00           O  \\nATOM   3862  CB  ILE A 247      -5.989   2.041  26.634  1.00  0.00           C  \\nATOM   3863  CG1 ILE A 247      -4.765   1.116  26.421  1.00  0.00           C  \\nATOM   3864  CG2 ILE A 247      -5.894   2.679  27.947  1.00  0.00           C  \\nATOM   3865  CD1 ILE A 247      -3.415   1.930  26.148  1.00  0.00           C  \\nATOM   3866  H   ILE A 247      -8.592   2.694  27.423  1.00  0.00           H  \\nATOM   3867  HA  ILE A 247      -7.335   0.768  25.509  1.00  0.00           H  \\nATOM   3868  HB  ILE A 247      -5.949   2.815  25.868  1.00  0.00           H  \\nATOM   3869 1HG1 ILE A 247      -4.626   0.489  27.301  1.00  0.00           H  \\nATOM   3870 2HG1 ILE A 247      -4.952   0.454  25.575  1.00  0.00           H  \\nATOM   3871 1HG2 ILE A 247      -4.957   3.231  28.016  1.00  0.00           H  \\nATOM   3872 2HG2 ILE A 247      -6.730   3.365  28.079  1.00  0.00           H  \\nATOM   3873 3HG2 ILE A 247      -5.924   1.916  28.725  1.00  0.00           H  \\nATOM   3874 1HD1 ILE A 247      -2.591   1.231  26.006  1.00  0.00           H  \\nATOM   3875 2HD1 ILE A 247      -3.532   2.540  25.251  1.00  0.00           H  \\nATOM   3876 3HD1 ILE A 247      -3.200   2.574  27.000  1.00  0.00           H  \\nATOM   3877  N   ALA A 248      -7.216  -1.155  26.890  1.00  0.00           N  \\nATOM   3878  CA  ALA A 248      -7.411  -2.402  27.614  1.00  0.00           C  \\nATOM   3879  C   ALA A 248      -6.127  -3.039  28.076  1.00  0.00           C  \\nATOM   3880  O   ALA A 248      -6.135  -3.818  29.061  1.00  0.00           O  \\nATOM   3881  CB  ALA A 248      -8.239  -3.423  26.731  1.00  0.00           C  \\nATOM   3882  H   ALA A 248      -6.930  -1.188  25.922  1.00  0.00           H  \\nATOM   3883  HA  ALA A 248      -7.972  -2.182  28.522  1.00  0.00           H  \\nATOM   3884 1HB  ALA A 248      -8.378  -4.352  27.284  1.00  0.00           H  \\nATOM   3885 2HB  ALA A 248      -9.212  -2.993  26.493  1.00  0.00           H  \\nATOM   3886 3HB  ALA A 248      -7.698  -3.628  25.808  1.00  0.00           H  \\nATOM   3887  N   LEU A 249      -5.039  -2.801  27.373  1.00  0.00           N  \\nATOM   3888  CA  LEU A 249      -3.733  -3.374  27.661  1.00  0.00           C  \\nATOM   3889  C   LEU A 249      -2.632  -2.354  27.383  1.00  0.00           C  \\nATOM   3890  O   LEU A 249      -2.756  -1.570  26.565  1.00  0.00           O  \\nATOM   3891  CB  LEU A 249      -3.404  -4.520  26.682  1.00  0.00           C  \\nATOM   3892  CG  LEU A 249      -4.285  -5.719  26.616  1.00  0.00           C  \\nATOM   3893  CD1 LEU A 249      -4.050  -6.572  25.419  1.00  0.00           C  \\nATOM   3894  CD2 LEU A 249      -4.282  -6.503  27.802  1.00  0.00           C  \\nATOM   3895  H   LEU A 249      -5.139  -2.176  26.586  1.00  0.00           H  \\nATOM   3896  HA  LEU A 249      -3.749  -3.780  28.672  1.00  0.00           H  \\nATOM   3897 1HB  LEU A 249      -3.386  -4.118  25.670  1.00  0.00           H  \\nATOM   3898 2HB  LEU A 249      -2.411  -4.903  26.916  1.00  0.00           H  \\nATOM   3899  HG  LEU A 249      -5.314  -5.405  26.440  1.00  0.00           H  \\nATOM   3900 1HD1 LEU A 249      -4.729  -7.425  25.441  1.00  0.00           H  \\nATOM   3901 2HD1 LEU A 249      -4.230  -5.989  24.516  1.00  0.00           H  \\nATOM   3902 3HD1 LEU A 249      -3.020  -6.928  25.423  1.00  0.00           H  \\nATOM   3903 1HD2 LEU A 249      -4.948  -7.357  27.675  1.00  0.00           H  \\nATOM   3904 2HD2 LEU A 249      -3.271  -6.858  28.001  1.00  0.00           H  \\nATOM   3905 3HD2 LEU A 249      -4.625  -5.896  28.640  1.00  0.00           H  \\nATOM   3906  N   MET A 250      -1.520  -2.468  28.092  1.00  0.00           N  \\nATOM   3907  CA  MET A 250      -0.362  -1.632  27.848  1.00  0.00           C  \\nATOM   3908  C   MET A 250       0.905  -2.474  27.752  1.00  0.00           C  \\nATOM   3909  O   MET A 250       1.041  -3.403  28.444  1.00  0.00           O  \\nATOM   3910  CB  MET A 250      -0.161  -0.647  29.005  1.00  0.00           C  \\nATOM   3911  CG  MET A 250      -1.233   0.286  29.106  1.00  0.00           C  \\nATOM   3912  SD  MET A 250      -0.976   1.582  30.336  1.00  0.00           S  \\nATOM   3913  CE  MET A 250      -1.037   0.597  31.797  1.00  0.00           C  \\nATOM   3914  H   MET A 250      -1.481  -3.161  28.826  1.00  0.00           H  \\nATOM   3915  HA  MET A 250      -0.531  -1.065  26.933  1.00  0.00           H  \\nATOM   3916 1HB  MET A 250      -0.079  -1.198  29.941  1.00  0.00           H  \\nATOM   3917 2HB  MET A 250       0.773  -0.103  28.862  1.00  0.00           H  \\nATOM   3918 1HG  MET A 250      -1.385   0.772  28.143  1.00  0.00           H  \\nATOM   3919 2HG  MET A 250      -2.150  -0.241  29.370  1.00  0.00           H  \\nATOM   3920 1HE  MET A 250      -0.896   1.233  32.671  1.00  0.00           H  \\nATOM   3921 2HE  MET A 250      -2.006   0.101  31.861  1.00  0.00           H  \\nATOM   3922 3HE  MET A 250      -0.247  -0.154  31.763  1.00  0.00           H  \\nATOM   3923  N   VAL A 251       1.829  -2.029  26.917  1.00  0.00           N  \\nATOM   3924  CA  VAL A 251       3.111  -2.615  26.911  1.00  0.00           C  \\nATOM   3925  C   VAL A 251       3.850  -2.190  28.179  1.00  0.00           C  \\nATOM   3926  O   VAL A 251       3.754  -1.083  28.581  1.00  0.00           O  \\nATOM   3927  CB  VAL A 251       3.881  -2.247  25.664  1.00  0.00           C  \\nATOM   3928  CG1 VAL A 251       3.934  -0.846  25.474  1.00  0.00           C  \\nATOM   3929  CG2 VAL A 251       5.265  -2.854  25.681  1.00  0.00           C  \\nATOM   3930  H   VAL A 251       1.631  -1.271  26.279  1.00  0.00           H  \\nATOM   3931  HA  VAL A 251       2.996  -3.699  26.932  1.00  0.00           H  \\nATOM   3932  HB  VAL A 251       3.342  -2.619  24.792  1.00  0.00           H  \\nATOM   3933 1HG1 VAL A 251       4.496  -0.623  24.567  1.00  0.00           H  \\nATOM   3934 2HG1 VAL A 251       2.922  -0.453  25.379  1.00  0.00           H  \\nATOM   3935 3HG1 VAL A 251       4.426  -0.381  26.328  1.00  0.00           H  \\nATOM   3936 1HG2 VAL A 251       5.796  -2.574  24.772  1.00  0.00           H  \\nATOM   3937 2HG2 VAL A 251       5.813  -2.487  26.550  1.00  0.00           H  \\nATOM   3938 3HG2 VAL A 251       5.185  -3.940  25.734  1.00  0.00           H  \\nATOM   3939  N   TYR A 252       4.579  -3.123  28.768  1.00  0.00           N  \\nATOM   3940  CA  TYR A 252       5.309  -2.901  30.012  1.00  0.00           C  \\nATOM   3941  C   TYR A 252       6.391  -1.888  29.820  1.00  0.00           C  \\nATOM   3942  O   TYR A 252       7.198  -2.025  28.955  1.00  0.00           O  \\nATOM   3943  CB  TYR A 252       5.964  -4.190  30.465  1.00  0.00           C  \\nATOM   3944  CG  TYR A 252       6.696  -4.125  31.785  1.00  0.00           C  \\nATOM   3945  CD1 TYR A 252       7.976  -3.681  31.878  1.00  0.00           C  \\nATOM   3946  CD2 TYR A 252       6.096  -4.540  32.908  1.00  0.00           C  \\nATOM   3947  CE1 TYR A 252       8.585  -3.622  33.098  1.00  0.00           C  \\nATOM   3948  CE2 TYR A 252       6.712  -4.502  34.109  1.00  0.00           C  \\nATOM   3949  CZ  TYR A 252       7.934  -4.055  34.198  1.00  0.00           C  \\nATOM   3950  OH  TYR A 252       8.475  -4.076  35.451  1.00  0.00           O  \\nATOM   3951  H   TYR A 252       4.625  -4.030  28.326  1.00  0.00           H  \\nATOM   3952  HA  TYR A 252       4.601  -2.578  30.776  1.00  0.00           H  \\nATOM   3953 1HB  TYR A 252       5.208  -4.971  30.555  1.00  0.00           H  \\nATOM   3954 2HB  TYR A 252       6.683  -4.516  29.714  1.00  0.00           H  \\nATOM   3955  HD1 TYR A 252       8.513  -3.373  30.980  1.00  0.00           H  \\nATOM   3956  HD2 TYR A 252       5.077  -4.925  32.871  1.00  0.00           H  \\nATOM   3957  HE1 TYR A 252       9.612  -3.266  33.178  1.00  0.00           H  \\nATOM   3958  HE2 TYR A 252       6.188  -4.845  35.001  1.00  0.00           H  \\nATOM   3959  HH  TYR A 252       7.832  -4.431  36.070  1.00  0.00           H  \\nATOM   3960  N   ARG A 253       6.421  -0.876  30.681  1.00  0.00           N  \\nATOM   3961  CA  ARG A 253       7.480   0.094  30.637  1.00  0.00           C  \\nATOM   3962  C   ARG A 253       8.390   0.078  31.872  1.00  0.00           C  \\nATOM   3963  O   ARG A 253       9.609   0.085  31.760  1.00  0.00           O  \\nATOM   3964  CB  ARG A 253       6.921   1.506  30.436  1.00  0.00           C  \\nATOM   3965  CG  ARG A 253       6.321   1.825  29.092  1.00  0.00           C  \\nATOM   3966  CD  ARG A 253       7.107   1.374  27.938  1.00  0.00           C  \\nATOM   3967  NE  ARG A 253       6.574   1.923  26.724  1.00  0.00           N  \\nATOM   3968  CZ  ARG A 253       6.806   1.457  25.535  1.00  0.00           C  \\nATOM   3969  NH1 ARG A 253       7.555   0.419  25.365  1.00  0.00           N  \\nATOM   3970  NH2 ARG A 253       6.274   2.046  24.498  1.00  0.00           N  \\nATOM   3971  H   ARG A 253       5.695  -0.781  31.377  1.00  0.00           H  \\nATOM   3972  HA  ARG A 253       8.131  -0.142  29.794  1.00  0.00           H  \\nATOM   3973 1HB  ARG A 253       6.142   1.697  31.172  1.00  0.00           H  \\nATOM   3974 2HB  ARG A 253       7.712   2.237  30.600  1.00  0.00           H  \\nATOM   3975 1HG  ARG A 253       5.343   1.350  29.008  1.00  0.00           H  \\nATOM   3976 2HG  ARG A 253       6.210   2.905  28.990  1.00  0.00           H  \\nATOM   3977 1HD  ARG A 253       8.140   1.701  28.051  1.00  0.00           H  \\nATOM   3978 2HD  ARG A 253       7.075   0.287  27.879  1.00  0.00           H  \\nATOM   3979  HE  ARG A 253       5.973   2.733  26.798  1.00  0.00           H  \\nATOM   3980 1HH1 ARG A 253       7.969  -0.041  26.163  1.00  0.00           H  \\nATOM   3981 2HH1 ARG A 253       7.727   0.067  24.434  1.00  0.00           H  \\nATOM   3982 1HH2 ARG A 253       5.686   2.859  24.625  1.00  0.00           H  \\nATOM   3983 2HH2 ARG A 253       6.450   1.690  23.570  1.00  0.00           H  \\nATOM   3984  N   CYS A 254       7.793   0.073  33.028  1.00  0.00           N  \\nATOM   3985  CA  CYS A 254       8.563   0.171  34.249  1.00  0.00           C  \\nATOM   3986  C   CYS A 254       7.880  -0.659  35.315  1.00  0.00           C  \\nATOM   3987  O   CYS A 254       6.740  -1.044  35.188  1.00  0.00           O  \\nATOM   3988  CB  CYS A 254       8.779   1.619  34.726  1.00  0.00           C  \\nATOM   3989  SG  CYS A 254       7.444   2.885  34.418  1.00  0.00           S  \\nATOM   3990  H   CYS A 254       6.787   0.001  33.085  1.00  0.00           H  \\nATOM   3991  HA  CYS A 254       9.549  -0.259  34.072  1.00  0.00           H  \\nATOM   3992 1HB  CYS A 254       8.940   1.627  35.804  1.00  0.00           H  \\nATOM   3993 2HB  CYS A 254       9.675   2.026  34.258  1.00  0.00           H  \\nATOM   3994  HG  CYS A 254       8.084   3.902  34.986  1.00  0.00           H  \\nATOM   3995  N   ALA A 255       8.637  -0.960  36.347  1.00  0.00           N  \\nATOM   3996  CA  ALA A 255       8.153  -1.771  37.425  1.00  0.00           C  \\nATOM   3997  C   ALA A 255       7.132  -0.958  38.146  1.00  0.00           C  \\nATOM   3998  O   ALA A 255       7.087   0.214  37.930  1.00  0.00           O  \\nATOM   3999  OXT ALA A 255       6.383  -1.486  38.920  1.00  0.00           O  \\nATOM   4000  CB  ALA A 255       9.274  -2.127  38.331  1.00  0.00           C  \\nATOM   4001  H   ALA A 255       9.584  -0.611  36.381  1.00  0.00           H  \\nATOM   4002  HA  ALA A 255       7.733  -2.683  37.001  1.00  0.00           H  \\nATOM   4003 1HB  ALA A 255       8.900  -2.743  39.148  1.00  0.00           H  \\nATOM   4004 2HB  ALA A 255      10.029  -2.682  37.775  1.00  0.00           H  \\nATOM   4005 3HB  ALA A 255       9.717  -1.218  38.736  1.00  0.00           H  \\nTER                                                                             \\nATOM   4007  N   TYR B 256      20.163  27.071  23.881  1.00  0.00           N  \\nATOM   4008  CA  TYR B 256      20.779  25.704  23.868  1.00  0.00           C  \\nATOM   4009  C   TYR B 256      22.287  25.786  23.845  1.00  0.00           C  \\nATOM   4010  O   TYR B 256      22.840  26.596  23.112  1.00  0.00           O  \\nATOM   4011  CB  TYR B 256      20.268  24.831  22.672  1.00  0.00           C  \\nATOM   4012  CG  TYR B 256      18.792  24.511  22.692  1.00  0.00           C  \\nATOM   4013  CD1 TYR B 256      18.223  23.933  23.826  1.00  0.00           C  \\nATOM   4014  CD2 TYR B 256      17.956  24.793  21.587  1.00  0.00           C  \\nATOM   4015  CE1 TYR B 256      16.848  23.646  23.884  1.00  0.00           C  \\nATOM   4016  CE2 TYR B 256      16.581  24.486  21.599  1.00  0.00           C  \\nATOM   4017  CZ  TYR B 256      16.067  23.948  22.811  1.00  0.00           C  \\nATOM   4018  OH  TYR B 256      14.731  23.636  22.786  1.00  0.00           O  \\nATOM   4019 1H   TYR B 256      19.167  26.988  23.896  1.00  0.00           H  \\nATOM   4020 2H   TYR B 256      20.468  27.567  24.694  1.00  0.00           H  \\nATOM   4021 3H   TYR B 256      20.443  27.571  23.062  1.00  0.00           H  \\nATOM   4022  HA  TYR B 256      20.507  25.193  24.792  1.00  0.00           H  \\nATOM   4023 1HB  TYR B 256      20.478  25.343  21.732  1.00  0.00           H  \\nATOM   4024 2HB  TYR B 256      20.807  23.884  22.657  1.00  0.00           H  \\nATOM   4025  HD1 TYR B 256      18.847  23.696  24.688  1.00  0.00           H  \\nATOM   4026  HD2 TYR B 256      18.372  25.259  20.694  1.00  0.00           H  \\nATOM   4027  HE1 TYR B 256      16.423  23.194  24.780  1.00  0.00           H  \\nATOM   4028  HE2 TYR B 256      15.975  24.663  20.710  1.00  0.00           H  \\nATOM   4029  HH  TYR B 256      14.371  23.846  21.921  1.00  0.00           H  \\nATOM   4030  N   PRO B 257      22.972  24.918  24.592  1.00  0.00           N  \\nATOM   4031  CA  PRO B 257      24.421  24.990  24.590  1.00  0.00           C  \\nATOM   4032  C   PRO B 257      24.989  24.604  23.211  1.00  0.00           C  \\nATOM   4033  O   PRO B 257      24.434  23.709  22.556  1.00  0.00           O  \\nATOM   4034  CB  PRO B 257      24.831  23.918  25.614  1.00  0.00           C  \\nATOM   4035  CG  PRO B 257      23.606  23.538  26.321  1.00  0.00           C  \\nATOM   4036  CD  PRO B 257      22.496  23.788  25.385  1.00  0.00           C  \\nATOM   4037  HA  PRO B 257      24.734  25.990  24.924  1.00  0.00           H  \\nATOM   4038 1HB  PRO B 257      25.288  23.061  25.098  1.00  0.00           H  \\nATOM   4039 2HB  PRO B 257      25.590  24.323  26.299  1.00  0.00           H  \\nATOM   4040 1HG  PRO B 257      23.654  22.481  26.623  1.00  0.00           H  \\nATOM   4041 2HG  PRO B 257      23.501  24.128  27.244  1.00  0.00           H  \\nATOM   4042 1HD  PRO B 257      22.333  22.896  24.762  1.00  0.00           H  \\nATOM   4043 2HD  PRO B 257      21.587  24.036  25.952  1.00  0.00           H  \\nATOM   4044  N   ILE B 258      26.048  25.284  22.778  1.00  0.00           N  \\nATOM   4045  CA  ILE B 258      26.786  24.989  21.531  1.00  0.00           C  \\nATOM   4046  C   ILE B 258      28.186  24.449  21.849  1.00  0.00           C  \\nATOM   4047  O   ILE B 258      28.981  25.052  22.624  1.00  0.00           O  \\nATOM   4048  CB  ILE B 258      26.936  26.246  20.594  1.00  0.00           C  \\nATOM   4049  CG1 ILE B 258      25.551  26.855  20.249  1.00  0.00           C  \\nATOM   4050  CG2 ILE B 258      27.669  25.883  19.341  1.00  0.00           C  \\nATOM   4051  CD1 ILE B 258      25.649  28.273  19.519  1.00  0.00           C  \\nATOM   4052  H   ILE B 258      26.351  26.052  23.359  1.00  0.00           H  \\nATOM   4053  HA  ILE B 258      26.236  24.231  20.975  1.00  0.00           H  \\nATOM   4054  HB  ILE B 258      27.492  27.024  21.116  1.00  0.00           H  \\nATOM   4055 1HG1 ILE B 258      25.003  26.170  19.603  1.00  0.00           H  \\nATOM   4056 2HG1 ILE B 258      24.969  26.977  21.163  1.00  0.00           H  \\nATOM   4057 1HG2 ILE B 258      27.762  26.764  18.706  1.00  0.00           H  \\nATOM   4058 2HG2 ILE B 258      28.661  25.512  19.595  1.00  0.00           H  \\nATOM   4059 3HG2 ILE B 258      27.117  25.108  18.808  1.00  0.00           H  \\nATOM   4060 1HD1 ILE B 258      24.646  28.643  19.306  1.00  0.00           H  \\nATOM   4061 2HD1 ILE B 258      26.167  28.981  20.167  1.00  0.00           H  \\nATOM   4062 3HD1 ILE B 258      26.200  28.164  18.586  1.00  0.00           H  \\nATOM   4063  N   ILE B 259      28.531  23.324  21.238  1.00  0.00           N  \\nATOM   4064  CA  ILE B 259      29.807  22.702  21.371  1.00  0.00           C  \\nATOM   4065  C   ILE B 259      30.473  22.735  19.985  1.00  0.00           C  \\nATOM   4066  O   ILE B 259      29.839  22.294  19.017  1.00  0.00           O  \\nATOM   4067  CB  ILE B 259      29.724  21.184  21.748  1.00  0.00           C  \\nATOM   4068  CG1 ILE B 259      29.077  20.938  23.065  1.00  0.00           C  \\nATOM   4069  CG2 ILE B 259      31.178  20.638  22.013  1.00  0.00           C  \\nATOM   4070  CD1 ILE B 259      27.632  20.865  23.025  1.00  0.00           C  \\nATOM   4071  H   ILE B 259      27.834  22.895  20.646  1.00  0.00           H  \\nATOM   4072  HA  ILE B 259      30.354  23.203  22.169  1.00  0.00           H  \\nATOM   4073  HB  ILE B 259      29.269  20.630  20.928  1.00  0.00           H  \\nATOM   4074 1HG1 ILE B 259      29.446  20.002  23.482  1.00  0.00           H  \\nATOM   4075 2HG1 ILE B 259      29.349  21.734  23.759  1.00  0.00           H  \\nATOM   4076 1HG2 ILE B 259      31.128  19.582  22.277  1.00  0.00           H  \\nATOM   4077 2HG2 ILE B 259      31.781  20.759  21.114  1.00  0.00           H  \\nATOM   4078 3HG2 ILE B 259      31.632  21.195  22.833  1.00  0.00           H  \\nATOM   4079 1HD1 ILE B 259      27.249  20.684  24.030  1.00  0.00           H  \\nATOM   4080 2HD1 ILE B 259      27.230  21.806  22.648  1.00  0.00           H  \\nATOM   4081 3HD1 ILE B 259      27.327  20.051  22.369  1.00  0.00           H  \\nATOM   4082  N   ASN B 260      31.728  23.183  19.883  1.00  0.00           N  \\nATOM   4083  CA  ASN B 260      32.328  23.386  18.557  1.00  0.00           C  \\nATOM   4084  C   ASN B 260      33.450  22.342  18.266  1.00  0.00           C  \\nATOM   4085  O   ASN B 260      34.133  21.819  19.182  1.00  0.00           O  \\nATOM   4086  CB  ASN B 260      33.050  24.758  18.501  1.00  0.00           C  \\nATOM   4087  CG  ASN B 260      32.227  25.901  18.966  1.00  0.00           C  \\nATOM   4088  OD1 ASN B 260      32.518  26.507  20.027  1.00  0.00           O  \\nATOM   4089  ND2 ASN B 260      31.250  26.257  18.188  1.00  0.00           N  \\nATOM   4090  H   ASN B 260      32.276  23.386  20.707  1.00  0.00           H  \\nATOM   4091  HA  ASN B 260      31.531  23.377  17.811  1.00  0.00           H  \\nATOM   4092 1HB  ASN B 260      33.950  24.721  19.116  1.00  0.00           H  \\nATOM   4093 2HB  ASN B 260      33.361  24.964  17.477  1.00  0.00           H  \\nATOM   4094 1HD2 ASN B 260      30.658  27.021  18.446  1.00  0.00           H  \\nATOM   4095 2HD2 ASN B 260      31.089  25.767  17.332  1.00  0.00           H  \\nATOM   4096  N   PHE B 261      33.605  22.029  16.984  1.00  0.00           N  \\nATOM   4097  CA  PHE B 261      34.721  21.149  16.533  1.00  0.00           C  \\nATOM   4098  C   PHE B 261      35.032  21.565  15.122  1.00  0.00           C  \\nATOM   4099  O   PHE B 261      34.098  21.846  14.328  1.00  0.00           O  \\nATOM   4100  CB  PHE B 261      34.315  19.669  16.588  1.00  0.00           C  \\nATOM   4101  CG  PHE B 261      35.403  18.758  16.057  1.00  0.00           C  \\nATOM   4102  CD1 PHE B 261      36.558  18.538  16.815  1.00  0.00           C  \\nATOM   4103  CD2 PHE B 261      35.224  18.056  14.840  1.00  0.00           C  \\nATOM   4104  CE1 PHE B 261      37.556  17.693  16.399  1.00  0.00           C  \\nATOM   4105  CE2 PHE B 261      36.200  17.186  14.434  1.00  0.00           C  \\nATOM   4106  CZ  PHE B 261      37.377  17.075  15.171  1.00  0.00           C  \\nATOM   4107  H   PHE B 261      32.957  22.392  16.299  1.00  0.00           H  \\nATOM   4108  HA  PHE B 261      35.570  21.297  17.201  1.00  0.00           H  \\nATOM   4109 1HB  PHE B 261      34.090  19.392  17.617  1.00  0.00           H  \\nATOM   4110 2HB  PHE B 261      33.409  19.519  16.003  1.00  0.00           H  \\nATOM   4111  HD1 PHE B 261      36.674  19.052  17.770  1.00  0.00           H  \\nATOM   4112  HD2 PHE B 261      34.301  18.146  14.266  1.00  0.00           H  \\nATOM   4113  HE1 PHE B 261      38.433  17.536  17.026  1.00  0.00           H  \\nATOM   4114  HE2 PHE B 261      36.060  16.582  13.538  1.00  0.00           H  \\nATOM   4115  HZ  PHE B 261      38.176  16.472  14.742  1.00  0.00           H  \\nATOM   4116  N   THR B 262      36.330  21.558  14.815  1.00  0.00           N  \\nATOM   4117  CA  THR B 262      36.727  21.708  13.455  1.00  0.00           C  \\nATOM   4118  C   THR B 262      37.588  20.577  12.951  1.00  0.00           C  \\nATOM   4119  O   THR B 262      38.427  20.041  13.717  1.00  0.00           O  \\nATOM   4120  CB  THR B 262      37.512  23.049  13.226  1.00  0.00           C  \\nATOM   4121  OG1 THR B 262      37.798  23.131  11.850  1.00  0.00           O  \\nATOM   4122  CG2 THR B 262      38.825  23.191  14.055  1.00  0.00           C  \\nATOM   4123  H   THR B 262      37.040  21.449  15.525  1.00  0.00           H  \\nATOM   4124  HA  THR B 262      35.831  21.730  12.834  1.00  0.00           H  \\nATOM   4125  HB  THR B 262      36.876  23.893  13.493  1.00  0.00           H  \\nATOM   4126  HG1 THR B 262      37.446  22.357  11.404  1.00  0.00           H  \\nATOM   4127 1HG2 THR B 262      39.296  24.148  13.830  1.00  0.00           H  \\nATOM   4128 2HG2 THR B 262      38.591  23.142  15.118  1.00  0.00           H  \\nATOM   4129 3HG2 THR B 262      39.507  22.382  13.797  1.00  0.00           H  \\nATOM   4130  N   THR B 263      37.353  20.161  11.713  1.00  0.00           N  \\nATOM   4131  CA  THR B 263      38.202  19.091  11.130  1.00  0.00           C  \\nATOM   4132  C   THR B 263      39.564  19.669  10.732  1.00  0.00           C  \\nATOM   4133  O   THR B 263      40.518  18.929  10.492  1.00  0.00           O  \\nATOM   4134  CB  THR B 263      37.591  18.376   9.970  1.00  0.00           C  \\nATOM   4135  OG1 THR B 263      37.181  19.413   9.005  1.00  0.00           O  \\nATOM   4136  CG2 THR B 263      36.366  17.599  10.373  1.00  0.00           C  \\nATOM   4137  H   THR B 263      36.608  20.556  11.157  1.00  0.00           H  \\nATOM   4138  HA  THR B 263      38.387  18.339  11.898  1.00  0.00           H  \\nATOM   4139  HB  THR B 263      38.317  17.682   9.547  1.00  0.00           H  \\nATOM   4140  HG1 THR B 263      37.404  20.280   9.354  1.00  0.00           H  \\nATOM   4141 1HG2 THR B 263      35.952  17.094   9.501  1.00  0.00           H  \\nATOM   4142 2HG2 THR B 263      36.636  16.860  11.127  1.00  0.00           H  \\nATOM   4143 3HG2 THR B 263      35.622  18.280  10.783  1.00  0.00           H  \\nATOM   4144  N   ALA B 264      39.649  21.006  10.640  1.00  0.00           N  \\nATOM   4145  CA  ALA B 264      40.932  21.678  10.229  1.00  0.00           C  \\nATOM   4146  C   ALA B 264      42.070  21.467  11.236  1.00  0.00           C  \\nATOM   4147  O   ALA B 264      41.995  21.989  12.304  1.00  0.00           O  \\nATOM   4148  CB  ALA B 264      40.691  23.194  10.005  1.00  0.00           C  \\nATOM   4149  H   ALA B 264      38.841  21.575  10.849  1.00  0.00           H  \\nATOM   4150  HA  ALA B 264      41.265  21.229   9.293  1.00  0.00           H  \\nATOM   4151 1HB  ALA B 264      41.625  23.671   9.708  1.00  0.00           H  \\nATOM   4152 2HB  ALA B 264      39.947  23.332   9.220  1.00  0.00           H  \\nATOM   4153 3HB  ALA B 264      40.331  23.645  10.928  1.00  0.00           H  \\nATOM   4154  N   GLY B 265      43.053  20.591  10.926  1.00  0.00           N  \\nATOM   4155  CA  GLY B 265      44.182  20.394  11.770  1.00  0.00           C  \\nATOM   4156  C   GLY B 265      43.711  19.527  12.956  1.00  0.00           C  \\nATOM   4157  O   GLY B 265      44.231  19.612  14.040  1.00  0.00           O  \\nATOM   4158  H   GLY B 265      42.981  20.062  10.069  1.00  0.00           H  \\nATOM   4159 1HA  GLY B 265      44.979  19.910  11.205  1.00  0.00           H  \\nATOM   4160 2HA  GLY B 265      44.563  21.360  12.100  1.00  0.00           H  \\nATOM   4161  N   ALA B 266      42.640  18.772  12.766  1.00  0.00           N  \\nATOM   4162  CA  ALA B 266      42.113  18.009  13.948  1.00  0.00           C  \\nATOM   4163  C   ALA B 266      43.158  17.067  14.493  1.00  0.00           C  \\nATOM   4164  O   ALA B 266      43.873  16.446  13.697  1.00  0.00           O  \\nATOM   4165  CB  ALA B 266      40.941  17.151  13.489  1.00  0.00           C  \\nATOM   4166  H   ALA B 266      42.169  18.688  11.876  1.00  0.00           H  \\nATOM   4167  HA  ALA B 266      41.773  18.730  14.692  1.00  0.00           H  \\nATOM   4168 1HB  ALA B 266      40.547  16.590  14.336  1.00  0.00           H  \\nATOM   4169 2HB  ALA B 266      40.158  17.792  13.082  1.00  0.00           H  \\nATOM   4170 3HB  ALA B 266      41.278  16.457  12.720  1.00  0.00           H  \\nATOM   4171  N   THR B 267      43.229  16.883  15.825  1.00  0.00           N  \\nATOM   4172  CA  THR B 267      44.104  15.921  16.364  1.00  0.00           C  \\nATOM   4173  C   THR B 267      43.362  14.849  17.163  1.00  0.00           C  \\nATOM   4174  O   THR B 267      42.179  15.072  17.459  1.00  0.00           O  \\nATOM   4175  CB  THR B 267      45.153  16.661  17.255  1.00  0.00           C  \\nATOM   4176  OG1 THR B 267      44.503  17.302  18.364  1.00  0.00           O  \\nATOM   4177  CG2 THR B 267      45.905  17.663  16.358  1.00  0.00           C  \\nATOM   4178  H   THR B 267      42.656  17.432  16.450  1.00  0.00           H  \\nATOM   4179  HA  THR B 267      44.611  15.415  15.542  1.00  0.00           H  \\nATOM   4180  HB  THR B 267      45.844  15.935  17.682  1.00  0.00           H  \\nATOM   4181  HG1 THR B 267      43.558  17.137  18.319  1.00  0.00           H  \\nATOM   4182 1HG2 THR B 267      46.647  18.197  16.952  1.00  0.00           H  \\nATOM   4183 2HG2 THR B 267      46.403  17.126  15.551  1.00  0.00           H  \\nATOM   4184 3HG2 THR B 267      45.197  18.376  15.936  1.00  0.00           H  \\nATOM   4185  N   VAL B 268      44.025  13.779  17.658  1.00  0.00           N  \\nATOM   4186  CA  VAL B 268      43.414  12.868  18.579  1.00  0.00           C  \\nATOM   4187  C   VAL B 268      42.916  13.661  19.837  1.00  0.00           C  \\nATOM   4188  O   VAL B 268      41.772  13.437  20.248  1.00  0.00           O  \\nATOM   4189  CB  VAL B 268      44.449  11.772  19.075  1.00  0.00           C  \\nATOM   4190  CG1 VAL B 268      43.913  10.930  20.154  1.00  0.00           C  \\nATOM   4191  CG2 VAL B 268      44.719  10.906  17.883  1.00  0.00           C  \\nATOM   4192  H   VAL B 268      44.979  13.619  17.368  1.00  0.00           H  \\nATOM   4193  HA  VAL B 268      42.594  12.359  18.070  1.00  0.00           H  \\nATOM   4194  HB  VAL B 268      45.354  12.271  19.422  1.00  0.00           H  \\nATOM   4195 1HG1 VAL B 268      44.665  10.199  20.453  1.00  0.00           H  \\nATOM   4196 2HG1 VAL B 268      43.655  11.555  21.009  1.00  0.00           H  \\nATOM   4197 3HG1 VAL B 268      43.023  10.410  19.801  1.00  0.00           H  \\nATOM   4198 1HG2 VAL B 268      45.429  10.125  18.154  1.00  0.00           H  \\nATOM   4199 2HG2 VAL B 268      43.788  10.450  17.546  1.00  0.00           H  \\nATOM   4200 3HG2 VAL B 268      45.137  11.513  17.079  1.00  0.00           H  \\nATOM   4201  N   GLN B 269      43.709  14.607  20.364  1.00  0.00           N  \\nATOM   4202  CA  GLN B 269      43.309  15.336  21.591  1.00  0.00           C  \\nATOM   4203  C   GLN B 269      42.060  16.184  21.288  1.00  0.00           C  \\nATOM   4204  O   GLN B 269      41.162  16.183  22.115  1.00  0.00           O  \\nATOM   4205  CB  GLN B 269      44.452  16.231  22.141  1.00  0.00           C  \\nATOM   4206  CG  GLN B 269      44.178  16.617  23.711  1.00  0.00           C  \\nATOM   4207  CD  GLN B 269      43.906  15.359  24.734  1.00  0.00           C  \\nATOM   4208  OE1 GLN B 269      44.605  14.321  24.725  1.00  0.00           O  \\nATOM   4209  NE2 GLN B 269      42.936  15.522  25.624  1.00  0.00           N  \\nATOM   4210  H   GLN B 269      44.591  14.830  19.924  1.00  0.00           H  \\nATOM   4211  HA  GLN B 269      43.058  14.606  22.360  1.00  0.00           H  \\nATOM   4212 1HB  GLN B 269      45.402  15.705  22.050  1.00  0.00           H  \\nATOM   4213 2HB  GLN B 269      44.522  17.140  21.543  1.00  0.00           H  \\nATOM   4214 1HG  GLN B 269      45.047  17.146  24.101  1.00  0.00           H  \\nATOM   4215 2HG  GLN B 269      43.296  17.254  23.769  1.00  0.00           H  \\nATOM   4216 1HE2 GLN B 269      42.724  14.790  26.274  1.00  0.00           H  \\nATOM   4217 2HE2 GLN B 269      42.416  16.376  25.647  1.00  0.00           H  \\nATOM   4218  N   SER B 270      42.070  16.848  20.129  1.00  0.00           N  \\nATOM   4219  CA  SER B 270      40.987  17.835  19.901  1.00  0.00           C  \\nATOM   4220  C   SER B 270      39.702  17.059  19.691  1.00  0.00           C  \\nATOM   4221  O   SER B 270      38.647  17.543  20.151  1.00  0.00           O  \\nATOM   4222  CB  SER B 270      41.204  18.852  18.803  1.00  0.00           C  \\nATOM   4223  OG  SER B 270      41.164  18.257  17.532  1.00  0.00           O  \\nATOM   4224  H   SER B 270      42.769  16.711  19.413  1.00  0.00           H  \\nATOM   4225  HA  SER B 270      40.852  18.418  20.813  1.00  0.00           H  \\nATOM   4226 1HB  SER B 270      40.437  19.623  18.865  1.00  0.00           H  \\nATOM   4227 2HB  SER B 270      42.168  19.338  18.943  1.00  0.00           H  \\nATOM   4228  HG  SER B 270      41.004  17.322  17.682  1.00  0.00           H  \\nATOM   4229  N   TYR B 271      39.733  15.933  18.984  1.00  0.00           N  \\nATOM   4230  CA  TYR B 271      38.486  15.129  18.896  1.00  0.00           C  \\nATOM   4231  C   TYR B 271      38.072  14.516  20.197  1.00  0.00           C  \\nATOM   4232  O   TYR B 271      36.910  14.522  20.478  1.00  0.00           O  \\nATOM   4233  CB  TYR B 271      38.688  14.042  17.822  1.00  0.00           C  \\nATOM   4234  CG  TYR B 271      37.544  13.121  17.650  1.00  0.00           C  \\nATOM   4235  CD1 TYR B 271      36.414  13.503  16.884  1.00  0.00           C  \\nATOM   4236  CD2 TYR B 271      37.582  11.877  18.200  1.00  0.00           C  \\nATOM   4237  CE1 TYR B 271      35.329  12.654  16.751  1.00  0.00           C  \\nATOM   4238  CE2 TYR B 271      36.519  11.024  18.070  1.00  0.00           C  \\nATOM   4239  CZ  TYR B 271      35.416  11.447  17.320  1.00  0.00           C  \\nATOM   4240  OH  TYR B 271      34.299  10.627  17.179  1.00  0.00           O  \\nATOM   4241  H   TYR B 271      40.563  15.607  18.510  1.00  0.00           H  \\nATOM   4242  HA  TYR B 271      37.669  15.789  18.603  1.00  0.00           H  \\nATOM   4243 1HB  TYR B 271      38.881  14.514  16.858  1.00  0.00           H  \\nATOM   4244 2HB  TYR B 271      39.562  13.441  18.073  1.00  0.00           H  \\nATOM   4245  HD1 TYR B 271      36.393  14.475  16.391  1.00  0.00           H  \\nATOM   4246  HD2 TYR B 271      38.464  11.552  18.751  1.00  0.00           H  \\nATOM   4247  HE1 TYR B 271      34.467  12.960  16.159  1.00  0.00           H  \\nATOM   4248  HE2 TYR B 271      36.546  10.043  18.546  1.00  0.00           H  \\nATOM   4249  HH  TYR B 271      34.443   9.806  17.656  1.00  0.00           H  \\nATOM   4250  N   THR B 272      38.973  13.895  20.938  1.00  0.00           N  \\nATOM   4251  CA  THR B 272      38.692  13.348  22.259  1.00  0.00           C  \\nATOM   4252  C   THR B 272      37.968  14.468  23.184  1.00  0.00           C  \\nATOM   4253  O   THR B 272      36.957  14.208  23.826  1.00  0.00           O  \\nATOM   4254  CB  THR B 272      39.958  12.898  22.880  1.00  0.00           C  \\nATOM   4255  OG1 THR B 272      40.455  11.821  22.041  1.00  0.00           O  \\nATOM   4256  CG2 THR B 272      39.560  12.345  24.210  1.00  0.00           C  \\nATOM   4257  H   THR B 272      39.903  13.802  20.555  1.00  0.00           H  \\nATOM   4258  HA  THR B 272      38.024  12.494  22.145  1.00  0.00           H  \\nATOM   4259  HB  THR B 272      40.636  13.745  22.978  1.00  0.00           H  \\nATOM   4260  HG1 THR B 272      39.847  11.678  21.311  1.00  0.00           H  \\nATOM   4261 1HG2 THR B 272      40.446  11.989  24.736  1.00  0.00           H  \\nATOM   4262 2HG2 THR B 272      39.077  13.125  24.799  1.00  0.00           H  \\nATOM   4263 3HG2 THR B 272      38.867  11.517  24.067  1.00  0.00           H  \\nATOM   4264  N   ASN B 273      38.583  15.666  23.232  1.00  0.00           N  \\nATOM   4265  CA  ASN B 273      38.055  16.692  24.115  1.00  0.00           C  \\nATOM   4266  C   ASN B 273      36.679  17.096  23.610  1.00  0.00           C  \\nATOM   4267  O   ASN B 273      35.824  17.276  24.419  1.00  0.00           O  \\nATOM   4268  CB  ASN B 273      38.974  17.886  24.076  1.00  0.00           C  \\nATOM   4269  CG  ASN B 273      40.245  17.673  24.847  1.00  0.00           C  \\nATOM   4270  OD1 ASN B 273      40.416  16.704  25.603  1.00  0.00           O  \\nATOM   4271  ND2 ASN B 273      41.143  18.631  24.695  1.00  0.00           N  \\nATOM   4272  H   ASN B 273      39.399  15.870  22.673  1.00  0.00           H  \\nATOM   4273  HA  ASN B 273      38.022  16.292  25.130  1.00  0.00           H  \\nATOM   4274 1HB  ASN B 273      39.229  18.115  23.041  1.00  0.00           H  \\nATOM   4275 2HB  ASN B 273      38.459  18.755  24.487  1.00  0.00           H  \\nATOM   4276 1HD2 ASN B 273      42.021  18.574  25.171  1.00  0.00           H  \\nATOM   4277 2HD2 ASN B 273      40.945  19.413  24.104  1.00  0.00           H  \\nATOM   4278  N   PHE B 274      36.445  17.144  22.293  1.00  0.00           N  \\nATOM   4279  CA  PHE B 274      35.133  17.458  21.691  1.00  0.00           C  \\nATOM   4280  C   PHE B 274      34.168  16.407  22.156  1.00  0.00           C  \\nATOM   4281  O   PHE B 274      33.142  16.760  22.646  1.00  0.00           O  \\nATOM   4282  CB  PHE B 274      35.271  17.510  20.192  1.00  0.00           C  \\nATOM   4283  CG  PHE B 274      33.982  17.461  19.442  1.00  0.00           C  \\nATOM   4284  CD1 PHE B 274      33.036  18.404  19.682  1.00  0.00           C  \\nATOM   4285  CD2 PHE B 274      33.734  16.418  18.533  1.00  0.00           C  \\nATOM   4286  CE1 PHE B 274      31.801  18.419  18.995  1.00  0.00           C  \\nATOM   4287  CE2 PHE B 274      32.499  16.441  17.762  1.00  0.00           C  \\nATOM   4288  CZ  PHE B 274      31.537  17.402  18.012  1.00  0.00           C  \\nATOM   4289  H   PHE B 274      37.230  16.950  21.688  1.00  0.00           H  \\nATOM   4290  HA  PHE B 274      34.810  18.434  22.055  1.00  0.00           H  \\nATOM   4291 1HB  PHE B 274      35.785  18.427  19.906  1.00  0.00           H  \\nATOM   4292 2HB  PHE B 274      35.882  16.674  19.854  1.00  0.00           H  \\nATOM   4293  HD1 PHE B 274      33.231  19.176  20.427  1.00  0.00           H  \\nATOM   4294  HD2 PHE B 274      34.488  15.657  18.331  1.00  0.00           H  \\nATOM   4295  HE1 PHE B 274      31.066  19.193  19.213  1.00  0.00           H  \\nATOM   4296  HE2 PHE B 274      32.334  15.695  16.985  1.00  0.00           H  \\nATOM   4297  HZ  PHE B 274      30.589  17.394  17.476  1.00  0.00           H  \\nATOM   4298  N   ILE B 275      34.458  15.119  21.953  1.00  0.00           N  \\nATOM   4299  CA  ILE B 275      33.462  14.059  22.350  1.00  0.00           C  \\nATOM   4300  C   ILE B 275      33.131  14.116  23.841  1.00  0.00           C  \\nATOM   4301  O   ILE B 275      31.977  13.994  24.242  1.00  0.00           O  \\nATOM   4302  CB  ILE B 275      33.948  12.695  21.915  1.00  0.00           C  \\nATOM   4303  CG1 ILE B 275      33.942  12.678  20.359  1.00  0.00           C  \\nATOM   4304  CG2 ILE B 275      33.027  11.629  22.546  1.00  0.00           C  \\nATOM   4305  CD1 ILE B 275      32.570  12.956  19.569  1.00  0.00           C  \\nATOM   4306  H   ILE B 275      35.335  14.839  21.537  1.00  0.00           H  \\nATOM   4307  HA  ILE B 275      32.516  14.269  21.852  1.00  0.00           H  \\nATOM   4308  HB  ILE B 275      34.974  12.550  22.253  1.00  0.00           H  \\nATOM   4309 1HG1 ILE B 275      34.643  13.423  19.984  1.00  0.00           H  \\nATOM   4310 2HG1 ILE B 275      34.281  11.704  20.004  1.00  0.00           H  \\nATOM   4311 1HG2 ILE B 275      33.360  10.636  22.244  1.00  0.00           H  \\nATOM   4312 2HG2 ILE B 275      33.067  11.711  23.631  1.00  0.00           H  \\nATOM   4313 3HG2 ILE B 275      32.003  11.787  22.208  1.00  0.00           H  \\nATOM   4314 1HD1 ILE B 275      32.752  12.905  18.495  1.00  0.00           H  \\nATOM   4315 2HD1 ILE B 275      31.831  12.205  19.848  1.00  0.00           H  \\nATOM   4316 3HD1 ILE B 275      32.197  13.946  19.826  1.00  0.00           H  \\nATOM   4317  N   ARG B 276      34.182  14.325  24.651  1.00  0.00           N  \\nATOM   4318  CA  ARG B 276      33.971  14.472  26.111  1.00  0.00           C  \\nATOM   4319  C   ARG B 276      33.109  15.646  26.416  1.00  0.00           C  \\nATOM   4320  O   ARG B 276      32.283  15.529  27.371  1.00  0.00           O  \\nATOM   4321  CB  ARG B 276      35.326  14.622  26.794  1.00  0.00           C  \\nATOM   4322  CG  ARG B 276      36.020  13.223  26.923  1.00  0.00           C  \\nATOM   4323  CD  ARG B 276      37.405  13.450  27.455  1.00  0.00           C  \\nATOM   4324  NE  ARG B 276      38.122  12.201  27.528  1.00  0.00           N  \\nATOM   4325  CZ  ARG B 276      39.390  12.136  27.921  1.00  0.00           C  \\nATOM   4326  NH1 ARG B 276      40.024  13.252  28.295  1.00  0.00           N  \\nATOM   4327  NH2 ARG B 276      39.987  10.954  27.957  1.00  0.00           N  \\nATOM   4328  H   ARG B 276      35.120  14.384  24.281  1.00  0.00           H  \\nATOM   4329  HA  ARG B 276      33.476  13.573  26.481  1.00  0.00           H  \\nATOM   4330 1HB  ARG B 276      35.952  15.299  26.215  1.00  0.00           H  \\nATOM   4331 2HB  ARG B 276      35.189  15.066  27.780  1.00  0.00           H  \\nATOM   4332 1HG  ARG B 276      35.449  12.595  27.607  1.00  0.00           H  \\nATOM   4333 2HG  ARG B 276      36.063  12.746  25.943  1.00  0.00           H  \\nATOM   4334 1HD  ARG B 276      37.941  14.130  26.794  1.00  0.00           H  \\nATOM   4335 2HD  ARG B 276      37.345  13.885  28.452  1.00  0.00           H  \\nATOM   4336  HE  ARG B 276      37.627  11.359  27.268  1.00  0.00           H  \\nATOM   4337 1HH1 ARG B 276      39.539  14.138  28.278  1.00  0.00           H  \\nATOM   4338 2HH1 ARG B 276      40.987  13.209  28.595  1.00  0.00           H  \\nATOM   4339 1HH2 ARG B 276      39.479  10.123  27.688  1.00  0.00           H  \\nATOM   4340 2HH2 ARG B 276      40.949  10.884  28.253  1.00  0.00           H  \\nATOM   4341  N   ALA B 277      33.238  16.747  25.649  1.00  0.00           N  \\nATOM   4342  CA  ALA B 277      32.407  17.945  25.923  1.00  0.00           C  \\nATOM   4343  C   ALA B 277      30.951  17.616  25.570  1.00  0.00           C  \\nATOM   4344  O   ALA B 277      30.041  18.048  26.289  1.00  0.00           O  \\nATOM   4345  CB  ALA B 277      32.902  19.084  25.135  1.00  0.00           C  \\nATOM   4346  H   ALA B 277      33.897  16.770  24.884  1.00  0.00           H  \\nATOM   4347  HA  ALA B 277      32.491  18.180  26.984  1.00  0.00           H  \\nATOM   4348 1HB  ALA B 277      32.289  19.962  25.339  1.00  0.00           H  \\nATOM   4349 2HB  ALA B 277      33.937  19.293  25.408  1.00  0.00           H  \\nATOM   4350 3HB  ALA B 277      32.848  18.843  24.075  1.00  0.00           H  \\nATOM   4351  N   VAL B 278      30.762  16.933  24.445  1.00  0.00           N  \\nATOM   4352  CA  VAL B 278      29.401  16.588  24.030  1.00  0.00           C  \\nATOM   4353  C   VAL B 278      28.756  15.686  25.116  1.00  0.00           C  \\nATOM   4354  O   VAL B 278      27.648  15.935  25.526  1.00  0.00           O  \\nATOM   4355  CB  VAL B 278      29.426  15.850  22.685  1.00  0.00           C  \\nATOM   4356  CG1 VAL B 278      28.027  15.251  22.410  1.00  0.00           C  \\nATOM   4357  CG2 VAL B 278      29.883  16.834  21.520  1.00  0.00           C  \\nATOM   4358  H   VAL B 278      31.540  16.646  23.868  1.00  0.00           H  \\nATOM   4359  HA  VAL B 278      28.831  17.510  23.913  1.00  0.00           H  \\nATOM   4360  HB  VAL B 278      30.129  15.019  22.750  1.00  0.00           H  \\nATOM   4361 1HG1 VAL B 278      28.038  14.725  21.455  1.00  0.00           H  \\nATOM   4362 2HG1 VAL B 278      27.767  14.553  23.206  1.00  0.00           H  \\nATOM   4363 3HG1 VAL B 278      27.289  16.052  22.373  1.00  0.00           H  \\nATOM   4364 1HG2 VAL B 278      29.896  16.295  20.573  1.00  0.00           H  \\nATOM   4365 2HG2 VAL B 278      29.185  17.668  21.453  1.00  0.00           H  \\nATOM   4366 3HG2 VAL B 278      30.882  17.213  21.735  1.00  0.00           H  \\nATOM   4367  N   ARG B 279      29.445  14.657  25.581  1.00  0.00           N  \\nATOM   4368  CA  ARG B 279      28.935  13.810  26.677  1.00  0.00           C  \\nATOM   4369  C   ARG B 279      28.597  14.617  27.917  1.00  0.00           C  \\nATOM   4370  O   ARG B 279      27.569  14.355  28.536  1.00  0.00           O  \\nATOM   4371  CB  ARG B 279      29.925  12.743  27.095  1.00  0.00           C  \\nATOM   4372  CG  ARG B 279      30.167  11.684  26.002  1.00  0.00           C  \\nATOM   4373  CD  ARG B 279      31.143  10.667  26.560  1.00  0.00           C  \\nATOM   4374  NE  ARG B 279      31.384   9.713  25.506  1.00  0.00           N  \\nATOM   4375  CZ  ARG B 279      32.575   9.267  25.196  1.00  0.00           C  \\nATOM   4376  NH1 ARG B 279      33.652   9.621  25.932  1.00  0.00           N  \\nATOM   4377  NH2 ARG B 279      32.765   8.511  24.167  1.00  0.00           N  \\nATOM   4378  H   ARG B 279      30.345  14.446  25.174  1.00  0.00           H  \\nATOM   4379  HA  ARG B 279      28.031  13.306  26.333  1.00  0.00           H  \\nATOM   4380 1HB  ARG B 279      30.877  13.208  27.344  1.00  0.00           H  \\nATOM   4381 2HB  ARG B 279      29.562  12.241  27.991  1.00  0.00           H  \\nATOM   4382 1HG  ARG B 279      29.223  11.204  25.745  1.00  0.00           H  \\nATOM   4383 2HG  ARG B 279      30.583  12.164  25.116  1.00  0.00           H  \\nATOM   4384 1HD  ARG B 279      32.065  11.169  26.850  1.00  0.00           H  \\nATOM   4385 2HD  ARG B 279      30.705  10.182  27.432  1.00  0.00           H  \\nATOM   4386  HE  ARG B 279      30.585   9.377  24.985  1.00  0.00           H  \\nATOM   4387 1HH1 ARG B 279      33.541  10.234  26.727  1.00  0.00           H  \\nATOM   4388 2HH1 ARG B 279      34.568   9.273  25.687  1.00  0.00           H  \\nATOM   4389 1HH2 ARG B 279      31.984   8.251  23.580  1.00  0.00           H  \\nATOM   4390 2HH2 ARG B 279      33.693   8.180  23.949  1.00  0.00           H  \\nATOM   4391  N   GLY B 280      29.492  15.546  28.278  1.00  0.00           N  \\nATOM   4392  CA  GLY B 280      29.312  16.344  29.500  1.00  0.00           C  \\nATOM   4393  C   GLY B 280      28.108  17.254  29.400  1.00  0.00           C  \\nATOM   4394  O   GLY B 280      27.488  17.590  30.461  1.00  0.00           O  \\nATOM   4395  H   GLY B 280      30.307  15.704  27.703  1.00  0.00           H  \\nATOM   4396 1HA  GLY B 280      29.194  15.678  30.355  1.00  0.00           H  \\nATOM   4397 2HA  GLY B 280      30.205  16.942  29.679  1.00  0.00           H  \\nATOM   4398  N   ARG B 281      27.695  17.634  28.213  1.00  0.00           N  \\nATOM   4399  CA  ARG B 281      26.551  18.496  28.032  1.00  0.00           C  \\nATOM   4400  C   ARG B 281      25.260  17.755  27.758  1.00  0.00           C  \\nATOM   4401  O   ARG B 281      24.151  18.302  27.922  1.00  0.00           O  \\nATOM   4402  CB  ARG B 281      26.804  19.621  26.987  1.00  0.00           C  \\nATOM   4403  CG  ARG B 281      28.030  20.410  27.375  1.00  0.00           C  \\nATOM   4404  CD  ARG B 281      27.736  21.281  28.624  1.00  0.00           C  \\nATOM   4405  NE  ARG B 281      28.191  22.597  28.284  1.00  0.00           N  \\nATOM   4406  CZ  ARG B 281      27.529  23.730  28.354  1.00  0.00           C  \\nATOM   4407  NH1 ARG B 281      26.283  23.803  28.840  1.00  0.00           N  \\nATOM   4408  NH2 ARG B 281      28.166  24.794  27.913  1.00  0.00           N  \\nATOM   4409  H   ARG B 281      28.199  17.309  27.401  1.00  0.00           H  \\nATOM   4410  HA  ARG B 281      26.326  18.977  28.984  1.00  0.00           H  \\nATOM   4411 1HB  ARG B 281      26.936  19.178  26.001  1.00  0.00           H  \\nATOM   4412 2HB  ARG B 281      25.933  20.275  26.938  1.00  0.00           H  \\nATOM   4413 1HG  ARG B 281      28.847  19.725  27.603  1.00  0.00           H  \\nATOM   4414 2HG  ARG B 281      28.321  21.060  26.549  1.00  0.00           H  \\nATOM   4415 1HD  ARG B 281      26.666  21.270  28.832  1.00  0.00           H  \\nATOM   4416 2HD  ARG B 281      28.276  20.881  29.481  1.00  0.00           H  \\nATOM   4417  HE  ARG B 281      29.139  22.694  27.944  1.00  0.00           H  \\nATOM   4418 1HH1 ARG B 281      25.819  22.968  29.169  1.00  0.00           H  \\nATOM   4419 2HH1 ARG B 281      25.807  24.693  28.877  1.00  0.00           H  \\nATOM   4420 1HH2 ARG B 281      29.105  24.706  27.550  1.00  0.00           H  \\nATOM   4421 2HH2 ARG B 281      27.717  25.697  27.937  1.00  0.00           H  \\nATOM   4422  N   LEU B 282      25.405  16.499  27.369  1.00  0.00           N  \\nATOM   4423  CA  LEU B 282      24.175  15.699  27.165  1.00  0.00           C  \\nATOM   4424  C   LEU B 282      23.624  15.244  28.455  1.00  0.00           C  \\nATOM   4425  O   LEU B 282      22.400  15.132  28.563  1.00  0.00           O  \\nATOM   4426  CB  LEU B 282      24.462  14.461  26.314  1.00  0.00           C  \\nATOM   4427  CG  LEU B 282      24.741  14.715  24.863  1.00  0.00           C  \\nATOM   4428  CD1 LEU B 282      25.299  13.515  24.208  1.00  0.00           C  \\nATOM   4429  CD2 LEU B 282      23.415  15.136  24.100  1.00  0.00           C  \\nATOM   4430  H   LEU B 282      26.304  16.069  27.208  1.00  0.00           H  \\nATOM   4431  HA  LEU B 282      23.445  16.315  26.641  1.00  0.00           H  \\nATOM   4432 1HB  LEU B 282      25.326  13.946  26.730  1.00  0.00           H  \\nATOM   4433 2HB  LEU B 282      23.603  13.792  26.373  1.00  0.00           H  \\nATOM   4434  HG  LEU B 282      25.471  15.519  24.768  1.00  0.00           H  \\nATOM   4435 1HD1 LEU B 282      25.491  13.730  23.157  1.00  0.00           H  \\nATOM   4436 2HD1 LEU B 282      26.232  13.235  24.696  1.00  0.00           H  \\nATOM   4437 3HD1 LEU B 282      24.587  12.694  24.286  1.00  0.00           H  \\nATOM   4438 1HD2 LEU B 282      23.642  15.316  23.049  1.00  0.00           H  \\nATOM   4439 2HD2 LEU B 282      22.679  14.336  24.181  1.00  0.00           H  \\nATOM   4440 3HD2 LEU B 282      23.012  16.046  24.545  1.00  0.00           H  \\nATOM   4441  N   THR B 283      24.477  14.930  29.432  1.00  0.00           N  \\nATOM   4442  CA  THR B 283      23.982  14.408  30.731  1.00  0.00           C  \\nATOM   4443  C   THR B 283      24.412  15.250  31.918  1.00  0.00           C  \\nATOM   4444  O   THR B 283      25.448  15.896  31.882  1.00  0.00           O  \\nATOM   4445  CB  THR B 283      24.433  12.952  31.071  1.00  0.00           C  \\nATOM   4446  OG1 THR B 283      25.857  12.847  31.117  1.00  0.00           O  \\nATOM   4447  CG2 THR B 283      23.889  11.939  30.162  1.00  0.00           C  \\nATOM   4448  H   THR B 283      25.471  15.046  29.295  1.00  0.00           H  \\nATOM   4449  HA  THR B 283      22.892  14.395  30.706  1.00  0.00           H  \\nATOM   4450  HB  THR B 283      24.108  12.697  32.079  1.00  0.00           H  \\nATOM   4451  HG1 THR B 283      26.247  13.704  30.927  1.00  0.00           H  \\nATOM   4452 1HG2 THR B 283      24.245  10.953  30.461  1.00  0.00           H  \\nATOM   4453 2HG2 THR B 283      22.800  11.960  30.204  1.00  0.00           H  \\nATOM   4454 3HG2 THR B 283      24.217  12.150  29.145  1.00  0.00           H  \\nATOM   4455  N   THR B 284      23.623  15.177  32.982  1.00  0.00           N  \\nATOM   4456  CA  THR B 284      23.907  15.848  34.240  1.00  0.00           C  \\nATOM   4457  C   THR B 284      25.108  15.222  34.967  1.00  0.00           C  \\nATOM   4458  O   THR B 284      25.754  15.878  35.793  1.00  0.00           O  \\nATOM   4459  CB  THR B 284      22.695  15.700  35.176  1.00  0.00           C  \\nATOM   4460  OG1 THR B 284      23.000  16.396  36.387  1.00  0.00           O  \\nATOM   4461  CG2 THR B 284      22.354  14.206  35.471  1.00  0.00           C  \\nATOM   4462  H   THR B 284      22.783  14.622  32.899  1.00  0.00           H  \\nATOM   4463  HA  THR B 284      24.078  16.905  34.037  1.00  0.00           H  \\nATOM   4464  HB  THR B 284      21.823  16.165  34.718  1.00  0.00           H  \\nATOM   4465  HG1 THR B 284      23.877  16.782  36.322  1.00  0.00           H  \\nATOM   4466 1HG2 THR B 284      21.492  14.154  36.135  1.00  0.00           H  \\nATOM   4467 2HG2 THR B 284      22.124  13.694  34.537  1.00  0.00           H  \\nATOM   4468 3HG2 THR B 284      23.208  13.725  35.947  1.00  0.00           H  \\nATOM   4469  N   GLY B 285      25.369  13.943  34.699  1.00  0.00           N  \\nATOM   4470  CA  GLY B 285      26.281  13.183  35.572  1.00  0.00           C  \\nATOM   4471  C   GLY B 285      25.725  12.794  36.952  1.00  0.00           C  \\nATOM   4472  O   GLY B 285      26.423  12.137  37.748  1.00  0.00           O  \\nATOM   4473  H   GLY B 285      24.951  13.482  33.904  1.00  0.00           H  \\nATOM   4474 1HA  GLY B 285      26.578  12.260  35.073  1.00  0.00           H  \\nATOM   4475 2HA  GLY B 285      27.188  13.762  35.742  1.00  0.00           H  \\nATOM   4476  N   ALA B 286      24.502  13.225  37.262  1.00  0.00           N  \\nATOM   4477  CA  ALA B 286      23.805  12.825  38.523  1.00  0.00           C  \\nATOM   4478  C   ALA B 286      23.482  11.336  38.483  1.00  0.00           C  \\nATOM   4479  O   ALA B 286      23.760  10.548  39.464  1.00  0.00           O  \\nATOM   4480  CB  ALA B 286      22.554  13.650  38.730  1.00  0.00           C  \\nATOM   4481  H   ALA B 286      24.034  13.845  36.617  1.00  0.00           H  \\nATOM   4482  HA  ALA B 286      24.484  13.005  39.357  1.00  0.00           H  \\nATOM   4483 1HB  ALA B 286      22.063  13.342  39.653  1.00  0.00           H  \\nATOM   4484 2HB  ALA B 286      22.821  14.705  38.796  1.00  0.00           H  \\nATOM   4485 3HB  ALA B 286      21.876  13.499  37.891  1.00  0.00           H  \\nATOM   4486  N   ASP B 287      23.003  10.907  37.286  1.00  0.00           N  \\nATOM   4487  CA  ASP B 287      22.310   9.622  37.168  1.00  0.00           C  \\nATOM   4488  C   ASP B 287      23.280   8.649  36.455  1.00  0.00           C  \\nATOM   4489  O   ASP B 287      23.534   8.698  35.230  1.00  0.00           O  \\nATOM   4490  CB  ASP B 287      20.985   9.781  36.462  1.00  0.00           C  \\nATOM   4491  CG  ASP B 287      20.206   8.438  36.303  1.00  0.00           C  \\nATOM   4492  OD1 ASP B 287      20.722   7.374  36.849  1.00  0.00           O  \\nATOM   4493  OD2 ASP B 287      19.119   8.419  35.690  1.00  0.00           O  \\nATOM   4494  H   ASP B 287      23.124  11.478  36.462  1.00  0.00           H  \\nATOM   4495  HA  ASP B 287      22.120   9.238  38.171  1.00  0.00           H  \\nATOM   4496 1HB  ASP B 287      20.359  10.480  37.017  1.00  0.00           H  \\nATOM   4497 2HB  ASP B 287      21.149  10.204  35.471  1.00  0.00           H  \\nATOM   4498  N   VAL B 288      23.914   7.807  37.257  1.00  0.00           N  \\nATOM   4499  CA  VAL B 288      24.914   6.906  36.754  1.00  0.00           C  \\nATOM   4500  C   VAL B 288      24.551   5.556  37.301  1.00  0.00           C  \\nATOM   4501  O   VAL B 288      24.234   5.449  38.513  1.00  0.00           O  \\nATOM   4502  CB  VAL B 288      26.342   7.346  37.178  1.00  0.00           C  \\nATOM   4503  CG1 VAL B 288      27.406   6.610  36.455  1.00  0.00           C  \\nATOM   4504  CG2 VAL B 288      26.515   8.836  36.936  1.00  0.00           C  \\nATOM   4505  H   VAL B 288      23.692   7.796  38.242  1.00  0.00           H  \\nATOM   4506  HA  VAL B 288      24.867   6.910  35.664  1.00  0.00           H  \\nATOM   4507  HB  VAL B 288      26.479   7.130  38.238  1.00  0.00           H  \\nATOM   4508 1HG1 VAL B 288      28.383   6.958  36.792  1.00  0.00           H  \\nATOM   4509 2HG1 VAL B 288      27.312   5.543  36.657  1.00  0.00           H  \\nATOM   4510 3HG1 VAL B 288      27.308   6.787  35.384  1.00  0.00           H  \\nATOM   4511 1HG2 VAL B 288      27.518   9.139  37.236  1.00  0.00           H  \\nATOM   4512 2HG2 VAL B 288      26.372   9.052  35.877  1.00  0.00           H  \\nATOM   4513 3HG2 VAL B 288      25.779   9.388  37.522  1.00  0.00           H  \\nATOM   4514  N   ARG B 289      24.593   4.552  36.429  1.00  0.00           N  \\nATOM   4515  CA  ARG B 289      24.277   3.165  36.805  1.00  0.00           C  \\nATOM   4516  C   ARG B 289      25.340   2.229  36.265  1.00  0.00           C  \\nATOM   4517  O   ARG B 289      25.658   2.249  35.072  1.00  0.00           O  \\nATOM   4518  CB  ARG B 289      22.913   2.803  36.252  1.00  0.00           C  \\nATOM   4519  CG  ARG B 289      21.969   3.894  36.674  1.00  0.00           C  \\nATOM   4520  CD  ARG B 289      20.559   3.533  36.674  1.00  0.00           C  \\nATOM   4521  NE  ARG B 289      19.794   4.726  36.466  1.00  0.00           N  \\nATOM   4522  CZ  ARG B 289      18.542   4.698  36.007  1.00  0.00           C  \\nATOM   4523  NH1 ARG B 289      17.952   3.544  35.735  1.00  0.00           N  \\nATOM   4524  NH2 ARG B 289      17.910   5.835  35.769  1.00  0.00           N  \\nATOM   4525  H   ARG B 289      24.852   4.756  35.474  1.00  0.00           H  \\nATOM   4526  HA  ARG B 289      24.255   3.098  37.893  1.00  0.00           H  \\nATOM   4527 1HB  ARG B 289      22.971   2.717  35.168  1.00  0.00           H  \\nATOM   4528 2HB  ARG B 289      22.608   1.832  36.642  1.00  0.00           H  \\nATOM   4529 1HG  ARG B 289      22.207   4.208  37.691  1.00  0.00           H  \\nATOM   4530 2HG  ARG B 289      22.070   4.744  35.998  1.00  0.00           H  \\nATOM   4531 1HD  ARG B 289      20.366   2.819  35.873  1.00  0.00           H  \\nATOM   4532 2HD  ARG B 289      20.299   3.083  37.631  1.00  0.00           H  \\nATOM   4533  HE  ARG B 289      20.235   5.611  36.680  1.00  0.00           H  \\nATOM   4534 1HH1 ARG B 289      18.450   2.676  35.875  1.00  0.00           H  \\nATOM   4535 2HH1 ARG B 289      17.004   3.531  35.388  1.00  0.00           H  \\nATOM   4536 1HH2 ARG B 289      18.376   6.717  35.935  1.00  0.00           H  \\nATOM   4537 2HH2 ARG B 289      16.963   5.823  35.422  1.00  0.00           H  \\nATOM   4538  N   HIS B 290      25.940   1.421  37.145  1.00  0.00           N  \\nATOM   4539  CA  HIS B 290      27.057   0.622  36.712  1.00  0.00           C  \\nATOM   4540  C   HIS B 290      28.173   1.458  36.153  1.00  0.00           C  \\nATOM   4541  O   HIS B 290      28.875   1.016  35.259  1.00  0.00           O  \\nATOM   4542  CB  HIS B 290      26.639  -0.581  35.861  1.00  0.00           C  \\nATOM   4543  CG  HIS B 290      25.643  -1.450  36.582  1.00  0.00           C  \\nATOM   4544  ND1 HIS B 290      26.053  -2.303  37.588  1.00  0.00           N  \\nATOM   4545  CD2 HIS B 290      24.284  -1.511  36.562  1.00  0.00           C  \\nATOM   4546  CE1 HIS B 290      24.995  -2.922  38.093  1.00  0.00           C  \\nATOM   4547  NE2 HIS B 290      23.905  -2.431  37.525  1.00  0.00           N  \\nATOM   4548  H   HIS B 290      25.631   1.356  38.104  1.00  0.00           H  \\nATOM   4549  HA  HIS B 290      27.581   0.227  37.582  1.00  0.00           H  \\nATOM   4550 1HB  HIS B 290      26.202  -0.231  34.925  1.00  0.00           H  \\nATOM   4551 2HB  HIS B 290      27.519  -1.172  35.610  1.00  0.00           H  \\nATOM   4552  HD2 HIS B 290      23.617  -0.934  35.920  1.00  0.00           H  \\nATOM   4553  HE1 HIS B 290      25.016  -3.705  38.850  1.00  0.00           H  \\nATOM   4554  HE2 HIS B 290      22.954  -2.684  37.754  1.00  0.00           H  \\nATOM   4555  N   GLU B 291      28.363   2.661  36.696  1.00  0.00           N  \\nATOM   4556  CA  GLU B 291      29.468   3.525  36.217  1.00  0.00           C  \\nATOM   4557  C   GLU B 291      29.120   4.075  34.854  1.00  0.00           C  \\nATOM   4558  O   GLU B 291      30.019   4.640  34.185  1.00  0.00           O  \\nATOM   4559  CB  GLU B 291      30.847   2.785  36.083  1.00  0.00           C  \\nATOM   4560  CG  GLU B 291      31.463   2.107  37.307  1.00  0.00           C  \\nATOM   4561  CD  GLU B 291      31.606   3.052  38.463  1.00  0.00           C  \\nATOM   4562  OE1 GLU B 291      31.355   4.261  38.260  1.00  0.00           O  \\nATOM   4563  OE2 GLU B 291      31.970   2.593  39.587  1.00  0.00           O  \\nATOM   4564  H   GLU B 291      27.761   2.996  37.435  1.00  0.00           H  \\nATOM   4565  HA  GLU B 291      29.611   4.334  36.934  1.00  0.00           H  \\nATOM   4566 1HB  GLU B 291      30.766   1.997  35.334  1.00  0.00           H  \\nATOM   4567 2HB  GLU B 291      31.605   3.488  35.738  1.00  0.00           H  \\nATOM   4568 1HG  GLU B 291      30.830   1.271  37.603  1.00  0.00           H  \\nATOM   4569 2HG  GLU B 291      32.440   1.710  37.036  1.00  0.00           H  \\nATOM   4570  N   ILE B 292      27.882   3.840  34.399  1.00  0.00           N  \\nATOM   4571  CA  ILE B 292      27.461   4.290  33.040  1.00  0.00           C  \\nATOM   4572  C   ILE B 292      26.424   5.405  33.181  1.00  0.00           C  \\nATOM   4573  O   ILE B 292      25.323   5.198  33.647  1.00  0.00           O  \\nATOM   4574  CB  ILE B 292      26.940   3.164  32.145  1.00  0.00           C  \\nATOM   4575  CG1 ILE B 292      27.990   2.056  31.916  1.00  0.00           C  \\nATOM   4576  CG2 ILE B 292      26.614   3.783  30.660  1.00  0.00           C  \\nATOM   4577  CD1 ILE B 292      27.335   0.718  31.693  1.00  0.00           C  \\nATOM   4578  H   ILE B 292      27.219   3.350  34.982  1.00  0.00           H  \\nATOM   4579  HA  ILE B 292      28.325   4.722  32.536  1.00  0.00           H  \\nATOM   4580  HB  ILE B 292      26.033   2.745  32.579  1.00  0.00           H  \\nATOM   4581 1HG1 ILE B 292      28.604   2.310  31.053  1.00  0.00           H  \\nATOM   4582 2HG1 ILE B 292      28.651   1.997  32.781  1.00  0.00           H  \\nATOM   4583 1HG2 ILE B 292      26.242   2.992  30.010  1.00  0.00           H  \\nATOM   4584 2HG2 ILE B 292      25.860   4.564  30.749  1.00  0.00           H  \\nATOM   4585 3HG2 ILE B 292      27.524   4.205  30.234  1.00  0.00           H  \\nATOM   4586 1HD1 ILE B 292      28.102  -0.041  31.535  1.00  0.00           H  \\nATOM   4587 2HD1 ILE B 292      26.738   0.455  32.567  1.00  0.00           H  \\nATOM   4588 3HD1 ILE B 292      26.691   0.770  30.816  1.00  0.00           H  \\nATOM   4589  N   PRO B 293      26.797   6.617  32.730  1.00  0.00           N  \\nATOM   4590  CA  PRO B 293      25.805   7.717  32.780  1.00  0.00           C  \\nATOM   4591  C   PRO B 293      24.559   7.488  31.907  1.00  0.00           C  \\nATOM   4592  O   PRO B 293      24.635   6.904  30.764  1.00  0.00           O  \\nATOM   4593  CB  PRO B 293      26.566   8.918  32.228  1.00  0.00           C  \\nATOM   4594  CG  PRO B 293      27.993   8.594  32.436  1.00  0.00           C  \\nATOM   4595  CD  PRO B 293      28.129   7.097  32.349  1.00  0.00           C  \\nATOM   4596  HA  PRO B 293      25.511   7.892  33.825  1.00  0.00           H  \\nATOM   4597 1HB  PRO B 293      26.319   9.064  31.166  1.00  0.00           H  \\nATOM   4598 2HB  PRO B 293      26.261   9.832  32.758  1.00  0.00           H  \\nATOM   4599 1HG  PRO B 293      28.611   9.094  31.676  1.00  0.00           H  \\nATOM   4600 2HG  PRO B 293      28.329   8.968  33.415  1.00  0.00           H  \\nATOM   4601 1HD  PRO B 293      28.384   6.811  31.318  1.00  0.00           H  \\nATOM   4602 2HD  PRO B 293      28.906   6.758  33.049  1.00  0.00           H  \\nATOM   4603  N   VAL B 294      23.440   7.944  32.467  1.00  0.00           N  \\nATOM   4604  CA  VAL B 294      22.137   7.825  31.814  1.00  0.00           C  \\nATOM   4605  C   VAL B 294      21.701   9.253  31.330  1.00  0.00           C  \\nATOM   4606  O   VAL B 294      21.924  10.270  32.001  1.00  0.00           O  \\nATOM   4607  CB  VAL B 294      21.106   7.261  32.779  1.00  0.00           C  \\nATOM   4608  CG1 VAL B 294      19.811   6.999  32.064  1.00  0.00           C  \\nATOM   4609  CG2 VAL B 294      21.674   5.981  33.465  1.00  0.00           C  \\nATOM   4610  H   VAL B 294      23.498   8.386  33.373  1.00  0.00           H  \\nATOM   4611  HA  VAL B 294      22.234   7.141  30.970  1.00  0.00           H  \\nATOM   4612  HB  VAL B 294      20.881   8.013  33.536  1.00  0.00           H  \\nATOM   4613 1HG1 VAL B 294      19.082   6.596  32.767  1.00  0.00           H  \\nATOM   4614 2HG1 VAL B 294      19.432   7.931  31.644  1.00  0.00           H  \\nATOM   4615 3HG1 VAL B 294      19.977   6.280  31.262  1.00  0.00           H  \\nATOM   4616 1HG2 VAL B 294      20.933   5.580  34.156  1.00  0.00           H  \\nATOM   4617 2HG2 VAL B 294      21.905   5.233  32.706  1.00  0.00           H  \\nATOM   4618 3HG2 VAL B 294      22.582   6.234  34.013  1.00  0.00           H  \\nATOM   4619  N   LEU B 295      21.160   9.297  30.098  1.00  0.00           N  \\nATOM   4620  CA  LEU B 295      20.657  10.543  29.539  1.00  0.00           C  \\nATOM   4621  C   LEU B 295      19.375  10.956  30.257  1.00  0.00           C  \\nATOM   4622  O   LEU B 295      18.713  10.149  30.929  1.00  0.00           O  \\nATOM   4623  CB  LEU B 295      20.329  10.308  28.061  1.00  0.00           C  \\nATOM   4624  CG  LEU B 295      21.606  10.023  27.275  1.00  0.00           C  \\nATOM   4625  CD1 LEU B 295      21.279   9.568  25.920  1.00  0.00           C  \\nATOM   4626  CD2 LEU B 295      22.501  11.264  27.147  1.00  0.00           C  \\nATOM   4627  H   LEU B 295      21.100   8.452  29.548  1.00  0.00           H  \\nATOM   4628  HA  LEU B 295      21.435  11.300  29.627  1.00  0.00           H  \\nATOM   4629 1HB  LEU B 295      19.641   9.468  27.987  1.00  0.00           H  \\nATOM   4630 2HB  LEU B 295      19.830  11.195  27.670  1.00  0.00           H  \\nATOM   4631  HG  LEU B 295      22.179   9.245  27.781  1.00  0.00           H  \\nATOM   4632 1HD1 LEU B 295      22.199   9.368  25.371  1.00  0.00           H  \\nATOM   4633 2HD1 LEU B 295      20.686   8.655  25.976  1.00  0.00           H  \\nATOM   4634 3HD1 LEU B 295      20.710  10.341  25.404  1.00  0.00           H  \\nATOM   4635 1HD2 LEU B 295      23.397  11.010  26.580  1.00  0.00           H  \\nATOM   4636 2HD2 LEU B 295      21.956  12.054  26.629  1.00  0.00           H  \\nATOM   4637 3HD2 LEU B 295      22.786  11.611  28.140  1.00  0.00           H  \\nATOM   4638  N   PRO B 296      18.982  12.218  30.127  1.00  0.00           N  \\nATOM   4639  CA  PRO B 296      17.677  12.711  30.651  1.00  0.00           C  \\nATOM   4640  C   PRO B 296      16.446  11.951  30.199  1.00  0.00           C  \\nATOM   4641  O   PRO B 296      16.353  11.578  29.069  1.00  0.00           O  \\nATOM   4642  CB  PRO B 296      17.620  14.147  30.106  1.00  0.00           C  \\nATOM   4643  CG  PRO B 296      19.118  14.622  29.982  1.00  0.00           C  \\nATOM   4644  CD  PRO B 296      19.743  13.281  29.423  1.00  0.00           C  \\nATOM   4645  HA  PRO B 296      17.703  12.704  31.751  1.00  0.00           H  \\nATOM   4646 1HB  PRO B 296      17.102  14.161  29.136  1.00  0.00           H  \\nATOM   4647 2HB  PRO B 296      17.041  14.786  30.789  1.00  0.00           H  \\nATOM   4648 1HG  PRO B 296      19.187  15.491  29.311  1.00  0.00           H  \\nATOM   4649 2HG  PRO B 296      19.496  14.947  30.963  1.00  0.00           H  \\nATOM   4650 1HD  PRO B 296      19.590  13.230  28.335  1.00  0.00           H  \\nATOM   4651 2HD  PRO B 296      20.815  13.246  29.667  1.00  0.00           H  \\nATOM   4652  N   ASN B 297      15.449  11.779  31.110  1.00  0.00           N  \\nATOM   4653  CA  ASN B 297      14.218  11.112  30.708  1.00  0.00           C  \\nATOM   4654  C   ASN B 297      13.381  11.997  29.769  1.00  0.00           C  \\nATOM   4655  O   ASN B 297      13.240  13.161  30.094  1.00  0.00           O  \\nATOM   4656  CB  ASN B 297      13.443  10.745  31.974  1.00  0.00           C  \\nATOM   4657  CG  ASN B 297      12.103  10.038  31.708  1.00  0.00           C  \\nATOM   4658  OD1 ASN B 297      11.729   9.651  30.580  1.00  0.00           O  \\nATOM   4659  ND2 ASN B 297      11.420   9.692  32.866  1.00  0.00           N  \\nATOM   4660  H   ASN B 297      15.544  12.105  32.061  1.00  0.00           H  \\nATOM   4661  HA  ASN B 297      14.479  10.206  30.158  1.00  0.00           H  \\nATOM   4662 1HB  ASN B 297      14.053  10.089  32.597  1.00  0.00           H  \\nATOM   4663 2HB  ASN B 297      13.239  11.648  32.550  1.00  0.00           H  \\nATOM   4664 1HD2 ASN B 297      10.536   9.227  32.805  1.00  0.00           H  \\nATOM   4665 2HD2 ASN B 297      11.811   9.907  33.760  1.00  0.00           H  \\nATOM   4666  N   ARG B 298      12.940  11.514  28.611  1.00  0.00           N  \\nATOM   4667  CA  ARG B 298      12.093  12.281  27.692  1.00  0.00           C  \\nATOM   4668  C   ARG B 298      10.827  12.756  28.417  1.00  0.00           C  \\nATOM   4669  O   ARG B 298      10.361  13.885  28.212  1.00  0.00           O  \\nATOM   4670  CB  ARG B 298      11.735  11.436  26.397  1.00  0.00           C  \\nATOM   4671  CG  ARG B 298      11.112  12.218  25.129  1.00  0.00           C  \\nATOM   4672  CD  ARG B 298      11.424  11.442  23.754  1.00  0.00           C  \\nATOM   4673  NE  ARG B 298      10.531  11.635  22.561  1.00  0.00           N  \\nATOM   4674  CZ  ARG B 298       9.204  11.436  22.527  1.00  0.00           C  \\nATOM   4675  NH1 ARG B 298       8.523  11.082  23.637  1.00  0.00           N  \\nATOM   4676  NH2 ARG B 298       8.524  11.596  21.389  1.00  0.00           N  \\nATOM   4677  H   ARG B 298      13.208  10.572  28.363  1.00  0.00           H  \\nATOM   4678  HA  ARG B 298      12.640  13.171  27.379  1.00  0.00           H  \\nATOM   4679 1HB  ARG B 298      12.630  10.937  26.030  1.00  0.00           H  \\nATOM   4680 2HB  ARG B 298      11.012  10.662  26.656  1.00  0.00           H  \\nATOM   4681 1HG  ARG B 298      10.031  12.297  25.245  1.00  0.00           H  \\nATOM   4682 2HG  ARG B 298      11.544  13.217  25.069  1.00  0.00           H  \\nATOM   4683 1HD  ARG B 298      12.411  11.727  23.391  1.00  0.00           H  \\nATOM   4684 2HD  ARG B 298      11.399  10.367  23.929  1.00  0.00           H  \\nATOM   4685  HE  ARG B 298      10.959  11.943  21.698  1.00  0.00           H  \\nATOM   4686 1HH1 ARG B 298       9.012  10.962  24.513  1.00  0.00           H  \\nATOM   4687 2HH1 ARG B 298       7.525  10.937  23.592  1.00  0.00           H  \\nATOM   4688 1HH2 ARG B 298       9.008  11.869  20.545  1.00  0.00           H  \\nATOM   4689 2HH2 ARG B 298       7.526  11.444  21.372  1.00  0.00           H  \\nATOM   4690  N   VAL B 299      10.268  11.874  29.269  1.00  0.00           N  \\nATOM   4691  CA  VAL B 299       8.977  12.210  29.917  1.00  0.00           C  \\nATOM   4692  C   VAL B 299       9.092  13.366  30.929  1.00  0.00           C  \\nATOM   4693  O   VAL B 299       9.792  13.340  31.933  1.00  0.00           O  \\nATOM   4694  CB  VAL B 299       8.333  10.950  30.555  1.00  0.00           C  \\nATOM   4695  CG1 VAL B 299       6.924  11.251  31.084  1.00  0.00           C  \\nATOM   4696  CG2 VAL B 299       8.273   9.863  29.536  1.00  0.00           C  \\nATOM   4697  H   VAL B 299      10.707  10.989  29.477  1.00  0.00           H  \\nATOM   4698  HA  VAL B 299       8.298  12.597  29.156  1.00  0.00           H  \\nATOM   4699  HB  VAL B 299       8.941  10.632  31.402  1.00  0.00           H  \\nATOM   4700 1HG1 VAL B 299       6.500  10.348  31.525  1.00  0.00           H  \\nATOM   4701 2HG1 VAL B 299       6.979  12.033  31.841  1.00  0.00           H  \\nATOM   4702 3HG1 VAL B 299       6.290  11.584  30.262  1.00  0.00           H  \\nATOM   4703 1HG2 VAL B 299       7.822   8.975  29.978  1.00  0.00           H  \\nATOM   4704 2HG2 VAL B 299       7.672  10.192  28.688  1.00  0.00           H  \\nATOM   4705 3HG2 VAL B 299       9.282   9.627  29.196  1.00  0.00           H  \\nATOM   4706  N   GLY B 300       8.421  14.461  30.599  1.00  0.00           N  \\nATOM   4707  CA  GLY B 300       8.455  15.614  31.467  1.00  0.00           C  \\nATOM   4708  C   GLY B 300       9.692  16.518  31.265  1.00  0.00           C  \\nATOM   4709  O   GLY B 300       9.880  17.473  32.030  1.00  0.00           O  \\nATOM   4710  H   GLY B 300       7.883  14.503  29.745  1.00  0.00           H  \\nATOM   4711 1HA  GLY B 300       7.562  16.218  31.305  1.00  0.00           H  \\nATOM   4712 2HA  GLY B 300       8.438  15.288  32.506  1.00  0.00           H  \\nATOM   4713  N   LEU B 301      10.569  16.199  30.297  1.00  0.00           N  \\nATOM   4714  CA  LEU B 301      11.752  17.024  30.063  1.00  0.00           C  \\nATOM   4715  C   LEU B 301      11.350  18.383  29.451  1.00  0.00           C  \\nATOM   4716  O   LEU B 301      10.676  18.439  28.412  1.00  0.00           O  \\nATOM   4717  CB  LEU B 301      12.754  16.290  29.161  1.00  0.00           C  \\nATOM   4718  CG  LEU B 301      14.086  17.011  29.020  1.00  0.00           C  \\nATOM   4719  CD1 LEU B 301      14.862  17.098  30.278  1.00  0.00           C  \\nATOM   4720  CD2 LEU B 301      14.943  16.281  27.932  1.00  0.00           C  \\nATOM   4721  H   LEU B 301      10.416  15.383  29.721  1.00  0.00           H  \\nATOM   4722  HA  LEU B 301      12.230  17.223  31.022  1.00  0.00           H  \\nATOM   4723 1HB  LEU B 301      12.933  15.299  29.575  1.00  0.00           H  \\nATOM   4724 2HB  LEU B 301      12.311  16.173  28.172  1.00  0.00           H  \\nATOM   4725  HG  LEU B 301      13.911  18.044  28.719  1.00  0.00           H  \\nATOM   4726 1HD1 LEU B 301      15.797  17.626  30.093  1.00  0.00           H  \\nATOM   4727 2HD1 LEU B 301      14.283  17.639  31.026  1.00  0.00           H  \\nATOM   4728 3HD1 LEU B 301      15.078  16.094  30.642  1.00  0.00           H  \\nATOM   4729 1HD2 LEU B 301      15.901  16.790  27.821  1.00  0.00           H  \\nATOM   4730 2HD2 LEU B 301      15.114  15.248  28.236  1.00  0.00           H  \\nATOM   4731 3HD2 LEU B 301      14.412  16.296  26.980  1.00  0.00           H  \\nATOM   4732  N   PRO B 302      11.747  19.485  30.120  1.00  0.00           N  \\nATOM   4733  CA  PRO B 302      11.386  20.814  29.606  1.00  0.00           C  \\nATOM   4734  C   PRO B 302      12.060  21.062  28.274  1.00  0.00           C  \\nATOM   4735  O   PRO B 302      13.194  20.654  28.044  1.00  0.00           O  \\nATOM   4736  CB  PRO B 302      11.992  21.760  30.637  1.00  0.00           C  \\nATOM   4737  CG  PRO B 302      12.005  20.996  31.877  1.00  0.00           C  \\nATOM   4738  CD  PRO B 302      12.184  19.542  31.509  1.00  0.00           C  \\nATOM   4739  HA  PRO B 302      10.290  20.908  29.583  1.00  0.00           H  \\nATOM   4740 1HB  PRO B 302      12.999  22.067  30.319  1.00  0.00           H  \\nATOM   4741 2HB  PRO B 302      11.385  22.675  30.711  1.00  0.00           H  \\nATOM   4742 1HG  PRO B 302      12.820  21.344  32.529  1.00  0.00           H  \\nATOM   4743 2HG  PRO B 302      11.067  21.153  32.430  1.00  0.00           H  \\nATOM   4744 1HD  PRO B 302      13.244  19.266  31.609  1.00  0.00           H  \\nATOM   4745 2HD  PRO B 302      11.557  18.918  32.162  1.00  0.00           H  \\nATOM   4746  N   ILE B 303      11.309  21.720  27.424  1.00  0.00           N  \\nATOM   4747  CA  ILE B 303      11.772  21.948  26.045  1.00  0.00           C  \\nATOM   4748  C   ILE B 303      13.135  22.720  25.993  1.00  0.00           C  \\nATOM   4749  O   ILE B 303      13.978  22.481  25.102  1.00  0.00           O  \\nATOM   4750  CB  ILE B 303      10.625  22.566  25.249  1.00  0.00           C  \\nATOM   4751  CG1 ILE B 303      11.012  22.722  23.790  1.00  0.00           C  \\nATOM   4752  CG2 ILE B 303      10.188  23.946  25.914  1.00  0.00           C  \\nATOM   4753  CD1 ILE B 303      11.357  21.370  22.986  1.00  0.00           C  \\nATOM   4754  H   ILE B 303      10.407  22.077  27.705  1.00  0.00           H  \\nATOM   4755  HA  ILE B 303      12.045  20.988  25.609  1.00  0.00           H  \\nATOM   4756  HB  ILE B 303       9.776  21.884  25.245  1.00  0.00           H  \\nATOM   4757 1HG1 ILE B 303      10.202  23.211  23.249  1.00  0.00           H  \\nATOM   4758 2HG1 ILE B 303      11.890  23.364  23.713  1.00  0.00           H  \\nATOM   4759 1HG2 ILE B 303       9.370  24.383  25.342  1.00  0.00           H  \\nATOM   4760 2HG2 ILE B 303       9.860  23.769  26.937  1.00  0.00           H  \\nATOM   4761 3HG2 ILE B 303      11.035  24.632  25.918  1.00  0.00           H  \\nATOM   4762 1HD1 ILE B 303      11.618  21.615  21.956  1.00  0.00           H  \\nATOM   4763 2HD1 ILE B 303      12.198  20.867  23.464  1.00  0.00           H  \\nATOM   4764 3HD1 ILE B 303      10.489  20.712  22.992  1.00  0.00           H  \\nATOM   4765  N   ASN B 304      13.415  23.540  26.996  1.00  0.00           N  \\nATOM   4766  CA  ASN B 304      14.618  24.312  26.967  1.00  0.00           C  \\nATOM   4767  C   ASN B 304      15.844  23.498  27.410  1.00  0.00           C  \\nATOM   4768  O   ASN B 304      16.972  23.977  27.341  1.00  0.00           O  \\nATOM   4769  CB  ASN B 304      14.491  25.634  27.707  1.00  0.00           C  \\nATOM   4770  CG  ASN B 304      14.020  25.479  29.116  1.00  0.00           C  \\nATOM   4771  OD1 ASN B 304      13.252  24.554  29.477  1.00  0.00           O  \\nATOM   4772  ND2 ASN B 304      14.407  26.442  29.929  1.00  0.00           N  \\nATOM   4773  H   ASN B 304      12.791  23.631  27.785  1.00  0.00           H  \\nATOM   4774  HA  ASN B 304      14.858  24.541  25.927  1.00  0.00           H  \\nATOM   4775 1HB  ASN B 304      15.458  26.139  27.720  1.00  0.00           H  \\nATOM   4776 2HB  ASN B 304      13.791  26.281  27.178  1.00  0.00           H  \\nATOM   4777 1HD2 ASN B 304      14.138  26.421  30.893  1.00  0.00           H  \\nATOM   4778 2HD2 ASN B 304      14.969  27.193  29.583  1.00  0.00           H  \\nATOM   4779  N   GLN B 305      15.613  22.259  27.870  1.00  0.00           N  \\nATOM   4780  CA  GLN B 305      16.677  21.304  28.197  1.00  0.00           C  \\nATOM   4781  C   GLN B 305      16.737  20.167  27.170  1.00  0.00           C  \\nATOM   4782  O   GLN B 305      17.486  19.184  27.459  1.00  0.00           O  \\nATOM   4783  CB  GLN B 305      16.408  20.650  29.553  1.00  0.00           C  \\nATOM   4784  CG  GLN B 305      16.480  21.614  30.710  1.00  0.00           C  \\nATOM   4785  CD  GLN B 305      15.856  21.071  31.978  1.00  0.00           C  \\nATOM   4786  OE1 GLN B 305      15.217  21.829  32.699  1.00  0.00           O  \\nATOM   4787  NE2 GLN B 305      16.047  19.772  32.271  1.00  0.00           N  \\nATOM   4788  H   GLN B 305      14.649  21.983  27.993  1.00  0.00           H  \\nATOM   4789  HA  GLN B 305      17.622  21.844  28.251  1.00  0.00           H  \\nATOM   4790 1HB  GLN B 305      15.418  20.194  29.547  1.00  0.00           H  \\nATOM   4791 2HB  GLN B 305      17.133  19.855  29.724  1.00  0.00           H  \\nATOM   4792 1HG  GLN B 305      17.526  21.836  30.920  1.00  0.00           H  \\nATOM   4793 2HG  GLN B 305      15.951  22.528  30.441  1.00  0.00           H  \\nATOM   4794 1HE2 GLN B 305      15.648  19.382  33.102  1.00  0.00           H  \\nATOM   4795 2HE2 GLN B 305      16.588  19.195  31.659  1.00  0.00           H  \\nATOM   4796  N   ARG B 306      16.016  20.274  26.030  1.00  0.00           N  \\nATOM   4797  CA  ARG B 306      15.814  19.089  25.182  1.00  0.00           C  \\nATOM   4798  C   ARG B 306      16.964  18.830  24.205  1.00  0.00           C  \\nATOM   4799  O   ARG B 306      17.109  17.717  23.734  1.00  0.00           O  \\nATOM   4800  CB  ARG B 306      14.414  19.099  24.512  1.00  0.00           C  \\nATOM   4801  CG  ARG B 306      14.047  18.010  23.497  1.00  0.00           C  \\nATOM   4802  CD  ARG B 306      13.987  16.696  24.239  1.00  0.00           C  \\nATOM   4803  NE  ARG B 306      13.643  15.703  23.266  1.00  0.00           N  \\nATOM   4804  CZ  ARG B 306      12.435  15.434  22.810  1.00  0.00           C  \\nATOM   4805  NH1 ARG B 306      12.315  14.526  21.855  1.00  0.00           N  \\nATOM   4806  NH2 ARG B 306      11.361  16.068  23.307  1.00  0.00           N  \\nATOM   4807  H   ARG B 306      15.612  21.155  25.748  1.00  0.00           H  \\nATOM   4808  HA  ARG B 306      15.884  18.199  25.808  1.00  0.00           H  \\nATOM   4809 1HB  ARG B 306      13.643  19.039  25.279  1.00  0.00           H  \\nATOM   4810 2HB  ARG B 306      14.272  20.039  23.979  1.00  0.00           H  \\nATOM   4811 1HG  ARG B 306      13.079  18.239  23.051  1.00  0.00           H  \\nATOM   4812 2HG  ARG B 306      14.806  17.970  22.715  1.00  0.00           H  \\nATOM   4813 1HD  ARG B 306      14.958  16.486  24.686  1.00  0.00           H  \\nATOM   4814 2HD  ARG B 306      13.232  16.755  25.022  1.00  0.00           H  \\nATOM   4815  HE  ARG B 306      14.392  15.143  22.880  1.00  0.00           H  \\nATOM   4816 1HH1 ARG B 306      13.136  14.060  21.495  1.00  0.00           H  \\nATOM   4817 2HH1 ARG B 306      11.403  14.299  21.486  1.00  0.00           H  \\nATOM   4818 1HH2 ARG B 306      11.474  16.758  24.037  1.00  0.00           H  \\nATOM   4819 2HH2 ARG B 306      10.441  15.856  22.952  1.00  0.00           H  \\nATOM   4820  N   PHE B 307      17.624  19.921  23.810  1.00  0.00           N  \\nATOM   4821  CA  PHE B 307      18.607  19.850  22.683  1.00  0.00           C  \\nATOM   4822  C   PHE B 307      19.971  20.473  23.061  1.00  0.00           C  \\nATOM   4823  O   PHE B 307      20.103  21.264  24.030  1.00  0.00           O  \\nATOM   4824  CB  PHE B 307      18.066  20.559  21.412  1.00  0.00           C  \\nATOM   4825  CG  PHE B 307      16.782  19.991  20.919  1.00  0.00           C  \\nATOM   4826  CD1 PHE B 307      16.764  18.744  20.358  1.00  0.00           C  \\nATOM   4827  CD2 PHE B 307      15.552  20.688  21.099  1.00  0.00           C  \\nATOM   4828  CE1 PHE B 307      15.549  18.159  19.835  1.00  0.00           C  \\nATOM   4829  CE2 PHE B 307      14.402  20.175  20.666  1.00  0.00           C  \\nATOM   4830  CZ  PHE B 307      14.342  18.902  19.972  1.00  0.00           C  \\nATOM   4831  H   PHE B 307      17.465  20.806  24.269  1.00  0.00           H  \\nATOM   4832  HA  PHE B 307      18.780  18.801  22.440  1.00  0.00           H  \\nATOM   4833 1HB  PHE B 307      17.915  21.617  21.622  1.00  0.00           H  \\nATOM   4834 2HB  PHE B 307      18.803  20.486  20.613  1.00  0.00           H  \\nATOM   4835  HD1 PHE B 307      17.690  18.172  20.303  1.00  0.00           H  \\nATOM   4836  HD2 PHE B 307      15.560  21.648  21.617  1.00  0.00           H  \\nATOM   4837  HE1 PHE B 307      15.578  17.181  19.356  1.00  0.00           H  \\nATOM   4838  HE2 PHE B 307      13.488  20.741  20.845  1.00  0.00           H  \\nATOM   4839  HZ  PHE B 307      13.412  18.513  19.561  1.00  0.00           H  \\nATOM   4840  N   ILE B 308      21.023  19.939  22.423  1.00  0.00           N  \\nATOM   4841  CA  ILE B 308      22.325  20.584  22.370  1.00  0.00           C  \\nATOM   4842  C   ILE B 308      22.673  20.779  20.902  1.00  0.00           C  \\nATOM   4843  O   ILE B 308      22.152  20.108  19.993  1.00  0.00           O  \\nATOM   4844  CB  ILE B 308      23.453  19.914  23.177  1.00  0.00           C  \\nATOM   4845  CG1 ILE B 308      23.852  18.573  22.606  1.00  0.00           C  \\nATOM   4846  CG2 ILE B 308      23.055  19.783  24.715  1.00  0.00           C  \\nATOM   4847  CD1 ILE B 308      25.175  18.091  23.215  1.00  0.00           C  \\nATOM   4848  H   ILE B 308      20.896  19.049  21.962  1.00  0.00           H  \\nATOM   4849  HA  ILE B 308      22.229  21.590  22.776  1.00  0.00           H  \\nATOM   4850  HB  ILE B 308      24.360  20.513  23.101  1.00  0.00           H  \\nATOM   4851 1HG1 ILE B 308      23.067  17.845  22.806  1.00  0.00           H  \\nATOM   4852 2HG1 ILE B 308      23.954  18.654  21.523  1.00  0.00           H  \\nATOM   4853 1HG2 ILE B 308      23.868  19.306  25.262  1.00  0.00           H  \\nATOM   4854 2HG2 ILE B 308      22.872  20.774  25.129  1.00  0.00           H  \\nATOM   4855 3HG2 ILE B 308      22.153  19.178  24.807  1.00  0.00           H  \\nATOM   4856 1HD1 ILE B 308      25.441  17.124  22.788  1.00  0.00           H  \\nATOM   4857 2HD1 ILE B 308      25.962  18.814  22.995  1.00  0.00           H  \\nATOM   4858 3HD1 ILE B 308      25.064  17.992  24.294  1.00  0.00           H  \\nATOM   4859  N   LEU B 309      23.511  21.767  20.666  1.00  0.00           N  \\nATOM   4860  CA  LEU B 309      23.921  22.025  19.294  1.00  0.00           C  \\nATOM   4861  C   LEU B 309      25.437  21.807  19.153  1.00  0.00           C  \\nATOM   4862  O   LEU B 309      26.230  22.039  20.071  1.00  0.00           O  \\nATOM   4863  CB  LEU B 309      23.647  23.465  18.979  1.00  0.00           C  \\nATOM   4864  CG  LEU B 309      22.293  24.047  19.268  1.00  0.00           C  \\nATOM   4865  CD1 LEU B 309      22.365  25.589  18.984  1.00  0.00           C  \\nATOM   4866  CD2 LEU B 309      21.262  23.325  18.485  1.00  0.00           C  \\nATOM   4867  H   LEU B 309      23.876  22.349  21.406  1.00  0.00           H  \\nATOM   4868  HA  LEU B 309      23.335  21.388  18.632  1.00  0.00           H  \\nATOM   4869 1HB  LEU B 309      24.354  24.082  19.532  1.00  0.00           H  \\nATOM   4870 2HB  LEU B 309      23.812  23.625  17.914  1.00  0.00           H  \\nATOM   4871  HG  LEU B 309      22.074  23.948  20.332  1.00  0.00           H  \\nATOM   4872 1HD1 LEU B 309      21.394  26.042  19.185  1.00  0.00           H  \\nATOM   4873 2HD1 LEU B 309      23.117  26.043  19.629  1.00  0.00           H  \\nATOM   4874 3HD1 LEU B 309      22.633  25.755  17.941  1.00  0.00           H  \\nATOM   4875 1HD2 LEU B 309      20.280  23.749  18.698  1.00  0.00           H  \\nATOM   4876 2HD2 LEU B 309      21.478  23.424  17.421  1.00  0.00           H  \\nATOM   4877 3HD2 LEU B 309      21.269  22.270  18.760  1.00  0.00           H  \\nATOM   4878  N   VAL B 310      25.810  21.252  18.011  1.00  0.00           N  \\nATOM   4879  CA  VAL B 310      27.192  20.951  17.698  1.00  0.00           C  \\nATOM   4880  C   VAL B 310      27.547  21.762  16.405  1.00  0.00           C  \\nATOM   4881  O   VAL B 310      27.097  21.416  15.275  1.00  0.00           O  \\nATOM   4882  CB  VAL B 310      27.351  19.450  17.431  1.00  0.00           C  \\nATOM   4883  CG1 VAL B 310      28.776  19.108  16.924  1.00  0.00           C  \\nATOM   4884  CG2 VAL B 310      27.141  18.666  18.731  1.00  0.00           C  \\nATOM   4885  H   VAL B 310      25.093  21.032  17.334  1.00  0.00           H  \\nATOM   4886  HA  VAL B 310      27.809  21.225  18.555  1.00  0.00           H  \\nATOM   4887  HB  VAL B 310      26.609  19.141  16.694  1.00  0.00           H  \\nATOM   4888 1HG1 VAL B 310      28.851  18.035  16.745  1.00  0.00           H  \\nATOM   4889 2HG1 VAL B 310      28.970  19.644  15.995  1.00  0.00           H  \\nATOM   4890 3HG1 VAL B 310      29.509  19.402  17.675  1.00  0.00           H  \\nATOM   4891 1HG2 VAL B 310      27.254  17.600  18.535  1.00  0.00           H  \\nATOM   4892 2HG2 VAL B 310      27.879  18.980  19.469  1.00  0.00           H  \\nATOM   4893 3HG2 VAL B 310      26.139  18.860  19.115  1.00  0.00           H  \\nATOM   4894  N   GLU B 311      28.453  22.764  16.542  1.00  0.00           N  \\nATOM   4895  CA  GLU B 311      28.866  23.573  15.419  1.00  0.00           C  \\nATOM   4896  C   GLU B 311      30.124  22.981  14.844  1.00  0.00           C  \\nATOM   4897  O   GLU B 311      31.173  22.966  15.497  1.00  0.00           O  \\nATOM   4898  CB  GLU B 311      29.048  25.035  15.837  1.00  0.00           C  \\nATOM   4899  CG  GLU B 311      29.491  25.887  14.618  1.00  0.00           C  \\nATOM   4900  CD  GLU B 311      30.180  27.188  15.024  1.00  0.00           C  \\nATOM   4901  OE1 GLU B 311      29.480  28.217  15.063  1.00  0.00           O  \\nATOM   4902  OE2 GLU B 311      31.413  27.182  15.350  1.00  0.00           O  \\nATOM   4903  H   GLU B 311      28.848  22.948  17.453  1.00  0.00           H  \\nATOM   4904  HA  GLU B 311      28.089  23.527  14.655  1.00  0.00           H  \\nATOM   4905 1HB  GLU B 311      28.110  25.417  16.241  1.00  0.00           H  \\nATOM   4906 2HB  GLU B 311      29.795  25.097  16.629  1.00  0.00           H  \\nATOM   4907 1HG  GLU B 311      30.177  25.300  14.008  1.00  0.00           H  \\nATOM   4908 2HG  GLU B 311      28.616  26.118  14.012  1.00  0.00           H  \\nATOM   4909  N   LEU B 312      30.000  22.464  13.630  1.00  0.00           N  \\nATOM   4910  CA  LEU B 312      31.154  21.870  12.931  1.00  0.00           C  \\nATOM   4911  C   LEU B 312      31.681  22.767  11.845  1.00  0.00           C  \\nATOM   4912  O   LEU B 312      30.914  23.264  10.982  1.00  0.00           O  \\nATOM   4913  CB  LEU B 312      30.620  20.587  12.184  1.00  0.00           C  \\nATOM   4914  CG  LEU B 312      29.984  19.569  13.137  1.00  0.00           C  \\nATOM   4915  CD1 LEU B 312      29.324  18.498  12.174  1.00  0.00           C  \\nATOM   4916  CD2 LEU B 312      31.073  18.894  14.113  1.00  0.00           C  \\nATOM   4917  H   LEU B 312      29.100  22.475  13.172  1.00  0.00           H  \\nATOM   4918  HA  LEU B 312      31.903  21.594  13.673  1.00  0.00           H  \\nATOM   4919 1HB  LEU B 312      29.882  20.897  11.445  1.00  0.00           H  \\nATOM   4920 2HB  LEU B 312      31.453  20.118  11.661  1.00  0.00           H  \\nATOM   4921  HG  LEU B 312      29.236  20.067  13.755  1.00  0.00           H  \\nATOM   4922 1HD1 LEU B 312      28.840  17.723  12.769  1.00  0.00           H  \\nATOM   4923 2HD1 LEU B 312      28.582  18.986  11.542  1.00  0.00           H  \\nATOM   4924 3HD1 LEU B 312      30.094  18.047  11.549  1.00  0.00           H  \\nATOM   4925 1HD2 LEU B 312      30.578  18.180  14.772  1.00  0.00           H  \\nATOM   4926 2HD2 LEU B 312      31.825  18.377  13.517  1.00  0.00           H  \\nATOM   4927 3HD2 LEU B 312      31.554  19.668  14.712  1.00  0.00           H  \\nATOM   4928  N   SER B 313      32.999  22.877  11.810  1.00  0.00           N  \\nATOM   4929  CA  SER B 313      33.673  23.697  10.761  1.00  0.00           C  \\nATOM   4930  C   SER B 313      34.741  22.831  10.119  1.00  0.00           C  \\nATOM   4931  O   SER B 313      35.149  21.808  10.687  1.00  0.00           O  \\nATOM   4932  CB  SER B 313      34.353  24.931  11.370  1.00  0.00           C  \\nATOM   4933  OG  SER B 313      33.388  25.773  11.998  1.00  0.00           O  \\nATOM   4934  H   SER B 313      33.563  22.402  12.500  1.00  0.00           H  \\nATOM   4935  HA  SER B 313      32.919  24.038  10.051  1.00  0.00           H  \\nATOM   4936 1HB  SER B 313      35.099  24.613  12.098  1.00  0.00           H  \\nATOM   4937 2HB  SER B 313      34.874  25.482  10.588  1.00  0.00           H  \\nATOM   4938  HG  SER B 313      32.540  25.341  11.875  1.00  0.00           H  \\nATOM   4939  N   ASN B 314      35.204  23.225   8.909  1.00  0.00           N  \\nATOM   4940  CA  ASN B 314      36.186  22.438   8.248  1.00  0.00           C  \\nATOM   4941  C   ASN B 314      37.261  23.288   7.598  1.00  0.00           C  \\nATOM   4942  O   ASN B 314      37.231  24.515   7.696  1.00  0.00           O  \\nATOM   4943  CB  ASN B 314      35.520  21.484   7.224  1.00  0.00           C  \\nATOM   4944  CG  ASN B 314      34.823  22.216   6.085  1.00  0.00           C  \\nATOM   4945  OD1 ASN B 314      34.841  23.472   5.961  1.00  0.00           O  \\nATOM   4946  ND2 ASN B 314      34.141  21.456   5.306  1.00  0.00           N  \\nATOM   4947  H   ASN B 314      34.867  24.067   8.464  1.00  0.00           H  \\nATOM   4948  HA  ASN B 314      36.708  21.838   8.994  1.00  0.00           H  \\nATOM   4949 1HB  ASN B 314      36.275  20.822   6.798  1.00  0.00           H  \\nATOM   4950 2HB  ASN B 314      34.786  20.859   7.733  1.00  0.00           H  \\nATOM   4951 1HD2 ASN B 314      33.648  21.849   4.529  1.00  0.00           H  \\nATOM   4952 2HD2 ASN B 314      34.103  20.472   5.476  1.00  0.00           H  \\nATOM   4953  N   HIS B 315      38.212  22.592   6.968  1.00  0.00           N  \\nATOM   4954  CA  HIS B 315      39.301  23.341   6.302  1.00  0.00           C  \\nATOM   4955  C   HIS B 315      38.791  24.278   5.184  1.00  0.00           C  \\nATOM   4956  O   HIS B 315      39.403  25.352   4.949  1.00  0.00           O  \\nATOM   4957  CB  HIS B 315      40.337  22.331   5.804  1.00  0.00           C  \\nATOM   4958  CG  HIS B 315      41.423  22.905   4.935  1.00  0.00           C  \\nATOM   4959  ND1 HIS B 315      42.665  23.223   5.422  1.00  0.00           N  \\nATOM   4960  CD2 HIS B 315      41.467  23.141   3.608  1.00  0.00           C  \\nATOM   4961  CE1 HIS B 315      43.413  23.695   4.438  1.00  0.00           C  \\nATOM   4962  NE2 HIS B 315      42.698  23.673   3.328  1.00  0.00           N  \\nATOM   4963  H   HIS B 315      38.214  21.583   6.930  1.00  0.00           H  \\nATOM   4964  HA  HIS B 315      39.770  24.015   7.018  1.00  0.00           H  \\nATOM   4965 1HB  HIS B 315      40.819  21.853   6.658  1.00  0.00           H  \\nATOM   4966 2HB  HIS B 315      39.838  21.550   5.231  1.00  0.00           H  \\nATOM   4967  HD2 HIS B 315      40.651  22.990   2.901  1.00  0.00           H  \\nATOM   4968  HE1 HIS B 315      44.442  24.042   4.529  1.00  0.00           H  \\nATOM   4969  HE2 HIS B 315      43.005  23.994   2.421  1.00  0.00           H  \\nATOM   4970  N   ALA B 316      37.684  23.906   4.518  1.00  0.00           N  \\nATOM   4971  CA  ALA B 316      37.057  24.707   3.464  1.00  0.00           C  \\nATOM   4972  C   ALA B 316      36.424  25.997   4.059  1.00  0.00           C  \\nATOM   4973  O   ALA B 316      35.863  26.781   3.338  1.00  0.00           O  \\nATOM   4974  CB  ALA B 316      36.023  23.900   2.737  1.00  0.00           C  \\nATOM   4975  H   ALA B 316      37.273  23.019   4.772  1.00  0.00           H  \\nATOM   4976  HA  ALA B 316      37.832  25.004   2.758  1.00  0.00           H  \\nATOM   4977 1HB  ALA B 316      35.567  24.510   1.957  1.00  0.00           H  \\nATOM   4978 2HB  ALA B 316      36.494  23.027   2.285  1.00  0.00           H  \\nATOM   4979 3HB  ALA B 316      35.256  23.576   3.438  1.00  0.00           H  \\nATOM   4980  N   GLU B 317      36.527  26.185   5.384  1.00  0.00           N  \\nATOM   4981  CA  GLU B 317      35.963  27.317   6.122  1.00  0.00           C  \\nATOM   4982  C   GLU B 317      34.402  27.431   6.041  1.00  0.00           C  \\nATOM   4983  O   GLU B 317      33.818  28.536   6.104  1.00  0.00           O  \\nATOM   4984  CB  GLU B 317      36.663  28.600   5.613  1.00  0.00           C  \\nATOM   4985  CG  GLU B 317      38.111  28.607   6.044  1.00  0.00           C  \\nATOM   4986  CD  GLU B 317      38.907  29.777   5.518  1.00  0.00           C  \\nATOM   4987  OE1 GLU B 317      38.403  30.601   4.707  1.00  0.00           O  \\nATOM   4988  OE2 GLU B 317      40.091  29.813   5.906  1.00  0.00           O  \\nATOM   4989  H   GLU B 317      37.039  25.475   5.888  1.00  0.00           H  \\nATOM   4990  HA  GLU B 317      36.168  27.175   7.184  1.00  0.00           H  \\nATOM   4991 1HB  GLU B 317      36.595  28.645   4.526  1.00  0.00           H  \\nATOM   4992 2HB  GLU B 317      36.150  29.476   6.009  1.00  0.00           H  \\nATOM   4993 1HG  GLU B 317      38.155  28.628   7.133  1.00  0.00           H  \\nATOM   4994 2HG  GLU B 317      38.584  27.686   5.706  1.00  0.00           H  \\nATOM   4995  N   LEU B 318      33.727  26.274   5.970  1.00  0.00           N  \\nATOM   4996  CA  LEU B 318      32.281  26.196   5.945  1.00  0.00           C  \\nATOM   4997  C   LEU B 318      31.901  25.669   7.339  1.00  0.00           C  \\nATOM   4998  O   LEU B 318      32.663  24.950   7.979  1.00  0.00           O  \\nATOM   4999  CB  LEU B 318      31.831  25.238   4.873  1.00  0.00           C  \\nATOM   5000  CG  LEU B 318      32.201  25.532   3.423  1.00  0.00           C  \\nATOM   5001  CD1 LEU B 318      31.525  24.560   2.568  1.00  0.00           C  \\nATOM   5002  CD2 LEU B 318      31.617  26.918   3.175  1.00  0.00           C  \\nATOM   5003  H   LEU B 318      34.266  25.421   5.933  1.00  0.00           H  \\nATOM   5004  HA  LEU B 318      31.885  27.186   5.720  1.00  0.00           H  \\nATOM   5005 1HB  LEU B 318      32.239  24.253   5.094  1.00  0.00           H  \\nATOM   5006 2HB  LEU B 318      30.743  25.173   4.899  1.00  0.00           H  \\nATOM   5007  HG  LEU B 318      33.286  25.515   3.314  1.00  0.00           H  \\nATOM   5008 1HD1 LEU B 318      31.778  24.754   1.526  1.00  0.00           H  \\nATOM   5009 2HD1 LEU B 318      31.846  23.554   2.837  1.00  0.00           H  \\nATOM   5010 3HD1 LEU B 318      30.447  24.646   2.702  1.00  0.00           H  \\nATOM   5011 1HD2 LEU B 318      31.831  27.225   2.151  1.00  0.00           H  \\nATOM   5012 2HD2 LEU B 318      30.538  26.891   3.328  1.00  0.00           H  \\nATOM   5013 3HD2 LEU B 318      32.064  27.631   3.868  1.00  0.00           H  \\nATOM   5014  N   SER B 319      30.746  26.072   7.829  1.00  0.00           N  \\nATOM   5015  CA  SER B 319      30.294  25.589   9.111  1.00  0.00           C  \\nATOM   5016  C   SER B 319      28.840  25.168   8.992  1.00  0.00           C  \\nATOM   5017  O   SER B 319      28.014  25.779   8.273  1.00  0.00           O  \\nATOM   5018  CB  SER B 319      30.376  26.751  10.074  1.00  0.00           C  \\nATOM   5019  OG  SER B 319      29.989  26.320  11.345  1.00  0.00           O  \\nATOM   5020  H   SER B 319      30.167  26.720   7.315  1.00  0.00           H  \\nATOM   5021  HA  SER B 319      30.957  24.784   9.431  1.00  0.00           H  \\nATOM   5022 1HB  SER B 319      31.395  27.136  10.093  1.00  0.00           H  \\nATOM   5023 2HB  SER B 319      29.729  27.556   9.728  1.00  0.00           H  \\nATOM   5024  HG  SER B 319      29.774  25.389  11.255  1.00  0.00           H  \\nATOM   5025  N   VAL B 320      28.481  24.090   9.694  1.00  0.00           N  \\nATOM   5026  CA  VAL B 320      27.060  23.718   9.886  1.00  0.00           C  \\nATOM   5027  C   VAL B 320      26.787  23.442  11.359  1.00  0.00           C  \\nATOM   5028  O   VAL B 320      27.722  23.063  12.072  1.00  0.00           O  \\nATOM   5029  CB  VAL B 320      26.599  22.464   9.082  1.00  0.00           C  \\nATOM   5030  CG1 VAL B 320      26.766  22.782   7.549  1.00  0.00           C  \\nATOM   5031  CG2 VAL B 320      27.377  21.157   9.498  1.00  0.00           C  \\nATOM   5032  H   VAL B 320      29.201  23.513  10.105  1.00  0.00           H  \\nATOM   5033  HA  VAL B 320      26.437  24.548   9.551  1.00  0.00           H  \\nATOM   5034  HB  VAL B 320      25.537  22.299   9.262  1.00  0.00           H  \\nATOM   5035 1HG1 VAL B 320      26.450  21.919   6.962  1.00  0.00           H  \\nATOM   5036 2HG1 VAL B 320      26.152  23.643   7.287  1.00  0.00           H  \\nATOM   5037 3HG1 VAL B 320      27.812  23.002   7.335  1.00  0.00           H  \\nATOM   5038 1HG2 VAL B 320      27.016  20.315   8.908  1.00  0.00           H  \\nATOM   5039 2HG2 VAL B 320      28.443  21.296   9.318  1.00  0.00           H  \\nATOM   5040 3HG2 VAL B 320      27.210  20.956  10.556  1.00  0.00           H  \\nATOM   5041  N   THR B 321      25.548  23.594  11.846  1.00  0.00           N  \\nATOM   5042  CA  THR B 321      25.328  23.293  13.263  1.00  0.00           C  \\nATOM   5043  C   THR B 321      24.308  22.180  13.371  1.00  0.00           C  \\nATOM   5044  O   THR B 321      23.160  22.291  12.880  1.00  0.00           O  \\nATOM   5045  CB  THR B 321      24.741  24.554  13.980  1.00  0.00           C  \\nATOM   5046  OG1 THR B 321      25.729  25.601  13.859  1.00  0.00           O  \\nATOM   5047  CG2 THR B 321      24.527  24.308  15.466  1.00  0.00           C  \\nATOM   5048  H   THR B 321      24.769  23.904  11.282  1.00  0.00           H  \\nATOM   5049  HA  THR B 321      26.287  23.039  13.716  1.00  0.00           H  \\nATOM   5050  HB  THR B 321      23.783  24.815  13.530  1.00  0.00           H  \\nATOM   5051  HG1 THR B 321      26.489  25.268  13.376  1.00  0.00           H  \\nATOM   5052 1HG2 THR B 321      24.118  25.206  15.928  1.00  0.00           H  \\nATOM   5053 2HG2 THR B 321      23.830  23.481  15.601  1.00  0.00           H  \\nATOM   5054 3HG2 THR B 321      25.479  24.061  15.934  1.00  0.00           H  \\nATOM   5055  N   LEU B 322      24.713  21.050  13.940  1.00  0.00           N  \\nATOM   5056  CA  LEU B 322      23.777  19.929  14.147  1.00  0.00           C  \\nATOM   5057  C   LEU B 322      23.050  20.138  15.430  1.00  0.00           C  \\nATOM   5058  O   LEU B 322      23.550  20.736  16.398  1.00  0.00           O  \\nATOM   5059  CB  LEU B 322      24.560  18.596  14.300  1.00  0.00           C  \\nATOM   5060  CG  LEU B 322      25.531  18.265  13.216  1.00  0.00           C  \\nATOM   5061  CD1 LEU B 322      26.177  16.925  13.618  1.00  0.00           C  \\nATOM   5062  CD2 LEU B 322      24.896  18.202  11.911  1.00  0.00           C  \\nATOM   5063  H   LEU B 322      25.673  20.950  14.237  1.00  0.00           H  \\nATOM   5064  HA  LEU B 322      23.127  19.856  13.275  1.00  0.00           H  \\nATOM   5065 1HB  LEU B 322      25.117  18.626  15.235  1.00  0.00           H  \\nATOM   5066 2HB  LEU B 322      23.844  17.776  14.354  1.00  0.00           H  \\nATOM   5067  HG  LEU B 322      26.308  19.029  13.178  1.00  0.00           H  \\nATOM   5068 1HD1 LEU B 322      26.902  16.628  12.859  1.00  0.00           H  \\nATOM   5069 2HD1 LEU B 322      26.683  17.038  14.577  1.00  0.00           H  \\nATOM   5070 3HD1 LEU B 322      25.406  16.160  13.702  1.00  0.00           H  \\nATOM   5071 1HD2 LEU B 322      25.643  17.959  11.154  1.00  0.00           H  \\nATOM   5072 2HD2 LEU B 322      24.124  17.433  11.919  1.00  0.00           H  \\nATOM   5073 3HD2 LEU B 322      24.445  19.167  11.679  1.00  0.00           H  \\nATOM   5074  N   ALA B 323      21.814  19.652  15.449  1.00  0.00           N  \\nATOM   5075  CA  ALA B 323      21.022  19.630  16.690  1.00  0.00           C  \\nATOM   5076  C   ALA B 323      20.844  18.189  17.170  1.00  0.00           C  \\nATOM   5077  O   ALA B 323      20.428  17.289  16.325  1.00  0.00           O  \\nATOM   5078  CB  ALA B 323      19.607  20.209  16.438  1.00  0.00           C  \\nATOM   5079  H   ALA B 323      21.406  19.290  14.599  1.00  0.00           H  \\nATOM   5080  HA  ALA B 323      21.531  20.251  17.428  1.00  0.00           H  \\nATOM   5081 1HB  ALA B 323      19.034  20.186  17.365  1.00  0.00           H  \\nATOM   5082 2HB  ALA B 323      19.692  21.238  16.090  1.00  0.00           H  \\nATOM   5083 3HB  ALA B 323      19.099  19.611  15.683  1.00  0.00           H  \\nATOM   5084  N   LEU B 324      21.281  17.932  18.412  1.00  0.00           N  \\nATOM   5085  CA  LEU B 324      21.119  16.584  18.959  1.00  0.00           C  \\nATOM   5086  C   LEU B 324      20.040  16.595  20.032  1.00  0.00           C  \\nATOM   5087  O   LEU B 324      19.994  17.511  20.877  1.00  0.00           O  \\nATOM   5088  CB  LEU B 324      22.465  16.118  19.599  1.00  0.00           C  \\nATOM   5089  CG  LEU B 324      23.551  15.611  18.577  1.00  0.00           C  \\nATOM   5090  CD1 LEU B 324      23.960  16.533  17.581  1.00  0.00           C  \\nATOM   5091  CD2 LEU B 324      24.760  15.394  19.586  1.00  0.00           C  \\nATOM   5092  H   LEU B 324      21.719  18.644  18.979  1.00  0.00           H  \\nATOM   5093  HA  LEU B 324      20.858  15.909  18.144  1.00  0.00           H  \\nATOM   5094 1HB  LEU B 324      22.891  16.950  20.157  1.00  0.00           H  \\nATOM   5095 2HB  LEU B 324      22.256  15.308  20.298  1.00  0.00           H  \\nATOM   5096  HG  LEU B 324      23.198  14.700  18.094  1.00  0.00           H  \\nATOM   5097 1HD1 LEU B 324      24.709  16.068  16.940  1.00  0.00           H  \\nATOM   5098 2HD1 LEU B 324      23.099  16.825  16.979  1.00  0.00           H  \\nATOM   5099 3HD1 LEU B 324      24.387  17.415  18.056  1.00  0.00           H  \\nATOM   5100 1HD2 LEU B 324      25.630  15.031  19.039  1.00  0.00           H  \\nATOM   5101 2HD2 LEU B 324      25.004  16.340  20.070  1.00  0.00           H  \\nATOM   5102 3HD2 LEU B 324      24.476  14.662  20.342  1.00  0.00           H  \\nATOM   5103  N   ASP B 325      19.222  15.548  20.087  1.00  0.00           N  \\nATOM   5104  CA  ASP B 325      18.201  15.345  21.169  1.00  0.00           C  \\nATOM   5105  C   ASP B 325      18.975  14.782  22.356  1.00  0.00           C  \\nATOM   5106  O   ASP B 325      19.643  13.757  22.222  1.00  0.00           O  \\nATOM   5107  CB  ASP B 325      17.170  14.358  20.538  1.00  0.00           C  \\nATOM   5108  CG  ASP B 325      15.940  14.140  21.401  1.00  0.00           C  \\nATOM   5109  OD1 ASP B 325      16.141  14.266  22.627  1.00  0.00           O  \\nATOM   5110  OD2 ASP B 325      14.864  13.796  20.782  1.00  0.00           O  \\nATOM   5111  H   ASP B 325      19.304  14.859  19.353  1.00  0.00           H  \\nATOM   5112  HA  ASP B 325      17.746  16.308  21.400  1.00  0.00           H  \\nATOM   5113 1HB  ASP B 325      16.848  14.739  19.569  1.00  0.00           H  \\nATOM   5114 2HB  ASP B 325      17.647  13.392  20.368  1.00  0.00           H  \\nATOM   5115  N   VAL B 326      18.867  15.456  23.514  1.00  0.00           N  \\nATOM   5116  CA  VAL B 326      19.590  14.994  24.719  1.00  0.00           C  \\nATOM   5117  C   VAL B 326      19.026  13.647  25.188  1.00  0.00           C  \\nATOM   5118  O   VAL B 326      19.773  12.928  25.918  1.00  0.00           O  \\nATOM   5119  CB  VAL B 326      19.613  15.973  25.877  1.00  0.00           C  \\nATOM   5120  CG1 VAL B 326      20.149  17.325  25.424  1.00  0.00           C  \\nATOM   5121  CG2 VAL B 326      18.209  16.101  26.487  1.00  0.00           C  \\nATOM   5122  H   VAL B 326      18.292  16.284  23.573  1.00  0.00           H  \\nATOM   5123  HA  VAL B 326      20.632  14.815  24.450  1.00  0.00           H  \\nATOM   5124  HB  VAL B 326      20.306  15.608  26.635  1.00  0.00           H  \\nATOM   5125 1HG1 VAL B 326      20.158  18.014  26.269  1.00  0.00           H  \\nATOM   5126 2HG1 VAL B 326      21.163  17.205  25.043  1.00  0.00           H  \\nATOM   5127 3HG1 VAL B 326      19.509  17.725  24.637  1.00  0.00           H  \\nATOM   5128 1HG2 VAL B 326      18.236  16.805  27.318  1.00  0.00           H  \\nATOM   5129 2HG2 VAL B 326      17.514  16.461  25.728  1.00  0.00           H  \\nATOM   5130 3HG2 VAL B 326      17.879  15.127  26.848  1.00  0.00           H  \\nATOM   5131  N   THR B 327      17.847  13.241  24.724  1.00  0.00           N  \\nATOM   5132  CA  THR B 327      17.313  12.003  25.298  1.00  0.00           C  \\nATOM   5133  C   THR B 327      17.933  10.764  24.630  1.00  0.00           C  \\nATOM   5134  O   THR B 327      17.776   9.663  25.155  1.00  0.00           O  \\nATOM   5135  CB  THR B 327      15.770  11.944  25.168  1.00  0.00           C  \\nATOM   5136  OG1 THR B 327      15.383  11.915  23.828  1.00  0.00           O  \\nATOM   5137  CG2 THR B 327      15.122  13.117  25.848  1.00  0.00           C  \\nATOM   5138  H   THR B 327      17.313  13.726  24.017  1.00  0.00           H  \\nATOM   5139  HA  THR B 327      17.568  11.975  26.358  1.00  0.00           H  \\nATOM   5140  HB  THR B 327      15.401  11.025  25.624  1.00  0.00           H  \\nATOM   5141  HG1 THR B 327      16.164  11.938  23.269  1.00  0.00           H  \\nATOM   5142 1HG2 THR B 327      14.040  13.048  25.740  1.00  0.00           H  \\nATOM   5143 2HG2 THR B 327      15.383  13.114  26.906  1.00  0.00           H  \\nATOM   5144 3HG2 THR B 327      15.473  14.041  25.391  1.00  0.00           H  \\nATOM   5145  N   ASN B 328      18.467  10.916  23.424  1.00  0.00           N  \\nATOM   5146  CA  ASN B 328      19.082   9.745  22.768  1.00  0.00           C  \\nATOM   5147  C   ASN B 328      20.424  10.062  22.124  1.00  0.00           C  \\nATOM   5148  O   ASN B 328      20.979   9.165  21.489  1.00  0.00           O  \\nATOM   5149  CB  ASN B 328      18.112   9.131  21.791  1.00  0.00           C  \\nATOM   5150  CG  ASN B 328      17.687  10.106  20.709  1.00  0.00           C  \\nATOM   5151  OD1 ASN B 328      18.265  11.125  20.503  1.00  0.00           O  \\nATOM   5152  ND2 ASN B 328      16.640   9.707  19.943  1.00  0.00           N  \\nATOM   5153  H   ASN B 328      18.465  11.804  22.944  1.00  0.00           H  \\nATOM   5154  HA  ASN B 328      19.325   9.007  23.534  1.00  0.00           H  \\nATOM   5155 1HB  ASN B 328      18.570   8.259  21.322  1.00  0.00           H  \\nATOM   5156 2HB  ASN B 328      17.226   8.787  22.325  1.00  0.00           H  \\nATOM   5157 1HD2 ASN B 328      16.303  10.298  19.209  1.00  0.00           H  \\nATOM   5158 2HD2 ASN B 328      16.205   8.823  20.112  1.00  0.00           H  \\nATOM   5159  N   ALA B 329      20.931  11.289  22.298  1.00  0.00           N  \\nATOM   5160  CA  ALA B 329      22.177  11.765  21.678  1.00  0.00           C  \\nATOM   5161  C   ALA B 329      22.161  11.847  20.156  1.00  0.00           C  \\nATOM   5162  O   ALA B 329      23.202  12.046  19.546  1.00  0.00           O  \\nATOM   5163  CB  ALA B 329      23.447  10.928  22.222  1.00  0.00           C  \\nATOM   5164  H   ALA B 329      20.410  11.913  22.897  1.00  0.00           H  \\nATOM   5165  HA  ALA B 329      22.310  12.813  21.946  1.00  0.00           H  \\nATOM   5166 1HB  ALA B 329      24.357  11.298  21.749  1.00  0.00           H  \\nATOM   5167 2HB  ALA B 329      23.526  11.047  23.303  1.00  0.00           H  \\nATOM   5168 3HB  ALA B 329      23.317   9.874  21.982  1.00  0.00           H  \\nATOM   5169  N   TYR B 330      21.004  11.663  19.516  1.00  0.00           N  \\nATOM   5170  CA  TYR B 330      20.928  11.477  18.033  1.00  0.00           C  \\nATOM   5171  C   TYR B 330      20.625  12.830  17.332  1.00  0.00           C  \\nATOM   5172  O   TYR B 330      19.984  13.710  17.928  1.00  0.00           O  \\nATOM   5173  CB  TYR B 330      19.805  10.452  17.716  1.00  0.00           C  \\nATOM   5174  CG  TYR B 330      20.046   9.686  16.401  1.00  0.00           C  \\nATOM   5175  CD1 TYR B 330      20.882   8.581  16.345  1.00  0.00           C  \\nATOM   5176  CD2 TYR B 330      19.389  10.071  15.207  1.00  0.00           C  \\nATOM   5177  CE1 TYR B 330      21.044   7.832  15.185  1.00  0.00           C  \\nATOM   5178  CE2 TYR B 330      19.567   9.371  14.061  1.00  0.00           C  \\nATOM   5179  CZ  TYR B 330      20.394   8.267  14.003  1.00  0.00           C  \\nATOM   5180  OH  TYR B 330      20.480   7.569  12.794  1.00  0.00           O  \\nATOM   5181  H   TYR B 330      20.152  11.650  20.058  1.00  0.00           H  \\nATOM   5182  HA  TYR B 330      21.885  11.088  17.684  1.00  0.00           H  \\nATOM   5183 1HB  TYR B 330      19.729   9.731  18.531  1.00  0.00           H  \\nATOM   5184 2HB  TYR B 330      18.849  10.970  17.648  1.00  0.00           H  \\nATOM   5185  HD1 TYR B 330      21.439   8.274  17.230  1.00  0.00           H  \\nATOM   5186  HD2 TYR B 330      18.732  10.940  15.203  1.00  0.00           H  \\nATOM   5187  HE1 TYR B 330      21.711   6.970  15.182  1.00  0.00           H  \\nATOM   5188  HE2 TYR B 330      19.051   9.676  13.150  1.00  0.00           H  \\nATOM   5189  HH  TYR B 330      19.914   7.989  12.142  1.00  0.00           H  \\nATOM   5190  N   VAL B 331      21.103  12.938  16.090  1.00  0.00           N  \\nATOM   5191  CA  VAL B 331      20.883  14.146  15.311  1.00  0.00           C  \\nATOM   5192  C   VAL B 331      19.398  14.205  14.882  1.00  0.00           C  \\nATOM   5193  O   VAL B 331      18.874  13.248  14.265  1.00  0.00           O  \\nATOM   5194  CB  VAL B 331      21.797  14.037  14.057  1.00  0.00           C  \\nATOM   5195  CG1 VAL B 331      21.511  15.243  13.117  1.00  0.00           C  \\nATOM   5196  CG2 VAL B 331      23.273  14.101  14.417  1.00  0.00           C  \\nATOM   5197  H   VAL B 331      21.624  12.176  15.680  1.00  0.00           H  \\nATOM   5198  HA  VAL B 331      21.168  15.006  15.918  1.00  0.00           H  \\nATOM   5199  HB  VAL B 331      21.605  13.086  13.560  1.00  0.00           H  \\nATOM   5200 1HG1 VAL B 331      22.147  15.175  12.234  1.00  0.00           H  \\nATOM   5201 2HG1 VAL B 331      20.465  15.226  12.812  1.00  0.00           H  \\nATOM   5202 3HG1 VAL B 331      21.722  16.173  13.644  1.00  0.00           H  \\nATOM   5203 1HG2 VAL B 331      23.872  14.020  13.511  1.00  0.00           H  \\nATOM   5204 2HG2 VAL B 331      23.486  15.049  14.910  1.00  0.00           H  \\nATOM   5205 3HG2 VAL B 331      23.520  13.279  15.089  1.00  0.00           H  \\nATOM   5206  N   VAL B 332      18.776  15.371  15.092  1.00  0.00           N  \\nATOM   5207  CA  VAL B 332      17.394  15.606  14.593  1.00  0.00           C  \\nATOM   5208  C   VAL B 332      17.359  16.614  13.434  1.00  0.00           C  \\nATOM   5209  O   VAL B 332      16.350  16.649  12.754  1.00  0.00           O  \\nATOM   5210  CB  VAL B 332      16.436  16.087  15.695  1.00  0.00           C  \\nATOM   5211  CG1 VAL B 332      16.455  15.051  16.873  1.00  0.00           C  \\nATOM   5212  CG2 VAL B 332      16.784  17.497  16.302  1.00  0.00           C  \\nATOM   5213  H   VAL B 332      19.248  16.107  15.597  1.00  0.00           H  \\nATOM   5214  HA  VAL B 332      17.002  14.665  14.205  1.00  0.00           H  \\nATOM   5215  HB  VAL B 332      15.429  16.156  15.284  1.00  0.00           H  \\nATOM   5216 1HG1 VAL B 332      15.778  15.383  17.661  1.00  0.00           H  \\nATOM   5217 2HG1 VAL B 332      16.134  14.077  16.506  1.00  0.00           H  \\nATOM   5218 3HG1 VAL B 332      17.466  14.974  17.273  1.00  0.00           H  \\nATOM   5219 1HG2 VAL B 332      16.054  17.753  17.070  1.00  0.00           H  \\nATOM   5220 2HG2 VAL B 332      17.780  17.467  16.743  1.00  0.00           H  \\nATOM   5221 3HG2 VAL B 332      16.759  18.249  15.513  1.00  0.00           H  \\nATOM   5222  N   GLY B 333      18.398  17.350  13.227  1.00  0.00           N  \\nATOM   5223  CA  GLY B 333      18.389  18.332  12.105  1.00  0.00           C  \\nATOM   5224  C   GLY B 333      19.647  19.128  12.120  1.00  0.00           C  \\nATOM   5225  O   GLY B 333      20.528  18.873  12.955  1.00  0.00           O  \\nATOM   5226  H   GLY B 333      19.220  17.276  13.809  1.00  0.00           H  \\nATOM   5227 1HA  GLY B 333      18.286  17.802  11.158  1.00  0.00           H  \\nATOM   5228 2HA  GLY B 333      17.524  18.987  12.202  1.00  0.00           H  \\nATOM   5229  N   TYR B 334      19.733  20.140  11.213  1.00  0.00           N  \\nATOM   5230  CA  TYR B 334      20.882  20.982  11.255  1.00  0.00           C  \\nATOM   5231  C   TYR B 334      20.580  22.320  10.613  1.00  0.00           C  \\nATOM   5232  O   TYR B 334      19.599  22.451   9.882  1.00  0.00           O  \\nATOM   5233  CB  TYR B 334      22.042  20.304  10.449  1.00  0.00           C  \\nATOM   5234  CG  TYR B 334      21.962  20.331   8.936  1.00  0.00           C  \\nATOM   5235  CD1 TYR B 334      21.218  19.438   8.256  1.00  0.00           C  \\nATOM   5236  CD2 TYR B 334      22.712  21.285   8.220  1.00  0.00           C  \\nATOM   5237  CE1 TYR B 334      21.231  19.464   6.798  1.00  0.00           C  \\nATOM   5238  CE2 TYR B 334      22.747  21.292   6.788  1.00  0.00           C  \\nATOM   5239  CZ  TYR B 334      21.990  20.373   6.122  1.00  0.00           C  \\nATOM   5240  OH  TYR B 334      21.957  20.294   4.770  1.00  0.00           O  \\nATOM   5241  H   TYR B 334      19.020  20.314  10.519  1.00  0.00           H  \\nATOM   5242  HA  TYR B 334      21.186  21.100  12.295  1.00  0.00           H  \\nATOM   5243 1HB  TYR B 334      22.990  20.776  10.709  1.00  0.00           H  \\nATOM   5244 2HB  TYR B 334      22.113  19.253  10.728  1.00  0.00           H  \\nATOM   5245  HD1 TYR B 334      20.615  18.707   8.796  1.00  0.00           H  \\nATOM   5246  HD2 TYR B 334      23.285  22.042   8.756  1.00  0.00           H  \\nATOM   5247  HE1 TYR B 334      20.634  18.747   6.235  1.00  0.00           H  \\nATOM   5248  HE2 TYR B 334      23.367  22.018   6.261  1.00  0.00           H  \\nATOM   5249  HH  TYR B 334      22.532  20.966   4.397  1.00  0.00           H  \\nATOM   5250  N   ARG B 335      21.477  23.238  10.852  1.00  0.00           N  \\nATOM   5251  CA  ARG B 335      21.384  24.640  10.262  1.00  0.00           C  \\nATOM   5252  C   ARG B 335      22.611  24.817   9.389  1.00  0.00           C  \\nATOM   5253  O   ARG B 335      23.717  24.466   9.810  1.00  0.00           O  \\nATOM   5254  CB  ARG B 335      21.361  25.666  11.382  1.00  0.00           C  \\nATOM   5255  CG  ARG B 335      21.342  27.104  10.852  1.00  0.00           C  \\nATOM   5256  CD  ARG B 335      21.306  28.114  11.918  1.00  0.00           C  \\nATOM   5257  NE  ARG B 335      22.587  28.357  12.578  1.00  0.00           N  \\nATOM   5258  CZ  ARG B 335      23.633  29.071  12.104  1.00  0.00           C  \\nATOM   5259  NH1 ARG B 335      23.648  29.628  10.895  1.00  0.00           N  \\nATOM   5260  NH2 ARG B 335      24.708  29.224  12.882  1.00  0.00           N  \\nATOM   5261  H   ARG B 335      22.258  23.006  11.449  1.00  0.00           H  \\nATOM   5262  HA  ARG B 335      20.457  24.715   9.692  1.00  0.00           H  \\nATOM   5263 1HB  ARG B 335      20.481  25.506  12.004  1.00  0.00           H  \\nATOM   5264 2HB  ARG B 335      22.238  25.533  12.015  1.00  0.00           H  \\nATOM   5265 1HG  ARG B 335      22.238  27.285  10.258  1.00  0.00           H  \\nATOM   5266 2HG  ARG B 335      20.458  27.251  10.230  1.00  0.00           H  \\nATOM   5267 1HD  ARG B 335      20.981  29.068  11.504  1.00  0.00           H  \\nATOM   5268 2HD  ARG B 335      20.609  27.799  12.694  1.00  0.00           H  \\nATOM   5269  HE  ARG B 335      22.727  27.954  13.495  1.00  0.00           H  \\nATOM   5270 1HH1 ARG B 335      22.851  29.527  10.282  1.00  0.00           H  \\nATOM   5271 2HH1 ARG B 335      24.456  30.152  10.590  1.00  0.00           H  \\nATOM   5272 1HH2 ARG B 335      24.728  28.810  13.804  1.00  0.00           H  \\nATOM   5273 2HH2 ARG B 335      25.500  29.753  12.549  1.00  0.00           H  \\nATOM   5274  N   ALA B 336      22.422  25.434   8.199  1.00  0.00           N  \\nATOM   5275  CA  ALA B 336      23.558  25.893   7.340  1.00  0.00           C  \\nATOM   5276  C   ALA B 336      23.142  27.308   6.904  1.00  0.00           C  \\nATOM   5277  O   ALA B 336      22.146  27.422   6.210  1.00  0.00           O  \\nATOM   5278  CB  ALA B 336      23.751  25.045   6.191  1.00  0.00           C  \\nATOM   5279  H   ALA B 336      21.474  25.587   7.885  1.00  0.00           H  \\nATOM   5280  HA  ALA B 336      24.469  25.862   7.938  1.00  0.00           H  \\nATOM   5281 1HB  ALA B 336      24.586  25.419   5.599  1.00  0.00           H  \\nATOM   5282 2HB  ALA B 336      23.966  24.029   6.521  1.00  0.00           H  \\nATOM   5283 3HB  ALA B 336      22.847  25.045   5.583  1.00  0.00           H  \\nATOM   5284  N   GLY B 337      23.879  28.279   7.473  1.00  0.00           N  \\nATOM   5285  CA  GLY B 337      23.591  29.740   7.294  1.00  0.00           C  \\nATOM   5286  C   GLY B 337      22.146  30.047   7.662  1.00  0.00           C  \\nATOM   5287  O   GLY B 337      21.761  29.896   8.791  1.00  0.00           O  \\nATOM   5288  H   GLY B 337      24.662  27.998   8.046  1.00  0.00           H  \\nATOM   5289 1HA  GLY B 337      24.268  30.324   7.918  1.00  0.00           H  \\nATOM   5290 2HA  GLY B 337      23.782  30.024   6.260  1.00  0.00           H  \\nATOM   5291  N   ASN B 338      21.331  30.483   6.681  1.00  0.00           N  \\nATOM   5292  CA  ASN B 338      19.970  30.980   6.974  1.00  0.00           C  \\nATOM   5293  C   ASN B 338      18.957  29.906   6.600  1.00  0.00           C  \\nATOM   5294  O   ASN B 338      17.778  30.183   6.408  1.00  0.00           O  \\nATOM   5295  CB  ASN B 338      19.683  32.333   6.229  1.00  0.00           C  \\nATOM   5296  CG  ASN B 338      20.723  33.426   6.538  1.00  0.00           C  \\nATOM   5297  OD1 ASN B 338      21.349  33.998   5.620  1.00  0.00           O  \\nATOM   5298  ND2 ASN B 338      20.910  33.726   7.814  1.00  0.00           N  \\nATOM   5299  H   ASN B 338      21.655  30.471   5.724  1.00  0.00           H  \\nATOM   5300  HA  ASN B 338      19.890  31.158   8.048  1.00  0.00           H  \\nATOM   5301 1HB  ASN B 338      19.671  32.160   5.152  1.00  0.00           H  \\nATOM   5302 2HB  ASN B 338      18.697  32.702   6.512  1.00  0.00           H  \\nATOM   5303 1HD2 ASN B 338      21.574  34.429   8.070  1.00  0.00           H  \\nATOM   5304 2HD2 ASN B 338      20.388  33.250   8.521  1.00  0.00           H  \\nATOM   5305  N   SER B 339      19.383  28.637   6.624  1.00  0.00           N  \\nATOM   5306  CA  SER B 339      18.468  27.568   6.368  1.00  0.00           C  \\nATOM   5307  C   SER B 339      18.577  26.448   7.418  1.00  0.00           C  \\nATOM   5308  O   SER B 339      19.715  26.133   7.838  1.00  0.00           O  \\nATOM   5309  CB  SER B 339      18.726  26.966   4.990  1.00  0.00           C  \\nATOM   5310  OG  SER B 339      18.487  27.976   3.975  1.00  0.00           O  \\nATOM   5311  H   SER B 339      20.349  28.421   6.822  1.00  0.00           H  \\nATOM   5312  HA  SER B 339      17.453  27.968   6.390  1.00  0.00           H  \\nATOM   5313 1HB  SER B 339      19.753  26.606   4.937  1.00  0.00           H  \\nATOM   5314 2HB  SER B 339      18.071  26.109   4.839  1.00  0.00           H  \\nATOM   5315  HG  SER B 339      18.225  28.770   4.448  1.00  0.00           H  \\nATOM   5316  N   ALA B 340      17.446  25.903   7.850  1.00  0.00           N  \\nATOM   5317  CA  ALA B 340      17.520  24.752   8.796  1.00  0.00           C  \\nATOM   5318  C   ALA B 340      16.753  23.610   8.225  1.00  0.00           C  \\nATOM   5319  O   ALA B 340      15.762  23.796   7.537  1.00  0.00           O  \\nATOM   5320  CB  ALA B 340      16.928  25.110  10.148  1.00  0.00           C  \\nATOM   5321  H   ALA B 340      16.541  26.243   7.559  1.00  0.00           H  \\nATOM   5322  HA  ALA B 340      18.570  24.496   8.936  1.00  0.00           H  \\nATOM   5323 1HB  ALA B 340      16.997  24.250  10.815  1.00  0.00           H  \\nATOM   5324 2HB  ALA B 340      17.480  25.947  10.577  1.00  0.00           H  \\nATOM   5325 3HB  ALA B 340      15.883  25.390  10.025  1.00  0.00           H  \\nATOM   5326  N   TYR B 341      17.195  22.391   8.566  1.00  0.00           N  \\nATOM   5327  CA  TYR B 341      16.733  21.152   7.991  1.00  0.00           C  \\nATOM   5328  C   TYR B 341      16.456  20.111   9.089  1.00  0.00           C  \\nATOM   5329  O   TYR B 341      17.296  19.943   9.950  1.00  0.00           O  \\nATOM   5330  CB  TYR B 341      17.746  20.532   6.982  1.00  0.00           C  \\nATOM   5331  CG  TYR B 341      17.966  21.474   5.855  1.00  0.00           C  \\nATOM   5332  CD1 TYR B 341      17.108  21.440   4.757  1.00  0.00           C  \\nATOM   5333  CD2 TYR B 341      18.994  22.403   5.908  1.00  0.00           C  \\nATOM   5334  CE1 TYR B 341      17.341  22.383   3.702  1.00  0.00           C  \\nATOM   5335  CE2 TYR B 341      19.245  23.256   4.915  1.00  0.00           C  \\nATOM   5336  CZ  TYR B 341      18.389  23.256   3.825  1.00  0.00           C  \\nATOM   5337  OH  TYR B 341      18.629  24.240   2.858  1.00  0.00           O  \\nATOM   5338  H   TYR B 341      17.906  22.366   9.283  1.00  0.00           H  \\nATOM   5339  HA  TYR B 341      15.809  21.347   7.446  1.00  0.00           H  \\nATOM   5340 1HB  TYR B 341      18.687  20.324   7.493  1.00  0.00           H  \\nATOM   5341 2HB  TYR B 341      17.357  19.583   6.613  1.00  0.00           H  \\nATOM   5342  HD1 TYR B 341      16.295  20.714   4.722  1.00  0.00           H  \\nATOM   5343  HD2 TYR B 341      19.636  22.455   6.787  1.00  0.00           H  \\nATOM   5344  HE1 TYR B 341      16.694  22.391   2.825  1.00  0.00           H  \\nATOM   5345  HE2 TYR B 341      20.103  23.926   4.975  1.00  0.00           H  \\nATOM   5346  HH  TYR B 341      19.381  24.774   3.126  1.00  0.00           H  \\nATOM   5347  N   PHE B 342      15.322  19.392   9.024  1.00  0.00           N  \\nATOM   5348  CA  PHE B 342      14.943  18.459  10.086  1.00  0.00           C  \\nATOM   5349  C   PHE B 342      14.526  17.154   9.466  1.00  0.00           C  \\nATOM   5350  O   PHE B 342      13.846  17.084   8.450  1.00  0.00           O  \\nATOM   5351  CB  PHE B 342      13.832  19.029  10.971  1.00  0.00           C  \\nATOM   5352  CG  PHE B 342      14.160  20.268  11.600  1.00  0.00           C  \\nATOM   5353  CD1 PHE B 342      14.822  20.323  12.807  1.00  0.00           C  \\nATOM   5354  CD2 PHE B 342      13.796  21.493  10.961  1.00  0.00           C  \\nATOM   5355  CE1 PHE B 342      15.136  21.558  13.333  1.00  0.00           C  \\nATOM   5356  CE2 PHE B 342      14.145  22.708  11.589  1.00  0.00           C  \\nATOM   5357  CZ  PHE B 342      14.739  22.728  12.736  1.00  0.00           C  \\nATOM   5358  H   PHE B 342      14.716  19.498   8.223  1.00  0.00           H  \\nATOM   5359  HA  PHE B 342      15.816  18.277  10.715  1.00  0.00           H  \\nATOM   5360 1HB  PHE B 342      12.934  19.184  10.374  1.00  0.00           H  \\nATOM   5361 2HB  PHE B 342      13.585  18.311  11.752  1.00  0.00           H  \\nATOM   5362  HD1 PHE B 342      15.081  19.394  13.315  1.00  0.00           H  \\nATOM   5363  HD2 PHE B 342      13.295  21.505   9.993  1.00  0.00           H  \\nATOM   5364  HE1 PHE B 342      15.718  21.626  14.252  1.00  0.00           H  \\nATOM   5365  HE2 PHE B 342      13.902  23.644  11.086  1.00  0.00           H  \\nATOM   5366  HZ  PHE B 342      14.922  23.682  13.229  1.00  0.00           H  \\nATOM   5367  N   PHE B 343      14.984  16.039  10.069  1.00  0.00           N  \\nATOM   5368  CA  PHE B 343      14.395  14.739   9.748  1.00  0.00           C  \\nATOM   5369  C   PHE B 343      12.920  14.655  10.033  1.00  0.00           C  \\nATOM   5370  O   PHE B 343      12.418  15.337  10.871  1.00  0.00           O  \\nATOM   5371  CB  PHE B 343      15.097  13.597  10.501  1.00  0.00           C  \\nATOM   5372  CG  PHE B 343      16.525  13.401  10.092  1.00  0.00           C  \\nATOM   5373  CD1 PHE B 343      16.924  13.175   8.748  1.00  0.00           C  \\nATOM   5374  CD2 PHE B 343      17.510  13.407  11.091  1.00  0.00           C  \\nATOM   5375  CE1 PHE B 343      18.264  12.972   8.421  1.00  0.00           C  \\nATOM   5376  CE2 PHE B 343      18.846  13.168  10.789  1.00  0.00           C  \\nATOM   5377  CZ  PHE B 343      19.265  12.984   9.467  1.00  0.00           C  \\nATOM   5378  H   PHE B 343      15.733  16.086  10.744  1.00  0.00           H  \\nATOM   5379  HA  PHE B 343      14.511  14.561   8.678  1.00  0.00           H  \\nATOM   5380 1HB  PHE B 343      15.071  13.797  11.571  1.00  0.00           H  \\nATOM   5381 2HB  PHE B 343      14.560  12.665  10.330  1.00  0.00           H  \\nATOM   5382  HD1 PHE B 343      16.167  13.162   7.963  1.00  0.00           H  \\nATOM   5383  HD2 PHE B 343      17.202  13.548  12.128  1.00  0.00           H  \\nATOM   5384  HE1 PHE B 343      18.552  12.806   7.383  1.00  0.00           H  \\nATOM   5385  HE2 PHE B 343      19.572  13.124  11.601  1.00  0.00           H  \\nATOM   5386  HZ  PHE B 343      20.317  12.850   9.218  1.00  0.00           H  \\nATOM   5387  N   HIS B 344      12.212  13.853   9.213  1.00  0.00           N  \\nATOM   5388  CA  HIS B 344      10.761  13.683   9.456  1.00  0.00           C  \\nATOM   5389  C   HIS B 344      10.489  13.114  10.856  1.00  0.00           C  \\nATOM   5390  O   HIS B 344      10.947  11.978  11.150  1.00  0.00           O  \\nATOM   5391  CB  HIS B 344      10.160  12.721   8.437  1.00  0.00           C  \\nATOM   5392  CG  HIS B 344       8.662  12.772   8.395  1.00  0.00           C  \\nATOM   5393  ND1 HIS B 344       7.869  11.956   9.177  1.00  0.00           N  \\nATOM   5394  CD2 HIS B 344       7.819  13.635   7.775  1.00  0.00           C  \\nATOM   5395  CE1 HIS B 344       6.600  12.285   9.005  1.00  0.00           C  \\nATOM   5396  NE2 HIS B 344       6.544  13.300   8.160  1.00  0.00           N  \\nATOM   5397  H   HIS B 344      12.642  13.366   8.440  1.00  0.00           H  \\nATOM   5398  HA  HIS B 344      10.260  14.645   9.352  1.00  0.00           H  \\nATOM   5399 1HB  HIS B 344      10.544  12.956   7.444  1.00  0.00           H  \\nATOM   5400 2HB  HIS B 344      10.467  11.702   8.674  1.00  0.00           H  \\nATOM   5401  HD2 HIS B 344       8.098  14.432   7.085  1.00  0.00           H  \\nATOM   5402  HE1 HIS B 344       5.746  11.802   9.479  1.00  0.00           H  \\nATOM   5403  HE2 HIS B 344       5.702  13.760   7.844  1.00  0.00           H  \\nATOM   5404  N   PRO B 345       9.821  13.890  11.715  1.00  0.00           N  \\nATOM   5405  CA  PRO B 345       9.583  13.403  13.101  1.00  0.00           C  \\nATOM   5406  C   PRO B 345       8.634  12.185  13.129  1.00  0.00           C  \\nATOM   5407  O   PRO B 345       7.667  12.097  12.298  1.00  0.00           O  \\nATOM   5408  CB  PRO B 345       8.950  14.607  13.799  1.00  0.00           C  \\nATOM   5409  CG  PRO B 345       8.492  15.490  12.692  1.00  0.00           C  \\nATOM   5410  CD  PRO B 345       9.332  15.277  11.518  1.00  0.00           C  \\nATOM   5411  HA  PRO B 345      10.547  13.146  13.565  1.00  0.00           H  \\nATOM   5412 1HB  PRO B 345       8.123  14.276  14.444  1.00  0.00           H  \\nATOM   5413 2HB  PRO B 345       9.689  15.096  14.450  1.00  0.00           H  \\nATOM   5414 1HG  PRO B 345       7.441  15.278  12.447  1.00  0.00           H  \\nATOM   5415 2HG  PRO B 345       8.539  16.544  13.003  1.00  0.00           H  \\nATOM   5416 1HD  PRO B 345       8.722  15.371  10.608  1.00  0.00           H  \\nATOM   5417 2HD  PRO B 345      10.151  16.012  11.514  1.00  0.00           H  \\nATOM   5418  N   ASP B 346       8.912  11.273  14.048  1.00  0.00           N  \\nATOM   5419  CA  ASP B 346       8.137  10.065  14.296  1.00  0.00           C  \\nATOM   5420  C   ASP B 346       6.741  10.246  14.885  1.00  0.00           C  \\nATOM   5421  O   ASP B 346       5.927   9.318  14.850  1.00  0.00           O  \\nATOM   5422  CB  ASP B 346       8.961   9.107  15.173  1.00  0.00           C  \\nATOM   5423  CG  ASP B 346      10.129   8.447  14.385  1.00  0.00           C  \\nATOM   5424  OD1 ASP B 346       9.895   7.420  13.645  1.00  0.00           O  \\nATOM   5425  OD2 ASP B 346      11.278   8.969  14.501  1.00  0.00           O  \\nATOM   5426  H   ASP B 346       9.733  11.455  14.608  1.00  0.00           H  \\nATOM   5427  HA  ASP B 346       7.927   9.586  13.339  1.00  0.00           H  \\nATOM   5428 1HB  ASP B 346       9.370   9.653  16.024  1.00  0.00           H  \\nATOM   5429 2HB  ASP B 346       8.311   8.325  15.567  1.00  0.00           H  \\nATOM   5430  N   ASN B 347       6.452  11.417  15.461  1.00  0.00           N  \\nATOM   5431  CA  ASN B 347       5.158  11.712  16.081  1.00  0.00           C  \\nATOM   5432  C   ASN B 347       5.042  13.221  16.307  1.00  0.00           C  \\nATOM   5433  O   ASN B 347       6.022  13.996  16.141  1.00  0.00           O  \\nATOM   5434  CB  ASN B 347       4.951  11.003  17.443  1.00  0.00           C  \\nATOM   5435  CG  ASN B 347       6.022  11.373  18.453  1.00  0.00           C  \\nATOM   5436  OD1 ASN B 347       6.053  12.513  18.959  1.00  0.00           O  \\nATOM   5437  ND2 ASN B 347       6.990  10.482  18.650  1.00  0.00           N  \\nATOM   5438  H   ASN B 347       7.171  12.126  15.462  1.00  0.00           H  \\nATOM   5439  HA  ASN B 347       4.367  11.366  15.414  1.00  0.00           H  \\nATOM   5440 1HB  ASN B 347       3.974  11.270  17.848  1.00  0.00           H  \\nATOM   5441 2HB  ASN B 347       4.960   9.923  17.296  1.00  0.00           H  \\nATOM   5442 1HD2 ASN B 347       7.722  10.675  19.305  1.00  0.00           H  \\nATOM   5443 2HD2 ASN B 347       6.986   9.620  18.144  1.00  0.00           H  \\nATOM   5444  N   GLN B 348       3.847  13.615  16.729  1.00  0.00           N  \\nATOM   5445  CA  GLN B 348       3.463  14.993  16.836  1.00  0.00           C  \\nATOM   5446  C   GLN B 348       4.275  15.717  17.907  1.00  0.00           C  \\nATOM   5447  O   GLN B 348       4.593  16.926  17.726  1.00  0.00           O  \\nATOM   5448  CB  GLN B 348       1.967  15.098  17.142  1.00  0.00           C  \\nATOM   5449  CG  GLN B 348       1.470  16.519  17.345  1.00  0.00           C  \\nATOM   5450  CD  GLN B 348       1.615  17.366  16.095  1.00  0.00           C  \\nATOM   5451  OE1 GLN B 348       1.137  16.996  15.019  1.00  0.00           O  \\nATOM   5452  NE2 GLN B 348       2.276  18.510  16.229  1.00  0.00           N  \\nATOM   5453  H   GLN B 348       3.183  12.898  16.985  1.00  0.00           H  \\nATOM   5454  HA  GLN B 348       3.617  15.471  15.869  1.00  0.00           H  \\nATOM   5455 1HB  GLN B 348       1.396  14.655  16.325  1.00  0.00           H  \\nATOM   5456 2HB  GLN B 348       1.739  14.530  18.045  1.00  0.00           H  \\nATOM   5457 1HG  GLN B 348       0.415  16.489  17.617  1.00  0.00           H  \\nATOM   5458 2HG  GLN B 348       2.049  16.985  18.142  1.00  0.00           H  \\nATOM   5459 1HE2 GLN B 348       2.403  19.111  15.439  1.00  0.00           H  \\nATOM   5460 2HE2 GLN B 348       2.647  18.772  17.121  1.00  0.00           H  \\nATOM   5461  N   GLU B 349       4.567  15.021  19.020  1.00  0.00           N  \\nATOM   5462  CA  GLU B 349       5.251  15.723  20.133  1.00  0.00           C  \\nATOM   5463  C   GLU B 349       6.711  16.060  19.672  1.00  0.00           C  \\nATOM   5464  O   GLU B 349       7.244  17.130  20.009  1.00  0.00           O  \\nATOM   5465  CB  GLU B 349       5.054  14.970  21.502  1.00  0.00           C  \\nATOM   5466  CG  GLU B 349       5.241  13.429  21.481  1.00  0.00           C  \\nATOM   5467  CD  GLU B 349       4.244  12.529  22.363  1.00  0.00           C  \\nATOM   5468  OE1 GLU B 349       4.526  12.230  23.574  1.00  0.00           O  \\nATOM   5469  OE2 GLU B 349       3.205  12.044  21.823  1.00  0.00           O  \\nATOM   5470  H   GLU B 349       4.338  14.043  19.122  1.00  0.00           H  \\nATOM   5471  HA  GLU B 349       4.820  16.720  20.230  1.00  0.00           H  \\nATOM   5472 1HB  GLU B 349       5.758  15.361  22.237  1.00  0.00           H  \\nATOM   5473 2HB  GLU B 349       4.049  15.158  21.879  1.00  0.00           H  \\nATOM   5474 1HG  GLU B 349       5.138  13.075  20.456  1.00  0.00           H  \\nATOM   5475 2HG  GLU B 349       6.249  13.191  21.818  1.00  0.00           H  \\nATOM   5476  N   ASP B 350       7.358  15.150  18.957  1.00  0.00           N  \\nATOM   5477  CA  ASP B 350       8.695  15.474  18.382  1.00  0.00           C  \\nATOM   5478  C   ASP B 350       8.611  16.574  17.312  1.00  0.00           C  \\nATOM   5479  O   ASP B 350       9.560  17.360  17.183  1.00  0.00           O  \\nATOM   5480  CB  ASP B 350       9.287  14.248  17.738  1.00  0.00           C  \\nATOM   5481  CG  ASP B 350       9.791  13.208  18.776  1.00  0.00           C  \\nATOM   5482  OD1 ASP B 350       9.866  13.537  19.976  1.00  0.00           O  \\nATOM   5483  OD2 ASP B 350      10.211  12.107  18.319  1.00  0.00           O  \\nATOM   5484  H   ASP B 350       6.960  14.236  18.793  1.00  0.00           H  \\nATOM   5485  HA  ASP B 350       9.350  15.800  19.191  1.00  0.00           H  \\nATOM   5486 1HB  ASP B 350       8.539  13.774  17.102  1.00  0.00           H  \\nATOM   5487 2HB  ASP B 350      10.122  14.539  17.100  1.00  0.00           H  \\nATOM   5488  N   ALA B 351       7.569  16.542  16.456  1.00  0.00           N  \\nATOM   5489  CA  ALA B 351       7.316  17.719  15.586  1.00  0.00           C  \\nATOM   5490  C   ALA B 351       7.191  19.006  16.275  1.00  0.00           C  \\nATOM   5491  O   ALA B 351       7.714  20.024  15.782  1.00  0.00           O  \\nATOM   5492  CB  ALA B 351       6.136  17.473  14.642  1.00  0.00           C  \\nATOM   5493  H   ALA B 351       6.957  15.741  16.390  1.00  0.00           H  \\nATOM   5494  HA  ALA B 351       8.208  17.893  14.984  1.00  0.00           H  \\nATOM   5495 1HB  ALA B 351       5.978  18.354  14.020  1.00  0.00           H  \\nATOM   5496 2HB  ALA B 351       6.351  16.614  14.007  1.00  0.00           H  \\nATOM   5497 3HB  ALA B 351       5.238  17.276  15.226  1.00  0.00           H  \\nATOM   5498  N   GLU B 352       6.499  19.037  17.405  1.00  0.00           N  \\nATOM   5499  CA  GLU B 352       6.381  20.248  18.133  1.00  0.00           C  \\nATOM   5500  C   GLU B 352       7.768  20.612  18.715  1.00  0.00           C  \\nATOM   5501  O   GLU B 352       8.248  21.791  18.620  1.00  0.00           O  \\nATOM   5502  CB  GLU B 352       5.311  20.111  19.208  1.00  0.00           C  \\nATOM   5503  CG  GLU B 352       4.970  21.500  19.779  1.00  0.00           C  \\nATOM   5504  CD  GLU B 352       5.908  22.051  20.879  1.00  0.00           C  \\nATOM   5505  OE1 GLU B 352       6.669  21.238  21.456  1.00  0.00           O  \\nATOM   5506  OE2 GLU B 352       5.868  23.283  21.183  1.00  0.00           O  \\nATOM   5507  H   GLU B 352       6.050  18.204  17.759  1.00  0.00           H  \\nATOM   5508  HA  GLU B 352       6.087  21.039  17.443  1.00  0.00           H  \\nATOM   5509 1HB  GLU B 352       4.422  19.649  18.779  1.00  0.00           H  \\nATOM   5510 2HB  GLU B 352       5.672  19.454  19.999  1.00  0.00           H  \\nATOM   5511 1HG  GLU B 352       4.976  22.227  18.967  1.00  0.00           H  \\nATOM   5512 2HG  GLU B 352       3.965  21.472  20.198  1.00  0.00           H  \\nATOM   5513  N   ALA B 353       8.533  19.593  19.162  1.00  0.00           N  \\nATOM   5514  CA  ALA B 353       9.748  20.003  19.847  1.00  0.00           C  \\nATOM   5515  C   ALA B 353      10.704  20.738  18.929  1.00  0.00           C  \\nATOM   5516  O   ALA B 353      11.347  21.684  19.334  1.00  0.00           O  \\nATOM   5517  CB  ALA B 353      10.452  18.774  20.444  1.00  0.00           C  \\nATOM   5518  H   ALA B 353       8.334  18.608  19.061  1.00  0.00           H  \\nATOM   5519  HA  ALA B 353       9.470  20.682  20.653  1.00  0.00           H  \\nATOM   5520 1HB  ALA B 353      11.362  19.089  20.956  1.00  0.00           H  \\nATOM   5521 2HB  ALA B 353       9.787  18.283  21.155  1.00  0.00           H  \\nATOM   5522 3HB  ALA B 353      10.707  18.078  19.646  1.00  0.00           H  \\nATOM   5523  N   ILE B 354      10.819  20.248  17.699  1.00  0.00           N  \\nATOM   5524  CA  ILE B 354      11.798  20.803  16.740  1.00  0.00           C  \\nATOM   5525  C   ILE B 354      11.469  22.221  16.314  1.00  0.00           C  \\nATOM   5526  O   ILE B 354      12.339  22.929  15.755  1.00  0.00           O  \\nATOM   5527  CB  ILE B 354      12.165  19.863  15.613  1.00  0.00           C  \\nATOM   5528  CG1 ILE B 354      10.954  19.593  14.710  1.00  0.00           C  \\nATOM   5529  CG2 ILE B 354      12.801  18.609  16.223  1.00  0.00           C  \\nATOM   5530  CD1 ILE B 354      11.297  18.916  13.403  1.00  0.00           C  \\nATOM   5531  H   ILE B 354      10.228  19.482  17.410  1.00  0.00           H  \\nATOM   5532  HA  ILE B 354      12.719  21.026  17.277  1.00  0.00           H  \\nATOM   5533  HB  ILE B 354      12.872  20.354  14.945  1.00  0.00           H  \\nATOM   5534 1HG1 ILE B 354      10.239  18.965  15.239  1.00  0.00           H  \\nATOM   5535 2HG1 ILE B 354      10.454  20.534  14.481  1.00  0.00           H  \\nATOM   5536 1HG2 ILE B 354      13.073  17.916  15.428  1.00  0.00           H  \\nATOM   5537 2HG2 ILE B 354      13.693  18.889  16.782  1.00  0.00           H  \\nATOM   5538 3HG2 ILE B 354      12.088  18.129  16.894  1.00  0.00           H  \\nATOM   5539 1HD1 ILE B 354      10.387  18.761  12.823  1.00  0.00           H  \\nATOM   5540 2HD1 ILE B 354      11.985  19.545  12.837  1.00  0.00           H  \\nATOM   5541 3HD1 ILE B 354      11.766  17.954  13.604  1.00  0.00           H  \\nATOM   5542  N   THR B 355      10.222  22.637  16.577  1.00  0.00           N  \\nATOM   5543  CA  THR B 355       9.894  24.031  16.277  1.00  0.00           C  \\nATOM   5544  C   THR B 355      10.671  24.997  17.139  1.00  0.00           C  \\nATOM   5545  O   THR B 355      10.699  26.227  16.835  1.00  0.00           O  \\nATOM   5546  CB  THR B 355       8.338  24.345  16.336  1.00  0.00           C  \\nATOM   5547  OG1 THR B 355       7.913  24.341  17.713  1.00  0.00           O  \\nATOM   5548  CG2 THR B 355       7.542  23.322  15.475  1.00  0.00           C  \\nATOM   5549  H   THR B 355       9.510  22.037  16.968  1.00  0.00           H  \\nATOM   5550  HA  THR B 355      10.228  24.255  15.264  1.00  0.00           H  \\nATOM   5551  HB  THR B 355       8.155  25.350  15.957  1.00  0.00           H  \\nATOM   5552  HG1 THR B 355       8.665  24.150  18.279  1.00  0.00           H  \\nATOM   5553 1HG2 THR B 355       6.478  23.553  15.527  1.00  0.00           H  \\nATOM   5554 2HG2 THR B 355       7.876  23.380  14.439  1.00  0.00           H  \\nATOM   5555 3HG2 THR B 355       7.713  22.316  15.855  1.00  0.00           H  \\nATOM   5556  N   HIS B 356      11.323  24.516  18.210  1.00  0.00           N  \\nATOM   5557  CA  HIS B 356      12.069  25.351  19.132  1.00  0.00           C  \\nATOM   5558  C   HIS B 356      13.543  25.489  18.767  1.00  0.00           C  \\nATOM   5559  O   HIS B 356      14.286  26.163  19.479  1.00  0.00           O  \\nATOM   5560  CB  HIS B 356      11.920  24.799  20.550  1.00  0.00           C  \\nATOM   5561  CG  HIS B 356      10.492  24.831  21.038  1.00  0.00           C  \\nATOM   5562  ND1 HIS B 356      10.043  25.791  21.912  1.00  0.00           N  \\nATOM   5563  CD2 HIS B 356       9.438  24.015  20.785  1.00  0.00           C  \\nATOM   5564  CE1 HIS B 356       8.764  25.574  22.172  1.00  0.00           C  \\nATOM   5565  NE2 HIS B 356       8.373  24.500  21.506  1.00  0.00           N  \\nATOM   5566  H   HIS B 356      11.284  23.520  18.370  1.00  0.00           H  \\nATOM   5567  HA  HIS B 356      11.671  26.365  19.108  1.00  0.00           H  \\nATOM   5568 1HB  HIS B 356      12.279  23.770  20.579  1.00  0.00           H  \\nATOM   5569 2HB  HIS B 356      12.538  25.381  21.234  1.00  0.00           H  \\nATOM   5570  HD2 HIS B 356       9.435  23.139  20.136  1.00  0.00           H  \\nATOM   5571  HE1 HIS B 356       8.135  26.178  22.825  1.00  0.00           H  \\nATOM   5572  HE2 HIS B 356       7.446  24.100  21.522  1.00  0.00           H  \\nATOM   5573  N   LEU B 357      13.957  24.779  17.712  1.00  0.00           N  \\nATOM   5574  CA  LEU B 357      15.316  24.810  17.196  1.00  0.00           C  \\nATOM   5575  C   LEU B 357      15.492  25.876  16.106  1.00  0.00           C  \\nATOM   5576  O   LEU B 357      14.622  25.943  15.191  1.00  0.00           O  \\nATOM   5577  CB  LEU B 357      15.645  23.402  16.594  1.00  0.00           C  \\nATOM   5578  CG  LEU B 357      16.033  22.471  17.729  1.00  0.00           C  \\nATOM   5579  CD1 LEU B 357      16.025  21.076  17.171  1.00  0.00           C  \\nATOM   5580  CD2 LEU B 357      17.434  22.761  18.295  1.00  0.00           C  \\nATOM   5581  H   LEU B 357      13.272  24.192  17.259  1.00  0.00           H  \\nATOM   5582  HA  LEU B 357      15.994  25.024  18.021  1.00  0.00           H  \\nATOM   5583 1HB  LEU B 357      14.767  23.037  16.064  1.00  0.00           H  \\nATOM   5584 2HB  LEU B 357      16.457  23.512  15.876  1.00  0.00           H  \\nATOM   5585  HG  LEU B 357      15.319  22.576  18.546  1.00  0.00           H  \\nATOM   5586 1HD1 LEU B 357      16.299  20.369  17.954  1.00  0.00           H  \\nATOM   5587 2HD1 LEU B 357      15.028  20.838  16.801  1.00  0.00           H  \\nATOM   5588 3HD1 LEU B 357      16.742  21.007  16.354  1.00  0.00           H  \\nATOM   5589 1HD2 LEU B 357      17.652  22.062  19.104  1.00  0.00           H  \\nATOM   5590 2HD2 LEU B 357      18.177  22.645  17.505  1.00  0.00           H  \\nATOM   5591 3HD2 LEU B 357      17.468  23.781  18.678  1.00  0.00           H  \\nATOM   5592  N   PHE B 358      16.613  26.598  16.184  1.00  0.00           N  \\nATOM   5593  CA  PHE B 358      17.032  27.534  15.100  1.00  0.00           C  \\nATOM   5594  C   PHE B 358      15.849  28.510  14.737  1.00  0.00           C  \\nATOM   5595  O   PHE B 358      15.511  28.672  13.549  1.00  0.00           O  \\nATOM   5596  CB  PHE B 358      17.435  26.760  13.839  1.00  0.00           C  \\nATOM   5597  CG  PHE B 358      18.505  25.654  14.080  1.00  0.00           C  \\nATOM   5598  CD1 PHE B 358      19.714  25.957  14.722  1.00  0.00           C  \\nATOM   5599  CD2 PHE B 358      18.241  24.384  13.703  1.00  0.00           C  \\nATOM   5600  CE1 PHE B 358      20.686  24.926  14.962  1.00  0.00           C  \\nATOM   5601  CE2 PHE B 358      19.221  23.352  13.970  1.00  0.00           C  \\nATOM   5602  CZ  PHE B 358      20.418  23.643  14.547  1.00  0.00           C  \\nATOM   5603  H   PHE B 358      17.194  26.505  17.005  1.00  0.00           H  \\nATOM   5604  HA  PHE B 358      17.896  28.102  15.448  1.00  0.00           H  \\nATOM   5605 1HB  PHE B 358      16.555  26.286  13.406  1.00  0.00           H  \\nATOM   5606 2HB  PHE B 358      17.831  27.453  13.098  1.00  0.00           H  \\nATOM   5607  HD1 PHE B 358      19.907  26.983  15.037  1.00  0.00           H  \\nATOM   5608  HD2 PHE B 358      17.289  24.124  13.239  1.00  0.00           H  \\nATOM   5609  HE1 PHE B 358      21.621  25.166  15.467  1.00  0.00           H  \\nATOM   5610  HE2 PHE B 358      18.984  22.324  13.698  1.00  0.00           H  \\nATOM   5611  HZ  PHE B 358      21.163  22.860  14.678  1.00  0.00           H  \\nATOM   5612  N   THR B 359      15.217  29.065  15.764  1.00  0.00           N  \\nATOM   5613  CA  THR B 359      13.902  29.778  15.551  1.00  0.00           C  \\nATOM   5614  C   THR B 359      14.019  31.099  14.779  1.00  0.00           C  \\nATOM   5615  O   THR B 359      13.013  31.603  14.248  1.00  0.00           O  \\nATOM   5616  CB  THR B 359      13.110  30.025  16.835  1.00  0.00           C  \\nATOM   5617  OG1 THR B 359      13.894  30.813  17.727  1.00  0.00           O  \\nATOM   5618  CG2 THR B 359      12.761  28.702  17.522  1.00  0.00           C  \\nATOM   5619  H   THR B 359      15.606  29.017  16.695  1.00  0.00           H  \\nATOM   5620  HA  THR B 359      13.275  29.162  14.906  1.00  0.00           H  \\nATOM   5621  HB  THR B 359      12.188  30.556  16.597  1.00  0.00           H  \\nATOM   5622  HG1 THR B 359      14.741  31.010  17.319  1.00  0.00           H  \\nATOM   5623 1HG2 THR B 359      12.197  28.903  18.433  1.00  0.00           H  \\nATOM   5624 2HG2 THR B 359      12.159  28.091  16.850  1.00  0.00           H  \\nATOM   5625 3HG2 THR B 359      13.678  28.170  17.773  1.00  0.00           H  \\nATOM   5626  N   ASP B 360      15.224  31.630  14.724  1.00  0.00           N  \\nATOM   5627  CA  ASP B 360      15.560  32.882  13.981  1.00  0.00           C  \\nATOM   5628  C   ASP B 360      15.960  32.622  12.529  1.00  0.00           C  \\nATOM   5629  O   ASP B 360      16.191  33.560  11.742  1.00  0.00           O  \\nATOM   5630  CB  ASP B 360      16.750  33.563  14.648  1.00  0.00           C  \\nATOM   5631  CG  ASP B 360      17.979  32.687  14.640  1.00  0.00           C  \\nATOM   5632  OD1 ASP B 360      17.949  31.564  15.208  1.00  0.00           O  \\nATOM   5633  OD2 ASP B 360      18.980  33.083  14.023  1.00  0.00           O  \\nATOM   5634  H   ASP B 360      15.955  31.143  15.224  1.00  0.00           H  \\nATOM   5635  HA  ASP B 360      14.698  33.549  14.019  1.00  0.00           H  \\nATOM   5636 1HB  ASP B 360      16.973  34.496  14.130  1.00  0.00           H  \\nATOM   5637 2HB  ASP B 360      16.496  33.813  15.678  1.00  0.00           H  \\nATOM   5638  N   VAL B 361      16.080  31.363  12.146  1.00  0.00           N  \\nATOM   5639  CA  VAL B 361      16.496  31.050  10.788  1.00  0.00           C  \\nATOM   5640  C   VAL B 361      15.433  31.461   9.760  1.00  0.00           C  \\nATOM   5641  O   VAL B 361      14.213  31.359  10.024  1.00  0.00           O  \\nATOM   5642  CB  VAL B 361      16.844  29.529  10.694  1.00  0.00           C  \\nATOM   5643  CG1 VAL B 361      15.671  28.703  10.151  1.00  0.00           C  \\nATOM   5644  CG2 VAL B 361      18.073  29.310   9.864  1.00  0.00           C  \\nATOM   5645  H   VAL B 361      15.886  30.607  12.787  1.00  0.00           H  \\nATOM   5646  HA  VAL B 361      17.385  31.637  10.554  1.00  0.00           H  \\nATOM   5647  HB  VAL B 361      17.021  29.143  11.698  1.00  0.00           H  \\nATOM   5648 1HG1 VAL B 361      15.957  27.652  10.103  1.00  0.00           H  \\nATOM   5649 2HG1 VAL B 361      14.811  28.814  10.811  1.00  0.00           H  \\nATOM   5650 3HG1 VAL B 361      15.411  29.053   9.152  1.00  0.00           H  \\nATOM   5651 1HG2 VAL B 361      18.294  28.244   9.815  1.00  0.00           H  \\nATOM   5652 2HG2 VAL B 361      17.905  29.693   8.857  1.00  0.00           H  \\nATOM   5653 3HG2 VAL B 361      18.915  29.834  10.316  1.00  0.00           H  \\nATOM   5654  N   GLN B 362      15.878  31.919   8.585  1.00  0.00           N  \\nATOM   5655  CA  GLN B 362      14.944  32.459   7.610  1.00  0.00           C  \\nATOM   5656  C   GLN B 362      14.060  31.414   6.981  1.00  0.00           C  \\nATOM   5657  O   GLN B 362      12.878  31.651   6.703  1.00  0.00           O  \\nATOM   5658  CB  GLN B 362      15.690  33.286   6.538  1.00  0.00           C  \\nATOM   5659  CG  GLN B 362      16.581  34.358   7.157  1.00  0.00           C  \\nATOM   5660  CD  GLN B 362      16.790  35.532   6.241  1.00  0.00           C  \\nATOM   5661  OE1 GLN B 362      17.255  35.381   5.111  1.00  0.00           O  \\nATOM   5662  NE2 GLN B 362      16.421  36.721   6.717  1.00  0.00           N  \\nATOM   5663  H   GLN B 362      16.863  31.895   8.362  1.00  0.00           H  \\nATOM   5664  HA  GLN B 362      14.244  33.114   8.128  1.00  0.00           H  \\nATOM   5665 1HB  GLN B 362      16.303  32.622   5.928  1.00  0.00           H  \\nATOM   5666 2HB  GLN B 362      14.966  33.763   5.877  1.00  0.00           H  \\nATOM   5667 1HG  GLN B 362      16.114  34.719   8.073  1.00  0.00           H  \\nATOM   5668 2HG  GLN B 362      17.555  33.922   7.381  1.00  0.00           H  \\nATOM   5669 1HE2 GLN B 362      16.534  37.542   6.156  1.00  0.00           H  \\nATOM   5670 2HE2 GLN B 362      16.031  36.793   7.635  1.00  0.00           H  \\nATOM   5671  N   ASN B 363      14.637  30.221   6.773  1.00  0.00           N  \\nATOM   5672  CA  ASN B 363      14.013  29.137   6.015  1.00  0.00           C  \\nATOM   5673  C   ASN B 363      14.098  27.814   6.778  1.00  0.00           C  \\nATOM   5674  O   ASN B 363      15.187  27.492   7.322  1.00  0.00           O  \\nATOM   5675  CB  ASN B 363      14.772  28.949   4.752  1.00  0.00           C  \\nATOM   5676  CG  ASN B 363      15.035  30.242   4.053  1.00  0.00           C  \\nATOM   5677  OD1 ASN B 363      16.190  30.673   3.864  1.00  0.00           O  \\nATOM   5678  ND2 ASN B 363      13.954  30.883   3.652  1.00  0.00           N  \\nATOM   5679  H   ASN B 363      15.555  30.079   7.169  1.00  0.00           H  \\nATOM   5680  HA  ASN B 363      12.985  29.422   5.786  1.00  0.00           H  \\nATOM   5681 1HB  ASN B 363      15.724  28.462   4.968  1.00  0.00           H  \\nATOM   5682 2HB  ASN B 363      14.212  28.293   4.085  1.00  0.00           H  \\nATOM   5683 1HD2 ASN B 363      14.040  31.758   3.175  1.00  0.00           H  \\nATOM   5684 2HD2 ASN B 363      13.050  30.494   3.825  1.00  0.00           H  \\nATOM   5685  N   ARG B 364      12.978  27.084   6.795  1.00  0.00           N  \\nATOM   5686  CA  ARG B 364      12.905  25.763   7.538  1.00  0.00           C  \\nATOM   5687  C   ARG B 364      12.314  24.758   6.628  1.00  0.00           C  \\nATOM   5688  O   ARG B 364      11.333  25.003   5.906  1.00  0.00           O  \\nATOM   5689  CB  ARG B 364      12.029  25.869   8.790  1.00  0.00           C  \\nATOM   5690  CG  ARG B 364      12.423  26.933   9.767  1.00  0.00           C  \\nATOM   5691  CD  ARG B 364      11.373  27.029  10.771  1.00  0.00           C  \\nATOM   5692  NE  ARG B 364      11.462  25.903  11.681  1.00  0.00           N  \\nATOM   5693  CZ  ARG B 364      12.287  25.889  12.721  1.00  0.00           C  \\nATOM   5694  NH1 ARG B 364      13.123  26.907  13.006  1.00  0.00           N  \\nATOM   5695  NH2 ARG B 364      12.270  24.839  13.489  1.00  0.00           N  \\nATOM   5696  H   ARG B 364      12.161  27.418   6.303  1.00  0.00           H  \\nATOM   5697  HA  ARG B 364      13.912  25.486   7.851  1.00  0.00           H  \\nATOM   5698 1HB  ARG B 364      10.999  26.066   8.497  1.00  0.00           H  \\nATOM   5699 2HB  ARG B 364      12.042  24.919   9.325  1.00  0.00           H  \\nATOM   5700 1HG  ARG B 364      13.370  26.664  10.236  1.00  0.00           H  \\nATOM   5701 2HG  ARG B 364      12.534  27.884   9.245  1.00  0.00           H  \\nATOM   5702 1HD  ARG B 364      11.489  27.956  11.332  1.00  0.00           H  \\nATOM   5703 2HD  ARG B 364      10.400  27.021  10.281  1.00  0.00           H  \\nATOM   5704  HE  ARG B 364      10.866  25.106  11.502  1.00  0.00           H  \\nATOM   5705 1HH1 ARG B 364      13.140  27.727  12.416  1.00  0.00           H  \\nATOM   5706 2HH1 ARG B 364      13.732  26.850  13.809  1.00  0.00           H  \\nATOM   5707 1HH2 ARG B 364      11.644  24.073  13.284  1.00  0.00           H  \\nATOM   5708 2HH2 ARG B 364      12.882  24.790  14.290  1.00  0.00           H  \\nATOM   5709  N   TYR B 365      12.861  23.560   6.686  1.00  0.00           N  \\nATOM   5710  CA  TYR B 365      12.403  22.502   5.908  1.00  0.00           C  \\nATOM   5711  C   TYR B 365      12.495  21.192   6.654  1.00  0.00           C  \\nATOM   5712  O   TYR B 365      13.546  20.920   7.239  1.00  0.00           O  \\nATOM   5713  CB  TYR B 365      13.302  22.404   4.684  1.00  0.00           C  \\nATOM   5714  CG  TYR B 365      12.837  21.383   3.705  1.00  0.00           C  \\nATOM   5715  CD1 TYR B 365      11.737  21.612   2.884  1.00  0.00           C  \\nATOM   5716  CD2 TYR B 365      13.489  20.155   3.603  1.00  0.00           C  \\nATOM   5717  CE1 TYR B 365      11.280  20.665   2.031  1.00  0.00           C  \\nATOM   5718  CE2 TYR B 365      13.067  19.199   2.687  1.00  0.00           C  \\nATOM   5719  CZ  TYR B 365      11.943  19.470   1.932  1.00  0.00           C  \\nATOM   5720  OH  TYR B 365      11.549  18.525   1.058  1.00  0.00           O  \\nATOM   5721  H   TYR B 365      13.636  23.408   7.315  1.00  0.00           H  \\nATOM   5722  HA  TYR B 365      11.379  22.716   5.601  1.00  0.00           H  \\nATOM   5723 1HB  TYR B 365      13.344  23.373   4.185  1.00  0.00           H  \\nATOM   5724 2HB  TYR B 365      14.315  22.152   4.996  1.00  0.00           H  \\nATOM   5725  HD1 TYR B 365      11.222  22.572   2.918  1.00  0.00           H  \\nATOM   5726  HD2 TYR B 365      14.340  19.935   4.247  1.00  0.00           H  \\nATOM   5727  HE1 TYR B 365      10.416  20.872   1.400  1.00  0.00           H  \\nATOM   5728  HE2 TYR B 365      13.619  18.265   2.580  1.00  0.00           H  \\nATOM   5729  HH  TYR B 365      12.148  17.776   1.106  1.00  0.00           H  \\nATOM   5730  N   THR B 366      11.440  20.376   6.577  1.00  0.00           N  \\nATOM   5731  CA  THR B 366      11.378  19.010   7.119  1.00  0.00           C  \\nATOM   5732  C   THR B 366      11.429  18.023   5.947  1.00  0.00           C  \\nATOM   5733  O   THR B 366      10.600  18.033   5.027  1.00  0.00           O  \\nATOM   5734  CB  THR B 366      10.119  18.809   8.012  1.00  0.00           C  \\nATOM   5735  OG1 THR B 366      10.121  19.741   9.102  1.00  0.00           O  \\nATOM   5736  CG2 THR B 366      10.055  17.336   8.534  1.00  0.00           C  \\nATOM   5737  H   THR B 366      10.633  20.754   6.102  1.00  0.00           H  \\nATOM   5738  HA  THR B 366      12.262  18.842   7.734  1.00  0.00           H  \\nATOM   5739  HB  THR B 366       9.223  19.024   7.431  1.00  0.00           H  \\nATOM   5740  HG1 THR B 366      10.904  20.294   9.048  1.00  0.00           H  \\nATOM   5741 1HG2 THR B 366       9.170  17.209   9.157  1.00  0.00           H  \\nATOM   5742 2HG2 THR B 366      10.005  16.652   7.687  1.00  0.00           H  \\nATOM   5743 3HG2 THR B 366      10.946  17.120   9.122  1.00  0.00           H  \\nATOM   5744  N   PHE B 367      12.402  17.126   5.988  1.00  0.00           N  \\nATOM   5745  CA  PHE B 367      12.580  16.138   4.935  1.00  0.00           C  \\nATOM   5746  C   PHE B 367      11.460  15.146   5.023  1.00  0.00           C  \\nATOM   5747  O   PHE B 367      10.813  14.982   6.082  1.00  0.00           O  \\nATOM   5748  CB  PHE B 367      13.898  15.348   5.145  1.00  0.00           C  \\nATOM   5749  CG  PHE B 367      15.118  16.177   4.931  1.00  0.00           C  \\nATOM   5750  CD1 PHE B 367      15.393  16.781   3.700  1.00  0.00           C  \\nATOM   5751  CD2 PHE B 367      16.063  16.309   5.961  1.00  0.00           C  \\nATOM   5752  CE1 PHE B 367      16.546  17.543   3.554  1.00  0.00           C  \\nATOM   5753  CE2 PHE B 367      17.184  17.033   5.789  1.00  0.00           C  \\nATOM   5754  CZ  PHE B 367      17.426  17.677   4.601  1.00  0.00           C  \\nATOM   5755  H   PHE B 367      13.038  17.129   6.773  1.00  0.00           H  \\nATOM   5756  HA  PHE B 367      12.635  16.658   3.978  1.00  0.00           H  \\nATOM   5757 1HB  PHE B 367      13.925  14.948   6.158  1.00  0.00           H  \\nATOM   5758 2HB  PHE B 367      13.929  14.503   4.458  1.00  0.00           H  \\nATOM   5759  HD1 PHE B 367      14.696  16.645   2.873  1.00  0.00           H  \\nATOM   5760  HD2 PHE B 367      15.882  15.802   6.909  1.00  0.00           H  \\nATOM   5761  HE1 PHE B 367      16.755  18.038   2.606  1.00  0.00           H  \\nATOM   5762  HE2 PHE B 367      17.904  17.105   6.604  1.00  0.00           H  \\nATOM   5763  HZ  PHE B 367      18.317  18.293   4.490  1.00  0.00           H  \\nATOM   5764  N   ALA B 368      11.179  14.559   3.858  1.00  0.00           N  \\nATOM   5765  CA  ALA B 368      10.181  13.523   3.803  1.00  0.00           C  \\nATOM   5766  C   ALA B 368      10.565  12.234   4.545  1.00  0.00           C  \\nATOM   5767  O   ALA B 368       9.707  11.485   4.944  1.00  0.00           O  \\nATOM   5768  CB  ALA B 368       9.773  13.230   2.347  1.00  0.00           C  \\nATOM   5769  H   ALA B 368      11.655  14.829   3.009  1.00  0.00           H  \\nATOM   5770  HA  ALA B 368       9.303  13.869   4.348  1.00  0.00           H  \\nATOM   5771 1HB  ALA B 368       9.018  12.444   2.332  1.00  0.00           H  \\nATOM   5772 2HB  ALA B 368       9.365  14.134   1.895  1.00  0.00           H  \\nATOM   5773 3HB  ALA B 368      10.646  12.905   1.783  1.00  0.00           H  \\nATOM   5774  N   PHE B 369      11.858  12.071   4.875  1.00  0.00           N  \\nATOM   5775  CA  PHE B 369      12.460  10.830   5.318  1.00  0.00           C  \\nATOM   5776  C   PHE B 369      12.953  11.108   6.755  1.00  0.00           C  \\nATOM   5777  O   PHE B 369      13.311  12.252   7.088  1.00  0.00           O  \\nATOM   5778  CB  PHE B 369      13.610  10.420   4.334  1.00  0.00           C  \\nATOM   5779  CG  PHE B 369      14.658  11.486   4.118  1.00  0.00           C  \\nATOM   5780  CD1 PHE B 369      15.705  11.710   5.054  1.00  0.00           C  \\nATOM   5781  CD2 PHE B 369      14.662  12.227   2.963  1.00  0.00           C  \\nATOM   5782  CE1 PHE B 369      16.693  12.671   4.774  1.00  0.00           C  \\nATOM   5783  CE2 PHE B 369      15.602  13.205   2.742  1.00  0.00           C  \\nATOM   5784  CZ  PHE B 369      16.599  13.440   3.613  1.00  0.00           C  \\nATOM   5785  H   PHE B 369      12.434  12.897   4.799  1.00  0.00           H  \\nATOM   5786  HA  PHE B 369      11.694  10.053   5.314  1.00  0.00           H  \\nATOM   5787 1HB  PHE B 369      14.112   9.530   4.712  1.00  0.00           H  \\nATOM   5788 2HB  PHE B 369      13.185  10.168   3.363  1.00  0.00           H  \\nATOM   5789  HD1 PHE B 369      15.727  11.131   5.978  1.00  0.00           H  \\nATOM   5790  HD2 PHE B 369      13.866  12.062   2.236  1.00  0.00           H  \\nATOM   5791  HE1 PHE B 369      17.526  12.811   5.462  1.00  0.00           H  \\nATOM   5792  HE2 PHE B 369      15.523  13.799   1.832  1.00  0.00           H  \\nATOM   5793  HZ  PHE B 369      17.328  14.225   3.417  1.00  0.00           H  \\nATOM   5794  N   GLY B 370      12.921  10.064   7.577  1.00  0.00           N  \\nATOM   5795  CA  GLY B 370      13.456  10.052   8.939  1.00  0.00           C  \\nATOM   5796  C   GLY B 370      14.952   9.855   8.855  1.00  0.00           C  \\nATOM   5797  O   GLY B 370      15.519   9.627   7.774  1.00  0.00           O  \\nATOM   5798  H   GLY B 370      12.490   9.228   7.211  1.00  0.00           H  \\nATOM   5799 1HA  GLY B 370      13.208  10.990   9.436  1.00  0.00           H  \\nATOM   5800 2HA  GLY B 370      12.983   9.252   9.508  1.00  0.00           H  \\nATOM   5801  N   GLY B 371      15.591   9.893   9.999  1.00  0.00           N  \\nATOM   5802  CA  GLY B 371      17.055   9.818  10.011  1.00  0.00           C  \\nATOM   5803  C   GLY B 371      17.557   8.506  10.591  1.00  0.00           C  \\nATOM   5804  O   GLY B 371      18.709   8.444  11.009  1.00  0.00           O  \\nATOM   5805  H   GLY B 371      15.095   9.973  10.875  1.00  0.00           H  \\nATOM   5806 1HA  GLY B 371      17.433   9.929   8.995  1.00  0.00           H  \\nATOM   5807 2HA  GLY B 371      17.457  10.645  10.595  1.00  0.00           H  \\nATOM   5808  N   ASN B 372      16.682   7.457  10.579  1.00  0.00           N  \\nATOM   5809  CA  ASN B 372      17.121   6.113  10.952  1.00  0.00           C  \\nATOM   5810  C   ASN B 372      18.038   5.588   9.879  1.00  0.00           C  \\nATOM   5811  O   ASN B 372      17.892   5.870   8.623  1.00  0.00           O  \\nATOM   5812  CB  ASN B 372      15.925   5.170  11.096  1.00  0.00           C  \\nATOM   5813  CG  ASN B 372      15.069   5.160   9.823  1.00  0.00           C  \\nATOM   5814  OD1 ASN B 372      14.307   6.121   9.539  1.00  0.00           O  \\nATOM   5815  ND2 ASN B 372      15.150   4.051   9.063  1.00  0.00           N  \\nATOM   5816  H   ASN B 372      15.719   7.603  10.311  1.00  0.00           H  \\nATOM   5817  HA  ASN B 372      17.633   6.171  11.914  1.00  0.00           H  \\nATOM   5818 1HB  ASN B 372      16.281   4.160  11.304  1.00  0.00           H  \\nATOM   5819 2HB  ASN B 372      15.315   5.483  11.943  1.00  0.00           H  \\nATOM   5820 1HD2 ASN B 372      14.616   3.982   8.220  1.00  0.00           H  \\nATOM   5821 2HD2 ASN B 372      15.743   3.297   9.342  1.00  0.00           H  \\nATOM   5822  N   TYR B 373      19.046   4.793  10.261  1.00  0.00           N  \\nATOM   5823  CA  TYR B 373      20.005   4.268   9.298  1.00  0.00           C  \\nATOM   5824  C   TYR B 373      19.423   3.535   8.036  1.00  0.00           C  \\nATOM   5825  O   TYR B 373      19.965   3.686   6.949  1.00  0.00           O  \\nATOM   5826  CB  TYR B 373      21.023   3.319  10.008  1.00  0.00           C  \\nATOM   5827  CG  TYR B 373      22.028   3.989  10.957  1.00  0.00           C  \\nATOM   5828  CD1 TYR B 373      22.767   5.096  10.559  1.00  0.00           C  \\nATOM   5829  CD2 TYR B 373      22.140   3.546  12.282  1.00  0.00           C  \\nATOM   5830  CE1 TYR B 373      23.643   5.726  11.457  1.00  0.00           C  \\nATOM   5831  CE2 TYR B 373      23.015   4.184  13.176  1.00  0.00           C  \\nATOM   5832  CZ  TYR B 373      23.763   5.173  12.729  1.00  0.00           C  \\nATOM   5833  OH  TYR B 373      24.689   5.781  13.529  1.00  0.00           O  \\nATOM   5834  H   TYR B 373      19.146   4.551  11.236  1.00  0.00           H  \\nATOM   5835  HA  TYR B 373      20.552   5.105   8.863  1.00  0.00           H  \\nATOM   5836 1HB  TYR B 373      20.480   2.576  10.594  1.00  0.00           H  \\nATOM   5837 2HB  TYR B 373      21.604   2.783   9.258  1.00  0.00           H  \\nATOM   5838  HD1 TYR B 373      22.669   5.481   9.544  1.00  0.00           H  \\nATOM   5839  HD2 TYR B 373      21.546   2.699  12.626  1.00  0.00           H  \\nATOM   5840  HE1 TYR B 373      24.219   6.594  11.138  1.00  0.00           H  \\nATOM   5841  HE2 TYR B 373      23.066   3.858  14.215  1.00  0.00           H  \\nATOM   5842  HH  TYR B 373      24.677   5.373  14.398  1.00  0.00           H  \\nATOM   5843  N   ASP B 374      18.399   2.705   8.241  1.00  0.00           N  \\nATOM   5844  CA  ASP B 374      17.849   1.971   7.094  1.00  0.00           C  \\nATOM   5845  C   ASP B 374      17.440   2.889   5.962  1.00  0.00           C  \\nATOM   5846  O   ASP B 374      17.872   2.579   4.769  1.00  0.00           O  \\nATOM   5847  CB  ASP B 374      16.653   1.136   7.580  1.00  0.00           C  \\nATOM   5848  CG  ASP B 374      17.061  -0.015   8.529  1.00  0.00           C  \\nATOM   5849  OD1 ASP B 374      18.262  -0.318   8.641  1.00  0.00           O  \\nATOM   5850  OD2 ASP B 374      16.141  -0.638   9.129  1.00  0.00           O  \\nATOM   5851  H   ASP B 374      17.992   2.566   9.155  1.00  0.00           H  \\nATOM   5852  HA  ASP B 374      18.623   1.310   6.703  1.00  0.00           H  \\nATOM   5853 1HB  ASP B 374      15.947   1.782   8.102  1.00  0.00           H  \\nATOM   5854 2HB  ASP B 374      16.134   0.710   6.721  1.00  0.00           H  \\nATOM   5855  N   ARG B 375      16.736   3.975   6.297  1.00  0.00           N  \\nATOM   5856  CA  ARG B 375      16.296   4.928   5.310  1.00  0.00           C  \\nATOM   5857  C   ARG B 375      17.490   5.700   4.682  1.00  0.00           C  \\nATOM   5858  O   ARG B 375      17.606   5.881   3.432  1.00  0.00           O  \\nATOM   5859  CB  ARG B 375      15.323   5.901   5.951  1.00  0.00           C  \\nATOM   5860  CG  ARG B 375      14.986   7.146   5.076  1.00  0.00           C  \\nATOM   5861  CD  ARG B 375      14.255   6.832   3.722  1.00  0.00           C  \\nATOM   5862  NE  ARG B 375      13.000   6.075   3.865  1.00  0.00           N  \\nATOM   5863  CZ  ARG B 375      12.815   4.851   3.360  1.00  0.00           C  \\nATOM   5864  NH1 ARG B 375      13.795   4.206   2.669  1.00  0.00           N  \\nATOM   5865  NH2 ARG B 375      11.633   4.240   3.568  1.00  0.00           N  \\nATOM   5866  H   ARG B 375      16.507   4.131   7.268  1.00  0.00           H  \\nATOM   5867  HA  ARG B 375      15.789   4.388   4.510  1.00  0.00           H  \\nATOM   5868 1HB  ARG B 375      14.389   5.387   6.174  1.00  0.00           H  \\nATOM   5869 2HB  ARG B 375      15.734   6.258   6.895  1.00  0.00           H  \\nATOM   5870 1HG  ARG B 375      14.332   7.817   5.634  1.00  0.00           H  \\nATOM   5871 2HG  ARG B 375      15.907   7.669   4.818  1.00  0.00           H  \\nATOM   5872 1HD  ARG B 375      14.005   7.765   3.219  1.00  0.00           H  \\nATOM   5873 2HD  ARG B 375      14.910   6.240   3.084  1.00  0.00           H  \\nATOM   5874  HE  ARG B 375      12.243   6.512   4.374  1.00  0.00           H  \\nATOM   5875 1HH1 ARG B 375      14.690   4.652   2.527  1.00  0.00           H  \\nATOM   5876 2HH1 ARG B 375      13.627   3.282   2.299  1.00  0.00           H  \\nATOM   5877 1HH2 ARG B 375      10.909   4.707   4.096  1.00  0.00           H  \\nATOM   5878 2HH2 ARG B 375      11.470   3.316   3.196  1.00  0.00           H  \\nATOM   5879  N   LEU B 376      18.436   6.160   5.524  1.00  0.00           N  \\nATOM   5880  CA  LEU B 376      19.527   6.945   5.029  1.00  0.00           C  \\nATOM   5881  C   LEU B 376      20.427   6.132   4.108  1.00  0.00           C  \\nATOM   5882  O   LEU B 376      20.973   6.643   3.153  1.00  0.00           O  \\nATOM   5883  CB  LEU B 376      20.280   7.552   6.258  1.00  0.00           C  \\nATOM   5884  CG  LEU B 376      19.507   8.563   7.050  1.00  0.00           C  \\nATOM   5885  CD1 LEU B 376      20.457   8.901   8.260  1.00  0.00           C  \\nATOM   5886  CD2 LEU B 376      19.168   9.850   6.278  1.00  0.00           C  \\nATOM   5887  H   LEU B 376      18.385   5.956   6.512  1.00  0.00           H  \\nATOM   5888  HA  LEU B 376      19.124   7.745   4.410  1.00  0.00           H  \\nATOM   5889 1HB  LEU B 376      20.557   6.742   6.930  1.00  0.00           H  \\nATOM   5890 2HB  LEU B 376      21.193   8.031   5.904  1.00  0.00           H  \\nATOM   5891  HG  LEU B 376      18.561   8.127   7.372  1.00  0.00           H  \\nATOM   5892 1HD1 LEU B 376      19.978   9.638   8.904  1.00  0.00           H  \\nATOM   5893 2HD1 LEU B 376      20.654   7.994   8.832  1.00  0.00           H  \\nATOM   5894 3HD1 LEU B 376      21.396   9.305   7.883  1.00  0.00           H  \\nATOM   5895 1HD2 LEU B 376      18.608  10.526   6.925  1.00  0.00           H  \\nATOM   5896 2HD2 LEU B 376      20.090  10.336   5.957  1.00  0.00           H  \\nATOM   5897 3HD2 LEU B 376      18.566   9.602   5.404  1.00  0.00           H  \\nATOM   5898  N   GLU B 377      20.574   4.832   4.434  1.00  0.00           N  \\nATOM   5899  CA  GLU B 377      21.410   3.968   3.654  1.00  0.00           C  \\nATOM   5900  C   GLU B 377      20.728   3.747   2.265  1.00  0.00           C  \\nATOM   5901  O   GLU B 377      21.484   3.783   1.287  1.00  0.00           O  \\nATOM   5902  CB  GLU B 377      21.566   2.617   4.334  1.00  0.00           C  \\nATOM   5903  CG  GLU B 377      22.549   2.630   5.555  1.00  0.00           C  \\nATOM   5904  CD  GLU B 377      22.528   1.313   6.285  1.00  0.00           C  \\nATOM   5905  OE1 GLU B 377      21.537   0.571   6.185  1.00  0.00           O  \\nATOM   5906  OE2 GLU B 377      23.539   1.016   7.048  1.00  0.00           O  \\nATOM   5907  H   GLU B 377      20.091   4.456   5.237  1.00  0.00           H  \\nATOM   5908  HA  GLU B 377      22.395   4.428   3.565  1.00  0.00           H  \\nATOM   5909 1HB  GLU B 377      20.593   2.272   4.685  1.00  0.00           H  \\nATOM   5910 2HB  GLU B 377      21.931   1.886   3.612  1.00  0.00           H  \\nATOM   5911 1HG  GLU B 377      23.557   2.834   5.195  1.00  0.00           H  \\nATOM   5912 2HG  GLU B 377      22.264   3.436   6.229  1.00  0.00           H  \\nATOM   5913  N   GLN B 378      19.383   3.668   2.275  1.00  0.00           N  \\nATOM   5914  CA  GLN B 378      18.673   3.517   0.953  1.00  0.00           C  \\nATOM   5915  C   GLN B 378      18.941   4.748   0.125  1.00  0.00           C  \\nATOM   5916  O   GLN B 378      19.286   4.661  -1.084  1.00  0.00           O  \\nATOM   5917  CB  GLN B 378      17.207   3.197   1.227  1.00  0.00           C  \\nATOM   5918  CG  GLN B 378      16.390   3.083  -0.121  1.00  0.00           C  \\nATOM   5919  CD  GLN B 378      15.701   4.356  -0.463  1.00  0.00           C  \\nATOM   5920  OE1 GLN B 378      15.154   5.049   0.403  1.00  0.00           O  \\nATOM   5921  NE2 GLN B 378      15.704   4.685  -1.777  1.00  0.00           N  \\nATOM   5922  H   GLN B 378      18.838   3.706   3.125  1.00  0.00           H  \\nATOM   5923  HA  GLN B 378      19.132   2.693   0.406  1.00  0.00           H  \\nATOM   5924 1HB  GLN B 378      17.136   2.260   1.779  1.00  0.00           H  \\nATOM   5925 2HB  GLN B 378      16.775   3.978   1.852  1.00  0.00           H  \\nATOM   5926 1HG  GLN B 378      17.076   2.833  -0.930  1.00  0.00           H  \\nATOM   5927 2HG  GLN B 378      15.639   2.301  -0.013  1.00  0.00           H  \\nATOM   5928 1HE2 GLN B 378      15.260   5.528  -2.083  1.00  0.00           H  \\nATOM   5929 2HE2 GLN B 378      16.150   4.086  -2.442  1.00  0.00           H  \\nATOM   5930  N   LEU B 379      18.776   5.921   0.735  1.00  0.00           N  \\nATOM   5931  CA  LEU B 379      18.962   7.162  -0.011  1.00  0.00           C  \\nATOM   5932  C   LEU B 379      20.396   7.338  -0.457  1.00  0.00           C  \\nATOM   5933  O   LEU B 379      20.674   7.917  -1.523  1.00  0.00           O  \\nATOM   5934  CB  LEU B 379      18.545   8.341   0.854  1.00  0.00           C  \\nATOM   5935  CG  LEU B 379      17.096   8.478   1.279  1.00  0.00           C  \\nATOM   5936  CD1 LEU B 379      16.862   9.648   2.195  1.00  0.00           C  \\nATOM   5937  CD2 LEU B 379      16.180   8.593   0.065  1.00  0.00           C  \\nATOM   5938  H   LEU B 379      18.523   5.967   1.712  1.00  0.00           H  \\nATOM   5939  HA  LEU B 379      18.331   7.130  -0.899  1.00  0.00           H  \\nATOM   5940 1HB  LEU B 379      19.122   8.316   1.777  1.00  0.00           H  \\nATOM   5941 2HB  LEU B 379      18.786   9.263   0.325  1.00  0.00           H  \\nATOM   5942  HG  LEU B 379      16.802   7.602   1.857  1.00  0.00           H  \\nATOM   5943 1HD1 LEU B 379      15.807   9.694   2.465  1.00  0.00           H  \\nATOM   5944 2HD1 LEU B 379      17.462   9.529   3.098  1.00  0.00           H  \\nATOM   5945 3HD1 LEU B 379      17.147  10.569   1.688  1.00  0.00           H  \\nATOM   5946 1HD2 LEU B 379      15.146   8.689   0.397  1.00  0.00           H  \\nATOM   5947 2HD2 LEU B 379      16.457   9.471  -0.518  1.00  0.00           H  \\nATOM   5948 3HD2 LEU B 379      16.281   7.701  -0.553  1.00  0.00           H  \\nATOM   5949  N   ALA B 380      21.400   6.951   0.377  1.00  0.00           N  \\nATOM   5950  CA  ALA B 380      22.776   7.145   0.047  1.00  0.00           C  \\nATOM   5951  C   ALA B 380      23.235   6.127  -1.007  1.00  0.00           C  \\nATOM   5952  O   ALA B 380      24.294   6.233  -1.643  1.00  0.00           O  \\nATOM   5953  CB  ALA B 380      23.641   6.944   1.337  1.00  0.00           C  \\nATOM   5954  H   ALA B 380      21.162   6.513   1.256  1.00  0.00           H  \\nATOM   5955  HA  ALA B 380      22.896   8.165  -0.318  1.00  0.00           H  \\nATOM   5956 1HB  ALA B 380      24.694   7.090   1.096  1.00  0.00           H  \\nATOM   5957 2HB  ALA B 380      23.339   7.668   2.094  1.00  0.00           H  \\nATOM   5958 3HB  ALA B 380      23.494   5.936   1.720  1.00  0.00           H  \\nATOM   5959  N   GLY B 381      22.377   5.127  -1.178  1.00  0.00           N  \\nATOM   5960  CA  GLY B 381      22.803   3.950  -1.969  1.00  0.00           C  \\nATOM   5961  C   GLY B 381      23.960   3.094  -1.473  1.00  0.00           C  \\nATOM   5962  O   GLY B 381      24.678   2.462  -2.261  1.00  0.00           O  \\nATOM   5963  H   GLY B 381      21.446   5.146  -0.786  1.00  0.00           H  \\nATOM   5964 1HA  GLY B 381      21.964   3.263  -2.080  1.00  0.00           H  \\nATOM   5965 2HA  GLY B 381      23.091   4.269  -2.970  1.00  0.00           H  \\nATOM   5966  N   ASN B 382      24.192   3.111  -0.173  1.00  0.00           N  \\nATOM   5967  CA  ASN B 382      25.306   2.395   0.416  1.00  0.00           C  \\nATOM   5968  C   ASN B 382      24.968   2.080   1.875  1.00  0.00           C  \\nATOM   5969  O   ASN B 382      24.394   2.968   2.553  1.00  0.00           O  \\nATOM   5970  CB  ASN B 382      26.611   3.211   0.352  1.00  0.00           C  \\nATOM   5971  CG  ASN B 382      27.302   3.165  -1.031  1.00  0.00           C  \\nATOM   5972  OD1 ASN B 382      27.512   2.099  -1.541  1.00  0.00           O  \\nATOM   5973  ND2 ASN B 382      27.641   4.313  -1.621  1.00  0.00           N  \\nATOM   5974  H   ASN B 382      23.574   3.639   0.427  1.00  0.00           H  \\nATOM   5975  HA  ASN B 382      25.459   1.471  -0.144  1.00  0.00           H  \\nATOM   5976 1HB  ASN B 382      26.401   4.253   0.596  1.00  0.00           H  \\nATOM   5977 2HB  ASN B 382      27.312   2.835   1.098  1.00  0.00           H  \\nATOM   5978 1HD2 ASN B 382      28.089   4.303  -2.516  1.00  0.00           H  \\nATOM   5979 2HD2 ASN B 382      27.448   5.183  -1.169  1.00  0.00           H  \\nATOM   5980  N   LEU B 383      25.281   0.865   2.339  1.00  0.00           N  \\nATOM   5981  CA  LEU B 383      25.209   0.533   3.781  1.00  0.00           C  \\nATOM   5982  C   LEU B 383      26.385   1.169   4.581  1.00  0.00           C  \\nATOM   5983  O   LEU B 383      27.407   1.546   4.023  1.00  0.00           O  \\nATOM   5984  CB  LEU B 383      25.167  -0.999   4.080  1.00  0.00           C  \\nATOM   5985  CG  LEU B 383      24.123  -1.729   3.247  1.00  0.00           C  \\nATOM   5986  CD1 LEU B 383      24.510  -3.207   3.312  1.00  0.00           C  \\nATOM   5987  CD2 LEU B 383      22.767  -1.453   3.681  1.00  0.00           C  \\nATOM   5988  H   LEU B 383      25.575   0.154   1.685  1.00  0.00           H  \\nATOM   5989  HA  LEU B 383      24.294   0.960   4.189  1.00  0.00           H  \\nATOM   5990 1HB  LEU B 383      26.150  -1.420   3.875  1.00  0.00           H  \\nATOM   5991 2HB  LEU B 383      24.947  -1.141   5.138  1.00  0.00           H  \\nATOM   5992  HG  LEU B 383      24.209  -1.421   2.205  1.00  0.00           H  \\nATOM   5993 1HD1 LEU B 383      23.799  -3.795   2.732  1.00  0.00           H  \\nATOM   5994 2HD1 LEU B 383      25.511  -3.340   2.900  1.00  0.00           H  \\nATOM   5995 3HD1 LEU B 383      24.496  -3.541   4.349  1.00  0.00           H  \\nATOM   5996 1HD2 LEU B 383      22.065  -2.000   3.051  1.00  0.00           H  \\nATOM   5997 2HD2 LEU B 383      22.645  -1.768   4.718  1.00  0.00           H  \\nATOM   5998 3HD2 LEU B 383      22.570  -0.384   3.601  1.00  0.00           H  \\nATOM   5999  N   ARG B 384      26.115   1.380   5.889  1.00  0.00           N  \\nATOM   6000  CA  ARG B 384      27.215   1.786   6.771  1.00  0.00           C  \\nATOM   6001  C   ARG B 384      28.501   1.032   6.504  1.00  0.00           C  \\nATOM   6002  O   ARG B 384      29.675   1.598   6.511  1.00  0.00           O  \\nATOM   6003  CB  ARG B 384      26.810   1.628   8.196  1.00  0.00           C  \\nATOM   6004  CG  ARG B 384      25.916   2.787   8.679  1.00  0.00           C  \\nATOM   6005  CD  ARG B 384      25.367   2.463  10.058  1.00  0.00           C  \\nATOM   6006  NE  ARG B 384      24.286   1.455   9.910  1.00  0.00           N  \\nATOM   6007  CZ  ARG B 384      23.778   0.741  10.899  1.00  0.00           C  \\nATOM   6008  NH1 ARG B 384      24.285   0.886  12.177  1.00  0.00           N  \\nATOM   6009  NH2 ARG B 384      22.673  -0.069  10.735  1.00  0.00           N  \\nATOM   6010  H   ARG B 384      25.189   1.269   6.277  1.00  0.00           H  \\nATOM   6011  HA  ARG B 384      27.440   2.836   6.582  1.00  0.00           H  \\nATOM   6012 1HB  ARG B 384      26.272   0.689   8.319  1.00  0.00           H  \\nATOM   6013 2HB  ARG B 384      27.700   1.580   8.824  1.00  0.00           H  \\nATOM   6014 1HG  ARG B 384      26.504   3.704   8.729  1.00  0.00           H  \\nATOM   6015 2HG  ARG B 384      25.089   2.924   7.982  1.00  0.00           H  \\nATOM   6016 1HD  ARG B 384      26.165   2.063  10.682  1.00  0.00           H  \\nATOM   6017 2HD  ARG B 384      24.970   3.369  10.514  1.00  0.00           H  \\nATOM   6018  HE  ARG B 384      23.909   1.302   8.984  1.00  0.00           H  \\nATOM   6019 1HH1 ARG B 384      25.040   1.535  12.350  1.00  0.00           H  \\nATOM   6020 2HH1 ARG B 384      23.901   0.343  12.937  1.00  0.00           H  \\nATOM   6021 1HH2 ARG B 384      22.222  -0.132   9.833  1.00  0.00           H  \\nATOM   6022 2HH2 ARG B 384      22.315  -0.599  11.516  1.00  0.00           H  \\nATOM   6023  N   GLU B 385      28.346  -0.272   6.262  1.00  0.00           N  \\nATOM   6024  CA  GLU B 385      29.581  -1.035   5.954  1.00  0.00           C  \\nATOM   6025  C   GLU B 385      30.431  -0.629   4.789  1.00  0.00           C  \\nATOM   6026  O   GLU B 385      31.602  -1.075   4.638  1.00  0.00           O  \\nATOM   6027  CB  GLU B 385      29.214  -2.506   5.743  1.00  0.00           C  \\nATOM   6028  CG  GLU B 385      30.408  -3.430   5.549  1.00  0.00           C  \\nATOM   6029  CD  GLU B 385      30.017  -4.878   5.447  1.00  0.00           C  \\nATOM   6030  OE1 GLU B 385      28.847  -5.165   5.536  1.00  0.00           O  \\nATOM   6031  OE2 GLU B 385      30.888  -5.698   5.279  1.00  0.00           O  \\nATOM   6032  H   GLU B 385      27.460  -0.756   6.276  1.00  0.00           H  \\nATOM   6033  HA  GLU B 385      30.311  -0.844   6.741  1.00  0.00           H  \\nATOM   6034 1HB  GLU B 385      28.648  -2.866   6.601  1.00  0.00           H  \\nATOM   6035 2HB  GLU B 385      28.574  -2.598   4.865  1.00  0.00           H  \\nATOM   6036 1HG  GLU B 385      30.933  -3.142   4.638  1.00  0.00           H  \\nATOM   6037 2HG  GLU B 385      31.093  -3.302   6.386  1.00  0.00           H  \\nATOM   6038  N   ASN B 386      29.837   0.209   3.915  1.00  0.00           N  \\nATOM   6039  CA  ASN B 386      30.550   0.679   2.737  1.00  0.00           C  \\nATOM   6040  C   ASN B 386      30.694   2.181   2.674  1.00  0.00           C  \\nATOM   6041  O   ASN B 386      30.955   2.743   1.615  1.00  0.00           O  \\nATOM   6042  CB  ASN B 386      29.786   0.191   1.477  1.00  0.00           C  \\nATOM   6043  CG  ASN B 386      29.837  -1.343   1.305  1.00  0.00           C  \\nATOM   6044  OD1 ASN B 386      28.784  -2.001   1.173  1.00  0.00           O  \\nATOM   6045  ND2 ASN B 386      31.032  -1.917   1.424  1.00  0.00           N  \\nATOM   6046  H   ASN B 386      28.888   0.518   4.072  1.00  0.00           H  \\nATOM   6047  HA  ASN B 386      31.554   0.252   2.746  1.00  0.00           H  \\nATOM   6048 1HB  ASN B 386      28.742   0.502   1.541  1.00  0.00           H  \\nATOM   6049 2HB  ASN B 386      30.213   0.657   0.589  1.00  0.00           H  \\nATOM   6050 1HD2 ASN B 386      31.123  -2.908   1.322  1.00  0.00           H  \\nATOM   6051 2HD2 ASN B 386      31.838  -1.358   1.615  1.00  0.00           H  \\nATOM   6052  N   ILE B 387      30.588   2.871   3.824  1.00  0.00           N  \\nATOM   6053  CA  ILE B 387      30.798   4.319   3.841  1.00  0.00           C  \\nATOM   6054  C   ILE B 387      32.007   4.609   4.770  1.00  0.00           C  \\nATOM   6055  O   ILE B 387      31.970   4.336   6.014  1.00  0.00           O  \\nATOM   6056  CB  ILE B 387      29.521   5.011   4.359  1.00  0.00           C  \\nATOM   6057  CG1 ILE B 387      28.401   4.877   3.323  1.00  0.00           C  \\nATOM   6058  CG2 ILE B 387      29.727   6.531   4.680  1.00  0.00           C  \\nATOM   6059  CD1 ILE B 387      26.983   5.218   3.853  1.00  0.00           C  \\nATOM   6060  H   ILE B 387      30.363   2.396   4.687  1.00  0.00           H  \\nATOM   6061  HA  ILE B 387      30.999   4.651   2.823  1.00  0.00           H  \\nATOM   6062  HB  ILE B 387      29.190   4.524   5.276  1.00  0.00           H  \\nATOM   6063 1HG1 ILE B 387      28.607   5.533   2.478  1.00  0.00           H  \\nATOM   6064 2HG1 ILE B 387      28.376   3.855   2.944  1.00  0.00           H  \\nATOM   6065 1HG2 ILE B 387      28.790   6.957   5.039  1.00  0.00           H  \\nATOM   6066 2HG2 ILE B 387      30.493   6.640   5.447  1.00  0.00           H  \\nATOM   6067 3HG2 ILE B 387      30.041   7.054   3.777  1.00  0.00           H  \\nATOM   6068 1HD1 ILE B 387      26.255   5.095   3.051  1.00  0.00           H  \\nATOM   6069 2HD1 ILE B 387      26.732   4.549   4.677  1.00  0.00           H  \\nATOM   6070 3HD1 ILE B 387      26.964   6.249   4.204  1.00  0.00           H  \\nATOM   6071  N   GLU B 388      33.061   5.161   4.188  1.00  0.00           N  \\nATOM   6072  CA  GLU B 388      34.265   5.438   5.001  1.00  0.00           C  \\nATOM   6073  C   GLU B 388      34.023   6.557   6.031  1.00  0.00           C  \\nATOM   6074  O   GLU B 388      33.384   7.597   5.780  1.00  0.00           O  \\nATOM   6075  CB  GLU B 388      35.476   5.825   4.163  1.00  0.00           C  \\nATOM   6076  CG  GLU B 388      36.054   4.633   3.329  1.00  0.00           C  \\nATOM   6077  CD  GLU B 388      37.068   5.043   2.251  1.00  0.00           C  \\nATOM   6078  OE1 GLU B 388      37.097   6.228   1.860  1.00  0.00           O  \\nATOM   6079  OE2 GLU B 388      37.826   4.146   1.762  1.00  0.00           O  \\nATOM   6080  H   GLU B 388      33.065   5.398   3.206  1.00  0.00           H  \\nATOM   6081  HA  GLU B 388      34.526   4.534   5.552  1.00  0.00           H  \\nATOM   6082 1HB  GLU B 388      35.203   6.627   3.477  1.00  0.00           H  \\nATOM   6083 2HB  GLU B 388      36.263   6.205   4.815  1.00  0.00           H  \\nATOM   6084 1HG  GLU B 388      36.546   3.934   4.006  1.00  0.00           H  \\nATOM   6085 2HG  GLU B 388      35.231   4.109   2.845  1.00  0.00           H  \\nATOM   6086  N   LEU B 389      34.606   6.317   7.207  1.00  0.00           N  \\nATOM   6087  CA  LEU B 389      34.670   7.345   8.306  1.00  0.00           C  \\nATOM   6088  C   LEU B 389      36.095   7.774   8.560  1.00  0.00           C  \\nATOM   6089  O   LEU B 389      37.004   7.021   8.285  1.00  0.00           O  \\nATOM   6090  CB  LEU B 389      34.108   6.705   9.588  1.00  0.00           C  \\nATOM   6091  CG  LEU B 389      32.680   6.108   9.439  1.00  0.00           C  \\nATOM   6092  CD1 LEU B 389      32.160   5.449  10.798  1.00  0.00           C  \\nATOM   6093  CD2 LEU B 389      31.626   7.183   8.996  1.00  0.00           C  \\nATOM   6094  H   LEU B 389      35.019   5.408   7.359  1.00  0.00           H  \\nATOM   6095  HA  LEU B 389      34.056   8.197   8.018  1.00  0.00           H  \\nATOM   6096 1HB  LEU B 389      34.781   5.909   9.902  1.00  0.00           H  \\nATOM   6097 2HB  LEU B 389      34.083   7.462  10.372  1.00  0.00           H  \\nATOM   6098  HG  LEU B 389      32.692   5.319   8.687  1.00  0.00           H  \\nATOM   6099 1HD1 LEU B 389      31.159   5.045  10.647  1.00  0.00           H  \\nATOM   6100 2HD1 LEU B 389      32.834   4.645  11.094  1.00  0.00           H  \\nATOM   6101 3HD1 LEU B 389      32.131   6.206  11.581  1.00  0.00           H  \\nATOM   6102 1HD2 LEU B 389      30.645   6.715   8.905  1.00  0.00           H  \\nATOM   6103 2HD2 LEU B 389      31.582   7.978   9.741  1.00  0.00           H  \\nATOM   6104 3HD2 LEU B 389      31.918   7.604   8.034  1.00  0.00           H  \\nATOM   6105  N   GLY B 390      36.289   8.939   9.180  1.00  0.00           N  \\nATOM   6106  CA  GLY B 390      37.513   9.594   9.421  1.00  0.00           C  \\nATOM   6107  C   GLY B 390      37.451  11.062   9.276  1.00  0.00           C  \\nATOM   6108  O   GLY B 390      36.353  11.585   9.010  1.00  0.00           O  \\nATOM   6109  H   GLY B 390      35.431   9.367   9.497  1.00  0.00           H  \\nATOM   6110 1HA  GLY B 390      37.855   9.366  10.431  1.00  0.00           H  \\nATOM   6111 2HA  GLY B 390      38.269   9.217   8.733  1.00  0.00           H  \\nATOM   6112  N   ASN B 391      38.567  11.729   9.515  1.00  0.00           N  \\nATOM   6113  CA  ASN B 391      38.530  13.151   9.448  1.00  0.00           C  \\nATOM   6114  C   ASN B 391      38.233  13.653   8.034  1.00  0.00           C  \\nATOM   6115  O   ASN B 391      37.590  14.737   7.848  1.00  0.00           O  \\nATOM   6116  CB  ASN B 391      39.836  13.729   9.981  1.00  0.00           C  \\nATOM   6117  CG  ASN B 391      39.659  15.177  10.422  1.00  0.00           C  \\nATOM   6118  OD1 ASN B 391      38.830  15.486  11.256  1.00  0.00           O  \\nATOM   6119  ND2 ASN B 391      40.481  16.040   9.910  1.00  0.00           N  \\nATOM   6120  H   ASN B 391      39.438  11.270   9.742  1.00  0.00           H  \\nATOM   6121  HA  ASN B 391      37.706  13.506  10.069  1.00  0.00           H  \\nATOM   6122 1HB  ASN B 391      40.182  13.130  10.825  1.00  0.00           H  \\nATOM   6123 2HB  ASN B 391      40.601  13.675   9.206  1.00  0.00           H  \\nATOM   6124 1HD2 ASN B 391      40.413  17.006  10.162  1.00  0.00           H  \\nATOM   6125 2HD2 ASN B 391      41.182  15.740   9.264  1.00  0.00           H  \\nATOM   6126  N   GLY B 392      38.711  12.955   7.017  1.00  0.00           N  \\nATOM   6127  CA  GLY B 392      38.458  13.315   5.593  1.00  0.00           C  \\nATOM   6128  C   GLY B 392      36.999  13.191   5.265  1.00  0.00           C  \\nATOM   6129  O   GLY B 392      36.434  14.139   4.761  1.00  0.00           O  \\nATOM   6130  H   GLY B 392      39.272  12.142   7.227  1.00  0.00           H  \\nATOM   6131 1HA  GLY B 392      38.795  14.336   5.410  1.00  0.00           H  \\nATOM   6132 2HA  GLY B 392      39.042  12.663   4.944  1.00  0.00           H  \\nATOM   6133  N   PRO B 393      36.370  12.044   5.544  1.00  0.00           N  \\nATOM   6134  CA  PRO B 393      34.929  11.929   5.380  1.00  0.00           C  \\nATOM   6135  C   PRO B 393      34.161  13.009   6.131  1.00  0.00           C  \\nATOM   6136  O   PRO B 393      33.145  13.497   5.615  1.00  0.00           O  \\nATOM   6137  CB  PRO B 393      34.588  10.525   5.897  1.00  0.00           C  \\nATOM   6138  CG  PRO B 393      35.857   9.777   5.453  1.00  0.00           C  \\nATOM   6139  CD  PRO B 393      36.959  10.707   5.788  1.00  0.00           C  \\nATOM   6140  HA  PRO B 393      34.679  12.011   4.312  1.00  0.00           H  \\nATOM   6141 1HB  PRO B 393      34.420  10.555   6.984  1.00  0.00           H  \\nATOM   6142 2HB  PRO B 393      33.653  10.171   5.437  1.00  0.00           H  \\nATOM   6143 1HG  PRO B 393      35.935   8.815   5.981  1.00  0.00           H  \\nATOM   6144 2HG  PRO B 393      35.806   9.546   4.379  1.00  0.00           H  \\nATOM   6145 1HD  PRO B 393      37.241  10.577   6.843  1.00  0.00           H  \\nATOM   6146 2HD  PRO B 393      37.818  10.510   5.130  1.00  0.00           H  \\nATOM   6147  N   LEU B 394      34.594  13.346   7.372  1.00  0.00           N  \\nATOM   6148  CA  LEU B 394      33.833  14.333   8.103  1.00  0.00           C  \\nATOM   6149  C   LEU B 394      34.059  15.738   7.464  1.00  0.00           C  \\nATOM   6150  O   LEU B 394      33.091  16.465   7.330  1.00  0.00           O  \\nATOM   6151  CB  LEU B 394      34.235  14.253   9.591  1.00  0.00           C  \\nATOM   6152  CG  LEU B 394      33.465  15.257  10.463  1.00  0.00           C  \\nATOM   6153  CD1 LEU B 394      31.973  14.892  10.462  1.00  0.00           C  \\nATOM   6154  CD2 LEU B 394      33.961  15.117  11.924  1.00  0.00           C  \\nATOM   6155  H   LEU B 394      35.414  12.941   7.800  1.00  0.00           H  \\nATOM   6156  HA  LEU B 394      32.774  14.101   7.997  1.00  0.00           H  \\nATOM   6157 1HB  LEU B 394      34.043  13.243   9.949  1.00  0.00           H  \\nATOM   6158 2HB  LEU B 394      35.304  14.448   9.673  1.00  0.00           H  \\nATOM   6159  HG  LEU B 394      33.650  16.269  10.102  1.00  0.00           H  \\nATOM   6160 1HD1 LEU B 394      31.424  15.603  11.080  1.00  0.00           H  \\nATOM   6161 2HD1 LEU B 394      31.590  14.928   9.442  1.00  0.00           H  \\nATOM   6162 3HD1 LEU B 394      31.844  13.888  10.864  1.00  0.00           H  \\nATOM   6163 1HD2 LEU B 394      33.426  15.823  12.559  1.00  0.00           H  \\nATOM   6164 2HD2 LEU B 394      33.777  14.101  12.275  1.00  0.00           H  \\nATOM   6165 3HD2 LEU B 394      35.030  15.328  11.967  1.00  0.00           H  \\nATOM   6166  N   GLU B 395      35.282  16.053   7.006  1.00  0.00           N  \\nATOM   6167  CA  GLU B 395      35.436  17.306   6.292  1.00  0.00           C  \\nATOM   6168  C   GLU B 395      34.494  17.398   5.120  1.00  0.00           C  \\nATOM   6169  O   GLU B 395      33.870  18.443   4.902  1.00  0.00           O  \\nATOM   6170  CB  GLU B 395      36.905  17.414   5.892  1.00  0.00           C  \\nATOM   6171  CG  GLU B 395      37.199  18.381   4.834  1.00  0.00           C  \\nATOM   6172  CD  GLU B 395      37.496  19.804   5.372  1.00  0.00           C  \\nATOM   6173  OE1 GLU B 395      38.089  19.966   6.516  1.00  0.00           O  \\nATOM   6174  OE2 GLU B 395      37.217  20.771   4.538  1.00  0.00           O  \\nATOM   6175  H   GLU B 395      36.088  15.459   7.137  1.00  0.00           H  \\nATOM   6176  HA  GLU B 395      35.166  18.122   6.963  1.00  0.00           H  \\nATOM   6177 1HB  GLU B 395      37.499  17.694   6.762  1.00  0.00           H  \\nATOM   6178 2HB  GLU B 395      37.262  16.442   5.551  1.00  0.00           H  \\nATOM   6179 1HG  GLU B 395      38.064  18.034   4.269  1.00  0.00           H  \\nATOM   6180 2HG  GLU B 395      36.350  18.428   4.153  1.00  0.00           H  \\nATOM   6181  N   GLU B 396      34.401  16.307   4.368  1.00  0.00           N  \\nATOM   6182  CA  GLU B 396      33.580  16.211   3.131  1.00  0.00           C  \\nATOM   6183  C   GLU B 396      32.127  16.363   3.492  1.00  0.00           C  \\nATOM   6184  O   GLU B 396      31.369  17.020   2.793  1.00  0.00           O  \\nATOM   6185  CB  GLU B 396      33.856  14.909   2.328  1.00  0.00           C  \\nATOM   6186  CG  GLU B 396      35.330  14.907   1.780  1.00  0.00           C  \\nATOM   6187  CD  GLU B 396      35.800  13.496   1.428  1.00  0.00           C  \\nATOM   6188  OE1 GLU B 396      35.028  12.492   1.601  1.00  0.00           O  \\nATOM   6189  OE2 GLU B 396      36.994  13.412   1.034  1.00  0.00           O  \\nATOM   6190  H   GLU B 396      34.929  15.501   4.672  1.00  0.00           H  \\nATOM   6191  HA  GLU B 396      33.827  17.054   2.485  1.00  0.00           H  \\nATOM   6192 1HB  GLU B 396      33.697  14.044   2.972  1.00  0.00           H  \\nATOM   6193 2HB  GLU B 396      33.151  14.836   1.500  1.00  0.00           H  \\nATOM   6194 1HG  GLU B 396      35.377  15.537   0.892  1.00  0.00           H  \\nATOM   6195 2HG  GLU B 396      35.987  15.338   2.534  1.00  0.00           H  \\nATOM   6196  N   ALA B 397      31.731  15.765   4.628  1.00  0.00           N  \\nATOM   6197  CA  ALA B 397      30.334  15.719   5.030  1.00  0.00           C  \\nATOM   6198  C   ALA B 397      29.856  17.111   5.413  1.00  0.00           C  \\nATOM   6199  O   ALA B 397      28.723  17.484   5.149  1.00  0.00           O  \\nATOM   6200  CB  ALA B 397      30.112  14.712   6.236  1.00  0.00           C  \\nATOM   6201  H   ALA B 397      32.428  15.335   5.219  1.00  0.00           H  \\nATOM   6202  HA  ALA B 397      29.748  15.372   4.179  1.00  0.00           H  \\nATOM   6203 1HB  ALA B 397      29.057  14.701   6.512  1.00  0.00           H  \\nATOM   6204 2HB  ALA B 397      30.418  13.710   5.935  1.00  0.00           H  \\nATOM   6205 3HB  ALA B 397      30.707  15.031   7.090  1.00  0.00           H  \\nATOM   6206  N   ILE B 398      30.737  17.849   6.088  1.00  0.00           N  \\nATOM   6207  CA  ILE B 398      30.413  19.257   6.509  1.00  0.00           C  \\nATOM   6208  C   ILE B 398      30.141  20.106   5.258  1.00  0.00           C  \\nATOM   6209  O   ILE B 398      29.166  20.856   5.249  1.00  0.00           O  \\nATOM   6210  CB  ILE B 398      31.514  19.829   7.462  1.00  0.00           C  \\nATOM   6211  CG1 ILE B 398      31.519  19.002   8.761  1.00  0.00           C  \\nATOM   6212  CG2 ILE B 398      31.394  21.381   7.692  1.00  0.00           C  \\nATOM   6213  CD1 ILE B 398      32.898  19.266   9.444  1.00  0.00           C  \\nATOM   6214  H   ILE B 398      31.639  17.460   6.323  1.00  0.00           H  \\nATOM   6215  HA  ILE B 398      29.466  19.248   7.047  1.00  0.00           H  \\nATOM   6216  HB  ILE B 398      32.499  19.636   7.037  1.00  0.00           H  \\nATOM   6217 1HG1 ILE B 398      30.684  19.309   9.390  1.00  0.00           H  \\nATOM   6218 2HG1 ILE B 398      31.377  17.948   8.522  1.00  0.00           H  \\nATOM   6219 1HG2 ILE B 398      32.187  21.712   8.362  1.00  0.00           H  \\nATOM   6220 2HG2 ILE B 398      31.487  21.898   6.738  1.00  0.00           H  \\nATOM   6221 3HG2 ILE B 398      30.425  21.609   8.136  1.00  0.00           H  \\nATOM   6222 1HD1 ILE B 398      32.957  18.701  10.375  1.00  0.00           H  \\nATOM   6223 2HD1 ILE B 398      33.701  18.951   8.777  1.00  0.00           H  \\nATOM   6224 3HD1 ILE B 398      33.000  20.329   9.658  1.00  0.00           H  \\nATOM   6225  N   SER B 399      31.039  20.011   4.287  1.00  0.00           N  \\nATOM   6226  CA  SER B 399      30.800  20.733   3.032  1.00  0.00           C  \\nATOM   6227  C   SER B 399      29.505  20.276   2.357  1.00  0.00           C  \\nATOM   6228  O   SER B 399      28.708  21.164   1.913  1.00  0.00           O  \\nATOM   6229  CB  SER B 399      31.956  20.546   2.044  1.00  0.00           C  \\nATOM   6230  OG  SER B 399      33.091  21.217   2.538  1.00  0.00           O  \\nATOM   6231  H   SER B 399      31.879  19.459   4.385  1.00  0.00           H  \\nATOM   6232  HA  SER B 399      30.715  21.797   3.256  1.00  0.00           H  \\nATOM   6233 1HB  SER B 399      32.160  19.483   1.920  1.00  0.00           H  \\nATOM   6234 2HB  SER B 399      31.669  20.940   1.070  1.00  0.00           H  \\nATOM   6235  HG  SER B 399      32.823  21.613   3.371  1.00  0.00           H  \\nATOM   6236  N   ALA B 400      29.253  18.970   2.323  1.00  0.00           N  \\nATOM   6237  CA  ALA B 400      28.021  18.461   1.673  1.00  0.00           C  \\nATOM   6238  C   ALA B 400      26.799  19.012   2.342  1.00  0.00           C  \\nATOM   6239  O   ALA B 400      25.776  19.482   1.693  1.00  0.00           O  \\nATOM   6240  CB  ALA B 400      28.025  16.975   1.668  1.00  0.00           C  \\nATOM   6241  H   ALA B 400      29.897  18.313   2.739  1.00  0.00           H  \\nATOM   6242  HA  ALA B 400      28.008  18.820   0.644  1.00  0.00           H  \\nATOM   6243 1HB  ALA B 400      27.116  16.610   1.189  1.00  0.00           H  \\nATOM   6244 2HB  ALA B 400      28.895  16.615   1.118  1.00  0.00           H  \\nATOM   6245 3HB  ALA B 400      28.067  16.608   2.692  1.00  0.00           H  \\nATOM   6246  N   LEU B 401      26.733  18.978   3.693  1.00  0.00           N  \\nATOM   6247  CA  LEU B 401      25.563  19.489   4.395  1.00  0.00           C  \\nATOM   6248  C   LEU B 401      25.458  20.981   4.096  1.00  0.00           C  \\nATOM   6249  O   LEU B 401      24.330  21.453   3.889  1.00  0.00           O  \\nATOM   6250  CB  LEU B 401      25.698  19.226   5.905  1.00  0.00           C  \\nATOM   6251  CG  LEU B 401      24.961  17.966   6.370  1.00  0.00           C  \\nATOM   6252  CD1 LEU B 401      25.508  16.774   5.829  1.00  0.00           C  \\nATOM   6253  CD2 LEU B 401      25.177  18.000   7.935  1.00  0.00           C  \\nATOM   6254  H   LEU B 401      27.500  18.596   4.228  1.00  0.00           H  \\nATOM   6255  HA  LEU B 401      24.682  18.967   4.024  1.00  0.00           H  \\nATOM   6256 1HB  LEU B 401      26.755  19.127   6.147  1.00  0.00           H  \\nATOM   6257 2HB  LEU B 401      25.302  20.086   6.444  1.00  0.00           H  \\nATOM   6258  HG  LEU B 401      23.910  18.034   6.087  1.00  0.00           H  \\nATOM   6259 1HD1 LEU B 401      24.944  15.916   6.195  1.00  0.00           H  \\nATOM   6260 2HD1 LEU B 401      25.450  16.808   4.741  1.00  0.00           H  \\nATOM   6261 3HD1 LEU B 401      26.549  16.680   6.135  1.00  0.00           H  \\nATOM   6262 1HD2 LEU B 401      24.688  17.137   8.389  1.00  0.00           H  \\nATOM   6263 2HD2 LEU B 401      26.244  17.970   8.156  1.00  0.00           H  \\nATOM   6264 3HD2 LEU B 401      24.748  18.916   8.342  1.00  0.00           H  \\nATOM   6265  N   TYR B 402      26.571  21.713   4.103  1.00  0.00           N  \\nATOM   6266  CA  TYR B 402      26.458  23.170   3.827  1.00  0.00           C  \\nATOM   6267  C   TYR B 402      25.870  23.459   2.438  1.00  0.00           C  \\nATOM   6268  O   TYR B 402      24.968  24.326   2.365  1.00  0.00           O  \\nATOM   6269  CB  TYR B 402      27.807  23.829   3.892  1.00  0.00           C  \\nATOM   6270  CG  TYR B 402      27.675  25.368   3.726  1.00  0.00           C  \\nATOM   6271  CD1 TYR B 402      27.323  26.170   4.798  1.00  0.00           C  \\nATOM   6272  CD2 TYR B 402      27.933  25.961   2.493  1.00  0.00           C  \\nATOM   6273  CE1 TYR B 402      27.163  27.538   4.654  1.00  0.00           C  \\nATOM   6274  CE2 TYR B 402      27.794  27.317   2.329  1.00  0.00           C  \\nATOM   6275  CZ  TYR B 402      27.402  28.093   3.403  1.00  0.00           C  \\nATOM   6276  OH  TYR B 402      27.286  29.470   3.183  1.00  0.00           O  \\nATOM   6277  H   TYR B 402      27.481  21.316   4.288  1.00  0.00           H  \\nATOM   6278  HA  TYR B 402      25.817  23.617   4.587  1.00  0.00           H  \\nATOM   6279 1HB  TYR B 402      28.279  23.600   4.849  1.00  0.00           H  \\nATOM   6280 2HB  TYR B 402      28.447  23.427   3.107  1.00  0.00           H  \\nATOM   6281  HD1 TYR B 402      27.166  25.727   5.782  1.00  0.00           H  \\nATOM   6282  HD2 TYR B 402      28.247  25.348   1.648  1.00  0.00           H  \\nATOM   6283  HE1 TYR B 402      26.885  28.148   5.514  1.00  0.00           H  \\nATOM   6284  HE2 TYR B 402      27.992  27.772   1.358  1.00  0.00           H  \\nATOM   6285  HH  TYR B 402      27.510  29.667   2.270  1.00  0.00           H  \\nATOM   6286  N   TYR B 403      26.268  22.708   1.464  1.00  0.00           N  \\nATOM   6287  CA  TYR B 403      25.864  23.121   0.073  1.00  0.00           C  \\nATOM   6288  C   TYR B 403      24.517  22.521  -0.297  1.00  0.00           C  \\nATOM   6289  O   TYR B 403      24.041  22.742  -1.396  1.00  0.00           O  \\nATOM   6290  CB  TYR B 403      26.912  22.681  -0.969  1.00  0.00           C  \\nATOM   6291  CG  TYR B 403      28.151  23.565  -0.997  1.00  0.00           C  \\nATOM   6292  CD1 TYR B 403      28.032  24.952  -1.113  1.00  0.00           C  \\nATOM   6293  CD2 TYR B 403      29.416  23.035  -0.711  1.00  0.00           C  \\nATOM   6294  CE1 TYR B 403      29.183  25.791  -1.105  1.00  0.00           C  \\nATOM   6295  CE2 TYR B 403      30.564  23.863  -0.695  1.00  0.00           C  \\nATOM   6296  CZ  TYR B 403      30.430  25.229  -0.883  1.00  0.00           C  \\nATOM   6297  OH  TYR B 403      31.520  26.088  -0.829  1.00  0.00           O  \\nATOM   6298  H   TYR B 403      26.825  21.875   1.587  1.00  0.00           H  \\nATOM   6299  HA  TYR B 403      25.789  24.208   0.043  1.00  0.00           H  \\nATOM   6300 1HB  TYR B 403      27.226  21.657  -0.760  1.00  0.00           H  \\nATOM   6301 2HB  TYR B 403      26.463  22.689  -1.962  1.00  0.00           H  \\nATOM   6302  HD1 TYR B 403      27.047  25.408  -1.212  1.00  0.00           H  \\nATOM   6303  HD2 TYR B 403      29.526  21.972  -0.498  1.00  0.00           H  \\nATOM   6304  HE1 TYR B 403      29.071  26.871  -1.197  1.00  0.00           H  \\nATOM   6305  HE2 TYR B 403      31.550  23.425  -0.536  1.00  0.00           H  \\nATOM   6306  HH  TYR B 403      32.316  25.582  -0.650  1.00  0.00           H  \\nATOM   6307  N   TYR B 404      23.871  21.746   0.563  1.00  0.00           N  \\nATOM   6308  CA  TYR B 404      22.461  21.426   0.309  1.00  0.00           C  \\nATOM   6309  C   TYR B 404      21.643  22.694   0.223  1.00  0.00           C  \\nATOM   6310  O   TYR B 404      20.713  22.777  -0.595  1.00  0.00           O  \\nATOM   6311  CB  TYR B 404      21.890  20.450   1.352  1.00  0.00           C  \\nATOM   6312  CG  TYR B 404      20.541  19.892   0.875  1.00  0.00           C  \\nATOM   6313  CD1 TYR B 404      20.513  18.913  -0.076  1.00  0.00           C  \\nATOM   6314  CD2 TYR B 404      19.348  20.383   1.390  1.00  0.00           C  \\nATOM   6315  CE1 TYR B 404      19.282  18.395  -0.565  1.00  0.00           C  \\nATOM   6316  CE2 TYR B 404      18.080  19.909   0.905  1.00  0.00           C  \\nATOM   6317  CZ  TYR B 404      18.075  18.888  -0.036  1.00  0.00           C  \\nATOM   6318  OH  TYR B 404      16.867  18.391  -0.523  1.00  0.00           O  \\nATOM   6319  H   TYR B 404      24.320  21.370   1.386  1.00  0.00           H  \\nATOM   6320  HA  TYR B 404      22.385  20.951  -0.669  1.00  0.00           H  \\nATOM   6321 1HB  TYR B 404      22.596  19.634   1.512  1.00  0.00           H  \\nATOM   6322 2HB  TYR B 404      21.765  20.966   2.303  1.00  0.00           H  \\nATOM   6323  HD1 TYR B 404      21.446  18.514  -0.475  1.00  0.00           H  \\nATOM   6324  HD2 TYR B 404      19.363  21.140   2.174  1.00  0.00           H  \\nATOM   6325  HE1 TYR B 404      19.283  17.614  -1.324  1.00  0.00           H  \\nATOM   6326  HE2 TYR B 404      17.155  20.351   1.276  1.00  0.00           H  \\nATOM   6327  HH  TYR B 404      16.139  18.860  -0.108  1.00  0.00           H  \\nATOM   6328  N   SER B 405      21.956  23.691   1.024  1.00  0.00           N  \\nATOM   6329  CA  SER B 405      21.147  24.842   1.055  1.00  0.00           C  \\nATOM   6330  C   SER B 405      21.233  25.644  -0.251  1.00  0.00           C  \\nATOM   6331  O   SER B 405      20.316  26.371  -0.542  1.00  0.00           O  \\nATOM   6332  CB  SER B 405      21.645  25.701   2.266  1.00  0.00           C  \\nATOM   6333  OG  SER B 405      23.006  26.136   2.058  1.00  0.00           O  \\nATOM   6334  H   SER B 405      22.770  23.646   1.620  1.00  0.00           H  \\nATOM   6335  HA  SER B 405      20.111  24.533   1.204  1.00  0.00           H  \\nATOM   6336 1HB  SER B 405      20.995  26.567   2.389  1.00  0.00           H  \\nATOM   6337 2HB  SER B 405      21.582  25.112   3.180  1.00  0.00           H  \\nATOM   6338  HG  SER B 405      23.271  25.780   1.207  1.00  0.00           H  \\nATOM   6339  N   THR B 406      22.304  25.473  -1.035  1.00  0.00           N  \\nATOM   6340  CA  THR B 406      22.506  26.254  -2.256  1.00  0.00           C  \\nATOM   6341  C   THR B 406      22.076  25.484  -3.462  1.00  0.00           C  \\nATOM   6342  O   THR B 406      21.980  26.049  -4.576  1.00  0.00           O  \\nATOM   6343  CB  THR B 406      24.022  26.569  -2.449  1.00  0.00           C  \\nATOM   6344  OG1 THR B 406      24.750  25.361  -2.758  1.00  0.00           O  \\nATOM   6345  CG2 THR B 406      24.647  27.255  -1.225  1.00  0.00           C  \\nATOM   6346  H   THR B 406      22.993  24.782  -0.773  1.00  0.00           H  \\nATOM   6347  HA  THR B 406      21.962  27.194  -2.160  1.00  0.00           H  \\nATOM   6348  HB  THR B 406      24.150  27.229  -3.307  1.00  0.00           H  \\nATOM   6349  HG1 THR B 406      24.142  24.618  -2.779  1.00  0.00           H  \\nATOM   6350 1HG2 THR B 406      25.702  27.450  -1.417  1.00  0.00           H  \\nATOM   6351 2HG2 THR B 406      24.132  28.196  -1.033  1.00  0.00           H  \\nATOM   6352 3HG2 THR B 406      24.551  26.606  -0.356  1.00  0.00           H  \\nATOM   6353  N   GLY B 407      21.726  24.229  -3.257  1.00  0.00           N  \\nATOM   6354  CA  GLY B 407      21.494  23.298  -4.416  1.00  0.00           C  \\nATOM   6355  C   GLY B 407      22.698  22.508  -4.909  1.00  0.00           C  \\nATOM   6356  O   GLY B 407      22.560  21.704  -5.843  1.00  0.00           O  \\nATOM   6357  H   GLY B 407      21.610  23.881  -2.316  1.00  0.00           H  \\nATOM   6358 1HA  GLY B 407      20.727  22.570  -4.150  1.00  0.00           H  \\nATOM   6359 2HA  GLY B 407      21.119  23.864  -5.268  1.00  0.00           H  \\nATOM   6360  N   GLY B 408      23.887  22.719  -4.330  1.00  0.00           N  \\nATOM   6361  CA  GLY B 408      25.105  22.054  -4.747  1.00  0.00           C  \\nATOM   6362  C   GLY B 408      25.270  20.608  -4.352  1.00  0.00           C  \\nATOM   6363  O   GLY B 408      26.119  19.885  -4.949  1.00  0.00           O  \\nATOM   6364  H   GLY B 408      23.923  23.377  -3.565  1.00  0.00           H  \\nATOM   6365 1HA  GLY B 408      25.185  22.090  -5.834  1.00  0.00           H  \\nATOM   6366 2HA  GLY B 408      25.966  22.586  -4.344  1.00  0.00           H  \\nATOM   6367  N   THR B 409      24.491  20.162  -3.378  1.00  0.00           N  \\nATOM   6368  CA  THR B 409      24.596  18.776  -2.874  1.00  0.00           C  \\nATOM   6369  C   THR B 409      23.318  18.034  -3.213  1.00  0.00           C  \\nATOM   6370  O   THR B 409      22.180  18.537  -2.973  1.00  0.00           O  \\nATOM   6371  CB  THR B 409      24.706  18.786  -1.338  1.00  0.00           C  \\nATOM   6372  OG1 THR B 409      25.879  19.494  -0.961  1.00  0.00           O  \\nATOM   6373  CG2 THR B 409      24.730  17.275  -0.742  1.00  0.00           C  \\nATOM   6374  H   THR B 409      23.808  20.784  -2.969  1.00  0.00           H  \\nATOM   6375  HA  THR B 409      25.496  18.324  -3.291  1.00  0.00           H  \\nATOM   6376  HB  THR B 409      23.853  19.317  -0.916  1.00  0.00           H  \\nATOM   6377  HG1 THR B 409      26.333  19.803  -1.749  1.00  0.00           H  \\nATOM   6378 1HG2 THR B 409      24.808  17.314   0.344  1.00  0.00           H  \\nATOM   6379 2HG2 THR B 409      23.812  16.759  -1.022  1.00  0.00           H  \\nATOM   6380 3HG2 THR B 409      25.586  16.738  -1.148  1.00  0.00           H  \\nATOM   6381  N   GLN B 410      23.519  16.884  -3.852  1.00  0.00           N  \\nATOM   6382  CA  GLN B 410      22.422  15.985  -4.259  1.00  0.00           C  \\nATOM   6383  C   GLN B 410      22.009  15.091  -3.127  1.00  0.00           C  \\nATOM   6384  O   GLN B 410      22.854  14.846  -2.253  1.00  0.00           O  \\nATOM   6385  CB  GLN B 410      22.850  15.135  -5.511  1.00  0.00           C  \\nATOM   6386  CG  GLN B 410      23.191  15.995  -6.825  1.00  0.00           C  \\nATOM   6387  CD  GLN B 410      22.069  16.924  -7.197  1.00  0.00           C  \\nATOM   6388  OE1 GLN B 410      21.011  16.486  -7.705  1.00  0.00           O  \\nATOM   6389  NE2 GLN B 410      22.273  18.229  -6.943  1.00  0.00           N  \\nATOM   6390  H   GLN B 410      24.472  16.624  -4.063  1.00  0.00           H  \\nATOM   6391  HA  GLN B 410      21.558  16.593  -4.527  1.00  0.00           H  \\nATOM   6392 1HB  GLN B 410      23.732  14.544  -5.265  1.00  0.00           H  \\nATOM   6393 2HB  GLN B 410      22.051  14.440  -5.769  1.00  0.00           H  \\nATOM   6394 1HG  GLN B 410      24.084  16.590  -6.635  1.00  0.00           H  \\nATOM   6395 2HG  GLN B 410      23.366  15.313  -7.657  1.00  0.00           H  \\nATOM   6396 1HE2 GLN B 410      21.565  18.900  -7.167  1.00  0.00           H  \\nATOM   6397 2HE2 GLN B 410      23.133  18.530  -6.530  1.00  0.00           H  \\nATOM   6398  N   LEU B 411      20.778  14.622  -3.117  1.00  0.00           N  \\nATOM   6399  CA  LEU B 411      20.330  13.774  -1.970  1.00  0.00           C  \\nATOM   6400  C   LEU B 411      21.218  12.563  -1.547  1.00  0.00           C  \\nATOM   6401  O   LEU B 411      21.456  12.364  -0.339  1.00  0.00           O  \\nATOM   6402  CB  LEU B 411      18.913  13.318  -2.240  1.00  0.00           C  \\nATOM   6403  CG  LEU B 411      18.271  12.591  -1.067  1.00  0.00           C  \\nATOM   6404  CD1 LEU B 411      18.192  13.440   0.179  1.00  0.00           C  \\nATOM   6405  CD2 LEU B 411      16.833  12.182  -1.528  1.00  0.00           C  \\nATOM   6406  H   LEU B 411      20.129  14.821  -3.865  1.00  0.00           H  \\nATOM   6407  HA  LEU B 411      20.354  14.378  -1.063  1.00  0.00           H  \\nATOM   6408 1HB  LEU B 411      18.308  14.190  -2.485  1.00  0.00           H  \\nATOM   6409 2HB  LEU B 411      18.920  12.653  -3.103  1.00  0.00           H  \\nATOM   6410  HG  LEU B 411      18.864  11.711  -0.816  1.00  0.00           H  \\nATOM   6411 1HD1 LEU B 411      17.725  12.867   0.981  1.00  0.00           H  \\nATOM   6412 2HD1 LEU B 411      19.197  13.735   0.483  1.00  0.00           H  \\nATOM   6413 3HD1 LEU B 411      17.597  14.330  -0.024  1.00  0.00           H  \\nATOM   6414 1HD2 LEU B 411      16.328  11.654  -0.719  1.00  0.00           H  \\nATOM   6415 2HD2 LEU B 411      16.266  13.076  -1.787  1.00  0.00           H  \\nATOM   6416 3HD2 LEU B 411      16.902  11.530  -2.399  1.00  0.00           H  \\nATOM   6417  N   PRO B 412      21.710  11.724  -2.458  1.00  0.00           N  \\nATOM   6418  CA  PRO B 412      22.556  10.622  -1.994  1.00  0.00           C  \\nATOM   6419  C   PRO B 412      23.779  11.066  -1.239  1.00  0.00           C  \\nATOM   6420  O   PRO B 412      24.062  10.398  -0.187  1.00  0.00           O  \\nATOM   6421  CB  PRO B 412      22.956   9.874  -3.245  1.00  0.00           C  \\nATOM   6422  CG  PRO B 412      21.733  10.241  -4.176  1.00  0.00           C  \\nATOM   6423  CD  PRO B 412      21.635  11.725  -3.957  1.00  0.00           C  \\nATOM   6424  HA  PRO B 412      21.967   9.966  -1.336  1.00  0.00           H  \\nATOM   6425 1HB  PRO B 412      23.934  10.231  -3.600  1.00  0.00           H  \\nATOM   6426 2HB  PRO B 412      23.066   8.802  -3.023  1.00  0.00           H  \\nATOM   6427 1HG  PRO B 412      21.947   9.952  -5.216  1.00  0.00           H  \\nATOM   6428 2HG  PRO B 412      20.839   9.680  -3.867  1.00  0.00           H  \\nATOM   6429 1HD  PRO B 412      22.484  12.225  -4.445  1.00  0.00           H  \\nATOM   6430 2HD  PRO B 412      20.682  12.094  -4.363  1.00  0.00           H  \\nATOM   6431  N   THR B 413      24.426  12.169  -1.646  1.00  0.00           N  \\nATOM   6432  CA  THR B 413      25.555  12.655  -0.902  1.00  0.00           C  \\nATOM   6433  C   THR B 413      25.121  13.237   0.429  1.00  0.00           C  \\nATOM   6434  O   THR B 413      25.867  13.087   1.416  1.00  0.00           O  \\nATOM   6435  CB  THR B 413      26.289  13.655  -1.784  1.00  0.00           C  \\nATOM   6436  OG1 THR B 413      26.812  12.925  -2.901  1.00  0.00           O  \\nATOM   6437  CG2 THR B 413      27.358  14.245  -1.075  1.00  0.00           C  \\nATOM   6438  H   THR B 413      24.130  12.667  -2.473  1.00  0.00           H  \\nATOM   6439  HA  THR B 413      26.206  11.812  -0.671  1.00  0.00           H  \\nATOM   6440  HB  THR B 413      25.597  14.434  -2.104  1.00  0.00           H  \\nATOM   6441  HG1 THR B 413      26.573  11.999  -2.817  1.00  0.00           H  \\nATOM   6442 1HG2 THR B 413      27.872  14.958  -1.719  1.00  0.00           H  \\nATOM   6443 2HG2 THR B 413      26.971  14.762  -0.197  1.00  0.00           H  \\nATOM   6444 3HG2 THR B 413      28.057  13.471  -0.760  1.00  0.00           H  \\nATOM   6445  N   LEU B 414      23.966  13.900   0.500  1.00  0.00           N  \\nATOM   6446  CA  LEU B 414      23.454  14.436   1.743  1.00  0.00           C  \\nATOM   6447  C   LEU B 414      23.217  13.286   2.734  1.00  0.00           C  \\nATOM   6448  O   LEU B 414      23.687  13.335   3.854  1.00  0.00           O  \\nATOM   6449  CB  LEU B 414      22.134  15.161   1.531  1.00  0.00           C  \\nATOM   6450  CG  LEU B 414      21.555  15.732   2.815  1.00  0.00           C  \\nATOM   6451  CD1 LEU B 414      22.453  16.848   3.258  1.00  0.00           C  \\nATOM   6452  CD2 LEU B 414      20.093  16.162   2.578  1.00  0.00           C  \\nATOM   6453  H   LEU B 414      23.433  14.030  -0.348  1.00  0.00           H  \\nATOM   6454  HA  LEU B 414      24.176  15.151   2.135  1.00  0.00           H  \\nATOM   6455 1HB  LEU B 414      22.293  15.971   0.821  1.00  0.00           H  \\nATOM   6456 2HB  LEU B 414      21.419  14.462   1.098  1.00  0.00           H  \\nATOM   6457  HG  LEU B 414      21.587  14.973   3.597  1.00  0.00           H  \\nATOM   6458 1HD1 LEU B 414      22.066  17.282   4.180  1.00  0.00           H  \\nATOM   6459 2HD1 LEU B 414      23.456  16.460   3.434  1.00  0.00           H  \\nATOM   6460 3HD1 LEU B 414      22.490  17.614   2.484  1.00  0.00           H  \\nATOM   6461 1HD2 LEU B 414      19.680  16.571   3.501  1.00  0.00           H  \\nATOM   6462 2HD2 LEU B 414      20.059  16.922   1.797  1.00  0.00           H  \\nATOM   6463 3HD2 LEU B 414      19.505  15.298   2.268  1.00  0.00           H  \\nATOM   6464  N   ALA B 415      22.595  12.214   2.250  1.00  0.00           N  \\nATOM   6465  CA  ALA B 415      22.281  11.075   3.126  1.00  0.00           C  \\nATOM   6466  C   ALA B 415      23.541  10.390   3.618  1.00  0.00           C  \\nATOM   6467  O   ALA B 415      23.654  10.062   4.826  1.00  0.00           O  \\nATOM   6468  CB  ALA B 415      21.281  10.096   2.433  1.00  0.00           C  \\nATOM   6469  H   ALA B 415      22.332  12.172   1.276  1.00  0.00           H  \\nATOM   6470  HA  ALA B 415      21.816  11.463   4.032  1.00  0.00           H  \\nATOM   6471 1HB  ALA B 415      21.065   9.263   3.102  1.00  0.00           H  \\nATOM   6472 2HB  ALA B 415      20.356  10.624   2.201  1.00  0.00           H  \\nATOM   6473 3HB  ALA B 415      21.722   9.717   1.512  1.00  0.00           H  \\nATOM   6474  N   ARG B 416      24.537  10.237   2.757  1.00  0.00           N  \\nATOM   6475  CA  ARG B 416      25.826   9.660   3.131  1.00  0.00           C  \\nATOM   6476  C   ARG B 416      26.465  10.555   4.222  1.00  0.00           C  \\nATOM   6477  O   ARG B 416      27.087  10.046   5.181  1.00  0.00           O  \\nATOM   6478  CB  ARG B 416      26.741   9.633   1.918  1.00  0.00           C  \\nATOM   6479  CG  ARG B 416      28.099   8.981   2.158  1.00  0.00           C  \\nATOM   6480  CD  ARG B 416      29.131   9.288   0.948  1.00  0.00           C  \\nATOM   6481  NE  ARG B 416      30.297   8.440   1.098  1.00  0.00           N  \\nATOM   6482  CZ  ARG B 416      30.392   7.208   0.600  1.00  0.00           C  \\nATOM   6483  NH1 ARG B 416      29.367   6.683  -0.069  1.00  0.00           N  \\nATOM   6484  NH2 ARG B 416      31.495   6.488   0.784  1.00  0.00           N  \\nATOM   6485  H   ARG B 416      24.390  10.535   1.803  1.00  0.00           H  \\nATOM   6486  HA  ARG B 416      25.661   8.639   3.476  1.00  0.00           H  \\nATOM   6487 1HB  ARG B 416      26.254   9.094   1.107  1.00  0.00           H  \\nATOM   6488 2HB  ARG B 416      26.919  10.652   1.574  1.00  0.00           H  \\nATOM   6489 1HG  ARG B 416      28.529   9.366   3.083  1.00  0.00           H  \\nATOM   6490 2HG  ARG B 416      27.975   7.900   2.238  1.00  0.00           H  \\nATOM   6491 1HD  ARG B 416      28.646   9.081  -0.005  1.00  0.00           H  \\nATOM   6492 2HD  ARG B 416      29.431  10.335   0.982  1.00  0.00           H  \\nATOM   6493  HE  ARG B 416      31.085   8.810   1.613  1.00  0.00           H  \\nATOM   6494 1HH1 ARG B 416      28.520   7.219  -0.198  1.00  0.00           H  \\nATOM   6495 2HH1 ARG B 416      29.436   5.750  -0.448  1.00  0.00           H  \\nATOM   6496 1HH2 ARG B 416      32.270   6.875   1.305  1.00  0.00           H  \\nATOM   6497 2HH2 ARG B 416      31.558   5.556   0.403  1.00  0.00           H  \\nATOM   6498  N   SER B 417      26.304  11.862   4.084  1.00  0.00           N  \\nATOM   6499  CA  SER B 417      26.929  12.851   4.948  1.00  0.00           C  \\nATOM   6500  C   SER B 417      26.325  12.788   6.300  1.00  0.00           C  \\nATOM   6501  O   SER B 417      27.019  12.903   7.317  1.00  0.00           O  \\nATOM   6502  CB  SER B 417      26.959  14.250   4.321  1.00  0.00           C  \\nATOM   6503  OG  SER B 417      27.738  14.172   3.135  1.00  0.00           O  \\nATOM   6504  H   SER B 417      25.709  12.174   3.330  1.00  0.00           H  \\nATOM   6505  HA  SER B 417      27.962  12.549   5.129  1.00  0.00           H  \\nATOM   6506 1HB  SER B 417      25.941  14.575   4.107  1.00  0.00           H  \\nATOM   6507 2HB  SER B 417      27.386  14.957   5.031  1.00  0.00           H  \\nATOM   6508  HG  SER B 417      28.024  13.258   3.066  1.00  0.00           H  \\nATOM   6509  N   PHE B 418      25.021  12.587   6.365  1.00  0.00           N  \\nATOM   6510  CA  PHE B 418      24.331  12.407   7.653  1.00  0.00           C  \\nATOM   6511  C   PHE B 418      24.824  11.152   8.300  1.00  0.00           C  \\nATOM   6512  O   PHE B 418      25.036  11.119   9.543  1.00  0.00           O  \\nATOM   6513  CB  PHE B 418      22.830  12.255   7.475  1.00  0.00           C  \\nATOM   6514  CG  PHE B 418      22.125  13.548   7.167  1.00  0.00           C  \\nATOM   6515  CD1 PHE B 418      22.413  14.711   7.874  1.00  0.00           C  \\nATOM   6516  CD2 PHE B 418      21.143  13.580   6.203  1.00  0.00           C  \\nATOM   6517  CE1 PHE B 418      21.651  15.902   7.592  1.00  0.00           C  \\nATOM   6518  CE2 PHE B 418      20.411  14.752   5.952  1.00  0.00           C  \\nATOM   6519  CZ  PHE B 418      20.693  15.882   6.636  1.00  0.00           C  \\nATOM   6520  H   PHE B 418      24.484  12.556   5.510  1.00  0.00           H  \\nATOM   6521  HA  PHE B 418      24.509  13.291   8.268  1.00  0.00           H  \\nATOM   6522 1HB  PHE B 418      22.629  11.555   6.665  1.00  0.00           H  \\nATOM   6523 2HB  PHE B 418      22.397  11.838   8.383  1.00  0.00           H  \\nATOM   6524  HD1 PHE B 418      23.206  14.697   8.622  1.00  0.00           H  \\nATOM   6525  HD2 PHE B 418      20.916  12.667   5.652  1.00  0.00           H  \\nATOM   6526  HE1 PHE B 418      21.847  16.818   8.149  1.00  0.00           H  \\nATOM   6527  HE2 PHE B 418      19.616  14.751   5.207  1.00  0.00           H  \\nATOM   6528  HZ  PHE B 418      20.132  16.787   6.406  1.00  0.00           H  \\nATOM   6529  N   ILE B 419      25.046  10.100   7.527  1.00  0.00           N  \\nATOM   6530  CA  ILE B 419      25.487   8.834   8.117  1.00  0.00           C  \\nATOM   6531  C   ILE B 419      26.863   8.987   8.743  1.00  0.00           C  \\nATOM   6532  O   ILE B 419      27.140   8.430   9.823  1.00  0.00           O  \\nATOM   6533  CB  ILE B 419      25.482   7.695   6.997  1.00  0.00           C  \\nATOM   6534  CG1 ILE B 419      24.040   7.252   6.834  1.00  0.00           C  \\nATOM   6535  CG2 ILE B 419      26.397   6.449   7.401  1.00  0.00           C  \\nATOM   6536  CD1 ILE B 419      23.904   6.385   5.477  1.00  0.00           C  \\nATOM   6537  H   ILE B 419      24.915  10.158   6.527  1.00  0.00           H  \\nATOM   6538  HA  ILE B 419      24.790   8.560   8.908  1.00  0.00           H  \\nATOM   6539  HB  ILE B 419      25.861   8.104   6.061  1.00  0.00           H  \\nATOM   6540 1HG1 ILE B 419      23.741   6.662   7.700  1.00  0.00           H  \\nATOM   6541 2HG1 ILE B 419      23.392   8.128   6.796  1.00  0.00           H  \\nATOM   6542 1HG2 ILE B 419      26.362   5.700   6.610  1.00  0.00           H  \\nATOM   6543 2HG2 ILE B 419      27.425   6.782   7.540  1.00  0.00           H  \\nATOM   6544 3HG2 ILE B 419      26.029   6.013   8.330  1.00  0.00           H  \\nATOM   6545 1HD1 ILE B 419      22.870   6.063   5.353  1.00  0.00           H  \\nATOM   6546 2HD1 ILE B 419      24.199   6.996   4.623  1.00  0.00           H  \\nATOM   6547 3HD1 ILE B 419      24.550   5.511   5.538  1.00  0.00           H  \\nATOM   6548  N   ILE B 420      27.740   9.748   8.086  1.00  0.00           N  \\nATOM   6549  CA  ILE B 420      29.141  10.017   8.585  1.00  0.00           C  \\nATOM   6550  C   ILE B 420      29.040  10.847   9.843  1.00  0.00           C  \\nATOM   6551  O   ILE B 420      29.604  10.466  10.868  1.00  0.00           O  \\nATOM   6552  CB  ILE B 420      29.983  10.758   7.548  1.00  0.00           C  \\nATOM   6553  CG1 ILE B 420      30.261   9.764   6.366  1.00  0.00           C  \\nATOM   6554  CG2 ILE B 420      31.286  11.330   8.185  1.00  0.00           C  \\nATOM   6555  CD1 ILE B 420      30.760  10.539   5.111  1.00  0.00           C  \\nATOM   6556  H   ILE B 420      27.441  10.159   7.213  1.00  0.00           H  \\nATOM   6557  HA  ILE B 420      29.624   9.062   8.790  1.00  0.00           H  \\nATOM   6558  HB  ILE B 420      29.406  11.585   7.135  1.00  0.00           H  \\nATOM   6559 1HG1 ILE B 420      31.007   9.032   6.674  1.00  0.00           H  \\nATOM   6560 2HG1 ILE B 420      29.348   9.218   6.127  1.00  0.00           H  \\nATOM   6561 1HG2 ILE B 420      31.865  11.851   7.423  1.00  0.00           H  \\nATOM   6562 2HG2 ILE B 420      31.026  12.025   8.982  1.00  0.00           H  \\nATOM   6563 3HG2 ILE B 420      31.879  10.513   8.596  1.00  0.00           H  \\nATOM   6564 1HD1 ILE B 420      30.948   9.835   4.300  1.00  0.00           H  \\nATOM   6565 2HD1 ILE B 420      30.000  11.257   4.800  1.00  0.00           H  \\nATOM   6566 3HD1 ILE B 420      31.681  11.067   5.353  1.00  0.00           H  \\nATOM   6567  N   CYS B 421      28.232  11.918   9.834  1.00  0.00           N  \\nATOM   6568  CA  CYS B 421      28.070  12.753  11.047  1.00  0.00           C  \\nATOM   6569  C   CYS B 421      27.471  12.087  12.236  1.00  0.00           C  \\nATOM   6570  O   CYS B 421      27.989  12.241  13.372  1.00  0.00           O  \\nATOM   6571  CB  CYS B 421      27.253  14.021  10.737  1.00  0.00           C  \\nATOM   6572  SG  CYS B 421      28.078  15.247   9.662  1.00  0.00           S  \\nATOM   6573  H   CYS B 421      27.725  12.164   8.996  1.00  0.00           H  \\nATOM   6574  HA  CYS B 421      29.058  13.056  11.393  1.00  0.00           H  \\nATOM   6575 1HB  CYS B 421      26.318  13.742  10.251  1.00  0.00           H  \\nATOM   6576 2HB  CYS B 421      26.999  14.527  11.668  1.00  0.00           H  \\nATOM   6577  HG  CYS B 421      27.078  16.123   9.657  1.00  0.00           H  \\nATOM   6578  N   ILE B 422      26.422  11.288  12.015  1.00  0.00           N  \\nATOM   6579  CA  ILE B 422      25.730  10.636  13.087  1.00  0.00           C  \\nATOM   6580  C   ILE B 422      26.732   9.651  13.722  1.00  0.00           C  \\nATOM   6581  O   ILE B 422      26.812   9.545  14.976  1.00  0.00           O  \\nATOM   6582  CB  ILE B 422      24.530   9.913  12.543  1.00  0.00           C  \\nATOM   6583  CG1 ILE B 422      23.468  10.918  12.254  1.00  0.00           C  \\nATOM   6584  CG2 ILE B 422      24.027   8.880  13.526  1.00  0.00           C  \\nATOM   6585  CD1 ILE B 422      22.393  10.334  11.362  1.00  0.00           C  \\nATOM   6586  H   ILE B 422      26.109  11.141  11.066  1.00  0.00           H  \\nATOM   6587  HA  ILE B 422      25.399  11.394  13.796  1.00  0.00           H  \\nATOM   6588  HB  ILE B 422      24.799   9.409  11.615  1.00  0.00           H  \\nATOM   6589 1HG1 ILE B 422      23.024  11.257  13.189  1.00  0.00           H  \\nATOM   6590 2HG1 ILE B 422      23.912  11.788  11.769  1.00  0.00           H  \\nATOM   6591 1HG2 ILE B 422      23.158   8.372  13.108  1.00  0.00           H  \\nATOM   6592 2HG2 ILE B 422      24.813   8.152  13.722  1.00  0.00           H  \\nATOM   6593 3HG2 ILE B 422      23.746   9.371  14.458  1.00  0.00           H  \\nATOM   6594 1HD1 ILE B 422      21.631  11.090  11.167  1.00  0.00           H  \\nATOM   6595 2HD1 ILE B 422      22.837  10.015  10.418  1.00  0.00           H  \\nATOM   6596 3HD1 ILE B 422      21.936   9.478  11.856  1.00  0.00           H  \\nATOM   6597  N   GLN B 423      27.520   8.909  12.952  1.00  0.00           N  \\nATOM   6598  CA  GLN B 423      28.418   7.950  13.596  1.00  0.00           C  \\nATOM   6599  C   GLN B 423      29.605   8.621  14.269  1.00  0.00           C  \\nATOM   6600  O   GLN B 423      29.934   8.203  15.387  1.00  0.00           O  \\nATOM   6601  CB  GLN B 423      28.917   6.905  12.626  1.00  0.00           C  \\nATOM   6602  CG  GLN B 423      27.753   5.991  12.062  1.00  0.00           C  \\nATOM   6603  CD  GLN B 423      28.360   5.046  11.121  1.00  0.00           C  \\nATOM   6604  OE1 GLN B 423      28.788   3.900  11.498  1.00  0.00           O  \\nATOM   6605  NE2 GLN B 423      28.438   5.469   9.890  1.00  0.00           N  \\nATOM   6606  H   GLN B 423      27.520   8.984  11.945  1.00  0.00           H  \\nATOM   6607  HA  GLN B 423      27.869   7.439  14.387  1.00  0.00           H  \\nATOM   6608 1HB  GLN B 423      29.416   7.394  11.789  1.00  0.00           H  \\nATOM   6609 2HB  GLN B 423      29.653   6.271  13.121  1.00  0.00           H  \\nATOM   6610 1HG  GLN B 423      27.284   5.464  12.893  1.00  0.00           H  \\nATOM   6611 2HG  GLN B 423      27.020   6.622  11.559  1.00  0.00           H  \\nATOM   6612 1HE2 GLN B 423      28.840   4.884   9.184  1.00  0.00           H  \\nATOM   6613 2HE2 GLN B 423      28.096   6.378   9.650  1.00  0.00           H  \\nATOM   6614  N   MET B 424      30.123   9.702  13.696  1.00  0.00           N  \\nATOM   6615  CA  MET B 424      31.301  10.290  14.290  1.00  0.00           C  \\nATOM   6616  C   MET B 424      30.997  11.195  15.437  1.00  0.00           C  \\nATOM   6617  O   MET B 424      31.906  11.531  16.174  1.00  0.00           O  \\nATOM   6618  CB  MET B 424      32.096  11.052  13.251  1.00  0.00           C  \\nATOM   6619  CG  MET B 424      32.639  10.080  12.129  1.00  0.00           C  \\nATOM   6620  SD  MET B 424      33.785  10.814  10.995  1.00  0.00           S  \\nATOM   6621  CE  MET B 424      35.240  11.072  12.009  1.00  0.00           C  \\nATOM   6622  H   MET B 424      29.727  10.124  12.868  1.00  0.00           H  \\nATOM   6623  HA  MET B 424      31.925   9.489  14.687  1.00  0.00           H  \\nATOM   6624 1HB  MET B 424      31.465  11.817  12.800  1.00  0.00           H  \\nATOM   6625 2HB  MET B 424      32.932  11.560  13.733  1.00  0.00           H  \\nATOM   6626 1HG  MET B 424      33.142   9.233  12.594  1.00  0.00           H  \\nATOM   6627 2HG  MET B 424      31.803   9.699  11.542  1.00  0.00           H  \\nATOM   6628 1HE  MET B 424      36.027  11.527  11.407  1.00  0.00           H  \\nATOM   6629 2HE  MET B 424      34.991  11.731  12.842  1.00  0.00           H  \\nATOM   6630 3HE  MET B 424      35.588  10.114  12.397  1.00  0.00           H  \\nATOM   6631  N   ILE B 425      29.753  11.549  15.602  1.00  0.00           N  \\nATOM   6632  CA  ILE B 425      29.346  12.438  16.680  1.00  0.00           C  \\nATOM   6633  C   ILE B 425      28.435  11.718  17.683  1.00  0.00           C  \\nATOM   6634  O   ILE B 425      28.848  11.483  18.823  1.00  0.00           O  \\nATOM   6635  CB  ILE B 425      28.733  13.748  16.142  1.00  0.00           C  \\nATOM   6636  CG1 ILE B 425      29.814  14.490  15.317  1.00  0.00           C  \\nATOM   6637  CG2 ILE B 425      28.300  14.613  17.324  1.00  0.00           C  \\nATOM   6638  CD1 ILE B 425      29.220  15.571  14.358  1.00  0.00           C  \\nATOM   6639  H   ILE B 425      29.053  11.197  14.964  1.00  0.00           H  \\nATOM   6640  HA  ILE B 425      30.227  12.695  17.267  1.00  0.00           H  \\nATOM   6641  HB  ILE B 425      27.869  13.517  15.519  1.00  0.00           H  \\nATOM   6642 1HG1 ILE B 425      30.519  14.974  15.993  1.00  0.00           H  \\nATOM   6643 2HG1 ILE B 425      30.376  13.770  14.721  1.00  0.00           H  \\nATOM   6644 1HG2 ILE B 425      27.866  15.542  16.955  1.00  0.00           H  \\nATOM   6645 2HG2 ILE B 425      27.559  14.076  17.915  1.00  0.00           H  \\nATOM   6646 3HG2 ILE B 425      29.166  14.839  17.946  1.00  0.00           H  \\nATOM   6647 1HD1 ILE B 425      30.029  16.054  13.809  1.00  0.00           H  \\nATOM   6648 2HD1 ILE B 425      28.536  15.096  13.654  1.00  0.00           H  \\nATOM   6649 3HD1 ILE B 425      28.682  16.317  14.940  1.00  0.00           H  \\nATOM   6650  N   SER B 426      27.223  11.313  17.258  1.00  0.00           N  \\nATOM   6651  CA  SER B 426      26.245  10.751  18.149  1.00  0.00           C  \\nATOM   6652  C   SER B 426      26.753   9.358  18.632  1.00  0.00           C  \\nATOM   6653  O   SER B 426      26.721   9.067  19.876  1.00  0.00           O  \\nATOM   6654  CB  SER B 426      24.972  10.463  17.434  1.00  0.00           C  \\nATOM   6655  OG  SER B 426      24.337  11.747  17.102  1.00  0.00           O  \\nATOM   6656  H   SER B 426      26.998  11.410  16.278  1.00  0.00           H  \\nATOM   6657  HA  SER B 426      26.039  11.475  18.938  1.00  0.00           H  \\nATOM   6658 1HB  SER B 426      25.183   9.884  16.535  1.00  0.00           H  \\nATOM   6659 2HB  SER B 426      24.327   9.858  18.069  1.00  0.00           H  \\nATOM   6660  HG  SER B 426      24.923  12.428  17.441  1.00  0.00           H  \\nATOM   6661  N   GLU B 427      27.210   8.490  17.685  1.00  0.00           N  \\nATOM   6662  CA  GLU B 427      27.574   7.156  18.172  1.00  0.00           C  \\nATOM   6663  C   GLU B 427      28.870   7.184  18.995  1.00  0.00           C  \\nATOM   6664  O   GLU B 427      29.043   6.408  19.932  1.00  0.00           O  \\nATOM   6665  CB  GLU B 427      27.761   6.141  17.033  1.00  0.00           C  \\nATOM   6666  CG  GLU B 427      26.462   6.049  16.166  1.00  0.00           C  \\nATOM   6667  CD  GLU B 427      25.290   5.286  16.773  1.00  0.00           C  \\nATOM   6668  OE1 GLU B 427      25.190   5.046  17.987  1.00  0.00           O  \\nATOM   6669  OE2 GLU B 427      24.343   4.987  15.981  1.00  0.00           O  \\nATOM   6670  H   GLU B 427      27.311   8.699  16.702  1.00  0.00           H  \\nATOM   6671  HA  GLU B 427      26.770   6.790  18.811  1.00  0.00           H  \\nATOM   6672 1HB  GLU B 427      28.603   6.444  16.411  1.00  0.00           H  \\nATOM   6673 2HB  GLU B 427      27.999   5.163  17.452  1.00  0.00           H  \\nATOM   6674 1HG  GLU B 427      26.108   7.057  15.953  1.00  0.00           H  \\nATOM   6675 2HG  GLU B 427      26.704   5.571  15.218  1.00  0.00           H  \\nATOM   6676  N   ALA B 428      29.784   8.049  18.599  1.00  0.00           N  \\nATOM   6677  CA  ALA B 428      30.999   8.196  19.367  1.00  0.00           C  \\nATOM   6678  C   ALA B 428      30.707   8.737  20.772  1.00  0.00           C  \\nATOM   6679  O   ALA B 428      31.365   8.299  21.710  1.00  0.00           O  \\nATOM   6680  CB  ALA B 428      31.965   9.166  18.654  1.00  0.00           C  \\nATOM   6681  H   ALA B 428      29.655   8.612  17.770  1.00  0.00           H  \\nATOM   6682  HA  ALA B 428      31.472   7.217  19.444  1.00  0.00           H  \\nATOM   6683 1HB  ALA B 428      32.877   9.269  19.242  1.00  0.00           H  \\nATOM   6684 2HB  ALA B 428      32.212   8.774  17.667  1.00  0.00           H  \\nATOM   6685 3HB  ALA B 428      31.490  10.140  18.549  1.00  0.00           H  \\nATOM   6686  N   ALA B 429      29.782   9.681  20.924  1.00  0.00           N  \\nATOM   6687  CA  ALA B 429      29.317  10.056  22.286  1.00  0.00           C  \\nATOM   6688  C   ALA B 429      28.734   8.891  23.024  1.00  0.00           C  \\nATOM   6689  O   ALA B 429      29.035   8.741  24.226  1.00  0.00           O  \\nATOM   6690  CB  ALA B 429      28.320  11.254  22.189  1.00  0.00           C  \\nATOM   6691  H   ALA B 429      29.387  10.151  20.122  1.00  0.00           H  \\nATOM   6692  HA  ALA B 429      30.187  10.359  22.869  1.00  0.00           H  \\nATOM   6693 1HB  ALA B 429      27.981  11.527  23.188  1.00  0.00           H  \\nATOM   6694 2HB  ALA B 429      28.820  12.107  21.730  1.00  0.00           H  \\nATOM   6695 3HB  ALA B 429      27.464  10.966  21.582  1.00  0.00           H  \\nATOM   6696  N   ARG B 430      27.950   8.051  22.342  1.00  0.00           N  \\nATOM   6697  CA  ARG B 430      27.279   6.964  23.003  1.00  0.00           C  \\nATOM   6698  C   ARG B 430      28.315   5.916  23.464  1.00  0.00           C  \\nATOM   6699  O   ARG B 430      27.990   5.195  24.411  1.00  0.00           O  \\nATOM   6700  CB  ARG B 430      26.231   6.311  22.040  1.00  0.00           C  \\nATOM   6701  CG  ARG B 430      25.005   7.199  21.906  1.00  0.00           C  \\nATOM   6702  CD  ARG B 430      24.243   6.469  20.713  1.00  0.00           C  \\nATOM   6703  NE  ARG B 430      22.898   7.095  20.537  1.00  0.00           N  \\nATOM   6704  CZ  ARG B 430      22.055   6.563  19.640  1.00  0.00           C  \\nATOM   6705  NH1 ARG B 430      22.455   5.479  18.893  1.00  0.00           N  \\nATOM   6706  NH2 ARG B 430      20.834   7.082  19.578  1.00  0.00           N  \\nATOM   6707  H   ARG B 430      27.823   8.176  21.348  1.00  0.00           H  \\nATOM   6708  HA  ARG B 430      26.757   7.359  23.875  1.00  0.00           H  \\nATOM   6709 1HB  ARG B 430      26.684   6.154  21.062  1.00  0.00           H  \\nATOM   6710 2HB  ARG B 430      25.941   5.334  22.427  1.00  0.00           H  \\nATOM   6711 1HG  ARG B 430      24.451   7.199  22.845  1.00  0.00           H  \\nATOM   6712 2HG  ARG B 430      25.317   8.217  21.669  1.00  0.00           H  \\nATOM   6713 1HD  ARG B 430      24.819   6.575  19.794  1.00  0.00           H  \\nATOM   6714 2HD  ARG B 430      24.127   5.412  20.949  1.00  0.00           H  \\nATOM   6715  HE  ARG B 430      22.676   7.901  21.106  1.00  0.00           H  \\nATOM   6716 1HH1 ARG B 430      23.378   5.089  19.022  1.00  0.00           H  \\nATOM   6717 2HH1 ARG B 430      21.826   5.073  18.216  1.00  0.00           H  \\nATOM   6718 1HH2 ARG B 430      20.577   7.844  20.191  1.00  0.00           H  \\nATOM   6719 2HH2 ARG B 430      20.163   6.716  18.919  1.00  0.00           H  \\nATOM   6720  N   PHE B 431      29.410   5.660  22.737  1.00  0.00           N  \\nATOM   6721  CA  PHE B 431      30.282   4.483  22.987  1.00  0.00           C  \\nATOM   6722  C   PHE B 431      31.706   4.905  23.004  1.00  0.00           C  \\nATOM   6723  O   PHE B 431      32.216   5.401  22.014  1.00  0.00           O  \\nATOM   6724  CB  PHE B 431      30.133   3.503  21.840  1.00  0.00           C  \\nATOM   6725  CG  PHE B 431      28.787   2.800  21.777  1.00  0.00           C  \\nATOM   6726  CD1 PHE B 431      28.554   1.687  22.555  1.00  0.00           C  \\nATOM   6727  CD2 PHE B 431      27.835   3.209  20.883  1.00  0.00           C  \\nATOM   6728  CE1 PHE B 431      27.375   0.983  22.431  1.00  0.00           C  \\nATOM   6729  CE2 PHE B 431      26.637   2.520  20.751  1.00  0.00           C  \\nATOM   6730  CZ  PHE B 431      26.397   1.405  21.501  1.00  0.00           C  \\nATOM   6731  H   PHE B 431      29.646   6.297  21.990  1.00  0.00           H  \\nATOM   6732  HA  PHE B 431      29.961   4.005  23.914  1.00  0.00           H  \\nATOM   6733 1HB  PHE B 431      30.279   4.024  20.895  1.00  0.00           H  \\nATOM   6734 2HB  PHE B 431      30.904   2.737  21.914  1.00  0.00           H  \\nATOM   6735  HD1 PHE B 431      29.313   1.369  23.271  1.00  0.00           H  \\nATOM   6736  HD2 PHE B 431      28.021   4.099  20.281  1.00  0.00           H  \\nATOM   6737  HE1 PHE B 431      27.199   0.103  23.050  1.00  0.00           H  \\nATOM   6738  HE2 PHE B 431      25.886   2.872  20.044  1.00  0.00           H  \\nATOM   6739  HZ  PHE B 431      25.471   0.843  21.390  1.00  0.00           H  \\nATOM   6740  N   GLN B 432      32.404   4.736  24.146  1.00  0.00           N  \\nATOM   6741  CA  GLN B 432      33.896   4.970  24.125  1.00  0.00           C  \\nATOM   6742  C   GLN B 432      34.541   4.101  23.085  1.00  0.00           C  \\nATOM   6743  O   GLN B 432      35.545   4.537  22.462  1.00  0.00           O  \\nATOM   6744  CB  GLN B 432      34.536   4.530  25.472  1.00  0.00           C  \\nATOM   6745  CG  GLN B 432      34.096   5.401  26.643  1.00  0.00           C  \\nATOM   6746  CD  GLN B 432      33.031   4.680  27.394  1.00  0.00           C  \\nATOM   6747  OE1 GLN B 432      32.205   3.962  26.769  1.00  0.00           O  \\nATOM   6748  NE2 GLN B 432      33.084   4.731  28.711  1.00  0.00           N  \\nATOM   6749  H   GLN B 432      31.962   4.459  25.011  1.00  0.00           H  \\nATOM   6750  HA  GLN B 432      34.077   6.036  23.985  1.00  0.00           H  \\nATOM   6751 1HB  GLN B 432      34.265   3.495  25.683  1.00  0.00           H  \\nATOM   6752 2HB  GLN B 432      35.622   4.574  25.390  1.00  0.00           H  \\nATOM   6753 1HG  GLN B 432      34.955   5.584  27.289  1.00  0.00           H  \\nATOM   6754 2HG  GLN B 432      33.712   6.345  26.256  1.00  0.00           H  \\nATOM   6755 1HE2 GLN B 432      32.390   4.262  29.259  1.00  0.00           H  \\nATOM   6756 2HE2 GLN B 432      33.818   5.238  29.163  1.00  0.00           H  \\nATOM   6757  N   TYR B 433      33.967   2.907  22.796  1.00  0.00           N  \\nATOM   6758  CA  TYR B 433      34.542   2.040  21.712  1.00  0.00           C  \\nATOM   6759  C   TYR B 433      34.514   2.762  20.364  1.00  0.00           C  \\nATOM   6760  O   TYR B 433      35.455   2.667  19.609  1.00  0.00           O  \\nATOM   6761  CB  TYR B 433      33.774   0.696  21.637  1.00  0.00           C  \\nATOM   6762  CG  TYR B 433      34.296  -0.257  20.647  1.00  0.00           C  \\nATOM   6763  CD1 TYR B 433      35.362  -1.086  21.008  1.00  0.00           C  \\nATOM   6764  CD2 TYR B 433      33.661  -0.469  19.447  1.00  0.00           C  \\nATOM   6765  CE1 TYR B 433      35.776  -2.055  20.145  1.00  0.00           C  \\nATOM   6766  CE2 TYR B 433      34.116  -1.371  18.542  1.00  0.00           C  \\nATOM   6767  CZ  TYR B 433      35.173  -2.190  18.894  1.00  0.00           C  \\nATOM   6768  OH  TYR B 433      35.638  -3.217  18.100  1.00  0.00           O  \\nATOM   6769  H   TYR B 433      33.153   2.588  23.301  1.00  0.00           H  \\nATOM   6770  HA  TYR B 433      35.586   1.835  21.950  1.00  0.00           H  \\nATOM   6771 1HB  TYR B 433      33.799   0.207  22.612  1.00  0.00           H  \\nATOM   6772 2HB  TYR B 433      32.729   0.887  21.393  1.00  0.00           H  \\nATOM   6773  HD1 TYR B 433      35.853  -0.954  21.972  1.00  0.00           H  \\nATOM   6774  HD2 TYR B 433      32.763   0.097  19.198  1.00  0.00           H  \\nATOM   6775  HE1 TYR B 433      36.606  -2.707  20.416  1.00  0.00           H  \\nATOM   6776  HE2 TYR B 433      33.650  -1.439  17.559  1.00  0.00           H  \\nATOM   6777  HH  TYR B 433      35.117  -3.258  17.294  1.00  0.00           H  \\nATOM   6778  N   ILE B 434      33.386   3.438  20.045  1.00  0.00           N  \\nATOM   6779  CA  ILE B 434      33.268   4.102  18.776  1.00  0.00           C  \\nATOM   6780  C   ILE B 434      34.111   5.395  18.740  1.00  0.00           C  \\nATOM   6781  O   ILE B 434      34.733   5.692  17.713  1.00  0.00           O  \\nATOM   6782  CB  ILE B 434      31.778   4.249  18.351  1.00  0.00           C  \\nATOM   6783  CG1 ILE B 434      31.235   2.800  18.187  1.00  0.00           C  \\nATOM   6784  CG2 ILE B 434      31.701   5.027  17.000  1.00  0.00           C  \\nATOM   6785  CD1 ILE B 434      29.790   2.759  17.666  1.00  0.00           C  \\nATOM   6786  H   ILE B 434      32.616   3.482  20.697  1.00  0.00           H  \\nATOM   6787  HA  ILE B 434      33.780   3.503  18.024  1.00  0.00           H  \\nATOM   6788  HB  ILE B 434      31.233   4.797  19.119  1.00  0.00           H  \\nATOM   6789 1HG1 ILE B 434      31.872   2.249  17.496  1.00  0.00           H  \\nATOM   6790 2HG1 ILE B 434      31.274   2.286  19.148  1.00  0.00           H  \\nATOM   6791 1HG2 ILE B 434      30.659   5.131  16.699  1.00  0.00           H  \\nATOM   6792 2HG2 ILE B 434      32.143   6.015  17.123  1.00  0.00           H  \\nATOM   6793 3HG2 ILE B 434      32.247   4.478  16.232  1.00  0.00           H  \\nATOM   6794 1HD1 ILE B 434      29.466   1.722  17.572  1.00  0.00           H  \\nATOM   6795 2HD1 ILE B 434      29.136   3.282  18.365  1.00  0.00           H  \\nATOM   6796 3HD1 ILE B 434      29.741   3.243  16.692  1.00  0.00           H  \\nATOM   6797  N   GLU B 435      34.189   6.161  19.847  1.00  0.00           N  \\nATOM   6798  CA  GLU B 435      35.130   7.278  19.949  1.00  0.00           C  \\nATOM   6799  C   GLU B 435      36.558   6.771  19.658  1.00  0.00           C  \\nATOM   6800  O   GLU B 435      37.296   7.386  18.882  1.00  0.00           O  \\nATOM   6801  CB  GLU B 435      35.028   7.848  21.405  1.00  0.00           C  \\nATOM   6802  CG  GLU B 435      36.040   8.988  21.654  1.00  0.00           C  \\nATOM   6803  CD  GLU B 435      36.266   9.134  23.114  1.00  0.00           C  \\nATOM   6804  OE1 GLU B 435      35.521   8.526  23.943  1.00  0.00           O  \\nATOM   6805  OE2 GLU B 435      37.291   9.808  23.394  1.00  0.00           O  \\nATOM   6806  H   GLU B 435      33.581   5.957  20.627  1.00  0.00           H  \\nATOM   6807  HA  GLU B 435      34.840   8.042  19.227  1.00  0.00           H  \\nATOM   6808 1HB  GLU B 435      34.019   8.222  21.579  1.00  0.00           H  \\nATOM   6809 2HB  GLU B 435      35.209   7.048  22.123  1.00  0.00           H  \\nATOM   6810 1HG  GLU B 435      36.972   8.750  21.143  1.00  0.00           H  \\nATOM   6811 2HG  GLU B 435      35.647   9.909  21.225  1.00  0.00           H  \\nATOM   6812  N   GLY B 436      36.837   5.582  20.198  1.00  0.00           N  \\nATOM   6813  CA  GLY B 436      38.115   4.929  19.977  1.00  0.00           C  \\nATOM   6814  C   GLY B 436      38.405   4.659  18.521  1.00  0.00           C  \\nATOM   6815  O   GLY B 436      39.562   4.877  18.074  1.00  0.00           O  \\nATOM   6816  H   GLY B 436      36.144   5.125  20.774  1.00  0.00           H  \\nATOM   6817 1HA  GLY B 436      38.915   5.549  20.381  1.00  0.00           H  \\nATOM   6818 2HA  GLY B 436      38.138   3.983  20.517  1.00  0.00           H  \\nATOM   6819  N   GLU B 437      37.405   4.161  17.769  1.00  0.00           N  \\nATOM   6820  CA  GLU B 437      37.555   3.856  16.353  1.00  0.00           C  \\nATOM   6821  C   GLU B 437      37.898   5.127  15.625  1.00  0.00           C  \\nATOM   6822  O   GLU B 437      38.667   5.185  14.645  1.00  0.00           O  \\nATOM   6823  CB  GLU B 437      36.225   3.302  15.830  1.00  0.00           C  \\nATOM   6824  CG  GLU B 437      35.935   1.912  16.266  1.00  0.00           C  \\nATOM   6825  CD  GLU B 437      36.929   0.923  15.641  1.00  0.00           C  \\nATOM   6826  OE1 GLU B 437      36.971   0.858  14.362  1.00  0.00           O  \\nATOM   6827  OE2 GLU B 437      37.708   0.375  16.436  1.00  0.00           O  \\nATOM   6828  H   GLU B 437      36.515   3.995  18.216  1.00  0.00           H  \\nATOM   6829  HA  GLU B 437      38.334   3.101  16.242  1.00  0.00           H  \\nATOM   6830 1HB  GLU B 437      35.407   3.939  16.167  1.00  0.00           H  \\nATOM   6831 2HB  GLU B 437      36.226   3.321  14.740  1.00  0.00           H  \\nATOM   6832 1HG  GLU B 437      35.998   1.863  17.353  1.00  0.00           H  \\nATOM   6833 2HG  GLU B 437      34.917   1.657  15.976  1.00  0.00           H  \\nATOM   6834  N   MET B 438      37.282   6.272  16.041  1.00  0.00           N  \\nATOM   6835  CA  MET B 438      37.562   7.507  15.370  1.00  0.00           C  \\nATOM   6836  C   MET B 438      38.907   8.101  15.697  1.00  0.00           C  \\nATOM   6837  O   MET B 438      39.534   8.662  14.789  1.00  0.00           O  \\nATOM   6838  CB  MET B 438      36.429   8.554  15.571  1.00  0.00           C  \\nATOM   6839  CG  MET B 438      34.990   8.030  15.197  1.00  0.00           C  \\nATOM   6840  SD  MET B 438      34.950   7.360  13.469  1.00  0.00           S  \\nATOM   6841  CE  MET B 438      33.699   6.209  13.960  1.00  0.00           C  \\nATOM   6842  H   MET B 438      36.631   6.268  16.813  1.00  0.00           H  \\nATOM   6843  HA  MET B 438      37.647   7.309  14.302  1.00  0.00           H  \\nATOM   6844 1HB  MET B 438      36.410   8.874  16.612  1.00  0.00           H  \\nATOM   6845 2HB  MET B 438      36.635   9.435  14.962  1.00  0.00           H  \\nATOM   6846 1HG  MET B 438      34.697   7.245  15.893  1.00  0.00           H  \\nATOM   6847 2HG  MET B 438      34.272   8.846  15.282  1.00  0.00           H  \\nATOM   6848 1HE  MET B 438      33.413   5.595  13.105  1.00  0.00           H  \\nATOM   6849 2HE  MET B 438      34.085   5.569  14.755  1.00  0.00           H  \\nATOM   6850 3HE  MET B 438      32.826   6.753  14.324  1.00  0.00           H  \\nATOM   6851  N   ARG B 439      39.312   7.896  16.932  1.00  0.00           N  \\nATOM   6852  CA  ARG B 439      40.654   8.353  17.390  1.00  0.00           C  \\nATOM   6853  C   ARG B 439      41.689   7.614  16.544  1.00  0.00           C  \\nATOM   6854  O   ARG B 439      42.655   8.199  16.098  1.00  0.00           O  \\nATOM   6855  CB  ARG B 439      40.908   8.054  18.873  1.00  0.00           C  \\nATOM   6856  CG  ARG B 439      40.104   8.802  19.897  1.00  0.00           C  \\nATOM   6857  CD  ARG B 439      40.446   8.126  21.282  1.00  0.00           C  \\nATOM   6858  NE  ARG B 439      41.684   8.645  21.795  1.00  0.00           N  \\nATOM   6859  CZ  ARG B 439      41.715   9.362  22.923  1.00  0.00           C  \\nATOM   6860  NH1 ARG B 439      40.586   9.517  23.627  1.00  0.00           N  \\nATOM   6861  NH2 ARG B 439      42.818   9.910  23.383  1.00  0.00           N  \\nATOM   6862  H   ARG B 439      38.705   7.421  17.584  1.00  0.00           H  \\nATOM   6863  HA  ARG B 439      40.715   9.434  17.257  1.00  0.00           H  \\nATOM   6864 1HB  ARG B 439      40.726   6.998  19.067  1.00  0.00           H  \\nATOM   6865 2HB  ARG B 439      41.952   8.255  19.111  1.00  0.00           H  \\nATOM   6866 1HG  ARG B 439      40.390   9.854  19.887  1.00  0.00           H  \\nATOM   6867 2HG  ARG B 439      39.043   8.713  19.663  1.00  0.00           H  \\nATOM   6868 1HD  ARG B 439      39.649   8.335  21.995  1.00  0.00           H  \\nATOM   6869 2HD  ARG B 439      40.539   7.049  21.148  1.00  0.00           H  \\nATOM   6870  HE  ARG B 439      42.528   8.449  21.274  1.00  0.00           H  \\nATOM   6871 1HH1 ARG B 439      39.727   9.095  23.304  1.00  0.00           H  \\nATOM   6872 2HH1 ARG B 439      40.593  10.056  24.481  1.00  0.00           H  \\nATOM   6873 1HH2 ARG B 439      43.687   9.797  22.879  1.00  0.00           H  \\nATOM   6874 2HH2 ARG B 439      42.797  10.444  24.239  1.00  0.00           H  \\nATOM   6875  N   THR B 440      41.461   6.319  16.263  1.00  0.00           N  \\nATOM   6876  CA  THR B 440      42.456   5.519  15.578  1.00  0.00           C  \\nATOM   6877  C   THR B 440      42.548   6.007  14.123  1.00  0.00           C  \\nATOM   6878  O   THR B 440      43.598   6.119  13.520  1.00  0.00           O  \\nATOM   6879  CB  THR B 440      42.146   4.032  15.664  1.00  0.00           C  \\nATOM   6880  OG1 THR B 440      42.146   3.604  16.986  1.00  0.00           O  \\nATOM   6881  CG2 THR B 440      43.256   3.157  14.996  1.00  0.00           C  \\nATOM   6882  H   THR B 440      40.586   5.892  16.531  1.00  0.00           H  \\nATOM   6883  HA  THR B 440      43.423   5.689  16.052  1.00  0.00           H  \\nATOM   6884  HB  THR B 440      41.201   3.828  15.161  1.00  0.00           H  \\nATOM   6885  HG1 THR B 440      42.343   4.347  17.562  1.00  0.00           H  \\nATOM   6886 1HG2 THR B 440      42.990   2.104  15.082  1.00  0.00           H  \\nATOM   6887 2HG2 THR B 440      43.345   3.424  13.943  1.00  0.00           H  \\nATOM   6888 3HG2 THR B 440      44.208   3.332  15.496  1.00  0.00           H  \\nATOM   6889  N   ARG B 441      41.416   6.306  13.550  1.00  0.00           N  \\nATOM   6890  CA  ARG B 441      41.412   6.850  12.178  1.00  0.00           C  \\nATOM   6891  C   ARG B 441      42.153   8.158  12.061  1.00  0.00           C  \\nATOM   6892  O   ARG B 441      42.862   8.376  11.086  1.00  0.00           O  \\nATOM   6893  CB  ARG B 441      39.997   6.872  11.597  1.00  0.00           C  \\nATOM   6894  CG  ARG B 441      39.465   5.431  11.317  1.00  0.00           C  \\nATOM   6895  CD  ARG B 441      38.003   5.591  11.235  1.00  0.00           C  \\nATOM   6896  NE  ARG B 441      37.309   4.427  10.707  1.00  0.00           N  \\nATOM   6897  CZ  ARG B 441      37.100   3.300  11.364  1.00  0.00           C  \\nATOM   6898  NH1 ARG B 441      37.551   3.209  12.640  1.00  0.00           N  \\nATOM   6899  NH2 ARG B 441      36.446   2.281  10.824  1.00  0.00           N  \\nATOM   6900  H   ARG B 441      40.537   6.172  14.029  1.00  0.00           H  \\nATOM   6901  HA  ARG B 441      42.031   6.208  11.550  1.00  0.00           H  \\nATOM   6902 1HB  ARG B 441      39.326   7.374  12.293  1.00  0.00           H  \\nATOM   6903 2HB  ARG B 441      39.994   7.444  10.669  1.00  0.00           H  \\nATOM   6904 1HG  ARG B 441      39.888   5.062  10.382  1.00  0.00           H  \\nATOM   6905 2HG  ARG B 441      39.758   4.770  12.133  1.00  0.00           H  \\nATOM   6906 1HD  ARG B 441      37.601   5.780  12.230  1.00  0.00           H  \\nATOM   6907 2HD  ARG B 441      37.765   6.430  10.582  1.00  0.00           H  \\nATOM   6908  HE  ARG B 441      36.956   4.479   9.761  1.00  0.00           H  \\nATOM   6909 1HH1 ARG B 441      38.032   3.990  13.063  1.00  0.00           H  \\nATOM   6910 2HH1 ARG B 441      37.406   2.360  13.167  1.00  0.00           H  \\nATOM   6911 1HH2 ARG B 441      36.089   2.352   9.881  1.00  0.00           H  \\nATOM   6912 2HH2 ARG B 441      36.304   1.434  11.355  1.00  0.00           H  \\nATOM   6913  N   ILE B 442      42.042   9.030  13.098  1.00  0.00           N  \\nATOM   6914  CA  ILE B 442      42.748  10.264  13.086  1.00  0.00           C  \\nATOM   6915  C   ILE B 442      44.237  10.019  13.274  1.00  0.00           C  \\nATOM   6916  O   ILE B 442      45.076  10.573  12.587  1.00  0.00           O  \\nATOM   6917  CB  ILE B 442      42.177  11.228  14.134  1.00  0.00           C  \\nATOM   6918  CG1 ILE B 442      40.727  11.566  13.773  1.00  0.00           C  \\nATOM   6919  CG2 ILE B 442      43.110  12.569  14.326  1.00  0.00           C  \\nATOM   6920  CD1 ILE B 442      40.015  12.221  14.941  1.00  0.00           C  \\nATOM   6921  H   ILE B 442      41.454   8.807  13.889  1.00  0.00           H  \\nATOM   6922  HA  ILE B 442      42.635  10.717  12.102  1.00  0.00           H  \\nATOM   6923  HB  ILE B 442      42.113  10.722  15.097  1.00  0.00           H  \\nATOM   6924 1HG1 ILE B 442      40.713  12.233  12.912  1.00  0.00           H  \\nATOM   6925 2HG1 ILE B 442      40.201  10.655  13.488  1.00  0.00           H  \\nATOM   6926 1HG2 ILE B 442      42.662  13.219  15.078  1.00  0.00           H  \\nATOM   6927 2HG2 ILE B 442      44.108  12.275  14.649  1.00  0.00           H  \\nATOM   6928 3HG2 ILE B 442      43.178  13.104  13.378  1.00  0.00           H  \\nATOM   6929 1HD1 ILE B 442      38.987  12.450  14.658  1.00  0.00           H  \\nATOM   6930 2HD1 ILE B 442      40.014  11.541  15.794  1.00  0.00           H  \\nATOM   6931 3HD1 ILE B 442      40.530  13.142  15.211  1.00  0.00           H  \\nATOM   6932  N   ARG B 443      44.534   9.059  14.154  1.00  0.00           N  \\nATOM   6933  CA  ARG B 443      45.944   8.819  14.517  1.00  0.00           C  \\nATOM   6934  C   ARG B 443      46.772   8.360  13.335  1.00  0.00           C  \\nATOM   6935  O   ARG B 443      47.887   8.802  13.211  1.00  0.00           O  \\nATOM   6936  CB  ARG B 443      45.936   7.788  15.649  1.00  0.00           C  \\nATOM   6937  CG  ARG B 443      47.300   7.369  16.118  1.00  0.00           C  \\nATOM   6938  CD  ARG B 443      47.132   6.432  17.266  1.00  0.00           C  \\nATOM   6939  NE  ARG B 443      46.923   7.161  18.510  1.00  0.00           N  \\nATOM   6940  CZ  ARG B 443      45.933   6.929  19.387  1.00  0.00           C  \\nATOM   6941  NH1 ARG B 443      45.008   5.997  19.259  1.00  0.00           N  \\nATOM   6942  NH2 ARG B 443      45.963   7.646  20.506  1.00  0.00           N  \\nATOM   6943  H   ARG B 443      43.814   8.492  14.578  1.00  0.00           H  \\nATOM   6944  HA  ARG B 443      46.378   9.759  14.860  1.00  0.00           H  \\nATOM   6945 1HB  ARG B 443      45.398   8.193  16.505  1.00  0.00           H  \\nATOM   6946 2HB  ARG B 443      45.406   6.893  15.323  1.00  0.00           H  \\nATOM   6947 1HG  ARG B 443      47.828   6.872  15.304  1.00  0.00           H  \\nATOM   6948 2HG  ARG B 443      47.863   8.249  16.432  1.00  0.00           H  \\nATOM   6949 1HD  ARG B 443      46.269   5.790  17.089  1.00  0.00           H  \\nATOM   6950 2HD  ARG B 443      48.026   5.818  17.368  1.00  0.00           H  \\nATOM   6951  HE  ARG B 443      47.573   7.902  18.735  1.00  0.00           H  \\nATOM   6952 1HH1 ARG B 443      45.011   5.390  18.451  1.00  0.00           H  \\nATOM   6953 2HH1 ARG B 443      44.297   5.890  19.968  1.00  0.00           H  \\nATOM   6954 1HH2 ARG B 443      46.703   8.318  20.656  1.00  0.00           H  \\nATOM   6955 2HH2 ARG B 443      45.247   7.519  21.206  1.00  0.00           H  \\nATOM   6956  N   TYR B 444      46.217   7.475  12.515  1.00  0.00           N  \\nATOM   6957  CA  TYR B 444      46.847   6.892  11.283  1.00  0.00           C  \\nATOM   6958  C   TYR B 444      46.436   7.597   9.989  1.00  0.00           C  \\nATOM   6959  O   TYR B 444      46.716   7.080   8.880  1.00  0.00           O  \\nATOM   6960  CB  TYR B 444      46.595   5.365  11.215  1.00  0.00           C  \\nATOM   6961  CG  TYR B 444      47.210   4.697  12.415  1.00  0.00           C  \\nATOM   6962  CD1 TYR B 444      48.594   4.498  12.514  1.00  0.00           C  \\nATOM   6963  CD2 TYR B 444      46.428   4.231  13.408  1.00  0.00           C  \\nATOM   6964  CE1 TYR B 444      49.163   3.915  13.645  1.00  0.00           C  \\nATOM   6965  CE2 TYR B 444      46.995   3.672  14.631  1.00  0.00           C  \\nATOM   6966  CZ  TYR B 444      48.372   3.500  14.695  1.00  0.00           C  \\nATOM   6967  OH  TYR B 444      48.963   2.872  15.770  1.00  0.00           O  \\nATOM   6968  H   TYR B 444      45.284   7.188  12.773  1.00  0.00           H  \\nATOM   6969  HA  TYR B 444      47.922   7.066  11.329  1.00  0.00           H  \\nATOM   6970 1HB  TYR B 444      45.521   5.174  11.183  1.00  0.00           H  \\nATOM   6971 2HB  TYR B 444      47.025   4.966  10.297  1.00  0.00           H  \\nATOM   6972  HD1 TYR B 444      49.248   4.801  11.696  1.00  0.00           H  \\nATOM   6973  HD2 TYR B 444      45.343   4.265  13.309  1.00  0.00           H  \\nATOM   6974  HE1 TYR B 444      50.242   3.770  13.699  1.00  0.00           H  \\nATOM   6975  HE2 TYR B 444      46.342   3.402  15.461  1.00  0.00           H  \\nATOM   6976  HH  TYR B 444      48.285   2.608  16.397  1.00  0.00           H  \\nATOM   6977  N   ASN B 445      45.759   8.757  10.120  1.00  0.00           N  \\nATOM   6978  CA  ASN B 445      45.331   9.591   9.020  1.00  0.00           C  \\nATOM   6979  C   ASN B 445      44.669   8.687   7.977  1.00  0.00           C  \\nATOM   6980  O   ASN B 445      45.047   8.646   6.798  1.00  0.00           O  \\nATOM   6981  CB  ASN B 445      46.540  10.401   8.504  1.00  0.00           C  \\nATOM   6982  CG  ASN B 445      46.146  11.520   7.523  1.00  0.00           C  \\nATOM   6983  OD1 ASN B 445      45.271  12.352   7.805  1.00  0.00           O  \\nATOM   6984  ND2 ASN B 445      46.813  11.544   6.356  1.00  0.00           N  \\nATOM   6985  H   ASN B 445      45.547   9.043  11.065  1.00  0.00           H  \\nATOM   6986  HA  ASN B 445      44.564  10.277   9.384  1.00  0.00           H  \\nATOM   6987 1HB  ASN B 445      47.065  10.850   9.348  1.00  0.00           H  \\nATOM   6988 2HB  ASN B 445      47.239   9.732   8.003  1.00  0.00           H  \\nATOM   6989 1HD2 ASN B 445      46.604  12.246   5.675  1.00  0.00           H  \\nATOM   6990 2HD2 ASN B 445      47.517  10.860   6.169  1.00  0.00           H  \\nATOM   6991  N   ARG B 446      43.736   7.844   8.431  1.00  0.00           N  \\nATOM   6992  CA  ARG B 446      43.100   6.850   7.585  1.00  0.00           C  \\nATOM   6993  C   ARG B 446      41.598   7.067   7.535  1.00  0.00           C  \\nATOM   6994  O   ARG B 446      41.014   7.531   8.510  1.00  0.00           O  \\nATOM   6995  CB  ARG B 446      43.368   5.412   8.091  1.00  0.00           C  \\nATOM   6996  CG  ARG B 446      43.196   4.458   6.905  1.00  0.00           C  \\nATOM   6997  CD  ARG B 446      43.091   3.047   7.306  1.00  0.00           C  \\nATOM   6998  NE  ARG B 446      44.423   2.467   7.265  1.00  0.00           N  \\nATOM   6999  CZ  ARG B 446      44.774   1.415   6.497  1.00  0.00           C  \\nATOM   7000  NH1 ARG B 446      43.859   0.803   5.718  1.00  0.00           N  \\nATOM   7001  NH2 ARG B 446      46.021   0.934   6.549  1.00  0.00           N  \\nATOM   7002  H   ARG B 446      43.468   7.907   9.403  1.00  0.00           H  \\nATOM   7003  HA  ARG B 446      43.512   6.937   6.579  1.00  0.00           H  \\nATOM   7004 1HB  ARG B 446      44.375   5.353   8.503  1.00  0.00           H  \\nATOM   7005 2HB  ARG B 446      42.671   5.173   8.895  1.00  0.00           H  \\nATOM   7006 1HG  ARG B 446      42.286   4.714   6.361  1.00  0.00           H  \\nATOM   7007 2HG  ARG B 446      44.055   4.547   6.238  1.00  0.00           H  \\nATOM   7008 1HD  ARG B 446      42.685   2.985   8.315  1.00  0.00           H  \\nATOM   7009 2HD  ARG B 446      42.431   2.521   6.617  1.00  0.00           H  \\nATOM   7010  HE  ARG B 446      45.134   2.881   7.853  1.00  0.00           H  \\nATOM   7011 1HH1 ARG B 446      42.904   1.132   5.707  1.00  0.00           H  \\nATOM   7012 2HH1 ARG B 446      44.129   0.016   5.146  1.00  0.00           H  \\nATOM   7013 1HH2 ARG B 446      46.701   1.359   7.164  1.00  0.00           H  \\nATOM   7014 2HH2 ARG B 446      46.283   0.147   5.974  1.00  0.00           H  \\nATOM   7015  N   ARG B 447      40.983   6.803   6.408  1.00  0.00           N  \\nATOM   7016  CA  ARG B 447      39.537   6.652   6.308  1.00  0.00           C  \\nATOM   7017  C   ARG B 447      39.243   5.181   6.079  1.00  0.00           C  \\nATOM   7018  O   ARG B 447      39.981   4.458   5.356  1.00  0.00           O  \\nATOM   7019  CB  ARG B 447      38.921   7.551   5.204  1.00  0.00           C  \\nATOM   7020  CG  ARG B 447      39.786   7.496   3.881  1.00  0.00           C  \\nATOM   7021  CD  ARG B 447      39.778   8.874   3.184  1.00  0.00           C  \\nATOM   7022  NE  ARG B 447      38.440   9.037   2.671  1.00  0.00           N  \\nATOM   7023  CZ  ARG B 447      37.863  10.209   2.333  1.00  0.00           C  \\nATOM   7024  NH1 ARG B 447      38.445  11.388   2.498  1.00  0.00           N  \\nATOM   7025  NH2 ARG B 447      36.651  10.144   1.861  1.00  0.00           N  \\nATOM   7026  H   ARG B 447      41.547   6.701   5.576  1.00  0.00           H  \\nATOM   7027  HA  ARG B 447      39.092   6.945   7.259  1.00  0.00           H  \\nATOM   7028 1HB  ARG B 447      37.905   7.221   4.992  1.00  0.00           H  \\nATOM   7029 2HB  ARG B 447      38.863   8.579   5.562  1.00  0.00           H  \\nATOM   7030 1HG  ARG B 447      40.813   7.227   4.129  1.00  0.00           H  \\nATOM   7031 2HG  ARG B 447      39.368   6.750   3.204  1.00  0.00           H  \\nATOM   7032 1HD  ARG B 447      40.024   9.650   3.909  1.00  0.00           H  \\nATOM   7033 2HD  ARG B 447      40.515   8.880   2.382  1.00  0.00           H  \\nATOM   7034  HE  ARG B 447      37.878   8.205   2.551  1.00  0.00           H  \\nATOM   7035 1HH1 ARG B 447      39.372  11.442   2.897  1.00  0.00           H  \\nATOM   7036 2HH1 ARG B 447      37.962  12.232   2.225  1.00  0.00           H  \\nATOM   7037 1HH2 ARG B 447      36.193   9.248   1.765  1.00  0.00           H  \\nATOM   7038 2HH2 ARG B 447      36.169  10.988   1.591  1.00  0.00           H  \\nATOM   7039  N   SER B 448      38.167   4.722   6.665  1.00  0.00           N  \\nATOM   7040  CA  SER B 448      37.740   3.347   6.472  1.00  0.00           C  \\nATOM   7041  C   SER B 448      36.379   3.140   7.031  1.00  0.00           C  \\nATOM   7042  O   SER B 448      35.925   3.850   7.986  1.00  0.00           O  \\nATOM   7043  CB  SER B 448      38.744   2.430   7.100  1.00  0.00           C  \\nATOM   7044  OG  SER B 448      38.746   2.640   8.482  1.00  0.00           O  \\nATOM   7045  H   SER B 448      37.621   5.328   7.261  1.00  0.00           H  \\nATOM   7046  HA  SER B 448      37.689   3.147   5.401  1.00  0.00           H  \\nATOM   7047 1HB  SER B 448      38.488   1.397   6.867  1.00  0.00           H  \\nATOM   7048 2HB  SER B 448      39.729   2.626   6.679  1.00  0.00           H  \\nATOM   7049  HG  SER B 448      38.093   3.324   8.647  1.00  0.00           H  \\nATOM   7050  N   ALA B 449      35.663   2.187   6.455  1.00  0.00           N  \\nATOM   7051  CA  ALA B 449      34.323   1.958   6.861  1.00  0.00           C  \\nATOM   7052  C   ALA B 449      34.268   1.213   8.211  1.00  0.00           C  \\nATOM   7053  O   ALA B 449      35.226   0.509   8.556  1.00  0.00           O  \\nATOM   7054  CB  ALA B 449      33.571   1.198   5.692  1.00  0.00           C  \\nATOM   7055  H   ALA B 449      36.060   1.615   5.724  1.00  0.00           H  \\nATOM   7056  HA  ALA B 449      33.852   2.925   7.037  1.00  0.00           H  \\nATOM   7057 1HB  ALA B 449      32.538   1.012   5.985  1.00  0.00           H  \\nATOM   7058 2HB  ALA B 449      33.588   1.810   4.790  1.00  0.00           H  \\nATOM   7059 3HB  ALA B 449      34.068   0.249   5.496  1.00  0.00           H  \\nATOM   7060  N   PRO B 450      33.178   1.364   8.972  1.00  0.00           N  \\nATOM   7061  CA  PRO B 450      33.103   0.651  10.249  1.00  0.00           C  \\nATOM   7062  C   PRO B 450      32.971  -0.865  10.074  1.00  0.00           C  \\nATOM   7063  O   PRO B 450      32.197  -1.368   9.213  1.00  0.00           O  \\nATOM   7064  CB  PRO B 450      31.801   1.145  10.861  1.00  0.00           C  \\nATOM   7065  CG  PRO B 450      30.981   1.629   9.671  1.00  0.00           C  \\nATOM   7066  CD  PRO B 450      31.990   2.204   8.725  1.00  0.00           C  \\nATOM   7067  HA  PRO B 450      33.960   0.939  10.875  1.00  0.00           H  \\nATOM   7068 1HB  PRO B 450      31.310   0.329  11.411  1.00  0.00           H  \\nATOM   7069 2HB  PRO B 450      32.007   1.944  11.588  1.00  0.00           H  \\nATOM   7070 1HG  PRO B 450      30.420   0.792   9.231  1.00  0.00           H  \\nATOM   7071 2HG  PRO B 450      30.239   2.372   9.998  1.00  0.00           H  \\nATOM   7072 1HD  PRO B 450      31.624   2.104   7.692  1.00  0.00           H  \\nATOM   7073 2HD  PRO B 450      32.166   3.261   8.974  1.00  0.00           H  \\nATOM   7074  N   ASP B 451      33.666  -1.593  10.947  1.00  0.00           N  \\nATOM   7075  CA  ASP B 451      33.591  -3.041  10.935  1.00  0.00           C  \\nATOM   7076  C   ASP B 451      32.336  -3.548  11.649  1.00  0.00           C  \\nATOM   7077  O   ASP B 451      31.596  -2.749  12.244  1.00  0.00           O  \\nATOM   7078  CB  ASP B 451      34.960  -3.666  11.395  1.00  0.00           C  \\nATOM   7079  CG  ASP B 451      35.178  -3.629  12.865  1.00  0.00           C  \\nATOM   7080  OD1 ASP B 451      34.203  -3.779  13.654  1.00  0.00           O  \\nATOM   7081  OD2 ASP B 451      36.395  -3.476  13.179  1.00  0.00           O  \\nATOM   7082  H   ASP B 451      34.255  -1.137  11.629  1.00  0.00           H  \\nATOM   7083  HA  ASP B 451      33.383  -3.368   9.916  1.00  0.00           H  \\nATOM   7084 1HB  ASP B 451      35.012  -4.706  11.071  1.00  0.00           H  \\nATOM   7085 2HB  ASP B 451      35.782  -3.133  10.917  1.00  0.00           H  \\nATOM   7086  N   PRO B 452      31.997  -4.878  11.567  1.00  0.00           N  \\nATOM   7087  CA  PRO B 452      30.777  -5.418  12.140  1.00  0.00           C  \\nATOM   7088  C   PRO B 452      30.613  -5.168  13.611  1.00  0.00           C  \\nATOM   7089  O   PRO B 452      29.489  -5.129  14.103  1.00  0.00           O  \\nATOM   7090  CB  PRO B 452      30.847  -6.973  11.921  1.00  0.00           C  \\nATOM   7091  CG  PRO B 452      31.909  -7.105  10.867  1.00  0.00           C  \\nATOM   7092  CD  PRO B 452      32.758  -5.882  10.784  1.00  0.00           C  \\nATOM   7093  HA  PRO B 452      29.911  -5.008  11.600  1.00  0.00           H  \\nATOM   7094 1HB  PRO B 452      31.099  -7.475  12.867  1.00  0.00           H  \\nATOM   7095 2HB  PRO B 452      29.864  -7.353  11.607  1.00  0.00           H  \\nATOM   7096 1HG  PRO B 452      32.552  -7.971  11.082  1.00  0.00           H  \\nATOM   7097 2HG  PRO B 452      31.449  -7.287   9.885  1.00  0.00           H  \\nATOM   7098 1HD  PRO B 452      33.740  -6.085  11.236  1.00  0.00           H  \\nATOM   7099 2HD  PRO B 452      32.870  -5.583   9.731  1.00  0.00           H  \\nATOM   7100  N   SER B 453      31.702  -5.047  14.395  1.00  0.00           N  \\nATOM   7101  CA  SER B 453      31.574  -4.787  15.795  1.00  0.00           C  \\nATOM   7102  C   SER B 453      30.925  -3.414  16.024  1.00  0.00           C  \\nATOM   7103  O   SER B 453      30.097  -3.245  16.978  1.00  0.00           O  \\nATOM   7104  CB  SER B 453      32.955  -4.944  16.543  1.00  0.00           C  \\nATOM   7105  OG  SER B 453      33.871  -3.946  16.140  1.00  0.00           O  \\nATOM   7106  H   SER B 453      32.624  -5.140  13.994  1.00  0.00           H  \\nATOM   7107  HA  SER B 453      30.875  -5.509  16.219  1.00  0.00           H  \\nATOM   7108 1HB  SER B 453      32.795  -4.879  17.619  1.00  0.00           H  \\nATOM   7109 2HB  SER B 453      33.372  -5.928  16.333  1.00  0.00           H  \\nATOM   7110  HG  SER B 453      33.408  -3.408  15.493  1.00  0.00           H  \\nATOM   7111  N   VAL B 454      31.341  -2.437  15.198  1.00  0.00           N  \\nATOM   7112  CA  VAL B 454      30.805  -1.057  15.368  1.00  0.00           C  \\nATOM   7113  C   VAL B 454      29.367  -0.999  14.947  1.00  0.00           C  \\nATOM   7114  O   VAL B 454      28.486  -0.452  15.668  1.00  0.00           O  \\nATOM   7115  CB  VAL B 454      31.608  -0.146  14.501  1.00  0.00           C  \\nATOM   7116  CG1 VAL B 454      31.037   1.274  14.501  1.00  0.00           C  \\nATOM   7117  CG2 VAL B 454      33.114  -0.174  14.931  1.00  0.00           C  \\nATOM   7118  H   VAL B 454      32.010  -2.618  14.463  1.00  0.00           H  \\nATOM   7119  HA  VAL B 454      30.917  -0.769  16.414  1.00  0.00           H  \\nATOM   7120  HB  VAL B 454      31.522  -0.476  13.466  1.00  0.00           H  \\nATOM   7121 1HG1 VAL B 454      31.646   1.911  13.859  1.00  0.00           H  \\nATOM   7122 2HG1 VAL B 454      30.014   1.255  14.126  1.00  0.00           H  \\nATOM   7123 3HG1 VAL B 454      31.045   1.670  15.516  1.00  0.00           H  \\nATOM   7124 1HG2 VAL B 454      33.688   0.494  14.290  1.00  0.00           H  \\nATOM   7125 2HG2 VAL B 454      33.203   0.152  15.967  1.00  0.00           H  \\nATOM   7126 3HG2 VAL B 454      33.501  -1.189  14.836  1.00  0.00           H  \\nATOM   7127  N   ILE B 455      29.086  -1.619  13.788  1.00  0.00           N  \\nATOM   7128  CA  ILE B 455      27.741  -1.622  13.247  1.00  0.00           C  \\nATOM   7129  C   ILE B 455      26.761  -2.304  14.252  1.00  0.00           C  \\nATOM   7130  O   ILE B 455      25.648  -1.828  14.604  1.00  0.00           O  \\nATOM   7131  CB  ILE B 455      27.658  -2.215  11.791  1.00  0.00           C  \\nATOM   7132  CG1 ILE B 455      28.489  -1.302  10.868  1.00  0.00           C  \\nATOM   7133  CG2 ILE B 455      26.120  -2.503  11.436  1.00  0.00           C  \\nATOM   7134  CD1 ILE B 455      28.837  -1.801   9.445  1.00  0.00           C  \\nATOM   7135  H   ILE B 455      29.821  -2.092  13.282  1.00  0.00           H  \\nATOM   7136  HA  ILE B 455      27.386  -0.593  13.198  1.00  0.00           H  \\nATOM   7137  HB  ILE B 455      28.227  -3.143  11.743  1.00  0.00           H  \\nATOM   7138 1HG1 ILE B 455      27.968  -0.356  10.727  1.00  0.00           H  \\nATOM   7139 2HG1 ILE B 455      29.446  -1.080  11.341  1.00  0.00           H  \\nATOM   7140 1HG2 ILE B 455      26.050  -2.914  10.429  1.00  0.00           H  \\nATOM   7141 2HG2 ILE B 455      25.710  -3.218  12.148  1.00  0.00           H  \\nATOM   7142 3HG2 ILE B 455      25.554  -1.573  11.489  1.00  0.00           H  \\nATOM   7143 1HD1 ILE B 455      29.424  -1.043   8.927  1.00  0.00           H  \\nATOM   7144 2HD1 ILE B 455      29.415  -2.723   9.514  1.00  0.00           H  \\nATOM   7145 3HD1 ILE B 455      27.918  -1.989   8.891  1.00  0.00           H  \\nATOM   7146  N   THR B 456      27.147  -3.503  14.763  1.00  0.00           N  \\nATOM   7147  CA  THR B 456      26.323  -4.159  15.740  1.00  0.00           C  \\nATOM   7148  C   THR B 456      26.088  -3.421  17.066  1.00  0.00           C  \\nATOM   7149  O   THR B 456      24.971  -3.367  17.622  1.00  0.00           O  \\nATOM   7150  CB  THR B 456      26.743  -5.624  15.976  1.00  0.00           C  \\nATOM   7151  OG1 THR B 456      26.483  -6.309  14.745  1.00  0.00           O  \\nATOM   7152  CG2 THR B 456      25.931  -6.261  17.002  1.00  0.00           C  \\nATOM   7153  H   THR B 456      28.006  -3.944  14.468  1.00  0.00           H  \\nATOM   7154  HA  THR B 456      25.294  -4.163  15.378  1.00  0.00           H  \\nATOM   7155  HB  THR B 456      27.786  -5.657  16.290  1.00  0.00           H  \\nATOM   7156  HG1 THR B 456      26.122  -5.690  14.105  1.00  0.00           H  \\nATOM   7157 1HG2 THR B 456      26.259  -7.291  17.138  1.00  0.00           H  \\nATOM   7158 2HG2 THR B 456      26.039  -5.718  17.941  1.00  0.00           H  \\nATOM   7159 3HG2 THR B 456      24.886  -6.251  16.696  1.00  0.00           H  \\nATOM   7160  N   LEU B 457      27.147  -2.760  17.568  1.00  0.00           N  \\nATOM   7161  CA  LEU B 457      26.952  -1.873  18.659  1.00  0.00           C  \\nATOM   7162  C   LEU B 457      25.956  -0.792  18.418  1.00  0.00           C  \\nATOM   7163  O   LEU B 457      25.059  -0.588  19.228  1.00  0.00           O  \\nATOM   7164  CB  LEU B 457      28.251  -1.282  19.226  1.00  0.00           C  \\nATOM   7165  CG  LEU B 457      29.065  -2.229  20.156  1.00  0.00           C  \\nATOM   7166  CD1 LEU B 457      30.462  -1.511  20.199  1.00  0.00           C  \\nATOM   7167  CD2 LEU B 457      28.496  -2.358  21.517  1.00  0.00           C  \\nATOM   7168  H   LEU B 457      28.076  -2.878  17.190  1.00  0.00           H  \\nATOM   7169  HA  LEU B 457      26.474  -2.422  19.470  1.00  0.00           H  \\nATOM   7170 1HB  LEU B 457      28.894  -0.996  18.395  1.00  0.00           H  \\nATOM   7171 2HB  LEU B 457      28.006  -0.385  19.794  1.00  0.00           H  \\nATOM   7172  HG  LEU B 457      29.096  -3.228  19.721  1.00  0.00           H  \\nATOM   7173 1HD1 LEU B 457      31.145  -2.081  20.830  1.00  0.00           H  \\nATOM   7174 2HD1 LEU B 457      30.870  -1.447  19.190  1.00  0.00           H  \\nATOM   7175 3HD1 LEU B 457      30.343  -0.508  20.607  1.00  0.00           H  \\nATOM   7176 1HD2 LEU B 457      29.115  -3.034  22.108  1.00  0.00           H  \\nATOM   7177 2HD2 LEU B 457      28.470  -1.378  21.995  1.00  0.00           H  \\nATOM   7178 3HD2 LEU B 457      27.484  -2.757  21.452  1.00  0.00           H  \\nATOM   7179  N   GLU B 458      26.109  -0.067  17.304  1.00  0.00           N  \\nATOM   7180  CA  GLU B 458      25.131   0.969  16.948  1.00  0.00           C  \\nATOM   7181  C   GLU B 458      23.688   0.373  16.971  1.00  0.00           C  \\nATOM   7182  O   GLU B 458      22.765   1.025  17.468  1.00  0.00           O  \\nATOM   7183  CB  GLU B 458      25.467   1.556  15.562  1.00  0.00           C  \\nATOM   7184  CG  GLU B 458      26.823   2.258  15.501  1.00  0.00           C  \\nATOM   7185  CD  GLU B 458      27.283   2.687  14.142  1.00  0.00           C  \\nATOM   7186  OE1 GLU B 458      26.600   2.319  13.158  1.00  0.00           O  \\nATOM   7187  OE2 GLU B 458      28.318   3.313  14.022  1.00  0.00           O  \\nATOM   7188  H   GLU B 458      26.900  -0.226  16.697  1.00  0.00           H  \\nATOM   7189  HA  GLU B 458      25.184   1.765  17.691  1.00  0.00           H  \\nATOM   7190 1HB  GLU B 458      25.463   0.759  14.819  1.00  0.00           H  \\nATOM   7191 2HB  GLU B 458      24.699   2.275  15.275  1.00  0.00           H  \\nATOM   7192 1HG  GLU B 458      26.787   3.151  16.125  1.00  0.00           H  \\nATOM   7193 2HG  GLU B 458      27.583   1.594  15.910  1.00  0.00           H  \\nATOM   7194  N   ASN B 459      23.541  -0.764  16.325  1.00  0.00           N  \\nATOM   7195  CA  ASN B 459      22.212  -1.307  16.168  1.00  0.00           C  \\nATOM   7196  C   ASN B 459      21.687  -1.796  17.539  1.00  0.00           C  \\nATOM   7197  O   ASN B 459      20.456  -1.858  17.694  1.00  0.00           O  \\nATOM   7198  CB  ASN B 459      22.269  -2.502  15.213  1.00  0.00           C  \\nATOM   7199  CG  ASN B 459      22.494  -2.106  13.752  1.00  0.00           C  \\nATOM   7200  OD1 ASN B 459      22.321  -0.930  13.376  1.00  0.00           O  \\nATOM   7201  ND2 ASN B 459      22.844  -3.102  12.942  1.00  0.00           N  \\nATOM   7202  H   ASN B 459      24.327  -1.266  15.937  1.00  0.00           H  \\nATOM   7203  HA  ASN B 459      21.570  -0.534  15.741  1.00  0.00           H  \\nATOM   7204 1HB  ASN B 459      23.075  -3.171  15.515  1.00  0.00           H  \\nATOM   7205 2HB  ASN B 459      21.337  -3.063  15.276  1.00  0.00           H  \\nATOM   7206 1HD2 ASN B 459      23.009  -2.923  11.972  1.00  0.00           H  \\nATOM   7207 2HD2 ASN B 459      22.942  -4.029  13.303  1.00  0.00           H  \\nATOM   7208  N   SER B 460      22.598  -1.999  18.546  1.00  0.00           N  \\nATOM   7209  CA  SER B 460      22.152  -2.571  19.812  1.00  0.00           C  \\nATOM   7210  C   SER B 460      22.089  -1.579  20.933  1.00  0.00           C  \\nATOM   7211  O   SER B 460      21.704  -1.933  22.061  1.00  0.00           O  \\nATOM   7212  CB  SER B 460      23.045  -3.703  20.257  1.00  0.00           C  \\nATOM   7213  OG  SER B 460      23.148  -4.743  19.185  1.00  0.00           O  \\nATOM   7214  H   SER B 460      23.573  -1.761  18.430  1.00  0.00           H  \\nATOM   7215  HA  SER B 460      21.145  -2.968  19.677  1.00  0.00           H  \\nATOM   7216 1HB  SER B 460      24.035  -3.314  20.493  1.00  0.00           H  \\nATOM   7217 2HB  SER B 460      22.643  -4.147  21.167  1.00  0.00           H  \\nATOM   7218  HG  SER B 460      22.607  -4.428  18.457  1.00  0.00           H  \\nATOM   7219  N   TRP B 461      22.328  -0.304  20.638  1.00  0.00           N  \\nATOM   7220  CA  TRP B 461      22.434   0.610  21.788  1.00  0.00           C  \\nATOM   7221  C   TRP B 461      21.216   0.668  22.655  1.00  0.00           C  \\nATOM   7222  O   TRP B 461      21.282   0.722  23.889  1.00  0.00           O  \\nATOM   7223  CB  TRP B 461      22.720   1.980  21.232  1.00  0.00           C  \\nATOM   7224  CG  TRP B 461      22.945   3.057  22.249  1.00  0.00           C  \\nATOM   7225  CD1 TRP B 461      24.048   3.175  23.121  1.00  0.00           C  \\nATOM   7226  CD2 TRP B 461      22.048   4.127  22.621  1.00  0.00           C  \\nATOM   7227  NE1 TRP B 461      23.833   4.239  23.992  1.00  0.00           N  \\nATOM   7228  CE2 TRP B 461      22.668   4.891  23.646  1.00  0.00           C  \\nATOM   7229  CE3 TRP B 461      20.819   4.580  22.076  1.00  0.00           C  \\nATOM   7230  CZ2 TRP B 461      22.044   6.040  24.224  1.00  0.00           C  \\nATOM   7231  CZ3 TRP B 461      20.191   5.742  22.673  1.00  0.00           C  \\nATOM   7232  CH2 TRP B 461      20.798   6.430  23.704  1.00  0.00           C  \\nATOM   7233  H   TRP B 461      22.438   0.065  19.704  1.00  0.00           H  \\nATOM   7234  HA  TRP B 461      23.258   0.277  22.419  1.00  0.00           H  \\nATOM   7235 1HB  TRP B 461      23.609   1.939  20.603  1.00  0.00           H  \\nATOM   7236 2HB  TRP B 461      21.889   2.298  20.603  1.00  0.00           H  \\nATOM   7237  HD1 TRP B 461      24.925   2.530  23.107  1.00  0.00           H  \\nATOM   7238  HE1 TRP B 461      24.433   4.500  24.762  1.00  0.00           H  \\nATOM   7239  HE3 TRP B 461      20.358   4.072  21.230  1.00  0.00           H  \\nATOM   7240  HZ2 TRP B 461      22.505   6.603  25.036  1.00  0.00           H  \\nATOM   7241  HZ3 TRP B 461      19.223   6.060  22.285  1.00  0.00           H  \\nATOM   7242  HH2 TRP B 461      20.291   7.300  24.123  1.00  0.00           H  \\nATOM   7243  N   GLY B 462      20.015   0.779  22.028  1.00  0.00           N  \\nATOM   7244  CA  GLY B 462      18.711   0.797  22.731  1.00  0.00           C  \\nATOM   7245  C   GLY B 462      18.542  -0.452  23.594  1.00  0.00           C  \\nATOM   7246  O   GLY B 462      18.174  -0.320  24.765  1.00  0.00           O  \\nATOM   7247  H   GLY B 462      20.032   0.853  21.021  1.00  0.00           H  \\nATOM   7248 1HA  GLY B 462      18.645   1.690  23.353  1.00  0.00           H  \\nATOM   7249 2HA  GLY B 462      17.905   0.855  22.001  1.00  0.00           H  \\nATOM   7250  N   ARG B 463      18.822  -1.576  22.985  1.00  0.00           N  \\nATOM   7251  CA  ARG B 463      18.654  -2.872  23.679  1.00  0.00           C  \\nATOM   7252  C   ARG B 463      19.670  -3.007  24.794  1.00  0.00           C  \\nATOM   7253  O   ARG B 463      19.341  -3.595  25.863  1.00  0.00           O  \\nATOM   7254  CB  ARG B 463      18.682  -4.055  22.725  1.00  0.00           C  \\nATOM   7255  CG  ARG B 463      17.313  -4.345  22.242  1.00  0.00           C  \\nATOM   7256  CD  ARG B 463      17.166  -4.867  20.815  1.00  0.00           C  \\nATOM   7257  NE  ARG B 463      18.423  -5.209  20.129  1.00  0.00           N  \\nATOM   7258  CZ  ARG B 463      19.053  -6.347  20.305  1.00  0.00           C  \\nATOM   7259  NH1 ARG B 463      18.566  -7.274  21.164  1.00  0.00           N  \\nATOM   7260  NH2 ARG B 463      20.184  -6.543  19.645  1.00  0.00           N  \\nATOM   7261  H   ARG B 463      19.157  -1.568  22.032  1.00  0.00           H  \\nATOM   7262  HA  ARG B 463      17.684  -2.874  24.178  1.00  0.00           H  \\nATOM   7263 1HB  ARG B 463      19.338  -3.829  21.885  1.00  0.00           H  \\nATOM   7264 2HB  ARG B 463      19.093  -4.925  23.237  1.00  0.00           H  \\nATOM   7265 1HG  ARG B 463      16.857  -5.106  22.876  1.00  0.00           H  \\nATOM   7266 2HG  ARG B 463      16.713  -3.435  22.279  1.00  0.00           H  \\nATOM   7267 1HD  ARG B 463      16.564  -5.775  20.820  1.00  0.00           H  \\nATOM   7268 2HD  ARG B 463      16.678  -4.112  20.200  1.00  0.00           H  \\nATOM   7269  HE  ARG B 463      18.812  -4.526  19.492  1.00  0.00           H  \\nATOM   7270 1HH1 ARG B 463      17.713  -7.092  21.674  1.00  0.00           H  \\nATOM   7271 2HH1 ARG B 463      19.055  -8.147  21.295  1.00  0.00           H  \\nATOM   7272 1HH2 ARG B 463      20.539  -5.826  19.027  1.00  0.00           H  \\nATOM   7273 2HH2 ARG B 463      20.692  -7.408  19.759  1.00  0.00           H  \\nATOM   7274  N   LEU B 464      20.899  -2.529  24.590  1.00  0.00           N  \\nATOM   7275  CA  LEU B 464      21.869  -2.573  25.675  1.00  0.00           C  \\nATOM   7276  C   LEU B 464      21.427  -1.703  26.830  1.00  0.00           C  \\nATOM   7277  O   LEU B 464      21.697  -2.048  28.029  1.00  0.00           O  \\nATOM   7278  CB  LEU B 464      23.252  -2.144  25.220  1.00  0.00           C  \\nATOM   7279  CG  LEU B 464      23.894  -3.062  24.202  1.00  0.00           C  \\nATOM   7280  CD1 LEU B 464      25.115  -2.550  23.409  1.00  0.00           C  \\nATOM   7281  CD2 LEU B 464      24.249  -4.404  24.943  1.00  0.00           C  \\nATOM   7282  H   LEU B 464      21.173  -2.137  23.701  1.00  0.00           H  \\nATOM   7283  HA  LEU B 464      21.938  -3.599  26.035  1.00  0.00           H  \\nATOM   7284 1HB  LEU B 464      23.181  -1.148  24.785  1.00  0.00           H  \\nATOM   7285 2HB  LEU B 464      23.904  -2.091  26.092  1.00  0.00           H  \\nATOM   7286  HG  LEU B 464      23.194  -3.248  23.387  1.00  0.00           H  \\nATOM   7287 1HD1 LEU B 464      25.455  -3.326  22.723  1.00  0.00           H  \\nATOM   7288 2HD1 LEU B 464      24.834  -1.663  22.841  1.00  0.00           H  \\nATOM   7289 3HD1 LEU B 464      25.919  -2.300  24.100  1.00  0.00           H  \\nATOM   7290 1HD2 LEU B 464      24.715  -5.096  24.240  1.00  0.00           H  \\nATOM   7291 2HD2 LEU B 464      24.940  -4.197  25.760  1.00  0.00           H  \\nATOM   7292 3HD2 LEU B 464      23.338  -4.851  25.342  1.00  0.00           H  \\nATOM   7293  N   SER B 465      20.970  -0.505  26.542  1.00  0.00           N  \\nATOM   7294  CA  SER B 465      20.489   0.350  27.548  1.00  0.00           C  \\nATOM   7295  C   SER B 465      19.436  -0.324  28.451  1.00  0.00           C  \\nATOM   7296  O   SER B 465      19.442  -0.170  29.679  1.00  0.00           O  \\nATOM   7297  CB  SER B 465      19.898   1.638  26.885  1.00  0.00           C  \\nATOM   7298  OG  SER B 465      20.943   2.426  26.260  1.00  0.00           O  \\nATOM   7299  H   SER B 465      20.962  -0.189  25.583  1.00  0.00           H  \\nATOM   7300  HA  SER B 465      21.323   0.627  28.195  1.00  0.00           H  \\nATOM   7301 1HB  SER B 465      19.156   1.355  26.139  1.00  0.00           H  \\nATOM   7302 2HB  SER B 465      19.390   2.236  27.641  1.00  0.00           H  \\nATOM   7303  HG  SER B 465      21.762   1.948  26.412  1.00  0.00           H  \\nATOM   7304  N   THR B 466      18.463  -0.967  27.815  1.00  0.00           N  \\nATOM   7305  CA  THR B 466      17.430  -1.746  28.544  1.00  0.00           C  \\nATOM   7306  C   THR B 466      18.063  -2.909  29.316  1.00  0.00           C  \\nATOM   7307  O   THR B 466      17.719  -2.954  30.557  1.00  0.00           O  \\nATOM   7308  CB  THR B 466      16.403  -2.273  27.550  1.00  0.00           C  \\nATOM   7309  OG1 THR B 466      15.832  -1.134  26.886  1.00  0.00           O  \\nATOM   7310  CG2 THR B 466      15.385  -3.076  28.207  1.00  0.00           C  \\nATOM   7311  H   THR B 466      18.426  -0.924  26.807  1.00  0.00           H  \\nATOM   7312  HA  THR B 466      16.935  -1.084  29.255  1.00  0.00           H  \\nATOM   7313  HB  THR B 466      16.903  -2.889  26.803  1.00  0.00           H  \\nATOM   7314  HG1 THR B 466      16.233  -0.331  27.228  1.00  0.00           H  \\nATOM   7315 1HG2 THR B 466      14.671  -3.435  27.466  1.00  0.00           H  \\nATOM   7316 2HG2 THR B 466      15.855  -3.927  28.700  1.00  0.00           H  \\nATOM   7317 3HG2 THR B 466      14.865  -2.469  28.948  1.00  0.00           H  \\nATOM   7318  N   ALA B 467      18.953  -3.687  28.651  1.00  0.00           N  \\nATOM   7319  CA  ALA B 467      19.540  -4.902  29.313  1.00  0.00           C  \\nATOM   7320  C   ALA B 467      20.262  -4.403  30.579  1.00  0.00           C  \\nATOM   7321  O   ALA B 467      20.103  -5.009  31.672  1.00  0.00           O  \\nATOM   7322  CB  ALA B 467      20.458  -5.598  28.480  1.00  0.00           C  \\nATOM   7323  H   ALA B 467      19.230  -3.459  27.707  1.00  0.00           H  \\nATOM   7324  HA  ALA B 467      18.725  -5.586  29.550  1.00  0.00           H  \\nATOM   7325 1HB  ALA B 467      20.848  -6.465  29.013  1.00  0.00           H  \\nATOM   7326 2HB  ALA B 467      19.949  -5.928  27.574  1.00  0.00           H  \\nATOM   7327 3HB  ALA B 467      21.281  -4.936  28.214  1.00  0.00           H  \\nATOM   7328  N   ILE B 468      20.979  -3.270  30.520  1.00  0.00           N  \\nATOM   7329  CA  ILE B 468      21.725  -2.852  31.716  1.00  0.00           C  \\nATOM   7330  C   ILE B 468      20.858  -2.299  32.792  1.00  0.00           C  \\nATOM   7331  O   ILE B 468      20.985  -2.670  33.953  1.00  0.00           O  \\nATOM   7332  CB  ILE B 468      22.780  -1.786  31.248  1.00  0.00           C  \\nATOM   7333  CG1 ILE B 468      23.929  -2.437  30.435  1.00  0.00           C  \\nATOM   7334  CG2 ILE B 468      23.527  -1.116  32.419  1.00  0.00           C  \\nATOM   7335  CD1 ILE B 468      24.783  -1.488  29.614  1.00  0.00           C  \\nATOM   7336  H   ILE B 468      21.023  -2.699  29.688  1.00  0.00           H  \\nATOM   7337  HA  ILE B 468      22.227  -3.724  32.132  1.00  0.00           H  \\nATOM   7338  HB  ILE B 468      22.278  -1.000  30.686  1.00  0.00           H  \\nATOM   7339 1HG1 ILE B 468      24.599  -2.967  31.112  1.00  0.00           H  \\nATOM   7340 2HG1 ILE B 468      23.515  -3.172  29.744  1.00  0.00           H  \\nATOM   7341 1HG2 ILE B 468      24.241  -0.391  32.028  1.00  0.00           H  \\nATOM   7342 2HG2 ILE B 468      22.811  -0.608  33.064  1.00  0.00           H  \\nATOM   7343 3HG2 ILE B 468      24.058  -1.875  32.994  1.00  0.00           H  \\nATOM   7344 1HD1 ILE B 468      25.553  -2.053  29.088  1.00  0.00           H  \\nATOM   7345 2HD1 ILE B 468      24.156  -0.968  28.889  1.00  0.00           H  \\nATOM   7346 3HD1 ILE B 468      25.255  -0.761  30.273  1.00  0.00           H  \\nATOM   7347  N   GLN B 469      19.893  -1.400  32.499  1.00  0.00           N  \\nATOM   7348  CA  GLN B 469      18.997  -0.876  33.472  1.00  0.00           C  \\nATOM   7349  C   GLN B 469      17.992  -1.822  34.078  1.00  0.00           C  \\nATOM   7350  O   GLN B 469      17.630  -1.638  35.250  1.00  0.00           O  \\nATOM   7351  CB  GLN B 469      18.312   0.351  32.812  1.00  0.00           C  \\nATOM   7352  CG  GLN B 469      19.269   1.585  32.745  1.00  0.00           C  \\nATOM   7353  CD  GLN B 469      18.652   2.679  31.937  1.00  0.00           C  \\nATOM   7354  OE1 GLN B 469      17.711   3.336  32.385  1.00  0.00           O  \\nATOM   7355  NE2 GLN B 469      19.235   2.914  30.770  1.00  0.00           N  \\nATOM   7356  H   GLN B 469      19.813  -1.092  31.540  1.00  0.00           H  \\nATOM   7357  HA  GLN B 469      19.574  -0.572  34.345  1.00  0.00           H  \\nATOM   7358 1HB  GLN B 469      17.992   0.090  31.803  1.00  0.00           H  \\nATOM   7359 2HB  GLN B 469      17.421   0.619  33.379  1.00  0.00           H  \\nATOM   7360 1HG  GLN B 469      19.453   1.945  33.757  1.00  0.00           H  \\nATOM   7361 2HG  GLN B 469      20.207   1.280  32.281  1.00  0.00           H  \\nATOM   7362 1HE2 GLN B 469      18.881   3.632  30.169  1.00  0.00           H  \\nATOM   7363 2HE2 GLN B 469      20.029   2.374  30.489  1.00  0.00           H  \\nATOM   7364  N   GLU B 470      17.575  -2.835  33.339  1.00  0.00           N  \\nATOM   7365  CA  GLU B 470      16.682  -3.855  33.968  1.00  0.00           C  \\nATOM   7366  C   GLU B 470      17.461  -5.039  34.501  1.00  0.00           C  \\nATOM   7367  O   GLU B 470      16.846  -6.036  34.910  1.00  0.00           O  \\nATOM   7368  CB  GLU B 470      15.665  -4.302  32.984  1.00  0.00           C  \\nATOM   7369  CG  GLU B 470      14.826  -3.084  32.507  1.00  0.00           C  \\nATOM   7370  CD  GLU B 470      13.404  -3.455  32.289  1.00  0.00           C  \\nATOM   7371  OE1 GLU B 470      12.891  -4.354  33.022  1.00  0.00           O  \\nATOM   7372  OE2 GLU B 470      12.798  -2.841  31.376  1.00  0.00           O  \\nATOM   7373  H   GLU B 470      17.838  -2.942  32.370  1.00  0.00           H  \\nATOM   7374  HA  GLU B 470      16.177  -3.395  34.818  1.00  0.00           H  \\nATOM   7375 1HB  GLU B 470      16.162  -4.774  32.136  1.00  0.00           H  \\nATOM   7376 2HB  GLU B 470      15.019  -5.050  33.443  1.00  0.00           H  \\nATOM   7377 1HG  GLU B 470      14.887  -2.297  33.259  1.00  0.00           H  \\nATOM   7378 2HG  GLU B 470      15.256  -2.700  31.583  1.00  0.00           H  \\nATOM   7379  N   SER B 471      18.782  -4.990  34.472  1.00  0.00           N  \\nATOM   7380  CA  SER B 471      19.592  -6.135  34.963  1.00  0.00           C  \\nATOM   7381  C   SER B 471      19.389  -6.398  36.437  1.00  0.00           C  \\nATOM   7382  O   SER B 471      18.915  -5.575  37.186  1.00  0.00           O  \\nATOM   7383  CB  SER B 471      21.068  -5.903  34.631  1.00  0.00           C  \\nATOM   7384  OG  SER B 471      21.654  -4.976  35.459  1.00  0.00           O  \\nATOM   7385  H   SER B 471      19.253  -4.171  34.115  1.00  0.00           H  \\nATOM   7386  HA  SER B 471      19.252  -7.041  34.459  1.00  0.00           H  \\nATOM   7387 1HB  SER B 471      21.612  -6.844  34.714  1.00  0.00           H  \\nATOM   7388 2HB  SER B 471      21.160  -5.563  33.600  1.00  0.00           H  \\nATOM   7389  HG  SER B 471      20.966  -4.691  36.066  1.00  0.00           H  \\nATOM   7390  N   ASN B 472      19.766  -7.645  36.767  1.00  0.00           N  \\nATOM   7391  CA  ASN B 472      19.707  -8.085  38.188  1.00  0.00           C  \\nATOM   7392  C   ASN B 472      21.050  -7.818  38.899  1.00  0.00           C  \\nATOM   7393  O   ASN B 472      21.958  -8.621  38.755  1.00  0.00           O  \\nATOM   7394  CB  ASN B 472      19.372  -9.550  38.208  1.00  0.00           C  \\nATOM   7395  CG  ASN B 472      19.458 -10.150  39.572  1.00  0.00           C  \\nATOM   7396  OD1 ASN B 472      19.330  -9.480  40.579  1.00  0.00           O  \\nATOM   7397  ND2 ASN B 472      19.639 -11.482  39.553  1.00  0.00           N  \\nATOM   7398  H   ASN B 472      20.091  -8.294  36.064  1.00  0.00           H  \\nATOM   7399  HA  ASN B 472      18.923  -7.519  38.693  1.00  0.00           H  \\nATOM   7400 1HB  ASN B 472      18.361  -9.698  37.826  1.00  0.00           H  \\nATOM   7401 2HB  ASN B 472      20.053 -10.088  37.548  1.00  0.00           H  \\nATOM   7402 1HD2 ASN B 472      19.711 -11.989  40.412  1.00  0.00           H  \\nATOM   7403 2HD2 ASN B 472      19.702 -11.965  38.680  1.00  0.00           H  \\nATOM   7404  N   GLN B 473      21.217  -6.591  39.387  1.00  0.00           N  \\nATOM   7405  CA  GLN B 473      22.479  -6.156  39.945  1.00  0.00           C  \\nATOM   7406  C   GLN B 473      23.620  -6.366  38.984  1.00  0.00           C  \\nATOM   7407  O   GLN B 473      24.697  -6.807  39.278  1.00  0.00           O  \\nATOM   7408  CB  GLN B 473      22.753  -6.773  41.351  1.00  0.00           C  \\nATOM   7409  CG  GLN B 473      21.605  -6.582  42.293  1.00  0.00           C  \\nATOM   7410  CD  GLN B 473      21.974  -6.750  43.736  1.00  0.00           C  \\nATOM   7411  OE1 GLN B 473      21.442  -7.695  44.416  1.00  0.00           O  \\nATOM   7412  NE2 GLN B 473      22.967  -6.009  44.175  1.00  0.00           N  \\nATOM   7413  H   GLN B 473      20.440  -5.946  39.367  1.00  0.00           H  \\nATOM   7414  HA  GLN B 473      22.452  -5.072  40.059  1.00  0.00           H  \\nATOM   7415 1HB  GLN B 473      22.951  -7.840  41.248  1.00  0.00           H  \\nATOM   7416 2HB  GLN B 473      23.643  -6.315  41.782  1.00  0.00           H  \\nATOM   7417 1HG  GLN B 473      21.210  -5.574  42.169  1.00  0.00           H  \\nATOM   7418 2HG  GLN B 473      20.833  -7.316  42.062  1.00  0.00           H  \\nATOM   7419 1HE2 GLN B 473      23.259  -6.077  45.130  1.00  0.00           H  \\nATOM   7420 2HE2 GLN B 473      23.432  -5.376  43.555  1.00  0.00           H  \\nATOM   7421  N   GLY B 474      23.395  -6.059  37.684  1.00  0.00           N  \\nATOM   7422  CA  GLY B 474      24.420  -6.181  36.666  1.00  0.00           C  \\nATOM   7423  C   GLY B 474      24.351  -7.402  35.821  1.00  0.00           C  \\nATOM   7424  O   GLY B 474      24.914  -7.454  34.702  1.00  0.00           O  \\nATOM   7425  H   GLY B 474      22.476  -5.733  37.420  1.00  0.00           H  \\nATOM   7426 1HA  GLY B 474      24.373  -5.322  35.997  1.00  0.00           H  \\nATOM   7427 2HA  GLY B 474      25.403  -6.168  37.136  1.00  0.00           H  \\nATOM   7428  N   ALA B 475      23.601  -8.427  36.273  1.00  0.00           N  \\nATOM   7429  CA  ALA B 475      23.477  -9.658  35.560  1.00  0.00           C  \\nATOM   7430  C   ALA B 475      22.254  -9.611  34.653  1.00  0.00           C  \\nATOM   7431  O   ALA B 475      21.165  -9.274  35.061  1.00  0.00           O  \\nATOM   7432  CB  ALA B 475      23.357 -10.829  36.602  1.00  0.00           C  \\nATOM   7433  H   ALA B 475      23.111  -8.314  37.149  1.00  0.00           H  \\nATOM   7434  HA  ALA B 475      24.375  -9.792  34.957  1.00  0.00           H  \\nATOM   7435 1HB  ALA B 475      23.261 -11.778  36.074  1.00  0.00           H  \\nATOM   7436 2HB  ALA B 475      24.248 -10.851  37.229  1.00  0.00           H  \\nATOM   7437 3HB  ALA B 475      22.479 -10.672  37.226  1.00  0.00           H  \\nATOM   7438  N   PHE B 476      22.473  -9.923  33.393  1.00  0.00           N  \\nATOM   7439  CA  PHE B 476      21.389  -9.736  32.426  1.00  0.00           C  \\nATOM   7440  C   PHE B 476      20.422 -10.872  32.540  1.00  0.00           C  \\nATOM   7441  O   PHE B 476      20.837 -12.027  32.622  1.00  0.00           O  \\nATOM   7442  CB  PHE B 476      21.919  -9.697  30.992  1.00  0.00           C  \\nATOM   7443  CG  PHE B 476      22.767  -8.496  30.672  1.00  0.00           C  \\nATOM   7444  CD1 PHE B 476      22.866  -7.382  31.563  1.00  0.00           C  \\nATOM   7445  CD2 PHE B 476      23.447  -8.489  29.451  1.00  0.00           C  \\nATOM   7446  CE1 PHE B 476      23.664  -6.268  31.181  1.00  0.00           C  \\nATOM   7447  CE2 PHE B 476      24.244  -7.391  29.101  1.00  0.00           C  \\nATOM   7448  CZ  PHE B 476      24.296  -6.286  29.986  1.00  0.00           C  \\nATOM   7449  H   PHE B 476      23.360 -10.285  33.074  1.00  0.00           H  \\nATOM   7450  HA  PHE B 476      20.900  -8.783  32.634  1.00  0.00           H  \\nATOM   7451 1HB  PHE B 476      22.517 -10.587  30.801  1.00  0.00           H  \\nATOM   7452 2HB  PHE B 476      21.082  -9.709  30.295  1.00  0.00           H  \\nATOM   7453  HD1 PHE B 476      22.331  -7.410  32.512  1.00  0.00           H  \\nATOM   7454  HD2 PHE B 476      23.388  -9.353  28.788  1.00  0.00           H  \\nATOM   7455  HE1 PHE B 476      23.761  -5.413  31.849  1.00  0.00           H  \\nATOM   7456  HE2 PHE B 476      24.815  -7.373  28.173  1.00  0.00           H  \\nATOM   7457  HZ  PHE B 476      24.878  -5.424  29.663  1.00  0.00           H  \\nATOM   7458  N   ALA B 477      19.144 -10.522  32.432  1.00  0.00           N  \\nATOM   7459  CA  ALA B 477      18.092 -11.548  32.415  1.00  0.00           C  \\nATOM   7460  C   ALA B 477      18.195 -12.470  31.181  1.00  0.00           C  \\nATOM   7461  O   ALA B 477      17.947 -13.668  31.257  1.00  0.00           O  \\nATOM   7462  CB  ALA B 477      16.773 -10.853  32.444  1.00  0.00           C  \\nATOM   7463  H   ALA B 477      18.883  -9.549  32.361  1.00  0.00           H  \\nATOM   7464  HA  ALA B 477      18.208 -12.167  33.305  1.00  0.00           H  \\nATOM   7465 1HB  ALA B 477      15.972 -11.592  32.432  1.00  0.00           H  \\nATOM   7466 2HB  ALA B 477      16.698 -10.252  33.350  1.00  0.00           H  \\nATOM   7467 3HB  ALA B 477      16.683 -10.207  31.572  1.00  0.00           H  \\nATOM   7468  N   SER B 478      18.598 -11.923  30.040  1.00  0.00           N  \\nATOM   7469  CA  SER B 478      18.890 -12.758  28.874  1.00  0.00           C  \\nATOM   7470  C   SER B 478      20.090 -12.148  28.157  1.00  0.00           C  \\nATOM   7471  O   SER B 478      20.315 -10.935  28.278  1.00  0.00           O  \\nATOM   7472  CB  SER B 478      17.657 -12.817  27.942  1.00  0.00           C  \\nATOM   7473  OG  SER B 478      16.869 -11.579  28.033  1.00  0.00           O  \\nATOM   7474  H   SER B 478      18.707 -10.921  29.968  1.00  0.00           H  \\nATOM   7475  HA  SER B 478      19.119 -13.767  29.220  1.00  0.00           H  \\nATOM   7476 1HB  SER B 478      17.985 -12.968  26.914  1.00  0.00           H  \\nATOM   7477 2HB  SER B 478      17.035 -13.668  28.215  1.00  0.00           H  \\nATOM   7478  HG  SER B 478      17.326 -11.021  28.667  1.00  0.00           H  \\nATOM   7479  N   PRO B 479      20.875 -12.987  27.453  1.00  0.00           N  \\nATOM   7480  CA  PRO B 479      22.074 -12.540  26.772  1.00  0.00           C  \\nATOM   7481  C   PRO B 479      21.758 -11.573  25.617  1.00  0.00           C  \\nATOM   7482  O   PRO B 479      20.656 -11.565  25.103  1.00  0.00           O  \\nATOM   7483  CB  PRO B 479      22.713 -13.829  26.256  1.00  0.00           C  \\nATOM   7484  CG  PRO B 479      21.621 -14.932  26.470  1.00  0.00           C  \\nATOM   7485  CD  PRO B 479      20.796 -14.465  27.581  1.00  0.00           C  \\nATOM   7486  HA  PRO B 479      22.738 -12.045  27.496  1.00  0.00           H  \\nATOM   7487 1HB  PRO B 479      22.993 -13.713  25.199  1.00  0.00           H  \\nATOM   7488 2HB  PRO B 479      23.638 -14.039  26.812  1.00  0.00           H  \\nATOM   7489 1HG  PRO B 479      21.033 -15.068  25.550  1.00  0.00           H  \\nATOM   7490 2HG  PRO B 479      22.096 -15.900  26.689  1.00  0.00           H  \\nATOM   7491 1HD  PRO B 479      19.767 -14.832  27.453  1.00  0.00           H  \\nATOM   7492 2HD  PRO B 479      21.221 -14.824  28.530  1.00  0.00           H  \\nATOM   7493  N   ILE B 480      22.705 -10.724  25.240  1.00  0.00           N  \\nATOM   7494  CA  ILE B 480      22.485  -9.921  24.044  1.00  0.00           C  \\nATOM   7495  C   ILE B 480      23.660 -10.267  23.185  1.00  0.00           C  \\nATOM   7496  O   ILE B 480      24.771 -10.372  23.656  1.00  0.00           O  \\nATOM   7497  CB  ILE B 480      22.360  -8.397  24.308  1.00  0.00           C  \\nATOM   7498  CG1 ILE B 480      21.023  -8.160  25.041  1.00  0.00           C  \\nATOM   7499  CG2 ILE B 480      22.513  -7.572  22.949  1.00  0.00           C  \\nATOM   7500  CD1 ILE B 480      20.519  -6.737  25.171  1.00  0.00           C  \\nATOM   7501  H   ILE B 480      23.569 -10.617  25.752  1.00  0.00           H  \\nATOM   7502  HA  ILE B 480      21.550 -10.237  23.583  1.00  0.00           H  \\nATOM   7503  HB  ILE B 480      23.141  -8.082  24.999  1.00  0.00           H  \\nATOM   7504 1HG1 ILE B 480      20.230  -8.712  24.538  1.00  0.00           H  \\nATOM   7505 2HG1 ILE B 480      21.093  -8.544  26.059  1.00  0.00           H  \\nATOM   7506 1HG2 ILE B 480      22.421  -6.506  23.158  1.00  0.00           H  \\nATOM   7507 2HG2 ILE B 480      23.489  -7.773  22.510  1.00  0.00           H  \\nATOM   7508 3HG2 ILE B 480      21.732  -7.873  22.251  1.00  0.00           H  \\nATOM   7509 1HD1 ILE B 480      19.571  -6.734  25.709  1.00  0.00           H  \\nATOM   7510 2HD1 ILE B 480      21.249  -6.141  25.719  1.00  0.00           H  \\nATOM   7511 3HD1 ILE B 480      20.373  -6.311  24.179  1.00  0.00           H  \\nATOM   7512  N   GLN B 481      23.339 -10.577  21.946  1.00  0.00           N  \\nATOM   7513  CA  GLN B 481      24.317 -10.920  20.943  1.00  0.00           C  \\nATOM   7514  C   GLN B 481      24.931  -9.726  20.309  1.00  0.00           C  \\nATOM   7515  O   GLN B 481      24.257  -8.760  19.885  1.00  0.00           O  \\nATOM   7516  CB  GLN B 481      23.669 -11.773  19.878  1.00  0.00           C  \\nATOM   7517  CG  GLN B 481      24.675 -12.445  19.038  1.00  0.00           C  \\nATOM   7518  CD  GLN B 481      23.978 -13.174  18.017  1.00  0.00           C  \\nATOM   7519  OE1 GLN B 481      23.588 -12.600  16.978  1.00  0.00           O  \\nATOM   7520  NE2 GLN B 481      23.812 -14.462  18.265  1.00  0.00           N  \\nATOM   7521  H   GLN B 481      22.360 -10.571  21.697  1.00  0.00           H  \\nATOM   7522  HA  GLN B 481      25.116 -11.489  21.419  1.00  0.00           H  \\nATOM   7523 1HB  GLN B 481      23.032 -12.522  20.349  1.00  0.00           H  \\nATOM   7524 2HB  GLN B 481      23.032 -11.150  19.251  1.00  0.00           H  \\nATOM   7525 1HG  GLN B 481      25.325 -11.691  18.594  1.00  0.00           H  \\nATOM   7526 2HG  GLN B 481      25.260 -13.122  19.661  1.00  0.00           H  \\nATOM   7527 1HE2 GLN B 481      23.335 -15.042  17.603  1.00  0.00           H  \\nATOM   7528 2HE2 GLN B 481      24.162 -14.857  19.114  1.00  0.00           H  \\nATOM   7529  N   LEU B 482      26.250  -9.785  20.313  1.00  0.00           N  \\nATOM   7530  CA  LEU B 482      27.029  -8.768  19.591  1.00  0.00           C  \\nATOM   7531  C   LEU B 482      27.906  -9.475  18.588  1.00  0.00           C  \\nATOM   7532  O   LEU B 482      27.859 -10.743  18.493  1.00  0.00           O  \\nATOM   7533  CB  LEU B 482      27.817  -7.865  20.550  1.00  0.00           C  \\nATOM   7534  CG  LEU B 482      27.053  -6.981  21.527  1.00  0.00           C  \\nATOM   7535  CD1 LEU B 482      28.126  -6.434  22.519  1.00  0.00           C  \\nATOM   7536  CD2 LEU B 482      26.216  -5.931  20.857  1.00  0.00           C  \\nATOM   7537  H   LEU B 482      26.737 -10.520  20.806  1.00  0.00           H  \\nATOM   7538  HA  LEU B 482      26.340  -8.140  19.028  1.00  0.00           H  \\nATOM   7539 1HB  LEU B 482      28.469  -8.490  21.159  1.00  0.00           H  \\nATOM   7540 2HB  LEU B 482      28.440  -7.192  19.961  1.00  0.00           H  \\nATOM   7541  HG  LEU B 482      26.383  -7.597  22.128  1.00  0.00           H  \\nATOM   7542 1HD1 LEU B 482      27.646  -5.787  23.254  1.00  0.00           H  \\nATOM   7543 2HD1 LEU B 482      28.608  -7.268  23.030  1.00  0.00           H  \\nATOM   7544 3HD1 LEU B 482      28.874  -5.864  21.968  1.00  0.00           H  \\nATOM   7545 1HD2 LEU B 482      25.700  -5.339  21.614  1.00  0.00           H  \\nATOM   7546 2HD2 LEU B 482      26.857  -5.280  20.262  1.00  0.00           H  \\nATOM   7547 3HD2 LEU B 482      25.483  -6.410  20.208  1.00  0.00           H  \\nATOM   7548  N   GLN B 483      28.626  -8.713  17.780  1.00  0.00           N  \\nATOM   7549  CA  GLN B 483      29.621  -9.225  16.761  1.00  0.00           C  \\nATOM   7550  C   GLN B 483      31.036  -8.822  17.017  1.00  0.00           C  \\nATOM   7551  O   GLN B 483      31.367  -7.622  17.317  1.00  0.00           O  \\nATOM   7552  CB  GLN B 483      29.274  -8.771  15.312  1.00  0.00           C  \\nATOM   7553  CG  GLN B 483      28.119  -9.455  14.681  1.00  0.00           C  \\nATOM   7554  CD  GLN B 483      28.122  -8.996  13.197  1.00  0.00           C  \\nATOM   7555  OE1 GLN B 483      27.733  -7.888  12.893  1.00  0.00           O  \\nATOM   7556  NE2 GLN B 483      28.708  -9.738  12.372  1.00  0.00           N  \\nATOM   7557  H   GLN B 483      28.482  -7.717  17.870  1.00  0.00           H  \\nATOM   7558  HA  GLN B 483      29.605 -10.315  16.780  1.00  0.00           H  \\nATOM   7559 1HB  GLN B 483      29.056  -7.703  15.308  1.00  0.00           H  \\nATOM   7560 2HB  GLN B 483      30.136  -8.932  14.664  1.00  0.00           H  \\nATOM   7561 1HG  GLN B 483      28.256 -10.533  14.769  1.00  0.00           H  \\nATOM   7562 2HG  GLN B 483      27.206  -9.152  15.194  1.00  0.00           H  \\nATOM   7563 1HE2 GLN B 483      28.737  -9.485  11.404  1.00  0.00           H  \\nATOM   7564 2HE2 GLN B 483      29.146 -10.582  12.681  1.00  0.00           H  \\nATOM   7565  N   ARG B 484      31.985  -9.762  16.782  1.00  0.00           N  \\nATOM   7566  CA  ARG B 484      33.401  -9.333  16.873  1.00  0.00           C  \\nATOM   7567  C   ARG B 484      33.798  -8.623  15.609  1.00  0.00           C  \\nATOM   7568  O   ARG B 484      32.997  -8.599  14.620  1.00  0.00           O  \\nATOM   7569  CB  ARG B 484      34.348 -10.524  17.126  1.00  0.00           C  \\nATOM   7570  CG  ARG B 484      34.016 -11.343  18.349  1.00  0.00           C  \\nATOM   7571  CD  ARG B 484      34.842 -12.710  18.257  1.00  0.00           C  \\nATOM   7572  NE  ARG B 484      34.189 -13.883  18.903  1.00  0.00           N  \\nATOM   7573  CZ  ARG B 484      34.218 -14.034  20.214  1.00  0.00           C  \\nATOM   7574  NH1 ARG B 484      34.863 -13.121  20.930  1.00  0.00           N  \\nATOM   7575  NH2 ARG B 484      33.633 -15.043  20.820  1.00  0.00           N  \\nATOM   7576  H   ARG B 484      31.776 -10.723  16.552  1.00  0.00           H  \\nATOM   7577  HA  ARG B 484      33.500  -8.641  17.710  1.00  0.00           H  \\nATOM   7578 1HB  ARG B 484      34.331 -11.191  16.265  1.00  0.00           H  \\nATOM   7579 2HB  ARG B 484      35.369 -10.160  17.238  1.00  0.00           H  \\nATOM   7580 1HG  ARG B 484      34.298 -10.789  19.245  1.00  0.00           H  \\nATOM   7581 2HG  ARG B 484      32.945 -11.548  18.370  1.00  0.00           H  \\nATOM   7582 1HD  ARG B 484      34.993 -12.974  17.211  1.00  0.00           H  \\nATOM   7583 2HD  ARG B 484      35.809 -12.584  18.743  1.00  0.00           H  \\nATOM   7584  HE  ARG B 484      33.727 -14.553  18.303  1.00  0.00           H  \\nATOM   7585 1HH1 ARG B 484      35.312 -12.342  20.469  1.00  0.00           H  \\nATOM   7586 2HH1 ARG B 484      34.905 -13.205  21.935  1.00  0.00           H  \\nATOM   7587 1HH2 ARG B 484      33.138 -15.739  20.279  1.00  0.00           H  \\nATOM   7588 2HH2 ARG B 484      33.678 -15.123  21.825  1.00  0.00           H  \\nATOM   7589  N   ARG B 485      35.014  -8.091  15.509  1.00  0.00           N  \\nATOM   7590  CA  ARG B 485      35.459  -7.440  14.265  1.00  0.00           C  \\nATOM   7591  C   ARG B 485      35.425  -8.328  13.008  1.00  0.00           C  \\nATOM   7592  O   ARG B 485      35.196  -7.848  11.928  1.00  0.00           O  \\nATOM   7593  CB  ARG B 485      36.873  -6.874  14.375  1.00  0.00           C  \\nATOM   7594  CG  ARG B 485      36.948  -5.830  15.495  1.00  0.00           C  \\nATOM   7595  CD  ARG B 485      38.272  -5.145  15.519  1.00  0.00           C  \\nATOM   7596  NE  ARG B 485      38.500  -4.281  16.690  1.00  0.00           N  \\nATOM   7597  CZ  ARG B 485      38.243  -2.968  16.649  1.00  0.00           C  \\nATOM   7598  NH1 ARG B 485      37.675  -2.438  15.584  1.00  0.00           N  \\nATOM   7599  NH2 ARG B 485      38.523  -2.199  17.685  1.00  0.00           N  \\nATOM   7600  H   ARG B 485      35.646  -8.132  16.296  1.00  0.00           H  \\nATOM   7601  HA  ARG B 485      34.786  -6.609  14.049  1.00  0.00           H  \\nATOM   7602 1HB  ARG B 485      37.574  -7.683  14.575  1.00  0.00           H  \\nATOM   7603 2HB  ARG B 485      37.158  -6.421  13.426  1.00  0.00           H  \\nATOM   7604 1HG  ARG B 485      36.174  -5.077  15.343  1.00  0.00           H  \\nATOM   7605 2HG  ARG B 485      36.796  -6.318  16.458  1.00  0.00           H  \\nATOM   7606 1HD  ARG B 485      39.067  -5.890  15.520  1.00  0.00           H  \\nATOM   7607 2HD  ARG B 485      38.370  -4.511  14.638  1.00  0.00           H  \\nATOM   7608  HE  ARG B 485      38.859  -4.714  17.531  1.00  0.00           H  \\nATOM   7609 1HH1 ARG B 485      37.433  -3.023  14.797  1.00  0.00           H  \\nATOM   7610 2HH1 ARG B 485      37.482  -1.447  15.555  1.00  0.00           H  \\nATOM   7611 1HH2 ARG B 485      38.934  -2.601  18.516  1.00  0.00           H  \\nATOM   7612 2HH2 ARG B 485      38.326  -1.210  17.647  1.00  0.00           H  \\nATOM   7613  N   ASN B 486      35.628  -9.621  13.190  1.00  0.00           N  \\nATOM   7614  CA  ASN B 486      35.574 -10.509  12.044  1.00  0.00           C  \\nATOM   7615  C   ASN B 486      34.163 -10.977  11.712  1.00  0.00           C  \\nATOM   7616  O   ASN B 486      33.954 -11.809  10.786  1.00  0.00           O  \\nATOM   7617  CB  ASN B 486      36.497 -11.684  12.296  1.00  0.00           C  \\nATOM   7618  CG  ASN B 486      36.124 -12.548  13.498  1.00  0.00           C  \\nATOM   7619  OD1 ASN B 486      35.033 -12.482  14.108  1.00  0.00           O  \\nATOM   7620  ND2 ASN B 486      37.071 -13.427  13.850  1.00  0.00           N  \\nATOM   7621  H   ASN B 486      35.819 -10.008  14.103  1.00  0.00           H  \\nATOM   7622  HA  ASN B 486      35.915  -9.960  11.164  1.00  0.00           H  \\nATOM   7623 1HB  ASN B 486      36.512 -12.330  11.417  1.00  0.00           H  \\nATOM   7624 2HB  ASN B 486      37.513 -11.321  12.453  1.00  0.00           H  \\nATOM   7625 1HD2 ASN B 486      36.921 -14.039  14.627  1.00  0.00           H  \\nATOM   7626 2HD2 ASN B 486      37.927 -13.471  13.337  1.00  0.00           H  \\nATOM   7627  N   GLY B 487      33.145 -10.482  12.395  1.00  0.00           N  \\nATOM   7628  CA  GLY B 487      31.773 -10.872  12.069  1.00  0.00           C  \\nATOM   7629  C   GLY B 487      31.259 -11.998  12.967  1.00  0.00           C  \\nATOM   7630  O   GLY B 487      30.018 -12.205  13.034  1.00  0.00           O  \\nATOM   7631  H   GLY B 487      33.305  -9.829  13.149  1.00  0.00           H  \\nATOM   7632 1HA  GLY B 487      31.116 -10.008  12.169  1.00  0.00           H  \\nATOM   7633 2HA  GLY B 487      31.724 -11.195  11.030  1.00  0.00           H  \\nATOM   7634  N   SER B 488      32.142 -12.705  13.663  1.00  0.00           N  \\nATOM   7635  CA  SER B 488      31.699 -13.831  14.562  1.00  0.00           C  \\nATOM   7636  C   SER B 488      30.765 -13.376  15.691  1.00  0.00           C  \\nATOM   7637  O   SER B 488      30.881 -12.226  16.130  1.00  0.00           O  \\nATOM   7638  CB  SER B 488      32.861 -14.656  15.115  1.00  0.00           C  \\nATOM   7639  OG  SER B 488      33.718 -13.848  15.894  1.00  0.00           O  \\nATOM   7640  H   SER B 488      33.126 -12.490  13.593  1.00  0.00           H  \\nATOM   7641  HA  SER B 488      31.068 -14.508  13.984  1.00  0.00           H  \\nATOM   7642 1HB  SER B 488      32.471 -15.473  15.721  1.00  0.00           H  \\nATOM   7643 2HB  SER B 488      33.418 -15.098  14.290  1.00  0.00           H  \\nATOM   7644  HG  SER B 488      33.340 -12.966  15.875  1.00  0.00           H  \\nATOM   7645  N   LYS B 489      29.769 -14.195  16.064  1.00  0.00           N  \\nATOM   7646  CA  LYS B 489      28.754 -13.778  17.005  1.00  0.00           C  \\nATOM   7647  C   LYS B 489      29.132 -14.256  18.423  1.00  0.00           C  \\nATOM   7648  O   LYS B 489      29.743 -15.348  18.600  1.00  0.00           O  \\nATOM   7649  CB  LYS B 489      27.385 -14.321  16.591  1.00  0.00           C  \\nATOM   7650  CG  LYS B 489      26.863 -13.769  15.271  1.00  0.00           C  \\nATOM   7651  CD  LYS B 489      25.485 -14.327  14.947  1.00  0.00           C  \\nATOM   7652  CE  LYS B 489      24.964 -13.783  13.624  1.00  0.00           C  \\nATOM   7653  NZ  LYS B 489      23.623 -14.332  13.286  1.00  0.00           N  \\nATOM   7654  H   LYS B 489      29.728 -15.127  15.677  1.00  0.00           H  \\nATOM   7655  HA  LYS B 489      28.710 -12.688  17.006  1.00  0.00           H  \\nATOM   7656 1HB  LYS B 489      27.436 -15.407  16.503  1.00  0.00           H  \\nATOM   7657 2HB  LYS B 489      26.653 -14.089  17.364  1.00  0.00           H  \\nATOM   7658 1HG  LYS B 489      26.801 -12.682  15.330  1.00  0.00           H  \\nATOM   7659 2HG  LYS B 489      27.550 -14.034  14.469  1.00  0.00           H  \\nATOM   7660 1HD  LYS B 489      25.537 -15.415  14.887  1.00  0.00           H  \\nATOM   7661 2HD  LYS B 489      24.787 -14.056  15.739  1.00  0.00           H  \\nATOM   7662 1HE  LYS B 489      24.896 -12.697  13.679  1.00  0.00           H  \\nATOM   7663 2HE  LYS B 489      25.660 -14.040  12.826  1.00  0.00           H  \\nATOM   7664 1HZ  LYS B 489      23.315 -13.948  12.404  1.00  0.00           H  \\nATOM   7665 2HZ  LYS B 489      23.679 -15.338  13.213  1.00  0.00           H  \\nATOM   7666 3HZ  LYS B 489      22.965 -14.083  14.009  1.00  0.00           H  \\nATOM   7667  N   PHE B 490      28.883 -13.430  19.427  1.00  0.00           N  \\nATOM   7668  CA  PHE B 490      29.092 -13.820  20.825  1.00  0.00           C  \\nATOM   7669  C   PHE B 490      28.025 -13.136  21.695  1.00  0.00           C  \\nATOM   7670  O   PHE B 490      27.369 -12.150  21.261  1.00  0.00           O  \\nATOM   7671  CB  PHE B 490      30.538 -13.502  21.373  1.00  0.00           C  \\nATOM   7672  CG  PHE B 490      30.825 -12.028  21.570  1.00  0.00           C  \\nATOM   7673  CD1 PHE B 490      31.062 -11.199  20.472  1.00  0.00           C  \\nATOM   7674  CD2 PHE B 490      30.886 -11.505  22.845  1.00  0.00           C  \\nATOM   7675  CE1 PHE B 490      31.286  -9.874  20.617  1.00  0.00           C  \\nATOM   7676  CE2 PHE B 490      31.147 -10.169  23.048  1.00  0.00           C  \\nATOM   7677  CZ  PHE B 490      31.372  -9.335  21.924  1.00  0.00           C  \\nATOM   7678  H   PHE B 490      28.539 -12.503  19.221  1.00  0.00           H  \\nATOM   7679  HA  PHE B 490      28.952 -14.899  20.908  1.00  0.00           H  \\nATOM   7680 1HB  PHE B 490      30.682 -14.000  22.331  1.00  0.00           H  \\nATOM   7681 2HB  PHE B 490      31.283 -13.897  20.684  1.00  0.00           H  \\nATOM   7682  HD1 PHE B 490      31.066 -11.630  19.470  1.00  0.00           H  \\nATOM   7683  HD2 PHE B 490      30.711 -12.162  23.698  1.00  0.00           H  \\nATOM   7684  HE1 PHE B 490      31.397  -9.241  19.737  1.00  0.00           H  \\nATOM   7685  HE2 PHE B 490      31.183  -9.749  24.053  1.00  0.00           H  \\nATOM   7686  HZ  PHE B 490      31.612  -8.282  22.066  1.00  0.00           H  \\nATOM   7687  N   SER B 491      27.919 -13.630  22.947  1.00  0.00           N  \\nATOM   7688  CA  SER B 491      26.919 -13.165  23.858  1.00  0.00           C  \\nATOM   7689  C   SER B 491      27.599 -12.453  25.005  1.00  0.00           C  \\nATOM   7690  O   SER B 491      28.635 -12.906  25.476  1.00  0.00           O  \\nATOM   7691  CB  SER B 491      26.146 -14.364  24.398  1.00  0.00           C  \\nATOM   7692  OG  SER B 491      25.481 -14.976  23.308  1.00  0.00           O  \\nATOM   7693  H   SER B 491      28.563 -14.347  23.248  1.00  0.00           H  \\nATOM   7694  HA  SER B 491      26.237 -12.508  23.317  1.00  0.00           H  \\nATOM   7695 1HB  SER B 491      26.837 -15.056  24.879  1.00  0.00           H  \\nATOM   7696 2HB  SER B 491      25.439 -14.029  25.155  1.00  0.00           H  \\nATOM   7697  HG  SER B 491      25.709 -14.456  22.533  1.00  0.00           H  \\nATOM   7698  N   VAL B 492      26.965 -11.371  25.443  1.00  0.00           N  \\nATOM   7699  CA  VAL B 492      27.307 -10.679  26.720  1.00  0.00           C  \\nATOM   7700  C   VAL B 492      26.221 -10.971  27.754  1.00  0.00           C  \\nATOM   7701  O   VAL B 492      25.012 -10.907  27.445  1.00  0.00           O  \\nATOM   7702  CB  VAL B 492      27.488  -9.117  26.583  1.00  0.00           C  \\nATOM   7703  CG1 VAL B 492      28.705  -8.752  25.660  1.00  0.00           C  \\nATOM   7704  CG2 VAL B 492      26.202  -8.444  26.106  1.00  0.00           C  \\nATOM   7705  H   VAL B 492      26.215 -11.008  24.873  1.00  0.00           H  \\nATOM   7706  HA  VAL B 492      28.255 -11.075  27.085  1.00  0.00           H  \\nATOM   7707  HB  VAL B 492      27.758  -8.705  27.555  1.00  0.00           H  \\nATOM   7708 1HG1 VAL B 492      28.797  -7.668  25.591  1.00  0.00           H  \\nATOM   7709 2HG1 VAL B 492      29.620  -9.167  26.083  1.00  0.00           H  \\nATOM   7710 3HG1 VAL B 492      28.544  -9.166  24.665  1.00  0.00           H  \\nATOM   7711 1HG2 VAL B 492      26.363  -7.369  26.024  1.00  0.00           H  \\nATOM   7712 2HG2 VAL B 492      25.921  -8.845  25.132  1.00  0.00           H  \\nATOM   7713 3HG2 VAL B 492      25.403  -8.637  26.822  1.00  0.00           H  \\nATOM   7714  N   TYR B 493      26.653 -11.330  28.978  1.00  0.00           N  \\nATOM   7715  CA  TYR B 493      25.772 -11.827  29.991  1.00  0.00           C  \\nATOM   7716  C   TYR B 493      25.647 -10.865  31.159  1.00  0.00           C  \\nATOM   7717  O   TYR B 493      24.804 -11.043  31.996  1.00  0.00           O  \\nATOM   7718  CB  TYR B 493      26.191 -13.249  30.511  1.00  0.00           C  \\nATOM   7719  CG  TYR B 493      26.165 -14.348  29.433  1.00  0.00           C  \\nATOM   7720  CD1 TYR B 493      27.266 -14.556  28.591  1.00  0.00           C  \\nATOM   7721  CD2 TYR B 493      25.031 -15.128  29.247  1.00  0.00           C  \\nATOM   7722  CE1 TYR B 493      27.255 -15.545  27.600  1.00  0.00           C  \\nATOM   7723  CE2 TYR B 493      24.984 -16.089  28.251  1.00  0.00           C  \\nATOM   7724  CZ  TYR B 493      26.091 -16.306  27.447  1.00  0.00           C  \\nATOM   7725  OH  TYR B 493      26.026 -17.298  26.477  1.00  0.00           O  \\nATOM   7726  H   TYR B 493      27.638 -11.243  29.182  1.00  0.00           H  \\nATOM   7727  HA  TYR B 493      24.771 -11.912  29.567  1.00  0.00           H  \\nATOM   7728 1HB  TYR B 493      27.201 -13.204  30.921  1.00  0.00           H  \\nATOM   7729 2HB  TYR B 493      25.524 -13.553  31.317  1.00  0.00           H  \\nATOM   7730  HD1 TYR B 493      28.159 -13.940  28.699  1.00  0.00           H  \\nATOM   7731  HD2 TYR B 493      24.161 -14.987  29.889  1.00  0.00           H  \\nATOM   7732  HE1 TYR B 493      28.126 -15.684  26.960  1.00  0.00           H  \\nATOM   7733  HE2 TYR B 493      24.076 -16.673  28.100  1.00  0.00           H  \\nATOM   7734  HH  TYR B 493      25.163 -17.718  26.507  1.00  0.00           H  \\nATOM   7735  N   ASP B 494      26.521  -9.876  31.225  1.00  0.00           N  \\nATOM   7736  CA  ASP B 494      26.554  -8.958  32.367  1.00  0.00           C  \\nATOM   7737  C   ASP B 494      27.135  -7.626  32.031  1.00  0.00           C  \\nATOM   7738  O   ASP B 494      27.814  -7.500  31.047  1.00  0.00           O  \\nATOM   7739  CB  ASP B 494      27.310  -9.551  33.578  1.00  0.00           C  \\nATOM   7740  CG  ASP B 494      28.651 -10.127  33.196  1.00  0.00           C  \\nATOM   7741  OD1 ASP B 494      29.528  -9.428  32.595  1.00  0.00           O  \\nATOM   7742  OD2 ASP B 494      28.855 -11.322  33.524  1.00  0.00           O  \\nATOM   7743  H   ASP B 494      27.182  -9.749  30.472  1.00  0.00           H  \\nATOM   7744  HA  ASP B 494      25.529  -8.759  32.681  1.00  0.00           H  \\nATOM   7745 1HB  ASP B 494      27.461  -8.775  34.329  1.00  0.00           H  \\nATOM   7746 2HB  ASP B 494      26.707 -10.336  34.035  1.00  0.00           H  \\nATOM   7747  N   VAL B 495      26.908  -6.632  32.901  1.00  0.00           N  \\nATOM   7748  CA  VAL B 495      27.319  -5.265  32.564  1.00  0.00           C  \\nATOM   7749  C   VAL B 495      28.837  -5.078  32.584  1.00  0.00           C  \\nATOM   7750  O   VAL B 495      29.377  -4.227  31.847  1.00  0.00           O  \\nATOM   7751  CB  VAL B 495      26.631  -4.254  33.517  1.00  0.00           C  \\nATOM   7752  CG1 VAL B 495      27.227  -4.358  34.931  1.00  0.00           C  \\nATOM   7753  CG2 VAL B 495      26.722  -2.800  32.939  1.00  0.00           C  \\nATOM   7754  H   VAL B 495      26.458  -6.806  33.788  1.00  0.00           H  \\nATOM   7755  HA  VAL B 495      27.008  -5.053  31.540  1.00  0.00           H  \\nATOM   7756  HB  VAL B 495      25.582  -4.530  33.628  1.00  0.00           H  \\nATOM   7757 1HG1 VAL B 495      26.733  -3.642  35.588  1.00  0.00           H  \\nATOM   7758 2HG1 VAL B 495      27.076  -5.366  35.316  1.00  0.00           H  \\nATOM   7759 3HG1 VAL B 495      28.294  -4.138  34.893  1.00  0.00           H  \\nATOM   7760 1HG2 VAL B 495      26.233  -2.106  33.622  1.00  0.00           H  \\nATOM   7761 2HG2 VAL B 495      27.769  -2.519  32.824  1.00  0.00           H  \\nATOM   7762 3HG2 VAL B 495      26.227  -2.763  31.968  1.00  0.00           H  \\nATOM   7763  N   SER B 496      29.563  -5.847  33.391  1.00  0.00           N  \\nATOM   7764  CA  SER B 496      30.975  -5.523  33.629  1.00  0.00           C  \\nATOM   7765  C   SER B 496      31.768  -5.341  32.361  1.00  0.00           C  \\nATOM   7766  O   SER B 496      32.474  -4.347  32.227  1.00  0.00           O  \\nATOM   7767  CB  SER B 496      31.676  -6.579  34.540  1.00  0.00           C  \\nATOM   7768  OG  SER B 496      31.337  -7.884  34.161  1.00  0.00           O  \\nATOM   7769  H   SER B 496      29.160  -6.654  33.846  1.00  0.00           H  \\nATOM   7770  HA  SER B 496      31.028  -4.558  34.135  1.00  0.00           H  \\nATOM   7771 1HB  SER B 496      32.757  -6.452  34.479  1.00  0.00           H  \\nATOM   7772 2HB  SER B 496      31.385  -6.415  35.577  1.00  0.00           H  \\nATOM   7773  HG  SER B 496      30.741  -7.796  33.414  1.00  0.00           H  \\nATOM   7774  N   ILE B 497      31.689  -6.306  31.432  1.00  0.00           N  \\nATOM   7775  CA  ILE B 497      32.426  -6.245  30.184  1.00  0.00           C  \\nATOM   7776  C   ILE B 497      32.103  -5.016  29.300  1.00  0.00           C  \\nATOM   7777  O   ILE B 497      32.887  -4.663  28.430  1.00  0.00           O  \\nATOM   7778  CB  ILE B 497      32.270  -7.604  29.361  1.00  0.00           C  \\nATOM   7779  CG1 ILE B 497      33.443  -7.760  28.411  1.00  0.00           C  \\nATOM   7780  CG2 ILE B 497      30.947  -7.716  28.684  1.00  0.00           C  \\nATOM   7781  CD1 ILE B 497      34.681  -8.478  28.951  1.00  0.00           C  \\nATOM   7782  H   ILE B 497      31.093  -7.101  31.613  1.00  0.00           H  \\nATOM   7783  HA  ILE B 497      33.481  -6.101  30.412  1.00  0.00           H  \\nATOM   7784  HB  ILE B 497      32.372  -8.452  30.037  1.00  0.00           H  \\nATOM   7785 1HG1 ILE B 497      33.124  -8.311  27.527  1.00  0.00           H  \\nATOM   7786 2HG1 ILE B 497      33.776  -6.776  28.079  1.00  0.00           H  \\nATOM   7787 1HG2 ILE B 497      30.896  -8.658  28.139  1.00  0.00           H  \\nATOM   7788 2HG2 ILE B 497      30.153  -7.685  29.429  1.00  0.00           H  \\nATOM   7789 3HG2 ILE B 497      30.823  -6.887  27.987  1.00  0.00           H  \\nATOM   7790 1HD1 ILE B 497      35.444  -8.522  28.174  1.00  0.00           H  \\nATOM   7791 2HD1 ILE B 497      35.071  -7.934  29.812  1.00  0.00           H  \\nATOM   7792 3HD1 ILE B 497      34.412  -9.490  29.252  1.00  0.00           H  \\nATOM   7793  N   LEU B 498      30.918  -4.423  29.545  1.00  0.00           N  \\nATOM   7794  CA  LEU B 498      30.430  -3.319  28.712  1.00  0.00           C  \\nATOM   7795  C   LEU B 498      30.796  -1.994  29.257  1.00  0.00           C  \\nATOM   7796  O   LEU B 498      30.724  -1.005  28.563  1.00  0.00           O  \\nATOM   7797  CB  LEU B 498      28.916  -3.403  28.412  1.00  0.00           C  \\nATOM   7798  CG  LEU B 498      28.440  -4.680  27.699  1.00  0.00           C  \\nATOM   7799  CD1 LEU B 498      26.967  -4.567  27.498  1.00  0.00           C  \\nATOM   7800  CD2 LEU B 498      29.093  -4.745  26.346  1.00  0.00           C  \\nATOM   7801  H   LEU B 498      30.352  -4.744  30.318  1.00  0.00           H  \\nATOM   7802  HA  LEU B 498      30.951  -3.351  27.756  1.00  0.00           H  \\nATOM   7803 1HB  LEU B 498      28.371  -3.329  29.352  1.00  0.00           H  \\nATOM   7804 2HB  LEU B 498      28.638  -2.555  27.787  1.00  0.00           H  \\nATOM   7805  HG  LEU B 498      28.721  -5.553  28.289  1.00  0.00           H  \\nATOM   7806 1HD1 LEU B 498      26.599  -5.460  26.993  1.00  0.00           H  \\nATOM   7807 2HD1 LEU B 498      26.474  -4.469  28.465  1.00  0.00           H  \\nATOM   7808 3HD1 LEU B 498      26.749  -3.691  26.889  1.00  0.00           H  \\nATOM   7809 1HD2 LEU B 498      28.766  -5.647  25.827  1.00  0.00           H  \\nATOM   7810 2HD2 LEU B 498      28.811  -3.868  25.763  1.00  0.00           H  \\nATOM   7811 3HD2 LEU B 498      30.176  -4.768  26.465  1.00  0.00           H  \\nATOM   7812  N   ILE B 499      31.244  -1.876  30.508  1.00  0.00           N  \\nATOM   7813  CA  ILE B 499      31.511  -0.593  31.129  1.00  0.00           C  \\nATOM   7814  C   ILE B 499      32.570   0.218  30.327  1.00  0.00           C  \\nATOM   7815  O   ILE B 499      32.372   1.388  30.176  1.00  0.00           O  \\nATOM   7816  CB  ILE B 499      31.828  -0.770  32.626  1.00  0.00           C  \\nATOM   7817  CG1 ILE B 499      30.584  -1.155  33.344  1.00  0.00           C  \\nATOM   7818  CG2 ILE B 499      32.181   0.553  33.239  1.00  0.00           C  \\nATOM   7819  CD1 ILE B 499      30.793  -1.685  34.841  1.00  0.00           C  \\nATOM   7820  H   ILE B 499      31.401  -2.721  31.038  1.00  0.00           H  \\nATOM   7821  HA  ILE B 499      30.622   0.029  31.034  1.00  0.00           H  \\nATOM   7822  HB  ILE B 499      32.669  -1.453  32.741  1.00  0.00           H  \\nATOM   7823 1HG1 ILE B 499      29.912  -0.299  33.391  1.00  0.00           H  \\nATOM   7824 2HG1 ILE B 499      30.072  -1.941  32.789  1.00  0.00           H  \\nATOM   7825 1HG2 ILE B 499      32.403   0.416  34.297  1.00  0.00           H  \\nATOM   7826 2HG2 ILE B 499      33.055   0.965  32.736  1.00  0.00           H  \\nATOM   7827 3HG2 ILE B 499      31.342   1.240  33.130  1.00  0.00           H  \\nATOM   7828 1HD1 ILE B 499      29.826  -1.939  35.275  1.00  0.00           H  \\nATOM   7829 2HD1 ILE B 499      31.429  -2.571  34.828  1.00  0.00           H  \\nATOM   7830 3HD1 ILE B 499      31.265  -0.907  35.439  1.00  0.00           H  \\nATOM   7831  N   PRO B 500      33.622  -0.411  29.761  1.00  0.00           N  \\nATOM   7832  CA  PRO B 500      34.563   0.376  28.984  1.00  0.00           C  \\nATOM   7833  C   PRO B 500      34.041   0.791  27.610  1.00  0.00           C  \\nATOM   7834  O   PRO B 500      34.715   1.531  26.928  1.00  0.00           O  \\nATOM   7835  CB  PRO B 500      35.789  -0.574  28.784  1.00  0.00           C  \\nATOM   7836  CG  PRO B 500      35.619  -1.559  29.907  1.00  0.00           C  \\nATOM   7837  CD  PRO B 500      34.148  -1.789  29.976  1.00  0.00           C  \\nATOM   7838  HA  PRO B 500      34.857   1.264  29.563  1.00  0.00           H  \\nATOM   7839 1HB  PRO B 500      35.751  -1.032  27.784  1.00  0.00           H  \\nATOM   7840 2HB  PRO B 500      36.724   0.002  28.839  1.00  0.00           H  \\nATOM   7841 1HG  PRO B 500      36.183  -2.479  29.694  1.00  0.00           H  \\nATOM   7842 2HG  PRO B 500      36.028  -1.145  30.840  1.00  0.00           H  \\nATOM   7843 1HD  PRO B 500      33.847  -2.482  29.177  1.00  0.00           H  \\nATOM   7844 2HD  PRO B 500      33.886  -2.197  30.963  1.00  0.00           H  \\nATOM   7845  N   ILE B 501      32.963   0.143  27.163  1.00  0.00           N  \\nATOM   7846  CA  ILE B 501      32.442   0.216  25.827  1.00  0.00           C  \\nATOM   7847  C   ILE B 501      31.350   1.264  25.609  1.00  0.00           C  \\nATOM   7848  O   ILE B 501      31.381   2.001  24.611  1.00  0.00           O  \\nATOM   7849  CB  ILE B 501      31.873  -1.149  25.452  1.00  0.00           C  \\nATOM   7850  CG1 ILE B 501      32.900  -2.267  25.562  1.00  0.00           C  \\nATOM   7851  CG2 ILE B 501      31.322  -1.150  23.978  1.00  0.00           C  \\nATOM   7852  CD1 ILE B 501      34.223  -1.929  24.906  1.00  0.00           C  \\nATOM   7853  H   ILE B 501      32.495  -0.440  27.842  1.00  0.00           H  \\nATOM   7854  HA  ILE B 501      33.258   0.469  25.151  1.00  0.00           H  \\nATOM   7855  HB  ILE B 501      31.057  -1.401  26.128  1.00  0.00           H  \\nATOM   7856 1HG1 ILE B 501      33.081  -2.493  26.612  1.00  0.00           H  \\nATOM   7857 2HG1 ILE B 501      32.505  -3.171  25.098  1.00  0.00           H  \\nATOM   7858 1HG2 ILE B 501      30.924  -2.136  23.739  1.00  0.00           H  \\nATOM   7859 2HG2 ILE B 501      30.531  -0.407  23.885  1.00  0.00           H  \\nATOM   7860 3HG2 ILE B 501      32.130  -0.908  23.287  1.00  0.00           H  \\nATOM   7861 1HD1 ILE B 501      34.910  -2.768  25.020  1.00  0.00           H  \\nATOM   7862 2HD1 ILE B 501      34.064  -1.732  23.845  1.00  0.00           H  \\nATOM   7863 3HD1 ILE B 501      34.648  -1.045  25.380  1.00  0.00           H  \\nATOM   7864  N   ILE B 502      30.383   1.269  26.509  1.00  0.00           N  \\nATOM   7865  CA  ILE B 502      29.262   2.165  26.391  1.00  0.00           C  \\nATOM   7866  C   ILE B 502      29.371   3.320  27.360  1.00  0.00           C  \\nATOM   7867  O   ILE B 502      29.684   3.143  28.614  1.00  0.00           O  \\nATOM   7868  CB  ILE B 502      27.960   1.415  26.610  1.00  0.00           C  \\nATOM   7869  CG1 ILE B 502      26.670   2.279  26.409  1.00  0.00           C  \\nATOM   7870  CG2 ILE B 502      27.911   0.818  27.995  1.00  0.00           C  \\nATOM   7871  CD1 ILE B 502      25.382   1.298  26.182  1.00  0.00           C  \\nATOM   7872  H   ILE B 502      30.426   0.636  27.295  1.00  0.00           H  \\nATOM   7873  HA  ILE B 502      29.259   2.584  25.385  1.00  0.00           H  \\nATOM   7874  HB  ILE B 502      27.876   0.612  25.879  1.00  0.00           H  \\nATOM   7875 1HG1 ILE B 502      26.517   2.912  27.282  1.00  0.00           H  \\nATOM   7876 2HG1 ILE B 502      26.802   2.936  25.549  1.00  0.00           H  \\nATOM   7877 1HG2 ILE B 502      26.969   0.286  28.129  1.00  0.00           H  \\nATOM   7878 2HG2 ILE B 502      28.741   0.124  28.121  1.00  0.00           H  \\nATOM   7879 3HG2 ILE B 502      27.987   1.613  28.737  1.00  0.00           H  \\nATOM   7880 1HD1 ILE B 502      24.485   1.902  26.042  1.00  0.00           H  \\nATOM   7881 2HD1 ILE B 502      25.546   0.681  25.298  1.00  0.00           H  \\nATOM   7882 3HD1 ILE B 502      25.255   0.658  27.054  1.00  0.00           H  \\nATOM   7883  N   ALA B 503      29.168   4.548  26.847  1.00  0.00           N  \\nATOM   7884  CA  ALA B 503      29.350   5.779  27.597  1.00  0.00           C  \\nATOM   7885  C   ALA B 503      28.127   6.440  28.077  1.00  0.00           C  \\nATOM   7886  O   ALA B 503      28.263   7.250  29.058  1.00  0.00           O  \\nATOM   7887  CB  ALA B 503      30.159   6.842  26.672  1.00  0.00           C  \\nATOM   7888  H   ALA B 503      28.872   4.599  25.883  1.00  0.00           H  \\nATOM   7889  HA  ALA B 503      29.935   5.549  28.488  1.00  0.00           H  \\nATOM   7890 1HB  ALA B 503      30.302   7.771  27.225  1.00  0.00           H  \\nATOM   7891 2HB  ALA B 503      31.130   6.426  26.404  1.00  0.00           H  \\nATOM   7892 3HB  ALA B 503      29.589   7.043  25.766  1.00  0.00           H  \\nATOM   7893  N   LEU B 504      26.992   6.175  27.400  1.00  0.00           N  \\nATOM   7894  CA  LEU B 504      25.734   6.916  27.659  1.00  0.00           C  \\nATOM   7895  C   LEU B 504      24.626   5.871  27.392  1.00  0.00           C  \\nATOM   7896  O   LEU B 504      24.716   5.013  26.538  1.00  0.00           O  \\nATOM   7897  CB  LEU B 504      25.429   8.042  26.563  1.00  0.00           C  \\nATOM   7898  CG  LEU B 504      26.321   9.231  26.678  1.00  0.00           C  \\nATOM   7899  CD1 LEU B 504      25.967  10.074  25.381  1.00  0.00           C  \\nATOM   7900  CD2 LEU B 504      26.330  10.047  27.881  1.00  0.00           C  \\nATOM   7901  H   LEU B 504      27.000   5.451  26.696  1.00  0.00           H  \\nATOM   7902  HA  LEU B 504      25.820   7.416  28.623  1.00  0.00           H  \\nATOM   7903 1HB  LEU B 504      25.548   7.604  25.573  1.00  0.00           H  \\nATOM   7904 2HB  LEU B 504      24.393   8.363  26.674  1.00  0.00           H  \\nATOM   7905  HG  LEU B 504      27.360   8.917  26.577  1.00  0.00           H  \\nATOM   7906 1HD1 LEU B 504      26.571  10.981  25.361  1.00  0.00           H  \\nATOM   7907 2HD1 LEU B 504      26.177   9.479  24.492  1.00  0.00           H  \\nATOM   7908 3HD1 LEU B 504      24.911  10.341  25.398  1.00  0.00           H  \\nATOM   7909 1HD2 LEU B 504      27.047  10.861  27.764  1.00  0.00           H  \\nATOM   7910 2HD2 LEU B 504      25.335  10.461  28.047  1.00  0.00           H  \\nATOM   7911 3HD2 LEU B 504      26.615   9.433  28.735  1.00  0.00           H  \\nATOM   7912  N   MET B 505      23.528   5.942  28.127  1.00  0.00           N  \\nATOM   7913  CA  MET B 505      22.343   5.108  27.851  1.00  0.00           C  \\nATOM   7914  C   MET B 505      21.094   5.930  27.826  1.00  0.00           C  \\nATOM   7915  O   MET B 505      20.985   6.930  28.587  1.00  0.00           O  \\nATOM   7916  CB  MET B 505      22.210   4.124  28.977  1.00  0.00           C  \\nATOM   7917  CG  MET B 505      23.286   3.098  29.085  1.00  0.00           C  \\nATOM   7918  SD  MET B 505      22.896   1.833  30.395  1.00  0.00           S  \\nATOM   7919  CE  MET B 505      22.964   2.774  31.934  1.00  0.00           C  \\nATOM   7920  H   MET B 505      23.504   6.590  28.902  1.00  0.00           H  \\nATOM   7921  HA  MET B 505      22.503   4.580  26.911  1.00  0.00           H  \\nATOM   7922 1HB  MET B 505      22.186   4.658  29.926  1.00  0.00           H  \\nATOM   7923 2HB  MET B 505      21.267   3.585  28.880  1.00  0.00           H  \\nATOM   7924 1HG  MET B 505      23.404   2.593  28.127  1.00  0.00           H  \\nATOM   7925 2HG  MET B 505      24.229   3.585  29.332  1.00  0.00           H  \\nATOM   7926 1HE  MET B 505      22.747   2.114  32.774  1.00  0.00           H  \\nATOM   7927 2HE  MET B 505      23.960   3.202  32.056  1.00  0.00           H  \\nATOM   7928 3HE  MET B 505      22.226   3.577  31.903  1.00  0.00           H  \\nATOM   7929  N   VAL B 506      20.180   5.548  26.975  1.00  0.00           N  \\nATOM   7930  CA  VAL B 506      18.859   6.139  26.978  1.00  0.00           C  \\nATOM   7931  C   VAL B 506      18.147   5.714  28.264  1.00  0.00           C  \\nATOM   7932  O   VAL B 506      18.273   4.604  28.686  1.00  0.00           O  \\nATOM   7933  CB  VAL B 506      18.054   5.749  25.717  1.00  0.00           C  \\nATOM   7934  CG1 VAL B 506      17.999   4.324  25.600  1.00  0.00           C  \\nATOM   7935  CG2 VAL B 506      16.664   6.389  25.698  1.00  0.00           C  \\nATOM   7936  H   VAL B 506      20.395   4.827  26.301  1.00  0.00           H  \\nATOM   7937  HA  VAL B 506      18.965   7.224  26.988  1.00  0.00           H  \\nATOM   7938  HB  VAL B 506      18.598   6.082  24.833  1.00  0.00           H  \\nATOM   7939 1HG1 VAL B 506      17.431   4.054  24.710  1.00  0.00           H  \\nATOM   7940 2HG1 VAL B 506      19.010   3.926  25.519  1.00  0.00           H  \\nATOM   7941 3HG1 VAL B 506      17.513   3.905  26.481  1.00  0.00           H  \\nATOM   7942 1HG2 VAL B 506      16.137   6.087  24.794  1.00  0.00           H  \\nATOM   7943 2HG2 VAL B 506      16.101   6.063  26.573  1.00  0.00           H  \\nATOM   7944 3HG2 VAL B 506      16.763   7.475  25.714  1.00  0.00           H  \\nATOM   7945  N   TYR B 507      17.458   6.641  28.878  1.00  0.00           N  \\nATOM   7946  CA  TYR B 507      16.734   6.385  30.098  1.00  0.00           C  \\nATOM   7947  C   TYR B 507      15.623   5.395  29.889  1.00  0.00           C  \\nATOM   7948  O   TYR B 507      14.852   5.514  29.008  1.00  0.00           O  \\nATOM   7949  CB  TYR B 507      16.157   7.678  30.626  1.00  0.00           C  \\nATOM   7950  CG  TYR B 507      15.403   7.598  31.920  1.00  0.00           C  \\nATOM   7951  CD1 TYR B 507      14.084   7.196  31.955  1.00  0.00           C  \\nATOM   7952  CD2 TYR B 507      15.980   7.994  33.058  1.00  0.00           C  \\nATOM   7953  CE1 TYR B 507      13.409   7.154  33.148  1.00  0.00           C  \\nATOM   7954  CE2 TYR B 507      15.307   7.964  34.230  1.00  0.00           C  \\nATOM   7955  CZ  TYR B 507      14.031   7.537  34.269  1.00  0.00           C  \\nATOM   7956  OH  TYR B 507      13.375   7.528  35.456  1.00  0.00           O  \\nATOM   7957  H   TYR B 507      17.436   7.568  28.478  1.00  0.00           H  \\nATOM   7958  HA  TYR B 507      17.428   5.978  30.834  1.00  0.00           H  \\nATOM   7959 1HB  TYR B 507      16.960   8.401  30.776  1.00  0.00           H  \\nATOM   7960 2HB  TYR B 507      15.472   8.098  29.890  1.00  0.00           H  \\nATOM   7961  HD1 TYR B 507      13.576   6.911  31.033  1.00  0.00           H  \\nATOM   7962  HD2 TYR B 507      17.010   8.350  33.052  1.00  0.00           H  \\nATOM   7963  HE1 TYR B 507      12.367   6.836  33.174  1.00  0.00           H  \\nATOM   7964  HE2 TYR B 507      15.797   8.287  35.149  1.00  0.00           H  \\nATOM   7965  HH  TYR B 507      13.959   7.851  36.146  1.00  0.00           H  \\nATOM   7966  N   ARG B 508      15.577   4.409  30.759  1.00  0.00           N  \\nATOM   7967  CA  ARG B 508      14.501   3.447  30.770  1.00  0.00           C  \\nATOM   7968  C   ARG B 508      13.661   3.462  32.057  1.00  0.00           C  \\nATOM   7969  O   ARG B 508      12.455   3.466  31.978  1.00  0.00           O  \\nATOM   7970  CB  ARG B 508      15.020   2.026  30.482  1.00  0.00           C  \\nATOM   7971  CG  ARG B 508      15.692   1.839  29.147  1.00  0.00           C  \\nATOM   7972  CD  ARG B 508      14.848   2.286  27.945  1.00  0.00           C  \\nATOM   7973  NE  ARG B 508      15.367   1.709  26.722  1.00  0.00           N  \\nATOM   7974  CZ  ARG B 508      15.139   2.147  25.514  1.00  0.00           C  \\nATOM   7975  NH1 ARG B 508      14.402   3.183  25.332  1.00  0.00           N  \\nATOM   7976  NH2 ARG B 508      15.650   1.526  24.492  1.00  0.00           N  \\nATOM   7977  H   ARG B 508      16.319   4.325  31.439  1.00  0.00           H  \\nATOM   7978  HA  ARG B 508      13.789   3.716  29.989  1.00  0.00           H  \\nATOM   7979 1HB  ARG B 508      15.738   1.739  31.249  1.00  0.00           H  \\nATOM   7980 2HB  ARG B 508      14.191   1.319  30.529  1.00  0.00           H  \\nATOM   7981 1HG  ARG B 508      16.615   2.418  29.120  1.00  0.00           H  \\nATOM   7982 2HG  ARG B 508      15.921   0.783  29.000  1.00  0.00           H  \\nATOM   7983 1HD  ARG B 508      13.818   1.959  28.082  1.00  0.00           H  \\nATOM   7984 2HD  ARG B 508      14.876   3.372  27.865  1.00  0.00           H  \\nATOM   7985  HE  ARG B 508      15.960   0.894  26.802  1.00  0.00           H  \\nATOM   7986 1HH1 ARG B 508      13.997   3.661  26.125  1.00  0.00           H  \\nATOM   7987 2HH1 ARG B 508      14.227   3.520  24.396  1.00  0.00           H  \\nATOM   7988 1HH2 ARG B 508      16.223   0.705  24.634  1.00  0.00           H  \\nATOM   7989 2HH2 ARG B 508      15.475   1.863  23.557  1.00  0.00           H  \\nATOM   7990  N   CYS B 509      14.295   3.447  33.196  1.00  0.00           N  \\nATOM   7991  CA  CYS B 509      13.550   3.374  34.430  1.00  0.00           C  \\nATOM   7992  C   CYS B 509      14.196   4.210  35.475  1.00  0.00           C  \\nATOM   7993  O   CYS B 509      15.269   4.695  35.279  1.00  0.00           O  \\nATOM   7994  CB  CYS B 509      13.359   1.939  34.876  1.00  0.00           C  \\nATOM   7995  SG  CYS B 509      14.727   0.929  34.603  1.00  0.00           S  \\nATOM   7996  H   CYS B 509      15.304   3.486  33.232  1.00  0.00           H  \\nATOM   7997  HA  CYS B 509      12.566   3.814  34.266  1.00  0.00           H  \\nATOM   7998 1HB  CYS B 509      13.131   1.916  35.942  1.00  0.00           H  \\nATOM   7999 2HB  CYS B 509      12.509   1.505  34.350  1.00  0.00           H  \\nATOM   8000  HG  CYS B 509      14.184  -0.177  35.101  1.00  0.00           H  \\nATOM   8001  N   ALA B 510      13.484   4.394  36.564  1.00  0.00           N  \\nATOM   8002  CA  ALA B 510      13.919   5.255  37.627  1.00  0.00           C  \\nATOM   8003  C   ALA B 510      15.092   4.583  38.336  1.00  0.00           C  \\nATOM   8004  O   ALA B 510      15.278   3.398  38.215  1.00  0.00           O  \\nATOM   8005  OXT ALA B 510      15.834   5.235  39.018  1.00  0.00           O  \\nATOM   8006  CB  ALA B 510      12.767   5.499  38.582  1.00  0.00           C  \\nATOM   8007  H   ALA B 510      12.601   3.911  36.652  1.00  0.00           H  \\nATOM   8008  HA  ALA B 510      14.232   6.203  37.189  1.00  0.00           H  \\nATOM   8009 1HB  ALA B 510      13.096   6.153  39.390  1.00  0.00           H  \\nATOM   8010 2HB  ALA B 510      11.944   5.971  38.046  1.00  0.00           H  \\nATOM   8011 3HB  ALA B 510      12.432   4.550  38.998  1.00  0.00           H  \\nTER                                                                             \\nATOM   8013  N   GLY C 511     -17.984   5.581  27.591  1.00  0.00           N  \\nATOM   8014  CA  GLY C 511     -16.735   6.403  27.701  1.00  0.00           C  \\nATOM   8015  C   GLY C 511     -16.107   6.703  26.341  1.00  0.00           C  \\nATOM   8016  O   GLY C 511     -15.043   7.369  26.241  1.00  0.00           O  \\nATOM   8017 1H   GLY C 511     -18.355   5.413  28.504  1.00  0.00           H  \\nATOM   8018 2H   GLY C 511     -18.660   6.073  27.043  1.00  0.00           H  \\nATOM   8019 3H   GLY C 511     -17.771   4.708  27.152  1.00  0.00           H  \\nATOM   8020 1HA  GLY C 511     -16.962   7.344  28.202  1.00  0.00           H  \\nATOM   8021 2HA  GLY C 511     -16.007   5.878  28.319  1.00  0.00           H  \\nATOM   8022  N   PHE C 512     -16.804   6.300  25.279  1.00  0.00           N  \\nATOM   8023  CA  PHE C 512     -16.252   6.464  23.927  1.00  0.00           C  \\nATOM   8024  C   PHE C 512     -16.621   7.833  23.358  1.00  0.00           C  \\nATOM   8025  O   PHE C 512     -16.106   8.237  22.298  1.00  0.00           O  \\nATOM   8026  CB  PHE C 512     -16.805   5.394  23.007  1.00  0.00           C  \\nATOM   8027  CG  PHE C 512     -16.371   4.036  23.357  1.00  0.00           C  \\nATOM   8028  CD1 PHE C 512     -16.962   2.951  22.730  1.00  0.00           C  \\nATOM   8029  CD2 PHE C 512     -15.412   3.819  24.372  1.00  0.00           C  \\nATOM   8030  CE1 PHE C 512     -16.551   1.641  23.011  1.00  0.00           C  \\nATOM   8031  CE2 PHE C 512     -14.980   2.506  24.682  1.00  0.00           C  \\nATOM   8032  CZ  PHE C 512     -15.585   1.408  24.043  1.00  0.00           C  \\nATOM   8033  H   PHE C 512     -17.715   5.879  25.391  1.00  0.00           H  \\nATOM   8034  HA  PHE C 512     -15.168   6.357  23.980  1.00  0.00           H  \\nATOM   8035 1HB  PHE C 512     -17.894   5.422  23.030  1.00  0.00           H  \\nATOM   8036 2HB  PHE C 512     -16.495   5.599  21.983  1.00  0.00           H  \\nATOM   8037  HD1 PHE C 512     -17.759   3.123  22.006  1.00  0.00           H  \\nATOM   8038  HD2 PHE C 512     -14.949   4.675  24.863  1.00  0.00           H  \\nATOM   8039  HE1 PHE C 512     -16.966   0.807  22.445  1.00  0.00           H  \\nATOM   8040  HE2 PHE C 512     -14.186   2.342  25.410  1.00  0.00           H  \\nATOM   8041  HZ  PHE C 512     -15.326   0.389  24.328  1.00  0.00           H  \\nATOM   8042  N   GLY C 513     -17.412   8.616  24.133  1.00  0.00           N  \\nATOM   8043  CA  GLY C 513     -17.780   9.984  23.768  1.00  0.00           C  \\nATOM   8044  C   GLY C 513     -16.681  10.939  23.452  1.00  0.00           C  \\nATOM   8045  O   GLY C 513     -16.885  11.857  22.649  1.00  0.00           O  \\nATOM   8046  H   GLY C 513     -17.758   8.228  24.999  1.00  0.00           H  \\nATOM   8047 1HA  GLY C 513     -18.427   9.965  22.891  1.00  0.00           H  \\nATOM   8048 2HA  GLY C 513     -18.351  10.436  24.579  1.00  0.00           H  \\nATOM   8049  N   LEU C 514     -15.458  10.789  24.037  1.00  0.00           N  \\nATOM   8050  CA  LEU C 514     -14.323  11.732  23.683  1.00  0.00           C  \\nATOM   8051  C   LEU C 514     -14.013  11.870  22.186  1.00  0.00           C  \\nATOM   8052  O   LEU C 514     -13.525  12.937  21.748  1.00  0.00           O  \\nATOM   8053  CB  LEU C 514     -12.961  11.519  24.445  1.00  0.00           C  \\nATOM   8054  CG  LEU C 514     -12.180  10.287  24.127  1.00  0.00           C  \\nATOM   8055  CD1 LEU C 514     -10.877  10.189  24.930  1.00  0.00           C  \\nATOM   8056  CD2 LEU C 514     -13.134   9.066  24.395  1.00  0.00           C  \\nATOM   8057  H   LEU C 514     -15.295  10.053  24.709  1.00  0.00           H  \\nATOM   8058  HA  LEU C 514     -14.633  12.750  23.918  1.00  0.00           H  \\nATOM   8059 1HB  LEU C 514     -12.310  12.366  24.236  1.00  0.00           H  \\nATOM   8060 2HB  LEU C 514     -13.162  11.497  25.516  1.00  0.00           H  \\nATOM   8061  HG  LEU C 514     -11.868  10.313  23.083  1.00  0.00           H  \\nATOM   8062 1HD1 LEU C 514     -10.350   9.274  24.658  1.00  0.00           H  \\nATOM   8063 2HD1 LEU C 514     -10.246  11.050  24.707  1.00  0.00           H  \\nATOM   8064 3HD1 LEU C 514     -11.106  10.173  25.995  1.00  0.00           H  \\nATOM   8065 1HD2 LEU C 514     -12.607   8.137  24.175  1.00  0.00           H  \\nATOM   8066 2HD2 LEU C 514     -13.444   9.069  25.440  1.00  0.00           H  \\nATOM   8067 3HD2 LEU C 514     -14.013   9.144  23.756  1.00  0.00           H  \\nATOM   8068  N   PHE C 515     -14.401  10.872  21.388  1.00  0.00           N  \\nATOM   8069  CA  PHE C 515     -14.087  10.889  19.937  1.00  0.00           C  \\nATOM   8070  C   PHE C 515     -15.187  11.420  19.022  1.00  0.00           C  \\nATOM   8071  O   PHE C 515     -15.032  11.567  17.846  1.00  0.00           O  \\nATOM   8072  CB  PHE C 515     -13.557   9.518  19.451  1.00  0.00           C  \\nATOM   8073  CG  PHE C 515     -12.395   9.020  20.234  1.00  0.00           C  \\nATOM   8074  CD1 PHE C 515     -11.140   9.544  19.996  1.00  0.00           C  \\nATOM   8075  CD2 PHE C 515     -12.542   8.049  21.236  1.00  0.00           C  \\nATOM   8076  CE1 PHE C 515     -10.001   9.065  20.692  1.00  0.00           C  \\nATOM   8077  CE2 PHE C 515     -11.418   7.563  21.899  1.00  0.00           C  \\nATOM   8078  CZ  PHE C 515     -10.147   8.095  21.647  1.00  0.00           C  \\nATOM   8079  H   PHE C 515     -14.916  10.092  21.771  1.00  0.00           H  \\nATOM   8080  HA  PHE C 515     -13.310  11.634  19.760  1.00  0.00           H  \\nATOM   8081 1HB  PHE C 515     -14.354   8.778  19.512  1.00  0.00           H  \\nATOM   8082 2HB  PHE C 515     -13.259   9.593  18.406  1.00  0.00           H  \\nATOM   8083  HD1 PHE C 515     -11.026  10.340  19.260  1.00  0.00           H  \\nATOM   8084  HD2 PHE C 515     -13.524   7.623  21.444  1.00  0.00           H  \\nATOM   8085  HE1 PHE C 515      -9.015   9.470  20.466  1.00  0.00           H  \\nATOM   8086  HE2 PHE C 515     -11.537   6.756  22.622  1.00  0.00           H  \\nATOM   8087  HZ  PHE C 515      -9.280   7.741  22.203  1.00  0.00           H  \\nATOM   8088  N   ASP C 516     -16.342  11.739  19.617  1.00  0.00           N  \\nATOM   8089  CA  ASP C 516     -17.472  12.287  18.852  1.00  0.00           C  \\nATOM   8090  C   ASP C 516     -17.462  13.787  18.725  1.00  0.00           C  \\nATOM   8091  O   ASP C 516     -16.724  14.500  19.447  1.00  0.00           O  \\nATOM   8092  OXT ASP C 516     -18.170  14.326  17.920  1.00  0.00           O  \\nATOM   8093  CB  ASP C 516     -18.786  11.877  19.531  1.00  0.00           C  \\nATOM   8094  CG  ASP C 516     -19.067  10.394  19.445  1.00  0.00           C  \\nATOM   8095  OD1 ASP C 516     -19.631   9.843  20.454  1.00  0.00           O  \\nATOM   8096  OD2 ASP C 516     -18.765   9.805  18.360  1.00  0.00           O  \\nATOM   8097  H   ASP C 516     -16.442  11.601  20.613  1.00  0.00           H  \\nATOM   8098  HA  ASP C 516     -17.442  11.871  17.845  1.00  0.00           H  \\nATOM   8099 1HB  ASP C 516     -18.756  12.162  20.583  1.00  0.00           H  \\nATOM   8100 2HB  ASP C 516     -19.616  12.413  19.070  1.00  0.00           H  \\nTER                                                                             \\nATOM   8102  N   GLY D 517      39.878  -2.302  27.465  1.00  0.00           N  \\nATOM   8103  CA  GLY D 517      38.527  -2.946  27.563  1.00  0.00           C  \\nATOM   8104  C   GLY D 517      37.960  -3.299  26.197  1.00  0.00           C  \\nATOM   8105  O   GLY D 517      36.916  -3.941  26.105  1.00  0.00           O  \\nATOM   8106 1H   GLY D 517      40.213  -2.087  28.382  1.00  0.00           H  \\nATOM   8107 2H   GLY D 517      40.515  -2.929  27.017  1.00  0.00           H  \\nATOM   8108 3H   GLY D 517      39.808  -1.460  26.930  1.00  0.00           H  \\nATOM   8109 1HA  GLY D 517      38.598  -3.851  28.167  1.00  0.00           H  \\nATOM   8110 2HA  GLY D 517      37.839  -2.273  28.074  1.00  0.00           H  \\nATOM   8111  N   PHE D 518      38.658  -2.930  25.123  1.00  0.00           N  \\nATOM   8112  CA  PHE D 518      38.118  -3.148  23.781  1.00  0.00           C  \\nATOM   8113  C   PHE D 518      38.488  -4.524  23.230  1.00  0.00           C  \\nATOM   8114  O   PHE D 518      38.064  -4.905  22.150  1.00  0.00           O  \\nATOM   8115  CB  PHE D 518      38.656  -2.102  22.853  1.00  0.00           C  \\nATOM   8116  CG  PHE D 518      38.253  -0.712  23.217  1.00  0.00           C  \\nATOM   8117  CD1 PHE D 518      38.782   0.344  22.518  1.00  0.00           C  \\nATOM   8118  CD2 PHE D 518      37.338  -0.466  24.264  1.00  0.00           C  \\nATOM   8119  CE1 PHE D 518      38.411   1.689  22.831  1.00  0.00           C  \\nATOM   8120  CE2 PHE D 518      36.944   0.884  24.569  1.00  0.00           C  \\nATOM   8121  CZ  PHE D 518      37.526   1.948  23.870  1.00  0.00           C  \\nATOM   8122  H   PHE D 518      39.565  -2.497  25.224  1.00  0.00           H  \\nATOM   8123  HA  PHE D 518      37.031  -3.062  23.827  1.00  0.00           H  \\nATOM   8124 1HB  PHE D 518      39.744  -2.150  22.843  1.00  0.00           H  \\nATOM   8125 2HB  PHE D 518      38.311  -2.303  21.840  1.00  0.00           H  \\nATOM   8126  HD1 PHE D 518      39.493   0.148  21.715  1.00  0.00           H  \\nATOM   8127  HD2 PHE D 518      36.889  -1.300  24.805  1.00  0.00           H  \\nATOM   8128  HE1 PHE D 518      38.823   2.516  22.253  1.00  0.00           H  \\nATOM   8129  HE2 PHE D 518      36.198   1.082  25.339  1.00  0.00           H  \\nATOM   8130  HZ  PHE D 518      37.287   2.976  24.140  1.00  0.00           H  \\nATOM   8131  N   GLY D 519      39.247  -5.318  23.996  1.00  0.00           N  \\nATOM   8132  CA  GLY D 519      39.654  -6.642  23.513  1.00  0.00           C  \\nATOM   8133  C   GLY D 519      38.568  -7.650  23.303  1.00  0.00           C  \\nATOM   8134  O   GLY D 519      38.701  -8.563  22.522  1.00  0.00           O  \\nATOM   8135  H   GLY D 519      39.547  -5.015  24.912  1.00  0.00           H  \\nATOM   8136 1HA  GLY D 519      40.173  -6.538  22.560  1.00  0.00           H  \\nATOM   8137 2HA  GLY D 519      40.359  -7.084  24.216  1.00  0.00           H  \\nATOM   8138  N   LEU D 520      37.424  -7.419  23.941  1.00  0.00           N  \\nATOM   8139  CA  LEU D 520      36.193  -8.141  23.678  1.00  0.00           C  \\nATOM   8140  C   LEU D 520      35.882  -8.413  22.195  1.00  0.00           C  \\nATOM   8141  O   LEU D 520      35.382  -9.485  21.876  1.00  0.00           O  \\nATOM   8142  CB  LEU D 520      35.105  -7.251  24.229  1.00  0.00           C  \\nATOM   8143  CG  LEU D 520      33.828  -7.971  24.403  1.00  0.00           C  \\nATOM   8144  CD1 LEU D 520      34.050  -9.182  25.384  1.00  0.00           C  \\nATOM   8145  CD2 LEU D 520      32.687  -7.040  24.899  1.00  0.00           C  \\nATOM   8146  H   LEU D 520      37.429  -6.696  24.646  1.00  0.00           H  \\nATOM   8147  HA  LEU D 520      36.232  -9.094  24.205  1.00  0.00           H  \\nATOM   8148 1HB  LEU D 520      35.433  -6.856  25.189  1.00  0.00           H  \\nATOM   8149 2HB  LEU D 520      34.961  -6.415  23.545  1.00  0.00           H  \\nATOM   8150  HG  LEU D 520      33.520  -8.402  23.450  1.00  0.00           H  \\nATOM   8151 1HD1 LEU D 520      33.111  -9.719  25.517  1.00  0.00           H  \\nATOM   8152 2HD1 LEU D 520      34.797  -9.857  24.966  1.00  0.00           H  \\nATOM   8153 3HD1 LEU D 520      34.395  -8.810  26.348  1.00  0.00           H  \\nATOM   8154 1HD2 LEU D 520      31.769  -7.617  25.010  1.00  0.00           H  \\nATOM   8155 2HD2 LEU D 520      32.963  -6.607  25.861  1.00  0.00           H  \\nATOM   8156 3HD2 LEU D 520      32.528  -6.241  24.174  1.00  0.00           H  \\nATOM   8157  N   PHE D 521      36.249  -7.499  21.305  1.00  0.00           N  \\nATOM   8158  CA  PHE D 521      35.868  -7.614  19.861  1.00  0.00           C  \\nATOM   8159  C   PHE D 521      36.974  -8.126  18.930  1.00  0.00           C  \\nATOM   8160  O   PHE D 521      36.755  -8.306  17.754  1.00  0.00           O  \\nATOM   8161  CB  PHE D 521      35.375  -6.244  19.347  1.00  0.00           C  \\nATOM   8162  CG  PHE D 521      34.193  -5.696  20.149  1.00  0.00           C  \\nATOM   8163  CD1 PHE D 521      32.899  -6.269  19.969  1.00  0.00           C  \\nATOM   8164  CD2 PHE D 521      34.394  -4.747  21.216  1.00  0.00           C  \\nATOM   8165  CE1 PHE D 521      31.831  -5.807  20.677  1.00  0.00           C  \\nATOM   8166  CE2 PHE D 521      33.295  -4.286  21.898  1.00  0.00           C  \\nATOM   8167  CZ  PHE D 521      32.012  -4.821  21.638  1.00  0.00           C  \\nATOM   8168  H   PHE D 521      36.798  -6.707  21.607  1.00  0.00           H  \\nATOM   8169  HA  PHE D 521      35.059  -8.341  19.773  1.00  0.00           H  \\nATOM   8170 1HB  PHE D 521      36.191  -5.524  19.393  1.00  0.00           H  \\nATOM   8171 2HB  PHE D 521      35.076  -6.333  18.303  1.00  0.00           H  \\nATOM   8172  HD1 PHE D 521      32.772  -7.085  19.257  1.00  0.00           H  \\nATOM   8173  HD2 PHE D 521      35.379  -4.328  21.423  1.00  0.00           H  \\nATOM   8174  HE1 PHE D 521      30.837  -6.212  20.489  1.00  0.00           H  \\nATOM   8175  HE2 PHE D 521      33.416  -3.501  22.644  1.00  0.00           H  \\nATOM   8176  HZ  PHE D 521      31.160  -4.447  22.204  1.00  0.00           H  \\nATOM   8177  N   ASP D 522      38.149  -8.466  19.509  1.00  0.00           N  \\nATOM   8178  CA  ASP D 522      39.284  -9.002  18.745  1.00  0.00           C  \\nATOM   8179  C   ASP D 522      39.299 -10.525  18.593  1.00  0.00           C  \\nATOM   8180  O   ASP D 522      38.530 -11.239  19.239  1.00  0.00           O  \\nATOM   8181  OXT ASP D 522      40.069 -11.045  17.834  1.00  0.00           O  \\nATOM   8182  CB  ASP D 522      40.575  -8.624  19.432  1.00  0.00           C  \\nATOM   8183  CG  ASP D 522      40.845  -7.173  19.396  1.00  0.00           C  \\nATOM   8184  OD1 ASP D 522      40.405  -6.532  18.442  1.00  0.00           O  \\nATOM   8185  OD2 ASP D 522      41.488  -6.665  20.360  1.00  0.00           O  \\nATOM   8186  H   ASP D 522      38.242  -8.343  20.507  1.00  0.00           H  \\nATOM   8187  HA  ASP D 522      39.268  -8.563  17.747  1.00  0.00           H  \\nATOM   8188 1HB  ASP D 522      40.541  -8.945  20.473  1.00  0.00           H  \\nATOM   8189 2HB  ASP D 522      41.407  -9.144  18.956  1.00  0.00           H  \\nTER                                                                             \\nHETATM 8191 C05  LG1 E 523     -14.825   3.833   3.998  1.00  0.00           C  \\nHETATM 8192 C06  LG1 E 523     -14.177   3.099   3.023  1.00  0.00           C  \\nHETATM 8193 C01  LG1 E 523     -13.256   3.701   2.173  1.00  0.00           C  \\nHETATM 8194 N14  LG1 E 523     -12.638   2.960   1.175  1.00  0.00           N  \\nHETATM 8195 C13  LG1 E 523     -11.863   3.560   0.196  1.00  0.00           C  \\nHETATM 8196 C18  LG1 E 523     -11.488   2.889  -0.957  1.00  0.00           C  \\nHETATM 8197 C17  LG1 E 523     -10.687   3.508  -1.907  1.00  0.00           C  \\nHETATM 8198 C16  LG1 E 523     -10.236   4.799  -1.732  1.00  0.00           C  \\nHETATM 8199 C15  LG1 E 523     -10.601   5.498  -0.586  1.00  0.00           C  \\nHETATM 8200 C19  LG1 E 523     -10.141   6.863  -0.307  1.00  0.00           C  \\nHETATM 8201 C20  LG1 E 523      -9.372   7.572  -1.229  1.00  0.00           C  \\nHETATM 8202 C21  LG1 E 523      -8.964   8.855  -0.934  1.00  0.00           C  \\nHETATM 8203 F30  LG1 E 523      -8.217   9.545  -1.824  1.00  0.00           F  \\nHETATM 8204 C22  LG1 E 523      -9.320   9.440   0.276  1.00  0.00           C  \\nHETATM 8205 C23  LG1 E 523     -10.067   8.760   1.195  1.00  0.00           C  \\nHETATM 8206 C24  LG1 E 523     -10.486   7.448   0.937  1.00  0.00           C  \\nHETATM 8207 NP5  LG1 E 523     -11.185   6.738   1.843  1.00  0.00           N  \\nHETATM 8208 C11  LG1 E 523     -11.865   5.657   1.548  1.00  0.00           C  \\nHETATM 8209 C02  LG1 E 523     -12.948   5.066   2.353  1.00  0.00           C  \\nHETATM 8210 C03  LG1 E 523     -13.654   5.801   3.301  1.00  0.00           C  \\nHETATM 8211 C04  LG1 E 523     -14.576   5.189   4.127  1.00  0.00           C  \\nHETATM 8212 F08  LG1 E 523     -15.226   5.912   5.065  1.00  0.00           F  \\nHETATM 8213 C12  LG1 E 523     -11.440   4.888   0.366  1.00  0.00           C  \\nHETATM 8214 C34  LG1 E 523     -11.041   7.131   3.250  1.00  0.00           C  \\nHETATM 8215 C27  LG1 E 523     -10.302   2.749  -3.152  1.00  0.00           C  \\nHETATM 8216 C33  LG1 E 523     -12.833   1.509   1.130  1.00  0.00           C  \\nHETATM 8217 H09  LG1 E 523     -15.523   3.345   4.660  1.00  0.00           H  \\nHETATM 8218 H10  LG1 E 523     -14.423   2.049   2.951  1.00  0.00           H  \\nHETATM 8219 H26  LG1 E 523     -11.817   1.880  -1.154  1.00  0.00           H  \\nHETATM 8220 H28  LG1 E 523      -9.607   5.260  -2.478  1.00  0.00           H  \\nHETATM 8221 H29  LG1 E 523      -9.095   7.121  -2.171  1.00  0.00           H  \\nHETATM 8222 H31  LG1 E 523      -9.005  10.450   0.494  1.00  0.00           H  \\nHETATM 8223 H32  LG1 E 523     -10.333   9.272   2.107  1.00  0.00           H  \\nHETATM 8224 H07  LG1 E 523     -13.535   6.873   3.376  1.00  0.00           H  \\nHETATM 8225 H35  LG1 E 523     -11.795   7.879   3.498  1.00  0.00           H  \\nHETATM 8226 H36  LG1 E 523     -11.173   6.257   3.887  1.00  0.00           H  \\nHETATM 8227 H37  LG1 E 523     -10.048   7.551   3.410  1.00  0.00           H  \\nHETATM 8228 H38  LG1 E 523     -11.048   2.922  -3.927  1.00  0.00           H  \\nHETATM 8229 H39  LG1 E 523      -9.329   3.093  -3.500  1.00  0.00           H  \\nHETATM 8230 H40  LG1 E 523     -10.252   1.683  -2.927  1.00  0.00           H  \\nHETATM 8231 H41  LG1 E 523     -13.752   1.283   0.588  1.00  0.00           H  \\nHETATM 8232 H42  LG1 E 523     -11.988   1.045   0.622  1.00  0.00           H  \\nHETATM 8233 H43  LG1 E 523     -12.904   1.120   2.145  1.00  0.00           H  \\nTER                                                                             \\nCONECT 8191 8192 8211 8217                                                      \\nCONECT 8192 8191 8193 8218                                                      \\nCONECT 8193 8192 8194 8209                                                      \\nCONECT 8194 8193 8195 8216                                                      \\nCONECT 8195 8194 8196 8213                                                      \\nCONECT 8196 8195 8197 8219                                                      \\nCONECT 8197 8196 8198 8215                                                      \\nCONECT 8198 8197 8199 8220                                                      \\nCONECT 8199 8198 8200 8213                                                      \\nCONECT 8200 8199 8201 8206                                                      \\nCONECT 8201 8200 8202 8221                                                      \\nCONECT 8202 8201 8203 8204                                                      \\nCONECT 8203 8202                                                                \\nCONECT 8204 8202 8205 8222                                                      \\nCONECT 8205 8204 8206 8223                                                      \\nCONECT 8206 8200 8205 8207                                                      \\nCONECT 8207 8206 8208 8214                                                      \\nCONECT 8208 8207 8209 8213                                                      \\nCONECT 8209 8193 8208 8210                                                      \\nCONECT 8210 8209 8211 8224                                                      \\nCONECT 8211 8191 8210 8212                                                      \\nCONECT 8212 8211                                                                \\nCONECT 8213 8195 8199 8208                                                      \\nCONECT 8214 8207 8225 8226 8227                                                 \\nCONECT 8215 8197 8228 8229 8230                                                 \\nCONECT 8216 8194 8231 8232 8233                                                 \\nCONECT 8217 8191                                                                \\nCONECT 8218 8192                                                                \\nCONECT 8219 8196                                                                \\nCONECT 8220 8198                                                                \\nCONECT 8221 8201                                                                \\nCONECT 8222 8204                                                                \\nCONECT 8223 8205                                                                \\nCONECT 8224 8210                                                                \\nCONECT 8225 8214                                                                \\nCONECT 8226 8214                                                                \\nCONECT 8227 8214                                                                \\nCONECT 8228 8215                                                                \\nCONECT 8229 8215                                                                \\nCONECT 8230 8215                                                                \\nCONECT 8231 8216                                                                \\nCONECT 8232 8216                                                                \\nCONECT 8233 8216                                                                \\n\",\"pdb\");\n",
              "\tviewer_15942021952474504.zoomTo();\n",
              "\tviewer_15942021952474504.setStyle({\"cartoon\": {\"color\": \"spectrum\"}, \"stick\": {\"colorscheme\": \"blackCarbon\", \"radius\": \"0.1\"}});\n",
              "\tviewer_15942021952474504.setStyle({\"hetflag\": true},{\"stick\": {\"colorscheme\": \"brownCarbon\", \"radius\": 0.2}});\n",
              "\tviewer_15942021952474504.addSurface(1,{\"opacity\": 0.7, \"color\": \"white\"},{\"resi\": [523], \"chain\": [\"E\"]});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.847, \"y\": -18.878, \"z\": 21.285}, \"end\": {\"x\": -2.334, \"y\": -20.299, \"z\": 22.404}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.688, \"y\": -19.457, \"z\": 17.396}, \"end\": {\"x\": -7.746, \"y\": -18.875, \"z\": 18.961}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.715, \"y\": -18.812, \"z\": 16.343}, \"end\": {\"x\": -9.079, \"y\": -19.532, \"z\": 15.408}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.524, \"y\": -19.004, \"z\": 12.445}, \"end\": {\"x\": -12.045, \"y\": -18.462, \"z\": 14.341}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.823, \"y\": -18.434, \"z\": 10.945}, \"end\": {\"x\": -15.714, \"y\": -19.85, \"z\": 11.981}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.122, \"y\": -15.85, \"z\": 12.289}, \"end\": {\"x\": -18.639, \"y\": -15.854, \"z\": 10.743}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.677, \"y\": -13.818, \"z\": 10.316}, \"end\": {\"x\": -18.639, \"y\": -15.854, \"z\": 10.743}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.823, \"y\": -18.434, \"z\": 10.945}, \"end\": {\"x\": -15.714, \"y\": -19.85, \"z\": 11.981}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.122, \"y\": -15.85, \"z\": 12.289}, \"end\": {\"x\": -18.639, \"y\": -15.854, \"z\": 10.743}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.616, \"y\": -12.287, \"z\": 18.717}, \"end\": {\"x\": -20.227, \"y\": -11.823, \"z\": 17.569}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.664, \"y\": -14.14, \"z\": 16.663}, \"end\": {\"x\": -19.099, \"y\": -14.953, \"z\": 17.657}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.721, \"y\": -13.499, \"z\": 19.531}, \"end\": {\"x\": -22.488, \"y\": -14.018, \"z\": 18.416}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.982, \"y\": -10.574, \"z\": 20.647}, \"end\": {\"x\": -19.808, \"y\": -10.112, \"z\": 20.355}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.091, \"y\": -12.531, \"z\": 25.185}, \"end\": {\"x\": -18.294, \"y\": -13.4, \"z\": 25.732}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.453, \"y\": -12.589, \"z\": 22.732}, \"end\": {\"x\": -19.163, \"y\": -12.933, \"z\": 22.251}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.664, \"y\": -14.14, \"z\": 16.663}, \"end\": {\"x\": -19.099, \"y\": -14.953, \"z\": 17.657}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.721, \"y\": -13.499, \"z\": 19.531}, \"end\": {\"x\": -22.488, \"y\": -14.018, \"z\": 18.416}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.155, \"y\": -13.551, \"z\": 21.66}, \"end\": {\"x\": -16.578, \"y\": -14.194, \"z\": 20.203}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.616, \"y\": -12.287, \"z\": 18.717}, \"end\": {\"x\": -20.227, \"y\": -11.823, \"z\": 17.569}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.3, \"y\": -11.427, \"z\": 21.614}, \"end\": {\"x\": -14.896, \"y\": -11.157, \"z\": 20.558}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.982, \"y\": -10.574, \"z\": 20.647}, \"end\": {\"x\": -19.808, \"y\": -10.112, \"z\": 20.355}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.137, \"y\": -11.097, \"z\": 24.321}, \"end\": {\"x\": -15.037, \"y\": -10.837, \"z\": 23.803}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -20.091, \"y\": -12.531, \"z\": 25.185}, \"end\": {\"x\": -18.294, \"y\": -13.4, \"z\": 25.732}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.453, \"y\": -12.589, \"z\": 22.732}, \"end\": {\"x\": -19.163, \"y\": -12.933, \"z\": 22.251}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.905, \"y\": -13.459, \"z\": 24.993}, \"end\": {\"x\": -13.721, \"y\": -13.92, \"z\": 24.398}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.155, \"y\": -13.551, \"z\": 21.66}, \"end\": {\"x\": -16.578, \"y\": -14.194, \"z\": 20.203}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.441, \"y\": -13.258, \"z\": 23.809}, \"end\": {\"x\": -11.024, \"y\": -13.337, \"z\": 22.625}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.3, \"y\": -11.427, \"z\": 21.614}, \"end\": {\"x\": -14.896, \"y\": -11.157, \"z\": 20.558}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.288, \"y\": -10.954, \"z\": 25.149}, \"end\": {\"x\": -10.003, \"y\": -10.512, \"z\": 24.251}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.137, \"y\": -11.097, \"z\": 24.321}, \"end\": {\"x\": -15.037, \"y\": -10.837, \"z\": 23.803}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.253, \"y\": -12.171, \"z\": 27.676}, \"end\": {\"x\": -10.189, \"y\": -12.148, \"z\": 27.305}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.905, \"y\": -13.459, \"z\": 24.993}, \"end\": {\"x\": -13.721, \"y\": -13.92, \"z\": 24.398}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.214, \"y\": -13.955, \"z\": 27.31}, \"end\": {\"x\": -7.926, \"y\": -14.601, \"z\": 26.216}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.441, \"y\": -13.258, \"z\": 23.809}, \"end\": {\"x\": -11.024, \"y\": -13.337, \"z\": 22.625}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.207, \"y\": -12.624, \"z\": 27.194}, \"end\": {\"x\": -5.563, \"y\": -12.476, \"z\": 25.456}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.288, \"y\": -10.954, \"z\": 25.149}, \"end\": {\"x\": -10.003, \"y\": -10.512, \"z\": 24.251}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.586, \"y\": -5.825, \"z\": 25.783}, \"end\": {\"x\": -13.59, \"y\": -5.122, \"z\": 24.075}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.777, \"y\": -4.718, \"z\": 24.065}, \"end\": {\"x\": -13.59, \"y\": -5.122, \"z\": 24.075}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.075, \"y\": -4.761, \"z\": 23.66}, \"end\": {\"x\": -9.416, \"y\": -4.917, \"z\": 21.754}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.386, \"y\": -11.721, \"z\": 29.308}, \"end\": {\"x\": -5.444, \"y\": -10.884, \"z\": 28.462}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.637, \"y\": -5.949, \"z\": 24.982}, \"end\": {\"x\": -7.049, \"y\": -5.334, \"z\": 24.214}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.253, \"y\": -12.171, \"z\": 27.676}, \"end\": {\"x\": -10.189, \"y\": -12.148, \"z\": 27.305}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.214, \"y\": -13.955, \"z\": 27.31}, \"end\": {\"x\": -7.926, \"y\": -14.601, \"z\": 26.216}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.207, \"y\": -12.624, \"z\": 27.194}, \"end\": {\"x\": -5.563, \"y\": -12.476, \"z\": 25.456}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.386, \"y\": -11.721, \"z\": 29.308}, \"end\": {\"x\": -5.444, \"y\": -10.884, \"z\": 28.462}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.679, \"y\": -10.388, \"z\": 36.665}, \"end\": {\"x\": -0.145, \"y\": -11.095, \"z\": 35.517}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.679, \"y\": -10.388, \"z\": 36.665}, \"end\": {\"x\": -0.145, \"y\": -11.095, \"z\": 35.517}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.942, \"y\": -2.119, \"z\": 36.475}, \"end\": {\"x\": 1.575, \"y\": -3.907, \"z\": 36.75}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.815, \"y\": -3.185, \"z\": 35.8}, \"end\": {\"x\": 3.052, \"y\": -4.915, \"z\": 35.527}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.422, \"y\": -4.909, \"z\": 33.317}, \"end\": {\"x\": -1.425, \"y\": -5.26, \"z\": 35.203}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.942, \"y\": -2.119, \"z\": 36.475}, \"end\": {\"x\": 1.575, \"y\": -3.907, \"z\": 36.75}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.815, \"y\": -3.185, \"z\": 35.8}, \"end\": {\"x\": 3.052, \"y\": -4.915, \"z\": 35.527}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.618, \"y\": -1.287, \"z\": 35.372}, \"end\": {\"x\": -3.241, \"y\": -1.689, \"z\": 33.658}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.169, \"y\": 0.139, \"z\": 34.954}, \"end\": {\"x\": -3.5, \"y\": 1.199, \"z\": 35.067}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.618, \"y\": -1.287, \"z\": 35.372}, \"end\": {\"x\": -3.241, \"y\": -1.689, \"z\": 33.658}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.169, \"y\": 0.139, \"z\": 34.954}, \"end\": {\"x\": -3.5, \"y\": 1.199, \"z\": 35.067}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.481, \"y\": -3.161, \"z\": 28.826}, \"end\": {\"x\": -2.518, \"y\": -3.408, \"z\": 30.737}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.422, \"y\": -4.909, \"z\": 33.317}, \"end\": {\"x\": -1.425, \"y\": -5.26, \"z\": 35.203}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.033, \"y\": -4.935, \"z\": 29.523}, \"end\": {\"x\": 1.041, \"y\": -3.403, \"z\": 28.444}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.325, \"y\": -7.568, \"z\": 29.475}, \"end\": {\"x\": 10.222, \"y\": -6.216, \"z\": 30.53}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.728, \"y\": -12.023, \"z\": 29.474}, \"end\": {\"x\": 11.793, \"y\": -10.327, \"z\": 28.078}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.441, \"y\": -8.357, \"z\": 22.091}, \"end\": {\"x\": 12.03, \"y\": -9.976, \"z\": 20.073}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.424, \"y\": -8.635, \"z\": 20.923}, \"end\": {\"x\": 12.03, \"y\": -9.976, \"z\": 20.073}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.325, \"y\": -7.568, \"z\": 29.475}, \"end\": {\"x\": 10.222, \"y\": -6.216, \"z\": 30.53}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.728, \"y\": -12.023, \"z\": 29.474}, \"end\": {\"x\": 11.793, \"y\": -10.327, \"z\": 28.078}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.458, \"y\": -18.535, \"z\": 27.87}, \"end\": {\"x\": 8.907, \"y\": -17.132, \"z\": 28.015}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.549, \"y\": -17.649, \"z\": 25.574}, \"end\": {\"x\": 8.144, \"y\": -18.987, \"z\": 25.041}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.458, \"y\": -18.535, \"z\": 27.87}, \"end\": {\"x\": 8.907, \"y\": -17.132, \"z\": 28.015}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.549, \"y\": -17.649, \"z\": 25.574}, \"end\": {\"x\": 8.144, \"y\": -18.987, \"z\": 25.041}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.689, \"y\": -11.525, \"z\": 22.746}, \"end\": {\"x\": 5.923, \"y\": -10.752, \"z\": 22.538}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.851, \"y\": -10.472, \"z\": 21.453}, \"end\": {\"x\": 7.114, \"y\": -10.358, \"z\": 20.663}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.727, \"y\": -12.768, \"z\": 23.45}, \"end\": {\"x\": 4.979, \"y\": -14.185, \"z\": 23.62}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.2, \"y\": -15.55, \"z\": 21.715}, \"end\": {\"x\": 2.047, \"y\": -13.99, \"z\": 20.753}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.464, \"y\": -15.139, \"z\": 18.862}, \"end\": {\"x\": -0.129, \"y\": -16.591, \"z\": 19.917}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.847, \"y\": -18.878, \"z\": 21.285}, \"end\": {\"x\": -2.334, \"y\": -20.299, \"z\": 22.404}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.023, \"y\": -17.565, \"z\": 17.221}, \"end\": {\"x\": -1.469, \"y\": -17.184, \"z\": 16.217}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.59, \"y\": -17.465, \"z\": 14.078}, \"end\": {\"x\": -5.015, \"y\": -17.971, \"z\": 15.247}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.688, \"y\": -19.457, \"z\": 17.396}, \"end\": {\"x\": -7.746, \"y\": -18.875, \"z\": 18.961}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.715, \"y\": -18.812, \"z\": 16.343}, \"end\": {\"x\": -9.079, \"y\": -19.532, \"z\": 15.408}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.029, \"y\": -19.041, \"z\": 13.096}, \"end\": {\"x\": -5.7, \"y\": -19.696, \"z\": 11.925}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.092, \"y\": -20.11, \"z\": 10.075}, \"end\": {\"x\": -8.895, \"y\": -19.938, \"z\": 11.049}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.524, \"y\": -19.004, \"z\": 12.445}, \"end\": {\"x\": -12.045, \"y\": -18.462, \"z\": 14.341}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.155, \"y\": -17.004, \"z\": 5.453}, \"end\": {\"x\": -12.008, \"y\": -14.916, \"z\": 4.802}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.905, \"y\": -20.806, \"z\": 8.516}, \"end\": {\"x\": -10.68, \"y\": -21.524, \"z\": 7.93}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.288, \"y\": -22.031, \"z\": 5.811}, \"end\": {\"x\": -12.771, \"y\": -19.997, \"z\": 6.023}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.656, \"y\": -18.385, \"z\": 4.534}, \"end\": {\"x\": -11.143, \"y\": -17.701, \"z\": 2.535}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.284, \"y\": -18.252, \"z\": 7.081}, \"end\": {\"x\": -16.076, \"y\": -16.514, \"z\": 6.454}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.905, \"y\": -20.806, \"z\": 8.516}, \"end\": {\"x\": -10.68, \"y\": -21.524, \"z\": 7.93}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.288, \"y\": -22.031, \"z\": 5.811}, \"end\": {\"x\": -12.771, \"y\": -19.997, \"z\": 6.023}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.029, \"y\": -19.041, \"z\": 13.096}, \"end\": {\"x\": -5.7, \"y\": -19.696, \"z\": 11.925}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.092, \"y\": -20.11, \"z\": 10.075}, \"end\": {\"x\": -8.895, \"y\": -19.938, \"z\": 11.049}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.776, \"y\": -20.332, \"z\": 11.067}, \"end\": {\"x\": -1.666, \"y\": -20.916, \"z\": 9.52}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.128, \"y\": -19.448, \"z\": 11.319}, \"end\": {\"x\": -1.136, \"y\": -18.74, \"z\": 12.682}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.023, \"y\": -17.565, \"z\": 17.221}, \"end\": {\"x\": -1.469, \"y\": -17.184, \"z\": 16.217}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.59, \"y\": -17.465, \"z\": 14.078}, \"end\": {\"x\": -5.015, \"y\": -17.971, \"z\": 15.247}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.741, \"y\": -12.512, \"z\": 15.496}, \"end\": {\"x\": 1.581, \"y\": -13.72, \"z\": 16.236}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.732, \"y\": -15.915, \"z\": 14.434}, \"end\": {\"x\": 1.488, \"y\": -15.284, \"z\": 12.801}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.2, \"y\": -15.55, \"z\": 21.715}, \"end\": {\"x\": 2.047, \"y\": -13.99, \"z\": 20.753}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.464, \"y\": -15.139, \"z\": 18.862}, \"end\": {\"x\": -0.129, \"y\": -16.591, \"z\": 19.917}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.689, \"y\": -11.525, \"z\": 22.746}, \"end\": {\"x\": 5.923, \"y\": -10.752, \"z\": 22.538}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.851, \"y\": -10.472, \"z\": 21.453}, \"end\": {\"x\": 7.114, \"y\": -10.358, \"z\": 20.663}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.702, \"y\": -11.299, \"z\": 19.239}, \"end\": {\"x\": 1.927, \"y\": -10.189, \"z\": 17.829}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.69, \"y\": -10.329, \"z\": 23.985}, \"end\": {\"x\": 5.923, \"y\": -10.752, \"z\": 22.538}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.457, \"y\": -8.328, \"z\": 22.915}, \"end\": {\"x\": 2.334, \"y\": -10.282, \"z\": 22.183}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.727, \"y\": -12.768, \"z\": 23.45}, \"end\": {\"x\": 4.979, \"y\": -14.185, \"z\": 23.62}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.69, \"y\": -10.329, \"z\": 23.985}, \"end\": {\"x\": 5.923, \"y\": -10.752, \"z\": 22.538}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.457, \"y\": -8.328, \"z\": 22.915}, \"end\": {\"x\": 2.334, \"y\": -10.282, \"z\": 22.183}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.75, \"y\": -4.404, \"z\": 21.029}, \"end\": {\"x\": 1.023, \"y\": -5.709, \"z\": 21.431}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.297, \"y\": -4.328, \"z\": 20.1}, \"end\": {\"x\": 1.023, \"y\": -5.709, \"z\": 21.431}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.864, \"y\": -8.28, \"z\": 19.979}, \"end\": {\"x\": 3.737, \"y\": -7.631, \"z\": 20.448}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.702, \"y\": -11.299, \"z\": 19.239}, \"end\": {\"x\": 1.927, \"y\": -10.189, \"z\": 17.829}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.864, \"y\": -8.28, \"z\": 19.979}, \"end\": {\"x\": 3.737, \"y\": -7.631, \"z\": 20.448}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.897, \"y\": -4.402, \"z\": 12.11}, \"end\": {\"x\": 3.038, \"y\": -4.789, \"z\": 10.934}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.741, \"y\": -12.512, \"z\": 15.496}, \"end\": {\"x\": 1.581, \"y\": -13.72, \"z\": 16.236}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.188, \"y\": -12.311, \"z\": 10.6}, \"end\": {\"x\": 5.631, \"y\": -13.001, \"z\": 12.595}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.732, \"y\": -15.915, \"z\": 14.434}, \"end\": {\"x\": 1.488, \"y\": -15.284, \"z\": 12.801}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.665, \"y\": -17.298, \"z\": 4.314}, \"end\": {\"x\": -2.395, \"y\": -17.859, \"z\": 3.793}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.056, \"y\": -16.616, \"z\": 10.387}, \"end\": {\"x\": 4.619, \"y\": -16.069, \"z\": 9.557}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.144, \"y\": -18.547, \"z\": 8.87}, \"end\": {\"x\": 2.454, \"y\": -18.778, \"z\": 9.646}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.776, \"y\": -20.332, \"z\": 11.067}, \"end\": {\"x\": -1.666, \"y\": -20.916, \"z\": 9.52}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.128, \"y\": -19.448, \"z\": 11.319}, \"end\": {\"x\": -1.136, \"y\": -18.74, \"z\": 12.682}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.716, \"y\": -25.919, \"z\": 10.077}, \"end\": {\"x\": 0.389, \"y\": -26.281, \"z\": 8.382}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.621, \"y\": -21.866, \"z\": 7.608}, \"end\": {\"x\": 2.499, \"y\": -22.462, \"z\": 7.581}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.844, \"y\": -24.584, \"z\": 6.516}, \"end\": {\"x\": 0.013, \"y\": -23.734, \"z\": 5.973}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.716, \"y\": -25.919, \"z\": 10.077}, \"end\": {\"x\": 0.389, \"y\": -26.281, \"z\": 8.382}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.778, \"y\": -25.959, \"z\": 6.503}, \"end\": {\"x\": 4.411, \"y\": -26.316, \"z\": 5.869}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.621, \"y\": -21.866, \"z\": 7.608}, \"end\": {\"x\": 2.499, \"y\": -22.462, \"z\": 7.581}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.844, \"y\": -24.584, \"z\": 6.516}, \"end\": {\"x\": 0.013, \"y\": -23.734, \"z\": 5.973}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.307, \"y\": -19.53, \"z\": 6.932}, \"end\": {\"x\": 6.404, \"y\": -19.926, \"z\": 7.141}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.056, \"y\": -16.616, \"z\": 10.387}, \"end\": {\"x\": 4.619, \"y\": -16.069, \"z\": 9.557}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.144, \"y\": -18.547, \"z\": 8.87}, \"end\": {\"x\": 2.454, \"y\": -18.778, \"z\": 9.646}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.396, \"y\": -15.788, \"z\": 8.051}, \"end\": {\"x\": 8.561, \"y\": -17.042, \"z\": 7.158}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.863, \"y\": -13.376, \"z\": 6.602}, \"end\": {\"x\": 8.157, \"y\": -13.304, \"z\": 8.325}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.188, \"y\": -12.311, \"z\": 10.6}, \"end\": {\"x\": 5.631, \"y\": -13.001, \"z\": 12.595}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.261, \"y\": -9.545, \"z\": 8.433}, \"end\": {\"x\": 8.596, \"y\": -8.49, \"z\": 7.052}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.054, \"y\": -10.671, \"z\": 18.67}, \"end\": {\"x\": 15.781, \"y\": -8.838, \"z\": 19.048}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.056, \"y\": -14.633, \"z\": 17.537}, \"end\": {\"x\": 17.443, \"y\": -13.377, \"z\": 17.699}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.891, \"y\": -15.129, \"z\": 19.062}, \"end\": {\"x\": 14.745, \"y\": -13.614, \"z\": 19.829}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.36, \"y\": -10.571, \"z\": 21.697}, \"end\": {\"x\": 20.329, \"y\": -11.416, \"z\": 21.47}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.055, \"y\": -8.147, \"z\": 21.295}, \"end\": {\"x\": 19.062, \"y\": -9.623, \"z\": 21.045}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.441, \"y\": -8.357, \"z\": 22.091}, \"end\": {\"x\": 12.03, \"y\": -9.976, \"z\": 20.073}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.424, \"y\": -8.635, \"z\": 20.923}, \"end\": {\"x\": 12.03, \"y\": -9.976, \"z\": 20.073}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.054, \"y\": -10.671, \"z\": 18.67}, \"end\": {\"x\": 15.781, \"y\": -8.838, \"z\": 19.048}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.734, \"y\": -15.767, \"z\": 17.233}, \"end\": {\"x\": 12.432, \"y\": -13.746, \"z\": 17.086}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.056, \"y\": -14.633, \"z\": 17.537}, \"end\": {\"x\": 17.443, \"y\": -13.377, \"z\": 17.699}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.463, \"y\": -18.507, \"z\": 17.008}, \"end\": {\"x\": 13.87, \"y\": -18.289, \"z\": 18.506}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.891, \"y\": -15.129, \"z\": 19.062}, \"end\": {\"x\": 14.745, \"y\": -13.614, \"z\": 19.829}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.706, \"y\": -19.93, \"z\": 17.997}, \"end\": {\"x\": 10.72, \"y\": -18.063, \"z\": 19.12}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.734, \"y\": -15.767, \"z\": 17.233}, \"end\": {\"x\": 12.432, \"y\": -13.746, \"z\": 17.086}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.742, \"y\": -20.592, \"z\": 16.884}, \"end\": {\"x\": 9.724, \"y\": -19.234, \"z\": 15.426}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.463, \"y\": -18.507, \"z\": 17.008}, \"end\": {\"x\": 13.87, \"y\": -18.289, \"z\": 18.506}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.706, \"y\": -19.93, \"z\": 17.997}, \"end\": {\"x\": 10.72, \"y\": -18.063, \"z\": 19.12}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.742, \"y\": -20.592, \"z\": 16.884}, \"end\": {\"x\": 9.724, \"y\": -19.234, \"z\": 15.426}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.302, \"y\": -23.183, \"z\": 13.402}, \"end\": {\"x\": 7.367, \"y\": -22.322, \"z\": 14.934}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.168, \"y\": -21.224, \"z\": 14.097}, \"end\": {\"x\": 7.367, \"y\": -22.322, \"z\": 14.934}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.329, \"y\": -27.025, \"z\": 12.085}, \"end\": {\"x\": 6.447, \"y\": -24.926, \"z\": 13.054}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.329, \"y\": -27.025, \"z\": 12.085}, \"end\": {\"x\": 6.447, \"y\": -24.926, \"z\": 13.054}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.778, \"y\": -25.959, \"z\": 6.503}, \"end\": {\"x\": 4.411, \"y\": -26.316, \"z\": 5.869}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.302, \"y\": -23.183, \"z\": 13.402}, \"end\": {\"x\": 7.367, \"y\": -22.322, \"z\": 14.934}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.168, \"y\": -21.224, \"z\": 14.097}, \"end\": {\"x\": 7.367, \"y\": -22.322, \"z\": 14.934}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.307, \"y\": -19.53, \"z\": 6.932}, \"end\": {\"x\": 6.404, \"y\": -19.926, \"z\": 7.141}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 7.396, \"y\": -15.788, \"z\": 8.051}, \"end\": {\"x\": 8.561, \"y\": -17.042, \"z\": 7.158}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.863, \"y\": -13.376, \"z\": 6.602}, \"end\": {\"x\": 8.157, \"y\": -13.304, \"z\": 8.325}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.261, \"y\": -9.545, \"z\": 8.433}, \"end\": {\"x\": 8.596, \"y\": -8.49, \"z\": 7.052}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.897, \"y\": -4.402, \"z\": 12.11}, \"end\": {\"x\": 3.038, \"y\": -4.789, \"z\": 10.934}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.391, \"y\": -0.864, \"z\": 5.261}, \"end\": {\"x\": 1.679, \"y\": -0.027, \"z\": 6.908}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.693, \"y\": -0.027, \"z\": 3.067}, \"end\": {\"x\": 3.815, \"y\": 1.042, \"z\": 4.723}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.334, \"y\": -2.277, \"z\": 1.623}, \"end\": {\"x\": 4.109, \"y\": -2.279, \"z\": 3.41}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.601, \"y\": -2.923, \"z\": 1.177}, \"end\": {\"x\": 0.747, \"y\": -2.995, \"z\": 3.079}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.391, \"y\": -0.864, \"z\": 5.261}, \"end\": {\"x\": 1.679, \"y\": -0.027, \"z\": 6.908}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.169, \"y\": -0.181, \"z\": 0.333}, \"end\": {\"x\": 0.138, \"y\": -0.033, \"z\": 1.209}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.708, \"y\": 2.324, \"z\": 6.188}, \"end\": {\"x\": -2.004, \"y\": 2.446, \"z\": 6.973}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.323, \"y\": 2.26, \"z\": 8.889}, \"end\": {\"x\": -2.004, \"y\": 2.446, \"z\": 6.973}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.693, \"y\": -0.027, \"z\": 3.067}, \"end\": {\"x\": 3.815, \"y\": 1.042, \"z\": 4.723}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.131, \"y\": -1.41, \"z\": -0.817}, \"end\": {\"x\": 2.258, \"y\": -0.992, \"z\": -1.108}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.334, \"y\": -2.277, \"z\": 1.623}, \"end\": {\"x\": 4.109, \"y\": -2.279, \"z\": 3.41}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.601, \"y\": -2.923, \"z\": 1.177}, \"end\": {\"x\": 0.747, \"y\": -2.995, \"z\": 3.079}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.131, \"y\": -1.41, \"z\": -0.817}, \"end\": {\"x\": 2.258, \"y\": -0.992, \"z\": -1.108}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.169, \"y\": -0.181, \"z\": 0.333}, \"end\": {\"x\": 0.138, \"y\": -0.033, \"z\": 1.209}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.255, \"y\": 2.997, \"z\": 3.871}, \"end\": {\"x\": -5.795, \"y\": 1.817, \"z\": 3.94}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.708, \"y\": 2.324, \"z\": 6.188}, \"end\": {\"x\": -2.004, \"y\": 2.446, \"z\": 6.973}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.323, \"y\": 2.26, \"z\": 8.889}, \"end\": {\"x\": -2.004, \"y\": 2.446, \"z\": 6.973}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.697, \"y\": 1.029, \"z\": 4.514}, \"end\": {\"x\": -8.082, \"y\": 1.739, \"z\": 6.357}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.344, \"y\": 2.03, \"z\": 12.263}, \"end\": {\"x\": -4.925, \"y\": 1.221, \"z\": 13.173}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.088, \"y\": 3.102, \"z\": 12.841}, \"end\": {\"x\": -0.599, \"y\": 4.459, \"z\": 13.361}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.598, \"y\": 4.009, \"z\": 11.408}, \"end\": {\"x\": -0.599, \"y\": 4.459, \"z\": 13.361}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.255, \"y\": 2.997, \"z\": 3.871}, \"end\": {\"x\": -5.795, \"y\": 1.817, \"z\": 3.94}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.697, \"y\": 1.029, \"z\": 4.514}, \"end\": {\"x\": -8.082, \"y\": 1.739, \"z\": 6.357}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.937, \"y\": -4.497, \"z\": 2.854}, \"end\": {\"x\": -15.799, \"y\": -2.545, \"z\": 2.238}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.444, \"y\": -2.133, \"z\": 7.434}, \"end\": {\"x\": -14.301, \"y\": -0.66, \"z\": 7.978}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.129, \"y\": -2.166, \"z\": 7.476}, \"end\": {\"x\": -15.396, \"y\": -3.811, \"z\": 8.462}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.697, \"y\": -9.587, \"z\": 7.805}, \"end\": {\"x\": -14.656, \"y\": -8.323, \"z\": 9.075}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.677, \"y\": -13.818, \"z\": 10.316}, \"end\": {\"x\": -18.639, \"y\": -15.854, \"z\": 10.743}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.204, \"y\": -12.071, \"z\": 7.146}, \"end\": {\"x\": -15.886, \"y\": -11.38, \"z\": 7.796}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.241, \"y\": -12.052, \"z\": 4.535}, \"end\": {\"x\": -14.694, \"y\": -10.752, \"z\": 4.814}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.601, \"y\": -11.731, \"z\": 5.148}, \"end\": {\"x\": -11.454, \"y\": -10.086, \"z\": 5.637}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.697, \"y\": -9.587, \"z\": 7.805}, \"end\": {\"x\": -14.656, \"y\": -8.323, \"z\": 9.075}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.793, \"y\": -13.98, \"z\": 6.262}, \"end\": {\"x\": -11.241, \"y\": -13.055, \"z\": 7.363}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.155, \"y\": -17.004, \"z\": 5.453}, \"end\": {\"x\": -12.008, \"y\": -14.916, \"z\": 4.802}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.284, \"y\": -18.252, \"z\": 7.081}, \"end\": {\"x\": -16.076, \"y\": -16.514, \"z\": 6.454}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.204, \"y\": -12.071, \"z\": 7.146}, \"end\": {\"x\": -15.886, \"y\": -11.38, \"z\": 7.796}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.985, \"y\": -15.812, \"z\": 4.309}, \"end\": {\"x\": -12.008, \"y\": -14.916, \"z\": 4.802}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.241, \"y\": -12.052, \"z\": 4.535}, \"end\": {\"x\": -14.694, \"y\": -10.752, \"z\": 4.814}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.906, \"y\": -8.396, \"z\": 1.714}, \"end\": {\"x\": -14.987, \"y\": -9.799, \"z\": 0.836}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.059, \"y\": -7.079, \"z\": 1.524}, \"end\": {\"x\": -13.12, \"y\": -8.975, \"z\": 1.582}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.032, \"y\": -14.695, \"z\": 2.675}, \"end\": {\"x\": -9.523, \"y\": -13.499, \"z\": 2.745}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.601, \"y\": -11.731, \"z\": 5.148}, \"end\": {\"x\": -11.454, \"y\": -10.086, \"z\": 5.637}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.607, \"y\": -15.062, \"z\": 4.205}, \"end\": {\"x\": -6.896, \"y\": -13.888, \"z\": 5.102}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.793, \"y\": -13.98, \"z\": 6.262}, \"end\": {\"x\": -11.241, \"y\": -13.055, \"z\": 7.363}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.521, \"y\": -17.716, \"z\": 4.203}, \"end\": {\"x\": -7.302, \"y\": -17.278, \"z\": 5.139}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.656, \"y\": -18.385, \"z\": 4.534}, \"end\": {\"x\": -11.143, \"y\": -17.701, \"z\": 2.535}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.985, \"y\": -15.812, \"z\": 4.309}, \"end\": {\"x\": -12.008, \"y\": -14.916, \"z\": 4.802}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.16, \"y\": -18.273, \"z\": 1.501}, \"end\": {\"x\": -6.852, \"y\": -17.543, \"z\": 1.804}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.032, \"y\": -14.695, \"z\": 2.675}, \"end\": {\"x\": -9.523, \"y\": -13.499, \"z\": 2.745}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.578, \"y\": -17.642, \"z\": 1.301}, \"end\": {\"x\": -3.926, \"y\": -15.82, \"z\": 1.58}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.665, \"y\": -17.298, \"z\": 4.314}, \"end\": {\"x\": -2.395, \"y\": -17.859, \"z\": 3.793}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.607, \"y\": -15.062, \"z\": 4.205}, \"end\": {\"x\": -6.896, \"y\": -13.888, \"z\": 5.102}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.521, \"y\": -17.716, \"z\": 4.203}, \"end\": {\"x\": -7.302, \"y\": -17.278, \"z\": 5.139}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.113, \"y\": -19.793, \"z\": 1.608}, \"end\": {\"x\": -3.081, \"y\": -20.554, \"z\": 2.103}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.16, \"y\": -18.273, \"z\": 1.501}, \"end\": {\"x\": -6.852, \"y\": -17.543, \"z\": 1.804}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.953, \"y\": -19.681, \"z\": -3.547}, \"end\": {\"x\": -2.211, \"y\": -19.072, \"z\": -1.503}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.977, \"y\": -17.182, \"z\": -2.956}, \"end\": {\"x\": -2.211, \"y\": -19.072, \"z\": -1.503}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.578, \"y\": -17.642, \"z\": 1.301}, \"end\": {\"x\": -3.926, \"y\": -15.82, \"z\": 1.58}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.306, \"y\": -20.343, \"z\": -2.292}, \"end\": {\"x\": 1.071, \"y\": -19.05, \"z\": -0.525}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.113, \"y\": -19.793, \"z\": 1.608}, \"end\": {\"x\": -3.081, \"y\": -20.554, \"z\": 2.103}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.306, \"y\": -20.343, \"z\": -2.292}, \"end\": {\"x\": 1.071, \"y\": -19.05, \"z\": -0.525}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.953, \"y\": -19.681, \"z\": -3.547}, \"end\": {\"x\": -2.211, \"y\": -19.072, \"z\": -1.503}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.977, \"y\": -17.182, \"z\": -2.956}, \"end\": {\"x\": -2.211, \"y\": -19.072, \"z\": -1.503}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.592, \"y\": -10.395, \"z\": -0.4}, \"end\": {\"x\": -0.988, \"y\": -11.173, \"z\": -2.234}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.51, \"y\": -8.484, \"z\": 1.221}, \"end\": {\"x\": 0.435, \"y\": -8.651, \"z\": -0.37}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.67, \"y\": -6.971, \"z\": 1.774}, \"end\": {\"x\": -2.265, \"y\": -6.751, \"z\": -0.229}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.028, \"y\": -8.628, \"z\": 3.211}, \"end\": {\"x\": -4.056, \"y\": -9.493, \"z\": 1.321}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.592, \"y\": -10.395, \"z\": -0.4}, \"end\": {\"x\": -0.988, \"y\": -11.173, \"z\": -2.234}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.662, \"y\": -8.943, \"z\": 5.423}, \"end\": {\"x\": -1.821, \"y\": -9.749, \"z\": 3.831}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.51, \"y\": -8.484, \"z\": 1.221}, \"end\": {\"x\": 0.435, \"y\": -8.651, \"z\": -0.37}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.179, \"y\": -6.56, \"z\": 6.496}, \"end\": {\"x\": -1.984, \"y\": -6.457, \"z\": 4.79}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.67, \"y\": -6.971, \"z\": 1.774}, \"end\": {\"x\": -2.265, \"y\": -6.751, \"z\": -0.229}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.749, \"y\": -6.559, \"z\": 7.157}, \"end\": {\"x\": -5.357, \"y\": -6.529, \"z\": 5.084}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.028, \"y\": -8.628, \"z\": 3.211}, \"end\": {\"x\": -4.056, \"y\": -9.493, \"z\": 1.321}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.941, \"y\": -8.731, \"z\": 8.972}, \"end\": {\"x\": -5.197, \"y\": -9.411, \"z\": 7.212}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.662, \"y\": -8.943, \"z\": 5.423}, \"end\": {\"x\": -1.821, \"y\": -9.749, \"z\": 3.831}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.251, \"y\": -7.753, \"z\": 11.056}, \"end\": {\"x\": -3.347, \"y\": -7.576, \"z\": 9.493}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.179, \"y\": -6.56, \"z\": 6.496}, \"end\": {\"x\": -1.984, \"y\": -6.457, \"z\": 4.79}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.754, \"y\": -5.59, \"z\": 11.988}, \"end\": {\"x\": -5.452, \"y\": -4.966, \"z\": 9.777}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.527, \"y\": -2.98, \"z\": 9.628}, \"end\": {\"x\": -5.452, \"y\": -4.966, \"z\": 9.777}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.749, \"y\": -6.559, \"z\": 7.157}, \"end\": {\"x\": -5.357, \"y\": -6.529, \"z\": 5.084}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.966, \"y\": -6.669, \"z\": 12.827}, \"end\": {\"x\": -7.871, \"y\": -7.084, \"z\": 10.874}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.941, \"y\": -8.731, \"z\": 8.972}, \"end\": {\"x\": -5.197, \"y\": -9.411, \"z\": 7.212}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.203, \"y\": -7.688, \"z\": 15.048}, \"end\": {\"x\": -6.117, \"y\": -8.797, \"z\": 13.286}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.251, \"y\": -7.753, \"z\": 11.056}, \"end\": {\"x\": -3.347, \"y\": -7.576, \"z\": 9.493}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.089, \"y\": -8.036, \"z\": 16.242}, \"end\": {\"x\": -4.966, \"y\": -6.071, \"z\": 14.962}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.327, \"y\": -5.264, \"z\": 16.73}, \"end\": {\"x\": -4.966, \"y\": -6.071, \"z\": 14.962}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.754, \"y\": -5.59, \"z\": 11.988}, \"end\": {\"x\": -5.452, \"y\": -4.966, \"z\": 9.777}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.527, \"y\": -2.98, \"z\": 9.628}, \"end\": {\"x\": -5.452, \"y\": -4.966, \"z\": 9.777}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.966, \"y\": -6.669, \"z\": 12.827}, \"end\": {\"x\": -7.871, \"y\": -7.084, \"z\": 10.874}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.203, \"y\": -7.688, \"z\": 15.048}, \"end\": {\"x\": -6.117, \"y\": -8.797, \"z\": 13.286}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.393, \"y\": -6.73, \"z\": 20.135}, \"end\": {\"x\": -6.887, \"y\": -8.082, \"z\": 18.812}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.089, \"y\": -8.036, \"z\": 16.242}, \"end\": {\"x\": -4.966, \"y\": -6.071, \"z\": 14.962}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.77, \"y\": -4.826, \"z\": 21.39}, \"end\": {\"x\": -4.811, \"y\": -5.653, \"z\": 19.843}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.327, \"y\": -5.264, \"z\": 16.73}, \"end\": {\"x\": -4.966, \"y\": -6.071, \"z\": 14.962}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.441, \"y\": -1.615, \"z\": 18.934}, \"end\": {\"x\": -3.352, \"y\": -1.552, \"z\": 17.958}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.648, \"y\": -2.874, \"z\": 22.009}, \"end\": {\"x\": -7.13, \"y\": -2.972, \"z\": 19.976}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.075, \"y\": -4.761, \"z\": 23.66}, \"end\": {\"x\": -9.416, \"y\": -4.917, \"z\": 21.754}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.637, \"y\": -5.949, \"z\": 24.982}, \"end\": {\"x\": -7.049, \"y\": -5.334, \"z\": 24.214}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.393, \"y\": -6.73, \"z\": 20.135}, \"end\": {\"x\": -6.887, \"y\": -8.082, \"z\": 18.812}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.75, \"y\": -4.404, \"z\": 21.029}, \"end\": {\"x\": 1.023, \"y\": -5.709, \"z\": 21.431}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.297, \"y\": -4.328, \"z\": 20.1}, \"end\": {\"x\": 1.023, \"y\": -5.709, \"z\": 21.431}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.77, \"y\": -4.826, \"z\": 21.39}, \"end\": {\"x\": -4.811, \"y\": -5.653, \"z\": 19.843}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.441, \"y\": -1.615, \"z\": 18.934}, \"end\": {\"x\": -3.352, \"y\": -1.552, \"z\": 17.958}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.93, \"y\": -1.188, \"z\": 25.922}, \"end\": {\"x\": -6.1, \"y\": -1.778, \"z\": 24.458}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.139, \"y\": -2.176, \"z\": 26.586}, \"end\": {\"x\": -6.1, \"y\": -1.778, \"z\": 24.458}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.648, \"y\": -2.874, \"z\": 22.009}, \"end\": {\"x\": -7.13, \"y\": -2.972, \"z\": 19.976}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.674, \"y\": -2.486, \"z\": 20.637}, \"end\": {\"x\": -10.288, \"y\": -1.952, \"z\": 22.071}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.249, \"y\": -1.676, \"z\": 20.919}, \"end\": {\"x\": -13.636, \"y\": -1.159, \"z\": 22.603}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.694, \"y\": -0.579, \"z\": 18.329}, \"end\": {\"x\": -13.69, \"y\": 0.719, \"z\": 19.743}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.258, \"y\": 0.834, \"z\": 23.301}, \"end\": {\"x\": -9.436, \"y\": 1.454, \"z\": 24.665}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.325, \"y\": 6.426, \"z\": 17.27}, \"end\": {\"x\": -12.134, \"y\": 7.187, \"z\": 16.168}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.713, \"y\": -2.846, \"z\": 16.83}, \"end\": {\"x\": -12.891, \"y\": -2.293, \"z\": 17.733}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.586, \"y\": -5.825, \"z\": 25.783}, \"end\": {\"x\": -13.59, \"y\": -5.122, \"z\": 24.075}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.777, \"y\": -4.718, \"z\": 24.065}, \"end\": {\"x\": -13.59, \"y\": -5.122, \"z\": 24.075}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.674, \"y\": -2.486, \"z\": 20.637}, \"end\": {\"x\": -10.288, \"y\": -1.952, \"z\": 22.071}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.784, \"y\": -4.083, \"z\": 17.731}, \"end\": {\"x\": -15.356, \"y\": -3.963, \"z\": 19.016}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.249, \"y\": -1.676, \"z\": 20.919}, \"end\": {\"x\": -13.636, \"y\": -1.159, \"z\": 22.603}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.894, \"y\": -2.572, \"z\": 16.998}, \"end\": {\"x\": -17.686, \"y\": -1.491, \"z\": 18.255}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.694, \"y\": -0.579, \"z\": 18.329}, \"end\": {\"x\": -13.69, \"y\": 0.719, \"z\": 19.743}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.747, \"y\": -2.894, \"z\": 14.157}, \"end\": {\"x\": -16.87, \"y\": -1.799, \"z\": 14.688}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.331, \"y\": -0.747, \"z\": 13.16}, \"end\": {\"x\": -16.87, \"y\": -1.799, \"z\": 14.688}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.928, \"y\": 0.95, \"z\": 13.294}, \"end\": {\"x\": -15.29, \"y\": 2.555, \"z\": 14.529}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.713, \"y\": -2.846, \"z\": 16.83}, \"end\": {\"x\": -12.891, \"y\": -2.293, \"z\": 17.733}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -19.443, \"y\": -5.559, \"z\": 14.015}, \"end\": {\"x\": -17.665, \"y\": -5.338, \"z\": 14.975}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.784, \"y\": -4.083, \"z\": 17.731}, \"end\": {\"x\": -15.356, \"y\": -3.963, \"z\": 19.016}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -21.913, \"y\": -5.422, \"z\": 15.008}, \"end\": {\"x\": -20.79, \"y\": -4.985, \"z\": 16.381}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.894, \"y\": -2.572, \"z\": 16.998}, \"end\": {\"x\": -17.686, \"y\": -1.491, \"z\": 18.255}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -23.403, \"y\": -4.155, \"z\": 13.054}, \"end\": {\"x\": -21.865, \"y\": -2.847, \"z\": 13.818}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -18.747, \"y\": -2.894, \"z\": 14.157}, \"end\": {\"x\": -16.87, \"y\": -1.799, \"z\": 14.688}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.331, \"y\": -0.747, \"z\": 13.16}, \"end\": {\"x\": -16.87, \"y\": -1.799, \"z\": 14.688}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.928, \"y\": 0.95, \"z\": 13.294}, \"end\": {\"x\": -15.29, \"y\": 2.555, \"z\": 14.529}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -21.822, \"y\": -4.784, \"z\": 9.676}, \"end\": {\"x\": -21.187, \"y\": -5.158, \"z\": 11.393}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.243, \"y\": -0.994, \"z\": 9.81}, \"end\": {\"x\": -14.301, \"y\": -0.66, \"z\": 7.978}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -19.443, \"y\": -5.559, \"z\": 14.015}, \"end\": {\"x\": -17.665, \"y\": -5.338, \"z\": 14.975}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -21.913, \"y\": -5.422, \"z\": 15.008}, \"end\": {\"x\": -20.79, \"y\": -4.985, \"z\": 16.381}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -23.403, \"y\": -4.155, \"z\": 13.054}, \"end\": {\"x\": -21.865, \"y\": -2.847, \"z\": 13.818}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -21.822, \"y\": -4.784, \"z\": 9.676}, \"end\": {\"x\": -21.187, \"y\": -5.158, \"z\": 11.393}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.937, \"y\": -4.497, \"z\": 2.854}, \"end\": {\"x\": -15.799, \"y\": -2.545, \"z\": 2.238}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.906, \"y\": -8.396, \"z\": 1.714}, \"end\": {\"x\": -14.987, \"y\": -9.799, \"z\": 0.836}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.059, \"y\": -7.079, \"z\": 1.524}, \"end\": {\"x\": -13.12, \"y\": -8.975, \"z\": 1.582}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.444, \"y\": -2.133, \"z\": 7.434}, \"end\": {\"x\": -14.301, \"y\": -0.66, \"z\": 7.978}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.129, \"y\": -2.166, \"z\": 7.476}, \"end\": {\"x\": -15.396, \"y\": -3.811, \"z\": 8.462}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.243, \"y\": -0.994, \"z\": 9.81}, \"end\": {\"x\": -14.301, \"y\": -0.66, \"z\": 7.978}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.944, \"y\": 6.285, \"z\": 15.156}, \"end\": {\"x\": -14.596, \"y\": 6.761, \"z\": 13.65}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.37, \"y\": 5.982, \"z\": 14.451}, \"end\": {\"x\": -9.906, \"y\": 6.117, \"z\": 12.315}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.37, \"y\": 5.982, \"z\": 14.451}, \"end\": {\"x\": -12.484, \"y\": 6.509, \"z\": 13.912}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.64, \"y\": 6.666, \"z\": 15.531}, \"end\": {\"x\": -12.484, \"y\": 6.509, \"z\": 13.912}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.279, \"y\": 7.325, \"z\": 14.346}, \"end\": {\"x\": -7.865, \"y\": 8.483, \"z\": 14.037}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.325, \"y\": 6.426, \"z\": 17.27}, \"end\": {\"x\": -12.134, \"y\": 7.187, \"z\": 16.168}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.64, \"y\": 6.666, \"z\": 15.531}, \"end\": {\"x\": -12.484, \"y\": 6.509, \"z\": 13.912}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.338, \"y\": 6.219, \"z\": 17.141}, \"end\": {\"x\": -8.423, \"y\": 6.687, \"z\": 16.852}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.37, \"y\": 5.982, \"z\": 14.451}, \"end\": {\"x\": -9.906, \"y\": 6.117, \"z\": 12.315}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.37, \"y\": 5.982, \"z\": 14.451}, \"end\": {\"x\": -12.484, \"y\": 6.509, \"z\": 13.912}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.176, \"y\": 3.63, \"z\": 16.728}, \"end\": {\"x\": -6.842, \"y\": 3.88, \"z\": 15.688}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.593, \"y\": 4.645, \"z\": 15.798}, \"end\": {\"x\": -3.963, \"y\": 5.19, \"z\": 14.499}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.279, \"y\": 7.325, \"z\": 14.346}, \"end\": {\"x\": -7.865, \"y\": 8.483, \"z\": 14.037}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.338, \"y\": 6.219, \"z\": 17.141}, \"end\": {\"x\": -8.423, \"y\": 6.687, \"z\": 16.852}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.708, \"y\": 5.136, \"z\": 18.433}, \"end\": {\"x\": -3.282, \"y\": 4.059, \"z\": 19.181}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -3.344, \"y\": 2.03, \"z\": 12.263}, \"end\": {\"x\": -4.925, \"y\": 1.221, \"z\": 13.173}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.176, \"y\": 3.63, \"z\": 16.728}, \"end\": {\"x\": -6.842, \"y\": 3.88, \"z\": 15.688}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.583, \"y\": 3.469, \"z\": 19.669}, \"end\": {\"x\": -1.03, \"y\": 2.446, \"z\": 17.386}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.088, \"y\": 3.102, \"z\": 12.841}, \"end\": {\"x\": -0.599, \"y\": 4.459, \"z\": 13.361}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.598, \"y\": 4.009, \"z\": 11.408}, \"end\": {\"x\": -0.599, \"y\": 4.459, \"z\": 13.361}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.593, \"y\": 4.645, \"z\": 15.798}, \"end\": {\"x\": -3.963, \"y\": 5.19, \"z\": 14.499}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.708, \"y\": 5.136, \"z\": 18.433}, \"end\": {\"x\": -3.282, \"y\": 4.059, \"z\": 19.181}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.74, \"y\": 5.593, \"z\": 23.674}, \"end\": {\"x\": 0.11, \"y\": 5.475, \"z\": 22.029}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.583, \"y\": 3.469, \"z\": 19.669}, \"end\": {\"x\": -1.03, \"y\": 2.446, \"z\": 17.386}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.515, \"y\": -1.052, \"z\": 24.635}, \"end\": {\"x\": -2.756, \"y\": -1.57, \"z\": 26.565}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.027, \"y\": 3.739, \"z\": 25.509}, \"end\": {\"x\": 0.659, \"y\": 2.753, \"z\": 23.741}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.393, \"y\": 4.576, \"z\": 26.672}, \"end\": {\"x\": 3.689, \"y\": 3.901, \"z\": 24.681}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.686, \"y\": 2.859, \"z\": 24.625}, \"end\": {\"x\": 3.689, \"y\": 3.901, \"z\": 24.681}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.689, \"y\": 7.008, \"z\": 27.619}, \"end\": {\"x\": 2.486, \"y\": 7.092, \"z\": 25.757}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.74, \"y\": 5.593, \"z\": 23.674}, \"end\": {\"x\": 0.11, \"y\": 5.475, \"z\": 22.029}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.832, \"y\": 6.253, \"z\": 29.636}, \"end\": {\"x\": 0.221, \"y\": 5.519, \"z\": 27.967}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.027, \"y\": 3.739, \"z\": 25.509}, \"end\": {\"x\": 0.659, \"y\": 2.753, \"z\": 23.741}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.631, \"y\": -1.271, \"z\": 26.279}, \"end\": {\"x\": 1.03, \"y\": 1.061, \"z\": 26.186}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.07, \"y\": 4.583, \"z\": 31.373}, \"end\": {\"x\": 2.539, \"y\": 3.681, \"z\": 29.534}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.393, \"y\": 4.576, \"z\": 26.672}, \"end\": {\"x\": 3.689, \"y\": 3.901, \"z\": 24.681}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.973, \"y\": 2.733, \"z\": 26.798}, \"end\": {\"x\": 6.101, \"y\": 4.672, \"z\": 26.717}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.988, \"y\": 6.545, \"z\": 32.294}, \"end\": {\"x\": 4.174, \"y\": 6.483, \"z\": 30.435}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.689, \"y\": 7.008, \"z\": 27.619}, \"end\": {\"x\": 2.486, \"y\": 7.092, \"z\": 25.757}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 0.832, \"y\": 6.253, \"z\": 29.636}, \"end\": {\"x\": 0.221, \"y\": 5.519, \"z\": 27.967}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.773, \"y\": 7.606, \"z\": 34.102}, \"end\": {\"x\": 0.983, \"y\": 6.142, \"z\": 33.882}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.07, \"y\": 4.583, \"z\": 31.373}, \"end\": {\"x\": 2.539, \"y\": 3.681, \"z\": 29.534}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.695, \"y\": -0.781, \"z\": 31.377}, \"end\": {\"x\": 4.263, \"y\": 0.155, \"z\": 32.27}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 3.988, \"y\": 6.545, \"z\": 32.294}, \"end\": {\"x\": 4.174, \"y\": 6.483, \"z\": 30.435}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 2.773, \"y\": 7.606, \"z\": 34.102}, \"end\": {\"x\": 0.983, \"y\": 6.142, \"z\": 33.882}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.425, \"y\": 9.212, \"z\": 37.403}, \"end\": {\"x\": 0.425, \"y\": 8.375, \"z\": 35.426}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.919, \"y\": 11.681, \"z\": 36.115}, \"end\": {\"x\": 0.753, \"y\": 12.754, \"z\": 34.984}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.08, \"y\": 11.739, \"z\": 37.129}, \"end\": {\"x\": 0.087, \"y\": 12.023, \"z\": 38.762}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -0.425, \"y\": 9.212, \"z\": 37.403}, \"end\": {\"x\": 0.425, \"y\": 8.375, \"z\": 35.426}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.497, \"y\": 10.22, \"z\": 33.793}, \"end\": {\"x\": -3.001, \"y\": 10.839, \"z\": 34.791}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.919, \"y\": 11.681, \"z\": 36.115}, \"end\": {\"x\": 0.753, \"y\": 12.754, \"z\": 34.984}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.08, \"y\": 11.739, \"z\": 37.129}, \"end\": {\"x\": 0.087, \"y\": 12.023, \"z\": 38.762}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.485, \"y\": 13.718, \"z\": 33.014}, \"end\": {\"x\": -2.923, \"y\": 14.47, \"z\": 32.052}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.341, \"y\": 17.653, \"z\": 29.536}, \"end\": {\"x\": 3.846, \"y\": 17.168, \"z\": 31.304}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.341, \"y\": 17.653, \"z\": 29.536}, \"end\": {\"x\": 3.846, \"y\": 17.168, \"z\": 31.304}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.677, \"y\": 14.03, \"z\": 25.818}, \"end\": {\"x\": -3.18, \"y\": 14.362, \"z\": 27.454}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.217, \"y\": 14.549, \"z\": 24.947}, \"end\": {\"x\": -3.025, \"y\": 13.772, \"z\": 23.646}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.045, \"y\": 13.877, \"z\": 20.751}, \"end\": {\"x\": -5.609, \"y\": 15.548, \"z\": 21.261}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.627, \"y\": 15.875, \"z\": 19.267}, \"end\": {\"x\": -6.149, \"y\": 13.997, \"z\": 18.375}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.689, \"y\": 13.331, \"z\": 13.328}, \"end\": {\"x\": -11.332, \"y\": 11.982, \"z\": 14.629}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.112, \"y\": 13.951, \"z\": 16.712}, \"end\": {\"x\": -9.228, \"y\": 15.554, \"z\": 16.197}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.069, \"y\": 17.57, \"z\": 17.789}, \"end\": {\"x\": -12.014, \"y\": 17.233, \"z\": 15.79}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -15.944, \"y\": 6.285, \"z\": 15.156}, \"end\": {\"x\": -14.596, \"y\": 6.761, \"z\": 13.65}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.995, \"y\": 11.453, \"z\": 16.297}, \"end\": {\"x\": -15.032, \"y\": 11.567, \"z\": 17.846}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.318, \"y\": 8.481, \"z\": 17.599}, \"end\": {\"x\": -18.765, \"y\": 9.805, \"z\": 18.36}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.389, \"y\": 15.77, \"z\": 13.829}, \"end\": {\"x\": -13.457, \"y\": 15.789, \"z\": 14.256}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.689, \"y\": 13.331, \"z\": 13.328}, \"end\": {\"x\": -11.332, \"y\": 11.982, \"z\": 14.629}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.112, \"y\": 13.951, \"z\": 16.712}, \"end\": {\"x\": -9.228, \"y\": 15.554, \"z\": 16.197}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -12.069, \"y\": 17.57, \"z\": 17.789}, \"end\": {\"x\": -12.014, \"y\": 17.233, \"z\": 15.79}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.389, \"y\": 15.77, \"z\": 13.829}, \"end\": {\"x\": -13.457, \"y\": 15.789, \"z\": 14.256}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.045, \"y\": 13.877, \"z\": 20.751}, \"end\": {\"x\": -5.609, \"y\": 15.548, \"z\": 21.261}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.627, \"y\": 15.875, \"z\": 19.267}, \"end\": {\"x\": -6.149, \"y\": 13.997, \"z\": 18.375}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.677, \"y\": 14.03, \"z\": 25.818}, \"end\": {\"x\": -3.18, \"y\": 14.362, \"z\": 27.454}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.217, \"y\": 14.549, \"z\": 24.947}, \"end\": {\"x\": -3.025, \"y\": 13.772, \"z\": 23.646}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.485, \"y\": 13.718, \"z\": 33.014}, \"end\": {\"x\": -2.923, \"y\": 14.47, \"z\": 32.052}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.262, \"y\": 10.12, \"z\": 33.809}, \"end\": {\"x\": -7.445, \"y\": 12.375, \"z\": 33.824}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -4.497, \"y\": 10.22, \"z\": 33.793}, \"end\": {\"x\": -3.001, \"y\": 10.839, \"z\": 34.791}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.401, \"y\": 8.138, \"z\": 30.364}, \"end\": {\"x\": -7.412, \"y\": 7.629, \"z\": 31.961}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -7.262, \"y\": 10.12, \"z\": 33.809}, \"end\": {\"x\": -7.445, \"y\": 12.375, \"z\": 33.824}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.314, \"y\": 6.093, \"z\": 31.194}, \"end\": {\"x\": -10.629, \"y\": 7.787, \"z\": 32.324}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.401, \"y\": 8.138, \"z\": 30.364}, \"end\": {\"x\": -7.412, \"y\": 7.629, \"z\": 31.961}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.392, \"y\": 3.526, \"z\": 27.974}, \"end\": {\"x\": -8.698, \"y\": 4.382, \"z\": 28.65}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.592, \"y\": 2.694, \"z\": 27.423}, \"end\": {\"x\": -8.698, \"y\": 4.382, \"z\": 28.65}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -9.314, \"y\": 6.093, \"z\": 31.194}, \"end\": {\"x\": -10.629, \"y\": 7.787, \"z\": 32.324}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -11.258, \"y\": 0.834, \"z\": 23.301}, \"end\": {\"x\": -9.436, \"y\": 1.454, \"z\": 24.665}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -10.392, \"y\": 3.526, \"z\": 27.974}, \"end\": {\"x\": -8.698, \"y\": 4.382, \"z\": 28.65}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -8.592, \"y\": 2.694, \"z\": 27.423}, \"end\": {\"x\": -8.698, \"y\": 4.382, \"z\": 28.65}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -6.93, \"y\": -1.188, \"z\": 25.922}, \"end\": {\"x\": -6.1, \"y\": -1.778, \"z\": 24.458}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -5.139, \"y\": -2.176, \"z\": 26.586}, \"end\": {\"x\": -6.1, \"y\": -1.778, \"z\": 24.458}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -2.515, \"y\": -1.052, \"z\": 24.635}, \"end\": {\"x\": -2.756, \"y\": -1.57, \"z\": 26.565}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -1.481, \"y\": -3.161, \"z\": 28.826}, \"end\": {\"x\": -2.518, \"y\": -3.408, \"z\": 30.737}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.033, \"y\": -4.935, \"z\": 29.523}, \"end\": {\"x\": 1.041, \"y\": -3.403, \"z\": 28.444}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 1.631, \"y\": -1.271, \"z\": 26.279}, \"end\": {\"x\": 1.03, \"y\": 1.061, \"z\": 26.186}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.686, \"y\": 2.859, \"z\": 24.625}, \"end\": {\"x\": 3.689, \"y\": 3.901, \"z\": 24.681}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.973, \"y\": 2.733, \"z\": 26.798}, \"end\": {\"x\": 6.101, \"y\": 4.672, \"z\": 26.717}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 5.695, \"y\": -0.781, \"z\": 31.377}, \"end\": {\"x\": 4.263, \"y\": 0.155, \"z\": 32.27}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.876, \"y\": 22.349, \"z\": 21.406}, \"end\": {\"x\": 24.434, \"y\": 23.709, \"z\": 22.556}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.957, \"y\": 22.392, \"z\": 16.299}, \"end\": {\"x\": 31.173, \"y\": 22.966, \"z\": 15.497}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.563, \"y\": 22.402, \"z\": 12.5}, \"end\": {\"x\": 34.098, \"y\": 21.846, \"z\": 14.328}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.841, \"y\": 21.575, \"z\": 10.849}, \"end\": {\"x\": 37.798, \"y\": 23.131, \"z\": 11.85}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.169, \"y\": 18.688, \"z\": 11.876}, \"end\": {\"x\": 40.518, \"y\": 18.929, \"z\": 10.492}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.413, \"y\": 17.006, \"z\": 10.162}, \"end\": {\"x\": 40.518, \"y\": 18.929, \"z\": 10.492}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.841, \"y\": 21.575, \"z\": 10.849}, \"end\": {\"x\": 37.798, \"y\": 23.131, \"z\": 11.85}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.169, \"y\": 18.688, \"z\": 11.876}, \"end\": {\"x\": 40.518, \"y\": 18.929, \"z\": 10.492}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.563, \"y\": 15.607, \"z\": 18.51}, \"end\": {\"x\": 42.179, \"y\": 15.072, \"z\": 17.459}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.656, \"y\": 17.432, \"z\": 16.45}, \"end\": {\"x\": 41.164, \"y\": 18.257, \"z\": 17.532}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.769, \"y\": 16.711, \"z\": 19.413}, \"end\": {\"x\": 44.503, \"y\": 17.302, \"z\": 18.364}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.903, \"y\": 13.802, \"z\": 20.555}, \"end\": {\"x\": 41.772, \"y\": 13.437, \"z\": 20.248}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.416, \"y\": 16.376, \"z\": 25.647}, \"end\": {\"x\": 40.416, \"y\": 16.704, \"z\": 25.603}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.399, \"y\": 15.87, \"z\": 22.673}, \"end\": {\"x\": 41.162, \"y\": 16.183, \"z\": 22.115}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.656, \"y\": 17.432, \"z\": 16.45}, \"end\": {\"x\": 41.164, \"y\": 18.257, \"z\": 17.532}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.769, \"y\": 16.711, \"z\": 19.413}, \"end\": {\"x\": 44.503, \"y\": 17.302, \"z\": 18.364}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.23, \"y\": 16.95, \"z\": 21.688}, \"end\": {\"x\": 38.647, \"y\": 17.543, \"z\": 20.151}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.563, \"y\": 15.607, \"z\": 18.51}, \"end\": {\"x\": 42.179, \"y\": 15.072, \"z\": 17.459}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.335, \"y\": 14.839, \"z\": 21.537}, \"end\": {\"x\": 36.91, \"y\": 14.522, \"z\": 20.478}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.903, \"y\": 13.802, \"z\": 20.555}, \"end\": {\"x\": 41.772, \"y\": 13.437, \"z\": 20.248}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.12, \"y\": 14.384, \"z\": 24.281}, \"end\": {\"x\": 36.957, \"y\": 14.208, \"z\": 23.826}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 42.416, \"y\": 16.376, \"z\": 25.647}, \"end\": {\"x\": 40.416, \"y\": 16.704, \"z\": 25.603}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 39.399, \"y\": 15.87, \"z\": 22.673}, \"end\": {\"x\": 41.162, \"y\": 16.183, \"z\": 22.115}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.897, \"y\": 16.77, \"z\": 24.884}, \"end\": {\"x\": 35.824, \"y\": 17.276, \"z\": 24.419}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.23, \"y\": 16.95, \"z\": 21.688}, \"end\": {\"x\": 38.647, \"y\": 17.543, \"z\": 20.151}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.54, \"y\": 16.646, \"z\": 23.868}, \"end\": {\"x\": 33.142, \"y\": 16.76, \"z\": 22.646}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.335, \"y\": 14.839, \"z\": 21.537}, \"end\": {\"x\": 36.91, \"y\": 14.522, \"z\": 20.478}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.345, \"y\": 14.446, \"z\": 25.174}, \"end\": {\"x\": 31.977, \"y\": 13.994, \"z\": 24.242}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.12, \"y\": 14.384, \"z\": 24.281}, \"end\": {\"x\": 36.957, \"y\": 14.208, \"z\": 23.826}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.307, \"y\": 15.704, \"z\": 27.703}, \"end\": {\"x\": 32.283, \"y\": 15.529, \"z\": 27.371}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.897, \"y\": 16.77, \"z\": 24.884}, \"end\": {\"x\": 35.824, \"y\": 17.276, \"z\": 24.419}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.199, \"y\": 17.309, \"z\": 27.401}, \"end\": {\"x\": 30.041, \"y\": 18.048, \"z\": 26.289}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.54, \"y\": 16.646, \"z\": 23.868}, \"end\": {\"x\": 33.142, \"y\": 16.76, \"z\": 22.646}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.304, \"y\": 16.069, \"z\": 27.208}, \"end\": {\"x\": 27.648, \"y\": 15.935, \"z\": 25.526}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.345, \"y\": 14.446, \"z\": 25.174}, \"end\": {\"x\": 31.977, \"y\": 13.994, \"z\": 24.242}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.984, \"y\": 8.251, \"z\": 23.58}, \"end\": {\"x\": 31.365, \"y\": 8.299, \"z\": 21.71}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.471, \"y\": 15.046, \"z\": 29.295}, \"end\": {\"x\": 27.569, \"y\": 14.355, \"z\": 28.536}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.585, \"y\": 9.377, \"z\": 24.985}, \"end\": {\"x\": 29.035, \"y\": 8.741, \"z\": 24.226}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.568, \"y\": 9.273, \"z\": 25.687}, \"end\": {\"x\": 35.521, \"y\": 8.526, \"z\": 23.943}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.693, \"y\": 8.18, \"z\": 23.949}, \"end\": {\"x\": 35.521, \"y\": 8.526, \"z\": 23.943}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.307, \"y\": 15.704, \"z\": 27.703}, \"end\": {\"x\": 32.283, \"y\": 15.529, \"z\": 27.371}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.199, \"y\": 17.309, \"z\": 27.401}, \"end\": {\"x\": 30.041, \"y\": 18.048, \"z\": 26.289}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.304, \"y\": 16.069, \"z\": 27.208}, \"end\": {\"x\": 27.648, \"y\": 15.935, \"z\": 25.526}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.471, \"y\": 15.046, \"z\": 29.295}, \"end\": {\"x\": 27.569, \"y\": 14.355, \"z\": 28.536}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.235, \"y\": 5.611, \"z\": 36.68}, \"end\": {\"x\": 20.722, \"y\": 7.374, \"z\": 36.849}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.376, \"y\": 6.717, \"z\": 35.935}, \"end\": {\"x\": 19.119, \"y\": 8.419, \"z\": 35.69}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.498, \"y\": 8.386, \"z\": 33.373}, \"end\": {\"x\": 23.534, \"y\": 8.698, \"z\": 35.23}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.235, \"y\": 5.611, \"z\": 36.68}, \"end\": {\"x\": 20.722, \"y\": 7.374, \"z\": 36.849}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.376, \"y\": 6.717, \"z\": 35.935}, \"end\": {\"x\": 19.119, \"y\": 8.419, \"z\": 35.69}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.852, \"y\": 4.756, \"z\": 35.474}, \"end\": {\"x\": 25.323, \"y\": 5.198, \"z\": 33.647}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.219, \"y\": 3.35, \"z\": 34.982}, \"end\": {\"x\": 25.658, \"y\": 2.249, \"z\": 35.072}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.852, \"y\": 4.756, \"z\": 35.474}, \"end\": {\"x\": 25.323, \"y\": 5.198, \"z\": 33.647}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.219, \"y\": 3.35, \"z\": 34.982}, \"end\": {\"x\": 25.658, \"y\": 2.249, \"z\": 35.072}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.504, \"y\": 6.59, \"z\": 28.902}, \"end\": {\"x\": 24.635, \"y\": 6.904, \"z\": 30.764}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.498, \"y\": 8.386, \"z\": 33.373}, \"end\": {\"x\": 23.534, \"y\": 8.698, \"z\": 35.23}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.1, \"y\": 8.452, \"z\": 29.548}, \"end\": {\"x\": 20.985, \"y\": 6.93, \"z\": 28.587}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.707, \"y\": 10.989, \"z\": 29.477}, \"end\": {\"x\": 11.729, \"y\": 9.651, \"z\": 30.58}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.959, \"y\": 11.943, \"z\": 21.698}, \"end\": {\"x\": 9.866, \"y\": 13.537, \"z\": 19.976}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.008, \"y\": 11.869, \"z\": 20.545}, \"end\": {\"x\": 9.866, \"y\": 13.537, \"z\": 19.976}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.416, \"y\": 15.383, \"z\": 29.721}, \"end\": {\"x\": 10.361, \"y\": 13.885, \"z\": 28.212}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.707, \"y\": 10.989, \"z\": 29.477}, \"end\": {\"x\": 11.729, \"y\": 9.651, \"z\": 30.58}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.416, \"y\": 15.383, \"z\": 29.721}, \"end\": {\"x\": 10.361, \"y\": 13.885, \"z\": 28.212}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.649, \"y\": 21.983, \"z\": 27.993}, \"end\": {\"x\": 13.194, \"y\": 20.654, \"z\": 28.044}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.612, \"y\": 21.155, \"z\": 25.748}, \"end\": {\"x\": 13.978, \"y\": 22.481, \"z\": 25.102}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.649, \"y\": 21.983, \"z\": 27.993}, \"end\": {\"x\": 13.194, \"y\": 20.654, \"z\": 28.044}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.612, \"y\": 21.155, \"z\": 25.748}, \"end\": {\"x\": 13.978, \"y\": 22.481, \"z\": 25.102}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.392, \"y\": 15.143, \"z\": 22.88}, \"end\": {\"x\": 16.141, \"y\": 14.266, \"z\": 22.627}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.136, \"y\": 14.06, \"z\": 21.495}, \"end\": {\"x\": 14.864, \"y\": 13.796, \"z\": 20.782}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.292, \"y\": 16.284, \"z\": 23.573}, \"end\": {\"x\": 17.109, \"y\": 17.717, \"z\": 23.734}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.896, \"y\": 19.049, \"z\": 21.962}, \"end\": {\"x\": 19.994, \"y\": 17.511, \"z\": 20.877}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.719, \"y\": 18.644, \"z\": 18.979}, \"end\": {\"x\": 22.152, \"y\": 20.108, \"z\": 19.993}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.876, \"y\": 22.349, \"z\": 21.406}, \"end\": {\"x\": 24.434, \"y\": 23.709, \"z\": 22.556}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.093, \"y\": 21.032, \"z\": 17.334}, \"end\": {\"x\": 23.55, \"y\": 20.736, \"z\": 16.398}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.673, \"y\": 20.95, \"z\": 14.237}, \"end\": {\"x\": 27.097, \"y\": 21.416, \"z\": 15.275}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.957, \"y\": 22.392, \"z\": 16.299}, \"end\": {\"x\": 31.173, \"y\": 22.966, \"z\": 15.497}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.1, \"y\": 22.475, \"z\": 13.172}, \"end\": {\"x\": 27.722, \"y\": 23.063, \"z\": 12.072}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.201, \"y\": 23.513, \"z\": 10.105}, \"end\": {\"x\": 30.914, \"y\": 23.264, \"z\": 10.982}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.563, \"y\": 22.402, \"z\": 12.5}, \"end\": {\"x\": 34.098, \"y\": 21.846, \"z\": 14.328}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.867, \"y\": 24.067, \"z\": 8.464}, \"end\": {\"x\": 32.663, \"y\": 24.95, \"z\": 7.979}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.266, \"y\": 25.421, \"z\": 5.933}, \"end\": {\"x\": 34.841, \"y\": 23.472, \"z\": 5.961}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.648, \"y\": 21.849, \"z\": 4.529}, \"end\": {\"x\": 33.091, \"y\": 21.217, \"z\": 2.538}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.103, \"y\": 20.472, \"z\": 5.476}, \"end\": {\"x\": 33.87, \"y\": 18.443, \"z\": 4.902}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.214, \"y\": 21.583, \"z\": 6.93}, \"end\": {\"x\": 38.089, \"y\": 19.966, \"z\": 6.516}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.867, \"y\": 24.067, \"z\": 8.464}, \"end\": {\"x\": 32.663, \"y\": 24.95, \"z\": 7.979}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.266, \"y\": 25.421, \"z\": 5.933}, \"end\": {\"x\": 34.841, \"y\": 23.472, \"z\": 5.961}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.1, \"y\": 22.475, \"z\": 13.172}, \"end\": {\"x\": 27.722, \"y\": 23.063, \"z\": 12.072}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.201, \"y\": 23.513, \"z\": 10.105}, \"end\": {\"x\": 30.914, \"y\": 23.264, \"z\": 10.982}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.769, \"y\": 23.904, \"z\": 11.282}, \"end\": {\"x\": 23.717, \"y\": 24.466, \"z\": 9.81}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.258, \"y\": 23.006, \"z\": 11.449}, \"end\": {\"x\": 23.16, \"y\": 22.291, \"z\": 12.88}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.093, \"y\": 21.032, \"z\": 17.334}, \"end\": {\"x\": 23.55, \"y\": 20.736, \"z\": 16.398}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.673, \"y\": 20.95, \"z\": 14.237}, \"end\": {\"x\": 27.097, \"y\": 21.416, \"z\": 15.275}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.248, \"y\": 16.107, \"z\": 15.597}, \"end\": {\"x\": 20.428, \"y\": 17.289, \"z\": 16.325}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.406, \"y\": 19.29, \"z\": 14.599}, \"end\": {\"x\": 20.528, \"y\": 18.873, \"z\": 12.955}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.22, \"y\": 17.276, \"z\": 13.809}, \"end\": {\"x\": 20.428, \"y\": 17.289, \"z\": 16.325}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.896, \"y\": 19.049, \"z\": 21.962}, \"end\": {\"x\": 19.994, \"y\": 17.511, \"z\": 20.877}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.719, \"y\": 18.644, \"z\": 18.979}, \"end\": {\"x\": 22.152, \"y\": 20.108, \"z\": 19.993}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.392, \"y\": 15.143, \"z\": 22.88}, \"end\": {\"x\": 16.141, \"y\": 14.266, \"z\": 22.627}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.136, \"y\": 14.06, \"z\": 21.495}, \"end\": {\"x\": 14.864, \"y\": 13.796, \"z\": 20.782}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.304, \"y\": 14.859, \"z\": 19.353}, \"end\": {\"x\": 19.984, \"y\": 13.71, \"z\": 17.928}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.313, \"y\": 13.726, \"z\": 24.017}, \"end\": {\"x\": 16.141, \"y\": 14.266, \"z\": 22.627}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.465, \"y\": 11.804, \"z\": 22.944}, \"end\": {\"x\": 19.643, \"y\": 13.757, \"z\": 22.222}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.292, \"y\": 16.284, \"z\": 23.573}, \"end\": {\"x\": 17.109, \"y\": 17.717, \"z\": 23.734}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.313, \"y\": 13.726, \"z\": 24.017}, \"end\": {\"x\": 16.141, \"y\": 14.266, \"z\": 22.627}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.465, \"y\": 11.804, \"z\": 22.944}, \"end\": {\"x\": 19.643, \"y\": 13.757, \"z\": 22.222}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.152, \"y\": 11.65, \"z\": 20.058}, \"end\": {\"x\": 18.265, \"y\": 11.125, \"z\": 20.503}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.676, \"y\": 7.901, \"z\": 21.106}, \"end\": {\"x\": 20.979, \"y\": 9.165, \"z\": 21.489}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.577, \"y\": 7.844, \"z\": 20.191}, \"end\": {\"x\": 20.979, \"y\": 9.165, \"z\": 21.489}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.304, \"y\": 14.859, \"z\": 19.353}, \"end\": {\"x\": 19.984, \"y\": 13.71, \"z\": 17.928}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.152, \"y\": 11.65, \"z\": 20.058}, \"end\": {\"x\": 18.265, \"y\": 11.125, \"z\": 20.503}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.914, \"y\": 7.989, \"z\": 12.142}, \"end\": {\"x\": 18.709, \"y\": 8.444, \"z\": 11.009}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.248, \"y\": 16.107, \"z\": 15.597}, \"end\": {\"x\": 20.428, \"y\": 17.289, \"z\": 16.325}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.733, \"y\": 16.086, \"z\": 10.744}, \"end\": {\"x\": 16.35, \"y\": 16.649, \"z\": 12.754}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.406, \"y\": 19.29, \"z\": 14.599}, \"end\": {\"x\": 20.528, \"y\": 18.873, \"z\": 12.955}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.22, \"y\": 17.276, \"z\": 13.809}, \"end\": {\"x\": 20.428, \"y\": 17.289, \"z\": 16.325}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.02, \"y\": 20.314, \"z\": 10.519}, \"end\": {\"x\": 17.296, \"y\": 19.943, \"z\": 9.95}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.906, \"y\": 22.366, \"z\": 9.283}, \"end\": {\"x\": 19.599, \"y\": 22.451, \"z\": 9.882}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.532, \"y\": 20.966, \"z\": 4.397}, \"end\": {\"x\": 24.33, \"y\": 21.453, \"z\": 3.889}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.769, \"y\": 23.904, \"z\": 11.282}, \"end\": {\"x\": 23.717, \"y\": 24.466, \"z\": 9.81}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.258, \"y\": 23.006, \"z\": 11.449}, \"end\": {\"x\": 23.16, \"y\": 22.291, \"z\": 12.88}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.851, \"y\": 29.527, \"z\": 10.282}, \"end\": {\"x\": 21.761, \"y\": 29.896, \"z\": 8.791}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.474, \"y\": 25.587, \"z\": 7.885}, \"end\": {\"x\": 19.715, \"y\": 26.133, \"z\": 7.838}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.349, \"y\": 28.421, \"z\": 6.822}, \"end\": {\"x\": 22.146, \"y\": 27.422, \"z\": 6.21}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.851, \"y\": 29.527, \"z\": 10.282}, \"end\": {\"x\": 21.761, \"y\": 29.896, \"z\": 8.791}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.555, \"y\": 30.079, \"z\": 7.169}, \"end\": {\"x\": 17.778, \"y\": 30.183, \"z\": 6.408}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.474, \"y\": 25.587, \"z\": 7.885}, \"end\": {\"x\": 19.715, \"y\": 26.133, \"z\": 7.838}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.349, \"y\": 28.421, \"z\": 6.822}, \"end\": {\"x\": 22.146, \"y\": 27.422, \"z\": 6.21}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.636, \"y\": 23.408, \"z\": 7.315}, \"end\": {\"x\": 15.762, \"y\": 23.796, \"z\": 7.537}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.02, \"y\": 20.314, \"z\": 10.519}, \"end\": {\"x\": 17.296, \"y\": 19.943, \"z\": 9.95}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.906, \"y\": 22.366, \"z\": 9.283}, \"end\": {\"x\": 19.599, \"y\": 22.451, \"z\": 9.882}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.038, \"y\": 17.129, \"z\": 6.773}, \"end\": {\"x\": 13.846, \"y\": 17.084, \"z\": 8.45}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.716, \"y\": 19.498, \"z\": 8.223}, \"end\": {\"x\": 13.546, \"y\": 20.92, \"z\": 7.239}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.733, \"y\": 16.086, \"z\": 10.744}, \"end\": {\"x\": 16.35, \"y\": 16.649, \"z\": 12.754}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.642, \"y\": 13.366, \"z\": 8.44}, \"end\": {\"x\": 13.311, \"y\": 12.252, \"z\": 7.088}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.96, \"y\": 14.236, \"z\": 18.793}, \"end\": {\"x\": 6.053, \"y\": 12.513, \"z\": 18.959}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.338, \"y\": 14.043, \"z\": 19.122}, \"end\": {\"x\": 6.053, \"y\": 12.513, \"z\": 18.959}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.05, \"y\": 18.204, \"z\": 17.759}, \"end\": {\"x\": 4.593, \"y\": 16.926, \"z\": 17.726}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 4.338, \"y\": 14.043, \"z\": 19.122}, \"end\": {\"x\": 6.053, \"y\": 12.513, \"z\": 18.959}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.334, \"y\": 18.608, \"z\": 19.061}, \"end\": {\"x\": 7.244, \"y\": 17.13, \"z\": 20.009}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.959, \"y\": 11.943, \"z\": 21.698}, \"end\": {\"x\": 9.866, \"y\": 13.537, \"z\": 19.976}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.008, \"y\": 11.869, \"z\": 20.545}, \"end\": {\"x\": 9.866, \"y\": 13.537, \"z\": 19.976}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.96, \"y\": 14.236, \"z\": 18.793}, \"end\": {\"x\": 6.053, \"y\": 12.513, \"z\": 18.959}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.228, \"y\": 19.482, \"z\": 17.41}, \"end\": {\"x\": 9.56, \"y\": 17.36, \"z\": 17.183}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 6.05, \"y\": 18.204, \"z\": 17.759}, \"end\": {\"x\": 4.593, \"y\": 16.926, \"z\": 17.726}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.51, \"y\": 22.037, \"z\": 16.968}, \"end\": {\"x\": 8.248, \"y\": 21.791, \"z\": 18.62}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 8.334, \"y\": 18.608, \"z\": 19.061}, \"end\": {\"x\": 7.244, \"y\": 17.13, \"z\": 20.009}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.284, \"y\": 23.52, \"z\": 18.37}, \"end\": {\"x\": 11.347, \"y\": 21.684, \"z\": 19.334}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 10.228, \"y\": 19.482, \"z\": 17.41}, \"end\": {\"x\": 9.56, \"y\": 17.36, \"z\": 17.183}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.272, \"y\": 24.192, \"z\": 17.259}, \"end\": {\"x\": 12.339, \"y\": 22.929, \"z\": 15.755}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 9.51, \"y\": 22.037, \"z\": 16.968}, \"end\": {\"x\": 8.248, \"y\": 21.791, \"z\": 18.62}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 11.284, \"y\": 23.52, \"z\": 18.37}, \"end\": {\"x\": 11.347, \"y\": 21.684, \"z\": 19.334}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.272, \"y\": 24.192, \"z\": 17.259}, \"end\": {\"x\": 12.339, \"y\": 22.929, \"z\": 15.755}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.732, \"y\": 26.85, \"z\": 13.809}, \"end\": {\"x\": 14.622, \"y\": 25.943, \"z\": 15.191}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.882, \"y\": 24.79, \"z\": 14.29}, \"end\": {\"x\": 14.622, \"y\": 25.943, \"z\": 15.191}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.886, \"y\": 30.607, \"z\": 12.787}, \"end\": {\"x\": 15.511, \"y\": 28.672, \"z\": 13.549}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.886, \"y\": 30.607, \"z\": 12.787}, \"end\": {\"x\": 15.511, \"y\": 28.672, \"z\": 13.549}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.555, \"y\": 30.079, \"z\": 7.169}, \"end\": {\"x\": 17.778, \"y\": 30.183, \"z\": 6.408}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.732, \"y\": 26.85, \"z\": 13.809}, \"end\": {\"x\": 14.622, \"y\": 25.943, \"z\": 15.191}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.882, \"y\": 24.79, \"z\": 14.29}, \"end\": {\"x\": 14.622, \"y\": 25.943, \"z\": 15.191}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.636, \"y\": 23.408, \"z\": 7.315}, \"end\": {\"x\": 15.762, \"y\": 23.796, \"z\": 7.537}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.716, \"y\": 19.498, \"z\": 8.223}, \"end\": {\"x\": 13.546, \"y\": 20.92, \"z\": 7.239}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 13.038, \"y\": 17.129, \"z\": 6.773}, \"end\": {\"x\": 13.846, \"y\": 17.084, \"z\": 8.45}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 12.642, \"y\": 13.366, \"z\": 8.44}, \"end\": {\"x\": 13.311, \"y\": 12.252, \"z\": 7.088}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.914, \"y\": 7.989, \"z\": 12.142}, \"end\": {\"x\": 18.709, \"y\": 8.444, \"z\": 11.009}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.091, \"y\": 4.456, \"z\": 5.237}, \"end\": {\"x\": 19.965, \"y\": 3.686, \"z\": 6.949}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.838, \"y\": 3.706, \"z\": 3.125}, \"end\": {\"x\": 17.872, \"y\": 2.579, \"z\": 4.769}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.69, \"y\": 4.652, \"z\": 2.527}, \"end\": {\"x\": 15.154, \"y\": 5.049, \"z\": 0.403}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.523, \"y\": 5.967, \"z\": 1.712}, \"end\": {\"x\": 17.606, \"y\": 5.881, \"z\": 3.432}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.162, \"y\": 6.513, \"z\": 1.256}, \"end\": {\"x\": 20.973, \"y\": 6.643, \"z\": 3.153}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.091, \"y\": 4.456, \"z\": 5.237}, \"end\": {\"x\": 19.965, \"y\": 3.686, \"z\": 6.949}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.574, \"y\": 3.639, \"z\": 0.427}, \"end\": {\"x\": 21.484, \"y\": 3.783, \"z\": 1.287}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.189, \"y\": 1.269, \"z\": 6.277}, \"end\": {\"x\": 23.539, \"y\": 1.016, \"z\": 7.048}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.909, \"y\": 1.302, \"z\": 8.984}, \"end\": {\"x\": 23.539, \"y\": 1.016, \"z\": 7.048}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.838, \"y\": 3.706, \"z\": 3.125}, \"end\": {\"x\": 17.872, \"y\": 2.579, \"z\": 4.769}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 14.69, \"y\": 4.652, \"z\": 2.527}, \"end\": {\"x\": 15.154, \"y\": 5.049, \"z\": 0.403}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.446, \"y\": 5.146, \"z\": -0.786}, \"end\": {\"x\": 19.286, \"y\": 4.661, \"z\": -1.084}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.523, \"y\": 5.967, \"z\": 1.712}, \"end\": {\"x\": 17.606, \"y\": 5.881, \"z\": 3.432}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.162, \"y\": 6.513, \"z\": 1.256}, \"end\": {\"x\": 20.973, \"y\": 6.643, \"z\": 3.153}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.446, \"y\": 5.146, \"z\": -0.786}, \"end\": {\"x\": 19.286, \"y\": 4.661, \"z\": -1.084}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.574, \"y\": 3.639, \"z\": 0.427}, \"end\": {\"x\": 21.484, \"y\": 3.783, \"z\": 1.287}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.888, \"y\": 0.518, \"z\": 4.072}, \"end\": {\"x\": 27.407, \"y\": 1.546, \"z\": 4.023}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.189, \"y\": 1.269, \"z\": 6.277}, \"end\": {\"x\": 23.539, \"y\": 1.016, \"z\": 7.048}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.909, \"y\": 1.302, \"z\": 8.984}, \"end\": {\"x\": 23.539, \"y\": 1.016, \"z\": 7.048}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.901, \"y\": 0.343, \"z\": 12.937}, \"end\": {\"x\": 22.321, \"y\": -0.93, \"z\": 13.376}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.315, \"y\": -0.599, \"z\": 11.516}, \"end\": {\"x\": 22.321, \"y\": -0.93, \"z\": 13.376}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.363, \"y\": 2.396, \"z\": 4.687}, \"end\": {\"x\": 29.675, \"y\": 1.598, \"z\": 6.511}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.04, \"y\": 1.535, \"z\": 12.35}, \"end\": {\"x\": 26.6, \"y\": 2.319, \"z\": 13.158}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.888, \"y\": 0.518, \"z\": 4.072}, \"end\": {\"x\": 27.407, \"y\": 1.546, \"z\": 4.023}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.363, \"y\": 2.396, \"z\": 4.687}, \"end\": {\"x\": 29.675, \"y\": 1.598, \"z\": 6.511}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.878, \"y\": 8.205, \"z\": 2.551}, \"end\": {\"x\": 37.097, \"y\": 6.228, \"z\": 1.86}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.019, \"y\": 5.408, \"z\": 7.359}, \"end\": {\"x\": 35.925, \"y\": 3.85, \"z\": 7.986}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.621, \"y\": 5.328, \"z\": 7.261}, \"end\": {\"x\": 37.004, \"y\": 7.021, \"z\": 8.285}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.414, \"y\": 12.941, \"z\": 7.8}, \"end\": {\"x\": 36.353, \"y\": 11.585, \"z\": 9.01}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.413, \"y\": 17.006, \"z\": 10.162}, \"end\": {\"x\": 40.518, \"y\": 18.929, \"z\": 10.492}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.088, \"y\": 15.459, \"z\": 7.137}, \"end\": {\"x\": 37.59, \"y\": 14.737, \"z\": 7.848}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.929, \"y\": 15.501, \"z\": 4.672}, \"end\": {\"x\": 36.434, \"y\": 14.139, \"z\": 4.761}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.428, \"y\": 15.335, \"z\": 5.219}, \"end\": {\"x\": 33.145, \"y\": 13.497, \"z\": 5.615}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.414, \"y\": 12.941, \"z\": 7.8}, \"end\": {\"x\": 36.353, \"y\": 11.585, \"z\": 9.01}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.639, \"y\": 17.46, \"z\": 6.323}, \"end\": {\"x\": 33.091, \"y\": 16.465, \"z\": 7.33}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.103, \"y\": 20.472, \"z\": 5.476}, \"end\": {\"x\": 33.87, \"y\": 18.443, \"z\": 4.902}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.214, \"y\": 21.583, \"z\": 6.93}, \"end\": {\"x\": 38.089, \"y\": 19.966, \"z\": 6.516}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.088, \"y\": 15.459, \"z\": 7.137}, \"end\": {\"x\": 37.59, \"y\": 14.737, \"z\": 7.848}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.879, \"y\": 19.459, \"z\": 4.385}, \"end\": {\"x\": 33.87, \"y\": 18.443, \"z\": 4.902}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.929, \"y\": 15.501, \"z\": 4.672}, \"end\": {\"x\": 36.434, \"y\": 14.139, \"z\": 4.761}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.897, \"y\": 18.313, \"z\": 2.739}, \"end\": {\"x\": 31.369, \"y\": 17.02, \"z\": 2.793}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.962, \"y\": 12.232, \"z\": 2.225}, \"end\": {\"x\": 36.994, \"y\": 13.412, \"z\": 1.034}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.169, \"y\": 10.988, \"z\": 1.591}, \"end\": {\"x\": 35.028, \"y\": 12.492, \"z\": 1.601}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.428, \"y\": 15.335, \"z\": 5.219}, \"end\": {\"x\": 33.145, \"y\": 13.497, \"z\": 5.615}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.5, \"y\": 18.596, \"z\": 4.228}, \"end\": {\"x\": 28.723, \"y\": 17.484, \"z\": 5.149}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.639, \"y\": 17.46, \"z\": 6.323}, \"end\": {\"x\": 33.091, \"y\": 16.465, \"z\": 7.33}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.481, \"y\": 21.316, \"z\": 4.288}, \"end\": {\"x\": 29.166, \"y\": 20.856, \"z\": 5.249}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.648, \"y\": 21.849, \"z\": 4.529}, \"end\": {\"x\": 33.091, \"y\": 21.217, \"z\": 2.538}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.879, \"y\": 19.459, \"z\": 4.385}, \"end\": {\"x\": 33.87, \"y\": 18.443, \"z\": 4.902}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.825, \"y\": 21.875, \"z\": 1.587}, \"end\": {\"x\": 28.708, \"y\": 21.164, \"z\": 1.913}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.897, \"y\": 18.313, \"z\": 2.739}, \"end\": {\"x\": 31.369, \"y\": 17.02, \"z\": 2.793}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.32, \"y\": 21.37, \"z\": 1.386}, \"end\": {\"x\": 25.776, \"y\": 19.482, \"z\": 1.693}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.532, \"y\": 20.966, \"z\": 4.397}, \"end\": {\"x\": 24.33, \"y\": 21.453, \"z\": 3.889}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.5, \"y\": 18.596, \"z\": 4.228}, \"end\": {\"x\": 28.723, \"y\": 17.484, \"z\": 5.149}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.481, \"y\": 21.316, \"z\": 4.288}, \"end\": {\"x\": 29.166, \"y\": 20.856, \"z\": 5.249}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.77, \"y\": 23.646, \"z\": 1.62}, \"end\": {\"x\": 24.968, \"y\": 24.326, \"z\": 2.365}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.825, \"y\": 21.875, \"z\": 1.587}, \"end\": {\"x\": 28.708, \"y\": 21.164, \"z\": 1.913}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.923, \"y\": 23.377, \"z\": -3.565}, \"end\": {\"x\": 24.041, \"y\": 22.742, \"z\": -1.396}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.32, \"y\": 21.37, \"z\": 1.386}, \"end\": {\"x\": 25.776, \"y\": 19.482, \"z\": 1.693}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.61, \"y\": 23.881, \"z\": -2.316}, \"end\": {\"x\": 20.713, \"y\": 22.777, \"z\": -0.595}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.77, \"y\": 23.646, \"z\": 1.62}, \"end\": {\"x\": 24.968, \"y\": 24.326, \"z\": 2.365}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.923, \"y\": 23.377, \"z\": -3.565}, \"end\": {\"x\": 24.75, \"y\": 25.361, \"z\": -2.758}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.61, \"y\": 23.881, \"z\": -2.316}, \"end\": {\"x\": 20.713, \"y\": 22.777, \"z\": -0.595}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.923, \"y\": 23.377, \"z\": -3.565}, \"end\": {\"x\": 24.041, \"y\": 22.742, \"z\": -1.396}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.923, \"y\": 23.377, \"z\": -3.565}, \"end\": {\"x\": 24.75, \"y\": 25.361, \"z\": -2.758}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.433, \"y\": 14.03, \"z\": -0.348}, \"end\": {\"x\": 22.854, \"y\": 14.846, \"z\": -2.253}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.332, \"y\": 12.172, \"z\": 1.276}, \"end\": {\"x\": 21.456, \"y\": 12.364, \"z\": -0.339}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.39, \"y\": 10.535, \"z\": 1.803}, \"end\": {\"x\": 24.062, \"y\": 10.398, \"z\": -0.187}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.709, \"y\": 12.174, \"z\": 3.33}, \"end\": {\"x\": 25.867, \"y\": 13.087, \"z\": 1.416}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.433, \"y\": 14.03, \"z\": -0.348}, \"end\": {\"x\": 22.854, \"y\": 14.846, \"z\": -2.253}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.484, \"y\": 12.556, \"z\": 5.51}, \"end\": {\"x\": 23.687, \"y\": 13.335, \"z\": 3.854}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.332, \"y\": 12.172, \"z\": 1.276}, \"end\": {\"x\": 21.456, \"y\": 12.364, \"z\": -0.339}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.915, \"y\": 10.158, \"z\": 6.527}, \"end\": {\"x\": 23.654, \"y\": 10.062, \"z\": 4.826}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.39, \"y\": 10.535, \"z\": 1.803}, \"end\": {\"x\": 24.062, \"y\": 10.398, \"z\": -0.187}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.441, \"y\": 10.159, \"z\": 7.213}, \"end\": {\"x\": 27.087, \"y\": 10.046, \"z\": 5.181}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.709, \"y\": 12.174, \"z\": 3.33}, \"end\": {\"x\": 25.867, \"y\": 13.087, \"z\": 1.416}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.725, \"y\": 12.164, \"z\": 8.996}, \"end\": {\"x\": 27.019, \"y\": 12.903, \"z\": 7.317}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.484, \"y\": 12.556, \"z\": 5.51}, \"end\": {\"x\": 23.687, \"y\": 13.335, \"z\": 3.854}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.109, \"y\": 11.141, \"z\": 11.066}, \"end\": {\"x\": 25.036, \"y\": 11.119, \"z\": 9.543}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.915, \"y\": 10.158, \"z\": 6.527}, \"end\": {\"x\": 23.654, \"y\": 10.062, \"z\": 4.826}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.52, \"y\": 8.984, \"z\": 11.945}, \"end\": {\"x\": 27.14, \"y\": 8.43, \"z\": 9.823}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.096, \"y\": 6.378, \"z\": 9.65}, \"end\": {\"x\": 27.14, \"y\": 8.43, \"z\": 9.823}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.441, \"y\": 10.159, \"z\": 7.213}, \"end\": {\"x\": 27.087, \"y\": 10.046, \"z\": 5.181}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.727, \"y\": 10.124, \"z\": 12.868}, \"end\": {\"x\": 29.604, \"y\": 10.466, \"z\": 10.868}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.725, \"y\": 12.164, \"z\": 8.996}, \"end\": {\"x\": 27.019, \"y\": 12.903, \"z\": 7.317}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.053, \"y\": 11.197, \"z\": 14.964}, \"end\": {\"x\": 27.989, \"y\": 12.241, \"z\": 13.372}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.109, \"y\": 11.141, \"z\": 11.066}, \"end\": {\"x\": 25.036, \"y\": 11.119, \"z\": 9.543}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.311, \"y\": 8.699, \"z\": 16.702}, \"end\": {\"x\": 26.812, \"y\": 9.545, \"z\": 14.976}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.998, \"y\": 11.41, \"z\": 16.278}, \"end\": {\"x\": 26.812, \"y\": 9.545, \"z\": 14.976}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.52, \"y\": 8.984, \"z\": 11.945}, \"end\": {\"x\": 27.14, \"y\": 8.43, \"z\": 9.823}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.096, \"y\": 6.378, \"z\": 9.65}, \"end\": {\"x\": 27.14, \"y\": 8.43, \"z\": 9.823}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.727, \"y\": 10.124, \"z\": 12.868}, \"end\": {\"x\": 29.604, \"y\": 10.466, \"z\": 10.868}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.053, \"y\": 11.197, \"z\": 14.964}, \"end\": {\"x\": 27.989, \"y\": 12.241, \"z\": 13.372}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.387, \"y\": 10.151, \"z\": 20.122}, \"end\": {\"x\": 28.848, \"y\": 11.483, \"z\": 18.823}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.998, \"y\": 11.41, \"z\": 16.278}, \"end\": {\"x\": 26.812, \"y\": 9.545, \"z\": 14.976}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.823, \"y\": 8.176, \"z\": 21.348}, \"end\": {\"x\": 26.721, \"y\": 9.067, \"z\": 19.876}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.311, \"y\": 8.699, \"z\": 16.702}, \"end\": {\"x\": 26.812, \"y\": 9.545, \"z\": 14.976}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.378, \"y\": 5.089, \"z\": 19.022}, \"end\": {\"x\": 25.19, \"y\": 5.046, \"z\": 17.987}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.646, \"y\": 6.297, \"z\": 21.99}, \"end\": {\"x\": 29.043, \"y\": 6.408, \"z\": 19.932}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.984, \"y\": 8.251, \"z\": 23.58}, \"end\": {\"x\": 31.365, \"y\": 8.299, \"z\": 21.71}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.585, \"y\": 9.377, \"z\": 24.985}, \"end\": {\"x\": 29.035, \"y\": 8.741, \"z\": 24.226}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.387, \"y\": 10.151, \"z\": 20.122}, \"end\": {\"x\": 28.848, \"y\": 11.483, \"z\": 18.823}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.676, \"y\": 7.901, \"z\": 21.106}, \"end\": {\"x\": 20.979, \"y\": 9.165, \"z\": 21.489}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.577, \"y\": 7.844, \"z\": 20.191}, \"end\": {\"x\": 20.979, \"y\": 9.165, \"z\": 21.489}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.823, \"y\": 8.176, \"z\": 21.348}, \"end\": {\"x\": 26.721, \"y\": 9.067, \"z\": 19.876}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.378, \"y\": 5.089, \"z\": 19.022}, \"end\": {\"x\": 25.19, \"y\": 5.046, \"z\": 17.987}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.872, \"y\": 4.599, \"z\": 25.883}, \"end\": {\"x\": 27.99, \"y\": 5.195, \"z\": 24.411}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.0, \"y\": 5.451, \"z\": 26.696}, \"end\": {\"x\": 27.99, \"y\": 5.195, \"z\": 24.411}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.646, \"y\": 6.297, \"z\": 21.99}, \"end\": {\"x\": 29.043, \"y\": 6.408, \"z\": 19.932}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.581, \"y\": 5.957, \"z\": 20.627}, \"end\": {\"x\": 32.216, \"y\": 5.401, \"z\": 22.014}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.144, \"y\": 5.125, \"z\": 20.774}, \"end\": {\"x\": 35.545, \"y\": 4.537, \"z\": 22.462}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.515, \"y\": 3.995, \"z\": 18.216}, \"end\": {\"x\": 35.455, \"y\": 2.667, \"z\": 19.609}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.117, \"y\": -3.258, \"z\": 17.294}, \"end\": {\"x\": 33.871, \"y\": -3.946, \"z\": 16.14}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.153, \"y\": 2.588, \"z\": 23.301}, \"end\": {\"x\": 31.381, \"y\": 2.001, \"z\": 24.611}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.631, \"y\": 6.268, \"z\": 16.813}, \"end\": {\"x\": 34.733, \"y\": 5.692, \"z\": 17.713}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 34.568, \"y\": 9.273, \"z\": 25.687}, \"end\": {\"x\": 35.521, \"y\": 8.526, \"z\": 23.943}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.693, \"y\": 8.18, \"z\": 23.949}, \"end\": {\"x\": 35.521, \"y\": 8.526, \"z\": 23.943}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.581, \"y\": 5.957, \"z\": 20.627}, \"end\": {\"x\": 32.216, \"y\": 5.401, \"z\": 22.014}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.705, \"y\": 7.421, \"z\": 17.584}, \"end\": {\"x\": 37.296, \"y\": 7.386, \"z\": 18.882}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.144, \"y\": 5.125, \"z\": 20.774}, \"end\": {\"x\": 35.545, \"y\": 4.537, \"z\": 22.462}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.586, \"y\": 5.892, \"z\": 16.531}, \"end\": {\"x\": 39.562, \"y\": 4.877, \"z\": 18.074}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.515, \"y\": 3.995, \"z\": 18.216}, \"end\": {\"x\": 35.455, \"y\": 2.667, \"z\": 19.609}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.537, \"y\": 6.172, \"z\": 14.029}, \"end\": {\"x\": 38.667, \"y\": 5.185, \"z\": 14.645}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.032, \"y\": 3.99, \"z\": 13.063}, \"end\": {\"x\": 38.667, \"y\": 5.185, \"z\": 14.645}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.406, \"y\": 2.36, \"z\": 13.167}, \"end\": {\"x\": 36.971, \"y\": 0.858, \"z\": 14.362}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.631, \"y\": 6.268, \"z\": 16.813}, \"end\": {\"x\": 34.733, \"y\": 5.692, \"z\": 17.713}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.454, \"y\": 8.807, \"z\": 13.889}, \"end\": {\"x\": 39.534, \"y\": 8.662, \"z\": 14.789}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.705, \"y\": 7.421, \"z\": 17.584}, \"end\": {\"x\": 37.296, \"y\": 7.386, \"z\": 18.882}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.814, \"y\": 8.492, \"z\": 14.578}, \"end\": {\"x\": 42.655, \"y\": 8.199, \"z\": 16.098}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.586, \"y\": 5.892, \"z\": 16.531}, \"end\": {\"x\": 39.562, \"y\": 4.877, \"z\": 18.074}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.284, \"y\": 7.188, \"z\": 12.773}, \"end\": {\"x\": 43.598, \"y\": 6.119, \"z\": 13.52}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 40.537, \"y\": 6.172, \"z\": 14.029}, \"end\": {\"x\": 38.667, \"y\": 5.185, \"z\": 14.645}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.032, \"y\": 3.99, \"z\": 13.063}, \"end\": {\"x\": 38.667, \"y\": 5.185, \"z\": 14.645}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.406, \"y\": 2.36, \"z\": 13.167}, \"end\": {\"x\": 36.971, \"y\": 0.858, \"z\": 14.362}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.468, \"y\": 7.907, \"z\": 9.403}, \"end\": {\"x\": 42.862, \"y\": 8.376, \"z\": 11.086}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.956, \"y\": 4.479, \"z\": 9.761}, \"end\": {\"x\": 35.925, \"y\": 3.85, \"z\": 7.986}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 41.454, \"y\": 8.807, \"z\": 13.889}, \"end\": {\"x\": 39.534, \"y\": 8.662, \"z\": 14.789}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.814, \"y\": 8.492, \"z\": 14.578}, \"end\": {\"x\": 42.655, \"y\": 8.199, \"z\": 16.098}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 45.284, \"y\": 7.188, \"z\": 12.773}, \"end\": {\"x\": 43.598, \"y\": 6.119, \"z\": 13.52}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 43.468, \"y\": 7.907, \"z\": 9.403}, \"end\": {\"x\": 42.862, \"y\": 8.376, \"z\": 11.086}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.878, \"y\": 8.205, \"z\": 2.551}, \"end\": {\"x\": 37.097, \"y\": 6.228, \"z\": 1.86}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.962, \"y\": 12.232, \"z\": 2.225}, \"end\": {\"x\": 36.994, \"y\": 13.412, \"z\": 1.034}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.169, \"y\": 10.988, \"z\": 1.591}, \"end\": {\"x\": 35.028, \"y\": 12.492, \"z\": 1.601}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.019, \"y\": 5.408, \"z\": 7.359}, \"end\": {\"x\": 35.925, \"y\": 3.85, \"z\": 7.986}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.621, \"y\": 5.328, \"z\": 7.261}, \"end\": {\"x\": 37.004, \"y\": 7.021, \"z\": 8.285}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.956, \"y\": 4.479, \"z\": 9.761}, \"end\": {\"x\": 35.925, \"y\": 3.85, \"z\": 7.986}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.624, \"y\": -5.14, \"z\": 13.994}, \"end\": {\"x\": 34.203, \"y\": -3.779, \"z\": 13.654}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.01, \"y\": -2.618, \"z\": 14.463}, \"end\": {\"x\": 31.596, \"y\": -2.749, \"z\": 12.244}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.433, \"y\": -3.023, \"z\": 14.797}, \"end\": {\"x\": 36.395, \"y\": -3.476, \"z\": 13.179}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.006, \"y\": -3.944, \"z\": 14.468}, \"end\": {\"x\": 29.489, \"y\": -5.129, \"z\": 14.103}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.117, \"y\": -3.258, \"z\": 17.294}, \"end\": {\"x\": 33.871, \"y\": -3.946, \"z\": 16.14}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.624, \"y\": -5.14, \"z\": 13.994}, \"end\": {\"x\": 34.203, \"y\": -3.779, \"z\": 13.654}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.076, \"y\": -2.878, \"z\": 17.19}, \"end\": {\"x\": 30.097, \"y\": -3.245, \"z\": 16.978}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.01, \"y\": -2.618, \"z\": 14.463}, \"end\": {\"x\": 31.596, \"y\": -2.749, \"z\": 12.244}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.9, \"y\": -0.226, \"z\": 16.697}, \"end\": {\"x\": 28.486, \"y\": -0.452, \"z\": 15.668}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.327, \"y\": -1.266, \"z\": 15.937}, \"end\": {\"x\": 25.648, \"y\": -1.828, \"z\": 14.604}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.006, \"y\": -3.944, \"z\": 14.468}, \"end\": {\"x\": 29.489, \"y\": -5.129, \"z\": 14.103}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.573, \"y\": -1.761, \"z\": 18.43}, \"end\": {\"x\": 24.971, \"y\": -3.367, \"z\": 17.622}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.076, \"y\": -2.878, \"z\": 17.19}, \"end\": {\"x\": 30.097, \"y\": -3.245, \"z\": 16.978}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.573, \"y\": -1.761, \"z\": 18.43}, \"end\": {\"x\": 25.059, \"y\": -0.588, \"z\": 19.228}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 25.04, \"y\": 1.535, \"z\": 12.35}, \"end\": {\"x\": 26.6, \"y\": 2.319, \"z\": 13.158}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.9, \"y\": -0.226, \"z\": 16.697}, \"end\": {\"x\": 28.486, \"y\": -0.452, \"z\": 15.668}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.438, \"y\": 0.065, \"z\": 19.704}, \"end\": {\"x\": 22.765, \"y\": 1.025, \"z\": 17.468}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.901, \"y\": 0.343, \"z\": 12.937}, \"end\": {\"x\": 22.321, \"y\": -0.93, \"z\": 13.376}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.315, \"y\": -0.599, \"z\": 11.516}, \"end\": {\"x\": 22.321, \"y\": -0.93, \"z\": 13.376}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.327, \"y\": -1.266, \"z\": 15.937}, \"end\": {\"x\": 25.648, \"y\": -1.828, \"z\": 14.604}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.573, \"y\": -1.761, \"z\": 18.43}, \"end\": {\"x\": 24.971, \"y\": -3.367, \"z\": 17.622}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.573, \"y\": -1.761, \"z\": 18.43}, \"end\": {\"x\": 25.059, \"y\": -0.588, \"z\": 19.228}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.173, \"y\": -2.137, \"z\": 23.701}, \"end\": {\"x\": 21.704, \"y\": -1.933, \"z\": 22.061}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.438, \"y\": 0.065, \"z\": 19.704}, \"end\": {\"x\": 22.765, \"y\": 1.025, \"z\": 17.468}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.962, \"y\": -0.189, \"z\": 25.583}, \"end\": {\"x\": 21.282, \"y\": 0.722, \"z\": 23.889}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.433, \"y\": 4.5, \"z\": 24.762}, \"end\": {\"x\": 24.716, \"y\": 5.013, \"z\": 26.538}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.426, \"y\": -0.924, \"z\": 26.807}, \"end\": {\"x\": 18.174, \"y\": -0.32, \"z\": 24.765}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.223, \"y\": 0.705, \"z\": 24.634}, \"end\": {\"x\": 18.174, \"y\": -0.32, \"z\": 24.765}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.055, \"y\": -8.147, \"z\": 21.295}, \"end\": {\"x\": 19.062, \"y\": -9.623, \"z\": 21.045}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.23, \"y\": -3.459, \"z\": 27.707}, \"end\": {\"x\": 19.341, \"y\": -3.595, \"z\": 25.863}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.173, \"y\": -2.137, \"z\": 23.701}, \"end\": {\"x\": 21.704, \"y\": -1.933, \"z\": 22.061}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.023, \"y\": -2.699, \"z\": 29.688}, \"end\": {\"x\": 21.697, \"y\": -2.048, \"z\": 28.029}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.962, \"y\": -0.189, \"z\": 25.583}, \"end\": {\"x\": 21.282, \"y\": 0.722, \"z\": 23.889}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.395, \"y\": 4.827, \"z\": 26.301}, \"end\": {\"x\": 20.943, \"y\": 2.426, \"z\": 26.26}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.813, \"y\": -1.092, \"z\": 31.54}, \"end\": {\"x\": 19.442, \"y\": -0.17, \"z\": 29.679}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 18.426, \"y\": -0.924, \"z\": 26.807}, \"end\": {\"x\": 18.174, \"y\": -0.32, \"z\": 24.765}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.96, \"y\": 0.894, \"z\": 26.802}, \"end\": {\"x\": 15.832, \"y\": -1.134, \"z\": 26.886}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.838, \"y\": -2.942, \"z\": 32.37}, \"end\": {\"x\": 17.719, \"y\": -2.954, \"z\": 30.557}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.23, \"y\": -3.459, \"z\": 27.707}, \"end\": {\"x\": 19.341, \"y\": -3.595, \"z\": 25.863}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.023, \"y\": -2.699, \"z\": 29.688}, \"end\": {\"x\": 21.697, \"y\": -2.048, \"z\": 28.029}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.253, \"y\": -4.171, \"z\": 34.115}, \"end\": {\"x\": 20.985, \"y\": -2.67, \"z\": 33.953}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.813, \"y\": -1.092, \"z\": 31.54}, \"end\": {\"x\": 19.442, \"y\": -0.17, \"z\": 29.679}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.319, \"y\": 4.325, \"z\": 31.439}, \"end\": {\"x\": 17.711, \"y\": 3.336, \"z\": 32.385}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 17.838, \"y\": -2.942, \"z\": 32.37}, \"end\": {\"x\": 17.719, \"y\": -2.954, \"z\": 30.557}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 19.253, \"y\": -4.171, \"z\": 34.115}, \"end\": {\"x\": 20.985, \"y\": -2.67, \"z\": 33.953}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.476, \"y\": -5.733, \"z\": 37.42}, \"end\": {\"x\": 21.654, \"y\": -4.976, \"z\": 35.459}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.091, \"y\": -8.294, \"z\": 36.064}, \"end\": {\"x\": 21.165, \"y\": -9.274, \"z\": 35.061}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.111, \"y\": -8.314, \"z\": 37.149}, \"end\": {\"x\": 21.958, \"y\": -8.621, \"z\": 38.755}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.476, \"y\": -5.733, \"z\": 37.42}, \"end\": {\"x\": 21.654, \"y\": -4.976, \"z\": 35.459}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.458, \"y\": -6.806, \"z\": 33.788}, \"end\": {\"x\": 24.914, \"y\": -7.454, \"z\": 34.702}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.091, \"y\": -8.294, \"z\": 36.064}, \"end\": {\"x\": 21.165, \"y\": -9.274, \"z\": 35.061}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.111, \"y\": -8.314, \"z\": 37.149}, \"end\": {\"x\": 21.958, \"y\": -8.621, \"z\": 38.755}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.36, \"y\": -10.285, \"z\": 33.074}, \"end\": {\"x\": 24.804, \"y\": -11.043, \"z\": 31.996}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.569, \"y\": -10.617, \"z\": 25.752}, \"end\": {\"x\": 25.012, \"y\": -10.907, \"z\": 27.445}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.215, \"y\": -11.008, \"z\": 24.873}, \"end\": {\"x\": 24.771, \"y\": -10.372, \"z\": 23.656}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 22.36, \"y\": -10.571, \"z\": 21.697}, \"end\": {\"x\": 20.329, \"y\": -11.416, \"z\": 21.47}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.737, \"y\": -10.52, \"z\": 20.806}, \"end\": {\"x\": 27.369, \"y\": -12.15, \"z\": 21.261}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.539, \"y\": -12.503, \"z\": 19.221}, \"end\": {\"x\": 27.859, \"y\": -10.743, \"z\": 18.493}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.146, \"y\": -10.582, \"z\": 12.681}, \"end\": {\"x\": 30.018, \"y\": -12.205, \"z\": 13.034}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.305, \"y\": -9.829, \"z\": 13.149}, \"end\": {\"x\": 32.997, \"y\": -8.599, \"z\": 14.62}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.776, \"y\": -10.723, \"z\": 16.552}, \"end\": {\"x\": 30.881, \"y\": -12.226, \"z\": 16.13}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 37.433, \"y\": -3.023, \"z\": 14.797}, \"end\": {\"x\": 36.395, \"y\": -3.476, \"z\": 13.179}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.646, \"y\": -8.132, \"z\": 16.296}, \"end\": {\"x\": 36.755, \"y\": -8.306, \"z\": 17.754}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.859, \"y\": -4.714, \"z\": 17.531}, \"end\": {\"x\": 40.405, \"y\": -6.532, \"z\": 18.442}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.126, \"y\": -12.49, \"z\": 13.593}, \"end\": {\"x\": 35.033, \"y\": -12.482, \"z\": 14.108}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 29.146, \"y\": -10.582, \"z\": 12.681}, \"end\": {\"x\": 30.018, \"y\": -12.205, \"z\": 13.034}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.305, \"y\": -9.829, \"z\": 13.149}, \"end\": {\"x\": 32.997, \"y\": -8.599, \"z\": 14.62}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.776, \"y\": -10.723, \"z\": 16.552}, \"end\": {\"x\": 30.881, \"y\": -12.226, \"z\": 16.13}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.126, \"y\": -12.49, \"z\": 13.593}, \"end\": {\"x\": 35.033, \"y\": -12.482, \"z\": 14.108}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.737, \"y\": -10.52, \"z\": 20.806}, \"end\": {\"x\": 27.369, \"y\": -12.15, \"z\": 21.261}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.539, \"y\": -12.503, \"z\": 19.221}, \"end\": {\"x\": 27.859, \"y\": -10.743, \"z\": 18.493}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.569, \"y\": -10.617, \"z\": 25.752}, \"end\": {\"x\": 25.012, \"y\": -10.907, \"z\": 27.445}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.215, \"y\": -11.008, \"z\": 24.873}, \"end\": {\"x\": 24.771, \"y\": -10.372, \"z\": 23.656}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.36, \"y\": -10.285, \"z\": 33.074}, \"end\": {\"x\": 24.804, \"y\": -11.043, \"z\": 31.996}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 26.458, \"y\": -6.806, \"z\": 33.788}, \"end\": {\"x\": 24.914, \"y\": -7.454, \"z\": 34.702}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.352, \"y\": -4.744, \"z\": 30.318}, \"end\": {\"x\": 29.377, \"y\": -4.227, \"z\": 31.847}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.401, \"y\": -2.721, \"z\": 31.038}, \"end\": {\"x\": 32.474, \"y\": -4.347, \"z\": 32.227}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.352, \"y\": -4.744, \"z\": 30.318}, \"end\": {\"x\": 29.377, \"y\": -4.227, \"z\": 31.847}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.495, \"y\": -0.44, \"z\": 27.842}, \"end\": {\"x\": 30.724, \"y\": -1.005, \"z\": 28.563}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.426, \"y\": 0.636, \"z\": 27.295}, \"end\": {\"x\": 30.724, \"y\": -1.005, \"z\": 28.563}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 31.401, \"y\": -2.721, \"z\": 31.038}, \"end\": {\"x\": 32.474, \"y\": -4.347, \"z\": 32.227}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 33.153, \"y\": 2.588, \"z\": 23.301}, \"end\": {\"x\": 31.381, \"y\": 2.001, \"z\": 24.611}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 32.495, \"y\": -0.44, \"z\": 27.842}, \"end\": {\"x\": 30.724, \"y\": -1.005, \"z\": 28.563}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 30.426, \"y\": 0.636, \"z\": 27.295}, \"end\": {\"x\": 30.724, \"y\": -1.005, \"z\": 28.563}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 28.872, \"y\": 4.599, \"z\": 25.883}, \"end\": {\"x\": 27.99, \"y\": 5.195, \"z\": 24.411}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 27.0, \"y\": 5.451, \"z\": 26.696}, \"end\": {\"x\": 27.99, \"y\": 5.195, \"z\": 24.411}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 24.433, \"y\": 4.5, \"z\": 24.762}, \"end\": {\"x\": 24.716, \"y\": 5.013, \"z\": 26.538}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 23.504, \"y\": 6.59, \"z\": 28.902}, \"end\": {\"x\": 24.635, \"y\": 6.904, \"z\": 30.764}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 21.1, \"y\": 8.452, \"z\": 29.548}, \"end\": {\"x\": 20.985, \"y\": 6.93, \"z\": 28.587}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 20.395, \"y\": 4.827, \"z\": 26.301}, \"end\": {\"x\": 20.943, \"y\": 2.426, \"z\": 26.26}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.223, \"y\": 0.705, \"z\": 24.634}, \"end\": {\"x\": 18.174, \"y\": -0.32, \"z\": 24.765}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 15.96, \"y\": 0.894, \"z\": 26.802}, \"end\": {\"x\": 15.832, \"y\": -1.134, \"z\": 26.886}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 16.319, \"y\": 4.325, \"z\": 31.439}, \"end\": {\"x\": 17.711, \"y\": 3.336, \"z\": 32.385}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.916, \"y\": 10.092, \"z\": 21.771}, \"end\": {\"x\": -16.106, \"y\": 8.237, \"z\": 22.298}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.442, \"y\": 11.601, \"z\": 20.613}, \"end\": {\"x\": -16.885, \"y\": 11.857, \"z\": 22.649}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -13.995, \"y\": 11.453, \"z\": 16.297}, \"end\": {\"x\": -15.032, \"y\": 11.567, \"z\": 17.846}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -14.916, \"y\": 10.092, \"z\": 21.771}, \"end\": {\"x\": -16.106, \"y\": 8.237, \"z\": 22.298}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -17.318, \"y\": 8.481, \"z\": 17.599}, \"end\": {\"x\": -18.765, \"y\": 9.805, \"z\": 18.36}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": -16.442, \"y\": 11.601, \"z\": 20.613}, \"end\": {\"x\": -16.885, \"y\": 11.857, \"z\": 22.649}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.798, \"y\": -6.707, \"z\": 21.607}, \"end\": {\"x\": 38.064, \"y\": -4.905, \"z\": 22.15}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.242, \"y\": -8.343, \"z\": 20.507}, \"end\": {\"x\": 38.701, \"y\": -8.563, \"z\": 22.522}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 35.646, \"y\": -8.132, \"z\": 16.296}, \"end\": {\"x\": 36.755, \"y\": -8.306, \"z\": 17.754}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 36.798, \"y\": -6.707, \"z\": 21.607}, \"end\": {\"x\": 38.064, \"y\": -4.905, \"z\": 22.15}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.859, \"y\": -4.714, \"z\": 17.531}, \"end\": {\"x\": 40.405, \"y\": -6.532, \"z\": 18.442}});\n",
              "\tviewer_15942021952474504.addLine({\"dashed\": true, \"color\": \"black\", \"start\": {\"x\": 38.242, \"y\": -8.343, \"z\": 20.507}, \"end\": {\"x\": 38.701, \"y\": -8.563, \"z\": 22.522}});\n",
              "viewer_15942021952474504.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {
            "tags": []
          }
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "UTPpGjjy2725",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "9d9f89c1-aa5e-4d59-bb62-2eef7e3ece76"
      },
      "source": [
        "# file name with the lowest energy\n",
        "print(lowest_energy_pdb_filename)"
      ],
      "execution_count": 19,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "./output/A0A088RHB8.0.apo_4.pdb\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "colab_type": "text",
        "id": "VC_sTp7V5gCq"
      },
      "source": [
        "If you think you achieved a good candidate, upload the pdb file to [Zindi](https://zindi.africa/competitions/indaba-grand-challenge-curing-leishmaniasis). you might be curing a Neglected Tropical Disease! To do this, click on the folder icon on the left of the colab notebook, under `output` directory download the above .pdb file and upload it to Zindi using the submit button on the competition page.\n",
        "\n",
        "*Exercise:*\n",
        "\n",
        "Re-run the above example with more sampling and with a different receptor protein and ligand (drug)."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "colab_type": "text",
        "id": "2IT2FL-_5gCr"
      },
      "source": [
        "***\n",
        "*Restart Jupyter Notebook kernel to properly re-initialize PyRosetta*\n",
        "***"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "7iYIiMpvoz9x",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        ""
      ],
      "execution_count": null,
      "outputs": []
    }
  ]
}